{
  "cells": [
    {
      "cell_type": "code",
      "execution_count": 52,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "xhnXwCwKOmBA",
        "outputId": "5f1e9f44-2ccc-467b-bd2f-f118c8d0cd4d"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: transformers in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (4.32.1)\n",
            "Collecting transformers\n",
            "  Obtaining dependency information for transformers from https://files.pythonhosted.org/packages/15/fc/7b6dd7e1adc0a6407b845ed4be1999e98b6917d0694e57316d140cc85484/transformers-4.39.3-py3-none-any.whl.metadata\n",
            "  Downloading transformers-4.39.3-py3-none-any.whl.metadata (134 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m134.8/134.8 kB\u001b[0m \u001b[31m3.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0ma \u001b[36m0:00:01\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: filelock in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from transformers) (3.9.0)\n",
            "Collecting huggingface-hub<1.0,>=0.19.3 (from transformers)\n",
            "  Obtaining dependency information for huggingface-hub<1.0,>=0.19.3 from https://files.pythonhosted.org/packages/05/c0/779afbad8e75565c09ffa24a88b5dd7e293c92b74eb09df6435fc58ac986/huggingface_hub-0.22.2-py3-none-any.whl.metadata\n",
            "  Downloading huggingface_hub-0.22.2-py3-none-any.whl.metadata (12 kB)\n",
            "Requirement already satisfied: numpy>=1.17 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from transformers) (1.24.3)\n",
            "Requirement already satisfied: packaging>=20.0 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from transformers) (23.2)\n",
            "Requirement already satisfied: pyyaml>=5.1 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from transformers) (6.0)\n",
            "Requirement already satisfied: regex!=2019.12.17 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from transformers) (2022.7.9)\n",
            "Requirement already satisfied: requests in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from transformers) (2.31.0)\n",
            "Collecting tokenizers<0.19,>=0.14 (from transformers)\n",
            "  Obtaining dependency information for tokenizers<0.19,>=0.14 from https://files.pythonhosted.org/packages/5f/4f/a4c12cc058a899c1caaa1e689c3df9a698e20e891d4005aa6ec2174a9339/tokenizers-0.15.2-cp311-cp311-macosx_11_0_arm64.whl.metadata\n",
            "  Downloading tokenizers-0.15.2-cp311-cp311-macosx_11_0_arm64.whl.metadata (6.7 kB)\n",
            "Collecting safetensors>=0.4.1 (from transformers)\n",
            "  Obtaining dependency information for safetensors>=0.4.1 from https://files.pythonhosted.org/packages/03/a0/67f2ed19dbc609ce1bf561a285a6cc8f804846a7f4d9c46b403be93022e4/safetensors-0.4.2-cp311-cp311-macosx_11_0_arm64.whl.metadata\n",
            "  Downloading safetensors-0.4.2-cp311-cp311-macosx_11_0_arm64.whl.metadata (3.8 kB)\n",
            "Requirement already satisfied: tqdm>=4.27 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from transformers) (4.65.0)\n",
            "Collecting fsspec>=2023.5.0 (from huggingface-hub<1.0,>=0.19.3->transformers)\n",
            "  Obtaining dependency information for fsspec>=2023.5.0 from https://files.pythonhosted.org/packages/93/6d/66d48b03460768f523da62a57a7e14e5e95fdf339d79e996ce3cecda2cdb/fsspec-2024.3.1-py3-none-any.whl.metadata\n",
            "  Downloading fsspec-2024.3.1-py3-none-any.whl.metadata (6.8 kB)\n",
            "Requirement already satisfied: typing-extensions>=3.7.4.3 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from huggingface-hub<1.0,>=0.19.3->transformers) (4.11.0)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from requests->transformers) (2.0.4)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from requests->transformers) (3.4)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from requests->transformers) (2.2.1)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from requests->transformers) (2024.2.2)\n",
            "Downloading transformers-4.39.3-py3-none-any.whl (8.8 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m8.8/8.8 MB\u001b[0m \u001b[31m51.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m00:01\u001b[0m00:01\u001b[0m\n",
            "\u001b[?25hDownloading huggingface_hub-0.22.2-py3-none-any.whl (388 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m388.9/388.9 kB\u001b[0m \u001b[31m37.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading safetensors-0.4.2-cp311-cp311-macosx_11_0_arm64.whl (393 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m393.4/393.4 kB\u001b[0m \u001b[31m33.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading tokenizers-0.15.2-cp311-cp311-macosx_11_0_arm64.whl (2.4 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.4/2.4 MB\u001b[0m \u001b[31m55.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m00:01\u001b[0m\n",
            "\u001b[?25hDownloading fsspec-2024.3.1-py3-none-any.whl (171 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m172.0/172.0 kB\u001b[0m \u001b[31m14.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: safetensors, fsspec, huggingface-hub, tokenizers, transformers\n",
            "  Attempting uninstall: safetensors\n",
            "    Found existing installation: safetensors 0.3.2\n",
            "    Uninstalling safetensors-0.3.2:\n",
            "      Successfully uninstalled safetensors-0.3.2\n",
            "  Attempting uninstall: fsspec\n",
            "    Found existing installation: fsspec 2023.4.0\n",
            "    Uninstalling fsspec-2023.4.0:\n",
            "      Successfully uninstalled fsspec-2023.4.0\n",
            "  Attempting uninstall: huggingface-hub\n",
            "    Found existing installation: huggingface-hub 0.15.1\n",
            "    Uninstalling huggingface-hub-0.15.1:\n",
            "      Successfully uninstalled huggingface-hub-0.15.1\n",
            "  Attempting uninstall: tokenizers\n",
            "    Found existing installation: tokenizers 0.13.2\n",
            "    Uninstalling tokenizers-0.13.2:\n",
            "      Successfully uninstalled tokenizers-0.13.2\n",
            "  Attempting uninstall: transformers\n",
            "    Found existing installation: transformers 4.32.1\n",
            "    Uninstalling transformers-4.32.1:\n",
            "      Successfully uninstalled transformers-4.32.1\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "s3fs 2023.4.0 requires fsspec==2023.4.0, but you have fsspec 2024.3.1 which is incompatible.\u001b[0m\u001b[31m\n",
            "\u001b[0mSuccessfully installed fsspec-2024.3.1 huggingface-hub-0.22.2 safetensors-0.4.2 tokenizers-0.15.2 transformers-4.39.3\n"
          ]
        }
      ],
      "source": [
        "!pip install --upgrade transformers\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 17,
      "metadata": {
        "id": "om3Kx8Kfoqvv"
      },
      "outputs": [],
      "source": [
        "def load_text_files(directory):\n",
        "    files = os.listdir(directory)\n",
        "    files = [file for file in files if not file.startswith('.DS')]\n",
        "    files = sorted(files, key=lambda x: int(re.split('\\.|_', x)[1]))\n",
        "    print(len(files))\n",
        "    data = []\n",
        "    for file in files:\n",
        "        if file.endswith('.txt'):\n",
        "            with open(os.path.join(directory, file), 'r', encoding='utf-8',errors='ignore') as f:\n",
        "                text = f.read()\n",
        "                data.append(text)\n",
        "    return data\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 18,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "3jSKIvOvBcel",
        "outputId": "dc4ffd37-0666-4cd7-94cd-8b16b0573208"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "32689\n",
            "32689\n",
            "1000\n",
            "1000\n"
          ]
        }
      ],
      "source": [
        "# Load text files from directories\n",
        "import os\n",
        "import re\n",
        "original_text = load_text_files('sumpubmed-master/text')[:1000]\n",
        "summaries = load_text_files('sumpubmed-master/shorter_abstract')[:1000]\n",
        "print(len(original_text))\n",
        "print(len(summaries))\n",
        "data = {'original_text': original_text, 'summary': summaries}"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 19,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "40907 1875\n",
            "25548 1422\n",
            "32083 2235\n",
            "31588 1677\n",
            "22660 1968\n",
            "45624 1617\n",
            "47588 2070\n",
            "29101 1728\n",
            "17390 1551\n",
            "26273 897\n"
          ]
        }
      ],
      "source": [
        "for i in range(10):\n",
        "    print(len(original_text[i]), len(summaries[i]))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 20,
      "metadata": {},
      "outputs": [],
      "source": [
        "data = {\n",
        "    \"text\": original_text,\n",
        "    \"summary\": summaries\n",
        "}\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 21,
      "metadata": {},
      "outputs": [],
      "source": [
        "from datasets import Dataset\n",
        "\n",
        "# Assuming data_dict is your initial dataset dictionary\n",
        "dataset = Dataset.from_dict(data)\n",
        "\n",
        "# Split the dataset into training and evaluation sets\n",
        "train_test_split = dataset.train_test_split(test_size=0.1)  # Adjust the split ratio as needed\n",
        "train_dataset = train_test_split['train']\n",
        "eval_dataset = train_test_split['test']\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 105,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Ignored unknown kwarg option special\n"
          ]
        }
      ],
      "source": [
        "from transformers import AutoTokenizer, AutoModelForSeq2SeqLM\n",
        "\n",
        "model_name = 'allenai/led-base-16384'\n",
        "tokenizer = AutoTokenizer.from_pretrained(model_name)\n",
        "model = AutoModelForSeq2SeqLM.from_pretrained(model_name)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 106,
      "metadata": {},
      "outputs": [
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "5a491d45e6fc41beabc5b9b791f14d23",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "Map:   0%|          | 0/900 [00:00<?, ? examples/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/Users/pavandukka/anaconda3/lib/python3.11/site-packages/transformers/tokenization_utils_base.py:3660: UserWarning: `as_target_tokenizer` is deprecated and will be removed in v5 of Transformers. You can tokenize your labels by using the argument `text_target` of the regular `__call__` method (either in the same call as your input texts if you use the same keyword arguments, or in a separate call.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "c7b6fd5ccffa41e8af40083aba82e3d8",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "Map:   0%|          | 0/100 [00:00<?, ? examples/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        }
      ],
      "source": [
        "def tokenize_function(examples):\n",
        "    model_inputs = tokenizer(examples['text'], max_length=4096, truncation=True, padding=\"max_length\")\n",
        "    with tokenizer.as_target_tokenizer():\n",
        "        labels = tokenizer(examples['summary'], max_length=256, truncation=True, padding=\"max_length\")\n",
        "\n",
        "    model_inputs['labels'] = labels['input_ids']\n",
        "    return model_inputs\n",
        "\n",
        "tokenized_train_datasets = train_dataset.map(tokenize_function, batched=True)\n",
        "tokenized_eval_datasets = eval_dataset.map(tokenize_function, batched=True)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 108,
      "metadata": {},
      "outputs": [],
      "source": [
        "from transformers import TrainingArguments\n",
        "\n",
        "training_args = TrainingArguments(\n",
        "    output_dir='./results',          \n",
        "    evaluation_strategy=\"epoch\",\n",
        "    learning_rate=5e-5,\n",
        "    per_device_train_batch_size=1,  # Start with 1 due to high memory demands; adjust as needed\n",
        "    weight_decay=0.01,\n",
        "    save_total_limit=2,\n",
        "    num_train_epochs=3,\n",
        "    report_to=\"none\"  # Reduce logging verbosity if preferred\n",
        ")\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 110,
      "metadata": {},
      "outputs": [],
      "source": [
        "from transformers import AutoModelForSeq2SeqLM, Trainer\n",
        "\n",
        "model = AutoModelForSeq2SeqLM.from_pretrained(model_name)\n",
        "\n",
        "trainer = Trainer(\n",
        "    model=model,\n",
        "    args=training_args,\n",
        "    train_dataset=tokenized_train_datasets,\n",
        "    eval_dataset=tokenized_eval_datasets,  # Provide the evaluation dataset here\n",
        "    tokenizer=tokenizer\n",
        ")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 111,
      "metadata": {},
      "outputs": [
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "90d11d52b91245c782b6f8f038cf03ef",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "  0%|          | 0/2700 [00:00<?, ?it/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "You're using a LEDTokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding.\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "{'loss': 2.8646, 'learning_rate': 4.074074074074074e-05, 'epoch': 0.56}\n"
          ]
        },
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "7404eed3d1f0464ba754e11ac0712ea5",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "  0%|          | 0/13 [00:00<?, ?it/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "{'eval_loss': 2.3731348514556885, 'eval_runtime': 70.5485, 'eval_samples_per_second': 1.417, 'eval_steps_per_second': 0.184, 'epoch': 1.0}\n",
            "{'loss': 2.4748, 'learning_rate': 3.148148148148148e-05, 'epoch': 1.11}\n",
            "{'loss': 2.0777, 'learning_rate': 2.2222222222222223e-05, 'epoch': 1.67}\n"
          ]
        },
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "fa94fee5f1dc4bf38fbc93f373a06d4f",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "  0%|          | 0/13 [00:00<?, ?it/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "{'eval_loss': 2.318596601486206, 'eval_runtime': 59.682, 'eval_samples_per_second': 1.676, 'eval_steps_per_second': 0.218, 'epoch': 2.0}\n",
            "{'loss': 1.9471, 'learning_rate': 1.2962962962962962e-05, 'epoch': 2.22}\n",
            "{'loss': 1.7481, 'learning_rate': 3.7037037037037037e-06, 'epoch': 2.78}\n"
          ]
        },
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "25fc2d01a32b4d94990387c07831dcbf",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "  0%|          | 0/13 [00:00<?, ?it/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "{'eval_loss': 2.3423874378204346, 'eval_runtime': 78.9431, 'eval_samples_per_second': 1.267, 'eval_steps_per_second': 0.165, 'epoch': 3.0}\n",
            "{'train_runtime': 4980.5246, 'train_samples_per_second': 0.542, 'train_steps_per_second': 0.542, 'train_loss': 2.1825097430193865, 'epoch': 3.0}\n"
          ]
        },
        {
          "data": {
            "text/plain": [
              "TrainOutput(global_step=2700, training_loss=2.1825097430193865, metrics={'train_runtime': 4980.5246, 'train_samples_per_second': 0.542, 'train_steps_per_second': 0.542, 'train_loss': 2.1825097430193865, 'epoch': 3.0})"
            ]
          },
          "execution_count": 111,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "trainer.train()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 112,
      "metadata": {},
      "outputs": [
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "6871070f229848c990acd0c504f931a6",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "  0%|          | 0/13 [00:00<?, ?it/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "{'eval_loss': 2.3423874378204346, 'eval_runtime': 102.0456, 'eval_samples_per_second': 0.98, 'eval_steps_per_second': 0.127, 'epoch': 3.0}\n"
          ]
        }
      ],
      "source": [
        "evaluation_results = trainer.evaluate()\n",
        "print(evaluation_results)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 23,
      "metadata": {},
      "outputs": [],
      "source": [
        "from transformers import AutoModelForSeq2SeqLM\n",
        "\n",
        "# Load the model\n",
        "model = AutoModelForSeq2SeqLM.from_pretrained('./results/checkpoint-2500')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 24,
      "metadata": {},
      "outputs": [],
      "source": [
        "from transformers import AutoTokenizer\n",
        "\n",
        "# Load the tokenizer\n",
        "tokenizer = AutoTokenizer.from_pretrained('./results/checkpoint-2500')\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 25,
      "metadata": {},
      "outputs": [
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "895dbf71ecaf4bc48f076ade2360ab52",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "Map:   0%|          | 0/100 [00:00<?, ? examples/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/Users/pavandukka/anaconda3/lib/python3.11/site-packages/transformers/tokenization_utils_base.py:3935: UserWarning: `as_target_tokenizer` is deprecated and will be removed in v5 of Transformers. You can tokenize your labels by using the argument `text_target` of the regular `__call__` method (either in the same call as your input texts if you use the same keyword arguments, or in a separate call.\n",
            "  warnings.warn(\n"
          ]
        }
      ],
      "source": [
        "def tokenize_function(examples):\n",
        "    model_inputs = tokenizer(examples['text'], max_length=4096, truncation=True, padding=\"max_length\")\n",
        "    with tokenizer.as_target_tokenizer():\n",
        "        labels = tokenizer(examples['summary'], max_length=256, truncation=True, padding=\"max_length\")\n",
        "\n",
        "    model_inputs['labels'] = labels['input_ids']\n",
        "    return model_inputs\n",
        "\n",
        "tokenized_eval_dataset = eval_dataset.map(tokenize_function, batched=True)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 26,
      "metadata": {},
      "outputs": [],
      "source": [
        "from datasets import load_metric\n",
        "\n",
        "rouge = load_metric('rouge')\n",
        "\n",
        "def compute_metrics(eval_pred):\n",
        "    predictions, labels = eval_pred\n",
        "    decoded_preds = tokenizer.batch_decode(predictions, skip_special_tokens=True)\n",
        "    decoded_labels = [label.split(tokenizer.pad_token, 1)[0] for label in tokenizer.batch_decode(labels, skip_special_tokens=True)]  # Remove padding\n",
        "\n",
        "    result = rouge.compute(predictions=decoded_preds, references=decoded_labels, use_aggregator=True)\n",
        "    return {\n",
        "        \"rouge1\": result['rouge1'].mid.fmeasure,\n",
        "        \"rouge2\": result['rouge2'].mid.fmeasure,\n",
        "        \"rougeL\": result['rougeL'].mid.fmeasure,\n",
        "        \"rougeLsum\": result['rougeLsum'].mid.fmeasure\n",
        "    }\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 27,
      "metadata": {},
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/Users/pavandukka/anaconda3/lib/python3.11/site-packages/accelerate/accelerator.py:436: FutureWarning: Passing the following arguments to `Accelerator` is deprecated and will be removed in version 1.0 of Accelerate: dict_keys(['dispatch_batches', 'split_batches', 'even_batches', 'use_seedable_sampler']). Please pass an `accelerate.DataLoaderConfiguration` instead: \n",
            "dataloader_config = DataLoaderConfiguration(dispatch_batches=None, split_batches=False, even_batches=True, use_seedable_sampler=True)\n",
            "  warnings.warn(\n"
          ]
        }
      ],
      "source": [
        "from transformers import Trainer, TrainingArguments\n",
        "\n",
        "training_args = TrainingArguments(\n",
        "    output_dir='./results',  # Directory to save the output results\n",
        "    do_predict=True,\n",
        "    per_device_eval_batch_size=2  # Adjust based on your hardware capabilities\n",
        ")\n",
        "\n",
        "trainer = Trainer(\n",
        "    model=model,\n",
        "    args=training_args,\n",
        "    compute_metrics=compute_metrics,\n",
        "    tokenizer=tokenizer,\n",
        "    eval_dataset=tokenized_eval_dataset  # Pass the tokenized evaluation dataset\n",
        ")\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 16,
      "metadata": {},
      "outputs": [
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "0f5c0c43835641baa8234cf2efcf99b0",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "  0%|          | 0/25 [00:00<?, ?it/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "\n",
            "KeyboardInterrupt\n",
            "\n"
          ]
        }
      ],
      "source": [
        "evaluation_results = trainer.evaluate()\n",
        "print(evaluation_results)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 50,
      "metadata": {},
      "outputs": [],
      "source": [
        "# Set the model to use the MPS device\n",
        "model = model.to('mps')\n",
        "\n",
        "def generate_summary(text):\n",
        "    # Ensure tensors are explicitly moved to the MPS device\n",
        "    inputs = tokenizer(text, return_tensors=\"pt\", max_length=4096, truncation=True).to('mps')\n",
        "    # Adding parameters to handle padding and EOS more gracefully\n",
        "    outputs = model.generate(\n",
        "        inputs['input_ids'],\n",
        "        max_length=100,  # Adjusted to match typical output lengths\n",
        "        num_beams=5,\n",
        "        early_stopping=True,\n",
        "        pad_token_id=tokenizer.pad_token_id,\n",
        "        eos_token_id=tokenizer.eos_token_id\n",
        "    )\n",
        "\n",
        "    return tokenizer.decode(outputs[0], skip_special_tokens=True)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 51,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "32689\n",
            "32689\n",
            "100\n"
          ]
        }
      ],
      "source": [
        "# Example usage\n",
        "test_text = load_text_files('sumpubmed-master/text')[1000:1100]\n",
        "val_summary = load_text_files('sumpubmed-master/shorter_abstract')[1000:1100]\n",
        "print(len(test_text))\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 52,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "a number of genes and transcripts were detected to be differentially expressed in endometrium tissue on day  <dig> of gestation, during maternal recognition of pregnancy.\n",
            "we identified  <dig>  differentially expressed genes and  <dig> differentially expressed transcripts in uterine endometrium tissue on day  <dig> of gestation, during maternal recognition of pregnancy.\n",
            "these data contribute toward a deeper understanding of the molecular events associated with reproductive success in pigs.\n",
            "the pig\n",
            "in this study, two error correction packages are analyzed by examining their effects on next-generation sequencing data from a heterozygous genome.\n",
            "two error correction packages are comparatively analyzed by examining their effects on next-generation sequencing data from a heterozygous genome.\n",
            "two error correction packages are comparatively analyzed by examining their effects on next-generation sequencing data from a heterozygous genome.\n",
            "two error correction packages are comparatively analyzed by examining their effects on next-generation sequencing data from a heter\n",
            "the bovine hapmap project was started with the goal of understanding the genetic structure and history of an important livestock species, insights that are not currently available from the study of humans or other animal species.\n",
            "the aim of this study was to compare genome wide signatures of selection found in the bovine hapmap project with genotypes collected from a similar set of animals for which phenotypes are known.\n",
            "the bovine hapmap project reported the sampling of  <\n",
            "sin <dig> is a little studied gene product of unclear function found in species ranging from mammals to fungi.\n",
            "sin <dig> is an amino acid protein that bound via polypeptide sequences in its c-terminal  <dig> amino acid domain to the stress-activated sty1/spc <dig> map kinase pathway in s. pombe, s. cerevisiae, and n. crassa.\n",
            "sin <dig> was found in\n",
            "marbofloxacin acts as a bactericidal concentration-dependent antibiotic similar to other fluoroquinolones.\n",
            "in this study, we evaluated the in vivo antimicrobial activity of marbofloxacin against p. multocida in tissue cage fluid in the presence of an active immune response.\n",
            "the in vivo antimicrobial activity of marbofloxacin against p. multocida in tissue cage fluid was determined using an integrated pk/\n",
            "we identified  <dig> putative abc genes in the tetranychus urticae genome, and  <dig> putative abc proteins belonging to the abca, abcb, abcc, abcd, abce, abcf, abcg and abch subfamilies.\n",
            "we identified  <dig> putative abc genes in the tetranychus urticae genome, and  <dig> putative abc proteins\n",
            "fus <dig> is a key transcriptional regulator of the precocious vegetative phase transition in arabidopsis thaliana.\n",
            "furthermore, fus <dig> downregulates a collection of genes involved in ethylene biosynthesis and signaling.\n",
            "furthermore, fus <dig> downregulates a collection of genes involved in ethylene biosynthesis and signaling.\n",
            "furthermore, fus <dig> downregulates a collection of genes involved\n",
            "the proposed algorithm is first applied to chromosome iii of caenorhabditis elegans, containing a total of  <dig> nucleotides with period- <dig> property.\n",
            "we show that the proposed algorithm enhances the accuracy of the identification while reducing noise.\n",
            "the proposed algorithm is first applied to chromosome iii of caenorhabditis elegans, containing a total of  <dig> nucleotides with period- <dig> property.\n",
            "the proposed algorithm\n",
            "inactivation of the pst system resulted in constitutive expression of the pho regulon, regardless of environmental phosphate availability, and affect virulence of many pathogenic bacteria.\n",
            "inactivation of the pst system resulted in constitutive expression of the pho regulon, regardless of environmental phosphate availability, and affect virulence of many pathogenic bacteria.\n",
            "inactivation of the pst system resulted in constitutive expression of the pho regulon, regardless of environmental phosphate availability, and\n",
            "teneurin transmembrane protein  <dig>  is a mouse homolog of the drosophila gene tenascin major  that encodes a transmembrane protein with tenascin-like egf  repeats.\n",
            "tenm4m1/m <dig> mutant embryos exhibited a wide array of phenotypes, ranging from embryonic death at gastrulation to viable with skeletal defects.\n",
            "tenm4m1/m <dig> mutant\n",
            "our method correctly identified five more actual perturbed tfs when the path length from the initial tf to the last tf being equal to or less than  <dig>  and six more tfs in overexpression experiments.\n",
            "here we present a tf-centered downstream gene set enrichment analysis to identify potential causal regulators responsible for gene expression changes.\n",
            "furthermore, our method correctly identified five more actual perturbed tfs when the path length from the initial tf to the last tf was\n",
            "we propose a model of amnion formation in the mouse involving a single acf growing and expanding laterally from the posterior side of the embryo.\n",
            "we propose a model of amnion formation in the mouse involving a single acf growing and expanding laterally from the posterior side of the embryo.\n",
            "we propose a model of amnion formation in the mouse involving a single acf growing and expanding laterally from the posterior side of the embryo.\n",
            "in mouse embryos, the\n",
            "we propose the use of the conditional probability distribution of disease genotypes given any marker genotype in the samples and to test for significance of the disequilibrium by making use of the samples that are not randomly collected from the population of interest.\n",
            "we demonstrate that bias in estimates of the disequilibrium parameter arising from the use of non-random samples can be substantially reduced by implementing the new method.\n",
            "we developed a new likelihood-based statistical approach to estimate the coefficient of linkage disequ\n",
            "the results suggest that chromosomal evolution in tilapia is dominated by alternate loss of gene duplicates, and by intra-chromosomal rearrangements.\n",
            "these results suggest that chromosomal evolution in tilapia is dominated by alternate loss of gene duplicates, and by intra-chromosomal rearrangements.\n",
            "the sequence of the tilapia genome will be a fundamental resource used for genetic selection, on traits such as growth performance and disease resistance, to create strains of\n",
            "in order to gain insight into the tissue specificity of somatic instability in hd, we developed a novel instability quantification/bioinformatics approach that is sensitive and applicable to high-throughput assays.\n",
            "our results suggest that somatic instability occurs as a result of ongoing pathogenesis, and suggests novel pathways that may be important in determining the tissue specificity of instability in hd.\n",
            "our results suggest that somatic instability occurs as a result of ongoing pathogenesis, and\n",
            "recombination is a fundamental process of bacterial evolution, capable of influencing the integrity of species.\n",
            "in this study, we present bioinformatic analyses of recombination events  between  <dig> publicly available genomes of escherichia coli isolates from different phylogroups  and ecological backgrounds.\n",
            "recombination events  between  <dig> publicly available genomes of escherichia coli isolates from different phylogroups  and ecological backgrounds provide a unique opportunity to\n",
            "narrow-leafed lupin  is a protein-rich grain crop for animal feed and human consumption that has excellent potential for sustainable crop rotation systems.\n",
            "narrow-leafed lupin  is a protein-rich grain crop for animal feed and human consumption that has excellent potential for sustainable crop rotation systems.\n",
            "narrow-leafed lupin  is a protein-rich grain crop for animal feed and human consumption that has excellent potential for sustainable crop rotation systems.\n",
            "in this study, we investigated whether the development of pharyngeal jaws differed between three types of diets:  intact snails with shell,  peeled snails without shell, and  finely ground up whole snails frozen in pellets, and  finely ground up whole snails frozen in pellets, and  finely ground up whole snails frozen in pellets, and  finely ground up whole snails frozen in pellets, and  finely ground up whole snails frozen in pellets, and \n",
            "in this paper, we present a new gene semantic similarity measure called intego <dig>  by choosing the most appropriate seed measures for each gene pair from a pool of candidate measures, and integrating the selected seed measures using a metaheuristic search method.\n",
            "in this paper, we present a new gene semantic similarity measure called intego <dig>  by choosing the most appropriate seed measures for each gene pair from a pool of candidate measures, and by integrating the selected seed measures using a\n",
            "adult neurogenesis in the central nervous system is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues during adulthood.\n",
            "adult neurogenesis in the central nervous system is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues during adulthood.\n",
            "adult neurogenesis in the central nervous system is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues during adulthood.\n",
            "adult neurogenesis in\n",
            "in order to better understand the mechanisms by which hox gene expression is initiated and regulated in nematodes, we sought to identify transcription factors that regulate lin- <dig> expression, with an emphasis on 1) regulation in the vulval precursor cells p <dig> p - p <dig> p and their descendants in the larva, and 2) expression in the vulval precursor cells p <dig> p - p <dig> p and their descendants in the embryo and lar\n",
            "here, we introduce three main-chain atom based surface representations which are found to better agree with the two existing structure classifications, the ce and the scop database  <dig>, as compared with the previous all-atom surface representation.\n",
            "the proposed representation also allows for a fast and accurate database search for em density maps where atomic coordinates are not available.\n",
            "the proposed representation also allows for a fast and accurate database search for em density maps where atomic coordinates are not available.\n",
            "\n",
            "the mrna sequence encoding muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <\n",
            "in the present study, we investigated the methylation dynamics of the major genes involved in myogenic determination and differentiation by comparing pluripotent embryonic stem cells, myoblasts, myotubes, myofibres and myogenic precursors from pax7-inducible escs, proliferating muscle stem cells, myogenic precursors from pax7-inducible escs, myogenic precursors from pax7-inducible escs,\n",
            "notch signaling in drosophila is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\n",
            "notch signaling is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\n",
            "notch signaling is regulated by two conserved glycosyltransferases, o-fucosyltransferase\n",
            "lactobacillus fermentum isolates were isolated from a relatively large geographic area including mongolia, four chinese provinces and two autonomous regions of china, and from various naturally fermented dairy products and acidic gruel.\n",
            "lactobacillus fermentum isolates were isolated from a relatively large geographic area including mongolia, four chinese provinces and two autonomous regions of china, and from various naturally fermented dairy products and acidic gruel.\n",
            "lactobac\n",
            "glioblastoma multiforme, a grade iv astrocytoma, is the most common form of primary brain cancer, with a median survival of approximately 1 year after multi-modal treatments.\n",
            "glioblastoma multiforme  is the most common form of primary brain cancer, with a median survival of approximately 1 year after multi-modal treatments.\n",
            "we identified  <dig> fusion events in  <dig> gbm samples by\n",
            "this study describes an application of statistical ir methodology to the analysis of a corpus consisting of  <dig>   <dig>   <dig> documents and a mixture of unigrams, unigrams, and random models.\n",
            "the lda model was trained on  <dig>   <dig>   <dig>   <dig> documents and a mixture of unigrams, unigrams, and random models.\n",
            "the lda model is\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3968 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "in this study we analysed the transcriptional changes in the developing humerus and associated joints at theiler stage  23- <dig> in muscle-less  compared to phenotypically normal littermate controls.\n",
            "in this study we analysed the transcriptional changes in the developing humerus and associated joints at theiler stage  23- <dig> in muscle-less  compared to phenotypically normal littermate controls.\n",
            "in this study we analysed the transcriptional changes in\n",
            "the aim of this study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\n",
            "the aim of the study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\n",
            "\n",
            "the crude enzyme was characterized for its catalytic activities and its components identified by proteomics.\n",
            "in this study, a biomass degrading enzyme system of the soft-rot fungus c. globosum bcc <dig> was studied.\n",
            "the crude enzyme was characterized for its catalytic activities and its components identified by proteomics.\n",
            "the crude enzyme was used for formulation of an efficient biomass-degrading enzyme mixture by a mixture design approach and applied for hydrolysis of alkaline\n",
            "to evaluate the impact of prophage induction on host gene expression, we constructed a cdna e. coli/bacteriophage lambda microarray, representing 99% of  <dig> e. coli open reading frames and  <dig> predicted lambda open reading frames.\n",
            "in the lysogenic state lambda expresses a small set of proteins that, among other things, repress lysis and confer fitness to the host while perturbing little else in host function.\n",
            "\n",
            "mutualistic interactions occur when two different organisms – referred to here as mutualistic partners - have reciprocal positive effects on each other’s growth.\n",
            "mutualistic interactions occur when two different organisms – referred to here as mutualistic partners - have reciprocal positive effects on each other’s growth.\n",
            "mutualistic interactions occur when two different organisms – referred to here as mutualistic partners - have reciprocal positive effects on each other’s growth.\n",
            "mutualistic interactions are\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3811 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "our results suggest that co-expression analysis is a powerful tool in the discovery of new targets for analysis in pathways or networks of interest.\n",
            "in addition to co-expressed genes, a large number of new candidate genes were identified that are co-expressed with important genes in the sa, ja and et signaling pathways.\n",
            "in addition to co-expressed genes, a large number of new candidate genes were identified that are co-expressed with important genes in the sa, ja\n",
            "we analyzed wes data from a family with a history of nshl by focusing on three categories of genetic information: cnv, linkage, and snvs.\n",
            "we identified  <dig>  coding autosomal snvs,  <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>  and  <dig>   <dig>   <dig> \n",
            "we here show that the large intracellular domain of human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\n",
            "the human pit <dig> histidine, h <dig>  located in the c-terminal pit family signature sequence, is critical for the pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\n",
            "our results\n",
            "we have recently shown, by an in silico approach, that the presence and abundance of secretory proteins, as well as their relative abundance in the secretomes of b. cinerea cultures, is crucial in determining the structure and function of numerous secreted and membrane-bound proteins.\n",
            "we have recently shown that the presence and abundance of secretory proteins, as well as their relative abundance in the secretomes of b. cinerea cultures, is crucial in determining the structure and\n",
            "to investigate the functional role of putative cis-elements in the transcriptional regulation of egcad <dig> and egcad <dig> genes in vascular tissues, we introduced several truncated versions of the egcad <dig> promoter fused to the uida reporter gene coding for β-glucuronidase  in tobacco by agrobacterium-mediated transformation.\n",
            "the egcad <dig> cis-element fused to the uida reporter gene coding\n",
            "in the present study, we have compiled a relatively large data set of bcrp substrates and non-substrates from literature and developed an svm-based in silico model for prediction of bcrp substrates and non-substrates of certain enzymes and transporters.\n",
            "in this study, we have developed a svm-based in silico model for prediction of bcrp substrates and non-substrates of certain enzymes and transporters\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3691 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "protein-protein interaction  networks integrate various biological signals including those used for energy generation, cell division and growth to give a few notable examples.\n",
            "protein-protein interaction  networks are suggested to provide several biological advantages.\n",
            "protein-protein interaction  networks are suggested to provide several biological advantages.\n",
            "protein-protein interaction  networks are suggested to provide several biological advantages.\n",
            "protein-protein interaction  networks are suggested to provide several biological advantages.\n",
            "protein-protein interaction  networks are proposed to\n",
            "our results suggest the presence of the human o <dig> allele already in the common ancestor of modern human and neandertal populations.\n",
            "the abo gene is organized in  <dig> exons that range in size from  <dig> to  <dig> nucleotides.\n",
            "our results suggest the presence of the human o <dig> allele already in the common ancestor of modern human and neandertal populations.\n",
            "our results suggest the presence of the human\n",
            "in this context, it is important to sample broadly among the charophytes if the homologs of key embryophyte genes are to be identified.\n",
            "we undertook here an analysis of two representative charophyte taxa: spirogyra pratensis and coleochaete orbicularis pringh.\n",
            "the results of this study highlight the importance of using plant gene models to annotate the putative putative putative putative putative putative putative\n",
            "camptothecin  is the precursor and backbone of many tias, including cpts, in plants such as c. acuminata.\n",
            "camptothecin  was first extracted from the stems of camptotheca acuminata in  <dig> and subsequently from nothapodytes foetida, ophiorrhiza pumila, ophiorrhiza japonica, and ophiorrhiza japonica.\n",
            "camptothe\n",
            "in this study,  <dig> serum or heparin plasma samples of  <dig> asian elephants located in  <dig> european zoos were used to assess seroprevalence against eehv-gb in serum and heparin plasma samples from two large cohorts of european and north-american captive asian elephants.\n",
            "the aim of this study was to assess seroprevalence against eehv-gb in serum and\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3225 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "in addition to bcr signaling, the splenic microenvironment and the ability to respond to locally produced growth factors play equally important roles in the development and maintenance of mature b cells.\n",
            "in addition to bcr signaling, the splenic microenvironment and the ability to respond to locally produced growth factors play equally important roles in the development and maintenance of mature b cells.\n",
            "in addition to bcr signaling, the splenic microenvironment and the ability to respond to locally produced growth factors play equally\n",
            "in this study, dna-microarrays and real-time rt-pcr analysis were carried out to characterize the transcriptional differences of s. mutans in various biofilms.\n",
            "the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm growth in s. mutans.\n",
            "the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose\n",
            "in this work, we describe the method to specifically identify all trans-sialidase-like sequences in the reference t. cruzi cl brener genome, raising the total number from the  <dig> initially annotated tcts genes to  <dig>  newly annotated tcts genes.\n",
            "the t. cruzi genome provides ample evidence of large gene families and thus are the focus of continued research to prevent and treat the diseases that they cause.\n",
            "\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 1904 to 2048 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "a high-density genetic map of sugarcane is a valuable tool to understand the genetic and genomic organisation of this complex polyploid crop.\n",
            "a genetic map of sugarcane was generated using a population of full-sib  individuals  with a total length of  <dig>  cm and an average distance of  <dig>  cm between markers.\n",
            "a genetic map of sugarcane was generated using a population of full-sib  individuals \n",
            "the purpose of this study is to show that serial skeletal muscle tractography in chick embryonic hind limb can be used as a powerful investigative tool to evaluate embryonic development with a dual-cooling technique, and investigate whether serial skeletal muscle tractography in dti correlates with ex vivo histology from day  <dig> of incubation through hatching.\n",
            "the tracked fiber length of the hind limb in dti correlates with histologic specimens from sacrificed embryos.\n",
            "the tracked fiber length of the hind\n",
            "in this study, we have generated a reference gene expression data set for non-neoplastic colonic epithelial cells from  <dig> different colectomy specimens with the clinical indication for each case being diverticular disease.\n",
            "in this study, we have generated a reference gene expression data set for non-neoplastic colonic epithelial cells from  <dig> different colectomy specimens with the clinical indication for each case being diverticular disease.\n",
            "in this study,\n",
            "here, we have investigated the genetic basis of c. gigas resistance to vibrio infections by high-throughput microfluidic qpcr approaches.\n",
            "our results reveal an extraordinary interindividual polymorphism in basal gene expression that is associated to an inherent capacity of some oysters to survive vibrio infections.\n",
            "the expression profiles of  <dig> hemocytes  have been collected from oysters before an infection by virulent vibrio strains isolated during summer mortality episodes in france\n",
            "a. marginale is a member of the order rickettsiales and a well-established obligate intracellular bacterial model.\n",
            "the level of diversity in the pan-genome is highly conserved, which, in spite of the highly conserved gene content, leads to increased plasticity.\n",
            "a. marginale is a member of the order rickettsiales and a well-established obligate intracellular bacterial model.\n",
            "a. marginale has a\n",
            "gas <dig> has been shown to inhibit cell proliferation and to mediate cell death, to mediate cell death, and to support growth of the cerebellum.\n",
            "gas <dig> was found to contain  <dig> amino acids, a potential signal peptide, one n-glycosylation site at asn <dig>  and an aminated ser <dig>.\n",
            "gas <dig> has been shown to colocalize to lipid rafts with ret\n",
            "to test whether egr <dig> regulates peripheral nerve myelination by egr <dig> in schwann cells, we examined the expression levels of egr <dig>  egr <dig>  etv <dig>  and egr <dig> in mouse and rat sciatic nerve of egr <dig> knockout mice.\n",
            "our results suggest that egr <dig> regulates peripheral nerve myelination by egr <dig> in schwann\n",
            "the aim of the present study was to determine the degree of the metabolism of aromatic amino acids by streptococcus mutans on the basis of one of the key enzymes of the aromatic amino acid pathway - prephenate dehydrogenase.\n",
            "the aim of the study was to identify streptococcus mutans pathogenicity determinants and to find an enzyme or an enzymatic biotype that could be targeted to hinder bacterial survival and therefore to prevent caries development.\n",
            "the\n",
            "in contrast to the pathogenic strain, l. licerasiae serovar copenhageni has  <dig> rfb locus genes, while l. licerasiae has  <dig> rfb locus genes.\n",
            "in contrast to the pathogenic strain, l. licerasiae serovar copenhageni has  <dig> rfb locus genes, while l. licerasiae has  <dig> rfb locus genes\n",
            "in nature, t. mirus has formed repeatedly, but only when t. dubius is the paternal parent and t. porrifolius is the maternal parent.\n",
            "in nature, t. mirus has formed repeatedly, but only when t. dubius is the paternal parent and t. porrifolius is the maternal parent.\n",
            "our results suggest that t. mirus has undergone at least one round of genome-wide duplication, and that this duplication may have\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 1550 to 2048 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "dact proteins have emerged as nodal points in the simultaneous control of the various wnt and tgfβ signaling pathways.\n",
            "dact proteins are modular proteins, using different structural domains to interact with their specific partners.\n",
            "dact proteins have so far only been found in bony vertebrates, but, using bioinformatic tools, they have emerged as nodal points in the simultaneous control of the various wnt and tgfβ signaling pathways.\n",
            "dact proteins\n",
            "in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> in drgn is not permanent but only transient.\n",
            "these data suggest that the transient gabaergic hyperpolarizing shift observed between e <dig> and e <dig> reflects oxytocin-induced fetal adaptation to delivery and contributes to fetal analgesia and protects the fetus against neuronal insult.\n",
            "in contrast to other neuronal types, the activity-dependent downregulation of nkcc\n",
            "seldi-tof-ms protein profiles were generated from three week old piglets experimentally infected with either a combination of pcv <dig> and prrsv, pcv <dig> and ppv, or phosphate buffered saline.\n",
            "seldi-tof-ms protein profiles were generated from three week old piglets experimentally infected with either a combination of pcv <dig> and ppv, pcv2/ppv and pigmap in the\n",
            "in this study we tested the hypothesis that the effects of amino acids and insulin on the ubiquitin-proteasome pathway in c2c <dig> myotubes are mediated by the ubiquitin-proteasome pathway.\n",
            "in this study we tested the hypothesis that amino acids and insulin act together to suppress the ubiquitin-proteasome pathway in c2c <dig> myotubes, via the ubiquitin-proteasome\n",
            "a sequential structure prediction algorithm, based on the most successful free modelling method of our time, rosetta, was developed and used to predict structure, first starting from the nitrogen terminus and then starting from the carbon terminus.\n",
            "in order to investigate whether directionality of synthesis can have an impact on protein structure prediction, a sequential structure prediction algorithm, based on the most successful free modelling method of our time, rosetta, was developed and used to predict structure, first starting from\n",
            "the solubility of α/β-hydrolases is predicted to be more soluble in e. coli than large α/β-hydrolases.\n",
            "the solubility of α/β-hydrolases is predicted to be more soluble in e. coli than large α/β-hydrolases.\n",
            "the solubility of α/β-hydrolases is predicted to be more than  <dig> % solubile in e. coli.\n",
            "\n",
            "over-expression of tbx <dig> may promote accelerated mammary gland development by promoting mammary epithelial cell proliferation in vivo.\n",
            "over-expression of tbx <dig> may promote accelerated mammary gland development through the inhibition of the nf-κb pathway and stimulation of both mammary epithelial cell and stem-like cell proliferation.\n",
            "over-expression of tbx <dig> may promote accelerated mammary gland development by promoting mammary epithelial cell\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3495 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "we present a k-spectrum based error correction algorithm to filter out erroneous reads in a high-coverage illumina dataset.\n",
            "our results show that if around 80% of the reads in a dataset are perfect, then our algorithm retains almost  <dig> % of the perfect reads with more than 90% precision rate.\n",
            "the results of our experiments show that if only kmers from perfect reads are used instead, this leads to an improvement of up to 10% on the percentage\n",
            "in this study, we applied our method to classify activities of cytochrome p <dig> c <dig> inhibitors and their activities were predicted with 100% accuracy.\n",
            "in this study, we developed an iterative approach to classify activities of cytochrome p <dig> c <dig> inhibitors and their activities were predicted with 100% accuracy.\n",
            "in addition to cytochrome p <dig> c <dig> inhibitors, we developed a new dataset for cytochrome p\n",
            "the amino acid sequences of gpt <dig> and gpt <dig> are novel plant-specific + tips that track growing mt plus ends independently of eb <dig> and spr <dig> in cultured nicotiana tabacum epidermal cells.\n",
            "gpt <dig> and gpt <dig> are novel plant-specific + tips that track growing mt plus ends independently of eb <dig> and spr <dig> \n",
            "salmonella enterica serovar typhimurium is a gram-negative zoonotic bacterium that can colonise the gut of humans and many species of food producing farm animals and cause enteric or septicaemic salmonella infection.\n",
            "salmonella shedding associated modules identified at day  <dig> and day  <dig> p.i.\n",
            "salmonella shedding associated modules identified at day  <dig> and day  <dig> p.\n",
            "fossil viruses  are a group of retroviruses that have a very stable and long co-speciation history with their hosts, and because of this, almost all of their divergence dates can be directly inferred from their measurement timescales.\n",
            "fossil viruses  are a group of complex retroviruses that have a very stable and long co-speciation history with their hosts, and because of this, almost all of their divergence dates can be directly inferred from those of\n",
            "the association of invasive and non-invasive isolates with the m phenotype of macrolide resistance was not statistically supported.\n",
            "in this study, a collection of  <dig> invasive gas isolates collected from portugal between  <dig> and  <dig>  was used to evaluate if the clone distribution among the invasive and non-invasive isolates in portugal reflected the clonal structure of the isolates causing pharyngitis, in terms of molecular properties and\n",
            "in the dba/2j strain, expression of iav ha mrna was up-regulated, followed by a procedure-dependent up-regulation.\n",
            "in the c57bl/6j strain, expression of iav ha mrna was up-regulated, followed by a procedure-dependent up-regulation.\n",
            "in the dba/2j strain, expression of iav ha mrna was up-regulated, followed by a procedure-dependent\n",
            "acid soils are the most important cause of low yield for many crops in the united states and europe.\n",
            "our results provide a more complete picture of the transcriptomic changes imposed by acid soil, and they may lead to the discovery of novel genes involved in al tolerance.\n",
            "in addition, our analysis identified genes in several metabolic pathways whose expression was modified when plants were growing in acid soil.\n",
            "acid soils are the most important cause of low yield for many crops in the united states and euro\n",
            "the c-terminal region of tspgwi is almost entirely disordered, with only a short segment comprising residues 960– <dig> predicted to form a stable helical secondary structure.\n",
            "in this paper we describe the cloning of the tspgwi gene without accompanying  mtase, based on a gene nucleotide sequence determination starting from internal amino acid sequences of rease proteolytic fragments followed by various pcr methods using combination of degenerated and non-degenerated primers\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3457 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "in this study, we analyzed the whole transcriptome of mouse liver and identified  <dig>  differentially expressed isoforms  that were significantly differentially expressed with age in mouse liver.\n",
            "using rna-sequencing, we were able to identify  <dig>  differentially expressed isoforms that were significantly differentially expressed with age in both young and old mouse livers.\n",
            "in total we found  <dig>  differentially expressed isoforms  that were significantly different\n",
            "in this study, we studied the effect of two- and three-strain coinfections on the virulence of flavobacterium columnare  using zebra fish as a model host.\n",
            "in this study, we investigated the effect of two- and three-strain coinfections on the virulence of flavobacterium columnare using zebra fish as a model host.\n",
            "we found a significant interaction between bacterial dose and strain identity on their effect on the host\n",
            "in this model, multiple preferences are controlled by two haploid loci, each with two alleles: the preferences locus, p, controls female preference, and the trait locus, t, controls male traits.\n",
            "in such scenarios, the strength of preference necessary to overcome the costs of choice is lower when multiple preferences are present versus a single preference.\n",
            "in this study, i first model the evolution of a single female preference in a system with costly intrasexual competition for mates\n",
            "in this context, the use of appropriate clustering algorithms allow the removal of foreign sequences to yield clean single-cell genome assemblies.\n",
            "the use of appropriate clustering algorithms allow the removal of foreign sequences to yield clean single-cell genome assemblies.\n",
            "acdc is a powerful tool in the quality assurance of single-cell draft genomes, especially for users who do not have the background to use the included techniques directly.\n",
            "the use of appropriate clustering algorithms allow the removal of foreign sequences to\n",
            "using network topological features, tissue expression profile and subcellular localization data, we show here that our meta-classifier trained on these systems-level data can reliably predict morbid and druggable genes on a genome-wide scale and also can define cellular rules governing morbidity and druggability in human.\n",
            "our meta-classifier trained on network topological features, tissue expression profile and subcellular localization data seems indeed to be a reliable predictor of morbid and druggable\n",
            "the proposed filtering procedure improves the sensitivity/specificity to detect small differential expression, especially in the range of low intensities.\n",
            "our results demonstrate that the proposed method improves the sensitivity/specificity to detect small differential expression, especially in the range of low intensities.\n",
            "the proposed filtering procedure improves the sensitivity/specificity to detect small differential expression, especially in the range of low intensities.\n",
            "the proposed method is robust in the sense that only few assumptions are made, and it\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3506 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "we present a new method for computing steady states of a boolean network, combining a graph theoretic reduction/transformation method with an approach using computational algebra.\n",
            "we show that our method performs favorably on some types of networks in comparison with other methods on a collection of benchmark networks, consisting of both published models and random networks with certain properties, namely kauffman networks and networks whose in-degree distribution satisfies a power law.\n",
            "we show that our method performs favorably on some types of\n",
            "the aim of this study was to investigate the efficacy of two skin antiseptics based on 85% ethanol on the skin of the forehead with application times of  <dig>   <dig>   <dig>  and  <dig> min on skin with a high density of sebaceous glands.\n",
            "the aim of this study was to investigate the efficacy of two skin antiseptics based on 85% ethanol on skin with a high density of sebaceous\n",
            "the hub classifier was trained on the existing go and protein-protein interaction data for escherichia coli, saccharomyces cerevisiae, drosophila melanogaster, and homo sapiens species.\n",
            "the hub classifier was trained on the existing go and protein-protein interaction data for escherichia coli, saccharomyces cerevisiae, drosophila melanogaster, and homo sapiens species.\n",
            "the hub classifier\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3120 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "chondrodysplasiachondrodys\n",
            "herpesviruses are members of herpesviridae family, a large family of dna viruses that cause chronic, latent and recurrent infections in animals and humans.\n",
            "herpesviruses preferentially target pin hubs and bottlenecks, a process similar to that of human-encoded mirnas.\n",
            "herpesviruses are double-stranded dna viruses with large genomes encoding complex virus particles and enzymes involved in a variety of cellular process, including nucleic acid metabolism,\n",
            "in silico analysis of celtos and human interleukin  <dig> fused to human interleukin- <dig>  and m cell-specific peptide ligand, co <dig> were used to design the designed construct.\n",
            "in silico analysis of celtos and human interleukin  <dig> fused to human interleukin- <dig>  and m cell-specific peptide ligand, co <dig> were\n",
            "site-directed mutagenesis is a tool used in rational protein design strategies to modify the structure or function of a protein to adapt it to particular performance requirements.\n",
            "site-directed mutagenesis is a tool used in rational protein design strategies to modify the structure or function of a protein to adapt it to particular performance requirements.\n",
            "site-directed mutagenesis is a tool used in rational protein design strategies to modify the structure or function of a protein to adapt it to particular performance requirements.\n",
            "in order to understand how a protein network gives rise to a cellular function it is essential to quantify the states of the involved proteins and their causal relations.\n",
            "our method identifies better than common clustering approaches the presence of two distinct subpopulations within a mixture of pc <dig> cells based on the levels of active raf, mek and erk in each subpopulation.\n",
            "our method identifies better than common clustering approaches the presence of two distinct subpopulations within a mixture of\n",
            "plant invasion into new environments is an extremely costly problem, which is why many species of invasive weeds have been introduced to new environments.\n",
            "plant performance and life cycle analysis revealed that introduced tetraploids exhibited lower expression of constitutive defense and secondary metabolite related genes compared to native tetraploids.\n",
            "plant performance and life cycle analysis revealed that introduced tetraploids exhibited lower expression of constitutive defense and secondary metabolite related genes compared to native tetraploids.\n",
            "plant\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 2538 to 3072 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "over-transcribed detoxification genes were found in adults of the vauclin strain, indicating a significant role of p450s and gsts in insecticide resistance in martinique.\n",
            "in addition,  <dig> detoxification genes were found to be over-transcribed in larvae and adults of the vauclin strain, indicating a significant role of p450s and gsts in insecticide resistance in martinique.\n",
            "in addition,  <dig>\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 1160 to 2048 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "our new normalization method does not require that the majority of genes be non-differentially expressed, and the numbers of over- and under-expressed genes are roughly equal; it appears much more effective than standard normalization methods when the numbers of over- and under-expressed genes are unequal, and the number of over- and under-expressed genes are roughly equal.\n",
            "our new normalization method does not require that the majority of genes be non-differentially expressed,\n",
            "single incision laparoscopic surgery  reduces surgical trauma and adhesion through implementing a small number and/or size of portals.\n",
            "single incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopic surgery for splenectomy in dogs.\n",
            "single incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopy for dogs undergoing elective splenectomy.\n",
            "single incision lapar\n",
            "in this study, we propose a \"parallel\" approach to measure the false discovery rate  in microarray experiments.\n",
            "this approach is based on the simultaneous multiplicity model, which is derived from the total number of p-values at the significance level, e.g. α =  <dig> - m.\n",
            "in this study, we propose a \"parallel\" approach to measure the false discovery rate  in a microarray experiment.\n",
            "this method uses the total number\n",
            "in this study, eight healthy adult, non-pregnant, dry buffaloes of jaffrabadi breed which had completed 1st lactation and had an average body weight of 350–400 kg were included in the study of rumen microbial diversity.\n",
            "the microbial diversity of the jaffrabadi breed of buffaloes was studied using high-throughput next-generation sequencing technologies.\n",
            "in this study, eight healthy adult, non-pregnant, dry\n",
            "in this study, we investigated the effect of neuronal jnk activation on synaptic change in the fly central nervous system; and b) identify transcriptional targets of neuronal jnk signaling in drosophila.\n",
            "in addition, we found five genes are responsive to the progesterone-related steroid ru <dig> commonly used for temporal control of gal4-restricted transgene expression in drosophila.\n",
            "in this study, we investigated the effect of neuronal jn\n",
            "in this study, we develop a novel algorithm to derive yeast complexes from weighted  ppi networks.\n",
            "we define a core-attachment based refinement method, mcl-ca, that is able to derive higher number of yeast complexes and with better accuracies than mcl-ca;  mcl-ca is able to derive higher number of yeast complexes and with better accuracies than mcl-ca; and  mcl-ca responds well to most available affinity scoring schemes for\n",
            "micrornas  are short, endogenously expressed rna molecules that regulate gene expression by binding directly and preferably to the 3' untranslated region  of protein coding genes.\n",
            "micrornas  are short, endogenously expressed rna molecules that regulate gene expression by binding directly and preferably to the 3' untranslated region  of protein coding genes.\n",
            "micrornas  are short, endogenously expressed rna molecules that regulate gene expression by\n",
            "the aim of the present study is to identify genes expressed in b. cenocepacia biofilms and detect the regulatory elements that might be involved in biofilm formation and survival, as a prerequisite to develop new strategies in treatment of b. cenocepacia infections.\n",
            "the distribution of internal tss across the b. cenocepacia genome can be used as a further criterion to identify genomic islands in b. cenocepacia, as a prerequisite to\n",
            "in the present study, we employed a systems biology approach and report the construction of a large-scale arabidopsis immune co-expression regulatory  network.\n",
            "in the present study, we employed a systems biology approach and report the construction of a large-scale arabidopsis immune co-expression regulatory  network.\n",
            "in this study, we employed a systems biology approach and report the construction of a large-scale arabidopsis immune co-expression regulatory  network.\n",
            "\n",
            "candida esophagitis represents a severe threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\n",
            "candida albicans are known to activate a subset of pattern recognition receptors, the toll-like receptors.\n",
            "candida esophagitis represents a severe threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\n",
            "the shorebirds and allies  present an exceptional group for studying numerous evolutionary hypotheses.\n",
            "our results demonstrate that the total evidence approach is a powerful tool for the reconstruction of the deep basal nodes of charadriiformes phylogeny.\n",
            "the total evidence approach combines complete taxonomic coverage and the inclusion of branch lengths provide the basis for future comparative analyses of charadriiformes evolution.\n",
            "our results demonstrate that the total evidence approach is a powerful tool for the reconstruction of the deep basal nodes\n"
          ]
        }
      ],
      "source": [
        "gen_summary = []\n",
        "for i in test_text:\n",
        "    summary = generate_summary(i)\n",
        "    gen_summary.append(summary)\n",
        "    print(summary)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 55,
      "metadata": {},
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
            "To disable this warning, you can either:\n",
            "\t- Avoid using `tokenizers` before the fork if possible\n",
            "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: rouge-score in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (0.1.2)\n",
            "Requirement already satisfied: absl-py in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from rouge-score) (2.1.0)\n",
            "Requirement already satisfied: nltk in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from rouge-score) (3.8.1)\n",
            "Requirement already satisfied: numpy in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from rouge-score) (1.24.3)\n",
            "Requirement already satisfied: six>=1.14.0 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from rouge-score) (1.16.0)\n",
            "Requirement already satisfied: click in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from nltk->rouge-score) (8.0.4)\n",
            "Requirement already satisfied: joblib in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from nltk->rouge-score) (1.2.0)\n",
            "Requirement already satisfied: regex>=2021.8.3 in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from nltk->rouge-score) (2022.7.9)\n",
            "Requirement already satisfied: tqdm in /Users/pavandukka/anaconda3/lib/python3.11/site-packages (from nltk->rouge-score) (4.65.0)\n"
          ]
        }
      ],
      "source": [
        "!pip install rouge-score\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 58,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "[{'rouge1': Score(precision=0.20816326530612245, recall=0.7391304347826086, fmeasure=0.3248407643312101), 'rouge2': Score(precision=0.06967213114754098, recall=0.25, fmeasure=0.10897435897435896), 'rougeL': Score(precision=0.11836734693877551, recall=0.42028985507246375, fmeasure=0.18471337579617836)}, {'rouge1': Score(precision=0.25842696629213485, recall=0.2839506172839506, fmeasure=0.2705882352941177), 'rouge2': Score(precision=0.10227272727272728, recall=0.1125, fmeasure=0.10714285714285715), 'rougeL': Score(precision=0.23595505617977527, recall=0.25925925925925924, fmeasure=0.24705882352941178)}, {'rouge1': Score(precision=0.20080321285140562, recall=0.6493506493506493, fmeasure=0.3067484662576687), 'rouge2': Score(precision=0.06451612903225806, recall=0.21052631578947367, fmeasure=0.09876543209876543), 'rougeL': Score(precision=0.10040160642570281, recall=0.3246753246753247, fmeasure=0.15337423312883436)}, {'rouge1': Score(precision=0.21348314606741572, recall=0.6129032258064516, fmeasure=0.31666666666666665), 'rouge2': Score(precision=0.04519774011299435, recall=0.13114754098360656, fmeasure=0.06722689075630252), 'rougeL': Score(precision=0.10674157303370786, recall=0.3064516129032258, fmeasure=0.15833333333333333)}, {'rouge1': Score(precision=0.20710059171597633, recall=0.5932203389830508, fmeasure=0.30701754385964913), 'rouge2': Score(precision=0.05952380952380952, recall=0.1724137931034483, fmeasure=0.08849557522123892), 'rougeL': Score(precision=0.11834319526627218, recall=0.3389830508474576, fmeasure=0.17543859649122806)}, {'rouge1': Score(precision=0.0968586387434555, recall=0.8222222222222222, fmeasure=0.17330210772833726), 'rouge2': Score(precision=0.04199475065616798, recall=0.36363636363636365, fmeasure=0.07529411764705882), 'rougeL': Score(precision=0.07329842931937172, recall=0.6222222222222222, fmeasure=0.13114754098360656)}, {'rouge1': Score(precision=0.18421052631578946, recall=0.5283018867924528, fmeasure=0.2731707317073171), 'rouge2': Score(precision=0.046357615894039736, recall=0.1346153846153846, fmeasure=0.06896551724137931), 'rougeL': Score(precision=0.11842105263157894, recall=0.33962264150943394, fmeasure=0.17560975609756097)}, {'rouge1': Score(precision=0.2097902097902098, recall=0.4838709677419355, fmeasure=0.2926829268292683), 'rouge2': Score(precision=0.07042253521126761, recall=0.16393442622950818, fmeasure=0.09852216748768473), 'rougeL': Score(precision=0.13286713286713286, recall=0.3064516129032258, fmeasure=0.18536585365853658)}, {'rouge1': Score(precision=0.15546218487394958, recall=0.5362318840579711, fmeasure=0.24104234527687296), 'rouge2': Score(precision=0.046413502109704644, recall=0.16176470588235295, fmeasure=0.0721311475409836), 'rougeL': Score(precision=0.1134453781512605, recall=0.391304347826087, fmeasure=0.17589576547231273)}, {'rouge1': Score(precision=0.1070110701107011, recall=0.58, fmeasure=0.1806853582554517), 'rouge2': Score(precision=0.05555555555555555, recall=0.30612244897959184, fmeasure=0.0940438871473354), 'rougeL': Score(precision=0.08856088560885608, recall=0.48, fmeasure=0.14953271028037382)}, {'rouge1': Score(precision=0.19047619047619047, recall=0.7088607594936709, fmeasure=0.3002680965147453), 'rouge2': Score(precision=0.10580204778156997, recall=0.3974358974358974, fmeasure=0.16711590296495957), 'rougeL': Score(precision=0.10884353741496598, recall=0.4050632911392405, fmeasure=0.17158176943699732)}, {'rouge1': Score(precision=0.15759312320916904, recall=0.6962025316455697, fmeasure=0.2570093457943925), 'rouge2': Score(precision=0.04310344827586207, recall=0.19230769230769232, fmeasure=0.07042253521126761), 'rougeL': Score(precision=0.10888252148997135, recall=0.4810126582278481, fmeasure=0.17757009345794394)}, {'rouge1': Score(precision=0.2835820895522388, recall=0.6705882352941176, fmeasure=0.3986013986013986), 'rouge2': Score(precision=0.11, recall=0.2619047619047619, fmeasure=0.15492957746478875), 'rougeL': Score(precision=0.1691542288557214, recall=0.4, fmeasure=0.23776223776223776)}, {'rouge1': Score(precision=0.32592592592592595, recall=0.6285714285714286, fmeasure=0.4292682926829269), 'rouge2': Score(precision=0.15671641791044777, recall=0.30434782608695654, fmeasure=0.20689655172413796), 'rougeL': Score(precision=0.2222222222222222, recall=0.42857142857142855, fmeasure=0.29268292682926833)}, {'rouge1': Score(precision=0.21611721611721613, recall=0.7866666666666666, fmeasure=0.3390804597701149), 'rouge2': Score(precision=0.11397058823529412, recall=0.4189189189189189, fmeasure=0.1791907514450867), 'rougeL': Score(precision=0.13186813186813187, recall=0.48, fmeasure=0.20689655172413796)}, {'rouge1': Score(precision=0.1624548736462094, recall=0.7258064516129032, fmeasure=0.26548672566371684), 'rouge2': Score(precision=0.03260869565217391, recall=0.14754098360655737, fmeasure=0.05341246290801186), 'rougeL': Score(precision=0.08664259927797834, recall=0.3870967741935484, fmeasure=0.1415929203539823)}, {'rouge1': Score(precision=0.049586776859504134, recall=0.16666666666666666, fmeasure=0.07643312101910829), 'rouge2': Score(precision=0.0, recall=0.0, fmeasure=0.0), 'rougeL': Score(precision=0.0371900826446281, recall=0.125, fmeasure=0.05732484076433122)}, {'rouge1': Score(precision=0.06906906906906907, recall=0.323943661971831, fmeasure=0.11386138613861387), 'rouge2': Score(precision=0.012048192771084338, recall=0.05714285714285714, fmeasure=0.01990049751243781), 'rougeL': Score(precision=0.06006006006006006, recall=0.28169014084507044, fmeasure=0.09900990099009901)}, {'rouge1': Score(precision=0.38461538461538464, recall=0.4268292682926829, fmeasure=0.40462427745664736), 'rouge2': Score(precision=0.12222222222222222, recall=0.13580246913580246, fmeasure=0.1286549707602339), 'rougeL': Score(precision=0.25274725274725274, recall=0.2804878048780488, fmeasure=0.26589595375722547)}, {'rouge1': Score(precision=0.1308016877637131, recall=0.41333333333333333, fmeasure=0.19871794871794876), 'rouge2': Score(precision=0.012711864406779662, recall=0.04054054054054054, fmeasure=0.01935483870967742), 'rougeL': Score(precision=0.08438818565400844, recall=0.26666666666666666, fmeasure=0.1282051282051282)}, {'rouge1': Score(precision=0.16923076923076924, recall=0.7432432432432432, fmeasure=0.2756892230576441), 'rouge2': Score(precision=0.07716049382716049, recall=0.3424657534246575, fmeasure=0.12594458438287154), 'rougeL': Score(precision=0.1076923076923077, recall=0.47297297297297297, fmeasure=0.1754385964912281)}, {'rouge1': Score(precision=0.25980392156862747, recall=0.6463414634146342, fmeasure=0.3706293706293706), 'rouge2': Score(precision=0.07881773399014778, recall=0.19753086419753085, fmeasure=0.11267605633802816), 'rougeL': Score(precision=0.12254901960784313, recall=0.3048780487804878, fmeasure=0.17482517482517482)}, {'rouge1': Score(precision=0.04395604395604396, recall=0.2926829268292683, fmeasure=0.07643312101910828), 'rouge2': Score(precision=0.01838235294117647, recall=0.125, fmeasure=0.032051282051282055), 'rougeL': Score(precision=0.04395604395604396, recall=0.2926829268292683, fmeasure=0.07643312101910828)}, {'rouge1': Score(precision=0.12152777777777778, recall=0.6862745098039216, fmeasure=0.20648967551622419), 'rouge2': Score(precision=0.05226480836236934, recall=0.3, fmeasure=0.08902077151335311), 'rougeL': Score(precision=0.08680555555555555, recall=0.49019607843137253, fmeasure=0.14749262536873156)}, {'rouge1': Score(precision=0.17796610169491525, recall=0.525, fmeasure=0.26582278481012656), 'rouge2': Score(precision=0.05982905982905983, recall=0.1794871794871795, fmeasure=0.08974358974358974), 'rougeL': Score(precision=0.13559322033898305, recall=0.4, fmeasure=0.20253164556962028)}, {'rouge1': Score(precision=0.14285714285714285, recall=0.4461538461538462, fmeasure=0.21641791044776118), 'rouge2': Score(precision=0.024752475247524754, recall=0.078125, fmeasure=0.03759398496240602), 'rougeL': Score(precision=0.08866995073891626, recall=0.27692307692307694, fmeasure=0.13432835820895522)}, {'rouge1': Score(precision=0.11328125, recall=0.48333333333333334, fmeasure=0.18354430379746833), 'rouge2': Score(precision=0.023529411764705882, recall=0.1016949152542373, fmeasure=0.03821656050955414), 'rougeL': Score(precision=0.06640625, recall=0.2833333333333333, fmeasure=0.10759493670886075)}, {'rouge1': Score(precision=0.2818181818181818, recall=0.5740740740740741, fmeasure=0.3780487804878049), 'rouge2': Score(precision=0.027522935779816515, recall=0.05660377358490566, fmeasure=0.03703703703703704), 'rougeL': Score(precision=0.15454545454545454, recall=0.3148148148148148, fmeasure=0.20731707317073172)}, {'rouge1': Score(precision=0.12956810631229235, recall=0.582089552238806, fmeasure=0.21195652173913043), 'rouge2': Score(precision=0.04, recall=0.18181818181818182, fmeasure=0.06557377049180327), 'rougeL': Score(precision=0.09634551495016612, recall=0.43283582089552236, fmeasure=0.15760869565217392)}, {'rouge1': Score(precision=0.17101449275362318, recall=0.7375, fmeasure=0.2776470588235294), 'rouge2': Score(precision=0.06976744186046512, recall=0.3037974683544304, fmeasure=0.11347517730496454), 'rougeL': Score(precision=0.12173913043478261, recall=0.525, fmeasure=0.19764705882352945)}, {'rouge1': Score(precision=0.21487603305785125, recall=0.7027027027027027, fmeasure=0.3291139240506329), 'rouge2': Score(precision=0.08713692946058091, recall=0.2876712328767123, fmeasure=0.1337579617834395), 'rougeL': Score(precision=0.1322314049586777, recall=0.43243243243243246, fmeasure=0.20253164556962025)}, {'rouge1': Score(precision=0.1977401129943503, recall=0.5303030303030303, fmeasure=0.2880658436213992), 'rouge2': Score(precision=0.03409090909090909, recall=0.09230769230769231, fmeasure=0.04979253112033195), 'rougeL': Score(precision=0.1016949152542373, recall=0.2727272727272727, fmeasure=0.14814814814814817)}, {'rouge1': Score(precision=0.1031390134529148, recall=0.3333333333333333, fmeasure=0.15753424657534246), 'rouge2': Score(precision=0.02252252252252252, recall=0.07352941176470588, fmeasure=0.034482758620689655), 'rougeL': Score(precision=0.05829596412556054, recall=0.18840579710144928, fmeasure=0.08904109589041097)}, {'rouge1': Score(precision=0.25821596244131456, recall=0.6790123456790124, fmeasure=0.3741496598639456), 'rouge2': Score(precision=0.07547169811320754, recall=0.2, fmeasure=0.10958904109589042), 'rougeL': Score(precision=0.14084507042253522, recall=0.37037037037037035, fmeasure=0.2040816326530612)}, {'rouge1': Score(precision=0.17272727272727273, recall=0.4634146341463415, fmeasure=0.2516556291390728), 'rouge2': Score(precision=0.05504587155963303, recall=0.15, fmeasure=0.08053691275167786), 'rougeL': Score(precision=0.11818181818181818, recall=0.3170731707317073, fmeasure=0.17218543046357618)}, {'rouge1': Score(precision=0.17142857142857143, recall=0.7619047619047619, fmeasure=0.2798833819241982), 'rouge2': Score(precision=0.10752688172043011, recall=0.4838709677419355, fmeasure=0.17595307917888564), 'rougeL': Score(precision=0.11785714285714285, recall=0.5238095238095238, fmeasure=0.1924198250728863)}, {'rouge1': Score(precision=0.19583333333333333, recall=0.6103896103896104, fmeasure=0.2965299684542587), 'rouge2': Score(precision=0.04602510460251046, recall=0.14473684210526316, fmeasure=0.06984126984126984), 'rougeL': Score(precision=0.125, recall=0.38961038961038963, fmeasure=0.1892744479495268)}, {'rouge1': Score(precision=0.1235632183908046, recall=0.7166666666666667, fmeasure=0.2107843137254902), 'rouge2': Score(precision=0.069164265129683, recall=0.4067796610169492, fmeasure=0.11822660098522168), 'rougeL': Score(precision=0.09482758620689655, recall=0.55, fmeasure=0.16176470588235292)}, {'rouge1': Score(precision=0.26339285714285715, recall=0.8939393939393939, fmeasure=0.4068965517241379), 'rouge2': Score(precision=0.1210762331838565, recall=0.4153846153846154, fmeasure=0.18750000000000003), 'rougeL': Score(precision=0.15625, recall=0.5303030303030303, fmeasure=0.2413793103448276)}, {'rouge1': Score(precision=0.15037593984962405, recall=0.26666666666666666, fmeasure=0.1923076923076923), 'rouge2': Score(precision=0.0, recall=0.0, fmeasure=0.0), 'rougeL': Score(precision=0.10526315789473684, recall=0.18666666666666668, fmeasure=0.13461538461538464)}, {'rouge1': Score(precision=0.32, recall=0.5797101449275363, fmeasure=0.4123711340206186), 'rouge2': Score(precision=0.12096774193548387, recall=0.22058823529411764, fmeasure=0.15625), 'rougeL': Score(precision=0.208, recall=0.37681159420289856, fmeasure=0.26804123711340205)}, {'rouge1': Score(precision=0.1423076923076923, recall=0.5967741935483871, fmeasure=0.22981366459627325), 'rouge2': Score(precision=0.03474903474903475, recall=0.14754098360655737, fmeasure=0.05625), 'rougeL': Score(precision=0.08076923076923077, recall=0.3387096774193548, fmeasure=0.13043478260869568)}, {'rouge1': Score(precision=0.0761904761904762, recall=0.5714285714285714, fmeasure=0.13445378151260506), 'rouge2': Score(precision=0.01592356687898089, recall=0.12195121951219512, fmeasure=0.028169014084507043), 'rougeL': Score(precision=0.05396825396825397, recall=0.40476190476190477, fmeasure=0.09523809523809525)}, {'rouge1': Score(precision=0.1834319526627219, recall=0.5254237288135594, fmeasure=0.2719298245614035), 'rouge2': Score(precision=0.041666666666666664, recall=0.1206896551724138, fmeasure=0.06194690265486725), 'rougeL': Score(precision=0.11242603550295859, recall=0.3220338983050847, fmeasure=0.16666666666666669)}, {'rouge1': Score(precision=0.15555555555555556, recall=0.4268292682926829, fmeasure=0.2280130293159609), 'rouge2': Score(precision=0.03571428571428571, recall=0.09876543209876543, fmeasure=0.05245901639344263), 'rougeL': Score(precision=0.12, recall=0.32926829268292684, fmeasure=0.1758957654723127)}, {'rouge1': Score(precision=0.25125628140703515, recall=0.684931506849315, fmeasure=0.3676470588235294), 'rouge2': Score(precision=0.09595959595959595, recall=0.2638888888888889, fmeasure=0.14074074074074072), 'rougeL': Score(precision=0.1507537688442211, recall=0.410958904109589, fmeasure=0.2205882352941176)}, {'rouge1': Score(precision=0.2096069868995633, recall=0.7058823529411765, fmeasure=0.32323232323232326), 'rouge2': Score(precision=0.06140350877192982, recall=0.208955223880597, fmeasure=0.09491525423728812), 'rougeL': Score(precision=0.11353711790393013, recall=0.38235294117647056, fmeasure=0.1750841750841751)}, {'rouge1': Score(precision=0.15483870967741936, recall=0.7164179104477612, fmeasure=0.2546419098143236), 'rouge2': Score(precision=0.05825242718446602, recall=0.2727272727272727, fmeasure=0.096), 'rougeL': Score(precision=0.1032258064516129, recall=0.47761194029850745, fmeasure=0.1697612732095491)}, {'rouge1': Score(precision=0.3313953488372093, recall=0.7125, fmeasure=0.45238095238095233), 'rouge2': Score(precision=0.10526315789473684, recall=0.22784810126582278, fmeasure=0.144), 'rougeL': Score(precision=0.1686046511627907, recall=0.3625, fmeasure=0.23015873015873015)}, {'rouge1': Score(precision=0.20074349442379183, recall=0.782608695652174, fmeasure=0.31952662721893493), 'rouge2': Score(precision=0.08955223880597014, recall=0.35294117647058826, fmeasure=0.14285714285714288), 'rougeL': Score(precision=0.12639405204460966, recall=0.4927536231884058, fmeasure=0.2011834319526627)}, {'rouge1': Score(precision=0.23963133640552994, recall=0.7428571428571429, fmeasure=0.3623693379790941), 'rouge2': Score(precision=0.06481481481481481, recall=0.2028985507246377, fmeasure=0.0982456140350877), 'rougeL': Score(precision=0.12442396313364056, recall=0.38571428571428573, fmeasure=0.18815331010452963)}, {'rouge1': Score(precision=0.13671875, recall=0.5645161290322581, fmeasure=0.22012578616352202), 'rouge2': Score(precision=0.0392156862745098, recall=0.16393442622950818, fmeasure=0.06329113924050633), 'rougeL': Score(precision=0.07421875, recall=0.3064516129032258, fmeasure=0.11949685534591194)}, {'rouge1': Score(precision=0.14285714285714285, recall=0.4838709677419355, fmeasure=0.22058823529411764), 'rouge2': Score(precision=0.03349282296650718, recall=0.11475409836065574, fmeasure=0.05185185185185186), 'rougeL': Score(precision=0.07142857142857142, recall=0.24193548387096775, fmeasure=0.11029411764705882)}, {'rouge1': Score(precision=0.16279069767441862, recall=0.875, fmeasure=0.2745098039215686), 'rouge2': Score(precision=0.07333333333333333, recall=0.4, fmeasure=0.12394366197183099), 'rougeL': Score(precision=0.09966777408637874, recall=0.5357142857142857, fmeasure=0.1680672268907563)}, {'rouge1': Score(precision=0.23369565217391305, recall=0.581081081081081, fmeasure=0.3333333333333333), 'rouge2': Score(precision=0.08196721311475409, recall=0.2054794520547945, fmeasure=0.11718749999999999), 'rougeL': Score(precision=0.14673913043478262, recall=0.36486486486486486, fmeasure=0.20930232558139533)}, {'rouge1': Score(precision=0.13559322033898305, recall=0.5217391304347826, fmeasure=0.21524663677130046), 'rouge2': Score(precision=0.05113636363636364, recall=0.2, fmeasure=0.08144796380090498), 'rougeL': Score(precision=0.1016949152542373, recall=0.391304347826087, fmeasure=0.16143497757847533)}, {'rouge1': Score(precision=0.14457831325301204, recall=0.7058823529411765, fmeasure=0.24), 'rouge2': Score(precision=0.04229607250755287, recall=0.208955223880597, fmeasure=0.07035175879396986), 'rougeL': Score(precision=0.0963855421686747, recall=0.47058823529411764, fmeasure=0.16)}, {'rouge1': Score(precision=0.14331210191082802, recall=0.6521739130434783, fmeasure=0.2349869451697128), 'rouge2': Score(precision=0.054313099041533544, recall=0.25, fmeasure=0.08923884514435695), 'rougeL': Score(precision=0.09554140127388536, recall=0.43478260869565216, fmeasure=0.1566579634464752)}, {'rouge1': Score(precision=0.2803030303030303, recall=0.5441176470588235, fmeasure=0.37), 'rouge2': Score(precision=0.06870229007633588, recall=0.13432835820895522, fmeasure=0.09090909090909091), 'rougeL': Score(precision=0.12878787878787878, recall=0.25, fmeasure=0.16999999999999998)}, {'rouge1': Score(precision=0.1346938775510204, recall=0.55, fmeasure=0.21639344262295082), 'rouge2': Score(precision=0.040983606557377046, recall=0.1694915254237288, fmeasure=0.066006600660066), 'rougeL': Score(precision=0.07755102040816327, recall=0.31666666666666665, fmeasure=0.12459016393442623)}, {'rouge1': Score(precision=0.23529411764705882, recall=0.7457627118644068, fmeasure=0.3577235772357723), 'rouge2': Score(precision=0.08064516129032258, recall=0.25862068965517243, fmeasure=0.12295081967213116), 'rougeL': Score(precision=0.13903743315508021, recall=0.4406779661016949, fmeasure=0.2113821138211382)}, {'rouge1': Score(precision=0.27631578947368424, recall=0.2625, fmeasure=0.2692307692307693), 'rouge2': Score(precision=0.013333333333333334, recall=0.012658227848101266, fmeasure=0.012987012987012988), 'rougeL': Score(precision=0.18421052631578946, recall=0.175, fmeasure=0.1794871794871795)}, {'rouge1': Score(precision=0.15467625899280577, recall=0.9148936170212766, fmeasure=0.2646153846153846), 'rouge2': Score(precision=0.07581227436823104, recall=0.45652173913043476, fmeasure=0.13003095975232196), 'rougeL': Score(precision=0.1079136690647482, recall=0.6382978723404256, fmeasure=0.1846153846153846)}, {'rouge1': Score(precision=0.19218241042345277, recall=0.8939393939393939, fmeasure=0.31635388739946385), 'rouge2': Score(precision=0.16013071895424835, recall=0.7538461538461538, fmeasure=0.2641509433962264), 'rougeL': Score(precision=0.18241042345276873, recall=0.8484848484848485, fmeasure=0.30026809651474534)}, {'rouge1': Score(precision=0.25089605734767023, recall=0.8860759493670886, fmeasure=0.3910614525139664), 'rouge2': Score(precision=0.14028776978417265, recall=0.5, fmeasure=0.2191011235955056), 'rougeL': Score(precision=0.17921146953405018, recall=0.6329113924050633, fmeasure=0.27932960893854747)}, {'rouge1': Score(precision=0.211864406779661, recall=0.7352941176470589, fmeasure=0.32894736842105265), 'rouge2': Score(precision=0.07234042553191489, recall=0.2537313432835821, fmeasure=0.11258278145695363), 'rougeL': Score(precision=0.1271186440677966, recall=0.4411764705882353, fmeasure=0.19736842105263158)}, {'rouge1': Score(precision=0.30434782608695654, recall=0.6140350877192983, fmeasure=0.4069767441860465), 'rouge2': Score(precision=0.10526315789473684, recall=0.21428571428571427, fmeasure=0.1411764705882353), 'rougeL': Score(precision=0.2, recall=0.40350877192982454, fmeasure=0.2674418604651163)}, {'rouge1': Score(precision=0.17105263157894737, recall=0.9122807017543859, fmeasure=0.2880886426592798), 'rouge2': Score(precision=0.10891089108910891, recall=0.5892857142857143, fmeasure=0.18384401114206128), 'rougeL': Score(precision=0.11513157894736842, recall=0.6140350877192983, fmeasure=0.19390581717451522)}, {'rouge1': Score(precision=0.15047021943573669, recall=0.631578947368421, fmeasure=0.2430379746835443), 'rouge2': Score(precision=0.031446540880503145, recall=0.13333333333333333, fmeasure=0.050890585241730284), 'rougeL': Score(precision=0.07836990595611286, recall=0.32894736842105265, fmeasure=0.12658227848101267)}, {'rouge1': Score(precision=0.2127659574468085, recall=0.7352941176470589, fmeasure=0.33003300330033003), 'rouge2': Score(precision=0.05128205128205128, recall=0.1791044776119403, fmeasure=0.07973421926910297), 'rougeL': Score(precision=0.11914893617021277, recall=0.4117647058823529, fmeasure=0.1848184818481848)}, {'rouge1': Score(precision=0.13225806451612904, recall=0.7068965517241379, fmeasure=0.22282608695652173), 'rouge2': Score(precision=0.042071197411003236, recall=0.22807017543859648, fmeasure=0.07103825136612023), 'rougeL': Score(precision=0.0935483870967742, recall=0.5, fmeasure=0.15760869565217392)}, {'rouge1': Score(precision=0.14204545454545456, recall=0.5681818181818182, fmeasure=0.2272727272727273), 'rouge2': Score(precision=0.042735042735042736, recall=0.1724137931034483, fmeasure=0.0684931506849315), 'rougeL': Score(precision=0.08806818181818182, recall=0.3522727272727273, fmeasure=0.14090909090909093)}, {'rouge1': Score(precision=0.14423076923076922, recall=0.4225352112676056, fmeasure=0.2150537634408602), 'rouge2': Score(precision=0.03864734299516908, recall=0.11428571428571428, fmeasure=0.057761732851985555), 'rougeL': Score(precision=0.08173076923076923, recall=0.23943661971830985, fmeasure=0.12186379928315412)}, {'rouge1': Score(precision=0.17222222222222222, recall=0.47692307692307695, fmeasure=0.25306122448979596), 'rouge2': Score(precision=0.03910614525139665, recall=0.109375, fmeasure=0.05761316872427983), 'rougeL': Score(precision=0.12222222222222222, recall=0.3384615384615385, fmeasure=0.1795918367346939)}, {'rouge1': Score(precision=0.1597222222222222, recall=0.6666666666666666, fmeasure=0.25770308123249297), 'rouge2': Score(precision=0.03484320557491289, recall=0.14705882352941177, fmeasure=0.05633802816901408), 'rougeL': Score(precision=0.09027777777777778, recall=0.37681159420289856, fmeasure=0.1456582633053221)}, {'rouge1': Score(precision=0.25882352941176473, recall=0.5714285714285714, fmeasure=0.35627530364372473), 'rouge2': Score(precision=0.0650887573964497, recall=0.14473684210526316, fmeasure=0.08979591836734693), 'rougeL': Score(precision=0.15294117647058825, recall=0.33766233766233766, fmeasure=0.2105263157894737)}, {'rouge1': Score(precision=0.29069767441860467, recall=0.6024096385542169, fmeasure=0.3921568627450981), 'rouge2': Score(precision=0.06432748538011696, recall=0.13414634146341464, fmeasure=0.08695652173913043), 'rougeL': Score(precision=0.18023255813953487, recall=0.37349397590361444, fmeasure=0.24313725490196078)}, {'rouge1': Score(precision=0.2052980132450331, recall=0.8378378378378378, fmeasure=0.32978723404255317), 'rouge2': Score(precision=0.132890365448505, recall=0.547945205479452, fmeasure=0.21390374331550804), 'rougeL': Score(precision=0.12582781456953643, recall=0.5135135135135135, fmeasure=0.20212765957446813)}, {'rouge1': Score(precision=0.19318181818181818, recall=0.21518987341772153, fmeasure=0.20359281437125748), 'rouge2': Score(precision=0.011494252873563218, recall=0.01282051282051282, fmeasure=0.012121212121212121), 'rougeL': Score(precision=0.125, recall=0.13924050632911392, fmeasure=0.1317365269461078)}, {'rouge1': Score(precision=0.16666666666666666, recall=0.4642857142857143, fmeasure=0.24528301886792456), 'rouge2': Score(precision=0.055793991416309016, recall=0.1566265060240964, fmeasure=0.08227848101265824), 'rougeL': Score(precision=0.10256410256410256, recall=0.2857142857142857, fmeasure=0.1509433962264151)}, {'rouge1': Score(precision=0.24166666666666667, recall=0.8405797101449275, fmeasure=0.37540453074433655), 'rouge2': Score(precision=0.14225941422594143, recall=0.5, fmeasure=0.2214983713355049), 'rougeL': Score(precision=0.15, recall=0.5217391304347826, fmeasure=0.23300970873786406)}, {'rouge1': Score(precision=0.11598746081504702, recall=0.6981132075471698, fmeasure=0.1989247311827957), 'rouge2': Score(precision=0.05660377358490566, recall=0.34615384615384615, fmeasure=0.0972972972972973), 'rougeL': Score(precision=0.09717868338557993, recall=0.5849056603773585, fmeasure=0.16666666666666666)}, {'rouge1': Score(precision=0.0521978021978022, recall=0.3877551020408163, fmeasure=0.09200968523002423), 'rouge2': Score(precision=0.011019283746556474, recall=0.08333333333333333, fmeasure=0.019464720194647202), 'rougeL': Score(precision=0.04120879120879121, recall=0.30612244897959184, fmeasure=0.07263922518159806)}, {'rouge1': Score(precision=0.1222707423580786, recall=0.4307692307692308, fmeasure=0.19047619047619047), 'rouge2': Score(precision=0.021929824561403508, recall=0.078125, fmeasure=0.03424657534246575), 'rougeL': Score(precision=0.07860262008733625, recall=0.27692307692307694, fmeasure=0.12244897959183673)}, {'rouge1': Score(precision=0.13357400722021662, recall=0.7115384615384616, fmeasure=0.22492401215805471), 'rouge2': Score(precision=0.06521739130434782, recall=0.35294117647058826, fmeasure=0.11009174311926605), 'rougeL': Score(precision=0.10108303249097472, recall=0.5384615384615384, fmeasure=0.1702127659574468)}, {'rouge1': Score(precision=0.15158924205378974, recall=0.7380952380952381, fmeasure=0.2515212981744422), 'rouge2': Score(precision=0.014705882352941176, recall=0.07228915662650602, fmeasure=0.024439918533604887), 'rougeL': Score(precision=0.08312958435207823, recall=0.40476190476190477, fmeasure=0.1379310344827586)}, {'rouge1': Score(precision=0.31724137931034485, recall=0.5679012345679012, fmeasure=0.40707964601769914), 'rouge2': Score(precision=0.08333333333333333, recall=0.15, fmeasure=0.10714285714285714), 'rougeL': Score(precision=0.21379310344827587, recall=0.38271604938271603, fmeasure=0.2743362831858407)}, {'rouge1': Score(precision=0.21982758620689655, recall=0.68, fmeasure=0.3322475570032573), 'rouge2': Score(precision=0.06493506493506493, recall=0.20270270270270271, fmeasure=0.09836065573770492), 'rougeL': Score(precision=0.1206896551724138, recall=0.37333333333333335, fmeasure=0.18241042345276873)}, {'rouge1': Score(precision=0.11890243902439024, recall=0.65, fmeasure=0.20103092783505155), 'rouge2': Score(precision=0.03669724770642202, recall=0.2033898305084746, fmeasure=0.062176165803108814), 'rougeL': Score(precision=0.07317073170731707, recall=0.4, fmeasure=0.12371134020618556)}, {'rouge1': Score(precision=0.3431372549019608, recall=0.4605263157894737, fmeasure=0.3932584269662921), 'rouge2': Score(precision=0.10891089108910891, recall=0.14666666666666667, fmeasure=0.125), 'rougeL': Score(precision=0.22549019607843138, recall=0.3026315789473684, fmeasure=0.2584269662921348)}, {'rouge1': Score(precision=0.23846153846153847, recall=0.5, fmeasure=0.3229166666666667), 'rouge2': Score(precision=0.06201550387596899, recall=0.13114754098360656, fmeasure=0.08421052631578947), 'rougeL': Score(precision=0.15384615384615385, recall=0.3225806451612903, fmeasure=0.20833333333333334)}, {'rouge1': Score(precision=0.16666666666666666, recall=0.6617647058823529, fmeasure=0.26627218934911245), 'rouge2': Score(precision=0.055762081784386616, recall=0.22388059701492538, fmeasure=0.08928571428571429), 'rougeL': Score(precision=0.1037037037037037, recall=0.4117647058823529, fmeasure=0.16568047337278105)}, {'rouge1': Score(precision=0.1271186440677966, recall=0.46875, fmeasure=0.19999999999999998), 'rouge2': Score(precision=0.02127659574468085, recall=0.07936507936507936, fmeasure=0.03355704697986577), 'rougeL': Score(precision=0.06779661016949153, recall=0.25, fmeasure=0.10666666666666667)}, {'rouge1': Score(precision=0.19008264462809918, recall=0.6764705882352942, fmeasure=0.2967741935483871), 'rouge2': Score(precision=0.05394190871369295, recall=0.19402985074626866, fmeasure=0.08441558441558442), 'rougeL': Score(precision=0.1115702479338843, recall=0.39705882352941174, fmeasure=0.17419354838709677)}, {'rouge1': Score(precision=0.15748031496062992, recall=0.821917808219178, fmeasure=0.2643171806167401), 'rouge2': Score(precision=0.08157894736842106, recall=0.4305555555555556, fmeasure=0.13716814159292035), 'rougeL': Score(precision=0.09973753280839895, recall=0.5205479452054794, fmeasure=0.16740088105726875)}, {'rouge1': Score(precision=0.2072072072072072, recall=0.3709677419354839, fmeasure=0.26589595375722547), 'rouge2': Score(precision=0.05454545454545454, recall=0.09836065573770492, fmeasure=0.07017543859649124), 'rougeL': Score(precision=0.14414414414414414, recall=0.25806451612903225, fmeasure=0.18497109826589597)}, {'rouge1': Score(precision=0.23076923076923078, recall=0.7285714285714285, fmeasure=0.3505154639175258), 'rouge2': Score(precision=0.11363636363636363, recall=0.36231884057971014, fmeasure=0.17301038062283736), 'rougeL': Score(precision=0.12217194570135746, recall=0.38571428571428573, fmeasure=0.18556701030927836)}, {'rouge1': Score(precision=0.22380952380952382, recall=0.6619718309859155, fmeasure=0.3345195729537367), 'rouge2': Score(precision=0.07177033492822966, recall=0.21428571428571427, fmeasure=0.10752688172043011), 'rougeL': Score(precision=0.13333333333333333, recall=0.39436619718309857, fmeasure=0.199288256227758)}, {'rouge1': Score(precision=0.17261904761904762, recall=0.4393939393939394, fmeasure=0.24786324786324784), 'rouge2': Score(precision=0.005988023952095809, recall=0.015384615384615385, fmeasure=0.008620689655172415), 'rougeL': Score(precision=0.08333333333333333, recall=0.21212121212121213, fmeasure=0.11965811965811965)}, {'rouge1': Score(precision=0.27972027972027974, recall=0.5, fmeasure=0.35874439461883406), 'rouge2': Score(precision=0.056338028169014086, recall=0.10126582278481013, fmeasure=0.07239819004524888), 'rougeL': Score(precision=0.15384615384615385, recall=0.275, fmeasure=0.19730941704035873)}]\n"
          ]
        }
      ],
      "source": [
        "from rouge_score import rouge_scorer\n",
        "scores = []\n",
        "# Initialize a Rouge scorer\n",
        "scorer = rouge_scorer.RougeScorer(['rouge1', 'rouge2', 'rougeL'], use_stemmer=True)\n",
        "\n",
        "for i in range(len(gen_summary)):\n",
        "    scores.append(scorer.score(gen_summary[i],val_summary[i]))\n",
        "print(scores)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 61,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "ROUGE-1 Mean Scores: {'precision': 0.18888690661058077, 'recall': 0.604345182108172, 'fmeasure': 0.27875036777401224}\n",
            "ROUGE-2 Mean Scores: {'precision': 0.06159820090090915, 'recall': 0.2127420660184897, 'fmeasure': 0.092803511412365}\n",
            "ROUGE-L Mean Scores: {'precision': 0.11850062983307419, 'recall': 0.3838589422134849, 'fmeasure': 0.17492864837603045}\n"
          ]
        }
      ],
      "source": [
        "import numpy as np\n",
        "\n",
        "# Initialize dictionaries to store total scores and count\n",
        "# Assume `data` is your list of dictionaries containing ROUGE scores\n",
        "\n",
        "# Initialize total dictionaries\n",
        "rouge1_totals = {'precision': 0, 'recall': 0, 'fmeasure': 0}\n",
        "rouge2_totals = {'precision': 0, 'recall': 0, 'fmeasure': 0}\n",
        "rougeL_totals = {'precision': 0, 'recall': 0, 'fmeasure': 0}\n",
        "\n",
        "# Sum up all scores\n",
        "for entry in scores:\n",
        "    rouge1_totals['precision'] += entry['rouge1'].precision\n",
        "    rouge1_totals['recall'] += entry['rouge1'].recall\n",
        "    rouge1_totals['fmeasure'] += entry['rouge1'].fmeasure\n",
        "    rouge2_totals['precision'] += entry['rouge2'].precision\n",
        "    rouge2_totals['recall'] += entry['rouge2'].recall\n",
        "    rouge2_totals['fmeasure'] += entry['rouge2'].fmeasure\n",
        "    rougeL_totals['precision'] += entry['rougeL'].precision\n",
        "    rougeL_totals['recall'] += entry['rougeL'].recall\n",
        "    rougeL_totals['fmeasure'] += entry['rougeL'].fmeasure\n",
        "\n",
        "# Calculate the number of entries\n",
        "num_entries = len(scores)\n",
        "\n",
        "# Calculate mean scores\n",
        "rouge1_means = {key: value / num_entries for key, value in rouge1_totals.items()}\n",
        "rouge2_means = {key: value / num_entries for key, value in rouge2_totals.items()}\n",
        "rougeL_means = {key: value / num_entries for key, value in rougeL_totals.items()}\n",
        "\n",
        "# Print the results\n",
        "print(\"ROUGE-1 Mean Scores:\", rouge1_means)\n",
        "print(\"ROUGE-2 Mean Scores:\", rouge2_means)\n",
        "print(\"ROUGE-L Mean Scores:\", rougeL_means)\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {},
      "outputs": [],
      "source": [
        "import os\n",
        "from datasets import Dataset\n",
        "import pandas as pd\n",
        "import re\n",
        "from transformers import GPT2Tokenizer, GPT2LMHeadModel, GPT2Config\n",
        "from transformers import TextDataset, DataCollatorForLanguageModeling\n",
        "from transformers import Trainer, TrainingArguments\n",
        "from transformers import pipeline\n",
        "from transformers import AutoTokenizer, AutoModelForSeq2SeqLM"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Cloning into 'sumpubmed'...\n",
            "remote: Enumerating objects: 130808, done.\u001b[K\n",
            "remote: Counting objects: 100% (44/44), done.\u001b[K\n",
            "remote: Compressing objects: 100% (38/38), done.\u001b[K\n",
            "remote: Total 130808 (delta 22), reused 17 (delta 6), pack-reused 130764\u001b[K\n",
            "Receiving objects: 100% (130808/130808), 346.82 MiB | 7.43 MiB/s, done.\n",
            "Resolving deltas: 100% (38368/38368), done.\n",
            "Updating files: 100% (130765/130765), done.\n"
          ]
        }
      ],
      "source": [
        "!git clone https://github.com/vgupta123/sumpubmed"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {},
      "outputs": [],
      "source": [
        "def load_text_files(directory):\n",
        "    files = os.listdir(directory)\n",
        "    files = [file for file in files if not file.startswith('.DS_Store')]\n",
        "    files = sorted(files, key=lambda x: int(re.split('\\.|_', x)[1]))\n",
        "    data = []\n",
        "    for file in files:\n",
        "        if file.endswith('.txt'):\n",
        "            with open(os.path.join(directory, file), 'r', encoding='utf-8',errors='ignore') as f:\n",
        "                text = f.read()\n",
        "                data.append(text)\n",
        "    return data"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 5,
      "metadata": {},
      "outputs": [],
      "source": [
        "# Load text files from directories\n",
        "original_text = load_text_files('sumpubmed/text')\n",
        "summaries = load_text_files('sumpubmed/shorter_abstract')\n",
        "# Create a dataset from the loaded text files\n",
        "data = {'original_text': original_text, 'summary': summaries}"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 6,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "(32689, 2)\n"
          ]
        }
      ],
      "source": [
        "# Create a dataset from the loaded text files\n",
        "df =pd.DataFrame({'original_text': original_text, 'summary': summaries})\n",
        "\n",
        "# dataset = Dataset.from_dict(data)\n",
        "\n",
        "print(df.shape)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 7,
      "metadata": {},
      "outputs": [],
      "source": [
        "import re\n",
        "\n",
        "def remove_words_followed_by_newline(input_string):\n",
        "    # Words to remove if followed by a newline character\n",
        "    words_to_remove = ['BACKGROUND', 'RESULTS', 'CONCLUSIONS', 'electronic supplementary material', 'keywords']\n",
        "\n",
        "    # Create a pattern to match the words followed by a newline\n",
        "    pattern = re.compile(r'(' + '|'.join(words_to_remove) + r')(?:\\n)')\n",
        "\n",
        "    # Remove the specified words followed by a newline character\n",
        "    cleaned_string = re.sub(pattern, '', input_string)\n",
        "\n",
        "    return cleaned_string"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 8,
      "metadata": {},
      "outputs": [],
      "source": [
        "def get_triples(doc):\n",
        "  triples = list(textacy.extract.subject_verb_object_triples(doc))\n",
        "  #print(triples)\n",
        "  return triples"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 9,
      "metadata": {},
      "outputs": [],
      "source": [
        "def get_nodes_relations(triples):\n",
        "  nodes = []\n",
        "  relations = []\n",
        "# iterate over the triples\n",
        "  for triple in triples:\n",
        "    # extract the Subject and Object from triple\n",
        "    node_subject = \"_\".join(map(str, triple.subject))\n",
        "    node_object  = \"_\".join(map(str, triple.object))\n",
        "    nodes.append(node_subject)\n",
        "    nodes.append(node_object)\n",
        "    # extract the relation between S and O\n",
        "    # add the attribute 'action' to the relation\n",
        "    relation = \"_\".join(map(str, triple.verb))\n",
        "    relations.append((node_subject,node_object,{'action':relation}))\n",
        "# remove duplicate nodes\n",
        "  nodes = list(set(nodes))\n",
        "  #print(nodes)\n",
        "# ['to_extract_SVO', 'I']\n",
        "  #print(relations)\n",
        "# [('I', 'to_extract_SVO', {'action': 'am_going'})]\n",
        "  return (nodes, relations)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 10,
      "metadata": {},
      "outputs": [],
      "source": [
        "def get_graph(nodes,relations):\n",
        "  G = nx.DiGraph()\n",
        "  G.add_nodes_from(nodes)\n",
        "  G.add_edges_from(relations)\n",
        "  return G"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 11,
      "metadata": {},
      "outputs": [],
      "source": [
        "def draw_graph(G):\n",
        "# extract the attribute 'action' from edges\n",
        "  edge_attribute = nx.get_edge_attributes(G, 'action')\n",
        "  edges, weights = zip(*edge_attribute.items())\n",
        "# resize figure\n",
        "  plt.rcParams[\"figure.figsize\"] = [5, 2]\n",
        "  plt.rcParams[\"figure.autolayout\"] = True\n",
        "# set figure layout\n",
        "  pos = nx.circular_layout(G)\n",
        "# draw graph\n",
        "  nx.draw(G, pos, node_color='b', width=2, with_labels=True)\n",
        "# draw edge attributes\n",
        "  nx.draw_networkx_edge_labels(G, pos,edge_attribute, label_pos=0.75 )\n",
        "  plt.show()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 15,
      "metadata": {},
      "outputs": [],
      "source": [
        "from datasets import Dataset\n",
        "dataset = Dataset.from_pandas(df[:1000])"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 21,
      "metadata": {},
      "outputs": [],
      "source": [
        "from transformers import AutoModelForSeq2SeqLM\n",
        "\n",
        "# Load the model\n",
        "model = AutoModelForSeq2SeqLM.from_pretrained('GraphembeddedModel')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 23,
      "metadata": {},
      "outputs": [],
      "source": [
        "from transformers import AutoTokenizer\n",
        "\n",
        "# Load the tokenizer\n",
        "tokenizer = AutoTokenizer.from_pretrained('GraphembeddedModel')\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 29,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "100\n"
          ]
        }
      ],
      "source": [
        "# Example usage\n",
        "test_text = load_text_files('sumpubmed-master/text')[1000:1100]\n",
        "val_summary = load_text_files('sumpubmed-master/shorter_abstract')[1000:1100]\n",
        "print(len(test_text))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 32,
      "metadata": {},
      "outputs": [
        {
          "data": {
            "text/plain": [
              "'BACKGROUND\\nthe pig  has been used as model animal in many fields, including biological  <cit> , agricultural, and biomedical studies  <cit> . pig litter size, which can be impacted by multiple factors including ovulation rate, embryonic survival, uterine capacity and fetal survival, is one of the most important economic traits affecting production efficiency in the pig industry  <cit> . improvements in litter size across the swine industry in recent decades has occurred through different selection schemes such as phenotypic selection, family index selection, blup-based selection, component trait selection, and hyperprolific selection  <cit> . however, despite current improvements, the low heritability of litter size has limited the genetic selection progress in substantially improving this trait  <cit> . improvement of litter size is ideally suited for application of molecular breeding approaches such as marker-assisted selection  and whole-genome selection, where molecular markers associated with superior trait values may accelerate genetic improvement. in addition, numerous studies have attempted to detect candidate genes underlying the molecular mechanism of sow prolificacy. although some candidate genes for sow prolificacy have been reported  <cit> , true causal genes remain scant. a deeper understanding of the physiological and molecular mechanisms associated with reproductive efficiency is needed to further the biological underpinnings that contribute to controlling litter size in pigs.\\n\\nit is now well established that prenatal mortality can be a major determinant of litter size in pigs  <cit> . when early conceptuses undergo rapid differentiation and expansion of their trophoblastic membranes during the first few weeks of gestation, abnormalities of embryogenesis that produce embryonic mortality can occur  <cit> . the extent and timing of embryonic and fetal losses during gestation has been well documented by previous studies, and it is estimated that about 25-45% of fertilized ova do not survive through gestation  <cit> . the majority of embryonic loss has been demonstrated to occur before day  <dig> of gestation, primarily around day 11– <dig>  <cit> . early events in conceptus development, including rapid trophoblastic elongation and successful establishment of the fetal: maternal interface, play a key part in determining litter size. the conceptus itself undergoes a dramatic transcriptomic reorganization during trophoblastic elongation and during initial interactions with the uterine endometrium  <cit> . additionally, the copious amounts of estrogen synthesized and released by the elongating conceptus serves as the maternal recognition of pregnancy signal around day  <dig> of gestation  <cit> . the timing of endometrial exposure to estrogen is critical as specific molecular cues associated with uterine receptivity occur simultaneously with conceptus estrogen synthesis and abnormal exposure to estrogen prior to day  <dig> results in compromised pregnancy establishment  <cit> .\\n\\nstudies have revealed that there are very different patterns of gene expression in uterine endometrium between successful and abortive embryo implantation, as well as between high and low litter size breeds  <cit> . it is now clear that the molecular events happening in the maternal endometrium during the early stages play a crucial role in determining the success of embryonic survival, and the early-stage endometrium provides an ideal window to investigate the molecular mechanisms that may be associated early conceptus loss and sow reproductive prolificacy. many genes involved in angiogenesis and immune responses are expressed at the endometrium-fetal surface  <cit> . interestingly, polymorphisms in genes expressed during embryonic implantation, such as in epha <dig> and tgf-β, have been associated with litter size  <cit> . however, what is lacking is a systematic comparison investigating global gene expression patterns in the uterine endometrium during the peri-implantation stage of pregnancy between pig breeds with established differences in litter size.\\n\\nmicroarrays have been widely used to detect differentially expressed genes hypothesized to contribute to controlling litter size. some interesting genes, such as steroidogenic acute regulatory protein and interleukin-1beta , were found to be expressed in during early development using multiple global expression profiling approaches  <cit> . differentially expressed genes have been detected in endometrium between pregnant and non-pregnant sows  <cit>  and between endometrium of normal and developmentally delayed fetuses  <cit> . furthermore, genes involved in several important pathways regulating fetal development have been identified comparing gestational day  <dig> and  <dig> placentae from erhualian and large white pig breeds  <cit> .\\n\\ndifferences in reproductive efficiency exist between different pig breeds. the prolific meishan pigs have higher embryo survival rates during early gestation,which may be due to the fact that the meishan conceptus exhibits a reduced trophectoderm mitotic rate during the preimplantation period, elongates from fewer cells and remains smaller throughout gestation compared with other low litter size breeds  <cit> . to further the comparison between meishan and domestic swine females, reciprocal embryo transfers were performed and revealed that litter size is controlled primarily by the recipient females rather than the sire or embryos  <cit> . these data suggest that developing a more comprehensive understanding of the maternal uterine microenvironment influencing embryonic survival and implantation during the establishment of pregnancy could be of significance towards understanding maternal mechanisms contributing to successful pregnancy establishment.\\n\\nin the present investigation we conducted a genome-wide screen to characterize gene expression of endometrial tissue on day  <dig> of gestation, during maternal recognition of pregnancy. we used the affymetrix genechip® to analyze endometrial gene expression in meishan and yorkshire, which have phylogenetically distinct genomes and also have striking prolificacy differences with respect to litter size. we analyzed these endometrial transcriptomic data through a comparative annotation approach to gain a deeper molecular insight into genes controlling the prolific nature of pigs. a number of genes and transcripts were detected to be differentially expressed in endometrial tissue between breeds. the data and our analysis in this investigation contribute toward a deeper understanding of the molecular events associated with reproductive success during early pregnancy establishment by comparing the molecular events between two separate breeds with different levels of prolificacy.\\n\\nRESULTS\\ntranscriptome analysis in early pregnancy endometrium\\nthe affymetrix porcine genechip® was used for transcriptomic analyses of endometrium from yorkshire and meishan pigs early in pregnancy with four biological replicates for each pig breed. to compare similar stages of endometrial development during gestation, we collected tissue from pregnant meishan and yorkshire gilts bred at their second estrous cycle to boars of the same breed. at the time of endometrial tissue collection, we determined the morphological stage conceptuses present with the uteri. we only processed endometrial tissue for this comparison when all embryos were at the filamentous stage. if the signal of a transcript from at least two of the four biological replicates were called “present” by the affymetrix algorithm, we considered this transcript expressed in the endometrium. using this criterion,  <dig>  transcripts were identified in meishan day  <dig>  endometrium, and  <dig>  transcripts were detected in yorkshire day  <dig>  endometrium.\\n\\nidentification of differentially expressed genes\\nfollowing statistical analysis, we identified  <dig>  transcripts that were differentially expressed between m <dig> and y <dig> . among these transcripts,  <dig> were annotated as known genes; thus about 15% of the transcripts were unable to be annotated. among the differentially expressed genes,  <dig> were significantly more abundant in m <dig> endometrium, while  <dig> genes were more abundant in y <dig> \\n\\n\\n1\\n2\\n3\\n4\\n1transcript abundance differences. meishan\\u2009>\\u2009yorkshire indicates the comparison identifying transcripts that are more abundantly expressed in meishan endometrium compared to yorkshire endometrium on day  <dig> of pregnancy. yorkshire\\u2009>\\u2009meishan indicates the comparison identifying transcripts that are more abundantly expressed in yorkshire endometrium compared to meishan endometrium on day  <dig> of pregnancy.\\n\\n2number of differentially expressed annotated genes with 2-fold or greater difference between breeds. number in parenthesis indicates total number of differentially expressed transcripts.\\n\\n3number of differentially expressed annotated genes with  <dig> -fold or greater difference between breeds. number in parenthesis indicates total number of differentially expressed transcripts.\\n\\n4total number of differentially expressed annotated genes based on a q-value of less than  <dig> . number in parenthesis indicates total number of differentially expressed transcripts.\\n\\ncluster analysis and go annotations\\nto gain insight into similarities between the transcriptome of the two breeds, data from all the differentially expressed genes in the endometrium were used in a systematic cluster analysis . although there is substantial variation in expression within the replicates of one breed, the heat map suggests clearly that the selected differential gene set associates the biological samples correctly into two groups each representing one breed. the quality of this association enabled further analysis to identify potential pathways and functions impacted in endometrium using the genes detected at statistically different abundance between these two breeds.\\n\\nthe database for annotation, visualization and integrated discovery  was utilized to annotate biological terms and processes preferentially represented in meishan and yorkshire endometrium during the peri-implantation stage of development. we used the gene lists of q\\u2009<\\u2009 <dig>  for david annotation, and tested using the annotation chart function in david separately for enrichment of genes with greater abundance in yorkshire endometrium compared to meishan endometrium, as well as those with greater abundance in meishan endometrium compared to yorkshire endometrium. terms enriched utilizing these gene lists are provided as additional file 1: table s <dig> and additional file 2: table s <dig> \\n\\nthe five most significant terms in each of the most enriched in the annotation clusters expressed at a higher level in yorkshire tissue were membrane-enclosed lumen, rna-splicing, rna binding, nucleus, and blood vessel development . the functions of these enriched genes are closely associated with the nucleus, transcriptional and rna processing functions in cells. other functions represented in additional significantly enriched clusters in this list of differentially expressed  genes include apoptosis, cell-cell tight junction, and other functions . it may be that the yorkshire endometrium is more transcriptionally responsive to during maternal: fetal crosstalk following exposure to the maternal recognition of pregnancy signals. it is also interesting that genes associated with angiogenesis appear to be more prominently expressed in yorkshire endometrium. these genes include transcription factors, such as jun, foxo <dig>  and angiogenesis genes, such as vegfc, kdr.\\n\\na\\nadavid annotation chart analysis was used to identify the most highly enriched group of related terms  in the yorkshire\\u2009>\\u2009meishan list of differentially expressed genes. see entire results in additional file 2: table s2a.\\n\\nalternatively, the five most significant gene ontology terms enriched in the genes expressed more abundantly in meishan endometrium were translational elongation, ribosome biogenesis, protein folding, chord-containing proteins and sgt <dig> domain, and intracellular organelle lumen . in meishan endometrium, enriched functions seem to focus on post-transcriptional processes, especially protein synthesis and maturation . genes such as ribosome proteins  and translation elongation factors  are more highly expressed in meishan endometrium.\\n\\na\\nadavid annotation chart analysis was used to find the most highly enriched group of related terms  in the meishan\\u2009>\\u2009yorkshire list of differentially expressed genes. see entire results in additional file 2: table s2b.\\n\\nvalidations by qrt-pcr\\nto validate the microarray results, we chose  <dig> differentially expressed genes identified following the microarray analysis which have also been reported in the literature as relevant to the physiology of implantation. the qrt-pcr results are shown in figure\\xa0 <dig>  of the  <dig> genes assayed,  <dig> confirmed statistically significant differential expression in the same direction as the microarray data, and the remaining four genes showed the same direction of expression between microarray and qrt-pcr analyses despite not being statistically different.\\n\\nregulatory network analysis\\nidentification of small molecules or proteins with a known regulatory link in the biological literature to multiple highly differentially expressed genes is of interest. to identify such sub-networks, a sub-network enrichment analysis  was performed on the gene lists representing those greater in the yorkshire breed and those greater in the meishan breed. in all,  <dig> proteins and  <dig> small molecules were identified following snea with a p-value less than  <dig>  . the most significantly enriched protein hub was regulated by rora, retinoic acid receptor-related orphan receptor alpha ) being linked to five genes that were significantly more abundant in endometrium on day  <dig> from yorkshire compared to meishan . another retinoic acid receptor, nr2f <dig>  was also highly ranked on the protein list in yorkshire, due to a link with seven differentially expressed genes .\\n\\nusing the list representing de genes with greater abundance in day  <dig> endometrium of meishan, the snea identified  <dig> proteins and  <dig> small molecules  as potential regulatory hubs .of interest, leptin was linked with  <dig> genes involved in multiple pathways, such as solute carrier family members , growth factors , and immune system ; all genes with greater abundance in meishan endometrium. also of interest, ghrl was linked with  <dig> genes more abundantly expressed in meishan .\\n\\ndiscussion\\nidentifying the endometrial mechanisms and pathways contributing to reproductive success in pigs has long been a research interest  <cit> . the majority of prenatal mortality in pigs occurs before gestation day  <dig>  particularly during the conceptus trophoblastic elongation and attachment to the uterine endometrium, beginning around day  <dig> of gestation  <cit> . establishing the interface between the developing conceptus and the maternal endometrium involves a number of complex signaling networks. it is well known that meishan pigs farrow 3– <dig> more piglets than yorkshire  <cit> . a study focused on physiological differences between meishan and yorkshire embryos showed that in vitro cultured meishan embryos had lower mitotic rate and fewer cells in day  <dig> conceptuses in vivo <cit> . morphological comparison studies revealed that variation of conceptuses diameter was smaller for meishan pigs, indicating that meishan conceptuses potentially develop more uniformly between days  <dig> and  <dig> of gestation  <cit> . meishan and yorkshire embryos transferred to meishan uterine environment developed slower and secreted less estrogen, compared to being transferred to a yorkshire recipient, suggesting that the recipient’s genotype was the major factor controlling embryonic growth and estrogen secretion  <cit> . comparison of gene expression during physiologically important events across different breeds could enable the identification of novel pathways contributing to reproductive success in pigs. this approach has previously been utilized with the meishan breed characterizing the ovarian gene expression networks of f <dig> lberian x meishan sows  <cit> .\\n\\nthe advantage of microarray analysis is the simultaneous measurement of the expression patterns of large number of genes  <cit> , thus enabling classification of molecular characteristics of phenotypes and generating hypotheses predicting phenotypic outcomes  <cit> . in the pig, a number of high throughput platforms have been established for global expression profiling analysis  <cit> . in this study, we conducted a genome-wide transcriptomic analysis on uterine endometrium collected on day  <dig> of gestation to identify and better understand the molecular basis of differences in conceptus survival between yorkshire and meishan pigs. collectively, we identified the presence of approximately  <dig>  expressed transcripts in the uterine endometrium on day  <dig> of gestation and following statistic alanalysis identified  <dig> transcripts with differentially abundance of which  <dig> demonstrated 2-fold or greater difference. following validation of the microarray results with a sub-set of  <dig> genes assayed with qrt-pcr, we utilized these lists to identify pathways and gene ontology terms associated with each breed by using those gene lists representing expression greater in one breed over another.\\n\\ninterpretation of gene annotation analysis: selected pathways of interest\\nhormone genes and transcriptional activity\\noxytocin  decreases uterine receptivity in rodents, as rats treated with the oxt antagonist, atosiban, before embryo transfer had improved implantation rates  <cit> . it is of interest that we observed that oxt abundance was greater in yorkshire compared to meishan as assessed by both microarray and qrt-pcr. given that mrna expression is greater in yorkshire, it is possible that oxt mediated biological mechanisms may exist that contribute to and/or impact conceptus attachment and interaction with the uterine endometrium.\\n\\nprostaglandin production is an important component contributing to the successful establishment of pregnancy in the pig. expression of prostaglandin endoperoxide synthase  <dig>  and ptgs <dig> are substantially increased during implantation  <cit> . our microarray results showed that ptgs <dig> was more abundant in meishan endometrium  whereas ptgs <dig> was more abundant in yorkshire endometrium , which is consistent with the results in erhualian endometrium compared to large white  <cit> . these results indicated that higher ptgs <dig> may have an important role in embryo survival and attachment in chinese breeds.\\n\\nas shown in additional file 2: table s <dig>   <dig> genes expressed more abundantly in yorkshire were clustered to the gene ontology  term membrane enclosed lumen,  <dig> genes to go term rna processing, and  <dig> genes to go term transcription factor binding pathway following david analysis. transcription factor transcripts more abundant in yorkshire, such as hoxa <dig> and foxo <dig>  are known to be regulated by steroid hormones, including estrogen, progesterone and testosterone  <cit> . the potential that increased abundance of steroid-hormone dependent factors may be related to the greater quantity of steroid hormone secretion from yorkshire embryos  <cit> .\\n\\nblood vessel development\\nsuccessful apposition and attachment of pig conceptuses followed by the early formation of a placental interface requires a maternal vascular support. we observed that genes detected at a greater abundance in yorkshire pigs are enriched for go terms related to blood vessel development. for example, cadherin  <dig>  type  <dig> vascular endothelium  is not only an essential adhesion molecule, but also contributes to angiogenesis which regulates cell proliferation and modulates vegf receptors  <cit> . the genes with greater abundance in meishan, such as phka <dig>  prkag <dig>  ppp1ca, pygl, phkg <dig>  cps <dig> were clustered by david analyses into the pathways of glycogen metabolism and glucan metabolism process.\\n\\ncell adhesion and tight junction genes\\ncell-to-cell communication through connexin protein function has been shown in porcine endometrium to be lower than in species with invasive implantation  <cit> . thus endometrial cell-to-cell interaction may be involved in limiting trophoblast invasiveness, as porcine implantation is superficial, or in developing specific channels for this implantation type. proteins such as matrix metalloproteinase  are endopeptidases that play an important role in the transient invasive property of trophoblastic cells during implantation. expression of mmp <dig> was lower in human endometrium in women with repeated implantation failure compared to fertile women, indicating a potential role in human embryonic implantation, an invasive process  <cit> . in pigs, our qrt-pcr and microarray results show that mmp <dig> expression is greater in yorkshire endometrium on day  <dig> of pregnancy compared to meishan. integrin beta  <dig>  protein has also been detected at porcine embryonic attachment sites on day  <dig>  <cit> . itgb <dig> has been reported to be involved with signal transduction and formation of cytoplasmic focal adhesions, which was interpreted as supportive for implantation  <cit> . itgb <dig> transcript was in greater abundance in yorkshire endometrium than meishan. we also observed other genes expressed more abundantly in yorkshire compared to meishan that are annotated as members of tight junction pathways, such as claudin  family members  and tight junction protein  <dig>  tight junction molecules, such as cldn <dig>  improved invasion activity in cancer cell lines through activation of mt1-mmp- <dig> and mmp- <dig>  <cit> , and peak expression of claudin- <dig> rna has been temporally associated with the implantation window in humans  <cit> . these data may suggest that development of membrane channels for specific ions, a major function of claudins, may be important to facilitate successful conceptus attachment in yorkshire pigs.\\n\\nimmune related pathways\\nwe also investigated the involvement of immune related pathways in de genes between these breeds based on the importance of immunity in early maternal: fetal interactions. successful pregnancy requires appropriate spatio-temporal regulation of genes contributing to the immune response during the peri-implantation period. although reports exist that aborted placentas expressed less th <dig> cytokines and greater th <dig> cytokines than successful pregnancies  <cit> , recent literature indicated that uterine receptivity is actually promoted by a th <dig> inflammatory response, whereas the th2-humoral response facilitated pregnancy maintenance  <cit> . we observed that genes expressed more abundantly in meishan endometrium compared to yorkshire are annotated as functioning in immune response pathways, such as cd <dig>  which may indicate a positive role of cd <dig> during implantation. cd <dig> expression has been significant positively correlated with th <dig> cytokines expression   <cit> . while reports on mice indicated that suppression of cd <dig> and other cytokines by cyclosporine a improved pregnancy rate  <cit> , the exact involvement of cd <dig> in abortion was not clear. in this context, it is interesting that three peptidyl-prolylisomerases  are more abundantly expressed in meishan . proteins in the ppi family are known to bind and potentially mediate cyclosporine a function, which in turn has been shown to improve pregnancy rate when administered at the implantation window stage in mice  <cit> . thus the ppi proteins may bind to natural cyclosporine analogs impacting the immunological regulation of conceptus attachment.\\n\\nwhile not sufficiently enriched at the level of immune annotation clustering, there were also other immune-related genes that are more highly expressed in yorkshire in the david analysis. among these, maps <dig> may function by activating complement component factors  <cit>  and alpha-2-macroglobin  is involved in immune cell migration in pregnancy  <cit> . additionally, transferrin  may be involved in removal of allergens from serum  <cit>  and it is expressed differently in the pregnant pig compared the non-pregnant pig  <cit> , indicating the potential role during pregnancy. higher expression of immunity associated genes such as maps <dig>  a2m and tf in meishan compared to yorkshire, but not the major cytokines, such as il <dig> and il <dig>  may be suggestive that the difference between these two breeds was not as large as that between females with normal versus aborted pregnancies. however, the snea does indicate that genes expressed more abundantly in meishan are linked to cytokines il <dig> and il <dig> .\\n\\nregulatory network analysis\\nrora  is a transcription factor belonging to the steroid hormone receptor superfamily  <cit>  and has been demonstrated to be expressed in human endometrium  <cit> . this transcription factor, as well as nr2f <dig>  has been suggested to play an important role in mediating the cellular response to retinoic acid  <cit> . rora is shown to activate hypoxia-inducible factor  <dig> alpha   <cit>  and hif- <dig> is an important mediator of estrogen-induced vegf expression in the uterus  <cit> . the function of these networks is concordant with the conclusion from our david analysis suggesting greater activation of angiogenesis at this stage in the yorkshire breed.\\n\\nleptin is a small peptide synthesized by white adipose tissue that regulates body weight homeostasis. leptin transcript abundance was significantly higher in the endometrium of fertile women as compared to women with recurrent implantation failure  <cit>  and thus higher meishan expression of leptin in our study is consistent with the prolificacy seen in this breed. ghrelin is thought as a novel paracrine/autocrine factor expressed in endometrium and placenta that is involved in crosstalk between endometrium and embryo during embryo implantation  <cit> , which may indicate that such intra-tissue communication is critical for implantation success.\\n\\nCONCLUSIONS\\nconceptus and fetal loss during early pregnancy establishment can be a critical determinant of litter size. however, identification of molecular pathways that may contribute to successful pregnancy establishment and the formation of a fetal: maternal interface has not yet been investigated in breeds with established differences in litter size performance. in this study, the affymetrix genechip® was used to compare gene expression profiles and identify associated pathways which may be differentially regulated in the uterine endometrium of meishan and yorkshire gilts during day  <dig> of gestation. the results have described the molecular portraits of uterine endometrium during the opening of the implantation window revealing differences between the meishan and yorkshire breeds. our results revealed several classifications of genes participating in regulation of endometrium function during early pregnancy. future analysis will require an exhaustive look into the protein repertoire of the uterine endometrium of the two breeds to strengthen our understanding of the relationship between the transcriptome and proteome in the uterine endometrium.\\n\\n'"
            ]
          },
          "execution_count": 32,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "test_text[0]"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 36,
      "metadata": {},
      "outputs": [],
      "source": [
        "# custome Dialogue Prediction\n",
        "\n",
        "from transformers import pipeline  # Import the pipeline function\n",
        "def generate_summary(text):\n",
        "    pipe = pipeline('summarization', model=model, tokenizer=tokenizer)\n",
        "    gen_kwargs = {'length_penalty': 0.8, 'num_beams': 8, \"max_length\": 1000}\n",
        "\n",
        "    custom_dialogue=text\n",
        "    print(pipe(custom_dialogue, **gen_kwargs))\n",
        "\n",
        "\n",
        "    returned_summary=pipe(text,**gen_kwargs) #do_sample=False)#max_length=130, min_length=30,\n",
        "    #print(\"Original text:\\n\"+text)\n",
        "    #print(\"\\n\\nGenerated summary:\\n\"+returned_summary[0]['summary_text'])\n",
        "    return returned_summary[0]['summary_text']"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 39,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "1\n",
            "[{'summary_text': 'a number of genes and transcripts were detected to be differentially expressed in the uterine endometrium of yorkshire and yorkshire gilts on day  <dig> of gestation.\\nwe identified  <dig> differentially expressed genes in the uterine endometrium of yorkshire and yorkshire gilts on day  <dig> of gestation, and  <dig> differentially expressed genes in the uterine endometrium of yorkshire gilts on day  <dig> of gestation.\\nthe uterine endometrium provides an ideal window to investigate the molecular mechanisms that may be associated early conceptus loss and sow reproductive prolificacy.\\na number of genes and transcripts were detected to be differentially expressed in the uterine endometrium of yorkshire and yorkshire gilts on day  <dig> of gestation.\\na number of genes and transcripts were detected to be differentially expressed in the uterine endometrium of yorkshire and yorkshire gilts on day  <dig> of gestation.\\na number of genes and transcripts were detected to be differentially expressed in the uterine endometrium of yorkshire and yorkshire gilts on day  <dig> of gestation.\\na number of genes and transcripts were'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the pig  has been used as model animal in many fields, including biological  <cit> , agricultural, and biomedical studies  <cit> . pig litter size, which can be impacted by multiple factors including ovulation rate, embryonic survival, uterine capacity and fetal survival, is one of the most important economic traits affecting production efficiency in the pig industry  <cit> . improvements in litter size across the swine industry in recent decades has occurred through different selection schemes such as phenotypic selection, family index selection, blup-based selection, component trait selection, and hyperprolific selection  <cit> . however, despite current improvements, the low heritability of litter size has limited the genetic selection progress in substantially improving this trait  <cit> . improvement of litter size is ideally suited for application of molecular breeding approaches such as marker-assisted selection  and whole-genome selection, where molecular markers associated with superior trait values may accelerate genetic improvement. in addition, numerous studies have attempted to detect candidate genes underlying the molecular mechanism of sow prolificacy. although some candidate genes for sow prolificacy have been reported  <cit> , true causal genes remain scant. a deeper understanding of the physiological and molecular mechanisms associated with reproductive efficiency is needed to further the biological underpinnings that contribute to controlling litter size in pigs.\n",
            "\n",
            "it is now well established that prenatal mortality can be a major determinant of litter size in pigs  <cit> . when early conceptuses undergo rapid differentiation and expansion of their trophoblastic membranes during the first few weeks of gestation, abnormalities of embryogenesis that produce embryonic mortality can occur  <cit> . the extent and timing of embryonic and fetal losses during gestation has been well documented by previous studies, and it is estimated that about 25-45% of fertilized ova do not survive through gestation  <cit> . the majority of embryonic loss has been demonstrated to occur before day  <dig> of gestation, primarily around day 11– <dig>  <cit> . early events in conceptus development, including rapid trophoblastic elongation and successful establishment of the fetal: maternal interface, play a key part in determining litter size. the conceptus itself undergoes a dramatic transcriptomic reorganization during trophoblastic elongation and during initial interactions with the uterine endometrium  <cit> . additionally, the copious amounts of estrogen synthesized and released by the elongating conceptus serves as the maternal recognition of pregnancy signal around day  <dig> of gestation  <cit> . the timing of endometrial exposure to estrogen is critical as specific molecular cues associated with uterine receptivity occur simultaneously with conceptus estrogen synthesis and abnormal exposure to estrogen prior to day  <dig> results in compromised pregnancy establishment  <cit> .\n",
            "\n",
            "studies have revealed that there are very different patterns of gene expression in uterine endometrium between successful and abortive embryo implantation, as well as between high and low litter size breeds  <cit> . it is now clear that the molecular events happening in the maternal endometrium during the early stages play a crucial role in determining the success of embryonic survival, and the early-stage endometrium provides an ideal window to investigate the molecular mechanisms that may be associated early conceptus loss and sow reproductive prolificacy. many genes involved in angiogenesis and immune responses are expressed at the endometrium-fetal surface  <cit> . interestingly, polymorphisms in genes expressed during embryonic implantation, such as in epha <dig> and tgf-β, have been associated with litter size  <cit> . however, what is lacking is a systematic comparison investigating global gene expression patterns in the uterine endometrium during the peri-implantation stage of pregnancy between pig breeds with established differences in litter size.\n",
            "\n",
            "microarrays have been widely used to detect differentially expressed genes hypothesized to contribute to controlling litter size. some interesting genes, such as steroidogenic acute regulatory protein and interleukin-1beta , were found to be expressed in during early development using multiple global expression profiling approaches  <cit> . differentially expressed genes have been detected in endometrium between pregnant and non-pregnant sows  <cit>  and between endometrium of normal and developmentally delayed fetuses  <cit> . furthermore, genes involved in several important pathways regulating fetal development have been identified comparing gestational day  <dig> and  <dig> placentae from erhualian and large white pig breeds  <cit> .\n",
            "\n",
            "differences in reproductive efficiency exist between different pig breeds. the prolific meishan pigs have higher embryo survival rates during early gestation,which may be due to the fact that the meishan conceptus exhibits a reduced trophectoderm mitotic rate during the preimplantation period, elongates from fewer cells and remains smaller throughout gestation compared with other low litter size breeds  <cit> . to further the comparison between meishan and domestic swine females, reciprocal embryo transfers were performed and revealed that litter size is controlled primarily by the recipient females rather than the sire or embryos  <cit> . these data suggest that developing a more comprehensive understanding of the maternal uterine microenvironment influencing embryonic survival and implantation during the establishment of pregnancy could be of significance towards understanding maternal mechanisms contributing to successful pregnancy establishment.\n",
            "\n",
            "in the present investigation we conducted a genome-wide screen to characterize gene expression of endometrial tissue on day  <dig> of gestation, during maternal recognition of pregnancy. we used the affymetrix genechip® to analyze endometrial gene expression in meishan and yorkshire, which have phylogenetically distinct genomes and also have striking prolificacy differences with respect to litter size. we analyzed these endometrial transcriptomic data through a comparative annotation approach to gain a deeper molecular insight into genes controlling the prolific nature of pigs. a number of genes and transcripts were detected to be differentially expressed in endometrial tissue between breeds. the data and our analysis in this investigation contribute toward a deeper understanding of the molecular events associated with reproductive success during early pregnancy establishment by comparing the molecular events between two separate breeds with different levels of prolificacy.\n",
            "\n",
            "RESULTS\n",
            "transcriptome analysis in early pregnancy endometrium\n",
            "the affymetrix porcine genechip® was used for transcriptomic analyses of endometrium from yorkshire and meishan pigs early in pregnancy with four biological replicates for each pig breed. to compare similar stages of endometrial development during gestation, we collected tissue from pregnant meishan and yorkshire gilts bred at their second estrous cycle to boars of the same breed. at the time of endometrial tissue collection, we determined the morphological stage conceptuses present with the uteri. we only processed endometrial tissue for this comparison when all embryos were at the filamentous stage. if the signal of a transcript from at least two of the four biological replicates were called “present” by the affymetrix algorithm, we considered this transcript expressed in the endometrium. using this criterion,  <dig>  transcripts were identified in meishan day  <dig>  endometrium, and  <dig>  transcripts were detected in yorkshire day  <dig>  endometrium.\n",
            "\n",
            "identification of differentially expressed genes\n",
            "following statistical analysis, we identified  <dig>  transcripts that were differentially expressed between m <dig> and y <dig> . among these transcripts,  <dig> were annotated as known genes; thus about 15% of the transcripts were unable to be annotated. among the differentially expressed genes,  <dig> were significantly more abundant in m <dig> endometrium, while  <dig> genes were more abundant in y <dig> \n",
            "\n",
            "\n",
            "1\n",
            "2\n",
            "3\n",
            "4\n",
            "1transcript abundance differences. meishan > yorkshire indicates the comparison identifying transcripts that are more abundantly expressed in meishan endometrium compared to yorkshire endometrium on day  <dig> of pregnancy. yorkshire > meishan indicates the comparison identifying transcripts that are more abundantly expressed in yorkshire endometrium compared to meishan endometrium on day  <dig> of pregnancy.\n",
            "\n",
            "2number of differentially expressed annotated genes with 2-fold or greater difference between breeds. number in parenthesis indicates total number of differentially expressed transcripts.\n",
            "\n",
            "3number of differentially expressed annotated genes with  <dig> -fold or greater difference between breeds. number in parenthesis indicates total number of differentially expressed transcripts.\n",
            "\n",
            "4total number of differentially expressed annotated genes based on a q-value of less than  <dig> . number in parenthesis indicates total number of differentially expressed transcripts.\n",
            "\n",
            "cluster analysis and go annotations\n",
            "to gain insight into similarities between the transcriptome of the two breeds, data from all the differentially expressed genes in the endometrium were used in a systematic cluster analysis . although there is substantial variation in expression within the replicates of one breed, the heat map suggests clearly that the selected differential gene set associates the biological samples correctly into two groups each representing one breed. the quality of this association enabled further analysis to identify potential pathways and functions impacted in endometrium using the genes detected at statistically different abundance between these two breeds.\n",
            "\n",
            "the database for annotation, visualization and integrated discovery  was utilized to annotate biological terms and processes preferentially represented in meishan and yorkshire endometrium during the peri-implantation stage of development. we used the gene lists of q <  <dig>  for david annotation, and tested using the annotation chart function in david separately for enrichment of genes with greater abundance in yorkshire endometrium compared to meishan endometrium, as well as those with greater abundance in meishan endometrium compared to yorkshire endometrium. terms enriched utilizing these gene lists are provided as additional file 1: table s <dig> and additional file 2: table s <dig> \n",
            "\n",
            "the five most significant terms in each of the most enriched in the annotation clusters expressed at a higher level in yorkshire tissue were membrane-enclosed lumen, rna-splicing, rna binding, nucleus, and blood vessel development . the functions of these enriched genes are closely associated with the nucleus, transcriptional and rna processing functions in cells. other functions represented in additional significantly enriched clusters in this list of differentially expressed  genes include apoptosis, cell-cell tight junction, and other functions . it may be that the yorkshire endometrium is more transcriptionally responsive to during maternal: fetal crosstalk following exposure to the maternal recognition of pregnancy signals. it is also interesting that genes associated with angiogenesis appear to be more prominently expressed in yorkshire endometrium. these genes include transcription factors, such as jun, foxo <dig>  and angiogenesis genes, such as vegfc, kdr.\n",
            "\n",
            "a\n",
            "adavid annotation chart analysis was used to identify the most highly enriched group of related terms  in the yorkshire > meishan list of differentially expressed genes. see entire results in additional file 2: table s2a.\n",
            "\n",
            "alternatively, the five most significant gene ontology terms enriched in the genes expressed more abundantly in meishan endometrium were translational elongation, ribosome biogenesis, protein folding, chord-containing proteins and sgt <dig> domain, and intracellular organelle lumen . in meishan endometrium, enriched functions seem to focus on post-transcriptional processes, especially protein synthesis and maturation . genes such as ribosome proteins  and translation elongation factors  are more highly expressed in meishan endometrium.\n",
            "\n",
            "a\n",
            "adavid annotation chart analysis was used to find the most highly enriched group of related terms  in the meishan > yorkshire list of differentially expressed genes. see entire results in additional file 2: table s2b.\n",
            "\n",
            "validations by qrt-pcr\n",
            "to validate the microarray results, we chose  <dig> differentially expressed genes identified following the microarray analysis which have also been reported in the literature as relevant to the physiology of implantation. the qrt-pcr results are shown in figure  <dig>  of the  <dig> genes assayed,  <dig> confirmed statistically significant differential expression in the same direction as the microarray data, and the remaining four genes showed the same direction of expression between microarray and qrt-pcr analyses despite not being statistically different.\n",
            "\n",
            "regulatory network analysis\n",
            "identification of small molecules or proteins with a known regulatory link in the biological literature to multiple highly differentially expressed genes is of interest. to identify such sub-networks, a sub-network enrichment analysis  was performed on the gene lists representing those greater in the yorkshire breed and those greater in the meishan breed. in all,  <dig> proteins and  <dig> small molecules were identified following snea with a p-value less than  <dig>  . the most significantly enriched protein hub was regulated by rora, retinoic acid receptor-related orphan receptor alpha ) being linked to five genes that were significantly more abundant in endometrium on day  <dig> from yorkshire compared to meishan . another retinoic acid receptor, nr2f <dig>  was also highly ranked on the protein list in yorkshire, due to a link with seven differentially expressed genes .\n",
            "\n",
            "using the list representing de genes with greater abundance in day  <dig> endometrium of meishan, the snea identified  <dig> proteins and  <dig> small molecules  as potential regulatory hubs .of interest, leptin was linked with  <dig> genes involved in multiple pathways, such as solute carrier family members , growth factors , and immune system ; all genes with greater abundance in meishan endometrium. also of interest, ghrl was linked with  <dig> genes more abundantly expressed in meishan .\n",
            "\n",
            "discussion\n",
            "identifying the endometrial mechanisms and pathways contributing to reproductive success in pigs has long been a research interest  <cit> . the majority of prenatal mortality in pigs occurs before gestation day  <dig>  particularly during the conceptus trophoblastic elongation and attachment to the uterine endometrium, beginning around day  <dig> of gestation  <cit> . establishing the interface between the developing conceptus and the maternal endometrium involves a number of complex signaling networks. it is well known that meishan pigs farrow 3– <dig> more piglets than yorkshire  <cit> . a study focused on physiological differences between meishan and yorkshire embryos showed that in vitro cultured meishan embryos had lower mitotic rate and fewer cells in day  <dig> conceptuses in vivo <cit> . morphological comparison studies revealed that variation of conceptuses diameter was smaller for meishan pigs, indicating that meishan conceptuses potentially develop more uniformly between days  <dig> and  <dig> of gestation  <cit> . meishan and yorkshire embryos transferred to meishan uterine environment developed slower and secreted less estrogen, compared to being transferred to a yorkshire recipient, suggesting that the recipient’s genotype was the major factor controlling embryonic growth and estrogen secretion  <cit> . comparison of gene expression during physiologically important events across different breeds could enable the identification of novel pathways contributing to reproductive success in pigs. this approach has previously been utilized with the meishan breed characterizing the ovarian gene expression networks of f <dig> lberian x meishan sows  <cit> .\n",
            "\n",
            "the advantage of microarray analysis is the simultaneous measurement of the expression patterns of large number of genes  <cit> , thus enabling classification of molecular characteristics of phenotypes and generating hypotheses predicting phenotypic outcomes  <cit> . in the pig, a number of high throughput platforms have been established for global expression profiling analysis  <cit> . in this study, we conducted a genome-wide transcriptomic analysis on uterine endometrium collected on day  <dig> of gestation to identify and better understand the molecular basis of differences in conceptus survival between yorkshire and meishan pigs. collectively, we identified the presence of approximately  <dig>  expressed transcripts in the uterine endometrium on day  <dig> of gestation and following statistic alanalysis identified  <dig> transcripts with differentially abundance of which  <dig> demonstrated 2-fold or greater difference. following validation of the microarray results with a sub-set of  <dig> genes assayed with qrt-pcr, we utilized these lists to identify pathways and gene ontology terms associated with each breed by using those gene lists representing expression greater in one breed over another.\n",
            "\n",
            "interpretation of gene annotation analysis: selected pathways of interest\n",
            "hormone genes and transcriptional activity\n",
            "oxytocin  decreases uterine receptivity in rodents, as rats treated with the oxt antagonist, atosiban, before embryo transfer had improved implantation rates  <cit> . it is of interest that we observed that oxt abundance was greater in yorkshire compared to meishan as assessed by both microarray and qrt-pcr. given that mrna expression is greater in yorkshire, it is possible that oxt mediated biological mechanisms may exist that contribute to and/or impact conceptus attachment and interaction with the uterine endometrium.\n",
            "\n",
            "prostaglandin production is an important component contributing to the successful establishment of pregnancy in the pig. expression of prostaglandin endoperoxide synthase  <dig>  and ptgs <dig> are substantially increased during implantation  <cit> . our microarray results showed that ptgs <dig> was more abundant in meishan endometrium  whereas ptgs <dig> was more abundant in yorkshire endometrium , which is consistent with the results in erhualian endometrium compared to large white  <cit> . these results indicated that higher ptgs <dig> may have an important role in embryo survival and attachment in chinese breeds.\n",
            "\n",
            "as shown in additional file 2: table s <dig>   <dig> genes expressed more abundantly in yorkshire were clustered to the gene ontology  term membrane enclosed lumen,  <dig> genes to go term rna processing, and  <dig> genes to go term transcription factor binding pathway following david analysis. transcription factor transcripts more abundant in yorkshire, such as hoxa <dig> and foxo <dig>  are known to be regulated by steroid hormones, including estrogen, progesterone and testosterone  <cit> . the potential that increased abundance of steroid-hormone dependent factors may be related to the greater quantity of steroid hormone secretion from yorkshire embryos  <cit> .\n",
            "\n",
            "blood vessel development\n",
            "successful apposition and attachment of pig conceptuses followed by the early formation of a placental interface requires a maternal vascular support. we observed that genes detected at a greater abundance in yorkshire pigs are enriched for go terms related to blood vessel development. for example, cadherin  <dig>  type  <dig> vascular endothelium  is not only an essential adhesion molecule, but also contributes to angiogenesis which regulates cell proliferation and modulates vegf receptors  <cit> . the genes with greater abundance in meishan, such as phka <dig>  prkag <dig>  ppp1ca, pygl, phkg <dig>  cps <dig> were clustered by david analyses into the pathways of glycogen metabolism and glucan metabolism process.\n",
            "\n",
            "cell adhesion and tight junction genes\n",
            "cell-to-cell communication through connexin protein function has been shown in porcine endometrium to be lower than in species with invasive implantation  <cit> . thus endometrial cell-to-cell interaction may be involved in limiting trophoblast invasiveness, as porcine implantation is superficial, or in developing specific channels for this implantation type. proteins such as matrix metalloproteinase  are endopeptidases that play an important role in the transient invasive property of trophoblastic cells during implantation. expression of mmp <dig> was lower in human endometrium in women with repeated implantation failure compared to fertile women, indicating a potential role in human embryonic implantation, an invasive process  <cit> . in pigs, our qrt-pcr and microarray results show that mmp <dig> expression is greater in yorkshire endometrium on day  <dig> of pregnancy compared to meishan. integrin beta  <dig>  protein has also been detected at porcine embryonic attachment sites on day  <dig>  <cit> . itgb <dig> has been reported to be involved with signal transduction and formation of cytoplasmic focal adhesions, which was interpreted as supportive for implantation  <cit> . itgb <dig> transcript was in greater abundance in yorkshire endometrium than meishan. we also observed other genes expressed more abundantly in yorkshire compared to meishan that are annotated as members of tight junction pathways, such as claudin  family members  and tight junction protein  <dig>  tight junction molecules, such as cldn <dig>  improved invasion activity in cancer cell lines through activation of mt1-mmp- <dig> and mmp- <dig>  <cit> , and peak expression of claudin- <dig> rna has been temporally associated with the implantation window in humans  <cit> . these data may suggest that development of membrane channels for specific ions, a major function of claudins, may be important to facilitate successful conceptus attachment in yorkshire pigs.\n",
            "\n",
            "immune related pathways\n",
            "we also investigated the involvement of immune related pathways in de genes between these breeds based on the importance of immunity in early maternal: fetal interactions. successful pregnancy requires appropriate spatio-temporal regulation of genes contributing to the immune response during the peri-implantation period. although reports exist that aborted placentas expressed less th <dig> cytokines and greater th <dig> cytokines than successful pregnancies  <cit> , recent literature indicated that uterine receptivity is actually promoted by a th <dig> inflammatory response, whereas the th2-humoral response facilitated pregnancy maintenance  <cit> . we observed that genes expressed more abundantly in meishan endometrium compared to yorkshire are annotated as functioning in immune response pathways, such as cd <dig>  which may indicate a positive role of cd <dig> during implantation. cd <dig> expression has been significant positively correlated with th <dig> cytokines expression   <cit> . while reports on mice indicated that suppression of cd <dig> and other cytokines by cyclosporine a improved pregnancy rate  <cit> , the exact involvement of cd <dig> in abortion was not clear. in this context, it is interesting that three peptidyl-prolylisomerases  are more abundantly expressed in meishan . proteins in the ppi family are known to bind and potentially mediate cyclosporine a function, which in turn has been shown to improve pregnancy rate when administered at the implantation window stage in mice  <cit> . thus the ppi proteins may bind to natural cyclosporine analogs impacting the immunological regulation of conceptus attachment.\n",
            "\n",
            "while not sufficiently enriched at the level of immune annotation clustering, there were also other immune-related genes that are more highly expressed in yorkshire in the david analysis. among these, maps <dig> may function by activating complement component factors  <cit>  and alpha-2-macroglobin  is involved in immune cell migration in pregnancy  <cit> . additionally, transferrin  may be involved in removal of allergens from serum  <cit>  and it is expressed differently in the pregnant pig compared the non-pregnant pig  <cit> , indicating the potential role during pregnancy. higher expression of immunity associated genes such as maps <dig>  a2m and tf in meishan compared to yorkshire, but not the major cytokines, such as il <dig> and il <dig>  may be suggestive that the difference between these two breeds was not as large as that between females with normal versus aborted pregnancies. however, the snea does indicate that genes expressed more abundantly in meishan are linked to cytokines il <dig> and il <dig> .\n",
            "\n",
            "regulatory network analysis\n",
            "rora  is a transcription factor belonging to the steroid hormone receptor superfamily  <cit>  and has been demonstrated to be expressed in human endometrium  <cit> . this transcription factor, as well as nr2f <dig>  has been suggested to play an important role in mediating the cellular response to retinoic acid  <cit> . rora is shown to activate hypoxia-inducible factor  <dig> alpha   <cit>  and hif- <dig> is an important mediator of estrogen-induced vegf expression in the uterus  <cit> . the function of these networks is concordant with the conclusion from our david analysis suggesting greater activation of angiogenesis at this stage in the yorkshire breed.\n",
            "\n",
            "leptin is a small peptide synthesized by white adipose tissue that regulates body weight homeostasis. leptin transcript abundance was significantly higher in the endometrium of fertile women as compared to women with recurrent implantation failure  <cit>  and thus higher meishan expression of leptin in our study is consistent with the prolificacy seen in this breed. ghrelin is thought as a novel paracrine/autocrine factor expressed in endometrium and placenta that is involved in crosstalk between endometrium and embryo during embryo implantation  <cit> , which may indicate that such intra-tissue communication is critical for implantation success.\n",
            "\n",
            "CONCLUSIONS\n",
            "conceptus and fetal loss during early pregnancy establishment can be a critical determinant of litter size. however, identification of molecular pathways that may contribute to successful pregnancy establishment and the formation of a fetal: maternal interface has not yet been investigated in breeds with established differences in litter size performance. in this study, the affymetrix genechip® was used to compare gene expression profiles and identify associated pathways which may be differentially regulated in the uterine endometrium of meishan and yorkshire gilts during day  <dig> of gestation. the results have described the molecular portraits of uterine endometrium during the opening of the implantation window revealing differences between the meishan and yorkshire breeds. our results revealed several classifications of genes participating in regulation of endometrium function during early pregnancy. future analysis will require an exhaustive look into the protein repertoire of the uterine endometrium of the two breeds to strengthen our understanding of the relationship between the transcriptome and proteome in the uterine endometrium.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "a number of genes and transcripts were detected to be differentially expressed in the uterine endometrium of yorkshire and yorkshire gilts on day  <dig> of gestation.\n",
            "we identified  <dig> differentially expressed genes in the uterine endometrium of yorkshire and yorkshire gilts on day  <dig> of gestation, and  <dig> differentially expressed genes in the uterine endometrium of yorkshire gilts on day  <dig> of gestation.\n",
            "the uterine endometrium provides an ideal window to investigate the molecular mechanisms that may be associated early conceptus loss and sow reproductive prolificacy.\n",
            "a number of genes and transcripts were detected to be differentially expressed in the uterine endometrium of yorkshire and yorkshire gilts on day  <dig> of gestation.\n",
            "a number of genes and transcripts were detected to be differentially expressed in the uterine endometrium of yorkshire and yorkshire gilts on day  <dig> of gestation.\n",
            "a number of genes and transcripts were detected to be differentially expressed in the uterine endometrium of yorkshire and yorkshire gilts on day  <dig> of gestation.\n",
            "a number of genes and transcripts were\n",
            "2\n",
            "[{'summary_text': 'both error correction software packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of read errors.\\nboth error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of read errors.\\nboth error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of read errors.\\nboth error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of reads.\\nboth error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of reads.\\nboth error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of reads.\\nboth error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of reads.\\nboth error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the prevalence of next-generation sequencing  has increased throughput for generating genomic data and our ability to perform genomic analysis. genomic analyses continue to increase our ability to understand genetic diseases and disorders.\n",
            "\n",
            "ngs technologies take short fragments of dna obtained from the genome of an organism of interest. the fragments are then sequenced. sequencing reads is the process of identifying a fragment's constituent nucleotides through chemical processes. a fragment with all bases identified, whether correctly or incorrectly, forms a read  <cit> .\n",
            "\n",
            "these reads are used in several different types of analyses. one example of this is in genome assembly. the generated reads are assembled into the source genome through a process of identifying reads that originated from the same regions and assembling them by merging them into longer, contiguous sequences called contigs. assembly is complex and difficult due to the short length of sequenced reads and requires volumes of high fidelity data to be accomplished accurately  <cit> .\n",
            "\n",
            "ngs technologies are, however, imperfect and misidentify some of the nucleotides contained in a dna fragment as they are being sequenced  <cit> . thus, errors are introduced into the reads. these errors are problematic because they introduce false genetic information into a dataset and complicate genome assemblies. error correction packages have been created that use different techniques in order to locate misidentified bases during sequencing and to correct them to their true sequence  <cit> . this allows for recovery of data that could otherwise confound a genome assembler. while error correction software is intended to correct errors, it can also introduce errors into a readset through mis-correction.\n",
            "\n",
            "genome assembly becomes even more complicated in the presence of heterozygosity. many organisms have a ploidy greater than one, including humans. diploid and polyploid organisms inherit genetic information from multiple sources . one set of genetic variation inherited from a parent is called a haplotype. when sequencing and assembling the genome of an organism, haplotype conservation is important for preserving and understanding the biological reality of an organism's genome. despite lower heterozygosity rates in humans compared to some organisms  <cit> , proper identification of polymorphism remains fundamental to genotype-phenotype analyses.\n",
            "\n",
            "many current genome assemblers do not maintain segregated haplotypes  <cit> . conservation of separate haplotypes during genome assembly allows for better understanding of complex diseases and phenotypes that have not been associated with single variations. haplotype-aware analyses allow for more powerful understanding of haplotypes and phenotypes. they also enable better protein structure prediction  <cit> . studies of the effects of heterozygosity on error correction performance are lacking.\n",
            "\n",
            "in this study, two different error correction techniques are comparatively analyzed by examining their effects on next-generation sequencing data from a heterozygous genome. these two packages are quake and echo.\n",
            "\n",
            "quake\n",
            "quake is a k-spectrum based approach to base-call error correction. quake analyzes the k-mer coverage distribution. then, a cut-off is determined from the distribution that identify trusted and untrusted k-mers. trusted k-mers are used to identify errors in the readset. a set of possible corrections is made and searched to find the correction with the maximum likelihood of making all k-mers trusted. in addition to correcting miscalled bases, quake also trims reads. trimming is the process of removing the ends of reads to remove low quality bases. quake requires that the value of k be specified at runtime. a formula is given by the creators of quake to calculate k based on genome size  <cit> .\n",
            "\n",
            "echo\n",
            "echo utilizes a multiple sequence alignment  approach in order to perform base-call error correction. based on a probabilistic model, echo first clusters reads from the same region of the source genome then corrects the reads. echo does not require input of genome size or any other input parameters to be run. the authors state that \" explicitly models heterozygosity in diploid genomes and provides a reference-free method for detecting bases that originated from heterozygous sites\"  <cit> .\n",
            "\n",
            "RESULTS\n",
            "synthetic datasets\n",
            "quake\n",
            "both haploid and diploid genome sizes of the genome were used when calculating the appropriate k for quake. the results from using the haploid and diploid genome sizes had nearly identical results. quake's general performance does not appear to change much as heterozygosity increases. quake's performance on both heterozygous and homozygous errors considered together does not appear to change much as heterozygosity increases .\n",
            "\n",
            "when correcting the datasets with all reads for a particular error and heterozygosity rate combined  and using the diploid and the haploid genome sizes as parameters, the error-corrected reads showed several of the same general trends for the first three error rates for errors at heterozygous positions. the rate of errors at heterozygous locations that were corrected correctly increased. the rate of errors at heterozygous locations that were not corrected decreased. the rate of errors at heterozygous locations corrected to the wrong haplotype was low  and varied little. the rate of errors at heterozygous locations corrected to neither of the haplotypes was low  and varied little. finally, the rate of heterozygous locations that had no error corrected to the other haplotype or to neither haplotype was low and varied little .\n",
            "\n",
            "a comparison of the first three error rates showed two variations. first, the rate of errors at heterozygous locations corrected to the wrong haplotype increased slightly as the error rate increased. second, the rate of non-error heterozygous locations corrected to the wrong haplotype was slightly increased with increased error rates.\n",
            "\n",
            "when correcting the datasets split by haplotype , several general trends hold for the first three error rates at all levels of heterozygosity. the rate of errors at heterozygous locations corrected correctly was high. the rate of errors at heterozygous locations not corrected was low. the rate of errors at heterozygous locations corrected to the wrong haplotype was low. lastly, the rate of errors at heterozygous locations corrected to neither of the haplotypes was low.\n",
            "\n",
            "echo\n",
            "echo differs from quake because it does not require the user to specify parameters at runtime. therefore, echo was run once on the heterozygous datasets and again on the homozygous datasets with no variation of parameters. the accuracy of echo was similar to quake as little variation in performance when observing all errors  was found. the rate of errors at heterozygous locations that were corrected correctly increased. the rate of errors at heterozygous locations corrected to the wrong haplotype decreased. the rate of errors at heterozygous locations that were not corrected decreased. observed variations between the first three error rates show that the rate of non-error heterozygous locations corrected to the other haplotype decreases as the error rate increases .\n",
            "\n",
            "echo's correction of the homozygous data for the first three error rates produced trends similar to quake's. the rate of errors at heterozygous locations corrected correctly was high. the rate of errors at heterozygous locations was low. the rate of errors at heterozygous locations corrected to the wrong haplotype or to neither haplotype was low.\n",
            "\n",
            "real sequencing data\n",
            "quake\n",
            "assemblies using quake-corrected reads were superior to the uncorrected assembly . the corrected assembly had a decreased number of contigs, larger n <dig>  and an increased largest contig length. the assembly using quake-corrected reads where the raw reads were segregated by strain during correction performed better than the assembly when all reads were corrected together. n <dig> nearly doubled when comparing the assembly of reads segregated by strain during correction to the assembly with reads together during correction. the length of the largest contig also increased significantly.\n",
            "\n",
            "there were five different assemblies. the assembly of raw reads involved no correction. for both quake and echo, the reads were corrected separately by strain and corrected together with reads from both strains present then assembled. the number of contigs, the n <dig> size and the length of the largest contig of each assembly is shown.\n",
            "\n",
            "echo\n",
            "assemblies using echo-corrected reads made improvements to the uncorrected assembly . the improvements were not as drastic as those made by quake. the n <dig> size remained at  <dig> base pairs for both assemblies after using echo. the largest contig created by both assemblies was better than the uncorrected assembly. it was also slightly improved in the assembly where strains were corrected together compared to when strains were corrected separately. the number of contigs also drops when compared to the uncorrected assembly and is additionally improved when the reads were corrected with strains segregated.\n",
            "\n",
            "discussion\n",
            "general trends\n",
            "both error correction software packages increased their rate of correctly correcting errors at heterozygous locations as the heterozygosity rate increased. this may be the case because as heterozygosity increases, the haplotypes become more unique. as the sequences become more unique, the error correction algorithms are able to treat the homologs as unique sequences. as can be seen, reads from an organism with high heterozygosity were error corrected better than reads from an organism with low heterozygosity. however, many interesting organisms, such as humans, have low heterozygosity  <cit> .\n",
            "\n",
            "quake\n",
            "quake differed from echo by its conservative nature when choosing whether or not to error correct. this trend manifested itself in the high rate of errors at heterozygous positions that quake did not correct when heterozygosity was low. at lower heterozygosity rates, the variations between homologs were sparse. there were, therefore, fewer surrounding heterozygous markers to indicate a proper correction for an error. quake was less certain of how to correct a putative error at these low heterozygosity rates and preferred to leave them uncorrected. a benefit of this can also be seen by the lower rate at which errors were introduced into the readset by quake . fewer errors being corrected is undesirable but also results in fewer introduced errors.\n",
            "\n",
            "quake has a low rate of correcting heterozygous errors to the wrong haplotype. additionally, it did not introduce errors at non-error heterozygous positions by correcting them to the wrong or neither haplotype . additionally, few chimeric reads were present after correction . the low levels of introduced errors and chimeric reads can be attributed to conservative correcting and trimming of reads.\n",
            "\n",
            "trimming reads may have given quake an advantage in introducing few errors and creating few chimeric reads. a downside to this, however, can be seen by looking at the proportion of reads that contain >  <dig> heterozygous marker. as heterozygosity increases, quake has much fewer reads with >  <dig> heterozygous marker compared to echo . reads with >  <dig> heterozygous marker are essential to haplotype specific genome assembly. variants contained within the same read can be used to associate them with each other. this association of variants can be used to form a haplotype  <cit> . upon further analysis, approximately 60% of all errors in reads that were corrected were removed through trimming . 40% of these errors were at heterozygous sites when an error rate of approximately  <dig> % was used.\n",
            "\n",
            "echo\n",
            "one of the strengths of echo is that no user input parameters are required in order to run. there is also no trimming. this leaves entire reads intact except for when corrected. the lack of trimming may or may not be a weakness depending on the needs of the user.\n",
            "\n",
            "echo showed itself to be aggressive in terms of error correction. this differentiates it from quake's conservative tendencies. aggressive correction proved to be a good feature when presented with homozygous data. however, it was detrimental when heterozygous data was present. at low heterozygosity rates, echo corrected errors and non-errors at heterozygous positions to the wrong or neither haplotype approximately 10% of the time in the worst cases at the lowest error rate. echo corrected many errors that quake did not correct at low heterozygosity rates. however, it also introduced more errors into the readset. the effects of introducing errors at heterozygous positions can be seen in the number of chimeric reads after correction.\n",
            "\n",
            "echo produced more chimeric reads than quake . as discussed earlier, this feature can be detrimental for downstream use of these corrected reads. the most likely reason for this increase in chimeric reads is the lack of read trimming in echo when errors are encountered near the end of a read. this leaves all heterozygous positions present in a read. having more heterozygous positions present is a key feature for haplotype-aware studies. compared to quake, the number of reads that contain >  <dig> heterozygous base is higher, especially as heterozygosity increases . though more reads may be chimeric, the higher number of reads with multiple heterozygous markers may be beneficial.\n",
            "\n",
            "real sequencing data\n",
            "using either quake or echo improved the real sequence data genomic assemblies. the overall number of contigs returned after assembly were decreased on all error-corrected assemblies. this is important because with fewer returned contigs, the scaffolding graph is much less complicated. more improvements can be found when using quake compared to echo. it is unclear how much of the improvement that was found after using quake is attributed to read trimming or due to the correction of errors. the improvements seen in all error-corrected assemblies shows how error correcting can improve downstream analysis. the assembly created from reads corrected by quake when the reads were separated by strain was significantly better than the assembly produced with mixed reads. a haplotype-aware error correction algorithm should be able to achieve this level of performance.\n",
            "\n",
            "CONCLUSIONS\n",
            "neither quake nor echo performed adequately as an error correction algorithm on heterozygous data. an acceptable error correction algorithm for heterozygous reads would provide results similar to figure  <dig> when correcting a heterozygous dataset.\n",
            "\n",
            "echo is a very aggressive corrector. quake, on the other hand, is much more conservative. aggressive correction provides the benefit of correcting many of the errors. it also results in the introduction of errors. many of these errors occur at heterozygous positions. conservative correction results in more of the original errors persisting through the correction phase but fewer new errors being introduced.\n",
            "\n",
            "an important distinction between the original errors and introduced errors should be made. original errors are much less detrimental to downstream analysis than introduced errors. introduced errors are placed in non-random locations and decrease the ability to compensate for these errors through high coverage.\n",
            "\n",
            "in addition to correction accuracy, the number of heterozygous markers present after correction should also be considered. because quake trims reads, a large portion of heterozygous bases were removed from the datasets. this created fewer errors but left fewer heterozygous markers in the corrected dataset. echo, although it had lower accuracy as a result of more introduced errors and mis-corrections of read errors, left all heterozygous positions in the corrected dataset.\n",
            "\n",
            "looking at the effects on real sequencing data and their subsequent assemblies was also enlightening and showed effects of error correction on real data. it is shown that both algorithms improve the assembly. the effects of haplotype-aware error correction can particularly be seen in the case of quake when the reads from each strain were segregated.\n",
            "\n",
            "there is much improvement that can be made to error correction algorithms to better handle heterozygous datasets. haplotype-aware correction algorithms should be able to enhance genome analysis. other approaches also include read classification and segregation by haplotype before error correcting. segregating the data creates datasets that resemble homozygous data. thus, many current bioinformatic tools could be used on heterozygous datasets that are segregated by haplotype.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "both error correction software packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of read errors.\n",
            "both error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of read errors.\n",
            "both error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of read errors.\n",
            "both error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of reads.\n",
            "both error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of reads.\n",
            "both error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of reads.\n",
            "both error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-corrections of reads.\n",
            "both error correction packages increased their rate of correctly correcting errors at heterozygous sites as a result of more introduced errors and mis-\n",
            "3\n",
            "[{'summary_text': 'the bovine hapmap project was started with the goal of understanding the genetic structure and history of an important livestock species.\\nthe bovine hapmap data were collected from  <dig> breeds of taurine, zebu, zebu-taurine and sanga-taurine.\\nthe bovine hapmap data were collected from  <dig> breeds of taurine, zebu, zebu, sanga-taurine and sanga-taurine.\\nthe bovine hapmap data were collected from  <dig> breeds of taurine, zebu, zebu-taurine and sanga-taurine.\\nthe bovine hapmap data were collected from  <dig> breeds of taurine, zebu, zebu, zebu-taurine and sanga-taurine.\\nthe bovine hapmap data were collected from  <dig> breeds of taurine, zebu, zebu, zebu, sanga-taurine and sanga-taurine.\\nthe bovine hapmap data'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the goal of genome wide analyses of polymorphisms is to achieve a better understanding of the link between genotype and phenotype. the study of a large number of polymorphisms spread across the genome will reveal aspects of the genetic structure of the population, including, in some cases, evidence of adaptive selection across the genome  <cit> . furthermore, if the individuals in the sample are measured for a range of traits, genome wide association  studies between the polymorphisms and the trait values can lead to the genetic dissection of traits  <cit> . this applies in particular to complex traits, where genetic and environmental factors combine to produce the phenotype  <cit> . a concordance between snp showing evidence of genetic selection and association to a trait may help define the phenotype that is under positive selection and may provide some evidence to support the association  <cit> , assuming that samples from populations that segregate the genetic variability in question have been included. there are studies of a few genes that give credibility to the approach  <cit> .\n",
            "\n",
            "genome wide studies of genetic selection  are generally performed separately to gwa despite the potential advantages of combining the information. the reasons for this separation are mainly operational. gwa studies are sampled to study a set of traits, and population stratification is avoided or tightly controlled  <cit> . often the studies are restricted to a particular population. much effort goes into replication of results in an independent sample of the same or similar populations, with a strong effort to perform meta-analyses across data sets. alternatively, studies of selection are recommended to use the most highly differentiated populations available  <cit> . these are not necessarily the best populations for gene discovery for any particular trait. one example is the human hapmap project, which reported the sampling of  <dig>  ×  <dig> single nucleotide polymorphisms  across the genome  <cit> , studied in three highly divergent population samples. many genomic regions showed signatures of selection, some identified using more than one method of analysis. however, it is not known what confidence could be placed in these signatures of selection if a different set of divergent populations was used, or if intermediate populations were included.\n",
            "\n",
            "the bovine hapmap project was started with the goal of understanding the genetic structure and history of an important livestock species. the expectation was that this study would provide information on the processes of domestication, insights that are not currently available from the study of humans or other animal species. to do this, animals from  <dig> breeds and two outgroup species were sampled from five continents and all three major branches of cattle  <cit> . this large number of groups is quite different to the human hapmap project. the  <dig> animals were genotyped for a panel of approximately  <dig> ×  <dig> snp and a variety of analyses were performed, including a genome wide assessment of positive selection. several methods were used to infer evidence of positive selection with many locations identified by more than one method. these methods were 1) the analysis of population divergence using fst  <cit> , 2) the analysis of ancestral states in conjunction with extended haplotype homozygosity  <cit>  of derived alleles using the ihs method  <cit> , and 3) the modelling of the distribution of allele frequencies along a chromosome under the expectation of a selective sweep analysed using a composite likelihood ratio   <cit> . however, these methods respond to different signals in the data. the lack of concordance is therefore not evidence of the absence of selection. it is unknown what proportion of these signals would be reproducible in another set of population samples.\n",
            "\n",
            "this study had two main aims. the first was to compare genome wide signatures of selection found in the bovine hapmap project with genotypes collected from a similar set of animals for which phenotypes are known  <cit> . the second was to explore the relationship between signals of selection and associations to traits. the traits were intramuscular fatness, residual food intake and meat yield, which have biological and economic importance. they relate to obvious characteristics of the phenotype and are under artificial selection in cattle. the australian samples include eight of the breeds used in the bovine hapmap study. we used fst to study selection partly because 1) it is a robust easily calculated measure with a long history  <cit> , 2) the amount of ancestral information on snp is low in cattle  <cit> , resulting in sparse information for ihs, and 3) the joint minor allele frequency distribution is not known in all breeds making the clr method complex to implement in cattle.\n",
            "\n",
            "methods\n",
            "samples\n",
            "cattle  have three major geographic groupings, the european and north asian taurine, south asian and east african zebu, and west and south african sanga  <cit> . sanga are often referred to as african taurine cattle. australian samples were collected from  <dig> breeds of taurine, zebu, zebu-taurine and sanga-taurine animals and known crossbreds  after the research plans were submitted for ethics approval . informed consent to use the samples was also obtained from the farmers who owned the cattle. the samples consisted of steers of the taurine british beef breeds of black angus , red and white faced hereford , the red shorthorn , and the white australian murray grey , a breed heavily influenced by the angus; the zebu brahman breed , which is usually grey but can be red; the composite sanga-taurine australian belmont red , a composite of the sanga africander cattle of the san and khoi of southern africa and hereford and shorthorn cattle, and the composite zebu-taurine cherry-red american santa gertrudis , a composite of the brahman and the shorthorn. the cows of the dairy breeds consisted of the taurine european brown swiss , black and white european/north american holstein , the channel isle red and white guernsey  and tawny jersey , the red australian illawarra shorthorn , derived from the milking shorthorn, the composite taurine red and white australian red cattle , consisting of illawarra shorthorn, ayreshire, scandanavian red and red holstein bloodlines which is being adapted to australian conditions, and the zebu-taurine composite black and red australian friesian sahiwal , a milking zebu composite, as well as industry cattle of known crossbred ancestry.\n",
            "\n",
            "1breed code is the 3-letter code for each breed, 2data code no is the numerical code for each breed in the analysis, and 3fst group code is the grouping used for the analysis of positive selection. code values of  <dig> mean coded as missing data.\n",
            "\n",
            "the beef cattle samples and their trait measurements have been described previously  <cit> . in short, the beef sample consisted of  <dig> individuals from  <dig> sires chosen to avoid close relatives. there were one to four offspring per sire and a median of one offspring per sire. this sub-sample was chosen from a larger group of  <dig> animals in half-sib pedigrees representing  <dig> sires. the detailed description of the method of choosing the sub-sample has been published  <cit> . the dairy samples were collected as part of an industry survey of cattle tick burden of  <dig> cattle in north eastern australia. the sub-sample of  <dig> animals consisted of offspring of  <dig> sires and  <dig> dams, with a median of one offspring per sire and a range of 1– <dig> offspring per sire. they were chosen for genotyping from  <dig> properties where tick counts had been obtained over several seasons. the animals were chosen to be from as many breeds and sire lines as possible, irrespective of their tick burdens. they were also chosen because they had full pedigree records in the australian dairy herd information service  database with full milk yield performance. because of this, obtaining equal numbers for each breed with such a specification was not possible, because the breeds do not have equal representation in australia. aliquots of  <dig> μl of blood were processed to dna using the qiaamp dna mini kit using the manufacturer's instructions .\n",
            "\n",
            "the bovine hapmap sample, genotypes and quality control have been described previously  <cit> . in brief, samples were obtained for the taurine beef breeds angus, charolais, hereford, limousin, piedmontese, red angus, and romagnola; the taurine dairy breeds brown swiss, guernsey, holstein, jersey, and norwegian red; the zebu brahman, gir, and nelore; the sanga n'dama breed; the zebu-taurine composite breeds of beefmaster and santa gertrudis; and the sanga-zebu composite sheko breed. these samples were genotyped for approximately  <dig> ×  <dig> snp derived from the bovine genome sequencing project. each breed sample consisted of  <dig> animals including two trios of sire, dam and offspring. the rest were animals as unrelated as possible using a pedigree analysis. the exceptions were the holstein, the limousin and the red angus, which had samples of  <dig>   <dig> and  <dig> respectively. for fst calculation the offspring were excluded. the correlation of fst with and without these offspring across all loci was r =  <dig> . a link to the snp can be found at  associated with the baylor college of medicine and the description of the snp is subject to the fort lauderdale agreement  <cit> . the order and distance of snp along the bovine genome was taken from the current assembly in genbank .\n",
            "\n",
            "genotyping\n",
            "the genotyping of the australian sample was performed using the megallele™ genotyping bovine  <dig> k snp panel  <cit> , a fully described set of snp, by parallele inc. on an affymetrix genechip scanner  <dig>  yielding an average spacing of  <dig> kb between snp. further details of the snp can be found at the link . the bulk of the snp on the snp array were obtained by comparing the genome sequence of the hereford animal to the partial sequence of the holstein  and the angus  animal, with  <dig> % csnp  obtained from the interactive bovine in silico snp database  <cit> , and the rest from the partial sequence of the limousin  and the brahman  animal. the major difference in breed origin of snp in the two snp sets is an increased proportion of snp obtained by comparing the genome sequence of the hereford animal to the partial genome sequence of the norwegian red , the brahman  and the jersey  animal, with a consequent decrease in the percentages of snp obtained from an animal of another breed  <cit> . in summary, the snp in both snp sets are mainly the pairwise differences between a taurine beef and dairy animal, with either a small or a moderate percentage of snp that are differences between a taurine and a zebu beef animal. the beef and dairy samples had been genotyped separately so the combined data set was rescored by parallele to ensure that genotypes were consistent. data quality assurance was investigated using duplicate samples  and the genotyping of a selection of loci using two alternative technologies. repeat genotyping of the same individual showed a  <dig> % concordance rate. all samples with more than 10% missing data were excluded and then all loci with more than 10% of missing data were excluded. the rationale for this is that dna that is not of high quality will be more likely to have incorrect genotype calls and have more missing data. the same will apply to snp with poor assays, which are more likely to have false or indeterminate calls, and so are also more likely to have missing data. the rejection of 10% was not arbitrary; the number of loci excluded was plotted against percent missing data and the cut off point was determined as the point in round numbers along the curve where the rate of change in the number of excluded loci became linear with every increased percent of missing data. this left  <dig> snp out of the panel of  <dig> scored snp. these snp were tested for departures from hardy-weinberg equilibrium within breed, and  <dig> % of comparisons were significant at the 5% level.\n",
            "\n",
            "analysis\n",
            "to determine the clustering of individuals in this study, a principal component analysis  was performed on the individual genotypes of the  <dig> animals using the  <dig> snp that had no missing data. the data were analysed using a covariance matrix in splus  <cit> . full records, without missing data, were used because individuals with missing data may appear as outliers  <cit> . the reduction in the number of loci has the benefit of reducing the correlation between genotypes due to linkage disequilibrium ; the average gap between adjacent loci increased from  <dig> kb to  <dig>  mb and the number of snp separated by less than  <dig> kb dropped from  <dig> to  <dig>  the first principal components  were highly significant and these were plotted against each other, encoding each coordinate with the breed or crossbred identity of the animal as contained in the database. during the analysis we found an angus and a murray grey individual clustered with the zebu-taurine composite animals. those animals were found to have a mismatch between the dna sample and breed designations in the beef crc database. they were excluded from the fst analysis although we do not expect that the exclusion would make much difference to the analysis because of the small numbers involved and the amount of variation explained by each principal component. the clustering was performed for two reasons. firstly, lumping of breeds was necessary in the dairy animals because some of the sample sizes were small . it has been shown that it is best to avoid small samples and to use samples of equal size when comparing estimates of fst  <cit> . having similar groups will generate less variable estimates of fst, while lumping groups that have divergent genotypic frequencies should act to reduce fst. nevertheless, it is the distribution of fst across the genome not the absolute value that is important in identifying evidence of positive selection  <cit> . secondly, the clustering identifies the most divergent breed samples in a convenient graphical output. comparison of the most divergent two to four groups will encompass most of the variability captured by fst  <cit> . the analysis of a common set of divergent breeds will allow a baseline comparison between the bovine hapmap and this study.\n",
            "\n",
            "to determine a genome wide pattern of positive selection, and to compare this to the bovine hapmap sample, the fst at each locus was calculated  <cit>  in the same way as in the bovine hapmap sample. all fst values in this study are per locus values. means and standard deviations were calculated not to find an unbiased average across all the data but to measure the dispersion of per locus fst in the sample. snps that were genotyped in all breeds were used. fst, the average heterozygosity and average allele frequency for each snp were plotted . examination of this figure allows one to see whether the fst values were likely to be affected by significant patterns in genotypic frequencies. the fst values were then plotted against genome location and values were averaged over  <dig> snp in a sliding window, as in the bovine hapmap project. signatures of selection can be recognized when adjacent snps all show high fst  <cit> , due to the hitch-hiking effect  <cit> , implying divergent selection between breeds, or where adjacent snps all show low fst, implying balancing selection between breeds  <cit> . several comparisons were made, varying the breeds and the snp that were included. the baseline comparison was for the three breeds angus, brahman and holstein, which are the most divergent samples in the pca in this study, and which were sampled in the bovine hapmap study. the pearson correlation coefficient was calculated between the fst values from this study and those from the bovine hapmap study. the fst values in the top and bottom  <dig> % of the distribution in both data sets were compared. to determine whether these loci were significant, the confidence limits for fst were calculated for both data sets. to obtain confidence limits for the per locus value of fst,  <dig>  bootstrap samples were taken each consisting of  <dig>  fst values sampled at random from the loci within each data set. the average fst was calculated for each bootstrap sample and the standard deviation of the resulting average fst values was used to set confidence limits  <cit> . to describe the relationship between the location and diversity along a chromosome, the difference between the fst  of pairs of immediately adjacent snp  was compared to the distance in base pairs between them. each interval between snp pairs was used once. snp pairs were binned into those below  <dig> kb, then into  <dig> kb bins up to  <dig> kb, and then into  <dig> kb bins up to  <dig>  mb. the mean δfst and standard error were plotted against mean distance between the snp pairs of each bin.\n",
            "\n",
            "the additive and allele substitution effects for the traits residual feed intake , percent meat yield , and percent intramuscular fat  in beef cattle were calculated as previously described and results published elsewhere  <cit> . in brief, residual trait values were obtained from a mixed model that incorporated fixed environmental effects as well as the random effect of breed  and sire, thereby accounting for possible stratification in the data, using asreml  <cit> . the additive effect  is half the distance between the two homozygotes, the dominance effect  is the difference between the heterozygotes and the average of the two homozygotes, and the allele substitution effect  = a + d where p+q =  <dig> and p, q are the allele frequencies  <cit> . standard errors were estimated using  <dig> bootstrap samples  <cit> . in this study the estimates for beef traits were reported within taurine breeds  and within zebu plus tropical composites . breed type was used because the sample sizes within each breed would be too small for meaningful analysis. as the trait values were adjusted for the effects of breed and contemporary group, a combined analysis should not produce spurious results. these results were searched using the list of snp that had extreme fst values.\n",
            "\n",
            "the number of association tests with high extreme fst versus low extreme fst was compared to the goodness of fit expectations for the same sample size using a chi-square test. the expected frequency was taken as the ratio of the observed number of snp in the low versus the high extreme.\n",
            "\n",
            "RESULTS\n",
            "before calculating fst, we used a pca to determine how the animals should be allocated to groups and the degree of divergence between samples. using the individual genotypes as the data in the pca , pc <dig> explained  <dig> % and pc <dig> explained  <dig> % of the variance. we found that pc <dig> separated out the zebu brahman breed from the taurine breeds. animals with known recent part zebu ancestry, such as the santa gertrudis and the australian friesian sahiwal, were also separated from the taurine breeds along the pc <dig> axis, although not to the same extent as the brahman. the belmont red, which in principle does not have recent zebu ancestry but in practise may have a small percentage of brahman ancestry, was also separated along pc <dig> to the same extent as the santa gertrudis. the taurine breeds were partly separated along pc <dig>  in general the animals of one taurine breed clustered together, but the clusters partly overlapped those of other breeds located near by. animals of the holstein dairy breed were located at one end and animals of the angus beef breed were located at the other end of the spread of breeds along pc <dig>  the hereford breed occurred at the intersection of the axes of pc <dig> and pc <dig>  the locations of the angus, hereford and holstein are consistent with the process of snp discovery, where most snp in the  <dig> k snp set were obtained by comparing the holstein and angus to the hereford. the complete overlapping of the angus and murray grey was expected given the role of the angus in the development of the murray grey. the slight separation between the angus and murray grey breeds and the other taurine breeds may be due to the absence of a full range of breeds rather than any particular distinctness of the angus and murray grey from the others. the breeds that were furthest apart on these two axes in this study are the angus, brahman and holstein. based on the pca, the dairy animals were lumped into four groups of reasonable size for the fst analysis, as indicated in table  <dig>  the australian friesian sahiwal were excluded from the fst calculations because of their small sample size and lack of affinity to other breeds.\n",
            "\n",
            "comparing this plot of pc <dig> and pc <dig> to the plot of the first two components in the bovine hapmap study, pc <dig> also separated the zebu from the taurine breeds in that study while pc <dig> partly separated the taurine breeds as a series of partly overlapping clusters. pc <dig> clearly separated the one sanga breed, the n'dama, from the zebu breeds, to the same distance as that between the zebu and taurine breeds. pc <dig> clearly separated the taurine from sanga breeds. there were no purebred sanga animals among the australian samples. the one major difference was the hereford breed in the bovine hapmap study, which was located well away from the other breeds as a loose, flat cluster. of the other taurine breeds in the bovine hapmap study, the angus and holstein were the furthest apart in the plot of pc <dig> and pc <dig> \n",
            "\n",
            "to characterise the differences in per locus fst between this and the bovine hapmap study, we compared several groupings of breeds and loci . there were minimums of  <dig> snp in the bovine hapmap data and  <dig> snp for the australian data with a per locus fst value when all populations were used. the australian data showed lower per locus fst values than the bovine hapmap data with a lower amount of dispersion around this mean value. in both studies, the mean per locus fst was larger when only the three divergent breeds angus, brahman and holstein were used. in the bovine hapmap data this was so whether all loci or only the loci in common to both studies  were used. the difference in mean per locus fst value between the australian sample and the bovine hapmap sample was less when only the three divergent breeds were compared. although these mean differences are small, they are all significant because of the large samples of loci used. the dispersion of fst values was also greater in the three breed comparisons than when all breeds where used. to determine if removing some breeds would make a difference to the estimates, we removed the two african breeds, n'dama and sheko, and the two zebu breeds, gir and nelore, from the bovine hapmap data. removing the n'dama and sheko removed animals that were highly divergent on pc <dig> of the bovine hapmap data, leaving only the partly overlapping european breeds. removing the gir and nelore reduced the number of zebu breeds to one. this made the breed composition more similar in the two studies. it also still included the angus, brahman and holstein. for all loci in the bovine hapmap data the average fst =  <dig> , s.d. =  <dig>  . this showed a reduction in the per locus fst, of nearly half the difference between the mean per-locus fst value of the full bovine hapmap and australian samples. this reduced set of breeds still showed the lower dispersion in per locus fst values found in the full sample of breeds compared to the three breed estimates.\n",
            "\n",
            "comparison\n",
            "to determine whether the global pattern of fst across the genome was the same in both this and the bovine hapmap study, the windowed fst was plotted against location in the genome . there were obvious differences in the locations of the major peaks. the difference in height along the ordinate was consistent with the average difference found for fst . in particular, locations on chromosomes  <dig>   <dig>   <dig>   <dig> and the x were obviously different. the windowed fst values between the two data sets at each locus were correlated with r =  <dig> , n =  <dig>  p =  <dig>  this indicated that genome location explained  <dig> % of the variance in that comparison.\n",
            "\n",
            "the relationship between genome location and fst was explored in more detail by using subsets of breeds and subsets of loci to determine which had the more important influence on whether a signal appeared in a particular location. using only the loci in common with all breeds in both data sets, the un-windowed fst values showed a correlation of r =  <dig>  . due to the number of common loci or common windowed points in common, n =  <dig>  all of the correlations reported below are highly significant, with p <<  <dig> . for the subset of three divergent breeds the un-windowed fst values showed a correlation of r =  <dig> , or  <dig> % of the variance in fst . the three breed comparison had a broader range of fst values. comparison of the three breed to the all breed correlation for un-windowed fst values showed that changing the breed composition in this experiment resulted in a  <dig> % decrease in the amount of the variance in fst explained across the genome.\n",
            "\n",
            "to determine the effect of the specific loci on the distribution of fst, we compared the windowed fst values in this study to the windowed fst values in the bovine hapmap study. the windows are for  <dig> adjacent loci so the composition and density of loci contributing to each reference point along the genome differed in the two data sets. for all loci the correlation was r =  <dig>  for the all breed fst values and r =  <dig>  for the three breed fst values. comparing the correlation of three breed windowed fst values of the bovine hapmap data and the australian data to the correlation of the un-windowed values for those data shows a reduction in the variance explained of 76% . comparing the all breed windowed to un-windowed fst values in the same way shows a reduction in the variance explained of 68% . this reduction was smaller than the three breed comparison but the full breed comparison includes not only differences in snp between studies but also differences in the number and composition of breeds.\n",
            "\n",
            "to determine the importance of differences in snp density between the two studies, windowed fst values for bta  <dig>   <dig> and  <dig> were compared to the other chromosomes. these three chromosomes have 2– <dig> times higher density than the other chromosomes in the bovine hapmap data, which in their turn have a 2– <dig> times higher density in the bovine hapmap data than in the australian data. for all breeds, comparing the australian to the bovine hapmap data, the correlations were r =  <dig>  for comparisons of bta <dig>   <dig> and  <dig> combined and r =  <dig>  for the other chromosomes combined. if the loci that are windowed are only the common ones between the two studies, that is, no difference in density, then the correlation between windowed fst values is r =  <dig> , essentially the same as the un-windowed values .\n",
            "\n",
            "we calculated the confidence interval for the per locus fst to be considered significant using bootstrap sampling. the  <dig> % confidence interval for the per locus fst in our data was  <dig>  ±  <dig>  while the confidence interval in the bovine hapmap data for the same loci was  <dig>  ±  <dig> . in the australian sample, the top  <dig> % corresponded to a threshold fst =  <dig>  and the bottom  <dig> % corresponded to a threshold fst =  <dig> , both outside the confidence interval for that dataset. in the bovine hapmap data, the top  <dig> % corresponded to a threshold fst =  <dig> , which is outside the  <dig> % confidence interval and the bottom  <dig> % corresponded to a threshold fst =  <dig> , which is outside a 99% confidence interval for that dataset. there were  <dig> loci that had an fst above the upper thresholds in both data sets, or  <dig> % of  <dig> snp. there were  <dig> loci that had an fst below the lower thresholds in both data sets. for the snp above the threshold in both data sets, the  <dig> snp were located in  <dig> genomic regions of  <dig> mb containing one or more snp with high fst . for the snp below the threshold in both data sets, almost none of those snp were close to another snp with low fst. some of the low fst values are negative, which may occur when estimating an fst value near zero, but which may also occur in some cases where the expected variance calculated from the average allele frequency for the entire sample is sufficiently less than the sample variance in allele frequencies across populations, or which may occur in some cases where the heterozygote frequencies within populations are higher than expected given the allele frequencies.\n",
            "\n",
            "the average δfst between the snp pairs was strongly influenced by the distance between them. the δfst increased quickly with distance and reached a plateau after approximately  <dig> kb . the average δfst associated with the bin sizes of < <dig> kb, 1– <dig> kb and 10– <dig> kb were all significantly different from each other, and each increase in average distance was accompanied by an increase in average δfst until the plateau was reached. the plot showed broader standard errors for comparisons in the 20– <dig> kb region, but this was due to smaller numbers of snp separated by those gaps. for example, there were  <dig> snp pairs separated by <  <dig> kb in that plot,  <dig> pairs separated by 10– <dig> kb, and between  <dig> and  <dig> pairs in each of the  <dig> kb bins from 20– <dig> kb. the plot shows that the bins between 20– <dig> kb had similar means. for the combined bin 20– <dig> kb, corresponding to a mean separation of  <dig>  kb between snp pairs, the mean δfst =  <dig> , se =  <dig> , n =  <dig> \n",
            "\n",
            "fourteen of the  <dig> snp with extreme fst in both data sets had an effect on one of the three traits , six of which had an effect on more than one trait . the same homozygote in one of the six snp increased both yield and imf, and the same homozygotes in the remaining five snp increased both imf and rfi. the effects that were significant for more than one trait were all in the same breed type. three of the six snp were located to one small region of bovine chromosome  <dig>  counting these three as one independent locus, four  of the  <dig>  independent snp had effects on more than one trait, or 33%. only  <dig> % of the snp in the entire experiment that had an effect on rfi also had an effect on imf and only  <dig> % that had on effect on yield also had an effect on imf. this difference in frequency, while large, was not significant . all of these snp had fst values greater than the upper threshold, representing divergence between the breeds. the observed frequency of  <dig> low fst versus  <dig> high fst, compared to the expected frequency of 35: <dig>  was significant . these loci all had much larger fst values in the three breed sub-sample than in the full breed sample.\n",
            "\n",
            "locus dbsnp identifier, chr chromosome position in bp, group breed type, temperate or tropical, n sample size, p <dig> is the allele higher up the alphabet, vr is the residual variance explained by the genotypes, a is the additive effect, sea is the standard error of a, α is the allele substitution effect, seαis the standard error of α.\n",
            "\n",
            "rfi residual food intake in kg per day, yield is the meat yield of the animal as a percentage, and imf is the intramuscular fat percentage\n",
            "\n",
            "the region on bta2: <dig>  mb that showed effects on two traits was examined in greater detail. there are eight snp in this region separated by a total of  <dig>  kb set in a region containing  <dig> snp separated by a total of  <dig>  mb. thirteen of the loci  showed a significant additive or allele substitution effect on rfi or imf in either temperate, tropical or the combined sample. six of the  <dig> snp showed an association but the fst value was within the confidence interval for the data set. nine of the  <dig> snp showed an association and had average allele frequencies of between  <dig>  and  <dig>  across all breeds. only the three snp with the highest fst values had effects on both traits.\n",
            "\n",
            "pave is the average allele frequency of p <dig> across the breeds, rfi is the ratio between the additive effect and its standard error, typerfi is the type of cattle in which the effect was estimated, imf is the ratio between the additive effect and its standard error, typeimf is the type of cattle in which the effect was estimated. tem temperate, tro tropical, ttl total. troα means the effect is the allele substitution effect.\n",
            "\n",
            "discussion\n",
            "in this study we report a high level of concordance between the fst values calculated from this study and the bovine hapmap data for the same snp and breeds. when the same divergent breeds, namely, the angus, brahman and holstein, are compared, the correlation in per locus fst values between the australian data and the bovine hapmap data was high. in both data sets we found that calculating fst using a large number of breeds resulted in per-locus fst values that were not as broadly dispersed around a mean value as those fst values calculated using only the three divergent breeds. this suggests that using many breeds or population groups acts to reduce the variability in estimated per-locus fst values.\n",
            "\n",
            "however, there are major differences between the two studies in which regions of the genome show the most divergence when all breeds and all loci are used. many of the strongest signals of selection in the bovine hapmap data were not found in our data, and the pattern of the signal across chromosomes was visibly different. the similarity between the fst values declined significantly when all breeds were used, amounting to 40% of the variance compared to when only the three divergent breeds were used; there are only eight breeds in common to both studies. the lower correlation with different breeds may suggest that each study had identified signals of divergence particular to the genetic history of those breeds, some of which may be due to selection. the relationship between fst values also decreases substantially when the composition of snp differed, amounting to 76% of the variance in these data sets when the three divergent breeds are compared.\n",
            "\n",
            "as there were only  <dig> loci in common to both studies, a large proportion of the difference in fst values between two studies will be due to different snp compositions in the data. for the common loci, the identity and density are the same, and, because the common loci form most of the loci in the australian set, density and identity are confounded when there is no difference in density between the two studies. nevertheless, above that baseline, the three higher density chromosomes of the bovine hapmap data , showed essentially the same relationship to the australian data as the rest of the data set. this suggests that snp identity was more important than density in explaining the differences between the two studies. the similarity between the fst values between the two datasets declined significantly when all breeds and snp were used, amounting to 68% of the variance in that comparison. this is less than the 76% when only the angus, holstein and brahman breeds of the two data sets are used. this difference suggests an interaction between breeds and snp. the composition of snp in the two panels is neither of uniform ascertainment nor of exactly the same ascertainment as each other. it is not uniform because snp were primarily generated by comparing a small number of animals, each of a different breed, to the hereford sequence, and it was different, because there were slightly more breeds used to define snp in the bovine hapmap data and a much larger percentage of zebu snp  <cit> .\n",
            "\n",
            "the first two axes of the pca separated breeds in a similar way in the australian and the bovine hapmap data with one major exception. the similarities are that the first axis clearly separated zebu from taurine and the second axis partially separated meat and dairy within european breeds in both data sets. the first axis also clearly separated zebu from purebred sanga in the bovine hapmap data, which were at the same co-ordinates on pc <dig> as the european taurine group. the second axis in the bovine hapmap study clearly separated the sanga  from the european taurine. the differences between european cattle were minor compared to the differences between european and african cattle. based on the bovine hapmap data, for diversity studies, a minimum three breed panel would probably have to consist of one taurine, one sanga and one zebu breed. sanga breeds are not common in the developed world. for breeds that are commonly available one could identify the angus, brahman and holstein as a minimum group for studies of breed or locus divergence based on the similarity between the two studies. however, the greater difference between angus and holstein compared to other european breeds may be a consequence of the snp ascertainment, and further study might be needed before a minimum panel of common breeds could be specified. the major difference between the two studies was that we found that the reference breed for snp discovery, the hereford, was located exactly where it was expected, at the intersection of pc <dig> and pc <dig>  because all breeds would in effect be compared to it . this location of the hereford cluster differs substantially from the analysis reported in the bovine hapmap study. there the hereford formed a flattened outlier group well separated from the other breeds. in that report, when only the loci originating from the jersey breed were analysed in the pca, the jersey animals were clustered outside the main group of taurine animals. this may indicate that the shifting clusters could be a generic factor associated with the analysis. while we have not explicitly modelled causes for the difference by performing a pca on the bovine hapmap data ourselves, if that explanation is correct then it may suggest that the use of a very large number of loci may lead to the unusual clustering of animals that acted as the source of the polymorphisms.\n",
            "\n",
            "higher densities of snp will be required to study genome wide signatures of selection in cattle. ld causes correlation between genotypes  <cit> , so fst values that are more similar at a close distance are likely to be similar due to ld. our results show that snp separated by more than  <dig> kb have δfst the same as those separated by more than  <dig> mb. this is consistent with the decay in ld as measured by r <dig> for these  <cit>  as well as the bovine hapmap data  <cit> , where r <dig> reaches a lower asymptote at around 20– <dig> kb. these results suggest that for cattle, fst values that are consistently high or low over distances much greater than  <dig> kb are likely to be due to factors other than baseline ld, and presumably some of these will be due to positive selection  <cit> . the density of markers required to demonstrate similarity beyond  <dig> kb implies snp spaced at less than  <dig> kb, so that the relationship of δfst and distance between snp can be quantified. it will require at least  <dig>  evenly spaced snp to quantify genome wide signals in cattle. as snp are not evenly spaced in current snp chips, it would require several fold more snp than that to explore all regions at that minimum density.\n",
            "\n",
            "the list of the snp with the most divergent fst values is a resource for the further investigation of the effects of positive selection on the bovine genome. comparing ends of the distribution in two studies does not necessarily mean that those values are more divergent than expected  <cit> . due to the correlation of genotypes in two samples from the same population, very similar fst values can be expected unless there are differences in the populations or errors in the way the genotypes are collected. instead, the loci in this list have fst outside the bootstrap confidence limits  <cit>  for both data sets. this indicates that they are unlikely to be merely the result of stochastic events during the evolution and domestication of cattle. one would predict that the process of domestication and breed formation would result in more divergent selection than balancing selection. this is because breeds would be formed and then selected for particular characteristics. breeds are selected to be different to each other. we found that there were more than twice as many divergent than convergent loci when the australian and bovine hapmap data were compared to each other, which is consistent with that prediction. these snp represent genomic regions that should be of general interest, because they represent the overlap of two studies that do not have the same breed composition. many of these loci are located within  <dig> mb of each other. one question that greater snp density will be able to answer is the detail of differences in fst around these snp. that should help define groups of divergent or convergent fst, which in turn would help to define the size and shape of these prospective signatures of positive selection.\n",
            "\n",
            "there appears to be significantly more loci with very high fst that are associated with rfi, yield and imf than those with very low fst. imf is strongly correlated to overall fatness of the animal and is negatively correlated with yield, larger rfi values have a positive relationship to imf and a negative relationship to measures of muscle area in cattle, while the mean values for these traits show large differences between breeds  <cit> . yield, fatness and efficiency have been valued for hundreds of years in some breeds  <cit> . much of this divergence is expected to be the result of artificial selection, because major differences in size and shape are known to have occurred during the domestication of cattle when compared to the extinct wild aurochs  <cit> . these breeds each have a specification of what the animals should look like and how they should perform that includes yield and fatness. one would therefore expect that some of the loci associated with these traits should show strong differences in genotype frequency between breeds, which would show up in a calculation of fst. it is interesting that of the snp with divergent fst five out of six of those with effects on more than one of these traits have effects that are consistent with the known correlations between rfi, yield and imf.\n",
            "\n",
            "examination of a region with enough density of snp shows that association to the trait is not just found for markers with high fst. associations are found for snp that do not have highly skewed allele frequencies, and associations are not just limited to a particular subsection of the sample. this suggests that these results are not merely a spurious intersection of high fst and trait association. functional evaluation of these and other snp will be needed to show where the causative mutations lie in this region. full genetic evaluation of these regions will require more snp, larger samples and more breeds. nevertheless, our data show that signatures of selection are a useful adjunct to trait mapping.\n",
            "\n",
            "the region associated with the peak in fst on bovine chromosome  <dig> contains several genes that have been associated with selective sweeps in humans. the genes r3hdm <dig>  and zranb <dig>  are associated with these cattle snp. the functions of these genes are inferred based on their sequence structures, and references to them point to the bioinformatics literature . for these particular snp with high fst values, most breeds are homozygous for the same allele, but the hereford, the santa gertrudis and the belmont red have a moderate allele frequency of the alternative allele. the hereford breed has been selected for efficiency and growth from its inception as a breed in the 18th century and continues to be more efficient than other british beef breeds  <cit> . polymorphisms in this region may therefore contribute to the differences seen in the hereford. this region has long been known to be associated with positive selection in humans due to the lactase gene  and human adaptation to drinking milk in adulthood  <cit> . it is unlikely that cattle are selected for lactase persistence because breeding stock are invariably weaned. recently, r3hdm <dig>  <cit>  was also shown to be under positive selection in european humans, it is not divergent just through hitchhiking to lct. these results in cattle provide a starting point for the kinds of trait that might be associated with the signal of selection in humans.\n",
            "\n",
            "CONCLUSIONS\n",
            "comparing the australian and bovine hapmap samples we found differences in the presumptive selective signatures when different breeds or snp are used. this leads to the conclusion that a full description of signatures of selection in a species will require a large number of populations to be sampled, not just those that have the most divergent genotypes or phenotypes. in addition, snp obtained from different genetic sources should also be used. a large amount of the variance in fst across the genome is due to differences in snp composition between studies, which suggests that evidence for selection in a region depends on the snp that are included in a study. it also suggests that when the same genetic region is compared in different studies the same snp should be used otherwise a large difference between selection signatures may be reported. these results also suggest that genome wide studies of selection are a useful adjunct to genome wide association studies. we found that useful signatures of selection can be obtained even when many of the sub-samples are not particularly divergent or cannot be clearly separated using a principal component analysis. if a sample used in a genome wide association study was genetically stratified, this stratification could be used to analyse gene frequency or fst distributions in sections of the genome in different breeds or populations. such an analysis may provide evidence for a selective signature or may even help to identify which populations are likely to contain the qtl even before functional nucleotide polymorphisms were identified. while many real qtl will not be accompanied by a signal of selection, such a signal will indicate a clear relationship between selection on some aspect of the phenotype and the effects of variation at a locus. the identification of the trait under selection will require animals of the appropriate breed or group measured for a wide range of traits. the use of evidence of presumptive selection on a genome wide scale means that panels of snp will need to be several fold larger in cattle than currently available. although several panels are now available that include  <dig> – <dig>  ×  <dig> snp in cattle , given the variation in fst with distance between snp, two to three times as many evenly spaced snp will be needed to study signals of selection in all regions in detail.\n",
            "\n",
            "authors' contributions\n",
            "wb planned and performed the analyses, and drafted the manuscript, bh bled cattle, extracted dna, rb bled cattle, extracted dna, mt extracted dna, lt, obtained cattle, led the bleeding of cattle. all authors reviewed the manuscript.\n",
            "\n",
            "supplementary material\n",
            "additional file 1\n",
            "plot of experiment-wide statistics for allele frequency, average heterozygosity and fst. two plots one above the other, the upper plot shows the average allele frequency plotted against the average heterozygosity for each snp, the lower plot shows average heterozygosity plotted against fst for each snp.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 2\n",
            "snp with extreme fst values in both the bovine hapmap and australian data sets. a table consisting of six columns listing the dbsnp identifier, the chromosome, the position in bp in the btau  <dig>  assembly, the per locus fst in the australian and then the bovine hapmap data, and the number of  <dig> mb blocks per chromosome with loci with high or low fst.\n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements\n",
            "this manuscript is a companion paper to the bovine hapmap project. we thank the bovine hapmap consortium for access to the bovine hapmap genotypes prior to publication. the comments of three anonymous reviewers greatly improved the analysis and manuscript. we thank the owners of the beef crc database, commonwealth scientific and industrial research organization, the queensland department of primary industries and fisheries, the new south wales department of primary industries and the university of new england, for access to the records and dna of the australian samples. dairy australia, and the members of the cooperative research agreement, csiro, qdpi&f, nswdpi and une, meat and livestock australia and catapult genetics, co-funded the collection of genotypes.\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the bovine hapmap project was started with the goal of understanding the genetic structure and history of an important livestock species.\n",
            "the bovine hapmap data were collected from  <dig> breeds of taurine, zebu, zebu-taurine and sanga-taurine.\n",
            "the bovine hapmap data were collected from  <dig> breeds of taurine, zebu, zebu, sanga-taurine and sanga-taurine.\n",
            "the bovine hapmap data were collected from  <dig> breeds of taurine, zebu, zebu-taurine and sanga-taurine.\n",
            "the bovine hapmap data were collected from  <dig> breeds of taurine, zebu, zebu, zebu-taurine and sanga-taurine.\n",
            "the bovine hapmap data were collected from  <dig> breeds of taurine, zebu, zebu, zebu, sanga-taurine and sanga-taurine.\n",
            "the bovine hapmap data\n",
            "4\n",
            "[{'summary_text': 'sin <dig> is a little-studied gene product of unclear function found in species ranging from mammals to fungi.\\nsin <dig> is encoded by a single gene from a yeast species, caenorhabditis elegans, that is ~ <dig> aa long.\\nsin <dig> was found in two yeast species, s. cerevisiae, and a number of other fungal species, including caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "sin <dig> was originally described as a human protein that modulated ras function in saccharomyces cerevisiae  <cit> . strains of yeast that expressed the constitutively activated ras2val  <dig> mutation had elevated levels of cyclic amp, impaired growth control and were acutely sensitive to heat shock. this phenotype was reversed when the yeast strain was transfected with a cdna  that encoded a then unknown protein. although the authors suggested that the inferred interaction between the jc <dig> product and ras might be fortuitous, they favored the possibility that that the unknown protein either was a true inhibitor of ras or that it was a ras target protein, which when over-expressed, had a protective action. a s. cerevisiae protein encoded by the avo <dig> gene showed distant similarity the human jc <dig> product  <cit> .\n",
            "\n",
            "approximately eight years after the identification of jc <dig> it was again identified, on this occasion in a yeast two-hybrid screen of a schizosaccharomyces pombe cdna library as a  <dig> amino acid protein that bound via polypeptide sequences in its c-terminal  <dig> amino acids to the sty1/spc <dig>  stress activated map kinase   <cit> . a fission yeast strain lacking the sin <dig> gene was sterile, sensitive to multiple types of stress, including heat shock, and had delayed cell cycles compared to a parental strain. sin <dig> acted downstream of activated sty1/spc <dig> and appeared to be necessary for normal function of the transcription factor atf <dig>  a homolog of human atf <dig>  wilkinson et al.  <cit>  found that an apparent full length homolog of sin <dig> from chicken allowed the heat sensitive strain of s. pombe to grow at 37°c, albeit very poorly. moreover, fusion of the first  <dig> amino acids of yeast sin <dig>  with the c-terminal  <dig> amino acids of the chicken sin <dig> sequence protected against heat shock. together, these data showed that sin <dig> functions as a component of the stress-activated sty1/spc <dig> map kinase pathway in s. pombe and that a functional homolog of sin <dig> exists in vertebrates.\n",
            "\n",
            "no further information concerning sin <dig> appeared since the paper of wilkinson et al.  <cit>  until our discovery that the ovine  sin <dig> associated via its c-terminus to the cytoplasmic domain of ifnar <dig>  a subunit of the type i ifn receptor  <cit> , and schroder et al.  <cit>  described transcripts for sin <dig> in human tissue and provided an analysis of the human gene. the latter study confirmed that sin <dig> was relatively well conserved across metazoa and fungi  and was also represented in amoebae, but not in other protozoan species.\n",
            "\n",
            "ovine sin <dig>  which is 88% identical in sequence to chicken sin <dig>  can be co-immunoprecipated with the ifn receptor subunit ifnar <dig> and shows a similar subcellular distribution to the receptor protein when co-expressed in mammalian cells  <cit> . although ovsin <dig> was identified from a cdna present in ovine endometrium and was initially considered to have a role in reproduction associated with the action of ifn-τ on the uterus during early pregnancy in the sheep, it became clear that the sin <dig> gene was expressed in tissues other than endometrium and might have a general role in the action of type  <dig> ifn. in particular, it seemed possible that sin <dig> might link the action of ifn to the stress activated sapk signal transduction pathways. such a linkage has been inferred from earlier studies in which early activation of p <dig> mapk had been noted following exposure of a variety of cell lines to ifn-α, -β, or -τ  <cit> .\n",
            "\n",
            "although the sapk pathway is itself ancient and is found in all the species in which the sin <dig> gene exists, the ifn system of receptors and ligands is restricted to vertebrates. we reasoned, therefore, that an analysis of sin <dig> gene sequences might not only provide insight into the function of sin <dig>  but indicate how the protein evolved to interact with ifnar <dig>  the fact that the sin <dig> gene appears to be expressed ubiquitously, that it is highly conserved across a wide range of taxa, and that it is a likely participant in several important signaling pathways, makes it an intriguing candidate for a functional/evolutionary analysis.\n",
            "\n",
            "RESULTS\n",
            "conservation of the sin <dig> gene from yeast to mammals\n",
            "a combination of searching methods was employed to locate sin <dig> genes in available cdna and genome data bases . sin <dig> sequences were found in two yeast species , the red bread mold  and a number of other fungal species , caenorhabditis elegans, a mosquito species , fruit fly , frog , two fish species , chicken , mouse , rat , human , sheep , cattle , and pig  . no apparent ortholog could be detected in the plant arabidopsis thaliana. nor could sequences corresponding to sin <dig> be found in protozoa other than amoebae and prokaryotic species.\n",
            "\n",
            "\"comments\" briefly describe the methods used to obtain the sequences.\n",
            "\n",
            "the marked dissimilarity in inferred amino acid sequence between sin <dig> from vertebrates and c. elegans , between the two yeast species  and between s. pombe and n. crassa  in the approximately  <dig> aa of overlap suggests that even if homologs existed in plants and prokaryotes they would likely be overlooked by the search methods employed.\n",
            "\n",
            "notes:  <dig>  numbers in the upper-right half above the diagonal are identity percentages for amino acid sequences.\n",
            "\n",
            " <dig>  numbers in the lower-left half below the diagonal are identity percentage for dna sequences.\n",
            "\n",
            " <dig>  numbers below the species names are the lengths of the sin <dig> protein.\n",
            "\n",
            " <dig>  na, not applicable, i.e. no significant similarity was found.\n",
            "\n",
            " <dig>  astericks, significant similarity occurs only in one region of the protein. for details, see the notes below:\n",
            "\n",
            "s. pombe-c. elegans: significant similarity occurs in one region .\n",
            "\n",
            "s. pombe- d. melanogaster : significant similarity occurs in one region .\n",
            "\n",
            "s. pombe- a. gambiae: significant similarity occurs in one region .\n",
            "\n",
            "n.crassa-c. elegans: significant similarity occurs in one region .\n",
            "\n",
            "n.crassa-d. melanogaster: significant similarity occurs in one region .\n",
            "\n",
            "n.crassa-a. gambiae: significant similarity occurs in one region .\n",
            "\n",
            "n.crassa-xenopus: significant similarity occurs in one region .\n",
            "\n",
            "n.crassa-m. musculus: significant similarity occurs in one region .\n",
            "\n",
            "c. elegans-d. melanogaster: significant similarity occurs in one region .\n",
            "\n",
            "c. elegans-a. gambiae: significant similarity occurs in one region .\n",
            "\n",
            " <dig>  sequences and their genbank accession numbers are: o. aries , b. taurus , h. sapiens , s. scrofa , m. muscus , r. norvegicus , g. gallus , x. laevis , f. rubripes , d. melanogaster , a. gambiae ; s. pombe ; n. crassa .\n",
            "\n",
            "sin <dig> from the yeast species, s. cerevisiae and s. pombe which consist of  <dig> aa and  <dig> aa, respectively, and also from the red bread mold, n. crassa  are much longer than sin <dig> from vertebrate and insect species, which are ~ <dig> aa long. the regions of similarity among these three fungal proteins are confined entirely to the carboxyl termini of these molecules, although several gaps have to be introduced to align them. no similarities are detectable in the amino terminal extensions, which, in the case of s. cerevisiae, is  <dig> aa long. it is the carboxyl regions of the fungal proteins that can also be aligned with the sin <dig> sequences from c. elegans, insects, and vertebrate species, including ovis aries, the sheep .\n",
            "\n",
            "a phylogenetic tree reconstructed from an alignment of amino acid sequences of sin <dig> is shown in fig.  <dig>  as anticipated, the sequences from the three fungi, c. elegans, the two insect species, and vertebrate species fell into distinct branches of the tree. the sequences for the mammalian species were tightly clustered, with identities ranging from 99%  to  <dig> %  . all the vertebrate cdna encoded polypeptides of  <dig> aa .\n",
            "\n",
            "there is considerable conservation of sin <dig> from mammals to birds , amphibians , and fish  . the insect sequences are rather longer than the ones from vertebrates, and several gaps have to be introduced to provide alignments . nevertheless, the insect amino acid sequences are approximately 33% identical to those of the mammals . five blocks of sequence  are significantly more conserved than others when two insects, a fish, an amphibian, a bird and several mammals are compared . three of these regions are located towards the n-terminus and two additional regions towards the c-terminus. the most diverse region is located centrally.\n",
            "\n",
            "the region of the greatest identity between these divergent insect and vertebrate sequences is an acidic region placed in conserved scd iii . in mammals, this region is completely conserved and corresponds to residues l232-k <dig> . however, an analysis of this sequence reveals no known functional motifs and no strong similarity to sequences represented in other known proteins. schroder et al.  <cit>  have also noted this conserved sequence in their analyses of sin <dig> sequences and have named it crim for conserved region in the middle.\n",
            "\n",
            "sin <dig> from c. elegans retains the highly conserved  <dig> amino acid scd i and the  <dig> amino acid domain iii . scd iii is also retained in the fission yeast and the red bread mold.\n",
            "\n",
            "vertebrates possess several unique sequences not present in insects and yeast, and, therefore, potentially implicated in the ifn signal transduction pathway including a carboxyl terminal region  immediately following the end of scd v.\n",
            "\n",
            "functional motifs in the sin <dig> primary sequence\n",
            "when the ovine sin <dig> sequence is scanned for functional motifs  <cit> , the structure appears unusually barren. two weak bipartite nuclear localization signals   <cit>  can be detected. one  is present in the scdii domain, and the other  is almost at the c-terminus within conserved domain v . nevertheless, data from the subcellular localization experiment showed that sin <dig> is excluded from the nucleus when transfected in cos <dig> or l <dig> cells  <cit> , suggesting these nls are probably not functional.\n",
            "\n",
            "there are numerous motifs that are recognized as potential but weak sites for phosphorylation by either casein kinase ii , protein kinase c, or protein kinase a . none of the  <dig> ck <dig> sites, the  <dig> protein kinase c, or the  <dig> protein kinase a sites present in the ovine sin <dig> primary sequence are conserved from mammals to fission yeast, although many are retained across all vertebrates. a weak site for myristylation , and hence for membrane association, is retained in all the vertebrate species examined, but is absent in insects and yeast. in absence of any data on the functional significance of these sites, they will not be discussed further.\n",
            "\n",
            "gene structure of sin <dig> from various species\n",
            "the genomic sequence encompassing the transcribed region of the gene could be retrieved from the genome data bases for s. pombe, s. cerevisiae, c. elegans, d. melanogaster, a. gambiae, f. rubripes, r. norvegicus, m. musculus, h. sapiens  <cit> . sin <dig> exists as a single copy gene in all these species. for example, the human sin <dig> gene is located on chromosome  <dig>   with the transcribed region composed of  <dig> exons and  <dig> introns and spanning a region of about  <dig> kb . exon  <dig> is spliced out of the shorter form of sin <dig>  <cit> . the lack of exon  <dig> does not cause a frame shift because the intron phases of the two introns on both sides of exon  <dig> are identical . schroder et al.  <cit>  have also demonstrated or predicted other minor splice variants for sin <dig> in the human. the  <dig> exons account for only  <dig> % of the gene sequence. it is, of course, unclear how many additional exons and introns are associated with the 5' utr beyond the transcription start site, whose location has not been determined.\n",
            "\n",
            "currently, the sheep and bovine genome sequences are not available, but it is likely that the sin <dig> gene organization will be similar to that in the human. the current comparative synteny maps between human, sheep and cattle  <cit>  predict that the sin <dig> gene is located on sheep chromosome  <dig>  and bovine chromosome  <dig> , respectively.\n",
            "\n",
            "a comparative map for all the genes is shown in fig.  <dig>  in fission yeast and insects, the sin <dig> gene consists of a single exon. in worm, fish, rat, mice, and human, sin <dig> has multiple exons. the exon/intron pattern, consisting of  <dig> exons, is observed in all vertebrates, including the two fish species . it is noteworthy that although the genomic sequences of sheep and cattle are not available, the exon/intron pattern of their sin <dig> genes is similar to that of other vertebrates based on the comparison between sheep or cattle sin <dig> cdna and human genomic sequence of sin <dig> . the lengths of these  <dig> exons are also remarkably conserved and fall within the normal range  . as expected, the sizes of the introns differ across species, and some are extremely long. intron sizes generally decrease in the order human > mouse > rat > fish . as expected, intron sizes were quite similar between rodents and human.\n",
            "\n",
            "the sin <dig> gene from c. elegans is organized quite differently from that in mammals. it consists of  <dig> exons interrupted by nine relatively short introns. the region of the c. elegans gene that contains regions of similarity with the mammalian protein sequences consists of exon  <dig>  and exon  <dig> . as noted above and in figure  <dig>  the sin <dig> gene from insects and s. pombe is comprised of only a single exon.\n",
            "\n",
            "discussion\n",
            "sin <dig> is a little studied gene product of unclear function found in species ranging from mammals to fungi. although the s. pombe gene product is longer than that of mammals, with an extension at its n-terminus, human sin <dig> can rescue the stress sensitivity noted in the phenotype of a s. pombe strain that expressed a constitutively active form of ras, indicating that function, as well as structure, has been conserved over hundreds of millions of years.\n",
            "\n",
            "two facts should be considered when attempting to infer a role for sin <dig> in vertebrates. the first, as discussed in the background, is the known ability of type  <dig> ifn to activate mapk/sapk in mammalian cells. the second is the proven involvement of sin <dig> in the yeast sapk  pathway and its involvement in controlling transcription of stress-activated genes  <cit> . the present analysis was conducted in an attempt to gain more detailed information about sin <dig> function from a phylogenetic analysis and comparison of sin <dig> genes and gene products in different taxonomic groups.\n",
            "\n",
            "the sin <dig> gene is remarkably divergent in both length and sequence identity within the fungi s. pombe, s. cerevisiae, and n. crassa, emphasizing the evolutionary distance between these three species. the regions of similarity are confined to the ~ <dig> amino acid c-terminal regions of the three sequences , and it is this region that is also conserved in insects and vertebrates . this diversity in structure within the fungi is probably reflected in divergence of function. avo <dig>  the apparent sin <dig> ortholog of s. cerevisiae, forms a membrane-associated complex with tor <dig> and other protein components , which control cell growth in response to nutrients  <cit> . cells with deletion of avo <dig> are unable to organize their actin cytoskeleton  <cit> . in contrast, the sin <dig> ortholog of s. pombe is involved in a stress response signaling pathway by interacting with sty <dig>  <cit> . a cross-species comparison of all the sin <dig> sequences available, indicates five regions of greatest conservation, only one of which, a ~ <dig> amino acid central region , was easily defined in all taxa . even this region is poorly conserved in the budding yeast, s. cerevisiae, although certain landmark amino acids are retained . interestingly, sin <dig> from insects and vertebrates, despite having only about 35% identity, are of similar length and possess the five regions of high identity. conceivably, the scd iii domain is functionally essential in all the species, while scds i, ii, iv, and v have evolved conserved function within the metazoa. a not unreasonable assumption is that that sin <dig> plays an evolutionarily conserved role in sapk signaling across a broad range of taxa, including all metazoan and fungal species  <cit>  but has assumed an additional function in vertebrates in mediating crosstalk with the ifn-signal transduction pathway.\n",
            "\n",
            "in vertebrates sin <dig> falls into a class of highly conserved gene products. its conservation is lower than that of two structural proteins, histone h <dig> and β-actin, but is comparable to that of cdk <dig> . however, while cdk <dig> in yeast and insects retains considerable sequence identity with the vertebrate orthologs, much of the conservation of sin <dig> is lost. it is tempting to speculate that sin <dig> has been subjected to powerful evolutionary constraint that has limited its amino acid sequence divergence within vertebrates. it should be noted that our analyses cannot exclude the possibility that conservation of sin <dig> among vertebrates reflects recent divergence of the sampled vertebrates relative to the other taxa examined. once data become available, it will be instructive to compare sin <dig> gene sequences from the invertebrate chordates  with those of the other metazoan taxa.\n",
            "\n",
            "values for percentage identities were obtained by aligning amino acid sequences from various species with their human counterparts. cdk, cyclin-dependent kinase. the genbank accession numbers for sequences are as follows. human : histone h <dig> , β-actin , cdk <dig> , sin <dig> . cattle : histone h <dig> , β-actin , cdk <dig> , sin <dig> . mouse : histone h <dig> , β-actin , cdk <dig> , sin <dig> . chicken : histone h <dig> , β-actin , cdk <dig> ; sin <dig> . frog : histone h <dig> , β-actin , cdk <dig> . fly : histone h <dig> , β-actin , cdk <dig> , sin <dig> . yeast : histone h <dig> , β-actin , cdk <dig> , sin <dig> .\n",
            "\n",
            "sin <dig> was shown to be associated with the cytoplasmic domain of ifnar <dig>  a subunit of the type i ifn receptor  <cit> . since insects appear to lack genes for type i ifn and their receptors , whereas vertebrates utilize this system primarily as an anti-viral response  <cit> , it should be theoretically possible to define a sequence in silico unique to vertebrates but clearly absent in both d. melanogaster and a. gambiae that might account for the association of sin <dig> with ifnar <dig>  sin <dig> binds to the carboxyl end of the cytoplasmic domain of ifnar <dig> via its own carboxyl  <dig> amino acids  <cit> . at least two candidate sequences exist in that part of sin <dig>  one is the rather basic carboxyl terminus , another a hdykhlyfesda  sequence, both of which are absent in the insect proteins . whether these sequences are participants in the interaction of sin <dig> with ifnar <dig> in vertebrates has not been examined experimentally. of course, it is quite possible that insect sin <dig> can bind vertebrate ifnar <dig> or that amino acid substitutions elsewhere in the carboxyl end of the vertebrate sequence have evolved to promote the interaction. these possibilities have also not been tested. in this regard, ifnar <dig>  with which sin <dig> interacts, has evolved much more rapidly than sin <dig> itself. the sequence of human ifnar <dig>  for example, shows only about 58% and 29% identity to those of ovine and chicken ifnar <dig>  respectively  <cit> , while orthologs have yet to be defined for ifnar <dig> in frogs and fish, even though these animals are believed to have a functional ifn system, which includes the production of type i ifn and downstream components in response to double stranded rna  <cit> . interestingly, the only highly conserved continuous sequence of chick and mammalian ifnar <dig> within the sin <dig> binding region is an acidic region  at the very carboxyl terminus of the molecule . it seems possible that this conserved sequence provides the scaffold for sin <dig> binding.\n",
            "\n",
            "as also observed by schroder et al.  <cit> , sin <dig> is represented by a single gene in all species where it exists. in both insects and the two yeast species, the gene is intronless, while in c. elegans and in vertebrate species introns are present . in budding yeast, only a small number  of genes have introns  <cit> , whereas in most other eukaryotes, including drosophila, intronic sequences are a feature of the majority of genes and must be excised to produce a functional mrna  <cit> . for d. melanogaster, for example, there is an average of  <dig> introns per gene  <cit> . these introns are short, averaging  <dig> bp in drosophila  <cit> . why the sin <dig> genes are intronless in these species is unclear, but there is considerable evidence that retrotransposition occurs in yeast, drosophila  <cit>  and mammals  <cit> . in this process, reverse transcription of mrna from a parental gene creates an intronless copy of the parental gene at a new position in the genome. if this mechanism created the sin <dig> gene, a remnant or evolved version of the parental gene might be anticipated to exist, particularly if the transposition event occurred in recent evolutionary time  <cit> . it is unclear whether the intronless sin <dig> gene in drosophila resulted from such a retrotransposition event since there is not a detectable intronic copy elsewhere in the genome. the sin <dig> gene from c. elegans has introns, but is organized very differently from that of vertebrates, where the intron/exon organization is highly conserved .\n",
            "\n",
            "unfortunately, the function of sin <dig> is unknown. its structural conservation from vertebrates to yeast  <cit>  and its expression in most, if not all tissues of mammals  <cit>  suggest a central, if elusive, role in life processes.\n",
            "\n",
            "CONCLUSIONS\n",
            "sapk-interacting protein  <dig> , a little-studied but widely expressed gene product, is encoded by a single gene in fungi, nematodes, insects, and all vertebrates analyzed and shows modest conservation of amino acid sequence that is consistent with some degree of conserved function in stress-activated signal transduction pathways. sin <dig> is highly conserved in vertebrates where it has been implicated in linking interferon responses to the sapk pathway.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "sin <dig> is a little-studied gene product of unclear function found in species ranging from mammals to fungi.\n",
            "sin <dig> is encoded by a single gene from a yeast species, caenorhabditis elegans, that is ~ <dig> aa long.\n",
            "sin <dig> was found in two yeast species, s. cerevisiae, and a number of other fungal species, including caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis elegans, caenorhabditis\n",
            "5\n",
            "[{'summary_text': 'we used an integrated pk/pd model to evaluate the in vivo antimicrobial activity of marbofloxacin against p. multocida in tissue cage fluid in the presence of an active immune response.\\nwe found an auc24tcf/mic ratio of  <dig> \\xa0±\\xa0 <dig> ,  <dig> ,  <dig> ,  <dig> ,  <dig> ,  <dig>  and  <dig> \\xa0mg/kg marbofloxacin in tissue cage fluid.\\nthe in vivo antimicrobial activity of marbofloxacin against p. multocida in tissue cage fluid was  <dig> \\xa0±\\xa0 <dig> ,  <dig> ,  <dig> ,  <dig>  and  <dig> \\xa0h·μg/ml.\\nthe in vivo antimicrobial activity of marbofloxacin against p. multocida in tissue cage fluid was  <dig> \\xa0±\\xa0 <dig> ,  <dig> ,  <dig> ,  <dig> ,  <dig>  and  <dig> % respectively and these values were independent of concentration in the range of  <dig> \\xa0'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the increase in drug-resistance coupled with the lack of new and effective antimicrobials indicates that modified drug-dosage regimens may be useful clinical alternatives. in the clinic, pharmacokinetic  and pharmacodynamic  outcomes are interrelated  <cit> . pk/pd modeling in veterinary medicine can be used to define the relationship between the effects of different drug concentrations and therapeutic outcomes. in short, pk/pd modeling provides a chance for dosage optimization  <cit> . we used the tissue cage  infection model to evaluate pk and pd profiles in interstitial fluids in the presence of an active immune response .\n",
            "\n",
            "\n",
            "p. multocida is a widespread pathogen that inhabits mucosal surfaces and upper respiratory tracts of clinically healthy animals  <cit> . it is the causative agent of fowl cholera in poultry, atrophic rhinitis in swine and hemorrhagic septicemia in buffalo and cattle . it also plays a major role in pneumonia in swine and ruminants as an opportunistic pathogen  <cit> .\n",
            "\n",
            "marbofloxacin is a synthetic third-generation fluoroquinolone that is used solely in veterinary medicine  <cit> . it is active against gram-negative and some gram-positive bacteria as well as mycoplasmas and other intracellular pathogens  <cit> . it is widely used against most of the swine respiratory pathogens including actinobacillus pleuropneumoniae, haemophilus parasuis and p. multocida .\n",
            "\n",
            "marbofloxacin acts as a bactericidal concentration-dependent antibiotic similar to other fluoroquinolones. the auc24/mic or cmax/mic  are the pk/pd surrogates that most closely reflect clinical outcomes.  <cit> . we used an integrated pk/pd model to evaluate the in vivo antimicrobial activity of marbofloxacin against p. multocida in swine. this approach gives an approximation of the most effective treatment for achieving a bacteriological cure and minimizing the emergence of bacterial resistance.\n",
            "\n",
            "methods\n",
            "animals and tissue-cages\n",
            "twenty clinically healthy castrated crossbred piglets ,purchased from guangdong pig breeding farm, with body weights of  <dig> to 18 kg were included in the study. these animals were housed in individual pens for a week prior to experiments with a controlled temperature of 26 °c. the animals were fed antimicrobial-free feed twice a day with water available ad libitum. tissue-cages were fabricated according to a previously published method  <cit> . two tissue-cages were implanted subcutaneously in the neck of each piglet after sedation and local anesthesia. about 3–4 weeks later and after wound healing,  <dig>  ml tissue-cage fluid  was sampled by percutaneous puncture. tcf samples were cultured for aerobic and anaerobic contaminants to ensure sterility before commencing the experiments.\n",
            "\n",
            "bacterial strain and inoculant preparation\n",
            "\n",
            "p. multocida strain cvcc <dig>  was purchased from the china veterinary culture collection centre . the original bacterial culture was isolated from a piglet that died of plague in jiang su, china. p. multocida was grown on tryptic soy agar  supplemented with 5% defibrinated sheep blood . bacteria were cultured in mueller–hinton broth , with shaking at 37 °c and grown to 109 cfu/ml. the cells were then concentrated by centrifugation at 3000×g for 10 min and suspended in sterile  <dig> % nacl to 108 cfu/ml.\n",
            "\n",
            "drugs\n",
            "marbofloxacin was obtained as a 10% injectable aqueous solution from yuan zhen . marbofloxacin standard was purchased from dr. ehrenstorfer  and ofloxacin was acquired from the national institute for food and drug control . pentobarbital sodium was from jian yang biotechnology and procainamide hydrochloride was purchased from xin zheng .\n",
            "\n",
            "mic determinations\n",
            "mic values were determined by an agar dilution method as a preliminary screening according to clinical and laboratory standards institute  reference methods. in this study, we measured the mic of marbofloxacin against the p. multocida in the tissue-cage fluid by a micro dilution assay in triplicate using the tissue-cage fluid as matrix.\n",
            "\n",
            "pk studies\n",
            "tissue cage fluid  was sampled 1 h after the injection of drug and thereafter at  <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig> and 96 h. all the samples were centrifuged at 6000×g at 4 °c for 10 min. the supernatants were stored at −20 °c. the concentrations of marbofloxacin in tissue cage fluid were determined using a waters  <dig> series high performance liquid chromatography  system with fluorescence detection  <cit> . the chromatographic separation was achieved on an agilent bds c <dig> column  at 28 °c with a thermostat column oven . all tcf samples were thawed at room temperature before analysis and 3 ml of trichloromethane was added to a 10 ml centrifuge tube with 200 μl tissue-cage fluids spiked with 10 μl ofloxacin internal standard, and vortexed for 1 min. the aqueous layer was discarded after centrifugation at 6000 g for 10 min and the organic layer was dried under a stream of nitrogen. the residue was suspended with  <dig>  ml mobile phase and a 10 μl aliquot was taken and injected into hplc for analysis after filtered through a  <dig>  μm nylon syringe filter . the determination of marbofloxacin in the tissue cage fluid was linear within a range of  <dig> –2 μg/ml and the correlation coefficient was > <dig> . the lower limit of quantitation was  <dig>  μg/ml. the recoveries of marbofloxacin in tissue-cage fluid were >85%. the standard deviations  and the coefficients of variability  for both interday and intraday were <8% in tcf.\n",
            "\n",
            "tissue-cage infection and in vivo kill curve of marbofloxacin\n",
            "twenty piglets were divided into ten groups randomly, with  <dig> tissue-cages for each group. 24 h after  <dig>  ml of p. multocida saline suspension  were injected into the sterile tissue cage; nine groups were treated with a series of marbofloxacin intramuscular injections  at  <dig> ,  <dig> ,  <dig> ,  <dig> ,  <dig> ,  <dig> ,  <dig> ,  <dig> ,  <dig>  mg/kg. the control group was given sterile physiological saline only. tcf samples were aspirated from the tissue-cages by percutaneous puncture at  <dig>   <dig>   <dig>   <dig>   <dig> and 24 h after dosing. 100 μl of tissue cage fluid was serially diluted 10-fold in saline within 1 h of sampling for cfu determinations. from each dilution, 20 μl was spread on btsa plated in triplicate and incubated for 16 h at 37 °c.the limit of detection was 500 cfu/ml.\n",
            "\n",
            "protein binding\n",
            "to determination the protein binding of marbofloxacin in tcf, marbofloxacin at  <dig> ,  <dig> ,  <dig> ,  <dig> and 2 mg/l was spiked into pooled uninfected tcf in triplicate. an amicon centrifree micropartition device with a  <dig> kda cutoff  was used to remove the protein-bound drug as previously described  <cit> . the marbofloxacin concentration before and after ultrafiltration and centrifugation was determined as described in the pk studies. the percent of drug bound to protein was calculated as the following equation: bound % =  × 100%, where cu is the filtrate concentration and ci is the initial concentration.\n",
            "\n",
            "pharmacokinetic-pharmacodynamic analysis\n",
            "the mic value was included in the calculation of the pk/pd index. according to standard methodology, an inhibitory sigmoid emax model was used to assess the pk/pd index with the highest predictive value to the different indices for each dosing regimen . the r <dig> value was computed for the correlation between antimicrobial effectiveness and each of the pk/pd parameters.  <dig> e=emax−emax−e0×cen/ec50n+cen \n",
            "\n",
            "\n",
            "\n",
            "e is the antibacterial effect; e <dig> represents maximum drug effect 24 h after i.m administration. e\n",
            "max is the change of bacterial load  in the infected tissue-cage with no drug present. c\n",
            "e is the pk/pd index magnitude. ec\n",
            " <dig> is the magnitude required for achieve 50% of e\n",
            "max and n is the hill coefficient that describes the sigmoid shape. pk and pd indices were calculated using a non-compartment model in winnonlin  <dig>  software ,\n",
            "\n",
            "RESULTS\n",
            "mic of marbofloxacin for p. multocida\n",
            "tissue cage fluid was sterile as assessed by aerobic and anaerobic culture. we then determined that the mic of marbofloxacin for p. multocida in tissue cage fluid was  <dig>  μg/ml ex vivo. this did not change after exposure to marbofloxacin for 24 h and this number was used for pk/pd calculations.\n",
            "\n",
            "tc infection and in vivo kill curve of marbofloxacin\n",
            "we determined the in vivo antibacterial activity of marbofloxacin against p. multocida cvcc <dig>  p. multocida inoculated into tissue cages in the absence of antibiotic was  <dig>  ±  <dig>  log <dig> cfu/ml averaged over all tissue cages. when marbofloxacin was administered intramuscularly, the bacterial counts for marbofloxacin-treated groups decreased by  <dig>  to  <dig> log <dig> cfu/ml. by this time, the control saline-treated cages had increased to  <dig> log <dig> cfu/ml. marbofloxacin exerted bactericidal activity when the dosage via intramuscular injection exceeded  <dig>  mg/kg .fig.  <dig> in vivo time killing curve of marbofloxacin against p. multocida in tcf. kb- <dig> means the group treated by saline. mean log10cfu/ml values only . \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pk study\n",
            "the concentration-time profile of marbofloxacin in the infected cages were calculated as auc24h and cmax of marbofloxacin in the tissue cage fluid following multiple dosages obtained from the measured marbofloxacin concentration data from each cage . the mean values of auc <dig> for marbofloxacin increased with dose escalation in the tissue cage fluid. a significant correlation  was observed between auc <dig> tcf and dosages .fig.  <dig> pharmacokinetic profiles of marbofloxacin in tcf. the tissue-cages were infected with p. multocida as described above, 24 h later administered with single intramuscularly doses of  <dig> ,  <dig> , <dig> , <dig> , <dig> , <dig> , <dig> , <dig>  and  <dig>  mg/kg marbofloxacin as bw. all tcf were sampled by puncture at  <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>  72and 96 h after dosing. marbofloxacin concentrations were determined using a hplc method with fluorescence detection. but a small number of samples sampled on 96 h were undetected. each symbol represents the mean ± standard deviation of the levels in the four tissue-cages. \n",
            "\n",
            "\n",
            "xa is the mean value.\n",
            "\n",
            "\n",
            "fig.  <dig> linear regression plots between administered doses and auc <dig> values. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pb\n",
            "the protein binding of marbofloxacin in tissue cage fluid was  <dig>  ±  <dig> % and  <dig>  ±  <dig> % in serum within the range of  <dig>  ~ 2 mg/l for marbofloxacin .table  <dig> protein binding of marbofloxacin in serum and tissue cage fluid\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "magnitude of the pk/pd parameter required for efficacy\n",
            "the pk parameters best predicting the in vivo of the drug were the auc24/mic , %t > mic  and cmax/mic  . the magnitudes of auc24/mic predicted for a 1-log reduction and a 3-log reduction  were  <dig>  h and  <dig>  h, respectively, and the slope of both parameters for antibacterial effectiveness was  <dig>  .fig.  <dig> relationships between pk/pd parameters  and antibacterial effect  after 24 h of therapy. r <dig> is the correlation coefficient. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "discussion\n",
            "respiratory diseases greatly affect the production and health of swine and result in high economic losses  <cit> . in europe, p. multocida is one of the most common bacterial species among porcine respiratory diseases and leads to atrophic rhinitis in young pigs  <cit> . marbofloxacin shows excellent effectiveness for treating p. multocida infection through in vitro tests  <cit> . the protein binding of marbofloxacin was low in previous studies and was 52% for serum, 40% for tissue cage fluid in cattle and  <dig> % for pig serum  <cit> . only free drug in the interstitial fluid is microbiologically active against extracellular bacteria  <cit> .\n",
            "\n",
            "in the present study, the protein binding of marbofloxacin in pig serum and tcf were  <dig> % and  <dig> % respectively and these values were independent of concentration in the range of  <dig>  ~ 2 μg/ml. the auc24tcf of marbofloxacin following single intramuscular administration  was estimated to be  <dig>  h·μg/ml, which was lower than that in plasma   <cit> . the cmax for tissue cage fluid was lower than previously reported in plasma  and tissue cage fluid  in pigs  <cit> .\n",
            "\n",
            "the pk/pd model integrating the pk parameter, the in vitro mic and pd outcome are all valuable in designing rational dosage regimens and in the measurement of susceptibility breakpoints  <cit> . our data indicated that auc24tcf/mic was the best parameter that correlated with the in vivo antibacterial effects of marbofloxacin against p. multocida. this agrees with previous studies of marbofloxacin activity against p. multocida  <cit> . we found an auc24tcf/mic value of  <dig>  h  that was lower than the reported value in plasma in piglets   <cit> .\n",
            "\n",
            "the auc24htcf/mic ratios that produced a 1-log reduction, and 50% and 90% of the maximum effect were  <dig>  h,  <dig>  h and  <dig>  h, respectively. the auc24tcf/mic ratio for bactericidal effects in this study was different from previous study values of  <dig> ,  <dig>  and  <dig>  h for 3-log reductions  <cit> . these differences may by accounted for by examining the ex vivo pk/pd model. this model holds the antibiotic concentration constant but this concentration would vary with time in the animal infection model  <cit> . in addition, the ex vivo and neutropenic murine lung infection models do not take the host immune responses into consideration. finally, the magnitude of pk/pd indices were influenced by the bacterial load and this may influence characteristics of the pk parameters. a higher antimicrobial concentration would be required for higher pathogen loads  <cit> .\n",
            "\n",
            "an auc24/mic ratio exceeding a value of 125 h was reported as useful for the treatment of infections caused by gram-negative organisms while reducing the risk of resistance induction . in combination with the mic <dig>  from two previous studies, and the pk characteristics in this study, we can infer that a dosage regimen of marbofloxacin  <dig>  mg/kg  was effective for the treatment of p. multocida infections  <cit> . however, considering that we found an auc24tcf/mic ratio of  <dig>  h that produced a 3-log reduction in this investigation, a marbofloxacin dose of  <dig>  mg/kg every 24 h is recommended for treatment of p. multocida infections with mic values lower than  <dig>  μg/ml. the dosage  <dig>  mg/kg was calculated by the equation auc\n",
            "24 =  <dig> ∗dose +  <dig>  . however, we did not assess the diffusion of marbofloxacin in lungs and the immune responses in lungs may be different from that in tissue cage fluid.\n",
            "\n",
            "CONCLUSIONS\n",
            "the present study characterized the in vivo effectiveness of marbofloxacin against p. multocida in a tissue cage model in pigs. the auc24tcf/mic proved to be the pk/pd index that predicted the antimicrobial activity of marbofloxacin against p. multocida. marbofloxacin presented excellent antibacterial activity with an auc24tcf/mic ratio of  <dig>  h for a 3-log reduction in bacteria. although this study needs to be validated by clinical treatment under practical conditions, the results indicated that it may be a critical step closer to clinical trials for optimization of dosage regimens.\n",
            "\n",
            "abbreviations\n",
            "aucarea under the curve\n",
            "\n",
            "auc24h/micthe ratio of area under the concentration-time curve over 24 h to mic\n",
            "\n",
            "bwbody weight\n",
            "\n",
            "cfucolony forming unit\n",
            "\n",
            "clsiclinical and laboratory standards institute\n",
            "\n",
            "cmax/micthe ratio of peak concentration to mic\n",
            "\n",
            "cmaxpeak concentration\n",
            "\n",
            "cvccchina veterinary culture collection\n",
            "\n",
            "hplchigh performance liquid chromatography\n",
            "\n",
            "micminimum inhibitory concentration\n",
            "\n",
            "p. multocidapasteurella multocida;\n",
            "\n",
            "pdpharmacodynamics\n",
            "\n",
            "pkpharmacokinetics\n",
            "\n",
            "t% > micthe percent time that marbofloxacin concentrations were above the mic\n",
            "\n",
            "tctissue cage\n",
            "\n",
            "tcftissue cage fluid\n",
            "\n",
            "not applicable.\n",
            "\n",
            "funding\n",
            "this work is supported financially by the national natural science foundation of china  and the national key basic research program of china . the funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.\n",
            "\n",
            "availability of date and materials\n",
            "the datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request.\n",
            "\n",
            "authors’ contributions\n",
            "zz conceived of the study helped to draft the manuscript. qz designed the experiment and drafted the manuscript. qz and js conducted the experiment and acquired the data. xm and xl analyzed the data. wx and yl participated in the revision of the manuscript. all authors read and approved the final manuscript.\n",
            "\n",
            "competing interests\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "consent for publication\n",
            "not applicable.\n",
            "\n",
            "ethics approval and consent to participate\n",
            "the experiments on pigs were approved by committee on the ethics for animal experiments of south china agricultural university \n",
            "\n",
            "publisher’s note\n",
            "springer nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "we used an integrated pk/pd model to evaluate the in vivo antimicrobial activity of marbofloxacin against p. multocida in tissue cage fluid in the presence of an active immune response.\n",
            "we found an auc24tcf/mic ratio of  <dig>  ±  <dig> ,  <dig> ,  <dig> ,  <dig> ,  <dig> ,  <dig>  and  <dig>  mg/kg marbofloxacin in tissue cage fluid.\n",
            "the in vivo antimicrobial activity of marbofloxacin against p. multocida in tissue cage fluid was  <dig>  ±  <dig> ,  <dig> ,  <dig> ,  <dig>  and  <dig>  h·μg/ml.\n",
            "the in vivo antimicrobial activity of marbofloxacin against p. multocida in tissue cage fluid was  <dig>  ±  <dig> ,  <dig> ,  <dig> ,  <dig> ,  <dig>  and  <dig> % respectively and these values were independent of concentration in the range of  <dig>  \n",
            "6\n",
            "[{'summary_text': 'we identified  <dig> abc genes  in t. urticae and  <dig> in two multi-pesticide resistant t. urticae strains, d. melanogaster cg <dig> and d. pulex dappu1- <dig>  we also identified  <dig> abc genes  in two multi-pesticide resistant t. urticae strains, d. melanogaster cg <dig> and d. pulex dappu1- <dig>  we also identified  <dig> abc genes  in two multi-pesticide resistant t. urticae strains, d. melanogaster cg <dig> and d. pulex dappu1- <dig>  we also identified  <dig> abc genes  in two multi-pesticide resistant t. urticae strains, d. melanogaster cg <dig> and d. pulex dappu1- <dig>  and  <dig> abc genes in two multi-pesticide resistant t. urticae strains, d. melanogaster cg <dig> and d. pulex dappu1- <dig>'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "atp-binding cassette  proteins form one of the largest protein families that are present in all living organisms on earth. the majority of abc proteins are membrane bound primary transporters, using atp to translocate substrates across extra- and intracellular membranes. in addition, these abc transporters are mostly uniporters, mediating the unidirectional translocation of a substrate  <cit> . the major facilitator superfamily  is another large transporter family present in all living organisms, but as opposed to abc transporters, it comprises secondary carriers that can be either uniporters, symporters or antiporters  <cit> . in most abc proteins two types of domains can be distinguished, an atp-binding domain ) and a transmembrane domain . the highly conserved nbd contains three motifs: a walker a and walker b domain and the abc signature . the nbd binds and hydrolyses atp and provides energy to transport substrates. the tmd consists of five to six membrane spanning helices and provides the specificity for the substrate. full transporters comprise two nbds and two tmds while half transporters have only one of each type and require homo- or heterodimerization to form a functional unit  <cit> . based on the homology of their nbds, abc proteins have been divided into seven subfamilies, abca to abch  <cit> . interestingly, the abch subfamily was discovered during analysis of the drosophila melanogaster genome and is present in all sequenced arthropod genomes to date and teleost fish, but not in mammals, plants or fungi  <cit> .\n",
            "\n",
            "in humans, abc proteins mainly function in the membrane transport of substrates, including amino acids, sugars, lipids, inorganic ions, polysaccharides, metals, peptides, toxic metabolites and drugs  <cit> . in addition to transporters, the human abc protein superfamily also contains ion channels , receptors  and proteins involved in translation   <cit> . mutations in abc genes have been linked to several human disorders, like cystic fibrosis, adrenoleukodystrophy, sitosterolemia and diabetes  <cit> . furthermore, within the abcb, c and g subfamilies, many genes code for proteins that contribute to resistance of cancer cells against chemotherapeutic agents: the multidrug resistance proteins or p-glycoproteins , the multidrug resistance-associated proteins  and the breast cancer protein   <cit> . in insects, it has been shown that abc transporters have functions that affect metabolism, development and resistance to xenobiotics including insecticides and plant secondary toxic compounds   <cit> . some abc transporters have specific functions that are well documented in arthropods. in d. melanogaster the abcc transporter mdr <dig> controls the export of a germ cell attractant  <cit> . d. melanogaster white, on the other hand, is a member of the abcg subfamily and is involved in the uptake of pigment precursors in the developing eye  <cit> . its orthologs in bombyx mori  and t. castaneum  have similar functions, and w-3oeb. mori mutants and tcabcg-9b dsrna injected adult beetles have white eyes  <cit> . in the tobacco hornworm, manduca sexta, orthologs of human p-gps are essential as they prevent the influx of nicotine across the blood brain barrier  <cit> . insect orthologs of human p-gps and mrps have also been frequently linked to pesticide resistance  <cit> . resistance to pesticides in insects is either related to reduced target-site sensitivity or sequestration/metabolism of the pesticide before it reaches the target site by quantitative or qualitative changes of genes involved in the detoxification process  <cit> . these “detoxification” genes comprise members of the p <dig> mono-oxygenases , glutathione-s-transferases , carboxyl/cholinesterases  and also the less known abc transporters. although the abc transporters have often been overlooked in studies that describe the detoxification toolkit in sequenced insect genomes  <cit> , clear examples of their importance in detoxification have been documented. for example, lanning et al.  <cit>  showed that increased expression of human p-gp orthologues in h. virescens was associated with resistance to thiodicarb, and a mutation in the same abcc member of four different lepidopteran species was recently associated with resistance to the cry1a toxin  <cit> .\n",
            "\n",
            "complete and correctly annotated gene inventories are a prerequisite to study the biological role and evolutionary history of abc genes. among arthropods, detailed studies of abc families have been published for members of several different insect orders  <cit>  and the crustacean daphnia pulex <cit> . in contrast, besides the identification of nine abc genes in the mange mite sarcoptes scabiei <cit> , there are no reported studies in the subphylum chelicerata , one of the most diverse groups of terrestrial animals  <cit> . recently, the first published draft genome sequence of a chelicerate, the two-spotted spider mite, tetranychus urticae, was reported  <cit> . the spider mite is one of the most polyphagous herbivores known, and has been documented to feed on more than  <dig>  plant species that belong to more than  <dig> different plant families, including many that produce toxic compounds  <cit> . in addition, spider mites are major agricultural pests and are the “resistance champion” among arthropods as they have the most documented instances of resistance to diverse pesticides  <cit> . the molecular mechanisms underlying spider mite resistance to xenobiotics  are less understood compared to insects  <cit> . however, the availability of the draft genome sequence now provides unique information and tools for the study of the role of gene families involved in xenobiotic metabolism in spider mites  <cit> . characterization of spider mite gene families associated with detoxification of xenobiotics, including abc genes, is the first step towards a better understanding of how spider mites cope with these compounds  <cit> . an initial preliminary analysis of abc genes in the spider mite genome focused solely on abcb and abcc subfamilies  <cit> , and lacked a complete description of the phylogenetic relationships with other metazoan abcs. in this study, we provide a detailed comparison of all abc subfamilies  in t. urticae with those of the insect d. melanogaster, the crustacean d. pulex, the nematode caenorhabditis elegans and the mammal homo sapiens. further, we show that expression levels of abc genes change in pesticide resistant strains and when new and challenging plant host are encountered. our analyses will facilitate biochemical and toxicological studies of the role of t. urticae abc transporters in spider mite physiology, and in particular the extraordinary host range and pesticide resistance development.\n",
            "\n",
            "RESULTS\n",
            "identification of spider mite abc genes\n",
            "we identified  <dig> putative abc genes in the genome of t. urticae . to our knowledge, this is the largest number of abc genes reported for any metazoan species so far  <cit> . of all organisms sequenced to date, only the protozoan ciliate tetrahymena thermophila has more abc genes  <cit> . a maximum likelihood phylogenetic analysis grouped the t. urticae abc proteins into each of the eight known abc subfamilies with high bootstrap support . we identified  <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig> and  <dig> abc proteins belonging to the abca, abcb, abcc, abcd, abce, abcf, abcg and abch subfamilies, respectively . significant homology  with one of the  <dig> putative t. urticae abc genes was found at an additional  <dig> loci in the t. urticae genome. however, gene models at these loci, most of which had homology to the abcc or abch subfamilies, lacked one or both vital domains  of canonical abc genes. these likely represent gene fragments or pseudogenized genes, and were excluded from detailed analysis . from the  <dig> full-length t. urticae abc genes, almost half  are located on only  <dig> genomic scaffolds . in addition, a high complexity was observed within gene structures, with exon numbers ranging from  <dig> to  <dig> . to determine the accurate evolutionary position of the  <dig> t. urticae abc proteins, phylogenetic analyses, including full-length abc protein sequences from the draft genomes of t. urticae and d. pulex and from the finished genomes of c. elegans, d. melanogaster and h. sapiens, were performed for each subfamily separately. the results of these analyses are discussed below.\n",
            "\n",
            "1\n",
            "2\n",
            "1t. urticae gene models can be accessed at the orcae genome portal, http://bioinformatics.psb.ugent.be/orcae/overview/tetur <cit> .\n",
            "\n",
            " <dig> transmembrane helices  were predicted using the scampi server  <cit>  while nucleotide binding domains  were determined using the scanprosite facility  <cit>  and prosite profile ps50893;  <dig> indicates that the t. urticae abc protein is a full transporter.\n",
            "\n",
            " <dig> n-glycosylation sites were predicted using netnglyc  <dig>  server  <cit> ; only n-glycosylation site with a “potential” score >  <dig>  and with a jury agreement were taken into account.\n",
            "\n",
            "4o-glycosylation sites were predicted using netoglyc  <dig>  server  <cit> ; if the g-score was higher than  <dig>  the residue was considered to be o-glycosylated; in this table the total number of o-glycosylated sites  is shown.\n",
            "\n",
            "s. \n",
            "\n",
            "cerevisiae\n",
            "c. \n",
            "\n",
            "elegans\n",
            "h. \n",
            "\n",
            "sapiens\n",
            "d. \n",
            "\n",
            "melanogaster\n",
            "t. \n",
            "\n",
            "castaneum\n",
            "d. \n",
            "\n",
            "pulex\n",
            "t. \n",
            "\n",
            "urticae\n",
            "* numbers were derived from  <cit>  and this study; one additional abcg transporter was identified for d. pulex  in this study resulting in  <dig> d. pulex abcs; a c. elegans abcb half transporter discussed by  <cit>  was added to the number of c. elegans abcb half transporters resulting in  <dig> abc transporters for c. elegans.\n",
            "\n",
            "the abca family comprises  <dig> full transporters  in t. urticae. in contrast to plants and some insects, no abca half transporters  were identified   <cit> . the t. urticae abca subfamily contains the largest t. urticae abc protein, tetur25g <dig>  . abcas share a distinct set of characteristics across species: an extracellular loop between the first and second transmembrane helices , a conserved motif downstream of each nbd  <cit>  and a conserved motif at the n-terminus . all t. urticae abcas have a characteristic extracellular loop between the first and the second tms of each tmd . the conserved motif downstream of each nbd was, except for tetur25g <dig>  present in all t. urticae abcas, while the n-terminus conserved motif could only be found in a single t. urticae abca . instead of xlxxkn, the remainder of t. urticae abcas harbor either a xmxxkd/s  or xlxxhr  n-terminal motif. a phylogenetic analysis of metazoan abcas is shown in additional file  <dig>  six t. urticae abcas  clustered together with high bootstrap support. these six abcas show high amino acid identity  and have identical exonic structure , indicating they might have arisen by recent duplication events. together with tetur01g <dig>  they form a sister-group with d. melanogaster cg <dig>  an abca reported to be down regulated in the salivary glands of an e <dig> mutant of d. melanogaster . d. melanogaster cg <dig> and the seven t. urticae abca genes cluster together, albeit with moderate bootstrap support, with a group of c. elegans abca transporters. the latter contains ced- <dig>  which is involved in the engulfment of cell corpses during programmed cell death in c. elegans <cit> .\n",
            "\n",
            "further, tetur27g <dig> and d. melanogaster cg <dig> form a sister clade of human abca <dig> and abca <dig>  while tetur25g <dig>  d. pulex dappu1- <dig> and dappu1- <dig> cluster with human abca <dig>  abca <dig>  abca <dig> and abca <dig> . these human abcas contain conserved predicted n-glycosylation sites at n <dig>  n <dig> and n <dig> of human abca <dig>  <cit> . in addition, it has been experimentally shown that d. melanogaster cg <dig> is also glycosylated at an asparagine  aligning with one of these conserved sites . the tetur27g <dig> and tetur25g <dig> proteins also have many predicted n-glycosylation sites  of which at least one  is shared with those conserved in human abca <dig>   <dig>   <dig>   <dig> and  <dig>  in humans, these abcas have highly specialized roles in phospho- and sphingolipid export  <cit> . for example, human abca <dig> controls the initial steps leading to high-density lipoprotein  formation at the cell membrane and is crucial for reverse cholesterol transport from peripheral tissues to the liver  <cit> . human abca <dig> works as an epidermal keratinocyte lipid transporter and a defective abca <dig> results in loss of the skin lipid barrier  <cit> . although we cannot assign such highly specific roles to the two t. urticae abca orthologues above, they may also be involved in lipid transport processes.\n",
            "\n",
            "the abcb subfamily consists of  <dig> fts and  <dig> hts in t. urticae . a phylogenetic analysis of abcb fts revealed that transporters of each species in the analysis clustered into separate clades, confirming an earlier hypothesis by sturm et al. that this subfamily has diversified through lineage-specific duplications  <cit>  . this diversification hypothesis is supported in mites by the fact that the t. urticae abcb fts, tetur11g <dig> and tetur11g <dig>  have well-supported phylogenetic clustering, similar exon patterns ) and high amino acid identity . t. urticae abcb fts form a sistergroup to a clade of c. elegans, h. sapiens and d. melanogaster abcb fts. the function of most members of this clade has been well documented in literature. human abcb ft, originally termed p-glycoproteins  but now also known as multiple drug resistance  proteins, are among the best characterized abc pumps and have been shown to be involved in transport of hydrophobic substrates including drugs, lipids, steroids, xenobiotics and peptides . the precise role of their orthologues in drosophila has been a focus of recent study. d. melanogaster mdr <dig> has been shown to function as an orthologue of human abcb1/mdr <dig>  a major abc transporter of cytotoxic xenobiotics at the human blood–brain barrier, and is required for chemical protection of the fruitfly brain  <cit>  while mdr <dig> has been shown to be essential in germ cell migration  <cit> . interestingly, arthropod abcb ft orthologues have frequently been linked to pesticide resistance  <cit> . for example, inhibition of a h. virescens orthologue of human abcb <dig> by the p-gp inhibitor quinidine decreased the toxicity of thiodicarb by  <dig> -fold in a resistant strain, compared to  <dig> -fold in a susceptible strain  <cit> . recently, it was found that pretreatment of d. melanogaster with the p-gp inhibitor verapamil reduced the toxicity of ddt by 10-fold in a resistant strain  <cit> . the involvement of p-gps in pesticide resistance is probably best documented for ivermectin resistance. this compound has been shown to be a substrate for both mammalian as insect-pgps and several cases of p-gp associated ivermectin resistance have been reported  <cit> .\n",
            "\n",
            "a phylogenetic analysis of abcb hts revealed, as was also shown by sturm et al.  <cit> , clear orthologous relationships between abcb hts, suggesting they have evolutionary conserved roles in metazoan species . in the case of t. urticae abcb hts, an orthologous relationship between tetur32g <dig> and d. melanogaster cg <dig>  d. pulex dappu1- <dig>  c. elegans abtm- <dig> and human abcb <dig> was found, while tetur17g <dig> groups together with d. melanogaster cg <dig>  d. pulex dappu1- <dig>  c. elegans haf- <dig>  and human abcb <dig>  as both tetur32g <dig> and tetur17g <dig> are predicted  to have a mitochondrial targeting signal, these t. urticae transporters are most likely trafficked to the mitochondria, as has been demonstrated for their human orthologues   <cit> . this suggests that tetur32g <dig> and tetur17g <dig> fulfill a similar role as their human orthologues. the human abcb <dig> protein plays a crucial role in iron homeostasis in the cytoplasm and mutations in this gene have been linked to several diseases  <cit> . recently, it was also shown that disruption of the c. elegans orthologue  of human abcb <dig> induced oxidative stress and premature cell death  <cit> . furthermore, an orthologue  of human abcb <dig> in the dengue vector, aedes aegypti, was reported to be upregulated in an insecticide resistant strain  <cit> . the function of human abcb <dig> is not well understood, but it was shown to mediate resistance against the chemotherapeutic agent doxorubicin in melanoma cells  <cit> . also, ichikawa et al. found that disruption of the mouse orthologue of human abcb <dig> lead to cardiomyopathy and decreased mitochondrial iron export  <cit> . orthologues of the remaining human mitochondrial transporters, abcb <dig> and abcb <dig>  could be identified in d. pulex, d. melanogaster and c. elegans but were not found in t. urticae <cit>  . interestingly, the localization of human abcb <dig> is currently under debate, as some studies suggest that it is located in lysosomes  <cit> . finally, similar to sturm et al.  <cit> , we did not identify arthropod orthologues of human abcb hts related to antigen processing  .\n",
            "\n",
            "the abcc subfamily consists of  <dig> transporters in t. urticae. to our knowledge, this is the largest number of abcc transporters reported in any metazoan species, including the flour beetle, t. castaneum, which also has an exceptionally large number of abccs  <cit>  . abcc proteins are fts, bearing  <dig> tmds and  <dig> nbds, with diverse functions: ion transport, cell-surface receptor activity and translocation of a broad array of substrates like drugs, cyclic nucleotides, endogenous compounds and their glutathione conjugates and glutathione  <cit> . because of their ability to extrude drugs, many of these abccs are also termed multidrug resistance associated proteins   <cit> . mrps can be categorized according to the presence or absence of a third n-terminal transmembrane-spanning domain  . in humans, “long” mrps like abcc <dig>   <dig>   <dig>   <dig> and  <dig>  have such a tmd <dig> while “short” human mrps like abcc <dig>   <dig>   <dig> and  <dig> do not   <cit> . in addition to mrps, the abcc family also harbors the cystic fibrosis transmembrane conductance regulator  and sulfonylurea receptors   <cit> .\n",
            "\n",
            "in our phylogenetic analysis,  <dig> t. urticae abccs clustered with d. melanogaster cg <dig>  d. pulex dappu1- <dig> and a group of human “long” mrps  . twenty-two of the transporters  from this t. urticae abcc clade also have a tmd <dig>  while we could not identify this additional transmembrane domain in tetur28g <dig>  , additional file 3). within this t. urticae abcc clade, transporters clustered into  <dig> distinct groups. one group  consists of  <dig> t. urticae abcc genes, each with  <dig> introns, and tetur04g <dig>  which has only  <dig> introns . interestingly, tetur04g <dig> is positioned at the basal node, indicating that intron gain events may have occurred in this group . the other group  consists of  <dig> t. urticae abcc genes:  <dig> with  <dig> introns, tetur25g <dig> with  <dig> introns and tetur28g <dig> having only  <dig> introns . the intron loss in tetur28g <dig> can be directly linked to the lack of the tmd <dig> , while the nature of the intron gain event in tetur25g <dig> is unclear . human mrp <dig>  mrp <dig>  mrp <dig> and mrp <dig> along with d. melanogaster cg <dig> and d. pulex dappu1- <dig> form a sister clade of t. urticae abcc groups  <dig> and  <dig> . these proteins have been extensively studied as transporters of natural product drugs like anthracyclines and plant alkaloids  <cit> . mrps also share many substrates with human p-gps  but, while p-gps transport drugs in their original form, mrps mostly transport their glucuronate, sulfate and glutathione  conjugates. in the latter case, gsh is fused by gsts with xenobiotics or their metabolites and finally transported out of the cell by mrps  <cit> . in humans, gsts from the alpha, mu and pi-class have been reported to act in synergy with mrps  <cit> . intriguingly, a clear expansion of gsts in the t. urticae genome was found for the delta and mu gst subclasses, the latter of which is not present in insects and until recently was believed to be vertebrate specific  <cit> . future studies should point out if there is a coordinated action between the gsts of these subfamilies and the many mrp orthologues of t. urticae.\n",
            "\n",
            " see the legend of figure  <dig> for procedure and display details.\n",
            "\n",
            "most of the biochemical properties of human mrp <dig> have been confirmed in its d. melanogaster orthologue cg <dig>  <cit> . recently, it was also found that exposure to the p <dig> mono-oxygenase inhibitor piperonylbutoxide and the antimetabolite/antifolate drug methotrexate alters the expression  of the d. melanogaster cg <dig> gene in the malpighian tubuli, which are organs known to play important roles in excretion and xenobiotic detoxification  <cit> . the orthologue  of cg <dig> in the human body louse, pediculus humanus, has also been reported to be upregulated after exposure to the pesticide ivermectin. moreover, injection of phabcc <dig> dsrna in p. humanus female lice increased their sensitivity to ivermectin by 20-30%  <cit> .\n",
            "\n",
            "fourteen spider mite abccs clustered with high bootstrap support in a group with  <dig> d. melanogaster abcc proteins, human abcc4/mrp <dig> and abcc <dig> and d. pulex dappu1- <dig> . all fourteen t. urticae abccs show the structural properties of “short” mrps , additional file 3). five of these t. urticae abccs group as a sister clade  of the  <dig> d. melanogaster abcc proteins . detailed physiological roles for most of the  <dig> d. melanogaster abcc proteins are unknown. d. melanogaster cg <dig> is regulated by heavy metals through the metal-responsive transcription factor  <dig>  and contributes to metal homeostasis  <cit> . d. melanogaster cg <dig> controls responsiveness to o <dig> deprivation and might also be involved in oxidative stress response  <cit>  while 31% of embryos of d. melanogaster cg <dig> mutants were unable to heal wounds  <dig> h postwounding  <cit> . recently, it was also shown that d. melanogaster cg <dig>  which is highly expressed in the midgut, was upregulated in cyp6g <dig> knockdown flies, indicating molecular crosstalk within a detoxification network  <cit> . furthermore, a point mutation in lepidopteran homologs of this d. melanogaster abcc group has been clearly linked with resistance against the b. thuringiensis cry1a toxin  <cit> . another  <dig> t. urticae abccs  form, together with d. pulex dappu1- <dig> and the  <dig> d. melanogaster and five t. urticae abccs  a sister clade of human abcc <dig> and abcc <dig> . the function of d. pulex dappu1- <dig> is not known, but human abcc7/cftr acts as a chloride channel, a unique function not found in any other abc transporter  <cit> . in our analysis, cftr clustered with human abcc4/mrp <dig>  which is its closest abc paralog according to jordan et al.  <cit> . human mrp <dig>  have the ability to transport a range of endogenous molecules involved in cellular signaling, like cyclic nucleotides, eicosanoids and conjugated steroid hormones  <cit> . as a drug transporter, mrp <dig> also stands out for its broad substrate specificity, covering antiviral, antibiotic, cardiovascular and cytotoxic agents  <cit> .\n",
            "\n",
            "interestingly, many transporter genes in t. urticae abcc groups 1– <dig> form closely related sister groups and show high amino acid identity between their corresponding protein sequences . together with their conserved exon pattern , this strongly suggests that multiple tandem duplications underlie the proliferation of these genes. this is in contrast to the crustacean d. pulex, which has only few abcc genes  <cit> , and more closely resembles what is generally observed for insects, where gene duplication of abcc genes occurs frequently  <cit> .\n",
            "\n",
            "furthermore, clear orthologous relationships were found for the two remaining t. urticae abccs: tetur03g <dig> and tetur11g <dig>  tetur03g <dig> clustered as an orthologue of d. melanogaster cg <dig>  d. pulex dappu1- <dig> and human abcc10/mrp <dig> . similar to its orthologues, tetur03g <dig> has a tmd <dig> <cit>  , additional file 3). the functions of d. melanogaster cg <dig> and d. pulex dappu1- <dig> are not known. a growing understanding of the physiological role of human abcc10/mrp <dig> is, on the other hand, beginning to emerge. human mrp <dig> is distinct from other human abccs in that it shows little or no activity towards glutathione, sulfate conjugates and cyclic nucleotides, substrates that can be handled by other human mrps . instead, human mrp <dig> is able to confer resistance to taxanes )  <cit> . however, the presence of one-to-one orthologues in other metazoans, might indicate a more conserved function of this abcc protein in this group of species.\n",
            "\n",
            "tetur11g <dig> is located in the same clade as d. pulex, human and d. melanogaster sulfonylurea receptors . in contrast to vertebrates, the n-terminal sur interpro-motif  is not present in tetur11g <dig> and other arthropod surs . however, the presence of a tmd <dig> typical for surs and “long” mrps  <cit> , and the well-supported clustering with human abcc8/sur <dig> and abcc9/sur <dig> support the idea that tetur11g <dig> is a sur homologue. four sur subunits assemble into an octameric complex with four pore-forming subunits, characteristic for inwardly rectifying potassium  channels, to form atp-sensitive potassium  channels  <cit> . three orthologues of these pore- forming subunits were also found in the t. urticae genome  of d. melanogaster), suggesting that a functional katp channel can be formed in t. urticae. katp channels are involved in multiple physiological processes, with roles in glucose homeostasis, ischemic protection and innate immunity  <cit> . intriguingly, in  <dig> it was suggested that the sur was the direct target of benzoylureas, a group of chitin synthesis inhibitors  <cit> . this was largely based on similar effects of glibenclamide, a well-known sur inhibitor in humans and anti-diabetic drug, on the inhibition of chitin synthesis. however, it was later shown by gangishetti et al.  <cit>  that sur is not expressed in the d. melanogaster epidermis, where chitin disruption is observed. recently, based on genetic mapping of etoxazole resistance genes, it was suggested that the action of chitin synthesis inhibitors is mediated by a direct interaction with chitin synthase, a processive glycosyl transferase  <cit> . the lack of a role for sur in chitin production, transport or metabolism is further confirmed by recent studies, where it was shown that the sur receptor is dispensable for chitin synthesis in d. melanogaster <cit> , and rnai knockdown of its orthologue in t. castaneum did not result into a phenotype  <cit> . elucidating the role of sur in t. urticae will therefore require additional studies.\n",
            "\n",
            "finally, no orthologues of human abcc <dig>   <dig> and  <dig> were identified in t. urticae, although three orthologues were found in the genome of d. pulex , confirming earlier findings by sturm et al.  <cit> . surprisingly, a single nucleotide polymorphism in human abcc <dig> was identified as the determinant of the human earwax type  <cit> . however, the potential roles of related transporters in other organisms  are not clear.\n",
            "\n",
            "the abcd subfamily harbors hts that in humans are located in the peroxisome where they are involved in the import of long and branched chain acyl-coa into this organelle  <cit> . the t. urticae genome has  <dig> abcd genes, tetur05g <dig> and tetur35g <dig> . this number of abcd genes equals those found in insects  <cit>  while  <dig>   <dig> and  <dig> are found in the genomes of d. pulex, h. sapiens and c. elegans, respectively . t. urticae abcds carry the eaa-like motif between tm <dig> and tm <dig> and the loop <dig> motif ), both considered to be essential for canonical abcd function  <cit> . the clear orthologous relationships we identified between t. urticae abcds and other metazoan abcds  suggests that the function of t. urticae abcds is likely to be conserved with those in other metazoans.\n",
            "\n",
            "the abce and f proteins are characterized by two linked nbds, but lack tmds and thus are involved in biological processes other than transport. the abce protein is essential in all eukaryotes examined to date and is one of the most conserved proteins known  <cit> . human abce <dig> was first discovered as an inhibitor of rnase l  <cit> , but was later found to have a more fundamental role in ribosome biogenesis and translation regulation  <cit> . in line with all eukaryotes to date, we found one abce protein  in t. urticae  and table 2) that has high amino acid identity  with d. melanogaster abce <dig>  . similar to human abce, abcf <dig> is involved in translation regulation, but probably does not play a role in ribosome biogenesis  <cit> . in most eukaryotes  <dig> abcf genes are found, and t. urticae conforms to this expectation  and table 2). the essential role of abce and abcf genes was recently shown in the flour beetle, t. castaneum, where rnai–mediated knockdown of members of the abce and f families resulted in 100% mortality in penultimate larvae  <cit> .\n",
            "\n",
            "a phylogenetic analysis of abce and abcf proteins was performed together . tetur30g <dig> grouped with metazoan abce <dig> orthologues, while each t. urticae abcf  clustered into well-supported separate clades with its metazoan orthologues, c. elegans f42a <dig>  excluded . the abce and abcf subfamilies are highly conserved, and t. urticae abce <dig> and abcfs probably have analogous roles as their orthologues in other metazoans.\n",
            "\n",
            "the abcg transporter family is present in most metazoan species, fungi and plants such as arabidopsis. for metazoan species, only abcg hts have been reported to date, while in plants and fungi also abcg fts are present  <cit> . in humans, abcg hts are primarily implicated in transport of endogenous and dietary lipids, while the human abcg <dig> functions as a multidrug efflux pump  <cit> . within the t. urticae genome we identified  <dig> abcgs, all having a typical reverse domain organization  , see additional file 3). a similar number of abcgs has also been found in d. pulex, and is the highest reported among metazoan species  <cit>  . according to sturm et al.  <cit> , the high number of abcg genes in d. pulex and d. melanogaster genomes is due to extensive lineage specific duplications. our phylogenetic analysis confirms this hypothesis not only for these two arthropod species but also for t. urticae, with twenty out of  <dig> abcgs grouping into one of the two t. urticae specific clades .\n",
            "\n",
            " see the legend of figure  <dig> for procedure and display details.\n",
            "\n",
            "one clade  consists of  <dig> t. urticae abcgs, each having a maximum of  <dig> intron . another t. urticae specific abcg clade comprises  <dig> transporters of which  <dig> are located next to each other on scaffold  <dig>  these seven abcgs show high amino acid identity  and have a conserved exon pattern , indicating a common origin by successive tandem duplication events. together with tetur06g <dig> and tetur02g <dig> they form a well-supported sister clade of d. melanogaster white and its d. pulex orthologues. interestingly, no orthologues of d. melanogaster abcgs brown and scarlet were found in t. urticae, while only one d. pulex orthologue of scarlet could be identified . about a century ago, the discovery of d. melanogaster white mutants with a remarkable eye-color phenotype marked the beginning of drosophila genetics. as a consequence, d. melanogaster white is one of the most intensively studied fruit fly genes  <cit> . d. melanogaster white dimerises with either d. melanogaster scarlet or brown to form a transporter involved in the uptake of pigment precursors  in cells of developing compound and simple eyes  <cit> . t. urticae has, in contrast to d. pulex and d. melanogaster, no compound eyes and only four simple eyes   <cit> . although no t. urticae orthologues of scarlet or brown were identified, dimerisation between the nine t. urticae co-orthologues of d. melanogaster white might result in a transporter capable of translocating pigment precursors into the cells of the spider mite ocelli . however, these transporters might also have other functions besides transporting pigment precursors, as in other species roles have been documented in courtship behavior  <cit> , transport of biogenic amines  <cit>  and uptake of uric acid  <cit>  as was shown for d. melanogaster white and/or its b. mori orthologue.\n",
            "\n",
            "in the middle of the abcg phylogenetic tree, tetur01g <dig> clustered with human abcg <dig>  d. melanogaster cg <dig> and d. pulex dappu1- <dig>  while tetur01g <dig> clustered with human abcg <dig>  d. melanogaster cg <dig>  and d. pulex dappu1- <dig>  c. elegans orthologues of human abcg5/ <dig> could not be identified . similar to human abcg5/ <dig>  d. melanogaster cg31121/cg <dig> <cit>  and d. pulex dappu1-258299/dappu1- <dig> <cit> , tetur01g <dig> and tetur01g <dig> are found juxtaposed in a head to head orientation. annilo et al.  <cit>  have suggested an evolutionary constraint on the separation of these genes, probably for the maintenance of shared regulatory regions. in humans, abcg <dig> and abcg <dig> are both glycoproteins and obligate heterodimers that limit intestinal absorption and promote biliary excretion of neutral sterols  <cit> . both tetur01g <dig> and tetur01g <dig> have at least one well-predicted glycosylation site . together with their head-to-head arrangement and the well-supported clustering with human abcg <dig> and  <dig>  it seems likely that these t. urticae abcgs have similar functions as their human counterparts.\n",
            "\n",
            "a clear orthologous relationship was found between tetur17g <dig>  d. melanogaster cg <dig> and d. pulex dappu1- <dig>  d. melanogaster cg <dig>  also known as e <dig> , is a 20-oh ecdysone  induced abc transporter that is capable of regulating 20e responses during metamorphosis, probably by removing 20e from cells  <cit> . recently, broehan et al.  <cit>  showed through rnai-mediated knockdown experiments and expression profiling that the t. castaneum orthologue of e <dig>  appears to serve a similar function in metamorphosis. in addition, it is also believed that e <dig> controls the circadian clock in adult flies through ecdysone-mediated expression of the clock gene vrille <cit> . interestingly, it was shown that not only the b. mori orthologue of e <dig>  but also four other midgut-specific b. mori abcg genes  could be induced by 20e  <cit> . as t. urticae uses a different molting hormone  compared to arthropods  <cit> , future experiments are required to establish if tetur17g <dig> has a similar function as its insect counterparts, and more specifically whether it can be induced by ponasterone a.\n",
            "\n",
            "t. urticae orthologues of human abcg <dig> and  <dig> were not identified in the phylogenetic analysis of abcg transporters, and only one was found in d. melanogaster  and d. pulex  . the function of human abcg <dig> is not well understood, while it is proposed that human abcg <dig> functions in conjunction with human abca <dig> and is involved in cholesterol homeostasis  <cit> . the d. melanogaster orthologue of human abcg <dig>  has been poorly characterized, but is expressed in the trachea  <cit> . finally, no clear orthologues of human abcg <dig> were identified . this transporter is the most thoroughly characterized human abcg and is capable of transporting an array of substrates, including anticancer drugs  <cit> . because of this feature, it has been proposed that arthropod abcgs could be involved in pesticide resistance  <cit> . however, to the best of our knowledge there has only been two studies that correlated increased arthropod abcg expression levels with resistance. in both reports, however, no functional evidence was obtained  <cit> . in fungi on the other hand, several cases of abcg fts involved in fungicide resistance, have been reported  <cit> .\n",
            "\n",
            "the abch subfamily was first discovered in d. melanogaster and is lacking in mammals, plants or fungi  <cit> . in addition to arthropods , members of this subfamily have also been reported in teleost fish  <cit> . most insects have only  <dig> abch genes, while  <dig> and  <dig> are present in d. pulex and t. urticae, respectively . liu et al.  <cit>  suggested that all insect abchs diversified from a common ancestral copy. according to our phylogenetic analysis, this insect ancestor seems not to be shared with t. urticae abch proteins . tetranychus abchs clustered, similar to d. pulex abchs , into a distinct clade, indicating that the diversity of the abch family in t. urticae has been due to lineage specific duplications. interestingly,  <dig> of the  <dig> t. urticae abchs appear to be intronless . although abchs have the same structural organization as metazoan abcgs , their physiological functions have remained enigmatic. in the zebrafish, d. rerio, abch <dig> has highest expression in brain, gills and kidney followed by lower expression in intestine, gonads, skeletal muscle and liver  <cit> . d. melanogaster abchs are enriched in the adult crop and hindgut  <cit>  and at least one of them  is glycosylated as shown by mass spectrometry of n-glycosylated peptides  <cit> . an rnai screen of d. melanogaster genes revealed that an rnai line that silences cg <dig> is lethal  <cit> . in addition, microarray analysis demonstrated an almost two-fold upregulation of a d. melanogaster abch  after cold hardening of adult fruit flies  <cit> . in the diamondback moth plutella xylostella, it was recently found that an abch transporter  was the most up-regulated abc gene in two resistant strains  <cit> . the most groundbreaking finding about insect abch function was just recently reported by the excellent study of broehan et al.  <cit> . rnai-mediated knockdown of an abch gene  in t. castaneum larvae resulted in dessication and 100% mortality. injection of tcabch-9c dsrna into adults also drastically reduced the number of eggs laid and all eggs failed to hatch. furthermore, cryosections of tcabch-9c dsrna injected larvae stained with nile red  revealed a lack of lipids in the epicuticle. based on these results, the authors suggested that tcabch-9c functions as a transporter of lipids to the cuticle and is required for the formation of a waterproof barrier in the epicuticle.\n",
            "\n",
            " see the legend of figure  <dig> for procedure and display details.\n",
            "\n",
            "expression profiling of abc genes\n",
            "we assessed expression of abc genes across development in the t. urticae london reference strain, as well as in london after transfer from a benign host  to two more challenging hosts . for the developmental and host transfer experiments, we used existing rna-seq reads, but we recalculated gene expression using newly described or corrected abc gene models curated as part of this study. we further examined previously published microarray data to assess the expression profiles of abc genes in two spider mite strains, mr-vp and mar-ab, that are resistant to multiple pesticides  <cit> .\n",
            "\n",
            "as assessed by rna-seq expression quantification, the majority of abc genes were found to be expressed –  <dig> of the  <dig> full length t. urticae abc genes had an rpkm of > <dig> in at least one of the spider mite life stages or on one of the plant hosts . in contrast, nearly all t. urticae abc fragments or pseudogenes were not expressed . most full-length t. urticae abc genes for which we detected no expression across development or on different hosts belonged to either abca, c or g subfamilies, many of which are tandem duplicated genes , figure 6). in c. elegans, tandem duplicated genes were shown to be subfunctionalized, with strong stage or tissue dependent expression  <cit> . whether t. urticae abc genes that lacked expression support in the existing data are expressed at low levels, in highly restricted expression domains, or alternatively are expressed under specific environmental conditions , remains to be determined.\n",
            "\n",
            "many t. urticae abc genes were broadly expressed, and more than half  were expressed across all developmental stages analyzed . however, the expression of many abcc and abch genes was restricted  in larvae and nymphs; overall, embryos and adult females had the highest number of non-expressed abcs ). furthermore, tetur30g <dig> and tetur20g <dig>  members of the abce and abcf subfamilies respectively, showed very high expression in all stages , coinciding with their presumed conserved role in translation regulation . similar high expression of the t. castaneum abce gene was reported in all developmental stages examined  <cit> . finally, within the other abc subfamilies, tetur27g <dig> , tetur11g <dig> , tetur04g <dig> , tetur05g <dig> , tetur02g <dig>  and tetur28g <dig>  had the highest average expression across developmental stages .\n",
            "\n",
            "upon host transfer,  <dig> and  <dig> abc genes were differentially expressed in strain london mites transferred from bean to arabidopsis and tomato plants, respectively . we found that 73%  of the differentially expressed abc genes in mites transferred to arabidopsis were also differentially expressed on tomato, and belonged to three subfamilies, abcc , -g  and -h . surprisingly, abc genes differentially expressed in two multi-pesticide resistant t. urticae strains as detected with expression microarrays  <cit>  belonged to the same abc subfamilies as identified with rna-seq data in the host transfer experiment . the abcc subfamily has been frequently linked with xenobiotic detoxification in arthropods . on the other hand, arthropod members of the abcg and h subfamilies have only very recently been reported to be associated with detoxification of xenobiotic compounds . the differential expression of members from these abc subfamilies  upon host transfer/exposure to xenobiotics should be further  validated in future studies.\n",
            "\n",
            "it is worth noting that the differentially expressed t. urticae abcc, -g and -h genes mentioned above were not among the most highly differentially expressed genes in these experiments, both in expression level and fold change. in total,  <dig> and  <dig> differentially expressed genes were identified in the two multi-resistant strains  <cit> . likewise,  <dig>  and  <dig>  differentially expressed genes were detected in mites when fed on arabidopsis and tomato in comparison with bean, respectively  <cit> . the major facilitator superfamily, another large and widespread transporter family  <cit> , showed overall a more pronounced response both in the number of genes differentially expressed, and in the fold change values of differentially expressed genes in both resistant strains and after host plant change  <cit> . thus, despite the exceptional number of abc transporters in the t. urticae genome, other transporters and non-transporter proteins also play key roles in the detoxification of xenobiotics  <cit> .\n",
            "\n",
            "CONCLUSIONS\n",
            "the spider mite t. urticae is among the most polyphagous pests worldwide and is notorious for its ability to develop resistance against numerous pesticides. one of the prerequisites to study xenobiotic metabolism  in this species is to inventory genes related to detoxification. here, we provide a survey of the abc gene superfamily, whose members have frequently been reported to play roles in detoxification, either by directly transporting toxicants out of cells, or after conjugation with glutathione. we identified  <dig> abc genes  in the genome of the spider mite t. urticae. to date, this is the largest number of abc genes reported in any metazoan species. the large number is mainly due to lineage-specific expansions in subfamilies c, g and h. of particular note, most of the differentially expressed abc genes in acaricide resistant strains and after introduction of mites to challenging host plants belong to these expanded abc subfamilies. this hints at their potential role in detoxification and may explain their retention after duplication in the mite genome. however, obtaining functional evidence that members of these abc subfamilies contribute to xenobiotic tolerance should be the priority of further research.\n",
            "\n",
            "due to the lineage specific expansions in the abcc, g, and h families, inferring the function of specific t. urticae abc family members based on phylogenetic relationships is not straightforward. nevertheless, we found clear orthologous relationships between some of the t. urticae abc proteins and human abcc <dig>  abcg <dig> and abcg <dig>  the d. melanogaster sulfonylurea receptor and the ecdysone-regulated transporter e <dig>  furthermore, we found a high conservation between t. urticae abc proteins and members of the abcb-half transporters and abcd, -e, and –f subfamilies, which are known to be involved in fundamental processes. to conclude, this study provides the first thorough abc gene analysis of a polyphagous arthropod herbivore and represents a useful resource for future biochemical and toxicological studies on the role of abc transporters in the extremely broad host range and development of pesticide resistance of t. urticae.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "we identified  <dig> abc genes  in t. urticae and  <dig> in two multi-pesticide resistant t. urticae strains, d. melanogaster cg <dig> and d. pulex dappu1- <dig>  we also identified  <dig> abc genes  in two multi-pesticide resistant t. urticae strains, d. melanogaster cg <dig> and d. pulex dappu1- <dig>  we also identified  <dig> abc genes  in two multi-pesticide resistant t. urticae strains, d. melanogaster cg <dig> and d. pulex dappu1- <dig>  we also identified  <dig> abc genes  in two multi-pesticide resistant t. urticae strains, d. melanogaster cg <dig> and d. pulex dappu1- <dig>  and  <dig> abc genes in two multi-pesticide resistant t. urticae strains, d. melanogaster cg <dig> and d. pulex dappu1- <dig>\n",
            "7\n",
            "[{'summary_text': 'fus <dig> plays an important role postembryonically by negatively regulating vegetative phase transitions through repression of ethylene action.\\nfurthermore, fus <dig> may act as a control point for vegetative phase transition by modulating the aba/ga ratio.\\nfurthermore, fus <dig> may act as a control point for vegetative phase transition by modulating the aba/ga ratio.\\nfurthermore, fus <dig> may act as a control point for vegetative phase transition by modulating the aba/ga ratio.\\nfurthermore, fus <dig> may act as a control point for vegetative phase transition by modulating the aba/ga ratio.\\nfurthermore, fus <dig> may act in the vegetative phase transition by modulating the aba/ga ratio.\\nfurthermore, fus <dig> may act in the vegetative phase transition by modulating the aba/ga ratio.\\nfurthermore, fus <dig> may act in the vegetative phase transition by modulating the aba/ga ratio.\\nfurthermore, fus <dig> may act in the developing vegetative mer'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "spatial patterning in most multicellular organisms requires genes to both establish regions of cell differentiation and specify cellular fate. in the early drosophila embryo, for example, cells are organized into boundaries by the pair rule and segment polarity genes, then they acquire distinct fates through homeotic gene expression  <cit> . homeotic genes are also required to establish boundaries during temporal patterning, whereas heterochronic genes define the timing of the cell fate decisions within those boundaries  <cit> . one challenge in developmental biology is to identify and understand the overall developmental role of genes involved in the temporal patterning of genetic programs.\n",
            "\n",
            "higher plants are well-suited for identifying genes involved in developmental timing because they continually produce easily distinguishable organs throughout the life cycle, whose fates are dependent on the time of emergence  <cit> . the types of leaves that emerge over time often show distinctive developmental changes that allow them to be classified into juvenile and adult leaves. later, when a plant enters reproductive development, the vegetative meristematic region switches to an inflorescence meristem that produces flower bracts with floral meristems in their axils  <cit> . genetic analysis in arabidopsis thaliana has identified a myriad of genes that converge to control the juvenile to adult leaf transitions and the switch of the vegetative meristem to reproductive development  <cit> .\n",
            "\n",
            "unlike flowers and leaves, which form from a shoot apical meristem, the developmental relationship between embryonic leaves  and adult foliar organs is complicated by cotyledon formation during embryonic patterning. furthermore, in many plants such as arabidopsis, cotyledons switch from a storage organ to a more leaflike photosynthetic organ soon after germination. despite these complexities, single loss-of-function mutations in arabidopsis have been identified in three genes, leafy cotyledon <dig> , leafy cotyledon <dig>  and fusca <dig> , whose mutations result in the replacement of cotyledons with organs more similar to vegetative leaves  <cit> . in lec <dig> and fus <dig> mutants, genes that encode markers of late embryogenesis are reduced or missing  <cit> . by contrast, germination markers that normally proceed late embryogenesis are precociously activated. these expression patterns suggest that lec <dig> and fus <dig> may establish temporal boundaries. although little is known about how these genes contribute to temporal patterns, it is known that fus <dig> regulates and is regulated itself by the synthesis of two terpenoid hormones, abscisic acid  and gibberellins   <cit> . the ratio of these two hormones contributes to proper cotyledon patterning by regulating the rates of cell cycling  <cit> .\n",
            "\n",
            "although extensive analyses of lec <dig>  lec <dig> and fus <dig> gene action have been carried out with respect to embryogenesis, the effects of these mutations on vegetative leaf development have not been studied extensively  <cit> . it has been shown that after germination, the first juvenile leaves of lec <dig> seedlings are shifted toward later leaf identities; however, this shift is not maintained, and successive leaves and flowering time were corrected back to a wild-type pattern  <cit> . this suggests that embryonic leaf development can have a restricted impact on future vegetative leaf identities. what remains unclear, however, is how cotyledon development impinges on later vegetative development, which is temporally and spatially distinct.\n",
            "\n",
            "to address such questions, we decided to use a combination of controlled fus <dig> activation during vegetative development with whole-genome transcript profiling. using this approach, we discovered that fus <dig> downregulates a collection of genes involved in ethylene biosynthesis and signaling. consistent with this finding, loss-of-function fus <dig> mutants show ectopic ethylene responses at both the developmental and molecular levels. the fus <dig> plants also show precocious vegetative phase change; however, unlike the lec <dig> mutants, this change is not corrected at later adult stages. more importantly, the accelerated vegetative phase transition can be suppressed by inhibiting ethylene action either genetically or pharmacologically. thus it appears that this previously defined embryonic regulator also has roles in vegetative development. one role of fus <dig> during early seedling growth is to dampen ethylene action, which in turn contributes to a slowing of subsequent vegetative phase transitions. these results add ethylene to the list of hormones that contribute to temporal patterning in arabidopsis.\n",
            "\n",
            "RESULTS\n",
            "downstream effectors of fus3\n",
            "the discovery that fus <dig> misexpression outside embryogenesis can influence vegetative leaf identity suggests that potential fus3-dependent downstream effectors can be identified through whole-genome microarray analysis  <cit> . we constructed an inducible misexpression system by transforming the fus3- <dig>  mutant with a fus3-glucocorticoid receptor  translational fusion under the control of the epidermal specific atml <dig> promoter   <cit> . transgenic seeds were germinated and grown in minimal medium  for  <dig> days, then plantlets were transferred to various concentrations of dexamethasone  to determine the minimal amount of activation needed to influence leaf identity. all concentrations of dex tested had an effect on vegetative leaf shape; increasing concentrations of dex resulted in a leaf with an increased paddle-like shape and a progressively shorter petiole . at higher than  <dig>  μm dex concentrations, the appearance of trichomes on the upper or adaxial surface of the leaves was completely inhibited . on the basis of these conditions,  <dig>  μm dex was chosen as the fus3-activating condition.\n",
            "\n",
            "putative fus <dig> targets involved in vegetative phase transitions were identified by examining the transcriptome of seedlings that transiently activate fus <dig> using the atml1:fus3-gr dex-inducible system  <cit> . to do this, transgenic seeds from atml1:fus3-gr were germinated and grown for  <dig> days in ms. after this time, half of the seedlings were transferred to ms media supplemented with  <dig>  μm dex , and the other half were transferred to ms media supplemented with dimethyl sulfoxide   for  <dig> or  <dig> days . putative fus <dig> targets were chosen on the basis of the following criteria. genes that increased  or decreased  at least twofold in expression after two days of fus <dig> activation  compared to the control  were considered to represent candidate fus <dig> targets involved in phase transition. utilizing this regime and the average of two replicate experiments,  <dig> genes increased  and  <dig> genes decreased  in expression  as the direct result of 2-day fus <dig> activation  compared to the control . to independently verify the expression changes of the 2-day dex induction experiment, a temporal replicate experiment was performed in which fus <dig> activation was extended by transferring 5-day-old seedlings onto dex for  <dig> days . all but  <dig> of the  <dig> genes selected by the 2-day experiment exhibited similar expression ratios in the 4-day experiment, demonstrating that activation of fus <dig> results in changes in expression levels of a relatively small but reproducible gene set.\n",
            "\n",
            "gene ontology  was utilized to categorize the activated and repressed genes by function . thirty-eight percent of genes upregulated by fus <dig> activation appear to be involved in enzymatic activity, whereas a number of repressed genes are annotated as having roles in hormone synthesis or action. for example, the cyp707a <dig> gene that encodes the cytochrome p <dig> monooxygenase that catabolizes aba is repressed more than twofold by fus <dig> activation   <cit> . this observation, in conjunction with the lack of fus3-dependent induction of known aba biosynthetic genes, suggests the increased aba levels observed previously in fus <dig> misexpression lines are due to decreased aba catabolism  <cit> . a second class of hormone-related genes that are repressed by fus <dig> activation at all time points and in both replicates are either involved in ethylene biosynthesis  or ethylene response . erfs and erebps are transcription factors which contain an ap <dig> dna binding domain, whereas edf  transcription factors contain both an ap <dig> and a b <dig> dna-binding domain. indeed, transcription factors constitute a dominant category  of fus3-repressed genes .\n",
            "\n",
            "potential downstream targets of fus <dig> would be expected to contain the ry promoter element motif catgca, to which the b <dig> domain of fus <dig> binds  <cit> . in fact, the ry sequence was found to be statistically enriched in both the up- and downregulated gene set compared to a randomized sample . together, these data strongly suggest that our experimental conditions identified a small gene set that is responsive to fus <dig>  interestingly, this set includes only a few genes that typically mark seed maturation and late embryogenesis, such as seed storage proteins and late embryogenesis abundant proteins .\n",
            "\n",
            "fus <dig> negatively regulates a subset of genes that are responsive to ethylene\n",
            "a closer inspection of our microarray data yielded additional ethylene-induced transcription factors  whose expression levels were repressed by fus <dig> . these genes did not quite meet the stringent criteria of exhibiting a twofold decrease following both 2- and 4-day periods of fus <dig> activation. of the  <dig> ap2/erebp/erf and rav/edf genes that are known to be induced by ethylene exposure, six genes  were dampened by fus <dig> activation, suggesting that fus <dig> plays a role in reducing the expression of genes involved in ethylene action  <cit> . this was verified by rt-pcr performed on acs <dig>  erf <dig>  edf <dig> and edf <dig> at  <dig> and  <dig> days of dex induction .\n",
            "\n",
            "fus3-mediated repression of genes involved in ethylene synthesis and action predict that fus <dig> loss-of-function mutants would exhibit an inverse expression pattern of these genes. indeed, expression of acs <dig>  edf <dig> and edf <dig> genes were consistently higher in germinating fus <dig> versus wild type at all time points surveyed after imbibition, whereas erf <dig> expression increased after  <dig> hours . these results again support a negative role for fus <dig> in regulating the expression of these ethylene responsive genes during germination. the presence of the ry element in the promoters of this gene subset may mean fus <dig> directly downregulates their expression. alternatively, fus <dig> may act through ethylene synthesis or signaling to modulate their expression. to test these possibilities, we repeated the rt-pcr analysis of acs <dig>  erf <dig>  edf <dig> and edf <dig> on 2-day old wild-type and fus <dig> seedlings germinated in the presence of either an ethylene biosynthesis inhibitor, aminoethoxyvinylglycine , or an ethylene signaling inhibitor, silver ions . in both cases, the inhibitors partially suppressed the increased levels of the gene transcripts in fus <dig> seedlings compared to untreated controls . furthermore, a key transcriptional regulator of ethylene signaling, ein <dig>  <cit> , showed increased protein stability in the emerging root cells of fus <dig> compared to wild type, and this stability was eliminated by the addition of silver ions . although these results do not exclude a direct role for fus <dig> on ethylene-responsive gene transcription, it does suggest that at least part of the aberrant gene expression observed in fus <dig> mutants does require functional ethylene synthesis or signaling.\n",
            "\n",
            "to further probe the connection of fus <dig> with ethylene action during germination, we studied the growth of dark-grown fus <dig> loss-of-function seedlings. when wild-type seedlings are germinated in the dark in the presence of ethylene, they show an exaggerated apical hook, a shortening of the hypocotyl and reduced root growth  <cit> . termed the \"triple response, \" this developmental assay has been extremely useful in genetically dissecting the role of various genes involved in ethylene synthesis or signaling in arabidopsis  <cit> . dark-germinated fus <dig> plantlets grown in the absence of ethylene exhibited shorter hypocotyl growth and hooked cotyledon development , which is consistent with their ectopic ethylene gene expression. moreover, these phenotypes were alleviated by the introduction of a mutation  that reduces ethylene signaling into the fus <dig> genetic background . to further differentiate ethylene synthesis and response, we repeated the experiment in the presence of the ethylene biosynthesis inhibitor avg. in contrast to the ein <dig> mutation, avg did not restore the hypocotyl length of fus <dig>  which suggests that the ectopic ethylene responses observed in dark-grown fus <dig> seedlings were not due to increased ethylene synthesis .\n",
            "\n",
            "fus <dig> functions to repress ethylene action during germination\n",
            "the heightened ethylene responses observed in fus <dig> after germination suggest that loss of this gene function might have other, uncharacterized effects on vegetative development. because fus <dig> is a regulator of embryonic leaf identity, we decided to examine the vegetative leaf identity and phase transitions of the fus <dig> loss-of-function mutant more closely. from the emergence of the first leaf after germination through to flowering, each wild-type leaf adopts a unique identity based on size and shape, and this graded growth variation is often referred to as the \"leaf heteroblastic series\"  <cit> . although rosette leaves always produce trichomes on the adaxial side of the wild-type leaf, trichomes on the abaxial side begin to appear on leaves only at later nodes . leaves that produce only adaxial trichomes are considered juvenile, whereas later leaves that show trichomes on both sides are regarded as adult  <cit> . also, the ratio of leaf blade-to-petiole lengths is generally lower in early juvenile leaves and becomes higher as the plant transitions to adult leaves and flowering  <cit> .\n",
            "\n",
            "comparisons of leaf profiles between the fus <dig> mutant and wild-type plants showed that the first two leaves from the mutant plant were more similar in shape and size to the third and fourth leaves of the wild type, and that this shift continued throughout the heteroblastic series . this precocious shift was reflected in the blade-to-petiole length ratio of each individual leaf  and in the leaf trichome distribution . abaxial trichomes frequently  appeared on leaf  <dig> in fus <dig> mutant plants, approximately two leaves earlier than the wild type.\n",
            "\n",
            "fus <dig> has functions outside embryogenesis\n",
            "there are two explanations for the vegetative phenotypes of the fus <dig> mutant. possibly the altered cotyledon development during embryogenesis in the mutant advances the progression of vegetative leaf identities after germination. alternatively, fus <dig> may function in the developing vegetative meristem. we tested these possibilities directly by measuring fus <dig> transcript levels using quantitative rt-pcr . in wild-type seeds, fus <dig> transcript is detected  <dig> hours postimbibition and declines over time, as previously shown  <cit> ; it is still detectable  <dig> days after germination . to further clarify the source tissue of postembryonic fus <dig> expression, a transgenic line containing a sensitive fus3:gus transcriptional fusion was germinated and sampled over time for β-glucuronidase -dependent blue histochemical staining  <cit> . the fus3:gus line used in these experiments was previously confirmed to reliably report embryonic fus <dig> expression patterns based on fus <dig> in situ hybridization  <cit> . whole-mount preparations of seedlings showed blue staining in emerging leaf primordia from 2- to 5-day-old seedlings . a similar expression pattern was found using a fus3:gfp reporter previously described  <cit>  .\n",
            "\n",
            "to functionally determine the effect of fus <dig> from embryonic and vegetative tissues on the precocious vegetative phase transition, we constructed transgenic plants in which fus <dig> could be activated either in the embryo or in vegetative tissue. fus <dig> was transformed with a fus3-gr translational fusion construct that was under the control of the native fus <dig> promoter . seeds produced from these lines in the absence of dex application were phenotypically indistinguishable from fus <dig> loss-of-function mutants, with approximately 95% of the seed being desiccation-intolerant after  <dig> weeks of storage . of the small number of seeds that did survive desiccation, all produced ectopic trichomes on their cotyledons, as expected. thus, in the absence of dex application, the fus3-gr fusion protein cannot rescue any of the known embryonic fus <dig> phenotypes. by contrast, in two independent experiments, approximately 50% of the seed produced by transgenic plants sprayed with  <dig> μm dex during flowering showed desiccation tolerance after  <dig> weeks of storage. of these, approximately 85% lacked ectopic trichomes on the cotyledons. the large reduction of fus <dig> seed phenotypes indicates that dex application during flowering is sufficient to rescue mutant embryos.\n",
            "\n",
            "from the transgenic seeds that were rescued, seven independent plants were randomly selected and grown to maturity in the absence of dex, and leaf profiles were performed. leaf profiles of all these lines still displayed accelerated heteroblastic development similar to that observed in fus <dig> leaf profiles . furthermore, the lines showed the same advanced vegetative phase transition as fus <dig> with respect to both individual leaf blade-to-petiole ratios . the abaxial trichomes, however, did show a partial reversion to a wild-type profile . together, these results suggest that specifically removing embryonic fus <dig> has a limited influence on the development of the first two juvenile leaves of arabidopsis, but not on later leaves.\n",
            "\n",
            "the inability of embryonic fus <dig> to fully rescue the precocious vegetative phase transitions of fus <dig> suggests that this gene may function outside seed development. we tested this possibility by using the fus <dig> fus3:fus3-gr transgenic lines. unlike in the previous experiment, however, we allowed transgenic lines to produce seed in the absence of dex application and then germinated the seed in the presence of  <dig> μm dex for  <dig> days. this short dex application did not influence the heteroblastic phase change in wild-type plants as measured by either the blade-to-petiole transition or the appearance of abaxial trichomes in comparison to untreated plants . by contrast, exposure of germinating fus <dig> fus3:fus3-gr transgenic seeds to dex for  <dig> days resulted in a partial  or full  shift of leaf heteroblasty back toward a wild-type profile . this rescue was partially reflected in trichome appearance  and in full-leaf measurements; in wild type, the largest leaf was leaf  <dig>  reaching a size of  <dig>  ±  <dig>  cm . similarly, the largest leaf in fus <dig> fus3:fus3-gr plants treated with dex was leaf  <dig>  which reached a size of  <dig>  ±  <dig>  cm . in contrast, leaf  <dig> was the largest leaf in fus <dig> fus3:fus3-gr untreated plants . in conclusion, a short pulse of fus <dig> during the first  <dig> days of postembryonic growth can influence leaf identity in juvenile leaves.\n",
            "\n",
            "fus3-dependent vegetative phase variation requires functional ethylene signaling\n",
            "loss-of-function fus <dig> mutants show advanced vegetative phase transition, increased expression of ethylene-regulated genes and phenotypes that are characteristic of increased ethylene signaling. to test if these phenotypes are related, a mutation conferring ethylene insensitivity was introduced into the fus <dig> background. the introduction of the ein <dig> mutation did suppress defects in vegetative phase change compared to the fus <dig> single mutant as measured by leaf profiles, blade-to-petiole ratios and abaxial trichome appearance . these results suggest that increased ethylene signaling does contribute to the advanced vegetative phase transition phenotype observed in fus <dig> \n",
            "\n",
            "to further refine the ethylene contribution, we germinated fus <dig> seeds on  <dig> μm agno <dig> for  <dig> days and transferred the resulting seedlings to soil. again, the fus <dig> phase transition phenotypes in the presence of agno <dig> were similar to those seen in the fus <dig> ein <dig> double-mutant . because in this experiment fus <dig> embryonic development occurred in the absence of the ethylene signaling inhibitor agno <dig>  any alterations in fus <dig> vegetative phase transition were due to the transient inhibition of ethylene signaling during germination. in conclusion, the inhibition of ethylene signaling during early germination is able to partially suppress the premature vegetative phase transitions observed in the fus <dig> mutant.\n",
            "\n",
            "discussion\n",
            "the fus <dig> hormonal framework\n",
            "four lines of evidence suggest that fus <dig> negatively regulates ethylene action. first, gain-of-function activation of fus <dig> dampens the expression of a collection of ethylene-responsive genes, whereas a loss-of-function fus <dig> allele shows the opposite effect. second, the ectopic expression of these ethylene-responsive genes in light-grown seedlings is dampened by the addition of ethylene synthesis or signaling inhibitors. third, loss-of-function fus <dig> seedlings show common ethylene-response phenotypes in the dark which are dependent on ethylene signaling and also have increased stability of the key positive regulator of ethylene signaling, ein <dig>  finally, the precocious vegetative phase variation observed in fus <dig> mutants is suppressed if ethylene signaling is inhibited genetically or pharmacologically.\n",
            "\n",
            "inhibition of ethylene biosynthesis does not rescue fus <dig> phenotypes in the dark, suggesting that alteration of ethylene signaling in fus <dig> is not merely a consequence of increased ethylene levels and requires functional ethylene signaling. however, genetic manipulation of fus <dig> did influence the expression of acs <dig>  encoding for an enzyme involved in ethylene biosynthesis. the altered expression of this gene might reflect a feedback response to changes in ethylene signaling. on this note, only a subset of ethylene signaling genes was regulated by fus <dig>  which suggests that the relationship between fus <dig> and ethylene is complex. many of the ethylene-responsive genes regulated by fus <dig> contain ry elements in their promoter, which suggests that this transcription factor may bind these promoters directly. a direct regulation of gene transcription may explain why the addition of ethylene inhibitors did not fully alleviate the ectopic expression of the ethylene-responsive genes. nevertheless, the negative effect of fus <dig> on the regulation of ethylene signaling adds this hormone to aba and ga as those that are dependent on fus <dig> during germination  <cit> .\n",
            "\n",
            "coordination of the synthesis and signaling of various hormones is important in regulating overall plant growth and development, and many examples of ethylene, aba and ga interactions have been reported  <cit> . in one of the best-studied cases, initial submergence of deepwater rice plants resulted in the accumulation of ethylene, which in turn enhanced ga sensitivity of the internodes to promote rapid growth  <cit> . the addition of aba had the opposite effect by decreasing the sensitivity of internode tissues to ga  <cit> . with respect to leaf identity, interactions between ethylene and aba have also been implicated in semiaquatic plants that exhibit heterophylly  <cit> . aba induces terrestrial leaf development and ethylene stimulates the formation of submerged leaves in part by reducing aba levels  <cit> . in arabidopsis, where genetic analysis can be applied, loss of ethylene response does increase aba levels in leaves, and, conversely, arabidopsis mutants deficient in aba synthesis show increased ethylene production  <cit> . the antagonistic and seemingly common relationships between aba and ga and between aba and ethylene imply that fus <dig> acts as a control point of these multiple hormone pathways.\n",
            "\n",
            "suppressing ethylene signaling in fus <dig> mutants partially rescued the defects in juvenile leaf identities and the accelerated vegetative phase variation of this mutant, but did not rescue the embryonic leaf identity phenotypes. perhaps this is not surprising, because there are major cellular differences beyond size and shape between embryonic cotyledons and juvenile and adult leaves. for example, genetic programs involved in storage reserves and desiccation appear to be embryo-specific. related to this is that controlled fus <dig> activation outside seed development was not sufficient to upregulate most of the standard seed-specific marker genes such as storage proteins, suggesting that other regulators are also required. the absence of seed-specific marker induction by the controlled activation of fus <dig> in seedlings is similar to results obtained by kagaya et al.  <cit> , who also utilized a transgene-based activation system. under their study conditions, vegetatively activated fus <dig> could induce ectopic seed storage gene expression only with the addition of aba. these observations are in contrast to constitutive fus <dig> misexpression experiments, which result in storage reserve accumulation in vegetative tissues. genetically reducing aba levels in plants that constitutively express fus <dig> also attenuates the ability of fus <dig> to activate embryonic programs ectopically, consistent with an aba requirement for full fus <dig> function  <cit> . it appears that there is a limited time during germination during which tissues are more sensitized to fus <dig> action. the presence of the transcription factor abi <dig> has been posited to define a 60-hour developmental window during germination during which aba can still arrest the growth of the emerging embryo and induce late-embryogenesis gene expression  <cit> . whether this 60-hour checkpoint also defines a window in which fus <dig> can reactivate late embryogenesis programming needs to be determined, but our expression studies place postembryonic fus <dig> within this developmental window.\n",
            "\n",
            "fus <dig> and vegetative phase change\n",
            "transitions from one leaf identity to the next depend on the coordination of at least two independent processes: the timing of leaf initiation and a program that determines the duration of a developmental phase  <cit> . in the latter case, the genes that define the juvenile boundary may repress the expression of adult leaf-promoting genes above a certain threshold. one prediction of this model is that these juvenile boundary regulators will decrease in activity over time to allow adult phase change to occur  <cit> . fus <dig> transcripts are detected after germination, and although they decrease, they can still be detected  <dig> days postgermination at a time when at least four to five leaf primordia have formed  <cit> . because leaf  <dig> is considered to be the first to show adult characteristics, the decreased temporal domain of fus <dig> expression correlates well with decreasing juvenile leaf identity. a simple model would suggest that the presence of fus <dig> in leaf primordia contributes to juvenile leaf identity and also that as fus <dig> activity drops, the transition to adult leaf phases can occur.\n",
            "\n",
            "although we have shown a vegetative role for fus <dig>  embryonic expression of fus <dig> also affects the leaf identity of the first two juvenile leaves. this is similar to studies involving lec <dig>  which suggests that the identity of the embryonic foliar organs influences the identity of subsequent vegetative organs despite their different developmental origins  <cit> . unlike fus <dig> mutants, however, aberrant vegetative phase transitions in lec <dig> plants are corrected in later leaves  <cit> . the lack of a detectable lec <dig> signal after germination, as shown in public microarrays  <cit> , may explain the difference between these mutants.\n",
            "\n",
            "in contrast to animals, plant hormones can function in many different tissues and at various times during development, resulting in various developmental outcomes. one of the mechanisms by which fus <dig> prolongs the juvenile phase appears to involve the downregulation of ethylene action, which suggests that this hormone is important in promoting adult-phase transitions . morphometric analysis of leaf growth of etr <dig> mutants have shown that this ethylene-insensitive plant produce shorter, broader leaves, which are more akin to a juvenile leaf identity  <cit> . consistent with this, the vegetative-to-floral transition in a collection of ethylene-insensitive mutants is delayed compared to the wild type  <cit> . in agreement with this, ml1:fus3-gfp plants that constitutively express fus <dig> postembryonically also show delayed flowering  <cit> . using ethylene to quicken vegetative phase transitions is consistent with the role of this hormone in promoting germination  <cit> . perhaps the short vegetative pulse of fus <dig> serves to dampen the action of this hormone after it has stimulated germination so that the leaf identities do not advance too quickly.\n",
            "\n",
            "CONCLUSIONS\n",
            "herein we provide evidence that the embryonic regulator fus <dig> also plays an important role postembryonically by negatively regulating vegetative phase transitions through repression of ethylene action. this study also implicates a role for ethylene in temporal patterning. together with previous findings showing that fus <dig> controls the embryonic-to-vegetative phase transition by modulating the aba/ga ratio, this highlights a pivotal role of fus <dig> in controlling the timing of expression of embryonic and vegetative programs through hormonal regulation. the pivotal role of fus <dig> in regulating hormone levels and responses introduces another dimension to understanding the transitions of leaf identity that occur throughout the plant's life cycle. aba and ethylene levels are also very dependent on both abiotic and biotic external signals, such as water availability, temperature and pathogen attack. hence, the phase transitions observed will be dependent not only on developmental regulators such as fus <dig> but also on environmental changes that influence hormone concentrations. in the future, it will be interesting to see if some of phenotypic plasticity observed in leaf shape can be linked not only to the relative timing of developmental regulators like fus <dig> but also to environmental conditions, both of which impinge on ethylene, aba and ga synthesis and/or signaling.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "fus <dig> plays an important role postembryonically by negatively regulating vegetative phase transitions through repression of ethylene action.\n",
            "furthermore, fus <dig> may act as a control point for vegetative phase transition by modulating the aba/ga ratio.\n",
            "furthermore, fus <dig> may act as a control point for vegetative phase transition by modulating the aba/ga ratio.\n",
            "furthermore, fus <dig> may act as a control point for vegetative phase transition by modulating the aba/ga ratio.\n",
            "furthermore, fus <dig> may act as a control point for vegetative phase transition by modulating the aba/ga ratio.\n",
            "furthermore, fus <dig> may act in the vegetative phase transition by modulating the aba/ga ratio.\n",
            "furthermore, fus <dig> may act in the vegetative phase transition by modulating the aba/ga ratio.\n",
            "furthermore, fus <dig> may act in the vegetative phase transition by modulating the aba/ga ratio.\n",
            "furthermore, fus <dig> may act in the developing vegetative mer\n",
            "8\n",
            "[{'summary_text': 'the proposed algorithm is first applied to chromosome iii of caenorhabditis elegans , containing a total of  <dig> nucleotides with  <dig> coding regions, and the results are compared with those of other popular methods.\\nthe proposed algorithm yields improvements ranging from 4% to 41% in terms of the area under the roc curve and the area under the roc curve, while the number of false negatives in a particular sensitivity is reduced by a factor of  <dig>  relative to other popular methods.\\nthe proposed algorithm is first applied to chromosome iii of caenorhabditis elegans , containing a total of  <dig> nucleotides with  <dig> coding regions.\\nthe proposed algorithm yields improvements ranging from 4% to 41% in terms of the area under the roc curve and the area under the roc curve, while the number of false negatives in a particular sensitivity is decreased by a factor of  <dig> .\\nthe proposed algorithm yields improvements ranging from 4% to 41% in terms of the area under the roc curve and the area under the roc curve, while the number of false negatives in a particular sensitivity is decreased by a factor of  <dig> .'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "when the dna sequence of a new eukaryotic organism is synthesized, the exonic  regions must be distinguished from the introns . the protein coding regions of dna have been observed to exhibit a period- <dig> property due to the non-uniform codon usage in the translation of codons into amino acids  <cit> . the aim of this paper is to use this property to identify exonic regions.\n",
            "\n",
            "several reasons for the existence of period- <dig> property have been presented in  <cit>  and  <cit> . some codons participate more in protein synthesis than others, giving rise to repetitions of a specific type of codon in the genome  <cit> . for example, the existence of a large number of gca codons in the exonic regions gives greater repetition of g, c and a nucleotides in the first, second and third codon position, respectively. in other words, the g, c and a nucleotides exhibit period- <dig> property in the exonic regions.\n",
            "\n",
            "gene finding methods based on genetic characteristics, such as promoter, cpg island, start and stop codon etc., tend to be of insufficient accuracy  <cit> . the characterization of coding and noncoding regions based on nucleotide statistics inside codons is described by bernaola et al., who employed a 12-symbol alphabet to identify the borders between coding and noncoding regions  <cit> . later, nicorici and astola segmented the dna sequence into coding and noncoding regions using recursive entropic segmentation and stop-codon statistics  <cit> .\n",
            "\n",
            "the use of signal processing techniques to identify exonic regions based on the period- <dig> property offers new opportunities for gene finding. tiawari used fourier transform spectrum to achieve this goal  <cit> . in tiawari's method, the discrete fourier transform  energy at a central frequency is calculated for a fixed length window, and the window is slid across the numerical sequence. vaidyanathan  <cit>  identified protein coding regions using an anti-notch filter which magnified regions with period- <dig> property. datta and asif  <cit>  presented a new algorithm using dft theory with a bartlett window. in another signal processing method, akhtar  <cit>  applied time domain algorithms, average magnitude difference function and time domain periodogram algorithms to identify period- <dig> property. some gene finding methods based on digital signal processing  techniques have been developed but the accuracy of these methods is low and requires improvement.\n",
            "\n",
            "in this paper, a new algorithm based on cross-correlation theory is presented. we show that the algorithm enhances the accuracy of the identification while reducing noise. the noisy waveform is cross-correlated with a periodic impulse train to provide the estimated signal. discrete wavelet transform is applied to remove extra frequencies.\n",
            "\n",
            "the remainder of the paper is organized as follows: in the methods section, the application of the cross-correlation to obtain the periodic signal plus noise is described, together with the period- <dig> behaviour detection using cross-correlation theory. the final part of this section details the use of wavelet transform to remove noise. the datasets used are introduced in the dataset section. thereafter, evaluation measures are introduced for the measurement and comparison of various methods. finally, in the results and discussion section, the results of the proposed algorithm are compared with those of the most common digital signal processing algorithms for exon prediction, in both time and frequency domains.\n",
            "\n",
            "methods\n",
            "cross-correlation\n",
            "the discrete nature of dna and the existence of period- <dig> behaviour in the exonic regions render it suitable for analysis by signal processing algorithms. we present an algorithm for the identification of the period- <dig> component based on cross-correlation techniques. the theory of cross-correlation theory is briefly explained below.\n",
            "\n",
            "correlation between two waveforms, x <dig> and x <dig>  each of length n, is defined as  <cit> :\n",
            "\n",
            "  r12= ∑n=0n-1x1x <dig> \n",
            "\n",
            "to estimate a periodic waveform that is contaminated with noise, this waveform is cross-correlated with an adjustable template waveform; the template waveform is adjusted until the cross-correlation is maximized. the resulting template is an estimate of the signal term of the periodic waveform.\n",
            "\n",
            "in our approach, a noisy waveform is cross-correlated with a periodic impulse train of period equal to that of the signal.\n",
            "\n",
            "let the signal of period np points  be s and the noise be q; therefore, the noisy waveform is s = s+q. periodic impulse train used for the cross-correlation is denoted δ, k =  <dig>   <dig>   <dig> ..., nδ, where nδ is the number of impulses. then\n",
            "\n",
            "  rsδ=1nδ ∑n=0n-1δ,k= <dig> , <dig> ...,nδ \n",
            "\n",
            "where j represents the lag, defined as the number of sampling points by which δ is shifted to the left. for j =  <dig>  and remembering that δ =  <dig> for all n ≠ knp\n",
            "\n",
            "  rsδ <cit> =1nδ   \n",
            "\n",
            "since the signal is periodic, s = s, and equation  becomes\n",
            "\n",
            "  rsδ <cit> =1nδ \n",
            "\n",
            "or\n",
            "\n",
            "  rsδ <cit> =s <cit> +1nδ ∑k=0nδq \n",
            "\n",
            "as nδ → ∞, 1nδ ∑k=0nδq→ <dig>  and therefore rsδ → s.\n",
            "\n",
            "now, the periodic impulse train is shifted on the signal by an amount depending on j. thus equation  can be written for all j's:\n",
            "\n",
            "  rsδ=s+1nδ ∑k=0nδq \n",
            "\n",
            "from equation , it can be concluded that\n",
            "\n",
            "  rsδ=s <cit> ,s <cit> ,...,sj= <dig> , <dig> ...,n- <dig> \n",
            "\n",
            "from which the periodic signal without noise can be extracted  <cit> .\n",
            "\n",
            "identification of exonic regions\n",
            "in this section, a new algorithm using the cross-correlation is proposed for the identification of exonic regions. the algorithm proceeds via the following steps:\n",
            "\n",
            " <dig>  dna sequences are converted into numerical sequences.\n",
            "\n",
            " <dig>  fir filter is applied to the numerical sequences representing dna sequences.\n",
            "\n",
            " <dig>  cross-correlation is applied to the filtered numerical sequences.\n",
            "\n",
            " <dig>  the noise effect is removed using discrete wavelet transform.\n",
            "\n",
            " <dig>  numerical conversion of the dna sequences\n",
            "to apply dsp techniques to the dna sequence to find nucleotide regions exhibiting period- <dig> behaviour, the dna sequence is first mapped onto the numerical sequence. the simplest conversion method maps four numerical sequences ia, it, ic and ig from dna sequences in binary format. in this mapping, the presence or absence of the respective nucleotides at the nth position is represented by '1' and '0', respectively. for example, given a section of dna sequence atccgatattc, the binary sequence of the nucleotide a, denoted ia, is  <cit> . the binary sequences for the other three nucleotides t, c and g are found similarly  <cit> .\n",
            "\n",
            " <dig>  applying fir filter to the numerical sequences\n",
            "after mapping the dna sequence onto its binary numerical sequence, the binary sequence is passed through a hamming window based fir filter of order  <dig> with central frequency set to 2π/ <dig>  to emphasize period- <dig> property in the exonic regions. lack of distortions in fir filters is one reason for their preferred use over iir filters in medical applications  <cit> .\n",
            "\n",
            " <dig>  applying cross-correlation theory to the numerical sequences\n",
            "most previous methods have used a window of fixed length to find the regions in dna sequences exhibiting period- <dig> property. in such methods, the window length directly affects the accuracy of the identification. typically, an appropriate window length is considered to lie within the range 240- <dig> . short length windows increase noise, while long length windows tend to miss short exonic regions.\n",
            "\n",
            "in our proposed method, the cross-correlation between the numerical dna sequence and an impulse train of periodicity  <dig>  is calculated to identify regions in the dna sequence with period- <dig> behaviour. the length of the impulse train is set at  <dig>  the impulse train signal length plays the same role as the window length in previous approaches. following the cross-correlation calculation, the impulse train slides across the numerical sequence by an amount j. the impulse train with periodicity of three considered as δ and ba, bt, bc and bg are the fir filter outputs for the input ia, it, ic and ig sequences respectively. then,\n",
            "\n",
            "  ma= ∑n=0n-1baδ \n",
            "\n",
            "  mt= ∑n=0n-1btδ \n",
            "\n",
            "  mc= ∑n=0n-1bcδ \n",
            "\n",
            "  mg= ∑n=0n-1bgδ \n",
            "\n",
            "different energy levels of the period- <dig> components exist in binary sequences ma, mt, mc and mg. thus, the output energy spectrum is the combination of the four separate outputs\n",
            "\n",
            "  m=ma+mt+mc+mg \n",
            "\n",
            "in this energy spectrum, a peak corresponds to the presence of a period- <dig> component on that region, implying that the region is exonic.\n",
            "\n",
            " <dig>  decreasing the noise using discrete wavelet transform\n",
            "decreasing noise increases the accuracy of exonic region identification. as seen from equation , a small window size, required for the detection of small exons, will not diminish noise sufficiently. hence we apply discrete wavelet transform  to decrease the noise in the output spectrum.\n",
            "\n",
            "dwt has been used for de-noising in various signal processing applications. in protein coding region detection, haar wavelet has previously been employed for noise suppression  <cit> . our proposed algorithm uses dmey wavelet to remove noise and thereby increase the accuracy of the exonic region identification.\n",
            "\n",
            "to this end, by down-sampling the output of low pass and high pass filters, samples are divided into two signals; high frequency samples  and low frequency samples , each embracing half the number of samples as the original signal. figure  <dig> shows this procedure operating over three levels.\n",
            "\n",
            "the signal x is passed first through the high pass filter, h, then through the low pass filter, g  <cit> .\n",
            "\n",
            "  shigh= ∑nx.h \n",
            "\n",
            "  slow= ∑nx.g \n",
            "\n",
            "approximation and detail signals for the output power spectrum of the sequence f56f <dig>   at positions 7021- <dig> are shown in figures 4b and 4c. by removing the detail signal and considering only the approximation signal, the extra frequencies are removed and the output power spectrum is smoothed. therefore, the noise effect is decreased, while the accuracy of the identification is enhanced.\n",
            "\n",
            "datasets\n",
            "standard datasets are used to compare the efficacy of different algorithms at identifying exonic regions. exon and intron positions in these databases are available and when dsp methods detect the position of exons, these positions are compared with real positions. the proposed algorithm is first applied to chromosome iii of caenorhabditis elegans , containing a total of  <dig> nucleotides with  <dig> coding regions, and the results are compared with those of other popular methods. the results of the proposed algorithm for the sequence f56f <dig>  of c. elegans  are separately presented. this sequence has five exonic regions at positions 928- <dig>  2528- <dig>  4114- <dig>  5465- <dig>  and 7255- <dig>  also analysed in this paper are the bg <dig>  <cit>  and hmr <dig>  <cit>  datasets. bg <dig> is a genomic test dataset of  <dig> single gene vertebrate sequences prepared by burset and guigo  <cit> . hmr <dig> comprises  <dig> single-gene human, mouse, and rat sequences selected in  <dig> by rogic et al.  <cit>  to test and evaluate the performance of gene structure prediction algorithms.\n",
            "\n",
            "evaluation measures\n",
            "to accurately compare different methods, the evaluation is performed at the nucleotide level. in the identification of exonic regions using dsp techniques, some parameters are defined by changing the threshold level in the output spectrum. those parameters which make the comparison possible are defined in this section. in the identification step, the number of nucleotides correctly predicted as exons is denoted true positive , while the number of nucleotides correctly predicted as introns is denoted true negative . similarly, the number of intron nucleotides predicted as exon nucleotide is the false positive  value, while the number of exon nucleotides predicted as intron nucleotides is the false negative  value. from these four defined quantities, the sensitivity and specificity parameters are determined as follows  <cit> :\n",
            "\n",
            "  sn=tptp+fn \n",
            "\n",
            "  sp=tptp+fp \n",
            "\n",
            "the sensitivity sn is the proportion of exon nucleotides that have been correctly predicted as exons, and the specificity sp is the proportion of predicted exon nucleotides that actually exist in the exonic regions. these parameters alone are not suitable for evaluation because at high sensitivity, the specificity is low and vice versa. therefore, another measure known as the approximate correlation  has been defined. this parameter combines sensitivity and specificity as shown  <cit> .\n",
            "\n",
            "  acp=14* \n",
            "\n",
            "  ac=* <dig> \n",
            "\n",
            "in applying dsp techniques to gene searching, other parameters have been described. a most popular evaluation measure is the receiver operating characteristic  curve. by selecting different threshold levels, different values of tp for a given fp are calculated at each threshold and the roc curve is constructed from the various tps and their corresponding fps. the area under the roc curve  is used as an evaluation measure; the greater the auc, the higher the accuracy of the gene finding algorithm  <cit> . another means by which to compare identification accuracy between methods is the calculation of specificity for different sensibilities. since the majority of genomes comprise intronic and intergenic regions, the calculation of fp can provide a useful comparison measure  <cit> .\n",
            "\n",
            "threshold selection method\n",
            "to discriminate between coding and noncoding regions, a threshold is imposed on the output power spectrum. the selection of a proper threshold can optimise the accuracy of the identification; however, the calculation of an optimum threshold value itself raises problems  <cit> . therefore, in this paper, the sensitivity, specificity and approximate correlation measures are defined by changing the threshold level, to accurately compare different methods. in this section, we discuss implementation of the threshold selection.\n",
            "\n",
            "to select an appropriate threshold, the method of kwan et al.  <cit>  is used. the mean and standard deviation of the period- <dig> values determined from a training set of exon and intron sequences are used to calculate the threshold level t, defined as:\n",
            "\n",
            "  t=sdp3e*meanp3i+sdp3i*meanp3esdp3e+sdp3i \n",
            "\n",
            "where meanp3e and sdp3e represent respectively the mean and standard deviation of the period- <dig> values obtained from the exon sequences of a training set, and meanp3i and sdp3i represent respectively the mean and standard deviation of the period- <dig> values obtained from the intron sequences of the same training set.\n",
            "\n",
            "the  <dig> multi exon genes from chromosome iii of c. elegans provide data for training. the calculated threshold level is  <dig>  this threshold level was applied to the f56f <dig>  gene in chromosome iii of c. elegans as shown in figure  <dig> a. clearly, at this threshold, all five regions are correctly identified as coding regions. however, there also exist small non-coding regions around position  <dig> which are misidentified as coding regions. since the characteristics of the dna sequence can change significantly at different positions, even within the same dataset, a static threshold may yield incorrect identifications at some positions. therefore, an adaptive threshold selection algorithm such as that described in  <cit>  is required for exon prediction. in tables , , , and  our proposed algorithm is compared with other algorithms over a range of thresholds.\n",
            "\n",
            "RESULTS\n",
            "in this section, the results of the proposed algorithm are compared with those of established methods, namely, average magnitude difference function , time domain periodogram   <cit> , anti-notch filter   <cit> , fourier transform spectrum   <cit>  and asif  <cit> . as mentioned in the previous section, to evaluate and compare the results, measures such as the area under the roc curve, the specificity and the number of false negatives in a particular sensitivity are computed. the approximate correlation measure for different threshold levels is also calculated. our proposed algorithm is first applied to the gene sequence f56f <dig> . in figure 5a, the output power spectrum computed by equation  is displayed. the output power spectrum of the an filter, tdp and dft methods is shown in figures  <dig> b, c and 5d respectively. exonic regions that should be identified in this figure are marked as shaded regions. it should be noted that figure  <dig> a is the output of equation  after de-noising with dwt, as shown in figure  <dig> b. the strongest feature of our proposed algorithm is the noise reduction. not only is the noise reduced by increasing the window length, but the small length exonic regions can be identified. unlike the established methods, the accuracy of identification does not decrease by changing the window length up to a specific value. figure  <dig> shows the effect of changing window length on area under the roc curve for the f56f <dig>  sequence. according to this curve, the identification accuracy of our algorithm is fixed for window lengths ranging from  <dig> to  <dig> bp, whereas that of the other tested methods depends on window length. the window length varies according to gene length. the decreasing noise effect and magnification of the period- <dig> component under fir filtering causes the peaks to coincide with exon positions and enables detection of small exons, such as the first exon in f56f <dig>  .\n",
            "\n",
            "in table  <dig>  the approximate correlation and specificity for specified sensitivities are presented for our proposed method and for the other tested methods . we observe that our algorithm yields the highest value of both parameters.\n",
            "\n",
            "sn\n",
            "sp\n",
            "the proposed algorithm is then applied to chromosome iii of caenorhabditis elegans , comprising  <dig> nucleotides with  <dig> coding regions, and the results are again compared with the outputs of other popular methods. different evaluation measures for the proposed algorithm, an filter and tdp methods are shown in table  <dig>  clearly, the proposed method outperforms an filter and tdp methods. it achieves a larger area under the roc curve, fewer false positives and higher specificities and approximate correlation compared with an filtering and tdp. by way of illustration, at a sensitivity of 20% the false positive output of our algorithm is  <dig> bp compared with  <dig> bp and  <dig> bp for an filtering and tdp, respectively. in addition, our proposed method exhibits relative improvements of 3% and 5% respectively over an filter and tdp methods in the approximate correlation measure.\n",
            "\n",
            "sn\n",
            "sp\n",
            "sp\n",
            "sp\n",
            "the proposed algorithm was finally applied to the hmr <dig> and the bg <dig> datasets. the output results are shown in table  <dig>  with regard to the hmr <dig> dataset, our algorithm outputs the least number of nucleotides incorrectly identified as exons. at a sensitivity of 30%, the number of false positives in the cross-correlation method is improved by a factor of  <dig>  relative to the next-best performing method, asif. our proposed algorithm shows relative improvements of 21%,  <dig> %, 24%, 18% and 4% over the dft, an filter, asif, amdf and tdp methods respectively, in terms of the area under the roc curve. similar superiority of our proposed algorithm is apparent for the bg <dig> dataset.\n",
            "\n",
            "sn\n",
            "sn\n",
            "sp\n",
            "sp\n",
            "sp\n",
            "sp\n",
            "sp\n",
            "sp\n",
            "sn\n",
            "sp\n",
            "sn\n",
            "sp\n",
            "CONCLUSIONS\n",
            "this paper presents a new algorithm based on cross-correlation theory, designed to increase the accuracy of exonic region identification. the fir filter makes it easier to identify the exonic regions. the main advantage of the proposed method is its reduced dependency on the window length as a result of the decreasing noise effect. the ability to detect small exonic regions is another advantage of this algorithm. the final step of the algorithm utilizes the discrete wavelet transform to reduce noise. compared with established time and frequency domain methods, the proposed algorithm yields improvements ranging from 4% to 41% in terms of the area under the roc curve for the hmr <dig> and bg <dig> datasets. our proposed method also minimises the number of nucleotides incorrectly predicted as exonic. this decrease in the number of false positives is responsible for the increase in specificity; for example, at a sensitivity of 30%, our proposed algorithm yielded 15% to 85% improvement in specificity over other tested methods. as can be seen from tables  <dig> and  <dig>  our algorithm confers significant improvement on the accuracy of exonic region identification.\n",
            "\n",
            "authors' contributions\n",
            "this work is carried out under the close guidance of ar and gk, who conceived of the study, and who participated in its design and coordination. oa implemented the method. the manuscript was written by oa and edited by ar and gk. all authors have read and approved the final manuscript.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the proposed algorithm is first applied to chromosome iii of caenorhabditis elegans , containing a total of  <dig> nucleotides with  <dig> coding regions, and the results are compared with those of other popular methods.\n",
            "the proposed algorithm yields improvements ranging from 4% to 41% in terms of the area under the roc curve and the area under the roc curve, while the number of false negatives in a particular sensitivity is reduced by a factor of  <dig>  relative to other popular methods.\n",
            "the proposed algorithm is first applied to chromosome iii of caenorhabditis elegans , containing a total of  <dig> nucleotides with  <dig> coding regions.\n",
            "the proposed algorithm yields improvements ranging from 4% to 41% in terms of the area under the roc curve and the area under the roc curve, while the number of false negatives in a particular sensitivity is decreased by a factor of  <dig> .\n",
            "the proposed algorithm yields improvements ranging from 4% to 41% in terms of the area under the roc curve and the area under the roc curve, while the number of false negatives in a particular sensitivity is decreased by a factor of  <dig> .\n",
            "9\n",
            "[{'summary_text': 'the expression of  <dig> genes involved in the pho regulon, including those involved in amino acid acquisition , energy metabolism , acid resistance  and fimbriae biosynthesis, was down-regulated in strain k <dig>  however, expression of  <dig> genes involved in the pho regulon was up-regulated in the pst mutant.\\nthe pst mutant showed a decreased capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\\nthe pst mutant showed a reduced capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\\nthe pst mutant showed a decreased capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\\nthe pst mutant showed a decreased capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\\nthe pst mutant showed a decreased capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\\nfurthermore, the pst mutant showed a decreased capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "in escherichia coli and many other bacterial species, the pho regulon is activated when cells face phosphate limitation, whereas its expression is inhibited in excess of phosphate. the two-component system  phor/phob responds to environmental phosphate concentration variations and has been shown to control expression of at least  <dig> genes  <cit> . phor is an inner membrane sensor protein that responds to periplasmic orthophosphate  concentrations and phob is the response regulator of the pho regulon. phor is activated when the pi concentration is below  <dig> μm. under pi-limited conditions, phospho-phob binds to specific dna sequences known as pho-boxes, located within pho-dependent promoter regions to either induce or repress gene transcription. the pstscab-phou operon encodes the pst system and belongs to the pho regulon. the pst system encodes an atp-binding cassette  transporter involved in the transport of pi. importantly, mutations in the pst system result in constitutive expression of the pho regulon, regardless of environmental phosphate availability, and affect virulence of many pathogenic bacteria  <cit> .\n",
            "\n",
            "extraintestinal pathogenic escherichia coli  strains are associated with various diseases, including urinary tract infections , newborn meningitis , abdominal sepsis and septicemia  <cit> . in poultry, apec strains are a frequent cause of extraintestinal infections, collectively called colibacillosis. in addition, e. coli o <dig> strains can also cause diseases in hosts other than poultry, such as swine, sheep, and humans  <cit> .\n",
            "\n",
            "we have previously shown that inactivation of the pst operon in porcine and avian expec strains resulted in constitutive expression of the pho regulon and rendered the strains avirulent  <cit> . moreover, it was reported that the phob gene of the avian pathogenic e. coli  χ <dig> strain is expressed during infection  <cit> . in accordance, in expec strains, other phob regulated genes were shown to be expressed in vivo  <cit> . a number of reports have described an association between the pst system, the pho regulon and bacterial virulence  <cit> . although inactivation of pst genes has been shown to affect the virulence of a number of bacterial pathogens, the mechanisms underlying this attenuation have not been elucidated.\n",
            "\n",
            "microarray studies have been conducted to understand how e. coli k- <dig> responds to growth in phosphate-rich or phosphate-limited conditions in phob mutant strains  <cit> . moreover, proteome profiles of cells grown under phosphate-rich or phosphate-limited conditions revealed that the overall phosphate response of e. coli may comprise up to  <dig> genes  <cit> . however, these studies investigated non-pathogenic e. coli k- <dig> grown in phosphate-limiting conditions. to understand global responses resulting from a mutation in the pst system, which constitutively activates the pho regulon, and its relationship with apec virulence, the transcriptional profile of the apec χ <dig> strain was compared with its isogenic pst mutant  using the affymetrix genechip® e. coli genome  <dig>  array. the pho regulon is clearly not a simple regulatory circuit for controlling phosphate homeostasis; it is part of a complex network important for both bacterial virulence and the global stress response. regulatory changes incurred due to inactivation of the pst system resulted in modulation of genes involved in cell surface modifications, production of fimbrial adhesins, and protection against environmental stresses in the apec mutant. these regulatory changes are likely to contribute to its reduced virulence and increased sensitivity to environmental stresses which may be encountered during host infection.\n",
            "\n",
            "methods\n",
            "bacterial strains, media and growth conditions\n",
            "the apec strain χ <dig>  the χ <dig> Δpstcab mutant , and the complemented χ <dig> Δpstcab mutant  were previously described  <cit> . the χ <dig> Δfim strain χ <dig> was generated by allelic exchange using suicide vector pdm <dig> as described by marc et al.  <cit> . e. coli strains were grown at 37°c in luria-bertani  medium. antibiotics or supplements were used at the following final concentration, when required:  <dig> μg/ml for nalidixic acid ,  <dig> μg/ml for kanamycin ,  <dig>  μg/ml for chloramphenicol  and  <dig> μg/ml for 5-bromo-4-chloro-3-indolylphosphate di-sodium .\n",
            "\n",
            "alkaline phosphatase assay\n",
            "alkaline phosphatase activity was measured as described previously with slight modifications  <cit> . briefly, cells grown under different conditions were adjusted to an optical density of  <dig>  at  <dig> nm , and  <dig> μg/ml of p-nitrophenyl phosphate  was added to cells permeabilized by  <dig> μl of  <dig> % sds and  <dig> μl of chloroform. color development was monitored at  <dig> nm and alkaline phosphatase activity was expressed in miller units , calculated as follows:  <dig>  <dig> × /t  × v  × od600].\n",
            "\n",
            "rna isolation\n",
            "rna extractions were performed using four biological replicates of cultures of strains χ <dig> and k <dig>  briefly, overnight cultures grown at 37°c in lb broth were diluted 100-fold into  <dig> ml of lb broth and were allowed to grow to mid-log phase . rna samples were extracted with the ribopure™-bacteria kit , according to the manufacturer's recommendations, with the exception that dnase i treatment was performed twice. rna concentration and purity were measured using the nanodrop nd- <dig> spectrophotometer  and the  <dig> bioanalyzer . quantitative rt-pcr  was performed on each rna sample using a smart cycler  to detect any dna contamination. for this purpose, primers targeting the rpod gene were used. rnas were stored at -80°c for future use.\n",
            "\n",
            "cdna synthesis and biotinylation\n",
            "ten μg of rna dissolved in  <dig> μl of rnase-, dnase- and pyrogen-free water  were supplemented with  <dig> μl of the genechip® eukaryotic poly-a rna control kit  and converted into cdna using superscript ii and random hexanucleotide primers , according to the manufacturer's instructions. following cdna synthesis,  <dig> μl of  <dig> mg/ml of rnase a  were added to the reaction mix and samples were incubated at 37°c for  <dig> min to degrade remaining rna. cdnas were purified using microcon ym- <dig> centrifugal filters . concentration and purity were measured using the nanodrop nd- <dig> spectrophotometer. a range between  <dig> to  <dig> μg of cdnas was fragmented using dnase i , at concentration of  <dig>  u/μg of cdna, and by incubating at 37°c for  <dig> min. fragmentation was stopped by heating the reaction mixture at 98°c for  <dig> min. fragmented cdnas were 3' biotinylated using genechip dna labelling reagent  at  <dig>  mm and  <dig> u of terminal deoxynucleotidyl transferase  at 37°c for  <dig> min. the reaction was stopped by adding  <dig> μl of  <dig>  m edta . a gel-shift assay on 14% polyacrylamide gel using immunopure neutravidin  was monitored to determine biotin incorporation.\n",
            "\n",
            "dna microarray hybridization and analysis\n",
            "the cdnas were hybridized onto the affymetrix genechip® e.coli genome  <dig>  array  as recommended by the manufacturer . hybridizations were performed at the génome québec innovation centre . data were processed using the robust multiarray average algorithm  for normalization, background correction and expression value calculation  <cit> . expression levels obtained from four independent biological replicates were compared using flexarray  <dig>  software  <cit> . robustness of the data was further enhanced by the eb  algorithm and p-value calculation. since the rma algorithm diminished the false positive rate and compressed the fold change, a  <dig> -fold change cut-off value was used for determination of the differentially expressed genes  <cit> . functional classification was done according to the tigr's comprehensive microbial resource   <cit> . pathogen-associated orfs were classified as such, since they represent sequences corresponding to one or more of the three pathogenic e. coli genomes , but which are absent from the genome of e. coli k- <dig> strain mg <dig> \n",
            "\n",
            "quantitative rt-pcr\n",
            "the qiagen quantitect® sybr® green rt-pcr kit was used for qrt-pcr according to the manufacturer's instructions. primer pairs were designed using the primerquest software from integrated dna technologies  to yield pcr products varying between  <dig> to  <dig> bp. the tus gene was used as a housekeeping control. each qrt-pcr run was done in triplicate and for each reaction, the calculated threshold cycle  was normalized to the ct of the tus gene amplified from the corresponding sample. the fold-change was calculated using the 2-ΔΔct method  <cit> . sequences of primers used for qrt-pcr analysis are available in additional file  <dig> \n",
            "\n",
            "in silico search for pho box\n",
            "we used the dna pattern search program available on the colibri© website . the search was performed by using the pattern n !c!c!g !a!g!g!gn!g!g   a nnn!g, where \"n\" represents any nucleotide, characters between square brackets \" \" means an ambiguity and \"!\" before a character indicates the negation of this position. this pattern was designed based on different e. coli pho box sequences presented by blanco et al.  <cit> . we performed the dna pattern search on the e. coli k- <dig> mg <dig> whole genome, with no mismatch allowed, narrowing our search to the  <dig> bp upstream of predicted start codons. we also used the gene list containing predicted phob binding sites using the pho box weight matrix elaborated by yuan et al.  <cit> . searches were performed against the e. coli mg <dig>  edl <dig>  sakai, and cft <dig> genomes which are represented on the affymetrix genechip® e. coli genome  <dig>  array.\n",
            "\n",
            "sensitivity of e. coli strains to reactive oxygen intermediate -generating agents\n",
            "sensitivity to oxidative stress generating agents was determined by an agar overlay diffusion method on lb plates  as described by sabri et al.  <cit> . briefly, overnight cultures grown in lb broth were adjusted to an od <dig> of  <dig> . then,  <dig> μl of each culture were suspended in molten top agar  and poured over the agar plates. filter paper disks  were added to the surface of the solidified overlays and  <dig> μl of hydrogen peroxide , plumbagin , phenazine methosulfate   or phenazine ethosulfate   were spotted onto the disks. the plates were then incubated overnight at 37°c and following growth, the diameters of inhibition zones were measured.\n",
            "\n",
            "yeast cell aggregation assay\n",
            "the yeast aggregation assay was derived from a micro-hemagglutination assay in 96-well round-bottom plates  <cit> . briefly, cultures were grown to mid-log phase  in lb broth at 37°c with shaking   or for  <dig> h without shaking at 37°c to enhance expression of type  <dig> fimbriae. bacterial cells were centrifuged, and pellets were suspended in phosphate buffered saline  to an initial suspension of approximately  <dig> ×  <dig> cells/ml. samples were then serially diluted two-fold in microtiter wells, and equal volumes of a 3% commercial yeast suspension were added to each of the wells. after  <dig> min of incubation on ice, yeast aggregation was monitored visually, and the agglutination titer was recorded as the most diluted bacterial sample giving a positive aggregation reaction. the Δfim type  <dig> fimbriae mutant strain χ <dig> was used as a negative control.\n",
            "\n",
            "preparation of fimbrial extracts and western blotting\n",
            "following growth of cultures with agitation at 37°c in lb to mid-exponential growth phase , fimbrial extracts were prepared and western blotting was performed as described previously  <cit> . briefly, fimbrial extracts were separated by sodium dodecyl sulfate -15% polyacrylamide gel electrophoresis in minigels as previously described by laemmli  <cit> . proteins were transferred to nitrocellulose membranes  using a mini trans-blot electrophoretic cell  for  <dig> min at  <dig> v. the membrane was blocked with startingblock  supplemented with  <dig> % tween  <dig> . incubations with primary  and secondary  antibodies were carried out for  <dig> h at room temperature. supersignal west pico chemiluminescent substrate  was used for detection. primary antibodies, raised against type  <dig> fimbriae from e. coli strain bam, were used, and react specifically with type  <dig> fimbriae from different apec strains  <cit> .\n",
            "\n",
            "electron microscopy\n",
            "cells for electron microscopy were grown as described above for microarray experiments. a glow-discharged formvar-coated copper grid was placed onto a drop of bacterial culture for  <dig> min to allow the cells to adsorb. the excess of liquid was then removed using a filter paper, just before a drop of 1% phosphotungstic acid  was placed onto the grid. samples were left to air dry and viewed using a phillips em <dig> transmission electron microscope.\n",
            "\n",
            "microarray accession numbers\n",
            "microarray data are available at the national center for biotechnology information gene expression omnibus database  under accession number gse <dig> \n",
            "\n",
            "RESULTS\n",
            "microarray design to identify differentially expressed genes in the apec pst mutant\n",
            "to assess the effects of a pst mutation as well as pho regulon activation on apec strain χ <dig>  a transcriptional profiling approach was used. strain χ <dig> and its isogenic pst mutant  were grown in a phosphate-rich  medium. in this medium, strains χ <dig> and k <dig> grew well, and they showed similar growth curves . the alkaline phosphatase activity of phoa is commonly used to evaluate the activation state of the pho regulon  <cit> . the pho regulon induction reaches its maximal rate at mid-log phase of growth in strain k <dig> as determined by phoa activity , whereas it was repressed in the wild-type parent strain χ <dig> . thus, in the k <dig> strain, the pho regulon is highly activated even during growth in phosphate-rich lb medium.\n",
            "\n",
            "the affymetrix genechip e. coli genome  <dig>  array contains oligonucleotides corresponding to four e. coli genomes  cft073). avian pathogenic escherichia coli , a frequent cause of extraintestinal infections in poultry, are categorized as expec. although the genome sequence of the χ <dig> apec strain is not yet available, it is now known that the apec o1:k <dig> strain genome shares 90% similarity with the cft <dig> orfs  <cit> . this suggests that probes on the genechip could represent at least in part the genome of χ <dig>  in addition, to determine the representative gene targets on the array that were present in strain χ <dig>  hybridization of χ <dig> genomic dna revealed that, among the  <dig>  orfs represented on the dna array,  <dig> were specific to the χ <dig> genome. the non-k- <dig> orfs were considered as pathogen-associated orfs.\n",
            "\n",
            "significant transcriptional changes in the k <dig> pst mutant\n",
            "the transcriptomic study identified  <dig> genes that were differentially expressed by at least  <dig> -fold in the pst mutant strain k <dig> when compared to the apec strain χ <dig>  with a p-value of ≤  <dig>  and an estimated false discovery rate  of  <dig> %. globally,  <dig> genes were up- and  <dig> were down-regulated. specific subsets of differentially expressed genes of known physiological relevance or putative function are discussed below and presented in tables  <dig> and  <dig>  a complete list of the differentially expressed genes is available online . these tables contain genes that are discussed below and represent significant change at the transcriptional level occurring in pho regulon activation. the known genes of the pho regulon were induced varying from 145- to 2-fold. besides those, the gadw gene showed the highest induction , whereas the greatest repression was observed for the yhcn gene . functional classification of the differentially expressed genes  indicated predominant transcriptional changes among genes associated with cellular homeostasis and metabolism. genes that were up-regulated included those encoding proteins of unknown function, transport and binding proteins, energy and central intermediary metabolism, and transcription . genes that were down-regulated also included the unknown function genes, as well as those involved with protein fate; protein synthesis; dna metabolism; purine, pyrimidine, nucleoside and nucleotide pathways; and cell envelope proteins . in addition,  <dig> small rnas  were differentially expressed. as expected, expression patterns of the genome-wide transcriptional response observed in the pst mutant in phosphate-rich medium shared similarities with transcriptomic analyses of non-pathogenic e. coli k- <dig> during phosphate starvation  <cit> . however, in contrast to e. coli k- <dig> which demonstrated marked phosphate limitation by early stationary phase , the apec pst mutant exhibited an early maximal induction of the pho regulon following growth to mid-log phase  in lb medium  <cit> . in addition, many genes associated with stress responses and metabolic functions were differentially expressed by the apec pst mutant when compared to its wild-type parent.\n",
            "\n",
            "a the first up-regulated gene of the operon is shown, b members of the pho regulon c indicates the presence of pho box in the gene promoter, d genes belonging to rpos regulon, e genes involved in acid stress response, f genes involved in oxidative stress response, g pathogen-associated orf, h genes in bold are up-regulated in k <dig> strain.\n",
            "\n",
            "a the first down-regulated gene of the operon is shown, b indicates the presence of pho box in the gene promoter, c genes belonging to rpos regulon, d genes involved in acid stress response, e genes involved in oxidative stress response, f pathogen-associated orf, g genes in bold are down-regulated in k <dig> strain.\n",
            "\n",
            "in contrast to the many differentially expressed e. coli genes involved in general metabolic or transport functions, only a few confirmed virulence- or stress-response associated genes demonstrated significant transcriptional changes in the pst mutant. of these, genes involved in acid and oxidative stress response were modulated. furthermore, genes encoding type  <dig> and f <dig> fimbriae were down-regulated in strain k <dig>  however, the majority of the pathogen-associated orfs identified are of unknown function. among the  <dig> differentially expressed genes,  <dig> were not found in e. coli k- <dig> strain mg <dig> and were considered as pathogen-associated orfs. of these,  <dig> were specific to cft <dig> and  <dig> to the ehec  genomes. overall we identified  <dig> pathogen-associated orfs that are influenced by constitutive expression of the pho regulon. however, the functions encoded by these orfs are currently unknown.\n",
            "\n",
            "modulation of the pathogen-associated orfs  could contribute to the reduced virulence of strain k <dig> observed by lamarche et al.  <cit> . interestingly, the virulence-associated gene systems sitabcd, irobcden, iucabcd-iuta, coding for metal transporters present in χ <dig>  that hybridized to the genomic array, were not differentially expressed in the attenuated pst mutant.\n",
            "\n",
            "blanco et al. showed that the c-terminal domain of phob interacts with a  <dig> bp region of dsdna that consists of two direct repeats of  <dig> bp  <cit> . each  <dig> bp repeat has a conserved  <dig> bp region  followed by a less conserved  <dig> bp segment. by in silico analysis and by using the list of genes identified in e. coli genomes by yuan et al., a number of genes with predicted pho boxes not associated with pi metabolism were identified among the differentially expressed genes  <cit> . ninety-one genes or transcriptional units possessed putative pho box, including the known pho regulon members . differentially regulated genes such as those involved in amino acid acquisition , energy metabolism , acid resistance  and f <dig> fimbriae biosynthesis  possess putative pho box. distribution of putative pho regulon members across different functional classes supports the hypothesis that the pho regulon overlaps and interacts with several other control circuits  <cit> . however, further studies will be required to establish whether these genes are directly regulated by phob.\n",
            "\n",
            "validation of microarray results by qrt-pcr\n",
            "validation of microarray results was achieved using qrt-pcr. fifteen genes representing a wide range of gene expression ratios  in k <dig> strain were selected for comparative qrt-pcr analysis . comparison of gene expression by microarray hybridizations and qrt-pcr demonstrated a very high level of concordance between the datasets, which is represented by a correlation coefficient of  <dig>  and a pearson correlation of  <dig>  .\n",
            "\n",
            "global stress response\n",
            "globally, the transcriptional profile of strain k <dig> indicated that in addition to up-regulating genes associated with scavenging pathways for phosphate acquisition and conservation, the pst mutant also demonstrated changes in expression of genes dealing directly with global stress. many lines of evidence suggest that the pho regulon and the stress response are interrelated  <cit> . the gene expression profile of strain k <dig> represents an exacerbated response in which the strain reacts to an inaccurate detection of phosphate-limiting conditions when, in fact, phosphate levels are actually abundant. this adaptive response entails the induction of different mechanisms to optimize the acquisition and the bioavailability of phosphate and to maintain the essential biochemical reactions active. in addition, during growth, the k <dig> mutant activates a generalized stress response adaptation.\n",
            "\n",
            "genes associated with the rpos regulon\n",
            "the pho regulon and the rpos regulon are interrelated regulatory networks of the bacterial adaptive response  <cit> . rpos is a sigma factor implicated in the cellular response to many stresses. it is also implicated in the stationary phase and the induction of genes in nutrient-limiting environments  <cit> . the rpos gene is expressed under a variety of growth conditions, but regulation and rpos production is largely dependent on post-transcriptional stability  <cit> . during exponential growth, in the k <dig> mutant, the rpos gene was not differentially expressed. however, the rpos-regulatory gene irap was induced  <dig> -fold. irap encodes an anti-adaptor protein that enhances rpos stability and accumulation by inhibiting its targeting to the clpxp degradosome  <cit> . moreover, the small regulatory rna  oxys, that inhibits the translation of rpos, was down-regulated . bougdour and gottesman have recently shown that the transcription of the irap gene is promoted by ppgpp accumulation during phosphate starvation  <cit> . these results suggest that in strain k <dig>  during exponential phase, the sigma factor rpos stability is increased, leading to regulatory expression of numerous rpos-dependent genes. accordingly,  <dig> genes known to be regulated by the rpos sigma factor are differentially expressed in strain k <dig> . hence, in the exponential growth phase, strain k <dig> may alter its gene regulation to respond to phosphate limitation, which includes establishment of a general stress response. quantification of the rpos protein would further demonstrate the establishment of the rpos response, at mid-log phase of growth, in the pst k <dig> mutant.\n",
            "\n",
            "oxidative stress response\n",
            "during aerobic growth, the cytotoxic by-products of the molecular oxygen metabolism, collectively known as reactive oxygen species , must be eliminated to reduce oxidative stress. it is generally thought that the primary endogenous source of ros is the respiratory chain, namely the nadh dehydrogenase ii, which can leak electrons to oxygen, thereby producing superoxide anion radicals . superoxide is normally detoxified through the activities of superoxide dismutases and catalases that dismute o2-• into molecular oxygen, hydrogen peroxide  and eventually to h2o. however, if they are not rapidly detoxified, superoxide radicals can damage iron-sulfur clusters in enzymes, thereby releasing fe2+, which can react with h2o <dig> and produce reactive hydroxyl radicals . ho• can eventually initiate the oxidation of proteins, lipids and dna  <cit> . several genes that belong to the rpos regulon are involved in defense against oxidative stress  and/or oxyr  and soxr  . in strain k <dig>  some genes involved in the oxidative stress response were found to be differentially expressed. among those, the dps , sodc , grxb , kate  and sufabcdse  genes were up-regulated from  <dig> - to  <dig> -fold. the down-regulated genes  were represented by grxa , trxb , trxc , pqia  and ybjc . in addition, the srna oxys regulator was down-regulated  in strain k <dig>  oxys has been proposed to play a role in protecting cells against the damaging effects of elevated hydrogen peroxide concentrations by controlling translation of >  <dig> genes associated with the oxidative stress response  <cit> . other, regulators involved in oxidative stress, such as oxyr and soxrs, were not differentially expressed in strain k <dig> \n",
            "\n",
            "since expression of some genes whose products exhibit antioxidant activities is modulated, the viability to oxidative stress of k <dig> cells was tested using different reactive oxygen intermediate -generating agents. we used h2o <dig> and various superoxide generators, such as plumbagin, phenazine methosulfate  and phenazine ethosulfate  to evaluate the sensitivity of the k <dig> pst mutant and its wild-type parent strain χ <dig>  strain k <dig> was more sensitive to oxidative stress than the parent strain χ <dig>  and wild-type resistance levels to all roi compounds, except for h2o <dig>  were restored by complementation  . these results show that the pst mutation affects bacterial resistance to oxidative stress, and modulation of certain genes implicated in the oxidative stress response is not sufficient to confer resistance to this stress. based on the transcriptional data, strain k <dig> may already be subjected to increased oxidative stress during growth, and is likely less able to cope with additional stresses incurred from exogenous roi-generating compounds. furthermore, moreau et al. have shown that glucose metabolism in non-growing cells starved for pi generates oxidative stress  <cit> . since oxidative stress is increased during infection of the host, a decreased capacity to resist ros could explain, at least in part, virulence attenuation observed for pst mutants  <cit> .\n",
            "\n",
            "a data presented are the means ± the standard deviations of six independent experiments.\n",
            "\n",
            "b compounds used were  <dig> μl of plumbagin , h2o <dig> , phenazine ethosulfate  , or phenazine methosulfate  .\n",
            "\n",
            "c values indicated in bold text are significantly different  compared to the mean of the wild-type strain.\n",
            "\n",
            "acid stress response\n",
            "the acid fitness island  contains genes encoding proteins that are known to provide protection against acid stress in e. coli. this broad acid response system helps the cell avoid self-imposed acid stresses that occur as a result of fermentation and enables the cell to survive to low ph conditions  <cit> . the microarray data demonstrate that the expression of the afi genes increased significantly in strain k <dig>  this includes the glutamate decarboxylase gene gada , the multiple transcriptional regulator genes that control expression of the glutamate dependent acid resistance  system, gade , gadx  and gadw , the two chaperones hdeab , the multidrug resistance efflux transporter mdte  and the transcriptional regulator yhif . in accordance, the gadbc operon , which encodes the glutamate-decarboxylase and an antiporter, respectively, were also up-regulated. regulation of the afi and gdar systems is extremely complex and multifactorial. the expression of these systems is controlled by a large number of regulators, including, among others, the evgas two-component system, the specific transcriptional regulators  and the rpos sigma factor  <cit> . up-regulation of genes involved in acid resistance suggests that the pst mutant may be subjected to increased acid stress during exponential growth. the effect of the pho regulon on induction of the acid resistance response is likely to be indirect, since no putative pho box were found upstream of these regulators.\n",
            "\n",
            "previous reports have described the relationship between the pho regulon and genes involved in acid stress resistance  <cit> . transcriptional activation of the gada/bc/e genes was observed in e. coli w <dig> grown in low phosphate media  <cit> . in spite of the induction of acid stress genes, strain k <dig> was previously shown to be more sensitive to acid stress  <cit> . it is possible that the pst mutant is already subjected to a decreased internal ph from an accumulation of organic acids produced by the glycolytic pathway, which could induce acid response genes during exponential growth  <cit> . as such, strain k <dig> which may already incur endogenous acid stress, may therefore be less able to cope when subjected to decreased external ph and acid stress, as observed by lamarche et al.  <cit> . based on these data, it would be interesting to compare the ph of the culture medium and the intracellular ph of the wild-type and pst mutant during growth, to assess whether induction of these acid response genes may be specifically due to increased acidity during growth.\n",
            "\n",
            "transport and binding proteins\n",
            "in addition to pho regulon members involved in the transport of phosphate and phosphorylated compounds, genes encoding amino acid and oligopeptide transport systems were up-regulated in strain k <dig>  indeed, the glutamine and the arginine transporters, encoded respectively by the glnhpq operon and by the artiqm/artj genes were up-regulated in strain k <dig> . down-regulation of the argr  gene, which encodes the transcriptional repressor of the arginine transporter, occurred at the same time as with the up-regulation of the arginine transporter artiqm/artj. the oligopeptide transporter, encoded by the oppabcdf genes, was also induced . this transporter is specific for small peptides of up to five amino acids. besides their roles in proteins synthesis, amino acids can serve as nitrogen, carbon and atp sources and they can protect cells from different stresses, such as acid stress  <cit> . furthermore, genes encoding the glutamine, arginine and oligopeptide transporters possess putative pho box, which suggest that the uptake of oligopeptides and amino acids via these transporters may be directly regulated by the pho regulon.\n",
            "\n",
            "energy metabolism\n",
            "some metabolic genes, involved in the catabolism of different carbon sources, were up-regulated. the srlaebd-gutm-srlr-gutq operon, involved in glucitol transport and metabolism and belonging to the phosphoenolpyruvate : carbohydrate phosphotransferase system , was induced in strain k <dig>   <cit> . in addition, up-regulation of genes enzymatically linked to the production of fructose-6-phosphate   and glucose  , and metabolism of dihydroxyacetone   and l-ascorbate   were also induced. these catabolic pathways link to the glycolytic pathway, and intermediates could take part in the tricarboxylic acid cycle. interestingly, the small regulatory rna  sgrs, which inhibits synthesis of the ptsg glucose-specific pts system and alleviates the effects of glucose-phosphate stress, was also up-regulated , suggesting that the strain k <dig> has preferentially shifted to transport and metabolism of carbon sources other than glucose  <cit> .\n",
            "\n",
            "on the other hand, repression of genes involved in catabolism of glycerol-3-phosphate  as a carbon source, such as the g3p transporter encoded by the glptq genes  and the glycerol-3-dehydrogenase glpd  was observed. the glp system mediates utilization of g3p or its precursors, glycerol and glycerophosphodiesters. since uptake of g3p by the glptq transporter leads to pi counter flow and an overall loss of pi from the cell, it is not surprising that this system is repressed in strain k <dig>  <cit> . by contrast, genes encoding the other g3p-specific transporter ugp were highly up-regulated . accordingly, the ugp genes are members of the pho regulon. under phosphate starvation conditions, g3p is transported by the ugp transporter, which leads to the use of g3p as a phosphate source  <cit> . identification of putative pho box upstream the srl and ula operons, the tala, trea, and ulag genes denotes a possible direct connection between these catabolic pathways and the pho regulon. furthermore, the presence of putative pho boxes upstream of the glpd gene suggests that the pho regulon inhibits the use of g3p as carbon source. thus, as the pst mutant falsely senses phosphate limitation, it may alter carbon uptake and utilization pathways in an effort to conserve phosphate sources. in order to specifically determine differences in carbon source utilization and transport by the pst mutant compared to its wild-type parent, strains could be compared for their capacity to metabolize and grow on different carbon sources.\n",
            "\n",
            "protein fate and synthesis\n",
            "many genes from the protein fate and synthesis functional classes were down-regulated in strain k <dig>  these genes included those encoding components of the 30s and 50s ribosomal subunits and many chaperone genes involved in protein fate, such as those of the dnak system and iron-sulfur cluster assembly. the dnakj , grpe  and ibpab  genes comprise the dnak system. this system is involved in many cytoplasmic events, such as folding of nascent polypeptide chains, rescue of misfolded proteins and assembly and disassembly of protein complexes  <cit> . iron-sulfur cluster complexes play several important roles in cellular processes, such as iron storage, electron transfer and regulation of enzyme activity and gene expression  <cit> . however, genes involved in iron-sulfur cluster assembly were down-regulated in strain k <dig>  they were represented by hscab-fdx  and iscu  genes. repression of the dnak system and iron-sulfur cluster assembly genes suggests a decrease in cellular processes occurring in strain k <dig>  down-regulation of chaperones and ribosomal components suggest a reduction in cellular processes and a potential increase of misfolded peptides, which could be deleterious for cells and may contribute, at least in part, to virulence attenuation of the apec pst mutant strain. such changes are likely to be indirectly influenced by the pho regulon, since there were no putative pho box identified upstream of these genes.\n",
            "\n",
            "cell membrane components\n",
            "in many bacteria, the pho regulon was shown to be involved in regulation of some cell surface component modifications such as teichuronic acid, phosphate-free lipids, phospholipids and exopolysaccharides  <cit> . among the differentially expressed genes in strain k <dig>   <dig> encode membrane components. of these,  <dig> genes were up- and  <dig> were down-regulated . many of the up-regulated genes belong to the transport and binding protein functional class and include members of the pho regulon that were increased by more than 15-fold. in contrast, the provwx operon , involved in glycine betaine osmoprotector transport, demonstrated a decreased expression. this system is implicated in protection from osmotic shock  <cit> . beside their role in energy metabolism, the tala  and trea  genes were also induced by osmotic stress under aerobic conditions  <cit> . this suggests that the pst k <dig> strain was faced with osmotic stress, and down-regulation of the provwx operon could reduce protection from this stress.\n",
            "\n",
            "among the down-regulated genes,  <dig> are involved in lipopolysaccharide  biosynthesis including lipid a modification. indeed, the rffcdgh  and the rfaj  genes, respectively involved in the enterobacterial common antigen  biosynthesis pathway and lps-core biosynthesis, were down-regulated. however, we previously did not observe any changes in o-antigen and/or eca profiles by sds-page and western blot analyses  <cit> . the results of this study suggest that subtle changes could indeed occur within the lps structure in strain k <dig> \n",
            "\n",
            "the epta and yeiu genes, involved in lipid a modification, were also down-regulated . the epta gene is the salmonella pmrc ortholog and is involved in phosphoethanolamine  covalent modification of lipid a  <cit> . the basrs two-component system that regulates epta was also down-regulated  in this study  <cit> . the basrs genes are orthologs of the pmrab genes of salmonella enterica, and pmrab has been shown to be required for resistance to polymyxin b and other antimicrobial compounds  <cit> . in addition, we have determined that, in strain k <dig>  the down-regulation of yeiu , encoding an undecaprenyl pyrophosphate phosphatase involved in the biosynthetic origin of the lipid a 1-pyrophosphate, correlated with a decrease in the hexa-acylated 1-pyrophosphate form of lipid a compared to the apec strain χ <dig>  <cit> . valvano ma and raetz et al. have proposed that modification of lipid a might contribute to the fine-tuning of the lipopolysaccharide molecule, which is often the target of modifications that might provide adaptive advantages to changing environmental conditions  <cit> . lps modification could contribute to the phenotypes observed in the pst mutant, such as decreased resistance to serum, acid and cationic antimicrobial peptides  <cit> . furthermore, down-regulation of lps modification genes may be a strategy used by the bacteria to optimize the pi availability. however, such cell surface modifications could play a role in the reduced virulence observed in pst mutants  <cit> .\n",
            "\n",
            "reduced expression of fimbrial genes and production of type  <dig> fimbriae\n",
            "fimbriae are adhesive organelles of paramount importance for successful bacterial recognition and colonization of specific host tissues. genes involved in type  <dig> and f <dig> fimbriae biosynthesis were down-regulated in the k <dig> strain . indeed, genes involved in the biosynthesis of type  <dig>   as well as f <dig> fimbriae   were repressed in strain k <dig>  type  <dig> and f <dig> fimbriae are important virulence factors involved in colonization and biofilm formation  <cit> . in apec strains, type  <dig> fimbriae are preferentially expressed in air sacs, which are the primary infection site  <cit> . f <dig> fimbriae are homologous to type  <dig> fimbriae but are immunologically and functionally distinct  <cit> . however, the production of f <dig> fimbriae and the possible role of these fimbriae in apec infections have not yet been investigated.\n",
            "\n",
            "in order to investigate whether the decreased expression of fimbrial genes correlated with decreased fimbrial production, strain χ <dig>  the pst mutant k <dig> and the complemented mutant ck <dig> were grown under conditions used for microarray analyses and were examined by electron microscopy. indeed, very few or no fimbriae were observed at the surface of cells of the k <dig> mutant, compared to wild-type strain χ <dig> and the complemented strain ck <dig> . the type  <dig> fimbrial adhesin recognizes d-mannose receptors on cells, and mediates aggregation of yeast cells which are rich in mannose surface molecules  <cit> .\n",
            "\n",
            "when cultures of strains χ <dig>  the pst mutant k <dig>  and the complemented ck <dig> were grown under conditions corresponding to those used in the microarray analyses, i.e. od <dig>  <dig> , the k <dig> strain did not demonstrate any yeast aggregation even at the most concentrated bacterial cell suspension , whereas χ <dig> and the complemented mutant ck <dig> agglutinated yeast at titers at least 10-fold less  . when strains were cultured for  <dig> h without agitation, to enhance production of type  <dig> fimbriae, the k <dig> mutant agglutinated yeast cells, but 51-fold less than wild-type strain χ <dig> and the complemented mutant . hence, the pst mutant was shown to be considerably less able to produce type  <dig> fimbriae compared to its wild-type parent even under different growth conditions. to determine whether decreased yeast agglutination was specifically due to a reduction in production of the type  <dig> fimbrial adhesin, cell surface extracts were tested for production of the type  <dig> major fimbrial subunit, fima, by western blotting, and this confirmed an important reduction of fima production in k <dig> strain, compared to χ <dig> and ck <dig> strains .\n",
            "\n",
            "several studies have provided evidence that the expression of type  <dig> fimbriae is altered in response to environmental stress conditions such as high osmolarity, ph and temperature, and that expression is induced upon entry into stationary phase  <cit> . putative pho box were identified upstream of the ydet and ydeq genes, which are involved in f <dig> fimbriae biosynthesis, whereas no putative pho box were observed upstream of the genes involved in type  <dig> fimbriae biosynthesis. taken together, results indicate that in the pst mutant, the production of type  <dig> fimbriae is repressed, and this loss of adhesin expression could be indirectly controlled by phobr. it was previously shown that in apec strains, type  <dig> fimbriae are preferentially expressed in the air sacs and lungs, where the air sacs are the primary site of initial infection  <cit> . thus, in the k <dig> strain, in addition to other previously established phenotypes, down-regulation of adhesins such as type  <dig> fimbriae could affect the initial step of colonization and possibly contribute to decreased virulence.\n",
            "\n",
            "CONCLUSIONS\n",
            "in conclusion, by assessing the global transcriptional response of a pst mutant, we determined that the effects of this mutation resulted in up-regulation of members of the pho regulon, which are involved in phosphate uptake and metabolism. transcriptional analyses also demonstrated the induction of a general stress response in the pst mutant, including genes involved in adaptation to acid stress, oxidative stress, and the general stress response . other changes included down-regulation of genes associated with cell surface composition. phenotypic tests confirmed a reduced capacity of the pst mutant to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress. altogether, our data provide further support that the pho regulon is an important part of a complex network that encompasses not only phosphate homeostasis, but also adaptive responses to stress and altered regulation of a diversity of genes including virulence factors such as those encoding fimbrial adhesins.\n",
            "\n",
            "abbreviations\n",
            "pst system: phosphate specific transport system; edta: ethylene diamine tetraacetic acid; cdna: complementary dna; orf: open reading frame; bp: base pair; fe2+: ferrous iron; fe-s: iron-sulfur cluster; sds-page: sodium dodecyl sulfate polyacrylamide gel electrophoresis.\n",
            "\n",
            "authors' contributions\n",
            "sc designed and performed the transcript profiling experiments, carried out downstream data analysis and drafted the manuscript. mgl constructed the k <dig> and ck <dig> strain, participated in the study design and revised the manuscript. pg executed the bioinformatics analysis and revised the manuscript. js carried out the characterization of fimbriae experiments. jp performed the oxidative stress experiments. cmd supervised the characterization of fimbriae and oxidative stress experiments, performed data analysis and revised the manuscript. jh participated in the conception and supervised the design of the study, performed data analysis and revised the manuscript. all authors read and approved the final manuscript.\n",
            "\n",
            "supplementary material\n",
            "additional file 1\n",
            "list of primers used in the qrt-pcr experiments. complete lists of primers used in the qrt-pcr experiments, including primer sequences are shown.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 2\n",
            "up-regulated genes in the k <dig> strain. the complete list of genes up-regulated by at least  <dig> -fold with a p-value of ≤  <dig>  and an estimated false discovery rate  of  <dig> %, in the pst k <dig> strain. probe number, gene name, fold change in log <dig>  and p-value were provided.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 3\n",
            "down-regulated genes in the k <dig> strain. the complete list of genes down-regulated by at least  <dig> -fold, with a p-value of ≤  <dig>  and an estimated false discovery rate  of  <dig> %, in the pst k <dig> strain. probe number, gene name, fold change in log <dig>  and p-value were provided.\n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements\n",
            "we thank andré ponton, and mathieu miron, génome québec innovation centre , and guillaume bruant, université de montréal for their help in the analysis of the microarray data. we are grateful to turlough m. finan and rahat zaheer, mcmaster university, for sharing their data on e. coli genes with predicted phob boxes, and to miria elias from the nrc-biotechnology research institute for technical assistance with the agilent  <dig> bioanalyzer. we also thank sébastien houle, inrs-iaf, for his technical help with fimbrial extraction and electron microscopy experiments. we thank daniel dubreuil  for his critical reading of the manuscript.\n",
            "\n",
            "this work was supported in part by grants to j.h. and c.m.d. from the natural sciences and engineering research council of canada  and a canada research chair to c.m.d.\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the expression of  <dig> genes involved in the pho regulon, including those involved in amino acid acquisition , energy metabolism , acid resistance  and fimbriae biosynthesis, was down-regulated in strain k <dig>  however, expression of  <dig> genes involved in the pho regulon was up-regulated in the pst mutant.\n",
            "the pst mutant showed a decreased capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\n",
            "the pst mutant showed a reduced capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\n",
            "the pst mutant showed a decreased capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\n",
            "the pst mutant showed a decreased capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\n",
            "the pst mutant showed a decreased capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\n",
            "furthermore, the pst mutant showed a decreased capacity to produce type  <dig> fimbriae and a decreased capacity to resist oxidative stress.\n",
            "\n",
            "10\n",
            "[{'summary_text': 'tenm4m1/m <dig> mutants have a phenotype that is distinct from other mutants with gastrulation failure, but is most similar to that of β-catenin, which affects both the central nervous system and epidermal stripes at stages when the growth cones of intersegmental neurons navigate to their targets.\\ntenm4m1/m <dig> mutants have a phenotype that is distinct from other mutants with gastrulation failure, but is most similar to that of β-catenin, which affects both the central nervous system and epidermal stripes at stages when the growth cones of intersegmental neurons  navigate to their targets.\\ntenm4m1/m <dig> mutants have a phenotype that is distinct from other mutants with gastrulation failure, but is most similar to that of β-catenin, which affects both the central nervous system and epidermal stripes at stages when the growth cones of intersegmental neurons  navigate to their targets.\\ntenm4m1/m <dig> mutants have a phenotype that is distinct from other mutants with gastrulation failure, but is most similar to that of β-catenin, which affects both the central nervous'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "tightly controlled signaling pathways play critical roles in the early stages of embryogenesis and body plan formation. during gastrulation, morphogenetic movements in addition to cell proliferation and differentiation transform an embryo with a single germ layer into one with three layers, the ectoderm, mesoderm and endoderm. in the mouse, gastrulation initiates at e <dig> , as cells of the primitive ectoderm  delaminate at the posterior side of the embryo and ingress through the primitive streak in an epithelial-to-mesenchymal transition  <cit> . genetic approaches have identified many genes that play a role in gastrulation and/or pattern formation  <cit> . the evolutionarily conserved signal transduction cascade of wnt/β-catenin regulates patterning of the visceral endoderm, the induction of the primitive streak, and the formation of anterior neural ectoderm  <cit> . nodal and brachyury  are among the genes downstream of wnt signaling that are required for primitive streak formation  <cit> .\n",
            "\n",
            "tenm <dig>  teneurin transmembrane protein  <dig>  is a mouse homolog of the drosophila gene tenascin major , also called odd oz   <cit> , which was originally associated with segmentation defects. ten-m is a member of the teneurin protein family and encodes a transmembrane protein with tenascin-like egf  repeats. ten-m is found in both secreted and membrane-bound forms at several compartment boundaries, suggesting a role in cell-cell communication. mutant ten-m drosophila embryos have abnormalities in neural ectoderm and mesoderm-derived tissues that result in death at the larval stage  <cit> . recently, new alleles of drosophila ten-m were identified and characterized establishing that mutations in this gene do not cause segmentation defects; instead, ten-m functions in motor neuron routing  <cit> . ten-m is expressed in the central nervous system and epidermal stripes at stages when the growth cones of intersegmental neurons  navigate to their targets. both mutation and over-expression of ten-m in epidermal cells leads to isn misrouting  <cit> . a related protein, tenascin a  has a trans-synaptic signaling role with ten-m: ten-a is presynaptic whereas ten-m is predominantly postsynaptic in neuromuscular synapse organization and target selection  <cit> .\n",
            "\n",
            "the mouse has four teneurin transmembrane protein family members  that lack signal peptides at the n-terminus, but contain a short hydrophobic domain characteristic of transmembrane proteins followed by a region with eight egf-like repeats, and a large c-terminal domain  <cit> . the vertebrate homolog of drosophilaten-a is called tenm <dig> and the homolog of drosophila ten-m is tenm <dig>  all four mammalian tenm genes are highly expressed in the brain and each gene produces many alternatively spliced transcripts, suggesting a variety of protein functions in different tissues  <cit> . novel mutant alleles having defects in early mouse embryonic development can be identified using ethylnitrosourea  mutagenesis  <cit> . a series of enu-induced alleles at mouse 17rn <dig> <cit>  contain mutations in the tenm <dig> gene  <cit> , which exhibit a wide array of phenotypes, ranging from embryonic death at gastrulation to viable with skeletal defects  <cit> . in two loss-of-function alleles , embryos fail to gastrulate. in less severe alleles , gastrulation occurs, but body axis formation, somitogenesis, vasculogenesis, cardiogenesis, and fusion of the allantois with the chorion are disrupted leading to death at early- to mid-gestation stages. consistent with these mutant phenotypes, tenm <dig> is ubiquitously expressed in the epiblast and extraembryonic regions as early as e <dig> . by e <dig> , tenm <dig> is highly expressed in the mesoderm of the developing embryo and extraembryonic tissues. later, tenm <dig> is expressed mainly in the neuroectoderm, but expression is maintained in the tail bud, somites and limbs  <cit> .\n",
            "\n",
            "to begin to address the biological function of tenm <dig>  the loss of function allele tenm4m <dig>  and a hypomorphic allele tenm4m <dig> were examined. tenm4m1/m <dig> mutant embryos failed to initiate gastrulation, failed to form a primitive streak, and failed to develop mesoderm. in addition, tenm4m1/m <dig> mutant embryos were incapable of forming any differentiated tissue. an analysis of the hypomorphic allele, tenm4m <dig>  along with the null allele, determined that mutant cells did not properly express e- or n-cadherin, suggesting that the epithelial-to-mesenchymal transition  did not occur. moreover, the mutants failed to up-regulate a topgal reporter gene, suggesting that wnt signaling failed to occur. finally, a gsk3β inhibitor rescued the ability of the embryos to form extraembryonic mesoderm and partially rescued their ability to form embryonic mesoderm.\n",
            "\n",
            "RESULTS\n",
            "tenm4m1/m <dig> mutant embryos arrest at the gastrulation stage\n",
            "at e <dig> , wildtype and tenm4m1/m <dig> embryos were similar in appearance . at e <dig> , mutant embryos were approximately half the size of littermates , and at e <dig> , mutant embryos were arrested in development at the e <dig>  stage . prior studies showed that most embryos were dead or dying by e <dig>  and that no mutant embryos survived past e <dig>   <cit> . histology revealed phenotypic differences between wildtype and mutant embryos . at e <dig>  wild-type embryos had a well-organized ectoderm, visceral endoderm, and extraembryonic mesoderm with proamniotic and extraembryonic cavities . in contrast, mutant embryos had no sign of mesoderm . at e <dig> , wildtype embryos developed embryonic mesoderm . by e <dig> , wildtype embryos developed three primitive embryonic cavities, including the amniotic, exocoelomic and ectoplacental cavities, head folds, and embryonic and extraembryonic mesodermal tissues, including the allantois . however, no mesoderm formed and the embryonic region did not expand in tenm4m1/m <dig> mutant embryos by e <dig>  or  <dig> , although some embryos appeared to form a morphologically abnormal extraembryonic cavity, perhaps caused by the expansion of the extraembryonic ectoderm and visceral endoderm .\n",
            "\n",
            "tenm4m1/m <dig> mutants do not produce mesoderm\n",
            "brachyury is expressed prior to the onset of gastrulation at e <dig>  in the extraembryonic ectoderm adjacent to the epiblast, and shifts to the developing primitive streak, where it is a marker of mesodermal-derived notochord  <cit> . no brachyury expression was found in the embryo of tenm4m1/m <dig> mutants, although a weak signal was observed in the extraembryonic tissue in half of the mutants . the cause of the inconsistent extraembryonic expression is not clear. this is an enu-induced point mutation, and often such alleles are somewhat leaky due to read-through of the point mutation. alternatively, this observation is consistent with expression in extraembryonic ectoderm, indicating developmental delay. importantly, none of the mutants had brachyury expression in the area of the potential primitive streak. additional primitive streak markers were examined. in wildtype embryos, foxa <dig>  expression is restricted to the anterior end of the primitive streak at e <dig>  .\n",
            "\n",
            "to determine how the tenm4m <dig> mutation affects the development of the anterior patterning of the embryo, the expression pattern of otx <dig> was examined, which is expressed uniformly in the epiblast and the anterior visceral endoderm  prior to gastrulation. the expression of otx <dig> becomes restricted to the anterior epiblast as mesoderm migrates from the primitive streak suppressing its expression in the posterior epiblast . in tenm4m1/m <dig> embryos, otx <dig> expression does not become fully restricted to the anterior epiblast, consistent with the lack of mesoderm influencing its restricted expression . bmp <dig> is expressed in the extraembryonic ectoderm adjacent to the proximal epiblast prior to gastrulation and later in the posterior primitive streak and extraembryonic mesoderm .\n",
            "\n",
            "proliferation and apoptosis of epiblast cells in tenm4m1/m <dig> is similar to wild type\n",
            "a threshold number of epiblast cells must be attained and maintained for gastrulation to initiate and progress  <cit> . mouse embryos that lack a normal number of cells due to abnormal cell proliferation or to cell losses will delay gastrulation until the embryo attains the appropriate number of cells. mutant embryos that fail to sustain the proliferation of epiblast cells do not initiate gastrulation or will arrest after the formation of a rudimentary primitive streak. to examine the effect of the tenm4m <dig> mutation on cellular proliferation in vivo, the incorporation of 5′-bromo-2′-deoxyuridine  into dna was analyzed. at e <dig>  , 48% of the nuclei of wildtype or heterozygous epiblast cells were labeled with brdu after a 20-minute exposure. in tenm4m1/m <dig> mutant embryos, 47% of the nuclei were labeled, suggesting that the proliferation of epiblast cells is normal at this time point. these data suggest that the gastrulation defect is not caused by insufficient cell proliferation in the epiblast. however, the total number of mutant epiblast cells was lower at e <dig>  , and a student’s t-test indicates that total cell numbers are significantly different . this suggests that the growth of the embryo starts to become impaired prior to e <dig> . increased apoptosis could also cause a reduced number of cells, so the tunel assay was used to examine cell death. normally, cell death occurs primarily in extraembryonic tissues, especially in trophoblast , but is rare in embryonic cells. very few apoptotic cells were observed in mutant embryos at e <dig>   or at e <dig>  .\n",
            "\n",
            "i) brdu incorporation at e <dig> . ii) tunel assay at e <dig> . numbers in parentheses are standard deviation .\n",
            "\n",
            "the epiblast contains a highly regulated stem cell population that adjusts to various perturbations including major alterations in cell number and cell position. because the cells remain in an undifferentiated state until late gastrulation  <cit> , one possibility for the low cell count in tenm4m1/m <dig> at the late gastrulation stage is a precocious differentiation of epiblast stem cells. therefore, the expression of pou5f <dig> at e <dig>  was examined. pou5f <dig> is distinguished by exclusive expression in blastomeres, pluripotent cells, and the germ cell lineage  <cit> . at e <dig> , pou5f <dig> expression should be restricted to the epiblast . in mutant embryos, the expression of pou5f <dig> is similar to wildtype, indicating that undifferentiated pluripotent epiblast cells are present .\n",
            "\n",
            "to examine differentiation potential, wildtype and mutant embryonic tissues at e <dig>  were transplanted into the testes of adult male mice and examined for the development of teratomas. after  <dig> weeks, the testes were fixed, sectioned and processed for histological analysis . teratomas derived from heterozygotes differentiated into many kinds of tissues, which are produced from all three germ layers . however, the teratomas derived from tenm4m1/m <dig> mutant embryos produced no differentiated tissues . the size of teratomas derived from mutant embryos never exceeded  <dig> mm, whereas teratomas from wildtype embryos grew to  <dig> - <dig> cm in diameter . furthermore, approximately half of the mutant-derived teratoma cells had degenerated when examined at  <dig> weeks. only four homozygous mutant-derived teratomas survived to be genotyped out of  <dig> teratomas examined , all of which had inflammatory reactions, indicating the decay of the cells. moreover, six teratomas could not be genotyped because they were severely decomposed. mutant-derived teratomas had signs of inflammation when teratomas were assessed at 1 week after transplantation, although again, no cell differentiation was observed . together, these results suggest that the failure to differentiate is cell autonomous. to further test this question, embryos were cultured to attempt to generate embryonic stem  cells for studies of chimeras with mutant cells. although multiple es cell lines were derived from wildtype  or heterozygous  embryos, none were derived from mutants , precluding experiments on such cells. blastocysts from matings of ten-m4m <dig> heterozygotes were collected at e <dig>  and cultured in vitro to observe any defects in the inner cell mass  and/or trophectoderm outgrowth. all blastocysts attached normally to gelatin-coated dishes and completely hatched from the zona pellucida after 2– <dig> days in culture. after hatching, the growth of the icm and the extent of trophoblast outgrowth was very abnormal in ten-m4m1/m <dig> mutants . mutant cells could be easily distinguished from those from wild type or heterozygotes because they failed to adhere and could not survive more than  <dig> days in culture.\n",
            "\n",
            "teratomas were assessed by histology after measuring the diameter of the teratoma. the teratoma sizes were categorized as large , middle  and small  and the tissue types in teratomas derived from normal and mutant embryos were categorized as ectoderm, endoderm and mesoderm. *not genotyped due to degrading of teratoma tissue.\n",
            "\n",
            "expression of e-cadherin, n-cadherin and snai1\n",
            "an epithelial-to-mesenchymal transition  is an essential step in gastrulation, which requires wnt signaling to proceed. down-regulation of e-cadherin is an early step of the emt. embryos that fail to gastrulate do not undergo the emt; however, the tenm <dig> allelic series includes a hypomorphic allele, tenm4m4/m <dig>  which gastrulates to form mesodermal tissues, including yolk sac, heart and blood vessels, yet arrests in development by e <dig>   <cit> . the m <dig> allele allows the emt to be assessed in a mutation that does not arrest before gastrulation occurs. the expression of e-cadherin in the loss of function allele tenm4m1/m <dig> and the hypomorphic allele tenm4m4/m <dig> was examined. e-cadherin was absent in the primitive streak and mesoderm of wildtype embryos, , whereas e-cadherin expression was not down regulated in the epiblast of both tenm4m1/m <dig> and tenm4m4/m <dig> . when epiblast cells delaminate at the primitive streak to form mesoderm, they switch expression from e- to n- cadherin. n-cadherin was expressed at high levels in the yolk sac and primitive streak in wildtype embryos ; however, no n-cadherin expression was observed in either tenm4m1/m <dig> or tenm4m4/m <dig> . snai <dig>  suppresses e-cadherin to initiate migration of trophoblast giant cells, which also requires the emt. no snai <dig> expression was observed in cultured ectoplacental cone mutant explants . all wildtype explants expressed snai <dig> , whereas only one tenm4m1/m <dig> explant had low snai <dig> expression, and five had none ,  <dig> %.\n",
            "\n",
            "the topgal assay suggests that wnt signaling does not occur\n",
            "drosophila ten-m affects wingless expression  <cit> . in mouse, canonical signaling by wnt <dig> is required for mesoderm induction in the early embryo  <cit> . a topgal transgenic expression reporter mouse strain, which expresses β-galactosidase under the control of three copies of the wnt-specific lef/tcf binding sites was used to assess wnt signaling in the mutants  <cit> . in wild-type e <dig>  embryos, topgal activity was detected in the primitive streak . in contrast, topgal activity was not present in tenm4m1/m <dig> e <dig>  embryos .\n",
            "\n",
            "normally, glycogen synthase kinase  <dig>  is inhibited by wnt signaling to promote stabilization of β-catenin and transcription of target genes  <cit> . to attempt to rescue the defective mesoderm induction in tenm4m1/m <dig> mutant embryos, wnt signaling was ectopically induced with a gsk <dig> specific inhibitor, 6-bromoindirubin-3′-oxime . its analog, 1-methyl-6-bromoindirubin-3′-oxime  is a useful negative control  <cit> . when mebio was added to the embryo culture medium, tenm4m1/m <dig> mutant embryos did not produce any mesodermal cells . however, when bio was added at  <dig> μm and  <dig> μm, extraembryonic tissue grew . even so, the embryonic region did not expand, although some cells migrated away from the epiblast . using a higher bio concentration , tenm4m1/m <dig> mutant embryos produced allantois and chorion, which fused, suggesting that extraembryonic mesoderm was produced and differentiated . moreover, topgal signaling was induced to various degrees in both embryonic and extraembryonic tissues , suggesting that wnt signaling was restored in some cells.\n",
            "\n",
            "discussion\n",
            "here we show using mouse mutants that the earliest functions of tenm <dig> are prior to gastrulation such that wnt signaling does not occur. a loss of function allele failed to gastrulate and produced no mesoderm. in vivo and in vitro experiments showed that loss of function mutant embryos did not have the potential to form differentiated tissues, a defect that was cell autonomous. further, e-cadherin and n-cadherin expression was abnormal in both loss of function and hypomorphic alleles, supporting the idea that tenm <dig> mutant cells fail to undergo the epithelial-to-mesenchymal transition, surprisingly even when a primitive streak forms and gastrulation occurs. the formation of embryonic cavities, along with weak expression of brachyury in extraembryonic regions and rescue of extraembryonic mesoderm by gsk inhibitors, suggests that extraembryonic mesoderm may remain competent in the mutants; however, mesoderm in the embryo proper may lack the potential to differentiate.\n",
            "\n",
            "the phenotype of tenm4m1/m <dig> mutant embryos is distinct from other mice with gastrulation failure. mutations in nodal, smad <dig>  smad <dig> or actr1b, which block signaling of members of the transforming growth factor beta  family, affect embryonic differentiation at the egg cylinder stage before gastrulation  <cit> . smad2-dependent nodal/activin/tgf-β signaling is essential for the maintenance of pluripotency in the epiblast and in human embryonic stem cells  <cit> . tenm4m <dig> deficient embryos maintain expression of pou5f <dig> in the epiblast, suggesting that the epiblast has pluripotent potential. foxh <dig> and cripto mediate the nodal-signaling pathway  <cit> . in contrast to tenm4m <dig>  foxh1- and cripto-deficient embryos produce a primitive streak. bone morphogenic protein  signaling is also required for gastrulation, and mesoderm induction fails in type-i bmp receptor  mutants  <cit> . in contrast to tenm <dig> mutants, teratomas derived from bmpr1-/- embryos produce mesoderm-derived tissues  <cit> . bmp signaling is required for visceral endoderm differentiation and the formation of cavities in the early mouse embryo  <cit> . tenm4m1/m <dig> embryos developed normal visceral endoderm and embryonic cavities, as they maintained bmp <dig> expression in the extraembryonic ectoderm. taken together, these data suggest that the first embryonic function of tenm <dig> may not be to target tgf-β or bmp signaling.\n",
            "\n",
            "canonical wnt signaling is essential for mesoderm formation, embryonic patterning, and epithelial-to-mesenchymal interactions. wnt <dig> plays a role in inducing mesoderm and forming the primitive streak  <cit> . although wnt <dig> mutants fail to induce mesoderm, the visceral endoderm and epiblast layers continue to grow and expand  <cit> . in contrast, tenm4m1/m <dig> embryos failed to expand the visceral endoderm and epiblast layers. mesd, which encodes the chaperone for the wnt co-receptors lrp5/ <dig>  as well as a double knockout of lrp5/ <dig>  shows phenotypes similar to the tenm4m <dig> mutant  <cit> . in these mutants, defects in extraembryonic tissues lead to failure to organize the proximal epiblast, similar to tenm4; however, in contrast, the visceral endoderm and epiblast layers continue to grow, similar to wnt <dig> mutants. β-catenin regulates cripto- and wnt3-dependent gene expression programs in mouse anterior posterior axis and mesoderm formation, controlling both the wnt and nodal pathways  <cit> . no mesoderm or head structures formed in β-catenin-deficient embryos, and markers of posterior mesoderm differentiation such as brachyury, as well as markers of a-p axis formation such as hex and hesx <dig>  were not expressed  <cit> . similar to β-catenin- and wnt3-deficient embryos, tenm4m1/m <dig> mutants failed to form a primitive streak, even though the epiblast maintained high pou5f <dig> expression, unlike mutations in smad <dig>  smad <dig> and nodal. the tenm4m1/m <dig> mutant phenotype is more similar to canonical wnt mutants than those of other pathways, yet seems to lack competency for the embryonic epiblast to differentiate into mesoderm  <cit> .\n",
            "\n",
            "fgfr <dig> mutants that do not activate snai <dig> show defective mesoderm migration and differentiation  <cit> . the suppressor of e-cadherin, snai <dig>  was not up-regulated in cultured mutant ectoplacental cone explants of the hypomorphic mutant tenm4m <dig>  these data, combined with knowledge of tenm family egf structural motifs, membrane orientation, and known secreted and membrane-bound forms, suggest that tenm <dig> may be an important molecule for proper adhesion of migrating extraembryonic cells. embryo cultures indicated that a gsk3β inhibitor induced the outgrowth of extraembryonic tissues, and partially rescued defective mesoderm induction, particularly in extraembryonic tissues. the activity of gsk inhibitors is not restricted to wnt signaling, and in fact, can influence cell adhesion molecules  <cit> . furthermore, the inhibitors did not fully rescue the mutant phenotypes, suggesting that other pathways are likely to be involved.\n",
            "\n",
            "the wnt pathway has evolved to interact with many receptors in a variety of signaling pathways that are independent of canonical signaling, see  <cit>  for review. for example, non-canonical wnt signaling is required for planar cell polarity and convergent extension  <cit> . in addition to its essential roles in gastrulation, stem cell homeostasis and axis patterning, the wnt pathway plays a key role in neuronal development and maintenance. in particular, non-canonical wnt signaling is crucial for path finding during axonal navigation as well as for synapse formation  <cit> . drosophila ten-m binds with filamins via its filamin-binding domain to reorganize the cytoskeleton during axon guidance  <cit> , a function of non-canonical wnt signaling. ten-m is expressed in muscle, but interacts with ten-a at the neuromuscular synapse, evidencing its role in cell-cell communication during synapse organization and function  <cit> . recently, a large-scale genome-wide association study  identified an intronic variant in tenm <dig>  as being associated with bipolar disorder, a severe mood disorder that affects over 1% of the population  <cit> . wnt signaling is implicated in many neuropsychiatric disorders, including schizophrenia, bipolar disorder and autism  <cit> . therefore, it will be intriguing to determine if tenm <dig> has a direct role in wnt signaling; however, evidence for this will require the analysis of other tenm <dig> alleles, which have abnormal body axis patterning or abnormal neural development  <cit> .\n",
            "\n",
            "CONCLUSIONS\n",
            "our data show that tenm <dig> is required for mesoderm induction during gastrulation, and that gastrulation arrests at a stage prior to that in which wnt signaling occurs. tenm4m1/m <dig> mutants have a phenotype that is distinct from other mutants with gastrulation failure, but is most similar to that of β-catenin, which affects both the wnt and nodal pathways. a gsk3β inhibitor rescued the ability of mutant embryos to form extraembryonic mesoderm and partially rescued the mutant embryos’ ability to form embryonic mesoderm. gsk <dig> can also affect adhesion complexes independent of wnt signaling. given that tenm <dig> is a cell adhesion molecule that acts at neuronal synapses in flies, it is possible that cell-cell interactions are perturbed in the mutants, preventing the communication required for gastrulation to proceed. further work will be required to fully understand the signaling pathways involved in this intriguing and important molecule.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "tenm4m1/m <dig> mutants have a phenotype that is distinct from other mutants with gastrulation failure, but is most similar to that of β-catenin, which affects both the central nervous system and epidermal stripes at stages when the growth cones of intersegmental neurons navigate to their targets.\n",
            "tenm4m1/m <dig> mutants have a phenotype that is distinct from other mutants with gastrulation failure, but is most similar to that of β-catenin, which affects both the central nervous system and epidermal stripes at stages when the growth cones of intersegmental neurons  navigate to their targets.\n",
            "tenm4m1/m <dig> mutants have a phenotype that is distinct from other mutants with gastrulation failure, but is most similar to that of β-catenin, which affects both the central nervous system and epidermal stripes at stages when the growth cones of intersegmental neurons  navigate to their targets.\n",
            "tenm4m1/m <dig> mutants have a phenotype that is distinct from other mutants with gastrulation failure, but is most similar to that of β-catenin, which affects both the central nervous\n",
            "11\n",
            "[{'summary_text': 'our method correctly identified  <dig> tfs with overlap p-value being equal to or less than  <dig> , while the overlap p-value was  <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "with the advance of high-throughput technologies such as dna microarray, chromatin immunoprecipitation dna chip   <cit> , yeast two-hybrid assays  <cit>  and co-immunoprecipitation screens  <cit> , various kinds of whole genome scale data are available, shedding light on the regulatory mechanisms in the biological system. several new computational methods have been developed to combine various kinds of data to construct regulatory networks  <cit> . in addition, several researchers have strived to infer regulatory pathways connecting the known causal perturbation to the affected genes using physical interaction networks  <cit> . these inferred pathways could explain consequences of perturbations such as gene knockout effects. if the causal factor is unknown, however, inference of the causal factor from the consequences ) is rather challenging.\n",
            "\n",
            "to address this, tu et al.  <cit>  and sutras et al.  <cit>  integrated tf-dna interactions and protein-protein interactions to map which gene among expression quantitative trait loci  was the causal factor responsible for the observed changes in the downstream gene expression. however, the candidate causal factor was restricted to genes located within eqtls, and their methods could not be widely applied if such information was not available. in another work, pollard et al.  <cit>  tried to discover underlying molecular causes of type  <dig> diabetes mellitus consistent with the expression changes based on  <dig>  molecular cause-and-effect relationships assembled from literature. yet the power of such kind of approach relies greatly on the size and quality of cause-and-effect relationships, which are often hard to collect.\n",
            "\n",
            "increasing amount of molecular interactions, including tf-dna interactions, protein-protein interactions  and protein post-translational modifications , mapped from high-throughput technologies may provide significant information about cause-and-effect relationships. previous approaches of associating tfs with expression changes were often based on direct binding targets of tfs  <cit> , which were derived either by upstream sequence matches to a consensus binding motif  <cit> , or by tf-dna interactions from chip-chip experiments  <cit> . several studies, however, have pointed out the low overlap between direct targets bound by a tf and transcriptionally affected genes caused by perturbation to the same tf  <cit> . backup in regulatory pathways is one possible reason for the low overlap, which leads to no expression changes observed for most direct targets of a tf under this tf knockout  <cit> . the ability of tfs to affect gene expression through ways other than direct transcription may be another reason. given the complexity of regulatory networks, if only tf-dna interactions were used and simple flat regulation pathway was modeled, the power of those methods for inference of associated tfs would be limited. integrating tf-dna interactions with other directed interactions and considering hierarchical and multi-layer regulatory pathways through which tfs affect expressions of their downstream genes may be helpful. protein-protein interactions provide limited information because ppis normally imply no regulation direction. protein post-translational modifications have rarely been considered for gene expression based causal inference, since ptm usually can not be detected at expression level.\n",
            "\n",
            "here we present a tf-centered downstream gene set enrichment analysis to identify potential causal regulators responsible for gene expression changes. integrating tf-dna interactions and ptm information, we constructed multi-layer regulation models progressively to derive possible downstream gene sets of a specific tf. ptm are incorporated because their regulation roles to proteins activation status are certain. tfs activity change would cause differential expressions of downstream genes, even though tfs themselves do not change at expression level. compared with the method using only direct tf-dna interactions, our method correctly identified five more actual perturbed tfs in knockout experiments  <cit>  and six more tfs in overexpression experiments  <cit> . the results suggest that in addition to direct transcription, indirect transcription and ptm are also responsible for the downstream effects of tfs perturbation, especially for tfs overexpression. potential regulatory pathways downstream of three perturbed regulators — snf <dig>  aft <dig> and sut <dig> — were given to demonstrate the power of incorporating indirect transcription and/or ptm information.\n",
            "\n",
            "furthermore, our method successfully identified several known and novel potential regulators involved in the transition from fermentative to glycerol-based respiratory growth  <cit>  and in the pheromone response  <cit> , some of which were validated by their changes in expression and/or phosphorylation status. additionally, downstream regulation pathways of sut <dig> in the transition process and aft <dig> in the pheromone response were also supported by the mrna and/or phosphorylation changes of their mediating tfs and/or “modulator” proteins.\n",
            "\n",
            "our results suggest that pathways through which tfs regulate the expression of downstream genes are condition dependent. with our methodology, many novel potential causal regulators and downstream regulation models may be proposed and evaluated. our method provides a valuable framework for analyzing gene expression data to identify causal regulators in the context of tf-dna interactions and ptm information, which may benefit disease mechanism studies and identification of potential interfering targets.\n",
            "\n",
            "methods\n",
            "the schema of our method is shown in figure  <dig>  for a perturbed condition, the method ranked the potential causal regulators based on the significance of the overlap between downstream gene sets of each tf and the changed expression profiles, namely, the list of degs. compared with most previous approaches using direct tf-dna interactions alone  <cit> , the downstream gene sets of each tf here were derived by integrating tf-dna interactions and ptm information.\n",
            "\n",
            "recent studies have shown that gene perturbation effects could be mediated by ppi networks as well  <cit> . it has been suggested that when the path length from the initial tf to the last tf is greater than  <dig>  the significance of the overlap between the observed degs and the expected targets of the actual perturbed tf is decreased  <cit> . thus, we constructed regulation models with the path length from the initial tf to the last tf being equal to or less than  <dig>  for examples, paths like tf-tf-tf, or tf-kinase-tf, are length  <dig> paths. models were built progressively by gradually adding tf-dna regulation layers and ptm information, as illustrated in figure  <dig>  six models were constructed in total and classified into one-layer regulation models, two-layer regulation models and three-layer regulation models, which were defined according to the number of tf-dna regulation layers. one-layer regulation models included simple direct model  and ptm-mediated direct model . simple direct model  only considered direct binding targets of tf a, while ptm-mediated direct model  extended model i to further include direct targets of other tfs  post-translationally modified by tf a. model ii could be further extended by another ptm layer, i.e., including direct targets of other tfs post-translationally modified by tf b. it had been found that downstream genes in this model were the same with those in model ii for all tfs and only upstream kinases, such as elm <dig>  reg <dig>  sak <dig>  glc <dig> and tos <dig>  were involved in the model. therefore this model was not included in our present work since the method was mainly aimed to identify tfs but not kinases responsible for expression changes. in the future, this model will be included when the work is extended to discover causal signal molecules and kinases. two-layer regulation models consisted of two-layer cascade regulation model , ptm-mediated two-layer cascade regulation model , and hybrid two-layer cascade regulation model . two-layer cascade regulation model  was built by extending model ii to further include second-layer targets of tf a mediated by tf c and tf d, while ptm-mediated two-layer cascade regulation model  was constructed by extending model ii to further include second-layer targets of tf a mediated by tf e which is modified by x . hybrid two-layer cascade regulation model  combined model iii and model i v, i.e., considering second-layer targets of tf a mediated by both protein expression changes  and protein modification changes . three-layer regulation model  was obtained by extending model v to further include third-layer targets of tf a mediated by tf f. model vi could not be further extended since the length of these paths tf a-tf c-tf f, tf a-‘modulator’ x-tf e, and tf a-tf b-tf d was all equal to  <dig>  thus there was only one three-layer regulation model.\n",
            "\n",
            "possible downstream genes of a tf were defined as the expected targets of this tf. six possible expected targets for tf a were denoted as . the observed set of degs was denoted as d={degs}. the overlap significance between six expected targets and the observed set of degs was calculated using the hypergeometric distributions .   \n",
            "\n",
            "six p-values were obtained for tf a to evaluate the significance of the overlap between six possible expected targets of tf a and the observed degs. to determine the most likely model through which tf a was responsible for expression changes, we selected the model in which the most significant overlap occurred . in this way, the overlap significance between the expected targets of tf a and the observed degs under this tf perturbation would be improved. however, the overlap significance between expected targets of other non-perturbed tfs and the degs might increase simultaneously, which would introduce much noise and descend the rank of the actual perturbed tf in the candidate list . we assumed that if a tf functioned through a model rather than model i, the overlap of its expected targets with the degs would be more significant in this model than that in model i, while the overlap of most of other tfs in this model would not be more significant than that in model i. on the contrary, if many tfs got more significant overlap between their expected targets and the observed degs in a model than that in model i, these tfs would rank before the actual perturbed tf and thus the actual perturbed tf would be missed. that is, the model in which many tfs obtained more significant overlap than that in model i was unacceptable. we compared the distribution of overlap p-values obtained by model i and other models using the wilcox rank sum test. those models whose distribution of overlap p-values was significant different from model i  were unacceptable. the minimum p-value obtained at all acceptable models for each tf was used to rank and evaluate the tf as the potential causal regulators . by selecting acceptable models based on the background distribution of overlap significance, we could correctly identify the actual perturbed tf and discover the most possible regulation pathway downstream of this tf perturbation .      \n",
            "\n",
            "for the observed list of degs, we inferred the most likely causal regulators and the underlying pathways. p-values being equal to or less than  <dig>  were considered to be significant and the corresponding tfs were reported as valid findings. if the actual perturbed tf was a valid finding and ranked at the top  <dig> of candidates, this tf was said to be correctly identified. we also tested our method using different numbers of top rank candidates to decide the correct identifications . we found that the criterion about the number of top ranks had little impact on the performance of the method , suggesting that the standard we used for the valid finding  was quite stringent and most of valid findings ranked at the top. for previous methods that downstream genes of tfs were obtained through simple direct model  using tf-dna interactions, only those tfs that affected gene expression through direct transcription could be correctly identified. in contrast, constructing hierarchical and multi-layer regulation models by the integration of tf-dna interactions and ptm information, our method was expected to find not only tfs that affected gene expression through direct transcription, but also those that affected gene expression through indirect transcription and post-translational modifications.\n",
            "\n",
            "we also built models in the following ways and compared the results from them with those from the six models: 1) we constructed models progressively without any constraint on the path length, i.e. with the path length from the initial tf to the last tf being greater than  <dig>  we found that many tfs eventually got almost the same set of downstream genes when the path length was long enough and the method failed to infer most of the actual perturbed tfs. 2) we did not construct all six models fixedly but tried to infer which model explains the expression data best. for example, unlike direct targets of tf a, b, c and d comprising the downstream genes of tf a in the two-layer cascade regulation model , which combinations of tf b, c and d were inferred to comprise the regulation models of tf a in the case that regulation effects of tf a could be mediated by tf b, tf c and tf d. if direct targets of tf a and tf b obtained the most significant overlap with observed degs, this p-value was considered as the overlap significance obtained by tf a as causal regulators and used to rank the candidates. it was found that much more noise was introduced in this way and many actual perturbed tfs ranked lower in the candidate list. it is very likely that six models may not cover all the possible regulation topologies. however, we think some topologies happen at very low frequencies in real biological systems. if all the possible models were considered, it would increase the model space and lower the performance. therefore, we limited our work to the six models with the belief that most frequent regulatory scenarios in the biological systems were well represented.\n",
            "\n",
            "data collection\n",
            "post-translational modifications  information was obtained from s. cerevisiae phosphorylation network  <cit>  and ptm-switchboard  <cit> . the former contains the majority of the well-characterized kinase- and phophatase-substrate relationships in s. cerevisiae  manually curated from literature. the latter constitutes the relationship between the tf and its “modulator” protein, which alters the tf’s activity through post-translational modifications. tf-dna interactions were obtained from  <cit> , which presented a framework for integrating seven distinct sources of evidences to score all possible tf-target interactions. we extracted all tf-target interactions with lls > <dig>  yielding a total of  <dig>  high confidence interactions. by integrating ptm and tf-dna interactions,  <dig> tfs with the number of downstream genes being no less than  <dig> in any one of the six models were selected as candidates .\n",
            "\n",
            "two tf perturbation data sets  <cit>  were used to verify the method. hu et al.  <cit>  profiled the transcriptional response in s. cerevisiae strains with individual deletions of  <dig> tfs. among  <dig> tfs,  <dig> are in aforementioned list of  <dig> candidates. the transcription responses of these  <dig> tfs knockout strains were selected to evaluate the power of the method. chua et al.  <cit>  provided the microarray expression data resulting from overexpression and/or deletion of  <dig> tfs, among which, those experiments of overexpression and/or deletion of tfs common with  <dig> candidates were selected. as a result, overexpression data of  <dig> tfs and deletion of  <dig> tfs were chosen. the standard p<= <dig>  was used to select degs for hu et al. data  <cit>  and |z|>= <dig> was used for chua et al. data  <cit>  here. we also tested our method using different standards. we found that similar overlap p-values were obtained for most tfs even though different standards were used. several tfs with very different overlap p-values often ranked at the top of the candidates. furthermore, their ranks at the candidate list changed little, though their overlap p-values between the expected targets and the observed degs  changed a lot . therefore, it may be concluded that our method is robust against the standard to select degs.\n",
            "\n",
            "our method was further applied to two datasets with no primary knockout or overexpression perturbation to discover important regulatory tfs involved in certain biological processes. expression profiles during a transition from fermentative to glycerol-based respiratory growth were obtained from  <cit> . expression data and phosphorylation information under pheromone response were from  <cit>  and  <cit> .\n",
            "\n",
            "RESULTS\n",
            "method verification by the knockout data of  <dig> tfs\n",
            "the results showed that  <dig> actual perturbed tfs out of  <dig> knockout data were correctly inferred. if only simple direct model  was considered,  <dig> were correctly inferred, among which, hsf <dig> was missed by our method due to its slightly dropped rank beyond the threshold. hsf <dig> ranked 20th if only model i was used but ranked 22nd by our method. it may be due to the increased noise level introduced by considering more regulation models. figure  <dig> shows the overlap significance of the expected targets of  <dig> tfs and the observed degs under these tfs’ knockout, obtained from the most likely model versus that obtained from model i. it could be seen from the figure that model i was employed by most tfs, suggesting that deletion of these tfs strongly affected expressions of their immediate direct targets. six tfs — pho <dig>  gln <dig>  aft <dig>  snf <dig>  aro <dig> and pdr <dig> — were correctly identified by our method but missed if only model i was used. all of the tfs with the exception of aft <dig> got more significant overlap in the most likely model, and their ranks in the candidate list were also improved. for example, pho <dig> ranked 64th if only model i was used, while it ranked 14th if model iv was also considered. as another example, pdr <dig> ranked 26th by using model i alone, but ranked 3rd if model iii was also considered. the more significant overlap and the improved ranks in the most likely model, rather than model i alone, suggested that the effects of these tfs knockout might transmit through more complex models. especially for pho <dig> and snf <dig>  the overlaps between the downstream genes from the most likely model  and the degs under their knockouts were  <dig> times more significant than that from model i. this finding suggested that it was highly likely that the regulation pathway of pho <dig> knockout was ptm-mediated two-layer cascade regulation model and pathway of snf <dig> knockout was highly likely to be ptm-mediated direct model. another two tfs, aft <dig> and gal <dig>  were both correctly identified by our method and by model i. however, their overlaps between the expected targets from the most likely model and the observed degs under their knockout were also much more significant than that from model i and their ranks were also improved, suggesting the potential regulation pathways of aft <dig>  and gal <dig>  knockout. these results demonstrated that ptm and cascade regulation helped explain tfs’ knockout effects. however, hu et al.  <cit>  did not observe any indication that indirect transcriptional or post-transcriptional regulation was responsible for the effects of a tf deletion. they drew the conclusion by comparing tf deletion target sets against each other or by comparing binding targets against deletion ones. this led to bias since the effects of a tf deletion might not be mediated by only one tf, but by cooperation or combination of several tfs. for example, the effects of snf <dig> deletion are mediated by msn <dig>  mig <dig>  sip <dig> and gln <dig> . in another example, the effects of aft <dig> deletion are mediated by cooperation of several tfs including msn <dig>  msn <dig>  dig <dig> and ste <dig> .\n",
            "\n",
            "in higher eukaryotes, multiple tfs simultaneously, cooperatively or competitively, regulate genes. for those tfs forming transcription complex to coordinate the expression of genes, such as arg80/arg <dig>  swi4/swi <dig> and mbp1/swi <dig>  they were all identified as causal regulators when one of their cooperative tfs was knocked out. for examples, arg <dig> ranked 1st  and arg <dig> ranked 6th  in the arg <dig> knockout experiment, and arg <dig> ranked 1st  and arg <dig> ranked 19th  in the arg <dig> knockout experiment. for those partially functionally redundant tfs  or promoter occupancy competitive tfs , however, they could not be correctly identified as causal regulators in their knockout experiments. it is possibly because that those tfs’ knockout effects can be partially or fully compensated by their backup tfs or competitive tfs.\n",
            "\n",
            "the results  of the knockout data of  <dig> tfs are listed in additional file  <dig> \n",
            "\n",
            "possible regulatory pathways downstream of snf <dig> knockout\n",
            "snf <dig> achieved the most significant overlap p-value  from ptm-mediated direct model , while the overlap p-value was  <dig>  when only simple direct model was used. this may suggest that snf <dig> affects gene expression through a ptm-mediated way, i.e., snf <dig> functions as a kinase rather than a tf. this finding is consistent with previous literature. the snf1p kinase complex, which phosphorylates serine and threonine residues, is essential for regulating the transcriptional changes associated with glucose depression through its deactivation of the transcriptional repressor mig <dig>  <cit> . snf <dig> is involved in the activation of s. cerevisiae heat shock transcription factor under glucose starvation conditions  <cit> . snf1p is also known or predicted to phosphorylate a wide range of substrates, including sip <dig>  <cit> , msn <dig>  <cit> , gat <dig> and gln <dig>  <cit> .\n",
            "\n",
            "to further verify whether or not snf <dig> functions through a ptm-mediated way, we selected those expression experiments in which snf <dig> was up/down-regulated  <cit>  assuming that differential expression of snf <dig> would lead to expression changes of its downstream genes, thus snf <dig> could be inferred as the causal regulator. we compared the overlap significance between the degs of those data and the expected targets of snf <dig> in the six models. when ptm-mediated direct model was used, the overlaps between the observed degs and the expected targets of snf <dig> were highly significant , while the number of the overlap was  <dig>  when other models were used. this further supports that snf <dig> functions as a kinase. figure  <dig> shows possible regulatory mechanisms downstream of snf <dig> knockout and in mas <dig> promoter mutant experiments  <cit> . though the overlap of degs between these two conditions is low, gene expressions are affected through similar pathways. snf <dig> regulates gene expressions through its modification of hsf <dig>  msn <dig>  mig <dig>  sip <dig>  gat <dig> and gln <dig> in the mas <dig> promoter mutant experiment, while snf <dig> affects expressions through msn <dig>  mig <dig>  sip <dig> and gln <dig> in the knockout experiment. the low overlap may be due to the involvement of other tfs unrelated to snf <dig> \n",
            "\n",
            "possible regulatory pathways downstream of aft <dig> knockout\n",
            "when simple direct model was used, the overlap p-value between the expected targets of aft <dig> and the observed degs under aft <dig> knockout was  <dig> , while the overlap p-value was  <dig> e- <dig> when ptm-mediated two-layer cascade regulation model  was used. figure  <dig> illustrates ptm-mediated two-layer cascade regulation pathway downstream of aft <dig> knockout. aft <dig> affects the expression of two protein kinases, tpk <dig> and kss <dig>  tpk <dig> modifies the protein states of msn <dig> and msn <dig>  while kss <dig> modifies the protein states of dig <dig> and ste <dig>  then these tfs lead to differential expressions of their target genes. tpk <dig> was differentially expressed in the aft <dig> knockout experiment, which supported the possibility of this regulatory pathway. the expression of kss <dig> was not changed, possibly due to the feedback regulation control of dig <dig>  ste <dig> and msn <dig> on kss <dig> \n",
            "\n",
            "to further verify whether or not aft <dig> functions through model iv under specific conditions, expression data with great down expression of aft <dig> were selected  <cit> . in the response to environmental changes under ypd stationary phase  <dig> d, aft <dig> showed  <dig>  fold change of down expression. we compared the degs under this condition with the expected targets in the six models of aft <dig>  the most significant overlap was also achieved at ptm-mediated two-layer cascade regulation model .\n",
            "\n",
            "method verification by the overexpression of  <dig> tfs and deletion of  <dig> tfs\n",
            " <dig> actual perturbed tfs out of  <dig> tfs deletion experiments were correctly identified, while  <dig> perturbed tfs out of  <dig> tfs overexpression data were correctly identified. in comparison,  <dig> perturbed tfs out of  <dig> tfs deletion experiments and  <dig> out of  <dig> tfs overexpression ones were correctly inferred if only model i was used. hap <dig> and rox <dig> were correctly identified by model i but missed by our method in their knockout experiments. although the same p-values were obtained , hap <dig> ranked 4th if model i was used but ranked 22nd by our method. the dropped rank of hap <dig> may be partially due to the increased noise level introduced by considering more models in our method. additionally, considering no cooperative or competitive interactions between tfs may be another reason. in the hap <dig> knockout experiment, hap <dig> forming hap complex with hap <dig> ranked 1st, though hap <dig> itself ranked 22nd. if such cooperative interaction between hap <dig> and hap <dig> was integrated into the method, hap <dig> would rank much higher. the situation was similar for rox <dig>  which ranked 4th if model i was used and ranked 29th by our method. our method correctly identified sok <dig> and sut <dig>  which were missed by model i in deletion experiments. our method also correctly identified six more perturbed tfs  from their corresponding overexpression experiments. the result showed that considering ptm and cascade regulation helped explain effects of tfs perturbation, especially effects of tfs overexpression. the results  of the deletion of  <dig> tfs and overexpression of  <dig> tfs are listed in additional file  <dig> \n",
            "\n",
            "with the exception of abf <dig>  hsf <dig>  met <dig> and rap <dig>  which appeared only in overexpression experiments, the remaining  <dig> tfs were performed with both overexpression and deletion experiments.  <dig> of  <dig> tfs  showed higher overlap significance between the direct targets of these tfs and the observed degs in overexpression experiments than those in deletion ones. this is partially due to the redundancy of tfs. for example, msn <dig> and msn <dig> are paralogous tfs with blastp e-value less than e- <dig>  swi <dig> and skn <dig> have paralogous tfs with blastp e-value less than e- <dig>  a subset of the homologous tfs bind to an overlapping group of targets, and thus it is not surprising that knocking out one of them has a small effect on the expression of its targets. in contrast to deletion, overexpression increases binding of tfs, and thus would activate most of their downstream targets.  <dig> out of  <dig> tfs in deletion experiments , and  <dig> out of  <dig> tfs in overexpression experiments showed more significant overlap between the degs and the expected targets in other models than in model i by  <dig> times. this may suggest that effects of tfs overexpression are more involved in indirect transcriptional or post-transcriptional regulation than those of tfs deletion.\n",
            "\n",
            "we also compared our results on deletion experiments from  <cit>  and those from  <cit> . simple direct model was responsible for the effects of two swi <dig> deletion experiments. for some tfs, however, different pathways were involved in the downstream of two deletion experiments of the same tf. for example, simple direct transcription explained the effects of pho <dig> deletion experiment provided by  <cit> , while ptm-mediated two-layer cascade regulation model was the most likely model for the effects of the pho <dig> deletion experiment provided by  <cit> . this may imply that indirect transcriptional or post-transcriptional regulatory pathways downstream of tfs are condition-dependent, even in the case of the same tf deletion.\n",
            "\n",
            "it should be noted that despite our prediction rate was less than 50%, it was much higher than that by random guess. if we randomly selected  <dig> tfs from the  <dig> candidates as causal regulators, the prediction rate was 14% . it would be even much lower if the quite stringent standard we used about valid findings  was also required since only small number of tfs satisfied the standard. in comparison, the prediction rate of our method achieved 28%  on hu et al. data, 31%  on chua et al. deletion data and 44%  on chua et al. overexpression data.\n",
            "\n",
            "possible regulatory pathways downstream of sut <dig> overexpression\n",
            "when simple direct model was used, the overlap p-value between the expected downstream genes of sut <dig> and the observed degs under sut <dig> overexpression was  <dig> , while p-value was  <dig> e- <dig> when hybrid two-layer cascade regulation model  was used, and sut <dig> ranked 2nd as the causal regulator out of  <dig> tf candidates. figure  <dig> shows the possible regulatory pathway downstream of sut <dig> overexpression. the effects of sut <dig> overexpression are mediated by gln <dig>  msn <dig>  msn <dig> and hap <dig>  among them, activity of gln <dig> is mediated by the phosphatase sit <dig>  sit <dig>  msn <dig>  msn <dig> and hap <dig> are all direct targets of sut <dig>  down expression was also observed for hap <dig> when sut <dig> was overexpressed, which further supported this regulatory pathway of sut <dig> \n",
            "\n",
            "method applied to transcriptome profiling during a transition from fermentative to glycerol-based respiratory growth\n",
            "to demonstrate the utility of the method to identify activated tfs in real biological process, we applied the method on transcriptome profiling during a transition from fermentative to glycerol-based respiratory growth  <cit> . we constructed a gold positive set as true activated tfs by selecting those tfs with two-fold or more differential expression from  <dig> candidates. the gold positive set consisted of  <dig> tfs.\n",
            "\n",
            "our method identified  <dig> tfs with overlap p-value being equal to or less than  <dig> . among  <dig> tfs,  <dig> tfs were true positives , expression of  <dig> tfs were changed less than two-fold , mal <dig> and mata <dig> were not profiled by the microarray.  <dig> tfs of the gold positive set were not detected by our method . in comparison, only  <dig> tfs with overlap significance p-value being equal to or less than  <dig>  were inferred if only simple direct model was used. among them,  <dig> tfs were two-fold or more differentially regulated ,  <dig> tfs were less than two-fold differentially expressed .  <dig> tfs of the gold positive set were missed . we compared the jaccard similarity score of our results with that obtained from simple direct model . the jaccard similarity score was defined as tp/  <cit> . the higher the jaccard similarity score, the better the method performs. the result showed that although our method got higher false positives, it achieved higher jaccard similarity score than that from model i, suggesting that taking multilayer-wise regulation models and ptm information into consideration was helpful for identifying activated tfs in the biological process.\n",
            "\n",
            "the high false positives of our method may be due to the incomplete selection of positive set, where only those tfs with differential mrna expression were chosen. this standard would miss those activated tfs with changed protein status . among the top  <dig> causal regulators identified by our method ,  <dig> regulators could not be inferred if only simple direct model was used .  <dig> of  <dig> regulators  were differentially expressed, which suggested the possibility of their activity changes in the process. the remaining  <dig> regulators were treated as false positives. however, the involvement of these  <dig> regulators in the diauxic shift is known or can be inferred by literature. for example, snf <dig> and ssn <dig> are both important regulators involved in the diauxic shift  <cit> . the snf <dig> protein kinase controls the induction of genes of the iron uptake pathway at the diauxic shift  <cit> . in glucose-grown cells, ssn <dig> negatively regulates  <dig> genes, including  <dig> that are induced during the diauxic shift  <cit> . moreover, ssn <dig> protein levels are depleted during the diauxic shift  <cit>  and ssn <dig> is also required for phosphorylation of sip <dig> during growth in nonfermentable carbon sources  <cit> . as another example, pho <dig> is an in vivo substrate of pho <dig>  one of the yeast nutrient-sensing kinases involved in the changes in gene expression profiles when yeast cells undergo a diauxic shift  <cit> . additionally, pho <dig> functions to repress or down-regulate the transcription of snz <dig>  which is expressed specifically in the postdiauxic to stationary phase  <cit> . in another example, cbf <dig> dna binding is necessary for repression of pho <dig> basal expression at the diauxic shift  <cit> . it has been reported that carbon deprivation caused the nuclear localization of yap <dig>  <cit>  and the proportion of cells with the nuclear yap <dig> concentrated during the diauxic shift  <cit> , which suggests that yap <dig> is involved in the process. put <dig> is also predicted to be involved in the process  <cit> . these regulators are possibly true positives and are activated in the process via changes of protein status.\n",
            "\n",
            "rox <dig> and sut <dig> ranked 6th and 4th, respectively, which were both differentially expressed. sut <dig> was the target of rox <dig>  and sut <dig> affected the expression of genes through hybrid two-layer cascade regulation model . this was consistent with the pathway of sut <dig> illustrated in figure  <dig>  where the perturbation of sut <dig> was mediated by gln <dig>  msn <dig>  msn <dig> and hap <dig>  sit <dig> , msn <dig> and hap <dig> were differentially expressed in the transition process, which further supported the possibility of the regulatory pathway of sut <dig> \n",
            "\n",
            "the overlap p-values of each tf in six models and the minimum p-value of each tf are listed in additional file  <dig>  tfs are ranked as causal regulators by their minimum p-values.\n",
            "\n",
            "results on the expression profiles in the transition from fermentative to glycerol-based respiratory growth.\n",
            "having demonstrated that our method identified those activated tfs with differential expression, we tested whether our method could find activated tfs that were both differentially expressed and post-translationally modified. we applied our method to infer activated tfs for mrna profiling data sets for yeast responding to the mating pheromone factor   <cit> .those identified tfs were evaluated by the transcriptional data and phosphoproteomics data after treatment with  <dig> mm α-factor for  <dig> min  <cit> . although the treatment concentrations were different between these two datasets, there was evidence that the transcriptional response to α-factor saturated at concentrations above  <dig>  nm  <cit> .\n",
            "\n",
            "the overlap p-values of each tf in six models and the minimum p-value of each tf are listed in additional file  <dig>  tfs were ranked as causal regulators by their minimum p-values.\n",
            "\n",
            "CONCLUSIONS\n",
            "our work provides an initial step toward analyzing gene expression data to find causal regulators by integrating tf-dna interactions and ptm information. we tested our method on large-scale tf deletion and overexpression experiments. the method correctly identified more actual perturbed tfs than the approach using only direct transcription, suggesting that indirect transcription and post-translation regulation are also responsible for tfs’ deletion/overexpression effects, especially for tfs’ overexpression effects. our method successfully identified known causal regulators and also inferred some novel tfs, which could lead to new hypotheses for future experiments during the processes of a transition from fermentative to glycerol-based respiratory growth and pheromone response. furthermore, possible regulatory pathways downstream of tfs in these processes were presented. expression and phosphorylation states of genes/proteins in the regulatory pathways provided further evidence to support the validity of these pathways.\n",
            "\n",
            "although our method was developed to find causal tfs, it could be easily extended to discover causal signal molecules and kinases. for example, cdc <dig> is a catalytic subunit of the main cell cycle cyclin-dependent kinase. if downstream gene set of cdc <dig> was constructed by ptm-mediated direct model, the number of the overlap between the observed degs  in pheromone response and expected targets of cdc <dig>  was  <dig> and the p-value was  <dig> e- <dig>  this suggests that cdc <dig> is involved in pheromone response, a finding that is supported by literature and consistent with experiments showing reduced phosphorylation of cdc <dig>  <cit> .\n",
            "\n",
            "in future work, we can further improve the method in several ways: 1) it could be beneficial to integrate into the framework information of transcriptional complexes and cooperative and competitive interactions. for example, although gcr <dig> was not correctly identified, rap <dig> was identified with the 2nd rank  in gcr <dig> knockout experiment. it is known that the rap1/gcr <dig> regulatory complex is required for efficient transcription of ribosomal protein  and glycolytic genes  <cit> . if such information of the cooperative interaction between rap <dig> and gcr <dig> was integrated into the framework, gcr <dig> could be correctly identified as the causal regulator. 2) it could be helpful to integrate into the framework proteomics data. protein abundance together with protein post-translational modifications status could be used to calculate the probability of existence of pathways, and then evaluate the possibility of the regulator as the cause. 3) it may be helpful to add sign in the model to represent the regulatory effects , to enhance the preciseness of the framework. 4) our method would miss those tfs only responsible for small subset of degs because overlaps between downstream genes of those tfs and the whole set of degs would not be significant. constructing downstream genes of combination of tfs may be beneficial in these cases. 5) the performance of our method relies heavily the size and quality of tf-dna interactions and ptm information. however, all of the interactions, especially ptm information, are incomplete and variable in different cellular conditions. limited and inaccurate information could mislead to biased paths and causal regulators. more abundant and accurate data available in the future would improve the power of the method.\n",
            "\n",
            "competing interests\n",
            "'the authors declare that they have no competing interests.\n",
            "\n",
            "authors' contributions\n",
            "ql developed software, analyzed data and wrote the manuscript. hd, yl and lx designed the project direction. th and gd participated in project discussions. yt, zt, ll, yl, hd and lx supervised the study. yt ,zt and lx revised the manuscript. all authors read and approved the final manuscript.\n",
            "\n",
            "supplementary material\n",
            "additional file 1\n",
            "rank distribution of the perturbed tf obtained by the method only using model i, by the method selecting the minimum p-value from the six models and by the method selecting the minimum p-value from acceptable models.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 2\n",
            "the effects of parameters to determine proper tfs and select degs on the results.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 3\n",
            " <dig> tfs with the number of downstream genes being no less than  <dig> in any one of the six models. orf names, gene names, and the number of downstream genes obtained by each model for  <dig> tfs are listed.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 4\n",
            "results on knockout data of  <dig> tfs.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 5\n",
            "results on deletion experiments of  <dig> tfs and overexpression ones of  <dig> tfs.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 6\n",
            "results on the expression profiles in the transition from fermentative to glycerol-based respiratory growth.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 7\n",
            "results on the expression data in the pheromone response.\n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements\n",
            "special acknowledgements go to merck research laboratories, usa, for funding a joint postdoctoral program. we acknowledge yvonne poindexter from the vanderbilt-ingram cancer center for her editing and the anonymous reviewers for their valuable comments and suggestions. other funding for this work include: the national basic research program of china , the national natural science foundation of china , and key infectious disease project .\n",
            "\n",
            "this article has been published as part of bmc bioinformatics volume  <dig> supplement  <dig>  2010: proceedings of the 21st international conference on genome informatics . the full contents of the supplement are available online at http://www.biomedcentral.com/1471-2105/11?issue=s <dig> \n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "our method correctly identified  <dig> tfs with overlap p-value being equal to or less than  <dig> , while the overlap p-value was  <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e- <dig> e\n",
            "12\n",
            "[{'summary_text': 'this model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\\nthis model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\\nthis model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\\nthis model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\\nthis model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\\nthis model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\\nthis model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "to develop and survive in utero, the mammalian conceptus develops a number of extraembryonic tissues and organs to provide nutritional support and protection to the embryo proper. these extraembryonic appendages are shed at birth. the amnion is the innermost extraembryonic membrane that surrounds the foetus of amniotes and delineates the fluid-filled amniotic cavity, thereby providing a confined niche within the conceptus and conferring protection and shock resistance  <cit> . of all the extraembryonic membranes it is morphologically the most conserved membrane. in contrast to the visceral yolk sac, the chorion and the allantois, the amnion is a thin transparent membrane  that is avascular in most amniotes. in mouse embryos, the amnion consists throughout gestation of a bi-layered membrane of squamous mesoderm and ectoderm facing the exocoelomic and the amniotic cavity respectively  <cit>  . the amniotic ectoderm is continuous with the embryonic ectoderm, whereas the amniotic mesoderm shares its borders with the mesothelium of the visceral yolk sac and the allantois. progressively, a basal lamina composed of collagen, laminin, nidogen and fibronectin fibers forms between the amniotic ectoderm and mesoderm  <cit> . the amniotic epithelium acquires an increasing number of microvilli at the surface, which may be associated with enhanced filtering and transport capacity across the membrane  <cit> .\n",
            "\n",
            "due to their lordotic position, mouse and rat embryos are peculiar in possessing inverted germ layers in which the ectoderm initially faces the inside of the egg cylinder  <cit> . starting at the 9-10-somite stage, mouse embryos undergo axial rotation, and hence achieve the regular flexed foetal position. consequently, the anterior junction between the embryo proper and the amnion on the one hand, and the embryo and the yolk sac on the other hand shifts progressively from anterior ectoderm over the heart field, to ventrally where the vitelline vein contacts the body wall  <cit> . when turning is finished , the embryo has become entirely enfolded in the amnion and visceral yolk sac  <cit>  .\n",
            "\n",
            "the amniotic membrane has low immunogenicity and hence high potential for regenerative medicine  <cit> . indeed, the amnion has been used for a century as a wound dressing  <cit> . recently, the amnion has gained attention due to the apparent presence of resident stem cells in term human amniotic ectoderm  <cit> . furthermore, cells isolated from human term amniotic ectoderm and mesoderm showed triple lineage differentiation capacity in cell culture  <cit> . similar studies on rat and mouse amniotic-membrane-derived cells have reported the existence of such pluripotent cells  <cit> . the origin of the amniotic stem cells is, however, unclear and, in the case of mice and rats, the source used to isolate the so-called amniotic stem cells has sometimes been controversial  <cit> .\n",
            "\n",
            "amnion formation is intimately related to the formation of the primitive streak early during gastrulation, but most investigators have focused on the analysis of the embryonic component of the conceptus, typically discarding the amnion in their studies. hence, the developmental origin of mouse amnion and its formation have been described fragmentarily  <cit> . the process of dividing the proamniotic cavity into the exocoelomic and the amniotic cavity by a membrane called the amnion is completed at the late streak/early bud to neural plate stage, depending on the mouse strain investigated.\n",
            "\n",
            "the primitive streak is the first morphological landmark of gastrulation at late ts  <dig> . it is characterized by a thickening at the posterior side of the epiblast, close to the embryonic-extraembryonic junction. streak ectoderm undergoes an epithelial-to-mesenchymal transition, and mesoderm emerges  <cit> . cells from the epiblast, via the streak, give rise to the mesoderm, including embryonic mesoderm and extraembryonic mesoderm of the chorion, amnion, yolk sac and allantois  <cit> . moreover, the epiblast gives rise to amniotic ectoderm as well as to embryonic ectoderm, endoderm and primordial germ cells  <cit> . the extraembryonic ectoderm will form the chorion and, with the ectoplacental cone, the chorionic disk of the placenta. visceral endoderm  becomes the endoderm component of the visceral yolk sac.\n",
            "\n",
            "fate mapping studies have revealed that amniotic mesoderm and amniotic ectoderm are derived from different regions of the epiblast. descendants of epiblast cells located at the posterior and posterolateral sides of the epiblast contribute to amniotic mesoderm  <cit> . indeed, labelling cells of the posterior primitive streak showed that the mesoderm derivative is mostly extraembryonic, part of which contributes to the formation of the amnion during early gastrulation   <cit> . in contrast proximal epiblast that is in the anterior half of the embryo at prestreak and streak stages, gives rise to amniotic ectoderm  <cit> .\n",
            "\n",
            "amnion formation begins with the accumulation of extraembryonic mesoderm leading to the formation of a posterior amniotic fold  <cit>  followed by folds along the sides of the egg cylinder like the progression of the lateral mesoderm wings  <cit> . bonnevie  disputed the role and existence of a posterior fold, but highlighted that the extraembryonic ectoderm at the anterior margin of the egg cylinder remains closely associated with the visceral endoderm, despite the eventual intercalation by extraembryonic mesoderm. snell & stevens  emphasized that extraembryonic mesoderm may accumulate at the anterior margin and regarded it as a small anterior fold. the exocoelomic cavity is then formed by the accumulation and coalescence of \"small cavities\", or \"small closed lumina\"  <cit> , within the posterior and lateral folds. according to snell & stevens , the posterior, lateral and anterior folds should, however, be thought of as a continuous constriction around the middle of the egg cylinder that tightens as the folds develop. this description differs from kaufman's authoritative description , which proposes the existence of separate posterior and anterior amniotic folds, each with an exocoelomic cavity. kaufman  described and illustrated the subsequent amnion expansion as follows: \"the rapid expansion of the posterior amniotic fold and its apposition and eventual fusion with the considerably smaller anterior amniotic fold results in the formation of the chorion and amnion, which divide the proamniotic cavity into ectoplacental, exocoelomic and amniotic cavities, respectively\" . several phenotypes observed in the amnion of mutant mouse models have been interpreted according to kaufman's description  <cit> . however, during routine analysis of serial sections, we came to the conclusion that this description might be inaccurate because we never observed an anterior amniotic fold with exocoelom. therefore, we re-examined the process of amnion formation in the mouse based on histological analysis of mouse embryos between the prestreak  and the neural plate stage . computer reconstruction of histological sections used for the atlas of mouse development confirmed the absence of an anterior fold. finally, we provide an animation that illustrates the single amniochorionic fold model and emphasizes the dynamics and arrangement of the tissues that contribute to amnion development.\n",
            "\n",
            "RESULTS\n",
            "amnion formation begins early during gastrulation in the mouse . at the midstreak stage, extraembryonic mesoderm accumulates between extraembryonic ectoderm and visceral endoderm at the posterior side of the embryo and a fold of extraembryonic and embryonic ectoderm bulges into the proamniotic cavity . this fold was historically called the \"posterior amniotic fold\", but we propose that it be named amniochorionic fold  because it gives rise to both the amnion and chorion. as the fold starts to form, the proamniotic canal becomes eccentric. subsequently, small lacunae appear within the extraembryonic mesoderm : the mechanism of lacunae formation is, however, unknown. it was possible that exocoelom formation could involve programmed cell death, similar to cavitation in the epiblast that leads to proamniotic cavity formation  <cit> . however, we did not detect apoptosis in posterior extraembryonic mesoderm during the process of exocoelom formation , making it likely that another process is involved. whatever the mechanism, the lacunae accumulate and fuse readily to form a larger cavity, the exocoelom, which is characteristically seen in late streak stage embryos .\n",
            "\n",
            "so far our observations fitted with the descriptions by kaufman and others in the field. the amniochorionic fold, which delineates the exocoelom, consists of extraembryonic mesoderm facing the exocoelom, and of a sheet of ectoderm facing the proamniotic cavity. however, often the ectoderm and mesoderm of the fold are transiently not fully aligned   <cit> . the ectoderm of the acf has an epiblast-derived component and an extraembryonic-ectoderm-derived one, as shown by the presence of oct3/ <dig> and expression of eomes in the respective layers   <cit> . the epiblast-derived component and the extraembryonic-ectoderm component of the acf form the prospective amniotic ectoderm and chorionic ectoderm, respectively. at the midstreak stage, the ectoderm of the acf consists mostly of extraembryonic ectoderm . however, as the fold enlarges, the contribution of embryonic ectoderm to the acf gradually increases .\n",
            "\n",
            "in contrast to what has been described by kaufman , we did not find evidence for a separate anterior amniotic fold with its own exocoelomic cavity. instead, a single cavity - the exocoelomic cavity - extends laterally around the egg cylinder as wing-like lateral extensions ; this process was further confirmed by transverse sections . the lateral extensions converge on the anterior midline at the embryonic-extraembryonic junction . on a parasagittal section of a late streak embryo, halfway between the midline and the lateral side of the egg cylinder , there appear to be two exocoeloms, a large posterior one and a smaller anterior one, which could fit with kaufman's interpretation. however, analysis of the neighbouring serial sections  reveals that this \"anterior cavity\" is actually continuous with the \"posterior cavity\", as represented in the scheme . hence, there is no anterior cavity in a midline section in the embryonic-extraembryonic junction region, when the allantoic bud and anterior endodermal furrow are in the same plane . therefore, we conclude that both \"cavities\" are the result of a cut through one single fold and the exocoelomic cavity.\n",
            "\n",
            "the proamniotic canal becomes localized anteriorly , close to where the lateral wings of the exocoelom converge. here, the extraembryonic and embryonic ectoderm from the acf will contact their counterparts at the anterior side of the egg cylinder resulting in the closure of the amniotic cavity and the separation of embryonic and extraembryonic ectoderm. we propose to name the latter region the anterior separation point  . the ectoderm of the embryo proper and the amniotic ectoderm now delineate the amniotic cavity completely, and the extraembryonic ectoderm is now called chorionic ectoderm. the junction between presumptive chorionic ectoderm and amniotic ectoderm remains distinct, without apparent cell mingling across the visible anatomical junction between extraembryonic ectoderm and embryonic ectoderm in the amniochorionic fold, as shown by the complementary expression patterns of eomes  and presence of oct3/ <dig>  .\n",
            "\n",
            "although the amniotic and chorionic ectoderm are now separated, the mesoderm of the fold that will intercalate between both ectoderm layers is not yet physically divided into chorionic and amniotic mesoderm. the exocoelomic cavity continues to enlarge with accumulation of extraembryonic mesoderm to form visceral yolk sac, allantois and blood islands further segregating the amnion and the chorion . the exocoelomic cavity is now delineated all round by extraembryonic mesoderm of the visceral yolk sac . as a consequence of the membrane segregation, the chorionic mesoderm and amniotic mesoderm become apposed to extraembryonic and epiblast-derived ectoderm, respectively. the amniotic and chorionic membranes then divide the proamniotic cavity of the egg cylinder into the amniotic, exocoelomic and ectoplacental cavities . in some mouse strains, the allantoic bud - the precursor of the allantois - is already visible before amnion closure  <cit>  , whereas it may appear after closure in other strains  <cit> . to challenge the single amniochorionic fold model further, also in an independent genetic background, we re-examined the original serial, longitudinal sections of two embryos that were prepared for the atlas of mouse development  <cit> , and generated 3d computer reconstructions thereof. an independent anterior amniotic fold was not found in transverse slices extracted from the 3d reconstructions . indeed, the reconstructions of these historical reference sections confirmed the presence of a single exocoelom extending around the egg cylinder .\n",
            "\n",
            "since exocoelom formation and its consequence for amnion and chorion formation are highly dynamic processes which are difficult to envision, we clarify the process in an animation .\n",
            "\n",
            "discussion\n",
            "earlier descriptions of amnion and exocoelom formation in the mouse have been partial and conflicting. both snell & stevens  and kaufman  describe a small anterior amniotic fold. however, snell & stevens consider this anterior fold a continuation of the posterior and lateral amniotic folds, and they do not describe it having lacunae. conversely, kaufman describes an anterior fold with an independent exocoelom. our present data demonstrates the absence of an independent, exocoelom containing, anterior fold. on the other hand, we show the presence of a single fold that is initiated posteriorly, and which we redefine as the amniochorionic fold . the fold expands laterally around the egg cylinder, like the progression of the lateral mesodermal wings. the lateral extensions converge on the anterior midline. the expansion of the exocoelomic cavity of the acf accompanies the lateral expansion of the fold around the egg cylinder, but does not reach the anterior side of the embryo. instead, a local accumulation of mesoderm can occur, forming what could be interpreted as a small anterior fold . we, however, propose not to call this small bulge an anterior fold because it risks being confusing. interestingly, while the epiblast grows directionally towards the primitive streak  <cit> , the proamniotic canal remains localized anteriorly, close to where the exocoelom wings converge, and maintains a relatively constant diameter before closure at the level of the embryonic-extraembryonic junction . this may promote the formation and expansion of the exocoelomic cavity within the extraembryonic mesoderm. compared with the growing embryo, relatively little cellular material is required in the developing amnion, chorion and yolk sac by virtue of exocoelom formation. ultimately, amnion closure is eccentric, close to the anterior margin of the egg cylinder, which we define as the anterior separation point .\n",
            "\n",
            "the differences in interpretation of amnion formation may be partly explained by the difficulties in correctly orienting and staging mouse embryos when sectioned within the deciduum, but also to slight variations in the expansion of the exocoelom on the left and right sides and to residual adjustment of axial symmetry of the embryo  <cit> . in our study, we analysed whole-mount embryos dissected free from the deciduum to better control the plane of section at the embryonic-extraembryonic junction. for instance, to visualize the asp in a midline section, it is crucial to examine a section in which the endodermal furrow and the base of the allantois are both present.\n",
            "\n",
            "we propose a model of amnion formation in the mouse involving a single acf growing and expanding laterally from the posterior side of the embryo: the single amniochorionic fold model. the 3d-reconstructions of kaufman's  original serial sections support our model further. the new material was in a cd <dig> background , and the model was confirmed in an f <dig>  background . however strain dependent differences in the formation of the amnion cannot be fully excluded.\n",
            "\n",
            "amnion development in the mouse is intimately related to exocoelom expansion. the initial establishment of the exocoelom is intriguing. the question remains as to what cellular and molecular mechanisms drive the formation of the lacunae in the extraembryonic mesoderm. selective cell survival and programmed cell death have been implicated in causing the cavitation in epiblast leading to the formation of the proamniotic cavity  <cit> . should a similar mechanism drive the formation of the exocoelomic cavity, cells at multiple sites throughout the extraembryonic mesoderm would have to undergo programmed cell death to generate the scattered small individual cavities. however, we did not detect apoptosis in the mesoderm of the fold, indicating that programmed cell death is likely not involved in the process of exocoelom formation. perhaps the formation of the exocoelomic cavity reflects merely the enlargement of extracellular spaces, or depends on the continuous rearrangement of cell adhesion molecules and extracellular matrix, allowing the formation of spaces in-between the mesodermal cells of the acf, similar to vascular lumen formation in invertebrates and vertebrates  <cit> . the accumulation and coalescence of these extracellular spaces or lacunae leads to the formation of a large extraembryonic coelom - the exocoelom - lined by extraembryonic mesoderm. to our knowledge, there are no mutants reported with explicitly impaired exocoelom formation in the newly formed extraembryonic mesoderm. nevertheless, the ectopic appearance of the cell adhesion molecule vcam and its receptor α1-integrin on the extraembryonic mesoderm lining the exocoelom  in foxf1-deficient mice leads to a compressed/ruffled exocoelom boundary  <cit> . conversely, reduced expression of a component of the extracellular matrix, fibronectin- <dig>  in chato-deficient mice results in expansion of the exocoelomic cavity  <cit> . this suggests that rearrangements of cell-cell and cell-extracellular matrix contacts may play a role in the formation/maintenance of the exocoelomic cavity and the tissues lining it.\n",
            "\n",
            "genetic studies in the mouse often provide valuable information on tissue morphogenesis. in contrast to the many mutants described for e.g. allantois  <cit>  and placenta  <cit> , remarkably few mutants appear relevant for our understanding of amnion formation . severe gastrulation mutants often lack an amniochorionic fold due to a general deficit of embryonic mesoderm, but some mutants display defects primarily related to amnion formation. remarkably, many of these seem related directly to bone morphogenetic protein  signaling , or putative modulators of bmp signaling . amnionless  mutants develop the most specific defects because they lack an amnion, whereas chorion, yolk sac blood islands, and allantois develop normally  <cit> . interestingly, embryonic ectoderm growth and differentiation are impaired, as well as the correct establishment of the middle - but not the posterior - region of the primitive streak  <cit> . bmp <dig> null embryos have a delayed amnion closure or fail to close the amnion, with a proamniotic duct remaining as late as early somite stages. the heart is formed within the exocoelomic cavity instead of the amniotic cavity  <cit> . smad <dig>  an intracellular mediator of bmp signaling, is also implicated in amnion closure: closure is delayed in smad5-deficient mice  <cit>  . this phenotype resembles not surprisingly a milder phenotype of the bmp <dig> null mice. moreover, the amnion of smad <dig> mutants often displays local thickenings that contain ectopic pgc-like cells, haematopoietic and endothelial cells; which could be attributed to ectopic bmp signaling via non-smad <dig> pathways  <cit> . this cell agglomerate cannot be related so far with any other loss-of-function model.\n",
            "\n",
            "* the observed amnion defects are considered secondary.\n",
            "\n",
            "we have provided here a morphological description and an animation of the poorly understood process of amnion formation. nevertheless, we are still a long way from understanding how the process is regulated at the molecular level. given the poor documentation of gene expression patterns in the amnion, it is at present also unclear if the amnion itself is differentially patterned in anterior versus posterior or lateral amnion. it is unclear what defines or distinguishes progressively embryonic and amniotic ectoderm, or yolk sac and amniotic mesoderm, at the molecular level. moreover, little is known about the amnion with respect to its impact on the development of the embryo and its surrounding extraembryonic tissues e.g. allantois, yolk sac and chorion, or vice versa. does the amnion then function exclusively as a container and filter for the amniotic fluid and as a shock absorber? or does it also signal actively to the surrounding tissues, and hence influences the patterning of the embryo?\n",
            "\n",
            "stem cell-like cells have been reported in the human amnion  <cit>  and recently also in the rat  <cit> . so far, their origin has been speculative. the origin, presence and potential of an amniotic stem cell-like population may differ in primate and rodent embryos because of the difference in topology between the disc-shaped primate and the cup-shaped rodent embryo, and the differences in developmental origin of amniotic layers  <cit> . however, if amniotic stem cell-like cells exist in mice, mouse genetic models will be extremely valuable for investigating the developmental origin of these cells, as well as in unravelling the complex cascade of molecular events that lead to the appearance of this cell population. the single amniochorionic fold model and the comprehensive animation reported here provide a new framework to investigate this cell population and to examine complex defects in the amnion of mouse mutants.\n",
            "\n",
            "CONCLUSIONS\n",
            "our histomorphological analysis revealed that only one amniotic fold is present in the mouse embryo, which we rename the \"amniochorionic fold\" . the acf emerges at the posterior side of the egg cylinder and expands laterally around the egg cylinder. exocoelom formation within this fold seems not to involve apoptosis. here we show that the acf and exocoelom do not expand through the anterior side of the embryo. amnion closure is eccentric and occurs close to the anterior margin of the egg cylinder, which we define as the \"anterior separation point\" . the 3d reconstructions of historical sections of e <dig>  embryos from kaufman  confirm the single amniochorionic fold model. this model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "this model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\n",
            "this model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\n",
            "this model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\n",
            "this model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\n",
            "this model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\n",
            "this model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion of mouse mutants.\n",
            "this model and the comprehensive animation provide a new framework for interpreting fate-map data, investigating amniotic stem cell populations and complex defects in the amnion\n",
            "13\n",
            "[{'summary_text': 'we propose the use of non-random samples to estimate the coefficient of linkage disequilibrium between genes at the two loci, genetic effects at the trait locus and the residual variance component to be modelled and inferred.\\nin this paper, we propose the use of non-random samples to estimate the coefficient of linkage disequilibrium between genes at the two loci, genetic effects at the trait locus and the residual variance component to be modelled and inferred.\\nin this study, we propose the use of non-random samples to estimate the coefficient of linkage disequilibrium between genes at the two loci, genetic effects at the trait locus and the residual variance component to be modelled and inferred.\\nin this study, we propose the use of non-random samples to estimate the coefficient of linkage disequilibrium between genes at the two loci, genetic effects at the trait locus and the residual variance component to be modelled and inferred.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "linkage disequilibrium  has long been one of the central topics in evolutionary and population genetics. linkage disequilibrium refers to non-random association of alleles at different linked or unlinked loci in a population. inference about ld provides useful information for distinguishing between alternative evolutionary models of genetic polymorphisms within or divergence between populations  <cit> . the current surge of population based association studies has reported identification of causal genetic variants of disease susceptibilities in humans  <cit>  and complex genetic variation in plants and animals  <cit> . the kernel of these studies is inference of ld between the genetic variants and functional loci that are closely genetically linked. thus, adequate prediction of ld is obviously crucial for reliability and accuracy of these studies.\n",
            "\n",
            "the coefficient of ld between two biallelic loci is defined as d = fab - fa fb in a randomly mating population, where fab, fa and fb are frequencies of gametes ab, alleles a and b in the population. the genetic parameter has been re-parameterized into different forms for various purposes of ld analysis  <cit> . hill proposed the well known \"chromosome counting\" method to estimate the parameter by using data of genotypes at the two loci from a random sample  <cit> . it has been widely used in population genetic analyses and studies on linkage disequilibrium based mapping  <cit> . the principle of the analysis has also been employed to develop widely used methods for predicting haplotypes of dna markers in natural populations  <cit> . in practice, however, it is very rare that the samples for ld analyses are truly randomly collected from the population under study. for example, the samples used in many association studies or population genomics analyses were so collected that the frequencies of some genotypes are artificially inflated to ensure that genotypes involving a rare allele are well represented  <cit> . weir and cockerham explored the consequences of implementing the method to estimate ld by using the samples in which some genotypes are missing and stressed that the method should not be used to estimate the parameter from non-random samples  <cit> , but neither appropriate theory nor method has been developed for inferring population disequilibrium from non-random samples.\n",
            "\n",
            "in this paper, we develop a new method to calculate the maximum likelihood estimate of the coefficient of linkage disequilibrium between genes at any pair of polymorphic loci in any randomly matting population by making use of samples of genotype data, which are non-randomly collected from the population of interest. on the basis of simulation studies, we demonstrate that bias in estimates of the disequilibrium parameter from hill's method arising from the use of non-random samples can be substantially reduced by implementing the new method. we compared analyses of a 'case and control' dataset of β-thalassemia using three different methods: hill's, haplotype prediction by the computer software phase <dig> . <dig> and the method developed in the present study.\n",
            "\n",
            "RESULTS\n",
            "we developed a likelihood-based statistical approach to estimate the coefficient of linkage disequilibrium between a pair of polymorphic loci in a natural population and to test for significance of the disequilibrium by making use of the samples that are not randomly collected from the population. the method uses information from the conditional distribution of genotypes at one locus given genotypes at the other in formulating the statistical analysis with non-random samples. this is in contrast to the approach proposed by hill  <cit> , which relies on information of a joint distribution of genotypes at the polymorphic loci and estimates the disequilibrium parameter from using the samples randomly collected from the population under question. hill's method has been extended or converted into various forms/approaches that are widely used in the current surge of genetic association studies and population genetic analyses  <cit> . here, our analysis is focused on hill's method  and the method  developed in the present study.\n",
            "\n",
            "to explore the adequacy of the methods h and l in estimating ld and their statistical power in detecting ld, we first conducted a simulation study to generate samples collected from the simulated population by sampling with various degrees of non-randomness. we then implemented the methods to analyze both the simulated datasets and the real data of  <dig>  kb dna sequence surrounding the β-globin gene from a 'case and control' study with β-thalassemia  <cit> .\n",
            "\n",
            "simulation study\n",
            "the study considered three schemes of sampling individuals from a simulated population in which the distribution of genotypes at the marker and disease loci was as described by table  <dig> for any given set of simulation parameters p, q and d. sampling scheme i involved n individuals being sampled completely randomly from the simulated populations. sampling scheme ii generated the samples in which either individuals with a specific marker genotype or individuals with a specific marker-disease genotype were missing. finally, sampling scheme iii mimicked the generation of the case-control samples used in most association studies, in which the cases and controls were present in the sample in equal proportions. in addition, we explored influence of various 'case and control' proportions on statistical power of the 'case and control' design for detecting ld. the computer programs designed for simulating linkage disequilibrium between two bi-allelic loci can be found elsewhere  <cit> , and were modified to generate the samples in the present study.\n",
            "\n",
            "q = q + d/p and r = q - d/\n",
            "\n",
            "linkage disequilibrium parameters were estimated based on  <dig> simulations of n =  <dig> individuals from  <dig> different populations: p and q are the frequencies of alleles of the marker  and trait , d is the coefficient of linkage disequilibrium between the marker and the disease loci, dmin and dmax are respectively the minimum and maximum possible coefficients of linkage disequilibrium given allelic frequencies p and q,  and  are the estimates from methods h and l respectively, and the means and standard deviations, s.d., calculated from  <dig> simulations.\n",
            "\n",
            "in sampling scheme ii, individuals were randomly sampled from  <dig> simulated populations but those with either a specific marker genotype  or a marker-disease genotype  were excluded in the sampling process. the sampling continued until the required number of individuals were obtained . from the samples so obtained, we calculated the allele frequencies, p and q, and then estimated d by use of both methods . means and standard deviations of the d estimates, based on  <dig>  simulations, are summarized in table  <dig>  in most cases, the two methods accurately estimate the disequilibrium parameters. however, when individuals with heterozygous genotypes at the marker locus were absent  in the sample, method h underestimated the parameters in the simulated populations 3- <dig>  in contrast, method l estimated the parameters adequately in these cases. in the cases where individuals with a specific marker-disease genotype were not present in the sample, method h severely underestimated the disequilibrium, for instance, when n <dig> =  <dig> in simulated population  <dig> and n <dig> =  <dig> in simulated population  <dig>  the biased estimates were substantially improved by making use of method l. one may argue that different performance of the two methods could be due to the limited sample size . we compared the methods with using much larger sample sizes  and observed that the same pattern in mean estimates of the disequilibrium parameter as demonstrated in table  <dig> with n =  <dig> . another question that may naturally rise from this sampling scheme is how much genotypic distribution at the simulated marker and disease loci deviates from that expected under hardy-weinberg equilibrium. the significance test shows that these genotype distributions in the samples generated from the sampling schemes highly significantly deviate from the hardy-weinberg expectations , revealing remarkable non-randomness of the samples with respect to the populations from which these samples were collected.\n",
            "\n",
            "means and standard deviations of coefficients of linkage disequilibrium were estimated from  <dig>  repeats for methods h and l when  <dig> individuals were generated from the sampling scheme ii in which either a marker genotype  or a marker-disease genotype  was missing. the population numbers are the same as those in table  <dig> and d is the true value of the disequilibrium coefficient.\n",
            "\n",
            "sampling scheme iii considered the scenario that the disease allele had a low frequency but generated an equal number of the case individuals  and the controls . to mimic the sampling scheme, we randomly generated a given number of 'case' or 'control' individuals for each of seven simulated populations. from these individuals,  <dig> 'case' and another  <dig> 'control' individuals were randomly collected, making a constant sample size of  <dig>  the samples so generated make a severely non-random collection of the cases but they present a typical example of the samples widely used in many current genetic association studies with a case-control design in which roughly an equal number of sporadic case individuals and control individuals were collected from the population under question  <cit> . use of the samples so collected raised a question of how the allele frequency parameters, p and q, can be calculated and used to estimate the disequilibrium parameter, d. we proposed to calculate p, the marker allele frequency, from the control sub-samples, and explored two alternative ways to obtain the value of q, the disease allele frequency. firstly, q was obtained from an independent population survey such as a prior epidemiological study or population survey. to assimilate this scenario, we used the stimulated value of q in estimation of d. secondly, we explored the use of q values directly estimated from the case-control sample in estimation of d.\n",
            "\n",
            "simulation parameters for the case-control sampling scheme and means and standard deviation of estimates of the disequilibrium coefficients,  and , from methods h and l respectively. p, q and d are simulated values of population allele frequencies at the marker and disease loci and linkage disequilibrium respectively.  are the theoretical minimum and maximum values of d. the disequilibrium coefficients were calculated when the marker allele frequency, p was calculated directly from the control sub-samples while the disease allele frequency q was either from population survey or directly estimated from the case-control samples.\n",
            "\n",
            "to explore influence of population size of the case-control samples on estimation and detection of the disequilibrium, we calculated estimates of the disequilibrium coefficient and the corresponding lod scores using method l in the case-control samples with an equal proportion of 'case and control' individuals but different total sample sizes. table  <dig> summarizes the means and standard deviations of  <dig>  repeated estimates of the disequilibrium coefficients and the corresponding lod scores. it can be seen that the disequilibrium parameters are estimated adequately from the method by use of the case-control sample with a size as small as only  <dig> individuals. the lod scores and the test statistic for significance of the disequilibrium, increase as the sample size increases.\n",
            "\n",
            "means and standard deviations of the estimates of the disequilibrium coefficients and the corresponding lod score from using various sizes of case-control samples.\n",
            "\n",
            "means and standard deviations of the estimates of the disequilibrium coefficients and the corresponding lod score from use of case-control samples of a constant size of  <dig> individuals but with varying proportions between the 'case and control' individuals, c:c.\n",
            "\n",
            "analysis of β-thalassemia dataset\n",
            "β-thalassemia is an autosomal recessive hemoglobinopathy caused by mutations in the β-globin  gene . the disorder is one of the most common inherited hemoglobinopathies in the world, with estimates of carrier frequencies ranging from  <dig> to 10% in some areas of the tropics and subtropics including southern china  <cit> . a frame shift mutation in codons  <dig> and  <dig>  a 4-bp deletion , of the human β-globin gene represents the most common β-thalassemia mutations in east and southeast asia. the population frequency of the deletion is as high as 3% in south china  <cit> . to survey the distribution of linkage disequilibrium among the polymorphic sites surrounding the β-globin gene, zhang et al collected a sample of  <dig> chinese individuals, including  <dig> βcd41/ <dig> thalassemia heterozygotes and  <dig> normal individuals  <cit> . they directly sequenced a  <dig> -kb dna region spanning  <dig>  kb of the β-globin cluster surrounding the deletion and detected  <dig> bi-allelic sites in the sequenced region. all individuals in the sample were genotyped at the polymorphic markers. this dataset represents a typical example of the selected sample in which disease carriers are deliberately enriched and no homozygote of the disease allele  is present in the sample.\n",
            "\n",
            "to evaluate linkage disequilibria across the sequenced region, the haplotypes in the sample were first predicted by use of the computer software phase <dig> . <dig> developed by stephens et al  <cit> , and the predicted haplotypes were then used to calculate the coefficients of linkage disequilibrium. we implemented the haplotype-based method and the two methods  to analyze the sequence data and calculated the coefficient of linkage disequilibrium between the disease mutation, which was set as position  <dig>  and each of the other polymorphic sites. we used the population frequency of 3% for the disease causing allele in the analysis and illustrate the distribution of estimates of the disequilibrium between each of the polymorphic sites and the disease causing mutation in figure  <dig> together with the lowest and highest bounds for the disequilibrium parameter which were inferred from the maker allele frequency estimates and the population frequency of the disease causing allele. it can be seen that the ld estimates from the present method could differ substantially from the other two methods. importantly, the ld estimates by the former lie properly between the corresponding theoretical bounds whilst the estimates by the latter two may be severely out of the bounds, suggesting serious bias in the estimates to the true parameters. figure  <dig> shows the lod score values for the ld estimates from the three methods and demonstrates that the present method provides much more likely estimates of linkage disequilibria in the vicinity of the disease causing site than the other methods under comparison. the data analysis clearly indicates that the method developed here is robust to non-randomness of the samples by which linkage disequilibrium is statistically inferred.\n",
            "\n",
            "discussion\n",
            "the past decade has witnessed great progress in high-throughput detection and genotyping of single nucleotide polymorphisms  in the genomes of plants, animals and humans. this has stimulated tremendous interest in mapping and identifying subtle genetic variants contributing to phenotypic variation of complex traits in natural populations through detecting linkage disequilibrium maintained in the populations by close linkage between alleles at genetic polymorphic sites and at trait loci. however, a common concern of association studies is the high proportion of false positive or false negative tests of association of trait phenotype with causal genetic polymorphisms as well as the limited statistical power in detecting genuine associations  <cit> . a rich pool of literature has been focused on exploring factors that cause these problems and seeking for solutions to them. the most prominent among these is population stratification that could cause both false positive and false negative inferences of association  <cit> .\n",
            "\n",
            "although it has been well established that skewed sampling from a population with a linkage equilibrium distribution of multi-locus genotypes may result in spurious linkage disequilibrium  <cit> , there has not been a comprehensive investigation of the consequences of non-random samples on statistical inference of ld from populations under disequilibrium. we demonstrate in the present study that the use of non-random samples could result in severely biased estimates of ld from the method proposed for random samples. the simulation study showed that the estimates could be so biased as to be outside the theoretical limits of the corresponding simulated values and the biases can be either up- or downwards. these results indicate that the non-randomness of the sample may result in considerable false positive or false negative inference of the disequilibrium parameter and, in turn, false positives or false negatives in association analyses in which significant degree of marker-disease association implies significant linkage disequilibrium. instead of considering the joint probability distribution of genotypes at the marker and disease loci in estimating ld from random samples  <cit> , we propose the use of the conditional probability distribution of disease genotypes given any marker genotype in developing a new method to estimate the parameter. the method avoids or effectively alleviates the influence of non-random presentation of any marker-disease genotype in the samples on the parameter estimation. on the basis of simulation studies, we show that the method yields equally adequate estimates of ld to the method previously proposed  <cit>  and currently widely cited in the literature when individual genotypes are randomly sampled from the populations. however, the method confers significant improvement over the current method when the parameter estimation is made from using artificially selected samples. methodologically, the improvement in the parameter estimation of the method developed in the present study over the current methods  <cit>  can be explained by their difference in extracting information of linkage disequilibrium between the two loci of interest from the samples under study. the present method uses the conditional probability distribution of genotypes at one locus on any given genotype at the other locus as illustrated in table  <dig>  when the sample of study is collected in such a way that genotype at one of the two loci undergo selection, for example, the case-control samples where genotypes at the disease locus are strongly selected, the joint distribution of genotypes at the two loci in the sample deviates greatly from the joint genotypic distribution in the population from which the sample is collected. however, the conditional distribution of genotypes at one locus given any genotype at the other selected or unselected locus in the sample remains approximately the same as that in the population. hence the analysis based on the conditional genotypic distribution is more robust to non-randomness of the samples to infer linkage disequilibrium in populations of interest than that based on the joint genotypic distribution.\n",
            "\n",
            "the 'case and control' design used in many association studies to screen for genetic polymorphisms in significant association with phenotypic variation probably illustrates the most popular example of ld analysis with non-random samples  <cit> . although the base populations under investigation may be randomly mating with respect to genotypes at marker-disease loci, the samples used in the studies are collected so that case individuals are well represented and thus create severely non-random presentation of the populations. given that the analysis of the samples is to infer the situation in the base populations, accurate inference of ld from the samples is obviously crucial for reliability of these analyses. the β-thalassemia data analysis in the present study represents a typical example of such studies. the analysis with the dataset, although very limited in sample size, highlights the importance of adequately tackling the non-randomness in the 'case and control' studies in order to achieve reliable assessment of ld. it clearly demonstrates the robustness of the method developed in the present study to the non-randomness. previous methods, which are widely implemented in the current literature of 'case and control' studies are only suitable for analysis with random samples and so can result in seriously biased inference of the disequilibrium parameter.\n",
            "\n",
            "a practical problem that is raised to accurately infer ld from 'case and control' samples is the need for accurate estimates of allelic frequencies at both marker and putative disease loci. we propose the use of control samples to calculate marker allelic frequency because there is usually not any prior selective criterion imposed on the marker genotypes of those individuals to be included in the control samples. it is shown that accurate estimates of the allele frequency are the basis for accurate estimation of the disequilibrium parameter , but neither the 'cases' or 'controls' separately or together are likely appropriate to estimate the allele frequency at the putative disease locus. in the scenario where epidemiological information is available for disease candidates such as the β-thalassemia data considered in the present study, the allele frequency estimate from a prior population survey may be used to meet this need. alternatively, one can search the lod score profile  for the mle of the disequilibrium parameter over various q values and choose the mle of d calculated from the q value at which the likelihood profile reaches the maximum. figure  <dig> illustrates such an example with the β-thalassemia data analysis and shows that the mle of d was obtained when q took a value of ~3%, which is near to the estimate from the previous population survey  <cit> .\n",
            "\n",
            "methodologically, the present study has been focused on the most prominent linkage disequilibrium measure, d as defined above. there are several other measures, such as d' or r <dig> etc, frequently used in the literature of genetic association study or population genetic analysis. these latter measures are either a scaled or standardized form of the basic disequilibrium parameter d. bias in estimate of d will be inherent to that of transformed versions of the parameter. for example, we re-analyzed the β-thalassemia data by using r <dig>  which is defined as d2/pq in the present notation, as the disequilibrium measure and showed the analysis in additional file  <dig>  the disequilibrium distribution shows an almost identical pattern to that demonstrated in figure  <dig>  in which d was the disequilibrium measure.\n",
            "\n",
            "one of the key properties of 'case and control' designs is its statistical power in detecting association between a genetic polymorphic marker to the phenotype of a disease trait. it has been widely accepted that level of ld is a critical factor in determining the power of 'case and control' designs to detect significance of genetic association  <cit> . several studies have focused on investigating factors affecting statistical power of the 'case and control' design, including marker allele frequency and errors in marker genotype and trait phenotype  <cit> . an equal proportion of cases and controls were proposed in these studies and implemented in many real 'case and control' experimental analyses  <cit> . the present study investigated the impact of using varying proportions of the cases and controls on the power of the 'case and control' design in detecting linkage disequilibrium and shows that, for a given size of the 'case and control' sample, increasing the proportion of the 'cases' decreases rather than increases the statistical power . in fact, increasing the proportion of control individuals in the 'case and control' samples alleviates the effects of non-randomness of the samples. this result indicates a need of reconsideration of the commonly accepted sampling strategy for 'case and control' study designs. we re-examined the question by implementing the chi-square based test proposed to test for significance of difference in marker allele frequency between cases and controls. the analysis was summarized as an additional file  <dig> and shows that use of an equal proportion of cases and controls in the case-control design is favoured for a higher statistical power to detect the genetic associations. however, it has been well established that the chi-square based association test is highly vulnerable to deviation of genotypic distribution from the hardy-weinberg equilibrium  <cit> . any violation to the equilibrium may result in severe type i error. moreover, it is clear from additional file  <dig> that the equilibrium does not usually hold in these samples.\n",
            "\n",
            "it is well established that ld based association analysis is effective only for the genes underlying mendelian traits or the genes with major genetic effects on polygenic traits  <cit> . in the present study we assumed that genotypes at the putative disease locus are observable as are those at the marker locus. the assumption holds for mendelian traits but may be questionable for quantitative traits. in fact, the theoretical model and analysis developed in the present study can be incorporated into the statistical framework we previously developed for detecting and estimating linkage disequilibrium between a genetic marker and a locus affecting a quantitative trait showing continuous or dichotomous phenotypic variation  <cit> . the quantitative genetic model allows allelic frequencies at the marker and trait loci, the coefficient of linkage disequilibrium between the two loci, genetic effects at the trait locus and the residual variance component to be modelled and inferred. integration of the two models enables the non-randomness of samples to be properly accounted for in the statistical inference on the genetic parameters.\n",
            "\n",
            "CONCLUSIONS\n",
            "we demonstrated that the non-randomness may cause seriously biased assessment of ld when using the current methods originally developed for random samples. we have developed a new approach for inferring ld from samples with various degrees of non-randomness, and showed the significantly improved robustness of the present approach over the current methods when non-random samples were to evaluate ld through intensive simulation studies and analysis of a case and control sample of β-thalasemia. as accurate estimation of the disequilibrium parameter is crucial for any association study, in which the case/control design represents a typical example of non-random samples, the present paper highlights the importance of tackling the problem of using non-random samples to the community of ld analysis, and in addition, provides a route to improve statistical reliability in association studies.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "we propose the use of non-random samples to estimate the coefficient of linkage disequilibrium between genes at the two loci, genetic effects at the trait locus and the residual variance component to be modelled and inferred.\n",
            "in this paper, we propose the use of non-random samples to estimate the coefficient of linkage disequilibrium between genes at the two loci, genetic effects at the trait locus and the residual variance component to be modelled and inferred.\n",
            "in this study, we propose the use of non-random samples to estimate the coefficient of linkage disequilibrium between genes at the two loci, genetic effects at the trait locus and the residual variance component to be modelled and inferred.\n",
            "in this study, we propose the use of non-random samples to estimate the coefficient of linkage disequilibrium between genes at the two loci, genetic effects at the trait locus and the residual variance component to be modelled and inferred.\n",
            "\n",
            "14\n",
            "[{'summary_text': 'the sequence of the tilapia genome will be a fundamental resource for genetic selection, on traits such as growth performance and disease resistance, to create strains of fish optimized for the unique culture conditions of each country.\\nthe sequence of the tilapia genome will be a fundamental resource for genetic selection, on traits such as growth performance and disease resistance, to create strains of fish optimized for the unique culture conditions of each country.\\nthe sequence of the tilapia genome will be a fundamental resource for genetic selection, on traits such as growth performance and disease resistance, to create strains of fish optimized for the unique culture conditions of each country.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "tilapia  are among the most important species in aquaculture and a primary source of animal protein for millions of people in the developing world  <cit> . only limited efforts have been made toward genetic improvement of these species  <cit> . the sequence of the tilapia genome will be a fundamental resource used for genetic selection, on traits such as growth performance and disease resistance, to create strains of fish optimized for the unique culture conditions of each country.\n",
            "\n",
            "tilapia and other closely related species of african cichlid fishes are also widely used in basic research. because of their intimate physiological relationship with the environment, tilapia are ideal for studies of ion regulation  <cit> , the accumulation of heavy metals  <cit> , and detoxification of biotoxins  <cit> . nile tilapia expressing a humanized insulin gene are being studied as a source of islet cells which might be transplanted into humans for control of type i diabetes  <cit> . tilapia are also an important model for studying environmental influences on sex differentiation  <cit> . the closely related haplochromine cichlids of the east african lakes are a model system for studying the genetic basis of behavior  <cit>  and evolutionary processes of adaptation and speciation  <cit> .\n",
            "\n",
            "cichlid genomics\n",
            "considerable progress has been made in developing genomic resources for tilapia and other east african cichlid fishes. genetic maps have been published for tilapia  <cit> , lake malawi haplochromines  <cit> , and astatotilapia burtoni  <cit> . there are also extensive collections of ests for lake victoria haplochromines  <cit> , a. burtoni  <cit>  and nile tilapia  <cit> . several bac libraries have been constructed for nile tilapia  <cit> , and fingerprinted to construct a physical map  <cit> . bac libraries have been constructed also for haplochromine cichlids from lakes malawi  <cit> , victoria  <cit>  and tanganyika  <cit> .\n",
            "\n",
            "comparative physical maps\n",
            "comparative maps have been a useful intermediate resource for many agricultural species before complete genome sequences were available  <cit> . most often these comparative maps have relied on mapping homologous gene markers in radiation hybrid panels  <cit> , but comparative maps have also been based on analysis of bac end sequences  <cit> . until a complete genome sequence is available for tilapia, comparative maps to the genome sequences of model fish species will provide the best organization of the partial sequence data for cichlid fishes.\n",
            "\n",
            "the utility of a comparative map is proportional to the extent to which synteny exists between the two genomes. useful comparative maps have been constructed between cattle and human  <cit> . the divergence among many fish lineages is much older, creating the potential for more extensive genome rearrangements. the ostariophysi  and acanthopterygii  diverged ~300my ago  <cit> . divergence among percomorph groups  occurred more than 100my ago  <cit> . the utility of comparative maps across these greater evolutionary distances is not yet clear.\n",
            "\n",
            "early research suggested that the rate of chromosome evolution is relatively low in non-mammalian vertebrates  <cit> . recently it has been suggested that the rate of chromosomal rearrangement increases immediately after episodes of whole-genome duplication  <cit> . teleost fishes experienced an additional round of whole genome duplication about  <dig> my ago  <cit> , and recent papers have suggested that fishes continue to have a high rate of chromosomal rearrangement  <cit> . however, the more extensive inter-chromosomal rearrangements detected in the zebrafish genome may be due to unique evolutionary processes in that lineage, and there appear to have been no major inter-chromosomal rearrangements in the medaka genome during the last 300my  <cit> . the green pufferfish shows relatively little inter-chromosomal rearrangement since divergence from the ancestral bony vertebrate  <cit> . most of the changes in the pufferfish lineage represent fusions that reduced the chromosome number after whole genome duplications.\n",
            "\n",
            "the goal of the present study was to construct a comparative physical map between tilapia and the latest sequence assemblies for three other percomorph species: stickleback, medaka and pufferfish. from this comparative map we estimate the extent of chromosomal rearrangement during the recent evolution of these species.\n",
            "\n",
            "RESULTS\n",
            "new bac library\n",
            "the bac library  constructed at the benaroya research institute consists of  <dig>  clones  with an average insert size of  <dig> kb. this represents a total of  <dig> gbp or approximately 10× coverage of the tilapia genome. the methods used to prepare this library are presented in additional file  <dig> \n",
            "\n",
            "sequencing statistics\n",
            "genoscope\n",
            "the construction of the bac libraries sequenced at genoscope was reported previously  <cit> . a total of  <dig>  clones from these libraries , have been restriction fingerprinted and assembled into  <dig>  contigs  <cit> . genoscope end sequenced a total of  <dig>  clones . from  <dig>  clones, a total of  <dig>  end sequences were obtained, representing  <dig> × clone coverage of the genome. the mean trimmed length of the sequences was  <dig>  bp, for a total dataset of  <dig> , <dig> bp representing  <dig> % sequence coverage of the genome.\n",
            "\n",
            "broad\n",
            "the broad institute end sequenced  <dig>  clones  from the benaroya library, obtaining a sequence for at least one end of  <dig>  clones, representing  <dig> × clone coverage of the genome. multiple attempts were made to sequence some clones and therefore, a total of  <dig>  end sequences were finally submitted to genbank. the mean length of the sequences was  <dig>  bp, for a dataset of  <dig> , <dig> bp. after quality trimming and vector removal with lucy, a total of  <dig>  sequences remained, with a mean length of  <dig>  bp, for a total of  <dig> , <dig> bp, representing  <dig> % sequence coverage of the genome. these sequences were previously analyzed for their repeat content  <cit> .\n",
            "\n",
            "microsatellites\n",
            "microsatellite motifs were identified in  <dig>   of the  <dig>  sequences. these included  <dig>  dinucleotide,  <dig>  trinucleotide, and  <dig> tetranucleotide repeats . over half of the repeats  were ac dinucleotides. at and ag dinucleotides were also abundant. aat was the most frequent trinucleotide. these microsatellites could be exploited to develop new genetic markers and could be used to anchor the fpc-based physical map  <cit>  to the genetic map  <cit> .\n",
            "\n",
            "genes\n",
            "a total of  <dig>   repeat-masked sequences had a significant  blastx hit to the uniprot database. we found that  <dig>   of the repeat-masked sequences had a significant  blastn hit to the  <dig>  nile tilapia est set  <cit> . therefore,  <dig>   of the sequences had either a significant blastx hit to uniprot or a significant blastn hit to the nile tilapia ests. there were  <dig>   sequences that had a significant hit to both uniprot and the nile tilapia ests.\n",
            "\n",
            "comparative mapping\n",
            "a total of  <dig>  bac end sequences were blasted against the genome assemblies of stickleback, medaka and pufferfish. the results are summarized in table  <dig>  the proportion of sequences that had hits with e-values less than e- <dig> ranged from  <dig> percent against pufferfish,  <dig> percent against medaka and  <dig> percent against stickleback. twenty-eight percent of the bacs had at least one hit to the stickleback genome assembly.\n",
            "\n",
            "*numbers for type  <dig>   <dig>   <dig> &  <dig> do not include hits to the contigs in the 'unordered chromosome' of each genome assembly. type  <dig> hits were scored when the two end sequences of a clone hit within  <dig> kb of each other in the target genome.\n",
            "\n",
            "we classified the bacs into one of four types, according to the pattern of blast hit. type  <dig> clones are those for which only a single sequence produced a hit in the target genome. type  <dig> clones are those in which the sequences from the two ends of the bac hit in the appropriate opposing orientation within  <dig> kb in the target genome. type  <dig> clones are those in which the two end sequences of a bac hit the same chromosome in the target genome outside of the  <dig> kb range. type  <dig> bacs are those in which the two sequences hit different chromosomes in the target genome.\n",
            "\n",
            "since the average blast hit rate against the stickleback genome is 17%, we expected the proportion of clones with hits on both ends would be  <dig> %. in fact we observed a slightly greater proportion , possibly reflecting a clustering of conserved sequences in the genomes. when both ends of a bac had blast hits, they were most often found within  <dig> kb on the same chromosome in the target genome . a much smaller proportion  were found at larger distances on the same chromosome in the target genome .\n",
            "\n",
            "conservation of gene order\n",
            "we can use the ratios of type  <dig>   <dig> and  <dig> hits  to estimate the number of rearrangements between genomes. across the three species, 27-41% of double hit clones are type  <dig> or  <dig>  if the bac clone inserts average  <dig> kb, and every third clone has a break in synteny, it would suggest a breakpoint every  <dig> ×  <dig> kb =  <dig> kb across the genome. this is equivalent to more than  <dig> breakpoints across the genome, or about  <dig> breakpoints per chromosome. we suspect this simple statistic overestimates the true number of chromosomal rearrangements.\n",
            "\n",
            "the best estimate of intra-chromosomal rearrangements is the number of type  <dig> bacs relative to the number of type  <dig> + type  <dig> bacs. this proportion is between  <dig> and 6%, suggesting an intra-chromosomal rearrangement every  <dig> ×  <dig> kb =  <dig> mb. if the average chromosome is  <dig> mb, this suggests about  <dig> breakpoints  per chromosome. we detected a mean of  <dig>  breakpoints per chromosome, with at least one rearrangement on each stickleback chromosome . the observed breakpoints were spanned by an average of  <dig>  bac clones. unfortunately, the relatively low clone coverage of the type  <dig> bacs does not allow us to identify all of the likely breakpoints, or precisely map their locations. still, the high end of these estimates  suggests there have been only  <dig> inversions since the divergence of tilapia and stickleback. the type  <dig> hits are visualized in circos plots in additional files  <dig>   <dig>   <dig>  figures s1-s <dig> \n",
            "\n",
            "type  <dig> bacs are possible evidence of inter-chromosomal rearrangements, and represent 24-37% of the two-hit bacs. this might suggest more than  <dig> breakpoints in synteny for each chromosome. however, we do not think this statistic is an indication of a large number of inter-chromosomal transfers of genes. rather, it probably includes many instances in which one of the blast matches is to a paralog on a second chromosome. for example, if the syntenic copy of the gene has been lost, blast will identify a paralog on another chromosome as the best hit. this kind of gene loss is a common feature of fish genomes, which underwent a whole-genome duplication about 300my ago. alternate loss of even a small proportion of genes from these duplicated regions would be sufficient to create the pattern. there are about  <dig>  genes/chromosome, and if only 5% of them  were deleted after the whole genome duplication, it would be sufficient to create the pattern we see in the bac data. the fact that type  <dig> blast hits have much lower e-values than type  <dig> blast hits  tends to reinforce this view.\n",
            "\n",
            "we mapped the rearrangements onto a phylogeny of the four species. the results suggest that approximately 15- <dig> rearrangements have occurred on each lineage since they diverged from their common ancestor. there is no indication that the rate of rearrangement is higher in one lineage than another.\n",
            "\n",
            "comparative physical maps\n",
            "these blast results are displayed in a gbrowse interface at http://www.bouillabase.org . separate tracks display the type  <dig>   <dig>   <dig> and  <dig> blast hits. an additional track displays the blast hits from each of the fingerprint contigs in the previous physical map  <cit> . because these fpc contigs contain multiple bac clones, they help to tie the physical map together at larger scales than the end sequences of individual clones.\n",
            "\n",
            "CONCLUSIONS\n",
            "end-sequencing of these bac libraries was a key step in preparing the tilapia genome for shotgun sequencing. together with the bac fingerprint database, these sequences will provide long-range structure for scaffolding the contigs of genome assemblies to construct a golden path across the genome.\n",
            "\n",
            "recent molecular phylogenies appear to have reached a consensus that cichlids are more closely related to medaka than to either pufferfish or stickleback  <cit> . nevertheless, a higher number of the tilapia bac end sequences hit stickleback  than either medaka  or pufferfish . this discrepancy might be due to variation in the quality of each assembly, or it might support an alternative phylogenetic reconstruction. regardless, it appears that the stickleback sequence is currently the best reference sequence for building comparative maps of tilapia  <cit> .\n",
            "\n",
            "finally, these data suggest that chromosomal evolution in recent teleosts is dominated by alternate loss of gene duplicates, and by intra-chromosomal rearrangements. the rate of these rearrangements is relatively slow, on the order of one per million years. so the prospects are good for building useful comparative maps between sequenced genomes and the large number of as yet unsequenced teleost species of commercial or scientific importance.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the sequence of the tilapia genome will be a fundamental resource for genetic selection, on traits such as growth performance and disease resistance, to create strains of fish optimized for the unique culture conditions of each country.\n",
            "the sequence of the tilapia genome will be a fundamental resource for genetic selection, on traits such as growth performance and disease resistance, to create strains of fish optimized for the unique culture conditions of each country.\n",
            "the sequence of the tilapia genome will be a fundamental resource for genetic selection, on traits such as growth performance and disease resistance, to create strains of fish optimized for the unique culture conditions of each country.\n",
            "\n",
            "15\n",
            "[{'summary_text': 'we have developed a novel, high-throughput method for quantifying somatic instability that combines a reliable, high-throughput method for quantifying somatic instability with mathematical modeling based on gene expression data.\\nour results suggest that somatic instability occurs as a consequence of ongoing pathogenesis, and suggest novel pathways that may be important in determining the tissue specificity of instability in hd.\\nwe have developed a novel, high-throughput method for quantifying somatic instability in hd.\\nour results suggest that somatic instability does not occur as a consequence of disease pathogenesis, suggesting that patterns of instability are determined by the combined effects/interactions of many genes.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "expansions of trinucleotide repeat sequences over certain thresholds cause more than  <dig> human diseases including huntington's disease , a number of spinocerebellar ataxias , myotonic dystrophy  <dig> , and fragile x syndrome. interestingly, expanded trinucleotide repeat sequences undergo progressive, expansion-biased tissue-specific somatic instability  <cit> . as the severity of these disorders is highly dependent on repeat length, somatic instability in tissues that are the pathogenic targets is predicted to contribute to disease. notably, in hd, striking somatic expansion of the hd cag repeat occurs in the striatum and cortex, brain regions that are major targets of the pathogenic process. furthermore, studies both in hd patients and in a knock-in mouse model of hd provide compelling evidence indicating that somatic expansion in these brain regions accelerates the ongoing pathogenic process  <cit> . therefore, understanding the mechanisms underlying tissue-specific somatic instability in hd may provide novel routes to therapies.\n",
            "\n",
            "somatic instability is critically dependent on dna repair genes and is also influenced by cis-factors  <cit> . however, it is unknown what determines its tissue specificity. it has been proposed that the expression levels of dna repair genes and/or the pathogenic process itself may underlie tissue patterns of instability  <cit> . given that somatic hd cag instability occurs in many tissues to varying extents  <cit> , we reasoned firstly, that tissue specificity may governed by many factors, and secondly, that studying a large cross-section of tissues with different instabilities would provide the most information concerning the major factors underlying tissue instability patterns. therefore, in order to gain insight into the factors that govern the tissue specificity cag instability in hd, we have taken quantitative, global and unbiased approaches.\n",
            "\n",
            "using accurate genetic knock-in mouse models of hd  <cit>  that exhibit similar tissue-specific patterns of somatic instability to those seen in hd patients  <cit> , we developed a novel instability quantification method that is sensitive and applicable to high-throughput assays. we then integrated this methodology with unbiased and global bioinformatic approaches to identify a gene expression \"signature\" and biological pathways that correlate with tissue instability. using these methods we have, a) tested the role played by factors previously proposed to contribute to the tissue specificity of somatic instability, and b) uncovered novel pathways that may be important in determining the tissue specificity of instability in hd.\n",
            "\n",
            "RESULTS\n",
            "instability quantification\n",
            "previous methods for determining instability following pcr amplification of repeats from 'bulk' genomic dna have either been qualitative, or have failed to adequately account for amplification efficiencies that differ between stable and unstable tissues. in contrast, quantitative small pool-pcr  methods  <cit>  are extremely labor-intensive and impractical for high-throughput analyses. in order to facilitate high-throughput, global analyses of somatic instability we therefore first developed a novel method for quantifying cag repeat sizes from 'bulk' genomic dna. pcr amplification of trinucleotide repeats generates multiple pcr products, viewed using genemapper software as a cluster of peaks differing by a single cag repeat unit . distinguishing signal peaks from noise peaks is critical for the accurate measurement of instability. in typical genemapper traces of pcr-amplified trinucleotide repeats, there is no clear boundary between signal and noise, making defining noise peaks  extremely difficult. to solve this problem, we developed a novel background correction method , where 20% of the height of the highest peak was set as the threshold for each analysis. for stringent analyses, peaks with heights lower than this threshold level were excluded from quantification. we used a conservative threshold factor  in this study as this detects peaks with good signal intensity , and is more resistant to amplification variation than lower thresholds . however, if peak signals are strong enough, a lower threshold  will provide more sensitive quantification.\n",
            "\n",
            "validation of the relative peak height threshold quantification method\n",
            "we first determined the reproducibility of our method by quantifying instability index in  <dig> tissues from 2- <dig> different hdhq111/+ mice at  <dig> months of age. as shown in figure  <dig>  the shift in the highest peak compared to tail  and the instability index  were highly reproducible between mice for all tissues tested. note that the instability indices of stable tissues  were negative because stable tissue genemapper traces were biased toward contraction likely due to the increased amplification efficiency of shorter cag alleles. instability indices of  <dig> tissues ranged from - <dig>   to  <dig>  .\n",
            "\n",
            "the relative peak height threshold method can also be applied to different types of instability quantification depending on the focus of the biological question. thus, after applying the relative peak height threshold, we can determine contraction and expansion indices , the number of contracted and expanded peaks  or the relative composition  of contracted, expanded and unchanged peaks . importantly, these measurements of different aspects of instability may be useful to capture the complexity of tissue instability. in all cases, measurements were reproducible for all tissues across multiple mice. to represent the levels of instability of tissues for further analysis, we used the instability index .\n",
            "\n",
            "next, to examine the effect of template dna amount on instability index, we calculated striatal instability indices using different amounts of template dna from striatum of an hdhq111/+ mouse at  <dig> months of age. as shown in figure 3a, instability indices calculated using the relative peak height method generated consistent instability indices  from a wide range of template dna amounts .\n",
            "\n",
            "we then compared instability indices using our relative peak height threshold method to somatic instability quantified using sp-pcr on genomic dna of tissues from the same mouse . figure 3b shows examples of tissues exhibiting high, medium and low instability indices, and the corresponding cag repeat length frequency distributions obtained by sp-pcr. these data indicated that the instability index broadly captured the bulk of the somatic variation detected by sp-pcr, but not the rare large expansions. however, there was a highly significant correlation between the instability index obtained using the bulk dna method and an instability index quantified from the small pool data , suggesting that although instability index using bulk dna may not be sensitive enough to detect rare molecules, it can give a good estimate of overall instability.\n",
            "\n",
            "together, our analyses suggest that the instability index, determined from genemapper traces of bulk genomic dna, is a reproducible measurement, relatively insensitive to input dna amount and well suited for high-throughput analyses where sp-pcr may be impractical.\n",
            "\n",
            "genome-wide identification of an instability-correlated gene expression signature\n",
            "with the aim of investigating the tissue specificity of somatic instability in a global and unbiased manner we then took a bioinformatics approach. using  <dig> different tissues from 5-month hdhq111/+ mice as our training set , with instability index as a quantitative phenotype, we analyzed mouse tissue gene expression data  to identify a gene expression signature that correlated with tissue repeat instability. hdhq <dig> somatic instability  increases over time  <cit> . we chose  <dig> months as this represents a time-point at which tissue differences in instability can be readily resolved. notably, the gene expression data is derived from mice that differ in age and genetic background  to the hdhq <dig> mice in this study. while age and genetic background-related gene expression changes will increase the noise in our system, this broad, tissue-based analysis allows us to pull out major tissue-specific gene expression differences that occur over and above age- and genetic background-related effects.\n",
            "\n",
            "thus, we modeled instability index as a function of gene expression using partial least square regression   <cit> . an instability-correlated gene expression signature was identified by leave-one-out cross validation  of training samples , and the signature, comprised of the  <dig> most highly correlated probes with tissue instability , reflected the instability index with a root mean squared error of prediction of  <dig>  .\n",
            "\n",
            "we then confirmed the predictive power of this instability-correlated gene expression signature by comparing measured instability indices with predicted instability indices from our regression model in new independent samples. for this, 1) we measured instability indices of four new independent hdhq111/+ tissues  and compared these with instability indices predicted from the regression model in the same tissues , and 2) we predicted instability indices using independent microarray data from hdhq <dig> striatum and cerebellum and compared these with measured instability indices . as shown in figure  <dig>  the predicted and measured instability indices matched closely in all cases  with a significant correlation , indicating that instability index can be relatively precisely predicted from the gene expression signature. furthermore, these data demonstrate that although the model was based on gene expression data and instability index data from mice that differed in age and genetic background, it nevertheless has significant predictive power. this indicates the presence of tissue-specific factors related to instability independent of age and genetic background.\n",
            "\n",
            "tissue instability prediction\n",
            "our sensitive quantification method and instability-correlated gene expression signature/regression model is a versatile tool. one of the advantages of our regression model is that the 'propensity' for instability can be predicted when instability can not be directly measured. for example, our approach allowed a prediction of an instability index in  <dig> different tissues and conditions in the mouse tissue gene expression data set , a far greater number than has ever been previously measured, providing a comprehensive view of tissue instability. interestingly, although most of the tissues  were predicted to be relatively stable, some degree of cag repeat instability was predicted for many tissues in the nervous system.\n",
            "\n",
            "the plsr model built with the instability-correlated gene expression signature from  <dig> training tissues was used to predict instability index for each of the tissues analyzed in the gnf mouse gene expression atlas. data represents mean of  <dig> replicates.\n",
            "\n",
            "pathogenesis and instability\n",
            "we are interested in understanding the factors that contribute to the tissue specificity of hd cag somatic expansion, particularly, why the repeat is so unstable in the striatum. our instability quantification/bioinformatics approach provides a novel, global and unbiased means of probing these factors. one possibility that could at least in part explain the tissue specificity is that somatic instability occurs as a result of the ongoing hd pathogenic process, as previously hypothesized  <cit> . we first used our ability to predict instability from gene expression to test this hypothesis. the instability-correlated gene expression signature reflects a cell or tissue state that is associated with instability. if instability occurred as a result of ongoing pathogenesis, one would expect an altered level of instability-correlated gene expression signature in cells expressing mutant huntingtin compared to wild-type cells. therefore, we performed gene expression profiling on striata and cerebella of 10-week hdhq111/ <dig> mice that exhibit an ongoing pathogenic process and somatic instability in striatum but not in cerebellum  <cit> , and on wild-type hdh+/+ littermates, and predicted instability using the regression model above. interestingly, as shown in figure 5a, predicted instability indices were greater in striatum than in cerebellum, but did not distinguish mutant from wild-type striatum. this finding suggested that mutant and wild-type striata have an equal propensity for somatic expansion that is unrelated to the hd cag pathogenic process. although wild-type striata possesses this propensity, the normal hd cag repeat does not actually expand because it does not present a sufficiently long target to be susceptible to the processes that mediate expansion.\n",
            "\n",
            "to test the prediction that somatic instability does not occur as a consequence of ongoing pathogenesis, we performed two genetic experiments. since the expanded hdh cag repeat is both a source of a pathogenic process and a target of instability, it is very difficult to delineate the relationship between the hd pathogenic process and somatic instability. therefore, we used genetic mouse models in which neurodegenerative processes are modulated or caused by factors independent of the hd cag repeat. we first investigated hdhq <dig> mice lacking the dopamine transporter , which show accelerated hd pathogenesis in the striatum  <cit> . as shown in figure 5b, striatal instability indices of hdhq92/+ dat-/- and hdhq92/+dat+/+ mice were not different, indicating that hd cag instability is not contributed by the disease process. we also tested whether inducing neurodegeneration in the cerebellum, a normally stable tissue, would cause instability in the cerebellum by crossing hdhq <dig> mice to harlequin  mice, a model of cerebellar granule cell degeneration  <cit> . as shown in figure 5c, hdhq111/+ hq/y mice and hdhq111/+ +/y control mice exhibited similar low cerebellar instability indices, indicating that neurodegeneration per se is insufficient to induce instability.\n",
            "\n",
            "taken together, these results support the prediction from our mathematical model, that the hd cag disease process is not responsible for the striatal specificity of hd cag repeat instability, arguing against the sequestration of dna repair proteins or other factors, as a contributor to somatic instability as previously suggested  <cit> . our results are also in agreement with similar levels of instability seen in knock-in and fragment transgenic models of hd that exhibit different rates of inclusion formation  <cit> , and with the observation that striatal instability occurs in sca <dig> and dm <dig>  although the striatum is not the target of pathogenesis in these disorders  <cit> .\n",
            "\n",
            "dna repair and repeat instability\n",
            "dna repair genes, particularly in the mismatch repair pathway, are required for somatic expansion of trinucleotide repeats  <cit>  and have previously been suggested as trans-acting tissue-specific factors responsible for tissue-specific somatic instability  <cit> . one possibility, therefore, is that dna repair gene expression levels are correlated with the levels of instability in tissues. our instability-associated gene expression signature gave us the opportunity to examine if expression levels of dna repair genes play a role in determining the tissue specificity of instability. thus, if dna repair gene expression levels were major determinants of the tissue specificity of somatic instability the expression levels of these genes would be predicted to correlate with instability levels across tissues. initial examination of the  <dig> probes comprising our instability-correlated gene expression signature did not highlight an important role for genes involved in dna repair in general . to probe these processes further, we examined whether expression levels of specific dna repair genes , previously shown to play important roles in cag repeat instability  <cit> , correlated with instability index measured in  <dig> tissues. the expression levels of msh <dig>  ogg <dig> and cbp did not correlate with instability index and msh <dig> expression level showed a weak negative correlation with instability index . in agreement with these findings, and further validating the predictive power of our signature, protein levels of msh <dig>  and cbp  did not correlate with instability index.\n",
            "\n",
            "selective neuronal expression of msh <dig> was recently proposed to contribute to the greater levels of instability in neurons compared to glia  <cit> , and therefore we explored this further. analyses of gene expression data revealed nearly identical msh <dig> expression levels in purified neurons and glia . together with the lack of correlation between instability index and msh <dig> expression levels across  <dig> tissues , the data argue against a major role for msh <dig> expression levels in determining tissue- or cell type-specific instability.\n",
            "\n",
            "therefore, our results suggest that although certain dna repair genes are absolutely critical for somatic instability  <cit> , their expression levels are unlikely to be the primary determinants of tissue specificity. clearly, posttranscriptional and/or posttranslational regulation of dna repair genes could still play a tissue-specific role. it would therefore be of further interest to determine whether there is a correlation between dna repair enzyme activities and tissue instability.\n",
            "\n",
            "genome-wide survey for pathways that correlate with tissue instability\n",
            "our findings indicate that neither hd pathogenesis, nor dna repair gene expression levels is a major determinant of the tissue specificity of somatic instability. we therefore sought to gain insight into the nature of the factors that might influence the tissue specific property of somatic instability. although, as a group, the genes comprising our signature are highly predictive of instability, on an individual gene basis, they do not have sufficient power to predict instability-related biological pathways. therefore, as an alternative strategy, we carried out a sensitive, unbiased and global gene set enrichment analysis , in which gene expression data is analyzed at a the level of biological pathways rather than individual genes  <cit> . confirming the findings above, dna metabolism pathways were not significantly correlated with instability index . in contrast, pathways broadly in the cell cycle category were negatively correlated, whereas pathways related to neurotransmitter activity  and cellular metabolism  were positively correlated with tissue instability index . confirming previous predictions, the negative correlation of cell cycle pathways with instability index is consistent with the instability seen in many tissues of the nervous system , its occurrence in postmitotic neurons  <cit>  and a dissociation of instability and cell division rate  <cit> . our results also suggest novel roles for additional pathways  in determining the tissue specificity of somatic instability.\n",
            "\n",
            "gene set enrichment analysis was performed using pearson correlation between expression level and instability index as a ranking metric. significant gene sets were identified by permutation-based nominal p value . nes, normalized enrichment score.\n",
            "\n",
            "it is possible that as striatum is particularly unstable, the highly correlated pathways are simply those that are predominantly present or absent in this tissue, and that the correlation with instability is coincidental. however, pathways significantly up-regulated or down-regulated in striatum compared to cerebellum  showed little overlap with those that correlated with instability; for example, the dopamine pathway is strongly up-regulated in striatum, but does not correlate with instability. this suggests that the instability-correlated pathways are directly related to instability rather than simply being striatal-specific.\n",
            "\n",
            "test of prediction from gsea\n",
            "instability-correlated pathways may either directly modify instability or may represent cells' secondary responses to instability. to distinguish these alternatives, we asked whether alteration of an instability-correlated pathway would influence instability. cell cycle pathways were negatively correlated with instability index , and our instability prediction in tissues  indicated intermediate levels of instability in many areas of the nervous system. this suggested that instability might be associated with the lack of mitotic activity in these tissues due to their high proportion of non-proliferating cells. therefore, we tested directly whether cell cycle block would result in increased instability. to test this hypothesis, we took advantage of a clonal striatal cell line  derived from striatal primordia of hdhq111/+ e <dig> embryos  <cit> . these cells divide at 33°c due to immortalization by the temperature sensitive sv <dig> large t antigen, but stop proliferating at 39°c due to the degradation of sv <dig> large t. we therefore compared the instability index of cells in cycling  and non-cycling  conditions over  <dig> weeks. as shown in figure  <dig>  the instability index increased over time only when the cells stopped cycling , but not when cells continuously proliferated , consistent with the prediction from the negative correlation between cell cycle and the instability index. it is notable that the hd cag repeat in sthdhq <dig> cells is extremely stable over multiple passages and under numerous different experimental conditions . cell cycle arrest is the only condition we have identified so far that has resulted in any expansion of the repeat. these findings indicate that the negative correlation of cell cycle pathways with the instability index more likely reflects a contribution of cell proliferation to preventing instability rather than a reduction of these pathways as a consequence of instability.\n",
            "\n",
            "contribution of multiple processes to somatic instability\n",
            "although cell cycle pathways may be directly involved in modifying instability, some tissues  with a high proportion of non-proliferating cells were relatively stable. this indicated that each correlated pathway may explain a small part of the tissue instability and that the contributions of each pathway may be different for each tissue. therefore, to investigate further the contributions of the different instability-correlated pathways, we compared the expression levels across different tissues of genes in the two most strongly correlated pathways . interestingly, although 'udp-galactose beta-n-acetylglucosamine beta- <dig> -galactosyltransferase activity' was the most significantly correlated pathway , liver which had a high instability index  showed a low level of gene expression in this pathway . in addition, similar levels of gene expression in the 'adrenoceptor activity' pathway, the second most significantly correlated pathway , occurred in hippocampus, cerebral cortex and striatum, with low , intermediate  and high  instability indices, respectively . these results indicate that no single pathway can fully explain tissue-specific instability, strongly implying that somatic instability requires multiple processes that may be different in different tissues.\n",
            "\n",
            "discussion\n",
            "we have developed a novel approach for use in investigations of tissue-specific somatic hd cag repeat instability that combines a reliable, high-throughput method for quantifying somatic instability with mathematical modeling based on gene expression data. predictions based on our modeling were confirmed using genetic, biochemical and cell culture-based experiments, indicating the validity of our bioinformatics approach.\n",
            "\n",
            "it has been proposed that somatic instability may be a consequence of disease pathogenesis  <cit> , potentially explaining the striatal specificity of somatic expansion in hd. our results directly demonstrate that hd pathogenesis does not explain the tissue specificity of hd cag instability. in addition, dna repair proteins have been found to be essential factors for somatic instability of trinucleotide repeats  <cit> . however, here we demonstrate that differences in expression levels of dna repair genes do not underlie the tissue-specific differences in hd cag instability. in addition, hdh expression levels did not correlate with instability index in tissues , confirming observations that although transcription through expanded repeats may be important in somatic instability  <cit> , tissue-specific patterns are not reflected in the steady state levels of hdh mrna. alternatively, our study suggests new pathways, notably metabolism, neurotransmitter, and cell cycle that may contribute, in combination, to the level of somatic instability in different tissues, providing a starting point to identify additional factors that contribute to somatic instability. notably, there was no predominant factor that could explain the tissue-specificity of hd cag instability, suggesting that patterns of instability are determined by the combined effects/interactions of many genes.\n",
            "\n",
            "somatic instability of trinucleotide repeats not only requires trans-acting factors, but has also been shown to depend on cis-acting sequences  <cit> . thus, while certain tissues are more predisposed to somatic expansion, the expandability of a particular repeat in a particular tissue is further modified by its context. this could at least in part explain differences in the precise tissue patterns of somatic expansion in different diseases  <cit> . it would therefore be of interest to determine instability-correlated gene expression signatures and instability-correlated biological pathways for other trinucleotide repeat diseases. instability-correlated genes/pathways that are shared between diseases would provide further insight into fundamental aspects of tissue-specific instability.\n",
            "\n",
            "our bioinformatics method based on gene expression data can only address aspects of tissue instability that are related to steady-state mrna levels. in principle, however, a similar bioinformatic approach could be also applied to proteomics data. irrespective of the particular method however, the strength of our approach is in its high-throughput, global and predictive nature, facilitating a number of important applications. our genemapper quantification method is readily applicable to high-throughput assays such as screening small molecules that modulate instability in cells, or screening for genetic modifiers in mice. a powerful application of our bioinformatics approach is that the instability-correlated gene expression signature can be used as a surrogate marker for instability in situations where repeat instability cannot be directly measured. for example, gene expression databases can be screened to identify cell or tissue states that have the propensity for somatic instability, even in the absence of an expanded cag repeat target as a read-out. similarly, databases can be screened for compounds that reduce the instability propensity. together, these approaches promise to accelerate the discovery of drugs that modulate instability and that are therefore candidate modifiers of disease.\n",
            "\n",
            "CONCLUSIONS\n",
            "our study demonstrates that multiple tissue factors including metabolism, neurotransmitter, and cell cycle combine to reflect the level of somatic instability in different tissues. our findings also indicate that dna repair proteins act largely in a non tissue-specific manner. in addition, the combination of our instability quantification method and mathematical modeling is a powerful strategy that has allowed us, in an unbiased manner, to gain critical new insights into the tissue specificity of trinucleotide repeat instability in hd. it opens the door to widespread downstream applications with the potential to make significant advances in novel avenues for therapeutic intervention in both huntington's disease and trinucleotide expansion disorders in general.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "we have developed a novel, high-throughput method for quantifying somatic instability that combines a reliable, high-throughput method for quantifying somatic instability with mathematical modeling based on gene expression data.\n",
            "our results suggest that somatic instability occurs as a consequence of ongoing pathogenesis, and suggest novel pathways that may be important in determining the tissue specificity of instability in hd.\n",
            "we have developed a novel, high-throughput method for quantifying somatic instability in hd.\n",
            "our results suggest that somatic instability does not occur as a consequence of disease pathogenesis, suggesting that patterns of instability are determined by the combined effects/interactions of many genes.\n",
            "\n",
            "16\n",
            "[{'summary_text': 'we applied a pipeline of statistical analyses to compare the sequences of  <dig> publicly available genomes of e. coli from different phylogroups  and ecological backgrounds.\\nwe found that homologous recombination happened more often in the middle of core regions at a small but highly significant level, and that homologous recombination happened more often in the middle of core regions at a small but highly significant level.\\nwe also found that homologous recombination happened more often in the middle of core regions at a small but highly significant level, and that homologous recombination happened less often in the middle of core regions at a small but highly significant level.\\nin this study, we applied a pipeline of statistical analyses to compare the sequences of  <dig> publicly available genomes of e. coli from different phylogroups  and ecological backgrounds.\\nwe found that homologous recombination happened more often in the middle of core regions at a small but highly significant level, and that homologous recombination happened less often in the middle of core regions at a small but highly significant level.\\nwe found that homologous recombination happened more often in the middle of core regions at a small but highly significant level, and that homologous recombination happened less often in the middle of core regions'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "recombination is a fundamental process of bacterial evolution, capable of influencing the integrity of species  <cit> . two types of recombination are typically distinguished: homologous recombination, where a fragment of a genome is replaced by the corresponding sequence from another genome  <cit> , and non-homologous recombination, which causes genetic additions of new material and is also called lateral gene transfer   <cit> . these two types of recombination may in fact often happen simultaneously, but they are usually studied separately because of the very different signatures they produce on the genomic sequences. both homologous and non-homologous types of recombination are key elements of the evolution of bacteria and can be linked to variations in fitness, and thus ecologies and lifestyles. there is indeed an ecological component in bacterial recombination, in the sense that bacteria with overlapping living environments, reservoirs or hosts  will have more opportunities for genetic exchange than species or lineages living in drastically distinct environments. recombination is therefore clearly conditioned by ecology, but conversely it is probable that recombination often drives ecological changes, for example by allowing favourable innovations to be exchanged by separate lineages adapting to a same lifestyle  <cit> .\n",
            "\n",
            "escherichia coli is a good example of an environmentally versatile and adaptable bacterial species. it encompasses some strains able to live commensally with their host and others causing a relatively wide variety of disease symptoms, from diarrhoea or renal failure to meningitis  <cit> . on top of this commensal versus pathogen duality, which may not represent a strict categorization, e. coli can be found in a wide range of hosts, as well as secondary non-host environments such as water, soils or plants  <cit> , in which it sometimes seems to maintain very well  <cit> . at the phylogenetic level, this plasticity is somewhat reflected by the population structure of e. coli, which is characterised by the presence of distinct phylogenetic groups  observable by phylogenetic reconstruction  <cit>  or the use of specific markers  <cit> . four major  and two minor  phylogroups have so far been described  <cit> . judging from the non-random isolation frequencies of different phylogroups in various hosts and environments  <cit> , it seems that the fitness in different environments varies among e. coli isolates from different phylogroups, which raises the question of the evolutionary nature of these phylogroups. are they the present reflection of e. coli subgroups undergoing speciation as a consequence of slightly variable ecologies? or, the primary environment of any e. coli being the gastrointestinal tract of endotherms, is there a relative cohesion of these phylogroups within the e. coli species after all? an indirect but efficient method to answer these questions is to look at the patterns of recombination  between different strains and members of the different phylogroups. as mentioned above, recombination should be conditioned by existing ecological differences between lineages, and may even be partly responsible for them in which case this approach also has the potential to identify the genes that play a key role in the adaptation.\n",
            "\n",
            "in this study, we contribute to the understanding of the association between genomic evolution and ecological adaptation by presenting bioinformatic analyses of recombination events  between  <dig> publicly available genomes of e. coli from different phylogroups  and ecological backgrounds . more generally, our extensive knowledge about e. coli compared to other microbial species provides a unique opportunity to study the mechanisms of genomic evolution in its biological context. we used a genomic analytical pipeline  which combined progressivemauve  <cit>  for aligning the genomes, clonalframe  <cit>  to establish their clonal relationships with one another, genoplast  <cit>  to study non-homologous recombination and clonalorigin  <cit>  to examine homologous recombination.\n",
            "\n",
            "methods\n",
            "genome sequences\n",
            "a total of  <dig> genomes of e. coli were available from the ncbi reference sequence database  <cit>  when this study was initiated. three of these genomes  were described as members of phylogroup d but did not cluster together in our preliminary phylogenetic analysis . furthermore, these three genomes showed evidence of deviation in the molecular clock rate which could have confused the analyses presented here since the models in clonalframe  <cit>  and clonalorigin  <cit>  assume a constant clock rate . these three genomes were therefore excluded so that we were left with a set of  <dig> genomes which is summarized in table  <dig>  several more genomes have recently become available on ncbi, but the complex analytical pipeline we used  could not easily accommodate them.\n",
            "\n",
            "multi-locus sequence typing data\n",
            "to assess the representativeness of the  <dig> strains included in this study, we compared them with the isolates from the e. coli reference collection   <cit>  which have been characterized by two independent multi-locus sequence typing  <cit>  schemes. fragments of 450-550bp from seven housekeeping genes  have been sequenced previously for a total concatenated length of 3423bp  <cit> . additional fragments of 450-600bp from eight genes  have subsequently been sequenced for a total concatenated length of 4095bp  <cit> . to achieve maximum robustness, we combined the data from both studies to obtain 7518bp of sequence from each isolate. blast  <cit>  was used to extract the sequences of each of the  <dig> gene fragments from each of the  <dig> genomes. a upgma dendrogram was then constructed to illustrate the phylogenetic relationship between the genomes and the ecor collection .\n",
            "\n",
            "analysis of genomic content\n",
            "the genomes of the  <dig> strains in table  <dig> were aligned using progressivemauve  <cit> . progressivemauve does not use annotations to guide the alignment. consequently, when there are multiple copies of a gene in the genome, progressivemauve will usually align the copy that fits best in the context of surrounding sequence, unless the identity to a sequence in a different context scores so much better that it exceeds the breakpoint penalty. in general this will have the effect of aligning orthologous copies of genes unless the gene conversion rate among paralogs is very high. the resulting alignment contained  <dig> locally colinear blocks . for all subsets of the genomes with cardinality ranging from  <dig> to  <dig>  the concatenated size of the homologous regions found in all or a fraction of the subset was counted directly from the output of progressivemauve. these values were used to generate figure  <dig>  furthermore, for each pair of strains, a pairwise distance was computed representing the proportion of genome content that they have in common. this matrix of pairwise distances was then used to build the upgma tree in figure 4b. the cophenetic correlation coefficient  <cit>  for this tree was  <dig>  indicating that it is a fairly good representation of the differences in genomic content between the genomes.\n",
            "\n",
            "from the complete alignment of the  <dig> genomes, a matrix of feature presence/absence was computed using the bbfilter script distributed with mauve, where each feature represented 50bp of unique sequence. this data was analyzed using genoplast  <cit>  which infers how the genomic composition of the genomes evolved on the branches of the clonal genealogy  assuming a model in which gain and loss of genetic material follow a relaxed molecular clock  <cit> . briefly, genoplast explores the space of gain and loss events happening on branches that are compatible with the observed patterns of sharing of genomic regions observed in the genomes at the leaves of the tree. genoplast was run for  <dig> , <dig> iterations with the first half discarded as burn-in. good convergence and mixing properties were found by comparing different runs. the results of the genoplast analysis are shown in figure  <dig> \n",
            "\n",
            "reconstruction of clonal genealogy\n",
            "all regions of at least 500bp found in all  <dig> genomes were extracted from the progressivemauve output using the stripsubsetlcbs script distributed with mauve. a total of  <dig> such regions were found, ranging in size from 501bp to  <dig> bp with a mean of 4322bp and a concatenated length of  <dig> mbp. these regions found in the  <dig> genomes represent the core-genome of e. coli . we applied clonalframe  <cit>  to this core-genome in order to reconstruct the clonal relationships between the genomes. clonalframe is a bayesian phylogenetic method which performs inference under an evolutionary model accounting for the effect of homologous recombination  <cit> . five runs of clonalframe were performed independently each consisting of  <dig>  iterations, the first half of which was discarded as burn-in. the results were compared between runs and found to be highly similar, indicating good convergence and mixing properties. the clonal genealogy inferred by clonalframe is shown in figure 4a. the analyses of homologous and non-homologous recombination described below were performed conditionally on this clonal genealogy. consequently, the fact that some genomes are more closely related to one another than others is fully accounted for in these analyses.\n",
            "\n",
            "analysis of homologous recombination\n",
            "in order to further analyse the role played by homologous recombination during the diversification of e. coli from a common ancestor, we applied the computer software clonalorigin  <cit>  which performs approximate inference under the coalescent with gene-conversion model  <cit> . clonalorigin detects recombination events, including their origin and destination on the clonal genealogy, and can therefore be used to reconstruct trends and patterns of homologous recombination  <cit> . the clonalorigin model rests on three global parameters which are the average length of recombination events δand the scaled rates of mutation and recombination events respectively equal to θs=2neμs and ρs=2ner where ne is the effective population size, μis the per site per generation mutation frequency and r is the per site per generation recombination frequency. a first run of clonalorigin was performed for each of the  <dig> core regions where each region independently infers the three parameters . the median values of the three parameters across all regions were as follows: δ=542bp, θs= <dig>  and ρs= <dig> . clonalorigin was then rerun for each region with the three parameters set equal to these estimates . in both steps, clonalorigin was run for  <dig> , <dig> iterations, the first half of which was discarded as burn-in.\n",
            "\n",
            "step  <dig> was only used to infer the values of the three global parameters, and all results presented here are based on the step  <dig> results from clonalorigin. for instance, figure  <dig> represents the number of recombination boundaries found in each of the  <dig> regions, with three hotspots  highlighted in grey. figures  <dig> and  <dig> compare the number of inferred recombination events between different parts of the genealogy with the number expected under the prior model. these two figures are based on the numbers of the observed and expected recombination events computed by clonalorigin for all pairs of potential donor and recipient branches of clonal genealogy. these values are compiled in additional file 2: table s <dig> \n",
            "\n",
            "RESULTS\n",
            "representativeness of the strains used in this study\n",
            "this study included  <dig> previously sequenced genomes of escherichia coli . to assess how representative these genomes are of the global diversity of the species, we compared them to the escherichia coli reference collection   <cit>  on the basis of two multi-locus sequence typing schemes which together spanned a total of  <dig> genes  <cit> . the resulting phylogeny  highlighted the six previously described lineages of e. coli, designated a, b <dig>  b <dig>  e, d and f  <cit> . overall, the  <dig> strains covered much of the diversity of e. coli, with eight strains in clade a, seven in clade b <dig>  five in clade e and seven in clade b <dig> . in each of these four clades, the strains seem to represent much of the within-clade diversity rather than being closely related within the clade. however, two clades were not represented in this genomic panel: clade d and clade f. three genomes from these phylogroups  were initially intended to be included, but were removed because they showed evidence for significant deviation from the assumption of a fixed molecular clock . figure  <dig> indicates how the diversity of the genomes in this study relates with that of the ecor strains, however, it should be noted that the issue of biased sampling of bacterial isolates is frequent and it is never possible to be sure of the representativeness of a sample  <cit> .\n",
            "\n",
            "reconstruction of the clonal genealogy\n",
            "aligning the  <dig> genomes using progressivemauve  <cit>  allowed us to compare their genomic content. as more genomes are considered in the analysis, the cumulative size of genomic regions shared by them decreased down to about  <dig> mbp, or about two thirds of each genome . since this length is roughly constant whether  <dig>   <dig>   <dig> or all  <dig> genomes are aligned , these regions are likely to represent the core-genome of e. coli, which means that homologs of these regions would be found in virtually any sequenced genome. we found  <dig> core regions present in the genomes of all  <dig> strains, with total length  <dig> ,899bp. these core regions were input into clonalframe  <cit>  in order to estimate the clonal genealogy in a way that accounts for homologous recombination which can confuse the signal of clonal inheritance  <cit> . this aspect is important because recombination has been reported to be frequent in e. coli by a large number of previous studies  <cit> . the inferred clonal genealogy  consisted of four clades corresponding to a, b <dig>  b <dig> and e. the relationships between genomes within clades were fully resolved, which is typically not achievable with mlst . the relationships between clades were unbalanced, with clade a and b <dig> most closely related to each other, and clade b <dig> most distant from any other clade. the stemminess  of this tree was compatible with expectation under the standard coalescent model , suggesting no evidence for population size variation during the evolution of e. coli <cit> .\n",
            "\n",
            "analysis of the dispensable genome\n",
            "in contrast to the core regions described above, non-core regions are found only in a strict subset of the genomes. the set of non-core regions is called the dispensable genome and together with the core genome forms the pan-genome  <cit> . the cumulative length of the non-core regions continues to increase up to the 27th genome, showing no sign of flattening, with each new genome adding about 250kbp of previously unobserved sequence . this distribution has been observed before, including in e. coli, and its pan-genome has consequently been called “open”  <cit> . however, an important difference between these previous studies and ours is that in figure  <dig> the lengths of genomic material are measured directly whereas previous studies counted the number of genes. our analysis is therefore robust to the problem of identifying homologous families of genes. nevertheless, this result indicates that the pan-genome of species with a high diversity and ecological plasticity such as e. coli draws from a large repertoire of genes that can be gained and lost through lateral gene transfer  <cit> .\n",
            "\n",
            "the similarity of the genomes in terms of genomic content was calculated from the patterns of presence and absence of non-core regions . compared with the clonal genealogy , the clade structure is only partly preserved in this tree of genome content, with clades b <dig> and e intact but clades a and b <dig> intermingled. clade b <dig> was split into three parts which were perfectly congruent with pathotypes. the three ehec strains  <dig>   <dig> and  <dig>  <cit>  formed one separate cluster. the two commensal strains iai <dig>  <cit>  and se <dig>  <cit>  and the only etec strain e24377a  <cit>  constituted another separate cluster, in which the two commensal strains were closest to each other. finally, the eaec strain  <dig>  <cit>  was on a separate branch in spite of its close relationship with the commensal strains iai <dig> and se <dig> in the clonal genealogy . this subdivision of b <dig> in terms of genomic content has been partially hinted at before  <cit>  and the fact that it is congruent with pathotypes suggests that it is linked with differences in ecological and pathogenic lifestyles. the presence or absence of genomic regions in the  <dig> observed genomes is the result of a process of gain and loss of content by the ancestors of the genomes since their evolution from a common ancestor. if gain and loss happened randomly and at constant rates, the tree based on genomic content  would be expected to to be very similar to the tree based on homology of the core-genome  since the evolution of both core and pan genomes would then follow the same molecular clock. the two trees were however highly different, indicating that the non-homologous recombination process  did not follow a strict molecular clock. genoplast  <cit>  was used to infer the non-homologous recombination events that happened in the context of the clonal genealogy inferred by clonalframe  under a model where the rates of gain and loss are allowed to change. the results of the genoplast analysis are shown in figure  <dig>  with differences in the rates of gain and loss on specific branches spanning two orders of magnitude. the rates of gain and  loss of genomic material were found to be higher on the short recent branches within clades a, e and b <dig> than on older and longer branches, which explained the higher stemminess of the genomic content tree  compared with the clonal genealogy .\n",
            "\n",
            "the branch directly above ehec strain  <dig> had the largest amount of gain of any branch  whereas the branch above the common ancestor of the other two b <dig> ehec strains  <dig> and  <dig> was the highest amount of gain for an internal branch . amongst the genomic material gained on these two branches,  <dig> kbp were shared by the three genomes, which explained why they clustered together in figure 4b. the distribution of this gain on the three genomes  indicated that their convergence in genomic content happened as a result of multiple gain events that happened both on the branch above  <dig> and on the branch above the common ancestor of  <dig> and  <dig>  the convergence in genomic content of the three ehec b <dig> strains was therefore reciprocal rather than unidirectional. few convergence events were found on the branches directly above  <dig> and  <dig>  in spite of considerable gain on these branches , which could indicate that the convergence in gene content with  <dig> is not on-going. unsurprisingly, this convergence involved several genes known to be ehec determinants, including shiga toxins  <cit>  and all genes from the locus of enterocyte effacement . however, it also included additional genes, such as flagellar genes  and a few metabolic clusters  with a notable presence of genes involved in aromatic compounds metabolism . these genes were not present in the other b <dig> strains examined in this study, which may indicate that acquiring ehec determinants via hgt is an important means of e. coli adaptation, possibly enhanced by the differences in host-associated selective pressures on ehec compared to commensals or more opportunistic pathotypes.\n",
            "\n",
            "homologous recombination hotspots in escherichia coli\n",
            "to quantify the propensity, genomic distribution and directionality of homologous recombination during the evolution of e. coli, we applied clonalorigin  <cit>  to the  <dig> core regions and assuming the clonal relationships between genomes estimated by clonalframe  <cit>  in figure 4a. the average length of fragments involved in homologous recombination was estimated at δ=542bp. this is almost ten times higher than a previous estimate in e. coli <cit> , but is of the same order as recent whole-genome estimates in bacillus cereus <cit> , helicobacter pylori <cit>  or chlamydia trachomatis <cit> . the relative rate of occurrence of recombination and mutation  <cit>  was estimated at ρs/θs= <dig> / <dig> = <dig>  which means that overall recombination happened just as frequently as mutation. the estimated rate of homologous recombination was fairly constant throughout the genome , with the exception of three clear hotspots  in which recombination rates were significantly higher. this included two large regions around the rfb operon involved in synthesis of the o antigen  and around the fima gene . these two regions had been reported previously as hotspots of diversity and recombination  <cit> .\n",
            "\n",
            "a smaller recombination hotspot was also detected, made of just two nearly adjacent core regions . this region had a similarly high recombination rate as the two regions above, but had not previously been detected as a hotspot, perhaps because of its small size . this hotspot contained genes yfclyfcmyfcamepaarocprmb and smrb. the gene mepa encodes for a murein endopeptidase  <cit>  whose role is presumably to restructure the bacterial cell wall during elongation or stabilise the peptidoglycan. mutational analyses on mepa <cit>  do not provide enough information to explain why recombination should be high for this gene. in the bacterial cell, aroc governs the synthesis of chorismate, a key precursor to the biosynthesis of aromatic compounds including the amino acids tryptophan and phenylalanine but also the siderophore enterobactin. the positive maintenance of a functional allele of aroc is arguably crucial for the cell to maintain appropriate levels of these amino acids and siderophores in natural conditions. in salmonella <cit> , as well as in brucella suis <cit> , aroc is required for virulence. incidentally, aroc is a common target to produce knocked-out attenuated vaccine strains  <cit> , for instance in salmonella serovars typhi  <cit>  and typhimurium  <cit> , pathogenic e. coli <cit> , brucella suis <cit> , burkholderia pseudomallei <cit>  or edwardsiella tarda <cit> . to our knowledge, this is the first mention of aroc being part of a recombination hotspot, giving additional clues on evolutionary dynamics at this locus. depending on how aroc is involved with virulence in e. coli, it may be under selective pressure from the immune system of the host, which could explain the observed peak in recombination rate  <cit> , but this hypothesis will need further work to be fully assessed.\n",
            "\n",
            "flux of homologous recombination\n",
            "the numbers of recombination events inferred by clonalorigin were counted for every combination of clades receiving and donating, and these values were compared with their expectation under the clonalorigin model which represents a close approximation to the coalescent model with gene conversion  <cit> . this comparison revealed significant non-uniformity in the homologous recombination flux within and between clades . the three clades a, b <dig> and b <dig> had higher numbers of within-clade recombination than expected, whereas clade e had almost exactly the expected number. on the other hand, the number of recombination events detected between clades was almost systematically below its expected value, with the only exception being recombination from clade a to b <dig> and vice-versa which had slightly higher than expected values. clades a and b <dig> are the two most closely related phylogroups  which may contribute to explain this observation. overall, inter-phylogroup recombination fluxes were lower than intra-phylogroup ones, which is compatible with the hypothesis that there is a preferred way of gene sharing within phylogroups  <cit> . this preferred exchange among strains of the same phylogroups could be explained by the possibility that the different e. coli phylogroups have slightly distinct ecological overlaps, which makes the likelihood of gene transfer higher among them than between them.\n",
            "\n",
            "a similar analysis as above was performed on a branch-by-branch basis rather than a clade-by-clade basis , the only added difficulty being that some donor/recipient pairs of branches have too low numbers of expected and observed recombination events for the comparison to be meaningful .this analysis confirmed the general pattern described above, with more recombination than expected within-clades and between a and b <dig>  and less recombination between all other clades. however, it also allowed the comparison of the individual behaviours of strains belonging to the same clade. for instance, strains bl <dig> and rel <dig>  <cit>  showed little history of importing recombination from clade b <dig>  contrasting with atcc <dig>  <cit>  or hs  <cit>  even though all four strains belong to clade a. this may be explained by the fact that these two strains are laboratory-adapted derived from e. coli strain b  <cit> , so that they would have had little or no opportunity for recent encounter and recombination with b <dig> strains.the four k- <dig> strains in this study  <cit>  were also laboratory-adapted, but had terminal branches too small to reliably estimate deviations in the number of recombination events. these four strains all originated from bioengineering manipulation on k- <dig> lineages over the last century and therefore harbour a very limited number of differences between them compared to what would be observed in natural populations.\n",
            "\n",
            "the b <dig> strains  <dig> and  <dig>  <cit>  showed significantly less sign of import from clade a  than other strains of b <dig>  this observation implies that these ehec strains have stopped recombining with strains of clade a  as they adapted to this new pathogenic lifestyle. two of the highest values throughout figure  <dig> were the ones corresponding to imports from strains  <dig> and  <dig> into strain  <dig>  <cit> . as previously noted, these are the only three ehec strains in b <dig>  and these three genomes have been converging in genomic content due to numerous non-homologous recombination events. this result indicates that the three genomes also have an extensive history of convergence through homologous recombination, which may have occurred at the same time as the gain of new shared genes. the evolutionary history of these three genomes seems therefore analogous to that of salmonella serovars typhi and paratyphi a, for which both core and pan genomes converged through recombination as they were progressively adapting to exclusive infection of the human host  <cit> .\n",
            "\n",
            "speciation in e. coli\n",
            "in the analysis of homologous recombination described above, three groups corresponding respectively to phylogroups e, b <dig>  and a+b <dig> exhibited more recombination within than between one another . this pattern is compatible with a definition of speciation in bacteria in which recombination plays the role of a cohesive force counterbalancing divergence by genetic drift and population structure, and where species appear when this force is weakened between lineages  <cit> . under such a model, patterns of genetic diversity can be generated in silico similar to those observed for example in salmonella enterica <cit> . the three groups might therefore represent lineages that, because of slightly distinct ecologies or notable variations in the species life cycle, have gradually diverged too far from one another for recombination to play its cohesive role, so that they might eventually become separate species, should these variations remain or increase. in other words, all e. coli phylogenetic backgrounds are found in the gut of endotherms  <cit>  which is their primary environment, and to some extent in nonhost secondary environments  <cit>  but it sounds plausible that phylogroup-associated variations in ecological fitness in different hosts or secondary environments could gradually decrease the physical and ecological overlap of strains from different phylogroups through time, and therefore the genetic flux between them. a number of studies seem to support this hypothesis, as different proportions of the different phylogroups are found in different environments and hosts  <cit> . additionally, some phylogroups seem to harbour strains that have either host-restricted or more generalist lifestyles  <cit> , as well as strains that are either resident or transients in their ability to colonize the gut  <cit> . our study contributes to highlight that such variations in ecology could potentially have an impact on genetic exchange in e. coli.\n",
            "\n",
            "an additional number of factors can be evoked to explain why the three groups would be diverging, including differences in their geographic distribution, adaptative selection, or simply as a result of the dependence of recombination on homology between donor and recipient  <cit> . the three groups are clearly separate in terms of genomic content  which could explain why they recombine less with each other and why clades a and b <dig> still recombine frequently since they are not differentiated in terms of genomic content. to test this hypothesis, we compared the distribution of the number of recombination events found in the middle and at the edge of core regions . we found that recombination happened more often in the middle of core regions at a small but highly significant level . if the variable genomic content is not just a random process, then homologous recombination would be expected to happen less often around these genes, a concept sometimes called fragmented speciation or “species in pieces”  <cit>  as it would predict that speciation can apply differentially across the genome. our results therefore demonstrate that fragmented speciation applies to e. coli, and that difference in genomic content is at least one of the factors driving the divergence of the three lineages.\n",
            "\n",
            "CONCLUSIONS\n",
            "we applied a pipeline of statistical analyses in order to compare the sequences of  <dig> e. coli genomes and reveal the ancestral history of clonal relationships, homologous recombination events and non-homologous recombination events that has led the ancestor of e. coli to diversify into the genomes we see today. the overall picture was one of divergence between three lineages  which were well differentiated on the basis of both genomic content and preference for homologous recombination, with the former apparently driving the latter as expected under a fragmented speciation scenario. however, against this divergence background, we observed the convergence of three ehec strains within b <dig> in both their core- and pan-genomes. these observations were correlated with the diversity of ecology and pathogenicity of the e. coli strains, and provide hypotheses for which genes and evolutionary processes are adaptively important.\n",
            "\n",
            "competing interests\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "authors’ contributions\n",
            "xd, df and aed conceived and designed the study. xd, gm and aed analyzed the data. all authors contributed to the writing of the paper and approved the final manuscript.\n",
            "\n",
            "supplementary material\n",
            "additional file 1\n",
            "figure s <dig>  test of molecular clock assumption. neighbour-joining phylogenetic reconstruction based on all  <dig> genomes available from ncbi and which shows that three of them  showed significant deviation from the assumption of constant molecular clock.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 2\n",
            "table s <dig>  detailed results of the clonalorigin analysis. this table contains all expected and observed values of the number of recombination events for all pairs of donor and recipient branches, as computed by clonalorigin. this is the data on which figure  <dig> is based. there is a cell for each donor/recipient combination, and the cells are ordered vertically and horizontally in the same way as in . in each cell, two values are given separated by a semi-colon: the first one is the observed value and the second one is the expected value.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 3\n",
            "figure s <dig>  test of ancestral population size dynamics. distribution of expected values of stemminess under the coalescent model. the observed value for the clonal genealogy estimated by clonalframe is shown as a vertical line and falls within the expected values.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 4\n",
            "figure s <dig>  gain in the three genomes  <dig>   <dig> and  <dig>  the genomic regions gained by the three genomes  <dig>   <dig> and  <dig> are colored. the regions in red are the ones that are uniquely shared by the three genomes, whereas the regions in green are not. for genome  <dig>  only the gain happening on the branch directly above is shown. for genomes  <dig> and  <dig>  the gain on the branches directly above are shown using lighter green and red, and the gain that happened on the branch above the common ancestor of  <dig> and  <dig> is shown using darker green and red.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 5\n",
            "figure s <dig>  test of the fragmented speciation model. boxplots of the distributions of the numbers of recombination events found in the middle  and at the edge of core regions . to generate the distribution on the left, the number of recombination events affecting the middle position was counted for each of the  <dig> core regions. to generate the distribution on the right, the number of recombination events affecting the site 10bp after the beginning of each core region was counted, as well as the number of recombination events affecting the site 10bp before the end of each core region.\n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements\n",
            "xd was supported by a fellowship from the ukcrc modernising medical microbiology consortium, which is funded under the ukcrc translational infection research initiative supported by the wellcome trust  and by the medical research council, biotechnology and biological sciences research council and the national institute for health research on behalf of the department of health . aed was supported in part by national science foundation award er  <dig> \n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "we applied a pipeline of statistical analyses to compare the sequences of  <dig> publicly available genomes of e. coli from different phylogroups  and ecological backgrounds.\n",
            "we found that homologous recombination happened more often in the middle of core regions at a small but highly significant level, and that homologous recombination happened more often in the middle of core regions at a small but highly significant level.\n",
            "we also found that homologous recombination happened more often in the middle of core regions at a small but highly significant level, and that homologous recombination happened less often in the middle of core regions at a small but highly significant level.\n",
            "in this study, we applied a pipeline of statistical analyses to compare the sequences of  <dig> publicly available genomes of e. coli from different phylogroups  and ecological backgrounds.\n",
            "we found that homologous recombination happened more often in the middle of core regions at a small but highly significant level, and that homologous recombination happened less often in the middle of core regions at a small but highly significant level.\n",
            "we found that homologous recombination happened more often in the middle of core regions at a small but highly significant level, and that homologous recombination happened less often in the middle of core regions\n",
            "17\n",
            "[{'summary_text': 'the lupin genome contains  <dig>  clones with an average insert length of  <dig> \\u2009kb, resulting in 12× coverage.\\nthe lupin genome is partitioned into many small and morphologically similar chromosomes, resulting in increased yields of successive crops.\\nthe lupin genome contains  <dig>  clones with a average insert length of  <dig> \\u2009kb, resulting in 12× coverage.\\nthe lupin genome is partitioned into many small and morphologically similar chromosomes, resulting in increased yields of successive crops.\\nthe lupin genome contains  <dig>  clones with a average insert length of  <dig> \\u2009kb, resulting in 12× coverage.\\nthe lupin genome contains  <dig>  clones with a average insert length of  <dig> \\u2009kb, resulting in 12× coverage.\\nthe lupin genome contains  <dig>  clones with a average insert length of  <dig> \\u2009kb, resulting in 12× coverage.\\nthe lupin genome contains  <dig>  clones with a average insert length of  <dig> \\u2009kb, resulting in 12× coverage'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "narrow-leafed lupin  is a protein-rich grain crop for animal feed and human consumption that has excellent potential for sustainable crop rotation systems. the most important advantage of lupin cultivation is its contribution of fixed nitrogen and organic matter to soil, resulting in increased yields of successive crops  <cit> .\n",
            "\n",
            "during the last decade, several sets of molecular markers linked to various agronomic traits in narrow-leafed lupin have been developed using an innovative method, microsatellite-anchored fragment length polymorphism  analysis  <cit> . in general, mflp integrates amplified fragment length polymorphisms   <cit>  and simple sequence repeat -anchor primer techniques  <cit> . mflp gels generally have arbitrary patterns. however, a particular mflp marker may be converted into a sequence-tagged site  by excising a single band from the gel, cloning it into a vector, and sequencing the insert  <cit> .\n",
            "\n",
            "mflp-derived sts markers have been generated for l. angustifolius genes tightly related to seed germination, flowering, and yield. examples are mollis, which confers soft seediness   <cit> ; tardus and le, which reduce pod shattering   <cit> ; and ku, which removes the vernalization requirement   <cit> . furthermore, the mflp method was used to develop sets of markers tagging hypothetical genes involved in lupin resistance to pathogenic fungi, including lanr <dig>  conferring anthracnose resistance   <cit> ; phr <dig> and phr <dig>  carrying resistance to phomopsis stem blight, a lupin disease caused by the necrotrophic fungus diaporthe toxica   <cit> ; hua’an yang, unpublished; and a pair of markers, rustm <dig> and rustm <dig>  linked to rust-resistance loci  <cit> .\n",
            "\n",
            "current advances in molecular biology have considerably accelerated progress on lupin genomics. first, a linkage map based on mflp markers was constructed, and the regions carrying genes conferring valuable agronomic traits were localized  <cit> . this map provided general insight into lupin genomics, but the arbitrary nature of mflp markers did not permit comparative mapping between narrow-leafed lupin and other legumes. subsequently, a genetic map of a large set of gene-based pcr markers was drawn up. the use of sequence-specific markers enabled cross-species analyses, which resulted in the detection of conserved synteny between l. angustifolius and a legume model species, medicago truncatula <cit> . consequently, these two maps were joined and supplemented with new molecular markers to create a reference genetic map of l. angustifolius aligned to the genome sequence of the model legume lotus japonicus <cit> .\n",
            "\n",
            "comparative genomics studies between l. angustifolius and other species were not restricted to analyses of linkage maps, but also included dna-hybridization methods. screening of a narrow-leafed lupin cdna library with glycine max and arabidopsis thaliana gene-derived probes showed a high level of gene structure conservation among these species  <cit> . in general, a syntenic network within conserved gene-rich regions of g. max, m. truncatula, and a. thaliana was discovered  <cit> . a further significant step was the completion of the g. max genome sequence  <cit> . the assembling and annotation of the g. max, m. truncatula, and l. japonicus genomes opened rich opportunities for translational genomics. a review of advanced genomic resources developed for legumes was reported by sato et al.  <cit> . most genes in papilionoid species occur within relatively large syntenic regions  <cit> , facilitating cross-species gene annotation and positional cloning based on the sequenced genomes. however, the substantial genome rearrangements between medicago and lotus, which probably occurred during a polyploidy event, limited the syntenic conservation between the genistoid and millettioid clades  <cit>  and has hampered comparative genomic studies between lupin and bean or pea.\n",
            "\n",
            "the ability to screen the narrow-leafed lupin genome and discover genes was greatly enhanced by the development of new nuclear genome bac libraries for the polish cultivar ‘sonet’  <cit>  and for the australian cultivar ‘tanjil’  <cit> . the sonet bac library comprises  <dig>  clones with an average insert size of 100 kb, representing approximately six haploid-genome equivalents. the tanjil bac library contains  <dig>  clones with an average insert length of  <dig>  kb, resulting in 12× coverage.\n",
            "\n",
            "recently, lupins have become the subject of extensive cytogenetic studies. molecular cytogenetics have bridged the gap between the molecular and chromosomal levels of genome organization. extended studies on genome size have been performed on both old and new world species  <cit> . further approaches have included chromosome structure analyses in l. angustifolius by fish  with different kinds of molecular probes  <cit>  and prins   <cit> . the nuclear genome of l. angustifolius is partitioned into many small and morphologically similar chromosomes , so unambiguous chromosome identification without the use of molecular probes is not possible  <cit> . critical for cytogenetic analysis was the construction of a nuclear genome bac library for this species  <cit> , because large insert libraries constitute unique resources for molecular probes in fish-based studies. the bac-fish procedure proved valuable for physically mapping small plant genomes with numerous, tiny chromosomes, such as in phaseolus vulgaris <cit> . furthermore, for g. max, whose whole-genome sequence is available, bac-fish with genetically anchored bacs allowed identification of all  <dig> chromosome pairs and their correlation with genetic and sequence-based markers  <cit> . in lupins, as in other plants without completely sequenced genomes, integrating gene sequences with physical localizations of bacs in chromosomes is of special importance. few l. angustifolius linkage groups have been associated with appropriate chromosomes by the bac-fish procedure  <cit> .\n",
            "\n",
            "in the present study, we used an mflp-derived marker as a probe to screen a narrow-leafed lupin bac library. because all publicly available molecular markers linked to l. angustifolius agronomic traits were developed by mflp, we decided to verify whether such markers were useful for genome screening and, eventually, for positional cloning of particular genes. complex molecular methods, such as dna fingerprinting, bac-fish, and genetic mapping, were used to identify gene-rich regions of the lupin genome and to assign them to particular chromosomes and linkage groups. we exploited bac-fish to support contig construction, genetic mapping, and selection of repeat-free bac clones for sequencing. then, bac sequences were functionally annotated and comparative mapping was performed to identify synteny blocks between l. angustifolius and g. max gene-rich genome segments.\n",
            "\n",
            "RESULTS\n",
            "bac library screening and construction of contigs\n",
            "the hybridization probe was designed based on the mflp-derived phtjm <dig> marker sequence. marker phtjm <dig> is apparently linked with phr <dig>  repeatmasker  <cit>  analysis of the probe sequence , with a. thaliana as a reference, revealed a short  section of simple repeats. a blast  <cit>  search of the ncbi non-redundant  dna database  <cit>  identified a region of 75– <dig> nucleotides  within the probe with significant identity  to numerous mouse and human sequences. because of the central position of this non-specific region within the marker sequence, designing of a new probe without this section was impossible. the radioactively labeled probe was hybridized to a set of three macroarrays containing dna of clones from the l. angustifolius nuclear genome bac library. the screening procedure tagged  <dig> bac clones . the positive hybridization signals were verified by pcr with dna isolated from bac clones as a template. of the bac clones,  <dig> generated only one pcr band with phtjm <dig> primers, with no visible nonspecific amplification products on agarose gels. the remaining six bacs did not amplify and were recognized as false-positive hybridization signals.\n",
            "\n",
            "\n",
            "considering that the l. angustifolius bac library has a 6× genome coverage, the number of bacs selected by the probe was equivalent to more than  <dig> copies in the genome. therefore, mflp-derived markers are not suitable for positional cloning. did the bac clones selected by the probe cluster in one locus within the genome or did they originate from various genome regions? to address this question, a restriction fingerprinting approach was used. digestion of clones with two endonucleases  yielded, on average,  <dig> products per clone. products shorter than  <dig> nt were not counted. bac clones that produced similar band patterns were considered to overlap and were grouped into contigs. from the  <dig> bacs,  <dig> contigs carrying  <dig> clones were assembled, while  <dig> bacs remained unlinked . the largest contig consists of five bac clones.\n",
            "\n",
            "functional annotation of bac-end sequences\n",
            "the  <dig> bacs selected by the phtjm <dig> probe were subjected to bac-end sequencing, generating  <dig> bac-end sequences  with an average insert read length of  <dig> nt. seventeen bac clones could not be sequenced from either end, whereas sequencing failed from the 5′-end for six and from the 3′-end for four. bess were deposited in dna data bank of japan  under consecutive accession numbers .\n",
            "\n",
            "during the repeatmasker annotation,  <dig>  nt  were identified as transposable elements:  <dig> % ltr/copia and  <dig> % ltr/gypsy. simple repeats accounted for  <dig> % of the bes nucleotide collection. these proportions roughly corresponded to the recently published preliminary characterization of the narrow-leafed lupin genome  <cit> . the general percentage of repetitive content in bacs selected by the mflp-based probe was higher than in the approach based on the annotation of  <dig>  randomly selected bess   <cit> . in that preliminary lupin genomic survey, the most abundant transposable elements were ltr/copia  and ltr/gypsy . considering these widespread repeats, the results of mflp-based genome screening converged with those based on random bac selection.\n",
            "\n",
            "next, repetitive sections of bess were sequence-masked to identify hypothetical gene homologs. fourteen gene-like sequences were identified. the presence of three genes  was validated by statistically significant alignments with ncbi expressed sequence tag   <cit>  sequences from lupinus spp., glycine spp., lotus japonicus, and medicago spp. . notably, the annotated genes have very diverse functions in plants and participate in a wide range of biochemical processes. moreover, they may be expressed during physiological as well as pathological conditions. such a result indicates that hybridization with the mflp-derived probe picked up bac clones representing dispersed sections of the l. angustifolius genome rather than a narrow set of clones linked with lupin resistance to phomopsis stem blight.\n",
            "\n",
            "identification of bac clones giving single-locus fish signals\n",
            "physical mapping of the bac clones and functional annotation of the bess allowed the selection of particular clones for cytogenetic analysis. the set of clones chosen for bac-fish contained:  all  <dig> clones carrying gene-like sequences in their bess,   <dig> clones with no significant identity to any gene or repetitive sequence in both bess,   <dig> clones carrying repetitive elements in one bes, and  five clones containing repetitive elements in both bess. bacs with various bes characteristics were chosen to verify whether bes sequence data could predict clone appearance in bac-fish . of the  <dig> bacs tested,  <dig> clones originated from  <dig> contigs, and  <dig> clones were singletons.\n",
            "\n",
            "bac-fish was performed to select bac clones for full-insert sequencing. the bacs were used as molecular probes in the bac-fish procedure to directly visualize their chromosomal positions in the cytological preparation. this step of the analysis identified bac clones with unique single-locus signals. twenty-nine bacs were repetitive in bac-fish and eight had single-locus signals . no correlation between the results of bac-fish  and the type of bes  was observed among the clones analyzed. the presence of transposable element in bes did not necessarily imply a repetitive appearance of a particular clone in bac-fish. only a few of the analyzed bac clones hybridized at one locus and were thus suitable for cytogenetic mapping. a large number of bacs gave fish signals dispersed over many chromosomes, probably indicating the presence of repetitive sequences within their inserts. such a pattern of hybridization has been reported in a number of other plant genome studies  <cit> . the results of bac-fish with two clones with dispersed signals, 055l <dig> and 067h <dig>  are presented in figure  <dig>  molecular cytogenetics offers some techniques to overcome such an obstacle, including hybridization with the c0t repetitive-dna fraction or subcloning and selecting low-copy bac clones. however, these approaches are not effective in all cases  <cit> . in the present experiment, to establish a collection of bacs suitable for subsequent multibac-fish comparative analyses, it was important to select clones showing specific hybridization in the absence of any reagent directed against repetitive content.\n",
            "\n",
            "\n",
            "* linkage group assignment confirmed by bac-fish with two clones.\n",
            "\n",
            "bac clone sequencing and annotation\n",
            "based on the results of contig construction, bess annotation, and preliminary bac-fish experiments, five clones were chosen for complete 454-type sequencing. this set included two clones with bac-fish signals dispersed over numerous chromosomes: 055l <dig>  representing the largest contig, and 067h <dig>  classified as a singleton. clones behaving as repetitive in fish were assumed to contain a big fraction of transposable elements, whereas clones with unique cytogenetic signals were expected to possess more gene sequences. based on these assumptions, three single-locus bacs were sequenced: 004g <dig>  057j <dig>  and 080b <dig>  sequencing of all five bacs resulted in the assembling of two contigs for the 055l <dig> clone , four for 067h <dig> , two for 004g <dig> , four for 057j <dig> , and two for 080b <dig> . full bac sequences were assembled by joining contigs together. gaps were indicated by separators of  <dig> ambiguous nucleotides.\n",
            "\n",
            "as a result of repeatmasker analysis ,  <dig> % of the 055l <dig> and  <dig> % of the 067h <dig> bac sequences were determined to carry repetitive content. the main groups of repeats were retrotransposons representing ty1/copia for 055l <dig> and ty3/gypsy for 067h <dig>  clones with single-locus bac-fish signals contained a considerably lower percentage of repetitive content in their sequences. the fraction of interspersed repeats comprised  <dig> % of bac clone 004g <dig> and  <dig> % of 057j <dig>  within the sequence of 080b <dig>  no transposable element was identified.\n",
            "\n",
            "\n",
            "bac sequences with masked repetitive regions were subjected to fgenesh  <cit> in silico gene prediction followed by blast against the est, nr, and swiss-prot  <cit>  collections . in the structures of 055l <dig> and 067h <dig>  the bacs containing the highest ratios of repetitive content, several gene and gene-like sequences were identified. the most significant similarities were to glucan endo- <dig> -beta-glucosidase  <dig>  a gras-family transcription factor, a sialyltransferase-like protein, an auxin-induced protein 6b, and endonuclease v. in particular, for the bac 055l <dig>  which was the richest in transposable sequences,  <dig> % of fgenesh predictions had no coverage in the est collection. within the three single-locus bac clone sequences, large gene-rich regions  were identified. the majority of genes predicted by fgenesh in these bacs  were validated by statistically significant similarities to sequences from the est and ncbi protein nr collections. for  <dig> predicted genes , the sequence equivalents were also recognized in swiss-prot database . the schematic structures of the sequenced bac clones, showing repetitive elements and predicted genes, are presented in figure  <dig>  bac sequences with annotations were deposited in the european molecular biology laboratory  nucleotide sequence database under accession numbers 004g <dig> , 055l <dig> , 057j <dig> , 067h <dig> , and 080b <dig> .\n",
            "\n",
            "\n",
            "microsynteny between l. angustifolius and glycine max\n",
            "two large repeat-free gene-rich regions from bac clones 004g <dig> and 080b <dig> served as anchors for comparative dna sequence analysis between the genomes of l. angustifolius and g. max. strong and conserved synteny to g. max was identified for both grrs . the sequence of bac clone 004g <dig> had two strikingly similar regions on soybean chromosomes gm <dig> and gm <dig>  the high level of sequence conservation was illustrated both by the identical order of microsyntenic blocks in soybean chromosomes and by their concerted orientation. no syntenic sequence was reversed in this region, indicating that these two segments of the g. max genome were not involved in chromosomal rearrangements that changed the gene order. however, those sections of soybean chromosome were approximately twice as big as their narrow-leafed lupin equivalents.\n",
            "\n",
            "\n",
            "sequences homologous to l. angustifolius grr 080b <dig> were identified in soybean chromosomes gm <dig> and gm <dig>  no evidence of a rearrangement or reorientation was found. the organizational parallels to the soybean chromosomes were not as precise as with clone 004g <dig>  two regions of hypothetical insertions in g. max chromosomes were localized around the  <dig>  kb locus in gm <dig> and the  <dig>  kb locus in gm <dig>  because these loci are not homologous, the insertions must have occurred after the duplication of the soybean genome.\n",
            "\n",
            "the genome of glycine is characterized by a relatively recent duplication event that took place about  <dig> mya. much of the dna generated in that process should remain in the soybean genome. this implies that a given region of the medicago or lotus genome should correspond to two regions in glycine <cit> . our comparison of two lupinus whole bac clone sequences with glycine sequences showed the high extent of microsynteny between them. each lupin clone fit surprisingly well two soybean sequences, with no inversions evident.\n",
            "\n",
            "comparative genome analyses and the possibility of information transfer among species depend on their phylogenetic distances. lupinus angustifolius and the three legume models for which genome sequence information is available – g. max, m. truncatula and l. japonicus – belong to different phylogenetic clades. thus, none of the model species is closely related to l. angustifolius. we chose g. max as a reference model because it has the same number of small, morphologically-similar chromosomes . furthermore, gao et al.  <cit>  sequenced both ends of  <dig>  randomly selected bac clones from their l. angustifolius library and generated  <dig>  bess, covering approximately 1% of the genome. they found that the average percent identity of blastn alignments between l. angustifolius bess and g. max, m. truncatula, and l. japonicus was  <dig> %,  <dig> %, and  <dig> %, respectively. our results indicated that not only were the nucleotide sequences of genes conserved between the soybean and lupin genomes, but also that the order and orientation of genes in syntenic blocks were maintained.\n",
            "\n",
            "genetic mapping of selected bac clones\n",
            "all clones showing single-locus bac-fish signals were genetically mapped. to address this task, pcr primers were designed on the basis of bess originating from particular bacs. pcr products amplified from dna isolated from parental lines of the mapping population were polymorphic in the following six bess: 004g15_ <dig>  057j20_ <dig>  080b11_ <dig>  074i10_ <dig>  077c13_ <dig>  and 076k16_ <dig>  notably, the bes 074i10_ <dig> had two polymorphic pcr products, both exposing a dominant type of segregation. polymorphism was observed neither for amplicons tagging both ends of bac clones 057k <dig> and 083c <dig> nor for pcr products derived from bess 004g15_ <dig>  057j20_ <dig>  080b11_ <dig>  074i10_ <dig>  077c13_ <dig>  and 076k16_ <dig>  considering the quality of bess and the monomorphic nature of particular pcr products, new primers were designed to elongate the bess by sanger sequencing. after the first cycle of bes elongation, polymorphism was detected for the 3′ bes of clone 057k <dig>  which was renamed 057k22_3f <dig>  the second round of bes extension identified a polymorphic site in the sequence of 076k16_3f <dig>  no polymorphism was detected in products derived from bac clone 083c <dig>  despite four cycles of bes elongation, primer design, and subsequent pcr product sequencing.\n",
            "\n",
            "additionally, primers were designed for selected “454” contigs of bac clones 055l <dig> and 067h <dig> to incorporate these sequences into the l. angustifolius genetic map. for clone 055l <dig>  pcr product polymorphism was observed in contig  <dig> . two polymorphic amplicons were identified for contig  <dig> of bac clone 067h <dig> .\n",
            "\n",
            "in the present work,  <dig> new genetic markers originating from nine bac clones and differing in the method of polymorphism detection were developed. six markers can be simply scored by pcr and agarose gel electrophoresis, because they segregated as present/absent or because the pcr products differed considerably in size. four markers required endonuclease digestion of the amplicons   <cit> , whereas for two markers, the restriction stage had to be preceded by pcr amplification with primers that introduced a restriction site based on a polymorphic locus   <cit> . sequences of the pcr products from the parental lines of the mapping population were compared with the source bess and bac sequences. the identity levels of at least 98% were observed for all but one marker, 074i10_3_ <dig> designed for bes 074i10_ <dig>  however, the primer pair based on the sequence of 074i10_ <dig> amplified two polymorphic pcr products, and the other, 074i10_3_ <dig>  showed  <dig> % nucleotide identity with 074i10_ <dig>  considering the similarity of marker sequences to reference bess and bacs, all markers except 074i10_3_ <dig> were considered to be specific labels of appropriate bac clones.\n",
            "\n",
            "molecular markers were allocated to five linkage groups of the recently published narrow-leafed lupin genetic map  <cit> . two markers, 067h16_ <dig> and 067h16_ <dig>  derived from two regions of bac clone 067h <dig>  were positioned in linkage group nll- <dig> at a reciprocal distance of  <dig>  cm. because the source sequences are physically linked, close genetic localization of derivative markers was expected. two other markers for sequenced bacs mapped to this chromosome: 055l12_ <dig>  tagging a contig of five clones, and 080b11_ <dig>  representing a singleton. merely  <dig> cm away from 080b11_ <dig>  a pair of markers  originating from bac clone 076k <dig> was also localized. two other markers, 077c13_ <dig> and 057k22_3f <dig>  based on singletons, were placed in the group nll- <dig> within an interval of  <dig>  cm. two genetic markers, 074i10_3_ <dig> and 074i10_3_ <dig>  derived from the same bes, mapped to the linkage groups nll- <dig> and nll- <dig>  respectively. as mentioned above, comparative sequence analysis of these two markers and bes 074i10_ <dig> suggested that only 074i10_3_ <dig> reflected the position of bac clone 074i <dig>  genetic markers 004g15_ <dig> and 057j20_ <dig>  derived from the two remaining sequenced bacs, were localized in linkage groups nll- <dig> and nll- <dig>  respectively. a supplementary file contains detailed marker data including primer sequences, pcr product sizes, sequence identity levels, applied detection methods, linkage group assignment, lod scores, and segregation data for l. angustifolius 83a: <dig>  and p <dig>  recombinant inbred lines [see additional file  <dig> \n",
            "\n",
            "bac-fish as a supporting tool in genetic mapping\n",
            "a comprehensive cytogenetic approach was applied to support the linkage mapping of selected bac clones. the co-localization of different clones was analyzed using the multibac-fish variant, i.e., simultaneous application of several differently labeled bacs on the same slide. the power of the fish technique is limited by two parameters: probe-size detection and axial-resolution. in advanced fish variants  both parameters are high, and even gene-sized dna fragments can be mapped  <cit> . the sensitivity and resolution of fish in mitotic metaphase are relatively low  <cit> . nevertheless, our study was based on observations of metaphase mitotic chromosomes, because we needed to analyze the clone positions with reference to individual chromosomes and linkage groups. this strategy involved all recognized phtjm <dig> single-locus bacs. additionally, one supplementary bac clone, 017b <dig>  that previously mapped to linkage group nll- <dig>  was included in this analysis. the specifications of the bac-fish combinations are presented in table  <dig> \n",
            "\n",
            "\n",
            "no: bac-fish signals for clones from this pair were observed in different chromosomes.\n",
            "\n",
            "yes: bac-fish signals for clones from this pair were observed in the same chromosome.\n",
            "\n",
            "the bac-fish signals served to determine clone location in chromosomes as well as to elaborate and verify their linkage group assignments. cytogenetic analyses were also used to validate the physical linkage of clones that were initially assembled into contigs by the restriction fingerprinting approach. bac-fish proved to be helpful in checking individual contig positions within the lupin genome. signals of bac clones  originating from two contigs  detected on different chromosomes confirmed that these contigs were separate. the location of bacs representing the same contig in different chromosomes proved that the contig did not reflect the real structure of the genome region, and a correction of bioinformatics analyses was necessary. such an outcome was observed for bacs 057j <dig> and 074i <dig> and resulted in the classification of these clones as singletons.\n",
            "\n",
            "bac fish was also indispensable during linkage mapping. bac-fish with clones 074i <dig> and 057j <dig> helped to resolve the issue of double genetic markers for bac clone 074i <dig>  which mapped to two different linkage groups. the preliminary assumption that the marker 074i10_3_ <dig> was not physically linked to a particular bac clone proved correct, because the fish signals specific for bacs 074i <dig> and 057j <dig> were observed on separate chromosomes. because marker 057j20_ <dig> mapped clone 057j <dig> to linkage group nll- <dig>  the fact that these two bacs were not co-localized indicated that bac 074i <dig> was not located in nll- <dig>  for the remaining bacs, every pair of clones that was not co-localized by genetic mapping had bac-fish signals on separate chromosomes.\n",
            "\n",
            "two clones, 057k <dig> and 077c <dig>  were localized in the same locus. based on the resolution of bac-fish on metaphase chromosomes, these sequences cannot be separated by more than a few megabases. this result converges with the linkage mapping outcome, because the markers 057k22_3f <dig> and 077c13_ <dig> were both located in linkage group nll- <dig> at a distance of  <dig>  cm. similarly, clones 080b <dig> and 076k <dig> mapped to nll- <dig>  their bac-fish signals overlapped each other in one chromosome, but these bacs were separated by a genetic distance of 10 cm.\n",
            "\n",
            "as mentioned above, the bac clone 083c <dig> remained unmapped, despite four rounds of bac-end sequence elongation and pcr-product analysis. to address its chromosome localization, this clone was used in bac-fish together with clone 017b <dig>  which mapped to linkage group nll- <dig>  hybridization signals for these two clones were observed on the same chromosome arm, in two close but separate loci. given the fish resolution and approximate ratio of physical to genetic distance, such a result indicates a genetic linkage of clones 083c <dig> and 017b <dig> of more than 10 cm. the linkage groups, supplemented with newly designed genetic markers and assigned chromosomes carrying specific bac-fish signals, are presented at figure  <dig> \n",
            "\n",
            "\n",
            "the first three genetic linkage groups of l. angustifolius were assigned to chromosomes using a similar approach, based on hybridization of ssr-rich probes with the bac library, selection of single-locus clones in bac-fish, and subsequent mapping of these clones using bes markers  <cit> . here, we significantly enhanced the integration of the chromosomal and genetic maps of narrow-leafed lupin by associating another five linkage groups with chromosomes. because 40% of the lupin karyotype has now been assigned to the genetic map, an alternate strategy can be used to complete the integration process. a commonly used approach to build an integrated plant genome map is fish of bac clones selected from the library by probes carrying sequences of genetically mapped markers  <cit> . however, the markers must be carefully selected to avoid incorporating repetitive sequences and duplicated genes, which can produce an excessive number of hybridization signals. the most recent genetic map of narrow-leafed lupin contains more than  <dig> gene-based markers  <cit> . a subset of these markers originates from putatively single- or low-copy genes and may be used in the future for a marker-oriented integration of the linkage and cytogenetic maps of this species.\n",
            "\n",
            "CONCLUSIONS\n",
            "probes based on sequences of mflp-derived markers are not suitable for positional cloning of genes linked to the particular markers, because they hybridize to numerous loci throughout a genome. however, such probes can be useful in general screening of a genome, because both the amount of repetitive dna and the proportions between the main groups of transposable elements in bac clones selected by mflp probe are similar to those in randomly chosen bacs.\n",
            "\n",
            "bac-fish can serve as a supporting tool to select bac clones for sequencing and tagging gene-rich regions in species with low quantities of sequence data. bac clones showing fish signals dispersed across numerous chromosomes contained many transposable elements, whereas in clones hybridizing to single loci, the amount of repetitive dna was negligible. moreover, single-locus bacs carried significantly more gene-like sequences than clones tagged by fish as repetitive.\n",
            "\n",
            "gene-rich regions identified in the l. angustifolius genome contain a low number of interspersed repeats and a high degree of synteny with the genome of the model legume g. max. this information is valuable for the lupin genome sequencing project. such sections of the genome can be readily assembled to large scaffolds and their gene content properly annotated.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the lupin genome contains  <dig>  clones with an average insert length of  <dig>  kb, resulting in 12× coverage.\n",
            "the lupin genome is partitioned into many small and morphologically similar chromosomes, resulting in increased yields of successive crops.\n",
            "the lupin genome contains  <dig>  clones with a average insert length of  <dig>  kb, resulting in 12× coverage.\n",
            "the lupin genome is partitioned into many small and morphologically similar chromosomes, resulting in increased yields of successive crops.\n",
            "the lupin genome contains  <dig>  clones with a average insert length of  <dig>  kb, resulting in 12× coverage.\n",
            "the lupin genome contains  <dig>  clones with a average insert length of  <dig>  kb, resulting in 12× coverage.\n",
            "the lupin genome contains  <dig>  clones with a average insert length of  <dig>  kb, resulting in 12× coverage.\n",
            "the lupin genome contains  <dig>  clones with a average insert length of  <dig>  kb, resulting in 12× coverage\n",
            "18\n",
            "[{'summary_text': 'phenotypic plasticity increases the probability of population persistence after colonization of a new environment, and suggests that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\\nour results suggest that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\\nour results suggest that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\\nour results suggest that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\\nin the midas cichlid species complex, the pharyngeal jaw of the cichlid amphilophus citrinellus is plastic.\\nthe pharyngeal jaw of the cichlid amphilophus citrinellus is plastic.\\nthe pharyngeal jaw of the cichlid amphilophus citrinellus is plastic.\\nour results suggest that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\\nour results suggest that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\\nthe pharyngeal jaw of the cichlid amphilophus citrinellus is plastic'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "adaptive radiations arise through the rapid divergence of an ancestral species into a multitude of morphologically and ecologically differentiated taxa  <cit> . this process is assumed to be driven by divergent natural selection and ecological speciation where the adaptation to different niches eventually results in the evolution of reproductive isolation  <cit> . for example, specialization to certain food resources might lead to divergent habitat preferences, which in turn might isolate the populations reproductively . specialization in diet is usually accompanied by morphological adaptations facilitating resource exploitation as has been shown in some textbook examples of adaptive radiation, e.g. the darwin finches on the galapagos islands  <cit> , the cichlid fishes in east african lakes  <cit> , or the cosmopolitan tiger beetles  <cit> .\n",
            "\n",
            "often, adaptive radiations are triggered by an altered adaptive landscape providing opportunity to invade previously not encountered ecological niches  or not accessible niches   <cit> . recent studies showed that these adaptive peak shifts might happen rapidly , and raise the question of how the adaptive morphological change drives the shift from one peak to another on the adaptive surface  <cit> . mutation in coding and regulatory sequences and selection might not be sufficient to explain the rapidity of ecological adaptation seen in some instances  <cit> . adaptation from standing genetic variation is also not likely to apply to all cases of adaptive radiations, particularly those with only a small number of founders  <cit> . adaptive phenotypic plasticity might play a key role allowing populations to enter the 'realm of attraction' of a new adaptive peak, in which genetic assimilation occurs through directional selection favoring genotypes that produce even more extreme phenotypes than what would be possible by plastic response of the ancestral genotype alone  <cit> . baldwin discussed this topic already in  <dig> and described it as 'a new factor in evolution'  <cit> . although its importance meanwhile became evident, phenotypic plasticity and genetic assimilation were dismissed as being unimportant during the modern evolutionary synthesis  <cit> . there has been a recent resurgence of interest in these phenomena  <cit> , but the link to diversification is still little explored and under debate  <cit> . not many investigations of phenotypic plasticity in model systems for speciation research, such as cichlid fishes, have been attempted .\n",
            "\n",
            "the neotropical midas cichlid species complex , is recognized among evolutionary biologists for its rapid phenotypic diversification and speciation  <cit> . this species complex has its center of its distribution in nicaragua, and is comprised of an array of very young species that inhabit both the large nicaraguan lakes, and several volcanic crater-lakes that contain small scale adaptive radiations  <cit> . the large nicaraguan lakes, characterized by relatively turbid and shallow waters, have repeatedly acted as source populations for the colonization of nearby crater-lakes newly formed in the calderas of extinguished volcanoes. in these lakes the midas cichlids encountered novel environmental conditions - i.e. presence of deeper zones and clearer water - and speciated in situ  <cit> . crater-lake species have separated along depth and benthic-limnetic axes  <cit> , with the open water column apparently being the first novel habitat invaded. also, the midas cichlid species have differentiated in their trophic adaptations. usage of food sources like stonewort, aufwuchs, evasive invertebrate prey, fish or snails differs species-specifically  <cit> . the midas cichlids species, as well as other neotropical and old world cichlids, often differ in the relative degree of hypertrophy of a second set of jaws in the throat - the pharyngeal jaw - derived from branchial arch components and important for food mastication . specialization for feeding on hard-shelled prey like snails, mussels, or crustaceans  through this hypertrophy of the pharyngeal jaw apparatus  has been found to be a common axis of differentiation in crater-lake midas cichlids as well as in other cichlid groups  <cit> . its frequency and independency of acquisition across the phylogenetic tree suggests an important role of this adaptation in cichlid speciation \n",
            "\n",
            "the midas cichlid species in the crater lakes are often well differentiated in the trophic apparatus and only a few thousand years old  <cit> . the trophic polymorphism in the midas crater-lake species could be derived from standing genetic variation, since the polymorphism is present in the large lakes, too  <cit> . however, the probably limited number of colonizing individuals would render a scenario of the evolution of trait divergence subsequent to colonization also plausible. this scenario is arguably more likely for remote crater-lakes with a monophyletic midas cichlid assemblage, e.g. lake apoyo . a plausible scenario could be that the divergence in the pharyngeal jaw apparatus in the crater lake midas cichlid species might have been initiated by phenotypic plasticity in the ancestor. reproductive isolation might then have occurred via habitat isolation through the heterogeneous distribution of snails in nicaragua's volcanic crater-lakes, where densities appear to be dependent on depth and substrate type  <cit> . during times of low food availability otherwise opportunistic individuals adapted for durophagy might confine to areas of high snail density and thereby encounter mates non-randomly in respect to their pharyngeal jaw type  <cit> . if the ancestor of derived species was phenotypically plastic in ecologically relevant traits, this plasticity might have triggered the diversification. the \"flexible stem\" model, proposed by west-eberhard  <cit> , predicts that the directions in phenotypic space in which plasticity is expressed influence the trajectories of phenotypic evolution via genetic accommodation, similar to evolution along \"genetic lines of least resistance\"  <cit> . therefore, it also predicts the outcomes of adaptive radiations seeded by the same ancestor and evolving in similar environments to be similar in terms of their phenotype composition.\n",
            "\n",
            "in several cichlid fish species , plasticity in different traits has been demonstrated: meyer experimentally induced changes in the oral jaw morphology in the neotropical cichlid parachromis managuensis by feeding different diets  <cit> , a similar procedure was followed by bouton and coworkers using the african cichlid neochromis greenwoodi  <cit> . the lake victoria cichlid haplochromis pyrrhocephalus was almost driven to extinction by the upsurge of the introduced, predatory nile perch in the 1980s, but was able to adapt morphologically to the new environmental conditions of high predatory pressure and eutrophication in only two decades  <cit> . it has been interpreted that the speed and complexity of these morphological changes relied on a joined action of phenotypic plasticity and genetic change. the molluscivorous astatoreochromis alluaudi naturally exhibits molariform pharyngeal jaws   <cit> . however, when raised on soft artificial food under laboratory conditions  <cit> , in natural conditions in lakes not inhabited by snails  <cit> , or in lakes inhabited by snails but also with a molluscivorous competitor present  <cit> , they develop less stout pharyngeal jaws with cuspid teeth .\n",
            "\n",
            "specializations matter most during ecological \"crunch times\", when resource availability is low and opportunistic feeding is precluded  <cit> . the ability to exploit resources then at all or more efficiently than other species can, matters for the individual's survival. but specializations come with a trade-off. the specialization of being able to feed on particular diets especially efficiently often comes at the cost of being much less efficient when dealing with alternative diets. apparently, such a trade-off exists in the neotropical midas cichlid  between two different types of pharyngeal jaws, molariform and papilliform. individuals with papilliform lower pharyngeal jaws are more effective when dealing with soft food items  <cit> . individuals with molariform jaws, on the other hand, can crack larger and harder snail shells and do this faster than papilliform individuals  <cit> .\n",
            "\n",
            "these cases of phenotypic plasticity, the basis of lacustrine cichlid radiations on trophic specialization  <cit>  and the possible causal linkage of plasticity and diversification  <cit>  call for examination of adaptive phenotypic plasticity in trophic traits in an adaptive radiation of cichlids comprising species differentiated in these traits. the lower pharyngeal jaw  might constitute 'an ideal component of cichlid trophic morphology' to be investigated in this respect  <cit> . preferably, the case in study should have a known and young history, involve colonization of new habitats and tests for plasticity in the ancestral or similar to the ancestral source population.\n",
            "\n",
            "here, we tested in a common garden experiment the developmental plasticity of the lower pharyngeal jaw of amphilophus citrinellus  exposed to diets differing in hardness. earlier work  <cit>  had suggested that the species in this species complex are phenotypically plastic and that the abundance of molariform fish correlates with the abundance of their major prey item, hard-shelled snails.\n",
            "\n",
            "the experiment was performed on a laboratory stock derived from the crater lake masaya, which was bred in captivity for several decades. although lake masaya is a volcanic crater-lake, its a. citrinellus population is very close to the populations of the lake nicaragua - which is probably the ancestral source population of most crater-lake radiations - in terms of body shape  <cit>  and phylogenetic relationships  <cit> . furthermore, it has been suggested that lake masaya might have been colonized as recently as  <dig> years ago  <cit> .\n",
            "\n",
            "we investigated whether the development of pharyngeal jaws differed between three types of diets:  intact snails with shell,  peeled snails without shell, and  finely ground up whole snails frozen in pellets, from which fish could nibble off the thawed, soft outer layer when those were given into the water. we aimed to verify whether a hard diet could induce changes in the pharyngeal jaw of the fish, and whether the generation of robust pharyngeal jaws with stout teeth  was determined by higher calcium content in the diet, or by mechanical stimulation of the jaws when crushing hard food items.\n",
            "\n",
            "our study finds that diet can induce changes on the trophic apparatus of the midas cichlids, and that this changes are related to the mechanical stimulation of the jaws.\n",
            "\n",
            "RESULTS\n",
            "geometric morphometric analyses\n",
            "the shape of the lower pharyngeal jaw differed significantly between the fish raised on a diet 'with shell' and the other two groups of fish as revealed by permutation testing of procrustes distances . the morphological differentiation measured by procrustes distance was significant and similarly large between the 'with shell' and the two other groups . the distance between 'ground' and 'no shell' was considerably smaller  and not significant. depicting the between group changes along discriminant functions by warped outline drawings revealed that shape was altered most in functionally relevant regions of the lpj, namely the posterior horns. in the 'with shell' group the horns  pointed more outward and were broader, and jaws were generally shorter along the anterior-posterior axis . additionally, the posterior outline  was less concave in the 'with shell' group as in the other groups. in the 'ground' group the posterior outline was as well less concave as in the 'no shell' group and the horns were directed outward slightly more, but horn width was smaller. the relative overlap on the first two principal components of shape variation between the treatment groups is illustrated in figure  <dig> \n",
            "\n",
            "distances between the group means in lpj shape space for data regressed on body weight . significance was assessed by permutation testing with  <dig> permutations.\n",
            "\n",
            "analyses of weights and lengths\n",
            "taking body weight as proxy for ontogenetic stage and correcting for it, measures not covered by the geometric morphometric shape analysis were investigated. the lpj weight showed significant differences between groups with 'no shell' having the lightest, 'with shell' having the heaviest and 'ground' having intermediate jaws. the centroid size, i.e. the scaling factor from the size-removing step in the alignment of landmark configurations, was found to differ significantly between the 'shell' and the 'no shell' group and between the 'shell' and the 'ground' group. differences were not significant between the 'ground' and the 'no shell' group . the dimension not assessed by centroid size, the jaw height, showed no group differentiation if fish body weight was taken as covariate, but showed strong group differentiation when corrected for lpj weight instead. in that case, the 'no shell' group had the highest, the 'with shell' group the most slender and the 'ground' group intermediate jaws relative to jaw weight. this points to an increase in bone density, moderate with high calcium diet and strong when mechanical impact acted also on the jaws during feeding.\n",
            "\n",
            "results of anovas for length and weight data using diet group and either body weight or lpj weight as factors. given are the p-values of the anovas and the p-values from a subsequent tukey honest significant difference-test for each group comparison. ws: 'with shell'-group; ns: 'no shell'-group; g: 'ground'-group\n",
            "\n",
            "the weight of the heavier of the fish's two largest otoliths - the sagittae - using fish body weight as covariate in an analysis of covariance, did not differ in the two high-calcium groups, but was significantly lower in the 'no shell' group . correcting for lpj weight, the 'with shell' group had significantly lower relative sagitta weight, while 'ground' and 'no shell' did not differ .\n",
            "\n",
            "discussion\n",
            "phenotypic plasticity has been hypothesized to be able to promote divergence only if it is not complete, i.e. sufficient to achieve the same fitness as if the trait was expressed constitutively  <cit> . a plastic response would be adaptive if it shifts the phenotype in the direction of a new peak on the adaptive surface, and non-adaptive or maladaptive responses to stressful environments would place the phenotype away from any optimum  <cit> . here, we were able to induce an adaptive plastic response in the lpj of a. citrinellus by feeding different diets. it qualitatively resembles interspecies differences found in nature, although less pronounced.\n",
            "\n",
            "in our common garden experiment, the changes induced on the fish exposed to a hard shell diet - i.e. horns of the lpj pointing more outwards, posterior outline less concave, lpj relatively heavier and possibly increased bone density - mirror those identified as adaptations for mollusk crushing in several other cichlid sister-species pairs  <cit> ,very closely related species in the midas cichlid complex in several crater lakes  <cit>  and in constitutively expressed  <cit>  or induced  <cit>  phenotypes in other species. the expression of a relatively hypertrophied pharyngeal jaw due to durophagy resembling adaptations found in specialized molluscivorous fish, and the result that hypertrophication is much weaker when fish are fed with high-calcium, low-impact diet leads to the conclusion that the observed phenotypic plasticity is indeed adaptive. the trade-off in feeding performance between different phenotypes further evidences the adaptive nature of plasticity in this trait  <cit> .\n",
            "\n",
            "a surprising finding is that lpj height did not differ between the experimental groups, since along this dimension divergence is commonly found in non-molluscivorous/molluscivorous species pairs  <cit> . a possible explanation would be that this trait behaves allometrically with larger and older molariform fishes expressing more re-growing molars thickening the lpj. a longer common garden experiment might reveal plasticity in this trait as well. an alternative is that lpj height is simply not plastic, and its evolution is solely governed by mutation and selection that might bring about developmental constraints. structural constraints and the lack of phenotypic accommodation would be a possible explanation as well. under this scenario, an increase in lpj height would not be possible due to prohibitive spatial demands.\n",
            "\n",
            "several findings suggest that no specific and adaptive shape difference was induced by a high-calcium diet alone. only small differences in shape were observed between 'no shell' and 'ground' groups, and those differences did not resemble known adaptations for durophagy. furthermore, the comparisons including otolith weight show that calcium allocation is strongly biased towards the lpj in the 'with shell' group but not in the 'ground' group. there, it appears to affect the skeleton evenly as indicated by the group comparison for sagittae weight when correcting for lpj weight. this corroborates the finding that the mechanical impact on the lpj during feeding triggers increased calcium allocation towards the jaw and suggests that a high-calcium diet leads to an unspecific increase in calcium deposition.\n",
            "\n",
            "the sagittae, as well as the other otoliths, grow in small increments throughout the fish's life  <cit>  and their weight is considered to reflect weight of the individual and availability of calcium during its life. however, ichii and mugiya  <cit>  showed that fish raised on a calcium depleted diet did not show different bone densities after a period of  <dig> days, but were able to substitute the lacking dietary input of calcium by increasing uptake through the gills from the water. farrell and campana  <cit>  observed that environmental availability of calcium does not affect its deposition on the otolith. these studies have background levels of calcium in both, supplied diet and water, which might differ from levels in our experiment, involve different species and their experiments were conducted significantly shorter. these differences in experimental setup might explain why in our study an effect of calcium availability on bone and otolith growth was observed as opposed to the other studies.\n",
            "\n",
            "the effects of the mechanical impact were strong enough to exceed anticipated effects of a higher availability of calcium in the 'ground' diet due to facilitated uptake of minerals from the readily processed shells. 'with shell' fish regularly spat out shell fragments during mastication, and hoogerhoud  <cit>  reports snail shell pieces to pass the digestive tract of cichlids apparently unharmed. such observations might explain the slight and non-significant shift towards relatively heavier otoliths in 'ground' fish when accounted for body weight. several studies on phenotypic plasticity express concerns about the influence of diet quality on developmental differences between treatment groups, so that detrimental effects of a low-quality diet might be mistaken for  phenotypic plasticity  <cit> . here, we addressed these concerns with our feeding regime. specifically, we are able to show that induced differences were not due to a lack of calcium in the diet. even though the studied individuals descended from an inbred line, which has not been subject to artificial selection favoring plasticity in the pharyngeal jaw apparatus, ability to express this trait plastically persisted. this suggests that the plasticity of the lpj in a. citrinellus might not be a trait under selection itself, but more likely an instance of a hidden reaction norm  <cit> .\n",
            "\n",
            "similarly to the midas cichlid, other cichlid species show pja adaptable or adapted to durophagy: in neotropical cichlids non-molluscivorous and molluscivorous species, having papilliform and molariform lpjs respectively, often represent closely related sister species pairs  <cit> . the same trajectory of divergence has been found between trophic morphs of the same species, herichthys minckleyi, occurring in the cuatro ciénegas basin, mexico. along the same axis allometric changes happen during the ontogeny of the mayan cichlid cichlasoma urophthalmus, introduced in southern florida  <cit> . the presence of hypertrophied pharyngeal jaws is not restricted to cichlids, or even to freshwater fishes: members of the marine families sciaenidae, haemulidae and carangidae express a similar type of pja, allowing them to feed on hard-shelled prey. the phylogenetic relationship to species with non-hypertrophied pharyngeal jaws can be close, e.g. congeneric, in these cases as well  <cit> .\n",
            "\n",
            "the number of cases of closely related species or trophic morphs of a single species exhibiting such divergent morphologies, as well as their phylogenetic dispersal, is astonishing. this trajectory in morphospace might be similarly important as the well-known deep-bodied vs. elongated body trajectory found in many benthic-limnetic fish species pairs . both phenotypic contrasts are usually accompanied by extensive diet and/or habitat preference differences, respectively. such ecological diversification has been shown to be a major factor in empirically studied speciation events and its importance in speciation is well supported by theoretical models  <cit> . in the midas cichlid species complex, ecological diversification has been shown to occur along both axes, even in correlation  <cit> , and probably led to speciation in several cases  <cit> .\n",
            "\n",
            "phenotypic plasticity and rates of diversification\n",
            "the importance of phenotypic plasticity in population divergence and speciation gained increasing attention in the last years  <cit> . both studies focusing on single species and studies within a larger comparative framework investigated this link: nylin & wahlberg found support for a 'plasticity scenario' for the diversification of nymphaline butterflies during the tertiary and argued that herbivorous taxa able to occupy several niches were more likely to diversify along with the angiosperm radiation  <cit> . in coastal san diego a population of montane dark-eyed juncos  was able to establish itself due to an adaptive plastic response in reproductive effort  <cit> . a recent review by pfennig et al.  <cit>  summarizes theoretical and empirical studies and diagnoses an important, but largely underappreciated, role of phenotypic plasticity in speciation and adaptive radiation. comparing sister clade pairs - with one clade being known to include cases of resource polyphenism, while the other does not - pfennig and mcgee found evidence that resource polyphenism is associated with greater species richness in fishes and amphibians  <cit> .\n",
            "\n",
            "the role of phenotypic plasticity in population divergence appears to be at least twofold:  plasticity increases the probability of population persistence after colonization of a new environment, thus making its split from the ancestral population more likely  <cit> , and  plasticity provides means of conquering other peaks on the adaptive landscape, possibly leading to assortative mating and speciation with parallel outcomes in repeated cases  <cit> .\n",
            "\n",
            "theoretical investigations support these predictions. probability of population persistence increases with plasticity while being dependent on the amount of environmental change and the costliness of plasticity  <cit> . at a moderate rate of environmental change and if plasticity is costly, high levels of plasticity are expected to lead to an increased probability of extinction while an intermediate level improves the ability of persistence  <cit> . access to novel ecological niches is improved because an increase in epigenetic variability does facilitate the circumvention of adaptive valleys and smoothes the fitness landscape  <cit> . using numerical simulations thibert-plante and hendry  <cit>  find plasticity to commence reduction in gene flow between populations in contrasting environments. to do so, plasticity must occur before dispersal but could then lead to reproductive isolation even prior to any adaptive genetic divergence.\n",
            "\n",
            "our demonstration of adaptive phenotypic plasticity in the lpj of a. cf. citrinellus suggests that this could be a crucial factor in ecological speciation and adaptive radiation in the repeated amphilophus crater-lake radiations and possibly in other cichlid clades as well  <cit> . the results of our experiment support the \"flexible stem\" hypothesis, in that the induced differences between treatment groups - more robust lpjs in the 'with-shell' group, less robust lpjs in groups fed soft food - resemble between-species differences in crater-lake radiations. however, we did not test for plasticity in the ancestor itself, nor in fish derived from the large nicaraguan lakes, but in a stock derived from lake masaya. the different history might have caused an alteration of the plastic response in experimental groups compared to the real ancestor. but since there is a considerable chance of the lake masaya a. citrinellus population being very young and since plasticity here seems not be lost easily , we suggest that our results endorse the \"flexible stem\" hypothesis for the midas cichlid assemblage. because the induced plasticity does not reach the extent of morphological divergence found between species in nature we conclude, that the expectations from the \"adaptive surface model\" are fulfilled as well.\n",
            "\n",
            "in which way exactly phenotypic plasticity and genetic accommodation in the pharyngeal jaw might abet diversification in the amphilophus species complex remains speculative. a direct influence on the formation of reproductive isolation might be given through enhancement of habitat preference. if individuals expressing the same type of pharyngeal jaw have a higher chance of mating with each other, and gene flow between groups is hampered strongly enough, population subdivision might be initiated. the heterogeneous distribution of snails, if it is stable over time and patches are sufficiently large, might be the basis for habitat preference by jaw type. alternatively, the hypothesized function of the pharyngeal jaw apparatus in sound production, e.g. during courtship, might bring about assortative mating according to jaw type if female sound preference is divergent as well  <cit> .\n",
            "\n",
            "however, even if phenotypic plasticity is less important in sympatric speciation scenarios it might still influence diversification in allopatry . by augmenting the probability of population persistence after colonization of a new environment, e.g. a crater-lake, and the possibility of genetic accommodation of plastic trait changes the likelihood of allopatric speciation between ancestral source population and the new colonizing population is increased. it remains unclear, whether or not the repeated endemic radiations of midas cichlids in nicaraguan crater-lakes are facilitated by phenotypic plasticity in the pharyngeal jaw or if the constitutively expressed differences in jaw shape between species are a secondary result of speciation driven by other factors. the best documented case of an in-crater-lake diversification, the origination of the arrow cichlid amphilophus zaliosus in lake apoyo, seems to have been driven by diverging habitat preferences with differences in pharyngeal jaw shape being probably secondary  <cit> . however, in other, less-well documented cases the hypothesis that adaptations in the pharyngeal jaw apparatus triggered divergence remains valid, but would need to be further investigated.\n",
            "\n",
            "CONCLUSIONS\n",
            "we demonstrated phenotypic plasticity in the pharyngeal jaw of the cichlid fish amphilophus citrinellus that is due not to differences in nutritional composition of the diet, but brought about largely by the mode of feeding. this finding might suggest that plasticity plays an important role in diversification.\n",
            "\n",
            "future research on how a plastic reaction in one trait could impact the expression of other traits through correlated plastic responses might contribute to the understanding of parallelisms so often encountered in nature. for example, it seems the papilliform pharyngeal jaw type is correlated with fusiform limnetic body shape whereas the molariform jaw type is correlated with deeper, benthic body shape  <cit> . the extent to which this 'integration of plastic responses'  <cit>  is determined, and by which factors, still remains to be elucidated. also, what role a stage of fixed polymorphism plays in the process of diversification, whether it is an intermediate step  <cit>  or a 'dead-end', remains to be investigated.\n",
            "\n",
            "how adaptive phenotypic plasticity is mediated genetically is another important issue. in cichlids, the family of bone morphogenetic proteins  is known to be involved in shaping bones of the oral and pharyngeal jaws  <cit>  and might constitute good candidates, along with respective transcription factors and ligands, for the elucidation of the genetics of phenotypic plasticity in the pja.\n",
            "\n",
            "cichlids, are a prime system for speciation research and have an important trophic trait expressed plastically, and therefore constitute a cogent group for investigating the role of adaptive phenotypic plasticity in diversification. research combining experimental and field studies with modern tools of analysis, such as sensitive group assignment methods or gene expression quantification, will be most rewarding avenues of research to elucidate the link between plasticity and speciation\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "phenotypic plasticity increases the probability of population persistence after colonization of a new environment, and suggests that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\n",
            "our results suggest that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\n",
            "our results suggest that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\n",
            "our results suggest that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\n",
            "in the midas cichlid species complex, the pharyngeal jaw of the cichlid amphilophus citrinellus is plastic.\n",
            "the pharyngeal jaw of the cichlid amphilophus citrinellus is plastic.\n",
            "the pharyngeal jaw of the cichlid amphilophus citrinellus is plastic.\n",
            "our results suggest that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\n",
            "our results suggest that adaptive phenotypic plasticity plays an important role in population divergence and speciation.\n",
            "the pharyngeal jaw of the cichlid amphilophus citrinellus is plastic\n",
            "19\n",
            "[{'summary_text': 'in this paper, we propose a new integrative measure called intego <dig>  by automatically selecting the most appropriate seed measures for each gene pair, and integrating the seed measures using a metaheuristic search method.\\nin this paper, we present a new integrative measure called intego <dig>  by automatically selecting the most appropriate seed measures for each gene pair, and integrating the seed measures using a metaheuristic search method.\\nin this paper, we present a new integrative measure called intego <dig>  by automatically selecting the most appropriate seed measures for each gene pair, and integrating the seed measures using a metaheuristic search method.\\nin this paper, we present a new integrative measure called intego <dig>  by choosing the most appropriate seed measures for each gene pair from a pool of candidate measures, and integrating the seed measures using a metaheuristic search method.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the gene ontology  provides a representation of biological knowledge through structured, controlled vocabulary of terms, which are interrelated forming a directed acyclic graph  for describing the functional information of gene products  <cit> . go consists of three categories that shared by all organisms: molecular function , biological process  and cellular component   <cit> . as a widely used bioinformatics resource, go provides rich information and a convenient way to study gene functional similarity, which has been successfully used in various aspects including predicting gene functional associations  <cit> , homology analysis  <cit> , assessing target gene functions  <cit> , and predicting subcellular localization  <cit> .\n",
            "\n",
            "since go was released, various computational measurements have been developed to compute gene functional similarities by comparing go terms with which the genes are annotated  <cit> . these term- comparison measurements can be classified into three categories based on the types of knowledge in go that they used: edge-based, node-based, and hybrid  <cit> .\n",
            "\n",
            "the measures in the edge-based category take the structure of go into account  <cit> . by using the topological information of go directed acyclic graph , a recently designed method relative specificity similarity  models both the distance of given term pair to its closest leaf terms and the distance to their most recent common ancestor   <cit> . the edge-based measures, however, are still fully dependent on the topology of go dag, and it is inappropriate to simply equalize the terms at the same topological level  <cit> .\n",
            "\n",
            "in the node-based category, methods originally designed for natural language processing  <cit>  are utilized for term comparisons. in the earlier developed measures, the similarity of two go terms is defined as the information content of their most informative common ancestor , indicating its specificity. it was further advanced by modeling the distance between a given term pair to its mica  <cit> . the results show strong correlations with yeast gene co-expressions and protein sequence similarities  <cit> . however, the node-based measures only consider the annotations and common ancestors, neglecting the complex topology of the go dag.\n",
            "\n",
            "hybrid measurements have been recently proposed to consider the more complete information in go.  <cit>  utilizes all of the parent terms of the target terms, which takes the topology of the go dag into account. hybrid relative specificity similarity  employs the concepts of information content, adapting topology, annotations and mica  <cit> . the experiment results show that both wang and hrss measures perform better than the traditional node-based measures  <cit> . however, these measures still only focus on several types of information in go but neglect others.\n",
            "\n",
            "since none of the existing measure can employ all the information in go, an integrative approach to unite all the strength of existing measures is preferred. in this direction,  <cit>  proposed a rank-based gene semantic similarity measure called intego by synergistically integrating multiple similarity measures  to take into account more aspects of go . intego first selects measures based on an evaluation set, and then integrates the selected measures using one of four straightforward methods . the experiment results showed that intego performs significant better than the seed measures  <cit> . however, the performance of intego is still limited, because it is vulnerable to the selection of low performance measures, and its fixed integration strategy may not be suitable for all gene pairs.\n",
            "\n",
            "in this paper, we aimed to present a new integrative measure called intego <dig>  by choosing the most appropriate seed measures for each gene pair from a pool of candidate measures using a grouping method, and by integrating the selected seed measures using a metaheuristic search method. the major contributions are:\n",
            "\n",
            "* our new integrative measure not only takes into account the state-of-the-art go based measures, but also selects the most appropriate seed measures for each gene pair.\n",
            "\n",
            "* a metaheuristic search method is presented in intego <dig> to flexibly integrate multiple seed measures.\n",
            "\n",
            "method\n",
            "the framework of intego <dig> is shown in figure  <dig>  the whole process includes two parts: 1) model training , in which the parameters of intego <dig> are obtained using a training set t , and 2) gene-to-gene similarity calculation  for the input gene set g. in intego <dig>  we solve two key problems, i.e, to select the most appropriate seed measures for each gene pair from all the candidate measures and to appropriately integrate the seed measures.\n",
            "\n",
            "intego <dig> has three steps. first, we calculate all the similarity scores using all the candidate measures and then rank them, resulting in a ranked matrix mr . second, a grouping process is applied on mr to identify the common features of all the ranked results, with which we define a set of seed measures for each gene pair saved in sseed. third, we integrate all the measures in sseed with an addition model, in which the parameter of each component is estimated by applying a learning process on training set t . we will introduce the three steps of intego <dig> in the following text.\n",
            "\n",
            "step  <dig>  computing similarities using all measures\n",
            "the similarity scores of all the gene pairs in a given gene set gs are calculated using all the candidate measures sall. and then for each measure, all the gene pairs are sorted incrementally according to their similarity scores, resulting in a ranked matrix mr , in which each row is a gene pair and each column is a measure, and mr  is the rank of gene pair i in measure j. subsequently, the ranked gene similarity score ranksim for genes g <dig> and g <dig> in gs is calculated as:\n",
            "\n",
            "  ranksimg1g <dig> m=2×mrg1g <dig> mgs <dig> \n",
            "\n",
            "where g <dig> and g <dig> are two target genes, m is a candidate measure in sall, |gs| is the number of genes in gene set gs, which according to figure  <dig>  is the input gene set g or the training set t . ranksim ∈  <cit> . ranksim indicates how similar g <dig> and g <dig> is, compared with all of the gene pairs in gs. note that although the similarities using each measure may at a different scale or have a different distribution, the ranked results are comparable. therefore, the integration of all the ranked results may better reflect functional similarity.\n",
            "\n",
            "step  <dig>  selecting seed measures\n",
            "since different similarity measures use different types of information in go, or model data in different ways, one measure may perform the best on certain functional categories but not on the others. alternatively, the integration of suitable measures makes it possible to calculate the overall similarity score by considering all the aspects of go. a key problem here is to select the most appropriate measures  for every gene pair from a pool of candidate measures.\n",
            "\n",
            "in this paper, we present a solution to this problem based on only one principle that the final ranked score should be the score that all the seed measures agree. to this end, a grouping algorithm to select the most appropriate seed measures for each gene pair is proposed as follows. let ranksim, ranksim, …, ranksim be the ranked similarity scores of n candidate measures for g <dig> and g <dig>  and mx ∈ sall. by putting them on a number axis, we group all the candidate measures agglomeratively based on their distances on the axis, forming a dendrogram d. and then we gradually reduce the distance threshold d in d to iteratively find the isolated measures and remove them until a core group of measures is leftover - which is called the seed measure group . mathematically, a seed measure group is the largest group with at least c measures, where c is a pre-defined value . and the distance between genes in the seed measure group is not larger than d′, where d′ is a pre-defined value .for g1g <dig>  only the measures in the seed measure group are considered as seed measures, saved in sseed.\n",
            "\n",
            "an illustration example of the seed measure group is shown in figure  <dig>  in the figure, with the decrease of d from d <dig> to d′, the isolated measures are in the order of m <dig>  m <dig>  m <dig>  and m <dig>  and the the seed measure group include m <dig>  m <dig>  m <dig>  and m <dig> \n",
            "\n",
            "it is clear that a seed measure group can be labeled as as high, low, or mix according to its distribution in the number axis. mathematically, we define the label of a seed measure group using the highest number of the isolated measures in the leftmost, middle or rightmost of the number axis. for example, the seed measure group in figure  <dig> is high, in figure  <dig> is low, and in figure  <dig> is mix. we label the seed measure groups, because the integration strategy could be different for different seed measure group types.\n",
            "\n",
            "step  <dig>  integrating seed measures\n",
            "in order to integrate the selected seed measures, we adopt an addition model which is one of the best known method for integrating a number of alternatives  <cit> . given a gene pair, we have learned its seed measures and the type of seed measure group from the previous step. for different types of seed measure groups, we build an addition model as shown in eq. 2:\n",
            "\n",
            "  sim=∑hi⋅ranksim+hα⋅max+hβ⋅min+hγ⋅aveif type = high;∑li⋅ranksim+lα⋅max+lβ⋅min+lγ⋅aveif type = low;∑mi⋅ranksim+mα⋅max+mβ⋅min+mγ⋅aveif type = mix. \n",
            "\n",
            "where type is the type of seed measure group; i is a seed measure in the seed measure group; ranksim is the similarity of given gene pair calculated with measure i ; xi is the parameter of seed measure i, where x is h, m or l; max, min and ave represent the maximum, minimum and average of all the ranksim values for g <dig> and g <dig> using all the seed measures; and xα, xβ , xγ are their parameters respectively. we include maximum, minimum and average in the eq.  <dig>  because the experiment results in  <cit>  show that maximal, minimal and average values are better than individual measure in the tested conditions.\n",
            "\n",
            "in order to use eq.  <dig> for seed measure integration, the parameters, e.g. xα, xβ , xγ , needs to be assigned. instead of leaving the difficult job to the end users, we estimate these parameters using a training data t . specifically, we adopt a metaheuristic search method to gradually update the parameters in eq.  <dig> to maximize the score of an objective function in t .\n",
            "\n",
            "there are a wide variety of metaheuristics, including simulated annealing, tabu search, iterated local search, variable neighborhood search, and greedy randomized adaptive search. it also includes a learning component to the search, such as ant colony optimization, evolutionary computation, and genetic algorithm. in this paper, we adopt the tabu search method. comparing with a simple local search procedure, tabu search carefully explores the neighborhood of each solution through the use of memory structures  to avoid sticking in the poor-scoring areas or areas where scores plateau  <cit> . specifically, given the training set t , we use the ec number  to explain molecular function with the criteria that the molecular functions of a group of genes are similar if they have the same ec numbers  <cit> . therefore, we can locate the best candidates of solutions for next move in the searching process.\n",
            "\n",
            "given all the genes in t grouped by their ec numbers, we compute both the intra-ec gene similarities and the inter-ec gene similarities using eq.  <dig> starting with a set of random parameters. we then gradually update the parameters to increase the difference between the intra- and inter-ec similarities. quantitatively, we utilize the logged fold change  measure which has been widely used in the gene expression studies  <cit> . the logfc score of ec number eiis defined in eq. 3:\n",
            "\n",
            "  logfc=1|ec|×∑ej∈ec;g∩g=θ∑g∈gdiffg|g| \n",
            "\n",
            "where g is set of all of genes which are assigned to ei; ec is a set of ecs which do not have any overlapped genes with ei  ∩ g = ∅) in the training set t ; and diffg  is calculated as:\n",
            "\n",
            "  diffg=ln|g|×∑g′∈g+c)|g|×∑g*∈g+c) \n",
            "\n",
            "where c is a constant small positive number, as a laplacian smoothing parameter; g is the set of all of the genes which ec number is ei except gene g; g is the set of all of the genes which ec number is ej ; g is a gene assigned to ei. sim and sim are defined in eq.  <dig>  in eq.  <dig>  the numerator and denominator represent the inter-ec distance and intra-ec distance respectively. the higher the diffg  is, the more obvious the positive difference between inter-ec difference and intra-ec difference is.\n",
            "\n",
            "finally, given training set t grouped by a set of ec numbers, the optimization function for each tabu search move is the average logfc score of all the involved ec numbers in the training set t :\n",
            "\n",
            "  optf=1|t|×∑ei∈tlogfc \n",
            "\n",
            "subsequently, we estimate the parameters in eq.  <dig> using the following tabu search process :\n",
            "\n",
            " <dig>  initialize tl as the empty tabu list, and a set of random parameters in eq.  <dig> as current solution s  satisfying ∑i∈mgxi + xα + xβ + xγ =  <dig> , where x is h, m , or l. the initial best solution is bs = s.\n",
            "\n",
            " <dig>  calculate the neighborhood solutions of s by increasing or decreasing one or multiple parameters in s. note that we learn one group of parameters at a time. for example, while learning parameters for hx, the other two groups lx and mx are fixed.\n",
            "\n",
            " <dig>  the best solution for next move s′ is selected from the neighborhood solutions of s using the optimization function .\n",
            "\n",
            " <dig>  if s′ > bs, let s′ be the current solution, update tl and bs = s′.\n",
            "\n",
            " <dig>  if s′ ≤ bs, we still let current best solution s = s′ and update tl if s′ ∉ tl. otherwise, we delete s′ from the neighborhood solutions and go back to step  <dig> \n",
            "\n",
            " <dig>  repeat step  <dig> to  <dig> till bs is stable.\n",
            "\n",
            " <dig>  to avoid bias, we repeat step  <dig> to  <dig> multiple times and choose the best result.\n",
            "\n",
            "RESULTS\n",
            "we evaluate intego <dig> on three model organisms  with different levels of go annotation scale and complexity  <cit> . for each of them, we use ec numbers and pathways as independent biological evidences for molecular function and biological process category in go respectively. finally, we test the robustness of intego <dig> by gradually removing seed measures with best performance.\n",
            "\n",
            "data preparation\n",
            "the go annotation and structure data were downloaded from the go website . the ec number and pathway information of human, arabidopsis and yeast were downloaded from the humancyc , plantcyc  and saccharomyces genome database  respectively. intego <dig> was implemented with python  <dig>  with networkx package .\n",
            "\n",
            "performance evaluation on molecular function\n",
            "proteins sharing the same ec numbers are considered to have similar molecular functions. for every manually curated pathway in human, arabidopsis and yeast, we grouped the genes based on their ec numbers  and tested the difference between the inter- and intra-group gene-gene similarities. there are in total  <dig>   <dig> and  <dig> ec groups with least three genes in human, arabidopsis and yeast respectively.\n",
            "\n",
            "in the experiments, we chose seven widely used measures in all the three categories as candidate measures. we also added a fake measure to simulate the situation where a wrong measure was included to test the robustness of intego <dig>  among the seven measures, simui  <cit>  and to  <cit>  measure use the go annotations information directly; resnik  <cit> , schlicker  <cit>  and simgic  <cit>  measure use annotation information to calculate the information content of go terms; wang  <cit>  measure considers the complex topology of go; hrss  <cit>  considers the shared path based on information content. more detail description is shown in additional file  <dig>  in the fake measure, a random half of the similarity scores were computed with resnik measure, and the other half were  <dig> or  <dig>  such that the similarity of two genes with the same ec is  <dig>  otherwise it is  <dig> .\n",
            "\n",
            "in order to evaluate intego <dig> systematically, we adopted the cross-validation strategy by randomly selecting 1/ <dig> of human ecs as the testing set  and the other 4/ <dig> of human ecs being the training set . the same training set was used for arabidopsis and yeast . using the training set, the parameters in eq.  <dig> were estimated, which were directly applied on the testing set to compute the ec-based logfc scores using eq.  <dig> \n",
            "\n",
            "we found that the parameters for the three types of seed measure groups  are significantly different, reflecting different integration strategies. the highest parameter in the high seed measure groups is maximum, in the low seed measure groups is minimum, and in the mix seed measure groups is simui measure.\n",
            "\n",
            "we compared the performance of intego <dig> with all the candidate measures, the average value of them and intego. figure  <dig> shows that intego <dig> performed the best among all the measures in all the three species. for example, the median, 75th and 25th percentile of logfc scores of intego <dig> on human were  <dig> ,  <dig>  and  <dig> , significantly higher than the seed measures it integrated  and supplementary table s <dig> in additional file 4). interestingly, the performance of intego <dig> was significantly higher than our previous measure intego, indicating that adding a weak measure has almost negligible effect to intego <dig>  but can significantly affect intego. comparing the logfc scores on every ec group using intego <dig>  intego and wang measure , we found that intego <dig> performed the best in all  <dig> ecs in the testing set, while intego and wang measure were being the best in  <dig> or  <dig> ecs only ). similarly, the median of logfc scores of intego <dig> in arabidopsis is  <dig> , which is  <dig> -fold higher than intego  and supplementary table s <dig> in additional file 4). intego <dig> performed the best in  <dig> of  <dig> ecs, while wang performed the best in  <dig> ecs ). we also evaluated intego <dig> on yeast which has richer information in go than human and arabidopsis. intego <dig> performed the best with the median logfc score being  <dig>   and supplementary table s <dig> in additional file 4). it was the best in  <dig> out of  <dig> total ec groups ).\n",
            "\n",
            "statistics analysis was carried out to test the significance of intego <dig> results. the p-values of t-test indicate that the results of intego <dig> are significantly different with the results of other measures except simgic, simui and wang measure on arabidopsis and yeast .\n",
            "\n",
            "performance evaluation on biological process\n",
            "given that genes annotated to the similar biological process may be involved in the same manually curated pathway, we grouped genes based on the pathway information, and on these gene groups we evaluated intego <dig>  there are in total  <dig>   <dig> and  <dig> pathways with at least two genes in humancyc, plantcyc and saccharomyces genome database respectively.\n",
            "\n",
            "the same logfc method  were used in the performance test. in human and arabidopsis, the median and 75th percentile of logfc scores of logfc scores were higher than other measures ,  and supplementary table s <dig> and s <dig> in additional file 4), indicating that integrating multiple gene similarity measures with intego <dig> could increase the overall performance. comparing the logfc scores from the intego <dig>  intego and wang measure for each pathway, figure  <dig> and  show that intego <dig> performs best in  <dig> of  <dig> pathways and  <dig> of  <dig> pathways on human and arabidopsis respectively. in yeast, the performance of intego <dig> is still the best. the median, 75th percentile and 25th percentile of logfc scores are  <dig> ,  <dig>  and  <dig> , which are significant higher than the second-best measure intego  and supplementary table s <dig> in additional file 4). in addition, intego <dig> performs best in  <dig> of  <dig>  yeast pathways ). although intego <dig> perform well in most datasets, its performance on arabidopsis is not good enough . the reason may be that all the result of seed measures are not good and very close to each other. therefore, the grouping process  in intego <dig> cannot select the appropriate seed measures from the seed measure. even though, intego <dig> also increase the performance of the similarity measures.\n",
            "\n",
            "statistics analysis was carried out to test the significance of intego <dig> results. the p-values of t-test indicate that the results of intego <dig> are significantly different with the results of other measures except simgic, simui and wang measure on arabidopsis .\n",
            "\n",
            "the results indicate that intego <dig> successfully utilizes the go information by integrating seed measures appropriately to better deliver functional similarities better genes.\n",
            "\n",
            "robustness of intego2\n",
            "to test the robustness of intego <dig>  we gradually removed a candidate measure  and then compute the logfc score. figure  <dig> shows that the performance reduced slowly by removing the first two measures . the median of logfc decreased less than  <dig>  after removing three best measures. this is because intego <dig> can select the most appropriate seed measures for each gene pair, since no measurement is suitable for every gene pair. to analysis the contribution of the different measures to the overall similarity, we applied leave-one-out measure on intego <dig>  the result shows that intego <dig> is overall robust to remove any integrated measure . the performance of intego <dig> decreases most after resnik measure is removed.\n",
            "\n",
            "performance evaluation on protein sequences\n",
            "in addition to use the logfc score as the evaluation criteria, we used protein sequence similarity as an independent evidence for further performance evaluation on the molecular function category  <cit> . in this experiment, the same human gene set in subsection \"performance evaluation on molecular function\" was used, and the sequence similarity scores ) were calculated with blast  <cit> . figure  <dig> shows that among all the go based semantic similarity measures, intego <dig> has the highest correlation score with the sequence based similarity with r-squared  <dig>  .\n",
            "\n",
            "generating functional association maps\n",
            "since intego <dig> computes gene-to-gene similarities more accurately than the tested existing measures, we computed the gene similarity scores for all the human, arabidopsis and yeast genes on both molecular function and biological process go categories, and generated a functional association map for each organism. as a demonstration, the human p <dig>  <cit>  gene functional association map  =  <dig> ) with  <dig> genes and  <dig> edges consists a tightly connected subgraph and several small or large but sparsely connected subgraphs . these networks provide a new platform for more advanced biomedical researches which could be beneficial in medical diagnostics.\n",
            "\n",
            "CONCLUSIONS\n",
            "the calculation of go-based gene functional similarity has already been widely applied  <cit> . however, since the existing measurements only use a subset of the go information , the demand to integrate these measurements is compelling.\n",
            "\n",
            "in this paper, we proposed a new integrative measure called intego <dig> by automatically selecting the most appropriate seed measures and by integrating the seed measures using an addition model. first, we calculate the ranked similarity scores using all the measures. second, seed measures are selected using a grouping process. third, the parameters of the addition model are estimated by optimizing an objective function on a training data. experimental results using ecs and pathways show that intego <dig> performs the best among all the measures. it also shows that intego <dig> is robust against the unavailability of candidate measures. note that we have proposed intego in the previous work to unify different measures  <cit> , which can be considered as a simplified case of intego <dig> \n",
            "\n",
            "to demonstrate the advantages of intego <dig>  we computed the gene similarity scores for all the human, arabidopsis and yeast genes on both molecular function and biological process go categories, and generated a functional association map for each organism. the new functional association maps, together with the existing biological networks, can be beneficial in medical diagnostics, and they also may provide more biological insights into gene function and regulation. in the future, we will apply intego <dig> to more organisms, data sets  and compare the new functional association maps with the existing biological network  to predict protein or genetic interaction based on the go similarity scores.\n",
            "\n",
            "competing interests\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "authors' contributions\n",
            "jc and yw designed the algorithm and experiments. jp and hl implemented the algorithm and finished the experiments. qj helped to design the algorithm to find the seed measure group.\n",
            "\n",
            "supplementary material\n",
            "additional file 1\n",
            "the effect of varying the least size of the seed measure group on intego <dig> performance. the x-axis is the least size of the seed measure group. the y-axis is the logfc scores. the top and bottom of the boxes represent 75th and 25th percentiles, red lines are the median, top and bottom whiskers represent greatest and lowest values except outliers. cross nodes represent outliers that are larger than the sum of 75th and  <dig>  interquartile range.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 2\n",
            "the effect of varying the threshold of the distance between genes in the seed measure group on intego <dig> performance. the x-axis is the threshold of the distance between genes in the seed measure group. the y-axis is the logfc scores. the top and bottom of the boxes represent 75th and 25th percentiles, red lines are the median, top and bottom whiskers represent greatest and lowest values except outliers. cross nodes represent outliers that are larger than the sum of 75th and  <dig>  interquartile range.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 3\n",
            "the description of the integrated measures. seven individual measures are described in this file. the reference papers of these measures are also listed.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 4\n",
            "supplementary tables. all the supplementary tables  are included in this file.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 5\n",
            "the effect of removing single integrated measure on intego <dig> performance. the x-axis is the individual measure removed. the y-axis is the logfc scores. the top and bottom of the boxes represent 75th and 25th percentiles, red lines are the median, top and bottom whiskers represent greatest and lowest values except outliers. cross nodes represent outliers that are larger than the sum of 75th and  <dig>  interquartile range.\n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements\n",
            "this project has been funded by the u.s. department of energy, grant no. de-fg02-91er <dig> to j.c; the national high technology research and development program of china grant  and the national natural science foundation of china grant  to y. w.\n",
            "\n",
            "declarations\n",
            "the publication costs for this article were funded by the corresponding author's institution.\n",
            "\n",
            "this article has been published as part of bmc systems biology volume  <dig> supplement  <dig>  2014: proceedings of the 25th international conference on genome informatics : systems biology. the full contents of the supplement are available online at http://www.biomedcentral.com/bmcsystbiol/supplements/8/s <dig> \n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this paper, we propose a new integrative measure called intego <dig>  by automatically selecting the most appropriate seed measures for each gene pair, and integrating the seed measures using a metaheuristic search method.\n",
            "in this paper, we present a new integrative measure called intego <dig>  by automatically selecting the most appropriate seed measures for each gene pair, and integrating the seed measures using a metaheuristic search method.\n",
            "in this paper, we present a new integrative measure called intego <dig>  by automatically selecting the most appropriate seed measures for each gene pair, and integrating the seed measures using a metaheuristic search method.\n",
            "in this paper, we present a new integrative measure called intego <dig>  by choosing the most appropriate seed measures for each gene pair from a pool of candidate measures, and integrating the seed measures using a metaheuristic search method.\n",
            "\n",
            "20\n",
            "[{'summary_text': 'the vno epithelium is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues during adulthood.\\nin this study, we investigated the cellular basis of adult neurogenesis in the vno of c57bl/ <dig> pregnant mice and their non-pregnant controls using high-throughput rna-sequencing and high-throughput rna-sequencing.\\nadult neurogenesis is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues into adulthood.\\nthe vno epithelium is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues into adulthood.\\nthe vno epithelium is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues into adulthood.\\nthe vno epithelium is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues into adulthood.\\nthe vno epithelium is a critical organ of the central nervous system.\\nin this study, we investigated the cellular basis of adult neurogenesis in the vno of c57bl/ <dig> pregnant mice and their non-pregnant controls using high-throughput rna-sequencing and high-throughput rna-sequencing.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the vertebrate olfactory system is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues during adulthood. this represents an exception within the central nervous system where neurogenesis is largely restricted to embryonic development and early postnatal stages. adult neurogenesis occurs almost exclusively in three specific areas, the hippocampal subgranular zone, the subventricular zone  and the olfactory epithelia . two of these regions, the svz and olfactory epithelia, supply new neurons directly to parts of the olfactory system in form of olfactory bulb interneurons and olfactory sensory neurons, respectively . within peripheral olfactory epithelia, the vomeronasal organ  conveys chemosensory information, such as pheromones and predator odors, that drive social and reproductive behaviors . vomeronasal sensory neurons  are constantly renewed throughout the life of the animal  <cit>  by neurons generated from stem cells located at the lateral and basal margins of the mature sensory epithelium . most of these margin newborn cells differentiate into mature vsns rarely undergoing apoptosis  <cit> , and project their axons to the accessory olfactory bulb   <cit> . whether these newly generated cells become functional vsns and thus are capable of transducing chemosensory cues has not been determined.\n",
            "\n",
            "in the olfactory epithelia, adult neurogenesis is often seen as a static, merely restorative process, not regarded as a real mechanism for plasticity. however, every aspect of adult-born cell production is carefully regulated and modulated, strongly suggesting that the olfactory system can tailor its production of new neurons to match the demands of its environment. newborn neurons in the vno derive from neural progenitors expressing mash <dig> and ngn <dig> genes  <cit> . during their proliferation phase, the majority of these progenitors express ki- <dig>  and proliferating cell nuclear antigen  proteins associated with the cell cycle  <cit> . postmitotic, immature vsns express markers associated with cytoskeletal remodeling such as doublecortin , β-tubulin iii, and gap <dig>  ncam/ocam adhesion molecules, and other lineage-related genes differentially expressed during the maturation process, including bcl11b/ctip <dig>  gnao <dig>  and gnai <dig>  <cit> . dcx is widely expressed in non-mitotic immature vsns and their axons during the whole vsn maturation period  <cit> . immature neurons migrate towards more superficial and central layers of the vno epithelium and mature into bipolar neurons characterized by the expression of olfactory marker protein  and vomeronasal receptors  of the v1r and v2r families  <cit> . proliferation of vsn precursors can be activated by a number of intrinsic signals, such as growth factors  <cit> , but also by external stimuli including exposure to urinary compounds  <cit> . however, it is not known whether adult neurogenesis in the vno operates in response to specific physiological conditions to modulate vno-dependent sensory function and behavior.\n",
            "\n",
            "adult neurogenesis in the brain has been shown to be regulated by a number of physiological, pathological, and environmental factors, including stress, synaptic activity, hormonal status, injury, and odor exposure  <cit> . hormonal changes that occur during pregnancy modulate forebrain and olfactory neurogenesis. prolactin, luteinizing hormone, and estradiol have been shown to promote neurogenesis and cell survival in the brain . this is important for the display of maternal behavior  <cit> , because olfactory discrimination and memory, both facilitated by adult-born neurons in the brain  <cit> , are critical for offspring recognition and care  <cit> . with few exceptions  <cit> , hormonal and pregnancy effects on vno neurogenesis have not been investigated, yet several lines of evidence associate sex hormones with modulatory effects on sensory function. hormones and hormone derivatives activate subsets of vsns  and regulate vsn sensory perception in female mice  <cit> . modulatory effects of sensory perception by motherhood and lactation have been recently reported in other systems. for example, hormonal changes associated with motherhood and pup care seem to enable plastic changes in sensory perception of the auditory system  <cit> .\n",
            "\n",
            "therefore, the aim of the present study was to elucidate the mechanisms controlling adult neurogenesis in the mouse vno. first, we performed a molecular characterization of the maturation phase of newly generated cells in the vomeronasal sensory epithelium. second, we evaluated the functional sensory properties of these cells. third, we found that neurogenesis in the vno is enhanced at the end of pregnancy. fourth, we determined that neuroblasts in the vno express estrogen receptor α  and that sustained high levels of the hormone estrogen enable a higher proliferation rate. these results show that neurogenesis in the vno epithelium can be modulated by an ovarian hormone, offering important insight to understand pregnancy-evoked changes in olfactory perception and behavior.\n",
            "\n",
            "RESULTS\n",
            "adult neurogenesis in the vomeronasal organ\n",
            "to investigate the cellular basis of adult neurogenesis in the mouse vomeronasal epithelium, we used a range of molecular markers  expressed at different stages of neuron generation and maturation: ki- <dig> and pcna, which are nuclear proteins present during dna synthesis in actively proliferating cells  <cit> ; dcx, which is expressed by neuronal precursor cells and immature neurons for 2– <dig> weeks after cell division  <cit> ; omp, which is expressed in all mature sensory neurons, and type- <dig> vomeronasal receptor , present in mature neurons of the basal layer of the vno  <cit> ; bromodeoxyuridine, 5-bromo-2’-deoxyuridine , a synthetic analog of thymidine that is incorporated into replicating dna, was used to label cells that were actively proliferating at the time of administration  <cit> . to visualize these markers in the vno epithelium we used either immunolocalization  or mouse strains expressing a fluorescent reporter under the control of a marker protein: dcx-dsred  <cit>  and omp-gfp mice  <cit>  . imaging on adult vno tissue slices revealed that progenitor cells expressing ki- <dig>  pcna, or brdu  are restricted to the marginal zones of the epithelium  whereas omp-expressing mature neurons localize in both marginal and central zones . this is consistent with previous studies identifying these marginal vno regions as active neurogenic sites  . immature neurons expressing dcx-dsred are predominantly distributed marginally although occupying a larger zone than ki-67/pcna-labeled cells , indicative of active migration to more central areas during cell maturation  <cit> . higher magnification analysis at single cell-resolution revealed that fractions of ki- <dig>  pcna, and brdu positive cells co-express dcx-dsred . however, cells positive for these early proliferation markers do not co-localize with v2r <dig> and omp+ cells . hence, dcx-dsred is widely expressed through neuron maturation from early proliferative stages to newly differentiated mature vsns.fig.  <dig>  neurogenesis and development of vomeronasal sensory neurons .  schematic of vsn maturation and markers used. ki-67/proliferating cell nuclear antigen -positive proliferating progenitors in the marginal zones of the vomeronasal organ  develop to immature doublecortin -expressing neurons and mature vsns in the central vno  expressing olfactory marker protein  and vomeronasal receptors . ki- <dig> is a nuclear antigen expressed during the whole cell cycle except at g <dig>  pcna participates in dna synthesis. dcx is a microtubule-associated protein expressed in immature neurons. omp is expressed in mature olfactory and vomeronasal sensory neurons. v2r <dig> antibody recognizes family-c v2rs expressed in most mature basal vno neurons.  coronal vno tissue slices labeled with antibodies against the early proliferation markers ki- <dig> and pcna  reveal expression on the vno marginal zones, whereas in maturity, the markers omp and v2r <dig>  localize on the center of the sensory epithelium on all or basal vsns, respectively. bottom panels: endogenous expression on omp-gfp and dcx-dsred transgenic mice in the vno. dotted lines show the limits of the sensory epithelium. scale bars, 50 μm.  double-labeling of ki- <dig>  pcna and brdu  immunostainings  with dcx-dsred in individual cells of the vno epithelial margins. scale bar, 10 μm.  triple-labeling with antibodies recognizing brdu  and omp  with the dsred  on vno coronal slices of dcx-dsred mice 14 days after injection with brdu. newborn vsns can be identified in an early maturity stage by double labeling of brdu and dsred  and in a later stage by triple labeling of brdu/dsred/omp . dotted lines show the limits of the sensory epithelium. scale bar, 25 μm.  fate-mapping analysis of brdu+ cells in the vno epithelium quantified at  <dig>   <dig>   <dig> and 21 days post injection  in vno coronal slices of dcx-dsred mice immunolabelled with brdu and omp, as shown in . omp+ cells increase, while dsred+ and omp– cells decrease over time. about 50 % of the newly generated cells reach a mature phenotype in 20 days. one-way anova: f <dig>  =  <dig> – <dig> ; ***p < <dig>  , *p < <dig>  , ***p < <dig>  , n =  <dig> mice per time point\n",
            "\n",
            "\n",
            "\n",
            "next, we analyzed the time course of vsn generation and maturation in adult dcx-dsred female mice. we injected mice with 100 mg/kg brdu and analyzed its incorporation in the vno of four animal groups:  <dig>   <dig>   <dig>  and 20 days post-injection . in addition to dcx-dsred reporter expression, vno sections were immunolabelled for brdu  and omp . brdu+ cells were classified in three developmental stages:  postmitotic progenitor cells that are positive for brdu and dsred, but negative for omp;  newly differentiated neurons positive for brdu, dsred, and omp; and  mature neurons positive for brdu and omp, but negative for dsred . cells positive for brdu and negative for dsred and omp probably account for either actively dividing progenitors or early differentiating glial cells. cell quantification revealed that at day  <dig> after brdu injection the most abundant brdu immunoreactivity  corresponds to postmitotic progenitors , indicating a rapid proliferation cycle. these cells decreased sharply after 7– <dig> days, and by day  <dig> represented only ~5 % of the total brdu+ cells. newly differentiated neurons  were nearly absent during the first day, but sharply increased to 28–35 % at 7– <dig> days, and were maintained stable  at 20 days. by contrast, fully mature neurons  were absent during the first 7 days, rapidly growing to 38 % after 14 days, and becoming the most abundant cell type  at 20 days . thus, most vsns maturate within a period of 14– <dig> days and only a very small fraction of progenitor cells remain undifferentiated during this time period.\n",
            "\n",
            "newly born vomeronasal cells give rise to functional sensory neurons\n",
            "we next asked whether newly generated cells become functional sensory neurons in the vomeronasal neuroepithelium. first, we studied the general morphology and axon conveyance of dcx-dsred+ cells. to provide a direct and simultaneous comparison of dcx and omp labeling, we crossed dcx-dsred mice with omp-gfp mice . utilizing en face confocal imaging in a vno whole-mount preparation  <cit> , we visualized vsn knobs at the dendritic tips of dxc-dsred/omp-gfp mice . the en face view of the vomeronasal epithelium reveals a homogeneous and punctate distribution of omp-gfp-expressing vsns . notably, dcx-dsred+ knobs were also present at dendritic tips of a substantial number of vsns . omp-gfp+ knobs were 5–10-fold more abundant than dcx-dsred+ knobs  and the level of overlap between omp-gfp and dcx-dsred represented ~12 % of gfp+ knobs . furthermore, imaging of vno axons and the aob nerve layer illustrates strong enrichment of dcx-dsred axons in both structures . dcx-dsred+ axons deriving from the nerve layer terminate into aob glomeruli that were identified by co-localization with omp-gfp . virtually all glomeruli in the anterior and posterior aob show dcx-dsred+ fibers. these data show that dcx-dsred+ vsns can extend their dendrites into the vno lumen and send axonal projections that reach the aob glomeruli.fig.  <dig>  proliferating progenitors in the vomeronasal organ  exhibit sensory neuron morphology.  imaging of knob layer on the luminal surface of the vno sensory epithelium of a doublecortin -dsred/omp-gfp double transgenic mouse . dsred+ knobs intermingle within the more mature population of gfp+ vomeronasal sensory neurons . scale bar, 20 μm. dsred+ axonal processes  are detected and extend towards the accessory olfactory bulb  together with mature gfp+ fibers. scale bar, 50 μm. medial view of the caudal part of the olfactory bulb  where the axons from the vno bend off laterally towards the aob. gfp expression in vsns is markedly lower than in the main olfactory epithelium and therefore axon bundles from the vno appear as substantially darker. in dsred images, vsn axon bundles are easily identified by their conspicuous parallel alignment. scale bar, 100 μm. sagittal view of aob glomeruli . dsred+ axons originating in the nerve layer terminate in several glomeruli , visualized by omp-gfp fluorescence. scale bar, 20 μm.  density measurements of dsred+ and gfp+ knobs reveal a sparser density of immature knobs compared to the mature population.  quantification of randomly sampled knobs shows that double-labeled gfp/dsred+ knobs represent  <dig>  % of the total gfp+ knob population. values in  and  were calculated using  <dig> areas from six dxc-dsred/omp-gfp mice; mean ± standard deviation  =  <dig>  ±  <dig>  dsred and  <dig>  ±  <dig>  omp-gfp knobs per 100 μm <dig>  dsred-gfp/gfp ratio =  <dig>  %,  <dig>  double positive knobs\n",
            "\n",
            "\n",
            "\n",
            "since maturating vsns have access to chemostimuli in the vno lumen and can deliver the olfactory information to the aob, we next asked whether these cells are functional sensory neurons capable of detecting chemosignals. we analyzed stimulus-induced activity in the vno of dcx-dsred mice using ratiometric ca2+ imaging on freshly dissociated vsns  <cit> . we compared the response pattern of dcx-dsred-positive vs. -negative cells in the same vsn preparation in response to three stimuli:  the major histocompatibility complex-peptide syfpeithi ;  fresh adult male c57bl/ <dig> urine ; and  its high molecular weight fraction , containing major urinary proteins  <cit> . these stimuli have been previously established as vsns stimuli  <cit> . activation by these three stimuli was detected in ~1–3 % of the  <dig>  cells screened, taken from  <dig> mice. importantly, all three stimuli induced similar levels of cell activation in both dcx-dsred+ and dcx-dsred– cells , suggesting that dcx-dsred+ cells exhibit functional sensory properties.fig.  <dig>  vomeronasal sensory neurons  activation by pheromonal compounds at different maturity stages.  dissociated vomeronasal organ  cells from doublecortin -dsred mice, either dsred+  and dsred– , show intracellular calcium transients in response to the major histocompatibility complex-derived peptide syfpeithi , whole male urine  and its high molecular weight fraction . both the number of activated cells  and the intracellular fura- <dig> fluorescent increases  indicate no differences in the response rate of dsred positive vs. negative cell populations for all three stimuli. example traces of ca2+ transients in single cells in response to the stimuli. high kcl  solution was delivered at the end of the experiment to assess cell viability.  post-hoc immunolabeling for ki- <dig>  v2r <dig> or endogenous omp-gfp in ca2+-imaged vno cells. representative images of vno cells taken with transmission light, bright field, fura- <dig> pseudocolor fluorescence 340/380 nm ratio , 488 nm excitation light of green-labeled secondary antibodies, or omp-gfp fluorescence , dsred fluorescence , and merged ex 488 nm/dsred color images . omp-gfp/dcx-dsred double transgenic mice were used to label omp-gfp cells. scale bar, 50 μm.  representative fura- <dig> ratio traces of dsred+ cells labeled with ki- <dig>  omp-gfp, or v2r <dig>  responses of cells positive vs. negative for ki- <dig>  omp-gfp, and v2r <dig> are shown.  percentage of activated dsred cells classified by marker expression. near-maximum response rates to the three stimuli are reached in cells positive for v2r <dig> and omp and negative for ki- <dig>  no responses were found in cells positive for ki- <dig>  cells negative for omp-gfp showed responses only to high k+\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "to assess a more precise maturity status of dcx-dsred + cells, we examined the expression of omp, v2r <dig>  and ki- <dig> using post-hoc immunostaining of responsive vsns directly in the ca2+ imaging recording chamber  <cit>  in independent parallel experiments. to determine the maturation stage, we classified the cells based on the combination of the specific markers at the time: cells positive for either omp or v2r <dig> or negative for ki- <dig> reached a late maturation stage, whereas cells positive for ki- <dig> or negative for omp and v2r <dig> were in an early stage . we imaged  <dig>  ,  <dig>   and  <dig>   cells taken from  <dig>   <dig>  and  <dig> mice in three parallel experiments and calculated the number of dcx-dsred+ cells activated by the three stimuli. we observed near-maximum rates of activation for syfpeithi, hmw, and whole urine in cells labeled with late maturity markers v2r <dig> and omp and negative for ki- <dig> . immature cells that were either negative for omp or positive for ki- <dig> showed no responses to any of the three pheromone stimuli. ki-67+ cells are a minority  within the total pool of dcx-dsred+, and therefore this lack or reduced responsiveness remains unnoticed when averaging all dcx-dsred+ responses . cells negative for v2r <dig> showed a 2- to 5-fold reduction in cell activity , suggesting intermediate maturation. stimulation with depolarizing high k+  solution confirmed the presence of voltage-dependent ca2+ channels in these cells . all cell groups exhibited activation to high k+  except for ki-67+ cells , confirming that neuron-like features are not yet fully developed in these cells. together, these results indicate that newly generated dcx-dsred+ vsns become chemosensitive neurons and that this process occurs at a late maturation stage subsequent to the expression of omp and v2r <dig>  but not in immature ki-67+ cells.\n",
            "\n",
            "vno neurogenesis is enhanced during pregnancy\n",
            "in mammals, adult stem cell division is physiologically stimulated during pregnancy in the forebrain svz and haematopoietic tissues which seem to favor the display of postpartum and maternal behaviors  <cit> . to test whether vno neurogenesis is upregulated during pregnancy, we examined the abundance of ki- <dig>  pcna, and dcx-dsred+ cells in the vnos of pregnant females at gestation days  19– <dig> and non-pregnant controls of the same age . we found that pregnant mice had significantly more cells positive for the three markers in the vno after 19– <dig> days of gestation , indicating a pregnancy-induced increase in neurogenesis. the increase on dcx-dsred+ cells occurred in the whole vno neuroepithelium with no obvious apical/basal gradients. female mice used in these experiments were previously exposed to adult males in order to be inseminated . to verify that contact to male odors does not influence vno stem cell proliferation, we exposed control-naïve females to male bedding for 20 days and counted the number of dcx-dsred cells . bedding-exposed females showed control levels of dcx-dsred cells in the vno  indicating that male-derived odors did not increase the generation of new vsns. we further determined whether high rates of immature neurons stay stable over time or decrease after parturition. we quantified the number of dcx-dsred+ cells 3 days after delivery  and observed no significant differences compared to controls , indicating that the number of dcx-dsred+ returns to pre-pregnancy levels after parturition.fig.  <dig>  pregnancy enhances vomeronasal organ  neurogenesis.  coronal vno sections immunolabelled with ki- <dig>  proliferating cell nuclear antigen , and endogenous doublecortin -dsred in 19/20-day pregnant females and control females of same age. dotted lines show the limits of the vno sensory epithelium. scale bars, 50 μm.  quantification of the densities of ki-67- and pcna-labeled cells, and relative abundance of dcx-dsred+ cells vs. total cells labeled with the nuclear dye dapi; 19/20-day pregnancy induced increase in immature neurons labeled with all markers. ki- <dig>  pcna, student’s t-test, *p < <dig> ; **p < <dig> . dcx-dsred, anova: f <dig>  =  <dig> ; *p < <dig> , **p < <dig> . the increase of dcx-dsred cells is not induced by exposure to male bedding for 20 days  and fades 3 days after delivery .  immunostaining of brdu+ cells in the vnos of control and gd <dig> pregnant mice injected with brdu at gd <dig>  scale bars, 50 μm.  quantification of brdu+ cells in the vnos of gd <dig> and gd <dig> pregnant mice and their non-pregnant controls injected with brdu at gd <dig>  the overall density of newborn cells marked with brdu decreases at gd <dig>  but not at gd <dig>  student’s t-test, **p < <dig> .  vno coronal sections labeled with ki- <dig> antibody at different gestation days . scale bar, 20 μm.  quantification of ki-67+ nuclei density in the vno at the different gestation days. two-way anova: f <dig>  =  <dig> ; **p < <dig> ; n = 34\n",
            "\n",
            "\n",
            "\n",
            "the increased neurogenesis could result from higher proliferation or, alternatively, from a higher cell survival rate. to distinguish between these possibilities, we injected females with brdu at gd <dig> and examined immunoreactivity at gd <dig> . vnos from pregnant females displayed lower numbers of brdu+ cells at gd <dig>  compared to non-pregnant controls , indicating a decrease in cell survival of cells produced 20 days earlier. therefore, the increase in neurogenesis  seems to be proportionally higher than the decrease of cell survival during late pregnancy . brdu+ cell numbers at day  <dig> were similar in both pregnant mice and controls , showing that proliferation is similar at this stage and mating has no major effect. together, these results suggest that the increase of immature vsns during pregnancy is caused by both higher proliferation and cell turn-over, but not by enhanced cell survival. this might be indicative of a cell-selective differentiation/survival process in pregnant females toward specific neuronal cell types.\n",
            "\n",
            "we next sought to establish when pregnancy modulates neurogenesis in the maternal vno by determining in which gestational period cell proliferation occurs. we therefore mated females with c57bl/ <dig> males  and collected maternal vnos at gd <dig>   <dig>   <dig>  and  <dig>  we measured vno cell proliferation following embryo implantation using immunodetection of ki- <dig> . in these experiments, we observed a significant increase in the number of ki-67+ cells only at gd <dig> . by contrast, maternal vnos that were collected at gd <dig>  gd <dig>  or gd <dig> showed no significant increase in ki-67+ cells compared to control female vnos .\n",
            "\n",
            "the vno transcriptomes of pregnant vs. non-pregnant females\n",
            "to further characterize the molecular and cellular processes occurring in the vno during pregnancy, we analyzed its transcriptome in pregnant  and non-pregnant control female mice via high-throughput rna-sequencing . we sequenced five pregnant and four age-matched control vno samples at high depth using the illumina hiseq platform . to ensure that our rna samples were captured from pregnant mice at the peak of vno neurogenesis, we first analyzed the levels of dcx expression . we found that three of the five pregnant females displayed ~ <dig> -fold increases in dcx expression compared to the control mice, consistent with the observed increase in neural proliferation . the other two females had no increase in dcx expression. interestingly, the embryos from these mice appeared to be 1– <dig> days younger than the other three litters, suggesting the development of their pregnancy was slightly delayed and the peak of neurogenesis had not yet been reached. we therefore excluded these two samples from our subsequent analyses.fig.  <dig>  pregnancy alters gene expression in the vomeronasal organ .  doublecortin  expression measured by high-throughput rna-sequencing  in control and pregnant mice. the two pregnant females  that did not display increased dcx expression compared to controls  were excluded. the three females that displayed increased neurogenesis  were used in further analyses. mean expression values are indicated by horizontal lines; the dotted pink line indicates the mean when all five pregnant mice are included.  differential gene expression of the whole vno transcriptome of pregnant mice compared to control mice. statistically significant differentially expressed genes  <5 %) are highlighted in red.  a scatter plot of relative vr gene and pseudogene expression levels  in the pregnant versus the control mice. statistically significant genes  are highlighted in red. the red line represents the 1: <dig> diagonal. each differentially expressed vomeronasal receptor  gene is detailed in fig.  <dig>   phylogenetic trees of all intact v1r and v2r genes, indicating the clades in which differentially expressed vr genes are located . filled circles indicated vr genes upregulated in pregnant females, open circles indicate vr genes downregulated in pregnant females\n",
            "\n",
            "\n",
            "\n",
            "to compare the transcriptomes of the remaining three pregnant females to the controls, we first performed a differential transcriptome-wide expression analysis. in total,  <dig> genes were identified as significantly differentially expressed  with a false discovery rate of 5 %;  <dig> of these were upregulated in the pregnant samples, whereas  <dig> were downregulated . a gene ontology analysis on the de genes revealed a significant enrichment of terms related to cell differentiation, regulation of transcription, cell proliferation, growth, and apoptosis, as well as central nervous system neuron development and axonogenesis . many of these de genes have been reported to interact  and form a regulatory network likely to underlie the observed alterations in proliferation and cell turn-over.fig.  <dig>  genes that are differentially expressed in the vomeronasal organ of pregnant females are predicted to interact in a network. a string  analysis of  <dig> of the  <dig> differentially expressed genes that encode proteins predicted to interact with at least one other. each protein is indicated by a circle, connecting lines indicate evidence of interaction. a red line indicates evidence of fusion; a green line is evidence of a shared neighborhood; a blue line is evidence of co-occurrence in phyletic profile; a purple line is experimental evidence of interaction; a yellow line is co-occurrence in published abstracts; a light blue line is co-occurrence in databases; a black line is evidence of co-expression\n",
            "\n",
            "\n",
            "\n",
            "we found no significant difference in the expression of the vr gene repertoire when considered together ; however, the vno contains hundreds of different sub-types of vsns, each defined by the expression of a given vr, or combination of vrs. if the neurogenesis or turn-over of neurons during pregnancy disproportionally affected specific vsn subtypes, the expression of the vr that define those sub-types may be altered relative to the others. to test this, we normalized for the number of total mature neurons in each sample using the expression of genes specifically expressed in mature vsns . we identified  <dig> vr genes that were significantly de; most  were proportionally upregulated in the vnos of the pregnant mice . from these,  <dig> are v1r genes and  <dig> are v2r genes , distributed across a number of vr subfamilies . together our results indicate that pregnancy induces observable differences in gene expression indicative of alterations in cell proliferation and cell turnover in the vno of pregnant mice. we find evidence of small biases towards a few subtypes of vsns, but the majority of the vsn repertoire remains unchanged.fig.  <dig>  the vomeronasal receptor genes and pseudogenes differentially expressed in pregnant mice. each of the  <dig> significantly differentially expressed genes from fig. 5c  are represented with a box-and-whisker plot of the rnaseq normalized counts  in control  and pregnant  mice\n",
            "\n",
            "\n",
            "\n",
            "estrogen stimulates vno neurogenesis\n",
            "neurophysiological signals that arise during pregnancy emerge in response to either embryo implantation or circulating maternal hormones. given that the neurogenesis increase appears in the last part of gestation, we reasoned that circulating hormones might have a more central role than implantation. using our rnaseq data, we analyzed the gene expression levels of hormone receptors in the vno . we observed high levels of expression of the progesterone receptors membrane-component  <dig> and  <dig> , androgen receptor , prolactin receptor , and estrogen receptor α . expression of the classical progesterone receptor , as well as estrogen receptor β , g-protein coupled estrogen receptor  <dig> , oxytocin receptor , luteinizing hormone/choriogonadotropin receptor , gonadotropin-releasing hormone receptor , and follicle-stimulating hormone receptor  was lower or not present . no significant hormone receptor expression differences were found in controls vs. pregnant females .fig.  <dig>  vomeronasal organ  neurogenesis is facilitated by estrogen.  a heat map of hormone receptor gene expression in the vno, from high-throughput rna-sequencing  data. expression values are in normalized counts. mean esr <dig> expression is  <dig> normalized counts and mean esr <dig> expression is  <dig> normalized counts, neither are differentially expressed between pregnant and control mice. pgrmc <dig>  pgrmc <dig>  progesterone receptors membrane-component  <dig> and 2; ar, androgen receptor; prlr, prolactin receptor; esr <dig>  estrogen receptor α; pgr, progesterone receptor; esr <dig>  estrogen receptor β; gpr <dig>  g-protein coupled estrogen receptor 1; oxtr, oxytocin receptor; lhcgr, luteinizing hormone/choriogonadotropin receptor; gnrhr, gonadotropin releasing hormone receptor; fshr, follicle stimulating hormone receptor.  immunolocalization of estrogen receptor α  in a wild type c57bl/ <dig> female vno coronal section. arrowheads indicate esr1+ cells at the lateral corners of the epithelium where neuronal progenitors congregate. nse, non-sensory epithelium; l, lumen. scale bar, 50 μm.  double labeling of vno epithelial margins with esr <dig> antibody and ki- <dig> antibody , doublecortin -dsred , and olfactory marker protein  antibody . vnos from c57bl/ <dig> females were used for ki- <dig> and omp staining. scale bar, 25 μm.  quantification of brdu+ cells in the vnos of ovariectomized dcx-dsred females after a 9-day treatment with either, vehicle , progesterone, or estradiol benzoate. one way-anova: f <dig>  =  <dig> ; **p < <dig> ; control vs. progesterone p =  <dig> ; n =  <dig> mice ,  <dig> mice , and  <dig> mice . brdu was administered 8 h before sacrifice\n",
            "\n",
            "\n",
            "\n",
            "the levels of prolactin, oxytocin, progesterone, and estrogen are known to be affected during pregnancy. prolactin and oxytocin remain at low levels during mid-pregnancy and rise sharply near the onset of parturition and lactation  <cit> , making them unlikely candidates for mediating the neurogenic response observed here. by contrast, circulating levels of two other hormones – progesterone and estradiol – are found at high levels during the second half of the pregnancy. progesterone circulating levels increase following mating and drop in the last 2– <dig> days before parturition, and estrogen levels rise and stay high during the last half of the pregnancy  <cit> . pgmrc <dig> has been shown to be expressed in mature vsns by immunohistochemistry  <cit> . esr <dig>  but not esr <dig>  protein and mrna expression has also been reported in the vno  <cit> . our rnaseq data revealed that esr <dig> is expressed in the vno of control and pregnant mice at similar levels and, on average, 8-fold higher than esr <dig> . we therefore analyzed the pattern of esr <dig> expression in the vno using esr <dig> immunoreactivity on wt c57bl/ <dig> female mice. consistent with previous studies  <cit> , esr <dig> was clearly detected in the non-sensory part of the vno epithelium . additionally, we detected high esr <dig> immunoreactivity at the lateral corners of the vno epithelium where neuronal progenitors congregate . double-labeling experiments showed that this esr <dig> immunoreactivity is co-localized with both proliferating ki- <dig> cells and immature dcx-dsred+ neurons . esr <dig> immunoreactivity was absent in mature omp+ neurons . these results strongly suggest that estrogen could exert its effect in the vno on neuronal progenitors by activation of esr <dig> \n",
            "\n",
            "to establish a functional link between high levels of circulating estrogen/progesterone present during late pregnancy and vno neurogenesis, we tested the effects of estrogen and progesterone treatment on stem cell proliferation in the vno. we performed six subcutaneous injections with estradiol benzoate  for a total period of 9 days into 8- to 10-week-old, ovariectomized dcx-dsred females. on the last day of hormone treatment we administered brdu and extracted the vnos after 8 h to monitor proliferation. a 9-day estradiol treatment increased brdu-labeled cells in the vno neuroepithelium, indicative of an increase in neurogenesis . this effect was highly specific as progesterone treatment  failed to induce a significant increase on the number of brdu+ cells, comparable to that in vehicle-treated females . thus, sustained high levels of circulating estrogen are sufficient to augment the proliferation of vno neural precursors in ovariectomized females.\n",
            "\n",
            "discussion\n",
            "our results demonstrate a remarkable degree of neural plasticity in sensory neurons of the vno during pregnancy and provide new insights into the molecular basis of this phenomenon. in a comprehensive approach, we used a wide range of molecular markers and high-throughput rna-sequencing to depict neurogenic events in the adult vno from early progenitor cells to mature sensory neurons. we show that newly generated cells can become mature neurons in a period of 14– <dig> days. these cells develop a mature neuronal morphology displaying dendrites and axonal connections with central brain targets. we demonstrate that neurons generated in the adult vno – after reaching a certain maturity stage characterized by the expression of omp and vrs – display sensitivity to natural chemostimuli, indicating that these cells are indeed functional sensory neurons. interestingly, the production of neural progenitors is stimulated in the vno of female mice during the final stage of pregnancy. employing transcriptome analysis in the vno of pregnant vs. control females, we observe differential expression of  <dig> genes, including  <dig> vrs and other genes involved in regulating cell proliferation and death. both pregnant and control female vnos are shown to express hormone receptors, of which estrogen receptor esr <dig> is prominently expressed in neural progenitor cells. consistent with an important role of esr <dig> in the neurogenic process, sustained high levels of estrogen are sufficient to increase cell proliferation in the vno epithelium.\n",
            "\n",
            "estrogen receptor participation – via esr <dig> receptor in particular – in cell proliferation and plasticity is well documented. estrogen affects hematopoietic stem cell renewal during pregnancy  <cit> , and neurogenesis in the hippocampus  <cit> . furthermore, estrogen promotes cell survival in bnst, mea  <cit> , and aob  <cit> , and has been implicated in neuron morphological plasticity  <cit> . the molecular targets of estrogen and its receptors capable of influencing neurogenesis are less clear. estrogen-induced transcriptional changes of mirnas are candidates to mediate vno neurogenesis. a large amount  of mirna transcription is found to be directly regulated by estrogen  <cit> , and many of these mirnas control olfactory neurogenesis in both the main olfactory epithelium and the vno  <cit> .\n",
            "\n",
            "the increase in new chemosensory neurons by pregnancy or estrogen likely has important functional consequences. the olfactory system is critical in the generation of maternal behaviors  <cit> . neurogenesis in central brain areas, particularly in the hippocampus and svz/aob, and its influence on parental behavior has been extensively studied . however, the behavioral consequences of neurogenesis in nasal sensory epithelia remain largely unexplored. the vno is specialized for recognizing social odors  that promote a variety of behavioral responses depending on age, gender, dominance, and hormonal status of the receiver or sender. for example, the same male major urinary protein pheromones that induce attraction in virgin females  <cit>  also provoke maternal aggression in lactating females  <cit> , likely as a consequence of hormone-induced plastic changes in the limbic system during pregnancy, parturition, and lactation. moreover, hormones may also modulate sensory signaling to induce behavioral changes, such as those provoked by progesterone, via directly silencing vsn responses to male major urinary proteins  <cit> . similarly, enhanced neurogenesis in the vomeronasal epithelium may have an impact on maternal behaviors displayed during and after pregnancy. in fact, it has long been hypothesized that an increase in vsn turnover during pregnancy could play a key role for the duration of memory formation in the context of the bruce effect, which is shorter in females undergoing pregnancy or when treated with estradiol implants  <cit> . the formation of new vsn-aob connections during pregnancy could provide a potential mechanism to increase the likelihood of pregnancy block in females mated with a previously exposed stud male, thus diminishing female olfactory memory.\n",
            "\n",
            "our data indicate that relative changes in gene expression of vrs at the end of pregnancy are rather small, affecting only 6 % of the approximately  <dig> intact vr genes. the majority of these genes were upregulated, suggesting that this effect may be a consequence of increased neurogenesis in pregnant mice. these data are consistent with an increase in the proportion of specific sub-types of vsns during pregnancy, perhaps those specialized in detecting maternal behavior-related pheromones. of the vr genes upregulated here, we found that the majority belong to the v1r family , and may therefore be considered as candidate receptors for mediating maternal behaviors. we cannot exclude that larger differences in receptor expression may appear at later stages, i.e. 1– <dig> weeks after parturition, once a larger number of newly generated neurons reach maturity. however, vno-dependent maternal behaviors are apparent near the time of delivery, arguing that the generation of mature vsns that detect maternal pheromones are likely to occur during late gestation. it is also possible that the proportion of vsns remains stable during pregnancy, and the differential expression we detect is a result of transcriptional upregulation of some vr genes, but not of others. in this case, the generation of new sensory neurons during pregnancy may be related to a more fundamental physiological response, such as an increase in sensitivity or establishment of new connections with aob targets. similarly, changes in smell perception reported by pregnant women have been largely classified as increases in sensitivity rather than smell distortions  <cit> . humans seem to lack a functional vno – vrs and transduction molecules are pseudogenized – thus narrowing a sensory target for hormone modulation to the main olfactory epithelium. further experiments must be conducted to determine whether a pregnancy-induced increase in neurogenesis also occurs in the main olfactory epithelium.\n",
            "\n",
            "methods\n",
            "mice\n",
            "animal care and experimental procedures were performed in accordance with the guidelines established by the animal welfare committees of the university of saarland and sanger institute. mice 6– <dig> weeks old were kept under standard light/dark cycle  with food and water ad libitum. dcx-dsred mice  were generated as described  <cit> . dcx-dsred mice were crossed with mice in which the coding region and part of the 3’ non-translated region of the omp gene was replaced by gfp as a histological reporter   <cit> . omp is an abundant cytosolic protein expressed in all mature vsns  <cit> . breeding established offspring that were heterozygous for dcx-dsred, gfp, and omp . c57bl/ <dig> mice served as additional controls.\n",
            "\n",
            "bromodeoxyuridine  administration\n",
            "singly-housed estrous females were mated overnight  with c57bl/ <dig> males. all females were in proestrus at the time of mating. injection of brdu was done in all females at the same time. mating was assessed by the presence of vaginal plugs. control females were kept isolated. brdu  was administered the next day  to female mice of both pregnant and control groups. two doses of brdu  were injected intraperitoneally within a 4-h interval.\n",
            "\n",
            "immunohistochemistry\n",
            "mice were anesthetized with ketamine  and xylazine  and perfused transcardially with  <dig>  % saline solution followed by 4 % paraformaldehyde in pbs. vnos were removed, postfixed for 2 h in 4 % paraformaldehyde, equilibrated overnight in pbs containing 30 % sucrose, embedded in oct , and snap-frozen in a dry ice/2-methylbutane bath. frozen tissue sections  were collected on glass slides  and stored at −80 °c until use. all immunocytochemical procedures were conducted at room temperature, except for tissue incubation in primary antibodies . sections were washed three times in pbs , incubated in blocking solution  for 1 h, and incubated overnight in blocking solution containing the primary antibody. the tissue was then washed thre times in pbs  followed by incubation in secondary antibody for 1 h. primary antibodies were: ki- <dig> , pcna , dcx , er-alpha , omp , v2r <dig> , brdu . secondary antibodies were: biotin conjugated horse anti-mouse , goat anti-rabbit , and rabbit anti-rat ; alexafluor  <dig> donkey anti-goat  and goat anti-rabbit . the incubation with biotin-conjugated secondary antibodies was followed by incubation with streptavidin alexa fluor  <dig> . for ki- <dig> and pcna immunoreactions, antigen retrieval was performed by heating the tissue slides to 70 °c in 2 % citrate buffer  for 10 min. for brdu, tissue slides were treated with 2 n hcl at 37 °c for 1 h, followed by ph neutralization with borate buffer  for 10 min. sections were mounted using vectashield mounting medium with dapi . fluorescence images were acquired on an olympus ix <dig> microscope. counts of the number of cells were evaluated blindly for each animal. brdu+ neurons  were quantified in the whole vno by averaging the counts from 8– <dig> non-consecutive sections per mouse.\n",
            "\n",
            "confocal laser scanning microscopy\n",
            "confocal images of dendritic knobs were taken with a leica sp <dig> clsm microscope. gfp was excited at 488 nm and dsred at 543 nm. emission was collected between  <dig> and 535 nm for gfp images and between  <dig> and 720 nm for illustration of dsred. the density of the dendritic knob endings at the surface of the sensory epithelium was determined manually from randomly selected areas of at least  <dig>  μm <dig> in whole-mount preparations of the laterally opened vno. confocal images of the aob of dcx-dsred/omp-gfp mice  were taken with a zeiss lsm  <dig> microscope. images are shown as maximum intensity projections , minimally adjusted in brightness and contrast using adobe photoshop elements  <dig> \n",
            "\n",
            "live-cell calcium imaging\n",
            "for ca2+ imaging of freshly dissociated vsns  <cit> , vno epithelium of adult dcx-dsred or dcx-dsred/omp-gfp mice of both sexes was detached from the cartilage and minced in pbs at 4 °c. the tissue was incubated  in pbs supplemented with papain  and dnase i , gently extruded in dmem  supplemented with 10 % fbs, and centrifuged at 100 × g . dissociated cells were plated on coverslips previously coated with concanavalin-a type v  and loaded with the ca2+ indicator fura-2/am . coverslips containing fura-2-loaded vsns were placed in a laminar-flow chamber  and constantly perfused with extracellular solution. ratiometric  fura- <dig> ca2+ imaging was performed using an olympus ix <dig> microscope equipped with a hamamatsu camera. image pairs were acquired at  <dig>  hz and analyzed using imagej . chemostimuli were delivered in random order through the bath for 30 s. post hoc immunostaining of vsns was performed directly in the recording chamber of the ca2+ imaging setup so that all responding cells could be easily identified by using the coordinates of acquired images  <cit> . virtually no cells were lost using this procedure. vsns were fixed using pbs containing 4 % paraformaldehyde , incubated for 10 min in blocking solution , incubated with anti-v2r <dig>  or anti-ki- <dig>  and then for 30 min at room temperature, and washed in blocking solution  followed by an incubation  in secondary antibody . no antigen retrieval was performed. cell nuclei were stained  with dapi .\n",
            "\n",
            "chemostimulation\n",
            "chemostimuli for ca2+ imaging were prepared fresh daily and diluted in hepes-buffered hank’s balanced salt solution  giving the following final concentrations: syfpeithi, 5 × 10− <dig> m; hmw fraction, 1: <dig> dilution; urine, 1: <dig> dilution; 90 mm kcl. fresh urine was collected daily from adult c57bl/ <dig> males . urine was size-fractionated to obtain hmw fraction from  <dig>  ml of urine by centrifugation  using microcon 10-kda molecular mass cutoff ultrafiltration columns . the centrifugation retentate was washed with  <dig>  ml of pbs three times and resuspended in  <dig>  ml of pbs. the major histocompatibility complex-peptide syfpeithi was initially dissolved in bicarbonate-containing bath solution -2-aminoethanesulfonic acid, 1 mm mgso <dig>  1 mm cacl <dig>  10 mm glucose] to give 100 μm stock solutions. further dilutions were made in hepes-buffered hank’s balanced salt solution.\n",
            "\n",
            "mating\n",
            "naïve dcx-dsred female mice were group housed  for 2 weeks prior to mating to enhance estrous cycle synchrony. females in proestrus were isolated and mated with an adult c57bl/ <dig> male for 12 h. the estrous phase was determined by vaginal smears  <cit> . mating occurrence was assessed by the presence of vaginal plugs 12 h after introduction of the male . the male was then removed from the cage for the rest of the pregnancy. females presenting plugs and that did not become pregnant  were used as controls. embryo implantation might occur within 72 h after mating. we therefore estimate a maximum error of −3 days in our pregnancy time line. for male bedding exposure, 150 ml of soiled c57bl/ <dig> adult male bedding was added every second day in the home cage of naïve, singly housed dcx-dsred adult females.\n",
            "\n",
            "rnaseq data processing and analysis\n",
            "vnos from pregnant and control females were dissected and immediately frozen on dry ice. rna was extracted using the rneasy mini kit  with on-column dnase digestion. mrna was prepared for sequencing using the truseq rna sample preparation kit . all samples were multiplexed together and sequenced on four lanes on the illumina hiseq  <dig> platform, to generate 100 bp paired-end reads. all data is available through the european nucleotide archive and the corresponding accession numbers can be found in additional file  <dig>  sequencing reads were mapped using star  <dig>  to the grcm <dig> mouse reference genome, annotation version  <dig> in the ensembl mouse genome database. the annotation file was modified to substitute the gene models for the olfactory and vomeronasal receptors by those described previously  <cit> , which include extended utrs and additional non-coding exons. the number of fragments aligned to each gene was counted using the htseq package, with the script htseq-count . any reads that map to multiple locations in the genome  were not counted towards the expression estimates since they cannot be assigned to any gene unambiguously. to compare the expression values across samples, raw count data was normalized to account for the depth of sequencing. size factors were calculated using deseq2’s function estimatesizefactorsformatrix, and raw counts were divided by the corresponding size factor for each sample. in order to analyze the molecular influence of neurogenesis on vr genes we used expression of dcx as criterion to determine the precise neurogenesis peak. pregnant females displaying less than  <dig> -fold increases in dcx expression compared to the control mice were not considered to be at the peak of neurogenesis and were therefore excluded from further analyses. to test for differential expression, deseq <dig> was used with standard parameters. genes were considered to be differentially expressed if they had an adjusted p value of ≤ <dig>  . to assess the relative proportion each vr gene contributes to the total vr repertoire, the geometric mean of three vsn-specific genes  was calculated. these genes display consistent ratios of expression across all samples, and were therefore used to calculate size factors to normalize the vr expression estimates . all vr genes were then tested for differential expression with deseq <dig>  providing the algorithm with size factors of  <dig> for all samples, to avoid further normalization. to find terms that are enriched in our list of de genes the over-/under-representation algorithm from genetrail  was used. the background provided were all those genes tested for differential expression. to assess whether the de genes form putative regulatory networks, string  was used with default settings, for the  <dig> de genes only. all normalized data and detailed results of the de and enrichment analyses can be found in the additional file  <dig> \n",
            "\n",
            "ovariectomy\n",
            "individually caged 6– <dig> week old mice were anesthetized with isoflurane . following a skin dorsal midline incision, caudal to the posterior border of the ribs, muscles of the posterior abdominal wall were separated to enter the abdominal cavity. blunt subcutaneous dissection, lateral to the skin incision, was made beneath the muscle layer to gain access to the ovaries and remove them. the uterine horn was reintroduced into the abdomen and the process was repeated bilaterally. the muscle incision was sutured and the skin incision closed using wound clips.\n",
            "\n",
            "hormonal treatments\n",
            "two weeks after ovariectomy, mice were injected with estradiol benzoate or progesterone as previously described  <cit> . briefly, either estradiol benzoate  or progesterone  were dissolved in sesame oil  and injected subcutaneously at days 1– <dig> and 6– <dig>  at the end of the hormonal treatment , two doses of brdu  were administered intraperitoneally with a 4-h interval between the two administrations.\n",
            "\n",
            "statistics\n",
            "independent student’s t-test was used for measuring the significance of difference between two independent distributions. multiple groups were compared using either one-way or two-way analysis of variance  with tukey’s test as a post hoc comparison. analysis was performed using origin <dig>   software. unless otherwise stated, results are presented as means ± sem.\n",
            "\n",
            "CONCLUSIONS\n",
            "in summary, our study contributes to determine the developmental profile of vsns and the modulation of neurogenesis by pregnancy, and directly demonstrates an effect of estradiol on this neurogenesis, which may serve to explain pregnancy-evoked changes in odor-guided behaviors. these results reinforce the view that factors including hormonal state and sensory input have the potential to finely tune sensory systems to adapt them to changing biological contexts.\n",
            "\n",
            "additional files\n",
            "additional file 1: \n",
            "excel workbook containing the raw data from fig. 4b, d\n",
            "and\n",
            "f. \n",
            "\n",
            "additional file 2: \n",
            "excel workbook containing the mapping statistics of the rnaseq data for each sample, along with the accession numbers for the raw data; the normalized counts for the whole transcriptome; the differential expression analysis between the pregnant and control samples; the gene ontology categories significantly overrepresented in the differentially expressed genes; the normalized counts for the vr genes, accounting for total vsn number; and the differential expression analysis on the vr genes only, after normalization for vsn number, between the pregnant and control samples. the column ‘length’ corresponds to the total exonic bases in all transcripts from the gene. for the differential expression sheets, the columns contain the following data: ‘basemean’, corresponds to the mean normalized expression value for the gene across all samples; ‘log2foldchange’, is the fold change between the pregnant and control samples, log <dig> transformed; ‘lfcse’, corresponds to the standard error associated with the fold change estimation; ‘stat’, is the wald statistic; ‘pvalue’ is the p value of the test; and ‘padj’ is the p value after adjusting for multiple testing . genes that have both their ‘pvalue’ and ‘padj’ set to na contain outliers; genes with only their ‘padj’ set to na were filtered prior to the test because their normalized counts were too low. \n",
            "\n",
            "\n",
            "\n",
            "abbreviations\n",
            "aobaccessory olfactory bulb\n",
            "\n",
            "brdubromodeoxyuridine, 5-bromo-2'-deoxyuridine\n",
            "\n",
            "dcxdoublecortin\n",
            "\n",
            "dedifferentially expressed\n",
            "\n",
            "esr1estrogen receptor α\n",
            "\n",
            "gdgestation day\n",
            "\n",
            "gfpgreen fluorescent protein\n",
            "\n",
            "hmwhigh molecular weight urine fraction\n",
            "\n",
            "mirnamicrorna\n",
            "\n",
            "ompolfactory marker protein\n",
            "\n",
            "pcnaproliferating cell nuclear antigen\n",
            "\n",
            "rnaseqhigh-throughput rna-sequencing\n",
            "\n",
            "svzsubventricular zone\n",
            "\n",
            "vnovomeronasal organ\n",
            "\n",
            "vrvomeronasal receptor\n",
            "\n",
            "vsnsvomeronasal sensory neurons\n",
            "\n",
            "competing interests\n",
            "\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "author contributions\n",
            "\n",
            "lo, tl-z, fz, and pc designed and initiated the study. lo, sk and apg performed immunohistochemistry and injections. as performed confocal knob and nerve imaging. mp analyzed confocal aob images. np and pc performed ca2+-imaging in vsns. xi-s and dwl prepared and analyzed the total vno rna-seq libraries. lo, apg, and pc analyzed data. all authors contributed to the preparation of the manuscript.\n",
            "\n",
            "we thank peter mombaerts for supplying omp-gfp mice, johanna roth for technical and animal care support, and matthieu keller for critical reading of the manuscript. this work was supported by deutsche forschungsgemeinschaft grants ch 920/2- <dig>  and sonderforschungsbereich  <dig> projects a <dig>  and a <dig> , a university of saarland homforexzellent grant , the volkswagen foundation , and the wellcome trust grant  <dig> . tl-z is a lichtenberg professor of the volkswagen foundation.\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the vno epithelium is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues during adulthood.\n",
            "in this study, we investigated the cellular basis of adult neurogenesis in the vno of c57bl/ <dig> pregnant mice and their non-pregnant controls using high-throughput rna-sequencing and high-throughput rna-sequencing.\n",
            "adult neurogenesis is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues into adulthood.\n",
            "the vno epithelium is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues into adulthood.\n",
            "the vno epithelium is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues into adulthood.\n",
            "the vno epithelium is characterized by a distinctive plastic capacity of cell renewal and axon rewiring that continues into adulthood.\n",
            "the vno epithelium is a critical organ of the central nervous system.\n",
            "in this study, we investigated the cellular basis of adult neurogenesis in the vno of c57bl/ <dig> pregnant mice and their non-pregnant controls using high-throughput rna-sequencing and high-throughput rna-sequencing.\n",
            "\n",
            "21\n",
            "[{'summary_text': 'in this work, we have identified  <dig> transcription factors that regulate lin- <dig> expression in the embryo and larva that we previously identified in drosophila.\\nin addition, we have identified  <dig> transcription factors that directly regulate lin- <dig> expression in the larva, including the orphan nuclear receptor nhr- <dig>  the hypodermal cell fate regulator lin- <dig>  and the gata factor elt- <dig>  these transcription factors may play a role in the initiation of lin- <dig> expression in the vulval lineage.\\nin addition, we have identified  <dig> transcription factors that directly regulate lin- <dig> expression in the larva, including lin- <dig>  the orphan nuclear receptor nhr- <dig>  the hypodermal cell fate regulator lin- <dig>  the gata factor elt- <dig> and the gata factor elt- <dig>  these transcription factors may play a role in the initiation of lin- <dig> expression in the larva.\\nin addition, we have identified  <dig> transcription factors that directly regulate lin- <dig> expression in the larva.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "hox genes encode evolutionarily conserved homeodomain-containing transcription factors that pattern cells and tissues along the anterior–posterior body axis during metazoan development . hox proteins perform this function by serving as master regulators of expression of batteries of genes that impart identity to a cell  <cit> , and the precise regulation of hox protein activity is therefore vital for proper development. due to their central and conserved role in regional identity and fate specification during metazoan development, the mechanisms underlying the initiation, maintenance and modulation of hox gene expression have been intensively studied  <cit> . in drosophila, an elaborate network of maternally supplied and zygotically expressed transcription factors act to initiate hox gene expression properly in the syncytial early embryo, while in the cellularized vertebrate embryo, secreted signaling molecules and growth factors are employed to coordinate precise hox gene expression in time and space  <cit> . in both vertebrates and drosophila, once hox gene expression is initiated, it is maintained by the trithorax and polycomb groups of chromatin regulatory proteins  <cit> , and can be further modulated by extracellular signaling pathways, hox protein autoregulation and cross-regulation, and other mechanisms  <cit> .\n",
            "\n",
            "as in other metazoans, hox genes are essential during the development of the nematode caenorhabditis elegans <cit> . c. elegans has only six hox genes, present in a dispersed cluster  <cit> . three hox genes, ceh- <dig>  nob- <dig>  and php- <dig>  are required for proper embryonic development  <cit> . the other three hox genes, lin- <dig>  mab- <dig>  and egl- <dig>  appear to be required only during post-embryonic development, however their expression begins in the embryo  <cit> . very little is known about the initiation of hox gene regulation in c. elegans in the embryo  <cit> , although subsequent regulation of hox gene expression in larval development by polycomb and trithorax-related proteins, signaling pathways, other transcription factors, micrornas, and hox proteins themselves, have all been noted  <cit> . elucidating the mechanisms by which hox gene expression is initiated and regulated in nematodes will broaden our understanding of this important class of developmental regulators across a larger range of animal phyla, giving us further insight into their use during the evolution of animal diversity and their function in gene regulatory networks controlling pattern formation.\n",
            "\n",
            "our laboratory and others have studied the function of the hox gene lin- <dig> during nematode larval development, in particular during formation of the vulva, which is part of the hermaphrodite egg-laying apparatus. vulval development begins in the first larval stage  when the twelve ventral hypodermal blast cells, p1–p <dig> , divide to generate posterior daughters known as pn.p cells  <cit> : the central six pn.p cells, p <dig> p–p <dig> p, become vulval precursor cells   <cit> . during the third larval stage  the action of wnt, ras and notch extracellular signaling pathways induces the vpcs to adopt distinct cell fates in the pattern 3°-3°-2°-1°-2°-3°, where the cells p <dig> p - p <dig> p adopt 1° and 2°  fates and divide to generate  <dig> cells that form the vulval opening, while p <dig> p, p <dig> p and p <dig> p adopt the non-vulval 3° fate, which is to divide once and fuse with the surrounding syncytial hypodermis . the hox gene lin- <dig> encodes a deformed/sex combs reduced ortholog expressed in the midbody region, including the six vpcs  <cit> . lin- <dig> acts twice during vulval development. lin- <dig> is first required to generate the vpcs; in lin- <dig> null mutants, the vpcs fuse with the hypodermis during the l <dig> stage, causing a vulvaless  phenotype  <cit> . little is known about the regulation of lin- <dig> expression at this time in development. lin- <dig> is also required at the time of vpc fate specification in the l3; loss of lin- <dig> activity at this time leads to defects in vpc fate specification  <cit> . at this later time, lin- <dig> acts downstream of rtk/ras and wnt extracellular signaling pathways  <cit> .\n",
            "\n",
            "trans-acting factors regulating lin- <dig> expression have been identified previously using a variety of methods including forward and reverse genetic analysis, evolutionary conservation, and transgenic reporter analysis. trans-acting factors regulating lin- <dig> expression during vulval development include the rtk/ras pathway transcriptional effectors lin- <dig> and lin- <dig>  <cit> , the gli family member tra- <dig> which acts downstream from the sex determination pathway  <cit> , the zinc finger protein sem- <dig>  <cit> , the novel protein lin- <dig>  <cit> , several chromatin regulators  <cit> , and lin- <dig> itself  <cit> . direct binding to sites within the lin- <dig> genomic region has been established in the case of lin- <dig> , lin- <dig>  lin- <dig> and tra- <dig>  <cit> .\n",
            "\n",
            "we previously undertook to identify cis-acting sites regulating lin- <dig> expression  <cit> . due to the large size of the lin- <dig> genomic region  we inserted fragments of lin- <dig> genomic dna upstream of an enhancerless gfp reporter. by that method, we identified three functional elements: a 340 bp upstream fragment  that directs expression in p cells in the embryo and in their larval descendants including the vpcs, a 247 bp site from the first lin- <dig> intron that directs expression in larval ventral cord neurons, and a  <dig>  kb promoter fragment  that drives expression in p <dig> p at the time of vulval induction. expression from the last element is dependent on ras pathway function and we showed that lin- <dig>  lin- <dig> and lin- <dig> directly bind this cis-regulatory module. using an alternative approach, kuntz et al. used phylogenetic analysis to identify a number evolutionarily-conserved regions in the lin- <dig> genomic region, and showed that several of these sites also drove expression in certain cells or tissue when inserted upstream of an enhancerless gfp reporter  <cit> . these approaches both require that the site in question be able to mediate proper transcription activation of the reporter gene in vivo, and this requirement may lead to false negative results. to circumvent this issue, here we use the yeast one-hybrid assay, in which transcription factors that directly bind to a site of interest are identified, and then the function of these factors in gene regulation in vivo can be assayed  <cit> .\n",
            "\n",
            "in our previous analysis of the lin- <dig> genomic region, we identified a number of short dna sequences in the lin- <dig> region that were strongly conserved between c. elegans and two other nematode species . starting with these evolutionarily conserved elements and the 340 bp jw <dig>  fragment, we used manual  <cit>  and robotically-assisted “enhanced y1h”   <cit>  screens to identify  <dig> c. elegans transcription factors that bind to specific lin- <dig> genomic dna fragments. seven of these factors were characterized further to determine a role in lin- <dig> regulation during vulval development. we found that the orphan nuclear receptor nhr- <dig>  the hypodermal fate regulator lin- <dig> and the gata factor elt- <dig> positively regulate lin- <dig> expression in the embryo and may play a role in initiation of lin- <dig> in the vulval lineage. in the larva, nhr- <dig>  lin- <dig>  elt- <dig>  the zinc finger proteins ztf- <dig> and bed- <dig>  and the t box factor tbx- <dig> positively regulate lin- <dig> expression in the vpcs. interestingly, we previously showed that the adjacent gata factor genes egl- <dig> and elt- <dig> are downstream targets of lin- <dig> in the larva vpcs  <cit> . combined with our current result that elt- <dig> binds to and regulates lin- <dig> expression in the embryo, this suggests that egl-18/elt- <dig> and lin- <dig> may form a positive feedback loop to initiate and maintain lin- <dig> gene expression during embryonic and larval life to ensure proper vpc fate specification.\n",
            "\n",
            "RESULTS\n",
            "identification of transcription factors that bind to lin- <dig> genomic regions using yeast one-hybrid screens\n",
            "we previously identified cis and trans-acting factors that regulate lin- <dig> expression in response to rtk/ras signaling in the larval vpcs  <cit> . to further understand hox gene lin- <dig> regulation in c. elegans, we sought to identify transcription factors that bind to and regulate lin- <dig> expression, with an emphasis on 1) regulation in the vulval precursor cells p <dig> p - p <dig> p  and their descendants in the larva, and 2) expression in the precursors to the vpcs, the p cells p <dig> - p <dig>  in the embryo. we used the yeast one-hybrid approach, in which dna sequences from the gene of interest are used as ‘bait’ to screen for factors that can bind these sequences in yeast leading to activation of reporter gene expression  <cit> . previously, in addition to identifying large genomic regions that drove gfp reporter expression in lin-39-expressing cells, we also identified  <dig> short dna elements  located upstream, downstream and in introns that are conserved in lin- <dig> from the species c. elegans, c. briggsae and c. remanei <cit>  . reasoning that some of these evolutionarily conserved regions  may be binding sites for transcription factors that regulate lin- <dig> expression, we used twelve small lin- <dig> genomic regions  that each encompassed one or more ecrs in yeast one-hybrid screens . we also used the element jw <dig> , a 340 bp fragment found  <dig>  kb upstream of lin- <dig> that drives gfp expression in the p cells of the embryo  <cit> , as this fragment may bind factors responsible for initiation of lin- <dig> expression . these thirteen dna fragments, which together represent ~ 13% of the genomic region between lin- <dig> and its neighboring genes, were used as separate ‘baits’ .\n",
            "lin- <dig> \n",
            "genomic dna in yeast one-hybrid assays\n",
            "\n",
            "the table indicates the  <dig> fragments  used in yeast one-hybrid screens, the evolutionarily-conserved regions  each contains, the size of the fragment used, and the transcription factors identified as binding to each fragment.\n",
            "\n",
            "we used two yeast one-hybrid assay procedures. one screen was performed by transformation of the thirteen bait strains with a library of  <dig> plasmids that each express one c. elegans transcription factor fused to the activation domain from the yeast transcription factor gal <dig>  <cit> . two other screens were performed using a robotically-assisted mating assay in which the thirteen haploid bait strains were mated to a collection of  <dig> strains, each of which expresses a single c. elegans transcription factor fused to the gal <dig> activation domain  <cit> . factors identified as positive from the primary screens were retransformed manually and tested a second time; only factors that showed a reproducible interaction in yeast were considered true positives. in total,  <dig> interactions were found in yeast between  <dig> transcription factors and eight different fragments : nhr- <dig> , alr- <dig> , and ztf- <dig>  were identified through the library transformation screen, while  <dig> factors were identified via the robotically-assisted mating screens: odr- <dig>  tbx- <dig> and tbx- <dig> ; lin- <dig> ; tbx- <dig>  tbx- <dig> and egl- <dig> ; flh- <dig> and nhr- <dig> ; tbx- <dig> and the protein encoded by b <dig>  ; bed- <dig> and flh- <dig> ; dmd- <dig> and elt- <dig>  .\n",
            "\n",
            "we describe here our characterization of seven transcription regulators  chosen because these factors had either known expression in lin-39-expressing cells or a phenotype affecting a lin-39-regulated process  <cit> , or because our preliminary data showed an effect on lin- <dig> reporter expression in vivo. for each factor we carried out the following analyses. to validate the yeast interactions we expressed and purified each factor from bacteria and assayed binding to the appropriate sites in vitro . to determine if these factors regulate lin- <dig> expression in vivo in the vulval precursor cells during larval life, we reduced function for each transcription factor and examined lin- <dig> expression using an integrated transcriptional lin-39::gfp reporter  that contains 250 kb of genomic dna around the lin- <dig> locus  <cit>  . we also examined lin- <dig> expression in rnai-treated and mutant animals at the l <dig> stage by qrt-pcr . the strain containing the integrated lin-39::gfp reporter was also used to examine the effects of reduced transcription factor activity on lin- <dig> expression in the embryonic p cells p <dig> - p <dig>  which divide in the larva to generate lin-39-expressing vulval precursor cells and ventral cord neurons  <cit>  . finally, we examined the effect of reduction of factor function in a lin-39-sensitized background on the fusion of the vulval precursor cells at the l <dig> stage . below we describe our results that show that six of the factors identified in the yeast one-hybrid screens regulate lin- <dig> expression in the larval vulval precursor cells, while three of them also regulate lin- <dig> expression in the embryo.\n",
            "lin-39::gfp \n",
            "expression in embryonic p cells in transcription factor rnai animals\n",
            "\n",
            "\n",
            "nhr-43\n",
            "\n",
            "alr-1\n",
            "\n",
            "ztf-17\n",
            "\n",
            "lin-26\n",
            "\n",
            "tbx-9\n",
            "\n",
            "bed-3\n",
            "\n",
            "elt-6\n",
            "\n",
            "egl-18\n",
            "smg-1; lin-39::gfp animals were grown on bacteria expressing dsrna for each transcription factor or the control vector and gfp expression in the p cells  of embryos laid by these animals was examined. the percentage of animals having a wild-type intensity of gfp in all four p cells based on visual observation is indicated. ‘n’ = number of embryos scored. ‘*’ indicates p <  <dig>  compared to control .\n",
            "pjw <dig> ::gfp \n",
            "expression in \n",
            "elt- <dig> \n",
            "and \n",
            "egl- <dig> \n",
            "mutants in embryonic p cells\n",
            "\n",
            "elt-6\n",
            "egl-18\n",
            "\n",
            "elt-6\n",
            "\n",
            "elt-6\n",
            "egl-18\n",
            "\n",
            "egl-18\n",
            "\n",
            "egl-18\n",
            "elt-6\n",
            "\n",
            "egl-18\n",
            "\n",
            "egl-18\n",
            "elt-6\n",
            "animals containing pjw <dig> ::gfp in either wild type, elt- <dig> mutant  or egl- <dig> mutant  backgrounds were synchronized and given the indicated rnai treatment. pjw <dig> ::gfp expression in p <dig> - p <dig> was scored at bean stage in the embryos laid by these animals. ‘n’ = number of embryos scored. the percentage of animals having a wild-type intensity of gfp in all four p cells based on visual observation is indicated in column four. ‘*’ indicates p <  <dig>  compared to control .\n",
            "lin- <dig> \n",
            "sensitized background affects vpc fusion\n",
            "\n",
            "\n",
            "lin- <dig> fv\n",
            "\n",
            "lin- <dig> nhr-43\n",
            "\n",
            "lin- <dig> alr-1\n",
            "\n",
            "lin- <dig> ztf-17\n",
            "\n",
            "lin- <dig> lin-26\n",
            "\n",
            "lin- <dig> tbx-9\n",
            "\n",
            "lin- <dig> bed-3\n",
            "\n",
            "lin- <dig> elt-6\n",
            "lin-39; wis <dig> animals were grown for two generations at 25° on e. coli expressing dsrna targeting the indicated gene . larvae at the mid l <dig> stage  were examined for ajm-1::gfp expression in the vulval precursor cells. wild type animals have either  <dig> or  <dig> vpcs with ajm-1::gfp expression at this time . the percentage of animals showing the wild type pattern is shown . ‘*’ indicates p <  <dig>  compared to feeding vector  control .\n",
            "\n",
            "orphan nuclear hormone receptor nhr- <dig> positively regulates lin- <dig> expression in the embryo and larva\n",
            "the nhr- <dig> gene encodes an orphan nuclear hormone receptor, and nhr-43::gfp reporter expression is present from late embryo to adult in hypodermis, excretory cells, posterior intestine cells, and two head neurons  <cit> . in yeast, nhr- <dig> bound to yf <dig>  a 342 bp fragment located  <dig>  kb upstream of lin- <dig> , and this binding was verified in vitro using nhr- <dig> protein purified from e. coli . some nuclear hormone receptors are known to bind sites containing the sequence tgac  <cit> , and there is a tgac site in yf <dig>  while the wild type yf <dig> sequence competed for nhr- <dig> binding in vitro, the same yf <dig> fragment with the tgac site mutated no longer competed effectively , suggesting that nhr- <dig> may bind to yf <dig> through this putative nhr binding site.\n",
            "\n",
            "when rnai was performed on worms carrying the lin-39::gfp transcriptional reporter to reduce nhr- <dig> function, their progeny showed a decrease in the number of animals expressing wild type levels of gfp in the vulval precursor cells at the l <dig> stage . qrt-pcr on nhr- <dig> mutant animals  <cit>  at the l <dig> stage also showed a decrease in lin- <dig> expression compared to control animals . we also found that fewer embryos derived from hermaphrodites treated with nhr- <dig> rnai showed lin-39::gfp expression in p <dig> -  <dig> compared to control animals , indicating that nhr- <dig> positively regulates lin-39::gfp in the embryonic p cells. this embryonic defect could explain the reduction in gfp expression in the vpcs in the larva, however a decrease in lin-39::gfp expression was also observed if nhr- <dig> rnai was performed on l <dig> worms and then the same animals were observed in the l <dig> stage  animals showed wild type expression in all vpcs versus 76% of rnai control, p <  <dig> ). in summary, orphan nuclear hormone receptor nhr- <dig> binds to a site located far upstream of the lin- <dig> initiation codon, and nhr- <dig> function is required for wild type levels of lin- <dig> expression in the embryo and larva in cells that will participate in vulval development.\n",
            "\n",
            "the aristaless homolog alr- <dig> may regulate larval lin- <dig> expression\n",
            "alr- <dig> encodes a factor homologous to the products of the drosophila aristaless and mouse arx  genes, and antibody staining and gfp reporter analysis have shown alr- <dig> expression in multiple neurons and hypodermal cells from embryo to adult  <cit> . in yeast, alr- <dig> bound to yf <dig>  a 311 bp dna fragment located  <dig>  kb upstream of lin- <dig> containing the evolutionarily conserved sequence ecr <dig> . recently, the modencode project identified genomic binding sites for a number of c. elegans transcription factors, including alr- <dig>  using chromatin immunoprecipitation from larval animals  <cit> . examination of this data shows binding of alr- <dig> to multiple sites in the lin- <dig> promoter region, including the ecr <dig> site. bacterially expressed alr- <dig> protein bound to a 40 bp dna fragment encompassing ecr <dig> in vitro, and this binding was not competed by  <dig> fold excess of an oligonucleotide of the same base composition and length but scrambled sequence . therefore, alr- <dig> binds a site in the upstream region of lin- <dig> in both in vitro and in vivo assays.\n",
            "\n",
            "we did not observe a significant effect of alr- <dig> rnai on lin- <dig> or lin-39::gfp expression in either the embryo or larva in the cells that will give rise to the vulva . however, we did observe an effect of alr- <dig> rnai on a lin- <dig> mediated process in the larval vpcs. during wild type development, the cells p <dig> p - p <dig> p are born in the l <dig> stage and because they express lin- <dig> they do not fuse with the surrounding hyp <dig> syncytium as more anterior and posterior pn.p cells do  <cit> . later, in approximately 50% of animals, p <dig> p can fuse with hyp <dig> in the l <dig> stage  <cit> . thus wild type animals have either five or six vpcs at the time of vulval induction in the late l <dig> stage. in lin- <dig> mutant animals, or in animals in which wnt signaling is compromised, additional vpcs are seen to adopt this fused fate  <cit> . to examine the role of potential lin- <dig> regulators in this process we used rnai to reduce their function in a sensitized strain containing the hypodermal junction marker ajm-1::gfp and the temperature sensitive mutation lin- <dig> and examined the fusion of the vpcs at the l <dig> stage as before  <cit> . we found an increased number of larva with vpc fusion defects in lin- <dig> alr- <dig> compared to lin- <dig> control animals . therefore, although we were unable to detect an effect of loss of alr- <dig> function on lin- <dig> expression in the vulval cells, we did see a weak effect on a lin- <dig> dependent process, leaving open its role as an in vivo regulator of lin- <dig> expression.\n",
            "\n",
            "the zinc finger protein ztf- <dig> positively regulates lin- <dig> expression\n",
            "ztf- <dig> encodes a zinc finger transcription factor, and a ztf-17::gfp transcriptional reporter is expressed in head muscle, pharynx, and the ventral nerve cord  <cit> . in yeast, ztf- <dig> bound to yf <dig>  a 372 bp dna fragment located 2 kb upstream of lin- <dig> that contains ecrs 7– <dig> . to identify a smaller ztf- <dig> binding region, we divided yf <dig> into smaller fragments, and showed by one-hybrid assays that ztf- <dig> interacts with yf4- <dig>  a 110 bp subfragment that overlaps ecr <dig> . in vitro binding and competition assays with bacterially-expressed ztf- <dig> protein validated this result, showing that ztf- <dig> binds to fragment yf4- <dig>  but not the adjacent fragment yf4- <dig> figure 1c and d).\n",
            "\n",
            "ztf- <dig> rnai performed on lin-39::gfp worms caused a decrease in the number of progeny animals showing wild type levels of lin-39::gfp expression in the vpcs, , and qrt-pcr analysis of these ztf- <dig> animals showed a statistically significant decrease in lin- <dig> at the l <dig> stage . lin-39::gfp expression in the vpc parent cells, p <dig> - p <dig>  was not altered in embryos derived from ztf- <dig> mothers , suggesting that ztf- <dig> may be a larval regulator of lin- <dig> expression. in summary, in vitro and yeast one-hybrid analyses indicate that the zinc finger protein ztf- <dig> binds to a 110 bp fragment located upstream of lin- <dig>  and positively regulates lin- <dig> expression in the vulval precursor cells during larval life.\n",
            "\n",
            "lin- <dig>  which is required for maintenance of hypodermal cell fates, may positively regulate lin- <dig> expression in the embryo and larva\n",
            "lin- <dig> encodes a zinc-finger protein which is expressed in all hypodermal cells based on antibody staining  <cit> . in lin- <dig> mutants, cell fate transformations from hypodermal to neuronal fate occur in many cells, including the vpcs  <cit> . lin- <dig> also bound to yf <dig> in the yeast one-hybrid assay , and further analysis showed an interaction with yf4- <dig>  a 103 bp dna fragment that is distinct from the fragment bound by ztf- <dig> . we divided fragment yf4- <dig> into two smaller fragments, and in vitro dna binding and competition experiments showed that lin- <dig> bound to yf4-3- <dig>  a 51 bp dna fragment that does not contain an evolutionarily conserved region .\n",
            "\n",
            "to assay lin- <dig> regulation of lin- <dig> expression in vivo, lin- <dig> rnai was performed on newly hatched lin-39::gfp larvae and gfp levels in vpcs p <dig> p-p <dig> p were examined at the l <dig> stage in the same animals; this ‘l <dig> feeding’ was performed because of the embryonic lethal phenotype caused by lin- <dig> rnai. lin- <dig> rnai caused a weak but significant decrease in expression of lin-39::gfp in p <dig> p and p <dig> p at the l <dig> stage , and qrt-pcr performed on lin- <dig> mutant animals  <cit>  also showed a small but significant decrease in lin- <dig> at the early l <dig> stage . finally, we investigated lin- <dig> regulation of lin- <dig> in the embryo, and found that fewer embryos expressed lin-39::gfp in the p cells from mothers treated with lin- <dig> rnai, compared to control embryos . consistent with effects on lin- <dig> expression, lin- <dig> performed on newly hatched l <dig> larvae in a sensitized lin- <dig> background caused a defect in vpc fusion at the l <dig> . thus, we have identified a binding site in the lin- <dig> upstream region for the general hypodermal transcription factor lin- <dig>  and our results suggest that lin- <dig> acts to positively regulate lin- <dig> expression in both the embryo and larva.\n",
            "\n",
            "the t-box protein tbx- <dig> binds in lin- <dig> intron  <dig> and positively regulates larval lin- <dig> expression in the vpcs\n",
            "tbx- <dig> encodes a t-box transcription factor, and a tbx-9::gfp translational reporter is expressed in lateral and ventral hypodermal cells, intestine and muscle  <cit> . tbx- <dig> mutants have a disorganized body shape beginning in the embryo, which was attributed to defects in hypodermal cells and body wall muscles  <cit> . in yeast, tbx- <dig> bound to yf <dig>  a 257 bp dna fragment located in the large second intron of lin- <dig> that contains ecrs 21– <dig> , and this binding was recapitulated in vitro with purified tbx- <dig> protein . we divided the yf <dig> fragment into four smaller fragments , and found that bacterially purified tbx- <dig> bound best to the 79 bp subfragment c, and that this fragment could also compete for binding of tbx- <dig> to yf <dig> .\n",
            "\n",
            "to test tbx- <dig> as a regulator of lin- <dig> in vivo, tbx- <dig> rnai was performed on l <dig> lin-39::gfp worms, and we observed a decrease in the number of animals with wild type levels of gfp expression in p <dig> p-p <dig> p in the same animals at the l <dig> stage . qrt-pcr analysis on tbx- <dig> mutant animals  <cit>  also showed a decrease in lin- <dig> levels compared to wild type l <dig> stage worms , and tbx- <dig> performed in a sensitized lin- <dig> background caused a defect in vpc fusion at the l <dig> . tbx- <dig> rnai treatment did not effect lin-39::gfp expression in the cells p5-p <dig> in the embryo . in summary, we found that tbx- <dig> may bind multiple sites within a 257 bp fragment from lin- <dig> intron  <dig>  and tbx- <dig> acts as a positive regulator of lin- <dig> expression in the vpcs in the larva.\n",
            "\n",
            "the zinc-finger protein bed- <dig> binds to site in lin- <dig> intron  <dig> and positively regulates lin- <dig> expression\n",
            "bed- <dig> encodes a bed zinc-finger protein that is expressed in most hypodermal cells, including the seam cells and the progeny of the vpcs at the time of l3/l <dig> molt  <cit> . in bed- <dig> mutants, the granddaughters of p <dig> p, p <dig> p and p <dig> p often fail to divide, which suggested that bed- <dig> acts late during vulval induction in the terminal divisions of the induced vpcs  <cit> . in yeast, bed- <dig> bound yf <dig>  a  <dig> base pair fragment from the second lin- <dig> intron that contains ecrs 24– <dig> . bacterially-expressed bed- <dig> protein bound to yf <dig> in vitro, however, we found that bed- <dig> protein also bound to several other unrelated dna fragments, suggesting the purified bed- <dig> protein may show non-specific binding in vitro .\n",
            "\n",
            "despite our inability to validate the bed- <dig> binding result in vitro, we observed that bed- <dig> rnai caused a strong decrease in the number of animals with wild type levels of lin-39::gfp expression in p <dig> p-p <dig> p at the l <dig> stage , and this decrease was also seen by qrt-pcr analysis on bed- <dig> l <dig> larvae . consistent with the decrease in lin- <dig> expression in vivo, bed- <dig> caused a defect in vpc fusion at the l <dig> in a sensitized lin- <dig> background . however lin-39::gfp expression did not change in embryos derived from mothers treated for bed- <dig> . therefore, although we could not localize a binding site for bed- <dig> beyond the lin- <dig> intron  <dig> fragment used in the yeast screen, our in vivo data indicate that bed- <dig> is likely to function as a positive regulator of lin- <dig> expression in the vulval precursor cells in the larva.\n",
            "\n",
            "the gata factor elt- <dig> binds to a lin- <dig> enhancer that directs expression in the p cells in the embryo\n",
            "we previously described a 340 bp cis-regulatory element from lin- <dig> that is sufficient to drive gfp expression in the embryo in p5-p <dig>  cells which divide to generate lin- <dig> expressing vpcs and neuroblasts in the ventral midbody region . a 24 bp sequence  in the pjw <dig>  enhancer is conserved between c. elegans and c. briggsae, and mutation of this site abolished embryonic expression from the pjw <dig>  reporter  <cit> . site s <dig> contains the sequence tgataa, a predicted binding site for a gata family transcription factor, which prefer the motif wgatar  <cit> . intriguingly, we found that the transcription factor elt- <dig> bound to the pjw <dig>  enhancer fragment in our yeast screen . elt- <dig> encodes a  <dig> amino acid gata transcription factor expressed in certain neurons and hypodermal cells, particularly the seam cells and vpcs  <cit> . we performed directed yeast one-hybrid assays with eight other c. elegans gata factors and found that elt- <dig> was the only gata factor that interacted with the pjw <dig>  enhancer fragment in yeast . when the gata site in pjw <dig>  was mutated in the yeast reporters, elt- <dig> no longer interacted with the dna fragment in yeast . elt- <dig> protein purified from e. coli bound the s <dig> site in vitro, but did not bind when the gata sequence was mutated . elt- <dig> binding to s <dig> was abolished when competed with excess wild type cold s <dig> probe, but when the s <dig> gata site was mutated, the resulting oligonucleotide  competed less well for elt-6/s <dig> binding . when a second gata site at the edge of the 40 bp oligonucleotide was also mutated, the ability of the mutated oligonucleotide  to compete was greatly reduced .\n",
            "\n",
            "to examine regulation of lin- <dig> expression by elt- <dig> in the embryo, we assayed expression of a reporter construct containing the wild type 340 bp element, pjw <dig> ::gfp, in elt- <dig> mutant animals. gk <dig> is an allele with 457 bp deletion covering the first intron and second exon of elt- <dig> and is a presumed null mutation  <cit> . only 56% of embryos from elt- <dig> mutant animals displayed a wild type pattern of pjw <dig> ::gfp expression, compared to 94% of control embryos . a decrease in the penetrance of expression was also seen in the embryos derived from pjw <dig> ::gfp mothers treated with elt- <dig> rnai . we also examined the effect of reduction of elt- <dig> function on embryonic expression of the large lin-39::gfp reporter that we used to assay other transcription factors. when elt- <dig> rnai was performed on lin-39::gfp hermaphrodites, only 82% of embryos showed the wild type level of gfp expression in p5- p <dig> . taken together, these results indicate that the gata factor elt- <dig> is necessary for proper expression of lin- <dig> in p5-p <dig> in the embryo, most likely via binding to the conserved gata site in the pjw <dig>  enhancer, which was previously shown to be necessary for enhancer driven gfp expression in the embryonic p cells  <cit> ).\n",
            "\n",
            "the gata factor egl- <dig> also regulates lin- <dig> expression in the p cells in the embryo\n",
            "the elt- <dig> open reading frame begins less than 600 bp downstream from the end of another gata factor gene, egl- <dig> , and these two genes are transcribed dicistronically in some tissues  <cit> . the dna binding domains of egl- <dig> and elt- <dig> are similar, and the two genes show genetic redundancy during fate specification of the hypodermal seam cells and vpcs  <cit> . in particular, reduction of function for both elt- <dig> and egl- <dig> in the larva causes the vpcs to adopt inappropriate cell fates and fuse with the hypodermal syncytium  <cit> , a phenotype also seen with reduction of lin- <dig> function  <cit> .\n",
            "\n",
            "although egl- <dig> did not bind to the 340 bp pjw <dig>  fragment in yeast assays , the known functional redundancy of elt- <dig> and egl- <dig> and their dicistronic expression in hypodermal cells led us to test for a role for egl- <dig> in the regulation of lin- <dig> expression. when the pjw <dig> ::gfp reporter was moved into two different egl- <dig> mutant strains, egl- <dig> and egl- <dig> <cit> , we found that gfp expression in p <dig> - p <dig> in the embryo decreased from 94% for control animals to 60% in egl- <dig> and 68% in egl- <dig> . these results suggest that egl- <dig> is also required for proper lin- <dig> expression in the embryo. to test for redundant function in lin- <dig> regulation, we performed rnai for one gene in the background of a mutant for the other . in neither case was there a significant decrease in the number of embryos showing pjw <dig> ::gfp expression compared to the egl- <dig> or elt- <dig> mutant strain treated with control vector . thus we did not obtain evidence for functional redundancy of elt- <dig> and egl- <dig> in regulating pjw <dig> ::gfp expression in the embryo, even though reduction of function for either gene alone affects embryonic gfp expression.\n",
            "\n",
            "elt- <dig> and egl- <dig> also regulate lin- <dig> expression post-embryonically\n",
            "we also examined whether elt- <dig> affects lin- <dig> expression post-embryonically. elt- <dig> rnai was performed on lin-39::gfp l <dig> larvae and we observed gfp expression in the vpcs in the same animals at the l <dig> stage. a statistically significant decrease in the number of vpcs  showing a wild type level of lin-39::gfp expression in the vpcs at the l <dig> stage was seen for elt- <dig> animals , indicating that elt- <dig> also positively regulates lin- <dig> in the larval vpcs. egl- <dig> is also likely to regulate lin- <dig> expression postembryonically, since qrt-pcr on animals in which both egl- <dig> and elt- <dig> function is compromised had lower expression of lin- <dig> than elt- <dig> mutant animals alone .\n",
            "\n",
            "to test the hypothesis that lin- <dig> is a downstream target of elt- <dig> and/or egl- <dig> in the larva, we asked if overexpression of the gata factors could increase lin- <dig> levels. we overexpressed egl- <dig> and elt- <dig> using the heat shock promoter and assayed lin- <dig> levels by qrt-pcr for lin- <dig> one hour after a single heat shock at the l2/l <dig> molt. we found that lin- <dig> levels went up  <dig>  fold when elt- <dig> was over-expressed,  <dig>  fold when egl- <dig> was over-expressed, and  <dig>  fold when both elt- <dig> and egl- <dig> were over expressed , supporting the hypothesis that lin- <dig> is a downstream target of both elt- <dig> and egl- <dig> in the larval vpcs.\n",
            "\n",
            "discussion\n",
            "the c. elegans hox gene lin- <dig> functions in the midbody region of the developing c. elegans larva, where it is expressed in the p cells and their descendants, including the hermaphrodite vulval precursor cells   <cit> . during vulval development, lin- <dig> expression is regulated by wnt and ras signaling pathways to facilitate vpc fate specification  <cit> . to further understand the function of lin- <dig> in hermaphrodite vulval development, we wish to identify cis-acting sites and trans-acting factors required for initiation, maintenance and regulated expression of lin- <dig> in the p cells and vpcs. previously, using an enhancerless gfp reporter assay, we identified a lin- <dig> enhancer fragment  that directs expression in the vpc p <dig> p at the time of vulval induction, which responds to ras pathway activity, and which is bound by the ras pathway effectors lin- <dig> and lin- <dig>  as well as by lin- <dig> itself  <cit> . we also identified a lin- <dig> enhancer  that directs expression in the p cells in the embryo and in their larval descendants , and showed that expression in the embryo depended on an evolutionarily conserved site that contains within it a putative gata transcription factor binding site.\n",
            "\n",
            "here we took a complementary approach to identify additional factors that may regulate lin- <dig> expression during vulval development. we concentrated on  <dig> short, evolutionarily conserved regions  that we previously identified in the lin- <dig> gene in c. elegans, c. briggsae and c. remanei <cit> , on the assumption that some may represent binding sites for phylogenetically conserved transcriptional regulators. we used the yeast one-hybrid technique to identify transcription factors that could bind to dna fragments containing one or more of these ecrs, circumventing the requirement that the dna site be sufficient to direct in vivo reporter expression  <cit> . having found multiple transcription factors that interact with lin- <dig> dna fragments, we looked for effects on endogenous lin- <dig> transcript levels and on expression from a lin-39::gfp reporter in vivo when the function of these proteins was reduced. using this approach we identified six transcription factors that bind lin- <dig> promoter sequences in yeast and in vitro and regulate lin- <dig> expression and/or function in vivo . three factors, the orphan nuclear receptor nhr- <dig>  the hypodermal cell fate regulator lin- <dig> and the gata factor elt- <dig> positively regulate lin- <dig> expression in the embryo. in addition to those three factors, we identified the zinc finger proteins ztf- <dig> and bed- <dig> and the t box factor tbx- <dig> as positive regulators of lin- <dig> expression in the larval vpcs. before this work, only four transcription factors were known to directly bind at the lin- <dig> gene and regulate its expression; lin- <dig>  lin- <dig>  lin- <dig> and tra- <dig>  all of which act during larval life   <cit> . therefore, by this approach combining phylogenetic conservation and yeast one-hybrid screening, we have more than doubled the number of factors known to directly bind to and regulate lin- <dig>  and for the first time we have identified factors that regulate expression of this hox gene in the embryo.\n",
            "\n",
            "we believe these factors regulate lin- <dig> in vivo because we observed a reproducible change in expression of the full length lin-39::gfp reporter and/or a decrease in endogenous lin- <dig> transcript levels when the function of each of these factors was reduced, and in several cases we observed a phenotype in a lin- <dig> regulated process . given their binding to sites from the lin- <dig> genomic region, the simplest model would be that these factors directly regulate lin- <dig> expression in the cells we examined . however, in the current work we did not pursue in vivo binding studies for any of these factors; although published results from the modencode project show binding of alr- <dig> to the evolutionarily conserved site we identified at the relevant time in development  <cit> . additionally, while existing gfp reporters for four of these proteins  are turned on in some lin-39-expressing cells, expression in the p cells or vpcs has not been directly observed  <cit> . gfp reporters suffer from the caveat that all the elements required to recapitulate endogenous expression may not be present, or the expression may be weak or dynamic. however, until we can verify expression of each transcription factor in the embryonic p cells or larval vpcs, and show evidence of binding to the sites we identified in vivo, it remains possible that some of these factors regulate lin- <dig> indirectly via another transcription regulator, or even act non cell-autonomously on lin- <dig> expression. the strongest case for direct regulation can be made for bed- <dig>  which regulates lin- <dig> expression during larval life, and for lin- <dig> and elt- <dig>  which regulate lin- <dig> expression in the embryonic p cells.\n",
            "\n",
            "bed-3\n",
            "bed- <dig> encodes a  <dig> amino acid zinc finger protein with a bed dna binding domain, and bed- <dig> mutants show an egl- laying  defective phenotype and defects in the terminal cell divisions of the descendants of the vpcs  <cit> . an intronic enhancer element from bed- <dig> directs gfp expression in vpc descendants, leading to the hypothesis that bed- <dig> functions in the terminal cell divisions of vulval cells during vulval induction  <cit> . we found that bed- <dig> showed the strongest effect on full length lin-39::gfp expression in the vpcs; in wild type animals the vpcs p <dig> p - p <dig> p showed lin-39::gfp expression 89% of the time , while in bed- <dig> animals these cells showed gfp expression only 37% of the time. this suggests that bed- <dig> could be acting at an earlier stage in vulval development. consistent with this, we observed that the bed- <dig> enhancer::gfp reporter does show expression in the vpcs before they divide , which would be consistent with bed- <dig> acting upstream of lin- <dig> in the vpcs themselves. therefore, although we could not verify bed- <dig> binding to the yf <dig> site in vitro with purified protein, we believe bed- <dig> is likely to be a positive regulator of lin- <dig> in the larval vpcs before vulval induction, as well as functioning in the subsequent cell division of their progeny, as previously reported  <cit> .\n",
            "\n",
            "lin-26\n",
            "lin- <dig> is a  <dig> amino acid zinc finger transcription factor that shows continuous expression in all embryonic and larval hypodermal cells after their birth  <cit> . loss of lin- <dig> causes hypodermal cells to adopt incorrect cell fates or degenerate after their birth, resulting in embryonic lethality  <cit> . ectopic expression of lin- <dig> in the early embryo can induce cells to adopt hypodermal-like fates  <cit> . these results suggest that lin- <dig> is required to specify and/or maintain the hypodermal cell fate  <cit> . lin- <dig> itself is positively regulated by the gata factor elt- <dig>  which is another global regulator of hypodermal cell fate  <cit> . we found that lin- <dig> rnai caused a weak but significant decrease in the expression of the full length lin-39::gfp reporter in the p cells in the embryo; 100% of wild type embryos show gfp expression, compared to 86% of lin- <dig> animals. lin- <dig> binds in vitro to a 51 bp sequence located approximately 2 kb upstream of the lin- <dig> start codon . recent data recording reporter gene expression from live developing embryos shows that a lin-26::mcherry transcriptional reporter shows expression in the mothers of the p cells shortly before they divide, at the same time as these cells also begin to show expression of a lin-39::mcherry reporter  <cit>  . based on these results, and the known function of lin- <dig> in hypodermal cell fate, we propose that lin- <dig> positively regulates lin- <dig> expression in the embryonic p cells, and that the function of this regulation is to aid in initial p cell fate specification and/or to maintain the p cell identity once established.\n",
            "\n",
            "elt-6\n",
            "elt- <dig> encodes a  <dig> amino acid gata family transcription factor. the elt- <dig> gene is immediately downstream from another gata factor gene, egl- <dig> , and reporter gene experiments suggest elt- <dig> may be expressed on its own and as part of a dicistronic message with egl- <dig> <cit> . these genes are expressed in many cells in the embryo, including the descendants of the ms and ab blastomeres . in the larva, they are expressed strongly in the lateral hypodermal seam cells and weakly in the vpcs  <cit> . these two gata factors share 76% identity in their dna-binding domains, and have been shown to act redundantly in seam cell development in both the embryo and larva  <cit> . we previously showed that a 340 bp lin- <dig> enhancer  directs gfp expression in the embryonic p cells, and a conserved gata site in the enhancer was necessary for expression  <cit> . here we identify the gata factor elt- <dig> as binding to this lin- <dig> enhancer in yeast and in vitro, and show that binding was dependent on the gata sequence. elt- <dig> rnai showed a decrease in expression of the full length lin-39::gfp reporter in embryonic p cells, and this result was recapitulated with the pjw <dig>  reporter in elt- <dig> mutant and elt- <dig> embryos. these data suggest that elt- <dig> is required for proper expression of the hox gene lin- <dig> in the embryonic p cells. consistent with this, recent data from live recordings of developing embryos shows that elt- <dig> expression in the mothers of p <dig> - p <dig> begins before lin- <dig> expression is first seen . this is the first example to our knowledge of a phenotype caused by reduction of elt- <dig> function alone. elt- <dig> rnai treatment of newly hatched l <dig> larvae also led to a weak reduction in lin-39::gfp expression in the l <dig> vpcs, and overexpression of elt- <dig> or egl- <dig> increased endogenous lin- <dig> expression in larvae. these results suggest that elt- <dig> and/or egl- <dig> may continue to positively regulate lin- <dig> expression in larval life in the vpcs.\n",
            "\n",
            "given these results, it is interesting that we previously showed, in collaboration with the rothmann laboratory, that the egl-18/elt- <dig> locus was likely to be a downstream target of lin- <dig> in the larval vpcs  <cit> . in that work an 800 bp enhancer element was identified in intron  <dig> of the egl- <dig> gene that directs gfp expression in the vpcs and their descendants starting in the l <dig> stage. this enhancer contains two hox protein-binding sites that are bound in vitro by lin- <dig> and its binding partner ceh- <dig>  and mutation of one site eliminated enhancer-driven gfp expression. finally, overexpression of egl- <dig> from the heat shock promoter was able to partially rescue vulval defects in lin- <dig> animals. these data led to the model that egl-18/elt- <dig> is a downstream target of lin- <dig> during vulval development.\n",
            "\n",
            "combining these previous data with our current results, our working model suggests lin- <dig> and elt- <dig> are involved in the initiation of expression of the hox gene lin- <dig> in the p cells in the embryo . lin- <dig> may also be regulated by egl- <dig> at this time, since egl- <dig> mutants had reduced enhancer-driven gfp expression in the p cells, and egl- <dig> protein bound to the pjw <dig>  gata site in vitro . once the fate of these cells is established, a positive feedback loop is established between elt- <dig>  acting via the upstream lin- <dig> enhancer pjw <dig> , and lin- <dig>  acting via the intronic enhancer in egl- <dig> <cit> . we propose that this positive feedback loop helps maintain expression of these genes and the fate of these cells and their descendants, the vpcs, during subsequent embryonic and larval development . interestingly, we have seen another example of lin- <dig> feedback regulation. the zinc finger protein sem- <dig> was previously shown to positively regulate lin- <dig> expression in vpcs  <cit> . we have found that in animals overexpressing either lin- <dig> or lin- <dig> and ceh- <dig>  sem- <dig> expression is decreased, suggesting that feedback mechanisms to decrease lin- <dig> levels when they are elevated may also exist . recent chromatin immunoprecipitation experiments from larvae verify binding of tagged lin- <dig> protein upstream and in introns of both the egl- <dig> and sem- <dig> genes  <cit> .\n",
            "\n",
            "CONCLUSIONS\n",
            "while much is known about the initiation and regulation of hox gene expression in drosophila and vertebrates, less is known outside of these well-studied groups. our laboratory and others have been studying the expression and function of the hox gene lin- <dig> during vulval development in the nematode c. elegans. we used the yeast one-hybrid  method to identify  <dig> transcription factors that interact with specific regions of the lin- <dig> gene, and further characterized several factors  showing that their function is required for proper expression of lin-39::gfp reporters and endogenous lin- <dig> in vivo. this work greatly expands the number of factors known to directly regulate lin- <dig> expression. given the known caveats of the yeast one-hybrid technique , and our emphasis on characterization of factors with known expression or phenotypes in the vulval cells, it is likely that there are additional factors that regulate lin- <dig> in the embryo and larva that we did not identify in our screens. given the important role of hox genes in patterning the developing metazoan body, it is not surprising that hox gene expression is found to be complicated in those species where it has been examined closely. our results suggest that expression of the c. elegans hox gene lin- <dig> in the p cells and vpcs during vulval development may be regulated by a large number of transcription factors, each making a small contribution to overall lin- <dig> expression on its own. this is consistent with in vivo data from c. elegans showing that the average worm gene is bound by several transcription factors at one time in larval development  <cit> . such a mechanism may ensure a robustness of expression for important developmental regulators like hox genes. transcription factors behaving in this manner would also not be identified in genetic screens, since each makes a small overall contribution to lin- <dig> expression, and feedback mechanisms may exist to compensate for reductions in lin- <dig> transcript levels. finally, for the first time, we have identified factors required for lin- <dig> expression in the embryo , and our results with egl- <dig> and elt- <dig>  combined with our earlier work on lin- <dig> regulation of egl-18/elt- <dig>  hint at a positive feedback mechanism to maintain lin- <dig> expression in the vulval lineages. the identification of lin- <dig>  elt- <dig>  and nhr- <dig> will help us further characterize the mechanisms for the initiation of hox gene expression in the nematode, allowing us to make comparisons across metazoan phyla about the mechanisms utilized to regulate this essential class of developmental regulators.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this work, we have identified  <dig> transcription factors that regulate lin- <dig> expression in the embryo and larva that we previously identified in drosophila.\n",
            "in addition, we have identified  <dig> transcription factors that directly regulate lin- <dig> expression in the larva, including the orphan nuclear receptor nhr- <dig>  the hypodermal cell fate regulator lin- <dig>  and the gata factor elt- <dig>  these transcription factors may play a role in the initiation of lin- <dig> expression in the vulval lineage.\n",
            "in addition, we have identified  <dig> transcription factors that directly regulate lin- <dig> expression in the larva, including lin- <dig>  the orphan nuclear receptor nhr- <dig>  the hypodermal cell fate regulator lin- <dig>  the gata factor elt- <dig> and the gata factor elt- <dig>  these transcription factors may play a role in the initiation of lin- <dig> expression in the larva.\n",
            "in addition, we have identified  <dig> transcription factors that directly regulate lin- <dig> expression in the larva.\n",
            "\n",
            "22\n",
            "[{'summary_text': 'here, we introduce three main-chain atom based surface representations which are found to better agree with the two existing structure classifications, the 3dzd and the scop database, as compared with the previous all-atom surface representation, the 3dzd.\\nthe 3dzd has the following properties that make it ideal for use in protein shape analyses.\\nthe 3dzd is a series expansion of an input 3d function, which allows for a compact representation of the protein structure and thus allows for a fast and accurate database search for em density maps.\\nthe 3dzd has the following properties: 1) rotation invariant; 2) sphericity; 3) sphericity; 3) sphericity; 4) sphericity; and 5) sphericity.\\nthe 3dzd has the following properties: 1) rotation invariant; 2) sphericity; 3) sphericity; 3) sphericity; 4) sphericity; and 5) sphericity.\\nthe 3dzd has the following properties: 1) rotation invariant; 2) sphericity; 3) sphericity; 3) sphericity;'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the number of protein structures deposited to the protein data bank  is increasing at a rapid pace. particularly, more and more protein structures of unknown function are being solved by structural genomics projects  <cit> . the flood of structures raises significant challenges to develop computational methods that will provide critical information for hypothesizing function and evolution of the proteins from their structural information  <cit> . the study of protein structures has now entered the informatics era, just as biological sequence analyses has done in the previous decade when efficient reuse of knowledge from existing databases became crucial. speed is an essence in such analyses, since biologists would need to run various database searches using different tools, and such analyses would be conveniently performed if they are done in real-time. most of the current structure comparison methods  <cit> , such as those that compare main-chain orientations or corresponding residue positions <cit> , are designed for pair wise comparison and are not suitable for a fast database search.\n",
            "\n",
            "there are other new tasks which have been brought up by recent experimental techniques, such as electron microscopy , which provides low-resolution structure data. here, challenges include how to use a low-resolution em density map for fitting high resolution structures  <cit>  or guiding protein structure prediction  <cit> , and how to efficiently and accurately compare global and local structures  <cit> . thus, development of a new generation of structure analysis tools, which allow a fast screening of large structure databases and can handle low resolution structure data, is needed. with this in mind, we represent protein structures as surface shapes by the 3d zernike descriptors   <cit> . there have been previous works which employ a protein surface representation  <cit> , such as volumetric representation  <cit> , convex hull  <cit> , and the spin image  <cit> . compared to those works, the 3dzd has the following properties that make it ideal for use in protein shape analyses. first, it is rotation and translation invariant. thus, prior time-consuming structure alignment is not necessary for structure comparison since their position in space does not change their 3dzds. this property enables direct comparison of the em density maps where atomic coordinates are not available. another advantage is its compact representation; a 3d shape is effectively represented in only  <dig>  numbers called the invariants. the 3dzd can also represent physicochemical properties such as the electrostatic potential and the hydrophobicity on the protein surfaces  <cit> . recently the 3dzd has been further applied for protein-protein docking prediction  <cit> , local surface comparison  <cit> , pocket shape matching for structure-based function prediction  <cit> , and ligand molecule screening  <cit> .\n",
            "\n",
            "in the previous work, we showed that the all atom surface representation by the 3dzd agrees sufficiently with the main-chain based structure comparison by ce  <cit> , but some differences were also observed due to the variations in the representation  <cit> . the new contribution of this paper is two fold. first, we introduce three main-chain atom based surface representations which are found to better agree with the two existing structure classifications, the ce and the scop database  <cit> , as compared with the previous all-atom surface representation. second, we show that the proposed representation also allows for a fast and accurate database search for em density maps.\n",
            "\n",
            "methods\n",
            "datasets\n",
            "to examine structure retrieval performance of the proposed methods, we use a data set of  <dig> representative protein structures, which are arbitrarily selected from  <dig> fold groups defined in a protein classification database by ce . ce is one of the frequently used protein structure comparison programs that compares cα positions of proteins using a dynamic programming algorithm. these representative structures have a resolution of  <dig>  Å or better, have no more than  <dig> missing residues in the solved structure, have all heavy atom positions solved, and are longer than  <dig> residues. in addition, the structure similarity of each pair is less than a z-score of  <dig>  by ce.\n",
            "\n",
            "this dataset also provides the scop classification code of proteins, which classifies the proteins into  <dig> class groups,  <dig> folds groups,  <dig> superfamily groups, and  <dig> family groups. we use both ce-based and scop classifications in our study since they have the following complementary features: the ce classification is automatic without human intervention and considers main-chain orientation, while scop is curated manually at a certain degree to take evolution into account.\n",
            "\n",
            "at this juncture, it is important to note that there is no golden standard in classification of proteins. the structure similarity measured for different representations can be largely different for distantly related proteins since they capture different aspects of the structures  <cit> . in our previous paper  <cit> , we showed that ce and scop do not fully agree and also that dali  <cit> , which compares distance maps of proteins, and ce have poorer agreement than ce and the 3dzd. each method has its own strength and thus an appropriate method should be selected depending on the purpose of each study. we have further shown examples of proteins whose surface shape similarity infers functional similarity, which are not detected by the conventional sequence or main-chain structure comparison methods  <cit> . in this study, we demonstrate that the new main-chain surface representations encoded by the 3dzd have a better agreement to ce and scop as compared to the original all-atom surface representation introduced in the previous study  <cit> .\n",
            "\n",
            "computing protein surfaces\n",
            "for a protein structure, four different surface representations are computed: one that uses all heavy atoms , the backbone conformation with all heavy atoms in the main-chain, i.e. cα, c, n, and o atoms , the backbone with cα, c, and n atoms , and the backbone cα atoms only . for the set of extracted atoms, the surface is generated using the msms program  <cit> . msms rolls a probe sphere on the atoms and defines the surface as the path of the center of the probe. the radius of the probe sphere is set to default value of  <dig> Å for aasurf, cacno, and cacn and the radius is set to  <dig> Å for ca to generate a smoother representation. the generated surface is then mapped on a 3d grid. a grid cell  is assigned a value of  <dig> if it is on the surface and  <dig> otherwise. because the 3dzd is defined within a unit sphere, the protein surfaces represented by voxels are scaled into a unit sphere. therefore, the size information of the protein is lost. the resulting voxels are considered as an input 3d function, f, which is used as input for computing the 3dzd as described in the next section.\n",
            "\n",
            "3d zernike descriptors\n",
            "the 3dzd is a series expansion of an input 3d function, which allows for a compact representation of the 3d object   <cit> . the mathematical foundation of the 3dzd was laid out by canterakis  and was applied on 3d object retrieval by novotni and klein . for readers’ convenience, a brief mathematical derivation of the 3dzd is shown below. for detailed derivations and discussions, refer to the aforementioned two papers canterakis  <cit>  and novotni and klein  <cit> .\n",
            "\n",
            "the first step of computing the 3dzd is derivation of the 3d zernike moments. for an input 3d function, f, the 3d zernike polynomials defined on order n, degree l, and repetition m, are given by   \n",
            "\n",
            "subjected to -l < m < l ,  <dig> ≤ l ≤ n , and  being even. the spherical harmonics, , are functions of a set of a polar angle, ϑ, and a azimuthal angle, ϕ. the radial function, rnl, incorporates the radius information into the basis function and are constructed so that  are polynomials when written in terms of the cartesian coordinates. the 3d zernike moments of f are defined as the coefficients of the expansion using this orthonormal basis in the following formula:   \n",
            "\n",
            "after computing the 3d zernike moments, a normalization step is necessary to obtain rotation invariance. this is done by taking the l <dig> norm of the 3d zernike moments as the descriptor. that is, the moments are collected into  dimensional vectors  and the rotational invariance is obtained by defining 3dzd, fnl , as the norm of vectors Ωnl :   \n",
            "\n",
            "the size of the 3dzd vector is set by the parameter n, called the order, which determines the resolution of the descriptor. the 3dzd is a series of invariants  for each combination of n and l, where n ranges from  <dig> to the specified order. for example, n ranges from  <dig> to  <dig> for a 3dzd of an order  <dig>  the order of n= <dig>  which yields a total of  <dig> numbers, or invariants, is used in our study based on the success of the previous works  <cit> . the last step is to normalize the descriptor by the norm of the descriptor. this normalization is found to reduce dependency of the 3dzd on the number of voxels used to represent a protein  <cit> . figures 1a through d show the surface generated from the four representations. figure 1e shows the 3dzd of the four representations for the protein pdb:1hdma. it can be seen that there is little difference in the 3dzd of cacno, cacn, and ca as compared to the 3dzd of aasurf in this particular case. the correlation coefficients among the three backbone representations  range between  <dig>  to  <dig> . the correlation coefficients between cacno, cacn, and ca to aasurf are  <dig> ,  <dig> , and  <dig> , respectively. although there is little difference between the four representations in this particular example, we will show later that the four representations make a difference in terms of overall database retrieval performance.\n",
            "\n",
            "evaluating database retrieval performance\n",
            "the database retrieval performance of the four surface representations is evaluated with precision-recall curves. the precision-recall curves are often confused with the receiver operator characteristic curves. although these two curves are related, the precision-recall curve is considered to be a better measure when the dataset is skewed  <cit> . the number of proteins in a group in the dataset used varies from  <dig> to  <dig> and thus a precision-recall is used here. for each protein in the dataset, the rest of the proteins described with the 3dzd are sorted by the euclidean distance  to the 3dzd of the query protein. then, the precision and the recall values are computed at each distance threshold value. the precision is defined as the fraction of the retrieved proteins of the same group with the query among all proteins retrieved above the distance threshold. the recall is defined as the fraction of the retrieved proteins of the same group with the query among all the proteins in the same group. finally, we calculate the average precision and recall for each distance threshold. the precision-recall curves of different representations are evaluated by the area under curve .\n",
            "\n",
            "as employed in the previous work  <cit> , we also apply pre-filtering of the proteins by their sequence length. for a query, a protein in the database is filtered out if it is longer than 135% or shorter than 65% to the length of the query protein. this is done because of the loss of the size of the proteins during the process of computing the 3dzd, since the proteins are scaled to fit into a unit sphere.\n",
            "\n",
            "combining 3dzd of aasurf and cacno\n",
            "we also examine database retrieval with combinations of the 3dzds of the aasurf and a backbone surface representation. among the three backbone representations, we choose cacno since no significant difference in performance was observed among the three . cacno would also be a natural choice since it is the full heavy atom representation of protein backbone. for the aasurf and cacno combination, the distances measured independently are linearly combined with weighting factors:   \n",
            "\n",
            "where y and x are the two proteins compared and i is the index of 3dzd invariants of aasurf, s, and cacno, b. wys and wyb are weights for aasurf and cacno of the query protein y, and m <dig> and m <dig> are the number of invariants in the 3dzd of aasurf and cacno, respectively. in this study, the 3dzd of aasurf and cacno is set to the same size, i.e. m1=m2= <dig>  eqn.  <dig> is asymmetric since the weights wys and wyb depend on the query protein, y.\n",
            "\n",
            "the weights for aasurf and cacno for a query protein are determined by two characteristics of its protein shape: 1) the existence of a tail-like structure and 2) the sphericity. the tail is an elongated region in the structure which is longer than three amino acids locating further than two times of the radius of gyration  of the protein from the center of the gravity. the radius of gyration is defined as follows:   \n",
            "\n",
            "where n is the number of atoms in protein xj, cog, is center of gravity of protein xj, and r is the approximate radius of atoms in which  <dig> Å is used  <cit> .\n",
            "\n",
            "the sphericity measures how compactly a protein structure fits to a sphere:   \n",
            "\n",
            "where rs is the radius of a sphere that has the same volume as the protein all-atom surface representation computed by the msms program. a larger value indicates that the protein is more spherical.\n",
            "\n",
            "RESULTS\n",
            "database retrieval performance\n",
            "the database retrieval performance of the four surface representations is first tested on the ce classification. in all the precision-recall curves shown in figure  <dig> , the four representations show significantly better performance than the random retrieval, which shows a precision-recall auc value of  <dig> . the random retrieval curve of the precision-recall graph is computed by generating random distance values between the pair of proteins. readers should not be confused with the auc value of the precision-recall used in this work with the one for the receiver operator characteristics  curves, which gives  <dig>  for a random retrieval. figure 2a and 2b show results without and with pre-filtering by protein length. all three backbone surface representations, cacno, cacn, and ca, show similar performance, which is significantly better than that of aasurf. note that aasurf showed much higher performance than dali and four other shape-comparison methods in our previous work  <cit> . the length-based pre-filtering improves the auc by around  <dig>  for all the backbone representations and by  <dig>  for aasurf representation. the length-based pre-filtering also improves the retrieval performance by incorporating the size information that is lost in the computation of 3dzd when a protein is scaled normalized. on the other hand, one could see that the improvement by the length filter is marginal, which implies it is not common to observe two proteins of different sizes with similar shape, which is consistent with the previous results  <cit> .\n",
            "\n",
            "although cacn performs the best, the difference in performance by cacno and ca is marginal with a average auc difference of  <dig> . since the three backbone representations show almost identical performance, we will only show results of cacno along with aasurf for further analyses.\n",
            "\n",
            "comparison of cacno and aasurf\n",
            "close examination of the individual cases of the database retrieval revealed interesting trends on the performance of the aasurf and the cacno representations. in figure  <dig>  the effect of the sphericity  and the tail-like structures  in the individual proteins are examined in respect of the database retrieval. here, the ce classification is used as the base truth. the y-axis shows the difference of the auc of the precision-recall curves with cacno being subtracted from that of aasurf. thus, a positive value indicates that the aasurf performs better than the cacno and a negative value indicates the opposite.\n",
            "\n",
            "the observed difference by aasurf and cacno for individual cases inspired us to combine the two representations to improve the retrieval performance. those results are shown in the next section. for this combined representation, spherical proteins  and proteins with tail-like structures  use different weights when combining the distance.\n",
            "\n",
            "improvement by combining aasurf and cacno representations\n",
            "next, we examine combining the distances defined by aasurf and cacno using eqn.  <dig>  query proteins with tail-like structures are given a higher aasurf weight  and ones with high sphericity are given a higher value for the cacno weight. all others are given fixed weights for aasurf  and cacno . with the threshold value of three residues for the tail-like structure and  <dig>  for the sphericity,  <dig> and  <dig> structures in the dataset fall into the category of structures with tails and spherical structures, respectively. table  <dig> shows the database retrieval results of the aasurf and cacno combination -back representation) with different weight values. again note that the random retrieval has the precision-recall auc value of  <dig> .\n",
            "\n",
            "note that a random retrieval has a precision-recall auc of  <dig> .\n",
            "\n",
            "a) difference of precision-recall auc of surf-back to cacno. difference to aasurf is shown in the parentheses.\n",
            "\n",
            "among the different weight values tested, the weight combination  <dig>  and  <dig>  performed the best. in this combination, the weights for aasurf  and cacno  are set to  <dig>  and  <dig> , respectively, for query proteins with a tail-like structure. on the other hand, if a query protein is spherical and has no tail-like structure, wys is set to  <dig>  and wyb is set to  <dig> . otherwise wys and wyb are set to  <dig>  and  <dig> , respectively. overall, the  <dig> / <dig>  weight combination results in an auc increase of  <dig>  and  <dig>  when compared with the retrieval results using cacno and aasurf, respectively. out of  <dig> fold groups in the dataset,  <dig> groups show improvement by surf-back. there are  <dig> groups where cacno shows a better performance and  <dig> groups where aasurf performs better than surf-back.\n",
            "\n",
            "application to em density maps\n",
            "finally, we show that the 3dzd is readily applicable for comparing em density maps. in figure  <dig>  low-resolution structures of isosurfaces of em density maps and reconstructed structures of the 3dzds  are visually compared. the em density maps are generated with the pdb2mrc program which simulates em density of protein structures  <cit> . the grid interval size is set to 1Å and three different resolutions  are employed. in fig. 6a, the original aasurf representations of proteins are reconstructed from their 3dzds of three different orders . the order which gives a similar resolution of the protein surface to each of the simulated em density maps is chosen. for figure 6b, 3dzds of the cacno representations are used for the reconstruction. figure  <dig> shows that surface representation by 3dzd is visually similar to em isosurfaces and thus would be suitable for describing em density maps at different levels of resolutions.\n",
            "\n",
            "observing the agreement between the em isosurface and structures coded by the 3dzd, we investigate further if the 3dzd can be used for searching similar em density maps of protein structures . to perform this experiment, we prepared datasets of the 3dzds of em density maps as follows: first, em density maps of the  <dig> representative protein structures are computed with pdb2mrc for two different resolutions,  <dig> and 15Å. then, for an em density map of a protein, two 3dzds are computed, one using voxels with a high density value range  and another one using voxels with a low density value range . thus, a total of four datasets of 3dzds of simulated em isosurfaces are prepared. voxels with a high density value in an em density map locate at the core of a protein and thus its isosurface resembles the cacno representation of the protein structure. on the other hand, the isosurface of a lower density value in the em map looks more similar to the aasurf representation of the protein structure.\n",
            "\n",
            "a) voxels with the specified density value range are chosen as input of computing 3dzd.\n",
            "\n",
            "speed improvement of 3dzd over ce\n",
            "a significant advantage of the 3dzd is its fast speed. due to its compact vector representation and rotation invariance, 3dzd allows real-time search of the entire protein data bank   <cit>  once 3dzds of structures are computed offline and stored in the database. to illustrate, the speed by 3dzd is compared with ce in table  <dig>  this evaluation was performed on a computer with intel core <dig> cpu  <dig>  processor with  <dig> giga bytes memory. the pairwise structure comparison with 3dzd takes only  <dig> x10-4s. simply multiplying this execution time by the current size of pdb  gives  <dig>  seconds, while the same procedure by ce results will need almost  <dig> days. the speed of 3dzd is significantly faster than previous similar works on em density map search  <cit> .\n",
            "\n",
            "the values are averaged over the  <dig> proteins. a) times for the search against  <dig> proteins.\n",
            "\n",
            "CONCLUSIONS\n",
            "in this work we examined the applicability of the 3dzd for two important tasks in structural bioinformatics. the first task is the real-time protein structure database search. in contrast to our previous work  <cit>  in which the 3dzd is used to represent an all-atom surface of protein structures , we have now examined backbone amino acid-based surface representations . the backbone-based representations showed significantly better performance than the aasurf when the retrieval performance was evaluated on the agreement to the ce and scop classifications. combinations of aasurf and cacno showed further improvement over cacno.\n",
            "\n",
            "the second task explores the applicability of the 3dzd on representation and comparison of low-resolution structural data by evaluating the database retrieval performance of simulated em density maps. intuitively, isosurfaces of the density maps and molecular surfaces represented by 3dzds look similar to each other. indeed, we showed that the 3dzd is well suited for database retrieval of em maps, achieving comparable accuracy to regular protein structure database retrieval in identifying proteins of the same fold to the query protein. this is the most comprehensive study so far published in identifying the fold class of proteins by comparing em density map of proteins. it is noteworthy that the 3dzd can identify proteins of the same fold with em maps even at 15Å resolution. using em maps of a higher resolution, which have now become more and more available, the retrieval accuracy will only get better. here we compared the em maps of single proteins as the proof of concept that 3dzd is suitable for handling em maps. we expect this work will stimulate further investigations for applying 3dzd or similar descriptors for handling em maps of multiple protein complexes and other low-resolution structure data, such as electron tomography.\n",
            "\n",
            "altogether, we are in a new exciting informatics era of structural biology, and we believe surface representation with the 3dzd will provide new directions and ideas that lead us to new findings through surfing ever expanding molecular structural information.\n",
            "\n",
            "list of abbreviations used\n",
            "em, electron microscopy; 3d, three dimensional; 3dzd, three dimensional zernike descriptor; scop, structural classification of proteins ; ce, combinatorial extension ; aasurf, all heavy atom surface; ca, alpha carbon; cacno, a surface representation using c-alpha, c, n, o atoms in the main-chain; cacn, surface representation using c-alpha, c, n, atoms in the main-chain; msms, molecular surface calculation program by michel sanner; auc, area under the curve; roc, receiver operator characteristics; dali, distance matrices alignment algorithm; pdb, protein data bank\n",
            "\n",
            "competing interests\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "authors' contributions\n",
            "dk conceived the study and sl and dk designed the experiments. sl executed the experiments. sl drafted the manuscript and dk revised it critically. all authors read and approved the manuscript.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "here, we introduce three main-chain atom based surface representations which are found to better agree with the two existing structure classifications, the 3dzd and the scop database, as compared with the previous all-atom surface representation, the 3dzd.\n",
            "the 3dzd has the following properties that make it ideal for use in protein shape analyses.\n",
            "the 3dzd is a series expansion of an input 3d function, which allows for a compact representation of the protein structure and thus allows for a fast and accurate database search for em density maps.\n",
            "the 3dzd has the following properties: 1) rotation invariant; 2) sphericity; 3) sphericity; 3) sphericity; 4) sphericity; and 5) sphericity.\n",
            "the 3dzd has the following properties: 1) rotation invariant; 2) sphericity; 3) sphericity; 3) sphericity; 4) sphericity; and 5) sphericity.\n",
            "the 3dzd has the following properties: 1) rotation invariant; 2) sphericity; 3) sphericity; 3) sphericity;\n",
            "23\n",
            "[{'summary_text': 'this study provides the first characterization of the mucin gene family in cattle, with the exception of human muc3b, muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig> and muc <dig> '}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "mucins  are heavily o-glycosylated proteins that cover all mucosal surfaces. they play an important protective role as they form a physical, chemical and immunological barrier between the environment and the organism. mucins can be largely divided into  <dig> structurally different families: the secreted and the membrane -associated mucins  <cit> . in humans,  <dig> members in the family of the secreted mucins have been identified, which can be further subdivided into gel-forming mucins  and non-gel-forming mucins   <cit> . the membrane-associated mucins, on the other hand, count  <dig> members   <cit> . structurally, all mucins have at least one mucin-like domain, named pts-domain, formed by a variable number of tandem repeats  rich in pro, thr and ser residues  <cit> . these pts domains carry the typical mucin o-glycosylations  <cit> . the secreted gel-forming mucins are further characterized by the presence of other typical domains  <cit> , such as n-terminal til domains, n- and/or c-terminal von willebrand d and c domains , c <dig> domains, and a c-terminal cystine knot  domain, all involved in the oligomerization through the formation of disulphide bridges between cysteine residues  <cit> . the capacity to form oligomers gives the secreted gel-forming mucins the ability to build up the dense, visco-elastic mucus gel that covers many epithelia  <cit> . secreted non-gel-forming mucins on the other hand are not able to oligomerize, and their structural and functional properties are not well described  <cit> . the cell surface-associated mucins also do not oligomerize, but they are characterized by specific domains such as the c-terminal sea urchin sperm protein-enterokinase-agrin  domain, cleaved after translation, an epidermal growth factor  or egf-like  domain involved in the three dimensional structure, and a transmembrane domain . muc <dig>  a member of the cell surface-associated mucins, together with the typical domains described above, contains a variant of the vwd domain that lacks cysteines, as well as an adhesion associated domain  with a possible role in cell adhesion, and a nido domain of unknown function. the membrane bound mucins are mostly present on the apical membrane of epithelial cells, where they have been suggested to play a role in cell signalling. some of them can also be found in the mucus layer, together with the secreted ones, probably due to proteolytic cleavage  <cit>  or to the expression of secreted splicing variants  <cit> .\n",
            "\n",
            "sequencing and annotation of mucin genes is known to be difficult due to the large size and repetitive structure of these molecules; moreover, among species, differences in the mucin gene family have been reported. comparison between humans and mice, two mammals with a completed annotated genome  <cit> , has shown that although the majority of mucins are commonly represented in both species, differences for few mucin genes are evident. in humans, for example, a muc <dig> and two muc <dig> genes  have been described, while in mice only one muc <dig> gene has been identified and it is still under debate if it is the homologue of human muc3a or muc <dig>  <cit> . moreover, the muc <dig> gene has so far only been described in humans  <cit> . in ruminant species, such as cattle, only few mucin genes have been described to date   <cit> ); therefore the first goal of the current study was to provide a comprehensive overview of annotated and non-annotated members of the bovine mucin family through database search and comparison with their potential human homologue. since mucins play a fundamental role in the gastrointestinal  tract defence mechanism, the distribution of the different mucins throughout the gi tract has been widely studied for monogastric mammalians  <cit> . however, in polygastric species, mucin distribution is still largely unclear. therefore, the second goal of the current study was to investigate the transcriptional distribution of the identified bovine mucins throughout the gi tract of adult healthy cows.\n",
            "\n",
            "methods\n",
            "identification and classification of mucin genes\n",
            "to identify mucin-encoding genes, the bovine genome assembly version btau_ <dig>  was used, accessible through the ncbi database  <cit> . first, the database was term-searched for automatically annotated mucin genes. in a second step, mucin-encoding genes were also identified by blastn and blastp queries, using the nucleotide and amino acid sequences of  <dig> previously described human mucin molecules. these sequences included muc <dig>  muc <dig>  muc3a, muc3b, muc <dig>  muc5ac, muc5b, muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig> and muc <dig>  the predicted bovine mucins were subsequently aligned with their putative human homologue to calculate amino acid sequence similarity. this was done using the megalign software .\n",
            "\n",
            "in the situation where multiple partial bovine sequences were found with either the same annotation or showing similarity to the same human mucin sequence, the respective genomic and mrna sequences were examined with seqman software  for possible overlaps. in the case of such overlap, the sequences were assembled to build up a contiguous genomic and transcript sequence and subsequently manually curated to identify open reading frames and protein sequences. the curations were based on comparisons with the human genomic and mrna sequences, blast searches and bovine est sequences, as well as sequencing of the overlap after pcr on samples collected from bovine gi tissues. when no overlap was found, a pcr approach was used on cdna samples in order to investigate whether the predicted partial mucin sequences belonged to the same gene or transcript. amplified fragments were extracted from agarose gel with the nucleospin® extract kit , cloned into the pgem®-t-easy vector  and transformed to escherichia coli dh5α cells . transformants were screened for inserts, according to the manufacturer's instructions and inserts were pcr-amplified with primers against the t <dig> and sp <dig> promoters located on the pgem®-t-easy vector. pcr products were cleaned-up and sequence-analyzed at the centre for medical genetics, belgium  <cit> . all the new sequences identified were then submitted to the embl database as mrna direct entries or as experimental tpa  entries.\n",
            "\n",
            "protein structure analysis\n",
            "the protein sequences of the human and bovine mucins were blasted against the smart database  <cit>  in the normal smart mode, searching for pfam domain and internal repeats. pts regions were identified either by smart as regions of low complexity  <cit>  or by hand by calculating the percentage of p + t  and s  residues in the repeats. protein sequences were analyzed for signal sequences using the signalp  <dig>  server  <cit> .\n",
            "\n",
            "sample collection\n",
            "tissue samples were obtained at the slaughter house from four healthy cows, i.e. two holstein and two mixed breed animals . the animals selected were four years old with an average weight of  <dig> kg. oesophagus, rumen, reticulum, omasum, abomasum , duodenum, jejunum, ileum, caecum, colon and rectum from the gi tract were included. tissues were snap frozen on liquid nitrogen and stored at -80°c until rna was extracted.\n",
            "\n",
            "rna and genomic dna extraction\n",
            "total rna was extracted from tissue samples using the rneasy mini kit . to remove contaminating genomic dna , on-column dnase digestion was performed using the rnase-free dnase set  according to the manufacturer's instructions. rna quality was verified using an experion™automated electophoresis system , and concentrations were determined using a nanodrop nd- <dig> spectrophotometer . for all samples, the rna quality indicator  calculated by the experion™ software  was > <dig> , indicating high rna integrity. genomic dna contamination was investigated with the superscript one-step rt pcr kit  using intron-spanning primers for glyceraldehyde-3-phosphate dehydrogenase  . genomic dna was extracted from one tissue sample using the dneasy blood and tissue kit  as recommended by the manufacturer and it was used as a positive control for primer specificity in subsequent pcr reactions.\n",
            "\n",
            "aal: cdna amplicon length\n",
            "\n",
            "bgapdh intron-spanning primers, used to investigate gdna contamination\n",
            "\n",
            "cdna synthesis and pcr\n",
            "one μg of total rna was converted to cdna using the iscript cdna synthesis kit , following the manufacturer's instructions. primers used to perform the mucin pcrs are listed in table  <dig> and were designed with an annealing temperature of 60°c using the primer <dig> software  <cit> . pcr amplifications were carried out using the gotaq® flexi dna polymerase kit . a total reaction volume of  <dig> μl was used, containing  <dig> μl of single-stranded cdna ;  <dig>  μm forward primer;  <dig>  μm reverse primer;  <dig>  u gotaq® dna polymerase;  <dig>  mm each dntp ;  <dig>  mm mgcl <dig>  1x green gotaq® flexi buffer. the pcr reactions were cycled as followed: an initial denaturing step of  <dig> min at 95°c, followed by  <dig> cycles of 30s at 95°c, 30s at 60°c,  <dig> s at 72°c and a final extension of  <dig> min at 72°c. pcr products were analyzed by electrophoresis in  <dig> % agarose  gel at a constant voltage of  <dig> v with 1x tris-acetate edta  buffer. dna was visualised by ethidium bromide and analyzed using image quant™  <dig>  image background was subtracted using the rolling ball method and band volumes were normalized against the known volume of the ladder  <dig> bp band  in each gel. values obtained were arbitrarily grouped and bands classified as absent , present weak  and present strong . to control the quality of the synthesized cdna in all the samples and to validate the classification based on band intensity, pcr on two housekeeping genes, gapdh and β-actin , was also performed.\n",
            "\n",
            "RESULTS\n",
            "cell surface associated mucins\n",
            "searching the latest bovine genome assembly resulted in the identification of  <dig> sequences encoding  <dig> different cell surface associated mucins . the percentage similarity, at amino acid level, between the predicted bovine mucins  and their human homologues is reported in table  <dig>  a single bovine gdna sequence encoding a predicted mrna sequence was found for muc <dig>  muc3a, muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig> and muc <dig> . two muc16-like sequences were identified and no bovine homologous sequences were found for human muc <dig> and hmuc3b. although all these bovine mucins showed low  similarity with their human homologues, the overall protein architecture was found to be conserved , with the exception of muc3a, muc <dig> and muc <dig> \n",
            "\n",
            "cds: coding sequence\n",
            "\n",
            "gdna: genomic dna accession number in genbank\n",
            "\n",
            "mrna: messenger rna accession number in genbank\n",
            "\n",
            "%: percentage amino acid similarity between human and bovine mucin homologue\n",
            "\n",
            "fl: full length\n",
            "\n",
            "3'p: 3'partial, 5'p: 5'partial\n",
            "\n",
            "nd: not determined\n",
            "\n",
            "* embl accession number assigned to the newly generated sequences\n",
            "\n",
            "** mrna sequence retrieved from the mucin biology group mucin database  <cit> \n",
            "\n",
            "the predicted bmuc3a was found to contain a signal peptide  and a pts repeat region. however, the c-terminal egf, sea and tm domains, present in the hmuc3a protein sequence, were not found in the predicted sequence . analysis of bmuc <dig> on the other hand, did show the presence of the c-terminal sea, egf-l and tm domains, as seen in hmuc <dig>  but there was no n-terminal sp or pts region . at genomic level, bmuc3a is located immediately upstream of bmuc <dig> on chromosome  <dig> and has the same orientation. however, there is gap in the genomic sequence of about  <dig> kb between both genes, explaining why the 3'end of bmuc3a and the 5'end of bmuc <dig> are still missing. for bmuc <dig>  a sp and pts repeat have been identified, but no other domains present in hmuc <dig> have been found. although the bmuc <dig> nucleotide sequence  was discontinued from the database, transcription of bmuc <dig> was evident in gi tissue samples. however, since bmuc <dig> has not been mapped on the current genome build btau_ <dig> , no further analysis to identify a putative 3'end was performed.\n",
            "\n",
            "two partial bmuc <dig> mrna , together with their respective gdna sequences , were found in the database. alignment of these sequences showed an overlapping region in both the gdna and mrna sequences, which was subsequently confirmed by pcr and sequencing. the new full-length gdna and mrna sequence was submitted to embl under accession number embl: bn <dig> . comparison of the domain architecture of bmuc <dig> with the human homologue showed that the protein structure was largely conserved , with the exception of a signal peptide only found in the bovine sequence.\n",
            "\n",
            "secreted mucins\n",
            "twelve bovine gdna and predicted mrna sequences were found in the sequence dataset, predicted to encode  <dig> secreted mucins . gel-forming muc <dig>  muc5ac, muc5b, muc <dig>  muc <dig>  bsm as well as the non gel-forming muc <dig> were identified. no putative bovine homologue was found for human muc <dig> .\n",
            "\n",
            "two partial bovine muc2-like mrna sequences , together with their corresponding gdna , were found in the database. alignment with the human muc <dig> homologue showed that one bmuc2-like mrna sequence  aligned to the 5'end of hmuc <dig> and the other one  to the 3'end, with no overlapping region. in an attempt to identify and sequence the missing central part of bovine muc <dig> mrna, a pcr was performed on cdna samples of the colon using a primer set spanning the missing region. the amplicon obtained in the pcr was subsequently cloned and sequenced. the two predicted sequences and the newly experimentally identified sequence were aligned and used to compose a new full-length bmuc <dig> mrna sequence that was submitted to embl under the accession number bn <dig> . at genomic level, the same approach was taken but without success. further blast search of the mrna sequence against the bovine genome did not result in the identification of this region; therefore the gdna sequence encoding the identified central part of the muc <dig> mrna still remains unknown.\n",
            "\n",
            "two predicted bmuc5-like sequences  were found in the database. sequence analysis indicated that the mrna sequence xm_ <dig> showed the highest similarity to hmuc5ac and was therefore annotated as bmuc5ac in the embl database  . the predicted mrna sequence with genbank accession number xr_ <dig> showed the highest similarity to hmuc5b. at genomic level, bmuc5ac and bmuc5b sequences were found to overlap, indicating that the bmuc5ac and bmuc5b genes are located in a cluster on chromosome  <dig>  according to the database, exons  <dig> to  <dig> of the predicted bmuc5b mrna sequence were identical to exons  <dig> to  <dig> of the predicted bmuc5ac mrna. to investigate whether bmuc5ac and bmuc5b are indeed alternatively spliced, or whether this has been wrongly annotated, the 5' end of the bmuc5b genomic sequence was analyzed for putative exons using the genscan server  <cit> . eight putative exons were identified encoding muc5b like sequences. transcription of this newly identified bmuc5b 5'end was experimentally shown by pcr amplification on cdna. the corrected bmuc5b mrna sequence has now been submitted under accession number bn <dig> .\n",
            "\n",
            "at protein level, the predicted bmuc5ac showed an overall similarity of  <dig> % with the human homologue, although the similarity increases if comparing the n-terminal region  before the pts repeats  and the c-terminal region  after the pts repeats  . similarly, bmuc5b showed a low overall similarity  with the human homologue, but  <dig> % and  <dig> % similarity at the n-terminal region , and the c-terminal region  respectively . some domains of the hmuc5ac and hmuc5b protein are not found in bmuc5ac and bmuc5b . hmuc5ac has an n-terminal sp, vwd, c <dig>  til and vwc domain , of which til is not present in bmuc5ac . these domains are followed by a second c <dig>  til and vwc domain in hmuc5ac , while only the second c <dig> domain in this region is found in bmuc5ac. a second vwd and third c <dig> domain before the pts region are present both in hmuc5ac  and bmuc5ac . at the c-terminus of hmuc5ac, a vwc, vwd, c <dig>  a second and third vwc and ct domain have been found, but in bmuc5ac, only the vwd, c <dig> and a vwc corresponding to the third vwc in hmuc5ac have been identified. likewise, the organization of the domains is different for hmuc5b and bmuc5b. while hmuc5b contains the typical n-terminal sp, vwd, c <dig>  til, vwc domain organization , only the sp and vwd  have been conserved in bmuc5b, while the c <dig>  til and vwc from hmuc5b have been replaced with a second vwd in bmuc5b . the domain organization of the remaining protein sequence is similar between hmuc5b and bmuc5b.\n",
            "\n",
            "two partial mrna sequences were found in the database for muc <dig>  one bovine sequence  showed to align to the 5' end of hmuc <dig> and one  to the 3'end. for both muc6-like sequences corresponding gdna sequences were also identified in the database. the alignment of all the bovine sequences found did not show any overlap at mrna or gdna level. therefore, an attempt to identify the missing central region was done using a pcr approach, but without success. although the similarity between hmuc <dig> and the n-terminal part of bovine muc6- <dig>  is  <dig> %, the domain organization was found to be different . in humans, sp, vwc, c <dig>  til and vwc domains  are present in the n-terminal region followed by c <dig>  vwc and c <dig> domains. instead, in bovine muc6- <dig> only a vwd and a til domain are present in the n-terminal region followed by c <dig>  til and vwd domains. in the central part of hmuc <dig> there is a pts repeat region, which is still not identified in cattle .\n",
            "\n",
            "three partial sequences with similarity to human muc <dig> were identified in genbank. two sequences  were found to align to the 5' end of hmuc <dig>  alignment of these two sequences showed an overlap at both the mrna and gdna level; hence a longer 5' partial bmuc <dig>  was constructed and submitted to the database under accession number bn <dig>  the third sequence identified in the database was the one encoding for bovine submaxillary mucin . bsm is one of the few bovine mucin previously described  <cit>  and it was found to have a high similarity  with the c-terminal part of hmuc <dig> at protein level. the predicted bmuc <dig> and bsm genes are both on chromosome  <dig> in cattle, separated by about  <dig> kb of genomic sequence, suggesting that bsm together with the predicted bmuc <dig> might belong together and form one unique muc <dig> gene. to confirm this hypothesis the genomic sequence between bsm and bmuc <dig> was analyzed using the genscan server  <cit> . only short exons  could be predicted in this region and none of them encoded for mucin domains or pts-rich peptides. also in humans the central pts region of muc <dig> is unknown, but a putative full-length hmuc <dig> has been proposed by chen et al.   <cit>  . on the other hand in mice and in pigs full-length genes homologous to hmuc <dig> have been previously identified, named muc <dig> and porcine submaxillary mucin , respectively. at protein level, alignment of both bmuc <dig> and bsm with human, mouse and pig muc <dig> suggests that all the expected domains, typical for muc <dig>  are represented in bmuc <dig> and bsm. assuming that the complete bovine muc <dig> has a similar length compared to psm and mouse muc <dig>  it would mean that the unknown region between bsm and bmuc <dig> is about  <dig> aa long. for this reason pcr was performed on jejunal samples in an attempt to identify this putative linkage region. however, no amplification was obtained, likely due to the presence of a repetitive sequence encoding the pts-rich regions.\n",
            "\n",
            "membrane associated and secreted mucins are differentially transcribed in the gastrointestinal tract\n",
            "to understand the transcription profile for all the known bovine mucins in the gastro-intestinal tract, pcr was performed on cdna of samples collected from several tissues. the results obtained showed a characteristic transcription profile for many of the mucins analyzed in both the membrane associated and the gel-forming groups . among the membrane associated mucins, muc <dig> and muc <dig> were majorly transcribed, with occurrence of bands in all the tissues examined. muc <dig>  muc <dig>  and muc <dig> were found to be transcribed in the first part of the gi tract with the strongest presence in the oesophagus of all the animals analyzed, while muc3a and muc <dig> were transcribed in the small and large intestine only. in contrast, muc <dig> was hardly present in any of the tissues analyzed with only weak bands in few samples collected from the large intestine. muc <dig> transcription was only detected in two samples collected in the oesophagus and one in the fundic region of the abomasum, caecum, colon and rectum.\n",
            "\n",
            "in regard to the secreted gel-forming mucins, muc5ac and muc <dig>  were found to be consistently transcribed in the abomasum, both fundic and pyloric regions, and in the first part of the intestinal tract . muc <dig> and muc5b were only transcribed in the intestinal section of the gi tract with bands present in the majority of the tissues collected from this area. muc <dig> and bsm showed a similar transcription pattern, with weak to strong bands in  <dig> animals in the jejunum and weak transcription in one sample in the rectum. muc <dig> was also found to be weakly transcribed in the omasum of one animal, whereas the bsm transcript was not detectable in this sample. the secreted non gel-forming mucin muc <dig> was transcribed in a limited number of tissues analyzed. finally, for the two control genes, gapdh and actb, strong bands were obtained in all the samples analyzed.\n",
            "\n",
            "discussion\n",
            "the current study identified  <dig> bovine mucin-encoding genes in the cattle genome , including nine membrane-associated mucins and six secreted ones. compared to human, no homologues could be identified for muc3b, muc <dig> and muc <dig>  this feature is not unique to cattle since previous studies showed that these three mucins were also absent in rodents  <cit> . in humans, the two genes encoding muc3a and muc3b show a high identity at nucleotide level both in the intronic and exonic sequences, suggesting that they are the product of a rather recent gene duplication after the divergence of humans and rodents  <cit> . this hypothesis would explain why muc3a and muc3b are found in chimpanzee, which diverged from humans later compared to rodents, but not in cows and other ruminants which diverged from humans earlier than rodents  <cit> . in addition, high similarity among human muc <dig> and rodent muc <dig> has previously been shown  <cit> . although it has previously been suggested that human muc <dig> is the true structural homologue of rodent muc <dig>  <cit> , the conserved localization of the muc <dig> gene upstream of muc <dig> and muc <dig> in humans and upstream of muc <dig> in mice and cattle, suggests that muc <dig> in the latter two species is the homologue of human muc <dig> rather than of muc <dig>  in regard to muc <dig>  blast analysis suggested that humans are the only species having this gene in their genome .\n",
            "\n",
            "as previously stated, sequencing mucin genes is technically difficult due to their size and the large number of repetitive sequences in the pts regions. therefore, some of the mucin gene and mrna sequences are still incomplete. for example for muc <dig>  sequence information for the central pts region is still missing. similar problems were encountered for muc3a, muc <dig>  muc <dig> and muc <dig> \n",
            "\n",
            "in humans, two clusters of mucins have been previously described. muc <dig>  muc <dig> and muc <dig>  encoding membrane bound mucins, are clustered on human chromosome 7q <dig>  <cit> , while muc <dig>  muc <dig>  muc5ac, and muc5b, encoding secreted gel-forming mucins, are clustered on human chromosome 11p <dig>  <cit> . with the exception of muc <dig>  for which no bovine homologue has been identified, both gene clusters appear to be maintained in cattle, on bovine chromosome bta <dig> and bta <dig>  respectively. this conservation has also been described for the muc <dig>  muc <dig>  muc5ac and muc5b gene cluster in mice  <cit> .\n",
            "\n",
            "alignment of the bovine and human sequences showed that, although the characteristic regions and domains of mucin glycoproteins are generally conserved between these species, the overall sequence similarity is very low with values included between  <dig> % and  <dig> %. the reason for this low similarity is probably related to the presence of the pts regions, which are characteristic of these proteins  <cit> . this is consistent with previous studies that compared human and rodent mucins  <cit> . in general, domain architecture of the membrane-associated mucins was found to be similar between humans and cattle, while the protein architecture of the gel-forming mucins appeared to be less conserved. the bovine muc5ac, muc5b, and muc <dig> proteins did not show the vwd-c8-til-vwc domain organization which is typical for the n-terminus of all human secreted gel-forming mucins. bovine muc5ac contained no til domains over its entire length, while bmuc <dig> was shown to have no vwc domains. however, all the domains responsible for the oligomerization of these molecules, such as vwd and cystein rich domains, are present. in view of the sequence variation detected compared to the human muc genes, it is still unclear what the variation is in the muc genes of different cattle breeds. high variability in the number of vntrs and the extent of vntr polymorphism within and across cattle breeds has only been reported for the muc <dig> gene  <cit> . the level of allelic variance for the other bovine muc genes is still unclear.\n",
            "\n",
            "bsm is one of the few mucins previously isolated and studied in cattle  <cit> . it is located on chromosome  <dig> and high sequence similarities have been shown with the c-terminal part of porcine and ovine submaxillary mucins . in  <dig>  <cit> , human and mouse muc <dig> were reported as new members of the gel-forming mucin family. their sequence similarity, similar expression pattern , and chromosomal location compared to bsm and psm suggested that they were potential homologues. the previously reported bsm sequence lacked the n-terminal conserved protein domains present in psm and other muc <dig> sequences  <cit> , suggesting that it was not complete. in the current study, a new mrna and corresponding gdna sequence  was identified showing homology to the 5' part of hmuc <dig>  this gdna sequence is located approximately  <dig> kb upstream of the bsm gene on chromosome  <dig>  the genomic localization and the almost identical transcription profile of both genes would suggest that bsm and this newly identified bmuc <dig> are actually part of one gene encoding the bovine muc <dig> homologue.\n",
            "\n",
            "at the moment, mechanisms regulating human mucin transcription are largely unknown and promotor sequences have not been identified for all human muc genes. the transcription factors nf-kappab and ap- <dig> seem to be involved in the regulation of a number of human muc genes, including the membrane bound mucin muc <dig>  <cit>  and the secreted mucins muc <dig>  <cit> , muc5ac  <cit> , muc5b  <cit>  and muc <dig>  <cit> . in cattle, promotor sequences are available only for muc <dig>  muc3a, muc5b, muc <dig>  muc <dig>  muc <dig> and muc <dig>  in silico promotor analysis showed potential binding sites for ap- <dig> in the secreted mucin muc5b and for ap- <dig> and c-rel/nf-kappab in all the membrane bound mucins, with the exception of muc <dig> . as more sequence data become available, it will be interesting to investigate if different regulatory mechanisms exist within the different classes of mucin genes and whether they can affect  physiological conditions.\n",
            "\n",
            "transcriptional analysis of the  <dig> identified mucin genes showed that most of them are transcribed in the gi tract of cattle. although some individual differences were observed, in general, for the membrane associated mucins, muc <dig> and muc <dig> transcripts were detected in all the tissues examined, whereas muc <dig> seemed to be mainly produced in the intestinal tissues. this is consistent with the observations made in humans  <cit> . muc <dig>  muc <dig> and muc <dig>  on the other hand, were the major membrane-associated mucins found in the oesophagus. a search of the human est dataset revealed the presence of muc <dig> and muc <dig> encoding ests derived from an oesophageal cdna library , suggesting that their transcription pattern might be conserved. concerning the secreted mucins, muc5ac and muc <dig> were primarily transcribed in the abomasum, whereas muc <dig> and muc5b were the main secreted mucins in the intestinal tissues. this is largely in agreement with the situation in humans, except for muc5b, for which evidence in the human intestine has only been reported in colonic tissue  <cit> . in contrast to our results in cattle, the human muc5b is also found in the oesophagus, where it is transcribed in the oesophageal submucosal glands . smgs are known to differ in distribution and structures among species  <cit> , so it is possible that the oesophageal samples collected in this study did not include regions rich in these glands. alternatively, it is also possible that in cattle smgs do not produce muc5b. interestingly, secreted mucins were not found to be produced in the pre-stomachs. since secreted gel-forming mucins are widely acknowledged as an important component in the formation of viscoelastic mucus in stomach and intestine to protect the mucosal epithelium against acidic and proteolytic damages  <cit> , the absence of gastric juices, hydrochloric acid and digestive enzymes in the pre-stomach may explain this lack of expression. on the other hand, the pattern of gene expression of both the membrane bound and secreted mucins in the pre-stomachs was very similar to the pattern observed in the oesophagus, which could be due to the common origin that these organs have during bovine embryogenesis  <cit> . although the transcription pattern of bovine mucins in the gi tract is genererally similar to what is observed in humans, transcription levels do not always correspond with protein levels and different factors can influence the functional properties of the mature protein. alternative splicing can generate different isoforms with different tissue distributions and properties. for several human membrane associated mucins, multiple isoforms have been identified, some of which are secreted  <cit> . in addition, posttranslational proteolysis of the sea domain can also release the extracellular domain of membrane bound mucins into the mucus layer  <cit> . since the large pts regions constitute o-glycosylation sites, variance in the number, length and sequence of the repeats can impact the extent and type of glycosylation and thus the biological functions of mucins that are largely defined by their carbohydrate constitution  <cit> . only a limited amount of studies have been performed to investigate these aspects in bovine mucins  <cit>  and future studies will need to include these different factors influencing mucin biology.\n",
            "\n",
            "CONCLUSIONS\n",
            "this study provides the first characterization of the mucin gene family in cattle and their transcriptional distribution in the gi tract. homologues were identified for all members of the human mucin family, with the exception of human muc3b, muc <dig> and muc <dig>  the protein domain architecture of the membrane-associated mucins was found to be conserved between humans and cattle, while protein architecture of the gel-forming mucins appeared to be less conserved. analysis of the transcription profile showed that the secreted mucins were transcribed from the abomasum onwards, whereas the membrane associated mucins muc <dig> and muc <dig> were transcribed throughout the whole gi tract.\n",
            "\n",
            "authors' contributions\n",
            "this article is a part of ph's phd thesis. mr helped in performing the transcription studies. rl contributed to the genome mining. bg, ec, jv and pg conceived and designed the project. all authors read and approved the final manuscript.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "this study provides the first characterization of the mucin gene family in cattle, with the exception of human muc3b, muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig>  muc <dig> and muc <dig> \n",
            "24\n",
            "[{'summary_text': 'our results suggest that apobec <dig> is required to acquire and maintain muscle-cell identity during myogenesis.\\nour results suggest that apobec <dig> is required to acquire and maintain muscle-cell identity during myogenesis, and that downregulation of the muscle-specific cytidine deaminase apobec <dig> in es-derived myogenic precursors reduces myogenic-associated dna demethylation and impairs muscle differentiation.\\nour results suggest that apobec <dig> is required to acquire and maintain muscle-cell identity during myogenesis, and that downregulation of the muscle-specific cytidine deaminase apobec <dig> in es-derived myogenic precursors reduces myogenic-associated dna demethylation and impairs muscle differentiation.\\nour results suggest that apobec <dig> is required to acquire and maintain muscle-cell identity during myogenesis, and that downregulation of the muscle-specific cytidine deaminase apobec <dig> in es-derived myogenic precursors reduces myogenic-associated dna demethylation and impairs muscle differentiation.\\nour results suggest that apobec <dig> is required for muscle-specific demethylation'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "developmental signals induce changes in gene expression and chromatin structure in order to define cell identity. epigenetic mechanisms, and dna methylation in particular, are essential for mammalian development being involved in the establishment and maintenance of cellular identity by creating cellular memory during embryonic development . dna methylation controls x-chromosome inactivation, expression of imprinted genes, represses the transcription of repeated sequences and prevents relocation of transposable elements . in addition, dna methylation affects gene expression by regulating promoters and distal regulatory elements, such as enhancers and insulators . in the last few years, global analyses of dna methylation dynamics in pluripotent stem cells and their differentiated progeny have begun to uncover the mechanisms involving coordinated changes occurring in the epigenome that are essential for lineage-specification and maintenance of cellular identity . very recently, we reported the genome-wide comparison of the dna methylation profiles of pluripotent embryonic stem cells and muscle stem cells showing, in agreement with other studies, that cellular differentiation is associated with a modest but global increase in dna methylation, which interestingly is accompanied by lineage-specific demethylation  <cit> . although the mechanisms controlling dna demethylation are still not well defined, several studies suggest the involvement of two major pathways in this process: the activity of tet methylcytosine dioxygenase proteins  <cit>  and aid/apobec cytidine deaminase enzymes . importantly, active dna demethylation was reported in human muscle-specific genes in heterokaryons derived from the fusion of human non-muscle cells with mouse muscle cells, indicating a dynamic dna methylation/demethylation interplay during muscle differentiation  <cit> .\n",
            "\n",
            "skeletal muscle is the tissue accounting for the largest percentage of body mass and contributes to multiple body functions including the voluntary movements. the skeletal muscle is mainly composed of highly specialized and terminally differentiated multinuclear, post mitotic and contractile myofibres, and a small pool of muscle stem cells, known as satellite cells , which are essential for the growth, maintenance and regeneration of skeletal muscle throughout life , as well as silencing of other lineage-specifying genes . in addition, members of the pax gene family pax <dig> and pax <dig> lie upstream of mrfs and their expression is crucial to regulate muscle progenitor cell functions  <cit> . pax <dig> is essential for the formation and maintenance of scs, being expressed in quiescent and activated scs, as well as in proliferating myogenic progenitors  <cit> .\n",
            "\n",
            "in the present study, we addressed for the first time the dna methylation dynamics of the major genes orchestrating myogenic determination and differentiation by comparing pluripotent escs, myogenic precursors from pax7-inducible escs, proliferating muscle stem cells, and their respective myotube derivatives. our results showed a common muscle-specific dna demethylation signature required to acquire and maintain the muscle-cell identity. notably, downregulation of the muscle-specific cytidine deaminase apobec <dig> in es-derived myogenic precursors reduced myogenin-associated dna demethylation and dramatically affected the expression of differentiation markers and, ultimately, muscle differentiation.\n",
            "\n",
            "RESULTS\n",
            "dna methylation profile of myogenic genes harbouring cpg island-promoters during myogenic differentiation\n",
            "to elucidate dna methylation dynamics associated with skeletal myogenic lineage commitment and terminal differentiation, we focused the analysis on genes known to be associated with this process: developmental genes , myogenic regulatory factors  and terminal differentiation genes . interestingly, these genes could also be classified according to their cpg distribution, which represent sites of potential dna methylation. pax <dig>  pax <dig> and myod contain cpg islands  in their promoters, belonging to the cpg-rich genes category; meanwhile, the other genes do not have cgi and their promoters are considered cpg-poor. using bisulphite sequencing analysis, we compared the dna methylation state of undifferentiated escs and muscle stem cells, isolated from adult skeletal muscle tissue, along with respective differentiated myotubes and mature myofibres . in addition, to assess whether the dna methylation events were muscle-lineage specific, we analysed the methylation profiles of the following non-myogenic cell lines: neuronal precursor cells , mouse embryonic fibroblasts  and cardiomyocytes . as shown in additional file 1a, b, all analysed cgis in pax <dig> and pax <dig> regulatory regions were completely unmethylated in all the samples. similar results were previously reported for myod cgi  <cit> . these results were expected since cgis, usually located in the promoter regions of housekeeping and developmental genes, are known to be largely resistant to dna methylation .fig.  <dig> epigenetic profile of myogenic genes harbouring a cpg island-promoter during myogenic differentiation. a diagram of the myogenic differentiation model and the main genes driving myogenesis. cpg-rich and cpg-poor promoter genes are indicated in green and in blue, respectively. b scheme of the pax <dig>  pax <dig> and myod genes and their associated cgis  and enhancer regions . dna methylation and expression of each gene were analysed in two biological duplicates of escs, myoblasts , myotubes , myofibres , and npc, hl <dig> and mef cell lines. each circle represents a cpg dinucleotide, and its position relative to the tss is indicated below. the colour gradient represents the level of methylation indicated in the legend. bar charts show gene expression values measured by qrt-pcr and normalized by 18s expression . n.d. means non detectable. c histone marks distribution and p <dig> binding in esc , mb  and mt  obtained from encode project and dynlatch’s lab  <cit> . cgis cpg islands, escs embryonic stem cells, npcs neuronal precursor cells, hl <dig> cardiomyocytes, mefs mouse embryonic fibroblasts, tss transcription start site\n",
            "\n",
            "\n",
            "\n",
            "since dna methylation often occurs in non-cgi regions, we investigated whether enhancers and promoters present a cell-specific deposition of this modification. previous studies of other groups identified two muscle-specific regulatory regions upstream of the myod transcription start site , located at -20 kb and -5 kb, respectively . importantly, it was shown that the distal myod enhancer located at -20 kb of the tss was modulated by dna methylation in mouse tissues  <cit> . therefore, we analysed the methylation status of pax <dig> hypaxial somatic enhancer  <cit>  located at -8 kb of the pax <dig> tss, the pax <dig> region containing the critical rbp-jκ binding site  <cit>  located at - <dig>  kb of pax <dig> tss, and the two myod enhancer regions. as shown in fig. 1b, although these regulatory regions were totally or partially methylated in escs and in non-muscle cells, pax <dig> and -20 kb myod enhancers were almost completely free of methylation in myogenic cells, which correlates with gene expression . on the contrary, the pax <dig> and -5 kb distal regulatory region of myod were both found highly methylated in muscle cells , suggesting that their activation would be independent of dna methylation. notably, npcs and hl <dig> cells presented high levels of pax <dig> expression, despite high dna methylation levels. this result would suggest that the pax <dig> hypaxial enhancer might be mainly associated to enhance pax <dig> expression in committed skeletal myogenic cells.\n",
            "\n",
            "next, to further characterize the epigenetic landscape involved in myogenic regulation, we took advantage of publicly available chip-seq data of histone post-translational modifications  <cit> . as schematized in fig. 1c, pax <dig>  pax <dig> and myod promoters showed a bivalent chromatin state characterized by histone  <dig> trimethylated on lysine  <dig>  and lysine  <dig>  in escs, which has been associated to poised transcription  <cit> . this bivalent state was clearly resolved in favour of the positive mark h3k4me <dig> at myoblast  and myotube  stages for myod, whereas the developmental genes pax <dig> and pax <dig> retained the bivalent state . the analysis of these loci, including enhancer and distal regions, showed a gain in deposition of histone  <dig> monomethyl-lysine <dig> , acetyl-lysine  <dig>  and increased recruitment of the p <dig> acetyltransferase at mb stage, and also at mts in the case of myod. since myod is already higher expressed in mbs, this maintenance of active enhancer marks in mts might be involved in keeping an open chromatin state of the region to maintain gene expression, rather than promoting further transcriptional activation. all together, these results indicate that the unmethylated state of cgis is conserved across the different cell types, whereas pax <dig> and -20 kb myod enhancer element show cell type-specific dynamic methylation patterns associated with changes in gene expression.\n",
            "\n",
            "myogenic-specific promoters undergo dna demethylation during muscle-lineage specification\n",
            "unlike myod, all the other mrfs myf <dig>  mrf <dig> and myog, as well as the muscle differentiation markers display low cpg content promoters. the analysis of the promoter region, containing e-box and mef <dig> binding sites, of myog and the other differentiation genes ckm, myh <dig>  myh <dig> and myh <dig> revealed high levels of methylation in escs and non-muscle cells, correlating with gene silencing, whereas the promoter regions in muscle cells were unmethylated . interestingly, these differentiation genes became demethylated at the mb stage but remained silenced. these myogenic genes were not bivalent at the esc stage and only presented active chromatin marks  in the promoter regions according to their expression timing. strikingly, the terminal differentiation genes myh <dig>  myh <dig> and myh <dig> gained the repressive mark h3k27me <dig> in mb cells after dna demethylation, suggesting that gene expression was maintained epigenetically repressed by prc <dig> complex until the onset of the terminal differentiation stage . interestingly, mrf <dig> followed a different dna methylation pattern, being highly demethylated only in mature myofibres, where it is specifically highly expressed .fig.  <dig> epigenetic profile of muscle-specific cpg-poor promoter genes during myogenesis. a scheme of the myog, mrf <dig>  ckm and myh <dig>   <dig> and  <dig> loci. regions analysed by sodium bisulphite sequencing are indicated in red. two biological replicates were performed for escs, myoblasts , myotubes  and myofibres  and in npc, hl <dig> and mef cell lines. each circle represents a cpg dinucleotide and its position relative to the tss is indicated below. the colour gradient represents the level of methylation indicated in the legend. bar charts show gene expression values measured by qrt-pcr and normalized by 18s expression . n.d. means non detectable. b histone marks distribution and p <dig> binding in esc , mb  and mt  obtained from encode project and dynlatch’s lab  <cit> . escs embryonic stem cells, npcs neuronal precursor cells, hl <dig> cardiomyocytes, mefs mouse embryonic fibroblasts, tss transcription start site\n",
            "\n",
            "\n",
            "\n",
            "we also investigated the epigenetic profile of highly expressed pluripotency genes in esc that become silenced in mb and mt. genes containing a cgi overlapping their promoter region, such as sox <dig> and fgf <dig>  were totally unmethylated in all cell types, whereas genes with cpg-poor content promoters, such as dppa <dig> and pou5f <dig>  became highly hypermethylated upon cellular differentiation . regarding histone marks, esc showed high levels of h3k4me <dig> in the promoter regions of sox <dig> and fgf <dig>  whereas mb and mt cells gained h3k27me <dig> in association with gene silencing . dppa <dig> and pou5f <dig> promoters presented also the active marks h3k4me <dig> and h3k27ac. however, in differentiating cells the h3k27me <dig> mark was not acquired  or in a very small extent , leaving the gain of dna methylation as the postulated epigenetic silencing mechanism .\n",
            "\n",
            "all together, these results suggest that myogenic differentiation genes become demethylated at cpg-poor regulatory regions during muscle-lineage determination, and are maintained silenced by the polycomb repressive histone mark h3k27me <dig> until gene activation. in contrast, pluripotency gene silencing involves complete dna methylation or gain of h3k27me <dig> mark, depending on promoter cpg context.\n",
            "\n",
            "pax7-induced es-derived muscle differentiation recapitulates myogenic dna methylation changes\n",
            "in order to recapitulate the commitment of myogenic precursors during embryonic development, we utilized mouse es cells engineered to express pax <dig> under the control of a doxycycline-inducible promoter   <cit> . at day  <dig> of embryoid body  differentiation pax <dig> transgene expression was induced by doxycycline , and at day  <dig> of eb differentiation the mesodermal precursor cells  were sorted, with or without dox, and grown for  <dig> more days . importantly, only the ipax7-es-derived myogenic precursors treated with dox expressed myogenic differentiation markers such as myogenin, ckm and myh <dig> and generated myotubes upon differentiation . to assess to what extent the ipax7-es-derived myogenic progenitors were epigenetically reprogrammed, we evaluated the dna methylation state of the main myogenic genes in three independent time courses. as expected, preliminary analysis revealed the complete unmethylated state of the cgi located at the pax <dig> and myod promoter regions during the differentiation process . importantly, the cpg-poor myogenic regulatory regions also showed a very similar dna methylation signature compared to primary muscle stem cells and, interestingly, a stepwise dna demethylation was identified . the pax <dig> hypaxial enhancer became completely demethylated at day  <dig> of eb formation, independently of pax <dig> induction and despite the lack of pax <dig> expression at any stage of differentiation . contrarily, the -20 kb enhancer of myod, the -111 kb enhancer of myf <dig>  and myf <dig> promoter became partially demethylated at a later time and exclusively in pax7-induced myogenic precursors, coinciding with the increase in mrna expression. notably, at day  <dig>  the demethylated state persisted, despite the dramatic reduction of myod and myf <dig> expression levels . myogenin promoter showed a similar demethylation time course, reaching a total loss of dna methylation at the ipax7-mb-precursor stage when myogenin was slightly expressed, and maintained in ipax7-mts, when the myogenin gene showed the maximum expression level . interestingly, the promoters of the late differentiation genes myh <dig> and ckm remained fully methylated until the ipax7-mt stage, when they became partially demethylated simultaneously with gene activation . importantly, we did not observe any loss of dna methylation in the absence of pax <dig> expression in the control cells , indicating that the observed demethylation was skeletal muscle-specific. finally, the analysis of the promoter regions of the pluripotency genes pou5f <dig> and dppa <dig>  which are progressively silenced, showed a similar gain of dna methylation during cell differentiation as observed in primary myoblasts .fig.  <dig> \n",
            "pax7-engineered escs efficiently acquired the myogenic dna methylation profile. a experimental scheme of the differentiation protocol to generate es-derived myogenic progenitors using ipax <dig> mes cells . pax <dig> induction assessed by qrt-pcr . b expression profiles of markers of pluripotency , muscle lineage commitment  and muscle differentiation  were measured by qrt-pcr at the successive time points of ipax <dig> esc-derived myogenic model and normalized by gapdh expression . c dna methylation dynamics of pax <dig>  myf <dig> and myod enhancers and cpg-poor promoters of muscle-specific genes  and pluripotency genes  during pax7-induced myogenesis . the results of three biological replicates are shown. each circle represents a cpg dinucleotide and the colour gradient represents the level of methylation indicated in the legend and assessed by sodium bisulphite sequencing. demethylation waves occurring during the pax7-induced myogenesis are denoted by red dotted boxes. escs embryonic stem cells, dox doxycycline; ipax <dig> inducible pax7\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "impaired myogenic differentiation in apobec <dig> knockdown myogenic precursors is associated with reduced dna demethylation\n",
            "dna demethylation at specific myogenic regulatory regions was clearly observed in primary myoblasts and in ipax <dig> esc-derived myogenic precursors . at first, we questioned whether the reported demethylation could result from a passive loss of methylation, in which the lack or the significant reduction of dna methyltransferases  coupled to the successive cell divisions would lead to the loss of this epigenetic mark. we analysed the expression levels of the dnmt <dig> responsible for the maintenance of the existent dna methylation pattern, and dnmt3a and dnmt3b, which add de novo methyl groups after dna replication. all dnmts were expressed, and, as shown in additional file  <dig>  the expression levels of dnmt <dig> and dnmt3a did not significantly change during eb differentiation, while dnmt3b expression decreased during cell differentiation independently of pax <dig> expression. these observations would suggest that specific dna methylation reduction on muscle-related genes might be mediated by an active mechanism rather than by a passive one.\n",
            "\n",
            "the two major pathways involved in controlling the dna demethylation process are the oxidation-mediated demethylation by tet proteins and the cytosine deamination by aid/apobec cytidine deaminase enzymes. the expression levels of tet <dig>  tet <dig> and tet <dig> did not change in the presence or absence of pax <dig> expression during the eb differentiation, as shown in additional file 4b. however, the expression levels of apobec <dig>  the skeletal muscle specific member of the aid/apobec family, were highly induced upon pax7-mediated muscle-commitment, being very high at the progenitor stage and derived mts . importantly, despite the lack of high specificity of the apobec <dig> antibody  a clear increase of apobec <dig> protein was observed by western-blot concomitant with the differentiation progression .fig.  <dig> impaired myogenic progression and differentiation and reduced dna demethylation in apobec <dig> knockdown myogenic precursors. a time course of apobec <dig> expression during differentiation of ipax <dig> es cells in presence or absence of pax <dig>  which was induced by adding doxycycline  to the culture media. apobec <dig> expression was measured by qrt-pcr , and by western blotting  . apobec <dig> mrna expression was normalized by gapdh expression  and as a reference the expression level in primary myoblasts  was measured . myog and actin wb were used as controls for myogenic differentiation and equal loading, respectively. b dna methylation and gene expression analyses of myogenin were performed in a daily time course after dox induction in the inducible myogenic model . each circle represents a cpg dinucleotide and its distance to the gene tss is indicated below. the colour gradient represents the methylation level indicated in the legend. n.d. means non detectable. c scheme of the experimental approach followed to knockdown apobec <dig> in the inducible pax <dig> model . gene expression and protein level to evaluate apobec <dig> knockdown efficiency using two independent clones were compared to control vector  . d\n",
            "myogenin expression was measured in pax7-induced myogenic precursors infected with apobec <dig> shrnas 4 days upon lentiviral transduction  . representative images of pax7-induced myogenic precursors infected with apobec <dig> shrnas under differentiation conditions stained with myogenin and mhc antibodies. scale bar: 100 μm. e dna methylation analysis by sodium bisulphite sequencing of the myogenin promoter in pax <dig> es-derived myogenic precursors transduced with apobec <dig> shrna <dig> and shrna <dig> in three biological replicates. each circle represents a cpg dinucleotide and its distance to the gene tss is indicated below. the colour gradient represents the level of methylation indicated in the legend. statistical significance with respect to the control samples is indicated with * for p-value <  <dig>  applying the kruskal-wallis test. esc embryonic stem cell, tss transcription start site, shrna short hairpin rna\n",
            "\n",
            "\n",
            "\n",
            "in order to test whether apobec <dig> is required for the myogenic process, we performed loss of function experiments by down-regulating apobec <dig> levels using lentiviral vectors encoding for specific shrnas. to identify the most appropriate time point for shrna viral-mediated transduction  we performed and analysed a daily time course starting from dox-treated pdgfrα+/flk1- sorted cells from day  <dig> ebs until ipax7-mb precursor stage . as shown in fig. 4b, we observed a perfect correlation between gradual dna demethylation of the myogenin locus and increased gene expression, which starts at eb day  <dig>  based on these data, we decided to transduce dox-treated pdgfrα+/flk1- cells at day  <dig> of differentiation using two different apobec <dig> shrnas . as expected, four days after transduction, apobec <dig> shrnas dramatically knocked-down apobec <dig> mrna and protein levels  without affecting pax <dig> expression . importantly, apobec <dig> knockdown was associated with a severe down-regulation of myogenin expression, both at mrna and protein level and, ultimately, impairment of terminal muscle differentiation . interestingly, the methylation analysis of the myogenin promoter upon apobec <dig> knockdown showed a significant methylation gain in 70 % of the promoter cpgs  compared to the control cells . finally, we also analysed the impact of apobec <dig> knockdown in myod and myf <dig> dna methylation levels, and, as shown in additional file 5a,b, although we observed a small increase of methylation in some of the cpgs, it was not statistically significant. all these results together, suggest that apobec <dig> might participate more actively in the dna demethylation process required to drive myogenic differentiation rather than in regulating the dna methylation-dependent myogenic commitment.\n",
            "\n",
            "discussion\n",
            "tissue-specific stem cells are lineage-restricted cells that have lost pluripotency capacities and gained cell-type identity. epigenetic regulation, including dna methylation and the post-translational modifications of histones, is crucial in the establishment and maintenance of cellular identity . in this study, we analysed for the first time the dna methylation dynamics of the principal genes orchestrating myogenic determination and differentiation programs by comparing es cells and their pax7-induced skeletal myogenic derivatives with muscle stem cells, both in proliferating and differentiating conditions. the detailed analysis of cpg-poor regulatory regions for the developmental gene pax <dig>  myod and myogenin genes as muscle cell-identity transcription factors, and ckm, mhy <dig>  myh <dig> and myh <dig> as myogenic terminal differentiation genes revealed a specific dna demethylation signature required to acquire and maintain muscle-cell identity during the establishment of the muscle lineage . these results reinforce observations reported in other cellular models that lineage-specific dna demethylation, together with pluripotency genes hypermethylation, occur during early cell fate decisions conferring unique dna methylation patterns which correlate with defined cellular identities . interestingly, our results showed that the crucial myogenic differentiation factor myogenin became fully demethylated in muscle-committed cells before gene expression . however, these results were in disagreement with previous work performed in c2c <dig> cells where loss of methylation was observed between myoblasts and myotubes  <cit> . to address this controversy, we isolated freshly satellite cells from adult mice, and, interestingly, the dna methylation analysis of the myogenin promoter revealed the complete demethylation of the promoter in ex vivo isolated muscle stem cells . these results would indicate that dna demethylation of myogenin promoter occurs during muscle-lineage determination, allowing the acquisition of a transcriptional poised state before gene expression during muscle differentiation. notably, gene silencing of terminal differentiation myosin genes in proliferating myoblasts upon dna demethylation is maintained by the acquisition of the polycomb repressive mark h3k27me <dig> , as previously reported for myog and ckm promoters  <cit> . during terminal differentiation, h3k27me <dig> is replaced by the positive marks h3k4me <dig> and h3k27ac, which result in gene transcription. this two-step repressive mechanism, where dna methylation is replaced by h3k27me <dig>  is also observed when escs are induced to differentiate  <cit>  and represents a switch from a stable repression mediated by dna methylation to a more facultative repression mediated by the histone code  <cit> .\n",
            "\n",
            "mrf <dig> is considered a determination and differentiation myogenic gene and is the only mrf expressed in mature myofibres  <cit> . curiously, mrf <dig> promoter is the only one still repressed by dna methylation in undifferentiated myoblasts, being highly demethylated only in mature myofibres where it is highly expressed . a recent paper demonstrated that adult satellite cells emerged from embryonic founder cells in which the mrf <dig> gene was activated  <cit> , but intriguingly, our results showed that mrf <dig> promoter was completely methylated in all cell types analysed except in mature myofibres. very recently we demonstrated that the multiple enhancers regulating the complex spatial-temporal expression of the myf5/mrf <dig> locus during embryogenesis were specifically demethylated in muscle-committed cells  <cit> . in that context, we speculate that complete demethylation of mrf <dig> promoter would be only required in mature myofibres to fully activate and maintain its high expression levels.\n",
            "\n",
            "it is well known that pax <dig> has a key role in the formation and maintenance of muscle committed progenitors. the engineered ipax7-esc model let us recapitulate the commitment of myogenic precursors during embryonic development and revealed the acquisition of a very similar dna methylation signature, compared to primary muscle cells . importantly, these results support the use of ipax <dig> es-derived myogenic progenitors as a bona fide model to generate in vitro myogenic precursors with therapeutic purposes. the detailed analysis of this stepwise model showed that the myogenic differentiation process occurred coupled to a gradual loss of dna methylation in myogenic regulatory regions in consecutive waves. initially, pax <dig> enhancer became demethylated before pax <dig> induction in line with its expression pattern, which is not restricted to the myogenic lineage  <cit> . next, myf <dig>  myod and myogenin cpg-poor regulatory regions became demethylated in ipax7-myoblast precursors correlating with transcriptional activation . finally, the terminal differentiation myogenic genes ckm and myh <dig> became simultaneously demethylated and activated during late differentiation, when ipax <dig> myogenic precursors differentiated and fused forming myotubes. this late demethylation wave only occurred in es-derived differentiating myocytes and not in primary myotubes, already demethylated in the mb stage. this observation points out an epigenetically accentuated myogenic commitment of adult muscle stem cells compared to myogenic precursors. importantly, the loss of dna methylation only took place upon pax <dig> expression  indicating that pax <dig> is required for skeletal muscle-specific demethylation. notably, the differentially methylated enhancer located at -111 kb of myf <dig> tss is a direct target of pax <dig> in satellite cells  <cit> , suggesting a link between pax <dig> and dna methylation. in the last years, several studies have shown the crosstalk between muscle transcription factors and chromatin modifiers. in this regard pax <dig> has been shown to recruit the wdr5-ash2l-mll <dig> histone methyltransferase complex to target genes, in order to stimulate myogenic transcription  <cit> . in the same line, myod interacts with the chromatin remodelling complex swi/snf  <cit> , and recruits the histone acetyltransferase p <dig> and the histone methyltransferase ash2l to activate muscle genes . in that context and taking into account our results we propose a model  where dna methylation prevents inappropriate expression of muscle genes in escs, being removed during the commitment to the skeletal muscle lineage. the expression of myogenic transcription factors might modulate both the recruitment, direct or indirect, of dna demethylases to reduce dna methylation and chromatin compaction, and histone modifiers/chromatin remodelers to activate transcription of muscle genes. in proliferating myoblasts, the differentiation genes would be kept in a silenced/poised transcriptional state by prc <dig> and hdacs complexes, whereas in myotubes they would be expressed upon the recruitment of chromatin activating complexes.fig.  <dig> model for the epigenetic control of muscle-specific gene expression during myogenesis. diagram proposing a simplified model where myogenic transcription factors would activate muscle genes by modulating both the recruitment, direct or indirect, of dna demethylases to reduce dna methylation levels and chromatin compaction, and histone modifiers/chromatin remodelers to activate transcription of muscle genes. in proliferating myoblasts, the differentiation genes would be kept in a silenced/poised transcriptional state by prc <dig> and hdacs complexes, whereas in myotubes they would be expressed upon the recruitment of chromatin activating complexes. abbreviations: ddms dna demethylases, hats histone acetyltransferases, hdacs histone deacetylases, hmts histone methyltransferases, prc <dig> polycomb repressive complex  <dig>  swi/snf swi/snf atp-dependent chromatin remodelling complex, yy <dig> yin and yang 1\n",
            "\n",
            "\n",
            "\n",
            "to investigate the possible pathway involved in dna demethylation during myogenesis we focused on apobec <dig> , a member of the cytidine deaminase family expressed exclusively in cardiac and skeletal muscle tissues with unknown functions  <cit> . the generation of apobec2-/- mice showed no major defects in mouse health, fertility or survival up to 1 year of age  <cit> . however, detailed analysis of these animals revealed a 15–20 % reduction in body mass from birth onwards, a clear histological evidence of a mild myopathy during aging, and a markedly increased ratio of slow to fast fibres in soleus apobec2-/- muscles  <cit> . although the cytidine deaminase activity of apobec <dig> is still controversial , it has been reported that during zebrafish development apobec2b stimulates dna demethylation  <cit> , and aid/apobec proteins promote demethylation of 5hmcs in mammals  <cit> . our results showed that apobec <dig> expression was highly induced upon muscle-commitment in ipax7-myogenic precursors  and, importantly, the downregulation of apobec <dig> dramatically impaired muscle differentiation, abolishing myogenin and myosins expression , in agreement with experiments performed on c2c <dig> myoblast cells showing that apobec <dig> was required for myotube formation  <cit> . recently, powell and co-workers reported that apobec <dig> had an essential role during retina and optic nerve regeneration  <cit> , although the small changes observed in dna methylation during tissue-regeneration were independent of apobec <dig> expression  <cit> . interestingly, our results showed that myogenin dna methylation levels were significantly higher in ipax7-myoblast precursors when apobec <dig> activity was abolished, correlating with gene silencing . however, the myod and myf <dig> dna methylation changes observed upon apobec <dig> knockdown were smaller, and not statistically significant. that could be due to the existence of some cell heterogeneity during the process of pax7-dependent myogenic induction, which could mask the demethylation events occurring in myod and myf <dig> genes. another explanation could be related to the fact that myod and myf <dig> play a very early role in muscle-lineage determination, as mouse embryos with targeted mutation in both genes lack myoblasts and differentiated skeletal muscles  <cit> . therefore, we cannot rule out the involvement of other mechanisms, such us tet-mediated dna demethylation, in the epigenetic activation of these two early-induced genes. altogether, our results suggest that apobec <dig> might participate in the active dna demethylation process required to drive myogenic differentiation. apobec <dig> has emerged as an important myogenic regulator that would merit further studies regarding how it is recruited to genomic loci affecting epigenetic regulation.\n",
            "\n",
            "CONCLUSIONS\n",
            "this study presents for the first time the dna methylation dynamics of the principal genes orchestrating the myogenic determination and differentiation programs during myogenesis. our results show that a common dna demethylation signature is required to acquire and maintain the muscle-cell identity, and that downregulation of the muscle-specific cytidine deaminase apobec <dig> in es-derived myogenic precursors reduces myogenic-associated dna demethylation and impairs muscle differentiation. importantly, we have also identified muscle epigenetic markers that may be useful to ensure the efficient and safe generation of es- and ips-derived myogenic progenitors for therapeutic applications.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "our results suggest that apobec <dig> is required to acquire and maintain muscle-cell identity during myogenesis.\n",
            "our results suggest that apobec <dig> is required to acquire and maintain muscle-cell identity during myogenesis, and that downregulation of the muscle-specific cytidine deaminase apobec <dig> in es-derived myogenic precursors reduces myogenic-associated dna demethylation and impairs muscle differentiation.\n",
            "our results suggest that apobec <dig> is required to acquire and maintain muscle-cell identity during myogenesis, and that downregulation of the muscle-specific cytidine deaminase apobec <dig> in es-derived myogenic precursors reduces myogenic-associated dna demethylation and impairs muscle differentiation.\n",
            "our results suggest that apobec <dig> is required to acquire and maintain muscle-cell identity during myogenesis, and that downregulation of the muscle-specific cytidine deaminase apobec <dig> in es-derived myogenic precursors reduces myogenic-associated dna demethylation and impairs muscle differentiation.\n",
            "our results suggest that apobec <dig> is required for muscle-specific demethylation\n",
            "25\n",
            "[{'summary_text': 'notch signaling in drosophila is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\\nnotch signaling in drosophila is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\\nnotch signaling in drosophila is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\\nnotch signaling in drosophila is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\\nnotch signaling in drosophila is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\\nnotch signaling is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\\nnotch signaling is regulated by two conserved glycosyltransferases, o-fuc'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "notch proteins are receptors for a conserved intercellular signaling pathway that mediates a wide variety of cell-fate decisions during animal development  <cit> . notch activity needs to be regulated precisely, and aberrant notch activity is associated with a number of human diseases including cancers and congenital syndromes.\n",
            "\n",
            "notch signaling is influenced by two conserved glycosyltransferases, o-fucosyltransferase <dig>  and fringe   <cit> . fng transfers n-acetylglucosamine  in a β <dig>  linkage onto o-linked fucose on egf domains  <cit> . fringe was first identified because of its role in modulating notch signaling during the development of the drosophila wing, where it both potentiates the activation of notch by one ligand, delta, and inhibits the activation of notch by another ligand, serrate  <cit> . these opposing effects of fringe on the activation of notch by its ligands, together with the restriction of normal fringe expression to dorsal wing cells, help to position a stripe of notch activation along the dorsal-ventral  compartment boundary. this stripe of notch activation is then essential for the further growth and patterning of the wing. fringe also helps to regulate notch activation in other drosophila tissues; however, there are many notch signaling events in drosophila, such as the role of notch in limiting the number of neural precursor cells  that are fringe-independent. similarly, vertebrate fringe proteins are important regulators of notch signaling in some contexts, but not in others  <cit> .\n",
            "\n",
            "ofut <dig> catalyzes the transfer of fucose from gdp-fucose, the universal donor for fucosyltransferases, onto egf domains  <cit> . it thus generates the o-linked fucose that is the substrate for fng. genetic studies of ofut <dig> in flies, and its homolog pofut <dig> in mice, have indicated that it has a much broader role in notch signaling than fng, and indeed appears to be universally required for all notch signaling  <cit> . while this was initially taken to reflect a universal requirement for o-fucose on notch, more recently a second function for ofut <dig> was identified  <cit> . ofut <dig> is a soluble er protein  <cit>  and, at least in drosophila, acts as a notch chaperone, facilitating the folding and secretion of notch <cit> .\n",
            "\n",
            "the fact that ofut <dig> possesses both fucosyltransferase and chaperone activity for notch raises the question of the respective contributions of these two activities to the genetic requirement for ofut <dig> in notch signaling. in this study, we have addressed this by examining the in vivo activity of notch produced in cells with ofut <dig> chaperone activity, but lacking fucosyltransferase activity. surprisingly, we find that notch expressed by these cells is a functional receptor. our observations indicate that o-fucosylation is dispensable for many notch signaling events during drosophila development.\n",
            "\n",
            "RESULTS\n",
            "o-fucosylation of notch is not required during embryonic neurogenesis\n",
            "we have taken two complementary approaches to evaluate the respective contributions of the fucosyltransferase and chaperone activities of ofut <dig> to notch signaling. first, we employed a mutant isoform of ofut <dig>  ofut1r245a. this mutation alters an invariant arginine within the putative gdp binding site, and it eliminates detectable fucosyltransferase activity, while retaining chaperone activity  <cit> . a genomic rescue construct carrying the ofut1r245a mutation was created, and then introduced into flies by p element-mediated transformation. the ability of the ofut1r245a isoform to rescue notch signaling phenotypes was then assayed in animals lacking wild-type ofut <dig>  as ofut <dig> is provided maternally to embryos, this was done by making germline clones with a null allele of ofut <dig>  ofut14r <dig>  in animals containing the ofut1r245a genomic construct inserted on the same chromosome arm. in the absence of any rescue construct, animals lacking maternal and zygotic ofut <dig> exhibit a strong neurogenic phenotype, in which impairment of lateral inhibition causes excess neurons to be produced at the expense of epidermal cells  <cit> . this phenotype is characteristic of mutations in notch pathway genes and can be visualized by staining with antibodies against a marker for neuronal cells, elav . significantly, when ofut1r245a is provided to ofut14r <dig> germline clones embryos, embryonic neurogenesis, as revealed by elav staining, appears indistinguishable from that in wild-type embryos . these embryos lacking o-fuct activity can hatch, but die as first instar larvae. this result implies that the fucosyltransferase activity of ofut <dig> is not essential for notch signaling during lateral inhibition.\n",
            "\n",
            "as an independent test of the requirement for notch fucosylation, we analyzed a mutant in which wild-type ofut <dig> is present but unable to fucosylate notch owing to the absence of its donor substrate, gdp-fucose. gdp-fucose is generated in the cytoplasm from gdp-mannose by enzymes including gdp-d-mannose dehydratase . since drosophila lack a salvage pathway for gdp-fucose synthesis  <cit> , and gdp-fucose is the common donor substrate for all fucosyltransferases, animals lacking gmd are expected to be unable to effect all types of fucosylation. animals that are mutant for a null allele of gmd, gmd <dig>  <cit> , die as larvae with no neurogenic phenotype , but the survival of gmd zygotic mutants could have been a result of maternally provided product. thus, requirements for gmd during embryonic development were evaluated by making germline clones. confirmation of the general deficit in fucosylation in these embryos was provided by staining with anti-hrp antibodies, which recognize a fucose-containing epitope on neuronal n-glycans  <cit> . anti-hrp staining was completely eliminated in animals lacking both maternal and zyogotic contributions of gmd . strikingly, however, notch signaling in these embryos, as visualized by elav staining, was indistinguishable from wild type  and gmd germline clone embryos are able to complete embryogenesis and hatch before dying as first instar larvae. these results establish that all fucose modifications are dispensable for lateral inhibition, as well as all other developmental processes essential for the hatching of a drosophila embryo.\n",
            "\n",
            "role of notch o-fucosylation in the wing imaginal disk\n",
            "notch signaling participates in many different processes throughout development and many of the factors that modulate notch signaling are context specific. for example, fng is dispensable for notch functions during embryonic neurogenesis, but plays a critical role in the developing wing. in the wing, fng is expressed by dorsal cells and its positive and negative effects on signaling by delta and ser, respectively, position a stripe of notch activation along the edge of fng expression at the d-v boundary  <cit> . the critical role of fng in positioning notch activation is evidenced by the observation that establishing novel fng expression boundaries by creating fng mutant clones in dorsal cells results in the induction of ectopic stripes of notch activation, which can be visualized by examining expression of targets of notch signaling, such as wingless   <cit> . in contrast, ofut <dig> mutant clones are associated with a complete loss of notch activation in the wing   <cit> , equivalent to that observed in notch mutant clones.\n",
            "\n",
            "to evaluate requirements for notch o-fucosylation during wing development, we again took advantage of the ability to rescue the chaperone, but not the fucosyltransferase, activities of ofut <dig> by expression of ofut1r245a. ofut1r245a was expressed in ofut14r <dig> null mutant clones using two different methods. in one set of experiments, we expressed ofut1r245a from a genomic rescue construct, such that ofut1r245a was expressed under its own promoter. in an alternative approach, ofut1r245a was over-expressed in ofut14r <dig> mutant clones using the mosaic analysis with a repressible cell marker  technique  <cit> , which employs the uas-gal <dig> system to drive expression under the control of a heterologous promoter. both methods yielded similar results: when ofut1r245a expression is similar to endogenous ofut <dig> levels, the rescued clones phenocopy fng mutant clones . thus, where the clones intersected the normal d-v boundary, normal wg expression is lost, whereas the borders of clones within the dorsal compartment can be associated with ectopic wg expression. this striking phenotype has two important implications. first, because wg expression can be induced within mutant cells expressing only ofut1r245a, it indicates that, as during embryonic neurogenesis, modification of notch with o-fucose is not required for it to function as a receptor. second, it indicates that the absence of o-fucose is functionally equivalent to the absence of elongated  o-fucose in the developing wing.\n",
            "\n",
            "we also attempted to extend these observations by examining gmd mutants. animals that are mutant for a null allele, gmd <dig>  exhibit decreased growth and loss of wg expression in the wing imaginal disks, consistent with a deficit in notch signaling  <cit> . to investigate whether this phenotype reflects a specific requirement for gmd in fng-dependent notch signaling, or a more general requirement for gmd in notch signaling, we created clones of cells homozygous for gmd <dig>  however, in most cases these did not show obvious notch-loss-of-function phenotypes . only when the minute technique was used to generate disks in which all, or almost all, of the wing was composed of mutant tissue was a loss of wg expression observed, and in all cases the loss of wg expression was non-autonomous . a non-autonomous phenotype in clones has been reported previously for a mutation in udp-glucose dehydrogenase, which is required for the synthesis of heparan sulfate  <cit> . we suggest that a general non-autonomy of mutations in genes that participate in nucleotide sugar biosynthesis could be explained if nucleotide sugars can diffuse through cells via gap junctions. owing this non-autonomy, gmd clones could not be directly compared with fng or ofut <dig> clones.\n",
            "\n",
            "over-expression of ofut <dig> can inhibit notch signaling independently of its fucosyltransferase activity\n",
            "loss of ofut <dig> impairs notch signaling, but over-expression of ofut <dig> can also impair notch signaling  <cit> . this observation led to the suggestion that high levels of notch o-fucosylation might suppress notch signaling. however, the determination that it is the chaperone activity, rather than the fucosyltransferase activity, that is universally required for notch signaling prompted us to re-examine the basis for the inhibition of notch signaling associated with ofut <dig> over-expression. towards that end, ofut1r245a was expressed at high levels in the developing notum under the control of ap-gal <dig>  expression of wild-type ofut <dig>  under ap-gal <dig> control interferes with notch-mediated lateral inhibition, and so results in the production of extra bristles   <cit> . the effect on bristles was only evident in microchaetes, but not in macrochaetes. over-expression of ofut1r245a  under ap-gal <dig> control results in a similar phenotype . this observation, together with another recent study  <cit> , indicate that the over-expression phenotype is independent of ofut1's fucosyltransferase activity. interestingly, when much stronger ofut <dig> expression  was induced in a thin stripe of cells along the anterior-posterior  compartment boundary using the ptc-gal <dig> driver with uas-ofut <dig> , wg expression was inhibited non-autonomously . as extracellular concentrations of nucleotide sugars are generally thought to be too low to support glycosylation, this non-autonomous affect is also consistent with a fucosylation-independent activity.\n",
            "\n",
            "ofut <dig> is required for secretion of notch from the er\n",
            "the observation that the fucosyltransferase activity of ofut <dig> is not required for all notch signaling events implies that the previously described notch chaperone activity of ofut <dig> is its most critical function. recently, however, our identification of this chaperone activity has been questioned and two additional activities for ofut <dig> have been proposed in its place: the first, independent of its fucosyltransferase activity, promotes endocytosis of notch from the plasma membrane to the early endosome  <cit> ; the second, dependent upon fucosyltransferase activity, promotes trafficking of notch from the plasma membrane to the sub-apical complex and adherens junctions  <cit> . the two most critical experiments distinguishing between the chaperone model and these trafficking models for ofut <dig> function relate to the localization of notch in cells lacking ofut <dig> \n",
            "\n",
            "first, it was claimed that in contrast to our report that notch is not secreted to the cell surface in the absence of ofut <dig>  <cit> , notch is secreted to the plasma membrane in ofut <dig> mutant cells  <cit> . this claim was based on an assay in which anti-notch antibodies were added to live disks. however, this is not an effective assay for cell surface localization, because a cell surface receptor is not required for bulk endocytosis , and once endocytosed, antibodies could be spread throughout the secretory pathway and then accumulate wherever there are significant epitope concentrations. a standard assay for cell surface localization is to determine whether the accessibility of epitopes requires membrane permeabilization. thus, we performed immunostaining using antibodies directed against the extracellular domain of notch, both in the presence and in the absence of detergent. when wing disk cells are permeabilized with detergent, wild-type cells exhibit both intracellular staining as well as apical hexagonal staining, corresponding to the normal cell surface localization of notch near the adherens junctions . ofut <dig> mutant cells show increased notch staining , which we have previously shown overlaps with er markers  <cit>  . to demonstrate that the notch accumulation near the apical surface in ofut <dig> mutant cells is not exposed on the cell surface, we stained disks without detergent treatment. under these conditions, no staining was observed in ofut <dig> mutant cells, whereas neighboring wild-type cells exhibit normal apical surface staining . these results are consistent with previous data using drosophila s <dig> cells  <cit>  and reconfirm that ofut <dig> is required for secretion of notch to the cell surface in wing disk cells.\n",
            "\n",
            "the second critical piece of evidence presented against the chaperone model was a claim that the notch that accumulates in ofut <dig> mutant cells is not actually in the er. this claim relies on the belief that the er is homogeneous organelle, that is, that all er proteins are homogeneously distributed. both our studies  <cit>  and those of sasaki et al. <cit>  reveal overlap between notch and er markers in ofut <dig> mutant or rnai-depleted cells. at the same time, we agree that notch does not overlap perfectly with er markers in all focal planes; indeed, it is for this reason that we originally thought that notch was not in the er in the absence of ofut <dig>  <cit> . however, we now think that differences between notch and other markers reflect er heterogeneity. in support of this idea, we have investigated the relative distributions of five different er markers in wing imaginal disks cells: two dedicated er chaperones , a classic er protein marker , a synthetic er protein  and bulk er proteins . each of these er markers exhibits partial, but not perfect, overlap with other er markers   <cit> . this observation of er heterogeneity emphasizes that the lack of perfect correspondence among markers cannot be taken as compelling evidence for the absence of er localization of notch in ofut <dig> mutant cells. together with observations that the accumulated notch in ofut <dig> cells does not overlap markers for any other vesicles or organelles  <cit> , that the distribution of notch in ofut <dig> mutant cells is similar to the distribution of ofut <dig> itself  <cit> , which by several criteria is an er protein  <cit> , and that notch does partially overlap with a variety of er markers in ofut <dig> mutant or depleted cells  <cit> , these observations are consistent with the conclusion that notch accumulation within ofut <dig> mutant cells is in the er.\n",
            "\n",
            "discussion\n",
            "the observations we describe here lead to the unexpected conclusion that the fucosyltransferase activity of ofut <dig> is not essential for all notch signaling in drosophila. the lack of requirement for notch o-fucosylation is evidenced by the observation that expression of a form of ofut <dig> that provides chaperone activity, but lacks fucosyltransferase activity, is sufficient to enable notch receptor activation in at least two different contexts: during embryonic neurogenesis, where notch signaling effects lateral inhibition, and during wing disk development, where inductive notch signaling establishes specialized d-v boundary cells. indeed, because ofut1r245a-rescued animals complete embryogenesis and hatch without grossly evident abnormalities, it is likely that many other notch signaling events also do not require notch fucosylation. although reagents that would enable direct visualization of the o-fucosylation state of notch in cells expressing ofut1r245a do not exist, prior characterization of this mutant isoform indicates that it is completely defective in fucosylation  <cit> . further, the observation of fng-like phenotypes in the wing provides a genetic argument that fucosylation is lacking in cells expressing only ofut1r245a, as if o-fucose were present on notch then it should be modified by fng. moreover, in a completely independent approach, the lack of requirement for notch fucosylation was demonstrated by the observation of normal neurogenesis and completion of embryogenesis in gmd mutants, whose general deficit in fucosylation was confirmed by anti-hrp staining. the absence of o-fucosylation in gmd mutants is also consistent with the prior observation that notch signaling at the d-v boundary of the wing is lost in gmd mutants  <cit> , as this is a fng-dependent process. collectively, these results indicate that notch receptors can transduce signals without modification by o-fucose. prior results suggesting that o-fucose is required for notch-ligand binding  <cit>  can be explained by the requirement for the chaperone activity of ofut <dig>  indeed, a bacterially expressed fragment of mammalian notch <dig> can bind to delta in spite of the absence of glycosylation  <cit> .\n",
            "\n",
            "the observation that o-fucose is not universally required for notch signaling does not mean that it has no effect. in acting as a substrate for fng, o-fucose clearly has an important modulatory role. it is, however, conceivable that this is the only absolute requirement for o-fucose, as genetic studies of the ofut1r245a allele, as well as genetic studies of gmd, are consistent with the possibility that all requirements for o-fucose might be explained by its role in fng-dependent modulation of notch: like ofut1r245a-rescued animals, or gmd maternal and zygotic mutant animals, fng mutant animals die as first instar larvae, without obvious affects on embryonic neurogenesis  <cit> . the apparently normal embryogenesis of gmd mutants is particularly striking, as fucose is a common component of insect n-glycans.\n",
            "\n",
            "the similarity between ofut1r245a-rescued clones and fng mutant clones in the wing is also informative in terms of the nature of the requirement for fng modification. that is, the observation that loss of o-fucose results in the same phenotype as loss of o-fucose elongation implies that modulation of notch signaling by fng is affected because a specific glycan created as a consequence of fng modification  alters notch-ligand interactions, rather than because elongation of o-fucose covers up a glycan  that promotes serrate-notch interactions or impairs delta-notch interactions.\n",
            "\n",
            "although the phenotypes of ofut1r245a and gmd might be explained simply by the requirement for fng-dependent elongation, there are nonetheless some indications that the monosaccharide form of o-fucose can have an influence on notch. in vitro binding revealed that elimination of the o-fucose modification site located in the ligand-binding region  of drosophila notch causes elevated serrate binding in the absence of fringe  <cit> . mutagenesis of the o-fucose site on egf <dig> of notch in mammalian cells also influenced notch signaling, although in this case the result was an impairment of delta-like  <dig> or jagged- <dig> activation of notch <dig> in cultured cells  <cit> , and impaired notch signaling in vivo . moreover, the introduction of a novel o-fucosylation site on egf repeat  <dig>  as observed in the nspl mutation, causes ectopic notch activation during eye development, independently of fringe activity  <cit> . these observations are, however, subject to the caveat that the affects of these mutations might not actually be a result of their effects on o-fucosylation. cell-based ligand binding assays have revealed that a soluble form of notch produced from cells over-expressing wild-type ofut <dig> results in increased binding to serrate but decreased binding to delta  <cit> , whereas notch from ofut1r245a over-expressing cells exhibits increased binding to both serrate and delta  <cit> . these differing affects of ofut <dig> versus ofut1r245a on delta-binding suggest that increased o-fucosylation can modulate notch-delta interactions. a study of a gmd-deficient mammalian cell line, lec <dig>  revealed that jagged- <dig> dependent activation of notch <dig> was reduced in the mutant cells, suggesting that o-fucose positively regulates notch <dig> activation by jagged- <dig> in this context  <cit> . finally, we note that the expression of ofut <dig> is developmentally regulated in drosophila, exhibiting a complex spatial and temporal pattern  <cit> . thus, while o-fucose is clearly not essential for all notch signaling, it might still have a modulatory role in some contexts.\n",
            "\n",
            "our studies have focused on the fucosyltransferase and the chaperone activities of ofut <dig> as distinct functions, genetically separable by the r245a mutation. nonetheless, from an evolutionary perspective it would be surprising if these activities were unrelated. given that ofut <dig> acts during the folding process and dissociates from egf domains upon o-fucosylation, it is tempting to speculate that o-fucose might serve as a tag for correctly folded egf domains, thus directing ofut <dig> to incompletely folded egf domains. therefore, although o-fucosylation is not absolutely required for notch receptor activity, it might still affect the efficiency of the chaperone activity of ofut <dig>  in support of this hypothesis, we have observed modest decreases in notch secretion in s <dig> cells treated with double-stranded rna corresponding to gmd .\n",
            "\n",
            "our studies also suggest the molecular basis for the previous observation that over-expression of ofut <dig> inhibits notch signaling  <cit> . we showed that over-expressed ofut <dig> inhibits notch signaling non-autonomously and non-enzymatically . although ofut <dig> is predominantly an er protein, a small fraction is secreted from cells  <cit> , which presents the possibility that the non-autonomous affect of ofut <dig> might be effected through direct interaction of secreted ofut <dig> with notch or its ligands, presumably mediated by its ability to associate with egf domains  <cit> . indeed it has recently been observed that secreted ofut <dig> can promote endocytosis of notch  <cit> , which could provide an explanation for the inhibition of notch signaling associated with ofut <dig> over-expression. we note, however, that the biological relevance of this phenomenon remains to be determined: it can occur when ofut <dig> is over-expressed, but it is not clear whether it is significant at endogenous expression levels. nonetheless, it is of potential pharmacological interest to note that ofut <dig> can act as a soluble inhibitor of notch signaling.\n",
            "\n",
            "CONCLUSIONS\n",
            "in summary, our findings demonstrated for the first time that o-fucose modification on notch receptors is not absolutely required for their activity in fringe-independent developmental processes in drosophila, and that successful folding mediated by chaperone activity of ofut <dig> is sufficient to generate functional notch receptors.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "notch signaling in drosophila is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\n",
            "notch signaling in drosophila is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\n",
            "notch signaling in drosophila is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\n",
            "notch signaling in drosophila is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\n",
            "notch signaling in drosophila is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\n",
            "notch signaling is regulated by two conserved glycosyltransferases, o-fucosyltransferase <dig> and gdp-fucose.\n",
            "notch signaling is regulated by two conserved glycosyltransferases, o-fuc\n",
            "26\n",
            "[{'summary_text': 'lactobacillus fermentum isolates were isolated from a relatively large geographic area including mongolia, four chinese provinces and two autonomous regions of china, and from various naturally fermented dairy products and acidic gruel.\\nlactobacillus fermentum isolates were isolated from a relatively large geographic area including mongolia, four chinese provinces and two autonomous regions of china, and from various naturally fermented dairy products and acidic gruel.\\nlactobacillus fermentum isolates were isolated from a relatively large geographic area including mongolia, four chinese provinces and two autonomous regions of china, and from various naturally fermented dairy products and acidic gruel.\\nlactobacillus fermentum is an economically important species of lactic acid bacterium  used in the production and preservation of fermented food as an acid-producing starter culture.\\nlactobacillus fermentum isolates were isolated from a relatively large geographic area including mongolia, four chinese provinces and two autonomous regions of china, and from various naturally fermented dairy products and acidic gruel.\\nlactobacillus fermentum is an economically important species of lactic acid bacterium  used in the production and preservation of fermented food as'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "lactobacillus fermentum is an economically important species of lactic acid bacterium  used in the production and preservation of fermented food as an acid-producing starter culture  <cit> . isolates of l. fermentum originate from a variety of habitats: traditionally fermented milk products, sourdough, fermenting plant materials, faeces and sewage amongst others . lactobacillus fermentum was first described by beijerink , as an obligate heterofermentative bacterium associated with the fermentation of hexoses to lactic acid  <cit> . another species, lactobacillus cellobiosus, which was first described by rogosa et al.,  is also heterofermentative and dna-dna hybridization studies showed it to be very similar to l. fermentum  <cit> . there is now strong evidence to support a close relationship between l. cellobiosus and l. fermentum and, in fact, l. cellobiosus has now been reclassified as a biovar of l. fermentum  <cit> .\n",
            "\n",
            "in recent years, molecular typing approaches have been used to characterize l. fermentum and the subspecies within it. for example, l. fermentum isolates could be differentiated from other lactobacillus species using randomly amplified polymorphic dna methods  <cit> . rapd-pcr has also been used in combination with amplified 16s rdna restriction analysis , pulsed-field gel electrophoresis with restriction fragment length polymorphism  and ribotyping to characterize  <dig> isolates from lactobacillus species in wine  <cit> . more recently, l. fermentum has been differentiated from l. gasseri and l. plantarum isolates from the human genital tract using pfge and fluorescence in situ hybridization   <cit> . however, these methods can sometimes be ambiguous because the majority of bacteria have very similar nutritional requirements and grow under similar environmental conditions  <cit> .\n",
            "\n",
            "multilocus sequence typing , a protocol that was based on partial nucleotide sequences of housekeeping genes, is commonly used to differentiate between isolates of the same microbial species  <cit> . in recent decades mlst has been developed as a technique to examine the evolution and genetic population structure of bacteria . it was initially evaluated for neisseria meningitidis but has subsequently been extended to many bacterial species  <cit> . most recently it has been used to characterize lactobacillus species including lactobacillus casei  <cit> , lactobacillus plantarum  <cit> , lactobacillus sanfranciscensis  <cit> , lactobacillus delbrueckii and lactobacillus sakei  <cit> . however, until this study, mlst had not been applied to characterizing l. fermentum isolates.\n",
            "\n",
            "here we developed an mlst scheme based on  <dig> housekeeping gene fragments to characterize  <dig> isolates of l. fermentum. the aim of this study was to develop an effective mlst method for l. fermentum and use this to describe the diversity, genetic population structure and evolutionary origins of isolates within this species.\n",
            "\n",
            "RESULTS\n",
            "sequence diversity in l. fermentum\n",
            "partial sequences of  <dig> gene fragments  were determined . the numbers of alleles, polymorphic sites, guanine-cytosine content, nucleotide diversity per site  and rate of dn/ds value  were all determined . fragment sizes of the  <dig> housekeeping gene fragments, which ranged from 589 bp  to 748 bp , were used for mlst analysis. the number of alleles per gene fragment varied between seven  and  <dig> . between six  and  <dig>  polymorphic sites were found for each gene fragment, and a total of  <dig> snps were identified. the mean guanine-cytosine content of the partial sequence of the  <dig> gene fragments varied between  <dig>  %  and  <dig>  % . pi was calculated for each individual gene fragment and varied from  <dig>   to  <dig>  . the dn/ds value for the  <dig> gene fragments varied between  <dig>   and  <dig>  .table  <dig> genes and primers used for mlst\n",
            "\n",
            "\n",
            "clpx\n",
            "\n",
            "\n",
            "dnaa\n",
            "\n",
            "\n",
            "dnak\n",
            "\n",
            "\n",
            "groel\n",
            "\n",
            "\n",
            "murc\n",
            "\n",
            "\n",
            "mure\n",
            "\n",
            "\n",
            "pepx\n",
            "\n",
            "\n",
            "pyrg\n",
            "\n",
            "\n",
            "reca\n",
            "\n",
            "\n",
            "rpob\n",
            "\n",
            "\n",
            "uvrc\n",
            "\n",
            "*positions correspond to complete genome sequence of lactobacillsu fermentum ifo 3956\n",
            "\n",
            "pi\n",
            "1\n",
            "d\n",
            "n\n",
            "/d\n",
            "s\n",
            "2\n",
            "\n",
            "clpx\n",
            "\n",
            "dnaa\n",
            "\n",
            "dnak\n",
            "\n",
            "groel\n",
            "\n",
            "murc\n",
            "\n",
            "mure\n",
            "\n",
            "pepx\n",
            "\n",
            "pyrg\n",
            "\n",
            "reca\n",
            "\n",
            "rpob\n",
            "\n",
            "uvrc\n",
            "\n",
            " <dig> mean pairwise nucleotide differences per site\n",
            "\n",
            "\n",
            "2\n",
            "d\n",
            "n\n",
            "/d\n",
            "s = the ratio of nonsynonymous to synonymous substitutions\n",
            "\n",
            "\n",
            "\n",
            "assignment of sequence types\n",
            "an mlst protocol was developed for the 203 l. fermentum isolates based on  <dig> housekeeping gene fragments and used to identify sts . fifty-seven sts were identified using combined data from the  <dig> gene fragments  . the  <dig> isolates were divided as follows: st- <dig> ; st- <dig> ; st- <dig> ; st- <dig> ; st- <dig>   <dig> ; st- <dig>   <dig>   <dig>   <dig> ; st- <dig>   <dig>   <dig> ; st- <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig> ; st- <dig>   <dig>   <dig>   <dig>   <dig>   <dig> ; the remaining  <dig> sts were each represented by a single isolate.\n",
            "\n",
            "relatedness of l. fermentum isolates\n",
            "to infer the evolutionary relationships amongst l. fermentum isolates, a phylogenetic tree based on the mlst data was obtained using the neighbour-joining  method . in the phylogenetic tree built, all of the isolates investigated were well clustered into two major groups, a and b. group a contained four clonal complexes  and  <dig> singletons, representing 78 l. fermentem isolates and one reference isolate ; most of the isolates  were isolated from acidic gruel. the reference isolate , which was also found in group a, was isolated from fermented plant material. group b was the larger of the two groups and included cc <dig> and nine singletons; almost all the isolates  were isolated from different dairy products such as yoghurt, kurut, qula, fermented camels’ milk, koumiss, whey, etc.. the ancestral type for group b was st- <dig>  which contained  <dig> isolates originating from dairy products, in addition to imau <dig>  which was an isolate from acidic gruel.fig.  <dig> neighbour-joining phylogenetic tree obtained from the concatenated nucleotide sequence of  <dig> sts. bootstrap value are shown for all branches. the numbering in the figure refers to the st. the two major phylogroups were designated as a and b\n",
            "\n",
            "\n",
            "\n",
            "recombination in l. fermentum\n",
            "the values of ia  and ias  for the  <dig> gene fragments were  <dig>  and  <dig>  , respectively. as the ias value was greater than  <dig>  this indicates that the genes investigated in this study were close to linkage disequilibrium  <cit> .\n",
            "\n",
            "simultaneously, a split decomposition analysis examining evidence for recombination amongst the 203 l. fermentum isolates revealed different structures in the split graphs for all  <dig> gene fragments . the split graphs for pepx, uvrc, and rpob were network-like with parallelogram structures indicative of recombination in the evolutionary history of those genes. however, the split graphs for groel, dnak, dnaa, mure, clpx, murc, reca and pyrg were tree-like structures, indicative of a clonal descent for these genes and an absence of recombination.fig.  <dig> split-decomposition analysis of 203 l. fermentum isolates with  <dig> housekeeping gene fragments. multi-parallelogram formations indicate recombination events. split-decomposition analysis of individual mlst loci. the numbering in the figure refers to allele numbers\n",
            "\n",
            "\n",
            "\n",
            "based on split decomposition analysis the relationships amongst the  <dig> sts could be described as having a network-like structure with rays of different lengths . the sts were divided into two main groups, a and b, and these groups were completely disconnected from each other. parallelogram-shaped groupings were detected suggesting recombination events had occurred frequently. isolates in group a were more distantly related to the ancestral isolate than the isolates in group b based on split decomposition analysis, suggesting that recombination had not occurred between isolates from the two groups, but that intergenic recombination may have occurred between isolates from the same group during their evolution. in addition,  <dig> sts indicated two distinct groups corresponding to group a and b in fig.  <dig> fig.  <dig> split-decomposition analysis of 203 l. fermentum isolates with  <dig> housekeeping gene fragments. multi-parallelogram formations indicate recombination events. combined split-decomposition analysis of all  <dig> mlst loci. the numbering in the figure refers to the st. gray line was drawn as boundaries of each group\n",
            "\n",
            "\n",
            "\n",
            "cluster analysis of the mlst data\n",
            "allelic profile-based phylogenetic analysis using the minimum spanning tree algorithm based on food-type and geographic origins, was used to explore genetic lineages amongst the l. fermentum isolates . in this representation, isolates with the same allelic profile were assigned to the same circle, the size of the circle is proportional to the number of isolates with that unique profile. the clonal complexes  were confirmed as groups of sts sharing  <dig> of the  <dig> gene fragments. the 203 l. fermentum isolates evaluated were assigned to  <dig> sts that were distributed amongst five ccs and  <dig> unique sts .fig.  <dig> minimum spanning tree analysis of 203 l. fermentum isolates based on allelic profiles of  <dig> gene fragments and according to food-type origin. each circle represents the sequence type, the size of the circle is proportional to the number of isolates within any given st. the strength of links are: black line = strong relationship; grey line = intermediate relationship; dotted line = weak relationship. sts belonging to the same clonal complex, are indicated by the surrounding shading. gray line was drawn as boundaries of each group\n",
            "\n",
            "\n",
            "\n",
            "most of the isolates recovered from acidic gruel were absent from the largest cc, i.e. cc <dig>  cc <dig> included  <dig> sts that mainly originated from traditional fermented dairy products from a wide range of geographic locations, including mongolia, seven regions of inner mongolia, tibet, gansu, sichuan, yunnan, qinghai and xinjiang. the only exception was isolate imau <dig>  which belonged to cc <dig>  but was isolated from acidic gruel. within cc <dig>  st- <dig> had the largest number of isolates  and was identified as the anscestral genotype surrounded by single-locus , or two-locus variants . all other ccs included only two sts, each with only a few isolates. cc <dig> included st- <dig> and st- <dig>  and contained seven isolates from yunnan. cc <dig> included st- <dig> and st- <dig>  and contained seven isolates from inner mongolia. cc <dig> included st- <dig> and st- <dig>  and contained six isolates from inner mongolia. cc <dig> included st- <dig> and st- <dig>  and contained five isolates from tibet and inner mongolia.\n",
            "\n",
            "discussion\n",
            "multilocus sequence typing is considered to be the best method for typing isolates using the dna sequences of selected housekeeping gene internal fragments  <cit> . in this study, we used the mlst method to analyze the natural diversity in l. fermentum based on the dna sequences of  <dig> housekeeping gene fragments. the isolates evaluated here were isolated from a relatively large geographic area including mongolia, four chinese provinces and two autonomous regions of china, and from various naturally fermented dairy products and acidic gruel. these isolates provide the relevant information required for a better understanding of the population structure and phylogenetic relationships amongst  <dig> isolates of l. fermentum.\n",
            "\n",
            "nucleotide sequences-based methods for bacterial typing are the most unambiguous methods by which isolates of any microorganism can be identified  <cit> . in this study, 203 l. fermentum isolates were divided into  <dig> sts, providing a clear indication of variability. we used the mlst method to compare nucleotide polymorphisms within regions of the  <dig> housekeeping gene fragments, which are under selective pressure to retain function. housekeeping genes used in this study had a certain number of polymorphisms and hence were useful for designing the mlst protocol. a total of  <dig> polymorphic sites were detected in the  <dig> gene fragments giving a polymorphism rate of  <dig>  % amongst the  <dig>  nucleotides present in l. fermentum isolates. this value was higher than that seen for other lab, such as oenococcus oeni, which had  <dig> snps  amongst  <dig>  base pairs sequenced  <cit> ; and other microbes such as aspergillus fumigatus, with  <dig> snps  amongst  <dig>  base pairs sequenced  <cit> . this result also indicated that, because most of the gene fragments used in this study had high nucleotide genetic diversity, they also had a strong discriminatory ability.\n",
            "\n",
            "the value of pi varied from  <dig>   to  <dig>   for each gene fragment. except for dnaa, most fragments showed similar nucleotide diversity of between  <dig>  and  <dig> . our estimate of pi was similar to those obtained for l. plantarum, which ranged from  <dig>  to  <dig>   <cit>  and l. delbrueckii, which ranged from  <dig>  to  <dig>   <cit> . these comparisons demonstrate the relatively high nucleotide diversity in these species and in the l. fermentum isolates used in this study. this relatively high nucleotide diversity in l. fermentum is possibly as a result of recombination by natural transformation and a low frequency of mutation. in our genetic analysis, the dn/ds ratios of the  <dig> gene fragments were all less than  <dig>  suggesting purifying selection as expected from relatively conserved housekeeping genes. in particular the values for clpx, dnaa, dnak and reca were zero, indicating that the amino acid composition of these genes did not change. similar results have also been found in other studies, supporting our conclusion that these housekeeping genes are all under stabilizing selection  <cit> .\n",
            "\n",
            "the phylogenetic tree constructed from the concatenated sequences of the  <dig> gene fragments indicated that the 203 l. fermentum isolates formed two distinct groups ; partitioning of the two groups was as a result of evolutionary changes in the clpx, dnaa, dnak, groel, murc, mure, pepx, pyrg, reca, rpob and uvrc gene sequences, as was clearly visible in the n-j phylogenetic tree constructed . this showed a strong evolutionary tendency for l. fermentum isolates from the same geographic areas, including mongolia, a number of chinese provinces and an autonomous region, and from the same naturally fermented products, to be similar. mlst analyses were valuable in identifying these differences in variation within the l. fermentum genome. hence, l. fermentum isolates that were included in the same group shared similarities in genome organization. notably, about eight of ten isolates from acidic gruel were grouped together within group a and almost all the isolates from fermented dairy products were grouped together within group b. the minimum spanning tree analysis supported the existence of two groups of isolates . although mlst analysis of 203 l. fermentum isolates indicated two groups here, we cannot exclude the possible that more complex population structures might be identified if a larger number of isolates from more diverse sources was used. in future work, mlst will be a useful tool to examine possible relationships amongst l. fermentum isolates, while simultaneously aiding the selection of industrial isolates with greatest potential for the production of fermented food.\n",
            "\n",
            "the analysis of population structure of l. fermentum isolates indicated substantial recombination phenomena. the ias value for the  <dig> gene fragments from 203 l. fermentum isolates was calculated as  <dig>  , which is indicative of a weak clonal population structure . several studies have already shown that lab isolated from fermented milk are structured in rather clonal populations, for example, xu et al.,  investigated  <dig> housekeeping genes in lactococcus lactis; the ias value was  <dig>  and indicative of a clonal population  <cit> .\n",
            "\n",
            "split-decomposition analysis also provided evidence of intraspecies recombination that could play a role in generating genotypic diversity amongst isolates according to the allelic profiles of the isolates evaluated. split graphs for individual loci indicated tree-like or network-like structures, suggesting that some genes were affected by intraspecies recombination . simultaneously, a split graph representation of the concatenated sequence of the  <dig> loci clearly indicated that two groups and their descents orginated from intraspecies recombination . the concatenated dendogram corresponded well with the allele-based dendogram . however, some small differences were found between the concatenated dendogram and the allele-based dendogram. for example, st- <dig> was positioned further away from st- <dig> in the latter than the former. although intraspecies recombination occurs frequently, isolates from fermented dairy products in group b and isolates from acidic gruel in group a in both the n-j phylogenetic tree  and the combined split graph , seem to be clonal. these results suggest that isolates from fermented dairy products and acidic gruel may have a common recent ancestor. in addition, the minimum spanning tree analysis result confirmed the above assumption that these isolates have a common recent ancestor .\n",
            "\n",
            "the clustering of isolates by food-type origin was evident in the minimum spanning tree  demonstrating strong patterns of specificity for source within the 203 l. fermentum isolates evaluated. the diversity found amongst isolates from fermented dairy products differed from that found amongst isolates from acidic gruel. almost all of the isolates from fermented dairy products were assigned to the largest cc, cc <dig>  whereas isolates from acidic gruel were dispersed among a larger number of sts. although the difference between isolates from fermented dairy products and acidic gruel was significant, we considered that evolution of l. fermentum was not correlated with food-type origins. we suggest that a more simple explanation is that the ecological niches from which the isolates were sampled are very narrow . as l. fermentum is a ubiquitous bacterium found in many different types of food and in animal faeces, it is likely that isolates from dairy products are only a small proportion of the natural diversity within this species, particularly as they would be selected for particular attributes associated with the fermentation of food. we also found no association between sts and geographical origin .\n",
            "\n",
            "mlst data are usually subdivided into nonoverlapping groups of related sts or ccs using an eburst algorithm approach to determine the most parsimonious patterns of descent of isolates within each cc from the predicted founder  <cit> . here we also used the eburst algorithm to analyze the mlst data, and found that st- <dig> inhabited a central location, and that other sts had relatives that were derived from the ubiquitous sts themselves and exhibited a limited genetic diversity, as has been found in other studies  <cit> . the great majority of sts in cc <dig> are single-locus variants  of st- <dig>  which strongly supported primary founders. furthermore, cc <dig> to cc <dig> were clearly formed based on food-type origins. for example, three of the seven ccs  consisted exclusively of isolates from fermented dairy products, with the exception of only one isolate , whereas three of five ccs  consisted exclusively of isolates from acidic gruel, with the exception of only one isolate . furthermore, we found that the l. fermentum isolates in cc <dig>  which were almost exclusively isolated from fermented dairy products, were located centrally, with isolates from acidic gruel being distributed around cc <dig> and not centrally .\n",
            "\n",
            "the genus lactobacillus, together with the genera paralactobacillus and pediococcus, is the largest group in the lactobacillaceae and order lactobacillales in the firmicutes  <cit> . the species l. fermentum and lactobacillus reuteri are phylogenetically closely related and are regarded together as the l. reuteri group. since maiden et al.,  first described an mlst technique for n. meningitides, mlst methods have been used to differentiate between isolates within species from the genus lactobacillus including l. casei, l. plantarum, l. sanfranciscensis, l. delbrueckii, and l. sakei, . to date, the mlst schemes used in these studies were distinct from each other because the isolates used in each study came from different habitats, and different housekeeping genes were selected for analysis. in these studies,  <dig> isolates of l. delbrueckii were assigned to  <dig> sts  <cit> ;  <dig> isolates of l. casei  were assigned to  <dig> sts  <cit> ;  <dig> isolates of l. sakei were assigned to  <dig> unique sts of which all isolates had evolved into three clades, each with a unique population structure  <cit> . in our study, we used mlst to identify  <dig> sts within a l. fermentum population of  <dig> isolates. our research indicated that this species had a clonal population structure and a pattern of diversity different from the previously mentioned species. moreover, two groups, each with a unique population structure, were identified amongst l. fermentum isolates . the l. fermentum mlst schemes in this study demonstrate that mlst is a useful tool for discrimination between isolates, and furthermore provides a method to analyze the evolution and population structure of lactobacillius species from various sources.\n",
            "\n",
            "CONCLUSIONS\n",
            "in this study, a novel mlst protocol was used to investigate the population genetic structure and evolutionary characteristics of l. fermentum. the mlst protocol presented provided high discriminatory power for molecular typing of l. fermentum isolates. furthermore, we were able to shed light on how this species has evolved into two unique groups. using a large number of isolates allowed us to better interpret the possible ecological differences underlying the two branches observed. finally, we also found that the evolution of l. fermentum sts was not correlated with geography or food-type origin. taken together, our results indicate that mlst of l. fermentum was an easy and valuable tool that, together with the construction of an mlst database, will contribute to further detailed studies on the evolution and population genetics of l. fermentum.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "lactobacillus fermentum isolates were isolated from a relatively large geographic area including mongolia, four chinese provinces and two autonomous regions of china, and from various naturally fermented dairy products and acidic gruel.\n",
            "lactobacillus fermentum isolates were isolated from a relatively large geographic area including mongolia, four chinese provinces and two autonomous regions of china, and from various naturally fermented dairy products and acidic gruel.\n",
            "lactobacillus fermentum isolates were isolated from a relatively large geographic area including mongolia, four chinese provinces and two autonomous regions of china, and from various naturally fermented dairy products and acidic gruel.\n",
            "lactobacillus fermentum is an economically important species of lactic acid bacterium  used in the production and preservation of fermented food as an acid-producing starter culture.\n",
            "lactobacillus fermentum isolates were isolated from a relatively large geographic area including mongolia, four chinese provinces and two autonomous regions of china, and from various naturally fermented dairy products and acidic gruel.\n",
            "lactobacillus fermentum is an economically important species of lactic acid bacterium  used in the production and preservation of fermented food as\n",
            "27\n",
            "[{'summary_text': 'glioblastoma multiforme , a grade iv astrocytoma, is the most common form of primary brain cancer, with a median survival of approximately 1\\xa0year after multi-modal treatments.\\nwe identified  <dig> fusion events in  <dig> gbm samples and  <dig> non-tumor samples from the tcga cohort.\\nwe also identified  <dig> fusion events in  <dig> gbm samples from illumina paired-end, 75\\xa0bp rna-seq data and  <dig> fusion events in  <dig> gbm samples from tcga and ivy center cohorts.\\nwe identified  <dig> fusion events in  <dig> gbm samples from  <dig> non-tumor samples and  <dig> gbm samples from two independent cohorts.\\nwe also identified  <dig> fusion events in  <dig> gbm samples from illumina paired-end, 75\\xa0bp rna-seq data and  <dig> fusion events in  <dig> gbm samples from tcga and ivy center cohorts.\\nwe identified  <dig> fusion events in  <dig> gbm samples and  <dig'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "cancers result from the accumulation of genomic mutations and epigenetic alterations that change gene expression and function. in particular, gene fusions have been recognized as an associated and significant feature of cancer since the characterization of the philadelphia chromosome  <cit> . the occurrence of gene fusions in solid tumors has long been noted, but their importance has been appreciated only recently, largely due to high throughput technologies such as transcriptome sequencing  . rna-seq permits genome-wide transcription analysis for novel transcript discovery.\n",
            "\n",
            "rna-seq has spurred important gene fusion discoveries for a number of different cancers, including lung  <cit> , prostate  <cit> , breast , brain  <cit> , thyroid  <cit>  and bladder carcinomas  <cit> . one obvious benefit from gene fusion discovery is the potential to develop novel treatments that target these genetic abnormalities. the eml4-alk translocation fusion is an example in which the fusion causes constitutive kinase activity. mouse fibroblasts transfected with eml4-alk formed tumors when this fusion was injected into nude mice, thus demonstrating the oncogenic activity of the resultant protein  <cit> . crizotinib, a competitive inhibitor of alk, has recently been granted fda approval for the treatment of specific late-stage, non-small cell lung cancers, and presently there are two phase  <dig> trials in progress  <cit> .\n",
            "\n",
            "glioblastoma multiforme , a grade iv astrocytoma, is the most common form of primary brain cancer, with a median survival of approximately 1 year after multi-modal treatments  <cit> . recent studies suggest that nearly 80% of all malignant brain tumors are accounted for by the broad category of gliomas, and 54% of all malignant brain tumors are gbm  <cit> . the first fusion protein discovered in glioblastoma was the fig-ros <dig> fusion, in which an intra-chromosomal deletion of 240 kb leads to a constitutively active kinase, suggesting oncogenic activity  <cit> . two more studies reported fusions of pdgfra-kdr  <cit>  and leo1-slc12a <dig>  <cit> , each in a single patient sample. the fgfr-tacc fusion is one of the recurrent fusions in gbm and it has been reported in three studies  <cit> . the oral administration of an fgfr inhibitor has been shown to prolong the survival of mice harboring intracranial fgfr-tacc-initiated glioma  <cit> . egfr-sept <dig> fusions present in about 4% of gbms were shown to be functional and sensitive to egfr inhibition in a recent study  <cit> .\n",
            "\n",
            "in this study, we focus on identification of gene fusion events from gbm transcriptome data. using our in-house pipeline, we identify and validate  <dig> fusion events in  <dig> gbm samples by analyzing solid single-end 50 bp data. we also identify  <dig> high-confidence gene fusion events in  <dig> gbm samples by analyzing tcga illumina hiseq paired-end 75 bp transcriptome data. we integrate gene fusion data with copy number data to elucidate fusion mechanisms in gbm.\n",
            "\n",
            "RESULTS\n",
            "gene fusion discovery pipeline for solid single-end 50 bp data\n",
            "we profiled the transcriptome of  <dig> gbm samples and  <dig> non-tumor samples using the solid sequencer. we generated 50 bp single-end rna-seq reads with sequencing depths ranging from  <dig> to  <dig> million reads . a variety of software packages are available for gene fusion discovery for illumina paired-end, illumina single-end and solid paired-end data . we developed an in-house gene fusion discovery pipeline, as no software package was available for single-end solid data . first, we aligned all the reads and calculated reads per kilobase per million  for each exon using bioscope  <dig>  software package  <cit> . gene annotations were combined from three databases: ensembl gene annotation version  <dig>  ucsc and refseq genes . cancer outlier profile analysis   <cit>  was performed for each exon to identify exons with substantially higher expression in a small set of samples. we evaluated the expression variation at 5′ and 3′ exons of all genes that had at least one exon with outlier expression levels. we extracted reads that partially mapped to the junction where there was a significant change in expression levels for a given gene. we then constructed a consensus sequence from the partially extracted reads. after converting the consensus sequence from color space to base space format, we used the ucsc blat tool  <cit>  to map the consensus sequence to the human genome . if part of the consensus sequence mapped to the original gene and the rest mapped uniquely to another region in the genome, then the sequence was considered a fusion sequence. we identified  <dig> such sequences  in eight samples. we were able to validate all of the  <dig> fusion transcripts using fusion qpcr followed by sanger sequencing. figure  <dig> illustrates the mon2-mars gene fusion as one example of a fusion transcript. the outlier expression of the mon <dig> and mars exons can be observed with a z-score >  <dig>  panel a shows the rna-seq read distribution across all exons for both genes. the mon <dig> read distribution shows higher 5′ expression relative to its 3′ expression, and the mars read distribution shows higher 3′ expression relative to its 5′ expression. partially mapped reads at exon  <dig> of mon <dig> and exon  <dig> of mars map to the mon2-mars fusion sequence. panel b shows the gel image for fusion qpcr. the product can be observed in sample sn <dig>  but not in non-tumor brain and mon2-mars fusion negative gbm samples. panel c shows the sanger sequencing trace of the fusion pcr product. detailed images for the other  <dig> fusion sequences are available in additional file  <dig> figure  <dig> \n",
            "gene fusion discovery pipeline for the ivy center solid single-end data. reads were aligned to the hg <dig> assembly using bioscope- <dig>  software by life technologies. rpkm values were calculated for each exon, followed by modified cancer outlier profile analysis . if any of the exons of a gene displayed outlier expression in a sample, then the read distribution across that gene was evaluated for that sample. if either the 3′ or 5′ end of the gene had a considerably lower rpkm value compared to the other, the gene was further evaluated for fusion. all partially mapped sequences to a potential fusion breakpoint were extracted. one or more consensus sequence was generated and translated to base space format from the color space format. the consensus sequences were then aligned to the hg <dig> human genome using ucsc blat. if part of the consensus sequence mapped to the known exon and the other part uniquely mapped to the genome, the sequence was considered a potential fusion sequence. all potential fusion sequences were validated with fusion qpcr followed by sanger sequencing.\n",
            "\n",
            "\n",
            "ivy center fusions\n",
            "\n",
            "\n",
            "fusions identified in the ivy center solid single-end dataset.\n",
            "example of a gene fusion identified in the ivy center solid single-end data. this figure illustrates the mon2-mars gene fusion. a. outlier expression of the mon <dig> and mars exons can be observed with a z-score >  <dig>  mon <dig> read distribution shows a drop after exon  <dig>  and mars read distribution shows a rise at exon  <dig>  partially mapped reads at exon  <dig> of mon <dig> and exon  <dig> of mars map to the mon2-mars fusion sequence. the sequence is represented both in color space and base space format. the purple color indicates a mismatch in the color space format. b. gel images showing the fusion qpcr result. the product can be observed in sample sn <dig>  but not in the non-tumor brain and mon2-mars fusion negative gbm samples. c. trace from sanger sequencing of the fusion pcr product.\n",
            "\n",
            "gene fusions in the ivy center solid dataset\n",
            "we identified  <dig> fusion transcripts in eight out of the  <dig> gbm samples . two samples, sn <dig> and sn <dig>  harbored multiple fusions. both fusion partners in eight of the fusion transcripts are well annotated genes. five  out of these eight transcripts are predicted to be in-frame fusions coding for a chimeric protein product. transcript creb1 → pard3b results in a c-terminal truncation of the 5′ fusion gene partner due to a frameshift. in transcript plekha6 → pik3c2b, the entire pik3c2b coding sequence is preserved, but the fusion junction is at a novel 5′ utr exon for plekha <dig>  in transcript lancl2 → rp11-745c <dig>  the 5′ partner gene fuses with a non-coding rna resulting in c-terminal truncation. for the other five fusion transcripts, the 5′ partner gene fuses with genomic sequence without gene annotation, denoted as “uar” in table  <dig>  three of these transcripts  result in c-terminal truncation of the 5′ partner genes. one transcript  is predicted to result in a shorter isoform with an extended 3′ utr, and one transcript does not have any predicted protein product . we had tissue available from surgery at recurrence for patient sn <dig>  and we were able to validate the presence of sec61g → uar at recurrence. predicted protein sequences are provided in additional file  <dig>  all fusions are intra-chromosomal in our cohort, and fusion partners are in close proximity, ranging from a distance of  <dig>  million base pairs to  <dig> kilo base pairs between the two partners. although there are no recurrent fusions in our small cohort, there are multiple genes, slc35e <dig>  pik3c2b and znf <dig>  that occurred in more than one fusion transcript. an fgfr3 → tacc <dig> fusion was recently reported by three independent studies as a recurrent gene fusion  <cit> . all our fusion transcripts are highly overexpressed compared to their wildtype gene partners, as is evident in the third column in table  <dig>  which shows a much higher number of reads spanning the fusion junction compared to the number of reads spanning the known wildtype junctions. one of the fusions, slc35e3 → uar, has two isoforms.\n",
            "\n",
            "gene fusions in the illumina hiseq tcga dataset\n",
            "we downloaded rna-seq data for  <dig> tcga samples from cghub  <cit>  to explore the gene fusion landscape of gbm beyond our cohort. the tcga transcriptome data are 75 bp paired-end reads generated using illumina hiseq with sequencing depths ranging from  <dig> to  <dig> million reads per sample. we used the tophat-fusion  <cit>  and snowshoes-ftd  <cit>  software packages to identify fusions because both packages are expected to have a very low false-positive rate. there were  <dig> and  <dig> fusion sequences identified by tophat-fusion and snowshoes ftd suggesting a large number of false positives . the number of fusion sequences could be reduced by increasing the threshold for the minimum number of fusion spanning reads, but this modification can lead to the failure to identify some truly important fusions, such as cep85l → ros <dig>  in our solid dataset, the fgfr3 → tacc <dig> fusion has the second lowest number of junction spanning reads. because our method resulted in a 100% validation rate, we applied filtering steps based on our method to the fusion sequences identified by both packages. we required that at least one of the breakpoints must be a known exon boundary. to reduce the likelihood of identifying passenger fusions  <cit> , we required that at least one of the fusion spanning reads must have a ratio of greater than two compared with its corresponding wild-type exon-exon spanning reads. we discarded gene fusions involving adjacent genes. exact details are provided in the methods section. after curating fusion sequences from both packages, we obtained a set of  <dig> high-confidence fusion sequences, which was referred to as the curated set. curated fusions were present in 53%  of patients, and 22%  of these patients harbored more than one fusion. the curated fusion set is available in additional file  <dig> \n",
            "\n",
            "gene fusions and copy number changes\n",
            "the circos plot of all curated fusions  shows specific genomic hotspots where fusions occur in gbm. two major genomic hotspots are on chromosomes  <dig>  and  <dig> . in our solid dataset,  <dig> of  <dig> validated fusions were located on 7p <dig> and 12q14- <dig>  other regions with higher frequency of fusions are on chromosomes  <dig>   <dig>   <dig> and  <dig>  these genomic hotspots for fusions are the regions that are frequently amplified in gbm, as observed in figure  <dig> <cit> . because affymetrix snp array data were available for all but two tcga samples, we looked for associations between fusion points and copy number data. we downloaded level  <dig> segmented copy number data from tcga  <cit> . the start and end points of each segment were considered to be the genomic breakpoints. for the curated set, copy number data were available for  <dig> fusion sequences, out of which at least one of the partner genes harbored a genomic breakpoint in  <dig> cases . we also predicted the fusion mechanism for each of the fusion sequences based on the copy number data. figure  <dig> shows the distribution of different fusion mechanisms for all curated fusions. we binned fusion mechanisms into six types:figure  <dig> \n",
            "gene fusions identified in the tcga illumina hiseq paired-end data and the ivy center solid single-end data. this circos plot shows all the curated fusions identified in tcga dataset. fusions are represented by arcs. the thickness of the arc represents the number of fusion-spanning reads. the colors of the arc represent the likely mechanism of the fusion formation. green arcs represent fusions formed by focal amplifications, cyan arcs represent fusions formed by tandem duplications, magenta arcs represent deletion-based fusions, dark orange arcs represent fusions formed by complex genomic rearrangements and blue arcs show the fusions formed by inter-chromosomal rearrangements. the outer ring shows the fusion breakpoint density histogram at a given genomic location. yellow triangles represent the fusions detected in the ivy center samples. the size of the triangle indicates the number of breakpoints in that location. the two rings outside of the ideogram show frequency of the copy number gain and loss in tcga samples. two major genomic hotspots for fusions can be observed on chromosomes  <dig>  and  <dig> . these regions also show frequent focal gains.\n",
            "association between gene fusions and copy number changes in the tcga dataset. a. distribution of each fusion mechanism over three chromosomes  with most fusions. majority of fusions due to focal amplifications  are found on chromosome  <dig>  fusions formed due to complex genomic rearrangements are mostly present on chromosomes  <dig> and  <dig>  b. fusion without associated copy number changes. c. fusion points within a genomic amplicon. d. fusions due to tandem duplications. e. black arc showing fusion due to interstitial genomic deletion. f. fusions due to complex genomic rearrangements. g. inter-chromosomal fusion with at least one fusion point near a genomic breakpoint. h. distribution of fusion mechanisms for  <dig> fusion sequences. eight percent of the fusion sequences do not have associated copy number changes. forty percent of fusion sequences are formed by complex genomic rearrangements. fusions in the proneural subtype are mostly formed by complex genomic rearrangements.\n",
            "\n",
            "\n",
            "\n",
            "no copy number changes - there are no genomic breakpoints around fusion points. these could be either inter- or intra-chromosomal fusions, see figure 4b.\n",
            "\n",
            "focal amplifications - fusion points are within a genomic amplicon, see figure 4c.\n",
            "\n",
            "tandem duplications - fusion points are around the start and end of an amplified genomic segment, see figure 4d.\n",
            "\n",
            "deletion-based - fusion points are around the start and end of a relatively deleted genomic segment, see figure 4e.\n",
            "\n",
            "complex genomic rearrangements - both fusion points are around genomic breakpoints with multiple segments between the two breakpoints, see figure 4f.\n",
            "\n",
            "inter-chromosomal - fusion partners are located on different chromosomes with at least one fusion point near a genomic breakpoint, see figure 4g.\n",
            "\n",
            "\n",
            "\n",
            "only 8% of the fusions are without accompanying copy number changes suggesting that the majority of the fusions in gbm are associated with unbalanced genomic rearrangements. majority of the fusions in focal amplicons are present on chromosome  <dig> and restricted to the egfr locus, see figure  <dig> and figure 4a. approximately 40% of all the fusions in gbm result from complex genomic rearrangements , see figure 4h. some of the inter-chromosomal rearrangements also display complex fusion mechanisms, see figure 4g. a recent study analyzed whole genome sequencing data and showed a high incidence of cgr in gbm resulting from chromothripsis—39% in gbm compared to 9% in other tumor types  <cit> . fusions generated through cgrs are largely present on chromosomes  <dig> and  <dig>  see figure 4a. the distribution of cgr based fusions on chromosome  <dig> is largely restricted to 12q14- <dig> amplicon, see figure  <dig>  even though partners of fusion sequences formed due to cgrs belong to different copy number segments, they have highly correlated copy number value, see figure 4f. this suggests co-amplification of segments involving fusion.\n",
            "\n",
            "gene fusions and molecular features\n",
            "we checked to see if samples with at least one curated fusion were enriched in any clinically associated molecular features. we did not find any association with presence of egfr viii, mutation/homozygous deletion of pten or tp <dig>  mutation of idh <dig> or g-cimp status . amplifications of egfr were more prevalent in samples with at least one fusion compared to samples with no fusions . we observed that the samples with classical subtype were more likely to have fusions  and that samples with mesenchymal subtype were less likely to have fusions , see figure 5a. this result can be explained by the association of genomic fusion hotspots with subtypes. almost all of the samples with a classical subtype have focal amplification of the egfr locus, and samples with a mesenchymal subtype have a much lower incidence of focal amplifications on chromosomes  <dig> and  <dig>  <cit> . figure 5b shows the chromosomal distribution of fusion breakpoints for each subtype. the majority of the fusions in the classical subtype are located on chromosomes  <dig>   <dig> and  <dig>  fusions in the mesenchymal subtype are mostly present on chromosomes  <dig>   <dig>   <dig> and  <dig>  whereas chromosomes  <dig> and  <dig> harbor the majority of the fusions with the proneural subtype. proneural subtype shows enrichment of fusions formed by complex genomic rearrangements, see figure 4h. gene fusions in samples with the neural subtype have a broader chromosomal distribution, with the majority of breakpoints on chromosomes  <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig> and  <dig> figure  <dig> \n",
            "association between gene fusions and molecular subtype in the tcga dataset. a. seventy-two percent of the samples with a classical subtype had at least one fusion event compared to 39% of samples with a mesenchymal subtype . b. chromosomal distribution of fusion events for different molecular subtypes.\n",
            "\n",
            "\n",
            "\n",
            "predicted structure of the curated fusions set\n",
            "we predicted the amino acid sequence of all curated fusions based on their chimeric nucleotide sequence. a significant portion of the fusions  were predicted to be in-frame fusions with amino acid sequences present from both fusion partner genes. another 18% were predicted to have c-terminal truncation due to either the out of frame fusion with another gene or fusion with an unannotated region. approximately 8% of fusions are predicted to result in the same protein product as its 3′ partner gene by borrowing only the promoter from the 5′ partner. in approximately 10% of the fusions, the 5′ gene partner is predicted to contribute only the promoter, but the n-terminal of the 3′ gene is truncated. we also observed another novel class of fusions that involve non-coding rna genes. in approximately 14%  of the fusions, the 5′ partner gene is predicted to have a c-terminal truncation due to fusion with a non-coding rna. these fusions also result in the expression of non-coding rnas that are not expressed in other samples. another important set of fusions involve tyrosine kinases. in  <dig> cases, the fusion sequences retained the tyrosine kinase domain. egfr , fgfr <dig>  and ntrk <dig>  were recurrently fused. other kinase genes included ephb <dig>  flt <dig> and ros <dig> \n",
            "\n",
            "recurrent gene fusions\n",
            "although more than half of gbms showed evidence of gene fusions, there were very few fusions that were present in more than one sample. one of those fusions is the already reported fgfr3 → tacc <dig> fusion, which was found in two patients in the tcga cohort and in one patient in the ivy center cohort. egfr → sept <dig>  an in-frame fusion with c-terminal deletion of egfr, was found in three patients in the tcga cohort. lancl2 → sept <dig>  an out of frame fusion that leads to the c-terminal truncation of lancl <dig>  was found in two tcga patients. two additional patients, one tcga and one ivy center, had fusions of lancl <dig> with non-coding rna rp11-745c <dig> , which also resulted in the c-terminal truncation of lancl <dig>  the same non-coding rna rp11-745c <dig>  fused with egfr in two tcga patients, resulting in the c-terminal truncation of egfr. there are  <dig> genes that are fusion partners in more than one patient sample . the majority of these genes  are on genomic fusion hotspots located on chromosomes  <dig> and  <dig> table  <dig> \n",
            "fusions involving genes that partner in more than one fusion\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "discussion\n",
            "our study highlights the prevalence of gene fusions as one of the major genomic abnormalities in gbm. fusions occur in approximately 30-50% of gbm patient samples. in the ivy center cohort of  <dig> patients, 33% of samples harbored fusions that were validated by qpcr and sanger sequencing. we were able to identify high-confidence gene fusions from rna-seq data in 53% of samples in a tcga cohort of  <dig> patients. we identified  <dig> cases  with fusions retaining the tyrosine kinase domain in the tcga cohort and one case in the ivy center cohort. recent advances in the development of tyrosine kinase inhibitors  have demonstrated that these drugs can provide significant benefit to patients whose tumors have a specific genetic abnormality. we also identified a novel class of fusions  that result in the c-terminal truncation of its 5′ partner due to fusion with non-coding rna genes. one such case was also present in the ivy center cohort. this study reveals the diversity of gene fusions in gbm samples. the majority of the fusions are private fusions occurring in one patient. there are a few fusions that recur at low frequency in gbm.\n",
            "\n",
            "our study is the first to provide a comprehensive view of the gene fusion landscape in gbm by examining sequences from  <dig> patients from two independent cohorts. we successfully utilized our in-house pipeline for fusion discovery using solid single-end, 50 bp rna-seq data with a 100% validation rate. for the tcga cohort, we used two different gene fusion detection software packages to comprehensively identify fusions from illumina paired-end, 75 bp rna-seq data. ours is the first study to describe recurrent fusions involving non-coding genes. we combined copy number data with gene fusion discovery to elucidate mechanisms of the formation of gene fusions in gbm. all of the fusions detected in this study can be further visualized and analyzed on our website .\n",
            "\n",
            "we were able to validate all of the fusions in our solid single-end rna-seq data by using strict filtering criteria. it is likely that we may have underestimated fusions for ivy center data. due to lack of access to the tissue samples, we could not determine the validation rate for our set of curated fusions in the tcga cohort. the curated fusion set did have a significantly higher percentage of fusions associated with copy number changes relative to the low-confidence set. we applied filters to discard likely passenger fusions  <cit> , but the functional significance of these fusions still needs to be evaluated.\n",
            "\n",
            "singh et al. was the first study to describe multiple fusions of fgfr-tacc in gbm, reporting this phenomenon in  <dig> of the  <dig> tumors examined. they showed that the fusion protein has oncogenic activity when introduced into astrocytes and oral administration of an fgfr inhibitor prolongs the survival of mice harboring intracranial fgfr-tacc-initiated glioma  <cit> . a second study by parker et al. showed that the fusion gene is overexpressed by escaping mir-99a regulation due to loss of the 3′ utr of fgfr <dig>  <cit> . in their cohort,  <dig> out of  <dig> samples harbored the fgfr3 → tacc <dig> fusion. in our ivy center cohort, the fgfr3 → tacc <dig> fusion was detected in one out of  <dig> samples. we tested for this fusion in an additional  <dig> samples in addition to the  <dig> rna-seq samples, but did not detect any fusion events. in the tcga cohort,  <dig> of  <dig> samples harbored the fgfr3 → tacc <dig> fusion. fusions of fgfr genes are identified in other cancers, including bladder cancer, cholangiocarcinoma, squamous lung cancer, breast cancer, thyroid cancer, oral cancer, head and neck squamous cell carcinoma and prostate cancer  <cit> . tropomyosin-receptor kinases  are known to play a role in cancer biology. rearrangements of the ntrk <dig> gene are consistently observed in a small fraction of papillary thyroid carcinomas  <cit> . we identified two cases of ntrk <dig> fusions in the tcga cohort. frattini et al.  <cit>  screened  <dig> samples for nfasc-ntrk <dig> fusion but did not find any. we identified a single case of a cep85l-ros <dig> fusion in the tcga patient samples. in a recent study by giacomini et al.  <cit> , a cep85l-ros <dig> fusion was detected for angiosarcoma. there have been two more reported cases, one angiosarcoma and one epithelioid hemangioendothelioma, with ros <dig> rearrangements. ros <dig> rearrangements also define a unique molecular subclass of lung cancer that may respond to an alk inhibitor  <cit> . we identified fusions of egfr in nine patient samples from the tcga cohort, out of which six retained the tyrosine kinase domain and resulted in a carboxyl-terminal truncation. a study by cho et al. has shown that cetuximab prolonged the survival of intracranially xenografted mice with oncogenic egfr carboxyl-terminal deletion mutants compared with untreated control mice  <cit> . it is likely that patients with fusions of egfr leading to carboxyl-terminal truncation will show sensitivity to egfr inhibitors. frattini et al.  <cit>  showed that egfr-sept <dig> fusions which occur in about 4% of gbms was a functional gene fusion in gbm and confers mitogen independence and sensitivity to egfr inhibition. a total of  <dig> cases from both cohorts have fusions of genes involved in chromatin remodeling and modification. these genes include arid1a, arid1b, ash1l, chd <dig>  hdac <dig>  hmga <dig>  jmjd1c, kdm4b, rere, setd1b and yeats <dig>  arid1a-mast <dig> fusion has been shown to be a critical driver fusion in an mda-mb- <dig> breast cancer cell line  <cit> . in  <dig> samples, the 5′ partner gene fuses with non-coding rna. these fusions are predicted to have a c-terminal truncation. these cases also have highly expressed non-coding rnas that are not expressed in other samples. a recent study by zhang et al.  <cit>  discovered a signature comprising of six long non-coding rna that predicts survival in gbm. there is now growing evidence of an oncogenic and tumor suppressive role for long, non-coding rnas in tumor biology  <cit> . their identification in gene fusion events has thus far been neglected, as most studies focus on fusions of the coding genes.\n",
            "\n",
            "even though gene fusion events in gbm are abundant with scarce recurrent events, they are not random events. majority of the fusion events occur at 7p <dig>  12q14- <dig>  1q <dig> and 4q <dig> which are also recurrently amplified regions in gbm. these fusion hotspots are consistent in both ivy and tcga cohorts. also majority of the fusion events are due to unbalanced genomic rearrangements. analysis of whole genome sequencing data also showed that 88% of genic rearrangements in gbm are associated with copy number alterations  <cit> . some of the key genes implicated in gbm biology within these hotspots are egfr, mdm <dig>  cdk <dig>  pik3c2b, mdm <dig> and pdgfra. a recent study  <cit>  identified a dense breakpoint pattern on 12q14- <dig> indicative of local chromosome instability and defined this region as “breakpoint enriched region” . they showed that patients with ber pattern had poor survival and this pattern was associated with mdm2/cdk <dig> co-amplification. there are two other cancers, dedifferentiated liposarcomas and lung adenocarcinomas that also show mdm2/cdk <dig> co-amplification in 90% and 4% of cases respectively  <cit> . all three types of cancer display distinct genomic aberration patterns in 12q14- <dig> region in spite of having mdm2/cdk <dig> co-amplification. gbm samples show shattering of the region with alternate high level deletions and gains, lung adenocarcinomas mostly contain large amplified segments and dedifferentiated liposarcomas contain multiple amplified segments . whole genome sequencing, copy number and rna-seq datasets show that gbms contain deletion bridges that connect these amplified segments and generate a large number of fusion transcripts. such complex genomic rearrangements are more prevalent on chromosome  <dig> but not limited to as shown in the study by malhotra et al.  <cit>  where they analyzed whole genome sequencing data of  <dig> gbms. about 40% of fusion transcripts are formed due to such complex genomic rearrangements. with the advent of rna-seq technology the list of fusion sequences in solid tumors is growing exponentially but little is known about the mechanisms that facilitate fusion events. the formation of the tmprss2-erg gene fusion that occurs in about 50% of prostate cancers has been shown to be facilitated by androgen signaling which induces proximity of the tmprss <dig> and erg genomic loci and then exposure to gamma irradiation which causes dna double-strand breaks  <cit> . the overview of the fusion landscape in gbm leads to questions about what mechanisms are responsible for generating highly site specific dna double-strand breaks and then joining of these breaks that result in complex genomic rearrangements.\n",
            "\n",
            "CONCLUSIONS\n",
            "gene fusions are frequent genomic abnormalities in gbm. the majority of the fusions are private fusions, with a minority recurring in multiple patients. complex genomic rearrangements are the major mechanism by which fusions are formed in gbm. due to the low frequency and rarity of clinically relevant fusions, rna-seq of gbm patient samples is an essential tool for the identification of patient specific fusions that can drive personalized therapy.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "glioblastoma multiforme , a grade iv astrocytoma, is the most common form of primary brain cancer, with a median survival of approximately 1 year after multi-modal treatments.\n",
            "we identified  <dig> fusion events in  <dig> gbm samples and  <dig> non-tumor samples from the tcga cohort.\n",
            "we also identified  <dig> fusion events in  <dig> gbm samples from illumina paired-end, 75 bp rna-seq data and  <dig> fusion events in  <dig> gbm samples from tcga and ivy center cohorts.\n",
            "we identified  <dig> fusion events in  <dig> gbm samples from  <dig> non-tumor samples and  <dig> gbm samples from two independent cohorts.\n",
            "we also identified  <dig> fusion events in  <dig> gbm samples from illumina paired-end, 75 bp rna-seq data and  <dig> fusion events in  <dig> gbm samples from tcga and ivy center cohorts.\n",
            "we identified  <dig> fusion events in  <dig> gbm samples and  <dig\n",
            "28\n",
            "[{'summary_text': 'the results indicate that a specific statistical ir model, an lda model, can be used to infer the hidden factors permeating a biomedical text corpus and that it can be used to synthesize and organize information about complex biological phenomena.\\nin particular, lda-based topic labels were generated from a corpus of  <dig>  documents and a mixture of unigram and random models.\\nin particular, lda-based topic labels were generated from a corpus of  <dig>  documents and a mixture of unigram and random models.\\nin particular, lda-based topic labels were generated from a corpus of  <dig>  documents and a mixture of unigram and random models.\\nthe results indicate that despite being estimated from a simple bag of words, lda-based topic labels are sufficient for the model to be used to enhance our understanding ofc. elegans biology.\\nin particular, lda-based topic labels were generated from a corpus of  <dig>  documents and a mixture of unigram and random models.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "in the design, analysis and interpretation of experiments, biomedical and clinical\n",
            "researchers encounter the problem of evaluating and summarizing prior knowledge on\n",
            "the subject under investigation. traditional solutions include examining articles in\n",
            "scientific journals, primarily via pubmed access to medline, and interrogating\n",
            "www-based sources such as entrez gene  <cit>  and online mendelian inheritance in man   <cit> . consider a scenario in which a researcher seeks enhanced knowledge about\n",
            "a protein implicated in aging. typically, the steps involved in addressing this\n",
            "problem include interrogating structured data resources: searching protein sequence\n",
            "databases to identify homologs in other species, querying warehouses of genomic\n",
            "information to determine key non-coding regions and polymorphisms, examining\n",
            "collections of high-throughput molecular profiling data sets to ascertain genes with\n",
            "similar patterns of expression, probing ontologies such as the gene ontology   <cit>  to uncover other genes with similar patterns of annotation, and so on.\n",
            "the entire procedure is accompanied by examination of the literature to determine,\n",
            "for example, classes of proteins mentioned in the same article as the putative\n",
            "gerontogene.\n",
            "\n",
            "advanced techniques and sophisticated tools for interacting with structured data are\n",
            "well known, widely available and include blast  <cit>  for sequence databases, ensembl  <cit>  and the ucsc browser  <cit>  for genomes, geo  <cit>  for transcript profiles, and tools available from the go that allow\n",
            "navigation of terms in the ontology. this is less true for the scientific\n",
            "literature. given the time-consuming yet critical importance of synthesizing\n",
            "information in text corpora such as medline, the problem of making data\n",
            "interpretation a more systematic and automated endeavor is emerging as an important\n",
            "topic of research . the development of strategies capable of providing a user the ability\n",
            "to assimilate and act upon information present in resources of structured and\n",
            "unstructured data remains an important goal.\n",
            "\n",
            "the primary aim of biomedical text mining is the systematic analysis of document\n",
            "collections such as medline abstracts and full-text journal articles with the goal\n",
            "of generating useful and unanticipated scientific discoveries . examples of tasks addressed by text mining methods include identifying\n",
            "literature relevant to specific molecules, finding associations between genes and\n",
            "diseases, determining putative functions for proteins, and predicting regulatory\n",
            "networks.\n",
            "\n",
            "a common approach to text mining is to treat the problem as one of natural language\n",
            "processing   <cit> . nlp methods concentrate on the linguistic structure of documents and\n",
            "make explicit use of syntactic, relational, and ontological knowledge. in biology,\n",
            "such approaches  <cit>  have been employed for information extraction: the task of ascertaining\n",
            "facts, relations, and entities in unstructured written language such as\n",
            "protein-protein interactions, protein subcellular location, and gene names. tools\n",
            "based on these ideas include textpresso  <cit>  and telemakus  <cit> . elsewhere, nlp has been used in conjunction with locuslink and go to\n",
            "compare omim to medline  <cit> .\n",
            "\n",
            "recently, nouns extracted from medline abstracts tagged with parts of speech were\n",
            "combined with knowledge from other sources, principal component analysis, and means\n",
            "linkage clustering to find associations between genes and phenotypes  <cit> . in general, nlp can be effective in circumscribed domains where\n",
            "linguistic knowledge is available and the terminology evolves slowly, is consistent,\n",
            "largely unambiguous, and relatively simple. however, the paucity and incomplete\n",
            "nature of such information for biomedical corpora suggests that the full potential\n",
            "of text mining in biology remains unrealized.\n",
            "\n",
            "an alternative to nlp is to frame the problem from the perspective of information\n",
            "retrieval   <cit> . statistical ir methods explore large quantities of information and often\n",
            "involve capabilities for clustering, classifying, categorizing, summarizing, and\n",
            "detecting novel, similar and relevant objects. the most successful testaments to the\n",
            "real-world utility of ir techniques are internet search engines. thus, statistical\n",
            "ir models of biological document collections could reveal rich, complex, and\n",
            "previously unappreciated relationships. such results would complement insights\n",
            "derived from analysis of molecular profiling, protein-protein interaction, gene\n",
            "knock-out, and similar types of data. with systematic deployment of ir tools, the\n",
            "interrogation of biomedical corpora could become as routine and indispensable a part\n",
            "of research as analyzing genomic and genetic data is today. the analogy is more than\n",
            "superficial and extends to the direct use of ir techniques such as singular value\n",
            "decomposition in bioinformatics . thus, the common mathematical foundations for algorithms that underpin\n",
            "ir and genome analysis make it possible to envision integrated procedures that\n",
            "combine primary biological data with biological corpora.\n",
            "\n",
            "this work describes an application of statistical ir methodology to the analysis of a\n",
            "biomedical text corpus, the caenorhabditis genetic center \n",
            "bibliography . the specific model at the heart of\n",
            "this study is the latent dirichlet allocation  model  <cit> , a hierarchical bayesian model employed previously to analyze text\n",
            "corpora and to annotate images  <cit> . recently, lda has been used to extract and analyze the topics present in\n",
            "a document corpus consisting of articles published in the journal proceedings of\n",
            "the national academy of sciences  <cit> .\n",
            "form.\n",
            "\n",
            "in general, ir methods assume that the order of words in a document can be neglected\n",
            "and view documents as \"bags of words.\" the loss of information incurred by ignoring\n",
            "word order is offset by the ability to devise efficient computational algorithms\n",
            "that are viable for large corpora. although there is no theoretical justification\n",
            "for casting a document in this manner, the practical benefits and utility of doing\n",
            "so are considerable. the lda model considered here is a model for a corpus viewed as\n",
            "a collection of bags of words. it assumes that each word of each document is\n",
            "generated by one of several \"topics\"; each topic is associated with a different\n",
            "conditional distribution over a fixed vocabulary. the same set of topics is used to\n",
            "generate the entire set of documents in a collection but each document reflects\n",
            "these topics with different relative proportions. thus, lda is a mixture of mixtures\n",
            "model, i.e., the mixture components are shared across all documents but\n",
            "each document exhibits different mixture proportions. as a generative probabilistic\n",
            "model, the lda can handle unseen or novel data, i.e., a document that was\n",
            "not one of the bag of words used to estimate the model.\n",
            "\n",
            "the fundamental entities in lda, random variables representing topics and words, are\n",
            "grouped together in such a way to form a corpus, i.e., a group of groups of\n",
            "words. the hierarchical nature of the model stems from the fact that documents are\n",
            "modeled as probability distributions across topics, and topics are modeled as\n",
            "probability distributions across words. a notable virtue of lda is that a given\n",
            "topic can occur with high probability in multiple documents, and that a given word\n",
            "can occur with high probability in multiple topics. topics are treated as latent\n",
            "variables, namely entities that are not present explicitly in the data , but are presumed to be present implicitly and are to be\n",
            "inferred by statistical analysis.\n",
            "\n",
            "to analyze the cgc bibliography, each item in the corpus was recast as a bag of words\n",
            "and the resultant data set of documents was used to estimate the parameters of three\n",
            "different statistical ir models. the predictive performance of the lda model was\n",
            "better than that of two simpler bag of words models, a unigram model and a mixture\n",
            "of unigrams model, trained on the same data set. the potential of lda in assisting\n",
            "biological studies was illustrated by considering the phenomenon of nematode aging.\n",
            "in order to illuminate the hidden factors permeating a corpus and captured by the\n",
            "topics discovered by a trained cgc lda model, lda topics were labeled via an\n",
            "automated process that assigned words from the cgc vocabulary \n",
            "and go terms  to each topic. examination of these labels\n",
            "indicated that the cgc topics captured meaningful and plausible facets of nematode\n",
            "biology. to investigate aging, topics whose corpus-based labels included many cgc\n",
            "words corresponding to the names of genes known to influence life span were\n",
            "identified. for the two topics with the greatest number of such cgc-based topic\n",
            "labels, novel candidates for age-related genes were equated with other cgc-based\n",
            "topic labels that corresponded to gene names . finally, an\n",
            "lda-based measure of pairwise document similarity was devised and used to address\n",
            "the problem of searching a database of documents to determine topic-space homologs\n",
            "of a query document. inspection of the \"document homologs\" of the cgc item shown inthe clk- <dig> gene.\n",
            "\n",
            "this work highlights the potential and utility of lda in organizing and exploiting\n",
            "one type of widely available information resource, a collection of documents in the\n",
            "form of free or unstructured text. however, researchers are faced with a plethora of\n",
            "resources including images and structured data such as molecular sequences,\n",
            "transcript profiles, disease information, and so on. thus, there is a compelling\n",
            "need for techniques and systems able to condense, integrate and present large\n",
            "amounts of disparate data to a user. this paper concludes with a discussion of how\n",
            "the family of probabilistic graphical models, of which lda is a specific example,\n",
            "provides a framework for integrating heterogeneous data and thus meets this\n",
            "challenge.\n",
            "\n",
            "RESULTS\n",
            "lda outperforms mixture of unigrams, unigram and random models\n",
            "in order to compare different models of text, a data set of c. elegans\n",
            "related documents was created. in particular, each cgc bibliography\n",
            "free-text item was transformed into a bag of words yielding a corpus of m\n",
            "=  <dig>   <dig> documents and a v =  <dig>   <dig> word vocabulary.\n",
            "\n",
            "the generalization performance of three statistical models was assessed: an lda\n",
            "model , a mixture of unigrams model , and a \"baseline\" unigram model . a model was trained using 90% of the  <dig> \n",
            "documents in the cgc corpus and tested on the remaining 10%. lda and mixture of\n",
            "unigrams models with k =  <dig>   <dig>   <dig>   <dig>  and  <dig> latent topics were\n",
            "estimated; a single unigram model was estimated because such models harbor no\n",
            "notion of topic. the perplexity  of the\n",
            "held-out test set of j =  <dig> documents  was computed for\n",
            "each trained model. figure  <dig> shows the generalization\n",
            "performance of each model as a function of the number of latent topics. lda has\n",
            "consistently smaller perplexity scores than the two extant models indicating\n",
            "better performance on unseen documents. since the perplexity of 50- and\n",
            "100-topic lda's is low and similar, a latent space with  <dig> topics appears to\n",
            "provide a parsimonious description of the cgc corpus.\n",
            "variational distribution used to approximate the posterior in lda\n",
            "  <cit> . lda defines a distribution on a collection of documents in\n",
            "much the same manner that a profile hidden markov model yields a\n",
            "distribution on a set of  sequences  <cit> . the corpus\n",
            "depicted contains m documents and each is a sequence of n\n",
            "words. open circles are parameters  or latent\n",
            "variables . the shaded circle is the\n",
            "observed word variable  and boxes  represent\n",
            "replicates. the dirichlet parameter, α, and topic-word\n",
            "matrix, β, are corpus-level parameters sampled once in the\n",
            "process of generating a corpus. the topic proportions, θ,\n",
            "is a document-level variable sampled from αonce per document.\n",
            "the topic, z, is a word-level variable sampled from θ\n",
            "once for each word in a document. formally, a k-topic lda\n",
            "specifies a two-level probabilistic process that generates a document as\n",
            "follows,  a k-dimensional vector, θ, is chosen\n",
            "from the distribution p, and  words\n",
            "are sampled repeatedly from the document-specific mixture distribution,\n",
            "p. exact inference and parameter\n",
            "estimation involve calculating the posterior distribution on a document\n",
            "p. this is intractable because the latent variables are\n",
            "coupled via the edge between θ and z. the\n",
            "posterior can be approximated by computing the variational dirichlet\n",
            "parameter γ and the variational multinomial parameter\n",
            "φ for each word in the document. the subscripts\n",
            "m, n, and k on a parameter  or variable \n",
            "donate the m th document, nth word and kth topic\n",
            "respectively. note that the dirichlet variable α is a\n",
            "distinct component of the probability model and not merely an expression\n",
            "of uncertainty about a parameter. this differs from profile hidden\n",
            "markov models where a mixture of dirichlet distributions is used as a\n",
            "prior for amino acid/nucleotide probability distributions.\n",
            "latent topics  and a unigram model . the corpus\n",
            "depicted contains m documents and each is a sequence of n\n",
            "words. open circles represent latent variables or\n",
            "parameters . each shaded circle is an\n",
            "observed word variable . boxes  represent\n",
            "replicates. the subscripts m, n abd k on a\n",
            "parameter  or variable  donate\n",
            "the mth document, nth word and kth topic respectively.a mixture of\n",
            "unigrams generates all the words in a given document from exactly one\n",
            "topic, z. this differs from the lda model where a single\n",
            "document can express multiple topics . note that the naive\n",
            "bayes model used to cluster transcript profiling data  <cit>  has the\n",
            "same topology as the mixture of unigrams but the observed variables are\n",
            "continuous-valued expression measurements rather than discrete\n",
            "words.\n",
            "and evaluated on the cgc corpus. the score of test documents is shown\n",
            "against the number of latent topics .\n",
            "\n",
            "the ability of three specific models to retrieve a set of  <dig> aging-related\n",
            "documents in the collection of  <dig>  cgc documents was assessed: a 50-topic lda\n",
            "model estimated using all documents in the corpus, a 50-topic mixture of\n",
            "unigrams model estimated using all documents, and a model which ordered all\n",
            "documents randomly. for each model, an average precision/recall  curve was\n",
            "constructed by computing the ranking of other documents given each aging-related\n",
            "document as a query, and the average f <dig> measure was computed. figure  <dig> shows average pr curves for the three models. the average\n",
            "f <dig> measure  for the lda model, the mixture of unigrams model and\n",
            "a random model is  <dig> ,  <dig> , and  <dig>  respectively. although the f <dig> values\n",
            "for the lda and random models are similar, the smaller standard error of the lda\n",
            "model indicates its superiority to the random model. in addition, the average pr\n",
            "curves indicate that the lda places more of the age-related documents higher up\n",
            "its rankings than the random model. thus, of the three models investigated, lda\n",
            "is best able to retrieve the set of related documents. overfitting by the\n",
            "mixture of unigrams model results in a performance worse than the random\n",
            "model.\n",
            "documents . precision is the fraction of documents in a list that are\n",
            "relevant  whereas recall is the fraction of relevant\n",
            "documents in the list. for a desired level of recall, for example 70%,\n",
            "there is a corresponding precision. the graph shows average precision\n",
            "against average recall. although each point is a mean of  <dig> pairs of\n",
            "precision and recall values, the standard error is negligible and so not\n",
            "depicted.\n",
            "\n",
            "all subsequent discussion of an lda model and/or a mixture of unigrams model\n",
            "pertain to a k =  <dig> topic model estimated using all m =  <dig>  225\n",
            "cgc documents in the corpus.\n",
            "\n",
            "lda latent topics embody concepts associated with nematode biology\n",
            "a systematic strategy for clarifying the nature of the hidden factors permeating\n",
            "a corpus was devised and applied to a cgc lda. topic annotation \n",
            "is defined as an automated process that creates a verbose  description\n",
            "of an lda topic. the method designed to annotate and label topics exploited the\n",
            "corpus-level parameter β . the\n",
            "k × v topic-word matrix β collates the\n",
            "multinomial distributions over the v words in the vocabulary that\n",
            "characterize the k topics. for a given lda model of a particular\n",
            "corpus, the kth row specifies the topic-specific word distribution for\n",
            "topic k and an element, βkv, denotes\n",
            "the likelihood of the vth word given the kth topic. for each\n",
            "of the k =  <dig> topics in the cgc lda model, the v =  <dig>  971\n",
            "βkv values were ordered and used to\n",
            "generate a word rank versus topic-specific word probability plot. in every case,\n",
            "the  <dig> top-ranked words accounted for most of the probability mass. thus, these\n",
            " <dig> high probability cgc words were designated topic annotation words .\n",
            "\n",
            "two different approaches were used to create labels for each topic. corpus-based\n",
            "topic labels are topic annotation words that are unique to a topic and represent\n",
            "descriptors applicable to only one topic. cgc-based topic labels were equated\n",
            "with topic annotation words that were not assigned to any of the other 49\n",
            "topics, i.e., the  <dig> sets of cgc-based topic labels formed disjoint\n",
            "subsets of words from the cgc vocabulary. ontology-based topic labels are the\n",
            "outcome of filtering topic annotation words using an external knowledge source\n",
            "and represent descriptors applicable to one or more topic. the ontology\n",
            "exploited here was the go. the relationship between go controlled vocabulary\n",
            "terms can be depicted as a directed acyclic graph . each node corresponds\n",
            "to a term from one of three aspects, for example, \"exodeoxyribonuclease,\"\n",
            "\"mitochondrial derivative\" and \"ethylene mediated signaling pathway\" are\n",
            "exemplars of go terms from the \"molecular function,\" \"cellular component\" and\n",
            "\"biological function\" aspects respectively. the structure of the dag\n",
            "underpinning the go vocabulary defines semantic relationships amongst terms so\n",
            "that, for example, the node for the go term \"intracellular\" is a parent of the\n",
            "node for the more specific go term \"nucleus.\" recall that the topic annotation\n",
            "words for topic k are the  <dig> words from the cgc vocabulary that best\n",
            "characterize the topic. these  <dig> words were mapped to nodes in the go dag. a\n",
            "node where a topic annotation word coincided with a go term was designated an\n",
            "explicit node. go-based topic labels were equated with the go terms for both\n",
            "explicit nodes and the children and grandchildren of explicit nodes.\n",
            "\n",
            "examination of the automatically generated cgc- and go-based labels suggests that\n",
            "lda topics capture meaningful and coherent facets of the molecular, cellular,\n",
            "and behavioral biology of c. elegans. figure 6\n",
            "shows results for four selected cgc topics . the hidden factors\n",
            "permeating the cgc corpus include one pertaining to sexual reproduction , chromosome structure and function , cell death  and\n",
            "locomotion .\n",
            "lda model estimated from a corpus with  <dig>  documents and a vocabulary\n",
            "of  <dig>  words. each panel shows results for a particular topic. the\n",
            "y-axis of the graph is topic-specific word probability\n",
            "and words are arranged along\n",
            "the x-axis according to this likelihood. only the  <dig> topic\n",
            "annotation words are plotted since the remaining words in the vocabulary\n",
            "have negligible probabilities. the words displayed explicitly are\n",
            "unigrams in the cgc vocabulary, including the names of c. elegans\n",
            "genes, and go terms. the position of a word along the\n",
            "x-axis represents its rank; the staggering of words along the\n",
            "y-axis is not significant and is designed only to improve\n",
            "legibility. the graph legend lists two types of automatically-generated\n",
            "topic labels. cgc-based topic labels are a subset of the  <dig> × 500\n",
            "topic annotation words that are unique to a topic and are words from the\n",
            "cgc vocabulary; these labels are ordered according to decreasing\n",
            "βkv values. go-based topic labels\n",
            "are the parents and grandparents go terms of go terms that are also\n",
            "topic annotation words. only go terms that occur four or more times are\n",
            "given and are listed in decreasing frequency . a cgc-based label is\n",
            "unique to a topic whereas a go-based label can be applied to one or more\n",
            "topic.\n",
            "\n",
            "ontology-based topic labels derived from structured knowledge for domains other\n",
            "than molecular and cellular biology are needed to clarify the nature of some cgc\n",
            "hidden factors. figure  <dig> shows topics that have cgg- but\n",
            "no go-based labels. inspection of the cgc-based labels for topics  <dig>   <dig>   <dig>  and\n",
            " <dig> suggest the presence of hidden factors that are concerned with scientific\n",
            "protocols and procedures that are independent of any biological question, and\n",
            "that allude to evolution.\n",
            "factors in the cgc corpus that pertain to the practical aspects of\n",
            "investigating biological mechanisms and processes. topics are\n",
            "represented in the same manner as in figure  <dig> \n",
            "\n",
            "interrogation of lda topics provides insights into genes influencing life\n",
            "span\n",
            "a guilt-by-association approach was devised to identify genes that may be\n",
            "involved in a phenomenon of interest and the procedure illustrated using genes\n",
            "implicated in modifying life span. a \"gene word\" is a word in the  <dig>  word\n",
            "cgc vocabulary that corresponds to the name of a c. elegans gene.\n",
            "cgc-based topic labels are the ≤  <dig> words in the vocabulary that best\n",
            "characterize a topic. if a number of the topic labels are gene words and most of\n",
            "the genes are known to be associated with a specific phenomenon, then the other\n",
            "gene words can be equated with genes likely to be involved in the same\n",
            "phenomenon. one factor influencing the biological insights that can be derived\n",
            "from this approach is the human curation component of the process used to create\n",
            "the cgc bibliography, i.e., the individual who defined the set of genes\n",
            "in the genes record believed to be discussed in the abstract record. in addition\n",
            "to limitations in the data used to estimate a statistical model of text, the lda\n",
            "remains a model based on the simple bag of words representation of a document.\n",
            "while this lda-based approach is not an automated method for formulating\n",
            "sophisticated and detailed hypotheses, it does highlight how a model that\n",
            "ignores syntax and semantics can organize information in a manner that provides\n",
            "a user the ability to exploit their background knowledge and enhance\n",
            "understanding of the subject in hand.\n",
            "life span and designated aging-related gene words. inspection of the two cgc lda\n",
            "topics with the greatest number of cgc-based labels that are aging-related gene\n",
            "words suggests that akt- <dig>  akt- <dig> and ges- <dig> may be associated\n",
            "with aging. figure  <dig> shows these two topics, topic  <dig> and\n",
            "topic  <dig>  in decreasing topic-specific word probability and with genes listed ingene words are age- <dig>  clk- <dig>  mev- <dig>  daf- <dig>  fer- <dig>  clk- <dig>  gro- <dig> \n",
            "daf- <dig>  akt- <dig>  akt- <dig>  clk- <dig>  pdk- <dig>  rad- <dig>  sod- <dig>  old- <dig>  ctl- <dig>  tkr- <dig> \n",
            "daf- <dig>  sod- <dig>  sir- <dig>  daf- <dig>  cln- <dig>  ins- <dig>  age- <dig>  and\n",
            "spe- <dig>  the genes depicted in a normal font have properties similar\n",
            "to known life span modifying genes such as dauer  phenotypes . for example, the wormbase  <cit>  annotations for akt- <dig> and akt- <dig> include \"protein\n",
            "serine/threonine kinase\" and \"inhibition of both akt- <dig> and akt- <dig> leads to\n",
            "dauer-constitutive phenotype.\"\n",
            "words . each topic is represented the same\n",
            "manner as figure  <dig>  in the graph, words in bold are the gerontogenes\n",
            "listed in table  <dig> \n",
            "is taken from the genes database of sageke\n",
            "http://www.sageke.org\n",
            "\n",
            "the mechanisms of action of putative gerontogenes suggested by different topics\n",
            "may not be identical. for topic  <dig>  the cgc-based labels that are gene words are\n",
            "ges- <dig>  osm- <dig>  osm- <dig>  che- <dig>  osm- <dig>  lov- <dig>  pkd- <dig>  daf- <dig>  che- <dig> \n",
            "unc- <dig>  che- <dig>  che- <dig>  osm- <dig>  che- <dig>  che- <dig>  pho- <dig>  dyf- <dig>  che- <dig>  and\n",
            "pkd- <dig>  ges- <dig> is a gut-specific carboxylesterase, a molecular function\n",
            "not ascribed by wormbase to life span modifying genes. since many topic labels\n",
            "are associated with the osm phenotype, osmoregulation may be a feature that\n",
            "differentiates topic  <dig> aging-related genes from those of topic  <dig> \n",
            "\n",
            "exhibition of multiple latent topics by an lda document reflects the\n",
            "complexity of issues discussed in documents\n",
            "by virtue of its superior generalization performance and retrieval ability, an\n",
            "lda model of the cgc corpus is a better statistical model of text than a mixture\n",
            "of unigrams model. a distinct advantage of lda is that although both models are\n",
            "generative, an lda document is the manifestation of many topics whereas a\n",
            "mixture of unigrams document is the product of only one topic. the subject\n",
            "matter of  documents is rarely limited to a single area so the\n",
            "benefit of a cgc lda is that words in a single document could come from, for\n",
            "example, a combination of topic  <dig>  and topic  <dig> . the mixing of lda topics in a cgc item was investigated\n",
            "by examining the document-specific, word-level parameter φ . the variational posterior topic probability\n",
            "φn indicates the extent to which the nth word is associated\n",
            "with the kth topic. a value that is both large and significant is an\n",
            "indicator of the topic most likely to have generated the word.\n",
            "\n",
            "the cgc item shown in figure  <dig> is primarily a mixture of\n",
            "two topics. figure  <dig> shows the topics most likely to have\n",
            "produced words in the document discussing the life span modifying clk-2\n",
            "gene . of the assigned words,  <dig> have\n",
            "posterior probabilities peaked on the aging-related topic  <dig>  and  <dig> on the general purpose topic  <dig> . three words are allocated to topic  <dig>  two to topic  <dig> \n",
            "two to topic  <dig>  and one to topic  <dig> \n",
            " <dig>  a word is identified with the topic k given in parenthesis\n",
            "when the document-specific, variational posterior topic probability\n",
            "exceeds a threshold,\n",
            "φn >  <dig> . as illustrated by \"telomere \", identical words\n",
            "within a document are generated by the same topic. note that only the\n",
            "title, genes and abstract records were concatenated and processed to\n",
            "generate the bag-of-words document used to estimate the lda.\n",
            "\n",
            "utility of lda in formulating hypotheses: insights into clk- <dig> function\n",
            "searching a document database to identify homologs of a query document yields\n",
            "insights that can complement those obtained from sequence-, structure- and\n",
            "function-based analysis of genes and proteins. prior studies of clk-2\n",
            "revealed that it encodes a sequence homolog of tel <dig>  a protein required for\n",
            "normal telomere length regulation in yeast . to enhance knowledge of how clk- <dig> might influence life\n",
            "span, homologs of a document discussing clk- <dig>  were identified by computing the topic-space pairwise similarity\n",
            "score between this query q and every cgc document t . although a given gene may appear in the genes record of many cgc\n",
            "items, the results described below are based on analysis of document homologs of\n",
            "the single cgc item shown in figure  <dig>  figure  <dig> shows the three most related items, cgc documents with\n",
            "the three largest  values. figure  <dig> shows the three topics most associated with them. the third best\n",
            "homolog is relatively uninformative: the text indicates a general review of\n",
            "aging mutants and topic  <dig> labels are general words pertaining to life span.\n",
            "clk- <dig> item is used as the query document, the three items\n",
            "shown have the largest topic-space pairwise similarity scores,\n",
            ". the documents are\n",
            "depicted in the same format as figure  <dig>  as illustrated by \"elegans\"\n",
            "and \"elegans\" in the first and second top-ranked documents,\n",
            "identical words may be attributed to different topics in different\n",
            "documents.\n",
            "the clk- <dig> query shown in figure  <dig> \n",
            "\n",
            "lda-based analysis leads to the hypothesis that clk- <dig> may have a role in\n",
            "coordinating signals between the outside and inside of cells. since the top two\n",
            "topic-space document homologs discuss nuclear receptors, clk- <dig> may have\n",
            "a direct or indirect involvement in receptor biology. topics  <dig> and  <dig> include\n",
            "the go-derived labels \"host cell plasma membrane,\" \"regulation of fgf receptor\n",
            "signaling pathway\" and \"regulation of beta  <dig> integrin biosynthesis.\"\n",
            "circumstantial evidence supports a possible role for clk- <dig> in signal\n",
            "transduction and tissue biology. fgf- <dig> regulates telomerase activity in human\n",
            "endothelial cells  <cit> . integrins are cell surface receptors important in communication\n",
            "between the extracellular environment and the nucleus  <cit> . the suggestion that clk- <dig> may influence telomere length via\n",
            "a mechanism not involving direct physical association with telomeres is\n",
            "plausible since a recent genome-wide screen for saccharomyces cerevisiae\n",
            "deletion mutants that affect telomere length identified genes with very\n",
            "diverse functions   <cit> . recent experimental results support a connection between vacuolar\n",
            "protein-sorting genes and telomere length homeostasis  <cit> .\n",
            "\n",
            "discussion\n",
            "this study demonstrates how a specific statistical ir model, an lda model, can be\n",
            "employed to infer the hidden factors permeating a biomedical text corpus and\n",
            "exploited to synthesize and organize information about complex biological phenomena.\n",
            "the results indicate that despite being estimated from a simple bag of words\n",
            "representation of items in the cgc bibliography, the intra-document statistical\n",
            "structure captured by an lda model is sufficient for the model to be used to enhance\n",
            "understanding of c. elegans biology. for example, analysis of the corpus-,\n",
            "document- and word-level parameters of a trained lda model enabled the exploration\n",
            "and creation of hypotheses about known and putative nematode aging-related\n",
            "genes.\n",
            "\n",
            "the cgc corpus studied here had m =  <dig>   <dig> documents and a v =  <dig> \n",
            " <dig> word vocabulary. estimating a 50-topic lda model from such training data took 3\n",
            "hours on a macintosh powerbook g <dig>  it should be straightforward to estimate a model\n",
            "for larger corpora such as medline where the number of documents is many orders of\n",
            "magnitude greater  and the vocabulary size is only one\n",
            "order of magnitude larger . in estimating an lda, the\n",
            "computational bottleneck is the variational e-step, i.e., computing the\n",
            "posterior topic dirichlet distribution for each document. fortunately, this\n",
            "procedure can be parallelized because given a model, the posterior for each document\n",
            "can be assessed independently. thus, it is feasible for the techniques described in\n",
            "this study to be applied to other corpora and to address questions other than life\n",
            "span modification.\n",
            "\n",
            "from an applications perspective, it is possible to envisage a scenario involving the\n",
            "creation of a library of lda models where each constituent model was estimated from\n",
            "a user- and/or computationally-defined corpus of documents focused on a specific\n",
            "area such as \"aging,\" \"cancer,\" \"yeast biology,\" \"response to stress,\"\n",
            "\"antibiotics,\" \"hiv,\" \"parkinson's disease,\" \"kinases,\" and so on. a query document\n",
            "would be compared to each subject-specific lda in the collection as opposed to a\n",
            "single model as described here. this approach seeks to mirror a common strategy in\n",
            "sequence analysis whereby a query sequence is rated against a library of hidden\n",
            "markov models   <cit>  estimated for domains of interest, for example, the pfam database of\n",
            "protein families  <cit> . since seaching a database of probabilistic models of proteins in order\n",
            "to identify \"remote\" sequence homologs is known to be effective, an analogous\n",
            "approach could prove useful in retrieving distant document homologs.\n",
            "\n",
            "the ideas discussed in this work can be extended and improved in a variety of ways.\n",
            "currently, the number of latent topics in an lda model k is a user-defined\n",
            "parameter but recent research has examined the task of choosing k  <cit> . in particular, if the lda is augmented with a non-parametric bayesian\n",
            "prior known as a hierarchical dirichlet process, both topic probabilities and the\n",
            "number of topics can be estimated from data. under the hierarchical dirichlet\n",
            "process prior, the number of topics grows as data are added to a collection  <cit> . the significance and practical importance of this feature\n",
            "is that when modeling scientific documents, the nature and size of the corpus is\n",
            "evolving constantly meaning that discovering topics is an ongoing task. further\n",
            "studies are required to devise rigorous methods for determining a set of words best\n",
            "able to characterize a topic: the current approach is somewhat arbitary in that\n",
            "corpus-based topic labels were defined as the  <dig> words in cgc vocabulary with the\n",
            "highest values for the likelihood of the word given the topic.\n",
            "\n",
            "in the lda model, topics represent entities that are presumed to permeate a corpus\n",
            "and these latent variables are to be inferred by statistical analysis. these\n",
            "implicit concepts are assumed to be equally related to one another, i.e.,\n",
            "the topics are not organized in any way and form a \"flat structure.\" it seems\n",
            "reasonble to believe that, for example, topics embodying the concepts of \"dna\n",
            "repair\" and \"chromosome structure and function\" should be more related to each other\n",
            "than to a topic focused on \"locomotion.\" thus, a model in which topics were\n",
            "themselves arranged as a hierarchy could prove useful and inspection of the\n",
            "relationship between topics might be informative. the current lda model\n",
            "represents a model for a particular instantiation of a corpus. it might be\n",
            "interesting to formulate dynamic models of corpora which sought to capture how a\n",
            "collection such as the cgc or medline changes over time. in addition to theoretical\n",
            "studies of their properties and behavior, such models might be useful not only for\n",
            "biomedical researchers and clinicians, but also policy makers, historians, and\n",
            "sociologists interested in the evolution of biology and its disciplines.\n",
            "\n",
            "statistical ir models are applicable not only to \"traditional\" document collections,\n",
            "but also to genome-scale biological data. in one example of such an lda model,\n",
            "\"documents\" could correspond to genes, \"topics\" to regulatory networks and \"words\"\n",
            "to motifs. since lda does not require mutually exclusive clustering, a given gene\n",
            "can participate in several networks and a given motif can appear in several genes.\n",
            "merging this type of analysis with that described here would result in evidential\n",
            "support for the functional behavior and role of a gene being derived both from\n",
            "primary biological data and from corpus-based analysis. this capacity to fuse\n",
            "support from disparate sources has been illustrated using a variant of lda known as\n",
            "\"correspondence lda\" and in the context of automated image annotation  <cit> .\n",
            "\n",
            "CONCLUSIONS\n",
            "lda is a special case of the family of probabilistic graphical models. this family\n",
            "includes a wide variety of other models that have proved useful in biology such as\n",
            "hmms, phylogenetic trees, and pedigrees  <cit> . the graphical model formalism allows such graphical components to be\n",
            "combined into heterogenous, large-scale statistical models that integrate evidence\n",
            "from multiple sources. doing so would yield a system for facilitating the\n",
            "formulation of ideas that could be interrogated and verified experimentally.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the results indicate that a specific statistical ir model, an lda model, can be used to infer the hidden factors permeating a biomedical text corpus and that it can be used to synthesize and organize information about complex biological phenomena.\n",
            "in particular, lda-based topic labels were generated from a corpus of  <dig>  documents and a mixture of unigram and random models.\n",
            "in particular, lda-based topic labels were generated from a corpus of  <dig>  documents and a mixture of unigram and random models.\n",
            "in particular, lda-based topic labels were generated from a corpus of  <dig>  documents and a mixture of unigram and random models.\n",
            "the results indicate that despite being estimated from a simple bag of words, lda-based topic labels are sufficient for the model to be used to enhance our understanding ofc. elegans biology.\n",
            "in particular, lda-based topic labels were generated from a corpus of  <dig>  documents and a mixture of unigram and random models.\n",
            "\n",
            "29\n",
            "[{'summary_text': 'in this study we identify  <dig> genes that are differentially expressed in the developing humerus at ts <dig>  early in the process of ossification and joint formation;  <dig> genes associated with development and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with cell adhesion and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with cell adhesion and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with cell adhesion and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with cell adhesion and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with cell adhesion and differentiation; and  <dig> genes associated with cell adhesion and differentiation.\\nthis work is an early step in unravelling the mechanistic basis of biophysical regulation of skeletal development and provides new insights into the biological processes that are affected by mechanical stimuli.'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "mechanical stimulation plays an important role in skeletal growth and repair reviewed in\n",
            " <cit>  and, although much less well studied, it is also required for normal skeletal development. this was initially indicated by observations that infants who experience decreased foetal movement in utero due to neuromuscular disorders present a range of skeletal anomalies including multiple joint fusions, craniofacial abnormalities and thin hypo-mineralised bones\n",
            " <cit> . direct evidence that mechanical stimuli generated by embryonic muscle contractions impacts skeletal development comes from a variety of experimental animal models that show similar abnormalities in ossification and joint formation, for example following muscle immobilisation in chick embryos, and in mouse embryos lacking muscle or with reduced or immobile muscle reviewed in\n",
            " <cit> . however little is known about the molecular mechanisms through which mechanical stimuli influence cellular events during skeletal development. the interplay between biophysical stimuli and gene regulation in differentiating cells is emerging as an important phenomenon in multiple developmental systems\n",
            " <cit> .\n",
            "\n",
            "a number of different strains of mutant mice have been studied that phenotypically lack limb muscle or show reduced stimuli from muscle contraction during development\n",
            " <cit> , including splotch  and splotch-delayed , where muscle precursor cells fail to migrate to the developing limbs and no limb muscle forms\n",
            " <cit> . common defects in muscle-less and immobilised embryos include abnormal initiation and/or progression of ossification\n",
            " <cit> , loss of definition of tissue territories in the joint region\n",
            " <cit>  and altered rudiment morphology\n",
            " <cit> , associated with reduced local cell proliferation\n",
            " <cit> . therefore, mechanical stimuli impact a variety of developmental processes and presumably must influence or integrate with signalling pathways and molecular changes known to guide these events. one clue to a signalling pathway impacted by mechanical stimulation comes from the work of kahn et al. <cit>  who showed that canonical wnt signalling is altered in the elbow joint of splotch-delayed embryos. several regulatory genes have been shown to have dramatically altered expression patterns in reduced mechanical stimuli including, ihh and colx at the site of ossification\n",
            " <cit>  and bmp <dig>  fgf <dig>  and pthlp at the joint line\n",
            " <cit> . whether expression of these genes is directly affected by the mechanical environment or as a more indirect consequence of altered cell behaviour is not known; a genome-wide, open ended screen is required to know more about the spectrum of molecular changes that occur when mechanical stimuli are altered.\n",
            "\n",
            "gene expression profiling to identify genome-wide changes under altered mechanical environments has been carried out on cells in culture using microarray technology, including osteoblast cell-lines subjected to weightlessness or microgravity conditions\n",
            " <cit> , chondrocyte-laden constructs and murine cartilage explants to which dynamic compression was applied\n",
            " <cit>  and chondrocyte cell lines exposed to hydrostatic pressure\n",
            " <cit> . gene expression profiling has the potential to uncover hundreds of genes that respond to mechanical stimuli simultaneously ; however no direct analyses of in vivo changes in gene expression during skeletal development following alteration of the mechanical environment have been performed. this is required to begin to assemble a picture of the molecular landscape impacted by mechanical stimuli in a developmental context.\n",
            "\n",
            "in this study we analysed the transcriptional changes in the developing humerus and associated joints at theiler stage  23\n",
            " <cit>    <dig> ) in muscle-less  compared to phenotypically normal littermate controls. we previously established that the humerus is the most strongly affected rudiment and ts <dig> the earliest time point at which the specific effects on ossification and joint line reduction in the elbow and shoulder regions are detected\n",
            " <cit> . we hypothesise that mechanical stimulation of the embryonic skeletal system impacts expression levels of genes implicated in a variety of regulatory pathways and biological processes, as would be expected when an integrated regulatory system is disturbed. the genes that show altered expression would include direct and indirect targets of mechanical stimulation. therefore, a genome wide analysis of altered transcript levels is required to indicate the principal molecular mechanisms disturbed and the most likely candidates for direct regulation. we have used both rna whole transcriptome sequencing analysis  and microarray technology to allow a comprehensive investigation of the altered transcriptome. microarray analysis is a more established technique\n",
            " <cit> , but rna-seq offers the potential of greater sensitivity\n",
            " <cit>  and analysing the same tissues in parallel allows direct comparison of the two assays and integration of the data sets. we also used rna-seq analysis of the normal developing humerus to explore the transcriptome at this specific stage of development. the humerus developing in the absence of muscle generated stimulation showed both up and down-regulation of gene expression. we reveal alteration of genes encoding components and targets of specific signalling pathways, in particular the wnt signalling pathway. genes associated with cytoskeletal rearrangement and extracellular matrix components are also affected. this analysis has allowed us to profile genome wide transcriptional changes giving an overview of the molecular processes and pathways most affected and identifying a set of putative direct target genes responding to mechanical stimulation during ossification and joint formation.\n",
            "\n",
            "RESULTS\n",
            "transcription profiling of the developing humerus during early ossification and joint formation \n",
            "rna sequencing of control humeri at ts <dig> gives an insight into the transcriptome at this key stage of development when the rudiment is undergoing early stages of ossification and tissue zones in the joint are being defined . transcripts from  <dig>  individual genes  were detected in this tissue at this stage. a minimum of  <dig> transcript reads , with at least one read from each replicate sample, was chosen as the cut off point to reliably indicate reproducible expression across biological replicates\n",
            " <cit> . expressed genes were divided into groups according to their relative expression level .  <dig> genes showed the lowest level of expression represented by between  <dig> and  <dig> read counts, while the majority of genes  showed between  <dig> and  <dig> read counts. only  <dig> genes are in the most highly expressed categories  , representing  <dig> % of expressed genes. expression levels of selected individual genes are represented in figure \n",
            "2b. the most abundantly transcribed gene is col2a <dig>  with a read count of  <dig> , and among the  <dig> genes with read counts of more than  <dig> , there are  <dig> other collagen encoding genes . in total  <dig> collagen subtype genes are expressed .\n",
            "\n",
            "the relative levels of expression of genes associated with signalling pathways involved in regulating skeletal development reviewed in\n",
            " <cit>  are highlighted in figure \n",
            "2b. this shows the potential components that can contribute to these signalling pathways at this stage of skeletal development. for example the hedgehog  pathway is known to play an important role in ossification through the action of ihh , binding to its receptor ptch <dig>  activating smo . similarly,  <dig> fgf ligand encoding genes and  <dig> fgf receptor encoding genes were detected, highlighting the potential for multiple fgf signalling interactions. the bmp signalling pathway genes also reveal potential for multiple signalling interactions with  <dig> bmp encoding genes expressed. bmp <dig> is by far the most highly expressed ; although it is not previously reported in this tissue at this stage in gene expression databases . the relative expression levels of wnt ligand , wnt receptor , extracellular wnt interactor , intracellular wnt pathway component  and wnt pathway transcription factor  encoding genes are represented. the detection of previously unreported wnt gene expression in the humerus  opens up new considerations for functional roles, especially as both genes are up-regulated in muscle-less rudiments . the low density lipoprotein receptor-related genes lrp <dig> and lrp <dig>  which are wnt co-receptors, are most highly expressed among the lrp gene family . interestingly eight fzd receptor encoding genes are detected. other genes known to be involved in skeletal development are highly expressed; sox <dig> , runx <dig> , spp <dig>  and mmp <dig> .\n",
            "\n",
            "identification of differentially expressed genes in muscle-less versus control developing humeri and associated joints\n",
            "microarray analysis of rna extracted from control and pax3\n",
            "spd/spd\n",
            " muscle-less humeri  detected expression of a similar proportion of individual genes on the array;  <dig>  independent genes from the control and  <dig>  from the muscle-less humeri . comparing hybridisation intensity between control and mutant derived cdnas, using cut off points of at least a 2-fold change and corrected p-value ≤ <dig> \n",
            " <cit>  for significance across replicates, identified  <dig> independent genes as differentially expressed . of these,  <dig> genes  were down-regulated and  <dig> genes  were up-regulated . rna-seq analysis  detected  <dig>  independent genes  in muscle-less humeri, compared to  <dig>  in control tissue . to determine differential expression, the same cut off points of a corrected p-value ≤ <dig>  and at least a 2-fold change were applied to the rna-seq data-set, identifying  <dig>  genes as de across replicates. of these,  <dig> genes  were down-regulated and  <dig> genes  were up-regulated in the muscle-less humeri and associated joints compared to that of phenotypically normal littermate control humeri .\n",
            "\n",
            "the venn diagrams in figure \n",
            " <dig> represent overlap of de genes identified by the two platforms. in total  <dig>  independent genes were identified as de in the muscle-less mutant compared to its phenotypic control . of the  <dig> genes identified by microarray ,  <dig> % of these genes were also represented in the rna-seq data. rna-seq detected a greater total number of de genes than the microarray . in addition if the stringency for de gene selection is weakened for the microarray data by moving the cut off point to corrected p-value of ≤ <dig> , the number of genes detected as de in common by the two platforms increases to  <dig> from  <dig> . this suggests that the rna-seq approach was more sensitive in detecting differential expression. grouping genes based on the degree of change showed that the spread of fold change values was similar across platforms although there were differences in the number of genes in different categories . additional file\n",
            "1: table s <dig> and s <dig> list all de genes and fold change values estimated by microarray and rna-seq.\n",
            "\n",
            "verification of fold change values for de genes was performed using qrt-pcr . the de genes chosen  include both down-regulated and up-regulated genes. the direction and degree of fold changes were similar in all cases for the microarray and the rna-seq. for all genes analysed there was a good correspondence across all platforms , although greater fold change differences were detected for the down-regulated gene rxrg and the up-regulated gene vstm2a by qrt-pcr.\n",
            "pax3\n",
            "\n",
            "\n",
            "spd\n",
            "\n",
            "\n",
            "/\n",
            "\n",
            "\n",
            "spd \n",
            "\n",
            "mutant humeri compared to control humeri revealed by microarray, rna seq and qrt-pcr\n",
            "\n",
            "#no numerical value for fold change as no transcripts detected in the mutant.\n",
            "\n",
            "***p ≤  <dig> , **p ≤  <dig> , *p ≤  <dig>  indicates significance of differential expression from qrt-pcr.\n",
            "\n",
            "p ≤  <dig>  for all microarray and rna-seq differences except spp <dig> in the microarray .\n",
            "\n",
            "biological interpretation of differentially expressed genes: down-regulated genes are associated with development and differentiation, cytoskeletal architecture and cell signalling\n",
            "to reveal any enrichment of functionally related genes among the de data sets, two web based tools, david and gostat, were used to analyse gene ontology  term associations\n",
            " <cit>  . a significant enrichment  indicates specific biological processes that are affected when mechanical stimuli are reduced. the strength of the enrichment is indicated by the calculated p-value. independent analysis of data sets from microarray and rna-seq showed the same enriched groups so analysis of the combined de gene sets is presented . the individual go terms found to be enriched have been grouped for the purpose of interpretation as indicated in tables \n",
            " <dig> and\n",
            " <dig>  analysis of the down-regulated gene set  indicated that genes associated with development and differentiation are most highly enriched . categories within this group are involved in signal transduction, including genes that encode signalling molecules, receptors, and transcription factors , for example signalling ligands fgf <dig> fgf <dig>  fgf <dig>  fgf <dig> from the fibroblast growth factor signalling pathway; receptors fzd <dig> and rxrg from the wnt and retinoic acid pathways respectively.  <dig> of these genes encode tfs including; barx <dig>  scx, hes <dig>  pitx <dig>  pitx <dig> and tead <dig>  the down-regulation of such signalling pathway component genes also underlies the enrichment of ontology groups related to cell signalling .\n",
            "\n",
            "1\n",
            "all go terms are associated with the subontology “biological process” except where indicated under cytoskeleton, where cellular component was also used.\n",
            "\n",
            "1p-value of enrichment of go terms using david analysis software; similar results were found with gostat.\n",
            "\n",
            "1\n",
            "1\n",
            "1p-value of enrichment of go terms using david analysis software; similar results were found with gostat.\n",
            "\n",
            " <dig> genes associated with the cytoskeleton were down-regulated in pax3\n",
            "spd/spd\n",
            " humeri, including those encoding microfilament, microtubule and intermediate filament components ;  <dig> are directly associated with microfilaments ,  <dig> with microtubules  and  <dig> with intermediate filaments ; other de genes associated with the cytoskeleton encode proteins that interact with myosin, or the extracellular matrix , including integrin and cadherin encoding genes ).\n",
            "\n",
            "up-regulated genes are associated with cell adhesion, cell signalling and development and differentiation\n",
            "genes up-regulated in muscle-less humeri revealed by microarray and rna-seq were similarly analysed for enrichment of genes associated with particular biological processes or cellular components, using associated go terms . for terms within the subontology biological process, the strongest enrichment was for cell adhesion and ecm associated genes . the genes identified in this category include ecm glycoproteins , ecm structural constituents , cell-adhesion molecules  and calcium-dependent cell adhesion proteins . this grouping in addition included genes involved in signalling pathways which overlap with the next most enriched terms; cell signalling  and cell-cell communication . the signalling pathway components identified in this category include: hedgehog , fibroblast growth factor , transforming growth factor , notch  and wnt signalling  and others, including receptors . other signals identified as up-regulated included c-fos induced gowth factor , hepatocyte growth factor  and insulin-like growth factor . the gene lists in the next most enriched set, development and differentiation, similarly show large overlap due again to the presence of the signalling pathway genes mentioned above, and also transcription factors foxc <dig>  foxo <dig>  lmx1a, lmx1b. z.\n",
            "\n",
            "under the subontology cellular component there was also striking enrichment of extracellular  and membrane associated gene products , including cell adhesion molecules , receptors , cell surface molecules , cadherins , trans-membrane proteins , voltage gated channels  and cell adhesion and extracellular components: alcam, cntn <dig>  epha <dig>  col8a <dig>  col8a <dig>  pappa, pcdh <dig> \n",
            "\n",
            "signalling pathway analysis of differentially expressed genes\n",
            "given the strong enrichment of genes associated with signalling pathways and development and differentiation functions , we sorted de genes according to participation in major developmental regulatory pathways . by far the most strongly impacted cell communication pathway is wnt signalling with  <dig> de genes encoding signalling molecules, receptors, pathway antagonists, known targets or potential targets of the pathway . from the diagrammatic representation of pathway components shown in figure \n",
            " <dig>  it is apparent that the encoded products of de genes act either at the cell surface in wnt signal generation/modulation/interpretation or are targets of the pathway. the genes listed include known targets of the pathway and seven potential target genes , included here due to their similarity to known targets; for example sall1is included because the orthologous gene sall <dig> is a known direct target of the pathway\n",
            " <cit> . in general, genes encoding cell surface components of the pathway are up-regulated, including signalling ligands and agonists , and extracellular antagonists , while down-regulated genes identified are more commonly targets of the pathway . interestingly nine known wnt target genes are up-regulated, including dkk <dig>  rspo <dig>  rspo <dig>  cldn <dig>  grem <dig>  kcnd <dig>  epha <dig> and sfrp <dig>  which encode membrane associated proteins, some of which regulate the wnt pathway .\n",
            "\n",
            "#no numerical value for fold change as no transcripts detected in the mutant.\n",
            "\n",
            "t -target, pt -potential target, r -receptor, co-rec -co-receptor, tf-act -transcription factor activator, l -ligand, ant -antagonist, co-act -co-activator.\n",
            "\n",
            "#no numerical value for fold change as no transcripts detected in the mutant.\n",
            "\n",
            "co-rec -co-receptor, t -target, l -ligand, l-act- ligand activator, r -receptor, ant -antagonist, co-tf- -co-transcription factor, pl –potential ligand.\n",
            "\n",
            "table \n",
            " <dig> lists de genes associated with other developmentally relevant signalling pathways, including the bmp, hedgehog, fibroblast growth factor, hippo and notch signalling pathways.\n",
            "\n",
            "spatial alteration of gene expression patterns\n",
            "while microarray and rna-seq analysis provides data on quantitative changes in gene expression levels across the whole developing rudiment, it does not reveal alterations in the spatial distribution of transcripts or give clues to the specific developmental events affected. we therefore performed in situ hybridisation on control and muscle-less mutant  limb sections at ts <dig> for a selected subset of genes. three of the genes selected encode components of wnt signalling pathways known to be important in skeletal development: cd <dig>  sfrp <dig> and wnt <dig>  spp <dig> encodes an ecm protein osteopontin which is a prominent component of mineralised matrices of bone and teeth\n",
            " <cit> . cd <dig> is a cell-surface glycoprotein involved in cell-cell interactions, cell adhesion and migration. it is a receptor for hyaluronic acid and can interact with other extracellular proteins, such as osteopontin, collagens and matrix metalloprteinases  reviewed in\n",
            " <cit> . it is a target gene of the wnt signalling pathway\n",
            " <cit> . cd <dig> gene expression is down-regulated  <dig>  fold in muscle-less humeri rna . the in situ hybridisation analysis reflects this down regulation dramatically with cd <dig> transcripts hardly detectable in either shoulder or elbow joints of muscle-less mutant embryos, compared to the clear joint line restricted expression seen in controls . sfrp <dig> encodes a secreted protein that acts as a modulator of the wnt signalling pathway, in particular during normal skeletal patterning in developing limbs\n",
            " <cit> . normal expression at ts <dig>  can be detected in the elbow and shoulder joints. the level of up-regulation from microarray and rna-seq analysis is  <dig>   and  <dig>   fold respectively . this up-regulation was reflected in the intensity of expression seen in mutant sections following in situ hybridisation . in addition to the increased level of expression a change in the spatial pattern is also evident. in both the elbow and the shoulder joints expression is expanded, particularly on the ventral aspect of the joint and the staining is unevenly distributed . a similar expression increase was seen in the phalangeal and carpal joints of the handplate, compared to control . wnt <dig> is a signalling ligand of the wnt signalling pathway. expression of this gene has previously indicated its role in joint formation during limb development\n",
            " <cit> . control wnt <dig> expression is seen at a low level in the ventral portion of the elbow joint  at ts23; there is also distinct expression in the epidermis, as previously detected\n",
            " <cit> . in the mutant there is a high level of expression in the elbow joint region; expression is spread across the whole joint line from ventral to dorsal, more extensive on the dorsal side .\n",
            "\n",
            "the spp <dig> gene is normally expressed in the hypertrophic zone and adjacent perichondrium\n",
            " <cit> . despite the appearance of hypertrophic chondrocytes at the mid-diaphysis of immobile spd embryos , no spp <dig> gene expression is detected in these cells. there is weak but detectable staining in the perichondrium at the site of the hypertrophic region, but again apparently lower than in control tissue . expression is also detected in the hypertrophic zone of the ulna  and this is reduced but not absent in the muscle-less mutant ulna , reflecting the reduced phenotypic effect seen in this rudiment\n",
            " <cit> . expression is seen only in the perichondrium of the radius in both the control and mutant .\n",
            "\n",
            "discussion\n",
            "here we describe the spectrum of genes expressed in the developing humerus at ts <dig>  early in the process of ossification and when territories of differentiating cells are being defined in the developing joint region. we also use microarray and rna-sequencing to identify genes that are differentially regulated when mechanical stimulation of the developing skeletal rudiment is altered, giving an insight into the genes that respond to mechanical stimuli generated by muscle contractions. we reveal that the genes altered are highly enriched for genes that regulate development and differentiation, are involved in cytoskeletal rearrangement and components of extracellular matrix including cell adhesion and signalling molecules. components of multiple signalling pathways important during development are affected, in particular  <dig> components of the wnt signalling pathway. although it is clear that appropriate mechanical stimulation from in utero muscle contractions is required for normal development of bone and cartilage, we know very little about the molecular mechanisms that incorporate mechanical cues with classical biochemical signalling pathways. the differentially regulated genes identified here, particularly those associated with signalling pathways and cytoskeletal changes represent a valuable focus for dissecting integrated regulation by biochemical and mechanical signals. these data represent an important resource that can be utilised to understand the molecular basis of mechanoregulation.\n",
            "\n",
            "the transcriptome of a developing skeletal rudiment\n",
            "utilising rna-sequencing technology to reveal the transcriptome in the normal developing humerus and associated joints at ts <dig> provides an insight to the processes that are occurring during this stage of skeletal development when chondrocytes are undergoing hypertrophy, the cartilaginous rudiment is beginning to ossify at the mid diaphysis reviewed in\n",
            " <cit>  and specific zones within the joint are differentiating reviewed in\n",
            " <cit> . this adds a valuable resource to a growing set of data that can be combined to explore skeletal development. previous transcriptome profiling studies have examined cartilage condensation in the tibia-fibula from e <dig>  to e <dig> \n",
            " <cit> , ossification of metatarsals\n",
            " <cit> , in different zones of the growth plate in  <dig> week old mice\n",
            " <cit>  and in runx2-/- mutant mice\n",
            " <cit> . earlier stages of limb development, although not specific to skeletal development, are informed by whole limb bud profiles between e <dig>  and e <dig> \n",
            " <cit> . another important resource that has recently become available examines the spatial expression patterns of  <dig>  coding genes and over  <dig> micrornas within the whole mouse embryo at the same developmental stage examined here\n",
            " <cit> . this digital transcriptome atlas is a powerful tool  that can be used to examine spatial analysis of specific genes, exploring possible functional associations. combining these resources gives information on quantitative and spatial expression of individual genes providing the basis to explore regulatory networks active during the development of skeletal rudiments.\n",
            "\n",
            "several of the findings of the transcriptome analysis are as expected; the most highly expressed genes  include  <dig> collagen encoding genes ; collagens have been shown to be the most abundant structural proteins in cartilage and show characteristic distribution patterns as skeletal rudiments develop\n",
            " <cit> . the insulin-like growth factor genes igf <dig> and igf <dig> and their associated receptors ifg2r and igf1r are also highly expressed ; these are reported to play a prominent regulatory role in skeletal development . similarly, aggrecan  and osteopontin , both involved in skeletal development, are highly expressed . although much is known about the regulatory network that controls early chondrogenesis and joint formation , open-ended whole transcriptome studies are required to add new information. centred on regulatory signalling pathways known to be involved in skeletal development; hh, fgf, tgfβ  and wnt; a complete list of components of these pathways expressed in the humerus as ts <dig> was extracted  indicating the potential role players in each of these pathways at this specific stage. the importance of indian hedgehog  expression in the early prehypertrophic and hypertrophic chondrocytes of cartilage condensations is well established\n",
            " <cit> . the full spectrum of possible interacting molecules in hh signalling  include the receptors ptch <dig>  ptch <dig>  smo and transcription factors gli <dig>  gli <dig>  and gli <dig>  in addition to ihh, desert hedgehog  expression was also detected  and, examining the data presented by cameron et al. confirms that dhh is up-regulated  in e <dig>  fibual and tibual cartilage\n",
            " <cit> . dhh has not previously been functionally linked to skeletogenesis but this opens the possibility of regulatory contribution, perhaps co-operating with ihh. no expression of shh was detected. similarly, novel components of the fgf, tgbβ and wnt pathways were identified .\n",
            "\n",
            "the wnt signalling pathway plays a central role during embryonic development and is known to be an important regulator of bone formation and bone remodelling reviewed in\n",
            " <cit> . it also plays a pivotal role in joint formation and maintenance, shown through gain and loss of function experiments\n",
            " <cit> . the key intracellular mediator of canonical wnt signalling, β-catenin  is the most highly expressed wnt signalling component in the ts <dig> humerus and associated joints. the most highly expressed wnt ligand is wnt5a, previously associated with expression in joints and perichondrium\n",
            " <cit>  and proliferating chondrocytes\n",
            " <cit> . other highly expressed ligands include wnt9a, wnt5b, wnt <dig>  and wnt <dig>  wnt5b and wnt <dig> expression has been shown in the pre-hypertrophic chondrocytes and wnt <dig>  wnt <dig> and wnt <dig> in the developing joints\n",
            " <cit> . high expression of wnt9a could be due to its role in the temporal and spatial regulation of ihh <cit> .\n",
            "\n",
            "numerous extracellular modulators of the pathway were detected; all five secreted frizzled related protein  genes, dickoff   <dig>   <dig> and  <dig> genes and four r-spondins , indicating a huge potential for pathway modulation. the most highly expressed antagonist modulators of the pathway were dkk <dig> and sfrp <dig> both of with are detected in joint cells at e <dig>  and e <dig> \n",
            " <cit> . the most highly expressed r-spondin agonist of the pathway was rspo <dig>  previously detected in phalanges\n",
            " <cit> .\n",
            "\n",
            "identification of mechanoresponsive genes\n",
            "the identification of differentially expressed genes between humeri from control and muscle-less embryonic limbs allows an investigation of the biological processes and the developmental regulatory signalling pathways that are affected by the removal of mechanical stimulation on skeletogenesis in vivo. we previously reported that muscleless  mutants display abnormal ossification in the humerus, altered humeral morphogenesis and altered elbow and shoulder joint formation and these effects were first observed at ts23\n",
            " <cit> . this was chosen as the point of analysis for differential gene expression because, although limb muscles begin to contract from approximately e <dig> \n",
            " <cit>  it is uncertain how much stimulation is transmitted to the skeletal rudiments when the developing tendons are at early stages of morphogenesis\n",
            " <cit> ; the sole indication that the force is functionally transmitted is the mutant phenotype seen at ts <dig>  although analysis at ts <dig> may miss some of the earliest effects, it is relatively early in the response and the earliest time at which it is certain that the system is disturbed. alteration in expression pattern of some selected candidate genes and pathways was previously revealed\n",
            " <cit> , but here we carry out the first genome wide study identifying a total of  <dig>  independent genes as differentially expressed: with approximately 60% down-regulated and 40% up-regulated. the finding of more genes being down-regulated than up-regulated and to a greater extent is consistent with the proposal that mechanical stimuli support the correct differentiation of cells, as observed in the ossification phenotype\n",
            " <cit> , and for the maintenance of tissue patterning, as seen in the developing joint\n",
            " <cit> . go annotation analysis identified specific biological processes that are affected when mechanical stimuli are removed. this type of analysis has been used previously to interpret biological processes associated with developing skeletal tissue\n",
            " <cit> . analysis of the down-regulated de gene set identified genes associated with development and differentiation as the most highly enriched categories, including developmental regulatory signalling pathway molecules and transcription factors. similarly, analysis of up-regulated de gene sets indicated genes associated with cell signalling and development and differentiation.\n",
            "\n",
            "de genes were also highly enriched for genes associated with the cytoskeleton. the cytoskeleton controls cell shape, organelle transport, cell motility and division, and connects the extracellular matrix to internal cell processes reviewed in\n",
            " <cit> . it maintains the mechanical integrity of cells and has been implicated in relaying mechanical signals to downstream biochemical responses\n",
            " <cit> . this was seen in the embryonic lung where cytoskeletal network inhibitors resulted in altered tissue morphogenesis and conversely when cytoskeletal tension was activated lung development was accelerated reviewed in\n",
            " <cit> , indicating the dynamic role the cytoskeleton has in morphogenesis.\n",
            "\n",
            "in chondrocytes the actin microfilaments are predominantly located at the periphery of the cytoplasm\n",
            " <cit> , tubulin microtubules are uniformly distributed throughout the cytoplasm\n",
            " <cit>  as are intermediate filaments, connecting the nuclear membrane with the cell periphery\n",
            " <cit> . in this study  <dig> genes annotated as cytoskeletal were down-regulated when mechanical stimulation was removed. these include  <dig> genes directly associated with actin microfilaments,  <dig> with microtubules and  <dig> with intermediate filaments . the most highly affected group, the filamentous-actin cytoskeleton, has been shown to be involved in articular cartilage chondrocyte mechanotransduction, converting a mechanical stimulus into a biochemical response\n",
            " <cit> . other studies have confirmed the involvement of the actin cytoskeleton in cartilage chondrocyte mechanotransduction via manipulation of the actin accessory proteins\n",
            " <cit> , but there are few reports on the affect of mechanical stimulation on microtubule and intermediate filaments\n",
            " <cit> . among the de genes is an actin-binding protein, cofilin <dig> ; cofilin was previously shown to be increased following cyclic mechanical loading of chondrocytes\n",
            " <cit> .\n",
            "\n",
            "the identification of cytoskeletal genes down-regulated following the removal of mechanical stimulation indicates that the cytoskeleton is affected, but is this because the mechanical integrity of the cell is altered or because mechanotransduction from the ecm is affected, or perhaps a combination of both? the finding that ecm and cell adhesion associated genes are also affected further supports changes in mechanotransduction pathways. the cell-adhesion associated integrins  and cadherins  are down-regulated and these proteins potentially function to physically couple cells to the ecm and play a role in mechanical signal transduction\n",
            " <cit> . articular chondrocytes have been shown to express both integrin\n",
            " <cit>  and non-integrin\n",
            " <cit>  ecm receptors. another actin-associated protein identified to be down-regulated is actinin-α <dig> ; this protein also couples the cytoskeleton to the ecm and may be involved in transducing mechanical stimulation.\n",
            "\n",
            "secreted phosphoprotein  <dig> , previously known as osteopontin , is one of the abundant non-collagenous proteins in bone matrix produced by osteoblasts and osteoclasts reviewed in\n",
            " <cit> . spp <dig> binds to hydroxyapatite and is a potent inhibitor of the mineralisation process, inhibiting the growth of bone matrix crystals\n",
            " <cit> . spp <dig> is expressed early in bone development, however it was concluded not to be required for normal development of bones as null mice  have no apparent effect on the structure or distribution of cells within bone tissue\n",
            " <cit> . however, spp <dig> expression has been shown to be regulated by mechanical stimulation both in vitro and in vivo <cit> . we found spp <dig> to be down-regulated in the developing humerus at ts <dig> in muscle-less embryos and in situ hybridisation showed a dramatic absence of detectable spp <dig> expression in hypertrophic chondrocytes whereas it is still detectable in the perichondrium , indicating a specific effect on expression in hypertrophic chondrocytes and not a delay in the onset of normal expression. it was previously shown that opn-/- mice did not suffer bone loss in response to mechanical unloading\n",
            " <cit> , suggesting that mice lacking spp <dig> could not sense the changes in mechanical stress, thus indicating its potential role in the signal transduction of mechanical stimulation. it has been suggested that mechanotransduction through spp <dig> is dependent on microfilament integrity, as mechanically stimulated increases in spp <dig> expression was blocked by disruption with cytochalasin-d in osteoblasts\n",
            " <cit> . this again highlights the link between an ecm component and the cytoskeleton in a mechanoresponse implicating these components in signal transduction, either directly through the cytoskeleton or through cell adhesion complexes via the cytoskeleton.\n",
            "\n",
            "an example of a non-integrin ecm component that is down-regulated in the absence of mechanical stimulation is cd <dig>  a target gene of the wnt signalling pathway\n",
            " <cit> , encoding a single-pass membrane glycoprotein that binds proteoglycan and hyaluronan  to produce a pericellular matrix surrounding chondrocytes reviewed in\n",
            " <cit> . cd <dig> has been implicated in joint cavitation through interaction with ha\n",
            " <cit> . we previously showed loss of expression of the cd <dig> gene in the interzone of the forming knee joints of immobilised chick embryos\n",
            " <cit> , one of a number of gene expression patterns reflecting a general loss of organisation of differentiating tissue territories, and here we show a similar effect on cd <dig> expression in the elbow and shoulder joint of muscle-less mouse limbs, where we previously showed a similar loss of tissue organisation\n",
            " <cit> . the very restricted expression of cd <dig> in the interzone of forming joints in control embryos at ts <dig> is barely detectable in muscle-less mutants . as well as the gene being sensitive to mechanical stimulation, as an integral part of the ecm and a regulator of joint formation, the gene product may also be an important mediator of mechanical stimuli.\n",
            "\n",
            "the link between cd <dig> and the wnt signalling pathway highlights perhaps the most striking finding of this analysis; the altered expression of  <dig> genes implicated in the wnt signalling pathway . canonical wnt signalling has been shown to be involved in maintaining joint integrity\n",
            " <cit>  and is disturbed in the joints of muscle-less mouse embryos\n",
            " <cit> . wnt signalling might also be involved in co-ordinating ossification and joint development; both processes altered in muscle-less embryos\n",
            " <cit> . non canonical signalling has also been implicated in planar cell polarity during growth plate regulation\n",
            " <cit> . there are also previous indications that the wnt pathway is responsive to mechanical stimulation in mesenchymal stem cells\n",
            " <cit>  in mature bone in vivo <cit>  and in response to injury of articular cartilage\n",
            " <cit> . here, genes encoding four wnt ligands are up-regulated in muscle-less embryos and in the case of wnt <dig> we show specific up-regulation in the elbow and shoulder joint region . two of the up-regulated genes  have not previously been associated with skeletal development. the wnt signalling antagonist sfrp <dig> is also up-regulated specifically in the joint region . the majority of known wnt target genes affected are down-regulated , perhaps due to increased expression of negative regulators sfrp <dig> and dkk <dig> and down-regulation of the fzd <dig> receptor; however ligands and agonists rspo <dig> and rspo <dig> are up-regulated as are some target genes indicating effects at multiple levels of regulation of the pathway. it is interesting that a number of the up-regulated targets feedback as negative regulators of the pathway . it is now important to functionally test the mechanisms linking mechanical stimulation with wnt signalling. this work provides sets of candidate genes to use in functional assays to excavate this important link. understanding how mechanical stimuli influence the wnt signalling pathway would be a major step forward in understanding how mechanical cues work together with classical molecular positional information to guide spatially appropriate tissue differentiation and provide indications of how conditions can be effectively recreated in vitro to guide stem cell differentiation.\n",
            "\n",
            "in situ hybridisation analysis showed altered gene expression of  <dig> wnt pathway genes in the developing shoulder and elbow joints; one down regulated  and two upregulated  . changes in the spatial restriction of sfrp <dig> and wnt <dig> expression were also seen. this does not represent a delay in normal expression in the mutant because the altered pattern is not reminiscent of earlier stages\n",
            " <cit>  and the changes are consistent with altered patterning of the tissue territories in the forming joint and the fusion phenotype seen in the mutant, with cartilage forming across the joint at later stages\n",
            " <cit> .\n",
            "\n",
            "in this study differential expression in developing skeletal rudiments is documented in the absence of limb muscle; this will include genes that respond to lack of mechanical stimulation but perhaps also as a paracrine response to adjacent muscle cells. we know that phenotypic effects on ossification and joint formation are due to the lack of mechanical stimulation rather than physical absence of muscle cells because phenotypic analysis of a range of mouse mutants where muscle is immobile\n",
            " <cit> , reduced\n",
            " <cit>  or absent\n",
            " <cit>  have similar effects and we see similar effects in immobilised chick embryos\n",
            " <cit> . therefore, although some of the genes identified here may respond to lack of adjacent muscle tissue, many must underlie the phenotypic effects seen in response to lack of mechanical stimulation. this is further supported by the overlap of some of the genes identified here and in skeletal cells in culture or adult tissues, in response to mechanical stimulation\n",
            " <cit> .\n",
            "\n",
            "another important limitation in this work is the possibility that a proportion of the down-regulated genes may be due to contamination of the dissected control humeri with adjacent mesenchyme/muscle, since this is being compared to tissue from muscle-less embryos. although care was taken with the dissections, it is impossible to be sure that all muscle tissue was eliminated from the control. the down-regulated gene set also showed enrichment for muscle associated genes , consistent with possible contamination of the dissected control humeri by neighbouring muscle. to inform this we also sequenced the transcriptome of mesenchyme adjacent to the humerus of control embryos at ts <dig> and compared it to the transcriptome of control humeri. we then cross referenced this to the down-regulated gene set in control versus muscle-less humeri , noting any genes enriched more than  <dig> fold in mesenchyme compared to control humeri; these are indicated in column  <dig> of additional file\n",
            "1: table s <dig>  it is possible that these genes are involved in both cartilage and muscle development so no genes have been removed from the data set, however, de genes also showing higher expression in mesenchyme compared to control humeri must be treated with caution with respect to a skeletal specific response to mechanical stimulation. such genes have not been prioritised in any of our subsequent exploration of candidate mechanosensitive genes.\n",
            "\n",
            "the developing humerus at ts <dig> constitutes different cell and tissue populations at different stages of differentiation including the joint region, the perichondrium and the organised zones within the cartilage rudiment. therefore the experimental design employed here will capture genes associated with different cells types at different stages of differentiation. it will now be important to sort out which cells and tissues have altered expression of specific genes. this can be addressed for a sub set of genes by in situ hybridisation, with an initial analysis of  <dig> genes presented in figure \n",
            " <dig>  it can be addressed in a high throughput manner by isolating specific cell populations using laser microdissection from tissue sections , purification of rna and quantitative rt-pcr gene expression profiling, comparing control and mutant tissue from,, for example the hypertrophic, prehypertrophic or the elbow joint region alone.\n",
            "\n",
            "we used both rna-sequencing  and microarray  technologies in parallel to determine differential expression. microarray technology has been utilised to determine expression of chondrogenic and osteogenic genes from developing whole tissues\n",
            " <cit> , and from in vitro differentiation procedures\n",
            " <cit> . the use of rna-seq technology to describe the transcriptome is more recent\n",
            " <cit> . previous direct comparisons between microarray and rna sequencing-based approaches to reveal alterations in gene expression between tissues reported that rna-seq identified more de genes\n",
            " <cit> . we also found that rna-seq is more sensitive in reproducibly detecting alterations in gene expression, detecting more genes altered at lower quantitative levels . this was further emphasised by reducing the stringency of the statistical analysis to p ≤  <dig> , which increased the number of genes detected by microarray specifically . an example of the importance of the increased sensitivity and reproducibility of rna-seq is shown by the spp <dig> gene which did not show statistical significance by microarray but has been verified by qrt-pcr and in situ hybridisation . the larger dynamic range\n",
            " <cit>  and higher reproducibility across replicates\n",
            " <cit>  has also been found in other studies.\n",
            "\n",
            "CONCLUSIONS\n",
            "this study examines the set of genes active at a key stage of skeletal development  and reveals the genes that are differentially regulated in the developing humerus when skeletal muscle is absent. since we previously showed that the lack of muscle contractions leads to common phenotypic defects in both ossification and joint formation in several chick and mouse models, this provides an insight into the genome wide alterations in gene transcription that take place when the mechanical environment is altered. given the importance of appropriate mechanical stimulation generated by embryo movement on skeletal development we postulated that mechanical stimuli must integrate with biochemical cell signalling pathways known to be essential for normal development. we show that multiple signalling pathways are affected, with components of the wnt signalling pathway most strongly disturbed including  <dig> wnt ligands and both down-regulation and up-regulation of target genes. down-regulated genes include cd <dig>  dll <dig> and fgf <dig> which are involved in further cellular interactions during joint formation or feed into other important cell communication events. among the up-regulated wnt targets are several genes that feed back into the wnt pathway itself as antagonists  or agonists . this finding, together with alteration of cytoskeletal components, indicates the biological processes involved in integrating biophysical stimuli during cell differentiation and patterning. understanding the mechanistic basis for how developing cells interpret and respond to biophysical cues is a major challenge, relevant to all developing systems, and will impact our ability to control differentiation of progenitor cells for regenerative therapies. this work is an early step in unravelling the mechanistic basis of biophysical regulation of skeletal development and provides a focus for future studies.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study we identify  <dig> genes that are differentially expressed in the developing humerus at ts <dig>  early in the process of ossification and joint formation;  <dig> genes associated with development and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with cell adhesion and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with cell adhesion and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with cell adhesion and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with cell adhesion and differentiation;  <dig> genes associated with development and differentiation;  <dig> genes associated with cell adhesion and differentiation; and  <dig> genes associated with cell adhesion and differentiation.\n",
            "this work is an early step in unravelling the mechanistic basis of biophysical regulation of skeletal development and provides new insights into the biological processes that are affected by mechanical stimuli.\n",
            "30\n",
            "[{'summary_text': 'the aim of this study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\\nthe aim of the study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\\nthe aim of the study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\\nthe aim of the study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\\nthe aim of the study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "in recent years, the applicability of oral fluid samples for diagnostics of infectious disease like porcine reproductive and respiratory syndrome , caused by the prrs virus  has seen increased discussion in scientific literature. several methods of detecting prrsv rna and prrsv antibodies , using both different molecular diagnostic methods and serological techniques, were developed  <cit> . sampling techniques were evaluated  <cit>  and the effect of the stabilization of the oral fluid  <cit>  and sample processing  <cit>  was determined with the intention to improve the results. different ropes for the oral fluid collection were tested  <cit> . some elisas, specifically developed for prrsv ab detection in oral fluid, show results comparable to serum elisas  <cit> . the usage of cotton ropes as chewing material for oral fluid collection was found to be the method of choice  <cit> . this system is highly suitable for pen-wise oral fluid collection in weaning pigs and fatteners. for individual oral fluid collection, however, especially from sows and boars, the animals have to be trained to chew on the cotton rope  <cit> . this is a time consuming measure and is not widely accepted among european pig producers. on the other hand, the continuous testing of individual animals via oral fluid sampling would be a substantial improvement in the monitoring in prrsv negative herds like boar studs. this presupposes an easy, rapid, animal friendly and efficient sampling method as well as the uncomplicated storage and transport of the samples. self-drying foam swabs like genotubes livestock  that were developed for the detection of minimal dna amounts in forensic medicine have a small sample volume and can be stored at room temperature for several weeks  <cit> .\n",
            "\n",
            "for the collected oral fluid samples, test systems with a high specificity and sensitivity are needed, as they are continuously developed and improved for serum samples  <cit> . a recently developed elisa detecting igg ab against prrsv in individual oral fluid collected with cotton ropes has according to kittawornrat et al.  <cit>  a specificity of 100%  and a sensitivity of  <dig> % . according to the manufacturer of the cited idexx prrs of elisa , the specificity is quoted at  <dig> %  in  <dig> tested individual oral fluid samples whereas the sensitivity is 100%  in  <dig> tested samples. for the idexx prrs x <dig> ab test , which is generally considered to be the de facto gold standard elisa in the detection of prrsv ab in serum, the manufacturer quotes a sensitivity of  <dig> % and a specificity of  <dig> %.\n",
            "\n",
            "the objective of the study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials. furthermore, the specificity of the idexx prrsv of elisa for the detection of prrsv ab in oral fluid collected with these methods was evaluated in comparison to the idexx prrs x <dig> in serum samples. to ensure the sensitivity of the oral fluid elisa, a number of prrsv positive pigs were tested as well.\n",
            "\n",
            "methods\n",
            "animals and serum samples\n",
            "a total of  <dig> pigs  were included in the study. the pigs consisted of  <dig> groups. group  <dig>  included  <dig> boars from four german boar studs,  <dig> boars from one austrian boar stud,  <dig> fatteners from one german pig breeding farm and  <dig> sows and gilts as well as  <dig> nursery piglets from two austrian pig breeding farms. all farms were classified as prrsv negative . group  <dig> included a total of  <dig> samples from the following pigs: a)  <dig> fatteners from one austrian and one german prrsv positive fattening farm, b)  <dig> nursery piglets injected with a prrsv type  <dig> strain at pre-vaccine stage and c)  <dig> fatteners challenged with a highly pathogenic prrsv type  <dig> strain. ten of the pigs mentioned under c) were the same as in b) and used twice for sampling with a time lag of 28 days between both sampling times. a blood sample was taken from each pig. all blood samples, except of the pigs mentioned under b) and c) in group  <dig>  were collected in the course of monitoring programs and not taken for the purpose of this study. housing, animal care and experimental protocol of the pigs mentioned under b) and c) were approved by the local ethics committee . blood samples were centrifuged for 10 minutes at 2400 g within 4 hours after sampling and serum was kept frozen at minus 20°c until analysis.\n",
            "\n",
            "collection and handling of oral fluid samples\n",
            "oral fluid was collected from the above mentioned pigs in two different ways while they were fixated for blood sampling or, in case of the boars, during semen collection:\n",
            "\n",
            " <dig>  individual oral fluid samples were collected via cotton gauze swab. for this purpose, the swab was held into the mouth of the respective pig with a serrefine and the oral fluid was allowed to soak into the swab . the swabs were stored in a 50-ml-falcon tube at minus 20°c until re-collection of the oral fluid and analysis. for the re-collection of the oral fluid, the swab was centrifuged for 10 minutes at 2500 g in a 50-ml-falcon tube with filter .figure  <dig> \n",
            "oral fluid collection via cotton gauze swabs , centrifugation for re-collection of oral fluid .\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " <dig>  individual oral fluid samples were collected via genotubes . the genotubes soaked with oral fluid were stored at room temperature up to four weeks until analysis.figure  <dig> \n",
            "oral fluid collection via genotubes , reconstitution of oral fluid by resuspension  and following centrifugation .\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "group 1: in  <dig> pigs oral fluid could be collected in both described ways. in  <dig> pigs  only cotton gauze swabs were used and in another  <dig> pigs  only genotubes were utilised. group 2: oral fluid samples were collected from  <dig> pigs both via cotton gauze swab and via genotube. in another  <dig> pigs only cotton gauze swabs and in  <dig> pigs  only genotubes were collected.\n",
            "\n",
            "oral fluid collection was done on the same day that the blood samples were taken from the respective pigs.\n",
            "\n",
            "detection of prrsv antibodies by elisa\n",
            "all serum samples were analysed with the idexx prrs x <dig> ab test for the presence of antibodies against prrsv.\n",
            "\n",
            "all oral fluid samples were analysed with the idexx prrs of elisa, designed for detection of antibodies against prrrsv in oral fluid. to test the reproducibility of results, samples of group  <dig> were tested in two different measures. the capacity of the foam swab of the genotube was measured experimentally. for this reason,  <dig> genotubes were dived into oral fluid for some seconds and the amount of fluid soaked into the swab was measured with weighing. the average was at approximate 200 μl with no considerable deviation. to reconstitute the dried oral fluid, the foam swab of the genotube was re-suspended in a  <dig>  ml microcentrifuge tube with 400 μl of the dilution buffer of the elisa kit  which means a 1: <dig> dilution of the contained oral fluid as is required in manufacturer’s instructions. to remove the remaining oral fluid from the foam swab, the genotubes were centrifuged for 10 minutes at 2500 g after removing the safedry medium from the tube . the gained fluid was added into the respective microcentrifuge tube.\n",
            "\n",
            "all serum and oral fluid elisas were conducted according to the manufacturer’s instructions. a brief description of the idexx prrsv of elisa is given in  <cit> . in both elisas, samples with sample-to-positive  ratios ≥ <dig>   were considered positive for prrsv antibodies.\n",
            "\n",
            "statistical analysis\n",
            "the specificity of the idexx prrs of elisa in oral fluid from cotton gauze swabs and genotubes compared to the idexx prrs x <dig> ab test in serum was estimated using group  <dig>  the sensitivity of the idexx prrs of elisa in oral fluid from cotton gauze swabs and genotubes compared to the idexx prrs x <dig> ab test in serum was tested using the samples from group  <dig>  the correlation of s/p values of the elisas were tested in group  <dig> with the correlation coefficient after spearman. over all samples, the accuracy of the idexx prrs of elisa in oral fluid from cotton gauze swabs and genotubes was calculated. in measure one, the agreement of the idexx prrs of elisa in oral fluid from cotton gauze swabs and genotubes with the idexx prrs x <dig> ab test in serum was determined with the kappa coefficient  and interpreted according to landis and koch  <cit> .\n",
            "\n",
            "RESULTS\n",
            "collection of oral fluid samples\n",
            "from all cotton gauze swabs, a sufficient amount of oral fluid  could be collected. the collection from each pig took between 30 seconds and three minutes. the limiting factor for the collection of oral fluid samples via cotton gauze swabs was the dryness of the mouth of the respective pig. this was more often the case in smaller pigs while the mouths of sows and especially breeding boars contained more oral fluid. the collection of oral fluid via cotton gauze swabs from boars during semen collection is possible without any fixation.\n",
            "\n",
            "the collection of oral fluid with genotubes took only a few seconds and went therefore much faster than with cotton gauze swabs. the usage of genotubes in fatteners and adult pigs is mostly possible without fixation of the pig.\n",
            "\n",
            "detection of prrsv antibodies by elisa\n",
            "the serum samples of all group  <dig> pigs tested negative for prrsv antibodies. the calculated specificity of the serum elisa was therefore 100%.\n",
            "\n",
            "the results of the oral fluid elisa from cotton gauze swabs and genotubes compared to the serum elisa are shown in table  <dig>  the s/p values in the prrsv antibody negative oral fluid samples ranged from  <dig>  to  <dig>  both in cotton gauze swabs and in genotubes. s/p values from false positive oral fluid samples from cotton gauze swabs ranged from  <dig>  to  <dig> , those from genotubes ranged from  <dig>  to  <dig> . in cotton gauze swabs as well as in genotubes, respectively, five false positive samples agreed between measure  <dig> and  <dig>  no agreement of false positive samples was found between cotton gauze swabs and genotubes.table  <dig> \n",
            "two-by-two contingency table comparing results of elisa for detection of prrsv antibodies in serum and oral fluid collected via cotton gauze swabs and genotubes\n",
            "\n",
            "\n",
            "\n",
            "idexx prrs of - oral fluid\n",
            "idexx prrs x <dig> ab - serum\n",
            "\n",
            "negative\n",
            "positive\n",
            "\n",
            "cotton gauze swabs\n",
            "\n",
            "genotubes\n",
            "\n",
            "\n",
            "the s/p values of the prrsv antibody elisas in group  <dig> can be seen in figure  <dig>   <dig> of the  <dig> serum samples, all cotton gauze samples and all genotubes were tested positive for prrsv antibodies. the s/p values of the positive samples in serum ranged from  <dig>  to  <dig> , in cotton gauze swabs from  <dig>  to  <dig>  and in genotubes from  <dig>  to  <dig> . the negative serum sample had a s/p value of  <dig> , the s/p value of the corresponding genotube was at  <dig> . there was a positive correlation of s/p values between serum and genotubes  and between cotton gauze swabs and genotubes  in the samples of group  <dig> figure  <dig> \n",
            "prrsv antibodies in serum and oral fluid collected via cotton gauze swab and genotubes in prrsv positive pigs.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "descriptive test parameters and measurements of agreement for all three samplings are shown in table  <dig> table  <dig> \n",
            "descriptive test parameters and measures of agreement of the idexx prrsv of using oral fluid collected via cotton gauze swabs and genotubes\n",
            "\n",
            "\n",
            "cotton gauze swabs\n",
            "genotubes\n",
            "idexx prrs x <dig> ab in serum was used as reference test.\n",
            "\n",
            "\n",
            "\n",
            "discussion\n",
            "in this study, two different ways of individual oral fluid sampling were evaluated for their effectiveness and practicability. using these oral fluid samples, the descriptive test parameters of the idexx prrsv of elisa for detection of prrsv antibodies in oral fluid were calculated in comparison to the idexx prrsv x <dig> elisa in serum samples.\n",
            "\n",
            "until now, besides the usage of genotubes livestock for dna analysis of the tested individuals, only a few studies described the use of genotubes for detection of infectious agents in animals  <cit> . in the mentioned studies, brachyspira dna in rectal swabs of pigs, respectively classical and african swine fever virus dna in wild boars were detected by pcr. one study is published that describes the usage of genotubes for sampling and detection of antibodies against african swine fever virus  <cit> . however, no study referring to the usage of genotubes for prrsv antibody detection by elisa was available until now.\n",
            "\n",
            "both of the sampling techniques used, cotton gauze swabs as well as genotubes, proved to be efficient for oral fluid collection from individual pigs. the collection by both sampling methods was successful in all pigs. individual oral fluid collection by cotton or polyester ropes is not always that successful even in trained pigs   <cit> . the collection via cotton gauze swabs, however, was time consuming and more difficult in smaller pigs than in fatteners or adult pigs. sampling adult boars without fixation is possible for instance during semen collection. it has to be considered, however, that the swab must be taken from within the mouth. collection of frothy saliva around the mouth was proven to be insufficient in other studies  <cit> . the collection of oral fluid with genotubes was easier and less time consuming than with cotton gauze swabs and can be done in larger and adult pigs mostly without the fixation of the animal. the re-collection of oral fluid from genotubes was efficient and can be standardised. the genotube contains a safedry medium that causes a rapid active drying of the sample. the absence of fluid makes the samples very stable. samples collected with a genotube can therefore be stored for several weeks and transported at room temperature  <cit> .\n",
            "\n",
            "according to the kappa coefficient, almost perfect agreement   <cit>  was found between elisa results in serum and oral fluid from cotton gauze swabs and genotubes. the sensitivity of the elisa was 100% in both oral fluid sample species. this number agrees with the sensitivity given by the manufacturer of the elisa for oral fluid collected with cotton ropes. the one serum sample of group  <dig> that was prrsv ab negative in serum had a s/p value slightly beneath the cut-off, whereas the corresponding genotube sample was found clearly positive. it has to be considered, however, that for an accurate analysis of sensitivity a larger number of samples must be analysed. other studies defined the sensitivity of the elisa with  <dig> %, tested in prrsv type  <dig> antibody positive samples exclusively  <cit> , and with  <dig> %  in cotton ropes and  <dig> %  in polyester ropes, tested in prrsv type  <dig> inoculated pigs  <cit> , and were thereby lower than calculated in this study. some false positive results can occur by analysing prrsv antibodies in oral fluid by elisa. the specificity of the elisa in this study is with  <dig> % for cotton gauze swabs and  <dig> % for genotubes within the confidence interval given by the manufacturer but lower than calculated in other studies for oral fluid samples collected with cotton ropes  <cit> . in group  <dig>  a correlation was found between the s/p values of serum samples and oral fluid collected with genotubes. this underlines the good agreement between serum and oral fluid samples found in other studies as well  <cit> . there was a strong correlation between s/p values of prrsv ab positive oral fluid samples collected with genotubes and with cotton gauze swabs, confirming the reproducibility of the results of the oral fluid elisa in samples collected with different sampling methods. the sampling techniques used in this study are therefore equally suitable for oral fluid collection and subsequent testing with the idexx prrs of elisa as cotton ropes.\n",
            "\n",
            "CONCLUSIONS\n",
            "this study shows that oral fluid samples can be used for the prrsv antibody detection on individual pig level. the use of genotubes proved to be an especially practical method both for oral fluid collection and for storage and transport of the samples. the elisa detecting prrsv ab in oral fluid collected by cotton gauze swabs and genotubes proved to be highly sensitive. it has to be considered, however, that false positive samples needing re-testing may occur. this can cause irritations, especially in prrsv negative farms like boar studs.\n",
            "\n",
            "competing interests\n",
            "\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "authors’ contributions\n",
            "\n",
            "ts: study coordination and design, performance of elisas, statistical analysis, drafting of the manuscript. ew: acquisition of data, participating in study design fs: conceived of the study, study design. all authors read and approved the final manuscript.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the aim of this study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\n",
            "the aim of the study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\n",
            "the aim of the study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\n",
            "the aim of the study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\n",
            "the aim of the study was to test the efficacy and practicability of oral fluid collection from individual pigs via cotton gauze swabs and a dry foam swab  as well as the re-collection of oral fluid from these materials.\n",
            "\n",
            "31\n",
            "[{'summary_text': 'in this study, a biomass-degrading enzyme system of the soft-rot fungus c. globosum bcc <dig> was characterized.\\nthe objective of this study was to identify enzymes with synergistic activities with cellulases and hemicellulose degrading enzymes in order to maximize the hydrolysis efficiency on lignocellulose and other lignocellulose degrading enzymes.\\nthe goal of this study was to identify enzymes with synergistic activities with cellulases and hemicellulose degrading enzymes in order to maximize the hydrolysis efficiency on lignocellulose and other lignocellulose degrading enzymes.\\nthe objective of this study was to identify enzymes with synergistic activities with cellulases and hemicellulose degrading enzymes in order to maximize the hydrolysis efficiency on specific lignocellulose and lignocellulose degrading enzymes.\\nthe objective of this study was to identify enzymes with synergistic activities with cellulases and hemicellulose degrading enzymes in order to maximize the hydrolysis efficiency on lignocellulose and lignocellulose degrading enzymes.\\nthe objective of this study was to identify enzymes with synergistic activities with cellul'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the shift from petroleum-based industry to a greener bio-based platform is expedited by an increasing concern of global warming. lignocellulosic plant biomass has attracted attention as a renewable resource for production of biofuels and commodity chemicals in biorefinery. lignocellulosic materials consist mainly of three different types of biopolymers:  cellulose, a linear homopolymer of d-glucose organized into highly crystalline microfibers which are intimately associated with an intricate network of  hemicellulose, an amorphous branched polymer comprising various pentoses, hexoses, and sugar acids, and  lignin, a heteropolymer of phenolic alcohols which shields the polysaccharide microstructure from external physical, chemical, and biological attacks. these biopolymers are organized into a complex and highly recalcitrant lignocellulosic structure  <cit> .\n",
            "\n",
            "in nature, lignocelluloses are degraded by the cooperation of various microorganisms, capable of producing an array of cellulolytic, hemicellulolytic, and ligninoloytic enzymes  <cit> . cellulases and hemicellulases are mainly composed of various hydrolytic enzymes in different glycosyl hydrolase  families acting synergistically. cellulases comprise three major groups of enzymes:  endoglucanases , which attack regions of low crystallinity in cellulose fibers, creating free chain-ends;  exo-glucanases or cellobiohydrolases  which further degrade the molecule by cleaving cellobiose from the free-chain ends; and  β-glucosidases  which hydrolyze cellobiose to produce glucose  <cit> . in addition to the three major groups of cellulases, there are a number of endo- and exo-acting enzymes that attack the heterogeneous hemicelluloses, such as endo-β- <dig> -xylanase, β-xylosidase, galactomannanase, glucomannanase, and acetylesterase. a challenge for development of a feasible biomass industry is identifying efficient lignocellulolytic microbes and developing an active enzyme system based on synergism of the major glycosyl hydrolases and the non-hydrolytic auxiliary components   <cit> . these molecular systems are required for efficient hydrolysis of lignocelluloses to sugars, the key intermediates for subsequent conversion to biorefinery products.\n",
            "\n",
            "aerobic fungi belonging to phylum ascomycota are important plant biomass degraders capable of secreting an array of glycosyl hydrolases and non-hydrolytic auxiliary components. several ascomycetes are commonly used for producing cellulases and hemicellulases in industry, e.g. trichoderma, aspergillus, and talaromyces  <cit> . enzymes from these fungi are different in their composite activities and enzyme components, which have different catalytic characteristics on lignocellulose decomposition. chaetomium is a saprophytic fungus belonging to ascomycota with high capability on degrading plant materials  <cit> . most studies on chaetomium, particularly on c. globosum have focused on laccases  while there have been few studies on their cellulolytic enzyme systems  <cit> . in this study, a biomass degrading enzyme system of the soft-rot fungus c. globosum bcc <dig> was studied. the crude enzyme was characterized for its catalytic activities and its components identified by proteomics. the crude enzyme was used for formulation of an efficient biomass-degrading enzyme mixture by a mixture design approach and applied for hydrolysis of alkaline-pretreated rice straw. the work provides an alternative lignocellulose-degrading enzyme system potent for on-site enzyme production for the development of a feasible biorefinery industry.\n",
            "\n",
            "methods\n",
            "strain and media\n",
            "rice straw was obtained from a paddy field in pathumthani province, thailand according to the required national guideline. it was physically processed by a cutting mill  and then sieved to particles smaller than  <dig>  mm. the biomass was delignified with 10%  naoh at  <dig> °c for 90 min at the solid/liquid ratio of 1: <dig>  washed with water until neutral ph was obtained, and dried at  <dig> °c for overnight before use as the substrate for enzymatic hydrolysis. the alkaline-pretreated rice straw contained  <dig> % cellulose,  <dig> % hemicellulose,  <dig> % lignin and  <dig> % ash according to the standard nrel analysis method  <cit> . empty palm fruit bunch  was obtained from kasetsart university, thailand and physically processed by a chopping machine  and subsequently on a cutting mill and sieved through a  <dig>  mm mesh. c. globosum bcc <dig> was identified and obtained from the biotec culture collection, thailand  and maintained on potato dextrose agar . polysaccharides used as substrates in enzymatic activity analysis were obtained from sigma-aldrich. accellerase®xy  and novozyme® <dig>  were obtained from dupont  and sigma-aldrich, respectively.\n",
            "\n",
            "optimization of cellulase production conditions\n",
            "\n",
            "c. globosum bcc <dig> was cultivated by submerged fermentation in 250-ml conical flasks. the inoculum was prepared from the culture grown on pda by plunging four agar pieces covered with profuse mycelia using a cock borer no.  <dig> and inoculated into 50 ml of the production medium  of microcrystalline cellulose avicel® and 1%  of soybean meal in water). the culture was incubated at  <dig> °c for  <dig> d with shaking at 200 rpm. culture media and conditions were varied as specified including concentration of avicel® ), concentration of lactose ) and ph . effect on addition of 1%  empty palm fruit bunch  as a co-carbon source for induction of cellulase was also studied. the cultures were collected periodically for determination of cellulase activity using the dinitrosalicylic acid  method  <cit> . the experiments were performed in triplicate and the average of cellulase activity was used as the response . the data were analyzed using spss  <dig>  .\n",
            "\n",
            "enzyme production and purification\n",
            "a 10%  inoculum grown in the optimal production medium containing 2% avicel®, 1% epfb, 1% soybean meal ,  <dig> % lactose, and initial ph of  <dig>  at  <dig> °c for  <dig> d was inoculated into a 5 l bioreactor  with a 3 l working volume of the same medium. the culture was incubated at  <dig> °c for  <dig> d with constant mixing at 200 rpm and oxygen feed of  <dig> vvm. the fungal mycelia were separated by filtration on gauze and clarified by centrifugation at  <dig>  × g for 10 min. the supernatant was then filtered through a  <dig>  μm supor®- <dig> membrane  followed by concentration  using a minimate™ tangential flow filtration  system equipped with a 10-kda mwco tff membrane . the enzyme was kept at  <dig> °c and used in subsequent experiments.\n",
            "\n",
            "enzyme activity assays\n",
            "polysaccharide-degrading enzyme activities were analyzed using the  <dig> -dinitrosalicylic acid  method by measuring the amount of reducing sugars liberated  <cit>  according to the standard procedure recommended by the commission on biotechnology, iupac  <cit>  with modifications on the total reaction volume. reactions of  <dig>  ml contained 100 mm sodium acetate phosphate buffers, ph  <dig>  with an appropriate dilution of the enzyme using a  <dig> × 6 cm whatman no.  <dig> filter paper as the substrate and incubated at  <dig> °c for 60 min for determining the filter paper activity  as filter paper unit . the carboxymethyl cellulase , xylanase, mannanase, amylase, and pectinase activities were assayed using 1%  carboxymethyl cellulose, 1%  birchwood xylan,  <dig> %  locust bean gum, 1%  soluble starch, and  <dig> %  pectin from citrus peels as the substrates, respectively. the reactions were incubated at  <dig> °c for 30 min. the amount of reducing sugars was determined at the end of the reaction by measuring the absorbance at 540 nm using a uv-vis spectrophotometer microplate reader  and interpolation from a standard curve prepared using dilutions of the corresponding sugar as standards. one enzyme activity unit  is defined as the amount of enzyme required to release 1 μmol of reducing sugars from a substrate in 1 min under the assay condition. the β-glucosidase and β-xylosidase activities were assayed using  <dig> %  p-nitrophenyl-β-d-glucopyranoside  and p-nitrophenyl-β-d-xylopyranoside  as the substrates, respectively in 3 ml reactions containing an appropriate amount of enzyme in 100 mm sodium acetate buffer . the reactions were incubated at  <dig> °c for 30 min and terminated by the addition of 2 ml of 1 m na2co <dig>  the quantity of p-nitrophenolate was measured spectrophotometrically at 405 nm at the end of the reaction. one activity unit  is defined as amount of enzyme that produces 1 μmol p-nitrophenolate per minute under the experimental conditions. the total protein concentration of the crude enzyme extracts was determined using bradford’s method with the biorad’s protein assay reagent  using bovine serum albumin  as the standard. the experiments were performed in triplicate.\n",
            "\n",
            "proteomic analysis\n",
            "the enzyme preparation was applied to a 10% sds-page gel and separated using a miniprotean ii cell . the protein bands were visualized by staining with coomassie blue r- <dig> and were manually excised into five fractions according to their apparent molecular weights . the polypeptides in gel were then digested with trypsin . the tryptic peptides were resuspended with  <dig> % formic acid and analyzed on a finigan ltq linear ion trap mass spectrometer  according to a method described in wongwilaiwalin et al.,  <cit> . all ms/ms spectra were searched using the mascot® search engine  against the ncbi-nr database following criteria: enzyme trypsin, static modification of cys , with differential modification of met . the search results were filtered by cross-correlation versus charge state  and protein probability . the candidate protein queries were mapped to the uniprot knowledge base  <cit> .\n",
            "\n",
            "synergistic action of c. globosum enzyme to commercial enzymes\n",
            "interactions among the c. globosum enzyme , novozyme® <dig> and accellerase®xy were studied using experimental mixture design approach  <cit>  with a fixed total enzyme volume  <cit> . the enzymatic hydrolysis reactions of 1 ml total volume contained 5%  alkaline-pretreated biomass in 100 mm sodium acetate buffer  supplemented with 1 mm sodium azide and incubated at  <dig> °c for 48 h with rotary shaking at 200 rpm. an optimal enzyme mixture releasing the highest reducing sugar yield was defined by a { <dig> }-augmented simplex lattice design using minitab  <dig>  software . the design contained  <dig> experimental points , which were performed in quadruplicate with three components and a lattice degree of three. the three independent variables in the mixture design consisted of cg-cel , novozyme® <dig> , and accellerase®xy . the sum of all enzyme components in the reactions were 100% with the total enzyme volume fixed at 80 μl. the amount of released reducing sugar  was used as dependent variables for simulation of the respondent model equation. the amount of reducing sugars liberated at the end of the reactions was determined using the dns method at the end of the reaction  <cit> . the sugar profile in the hydrolyzates was analyzed by high performance liquid chromatography  equipped with a differential refractometer using an aminex hpx-87h column . the column temperature was  <dig> °c. h2so <dig> solution  was used as the mobile phase at a flow rate of  <dig>  ml/min. concentration of sugar was determined from the calibration curve of standard solution. the experiments were performed in quadruplicate. sugars from control reactions containing heat-inactivated enzymes were subtracted from the data.\n",
            "\n",
            "mixture design analysis\n",
            "after regression analysis, the full cubic model was used to simulate the optimized ratio of the mixture components. the canonical correlation of the full cubic model is shown in eq. :  <dig> y=∑i=13βixi+∑∑i<j3βijxixj+∑∑i<j3δijxixjxi−xj+∑∑∑i<j<k3βijkxixjxk where y is a predicted response, βi is a linear coefficient, βij is a quadratic coefficient, and βijk is a cubic coefficient. βij is a parameter of the model. βixi represents the linear blending portion, and the parameter βij represents either synergistic or antagonistic blending.\n",
            "\n",
            "RESULTS\n",
            "optimization of enzyme production conditions\n",
            "production of cellulolytic enzyme from c. globosum bcc <dig> by submerged fermentation was studied. the optimization study included three variables:  concentration of avicel®,  concentration of lactose, and  initial ph previously identified to show significant effects  on cellulase level of the fungus in the preliminary screening of factors influencing enzyme production. according to table  <dig>  the basic fermentation condition containing avicel® as the sole carbon source  resulted in the cmcase and fpase activity of  <dig>  and  <dig>  u/ml, respectively. further increases in avicel®  did not enhance the target enzymatic activities. addition of lactose  was found to significantly  induce cellulase production level when compared with the same conditions in the absence of lactose. higher cellulase productivity was found at acidic ph  compared with that observed under neutral conditions. the highest cmcase and fpase activities of  <dig>  u/ml and  <dig>  fpu/ml, respectively, were recorded after 144 h of fermentation in run no.  <dig> containing 1% lactose and an inducer with controlled ph at  <dig> . this up-regulation of cellulase by lactose was previously reported in c. papyrosolvens  <cit>  and a. cellulolyticus  <cit>  where lactose was shown to act as an effective inducer of cellulase biosynthesis, which is controlled by a regulator protein responsive to the concentration of the target substrate  <cit> . the preference for acidic ph for production of cellulase and other lignocellulose degrading enzymes has been reported for most cellulase producing ascomycetes fungi e.g. t. reesei  <cit> , a. cellulolyticus  <cit> , and c. globosum  <cit> . this preference is related to ph for optimal growth and catalytic activity of the enzymes.table  <dig> effects of variables to cellulase activities in fermentation of bcc5776\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "addition of epfb as a co-substrate at 1%  concentration under submerged fermentation was found to further improve the cellulase production by c. globosum bcc <dig>  this led to the enhancement of endo-glucanase  and total cellulase  activities to  <dig>  u/ml and  <dig>  fpu/ml, equivalent to  <dig> and 25% increases, respectively, compared with those obtained in its absence . the use of agro-industrial wastes as the sole carbon source or in combination with pure cellulose as an effective and cost-efficient substrate for production of plant biomass degrading enzymes e.g. cellulase, hemicellulase, and pectinase production has been demonstrated in many fungal strains . the cellulase activities produced by c. globosum in this study were in the same range  compared to several wild type fungi in genera aspergillus, acremonium, trichodema, and schizophyllum reported in many recent publications, which were in the range of  <dig> – <dig>  fpu/ml. the very high cellulase activities  were achieved by mutants or genetically modified fungal strains. comparison of cellulase activities produced by different fungal strains recently reported is shown in the additional file 1: table s <dig> fig.  <dig> comparison of  cmcase and  fpase activities from c. globosum bcc <dig> under various conditions. the cultures were incubated at  <dig> °c for 144 h with continuous shaking at 200 rpm. control medium  of microcrystalline cellulose avicel® and 1%  of soybean meal in water); optimized medium  microcrystalline cellulose avicel®,  <dig> %  lactose, and 1%  soybean meal with 50 mm potassium phosphate, ph  <dig> ); optimized medium + 1%  epfb\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "the optimized enzyme production medium  was used for production of the crude enzyme  in a laboratory-scale fermenter. the enzyme showed similar composite activities to those obtained in the small scale experiments. the crude enzyme preparation  contained a relatively high level of endo-glucanase as shown by cmcase activity  with fpase activity . strong endo-β- <dig> -xylanase  and endo-β- <dig> -mannanase  activities were found together with other enzymes involved in plant polysaccharide degradation, such as pectinase  and amylase . however, regardless of the high activities of endo-acting glycosyl hydrolases, the crude enzyme exhibited substantially weak downstream activities i.e. β-glucosidase  and β-xylosidase  activities, which could limit degradation of cellulose and hemicelluloses to sugars. the 5× concentrated enzyme contained the cellulase activity of  <dig>  fpu/ml while >90% activities of other enzymes were retained.table  <dig> the composite enzyme activity profiles of c. globosum bcc <dig> enzyme extracts\n",
            "\n",
            "\n",
            "aprotein concentration =  <dig>  mg/ml\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "synergistic action of cg-cel and commercial enzymes\n",
            "the mixture design approach was applied to study synergistic and cooperative interactions among cg-cel and a hemicellulase, accellerase®xy, and a β-glucosidase, novozyme® <dig>  chosen from our pre-screening study on identification of additive enzymes with complementing activities to the core cellulase . the measured response of this method was presumed to depend only on the enzyme proportion of the complements in the mixture. a { <dig> }-simplex lattice model was applied which comprised  <dig> experimental points located inside the triangular graph, in which the sum of the three component loading for every experimental point was always 100% based on a volumetric basis. according to this design, the cellulase dosages as fpu loading/g substrate of each experimental point was varied according to different proportions of cg-cel: novozyme®188:accellerase®xy. each point was operated in quadruplicate to minimize the effect of experimental error.fig.  <dig> comparative activity profiles of  cg-cel and the synergized enzymes  accellerase®xy and  novozyme® <dig>  the y-axis is in a logarithm base  <dig> scale \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "the reducing sugar yield obtained from each experimental condition in the mixture design is summarized in table  <dig>  the standard deviation of all the data points was ≤ 5% of the mean. cg-cel, equivalent to the fpase activity of  <dig>  fpu/g , demonstrated a higher reducing sugar yield  than those obtained with the β-glucosidase novozyme® <dig>  and the hemicellulase accellerase®xy  alone. the binary combinations between cg-cel and accellerase®xy  led to slight increases in reducing sugar yields  compared with that of cg-cel alone, despite the lower total fpase dosage in the reaction . ternary mixtures of all three enzymes at specific compositions resulted in further increases in the total reducing sugar yields, with the maximal reducing sugar yield of  <dig>  mg/g  which contained a 4:1: <dig> ratio of bcc5776: novozyme®188: accellerase®xy, suggesting synergism of the three enzymes.table  <dig> design of experiment for enzymatic hydrolysis of pretreated rice straw and the associated response data\n",
            "\n",
            "reactions  contained 5%  alkaline-pretreated rice straw in 100 mm sodium acetate buffer, ph  <dig>  with a total enzyme volume of 80 μl and incubated at  <dig> °c for 48 h\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "the response data for the reducing sugar were then analyzed using multiple regression analysis from the linear to full cubic model. the full cubic model was found to be the best fitted model for the reducing sugar . the anova analysis of the full cubic model is illustrated in table  <dig>  for a single factor, cg-cel, novozyme® <dig>  accellerase®xy coefficients showed a positive correlation with the reducing sugar yield. marked synergisms between cg-cel and novozyme® <dig> or accellerase®xy were observed , with a weak synergistic interaction between novozyme® <dig> and accellerase®xy. the highest coefficient was observed for the cg-cel*novozyme®188*accellerase®xy, indicating strong interactions among these components. the fitted equation for the reducing sugar yield based on the significant terms is shown in eq. :table  <dig> the regression model analysis of the { <dig> } full cubic model\n",
            "\n",
            "\n",
            "  <dig> reducingsugarmg/g= <dig> *bcc5776+ <dig> *novozyme®188+ <dig> *accellerase®xy+ <dig> *bcc5776*novozyme®188+ <dig> *bcc5776*accellerase®xy+ <dig> *novozyme®188*accellerase®xy+ <dig> *bcc5776*novozyme®188*accellerase®xy−− <dig> *bcc5776*novozyme®188−− <dig> *bcc5776*accellerase®xy− \n",
            "\n",
            "\n",
            "the responses of reducing sugar yield with respect to component combinations are represented by a ternary mixture contour plot . the area that emphasized the greatest reducing sugar was in the middle of the cg-cel and accellerase®xy axes, and near the bottom of the novozyme® <dig> vertex. this implies that a high level of reducing sugars could be achieved when cg-cel was the major component with novozyme® <dig> and accellerase®xy in the minority. the optimal enzyme combination based on the maximal reducing sugar yield was determined to be  <dig> % cg-cel,  <dig> % novozyme® <dig>  and  <dig> % accellerase®xy with a predicted reducing sugar yield of  <dig>  mg/g pretreated rice straw with the total cellulase activity of  <dig>  fpu/g. an experimental reducing experimental sugar yield of  <dig>  mg/g was obtained, validating the model. the ternary enzyme mixture was found to increase the yields of all major composite sugars  from hydrolysis of the pretreated rice straw. the glucose yield  obtained using the optimal enzyme mixture was  <dig>  mg/g, which was higher than that obtained using the individual enzymes . this led to the increase in glucose releasing efficiency from  <dig>  mg glucose/fpu from cg-cel alone by  <dig>  fold to  <dig>  mg glucose/fpu of the ternary optimal enzyme mixture.   <cit> . further increases in the total enzyme loading from 1×  to 4×  led to a stepwise increase in reducing sugar yield to  <dig>  mg/g and glucose released to  <dig>  mg/g; however, respective decreases in glucose releasing efficiency were observed at higher enzyme loadings .fig.  <dig> the contour plot of the experimental design optimization of the ternary enzyme complex. one hundred percent component amount is equal to 80 μl total reaction volume\n",
            "\n",
            "\n",
            "fig.  <dig> saccharification of pretreated rice straw using the optimal enzyme mixture compared with individual enzymes. the reactions  contained 5%  alkaline-pretreated rice straw in 100 mm sodium acetate buffer, ph  <dig>  with a fixed total enzyme volume of 80 μl  and incubated at  <dig> °c for 48 h.  sugar profile of optimal enzyme mixture;  the total reducing sugars  and individual sugar profile  glucose;  xylose;  arabinose) versus fpu loading/g on pretreated rice straw hydrolysis. *1× enzyme loading equaled to  <dig>  fpu/g\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "proteomic analysis of bcc <dig> crude enzyme\n",
            "in order to analyze the composite glycosyl hydrolases and accessory non-hydrolytic enzymes with functions on attacking lignocellulosic materials in cg-cel, the composite proteins in the secretome of bcc <dig>  was analyzed using lc/ms/ms. twenty seven different proteins were identified in cg-cel, from which 81% were annotated as functional proteins while the rest were classified as hypothetical proteins. the majority of them are hydrolytic enzymes attacking cellulose and hemicelluloses, with a substantial fraction of non-cellulosic polysaccharide hydrolyzing enzymes . most of them are closely related to homologous enzymes in neofusicoccum parvum ucrnp <dig> and other ascomycetes. the cellulose degrading enzymes were classified into various glycosyl hydrolase families and non-hydrolytic enzymes in different classes. the cellulolytic components are annotated as  <dig> endo-glucanases  and  <dig> cellobiohydrolases, including cbh-i , which attacks the reducing end of cellulose chain, and cbh-ii, which attacks the non-reducing end, releasing cellobiose units, in addition to one carbohydrate binding module .table  <dig> identification of the polysaccharide degrading enzymes in the secretome of chaetomium globosum bcc5776\n",
            "\n",
            "uncultured fungus\n",
            "uncultured fungus\n",
            "fusariam fujikuroi\n",
            "uncultured fungus\n",
            "neofusicocum parvum\n",
            "collectotrichum graminicola\n",
            "irpex lacteus\n",
            "macrophomina phaseolina\n",
            "collectotrichum graminicola\n",
            "theilavia terrestris\n",
            "volvariella volvacea\n",
            "uncultured fungus\n",
            "heterobasidion irregulare\n",
            "neofusicocum parvum\n",
            "macrophomina phaseolina\n",
            "macrophomina phaseolina\n",
            "aspergillus oryzae\n",
            "macrophomina phaseolina\n",
            "neofusicocum parvum\n",
            "neofusicocum parvum\n",
            "neofusicocum parvum\n",
            "phaeoacremonium minimum\n",
            "neofusicocum parvum\n",
            "neofusicocum parvum\n",
            "neofusicocum parvum\n",
            "macrophomina phaseolina\n",
            "phialophora sp.\n",
            "\n",
            "\n",
            "\n",
            "a few hemicellulolytic enzymes were also identified in the bcc <dig> secretome. these included gh <dig> and  <dig> endo-β- <dig> -xylanase, a key endo-acting hemicellulase attacking xylan, the major component in hemicellulose in addition to exo-acting enzymes including a β-galactosidase  and α-mannosidase . non-hydrolytic counterparts included a cellobiose dehydrogenase acting on oxidizing cellodextrins, an intermediate in cellulose hydrolysis to their corresponding lactones, and a polysaccharide deacetylase functioning on acetyl group branches in hemicelluloses. the functions of both enzymes on synergistic and cooperative actions with core cellulases have been demonstrated  <cit> . other hydrolases in the secretome are mainly related to different classes of proteases, e.g. carboxypeptidase s <dig>  leucyl aminopeptidase, and tripeptidyl-peptidase  <dig>  no other non-cellulosic polysaccharide degrading enzymes e.g. amylases or pectinases were identified. the proteomic profiles of the secretome thus point to a lack of downstream cellulose degrading enzyme  and various hemicellulolytic components. the lack of these activities could explain why cg-cel is strongly synergistic with supplemented pure β-glucosidase and hemicellulases.\n",
            "\n",
            "synergism of core cellulases with hemicellulases and auxiliary components in various classes of carbohydrate processing enzymes according to the carbohydrate active enzymes database   <cit>  has been demonstrated and could be applied for developing efficient enzyme systems for bioindustries. their synergistic interactions can be explained by several mechanisms . activities of upstream enzymes can be enhanced by cooperative action with downstream enzymes acting on degrading smaller substrates, for example, alleviation of cellobiohydrolase  inhibition by a β-d-glucosidase which cleaves cellobiose, the released product with an inhibitory effect to cbhs to glucose molecules  <cit> . synergism can also be resulted from the endo/exo effect between an endoglucanase which creates new, free cellodextrin chain ends for an exo-acting cbh  <cit>  as well as synergic action between different exoglucanases attacking the reducing and non-reducing ends of cellulose chains  <cit> . cooperative action of cellulases and hemicellulases can also increases accessibility to the target substrates of each other, providing synergisms between glycosyl hydrolases attacking different biopolymers in the plant cell wall  <cit> . physical modification of the substrates e.g. loosening of the crystalline region of cellulose by auxiliary proteins   <cit>  and non-hydrolytic enzymes   <cit>  which in overall, results in increased accessibility of the hydrolytic enzymes to the substrates has also been demonstrated. however, they were not annotated in the secretome of c. globosum in this study.\n",
            "\n",
            "synergistic enzyme systems are developed by empirically determining combinations of different enzymes for optimal digestibility on specific lignocellulosic substrates in order to reduce enzyme usage without sacrificing the rate or yield from substrate hydrolysis. t. reesei cellulase, a well-known cellulase in biomass industry was shown to lack β-glucosidase activity, resulting in strong inhibition of the cellulase by accumulated cellobiose in the reaction. this inhibition can be overcome by addition of external β-glucosidase enzymes from different microbial origins  <cit> . the synergistic action of t. reesei cellulase with a crude enzyme mixture from a. aculeatus containing various cell wall polysaccharide degrading enzymes with strong downstream cellulolytic and hemicellulolytic activities and a non-catalytic bacterial expansin, acting on physical loosening of the crystalline region of cellulose fibers, on saccharification of alkaline pretreated rice straw was reported  <cit> . glycosyl hydrolases from the metagenome of microflora present in sugarcane bagasse can enhance reducing sugar yield of t. reesei cellulase  <cit> . a recent work also showed strong synergistic action of a commercial cellulase, acellerase® <dig> with an endo-xylanase, pectate lyase, and a hemicellulose side chain cleaving α-arabinofuranosidase  <cit> . these findings demonstrate the potential of formulating active synergistic enzyme systems from various microbial sources in order to maximize the hydrolysis efficiency on specific lignocellulosic substrates. the glucose releasing efficiency of the synergistic ternary enzyme system developed in this study was higher than that reported for accellerase® <dig>  and the synergistic enzyme systems based on that enzyme, which was in the range of 122–173 mg/fpu  <cit> . the efficiency of the ternary enzyme mixture is in the same range to that reported for a synergistic system comprising t. reesei cellulase, a crude enzyme from a. aculeatus and a bacterial expansin  <cit>   and between t. reesei cellulase and enzyme from a. awamori  <cit>  . the greater efficiency of the c. globosum enzyme system may depend though on the nature of the substrates and hydrolysis conditions. elucidation of synergized enzyme activities in this study thus provides a basis for further modification of the cg-cel enzyme either by complementation of lack activities by external in-house enzymes or genetic modification of the strain.\n",
            "\n",
            "CONCLUSIONS\n",
            "a biomass-degrading enzyme system from c. globosum bcc <dig> , a potent fungus for development of a cellulase producing strain for on-site enzyme production, was characterized. the primary cellulase activity can be augmented by other hydrolytic and non-hydrolytic accessory enzymes. the enzyme system shows a high degree of synergism with commercial β-glucosidase and hemicellulase. the optimal combination of cg-cel, β-glucosidase and hemicellulase was found, leading to marked improvement in the sugar releasing efficiency. the work provides an approach for designing an effective enzyme system with specificity for saccharification of lignocellulosic materials, including agricultural biomass for the biorefinery industry.\n",
            "\n",
            "additional files\n",
            "\n",
            "additional file 1: table s <dig>  comparison of cellulose production by c. globosum bcc <dig> and other fungi in recent selected publications. \n",
            "\n",
            "\n",
            "additional file 2: table s <dig>  sugar yields from hydrolysis of pretreated rice straw using various enzyme combinations and dosages. \n",
            "\n",
            "\n",
            "additional file 3: figure s <dig>  sds-page analysis of the bcc <dig> crude enzyme extracts. lane 1- marker proteins; lane  <dig> –the cg-cel crude enzyme extracts stained with coomassie brilliant blue. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "abbreviations\n",
            "cg-celcellulase system of chaetomium globosum bcc5776\n",
            "\n",
            "cmcasecellulase activity on carboxymethyl cellulose\n",
            "\n",
            "dnsdinitrosalicylic acid\n",
            "\n",
            "epfbempty palm fruit bunch\n",
            "\n",
            "fpasecellulase activity on filter paper\n",
            "\n",
            "ghglycosyl hydrolase\n",
            "\n",
            "sbmsoybean meal\n",
            "\n",
            "tfftangential flow filtration\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study, a biomass-degrading enzyme system of the soft-rot fungus c. globosum bcc <dig> was characterized.\n",
            "the objective of this study was to identify enzymes with synergistic activities with cellulases and hemicellulose degrading enzymes in order to maximize the hydrolysis efficiency on lignocellulose and other lignocellulose degrading enzymes.\n",
            "the goal of this study was to identify enzymes with synergistic activities with cellulases and hemicellulose degrading enzymes in order to maximize the hydrolysis efficiency on lignocellulose and other lignocellulose degrading enzymes.\n",
            "the objective of this study was to identify enzymes with synergistic activities with cellulases and hemicellulose degrading enzymes in order to maximize the hydrolysis efficiency on specific lignocellulose and lignocellulose degrading enzymes.\n",
            "the objective of this study was to identify enzymes with synergistic activities with cellulases and hemicellulose degrading enzymes in order to maximize the hydrolysis efficiency on lignocellulose and lignocellulose degrading enzymes.\n",
            "the objective of this study was to identify enzymes with synergistic activities with cellul\n",
            "32\n",
            "[{'summary_text': 'to evaluate the impact of prophage induction on host gene expression, we constructed a whole-genome cdna e. coli/bacteriophage lambda microarray, representing 99% of  <dig> e. coli open reading frames and  <dig> predicted lambda open reading frames.\\nto evaluate the impact of prophage induction on host gene expression, we constructed a whole-genome cdna e. coli/bacteriophage lambda microarray, representing 99% of  <dig> e. coli open reading frames and  <dig> predicted lambda open reading frames.\\nin addition to altered transcription of the cpx regulon, we observed altered expression of several genes associated with initiation of replication and cell division.\\nin addition to altered transcription of the cpx regulon, we observed altered expression of  <dig> genes associated with initiation of replication and cell division.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "bacteriophage lambda has been studied for over  <dig> years and has served as a model for understanding genetic networks, control and development. lambda is a temperate phage capable of undergoing divergent developmental pathways: lysis and lysogeny. lytic development is lethal to host escherichia coli, resulting in amplification and release of progeny phage. in the lysogenic state the phage integrates into the host chromosome, where it can silence lytic promoters and replicate quiescently as a prophage. induction of lysis from the lysogenic state can be triggered by agents that damage dna or interfere with replication, such as mitomycin c and uv light. the gene regulatory network underlying the lambda lifecycle has been studied in exhaustive detail, yet the switch continues to reveal new levels of complex regulation  <cit> . the mechanistic details of this switch have been elegantly reviewed elsewhere  <cit> .\n",
            "\n",
            "the lambda gene regulatory network is composed of both phage and host factors, many of which interact with each other  <cit> . host proteases, replication, transcription and translation machinery are necessary for phage replication and development. proteins that interact with lambda antiterminator n play an essential role in regulating the temporal expression of delayed and late phage genes. several host factors have been shown to alter phage gene expression by unknown mechanisms. for example integration host factor  appears to enhance transcription at lambda promoter pl  <cit> . lambda gene products can, in turn, alter the state of host gene expression. for example, the lambda repressor ci directly represses pcka, a host gene involved in glucogenesis  <cit> . additionally, lambda-n can antiterminate host transcripts in the galactose operon region during prophage induction, resulting in elevated levels of galactose enzymes  <cit> . non-essential phage gene products of unknown function appear to alter host physiology, cell-cycle and macromolecular synthesis  <cit> . overall, lambda development is sensitive to the physiological state of the host and possibly to stochastic fluctuations in host and phage protein levels  <cit> .\n",
            "\n",
            "the focus of this study is to identify interactions between host and phage during a dynamic phage process. these interactions are also of interest to the emerging field of synthetic biology in which new genetic circuits are engineered into cells, often from parts found in lambda  <cit> , and must be designed to have minimal impact on the host. in the lysogenic state lambda expresses a small set of proteins that, among other things, repress lysis and confer fitness to the host while perturbing little else in host function  <cit> . this makes sense since advancing host fitness favors survival of the prophage. on the other hand, during lytic growth there is no such evolutionary concern for the host, other than a selection pressure to maintain host infrastructure for producing progeny phage. hence, we wondered how host functions are modulated by the lambda phage gene expression program.\n",
            "\n",
            "to address this question, here we characterize the effect of prophage induction on host physiology. since much of the lambda regulatory circuitry is controlled at the transcriptional level, we surveyed gene expression profiles of the escherichia coli and lambda phage transcriptomes throughout the timecourse of prophage induction by uv light. we constructed a whole-genome cdna e. coli/bacteriophage lambda microarray, representing 99% of  <dig> e. coli open reading frames and  <dig> predicted lambda open reading frames. we characterized the response of wild-type lambda lysogens and non-lysogens to uv light, and studied the impact of prophage induction on phage and host gene expression.\n",
            "\n",
            "RESULTS\n",
            "expression profiles of escherichia coli and bacteriophage lambda orfs\n",
            "to evaluate gene expression of host and phage in a lysogen and during prophage induction, total rna was harvested at 20-minute intervals from log-phase cultures of wild-type lysogenic and non-lysogenic e. coli. rna samples and genomic dna were fluorescently labeled and competitively hybridized on cdna arrays containing 99% of e. coli open reading frames  and  <dig> predicted orfs of bacteriophage lambda. lysogenic and non-lysogenic wild-type e. coli were compared at each time point, via a genomic control. indirect comparison to a genomic dna reference was useful for making multiple comparisons within and between the two strains.\n",
            "\n",
            "we evaluated three time courses to distinguish the responses of host and phage to the following conditions: irradiation of non-lysogens with uv light, irradiation of lysogens with uv light, and a \"mock induction\" of lysogens in which cells were taken through the induction protocol but were not irradiated. e. coli genes were regarded as having statistically significant differential expression if they met the following criteria: average fold change >  <dig>  and p-value ≤  <dig>  . by applying these criteria to each time course we identified differentially changing genes for each condition .\n",
            "\n",
            "gene expression of escherichia coli and bacteriophage lambda in the lysogenic state\n",
            "we observed eighteen e. coli genes to have significantly altered expression levels in lambda lysogens relative to uninfected bacteria . many of these changes were consistent with other genes in the same operon. eleven of these transcripts encoded transport-related gene products, including the proline abc transporter , the phosphate abc transporter , spermidine transport permease , and the l-lactate utilization system . six transcripts were of unknown function. only three significantly changing e. coli genes had expression differences greater than two-fold . it has previously been shown that pcka is repressed in lysogen-containing bacteria  <cit> . we do not observe lysogen-specific repression of pcka in our analysis because the gene is repressed in both wild type and lysogenic strains under our growth conditions, in which glucose is present.\n",
            "\n",
            "in the lysogenic state most of the lambda genome is silenced by the repressor ci. at 95% confidence, we detected significant expression of eight phage genes . the most highly expressed genes were on the prm operon: ci, rexa and rexb. phage genes lom and bor, virulence factors known to be expressed in a lysogen  <cit> , were not detected at significant levels in our microarrays. bor is not detected because the coding region in our strain was deleted and replaced with an antibiotic marker. it is unknown why lom is not expressed at detectable levels in our experiments: little is known about regulation of the lom promoter; it may be repressed under the growth conditions in this study. a previous study by chen et al, examining gene expression of the lysogenic state during growth on minimal media, observed high expression levels of kil, int, and gpe, in addition to ci, rexab, lom and bor  <cit> . our expression study confirms the chen et al data for ci and rexab gene expression but differs in two ways: first, we do not observe significant expression of kil, int or gpe; second, we observe low-level induction of gpg, gpt, bet, xis, and orf <dig>  despite these differences, changes in non-lysogenic genes in both studies were small  and the measurements are particularly sensitive to noise, as expression ratios compare samples with and without the lambda genome.\n",
            "\n",
            "gene expression of bacteriophage lambda during prophage induction\n",
            "prophage induction is known to proceed in a temporal cascade of regulation, initiated by depletion of repressor ci at upstream binding sites in or and ol. upon exposure to uv light, ci undergoes an auto-cleavage reaction stimulated by an activated form of reca, probably activated by binding to single-stranded dna at sites of damage  <cit> . degradation of ci results in the de-repression of early lytic promoters pr and pl and transcription of anti-terminators n and q. n is a regulator of delayed-early transcripts that anti-terminates sites in both pr and pl operons, allowing transcription of the full program of delayed early genes including replication and excision machinery. the q gene product anti-terminates the late pr' lytic promoter. accumulation of q is necessary for transcription of capsid, assembly, tail and lysis genes from pr'.\n",
            "\n",
            "to characterize the lambda gene expression program during induction, transcript levels at each time point after induction were compared to basal expression levels of cells undergoing a mock induction protocol. basal expression levels are detectable above background noise in all lambda probes due to a low frequency of spontaneous de-repression. most measured phage transcripts were significantly up-regulated during the time course of prophage induction . early in induction, very little phage gene expression was detected. at  <dig> minutes several delayed early and early lytic genes were detected at significant levels including n, cro, ciii, gam and bet. at  <dig> minutes all early lytic and some late lytic genes were significantly upregulated. by  <dig> minutes, all late phage genes were highly expressed . at this time in induction, 20% of the total detected transcripts encoded phage proteins .\n",
            "\n",
            "we also grouped the phage genes by average linkage hierarchical clustering . in the expression data presented here, four main clusters of genes were clearly distinguished. each cluster corresponded to different developmental stages of the phage lifecycle: lysogeny, early lytic, and two clusters of lytic transcription. the observed expression timing of phage genes proceeded in a temporal cascade, as expected.\n",
            "\n",
            "verification of lambda phage gene expression by rt-pcr\n",
            "several genes from the lambda phage genome were analyzed by rt-pcr, including repressors ci and cro, p, n, q, and lom. one of these genes, p, was not present on our arrays but exhibited the same qualitative behavior as other early lytic genes. rt-pcr data  indicate that the fold-change of several genes was higher than that measured on microarrays, but that the direction of change was the same.\n",
            "\n",
            "e. coli gene expression is insensitive to \"mock irradiation\" protocol\n",
            "as a negative control, to confirm that the irradiation protocol has no effect on cell physiology, irrespective of uv light, we performed \"mock inductions\" on lysogens using a standard protocol for inducing phage. as expected, none of the host or phage transcripts were differentially expressed over the course of  <dig> minutes following the mock induction protocol.\n",
            "\n",
            "gene expression program of the e. coli sos response\n",
            "the e. coli sos response has been studied extensively and global expression levels have been profiled using microarrays  <cit> . in the transcriptional profiling study by courcelle et al, a dose of  <dig> j/m <dig> was applied to induce an sos response in wild-type e. coli. in results presented here, we selected a dose of  <dig> j/m <dig> uv light because it elicits a less intense sos response but is a sufficiently high dose to induce most lysogens . we observed differential regulation of  <dig> genes in wild-type e. coli following irradiation by uv light . in our study,  <dig> genes known to be regulated by lexa were significantly up-regulated  <dig> minutes after uv exposure, in agreement with prior studies  <cit> . the gene expression profiles of the uv-response genes in lysogenic and non-lysogenic e. coli were similar for the first  <dig> minutes; at  <dig> and  <dig> minutes the response diverged, presumably due to the impact of prophage induction . our results are not directly comparable to other studies as we used different strains and media and a much lower uv dose.\n",
            "\n",
            "host gene expression during prophage induction\n",
            "to evaluate significantly changing host genes during prophage induction, we compared temporal expression profiles of lysogenic and non-lysogenic e. coli at  <dig> minute intervals, following exposure to uv light. using the selection criteria described previously, we observed differential expression of  <dig> host genes . most changes were small  and occurred late in induction, with reduced expression of genes in several cog functional groups.\n",
            "\n",
            "transcriptional enhancement observed near lambda integration site during prophage induction\n",
            "genes in the gal operon, adjacent to the phage integration site attb, are amplified during prophage induction in e. coli; this amplification has been attributed to increase in gene dosage due to escape replication from the prophage origin  <cit>  and trans-acting antitermination by lambda-n  <cit> . in data presented here, transcription of gal operon genes was enhanced nearly 13-fold in lysogens relative to non-lysogens, as expected. in addition to the gal operon, we observed enhancement of transcription in a 300-kb region surrounding the prophage integration site, attb . the boundaries of this region were defined by binning the average expression fold-change at defined intervals. in the attb region,  <dig> genes demonstrated  <dig> -fold or higher up-regulation; no genes in the region were down-regulated. as distance from attb increased, the average fold-change of host genes decreased.\n",
            "\n",
            "host dna adjacent to attb is amplified at high copy number\n",
            "it was recently demonstrated that induction of lambdoid phages in salmonella enterica can lead to amplification of large regions of the host genome next to phage integration sites via escape replication  <cit> . sternberg observed escape replication in bacteriophage lambda extending to chromosomal markers up to  <dig> min from the attb integration site  <cit> . to examine the impact of prophage induction on dna copy number under our conditions, we competitively hybridized genomic dna from untreated lysogens with dna isolated from lysogens one hour after irradiation with uv light. over  <dig> genes in a 400-kb region surrounding attb showed an increase in copy number , indicating that replication is initiating at the lambda origin and extending into the host chromosome. as distance from the prophage increases the dna copy number is reduced. increased gene dosage via escape replication from attb likely contributes to enhanced levels of host transcripts in this region.\n",
            "\n",
            "a replication-defective phage has reduced impact on the host in the attb region\n",
            "to evaluate the impact of dna copy number on transcriptional response in the attb region, we examined the gene expression profile of a uv-induced lambda prophage defective in replication. in comparison to non-lysogenic bacteria,  <dig> genes were differentially regulated during the course of prophage induction, less than half that observed in wild type phage. in the attb region we observed differential regulation of  <dig> genes, with no evident bias for upregulated genes or position . we did not observe an increase in dna copy number in this region . most genes with known function encoded protiens involved in energy metabolism. the direction of change for these genes was negative, consistent with our observation of energy metabolism genes during induction of wild type phage discussed below.\n",
            "\n",
            "functional categorization of differentially regulated host genes during prophage induction\n",
            "excluding the up-regulated genes in the attb region, we examined the functional categorization of the remaining  <dig> genes responsive to lambda phage.  <dig> of these genes were putative open reading frames with no known function. few genes exhibited altered expression at early timepoints; the most significant change  <dig> minutes after uv exposure was 8-fold induction of the fruabk operon, involved in fructose transport and metabolism. most host genes were differentially regulated at later timepoints . a fisher exact test  demonstrated that after sixty minutes six out of  <dig> functional groups were enriched in down-regulated genes: biosynthesis, carbon catabolism, cell processes, cell structure, energy metabolism and transport. no functional groups were enriched in up-regulated genes.\n",
            "\n",
            "in addition to this functional coherence, several genes known to interact with lambda showed altered transcriptional regulation. it was previously shown that production of early lambda gene products leads to increased levels of heat shock gene products  <cit> . induction of the heat shock regulon is mediated indirectly by lambda-ciii, which inhibits protease ftsh and thereby stabilizes σ <dig>  and other ftsh-sensitive proteins  <cit> . in data presented here, we observed altered transcription of several genes in the heat shock regulon, including molecular chaperones encoding proteins essential for bacteriophage replication and morphogenesis  <cit> . transcription of dnak, mopa , mopb , htpgx and hsluv was significantly enhanced forty minutes after induction and repressed at sixty minutes. expression of additional heat shock genes htra and hsit was increased four-fold at  <dig> and  <dig> minutes after induction. up-regulation of the heat shock regulon coincided with peak levels of the lambda-ciii transcript.\n",
            "\n",
            "expression of accessory genes on the lambda-pl operon has been shown to alter the e. coli cell cycle by transiently blocking initiation of replication and cell division  <cit> . we investigated the extent of this effect on host gene expression during lambda induction and found altered expression of several genes associated with initiation of replication and cell division. cell division genes ftsj, sula, tig, dicb, mind and xerc were down-regulated late in induction. the tig gene encodes trigger factor , an important molecular chaperone involved in cell division  <cit> . changes in tf levels cause defective cell division  <cit> . xerc is a recombinase that effects chromosome segregation at cell division. inhibitors of cell division, dicb, sula, and mind, were down-regulated at forty and sixty minutes. increased expression was observed at sixty minutes in genes involved in cell division  and initiation of replication . these genes are located near the lambda integration site, so any change in their expression levels late in induction is complicated by the increase in copy-number in this region.\n",
            "\n",
            "differential regulation of two-component signaling pathways and global regulators\n",
            "we observed altered regulation of several genes in the cpx and arc two-component regulatory systems. the cpx response network is estimated to regulate the expression of over  <dig> genes, involved in functionally diverse processes such as management of protein distress, motility and chemotaxis, biofilm formation and response to envelope stress  <cit> . cpxa and cpxr, co-transcribed on an operon activated by σs  and cpxr-p  <cit> , were upregulated in lambda lysogens  <dig> minutes after induction. in addition to cpxar,  <dig> genes in the cpx regulon were differentially regulated in our data. these genes encode proteins involved in diverse physiological activities, including envelope distress , management of protein distress , the starvation response , the pts system , glycine cleavage , and regulation of sigma-e . of the  <dig> genes known to be under positive control of cpx-p, six were upregulated including cpxar, yihe-dsba, and rpoerseab. the direction of cpx regulation is unknown for the remaining genes. there is overlap between genes expressed in the cpx and rpoh regulons, and it is speculated that these systems, along with the rpoe regulon, respond to a variety of stresses in a concerted manner  <cit> .\n",
            "\n",
            "the arc system responds to respiratory growth conditions to modulate expression of an estimated 100– <dig> genes, including metabolic regulons  <cit> . normally activated under anaerobic conditions, the arc response can be also be activated under aerobic conditions in the presence of reducing agents  <cit> . we identified differential regulation of  <dig> genes in the arc regulon. nearly all the genes were repressed  <dig> minutes after prophage induction, and all but five were expressed in the same direction of change consistent with previously published data. several multi-gene operons known to be repressed by phosphorylated arca were down-regulated including aceab, agacdi, cyoabcde, lctprd, nuoabcefghijklm, and trebc. sura, encoding a protein that mediates stress-induced survival, was also downregulated. further experimentation is required to test whether these shifts in gene expression are a response mediated by the arc system or a pleiotropic stress response.\n",
            "\n",
            "in addition to altered transcription of the cpx and arc regulons, we observed changes in other global regulators in lambda lysogens, including global regulators crp and lrp. lrp was upregulated at  <dig> and  <dig> minutes. the crp transcript was downregulated at  <dig> and  <dig> minutes. since these changes occurred late in induction, little effect on downstream pathways was observed before bacteria started lysing en masse.\n",
            "\n",
            "discussion\n",
            "in this study we measured gene expression changes in wild-type escherichia coli with and without an integrated prophage, during exponential growth and following exposure to a mild dose of uv light. we observed that the lambda phage genes were up-regulated in a temporal cascade. we also found that  <dig> e. coli genes were differentially expressed in response to prophage induction. most of these changes, while significant, were small in magnitude and occurred late in induction.\n",
            "\n",
            "it was expected that the phage gene expression would progress in a temporally organized manner and for the most part, lambda behaved as expected. we observed lysogenic transcripts to be up-regulated  <dig> minutes after induction. this result was confirmed by rt-pcr. in models of promoter activity at prm by dodd et al, a decrease in repressor concentration from lysogenic levels can stimulate activity of prm  <cit> . the increase in lysogenic transcript abundance could result from transient de-repression of the prm promoter or by an increase in gene dosage due to lambda dna replication.\n",
            "\n",
            "the host response to lambda phage induction by uv light is more complex than previously thought, causing shifts in expression of  <dig> genes. that complexity was compounded by the necessity to consider multiple conditions as controls. in a mock induction control, we found that our induction protocol does not have a significant impact on host or phage gene expression. controlling for response to uv light exposure, we found the sos response in non-lysogenic and lysogenic e. coli to be in good agreement with previous findings.\n",
            "\n",
            "of the genes that were up-regulated during prophage induction, nearly all were proximal to the attb integration site, likely due to an increase in copy number of the dna in that region. escape replication appears to be a common strategy for some, but not all, lambdoid phages  <cit> . prophage induction with a replication-defective mutant prophage eliminated both transcriptional and dna copy-number enhancement in this region.\n",
            "\n",
            "several known mechanisms of host-phage interaction were confirmed by our data, including induction of the heat shock regulon, escape replication and transient inhibition of cell division and initiation of replication. it has been speculated that these interactions benefit lytic development by increasing phage gene dosage and the available pool of host resources.\n",
            "\n",
            "at the peak of late lytic-phase lambda gene expression, several hundred host genes were down-regulated. these genes were involved in diverse cellular processes including biosynthesis, cell structure, central and energy metabolism, and transport. due to replication and high levels of phage gene expression, one would expect that expression of many host genes would be altered simply by mass action, especially late in induction when approximately 20% of the transcriptional output encodes phage proteins. a follow-up experiment in which late lambda gene expression is blocked may reveal other interesting and specific effects of early lambda gene products.\n",
            "\n",
            "in results presented here, late lambda phage induction appears to induce general stress response pathways in the host, including the arc and cpx two-component systems, and global regulators crp and lrp. the arc system is activated by alterations in the redox state of the cell. the cpx regulon is a pleiotropic stress response system, responsive to diverse kinds of stress including envelope and nutritional distress. the cpx system interacts with the heat shock regulon, also upregulated during lambda induction. crp and lrp are both sensitive to the nutritional state of the bacterium. depletion of cellular resources by the phage, along with envelope distress due to pending cell lysis, may contribute indirectly to these changes in host gene expression. global regulators can be induced by various intra- and extracellular cues, so it is unclear whether expression of these genes is pleiotropic or mediated by a specific factor. the host does not appear to deploy specific defense mechanisms in response to prophage induction at the gene expression level; rather the observed transcriptional shifts are directed by the phage, both directly and indirectly.\n",
            "\n",
            "CONCLUSIONS\n",
            "this study identified  <dig> e. coli genes responsive to lambda phage induction by uv light and confirmed known host-phage interactions. most host genes are differentially expressed late in induction at low fold-change. overall, prophage induction had a surprisingly low impact on host gene expression. these results imply that bacteriophage lambda, as a module, is relatively independent from its host during prophage induction, exerting relatively little load on host physiology until the lytic process is nearly complete.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "to evaluate the impact of prophage induction on host gene expression, we constructed a whole-genome cdna e. coli/bacteriophage lambda microarray, representing 99% of  <dig> e. coli open reading frames and  <dig> predicted lambda open reading frames.\n",
            "to evaluate the impact of prophage induction on host gene expression, we constructed a whole-genome cdna e. coli/bacteriophage lambda microarray, representing 99% of  <dig> e. coli open reading frames and  <dig> predicted lambda open reading frames.\n",
            "in addition to altered transcription of the cpx regulon, we observed altered expression of several genes associated with initiation of replication and cell division.\n",
            "in addition to altered transcription of the cpx regulon, we observed altered expression of  <dig> genes associated with initiation of replication and cell division.\n",
            "\n",
            "33\n",
            "[{'summary_text': 'in this study, we investigated the evolution of a passive mutualistic interaction between two auxotrophic strains of the bacterium escherichia coli bw <dig>  and two evolved mutualistic partners of bw <dig>  and tested whether the passive mutualistic interaction itself promotes the evolution of cell aggregation.\\nin this study, we investigated the evolution of a passive mutualistic interaction between two auxotrophic strains of the bacterium escherichia coli bw <dig>  and two evolved mutualistic partners of bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "mutualistic interactions are pervasive in the natural environment and shape the assembly and functioning of nearly every ecological community  <cit> . a mutualistic interaction occurs when two  different organisms – referred to here as mutualistic partners - have reciprocal positive effects on each other’s growth. examples of mutualistic interactions include the associations between legumes and nitrogen-fixing bacteria, between plants and pollinating insects, and between humans and members of their gut microbiota  <cit> . mutualistic interactions are also common between members of microbial communities and are important determinants of their ecological dynamics and processes  <cit> . a typical feature of mutualistic interactions within microbial communities is that they often require one or both mutualistic partners to excrete metabolites that positively affect the growth of others  . while we mainly focus on this type of mutualistic interaction in this manuscript, the main idea that we develop is potentially relevant for other types of mutualistic interactions and organisms.fig.  <dig> spatial structure promotes the evolution of active metabolite excretion. a a mutualistic interaction between two mutualistic partners. partner  <dig>  excretes metabolite a  that promotes the growth of partner  <dig>  partner  <dig>  excretes metabolite b  that promotes growth of partner  <dig>  b in a well-mixed environment, a mutant green cell that actively excretes more metabolites  should decrease in frequency. this is because the mutant cell would pay all of the cost for increased metabolite excretion but would receive the same benefit as all the other green cells. c in a spatially structured environment, a mutant green cell that actively excretes more metabolites  could potentially increase in frequency. this is because spatial structure might cause the mutant cell to receive a disproportionate amount of the benefits from increased metabolite excretion. these benefits originate from the positive effects of increased excretion of metabolite a on partner  <dig>  thus resulting in increased excretion of metabolite b\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "how mutualistic interactions emerge and persist is not clear . a first question pertains to the origin of mutualistic interactions. starting from a situation where two microbial genotypes exist in the same environment but do not affect each other’s growth, how could a mutant of one genotype emerge that excretes a metabolite that positively affects the growth of another genotype and increases in frequency, thus laying the foundation for a mutualistic interaction to evolve? in this scenario, the mutant might not receive an immediate benefit but would potentially carry a metabolic cost associated with excreting the metabolite. the mutant might therefore not be able to increase in frequency relative to its ancestor and a mutualistic interaction could not establish.\n",
            "\n",
            "one possible solution to this problem is that mutually beneficial interactions could originally be passive in nature  <cit> . we use the term ‘passive’ to refer to a behavior or property of an organism that, while potentially being beneficial to another organism, did not evolve because of its positive effects on that other organism. this is particularly evident for metabolic interactions between microorganisms, which often passively excrete metabolites that positively affect the growth of other microorganisms . for example, microorganisms might excrete metabolites via cell leakage or as side-products or end-products of their own metabolism , which could then be taken up by other microorganisms. if such metabolic interactions are reciprocal between two partners , they constitute as a mutually beneficial but passive mutualistic interaction.\n",
            "\n",
            "while the passive scenario described above could be important for the origin of a mutualistic interaction, many mutualistic interactions in nature are not passive but are rather based on the active excretion of metabolites that positively affect the growth of a mutualistic partner . the active excretion of metabolites must, at least to some extent, divert cellular resources away from the growth and reproduction of the excreting microorganism . these metabolic costs then lead to a second fundamental question: starting from a situation where two microorganisms are coupled by an initially passive mutualistic interaction, how could investment of metabolic resources into each mutualistic partner evolve? such an evolutionary transition requires a mutant that actively excretes metabolites to increase in frequency. again, this is not trivial to explain. a mutant that actively excretes metabolites would increase the growth of its mutualistic partner, which in turn would lead to a benefit that is accessible to both the mutant and its ancestor . while the benefits of an increased investment are thus distributed uniformly, the costs are borne alone by the actively excreting mutant. the mutant might therefore have a growth disadvantage relative to its ancestor and decrease in frequency.\n",
            "\n",
            "ultimately, the evolutionary transition of a passive into an active mutualistic interaction requires that the mutant that actively invests metabolic resources into the growth of its mutualistic partner receive an exclusive benefit  <cit> . such coupling between investment and return arises if individuals of different mutualistic partners are spatially associated with each other for long periods of time  <cit> . in this case, a mutant that invests metabolic resources into the growth of its mutualistic partner would receive an increased return if the partner’s growth leads to an increased production of the benefit. this situation has been referred to as the “partner-fidelity model”  <cit> . a general scenario under which such a long-term association between individuals could arise is in the presence of spatial structure  ; that is, in a situation where physical barriers constrain the movement of individuals and metabolites. in spatially structured environments, the benefits arising from an investment into another organism are not distributed evenly but are instead disproportionally directed back towards the investor, thus allowing those individuals to potentially increase in frequency and spread  .\n",
            "\n",
            "while there is accumulating theoretical and experimental evidence to support the importance of spatial structure on the evolution of mutualistic interactions, these studies have largely focused on imposing abiotic spatial structure on a mutualistic consortium and analyzing the evolutionary outcomes . mutualistic interactions, however, are also observed in habitats with relatively little abiotic spatial structure, for example between microorganisms that reside within the water columns of open oceans and lakes . this then underscores an important gap in our knowledge: how can we explain the evolution of an active mutualistic interaction in the absence of extensive abiotic spatial structure?\n",
            "\n",
            "the central idea that we are addressing in this manuscript is that microorganisms can readily evolve to create their own spatial structure based on physical attachment between individuals   <cit> . more specifically, we test the hypothesis that an initially passive mutualistic interaction selects for mutants that aggregate together with other individuals and thereby benefit from increased local concentrations of excreted metabolites. this scenario could have important consequences because cell aggregation leads to “partner fidelity” and could set the stage for the evolution of an active mutualistic interaction. indeed, cell aggregates are prevalent within the mixed layers of oceans and non-stratified lakes and can harbor mutualistic interactions . whether a passive mutualistic interaction itself could promote the evolution of cell aggregation, however, is not clear.\n",
            "\n",
            "to test this hypothesis, we experimentally created a passive mutualistic interaction between two auxotrophic strains of the bacterium escherichia coli and tested for the evolution of cell aggregation. each strain is defective in the biosynthesis of a different amino acid; they can only grow if the required amino acid is exogenously supplied from an abiotic source or if the strains are grown together in co-culture and passively excrete or release small amounts of the amino acid required by the other. this passive mutualistic interaction was based on a single genetic mutation in each mutualistic partner; it could therefore originate spontaneously via random mutation within large populations. we then propagated the mutualistic co-cultures in the absence of extensive abiotic spatial structure  and tested whether the passive mutualistic interaction itself promotes the evolution of cell aggregation. we are not investigating how spatial structure influences the evolutionary transition to an active mutualistic interaction in this manuscript. instead we ask how spatial structure itself evolves, and thereby focus on a process that has potentially profound implications for interactions both within and between populations of organisms.\n",
            "\n",
            "methods\n",
            "bacterial strains\n",
            "we obtained amino acid-auxotrophic strains of e. coli bw <dig> from the keio single-gene knockout library  <cit>  . strain bw <dig>  contains a complete deletion of the open-reading frame of the proc gene and is predicted to be defective in proline biosynthesis. strain bw <dig>  contains a complete deletion of the open-reading frame of the trpc gene and is predicted to be defective in tryptophan biosynthesis. we refer to these strains as ‘mutualistic partners’ because they can grow together in co-culture but cannot grow alone; the ability to grow together is based on the passive excretion – presumably via cell leakage - of the amino acid that the other strain cannot biosynthesize.table  <dig> bacterial strains and plasmids used in this study\n",
            "\n",
            "\n",
            "r; kmr\n",
            "r; kmr\n",
            " puc18t-mini-tn7t-lac-gm-egfp\n",
            " puc18t-mini-tn7t-lac-gm-echerry\n",
            "\n",
            "\n",
            "\n",
            "we verified that the mutualistic partners are auxotrophic for the predicted amino acid by growing them in isolation with liquid minimal medium that was or was not supplemented with the required amino acid. the liquid minimal medium consisted of  <dig>  g l− <dig> na2hpo4 × 7h2o, 3 g l− <dig> kh2po <dig>   <dig>  g l− <dig> nacl, 1 g l− <dig> nh4cl,  <dig>  g l− <dig> glucose,  <dig>  g l− <dig> mgso <dig>  and 10 mg l− <dig> gentamycin . we streaked each mutualistic partner onto a different lysogeny broth  agar plate, picked three colonies from each lb agar plate, inoculated each colony into a different test tube containing 3 ml of mm, and incubated the test tubes for 24 h at 37 °c with continuous shaking . as expected, neither mutualistic partner could grow in isolation with mm. however, strain bw <dig>  could grow in isolation when we supplemented mm with 50 mg l− <dig> l-proline while strain bw <dig>  could grow in isolation when we supplemented mm with 20 mg l− <dig> l-tryptophan, thus verifying that each mutualistic partner is indeed auxotrophic for the predicted amino acid.\n",
            "\n",
            "genetic manipulations\n",
            "we introduced a different plasmid into each mutualistic partner that carries a gene encoding for a different florescent protein, thus allowing us to distinguish and individually quantify each mutualistic partner when they are grown together in co-culture. to accomplish this, we constructed two derivatives of the puc18t-mini-tn7t-lac-gm conditionally replicative plasmid   <cit> . this plasmid contains an isopropyl-β-d-thiogalactopyranosid -inducible plac promoter located immediately upstream of a multiple cloning site . we first purified the puc18t-mini-tn7t-lac-gm plasmid from an overnight culture of e. coli dh5α/λpir   <cit> . we next used gotaq dna polymerase  to pcr amplify the egfp or echerry gene  <cit> , which encode for green or red fluorescent protein respectively. the pcr amplification primers contain the bamhi and kpni restriction sites that we used to clone the pcr products into the mcs of the puc18t-mini-tn7t-lac-gm plasmid . we then digested the pcr products and the puc18t-mini-tn7t-lac-gm plasmid with bamhi and kpni  and ligated the pcr products into the puc18t-mini-tn7t-lac-gm plasmid. we designated the assembled derivative plasmids as puc18t-mini-tn7t-lac-gm-egfp and puc18t-mini-tn7t-lac-gm-echerry . we finally replicated the assembled derivative plasmids in e. coli dh5α/λpir   <cit> , introduced the derivative plasmids into the mutualistic partners via electroporation, and selected for transformants carrying the derivative plasmids by plating on lb agar plates supplemented with 10 μg ml− <dig> gentamycin and 1 mm iptg. we introduced each derivative plasmid into each mutualistic partner, resulting in both egfp- and echerry-expressing variants of strains bw <dig>  and bw <dig> .\n",
            "\n",
            "evolution experiment\n",
            "we performed an evolution experiment with replicated co-cultures of the two mutualistic partners. we streaked the egfp-expressing bw <dig> , echerry-expressing bw <dig> , echerry-expressing bw <dig> , and egfp-expressing bw <dig>  strains onto different lb agar plates that were supplemented with 10 μg ml− <dig> gentamycin and 1 mm iptg. we then picked one colony from each lb agar plate, inoculated each colony into a different test tube containing  <dig>  ml of mm supplemented with 300 μl of liquid lb medium, and incubated the test tubes for 24 h at 37 °c with continuous shaking . after reaching stationary phase, we centrifuged the cultures, discarded the spent medium, washed the cells twice with h2o containing 9 g l− <dig> nacl, and suspended the washed cells in h2o containing 9 g l− <dig> nacl. we next prepared two binary mixes  of the mutualistic partners ; one set of co-cultures consisted of the egfp-expressing bw <dig>  and echerry-expressing bw <dig>  mutualistic partners while the other set of co-cultures consisted of the echerry-expressing bw <dig>  and egfp-expressing bw <dig>  mutualistic partners. we finally inoculated 300 μl of each co-culture into eight replicated test tubes containing  <dig>  ml of mm that was supplemented with 1 mm iptg but not with amino acids, resulting in a total of  <dig> replicated mutualistic co-cultures. we designated the co-cultures consisting of the egfp-expressing bw <dig>  and echerry-expressing bw <dig>  mutualistic partners as mutualistic co-cultures a <dig> to a <dig> and the co-cultures consisting of the echerry-expressing bw <dig>  and egfp-expressing bw <dig>  mutualistic partners as mutualistic co-cultures b <dig> to b <dig>  the initial od <dig> of each mutualistic co-culture was approximately  <dig> . after incubating the mutualistic co-cultures for  <dig>  days at 37 °c with continuous shaking , we transferred 300 μl of each co-culture to a new test tube containing  <dig>  ml of fresh mm that was supplemented with 1 mm iptg but not with amino acids to achieve a 1: <dig>  dilution. we then repeated the incubation and transfer steps in the same mm supplemented with 1 mm iptg for a total of  <dig> serial transfers. immediately before each transfer, we measured the od <dig> of each mutualistic co-culture and archived a portion of each mutualistic co-culture in 20% glycerol at −80 °c for further analyses. we quantified the magnitude of cell aggregation within each mutualistic co-culture immediately before the fourteenth transfer as described below. if all cells in these co-cultures would have grown at the same rate, then the fourteen transfers with ten-fold dilution would correspond to approximately  <dig> generations of growth ; however, if only a portion of the cells would have grown efficiently under our experimental conditions , then the number of cell generations during the evolution experiment could have been potentially much larger. we performed control experiments with the ancestral mutualistic partners using growth conditions identical to those described above, except that the mm was supplemented with 50 mg l− <dig> l-proline and 20 mg l− <dig> l-tryptophan.\n",
            "\n",
            "quantification of cell aggregation\n",
            "we imaged each mutualistic co-culture using a leica tcs sp <dig> confocal laser-scanning microscope  with a 63 ×  oil-immersion lens . we removed 5-μl liquid aliquots from each mutualistic co-culture immediately after removal from the shaking incubator and deposited the liquid aliquots onto the surface of a glass slide. we imaged egfp-expressing cells using 488 nm excitation wavelength and 500–530 nm emission wavelengths. we imaged echerry-expressing cells using 633 nm excitation wavelength and 657–757 nm emission wavelengths. we collected images at a resolution of 1024 ×  <dig> using las af v <dig>  software .\n",
            "\n",
            "we quantified cell aggregation using the statcoloc plugin of the icy software  <cit> . this algorithm computes the two-dimensional co-localization of different objects  using the ripley’s k function  <cit> . the resulting k-value measures the degree to which a set of objects deviates from spatial homogeneity. in our experiments with completely mixed batch reactors, a deviation from spatial homogeneity is most likely caused by cell aggregation, which we indeed confirmed by microscopy. we first detected e\n",
            "gfp- and echerry-expressing cells using the spot detector plugin of icy and translated the images into binary data using the nih imagej analysis software . we then applied the statcoloc plugin using a radius of  <dig>  μm to  <dig>  μm. we analyzed between five and nine randomly selected microscope fields for each mutualistic co-culture and obtained ten k-values as a function of distance for each microscope field. we finally identified the maximum observed k-value for each microscope field and tested whether the maximum observed k-values are significantly different between test and reference mutualistic co-cultures using the non-parametric mann-whitney u test. we performed the same statistical tests with the sum-of-the-ten k-values rather than the maximum observed k-value and obtained qualitatively identical results. we chose here to report the maximum observed k-values because they had better statistical properties. namely, many of the microscopy fields did not contain any cell aggregates, which would be expected when aggregation is extensive and consists of a few sparsely distributed objects. this increases the variance when using the sum-of-the-ten k-values. we implemented the mann-whitney u test with the statcoloc plugin and considered a two-sided p <  <dig>  to be statistically significant.\n",
            "\n",
            "isolation of evolved mutualistic partners\n",
            "we isolated mutants emerging within each archived mutualistic co-culture that remained viable immediately before the fourteenth transfer of experimental evolution. we first streaked each archived mutualistic co-culture onto different lb agar plates that were supplemented with 10 μg ml− <dig> gentamycin and 1 mm iptg. in many cases, the colonies expressed multiple fluorescent proteins, and we therefore had to perform a second streaking. after obtaining single colonies that only express one fluorescent protein, we qualitatively distinguished different evolved mutualistic partners within each co-culture based on colony morphology and the fluorescent gene that they expressed . for some co-cultures, we identified more than one colony morphology for each fluorescent gene. we finally picked one colony for each morphology, inoculated each colony into a different test tube containing 3 ml of liquid lb medium supplemented with 10 μg ml− <dig> gentamycin and 1 mm iptg, incubated the test tubes for 24 h at 37 °c with continuous shaking , and archived a portion of each culture in 20% glycerol at −80 °c for further analyses.\n",
            "\n",
            "genome sequencing of evolved mutualistic partners\n",
            "we sequenced the genomes of all the isolated evolved mutualistic partners . in parallel, we sequenced the genomes of all the ancestral mutualistic partners , echerry-expressing bw <dig> , echerry-expressing bw <dig> , and egfp-expressing bw <dig>  strains). we first streaked each mutualistic partner from the glycerol-archived samples onto different lb agar plates supplemented with 10 μg ml− <dig> gentamycin and 1 mm iptg. we then picked one colony from each lb agar plate, inoculated each colony into a test tube containing 3 ml of liquid lb medium, incubated the test tubes for 24 h at 37 °c with continuous shaking , and extracted genomic dna from each culture using the archivepure dna purification kit . we then prepared one sequence library for each mutualistic partner using 1 ng of genomic dna, the nextera xt dna sample preparation kit , and a different sample-specific multiplex adapter. we next pooled the libraries together, loaded the pool onto a single miseq flow cell , and sequenced the libraries using a miseq sequencer  operated by the genomic diversity center at eth zürich  we performed paired-end 150-cycle sequencing with the miseq reagent kit  . all of the sequence reads are publically available in the ncbi sequence read archive  under bioproject id number sub <dig> \n",
            "\n",
            "we analyzed the resulting sequence reads by first binning the raw sequence reads into libraries using the automated run protocol on the miseq sequencer . we then performed quality control with fastqc version  <dig> . <dig> and quality filtering using prinseq lite version  <dig> . <dig> software  <cit> . the parameters used for quality filtering are as follows: out_format, 3; min_qual_mean, 28; min_len, 50; range_gc 15–85; ns_max_n, 1; derep, 14; derep_min, 2; trim_ns_left, 1; trim_ns_right, 1; trim_qual_left, 28; trim_qual_right, 28; trim_left,  <dig>  in summary, we trimmed or discarded all sequence reads with mean quality scores below  <dig> or had ambiguous nucleotides. we further discarded all sequence reads that were shorter than  <dig> bases. we finally applied the breseq pipeline  and the utility program gdtools  <cit>  to predict genetic changes between each evolved mutualistic partner and its corresponding ancestral mutualistic partner. this included nucleotide polymorphisms, deletions, insertions, and multiplications. we used the genome sequence of e. coli k- <dig> mg <dig>  <cit>  as a reference for the mapping and assembly of the sequence reads. the parameters used for calling genetic changes are as follows: reference, nc_000913; base-quality-cutoff, 15; require-match-length,  <dig> \n",
            "\n",
            "crossing experiment\n",
            "we randomly selected the b <dig> mutualistic co-culture to test whether one or more of the evolved mutualistic partners were responsible for the evolution of cell aggregation. based on colony morphology, fluorescent protein production, and genome analyses, we identified one evolved mutualistic partner of strain bw <dig>  and two different evolved mutualistic partners of strain bw <dig>  within the b <dig> mutualistic co-culture immediately before the fourteenth transfer of experimental evolution . we first streaked each of the evolved mutualistic partners and their corresponding ancestral mutualistic partners from the archived isolate samples onto different lb agar plates that were supplemented with 10 μg ml− <dig> gentamycin and 1 mm iptg. we then picked one colony from each lb agar plate, inoculated each colony into a different test tube containing  <dig>  ml of mm supplemented with 300 μl of liquid lb medium, and incubated the test tubes for 24 h at 37 °c with continuous shaking . after reaching stationary phase, we washed and suspended the cells in water containing 9 g l− <dig> nacl as described above for the evolution experiment. we next prepared different binary  or ternary  mixes of different evolved and ancestral mutualistic partners as described in the results section. we finally inoculated 300 μl of each mix into replicated test tubes containing  <dig>  ml of mm that was supplemented with 1 mm iptg but not with amino acids. the initial od <dig> of each mutualistic co- or tri-culture was approximately  <dig> . after incubating the mutualistic co- or tri-cultures for 7 days at 37 °c with continuous shaking , we measured the od <dig> of each co-or tri-culture and quantified the magnitude of cell aggregation as described above.\n",
            "\n",
            "RESULTS\n",
            "experimental creation of a passive mutualistic interaction\n",
            "we created a passive and obligate mutualistic interaction by inoculating pairs of the bw <dig>  and bw <dig>  mutualistic partners  together into mm that was not supplemented with amino acids. we found that the mutualistic partners could grow when they were inoculated together but could not grow when they were inoculated in isolation, thus demonstrating that we could indeed create the expected passive mutualistic interaction. we use the term ‘passive’ because the interaction emerges spontaneously when inoculating the two auxotrophic strains together that were neither engineered  nor evolved to actively excrete the amino acid that the other strain requires. we further tested whether access to these two amino acids limits the growth of the mutualistic co-cultures. we found that the co-cultures reached stationary phase approximately four-times more rapidly when we provided exogenous supplements of the required amino acids   than when we did not , thus verifying that the supply of the required amino acids did indeed limit the growth of the mutualistic co-cultures.\n",
            "\n",
            "evolutionary dynamics\n",
            "we next investigated the evolutionary dynamics of the mutualistic co-cultures. we performed an evolution experiment by serially transferring the  <dig> replicated mutualistic co-cultures every  <dig>  days into fresh mm that was not supplemented with amino acids, corresponding to  <dig> weeks of experimental evolution. mutualistic co-cultures a1-a <dig> consisted of the egfp-expressing bw <dig>  and echerry-expressing bw <dig>  mutualistic partners while mutualistic co-cultures b <dig> to b <dig> consisted of the echerry-expressing bw <dig>  and egfp-expressing bw <dig>  mutualistic partners. we used two different combinations of fluorescent protein-encoding genes for two reasons. first, it allowed us to assess whether differences in the fluorescent protein-encoding genes themselves affect the outcome of the evolution experiment. second, it allowed us to monitor for cross-contamination among the mutualistic co-cultures during the evolution experiment, which we never detected. we measured the od <dig> of each mutualistic co-culture immediately before each subsequent transfer as a proxy of total cell numbers.\n",
            "\n",
            "we observed three qualitatively different evolutionary dynamics. for all  <dig> mutualistic co-cultures, the od <dig> decreased by about  <dig> -fold between the first and second transfers , indicating a substantial reduction in total cell numbers. thereafter, however, the mutualistic co-cultures exhibited different dynamics. for three of the mutualistic co-cultures , the od <dig> continued to decrease and fell below detection levels at the fourth transfer , indicating a persistent reduction in total cell numbers and eventual extinction. for another three of the mutualistic co-cultures , the od <dig> increased after the second transfer but then decreased again and fell below detection levels at the sixth, fourteenth, or fifteenth transfer , indicating an initial increase in total cell numbers followed by a rapid shift in growth dynamics and eventual extinction. finally, for the remaining ten mutualistic co-cultures, the od <dig> increased nearly continuously and significantly by about  <dig> -fold between the second and final transfers , demonstrating a progressive and substantial increase in total cell numbers and avoidance of extinction. these results are qualitatively consistent with a previous evolution experiment that imposed a mutualistic interaction between a methanogenic and a sulfate-reducing microorganism  <cit> . in that experiment, the authors similarly found that some mutualistic co-cultures went extinct while others progressively increased in total cell numbers and avoided extinction.fig.  <dig> experimental evolution of the mutualistic consortia. sixteen replicate mutualistic consortia were assembled from the egfp-expressing bw <dig>  and echerry-expressing bw <dig>  mutualistic partners  or from the echerry-expressing bw <dig>  and egfp-expressing bw <dig>  mutualistic partners . total cell numbers were then measured as od <dig> and tracked over time. after the mutualistic interaction was imposed, ten consortia improved in growth performance over evolutionary time while six consortia went extinct, suggesting that different consortia undergo different evolutionary trajectories\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "one limitation of our analysis above is our use of od <dig> measures as proxy measures of total cell numbers, as cell aggregation can prevent a linear correspondence between od <dig> and total cell numbers. in general, however, cell aggregation tends to decrease od <dig>  thus, the fact that we observed a general increase in od <dig> over evolutionary time as cell aggregation emerged suggests that total cell numbers also increased over the same evolutionary time. the increase in total cell numbers, however, should be interpreted as a qualitative observation rather than a quantitative measure.\n",
            "\n",
            "evolution of cell aggregation\n",
            "we next examined whether the created passive mutualistic interaction promotes the subsequent evolution of cell aggregation. to accomplish this, we used a confocal microscope to analyze the spatial structure of the mutualistic co-cultures that avoided extinction immediately before the fourteenth transfer. we found that the cells within all of these mutualistic co-cultures were significantly more aggregated together than were the cells within their corresponding ancestral mutualistic co-culture . all observed aggregates contained mixtures of both mutualistic partners. all of the evolved mutualistic co-cultures had significantly larger maximum observed k-values  than did their corresponding ancestral mutualistic co-cultures . this indicates that cells within the evolved mutualistic co-cultures were more heterogeneously distributed in space via cell aggregation than were the cells within their corresponding ancestral mutualistic co-culture. in general, aggregation was qualitatively similar within lineages but varied across lineages . we further found that cell aggregation only evolved in co-cultures consisting of strains coupled by the passive mutualistic interaction. spatial structure never evolved when we prevented the mutualistic interaction from establishing by growing the ancestral mutualistic partners together in lb liquid medium or in mm supplemented with the required amino acids . finally, there was no statistically detectable effect of the combination of fluorescent proteins used to mark each strain on the maximum observed k-values reported in fig.  <dig>  . taken together, our results demonstrate that the creation of the passive mutualistic interaction was necessary for and promoted the subsequent evolution of cell aggregation.fig.  <dig> representative images of cell aggregation within the mutualistic consortia. images were obtained immediately before the fourteenth transfer of experimental evolution. for the a <dig>  a <dig>  and a <dig> consortia, green cells are the egfp-expressing bw <dig>  mutualistic partner and red cells are the echerry-expressing bw <dig>  mutualistic partner. for the b <dig> consortium, red cells are the echerry-expressing bw <dig>  mutualistic partner and green cells are the egfp-expressing bw <dig>  mutualistic partner. note that all cell aggregates contained both mutualistic partners, but the qualitative structure and organization of the cell aggregates varied across the different consortia\n",
            "\n",
            "\n",
            "fig.  <dig> magnitude of cell aggregation within the ancestral and evolved mutualistic consortia. cell aggregation was measured as the maximum observed k-value. definitions: panc, ancestral strain of bw <dig> ; tanc, ancestral strain of bw <dig> . the data are presented as tukey box plots. all the mutualistic consortia had larger k-values than the ancestral consortia, indicating an increase in cell aggregation over the course of the evolution experiment\n",
            "\n",
            "\n",
            "b\n",
            "p\n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            "\n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            "\n",
            "aonly those consortia that avoided extinction were analyzed. bbold font: statistically significant differences between the test and ancestral mutualistic consortia . cdefinitions: panc, ancestral strain of bw <dig> ; tanc, ancestral strain of bw <dig> ; pevol, evolved strain of bw <dig> ; tevol,a, evolved strain of bw <dig> ; tevol,b, evolved strain of bw <dig> . dtevol,a and tevol,b are two genetically different strains of bw <dig>  that emerged within the b <dig> mutualistic consortium\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "one alternative explanation is that the cell aggregates consisted of dead cells rather than viable cells. to test this, we routinely plated the cultures onto agar plates. after incubation, we always observed individual colonies that expressed both fluorescent proteins . this indicates that the mutualistic partners were indeed physically attached to each other and that cells within the aggregates were viable. the cell aggregation phenotype therefore clearly contributed towards population-level phenotype. however, as shown in fig.  <dig>  individual planktonic cells remained. thus, while aggregation significantly contributed to population-level phenotype, it remained incomplete.\n",
            "\n",
            "while we observed the emergence of cell aggregation, we note that our results do not suggest that the emergence of cell aggregation was directly caused by the mutualistic interaction itself. this could occur via mechanisms such as specific partner recognition, where each mutualistic type aggregates with its partner. however, such adaptations would likely require long evolutionary times. instead, the emergence of cell aggregation in our study is likely an indirect consequence of the small amounts of amino acids that leak or are released from the mutualistic partners. in other words, the mutualistic interaction creates an environment with low concentrations of amino acids, which then promotes the evolution of non-specific cell aggregation. regardless, non-specific cell aggregation might then set the stage for further evolutionary changes, such as the emergence of partner recognition and specific attachment between the different mutualistic partners.\n",
            "\n",
            "genetic changes during emergence of cell aggregation\n",
            "we investigated the genetic changes that occurred during the evolution experiment. the goal here was not to identify the specific genetic changes that cause cell aggregation, but rather to investigate whether each lineage accumulated similar or different genetic changes during the acquisition of the cell aggregation phenotype. this then allows us to hypothesize whether there is a single or multiple evolutionary pathways to cell aggregation. to accomplish this, we reconstructed the ancestral and evolved mutualistic co-cultures from isolates. we first grew each ancestral or evolved mutualistic partner on amino acid-rich lb agar plates, assembled the mutualistic partners together into mutualistic co-cultures, and inoculated the co-cultures into mm that was not supplemented with amino acids. we found that the reconstructed evolved mutualistic co-cultures formed cell aggregates after  <dig> days of incubation while the ancestral mutualistic co-cultures did not . thus, cell aggregation was heritable and therefore likely had a genetic basis.\n",
            "\n",
            "we next sequenced the genomes of the ancestral and evolved isolates and identified genetic changes in the evolved mutualistic partners that might have caused the observed emergence of cell aggregation. we provide a complete list of all the observed genetic changes in additional file  <dig>  we excluded mutualistic co-culture a <dig> from the analysis because we could not separate the mutualistic partners into individual cells . each evolved mutualistic co-culture contained a mutualistic partner that fixed at least one genetic change located within or immediately upstream of a gene known to be involved with biofilm formation . however, as opposed to comparable experimental evolution studies and analytical methods , we observed relatively limited evolutionary parallelism of the genetic changes. more than 75%  of the genes or upstream regions that contained genetic changes were changed in only one co-culture . thus, there appears to be a large number of possible genetic targets that could result in the evolution of cell aggregation. this is not unexpected given the large number of gene products required for the regulation, initiation, development and maturation of e. coli biofilms .table  <dig> genetic changes within or upstream of genes that have experimentally verified roles in e. coli biofilm formation in other studies\n",
            "\n",
            "\n",
            "spot\n",
            "c\n",
            "hdfr\n",
            "hdfr\n",
            "flhc\n",
            "spot\n",
            "c-trmh\n",
            "spot\n",
            "c-trmh\n",
            "glmu\n",
            "c\n",
            "spot\n",
            "c\n",
            "spot\n",
            "c\n",
            "spot\n",
            "c\n",
            "glmu\n",
            "c\n",
            "gspa\n",
            "rcsf\n",
            "bama\n",
            "nlpi\n",
            "dksa\n",
            "yeap\n",
            "gpp\n",
            "blur\n",
            "\n",
            "aonly those consortia that avoided extinction were analyzed. bsubscripts indicate that more than one clone of that mutualistic partner was sequenced from the corresponding mutualistic consortium. cgenetic changes in spot and glmu have been observed in other studies where cell aggregation did not emerge\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "while we observed limited evolutionary parallelism in the genetic changes, a few genes or upstream regions were changed in mutualistic partners from more than one co-culture. genetic changes in spot or its upstream region occurred in mutualistic partners from three co-cultures . spot affects biofilm formation by modifying levels of  ppgpp  <cit>  and, under certain conditions, the inactivation of spot can enhance biofilm formation  <cit> . genetic changes in glmu occurred in mutualistic partners from two co-cultures . glmu affects biofilm formation by controlling the biosynthesis of surface adhesion molecules  <cit> . we note here, however, that genetic changes in spot and glmu have been reported in other evolution experiments where cell aggregation did not emerge  <cit> , and that further molecular work would therefore be required to test their role here. genetic changes in three flagellar genes  occurred in mutualistic partners from two co-cultures . these genes affect biofilm formation and surface attachment by regulating the biosynthesis of flagella  <cit> . finally, genetic changes in three other genes or upstream regions  occurred in mutualistic partners from two co-cultures . these genes affect biofilm formation by activating the rcs system, which regulates the biosynthesis of surface adhesion molecules and curli fimbre  <cit> . we did observe some mutations in laci, which is involved with the transcriptional regulation of the fluorescent proteins. these mutations were not unique to one mutualistic partner or another, and they are therefore unlikely to have created confounding factors that compromise our main conclusions.\n",
            "\n",
            "both mutualistic partners are required for cell aggregation\n",
            "the genome sequences demonstrate that both mutualistic partners could acquire mutations in biofilm-related genes or in regions immediately upstream of those genes. this observation then leads to the following hypothesis: both mutualistic partners could contribute towards the observed evolution of cell aggregation. to test this hypothesis, we performed a crossing experiment with the b <dig> mutualistic co-culture. based on colony morphology and genome sequences, we determined that this mutualistic co-culture had one evolved mutualistic partner of bw <dig>   and two different evolved mutualistic partners of bw <dig>   . tevol,a contains one genetic change not present in tevol,b while tevol,b contains seven genetic changes not present in tevol,a . we first grew each mutualistic partner in isolation and then mixed the evolved mutualistic partners together or mixed each evolved mutualistic partner with its corresponding ancestral mutualistic partner of bw <dig>   or bw <dig>  . we finally quantified cell aggregation of the resulting mutualistic consortia using a confocal microscope . we acknowledge here that we only performed these experiments for one lineage; the results may therefore be lineage-specific and we cannot make general statements. instead, our objective here was to investigate a single lineage in order to provide initial insight into the observed phenomena and set the stage for future investigations .fig.  <dig> magnitude of cell aggregation for different combinations of ancestral and evolved mutualistic partners from the b <dig> mutualistic consortium. cell aggregation was measured as the maximum observed k-value. definitions: panc, ancestral strain of bw <dig> ; tanc, ancestral strain of bw <dig> ; pevol, evolved strain of bw <dig> ; tevol,a, evolved strain of bw <dig> ; tevol,b, evolved strain of bw <dig> . tevol,a and tevol,b are two genetically different strains of bw <dig>  that emerged within the b <dig> mutualistic consortium. the data are presented as tukey box plots. note that the k-values  were the largest for pairs of evolved mutualistic partners, suggesting that genetic changes in both partners contributed to the observed cell aggregation\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "we found that heritable changes in both evolved mutualistic partners are required to maximize cell aggregation. mutualistic consortia of panc and tevol,a or tevol,b produced significantly more cell aggregation than mutualistic consortia of panc and tanc . in contrast, mutualistic consortia of pevol and tanc did not form significantly more cell aggregation than mutualistic consortia of panc and tanc . however, mutualistic consortia of pevol and tevol,a or tevol,b produced the most significant increase in cell aggregation . thus, within the b <dig> mutualistic consortium, the most substantial increase in cell aggregation occurred when mutations in both mutualistic partners were present together within the consortium.\n",
            "\n",
            "only one mutualistic partner contributes towards increased cell numbers\n",
            "we next asked whether both evolved mutualistic partners contribute towards increased cell numbers of the mutualistic co-cultures . to test this, we performed the same crossing experiment with the b <dig> mutualistic consortium as described above. we found that mutations in only one evolved mutualistic partner contributed towards increased cell numbers. mutualistic co-cultures of panc and tevol,a or tevol,b had significantly higher od <dig> than mutualistic co-cultures of panc and tanc and could account for all or a substantial portion of the observed increase in growth of the evolved mutualistic co-culture . in contrast, mutualistic co-cultures of pevol and tanc had significantly lower od <dig> than mutualistic co-cultures of panc and tanc . thus, genetic changes in the pevol mutualistic partner resulted in a growth defect when combined with its ancestral mutualistic partner. this suggests that the evolutionary changes in pevol were acquired in direct response to evolutionary changes in tevol,a or tevol,b, and are therefore a result of co-evolutionary dynamics between the mutualistic partners. thus, within the b <dig> mutualistic co-culture, mutations in only one mutualistic partner was sufficient to explain the observed increase in cell numbers – but not necessarily the growth rate – of the b <dig> mutualistic co-culture over evolutionary time. after these initial mutants emerge, this may then set the stage for future mutants to emerge that reinforce the mutualistic interaction. we believe these dynamics are generalizable, in that the origin of any mutualistic interaction must necessarily emerge via single genetic changes. the dynamics with respect to the further strengthening of the mutualistic interaction, where mutants of both cell-types increase in frequency, would require longer evolutionary times than investigated in this study. we acknowledge here that we are not investigating the effects of individual genetic changes, which would require introducing those genetic changes into the ancestral background. it is possible that the presence of multiple genetic changes may be critically important for our observations, such as reciprocal adaptations between the mutualistic partners.fig.  <dig> total cell numbers for different combinations of ancestral and evolved mutualistic partners from the b <dig> mutualistic consortium. total cell numbers were measured as the od <dig>  definitions: panc, ancestral strain of bw <dig> ; tanc, ancestral strain of bw <dig> ; pevol, evolved strain of bw <dig> ; tevol,a, evolved strain of bw <dig> ; tevol,b, evolved strain of bw <dig> . tevol,a and tevol,b are two genetically different strains of bw <dig>  that emerged within the b <dig> mutualistic consortium. the data are presented as tukey box plots. note that the od <dig> measurements  were the largest for consortia containing the evolved ∆trpc partner, regardless of the evolutionary history of the ∆proc partner. thus, mutations in only the ∆trpc partner are sufficient to explain the increase in od <dig> over the course of the evolution experiment\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "CONCLUSIONS\n",
            "we propose that the scenario we investigated here might be applicable to different types of organisms and interactions: for many organisms, proximity to members of the same or another genotype might be advantageous, for example because these other individuals provide resources or protection. this is expected to impose selection for biological traits that ensure proximity, through physical attachment, behavior, or by other means. this would lead to a situation that is equivalent to spatial structure; that is, a situation where individuals within or between genotypes are associated with each other for extended periods of time, as assumed in the “partner fidelity” model  <cit> . according to this scenario, mutually beneficial interactions within and between organisms are expected to evolve more readily than often assumed, because the organisms themselves can readily generate the spatial structure that is necessary for these beneficial interactions to emerge and be stable. while this scenario is consistent with our results, a conclusive test would require tracking the spatial positioning of individual cells over evolutionary time, which is not possible with our current data set. however, recent developments in analytical microbiology should now allow for such tracking and for explicit tests of this hypothesis.\n",
            "\n",
            "additional files\n",
            "\n",
            "additional file 1: table of oligonucelotide pcr primers used in this study. \n",
            "\n",
            " \n",
            "additional file 2: genetic changes observed after experimental evolution in this study. \n",
            "\n",
            " \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study, we investigated the evolution of a passive mutualistic interaction between two auxotrophic strains of the bacterium escherichia coli bw <dig>  and two evolved mutualistic partners of bw <dig>  and tested whether the passive mutualistic interaction itself promotes the evolution of cell aggregation.\n",
            "in this study, we investigated the evolution of a passive mutualistic interaction between two auxotrophic strains of the bacterium escherichia coli bw <dig>  and two evolved mutualistic partners of bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>  bw <dig>\n",
            "34\n",
            "[{'summary_text': 'our study shows that co-expression analysis using a selection of publicly available stress related data sets resulted in many new, potential components of the signal transduction pathways involved in arabidopsis defense responses.\\nthe co-expression network around ics <dig> and pbs <dig> is highly co-expressed and consists of a large number of co-expressed genes.\\nthe co-expression network around ics <dig> and pbs <dig> is highly co-expressed and consists of a large number of co-expressed genes.\\nthe co-expression network around ics <dig> and pbs <dig> is highly co-expressed and consists of a large number of co-expressed genes.\\nthe co-expression network around ics <dig> and pbs <dig> is highly co-expressed and consists of a large number of co-expressed genes.\\nthe co-expression network around ics <dig> and pbs <dig> is highly co-expressed and consists of a large number of co-expressed genes.\\nthe co-expression network around ics <dig> and pbs <dig> consists of a large number of co-expressed genes'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "plants exposed to biotic and abiotic stress activate various signal transduction pathways, like the salicylic acid -, jasmonic acid -, ethylene -, and abscisic acid -mediated signaling pathways that act singly or in combinations to evoke the most appropriate defense response  <cit> . for example, attack by pathogens results in extensive crosstalk between the sa-, ja- and et-signaling pathways, implicating complex regulatory networks underlying the plant's pathogen defense  <cit> . arabidopsis contains almost  <dig> genes encoding transcription factors  <cit>  and it is safe to assume that many are involved in regulation of these defense-signaling pathways. however, the precise regulatory mechanisms and the transcription factors involved are mostly still unknown. to fine-tune the initiated defense responses the biosynthesis and signaling pathways influence each other via crosstalk. this makes discovery of novel regulatory elements within these pathways even more challenging.\n",
            "\n",
            "the signaling that leads to defense proceeds via interactions of signaling pathway components and because of this, the genes involved are often expressed under similar conditions. this makes their expression cooperatively regulated and their expression patterns highly similar. based on this concept, an analysis of co-regulated genes under a variety of conditions can give valuable information for understanding the possible regulatory mechanisms involved in defense responses. any dataset consisting of at least two experiments can be used to perform a co-expression analysis, although for an analysis that is independent of the experimental conditions, a minimum of approximately  <dig> experiments is needed  <cit> .\n",
            "\n",
            "to investigate co-expressed genes in arabidopsis many co-expression databases from different micro-array sources with hundreds of experimental conditions per dataset have been developed in the last couple of years, such as gene expression omnibus , arrayexpress , athcor@csb.db , genevestigator , the botany array resource , arabidopsis co-expression data mining tool , atted-ii , atgenexpress/prime , and cressexpress . many of these databases only accept single-gene queries for analysis of a correlation coefficient. to obtain full flexibility in analysis method, data selection, filtering, etc., a more tailor made approach is needed. this can only be achieved after downloading the datasets and perform a manual analysis, which requires considerable computer power and knowledge about analysis methods, which is not essential for most of the available online tools.\n",
            "\n",
            "within the plant field there is an increasing number of publications that report the finding of biologically relevant genes involved in certain pathways via co-expression analysis. examples are: genes involved in root development  <cit> , genes involved in mitochondrial functions  <cit> , clusters of genes involved in primary and secondary cell wall formation  <cit> , myb transcription factors responsible for initiation of aliphatic glucosinolate biosynthesis  <cit> , and clusters of genes in a network related to cold stress and biochemical pathways  <cit> . in all these cases co-expression analysis assisted in building a network that linked unknown regulatory elements to already described pathways and helped expand hypotheses on how the genes in the network were regulated.\n",
            "\n",
            "although co-expression analysis tools are powerful in lead discovery, they cannot guarantee that observed co-expression of genes is biologically relevant. further analysis using the 'classical' genomic and/or metabolomic approaches will still be necessary to confirm the involvement of the discovered genes. despite this, co-expression analysis has proven itself as a very powerful tool in the discovery of new targets for analysis in pathways or networks of interest, as it can much more rapidly provide insight into potentially important network genes than random gain of function or loss of function approaches.\n",
            "\n",
            "here we report findings from a co-expression analysis covering a large number of microarray data sets derived from stress-induced arabidopsis. in addition to genes already known to be involved in various stress-response pathways, a large number of new candidate genes were identified that potentially participate in regulation of stress-responses.\n",
            "\n",
            "RESULTS\n",
            "public microarray data selection\n",
            "to discover new leads in the transcriptional regulation of the sa, ja and et biosynthesis and signaling pathways under stress conditions an analysis of multiple transcriptome co-expression profiles was setup. for a flexible setup that is not limited to predefined settings, datasets or processing of samples, a dataset was downloaded from the tair website . this dataset consists of  <dig> affymetrix arabidopsis 25k arrays obtained from nascarrays and atgenexpress. all microarrays were normalized by tair using the robust multi-array method .\n",
            "\n",
            "to focus on stress-related sa, ja and et biosynthesis and signaling pathways we performed a bi-clustering of all wrky transcription factors spotted on the affymetrix arrays versus a selected set of microarray data obtained from a variety of stress conditions. the stress data set of  <dig> microarrays as listed in figure 1d was selected from the total of  <dig> currently available microarrays. an overview of these  <dig> microarrays is given in additional file  <dig>  table  <dig>  for comparison, a set consisting of  <dig> development-related microarrays and a set consisting of all  <dig> available microarrays were also analyzed. hierarchical cluster trees with complete linkage and dendogram cutoffs of  <dig>  were added for both the experimental conditions and the wrky genes, and visualized using different colors. the result of this bi-clustering is shown in figure 1a. the colors of the bar below the bi-clustering matrix correspond to the colored sets of arrays as denoted in figure 1d. similar bi-clusterings of wrky gene expression profiles were performed with the subset of development-related microarrays and with the set containing all micro-arrays. the hierarchical cluster trees for the latter bi-clusterings are shown in figures 1b and 1c, respectively.\n",
            "\n",
            "it is evident that substantial differences occur in the hierarchical clustering of the wrkys between the three sets of arrays. wrky genes with coordinated expression patterns clustering close together under conditions of stress  appeared not necessarily also co-regulated during development . e.g., wrkys  <dig> and  <dig>  were clustered close together in the same sub-tree when the bi-clustering was done with the set of stress microarrays, but were situated far apart in separate sub-trees when the development-related arrays were used. the same is the case for wrkys  <dig> and  <dig> . therefore, to maximize the probability that only biologically relevant correlations were obtained, we chose to use the dataset of the stress-related microarrays listed in figure 1d to investigate co-expression of genes involved in the sa, ja and et pathways.\n",
            "\n",
            "target gene selection and co-expression cutoff determination\n",
            "to elucidate new transcription factors regulating sa, ja and et biosynthesis and signaling pathways we composed a set of genes consisting of all color-coded genes indicated in figure  <dig>  this set comprises many well-documented genes attributed to the respective stress-signaling pathways  <cit> . this set was supplemented with a set of genes encoding almost  <dig> transcription factors according to czechowski et al.  <cit>  and with the genes for the known jaz repressor proteins and a number of other known regulators of these pathways. a listing of the genes in the set is given in additional file  <dig>  table  <dig>  to determine the pearson correlation coefficient  cutoff for finding biologically relevant co-expressed genes and networks, various approaches can be applied. several of these approaches are reviewed by borate et al.  <cit>  including maximal cliques, spectral graph clustering, correlation of control spots with expressed genes, top 1% of correlations, bonferroni corrected p-values, and statistical power. the first two methods resulted in the most biological reliable pcc cutoffs. since a maximal cliques approach required more computational power than we had available and the spectral graph clustering easily results in cutoffs that are  <dig>  off, we chose to apply the approach as described by aoki et al.  <cit> . their method, based on density of the network combined with decreasing number of nodes and edges with higher pcc values, closely approaches the biological relevant pcc and is easy to implement for biologists with modest computing power. the number of nodes , edges , the network density , and the number of individual clusters obtained using the mcode algorithm was determined for different pcc cutoffs using the genes listed in additional file  <dig>  table  <dig> and figure  <dig>  the results are visualized in figure 3a-d. the total number of nodes and edges increased with a decreasing pcc threshold . in figure 3a a linear increase in the number of nodes that have at least one link with another node is found between  <dig>  and  <dig> . on the other hand, the number of edges below a cutoff of  <dig>  starts to rapidly increase , indicating that the available nodes become more densely connected as can also be seen with the increase in network density below this cutoff . the region from  <dig>  to  <dig>  in figure 3c indicates the minimum network density. according to the analysis of aoki et al.  <cit>  the most biological relevant pcc cutoff is found above these values. combined, the data of figure 3a-c leaves a relevant range for the cutoff between  <dig>  to  <dig> . to evaluate the number of clusters related to this range of closely co-regulated genes inside the network, the mcode algorithm was used to determine the number of clusters for decreasing pcc values between  <dig>  and  <dig>  at  <dig>  intervals . the number of clusters increases steadily when lowering the pcc cutoff from  <dig>  to approximately  <dig>  after which it stabilizes between  <dig>  and  <dig>  and at lower thresholds even decreases. combining the ranges of  <dig>  to  <dig>  and  <dig>  to  <dig>  made us choose the lowest overlapping cutoff of  <dig>  for where biologically significant modules are most likely to be expected. we have not investigated networks of genes that are up-regulated in one set and down-regulated in the other .\n",
            "\n",
            "using the pcc threshold of  <dig>  a co-expression network was constructed and visualized with cytoscape . the blue dots represent the selection of transcription factors and jaz proteins having at least one edge , and the colored dots represent the correspondingly colored genes from figure  <dig>  the total co-expression network thus obtained consists of  <dig> nodes that share  <dig> edges. statistical verification of our choice of cutoff by calculation of bonferroni corrected p-values cannot be applied with data sets of this size, since cutoffs of as little as  <dig>  can easily become statistically highly significant, while biological relevance at this low cutoff would be unlikely  <cit> . however, close co-expression of genes as deduced from our constructed network matched well with correlations found in literature . moreover, biochemical and functional analysis with gene sets selected from our network further supported its robustness  <cit> .\n",
            "\n",
            "exploration of co-expressed closest neighbor transcription factors of regulatory genes\n",
            "the closest neighbors with a single edge distance from the regulatory genes shown in figure  <dig> were separated in multiple sub cluster networks . the map kinase pathway from flagellin to defense genes  is depicted in figure 4a, and the map kinase pathway leading to the suppression of sa and induction of ja defense genes  is shown in figure 4b. the network of genes co-expressed with the ja biosynthesis genes  is depicted in figure  <dig>  networks of et biosynthesis  and et signaling  are shown in figures 6a and 6b, respectively. figure  <dig> shows the network of genes co-expressed with the genes leading to sa biosynthesis . a detailed description of the above networks is given in the following paragraphs.\n",
            "\n",
            "the map kinase pathways\n",
            "the response to flagellin fragment flg <dig> as part of the pamp signaling pathway is mediated via a mapk cascade  <cit> . this signal transduction via mapkkk/mekk1?-mkk4/mkk5-mpk3/mpk <dig> leads to transcriptional activation of downstream wrky <dig> and wrky <dig> genes, which results in the induction of resistance to both bacterial and fungal pathogens . our results show that the genes encoding the mapk components are highly co-expressed and form a network with a large number of co-expressed transcription factors . the known downstream target of this cascade, wrky <dig>  is connected to mekk <dig> and mkk4/mkk <dig>  surprisingly, mpk <dig> was not linked to any of the genes in the network, but appeared to be co-expressed with ein <dig> and etr <dig>  both involved in the et signaling pathway . as revealed by  <cit> , multiple different models are possible of how mpk <dig> could be regulated directly under mekk <dig>  on the other hand, mpk <dig> has been described as the map kinase substrate of mkk <dig> and the mkk3-mpk <dig> cascade is important for the ja-dependent negative regulation of myc <dig> <cit> . myc <dig> has the opposite effect on the mkk4/mpk <dig> branch. induction of erf <dig> activates a variety of wound response/insect resistance genes in ja-treated plants and regulates ja-dependent responses. erf <dig> is positively regulated by myc <dig> and in our analysis is connected to mkk <dig> and mpk <dig> <cit> . besides this connection, mkk <dig> is co-expressed with aos and opr <dig>  that are both important genes in the biosynthesis pathway of ja, suggesting that erf <dig> might activate the mkk4/mpk <dig> cascade and via this route induce ja biosynthesis. with the biosynthesis of ja, in many cases also the jaz repressor genes are positively regulated  <cit> . the connection between mkk <dig> and jaz <dig> might indicate that this branch is under control of the jaz <dig> repressor.\n",
            "\n",
            "the flagellin fragment flg <dig> not only affects the regulation of ja and et pathways, but also activates the sa pathway. many wrky genes are co-expressed with mekk <dig> and mkk <dig>  wrky <dig> is rapidly induced to very high levels upon flg <dig> treatment  <cit> . together with wrky <dig>  wrky <dig> is also co-regulated and they are both found as co-expressed genes with important genes in the sa biosynthesis pathway .\n",
            "\n",
            "both wrky <dig> and wrky <dig> are positive regulators of pr-gene expression and systemic acquired resistance . functional wrky <dig> is required for full induction of sar and is linked to the activation of pr- <dig>  <cit> . wrky <dig> enhances resistance against pseudomonas syringae  <cit> . the link between wrky <dig> and mek <dig> can be explained via mekk <dig> . mekk <dig> is upstream of mek <dig> and interacts with an activation domain protein that can be phosphorylated and binds to the promoter of wrky <dig> to activate gene expression  <cit> . this links wrky <dig> and wrky <dig> to flg <dig> and the initiation of sar mediated defense within our co-expression network.\n",
            "\n",
            "the mapk cascade , induced by challenge inoculation with p. syringae or treatment with flg <dig>  leads to phosphorylation of map kinase substrate  <dig> , which forms a complex with mpk <dig> and wrky <dig> and possibly wrky <dig>  upon phosphorylation of mks <dig>  wrky <dig> is released inthe nucleus to initiate positive regulation of ja-induced defense genes and negative regulation of sa-related defense genes. also other wrkys, like wrky <dig> and wrky <dig>  act as negative regulators of basal resistance responses  <cit> . almost all of the genes encoding these wrkys were found interconnected in the co-expression network . wrky <dig> is also stress and pathogen inducible and acts as a transcription factor that represses basal defense and pr-gene expression. when considering its location in the co-expression network, wrky <dig> could function in a similar manner as wrky11/ <dig> and/or wrky25/ <dig>  <cit> .\n",
            "\n",
            "wrky <dig> and the functional homolog wrky <dig> have dual roles in sa-mediated gene expression and resistance. upon high accumulation of sa, wrky54/ <dig> act as negative regulators of sa biosynthesis. besides this negative role, they activate other sa-regulated genes  <cit> . the route via which wrky <dig> and wrky <dig> repress sa biosynthesis is unknown. within the co-expression network both these wrkys link to both mek <dig> and mkk <dig>  two important kinases in the cascade that leads to repression of sa defense genes. it may be that negative regulation of sa biosynthesis is brought about by activation of this map kinase cascade by wrky <dig> and wrky <dig> \n",
            "\n",
            "the ja biosynthesis pathway\n",
            "the jaz repressor proteins play an important role in ja signaling. the initial jaz repressor that is released from myc <dig> to activate transcription of target genes is jaz <dig>  <cit> . myc <dig>  jaz <dig> and jaz <dig> are directly linked in the co-expression network with opr <dig>  encoding 12-oxophytodienoate reductase, an essential enzyme in ja biosynthesis . several other genes encoding jaz repressors are also connected to opr <dig> and to the gene encoding ja methyl transferase , while others link to both jmt and the gene for allene oxide synthase . the various connections of these jaz genes may hint at which levels the different jaz repressors are operational .\n",
            "\n",
            "surprisingly, many of the wrky transcription factors that are involved in positive or negative regulation of pr-genes and sar are also connected to the ja biosynthesis pathway , like the positive regulatory combinations wrky18/ <dig> , wrky54/ <dig> , wrky28/ <dig> that are possibly involved in the regulation of sa biosynthesis  and wrky11/ <dig> that act as negative regulators of sa defense genes.\n",
            "\n",
            "several members of the myb transcription factor family were also found to be closely co-expressed with the ja biosynthesis genes aos, opr <dig> and jmt. most of the co-expressed myb transcription factors have no known function. using publicly available online co-expression analyses, a link was found between myb <dig> and the regulation of aliphatic glucosinolate biosynthesis  <cit> . since methyl-ja is involved in regulation of glucosinolate biosynthesis this could indicate that myb <dig> is co-expressed at the level of jmt or below. however, the upstream connection of myb <dig> with aos suggests that activation of the glucosinolate pathway by myb <dig> is already initiated before methyl-ja is synthesized.\n",
            "\n",
            "the et biosynthesis and signaling pathway\n",
            "et is produced from s-adenosyl-methionine in a two-step reaction catalyzed by the enzymes aminocyclopropane carboxylic acid -synthase  and acc-oxidase , respectively. genes co-expressed with the et biosynthesis genes are depicted in figure 6a. we found a connection between acs2/ <dig> and mekk1/mks <dig> of the map kinase pathway. mekk <dig> has been proposed to lead to phosphorylation of mpk <dig>  although the mechanism through which this might occur has not yet been established. different models for this regulation have been proposed  <cit> . acs <dig> and acs <dig> can be phosphorylated by mpk <dig> . this phosphorylation stabilizes the protein, which results in increased et production  <cit> . other genes co-expressed with the et biosynthesis genes acs <dig>  acs <dig> and aco encode a variety of aux/iaa and arf factors. in a review by reed  <cit>  it is proposed that targets of aux/iaa and arf might include genes encoding acc synthase. various other aux/iaa and arf genes were found to be closely co-expressed with a number of other regulator genes  involved in et biosynthesis, indicative of a possible function in the integration of et and auxin signaling pathways.\n",
            "\n",
            "the map kinases in the et signaling pathway  are connected to a limited number of other nodes. the link between mpk <dig> and erf <dig> was discussed above. mutant studies with the etr1- <dig> gain-of-function et-insensitive mutant placed mpk <dig> directly downstream of etr <dig>  <cit> . this is also observed within the co-expression network. in the network ein <dig> is also connected to mpk <dig>  in the mkk9-mpk3/ <dig> cascade it was shown that direct phosphorylation in the nucleus via this cascade stabilizes the ein <dig> protein, which may be a key step in et signaling . the involvement of mkk <dig> at this point of the pathway has recently been questioned  <cit> . notably, in the co-expression network mkk <dig> doesn't correlate with any genes known to be involved at this point of the pathway, further undercutting the suggested involvement of mkk <dig> in et-signaling. within the co-expression network depicted in figure 3e both genes for etr <dig> and mpk <dig> , are in between the super cluster with the genes encoding proteins involved in sa signaling , flg22-initiated mapk kinase cascade  and the ja biosynthesis genes , and the super cluster with several genes involved in the et signaling pathway . the central location of mpk <dig> and etr <dig> between the super clusters with the other signaling genes might be indicative for a role of the combination of etr1/mpk <dig> in crosstalk between these clusters.\n",
            "\n",
            "within the ethylene-signaling network  we found many genes co-expressed with ein <dig>  for almost none of these genes a clear function has been described in literature so far. recently, it was found that the modulation of npr <dig> dependency of sa-ja crosstalk by et is dependent on ein <dig>  <cit> . most of the genes involved in the crosstalk have not yet been assigned to a particular function. surprisingly, in our analysis many of the genes that are co-expressed with ein <dig>  are also connected to cpr <dig> in the sa biosynthesis pathway . it is tempting to assume that some of these genes are involved in the ein2-dependent crosstalk with sa.\n",
            "\n",
            "the sa biosynthesis pathway\n",
            "heterodimerization of eds <dig> and pad <dig> and their nuclear localization are important for subsequent steps in the sa signaling pathway  <cit> . recently, it was found that eds <dig> expression is repressed by the ca2+/calmodulin-binding transcription factor atsr <dig>  indicating that sa levels are regulated by ca2+  <cit> . we found that the gene encoding the ca2+/calmodulin-binding transcription factor myb <dig>  is co-expressed with pbs <dig> . if myb <dig>  like atsr <dig>  acts as a repressor of sa accumulation, this might indicate another point of regulation. in addition to the link to pbs <dig>  myb <dig> is also connected to jmt in the methyl-ja synthesis pathway and to acs <dig> in the et biosynthesis pathway, suggestive for a role of myb <dig> in fine-tuning sa, ja, and et biosynthesis. besides the connections of wrky <dig> and wrky <dig> that are already known to influence biosynthesis of sa, we found two new wrky genes  to be co-expressed with isochorismate synthase  <dig> , a key enzyme in the biosynthesis of sa. as described above, wrky <dig> is known to be rapidly induced by flg <dig>  while wrky <dig> is rapidly induced downstream of avirulence effectors  <cit> . this might indicate a direct role for these wrkys in flagellin and avirulence effector-induced biosynthesis of sa. another wrky gene that we found to be co-expressed with pbs <dig> is wrky <dig>  this wrky is described in literature as one that is evolutionary highly related to wrky <dig>  <cit> .\n",
            "\n",
            "to illustrate the validity of our choice to limit the co-expression analysis to the set of stress-related micro-arrays, in figure  <dig> we focused on the sub network around ics1/pbs <dig>  in figure 8a, all genes that were found co-expressed in the stress-related set within one edge at the pcc cutoff of  <dig>  are displayed. among the co-expressed genes are wrky <dig> and pad <dig>  which are proven factors in the sa-signaling pathway  <cit> . this sub-network degraded when only the set of development-related genes  or the set of all  <dig> available micro-arrays were considered . this supports the notion that also other genes in the dataset may play roles in the stress-related pathways investigated. based on the results of the co-expression sub-network around ics <dig> and pbs <dig>  in a follow-up paper we investigated the possible role of transcription factors wrky <dig> and wrky <dig> in ics <dig> and pbs <dig> gene expression  <cit> .\n",
            "\n",
            "in figures  <dig>   <dig>   <dig> and  <dig> only co-expressed, established transcriptional regulators are depicted. a full list of all genes found to be closely co-expressed with the pathway components in figure  <dig> is given in additional file  <dig>  table  <dig> \n",
            "\n",
            "CONCLUSIONS\n",
            "our study shows that co-expression analysis using a selection of publicly available stress related data sets resulted in many new, potential components of the signal transduction pathways involved in stress responses. this could aid in the further characterization of these pathways.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "our study shows that co-expression analysis using a selection of publicly available stress related data sets resulted in many new, potential components of the signal transduction pathways involved in arabidopsis defense responses.\n",
            "the co-expression network around ics <dig> and pbs <dig> is highly co-expressed and consists of a large number of co-expressed genes.\n",
            "the co-expression network around ics <dig> and pbs <dig> is highly co-expressed and consists of a large number of co-expressed genes.\n",
            "the co-expression network around ics <dig> and pbs <dig> is highly co-expressed and consists of a large number of co-expressed genes.\n",
            "the co-expression network around ics <dig> and pbs <dig> is highly co-expressed and consists of a large number of co-expressed genes.\n",
            "the co-expression network around ics <dig> and pbs <dig> is highly co-expressed and consists of a large number of co-expressed genes.\n",
            "the co-expression network around ics <dig> and pbs <dig> consists of a large number of co-expressed genes\n",
            "35\n",
            "[{'summary_text': 'in this study, we analyzed wes data from a family with a history of nshl by focusing on three categories of genetic information: cnv, linkage, and snvs.\\nwes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\\nwes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\\nwes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\\nwe analyzed wes data from a family with a history of nshl by focusing on three categories of genetic information: cnvs, linkage, and snvs.\\nwes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\\nwes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\\nwes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\\nwes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\\nwes data can be expanded to obtain more information useful for identifying causative mutations in mendelian'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "by virtue of the recent development of massively parallel dna sequencing technologies, access to genomic composition has become easier than ever. with the advantage of exome sequencing, many studies have identified causal variants responsible for numerous disorders. exome sequencing provides a particularly powerful method with which to identify disease-causing single nucleotide variations  in mendelian disorders\n",
            " <cit> . though whole exome sequencing  has been used to successfully discover many genes that cause mendelian disorders, analysis of wes data remains challenging\n",
            " <cit> . an individual exome has more than  <dig>  variants compared with the reference genome. even in familial mendelian disorders, the overall success rate for identifying disease-causing genes is around 50%\n",
            " <cit> . thus, many of the potential reasons for failure in the wes approach need to be solved fully in order to realize the promise of wes for routine diagnosis of mendelian disorders.\n",
            "\n",
            "filtering patient data against normal populations and inferring identity-by-descent  in family studies can enrich candidate genes\n",
            " <cit> . genetic linkage analysis has also been a powerful tool for isolating potential causal candidate variants. a two-step approach of linkage analysis using single nucleotide polymorphism  microarrays to detect high logarithm of odds  score regions and subsequent targeted re-sequencing of regions of interest has been utilized in many genomic studies to intensify the power of detection\n",
            " <cit> . classically, microsatellite markers have been used for linkage analysis, and now millions of dimorphic snp markers can be used to provide higher resolution in order to pinpoint candidate loci\n",
            " <cit> . currently, there are many efforts to use coding snp information from wes data to facilitate genetic linkage mapping. specifically, coding snp data from wes can be used to establish multiphasic exome analyses based on linkages and snvs\n",
            " <cit> .\n",
            "\n",
            "copy number variation  has been implicated in both mendelian diseases\n",
            " <cit>  and common diseases such as obesity\n",
            " <cit>  and schizophrenia\n",
            " <cit> . the presence of large insertions or deletions in patients is typically investigated prior to snv analysis by karyotyping, fluorescence in situ hybridization , and/or array comparative genome hybridization . estimation of cnv is a challenging aspect of wes analysis, in which local depths of coverage must be mapped to copy numbers. indeed, acgh has limitations in detecting high cnv regions. conversely, cnv data based on wes provides more accurate copy numbers because the depths of exon coverage from wes data vary linearly with real copy numbers\n",
            " <cit> . bioinformatics tools to analyze copy numbers from wes data are now publicly available\n",
            " <cit> .\n",
            "\n",
            "nonsyndromic hearing loss  contributes to more than 70% of inherited cases of hearing loss. to date, approximately  <dig> genes have been shown to be causally related to nshl. many studies have identified more than  <dig> loci responsible for nshl; however,  <dig> loci have not yet been mapped to proper genes\n",
            " <cit> . the complexity of the auditory system may explain why so many genes and loci are linked to hearing loss. the genetic causes of hearing loss can be detected by sequence analysis, which helps clinicians and patients to delineate the basis of the disease. given that hearing loss in early childhood can affect linguistic development\n",
            " <cit> , it is important to improve current techniques for identifying genetic alterations that cause nshl. earlier identification of such alterations in patients and families may allow for better clinical management of nshl.\n",
            "\n",
            "analysis of wes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases. in this paper, in order to analyze wes data from an entire family, we applied three different methods, namely, cnv, linkage, and segregation analysis. by combining the results obtained from these methods, we efficiently identified a causative mutation from the family data. we applied this approach to wes data from a nshl family to identify candidate disease-causing variants.\n",
            "\n",
            "RESULTS\n",
            "clinical features of a nshl family\n",
            "we identified a korean family with six members affected by nshl and seven unaffected members . pure tone audiometry  was performed on nine family members, three of whom  exhibited profound post-lingual hearing loss. the three members had normal cognitive function and no anomalous-looking features. they went through a battery of clinical tests ranging from general physical examinations, chest x-rays, and simple blood tests to detailed imaging studies including brain mri and temporal bone ct scans. no abnormalities were detected in the tests, excluding the possibility that hearing loss in these patients was syndromic. the other siblings  exhibited normal hearing . patients ii- <dig>  ii- <dig>  and ii- <dig> estimated that their hearing loss became severe in their 30’s, during which time they started to wear hearing aids. their hearing loss was further aggravated and became profound in their late forties. ultimately, patients ii- <dig> and ii- <dig> no longer benefited from hearing aids and underwent cochlear implantation. they achieved recognition of common sentences without lip reading one year after implantation. gjb <dig> is one of the most frequently detected genes in individuals with nshl, and thus we first investigated the sequence of gjb <dig> in the nshl patients. after failing to identify any mutations in gjb <dig>  we performed wes on several members of the korean family in order to identify a disease-causing mutation.\n",
            "\n",
            "copy number analysis using wes data\n",
            "wes data was obtained from the parents and six siblings . the mean coverage of each sample ranged from  <dig> x to  <dig> x, and  <dig> % to  <dig> % of the targeted exome had at least  <dig> reads. a multiphasic wes analysis was designed to find a causative nshl mutation . first, we investigated co-segregation of copy number duplication or deletion in exomes of patients using contra software. we detected five cnv loci with distinct features in the plots . none of the cnvs co-segregated with affected or unaffected family members. one cnv locus of the cnvs from three members  was located in 8p <dig> , a region that contains beta-defensin genes and spag <dig> . the following genes were identified as being located at regions of distinct cnvs in the indicated family members: gstm <dig> in 1p <dig>   , ugt2b <dig> in 4q <dig>  , bntl <dig> in 5q <dig>  , and lilrb <dig> in 19q <dig>   . we also applied fisher’s exact test for the lod score per exon to detect co-segregated regions of cnvs, but there were no peaks with values reaching significance. we identified two groups based on the pattern of segregation of spag <dig>  gstm <dig>  and beta-defensin genes to validate the relevance of this method .\n",
            "\n",
            "exome linkage analysis\n",
            "because the pedigree strongly suggested an autosomal dominant mode of inheritance, we identified  <dig>  coding autosomal snvs from wes data and performed single-point linkage analysis. we identified six hot spots where a number of peaks were closely clustered . specifically, we identified peaks on chromosomes  <dig>   <dig>   <dig>   <dig>   <dig>  and  <dig> consisting of  <dig>   <dig>   <dig>   <dig>   <dig>  and  <dig> exons, respectively.\n",
            "\n",
            "we validated single-point linkages using a snp microarray containing  <dig>  snps. along with the eight initial family members recruited for wes analysis, we included three additional subjects  to validate the significance of peaks obtained from exome linkage analysis. the six hot spots detected from sequencing data were also detected in microarray analysis with a relatively high lod score . adding three more subjects to the linkage analysis enhanced the peaks at chromosomes  <dig> and  <dig>  which consisted of one and three snps , respectively. the genotype patterns of these four peaks were perfectly matched with an autosomal dominant mode of inheritance.\n",
            "\n",
            "snv analysis\n",
            "based on the wes analysis of four affected and four unaffected family members, we identified  <dig> ~ <dig>  snvs and 413~ <dig> indels. these were reduced to 962~ <dig>  snvs and 140~ <dig> indels after filtering through the dbsnp <dig> and  <dig> genome databases. fifteen variants causing amino acid changes were selected based on their co-segregation pattern within the family . all of the  <dig> variants on chromosomes  <dig>   <dig>   <dig>   <dig>  and  <dig> corresponded to regions with high lod scores . one novel mutation in actin gamma  <dig>  was identified, consisting of a methionine to threonine substitution at amino acid  <dig> , this candidate variant was validated by sanger sequencing and co-segregated with hearing loss in all family members .\n",
            "\n",
            "actg <dig>  was strictly conserved in  <dig> of  <dig> eukaryotes analyzed , with the m <dig> codon being conserved in  <dig> species. protein damage prediction analysis identified p.m305t as “possibly damaging” by humdiv, “probably damaging” by humvar in polyphen2\n",
            " <cit> , and “disease causing” by mutationtaster\n",
            " <cit> . the mutation site, met <dig>  was visualized using the 3d structure of bovine beta-actin bound by adenosine triphosphate  with profilin . the methionine was closely located to the atp molecule. additionally, met <dig> is listed as a predicted residue for the atp binding site by the protein data bank .\n",
            "\n",
            "discussion\n",
            "wes is a powerful technique that can be used to discover causative genes in human diseases. although wes has been integral in identifying more than  <dig>  novel genes in mendelian disorders\n",
            " <cit> , there is still a need for increased efficiency of gene discovery using wes data. in this regard, we analyzed wes data from a family with a history of nshl by focusing on three categories of genetic information: cnvs, linkage analysis, and snvs. utilizing these data, we undertook a stepwise multiphasic approach to identify disease-causing variations in the family.\n",
            "\n",
            "8p <dig> , which contains a beta-defensin cluster, was detected as a region with high copy number  and low copy number  . the defensin cluster, containing both alpha- and beta-defensins, was previously studied as a dynamic genomic region with varying copy numbers ranging from one to twelve\n",
            " <cit> . the parents had normal copy numbers, which was in contrast to the low copy numbers seen in two children and high copy number observed in one child. a total of four haplotypes of 8p <dig>  may have been inherited in this family, and each parent may have had both under- and over-amplified alleles of 8p <dig> . the overall copy number of a parent can appear to be normal due to compensation of copy number from over- to under-amplified alleles\n",
            " <cit> . in the family in this study, gstm <dig> and ugt2b <dig>  genes with frequently reported deletions\n",
            " <cit> , as well as bntl <dig> and lilrb <dig>  exhibited cnvs. we used fisher’s exact test on the affected and unaffected family members after validating this method for 8p <dig>  and gstm <dig> groups to determine the amplification or deletion of multiple exons that matched the co-segregation pattern of the disease. multiple statistically significant peaks at 8p <dig>  and gstm <dig> were identified, and were identical to plots from the first approach. however, there was only one statistically significant peak identified by testing the two groups that segregated with the disease, and this peak did not correlate with disease status. thus, while wes may provide a method to identify cnv regions with highly similar sequences, determining accurate copy numbers can prove difficult.\n",
            "\n",
            "linkage analysis was performed to narrow down the number of candidates based on wes data. importantly, linkage analysis with a relatively small number of markers still provides useful information. fewer markers from wes data are available and can be obtained from a snp microarray, and the markers that are identified may not be evenly distributed. given these limitations, it is necessary to consider the potential disadvantages of this approach. because we analyzed only exonic snps , we may have lost critical information located outside of exons. in addition, potential genotyping errors in linkage analyses can reduce statistical power for detecting linkage peaks or result in false positive linkage peaks\n",
            " <cit> . even so, the results obtained from the different data sets in this study confirmed the validity of our approach. linkage analysis requires a large number of subjects to help identify putative loci. unless a proper number of subjects are available, an informative result is difficult to obtain.\n",
            "\n",
            "after applying linkage analysis results, the co-segregated variants were all found to be located in the loci of high lod scores. however, linkage analysis can decrease the number of candidate variants, particularly in instances where candidate variants are widely distributed. additional linkage analysis of wes data demonstrated a similar performance to that of snp microarray data and simultaneously generated results during variant calling. considering that cnvs could be also detected using this approach, the multiphasic analysis of wes data efficiently narrowed and identified candidate variants and was advantageous compared with established methods such as initial acgh, variant calling according to wes data alone, or linkage analysis based on snp microarray data.\n",
            "\n",
            "actin is a highly conserved cytoskeletal protein that plays important roles in eukaryotic cell processes such as cell division, migration, endocytosis, and contractility. actin isoforms are classified into two groups based on expression patterns. acta <dig>  acta <dig>  actc, and actg <dig> are “muscle” actins, predominantly expressed in striated or smooth muscle, whereas actb and actg <dig> are cytoplasmic “non-muscle” actins\n",
            " <cit> . autosomal dominant progressive sensorineural hearing loss, dfna20/ <dig> , is caused by a mutation in the gamma-actin gene on chromosome  <dig> at q <dig> . some actg <dig> mutations are associated with baraitser-winter syndrome, which is characterized by developmental delay, facial dysmorphologies, brain malformations, colobomas, and variable hearing loss. the constellation of these abnormalities is suggested as the most severe phenotype of actg <dig> mutations\n",
            " <cit> . a genome-wide screen of dfna <dig> localized candidates to 17q <dig> \n",
            " <cit>  and mapped the causative missense mutations to highly conserved actin domains of the gamma-actin gene \n",
            " <cit> . in vivo and in vitro studies of actg <dig> indicate that it is required for reinforcement and long-term stability of actin filamentous structures of stereocilia, but not for auditory hair cell development, which is in line with the progressive nature of hearing loss related to actg <dig> mutations in humans\n",
            " <cit> . further, missense mutations in either actb or actg <dig> have recently been reported to cause baraitser-winter syndrome. interestingly, of the  <dig> mutations that cause dfna20\n",
            " <cit>  and  <dig> mutations that cause baraitser-winter syndrome  that have been reported, are all missense mutations. the predicted interaction between met <dig> and atp in bovine beta-actin, a protein with a 99% identity to actg <dig>  implies that the mutation of met <dig> may influence atp binding of actg <dig>  which is essential for polymerization of g-actin to f-actin.\n",
            "\n",
            "actg <dig> is predominantly expressed in intestinal epithelial and auditory hair cells\n",
            " <cit> . detection of exclusively missense mutations in this gene may imply that truncating mutations have more severe effects and might cause embryonic lethality. the hearing impaired subjects in this study  did not report any gastrointestinal complaints. the subjects in this study required cochlear implants, recapitulating what has previously been reported regarding the management of patients with mutations in actg <dig> and resultant nshl\n",
            " <cit> . the severe phenotype and rapid progression of hearing loss to a profound level within one or two decades associated with mutations in actg <dig> necessitates an early molecular genetic diagnosis and timely auditory rehabilitation.\n",
            "\n",
            "CONCLUSIONS\n",
            "two or more platforms  have previously been required to generate complex genetic information such as cnvs, linkages, snvs, and indels. in general studies of mendelian disorders, wes has primarily been utilized to obtain only snvs and indels. our study agrees well with other work demonstrating that analysis of wes data also allows for cnv and linkage determination due to its quantitative traits. given the robust nature of wes data, it is clear that the full capabilities of this relatively new technology have not yet been fully realized. our multiphasic wes analysis proved very powerful for the interpretation and narrowing of wes results, in particular when a large amount of family data is available.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study, we analyzed wes data from a family with a history of nshl by focusing on three categories of genetic information: cnv, linkage, and snvs.\n",
            "wes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\n",
            "wes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\n",
            "wes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\n",
            "we analyzed wes data from a family with a history of nshl by focusing on three categories of genetic information: cnvs, linkage, and snvs.\n",
            "wes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\n",
            "wes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\n",
            "wes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\n",
            "wes data can be expanded to obtain more information useful for identifying causative mutations in mendelian diseases.\n",
            "wes data can be expanded to obtain more information useful for identifying causative mutations in mendelian\n",
            "36\n",
            "[{'summary_text': 'our results suggest that human pit <dig> is dispensable for pi transport function, and that the overall structure of the pit family proteins has remained unchanged during evolution.\\nour results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\\nour results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\\nour results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\\nour results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\\nour results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\\nour results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "phosphate is needed by any living cell for structural and metabolic purposes. inorganic phosphate  has to be actively transported across the cell membrane against a chemical and electrical gradient. in mammalian cells this task is managed by the type iii sodium-dependent pi  symporters, pit <dig> and pit <dig>  which utilize the free energy provided by the na+ concentration gradient as the driving force for uphill import of pi <cit> , reviewed in  <cit> .\n",
            "\n",
            "the mammalian type iii transporters are part of the pi transport  family , but several members were originally identified as receptors for different retroviruses belonging to the gamma-retrovirus genus  <cit> ; thus, pit <dig> and pit <dig> are proteins with dual functions. the pit family also comprises non-mammalian members, e.g., fungus pho-4+ )  <cit>  and yeast pho <dig> )  <cit>  as well as the proton -dependent pi transporters from bacteria, pita and pitb )  <cit> , and plant pht2_ <dig> )  <cit> . furthermore, there is an increasing number of entries in the national center for biotechnology information  protein database  that show similarity to the known members of the pit family and therefore are denoted putative phosphate permeases; and pit family members have been found in all kingdoms  <cit> , reviewed in  <cit> . altogether, this suggests that the pit proteins developed very early in evolution and that this family of proteins has important function in all kingdoms of life.\n",
            "\n",
            "the first membrane topology model of pit proteins was based on kyte-doolittle hydropathy plots. analyses of human pit <dig> and pho-4+ protein sequences predicted  <dig> transmembrane  domains,  <dig> loops  hereof  <dig> extracellular, internal n- and c-terminal ends, and a large hydrophilic domain  intracellularly positioned between the putative 6th and 7th tm domains  <cit> . due to profound similarity  between the human pit paralogs, the same membrane topology model was proposed to also apply for human pit <dig>   <cit> . the model was, moreover, supported by the experimental assignment of the large intracellular domain of rat pit <dig> to the cytoplasmic space  <cit> . other topology models have, however, been proposed for pit <dig>  <cit>  and pit <dig>  <cit>  ; please see legend to figure  <dig> for more details. nevertheless, we have shown that exchanging as little as  <dig> or  <dig> amino acids in the fungal pit protein, pho-4+, with human pit <dig> or human pit <dig> sequences, respectively, results in proteins that support infection by human pit <dig> or pit <dig> cognate gamma-retroviruses  <cit> ; results, which suggest that these transporters are structurally highly related.\n",
            "\n",
            "analyzing human pit <dig> and pho-4+ sequences, johann and coworkers discovered an internal sequence repeat, which they suggested had originated from an ancient gene duplication  <cit> . both regions were shown to harbor a prodom domain, pd <dig>   <cit>  , characteristic for all members of the pit family. interestingly, all amino acids in human pit <dig>  and in human pit <dig>  identified to be critical for pi transport function are located in these prodom domains   <cit> . it should be noted, that, the prodom domain pd <dig> has changed and now consists of what corresponds to human pit <dig> i53-l <dig> http://prodom.prabi.fr . in an attempt to narrow down a pit family trait, saier aligned the n-terminal protein sequences from  <dig> members representing all kingdoms  <cit> . the author noted the existence of an 11-amino-acid-long sequence in the n-terminal region containing the conserved core sequence  and proposed it to be a signature sequence for the pit family  <cit> . however, refined studies of the n- and c-termini of  <dig> protein sequences representing pit family members from all kingdoms revealed that these proteins harbor a 12-amino-acid-long pit family signature sequence - with the common core consensus sequence  - within each of the pd <dig> prodom domains proposed in  <dig>  <cit> . furthermore, d <dig> and d <dig> shown to be critical for pit <dig> pi transport are placed in either of the pit family signature sequences  <cit> .\n",
            "\n",
            "to further investigate the importance of the pit family signature sequences, we have analyzed the human pit <dig> histidine, h <dig>  located in the c-terminal pit family signature sequence, and we show that it is indeed critical for the pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses. the human pit <dig> h <dig> is the second amino acid in this sequence to be identified as critical for pi transport function. in addition, we also show that the human pit <dig> glutamate, e <dig>  located in the pd <dig> prodom domain, is critical for the pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\n",
            "\n",
            "we have, moreover, combined studies of the gene structure of the human pit genes , alignment and tm domain prediction of protein sequences of pit family members from all kingdoms of life, and studies of the dual functions of the human pit paralogs as pi transporters and gamma-retroviral receptors, and we found that these proteins are excellent as models for studying the evolution of protein structure-function relationship. specifically based on the observation that bacterial and archaeal pit family members are substantially smaller than eukaryotic members  <cit>  and our alignment , we analyzed truncation mutants of human pit <dig>  our results clearly show that the large intracellular domain of human pit <dig> is dispensable for pi transport function, and that a fully functional pi-transporting unit can be created by the  <dig> tm domains and the small loop sequences connecting them . a further truncated human pit <dig> protein with the 5th and 6th tm domains and the large intracellular domain removed resembles the structures of as well a putative phosphate permease from archaea as of pita from bacteria  and e. coli, respectively). this mutant  was an excellent gamma-retroviral receptor  <cit> , and we here show that it can support low levels of pi transport. altogether, these results suggest that the overall structure of the pi-transporting unit of the pit family proteins has remained unchanged during evolution.\n",
            "\n",
            "methods\n",
            "sequence alignment\n",
            "protein sequence alignment of nine pit family members representing all kingdoms was made using the clustalw alignment program version  <dig> . <dig> available at the european bioinformatics institute server   <cit> . the swiss-prot protein sequences were retrieved from the ncbi protein server . accession numbers are: homo sapiens  pit <dig> , h. sapiens pit <dig> , caenhorabditis elegans  putative phosphate permease , drosophila melanogaster  putative phosphate permease , n. crassa pho-4+ , trypanosoma brucei  putative phosphate permease , a. fulgidus putative phosphate permease , a. thaliana pht2_ <dig> , and e. coli pita . all sequences encompass two 12-amino-acid-long sequences, which based on comparison of  <dig> sequences, were identified in pit proteins and related proteins and suggested to be pit family signature sequences  <cit> . we, however, observed that the c-terminal pit family signature sequence of e. coli pita did not group together with the c-terminal pit family signature sequences of the eight other species in the alignment . in order to group all the c-terminal pit family signature sequences together, the alignment was adjusted manually after an alignment of h. sapiens pit <dig> amino acids s422-v <dig> , a. fulgidus putative phosphate permease , and e. coli pita . for the adjusted protein sequence alignment of the pit family members, see additional file  <dig> figure a.\n",
            "\n",
            "prediction of tm domains in the pit family members and related proteins\n",
            "putative tm domains were predicted using the tmhmm server v.  <dig>  available at the center for biological sequence analysis, technical university of denmark , and the dense alignment surface  transmembrane prediction server available at the stockholm bioinformatics center, stockholm university . tmhmm is based on a hidden markov model  that is cyclic with seven types of states for helix core, helix caps on either side, loop on the cytoplasmic side, two loops for the non-cytoplasmic side, and a globular domain state in the middle of each loop  <cit> , and das is based on low-stringency dot-plots of the query sequence against a collection of non-homologous membrane proteins using a previously derived special scoring matrix  <cit> .\n",
            "\n",
            "in general, the predictions using both servers correspond well to each other when compared , however, the das server tends to predict shorter tm domains in agreement with the tendency for prokaryotic tm domains to be shorter in length when compared to the length of eukaryotic tm domains  <cit> . therefore, we chose to use the das server over the tmhmm server when predicting tm domains in the prokaryotic protein sequences for e. coli pita and a. fulgidus putative phosphate permease. the predicted tm domains are shown in additional file  <dig> figure a.\n",
            "\n",
            "intron-exon border analysis of human pit genes slc20a <dig> and slc20a2\n",
            "the spidey mrna-to-genome dna alignment program version  <dig>  available from the ncbi homepage   <cit>  was used to determine the location of intron-exon borders in the human pit genes. spidey takes as input an mrna sequence and the corresponding genomic sequence, and it generates an alignment that establishes the gene structure. the genbank mrna sequences were retrieved from the ncbi nucleotide server . accession numbers are: h. sapiens pit <dig> mrna  and h. sapiens pit <dig> mrna . the genomic genbank sequences were retrieved from the ncbi human genome server . accession numbers are: h. sapiens chromosome  <dig>   and h. sapiens chromosome  <dig>  . the intron-exon borders are shown in additional file  <dig> figure a on the protein sequence alignment of nine pit family members.\n",
            "\n",
            "expression plasmids\n",
            "the pcdna1artkpa-derived expression plasmids poj <dig> and poj <dig>  encoding human pit <dig> and pit <dig>  respectively, have been described  <cit> .\n",
            "\n",
            "the plasmid encoding the human pit <dig> h502a mutant was made by using the quickchange® xl site-directed mutagenesis kit  according to the manufacturer's instructions. besides the mutations creating h502a, the forward primer 5'-ttcgggtcctttgctgccggcggcaatgacgt-3' and reverse primer 5'-acgtcattgccgccggcagcaaaggacccgaa-3' also generated, by introduction of a silent mutation, an ngom iv restriction enzyme cleavage site in poj <dig>  which was used for screening. the plasmid encoding the human pit <dig> e70k mutant was made by using the altered sites ii kit  according to the manufacturer's instructions. a mutation creating e70k as well as a dra i restriction enzyme cleavage site was introduced into a palter- <dig> vector  harboring the pst  <dig> - hind iii fragment of poj <dig>  using the primer 5'-gacagagcccactgttttaaagatgctagctag-3'. finally, this construct was digested with kpn i and hind iii generating a fragment, which was used to replace the corresponding fragment in poj <dig> resulting in the desired plasmid.\n",
            "\n",
            "the plasmid encoding the human pit2Δl183-v <dig> mutant has previously been described  <cit> . the plasmid encoding the human pit2Δr254-v <dig> mutant was made using a palter- <dig> vector harboring the pst i - hind iii fragment of poj <dig>  as template in a polymerase chain reaction  with the forward primer 5'ctatagggagacccaagctttgtttatttaa3' and the reverse primer 5'gaggacctggaggaaatggaacaggaggtgtgataaagcaccttctttttg3'; the latter primer was used to create the link between the 5' sequence encoding kegals <dig> and the 3' sequence encoding h484llfh . the amplification product was digested with sse  <dig> i and hind iii and used to replace the corresponding fragment in poj <dig> resulting in the desired plasmid.\n",
            "\n",
            "the authenticities of all the nucleotide sequences were confirmed.\n",
            "\n",
            "the plasmids were purified using either cesium chloride  according to the protocol described by maniatis and coworkers  <cit> , or using nucleobond  or qiagen maxiprep  according to the manufacturer's instructions.\n",
            "\n",
            "cell cultures\n",
            "chinese hamster ovary k <dig> cells, cho k <dig>  and dog osteosarcoma cells, d <dig> , were cultivated as described  <cit> ; mus dunni tail fibroblasts, mdtf  were cultivated in dulbecco's modified eagle's medium supplemented with 10% fetal bovine serum , 100iu per ml of penicillin , and  <dig> μg of streptomycin  per ml . a-mlv , 10a <dig> mlv, and gibbon ape leukemia virus  pseudotypes of the β-galactosidase-encoding transfer vector g1bgsvn  <cit>  were obtained from the producer cell lines pa317gbn, pt67gbn, and pg13gbn, respectively  <cit> . pt67gbn was established as described  <cit> . all packaging cells were cultivated in dmem supplemented with 10% newborn calf serum  and ps . feline leukemia virus subgroup b  vector pseudotypes carrying the g1bgsvn transfer vector were made essentially as described  <cit> . vectors were harvested as supernatants from confluent producer cells, and the vector containing supernatants were filtered  and stored at -80°c.\n",
            "\n",
            "transient transfection and infection assay\n",
            "transient transfection-infection assays were performed essentially as described  <cit> . briefly, cho k <dig> cells seeded in 60-mm-diameter dishes at  <dig> ×  <dig> cells per dish were transfected with  <dig> μg per dish of plasmid dna encoding human pit <dig> , human pit <dig> , human pit <dig> h502a, human pit <dig> e70k, or equimolar amounts to human pit <dig> of human pit2Δr254-v <dig> or human pit2Δl183-v <dig>  mock treated cells were transfected with empty vector dna . three independent precipitates were made per construct. forty-eight hours after transfection, approx.  <dig> to  <dig> ×  <dig> 10a <dig> mlv or a-mlv pseudotypes carrying the g1bgsvn transfer vector were added per dish in the presence of polybrene. forty-eight hours later, the dishes were stained and evaluated. infection was analyzed by counting the number of β-galactosidase-positive  cells per dish. analyses for felv-b and galv receptor functions were performed on mdtf cells using  <dig>  ×  <dig> cells and approx.  <dig>  to  <dig>  ×  <dig> vector pseudotypes per dish. numbers of vector pseudotypes used in the experiments were calculated from the number of β-galactosidase-positive colonies per ml obtained on d <dig> cells as described  <cit> .\n",
            "\n",
            "32pi transport assay\n",
            "female xenopus laevis  frogs were obtained from nasco  and kept and handled according to guidelines from the danish animal experiments inspectorate. oocytes were isolated from frogs anesthetized in a  <dig> - <dig> % ms. <dig>   solution for 10- <dig> minutes. a 1- <dig>  centimeters incision was made in the abdomen and several ovaries were removed surgically by authorized personnel. the oocytes were manually dissected and subsequently collagenase  treated and maintained in modified barth's solution  at 18°c as described  <cit> . the following day, the oocytes were used for crna injection and subsequent analyses of 32pi uptake essentially as described previously  <cit> . briefly, crnas were prepared from apa  <dig>  or bln  <dig>  linearized plasmid preparations applying the mmessage mmachine kit . stage v-vi oocytes were microinjected with  <dig>  ng of crna  and incubated at 18°c. after two to three days, the oocytes were washed in phosphate-free uptake solution , and hereafter incubated in uptake solution containing  <dig>  mm kh232po <dig>  at rt for  <dig> hour. the oocytes were washed in ice-cold uptake solution containing  <dig> mm kh2po <dig> and the 32pi uptake of each oocyte measured in a liquid scintillation counter as described previously  <cit> . it should be noted that factors coupled to the health and husbandry of the female x. laevis frogs can influence the oocyte batches. these factors include nutrition, season of the year , water temperature, salinity and hardness of the water, water contaminants or toxins, and diseases  <cit> , and the impact is that different batches of oocytes injected with crnas encoding the same proteins exhibit different average transport capacities.\n",
            "\n",
            "statistical analysis\n",
            "the null hypothesis that two mean values are identical was tested by a two-tailed student's t-test. the test compares the actual difference between two mean values in relation to the variation in the data . the null hypothesis was rejected, e.g., the mean values were considered different when p< <dig> .\n",
            "\n",
            "RESULTS\n",
            "human pit <dig> e <dig> and human pit <dig> h <dig> are critical for pi transport function but dispensable for gamma-retroviral receptor function\n",
            "in a former study, we identified the putative 2nd-tm domain-positioned human pit <dig> e <dig> as being critical for pit <dig> pi transport function   <cit> . the human pit <dig> paralog, human pit <dig>  harbors a corresponding glutamate in position  <dig>  e <dig>  to investigate whether this conserved residue was important for pit <dig> pi transport function, it was mutated to a lysine generating the mutant human pit <dig> e70k. in the experiment shown in figure 3a, oocytes injected with crna encoding human pit <dig> supported a 32pi uptake of  <dig>  ± <dig>  pmol/oocyte-hour at ph  <dig>  in agreement with previous results obtained addressing the na32pi uptake function of human pit <dig> in x. laevis oocytes  <cit> . the pi transport function of human pit <dig> e70k was severely impaired when compared to that of wildtype pit <dig> ) ; see additional file  <dig> for data and statistics to figure  <dig> \n",
            "\n",
            "besides being pi-transporting proteins, the mammalian pit proteins also serve as gamma-retroviral receptors, and this dual-function allows for analyzing whether a mutated pit protein is properly processed, folded and translocated to the cell surface  <cit> . the human pit <dig> e70k mutant was therefore analyzed for gamma-retroviral receptor function using a transient transfection-infection assay  <cit> . for the infection assay, retroviral vectors harboring a β-galactosidase encoding transfer vector and carrying viral surface proteins responsible for receptor recognition were used; vectors carrying, e.g., 10a <dig> mlv surface proteins are referred to as 10a <dig> mlv vector pseudotypes. eukaryotic expression plasmids encoding human pit <dig> and human pit <dig> e70k mutant protein were transfected into cho k <dig> cells non-permissive for infection by 10a <dig> mlv vector pseudotypes   <cit> . the abilities of these proteins to support infection by 10a <dig> mlv vector pseudotypes were analyzed; the infection levels were evaluated as the number of β-galactosidase positive  cells per 60-mm-diameter dish. cho k <dig> cells expressing human pit <dig> were permissive for infection by 10a <dig> mlv vector pseudotypes  in agreement with pit1's well-described receptor function for 10a <dig> mlv  <cit> . moreover, the human pit <dig> e70k mutant supported wildtype pit <dig> levels of 10a <dig> mlv infection ,  <dig> ± <dig> blue cells per dish , p =  <dig> ) . besides being a receptor for 10a <dig> mlv, pit <dig> is also a receptor for galv  <cit>  and for felv-b  <cit> . the human pit <dig> e70k protein was analyzed in parallel for receptor function for vector psedotypes of these two viruses in non-permissive mus dunni tail fibroblasts and found to sustain wildtype pit <dig> infection levels of galv ,  <dig> ± <dig> blue cells per dish , p =  <dig> ) and felv-b ,  <dig> ± <dig> blue cells per dish , p =  <dig> ). the wildtype receptor functions of pit <dig> e70k confirm that the overall membrane topology is preserved and that the processing to the cell surface was unaffected by the e70k-mutation.\n",
            "\n",
            "the glutamate e <dig> in human pit <dig> is conserved in eukaryotic pit family members as are the other two human pit <dig> residues   previously shown to be critical for pi transport function  <cit> . since the corresponding glutamate and serine residues in human pit <dig> have already been identified as being critical for pi transport function  <cit> , this demonstrate that equivalent glutamate or serine residues in the human pit paralogs both are critical for their pi transport functions. these observations illustrate that it is highly likely that the other conserved amino acids identified in human pit <dig> as being critical for pi transport function also are important for the transport function of human pit <dig> and other pit family members.\n",
            "\n",
            "the histidine residue, human pit <dig> h <dig> is positioned in the 7th tm domain according to the johann topology model   <cit> . it is, moreover, located in the c-terminal pit family signature sequence and conserved in eukaryotic pit family members  <cit>  . moreover, analysis of  <dig> sequences of bacterial pit family members revealed only  <dig> sequences without the histidine residue illustrating that this residue is also highly preserved in the c-terminal pit family signature sequence of pit family members belonging to this kingdom  <cit>  . since the conserved aspartic acid in the c-terminal pit family signature sequence, that is human pit <dig> d <dig>  is critical for pi transport of pit <dig>  <cit> , we hypothesized that other conserved amino acids in this motif might be critically involved in pi transport function of human pit <dig> and other members of the pit family as well. mutation of human pit <dig> h <dig> to alanine created the mutant denoted pit <dig> h502a. this mutant was analyzed for 32pi transport function in x. laevis oocytes  and 10a <dig> mlv and a-mlv receptor functions in cho k <dig> cells .\n",
            "\n",
            "in the experiment shown in figure 3b, oocytes injected with crna encoding human pit <dig> supported a 32pi uptake of  <dig>  ± <dig>  pmol/oocyte-hour at ph  <dig>  in agreement with former studies addressing the na32pi uptake function of human pit <dig> in x. laevis oocytes  <cit> . the pi transport function of human pit <dig> h502a was severely impaired when compared to that of wildtype pit <dig> ) .\n",
            "\n",
            "to analyze whether the human pit <dig> h502a mutant is properly folded and processed to the cell surface, it was also analyzed for gamma-retroviral receptor function using the transient transfection-infection assay  <cit> . eukaryotic expression plasmids encoding human pit <dig> and human pit <dig> h502a mutant protein were transfected into cho k <dig> cells non-permissive for infection by a-mlv and 10a <dig> mlv vector pseudotypes   <cit> . cho k <dig> cells expressing human pit <dig> were permissive for infection by both a-mlv and 10a <dig> mlv vector pseudotypes  in agreement with pit2's well-described receptor function for a-mlv and 10a <dig> mlv  <cit> . moreover, the human pit <dig> h502a mutant supported wildtype pit <dig> levels of 10a <dig> mlv infection ,  <dig>  ± <dig> blue cells per dish , p =  <dig> )  and a-mlv infection ,  <dig>  ± <dig> blue cells per dish , p =  <dig> ) . these results demonstrate that the overall membrane topology of human pit <dig> h502a is preserved, and that the processing of human pit <dig> h502a to the membrane surface is unaffected by the mutation. thus, histidine  <dig> in the 7th tm domain is the second amino acid - besides d <dig> - in the c-terminal pit family signature sequence , which has been shown to be essential for human pit <dig> pi transport function. while the exact role of the histidine residue in the c-terminal signature sequence still needs to be revealed, its critical role for human pit <dig> pi transport function emphasizes the importance of the c-terminal pit family signature sequence in the physiological function of the pit proteins.\n",
            "\n",
            "besides human pit <dig> e <dig> and human pit <dig> h <dig>  six conserved amino acids in human pit <dig> and two corresponding positions in human pit <dig> have previously been identified as being critical for pi transport function  <cit> . all these amino acids are located in the prodom domains  suggested in  <dig> to define members of the pit family   <cit> . therefore it is likely that sequences outside these two domains might be dispensable for the pi transport function of the pit proteins, and that a minimal pi-transporting unit of the pit proteins can be identified.\n",
            "\n",
            "alignment of protein sequences of pit family members from all kingdoms\n",
            "a previously published alignment of human pit <dig> and human pit <dig> protein sequences shows that the l <dig> loop - the large intracellular domain - is the region where these sequences diverge the most  <cit>  . moreover, alignment of human pit <dig> and n. crassa pho-4+ shows that the large intracellular domain  is smaller in pho-4+, whereas the rest of the pho-4+ protein sequence aligns well with the protein sequence of human pit <dig>  <cit>  . to further address this, we counted the number of amino acids in the large intracellular domain  of nine different pit family members and plotted the lengths according to their phylogenetic relationship in figure 4a. the figure shows that pit family members from archaea and bacteria harbor the shortest l <dig> loops whereas the pit-proteins from chordates harbor the longest l <dig> loops . note that the l <dig> loop of the c. elegans putative phosphate permease is unexpectedly short , and according to the plot we would have expected a l <dig> loop length for this protein in the interval between  <dig> and  <dig> amino acids . the observed differences in the l <dig> loop lengths of pit family members from different species thus suggest that the l <dig> loop evolved from being a regular loop to become a regular domain during evolution. in order to address this issue, we counted the number of amino acids in all loops  in the nine pit family members and plotted the average loop lengths ±sem in figure 4b. the figure shows that the l <dig> loop in average is much larger than all other loops ; see additional file  <dig> for data to figure 4b. the figure also shows that the variation in the l <dig> loop lengths is substantial , see figure 4b legend for discussion. thus, with 95% confidence the longest regular loop is the l <dig> loop with a maximum length of  <dig> amino acids, see legend to figure 4b for discussion. the definition of the maximum length of a loop also has the impact that the l <dig> of e. coli pita consisting of  <dig> amino acids  has to be considered a domain. in summary, analysis of the sizes of the loop sequences l <dig> to l <dig> in nine pit family members from all kingdoms led to the determination of a limit of maximum  <dig> amino acids in a regular loop sequence - and sequences longer than  <dig> amino acids are highly likely domains. in support of our calculations of the maximum loop length for pit-proteins is a previous study of  <dig> transmembrane domain-containing sequences, with  <dig> sequences being multi-transmembrane spanning, showing that ~90% of the loops are shorter than  <dig> amino acid residues  <cit> . another study supporting our finding is the analysis of loops in  <dig> existing 3d structures of transmembrane proteins showing that the majority of loops connecting transmembrane domains are shorter than  <dig> amino acid residues  <cit> .\n",
            "\n",
            "the proteins in figure 4a with l <dig> loop sizes smaller than  <dig> amino acids are the archaeal putative phosphate permease and the bacterial pita protein, implying that single cell organisms without nuclei that rarely harbor membrane-bound organelles cope without the large intracellular domain, whereas single cell animals  with nuclei and membrane-bound organelles have distinct l <dig> domains as shown for the t. brucei putative phosphate permease . altogether this suggest a role for the large intracellular domain, which is not directly related to pi transport per se, and it also suggest that the large intracellular domain  may have increased in length during the evolution from archaea to chordata as a consequence of adaptation to more complex environments.\n",
            "\n",
            "besides a difference in the lengths of l <dig>  a difference in the number of tm domains in the pit family members was observed . the illustration of tm domain conservedness  and tm domains, which are suggested by us to be present but not predicted by protein sequence analysis using the tmhmm server , shows the following conservedness of tms: tm  <dig>  tm  <dig>  tm  <dig>  > tm  <dig>  tm  <dig>  > tm  <dig>  tm  <dig>  tm  <dig> > tm  <dig> > tm  <dig>  . the most prominent observation is that e. coli pita and a. fulgidus putative phosphate permease both lack the 5th and 6th tm domains . this in addition to the previous observation that these two proteins also lack the l <dig> domain , suggest that the 5th and 6th tm domains and the l <dig> domain are dispensable for pi transport function, and that a basic pi-transporting unit of the pit family members can be identified. this unit would consist of regions flanking the large intracellular domain  but highly likely also be devoid of the 5th and 6th tm domains. interestingly, in support of this theory, drawing of the putative topology models for human pit <dig>  e. coli pita, and a. fulgidus putative phosphate permease based on the alignment in additional file  <dig> figure a, shows that the bacterial and archaeal proteins have a predicted eight tm backbone where the n-terminal pit-family signature sequence is placed in the 1st extracellular loop  and the c-terminal pit family signature sequence is placed in the 3rd extracellular loop  . in comparison, the drawing of the putative topology model for human pit <dig> shows a backbone of  <dig> tm domains where the n-terminal and c-terminal pit-family signature sequences are placed in the 1st extracellular loop  and the 4th extracellular loop , respectively . an interpretation of these drawings could be that the intra-protein locations of the n-terminal and c-terminal pit-family signature sequences are of importance, and that tm  <dig> to tm  <dig> and tm  <dig> to tm  <dig> constitute a core sustaining the pi-transporting function whereas tm  <dig> and tm  <dig> and the large intracellular domain  constitute a regulatory unit. finally, the amino acids identified as being critical for pi transport function are located in the prodom domains suggested in  <dig>   <cit>   in agreement with the 5th and 6th tm domains and the large intracellular domain  might be dispensable for the pi transport function.\n",
            "\n",
            "design of human pit <dig> truncation mutants\n",
            "to identify the minimal pi-transporting unit, two human pit <dig> truncation mutants were analyzed. they were designed to address the pi transport function and the gamma-retroviral receptor functions of: 1) a human pit <dig> mutant protein, which consists of the  <dig> tm domains and a l <dig> loop of  <dig> amino acids  creating the mutant human pit2Δr254-v <dig>  the human pit2Δr254-v <dig> mutant does not resemble a naturally occurring homolog found in lower species, and it is merely designed to address if the large intracellular domain is dispensable for na+-dependent pi-uptake , and 2) a human pit <dig> mutant protein that resembles an archaeal and bacterial homolog with respect to protein composition, i.e., lacking the 5th and 6th tm domains and the large intracellular domain   . note that in the salaün model the 5th and 6th tm domains correspond to tmvi and tmvii .\n",
            "\n",
            "the large intracellular domain  of human pit <dig> is dispensable for pi transport function whereas the fragment l183-v <dig> is more critical for pi transport function\n",
            "the na+-dependent 32pi transport function of wildtype human pit <dig> and the human pit2-derived truncation mutants pit2Δl183-v <dig> and pit2Δr254-v <dig>  were analyzed in x. laevis oocytes .\n",
            "\n",
            "oocytes injected with crna encoding human pit <dig> supported a 32pi uptake of  <dig>  ± <dig>  pmol/oocyte-hour  and  <dig>  ± <dig>  pmol/oocyte-hour  at ph  <dig>  in agreement with previous results  <cit> .\n",
            "\n",
            "the 32pi transport activities of the pit <dig> mutant lacking the major part of the large intracellular domain, human pit2Δr254-v <dig>   and  <dig>  ± <dig>  pmol/oocyte-hour ) were indistinguishable from those of pit <dig>   and p =  <dig>  ); see additional file  <dig> for data and statistics to figure  <dig>  thus, the large intracellular domain of human pit <dig> but  <dig> amino acids  is dispensable for its pi transport function.\n",
            "\n",
            "the 32pi transport activity of the human pit <dig> mutant lacking the large intracellular domain as well as the 5th and 6th tm domains, pit2Δl183-v <dig> , was severely impaired  and  <dig>  ± <dig>  pmol/oocyte-hour ) when compared to the pi transport function of wildtype pit <dig>  and p =  <dig>  ). however, interestingly the mutant did support low levels of pi uptake significantly different from h2o-injected oocytes  and  <dig>  ± <dig>  pmol/oocyte-hour )  and p =  <dig>  ).\n",
            "\n",
            "viral receptor function of mutant pit <dig> proteins\n",
            "using the transient transfection-infection assay, we analyzed whether the deletions in human pit <dig> affected their viral receptor functions for a-mlv and 10a <dig> mlv. eukaryotic expression plasmids encoding human pit <dig> and the mutant proteins were transfected into cho k <dig> cells. as expected, human pit <dig> transfected cells were permissive for infection by both 10a <dig> mlv and a-mlv vector pseudotypes . while the human pit <dig> truncation mutant lacking the large intracellular domain, human pit2Δr254-v <dig>  was a fully functional pi transporter , it only supported low levels of pit <dig> cognate gamma-retroviral infection . note that human pit2Δr254-v <dig> was tested once for a-mlv receptor function and twice for 10a <dig> mlv receptor function. the a-mlv study was done in parallel to a 10a <dig> mlv receptor function study using the same set of plasmid precipitates. interestingly, the human pit <dig> truncation mutant lacking the 5th and 6th tm domains in addition to the large intracellular domain, human pit2Δl183-v <dig> , supported substantial levels of pit <dig> cognate gamma-retroviral infection   <cit>  showing that its low levels of pi transport function were not due to incorrect processing of this mutant to the cell surface.\n",
            "\n",
            "athe experimental setup is described in the text. a-mlv and 10a <dig> mlv vector pseudotypes were tested on the same precipitates made from cscl-purified plasmids in experiment  <dig>  in experiment  <dig>  qiagen maxiprep-purified plasmids were used for preparing precipitates and only the 10a <dig> mlv vector pseudotype was tested.\n",
            "\n",
            "breceptor and mutant receptor sequences were cloned into pcdna1artkpa.\n",
            "\n",
            "cthe data are averages of three independent transfections ± sem. the average number of blue cells per three 60-mm-diameter dishes transfected with a plasmid encoding pit <dig> was assigned a value of 100% .\n",
            "\n",
            "ddata are from bøttger and pedersen  <dig>  <cit> ; please see article for details.\n",
            "\n",
            "end, not determined.\n",
            "\n",
            "fvalues are based on the detection limit of  <dig> blue cell per three 60-mm-diameter dishes.\n",
            "\n",
            "pit <dig> regions directly involved in receptor function for 10a <dig> mlv and a-mlv have also been identified by expression of chimeric proteins in cho k <dig> cells and were found to be located in the putative extracellular loops  <dig>  and  <dig>    <cit> . both of the human pit <dig> mutants, pit2Δr254-v <dig> and pit2Δl183-v <dig>  harbor extracellular loops  <dig>  and  <dig>  according to the johann pit <dig> model . based on their - here identified - pi transport abilities, it is unlikely that pit2Δr254-v <dig> is less expressed at the cell surface than pit2Δl183-v <dig>  and the observation that the less truncated human pit <dig> mutant protein is a worse gamma-retroviral receptor than a more heavily truncated human pit <dig> mutant protein might instead reflect a disturbance of the folding and/or conformation of the extracellular loops  <dig>  and  <dig>  due to the sole presence of the extracellular loop  <dig>  without the large intracellular domain in pit2Δr254-v <dig> \n",
            "\n",
            "intron-exon borders of the human pit genes slc20a <dig> and slc20a2\n",
            "the human pit proteins are encoded by genes that localize to different chromosomes. the human gene, slc20a <dig>  encoding the pit <dig> protein is located on chromosome  <dig> at position q <dig>  <cit> , and the human gene, slc20a <dig>  encoding the pit <dig> protein is located on chromosome  <dig> at position p <dig>   <cit> .\n",
            "\n",
            "to analyze the gene structure of slc20a <dig> and slc20a <dig>  the intron-exon borders in each of the genes were determined using the spidey mrna-to-genome dna alignment as described in \"methods\". the intron-exon borders are marked with stars  and vertical lines in the pit <dig> and pit <dig> protein sequences in the alignment of nine pit family members in additional file  <dig> figure a.\n",
            "\n",
            "eight out of nine intron-exon borders  in slc20a <dig> and slc20a <dig> are predicted to be homologous. one intron-exon border  and f <dig> ) are displaced giving a gap corresponding to  <dig> amino acids . these two borders are placed in the middle of the genome sequences, which encode the large intracellular domain  of the human pit proteins. as seen from additional file  <dig> figure a, the alignment between the human pit proteins in this region is poor and the gap highly likely reflects this, and not a significant difference in intron-exon structure between slc20a <dig> and slc20a <dig> \n",
            "\n",
            "interestingly, in support of the theory that the 5th and 6th tm domains can be dispensable for pi transport function, is the observation that these tm domains are encoded by two different exons, see additional file  <dig> figure a , and therefore the possibility exists that the sequences in these two exons have entered later in evolution.\n",
            "\n",
            "specialized functions of the mammalian pit proteins\n",
            "mammalian pit proteins are expressed in all tissues investigated and due to their broad expression profiles, they have been suggested to accommodate house-keeping functions, i.e., supplying cells with pi to maintain basic cellular functions  <cit> . however, in recent years additional specialized functions of the pit proteins have been reported. these include roles for pit <dig> in proximal tubule phosphate reabsorption  <cit> , and for pit <dig> in regulation of parathyroid gland pth production  <cit> , cell proliferation  <cit> , and in tumor necrosis factor  induced apoptosis  <cit> . recent studies also indicate that both the pit proteins function as pi sensors  <cit> , reviewed in  <cit> . interestingly, some of these functions, that is, pit2's suggested role in pi sensing  <cit>  and pit1's role in cell proliferation and tnf-induced apoptosis  <cit>  have been shown to be independent of the pi transport functions of the proteins.\n",
            "\n",
            "pit <dig> has also been implicated in normal chondroblastic and osteoblastic differentiation and mineralization processes  <cit> , as well as trans-differentiation of vascular smooth muscle cells to cells with characteristics of chondro-/osteoblasts in the pathologic process of vascular calcification at hyperphosphatemia  <cit> . more rodent in vivo models have been used to study the role of pit <dig> in normal bone formation and/or embryonic development. rats with transgenic overexpression of pit <dig> showed no major bone deformity during skeletal development  <cit> . however, these rats displayed a slight but significant decrease in the bone mineral content of the whole skeleton together with a reduction albeit non-significant in the total bone area  <cit> . the role of pit <dig> during embryonic mouse development has been studied by two different groups employing early conditional excision of slc20a <dig> exons 3- <dig>  <cit>  and slc20a <dig> exon  <dig>  <cit> , which resulted in homozygous embryonic lethality. both studies find that the embryos are anemic and do not survive past e <dig> , at which stage the morphology shows reduced growth  <cit> ; the anemia was found to be due to severe defects in liver development  <cit> . comparison of wildtype mice to mice with low  expression of pit <dig> mrna showed that some of the latter mice displayed impaired bone mineralization at birth, while 15-days old mice showed no major differences in mineralization  <cit> . interestingly, in embryos  lacking\n",
            "\n",
            "pit <dig> expression beck and coworkers found an upregulated pit <dig> expression, which however could not rescue the embryos past e <dig> , and the authors therefore suggest that the critical non-redundant role of pit <dig> in development is not pi-uptake  <cit> . altogether, the in vivo studies do not exclude a role for pit <dig> in normal bone formation, although they imply that pit <dig> is not critical for the early skeletal developmental processes.\n",
            "\n",
            "the alignment and analyses of exon structure together with the observed pi transport functions of the pit <dig> deletion mutants presented here might suggest that the regions of the pit proteins involved in the pi-transport independent functions map to sequences in the 5th and 6th tm domains and/or in the large intracellular domain. in line with this, we are currently investigating the function of the large intracellular domain of the human pit <dig> protein and our results support the hypothesis that the large intracellular domain has other functions than pi transport.\n",
            "\n",
            "CONCLUSIONS\n",
            "investigation of the pi transport and retroviral receptor functions of the human pit proteins has allowed for identification of a histidine residue  in the c-terminal pit family signature sequence as being critically involved in pi transport function. moreover, we show that a pit <dig> glutamate residue  positioned in the 2nd tm domain is critical for pi transport function in agreement with the former identification of the equivalent glutamate in human pit <dig>  as being critical for pi transport function  <cit> .\n",
            "\n",
            "we have shown that a human pit <dig> mutant consisting of the  <dig> tm domains and minor loops  transports pi as wildtype pit <dig>  proving that the large intracellular domain  is dispensable for pi transport function. a further truncated human pit <dig> mutant consisting of the 1st to 4th tm domains linked to the 7th to 10th tm domains and the minor loop sequences connecting the tms , and which resembles archaeal and bacterial homologs, sustained low levels of pi transport. this protein harbors the prodom domains defining the pit family members and, moreover, harbors all the amino acids so far identified as being critical for pi transport function.\n",
            "\n",
            "the above results showing that truncated human pit <dig> mutant proteins - one of which resembles a phosphate permease from bacteria and a putative phosphate permease from archaea - support pi transport, point to the conclusion that the overall structure of the pit family proteins has remained unchanged during evolution and that a basic pi-transporting unit exists.\n",
            "\n",
            "list of abbreviations used\n",
            "10a <dig> mlv: retrovirus closely related to a-mlv, a-mlv: amphotropic murine leukemia virus, cho k1: chinese hamster ovary k <dig>  galv: gibbon ape leukemia virus, felv-b: feline leukemia virus subgroup b, pi: inorganic phosphate, pit: pi transporter, tm: transmembrane.\n",
            "\n",
            "authors' contributions\n",
            "pb and lp conceived the study and designed the experiments. pb did the experimental work, and drafted the manuscript. pb and lp edited and approved the final version of the manuscript.\n",
            "\n",
            "supplementary material\n",
            "additional file 1\n",
            "protein sequence alignment of nine pit family members from all kingdoms. a the  <dig> putative tm domains according to the johann topology model are shown on the human pit <dig> sequence using black boxes with white filling  <cit> ; the putative large intracellular domain  of human pit <dig>  according to this model, spans the amino acid sequence: p236-v <dig>  the n-terminal and c-terminal pit family signature sequences  <cit>  are shown on the alignment in black boxes with grey filling. human pit <dig> e <dig> in the 2nd tm domain and human pit <dig> h <dig> in the 7th tm domain are indicated with circles. the tmhmm-predicted tm domains of the eukaryotic protein sequences for pit family members and the das-predicted tm domains of the prokaryotic protein sequences for pit family members are shown in black bold. the red bold sequences represent tm-domains, which we suggest exist, however, they were not predicted by the servers: n. crassa pho-4+ tm  <dig>  is suggested to be homologous to the tm  <dig> predicted in the c. elegans putative phosphate permease protein sequence. the presence of pho-4+ tm  <dig> is also based on the assumption that the n-terminal pit-family signature sequences should be placed equivalently  in all pit family members. a. thaliana pht2_ <dig> tm  <dig>  is suggested to be homologous to the tm  <dig> predicted in the t. brucei putative phosphate permease protein sequence. the presence of pht2_ <dig> tm  <dig> is also based on experimental assignment of the l <dig> for rat pit <dig> to the cytoplasmic space  <cit> , and pht2_ <dig> therefore requires a tm  <dig> to fulfill this criteria. h. sapiens pit <dig> tm  <dig>  is suggested to be homologous to the tm  <dig> predicted in the h. sapiens pit <dig> protein sequence. investigation of a human pit1/pit <dig> chimera where the pit <dig> backbone harbors the human pit <dig> sequence g120-v <dig> showed that this sequence conferred a-mlv receptor function upon human pit <dig>  <cit> , and the g120-v <dig> sequence is therefore highly likely extracellular in both human pit paralogs and this requires the presence of tm  <dig> in human pit <dig>  tm  <dig> domains in putative phosphate permeases from c. elegans , d. melanogaster , t. brucei , and n. crassa pho-4+  are suggested to be homologous to the tm  <dig> predicted in h. sapiens pit <dig> and pit <dig> sequences. the presence of tm  <dig> in putative phosphate permeases from c. elegans, d. melanogaster, t. brucei, and n. crassa pho-4+ is also based on the assumption that the c-terminal pit-family signature sequences should be placed equivalently  in all pit family members. moreover, investigation of a pho-4+/human pit <dig> chimera where the pho-4+ backbone harbors the human pit <dig> sequences c117-i <dig>  and l512-a <dig>  showed that these sequences confer a-mlv receptor function upon pho-4+ <cit> , and these sequences are therefore highly likely extracellular and this requires the presence of a tm  <dig>  similarly, investigation of a pho-4+/human pit <dig> chimera where the pho-4+ backbone harbors the human pit <dig> sequence l545-s <dig>  showed that these sequences confer galv receptor function upon pho-4+ <cit> . h. sapiens pit <dig> tm  <dig>  and h. sapiens pit <dig> tm  <dig>  are suggested to be homologous to the tm  <dig> predicted in rpho- <dig> r. norvegicus  protein sequence g601-s <dig>  using the tmhmm server . n. crassa pho-4+ tm  <dig>  is suggested to be homologous to the tm  <dig> predicted in c. elegans putative phosphate permease protein sequence. the tm  <dig> is required to orient the tm  <dig> equivalently in all pit family members. lower case letters represent tmhmm- or das-predicted tm sequences, which we based on either too small length to comprise a tm or due to suggested extracellular position  found were non-compatible with regular tm domains; however, these sequences might instead \"dip\" into the membrane lipid bilayer. it should be noted that these sequences are counted as being part of the loop sequences in figure  <dig>  a star   and a vertical line indicate the position of an intron-exon border in each of the human pit genes determined by use of the spidey mrna-to-genome dna alignment as described in \"methods\". below the alignment, the names, species, phylas, kingdoms, swiss-prot accession numbers, and the amino acid lengths of the nine proteins are given. b the server-predicted tm domains  and the by us suggested tm domains  for each of the nine pit family members are depicted in order to illustrate the conservedness of the tm domains: tm  <dig>  tm  <dig>  tm  <dig>  > tm  <dig>  tm  <dig>  > tm  <dig>  tm  <dig>  tm  <dig> > tm  <dig> > tm  <dig> . the white asterisk indicates a prediction of a unique tm domain in the unusually long n-terminal sequence of a. thaliana pht2_ <dig> \n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 2\n",
            "data and statistics. average 32pi uptakes in oocytes given as pmol/oocyte-hour ±sem, information regarding the number  of oocytes measured, and the statistics  for figures 3a-b and figure  <dig> are available in additional file  <dig>  average numbers of blue  cells per dish from three dishes ±sem and the statistics  for figures 3c-e are available in additional file  <dig>  average loop lengths given as amino acids ±sem and information regarding the number  of loops counted for figure 4b are available in additional file  <dig> \n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements and funding\n",
            "we thank drs. bryan o'hara for poj <dig> and poj <dig>  maribeth v. eiden for the pa317gbn and pg13gbn cell lines, joyce dunn for the felv-b virus stock, and jan egebjerg jensen for use of his x. laevis oocyte facilities. we furthermore thank bente andersen for excellent technical assistance.\n",
            "\n",
            "this work was supported by the lundbeck foundation , the novo nordisk foundation, the danish medical research foundation  22-03- <dig>  09- <dig> , 09- <dig> , 09- <dig> , an engineer arne hansen grant, and the intramural budget at the institute of clinical medicine at aarhus university.\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "our results suggest that human pit <dig> is dispensable for pi transport function, and that the overall structure of the pit family proteins has remained unchanged during evolution.\n",
            "our results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\n",
            "our results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\n",
            "our results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\n",
            "our results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\n",
            "our results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\n",
            "our results suggest that human pit <dig> is dispensable for pi transport function but dispensable for infection by pit <dig> cognate gamma-retroviruses.\n",
            "\n",
            "37\n",
            "[{'summary_text': 'the glyco-secretome of b. cinerea is composed mainly of polysaccharide hydrolases, polysaccharide hydrolases, oxidoreductases, and plenty of proteins with unknown functions.\\nthe glyco-secretome of b. cinerea is composed mainly of polysaccharide hydrolases, polysaccharide hydrolases, polysaccharide hydrolases, and plenty of proteins with unknown functions.\\nthe glyco-secretome of b. cinerea is characterized by the prediction of a high number of secretory proteins, about 10% of the polypeptides coded by the genome.\\nthe glyco-secretome of b. cinerea is composed mainly of polysaccharide hydrolases, polysaccharide hydrolases, oxidoreductases, and plenty of proteins with unknown functions.\\nthe glyco-secretome of b. cinerea is characterized by the prediction of a high number of secretory proteins, about 10% of the polypeptides coded by the genome.\\nthe glyco-secretome of b. cinerea is characterized by the prediction of a high number of secretory proteins, about 10% of which'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "botrytris cinerea has been considered the second most important plant pathogenic fungus according to its economic and scientific importance  <cit> , and is able to infect more than  <dig> plant species including many with high agronomic value  <cit> . the genome of two b. cinerea strains have been sequenced  <cit>  and one of its key features is the prediction of a high number of secretory proteins, about 10% of the polypeptides coded by the genome. proteomic studies have actually revealed an abundant and diverse set of proteins in the extracellular medium for b. cinerea cultures  <cit> . this set contains mainly proteins involved in the degradation of plant structures , but also proteins with other functions such as the induction of cell death in the host  <cit> , as well as plenty of proteins with unknown function  <cit> . the role of some of these extracellular proteins has been studied by gene knock-out to identify secretome members contributing to virulence, i.e. virulence factors, but only a few have shown to contribute modestly to the infection process  <cit> .\n",
            "\n",
            "it is known that o-glycosylation is crucial in determining the structure and function of numerous secreted and membrane-bound proteins  <cit> . carbohydrate chains have been proposed to enhance the stability and solubility of proteins, to confer protection against proteases, to act as a sorting determinant, and to be involved in various development and differentiation processes  <cit> . we have recently shown, by an in silico approach, that about half of predicted proteins in fungal secretomes display ser/thr-rich regions, i.e. regions with at least 40% ser/thr in a minimum of  <dig> contiguous residues  <cit> , which are usually considered to display high-density of o-glycosylation  <cit> . indeed, about one fourth of secretory proteins were predicted to display hyper-o-glycosylated regions, i.e. regions with at least 25% of residues predicted to be o-glycosylated  <cit> . in fungi, o-glycosylation begins with the addition of a mannose residue by protein o-mannosyltransferases  in the lumen side of the endoplasmic reticulum  membrane  <cit> , a process which has recently been shown to occur, at least partially, concomitantly with the translocation of nascent polypeptides into the er  <cit> . three pmt families exist in fungi   <cit> , and filamentous fungi usually have only one pmt gene per family. the deletion of one or more of the pmt genes usually results in loss of viability or strong defects such as reduced conidia production, changes in fungal morphology, etc., emphasizing the importance of o-glycosylation for the fungal biology. the b. cinerea genome also contains three pmt genes  <cit> , bcpmt <dig>   <dig> and  <dig>  and the three of them, but especially bcpmt <dig>  are critical for the stability of the cell wall, are necessary for sporulation, and are required for the generation of the extracellular matrix. besides, bcpmts are also required for full virulence in a variety of hosts, with a special role in the adhesion to, and penetration of, intact plant leaves  <cit> .\n",
            "\n",
            "since pmts have been found in both prokaryotes and eukaryotes  <cit> , but not in plants, these proteins are promising targets in the design of novel control strategies against fungal phytopathogens. however, the er-associated topology of these proteins  <cit>  poses strong problems for the elucidation of their structure and the design of specific inhibitors. an indirect approach is the identification of specific pmt substrates that could also be important for fungal biology and virulence. since we have previously observed significant changes in the patterns of protein secretion and glycosylation by the bcpmt mutants  <cit> , we have addressed here the characterization of glyco-secretome differences between wild-type b. cinerea and bcpmt mutants.\n",
            "\n",
            "RESULTS\n",
            "many proteins in the b. cinerea secretome are glycosylated\n",
            "we have previously identified more than one hundred proteins in the b. cinerea secretome  <cit> , and we have also observed that the appearance of the secretome in 1d- and 2d-page changes radically for mutants affected in pmts, which catalyse the first step of o-glycosylation  <cit> . this prompted us to better characterize the set of glycoproteins secreted by wild-type b. cinerea strain b <dig> , as well as by the mutants lacking each one of the three b. cinerea pmts  <cit> . preliminary observations allowed us to establish static liquid cultures in petri dishes with ygg-low medium, inoculated with mycelial plugs and incubated for 4 days, as the optimum conditions to maximize isolation of secretory proteins in the case of Δbcpmt mutants. culture media obtained from these plates contained plenty of proteins  and the band pattern in sds-page was different for the wild type and for the Δbcpmt mutants. purification of glycoproteins from the culture medium by affinity chromatography with concanavalin-a resulted in quite different band patterns , with some proteins clearly disappearing and others being enriched. notably, the band corresponding to the most abundant protein in the secretome of b. cinerea, the 35-kda aspartic protease bcap <dig>  <cit>  , is completely absent after purification of glycoproteins, in accordance with the fact that no n- or o-glycosylated sites are predicted in silico for this protein by the netnglyc  <dig>   <cit>  and netoglyc  <dig>   <cit>  servers. moreover, an enrichment in proteins with high molecular weight  was observed for all samples after glycoprotein purification, in agreement with the fact that lectin-blot experiments also show that glycosylation is more prominent for these proteins  <cit> . as expected, the sds-page band pattern of purified glycoprotein samples is different for the wild type and the three Δbcpmt mutants, especially in the case of Δbcpmt <dig> and Δbcpmt <dig> figure  <dig> \n",
            "glyco-secretome purification. electrophoretic analysis of glycoproteins purified from the extracellular fraction in cultures of wild-type b. cinerea  and the indicated Δbcpmt mutants. a: sds-page showing all proteins precipitated from 150 μl of culture medium. b: sds-page of the purified glycoprotein samples . black arrow: bcap <dig>  a protein with no predicted glycosylation sites. white arrows: example bands appearing/disappearing in the mutant samples as compared with the corresponding wild-type sample.\n",
            "\n",
            "\n",
            "\n",
            "2d electrophoresis of purified glycoproteins was carried out for b <dig>  and the Δbcpmt <dig> mutant, and resulted in a relatively simple spot pattern , with not too many differences for the two samples. some spots, however, showed clear differences in intensity, such as spot  <dig>  which is over-expressed in the Δbcpmt <dig> sample. a total of  <dig> spots obtained from the wild-type or the Δbcpmt <dig> 2d gels were excised and analyzed by maldi-tof/tof, resulting in the identification of  <dig> proteins . in some cases , the same protein was identified in several spots forming charge trains , or even in spots  with different mobility in the second dimension .figure  <dig> \n",
            "2d electrophoresis of the glyco-secretome. 10 μg of purified protein sample were fractionated by 2d electrophoresis and stained with silver. numbers indicate proteins identified in each spot by maldi-tof/tof . boxes identify charge trains for which the same protein was identified in several spots.\n",
            "\n",
            "\n",
            "\n",
            "identification of glycoproteins in the secretome by lc-ms/ms\n",
            "the purified glycoprotein samples obtained for the four strains were also analyzed by shotgun proteomics. the total number of proteins identified was  <dig> in the four samples . only one of the  <dig> proteins identified previously from the 2d electrophoresis gels  was not identified by lc-ms/ms, so the total number of glycoproteins identified by the two methods is  <dig>  most proteins identified  showed signal peptide according to signalp  <dig>   <cit> , and  <dig> others showed alternative secretion features according to secretomep  <dig>   <cit> . forty-eight of the  <dig> proteins are described here for the first time as components of the b. cinerea secretome, while the rest have been reported previously  <cit> .\n",
            "\n",
            "contrary to our initial expectations, the composition of the glyco-secretome was not radically different for the wild type and for the three Δbcpmt mutants, so that proteins clearly absent in one or more of the mutants, and therefore putative substrates of the corresponding bcpmt, are scarce. restricting the comparative analysis to highly expressed proteins , so that the number of spectral counts accumulated for each protein makes the comparison more significant, it results that the  <dig> proteins considered are all expressed in the three Δbcpmt mutants, as well as in the wild type. only two proteins, pectin methylesterase bcpme <dig> and b0510_ <dig> , have an abundance of less than 25% that of the wild type in at least one of the mutants. noticeably, some of these proteins are actually found at elevated levels in the mutants, in comparison with the wild type. overall, these results confirm a prominent role for pmts in protein glycosylation, but in a way that may be more complex than anticipated. it does not seem, for example, that individual proteins are substrates of only one bcpmts.table  <dig> \n",
            "abundant proteins identified by lc-ms/ms in wild-type\n",
            "b. cinerea\n",
            "and the three Δ\n",
            "bcpmt\n",
            "mutants\n",
            "\n",
            "\n",
            "\n",
            "protein name\n",
            "1\n",
            "– gene id\n",
            "spectral counts\n",
            "2\n",
            "\n",
            "b05\n",
            "Δ\n",
            "bcpmt1\n",
            "Δ\n",
            "bcpmt2\n",
            "Δ\n",
            "bcpmt4\n",
            "\n",
            "1protein family and/or conserved domains found in sequences are included between brackets where possible: glycosyl hydrolase family , merops family , carbohydrate esterase family , or carbohydrate binding modules .\n",
            "\n",
            "\n",
            "2only proteins fulfilling the following two conditions are displayed: they have at least  <dig> spectral counts in one of the four strains, and they account for more than 2% of all spectral counts in one of the strains.\n",
            "\n",
            "\n",
            "\n",
            "the  <dig> proteins were classified into families previously defined  <cit>  . family assignments were made using sequence similarities with proteins of known function detected by blast  <cit> , and the presence of conserved domains according to pfam  <cit> . distribution of glycoproteins in families was similar for the wild type and the three mutants . the glyco-secretome is composed mainly of polysaccharide hydrolases , followed by proteases  and oxidoreductases . the rest of the families  are represented by a similar amount of proteins and contribute always less than 10% to the overall composition. as compared with the complete secretome  <cit> , the glyco-secretome showed an increase in polysaccharide hydrolases and oxidoreductases, but a reduction in pectinases.figure  <dig> \n",
            "family distribution of the glycoproteins identified in the four\n",
            "b. cinerea\n",
            "strains. numbers indicate the number of proteins in each category.\n",
            "\n",
            "\n",
            "\n",
            "ser/thr-rich and highly o-glycosylated regions in glycoproteins\n",
            "we have recently analyzed in sílico the presence and distribution of ser/thr-rich regions, as well as hyper-o-glycosylated regions, in the predicted b. cinerea secretome  <cit> . we now have the opportunity to compare those results with a similar analysis for the experimental set of actually secreted and glycosylated proteins reported here . the results show, in first place, that ser and thr residues are extremely abundant among secreted glycoproteins. on average, 38% of residues in these proteins are either ser or thr in the experimental glycoprotein set, while this number is just 17% for the whole predicted secretome. when the ms excel xrr macro described previously  <cit>  was used to study if these residues are grouped forming ser/thr-rich regions  a big number of these was found, even when a very high ser/thr content was necessary for a region to be considered as ser/thr-rich . the proportion of proteins displaying ser/thr-rich regions was considerably higher for the experimental set of glycoproteins than for the predicted whole secretome . for example, 80% of glycoproteins displayed regions with at least 40% ser/thr, while only 46% of proteins in the whole predicted secretome exhibited them . moreover, the total number of ser/thr-rich regions was comparatively higher for the glycoprotein set: among the  <dig> glycoproteins  <dig> ser/thr-rich regions  were found, almost  <dig> per protein, while the whole predicted secretome  contained only  <dig> of these ser/thr-rich regions. notably, seven proteins were found with regions of 20– <dig> residues in which at least 80% of them are ser or thr.figure  <dig> \n",
            "ser/thr-rich regions and hyper-\n",
            "o\n",
            "-glycosylated regions among proteins in the glyco-secretome. a: percentages of proteins displaying regions with the indicated minimal ser/thr content , both among the experimental glycoprotein set  and among all predicted secretome proteins . b: percentage of proteins with predicted hyper-o-glycosylated regions in the same two proteins sets, also analyzed for the indicated minimal abundances of predicted o-glycosylation sites . c: frequency distribution of ser/thr-rich and hyper-o-glycosylated regions along proteins for the experimental glycoprotein set. the total number of regions found in every possible 10% interval along protein length is represented. d: relative abundance of proteins in the Δbcpmt mutants, as compared with the wild-type strain b <dig> , estimated from the spectral counts for two protein sets: those displaying hyper-o-glycosylated regions  and those without them . asterisks indicate a significant difference for the two sets  at either  <dig>   or  <dig>   confidence level.\n",
            "\n",
            "\n",
            "\n",
            "similar results were obtained when glycosylation was predicted for the set of purified glycoproteins. both n-glycosylation and o-glycosylation was predicted for most of them , but the average number of predicted o-glycosylated residues in proteins was about three times higher than the number of n-glycosylated ones. the predicted o-glycosylation positions were frequently grouped in highly-o-glycosylated regions , but not the n-glycosylation sites . using the less stringent definition of highly-o-glycosylated regions , it results that about half of proteins  in the glyco-secretome display these kind of regions, about twice the number obtained for the whole predicted secretome   <cit> .\n",
            "\n",
            "we observed previously, for the whole predicted secretome, that ser/thr-rich and predicted highly o-glycosylated regions have a tendency to be located toward the two ends of polypeptide chains  <cit> . this tendency is clearer in the case of glycoproteins and especially evident for predicted highly o-glycosylated regions , which are frequently found in the c-terminal region of proteins and are also common at the n-terminus right after the signal peptide.\n",
            "\n",
            "interestingly, we found a relationship between the presence of predicted hyper-o-glycosylated regions in proteins and their relative abundance in the secretomes of the Δbcpmt mutants, estimated from the spectral counts. this was done by first choosing a set of abundant proteins, for which we could calculate more reliable fold changes from spectral counts, consisting of  <dig> proteins with at least  <dig> spectral counts in one of the four botrytis strains and a relative abundance of at least 1% in one of the four strains. the fold change in the amount of protein in the Δbcpmt mutants, relative to the wild type, was then estimated from the spectral counts for each individual protein. next, the average fold change was calculated separately for those proteins with predicted hyper-o-glycosylated regions and for those without them, and are displayed in figure 4d. statistically significant differences were found for the two groups in the three mutants, so that the relative abundance of proteins predicted to have hyper-o-glycosylated regions is, on average,  <dig>  times higher in the secretomes of the Δbcpmt mutants than in the secretome of the wild type. on the contrary, no difference was observed for the proteins without predicted hyper-o-glycosylated regions.\n",
            "\n",
            "one possible explanation for the fact that proteins displaying hyper-o-glycosylated regions tend to be found at higher levels in the secretomes of the Δbcpmt mutants is that o-glycosylation causes retention of the proteins in the cell wall or the extracellular matrix. these proteins, being glycosylated to a less extent in the Δbcpmt mutants, would escape easier and accumulate to higher levels in the culture medium. to test this hypothesis, two fusion proteins were expressed in b. cinerea which both contained the sequence of bcspl <dig>  <cit> , an abundant component of the b. cinerea secretome partially retained in the cell wall  <cit> , fused to gfp, but differed in the presence of a ser/thr-rich region in the c-terminal end originally coming from the endoglucanase cel5a  <cit> . unfortunately, the level of expression seemed quite different for the two fusion proteins, as judged from the green fluorescence in the mycelium  and from the western-blots with monoclonal anti-gfp antibodies , and both gave various bands in the western-blot. by considering all the bands obtained in the blots, we determined for liquid cultures of each of these two strains the ratio of protein secreted to the culture medium versus the amount associated with the mycelium . although the effect of the introduction of the ser/thr-rich tail in the protein was not spectacular, the fusion protein displaying it showed a higher tendency to be associated with the mycelium which was statistically significant in the three repetitions of the experiment shown in figure 5c.figure  <dig> \n",
            "ser/thr-rich regions partially retain proteins with the mycelium. a: fluorescence of transformants expressing a fusion proteins with bcspl <dig>  gfp, and with , or without , a ser/thr-rich region from cel5a. the wild-type strain b <dig>  was used as control. b: sds-page and western blot  comparing the amount of recombinant protein released to the culture medium  and the amount associated with the mycelium  in cultures  of the same strains as in . amounts of proteins loaded were those contained in  <dig>  ml of medium  or associated with  <dig>  mg of mycelium . c: percentages of recombinant protein released to the medium as soluble protein for the two strains, averaged for three independent cultures. asterisk indicates a statistically significant difference between the two strains with  <dig>  confidence .\n",
            "\n",
            "\n",
            "\n",
            "overexpression of o-glycosylated proteins in b. cinerea\n",
            "three of the proteins detected as components of the glyco-secretome were overexpressed in b. cinerea to further study them. these proteins were selected because they display ser/thr regions and are glycosylated in different degrees . the three proteins were the endopolygalacturonase bcpg <dig>  previously reported to be required for full virulence in b. cinerea  <cit>  and to be perceived as a pamp  by arabidopsis  <cit> , and two proteins of unknown function that we have named bcieb <dig>  and bcsun <dig> . these proteins differed in the number or length of the ser/thr-rich regions, with bcsun <dig> being predicted to be highly glycosylated by netoglyc  <dig>  and the other two poorly glycosylated .\n",
            "\n",
            "expression of the three genes under the control of the strong promoter olic resulted in the accumulation of decent amounts of proteins in the culture medium . in order to check if, and how, these proteins were glycosylated we treated them with two glycosyl hydrolases: exo α1- <dig>  and exo α1- <dig> mannosidases. since all attempts to purify them from the culture media were unsuccessful, the treatment was done with a whole secretome sample and analyzed by western-blot with anti-c-myc antibodies . reduction of protein size, as a consequence of treatment, was considerable for bcsun <dig> , limited for bcieb <dig> , and not detectable for bcpg <dig>  in good accordance with the number of glycosylation sites predicted for the three proteins:  <dig>   <dig> and  <dig> . this reduction was seen only for the incubation with the exo α1- <dig>  enzyme. treatment with exo α1- <dig> mannosidase did not produced any effect by itself nor did it potentiate the effect of exo α1- <dig> . these results indicate, therefore, that at least bcsun <dig> and bcieb <dig> are actually glycosylated, that the amount of sugars incorporated is in accordance with the predictions carried out by netoglyc, and that glycosidic chains contain primordially mannoses linked by α1- <dig> or α1- <dig> glycosidic bonds. additionally, the same strategy was used to assess the incorporation of mannose residues to the bcspl1-gfp-st fusion protein described above , and also in this case treatment with exo α1- <dig>  mannosidase  produced a considerable reduction in size, but not in the case of the control fusion protein bcspl1-gfp.figure  <dig> \n",
            "deglycosylation of\n",
            "b. cinerea\n",
            "proteins with exo α1- <dig>  and exo α1- <dig> mannosidases. extracellular media from cultures of the strains overexpressing the indicated proteins were treated with the enzymes and analysed by western-blot with the indicated antibodies. sds-page of the untreated samples is also shown . the samples treated with the different enzymes, as well as the untreated controls, were identical and corresponded to 900 μl of culture medium. arrows point to the positions in the coomassie-stained gels that correspond to the bands observed in the western blots.\n",
            "\n",
            "\n",
            "\n",
            "culture media from the three strains overexpressing the o-glycosylated proteins were also tested for the ability to elicit defense responses in plants by seedling growth inhibition assay  <cit> , a sensitive and quantitative test which correlates with typical plant defence responses such as callose deposition, production of reactive oxygen species, or pathogenesis-related gene expression  <cit> . in this assay, culture media from the three overexpressing strains revealed considerably more efficient in inhibiting growth of tobacco seedling, as compared with the media obtained with the wild-type strain . moreover, the seedlings treated with media from the overexpressing strains showed necrotic symptoms that were clearly more intense than with the wild-type strain.figure  <dig> \n",
            "growth inhibition and necrosis of seedlings caused by\n",
            "o\n",
            "-glycosylated proteins. tobacco seedlings were treated for 9 days with culture media from the strains overexpressing the indicated o-glycosylated proteins and then assessed for necrosis and growth inhibition. a: western-blot  showing the relative amounts of recombinant proteins in the culture media from the overexpressing strains. medium from the wild-type strain  and uninoculated medium  were used as controls. each lane contained proteins precipitated from 1 ml of medium. molecular weight makers are show to the left of each lane . b: example seedlings treated with the culture media. c: average weight  of treated seedling after the 9-day incubation. different letters on bars indicate statistically significant differences with  <dig>  confidence.\n",
            "\n",
            "\n",
            "\n",
            "discussion\n",
            "o-glycosylated proteins are crucial components of fungal secretomes. not only has it been predicted that more than half of secretome components display o-glycosylation sites  <cit> , frequently in the form of highly-o-glycosylated regions, but it has also been shown that mutants affected in the first step of the o-glycosylation machinery display a vast arrays of defects including decreased virulence in the case of phytopathogens  <cit> . by purification and identification of glycoproteins, we have shown here that b. cinerea indeed secretes a substantial amount of these proteins. altogether,  <dig> proteins were identified in the isolated glycoprotein sample for the wild-type strain b <dig> , which include  <dig> proteins for which experimental evidence is provided here, for the first time, as components of the secretome. it is worth noting the absence, among these proteins, of the aspartic protease bcap <dig>  whose band is not detected in sds-page of the purified glycoprotein sample  nor does it appear among the  <dig> proteins identified by lc-ms/ms for the same sample. this protein serves as an internal negative control for the purification of glycoproteins, since it is the most abundant protein in the secretome  <cit>  but it lacks predicted glycosylation sites. its absence in the purified sample indicates that the purification is working properly. family distribution of glycoproteins  is not very different from that obtained for the whole secretome sample  <cit> , with a low number of proteins classes containing most of the proteins: mainly proteases, polysaccharide hydrolases, oxidoreductases, and plenty of proteins with unknown functions. it does not seem, therefore, that glycosylation acts preferentially on specific types of proteins.\n",
            "\n",
            "contrary to what we expected, the composition of the glyco-secretome was not very different for the wild type and the three Δbcpmt mutants affected in o-glycosylation. we reasoned that lack of glycosylation of specific proteins in one or more of the Δbcpmt mutants would prevent its retention by the lectin affinity column and so would result in its absence in the corresponding glycoprotein sample. although there are plenty of proteins  which are absent in one or more of the four strains, these correspond almost always to those with just one or a few spectral counts, so that their absence/presence may be the result of mere chance. the comparison of spectral counts for those proteins with the highest expression , however, does not show any protein which is absent in the mutant samples. this may indicate, at least for these proteins, that they are not substrates of single pmts  but are glycosylated by the combined action of several of them. pmts have been shown to act as specific dimers in saccharomyces cerevisiae, pmt1/pmt <dig> heterodimer and pmt4/pmt <dig> homodimer  <cit> , and as almost all possible hetero- and homo-dimeric combinations in aspergillus nidulans  <cit> . our results agree better with the situation in a. nidulans, since a more promiscuous association of pmt monomers, to form functional dimers, would inherently imply an easier substitution of one of the pmts by the other isoforms.\n",
            "\n",
            "surprisingly, more often than not, spectral counts for the most abundant proteins are higher in the Δbcpmt mutants than in the wild-type strain b <dig>  , possibly because proteins correctly glycosylated are retained easier in the cell surroundings  while those with an incomplete glycan structure escape easier. it was actually shown  that proteins predicted to have hyper-o-glycosylated regions are, on average, more abundant in the glyco-secretomes of the Δbcpmt mutants, as compared with the wild type, while the same is not true for the proteins not predicted to have hyper-o-glycosylated regions. moreover, addition of a ser/thr rich region to a fusion protein expressed in b. cinerea increases the amount of it that remains associated with the mycelium , as compared with the amount found in the extracellular medium, although these results need to be taken with caution because of the different levels of gfp expression in the two strains. if confirmed, this effect of glycosylation maybe physiologically important for b. cinerea, as enzymes acting on soluble substrates and producing assimilable nutrients for the fungal cells may be more efficient if retained closer to the cells.\n",
            "\n",
            "the overexpression of three of the proteins predicted to be o-glycosylated allowed experimental confirmation of the post-translation modification for two of them, as a decrease in the apparent molecular mass observed in sds-page was caused by enzymatic deglycosylation . although prediction of o-glycosylation carried out by netoglyc is only approximate for fungal proteins  <cit> , the changes in the apparent molecular mass obtained for the three proteins are in accordance with the number of o-glycosylation sites predicted . incubation with two different deglycosylation enzymes showed the presence of mannose α1- <dig> and/or α1- <dig> bonds , but no mannose α1- <dig> bonds were detected. these results are in agreement with the glycosyl linkages observed for other fungal and yeast proteins, usually displaying mannose α1- <dig> and α1- <dig> linkages  <cit> . although mannose α1- <dig> bonds have been observed in other fungi such as aspergillus  <cit> , they either do not exist in b. cinerea, at least in the proteins analyzed, or their contribution to the proteins molecular weight is not significant.\n",
            "\n",
            "the extracellular media obtained after growing the strains overexpressing the three glycoproteins showed an enhanced ability to elicit plant defenses, as detected in seedling growth inhibition assays . this was not surprising in the case of bcpg <dig>  since this proteins has recently been shown to be recognized as a pamp by the arabidopsis pattern recognition receptor rbpg <dig>  <cit> , but was completely unexpected for the other two. it is tempting to speculate that it is the sugar fraction on these proteins the part responsible for the elicitation of the plant defenses, since this feature is shared by the three proteins, but clearly the alternative explanation, i.e. different amino acid sequences in the three proteins act as elicitors, cannot be ruled out at this point.\n",
            "\n",
            "CONCLUSIONS\n",
            "b. cinerea secretes plenty of glycosylated proteins belonging to a diverse set of families, with polysaccharide hydrolases, proteases, and oxidoreductases being the most abundant groups. as expected, ser/thr-rich regions, considered to be substrates of the o-glycosylation machinery, are twice more abundant in the glyco-secretome than in the whole secretome. surprisingly, proteins predicted to be hyper-o-glycosilated are more abundant in the glyco-secretomes of o-glycosylation deficient mutants, possibly because o-glycosylation causes retention in the cell wall or extracellular matrix. o-glycosylated proteins seem to have a prominent role in plant-pathogen interaction, since the independent overexpression of three of them in b. cinerea increases elicitation of plant defenses by the fungus.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the glyco-secretome of b. cinerea is composed mainly of polysaccharide hydrolases, polysaccharide hydrolases, oxidoreductases, and plenty of proteins with unknown functions.\n",
            "the glyco-secretome of b. cinerea is composed mainly of polysaccharide hydrolases, polysaccharide hydrolases, polysaccharide hydrolases, and plenty of proteins with unknown functions.\n",
            "the glyco-secretome of b. cinerea is characterized by the prediction of a high number of secretory proteins, about 10% of the polypeptides coded by the genome.\n",
            "the glyco-secretome of b. cinerea is composed mainly of polysaccharide hydrolases, polysaccharide hydrolases, oxidoreductases, and plenty of proteins with unknown functions.\n",
            "the glyco-secretome of b. cinerea is characterized by the prediction of a high number of secretory proteins, about 10% of the polypeptides coded by the genome.\n",
            "the glyco-secretome of b. cinerea is characterized by the prediction of a high number of secretory proteins, about 10% of which\n",
            "38\n",
            "[{'summary_text': 'in this study, we investigated the functional role of putative cis-elements in the transcriptional regulation of egcad <dig> and egccr genes in vascular tissues.\\nthe egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes.\\nthe egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes.\\nthe egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes.\\nthe egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes.\\nthe egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes.\\nthe egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes in planta.\\nthe egcad <dig> promoter is'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "vascular cambium is a cylindrical secondary meristem that produces both secondary phloem and secondary xylem . the most characteristic components of secondary cell walls are lignins, complex phenolic polymers, which play fundamental roles in mechanical support, water and solute conductive properties and disease resistance in higher plants  <cit> . the biosynthesis of the lignin polymers derives from the general phenylpropanoid pathway which provides precursors for several branch pathways leading to the elaboration of a wide range of compounds involved in various aspects of plant development and defence  <cit> . in the lignin-specific branch pathway, the conversion of hydroxycinnamoyl coa esters into cinnamyl alcohols , the monomeric units that are incorporated into the lignin heteropolymer, is catalyzed by cinnamoyl coa reductase  and cinnamyl alcohol dehydrogenase  successively.\n",
            "\n",
            "the spatial and temporal expression of the eucalyptus gunnii ccr and cad <dig> genes was investigated by fusing the gene promoters to the uida reporter gene coding for β-glucuronidase  and expressing these constructs in transgenic woody and herbaceous plants  <cit> . both egcad <dig> and egccr gene promoters have been reported to direct gus activity in the vascular tissues of all organs in all the plant species examined. similar expression patterns have been reported for the arabidopsis thaliana atcad-c and atcad-d genes, which encode proteins that are closely related to the egcad <dig> enzyme  <cit> . the gus activities were found to be consistent with tissue and cell locations of the ccr and cad <dig> transcripts and proteins obtained by the use of in situ hybridization and immunolocalization  <cit> . these observations showed that the two genes have the same expression pattern, suggesting that the control of monolignol production is, at least in part, achieved through their coordinated transcriptional regulation.\n",
            "\n",
            "deletion studies of the egccr and egcad <dig> promoters showed that the egccr promoter deleted to - <dig> bp upstream of the transcription start site retained its ability to direct gus expression in a pattern similar to that obtained with the full-length promoter whereas the promoter deleted to - <dig> bp was inactive  <cit> . in the egcad <dig> promoter, a region necessary for gus expression in cambium and secondary xylem was mapped between - <dig> and - <dig> bp upstream of the transcription start site  <cit> . these results indicate that expression of the egccr and egcad <dig> genes is under the control of a promoter region  proximal to the transcription start site. as revealed by in silico studies, both regulatory regions contain putative binding sites for myb transcription factors.\n",
            "\n",
            "plant myb proteins belong to one of the largest families of transcription factors. according to their predicted sequences and structural features, they have been classified into several subgroups  <cit> . the r2r <dig> two-repeat myb family occurs specifically in plant lineages and its members have been postulated to participate in the regulation of a wide range of developmental and metabolic processes, notably the phenylpropanoid biosynthetic pathway  <cit> . however, despite the conservation of myb consensus motifs  in most phenylpropanoid gene promoters, functional evidence for their contribution to the promoter activity has only been directly proven for a few of them . although the involvement of myb proteins in the monolignol-specific branch pathway has been suggested , the functional role of myb cis-elements and the molecular mechanism by which they participate in promoter activities have not yet been investigated.\n",
            "\n",
            "the main goal of this study was to define the functional role of putative cis-elements in the transcriptional activities of egcad <dig> and egccr promoters. firstly, we restricted the regulatory region of the egcad <dig> promoter to an  <dig> bp region necessary for expression both in secondary xylem and in cambium. this  <dig> bp region contains two myb sites and two unreferenced sites that are revealed by in vivo footprinting. a straightforward approach to investigate the functional role of promoter elements is to introduce block mutations into the putative cis-elements in the context of the shortest active promoter. thus, by using egccr and egcad <dig> short promoter versions carrying single or combined mutations in their putative cis-elements, we clearly identified the myb elements in the egccr and egcad <dig> promoters as being crucial not only for the formation of protein-dna complexes in vitro either with recombinant myb protein or with cellular protein extracts, but also for the transcriptional regulation of egccr and egcad <dig> genes in planta. in addition, the experimental data showed that a newly identified cis-element is required for egcad <dig> expression in the cambial zone.\n",
            "\n",
            "RESULTS\n",
            "an  <dig> bp egcad <dig> cis-regulatory region is involved in egcad <dig> expression in xylem tissues and in dividing cells\n",
            "to further localize the cis-elements in the  <dig> bp region  involved in the transcriptional regulation of egcad <dig>  <cit> , we introduced several truncated versions of the promoter fused to the gus reporter gene into tobacco by agrobacterium-mediated transformation. tobacco, a model herbaceous plant which undergoes secondary thickening, was previously shown to be a suitable model plant for studying the egcad <dig> promoter from trees  <cit>  and also phenylpropanoid promoters from herbaceous or woody plants . the shortest tested 5'-deletion showing a similar vascular expression pattern to the full-length  promoter was deletion - <dig> . stem cross-sections showed gus expression in cells connected to lignification, such as differentiating xylem and parenchyma cells of the primary and secondary xylem , but also in cells apparently unconcerned by lignification, such as the dividing cells of the vascular cambium and the cells of the external and internal phloems, especially the companion cells . gus expression in xylem and cambium was lost upon deletion of the promoter to - <dig> which retained gus activity only in the internal phloem of tobacco stem  <cit> , delimiting a shorter  regulatory region  involved in promoter activity in xylem and vascular cambium.\n",
            "\n",
            "to confirm the importance of this  <dig> bp region in egcad <dig> expression in dividing cells, we transformed tobacco by <dig> suspension-cultured cells with the - <dig> promoter deletion construct. fluorimetric measurements of gus activity indicated that this short promoter was active in by <dig> cells at a level close to that of the egcad <dig> full-length promoter or of the constitutive camv 35s promoter used as a reference . gus activity decreased drastically with the - <dig> promoter deletion construct . these data confirm that the  region contains cis-elements required for activity of the promoter in dividing cells  as well as in xylem.\n",
            "\n",
            "fine mapping of cis-elements in the egcad <dig> promoter by in vivo dms footprinting\n",
            "to precisely map the cis-elements in the egcad <dig> promoter interacting with trans-acting factors, we used in vivo dimethyl sulphate  footprinting, a technique allowing location of protein-dna interactions with single nucleotide resolution  <cit> . the chemical agent dms methylates the unprotected guanine residues rendering them susceptible to cleavage by piperidine.\n",
            "\n",
            "the footprints were examined over a broad region  encompassing the regulatory region . in exponentially-dividing cells that highly expressed egcad <dig> , several guanine residues of the promoter sequence showed differences in the footprint relative to the control , whereas in stationary phase cells with low expression of egcad <dig>  the footprint was almost identical to the control . the protected  or hypersensitive  sites, corresponding to lower or higher intensity bands respectively relative to the control, are indicated on the promoter sequence . two protected areas were identified, each composed of two adjacent guanines , surrounded by hypersensitive residues . hypersensitivity to dms methylation can indeed result from alterations in the local dna topology induced by protein binding  <cit> . the protected guanines fall within two identical sequences , which we call bsa  and bsb , and which have no homology with consensus binding sequences in databases. just downstream of both bsa and bsb, however, are sequences that correspond to mbsiig, one of the plant myb transcription factor binding sites tggt;  <cit> ). these elements are named myba  and mybb . all the footprints are included within the  <dig> bp regulatory region delineated above. no other footprint was observed within the investigated  broad promoter region , indicating the absence of additional interaction sites upstream and downstream of the  <dig> bp regulatory region.\n",
            "\n",
            "when surveying the cad <dig> regulatory region in several eucalyptus species, a few base substitutions were found in bsa and bsb sequences whereas the myb sites were perfectly conserved, arguing for their possible functional importance . it is worthy of note that similar conservation across eucalyptus species was also found for the unique myb site in the ccr regulatory region .\n",
            "\n",
            "these results suggest that four relatively close putative cis-elements in the regulatory region  of the egcad <dig> promoter could interact with several transcription factors in cells that are transcriptionnally active for the egcad <dig> gene.\n",
            "\n",
            "effect of mutations on in vitro and in vivo binding of egmyb <dig> to the egcad <dig> and egccr promoters\n",
            "to investigate the ability of myb sites of egcad <dig> and egccr promoters to interact in vitro and in vivo with myb factor, site-directed mutagenesis was performed to disrupt these putative regulatory sites individually or in combination. electrophoretic mobility gel assays  and transactivation experiments were carried out in the presence of the eucalyptus protein egmyb <dig>  a r2r <dig> myb factor, that was hypothesized to positively regulate the transcription of genes belonging to the monolignol pathway  <cit> .\n",
            "\n",
            "emsas were performed using recombinant egmyb <dig> and dna probes corresponding to  egcad <dig> and  egccr regulatory fragments with mutated myb sites . the specificity of the dna-protein complex was verified by competition with a 100-fold molar excess of unlabelled specific competitor  whereas an unlabelled non-specific dna  did not affect the formation of the complexes. mutation of the myba and mybb sites in egcad <dig> and mutation of the myb site in egccr abolished the binding of recombinant egmyb <dig> to the regulatory fragments. this loss of binding occurred even when a single myb site remained intact in the egcad <dig> promoter, indicating that both myb sites are required for maximal binding of egmyb <dig> \n",
            "\n",
            "transactivation experiments were performed by co-transfecting both the egmyb <dig> cdna expressed under the control of the camv 35s promoter  and wild-type or mutated - <dig> egcad <dig> or - <dig> egccr promoters-gus fusions  into leaf mesophyll cells . when the wild-type - <dig> egcad <dig> promoter was co-transfected with the egmyb <dig> construct, gus activity was induced  <dig> -fold when compared to the control construct . interestingly, the endogenous activity of the myba-mutated promoter was notably higher than that of the wild-type promoter, suggesting that the myba element might bind a repressor, at least in leaf mesophyll cells. single or combined mutations of the myba and mybb elements reduced the gus expression driven by egmyb <dig> to the same level as their respective controls, thereby reflecting the loss of activation by egmyb <dig>  again, this effect was observed even with a single mutated myb site, indicating that both myb sites are required for transactivation of the egcad <dig> promoter by egmyb <dig>  transactivation of the - <dig> egccr wild-type promoter by egmyb <dig> by about 2-fold was also abolished upon mutation of the myb site .\n",
            "\n",
            "together, these in vitro and in vivo data indicate that, in the experimental conditions used, the three myb cis-elements are able to directly interact with egmyb <dig> transcription factor and are involved in the transcriptional activation of egcad <dig> and egccr genes mediated by egmyb <dig> \n",
            "\n",
            "effect of mutations in cis-elements on the protein binding activities of the egcad <dig> and egccr regulatory regions\n",
            "to investigate the role of myb and bs putative cis-elements of egcad <dig> and egccr promoters in the binding of transcription factors to the regulatory regions, emsas were performed using the regulatory fragments carrying several combinations of block mutations and tobacco protein extracts . for both wild-type promoter fragments, a large fast-migrating band was observed and another slower-migrating complex  was also detected when the protein extracts were used immediately after extraction . the large band was composed of two closely migrating complexes already described for egccr  and named l <dig> and l <dig> for egcad <dig> \n",
            "\n",
            "competition with a 100-fold molar excess of the corresponding unlabelled specific competitor dna fragments blocked the formation of all the complexes  whereas a similar amount of an unlabelled non-specific dna had no effect , indicating the specificity of the interactions.\n",
            "\n",
            "whereas mutation of the myba site in egcad <dig> had no major effect on the emsa result , mutation of the mybb site prevented the formation of both the slower migrating complex  and the lowest part of the faster migrating large band  . single or double mutations in the bs elements of egcad <dig> somewhat perturbed the pattern of the complexes with a notable decrease in h and a concomitant increase in l <dig> , especially when bsb was mutated . this concomitant increase in l <dig> was lost with the additional mutation in mybb . thus, when combined mutations including mybb were tested , similar results were obtained as with the mutation of mybb alone, ie loss of the h and l <dig> complexes. taken together, these data suggest that mybb is a key cis-element involved in the formation of the h and l <dig> complexes and that bsb, and to a lesser extent bsa, apparently influence the ratio between the h and l <dig> complexes. concerning the egccr promoter, only the mutation of the myb element led to an overall decrease in all the complexes .\n",
            "\n",
            "effect of mutations in cis-elements on vascular expression of egcad <dig> and egccr promoters\n",
            "to investigate the role of each cis-element on tissue-specific expression from egcad <dig> and egccr promoters, transgenic tobacco plants expressing gus under the control of the wild-type or mutated - <dig> egcad <dig> or - <dig> bp egccr promoters were generated.\n",
            "\n",
            "the bsa- or myba-mutated egcad <dig> promoters  drove a gus expression pattern similar to the wild-type - <dig> egcad <dig> promoter . in contrast, mutation of mybb resulted in a considerable global diminution of gus staining in all vascular tissues, suggesting that mybb could be bound by an activator factor . this decrease in gus staining was particularly pronounced in the xylem rays and in the parenchyma cells surrounding the primary xylem. although with a lower intensity, the cambial-phloem zone still appeared as a continuous ring like with the wild-type promoter . this gus expression pattern was also observed with the myba-mybb double mutant , confirming that myba does not play a crucial role in the promoter activity under these experimental conditions.\n",
            "\n",
            "upon mutation of the bsb cis-element, a substantial decrease in gus activity was observed in the vascular cambium and, to a lesser extent, in the immediately derived parenchymatous cells, ie the xylem rays on the inner side and the parenchyma cells between phloem islands on the outer side . thus, the cambial-phloem zone resulted in a spotted pattern with the spots showing gus activity restricted to the external phloem area. the internal phloem still displayed gus staining .\n",
            "\n",
            "the double mutation bsb-mybb reflected the combined actions of the single mutations with an overall reduction of gus activity.\n",
            "\n",
            "gus activity driven by the wild-type - <dig> egccr promoter construct showed a vascular pattern similar to that driven by the - <dig> egcad <dig> promoter and to that reported in  <cit>  using the full-length egccr promoter . this staining pattern was not affected by mutation of the bs <dig> cis-element. in contrast, gus activity in plants carrying the single myb mutant construct or the bs1-myb double mutant construct notably diminished in all vascular tissues. as already observed for egcad <dig>  gus expression remained evident in the internal phloem of stem .\n",
            "\n",
            "these observations were further corroborated by fluorimetric measurement of gus activities . egcad <dig> mybb and bsb single mutants showed a strong reduction of gus activity  when compared to the wild-type - <dig> promoter. similarly, mutation of the myb site in the egccr promoter resulted in a marked decrease of gus activity  when compared to the wild-type - <dig> promoter .\n",
            "\n",
            "taken together, these data provide evidence for a functional module comprising bsb-mybb in the egcad <dig> promoter: mybb appears to bind a general activator factor in all cell types whereas bsb might bind a protein specific to the cambium dividing cells. in the egccr promoter, the unique myb cis-element might fulfil these dual functions.\n",
            "\n",
            "discussion\n",
            "contribution of the egcad <dig> and egccr cis-elements to the protein binding and to transcriptional activities of the promoters\n",
            "here, we delineated a short regulatory region  in the egcad <dig> promoter which is required for expression both in xylem tissues and in dividing undifferentiated cells either cultivated in vitro or already present in planta embedded within other tissues such as the cambial cells. four cis-elements potentially involved in the regulation of egcad <dig> expression were identified within this region. they are arranged in two modules, each comprising an identical repeat of an unreferenced protein binding site  with the sequence ctggtt and an mbsiig consensus site for myb proteins. a simpler organisation exists in the  regulatory region of the egccr promoter with two putative cis-elements: a g-rich box  and a mbsiig myb consensus site  <cit> .\n",
            "\n",
            "our results with emsas and transgenic plants with mutated cis-elements revealed the importance of the mybb cis-element in egcad <dig> and of the myb cis-element in egccr for the formation of the high and low mobility complexes and for the transcriptional activation of the egcad <dig> and egccr promoters in vascular tissues.\n",
            "\n",
            "it is intriguing that neither emsas nor the transgenic plant approaches provided clear evidence of a functional role for myba in egcad <dig> promoter activity. this does not exclude its participation in other promoter functions, such as the response to external stimuli  <cit> , or mechanisms specific to woody plant species  <cit> , or expression in other parts of the plant, as does, for example, the aciii/myb element of the bean pal <dig> promoter, which drives expression specifically in flowers  <cit> . moreover, our observation in transactivation assays that the endogenous activity of the myba-mutated promoter in the absence of egmyb <dig> was notably higher than the wild-type promoter activity, suggests that the myba element might bind a repressor, at least in leaf mesophyll, a tissue where egcad <dig> and egccr promoters are poorly expressed. such repressors of lignin biosynthesis genes have already been demonstrated in the myb family; some of them have broad tissue expression patterns whereas others are tissue-specific or are preferentially expressed in particular tissues  <cit> .\n",
            "\n",
            "mutation of the bsa site in the egcad <dig> promoter or of the g-rich bs <dig> site in the egccr promoter had no major effect on complex formation in vitro or on promoter activity in transgenic plants. in contrast, mutation of bsb in egcad <dig> drastically reduced the formation of the h complex in emsas and altered the expression pattern in transgenic plants, leading to decreased promoter activity in cambial cells and in the immediately derived parenchyma cells.\n",
            "\n",
            "ongoing studies show that the proteins of the slow-migrating h complex are found in the phosphorylated fraction of tobacco nucleus proteins whereas those of the faster-migrating l <dig> complex are found in the unphosphorylated one . we believe that the h and l <dig> complexes which both bind the mybb element of egcad <dig> could differ from each other by the addition of one or more protein which can be subjected to phosphorylation. bsb that is located close to mybb in egcad <dig> could be involved in the formation of the phosphorylated complexes since its disruption affects the ratio between h and l <dig>  the possible involvement of the bsb site both in phosphorylated transcriptional complexes and in egcad <dig> expression in cambial cells  is consistent with the crucial and widely known role of phosphorylation mechanisms in metabolism connected to cell division  <cit> .\n",
            "\n",
            "the similar protein binding activities of the egcad <dig> and egccr regulatory fragments in emsas and the similar expression patterns driven by the promoters in vascular tissues suggest that the transcriptional mechanisms necessary for their coordinated expression may be mediated by their common myb cis-elements.\n",
            "\n",
            "functional architecture of the promoters\n",
            "the complex organisation of the egcad <dig> promoter with its cis-elements arranged in two similar modules  suggests that redundancy and mechanisms of cooperation or competition between cis-elements and trans-acting factors might be involved in the regulation of the promoter activity. in particular, emsas with recombinant egmyb <dig> and transactivation assays clearly demonstrated that the two myb sites of the egcad <dig> promoter cooperated for binding and activation. when studying in silico the distribution of myb sites in the proximal  <dig> bp of the arabidopsis phenylpropanoid gene promoters , it appears that two copies of myb elements and, more specifically, mbsiig or mbsii sites  are found in most promoters, generally separated by  <dig> to  <dig> bp . although the ccr gene involved in lignin biosynthesis in arabidopsis follows this rule, intriguingly the promoter of the ccr gene in eucalyptus contains a single myb element . the two nearby mbsiig of bean pal <dig> and pal <dig> genes  were demonstrated to be crucial for the transcriptional regulation of these genes in planta  <cit> , suggesting a functional significance for this promoter architecture. furthermore, as in our study, cooperative binding was also reported for the two myb elements of the bean pal <dig> promoter in the presence of the pine ptmyb <dig> protein  <cit> .\n",
            "\n",
            "combinatorial interactions between the transcription factors binding to myb elements and those binding to other closely located cis-elements are crucial for transcriptional regulation of the flavonoid branch pathway genes  <cit> . other putative cis-elements that could cooperate with myb elements were detected in the regulatory regions of egcad <dig> and egccr by using the place database . gt- <dig> elements, putative targets for trihelix-related transcription factors  <cit> , appear to be well conserved in the regulatory regions of cad <dig> and ccr in various eucalyptus species . interestingly, the hypersensitive site at - <dig> revealed in this study by in vivo footprinting falls within a wrky transcription factor binding site  <cit>  that is conserved in the cad <dig> regulatory regions; however it is more variable in ccr, especially in the invariant core which is essential for wrky protein binding. in addition, the cad <dig> regulatory region contains a conserved motif similar to the tere cis-element that is involved in secondary wall formation during differentiation of tracheary elements  <cit> ; this motif is, however, not found in the egccr regulatory region. presently, we cannot exclude the involvement of these putative cis-elements in cad or ccr expression. however, the strong conservation of the myb elements together with their key role in promoter activity, as demonstrated in this paper, indicates that the coordinated developmental regulation of egcad <dig> and egccr expression is mediated through myb transcription factors.\n",
            "\n",
            "myb candidates for transcriptional regulation of egcad <dig> and egccr genes\n",
            "egcad <dig> and egccr genes which, as demonstrated in this study, required functional myb binding sites in their promoters, exhibit expression in cell types consistent with their function in monolignol biosynthesis. their expression was indeed found  in xylem tissues undergoing lignification and in parenchyma cells surrounding and feeding the lignified tissues with monolignols and  in the vascular cambium in which small monolignol oligomers may be components of a signal transduction pathway leading to the cell division  <cit> .\n",
            "\n",
            "myb transcription factors responsible for egcad <dig> and egccr transcriptional activation should be found in the cell types exhibiting cad <dig> and ccr expression and the interplay between various myb factors or other transcription factors might contribute to the spatio-temporal expression patterns of egcad <dig> and egccr genes within specific cell types.\n",
            "\n",
            "particular myb transcription factors have been detected in the cambium  <cit>  and in xylem rays  <cit> , and might contribute to egcad <dig> and egccr promoter activities in such parenchymatous tissues. other myb factors appear more specifically expressed in relation to secondary wall formation in cells that synthesize lignins. the eucalyptus egmyb <dig> and the pine  ptmyb <dig> and ptmyb <dig> transcription factors are preferentially expressed in secondary xylem. when overexpressed, these transcription factors affect the expression of monolignol biosynthesis genes and the amount or composition of lignins  <cit> . in arabidopsis, the transcriptional network regulating secondary wall synthesis has been investigated  <cit> . atmyb <dig>  a close homologue of egmyb <dig> and ptmyb <dig>  appears to be a key switch in mediating the biosynthesis of the three major components of secondary walls  through the control of downstream, pathway-specific myb and other transcription factors  <cit> . recently, other arabidopsis myb factors, more closely related to ptmyb <dig>  were shown to directly regulate the promoters of lignin biosynthesis genes  <cit> . at the present time, the role of egmyb <dig> in the regulation of egcad <dig> and egccr gene expression must be clarified in order to determine whether egmyb <dig> activates the transcription of these genes by direct binding on their promoters in vivo or by an upstream control as suggested for atmyb <dig> \n",
            "\n",
            "a complex picture of the transcriptional regulatory network controlling lignin biosynthesis genes is therefore emerging, with various myb factors acting at different levels. as in the flavonoid branch pathway  <cit> , the myb factors controlling the monolignol pathway are likely to cooperate within multiprotein complexes with other protein partners that remain to be characterised.\n",
            "\n",
            "CONCLUSIONS\n",
            "the overall goal of this project was to characterize the functional role of putative cis-elements from promoters of genes involved in the monolignol biosynthetic pathway. in this study, we first demonstrated that the myb cis-elements found in the delineated regulatory regions of egcad <dig> and egccr promoters are able to be bound by myb factors to form specific dna-protein complexes in vitro. myb cis-elements play a key role in the vascular expression of these two genes and support the hypothesis that myb elements serve as a common means for the coordinated regulation of genes in the entire lignin biosynthetic pathway. moreover, a novel functional cis-element was identified and shown to be involved, possibly in combination with the myb cis-element, in the transcriptional regulation of egcad <dig> in cambial cells. a comprehensive understanding of the functional cis-elements involved in the transcriptional regulation of egcad <dig> and egccr is the first step to dissect the regulatory network controlling lignin biosynthesis. future studies will aim at the identification of the transcription factors that bind these cis-elements and at the mechanisms by which these transcription factors interplay with each other to provide the final spatio-temporal regulation of the lignin pathway.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study, we investigated the functional role of putative cis-elements in the transcriptional regulation of egcad <dig> and egccr genes in vascular tissues.\n",
            "the egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes.\n",
            "the egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes.\n",
            "the egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes.\n",
            "the egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes.\n",
            "the egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes.\n",
            "the egcad <dig> promoter is a key cis-element involved in the transcriptional regulation of egcad <dig> and egccr genes in planta.\n",
            "the egcad <dig> promoter is\n",
            "39\n",
            "[{'summary_text': 'in the present study, using a carefully defined and relatively large data set with  <dig> known wild-type bcrp substrates and non-substrates, we have developed a svm-based in silico model for prediction of wild-type bcrp substrates and non-substrates of certain enzymes and transporters.\\nin this study, we have developed a svm-based in silico model for prediction of wild-type bcrp substrates and non-substrates based on the selected svm prediction model.\\nin this study, we have developed a svm-based in silico model for prediction of wild-type bcrp substrates and non-substrates with an overall prediction accuracy of ~73% for an independent external validation data set of  <dig> compounds.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "human breast cancer resistance protein  is an atp-binding cassette  efflux drug transporter  <cit> . bcrp is one of the abc transporters that confer resistance to a large number of structurally and chemically unrelated chemotherapeutic agents through atp hydrolysis-dependent efflux transport of these drugs  <cit> . the substrates of bcrp have been rapidly expanding to include not only chemotherapeutics such as mitoxantrone, topotecan and imatinib, but also non-chemotherapeutic drugs such as prazosin, glyburide, nitrofurantoin and statins as well as non-therapeutic compounds such as dietary flavonoids, porphyrins, estrone 3-sulfate, and the dietary carcinogen 2-amino-1-methyl-6-phenylimidazopyridine  <cit> . bcrp is also highly expressed in organs important for the absorption , elimination , and distribution  of drugs and xenobiotics  <cit> , and has recently been recognized by the fda as one of the most important drug transporters involved in clinically relevant drug disposition and drug-drug interactions  <cit> . due to the clinical importance of bcrp in drug resistance and drug disposition, it should be of high value to develop cost-effective methods for evaluation of transport of drugs or drug candidates by bcrp so that the pharmacokinetics, efficacy, safety, and tissue levels of these compounds may be predicted. one of such methods would be the development of in silico models for prediction of bcrp substrates.\n",
            "\n",
            "indeed, in the recent years, in silico prediction models have emerged into the pipeline of drug discovery which allow initial screening and selection of promising compounds from chemical libraries and large databases. in addition, these models could provide information concerning the mechanism of protein-ligand interactions. in silico methods for prediction of protein-ligand interactions including transport characteristics can be divided into ligand-based and protein structure-based approaches. with protein structure-based methods such as molecular docking, structures and physicochemical characteristics of an intermolecular complex formed between interacting protein and ligand could be predicted if high resolution structures of both the protein and the ligand under question are available. high resolution structures of bcrp have not been resolved. homology models of bcrp have recently been developed and await further experimental validation  <cit> . although these homology models can be used for docking calculations and interpretation of biochemical data, results obtained are unlikely reliable for drug design and screening. in contrast, ligand-based methods based on structural similarity of ligands to known substrates generally yield much greater prediction accuracies than protein structure-based methods.\n",
            "\n",
            "among ligand-based methods, one common approach is to develop quantitative structure-activity relationship models . the objective of sar and qsar analysis is to establish a correlation between descriptors which represent information of molecular structures of ligands and biological activities for a series of biologically and structurally characterized compounds. various sar and qsar models for bcrp inhibitors have been published  <cit> . several sar and qsar studies suggest that lipophilicity of ligands is a good predictor for bcrp inhibition  <cit> , but other studies argue that this property is not significant  <cit> . a planar structure of inhibitors seems to be necessary for binding to the active site of bcrp  <cit> . with respect to prediction of bcrp substrates, only one sar study of camptothecin analogues revealed that hydrogen bond formation might be important for substrate recognition by bcrp  <cit> . one common feature of these sar and qsar models is that these models are usually built using a congeneric series of molecules and thus may not be valid for other classes of compounds. for this reason, more sophisticated techniques are required for classification of bcrp ligands.\n",
            "\n",
            "another ligand-based approach is to use statistical learning methods to predict features based on properties of examples, and compounds of any chemical structures can be used. of these methods, the support vector machine  method is most frequently used and has proved valuable in a wide range of applications. svm has gained popularity in the chemo- and bioinformatics field due to its ability to classify objects into two classes based on their structural features. in particular, the svm method was useful for classification of molecules as substrates or non-substrates of enzymes or transporters. for example, several studies have been reported for prediction of substrates and non-substrates of p-glycoprotein  using svm with generally greater than 70% prediction accuracies  <cit> . zhong et al. recently reported a genetic algorithm-conjugate gradient-support vector machine  procedure for prediction of bcrp substrates and non-substrates  <cit> . although these studies are highly valuable, the scientific community has no open access to most of these published in silico models. there are a few svm-based free web servers for predicting substrates and non-substrates of certain enzymes and transporters. for example, mishra et al. reported a web server for cytochrome p <dig> enzymes  <cit> , and our laboratories published a free web server for prediction of p-gp substrates and non-substrates using the svm method   <cit> .\n",
            "\n",
            "therefore, in the present study, we have compiled a relatively large data set of bcrp substrates and non-substrates collected from literature and developed an svm-based in silico model for prediction of wild-type bcrp substrates and non-substrates. this prediction model has been integrated into a free web server  which allows the users to predict the capability of wild-type bcrp to transport the query ligands and calculate their physiochemical properties including molecular weight, logp value, and polarizability.\n",
            "\n",
            "methods\n",
            "data set\n",
            "all known wild-type bcrp substrates and non-substrates used in this study were taken from published data in the literature. information for some of these compounds in the data set was obtained through searching the university of washington metabolism & transport drug interaction database . this data set is based on results of in vitro transport assays such as the membrane vesicle uptake assay, the efflux assay using intact mammalian cells over-expressing bcrp, and transwell transport assay using mdckii/bcrp cells. results from in vitro drug resistance assays were also used. however, results from drug-stimulated atpase assays were not used because many substrates do not stimulate atpase activity of bcrp. in the case of conflicting evidence, only the results confirmed by at least two independent studies were accepted. this data set contains  <dig> bcrp substrates and  <dig> non-substrates with highly diverse chemical structures. we noticed that  <dig> out of the  <dig> substrates had multiple reports. however, only about  <dig> out of the  <dig> non-substrates had multiple reports. it is worth noting that the drug-selected bcrp mutants with amino acid substitutions at position  <dig> exhibit altered substrate specificity. for example, doxorubicin, rhodamine  <dig> and lysotracker green are substrates of the mutant r482g or r482t, but cannot be efficiently transported by wild-type bcrp  <cit> . therefore, such compounds were classified as non-substrates of wild-type bcrp which was the subject of this study. of the  <dig> compounds ,  <dig> compounds  were randomly used in the training and test subsets in various training/test ratios, and  <dig> compounds  were defined as the independent external validation subset. all compounds are listed in additional file 1: table s <dig>  the chemical structures of all these molecules are shown in two sdf files provided as additional files  <dig> and  <dig> which can be viewed using the free marvinview software .\n",
            "\n",
            "support vector machine \n",
            "the svm method we used in this study is essentially the same as previously described  <cit> . briefly, the standard procedure of classification by svm can be divided into four stages. in the first stage, all compounds in the data set were defined as substrates and non-substrates of wild-type bcrp. then, the molecules were characterized using molecular descriptors. the data set was then split into the training and test subsets, and an independent external validation subset was also created. in the second stage, the compounds in the training set were presented as points in a high-dimensional space according to their molecular descriptors. in this high-dimensional space, a hyperplane was determined to separate objects into substrate and non-substrate groups. since various hyperplanes allow separation of objects, a hyperplane that maximizes the margin needs to be constructed. in the third stage, the models constructed using the training data set were used to calculate prediction accuracy for a test set to evaluate the models. finally, the models were validated using the independent external data set.\n",
            "\n",
            "chemical structures of all wild-type bcrp substrates or non-substrates used in this study were downloaded from the pubchem database . some compounds were extracted from the original publications and redrawn by means of marvinview . all molecules were subject to geometry optimization using the molconvert software , which applies the dreiding molecular mechanics force field, and to calculation of the gasteiger partial charges  <cit> . the dragonx software  was used to calculate a total of  <dig> molecular descriptors for each molecule. the descriptors with more than 80% zero values and too small standard deviation values  were eliminated. the libsvm software  was then used for svm calculations. linear, polynomial, and radial basis function  kernels were tested in this study. rbf is calculated using the equation k = exp, where γ is a kernel parameter, xi and xj are instance label pairs, and k is the kernel function. the prediction power of svm is greatly influenced by the selection of kernel, the kernel parameter γ, and soft margin parameter c.\n",
            "\n",
            "the best combination of c and γ was selected by a grid-search with exponentially growing sequences of c and γ. each combination of parameter choices was checked, and the parameters with the best validation accuracy were selected. after the best parameters c and γ were found, the whole training set was trained again to generate the final model. the feature selection tool fselect.py  provided by the libsvm developer was used to measure the relative importance of each feature. for each feature, an f-score can be calculated using fselect.py. generally, the larger the f-score, the more likely the feature is discriminative. therefore, f-score was used as a feature selection criterion. features with high f-scores were selected and then svm was applied. high f-score features were gradually added until the validation accuracy decreased. descriptors were checked for their correlation. among the descriptors with a correlation of  <dig> , the descriptors with higher f-scores were kept for further svm calculations. prediction power of the above-described svm method was evaluated based on the number of true positive , true negative , false positive , and false negative  predictions. additional parameters that are widely used, namely accuracy , sensitivity , specificity , and the matthews correlation coefficient , were also calculated using the following equations:\n",
            "\n",
            " acc=tp+tn/tp+tn+fp+fn×100se=tp/tp+fn×100sp=tn/fp+tn×100mcc=tp×tn-fp×fn/tp+fntp+fptn+fptn+fn1/ <dig> \n",
            "\n",
            "the web server\n",
            "the best prediction model generated using the svm method described above has been integrated into a free web server . this web server allows the users to predict as to whether a query compound is likely to be a bcrp substrate. the chemical structure of the query compounds can be uploaded or drawn in by the users using the built-in chemaxon marvin java applet. the web server is linked to pubchem so that any query compounds can be directly retrieved with text search. any compounds of interest can be searched by their names, uploaded in pdb, mol, mol <dig>  hin, or smiles format or drawn in using a marvin applet by the users. structural conversions and 3-dimensional geometry optimization by the dreiding method are carried out using the molconvert software. two-dimensional and 3-dimensional molecular descriptors are calculated using the dragonx software.\n",
            "\n",
            "RESULTS\n",
            "since svm tends to find a linear separating hyperplane with the maximal margin in a high-dimensional space by using a penalty parameter of the error term and a kernel function, we first investigated the influence of kernel function on the performance parameters. svm prediction performance parameters of  <dig> runs with different kernel functions  are provided in table  <dig>  the data shown in table  <dig> were obtained using a training set of  <dig> compounds, a test set of  <dig> compounds, and an external validation set of  <dig> compounds. the test set was used for choosing the best kernel function, and test performances were used as the criteria for selecting the best kernel. it should be emphasized that the external validation set was not used in the model building steps. it appeared that polynomial kernel function produced generally lower prediction accuracy compared to linear kernel function and rbf. although performance parameters associated with linear and rbf kernels were comparable, rbf provided slightly better prediction results. this is consistent with a general practice that rbf is the most popular choice of kernel function in svm. based on results of this preliminary evaluation, only rbf was used in further calculations.\n",
            "\n",
            "acc, accuracy ; sp, specificity ; se, sensitivity ; mcc, the matthews correlation coefficient . the external data set was only used to validate the prediction power of the models constructed, and was not used for model selection.\n",
            "\n",
            "due to the limited number of currently known wild-type bcrp substrates and non-substrates, if more compounds are used in the training set, fewer compounds can be used in the test set, likely resulting in less reliable test prediction outcome. therefore, we next investigated the influence of the number of compounds in the training and test sets on prediction accuracy. the results of svm calculations performed with varying training/test set ratios are shown in table  <dig>  overall, we did not observe significant differences in the performance parameters with different training/test ratios. however, the mcc values at the training/test ratios of  <dig> / <dig> ;  <dig> / <dig>  and  <dig> / <dig>  appeared to be comparable and slightly better than those at other ratios. similar to the kernel selection, only the test and training data sets were used for model construction, and the external validation data set was only used for validation of the models constructed. thus, the training/test ratio of  <dig> / <dig>  was chosen for further calculations in order to maximize the chemical space occupied by the molecules in the training set.\n",
            "\n",
            "the total number of molecules used in the training and test data sets were  <dig>  the number of molecules in the external validation data set was  <dig>  acc, accuracy ; sp, specificity ; se, sensitivity ; mcc, the matthews correlation coefficient . the external data set was only used to validate the prediction power of the models constructed, and was not used for model selection.\n",
            "\n",
            "there is no general rule regarding selection of the best svm prediction model. the run that provides the highest prediction accuracy for the training set may be selected. however, such an approach could be misleading because a model with the highest prediction accuracy  for a training set does not necessarily produce the highest prediction accuracy  neither for the test set nor for the independent external validation set due to a phenomenon called overfitting. it is therefore necessary to consider prediction characteristics of both the training and test sets when the best svm model is to be selected. in our study for prediction of p-gp substrates  <cit> , we proposed the following approach. first, the differences in prediction accuracy between the training and test sets were calculated, and the models with the smallest difference were taken into account. second, of the models with the smallest difference in prediction accuracy between the training and test sets, those built with the smallest number of molecular descriptors were considered because inclusion of too many descriptors may again produce overfitted models and the inclusion of unnecessary or irrelevant descriptors creates noise in the model. we showed that classification of individual compounds in the independent external validation set as substrates or non-substrates of p-gp was very similar among the potentially best models  <cit> . using the same approach, the best svm prediction model was selected for substrates of wild-type bcrp and the prediction performance parameters of the selected model are shown in table  <dig>  the selected model showed an overall prediction accuracy of ~73% for the external validation data set. also, wild-type bcrp substrates were generally predicted with a higher accuracy than non-substrates . in the svm prediction, there is generally a borderline region where the structural properties of substrates and non-substrates are very similar, and therefore compounds in this borderline region cannot be separated by their structural properties alone. in our model, molecules in this borderline region were mainly predicted as substrates. this may contribute to the higher prediction accuracy for substrates.\n",
            "\n",
            "tp, true positive; tn, true negative; fp, false positive; fn, false negative; acc, accuracy ; sp, specificity ; se, sensitivity ; mcc, the matthews correlation coefficient .\n",
            "\n",
            "classification of individual compounds as wild-type bcrp substrates or non-substrates was very similar among the  <dig> selected models in the first step. table  <dig> shows the overlap in classification  among these  <dig> models. as can be seen in table  <dig>  classification of compounds to be wild-type bcrp substrates or non-substrates in different models were highly similar, that is, the overlap values varied between  <dig> % and  <dig> %, with an average of  <dig> %. for example,  <dig> % of compounds were predicted to be in the same category  by the model  <dig> compared to the model  <dig>  which has the highest prediction accuracy. this finding suggests that, to obtain reliable predictions, selection of the best model is likely not as important as, for example, the compilation of high quality data sets. in this regard, we would like to point out that the data set collected in this study regarding whether a specific compound is a substrate of wild-type bcrp could be obtained using different transport methods. this does not affect the prediction accuracy of our model as long as the different transport methods produce the same result as to if the compound is a bcrp substrate. this is because the svm model only makes qualitative , not quantitative  prediction.\n",
            "\n",
            "the overall prediction accuracies  of the  <dig> experimental models  <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>   <dig>  and  <dig> were  <dig> %,  <dig> %,  <dig> %,  <dig> %,  <dig> %,  <dig> %,  <dig> %,  <dig> %,  <dig> %, and  <dig> %, respectively.\n",
            "\n",
            "recently, an svm study based on a different data set was published by zhong et al.  <cit>  and reported a higher overall prediction accuracy for bcrp substrates and non-substrates . it should be noted that the compounds used by zhong et al. were only divided into two sets, namely a training set and a test set, without an independent external validation data set. also, the test set used by zhong et al. was not independent when it was used for the selection of the best model. therefore, their results cannot be directly compared to the data of this study. this is because, besides the training and test sets, we also used an independent external validation data set to evaluate prediction outcome and calculate prediction accuracies of the selected best model. moreover, certain compounds in the data sets of zhong et al. were actually the same under different names . additionally, a number of compounds were classified as bcrp substrates  by zhong et al., but as non-substrates in this study as explained in the methods section.\n",
            "\n",
            "we found that the final svm model selected in this study used the molecular descriptors shown in table  <dig>  that is, the following descriptors were found to be used in the final model: mean information index on atomic composition , spherosity , morse signals, and a mass weighed gateway descriptor. these descriptors suggest that the 3-dimensional structure of a substrate is likely the determining factor for bcrp/substrate interactions. the results of classification by this svm model for all compounds used in this study are shown in additional file 1: table s <dig> \n",
            "\n",
            "in order to make the svm model publicly available, we developed a free web server . this web server enables the users to predict if a query compound is a bcrp substrate based on the selected svm prediction model of this study.\n",
            "\n",
            "CONCLUSIONS\n",
            "in summary, bcrp is an abc drug transporter that confers multidrug resistance in cancers and plays an important role in drug disposition. therefore, it is important to develop in silico prediction models for bcrp substrates that could be used as cost-effective tools for screening of drug candidates in early drug discovery stage and for identification of bcrp substrates among existing drugs so that potential drug-drug interactions may be predicted. in the present study, using a carefully defined and relatively large data set with  <dig> known wild-type bcrp substrates and non-substrates, we have developed an svm model for prediction of wild-type bcrp substrates and non-substrates with an overall prediction accuracy of ~73% for an independent external validation data set of  <dig> compounds. the prediction accuracy for wild-type bcrp substrates was ~76%, which is higher than that for non-substrates. the molecular descriptors used by this svm model suggest that the 3-dimensional structure of a compound is possibly a predominant factor in determining bcrp/substrate interactions. this svm prediction model has been integrated into a web server  which is freely available to the scientific community. we believe that availability of such a prediction model will facilitate drug discovery as well as basic research investigating the role of bcrp in drug transport.\n",
            "\n",
            "abbreviations\n",
            "abc: atp-binding cassette; bcrp: breast cancer resistance protein; abcg2: the second member of abc transporter subfamily g; svm: support vector machine; acc: accuracy ; sp: specificity ; se: sensitivity ; mcc: matthews correlation coefficient ; sd: standard deviation; tp: true positive; fp: false positive; tn: true negative; fn: false negative.\n",
            "\n",
            "competing interests\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "authors’ contributions\n",
            "eh: data analysis and interpretation, manuscript preparation, and final approval of the version to be published; ih: collection of data sets, data analysis and interpretation, manuscript preparation; sc: collection of data sets and critical revision of the manuscript; ir-m: collection of data sets and critical revision of the manuscript; zb: data analysis and interpretation, manuscript preparation, and final approval of the version to be published; qm: collection of data sets, data analysis and interpretation, manuscript preparation, and final approval of the version to be published. all authors read and approved the final manuscript.\n",
            "\n",
            "supplementary material\n",
            "additional file 1\n",
            "all the wild-type bcrp substrates and non-substrates used in this study and classification of these compounds by the selected svm prediction model developed in this study are shown in the supplemental table s <dig> \n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 2\n",
            "the chemical structures of all wild-type bcrp substrates are shown in this sdf file which can be viewed using the free marvinview software . the order of molecules in the sdf file is according to the supplementary table s <dig> \n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 3\n",
            "the chemical structures of all wild-type bcrp non-substrates are shown in this sdf file which can be viewed using the free marvinview software . the order of molecules in the sdf file is according to the supplementary table s <dig> \n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements\n",
            "this work was supported by the hungarian state and the european union , under the aegis of new hungary development plan , and by a grant from the national institutes of health, gm <dig> .\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in the present study, using a carefully defined and relatively large data set with  <dig> known wild-type bcrp substrates and non-substrates, we have developed a svm-based in silico model for prediction of wild-type bcrp substrates and non-substrates of certain enzymes and transporters.\n",
            "in this study, we have developed a svm-based in silico model for prediction of wild-type bcrp substrates and non-substrates based on the selected svm prediction model.\n",
            "in this study, we have developed a svm-based in silico model for prediction of wild-type bcrp substrates and non-substrates with an overall prediction accuracy of ~73% for an independent external validation data set of  <dig> compounds.\n",
            "\n",
            "40\n",
            "[{'summary_text': 'protein-protein interaction  networks are a common mechanism by which hub proteins bind to their multitude of partners.\\nprotein-protein interaction  networks have been proposed to provide several biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks have been proposed to provide biological advantages.\\nprotein-protein interaction  networks'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "protein-protein interaction  networks integrate various biological signals including those used for energy generation, cell division and growth to give a few notable examples. the architectures of the ppi networks indicate that they are nearly scale free  <cit> . that is, a log-log plot of the number of nodes versus the number of links  at each node gives a straight line with a negative slope. the negative slope means that these sets of interactions contain a few proteins  with many links and many proteins  with only a few links. the term ‘hub protein’ is relative to the other proteins in a given ppi network, with no agreed upon number of links separating hubs and non-hubs.\n",
            "\n",
            "several networks such as the internet, cellular phone systems, social interactions, author citations, and so on, exhibit scale-free architecture. with regard to ppis, scale-free network architecture is suggested to provide several biological advantages. for example, given the small fraction of hub proteins, random deleterious mutations will more likely occur in non-hub proteins. the elimination of the functions of such non-hub proteins typically have small effects and so, generally, are not serious. in contrast, a deleterious mutation of a hub protein is more likely to be lethal  <cit> . another advantage is that signals can traverse these networks in a small number of steps, so signal transduction efficiency is improved compared to that expected for random networks  <cit> .\n",
            "\n",
            "understanding ppi network evolution across different species is an important problem  <cit> . from this body of work, hub proteins appear to evolve more slowly than non-hub proteins, an observation that is consistent with fisher's classic proposal that pleiotropy constrains evolution  <cit> . some proteins have multiple, simultaneous interactions   <cit>  while others have multiple, sequential interactions   <cit> . date hubs appear to connect biological modules to each other  <cit>  while party hubs evidently form scaffolds that assemble functional modules  <cit> .\n",
            "\n",
            "the idea that ppi networks use scale-free network topology is receiving considerable attention, but some caution is in order. currently constructed networks are noisy, with both false positive and false negative interactions  <cit> . also, network coverage to date  <cit>  is not sufficient to prove scale-free architecture  <cit> . whether ppi networks are truly scale-free or only approximately so, it nevertheless appears to be true that a relatively small number of proteins interact with many partners, either as date hubs or party hubs, while many proteins interact with just a few partners.\n",
            "\n",
            "the ability of a protein to bind to multiple partners was suggested to involve new principles  <cit> . indeed, neither the lock-and-key  <cit>  nor the original induced-fit  <cit>  readily explains how one protein can bind to multiple partners. note that the original induced fit mechanism was defined as changes in a structured binding site upon binding to the partner  <cit> , changes that are analogous to a glove altering its shape to fit a hand. on the other hand, both theoretical and experimental studies over many years suggested that natively unstructured or intrinsically disordered proteins form multi-structure ensembles that present different structures for binding to different partners  <cit> . based on these prior studies, we proposed that molecular recognition via disorder-to-order transitions provides a mechanism for hub proteins to specifically recognize multiple partners  <cit> . we pointed out earlier that intrinsic disorder could enable one protein to associate with multiple partners  and could also enable multiple partners to associate with one protein   <cit> .\n",
            "\n",
            "recent bioinformatics studies support the importance of protein disorder for hubs  <cit> . while disorder appears to be more clearly associated with date hubs  <cit>  than with party hubs, some protein complexes clearly use long regions of disorder as a scaffold for assembling an interacting group of proteins  <cit> . thus, the importance of disorder for party hubs needs to be examined further. additional evidence for the importance of disorder for highly connected hub proteins comes from a structure-based study of the yeast protein interaction network  <cit> . the authors considered only interactions that could be mediated by domains with known structures and found that the degree distribution of the resulting network contained no proteins with more than  <dig> interactions, which is more than an order of magnitude less than is observed in one unfiltered, high confidence dataset . this result indicates that a structure-based view of hub proteins is insufficient to explain the multitude of partners that interact with hub proteins.\n",
            "\n",
            "to improve understanding of the use of disorder for binding diversity, we studied two prototypical examples: p <dig> and 14-3- <dig>  both are hubs that are clearly involved in crucial biological functions. for example, p <dig> is a key player in a large signaling network involving the expression of genes carrying out such processes as cell cycle progression, apoptosis induction, dna repair, response to cellular stress, etc.  <cit> . loss of p <dig> function, either directly through mutation or indirectly through several other mechanisms, is often accompanied by cancerous transformation  <cit> . cancers with mutations in p <dig> occur in colon, lung, esophagus, breast, liver, brain, reticuloendothelial tissues and hemopoietic tissues  <cit> . the p <dig> protein induces or inhibits over  <dig> genes, including p <dig>  gadd <dig>  mdm <dig>  igfbp <dig>  and bax <cit> .\n",
            "\n",
            "the four regions or  domains in p <dig> are the n-terminal transcription activation domain, the central dna binding domain, the c-terminal tetramerization domain, and the c-terminal regulatory domain. the last two could be considered to be a single c-terminal domain with two subregions. the transactivation region interacts with tfiid, tfiih, mdm <dig>  rpa, cbp/p <dig> and csn5/jab <dig> among many other proteins  <cit> . the c-terminal domain interacts with gsk3β, parp- <dig>  taf <dig>  trrap, hgcn <dig>  taf, 14-3- <dig>  s100b and many other proteins  <cit> .\n",
            "\n",
            "as for 14-3- <dig> proteins, they contribute to a wide range of crucial regulatory processes including signal transduction, apoptosis, cell cycle progression, dna replication, and cell malignant transformation  <cit> . these activities involve 14-3- <dig> interactions with various proteins in a phosphorylation-dependent manner. more than  <dig> proteins have been shown to interact with members of 14-3- <dig> family  <cit> , with these 14-3-3-interacting proteins amounting to approximately  <dig> % of the human proteome  <cit> . one proposed functional model is that 14-3- <dig> binds to the specific target as a molecular anvil causing conformational changes in the partner. in their turn, these changes can affect enzymatic  activity of a target protein, or mask or reveal specific motifs that regulate its localization, activity, phosphorylation state, and/or stability  <cit> .\n",
            "\n",
            "the 14-3- <dig> protein has at least nine sequence isomers, called α, β, γ, δ, ε, η, σ, τ, and ζ  <cit> . all isomers are structured dimers with grooves that bind to more than  <dig> different partners, and the different partners have different sequences for their binding regions. screening experiments have identified individual peptides that bind to all the different isomers, suggesting that the binding grooves in the different isomers have some common features  <cit> . a recent bioinformatics study suggests that the partners of 14-3- <dig> utilize intrinsic disorder for binding  <cit> .\n",
            "\n",
            "the interactions of p <dig> and 14-3- <dig> with their partners as reported previously  <cit>  are examined herein but from an order-disorder point of view. in the case of p <dig>  different regions in the disordered tails enable this protein to bind to multiple partners at the same time. in addition, one single region of disorder adopts clearly different secondary structures and uses the same amino acids to different extents in different binding interactions. for this case the plasticity of the disordered region clearly enables the binding to multiple partners. in the case of 14-3- <dig>  the different partners have distinct sequences. their interactions with 14-3- <dig> show characteristics, such as hydrogen bonds between side chains of 14-3- <dig> and the backbone of the partners and such as hydrogen bonds between the backbone of the partners and water, indicating that the two partners were very likely unfolded in water just prior to association with 14-3- <dig>  the distinct sequences of the partners do not adopt identical backbone structures, and the various side chain interactions between 14-3- <dig> and the two different partners involve induced-fit adjustments of the 14-3- <dig> structure. overall, these studies show how the plasticity of disordered proteins is used to enable the binding diversity of hub proteins, both for a single disordered region binding to multiple partners and for multiple disordered regions binding to the same partner. an earlier, less complete version of this work was reported at the biocomp’ <dig> meeting  <cit> .\n",
            "\n",
            "RESULTS\n",
            "intrinsic disorder and the molecular interactions of p53\n",
            "the p <dig> molecule interacts with many other proteins in order to carry out its signal transduction function. a number of these are downstream targets, such as transcription factors, and others are activators or inhibitors of p53's transactivation function. many of these interactions have been mapped to regions of the p <dig> sequence : the n-terminal domain , the c-terminal domain , and the dna binding domain . these domains have also been characterized in terms of their structure or lack thereof  and blue  segments), where the dna binding domain is intrinsically structured and the terminal domains are intrinsically disordered  <cit> . while the tetramerization domain is structured, the structure is acquired upon the formation of the complex. additionally, multiple different posttranslational modifications have been identified in p <dig> . these modifications are relevant here because they are a common method for altering protein interactions.\n",
            "\n",
            "comparing the regions of order and disorder reveals a strong bias towards the localization of the interactions within the intrinsically disordered regions. overall, 60/ <dig> = 71% of the interactions are mediated by intrinsically disordered regions in p <dig>  a bias toward intrinsically disordered regions is even more pronounced in the sites of posttranslational modifications, with 86%, 90%, and 100% of observed acetylation, phosphorylation, and protein conjugation sites, respectfully, found in the disordered regions. this is consistent with previous observation of a strong bias for post translational modifications toward intrinsically disordered regions  <cit> . this concentration of functional elements within intrinsically disordered regions compares to just 29% of the residues being disordered  <cit> . clearly, p <dig> exhibits a highly biased use of disordered regions for mediating and modulating interactions with other proteins.\n",
            "\n",
            "in addition to experimentally characterized disorder, predictions of intrinsic disorder for p <dig> using both pondr vl-xt  <cit>  and vsl <dig> predictions  <cit>  were carried out . the latter is one of the highest accuracy prediction algorithms available  <cit> , whereas the former has been observed to be especially useful in identifying binding regions within longer regions of disorder  <cit>  and to be much better at identifying such sites as compared to a number of different disorder predictors  <cit> . both predictors give good agreement with the experimental determination of intrinsic disorder  <cit> , and in the case of p <dig> both of their predictions agree well with experimental characterization.\n",
            "\n",
            "analysis of associations involving p <dig> using 3d structures\n",
            "the structures of  <dig> complexes between various regions of p <dig> and unique binding partners have been determined . for  <dig> of these partners, the interactions are mediated by regions experimentally characterized as intrinsically disordered, where pondr vl-xt detects the majority of these binding regions as short predictions of order within a longer prediction of disorder. these structures are complexes between p <dig> and endogenous partners: cyclin a  <cit> , sirtuin  <cit> , cbp  <cit> , s100ββ  <cit> , set <dig>  <cit> , tgcn <dig>  <cit> , rpa <dig>  <cit> , mdm <dig>  <cit> , tfb <dig>  <cit> , and itself  <cit> . the remaining  <dig> interactions are mediated by the structured dbd, namely between p <dig> and  <dig> endogenous partners – dna  <cit> , 53bp <dig>  <cit> , and 53bp <dig>  <cit>  – and one exogenous partner – the large-t antigen  from simian virus  <dig>  <cit> .\n",
            "\n",
            "protein complexes can be formed from the association of structured proteins, by the folding of one disordered protein onto the surface of a structured partner, or by the coupled folding and binding of intrinsically disordered proteins  <cit> . nussinov and collaborators  <cit>  showed that a plot of normalized monomer area  versus normalized interface area  nicely separates complexes formed from structured proteins as compared to complexes formed from unfolded proteins by coupled binding and folding. that is, associations of structured proteins exhibit small nmas and nias and so lie near the origin of the nma-nia plot. conversely, complexes formed by coupled binding and folding have much larger nmas and nias, and so are spread out and lie far from the origin of the nma versus nia plot. indeed, a linear boundary separates the two groups  <cit> . it should be emphasized that the nma-nia plot approach is a global measure of a proteins order-disorder monomeric state, and has not been characterized on local order-disorder transitions .\n",
            "\n",
            "as described in more detail in the implementation, by developing two separate nma-nia plots, one for each partner of a complex , and then by determining the distance to the linear boundary in each plot, a double nma-nia plot  can be produced. interacting pairs can be divided into the  <dig> groups given above, namely:  both partners are structured  of 2b), i.e. both distances are negative;  one partner is structured and the second partner is disordered, i.e. the ordered partner has a negative distance and the disordered partner has a positive distance  of 2b); and  both partners are intrinsically disordered, i.e. both distances are positive  of 2b).\n",
            "\n",
            "a double nma-nia plot was calculated for  <dig> of the p <dig> complex structures . the p53-dna complex was excluded since the nma-nia analysis is not relevant for nucleic acids. in the general case, the distinction between the distances of the two partners is arbitrary, so that the double nma-nia plot is symmetric about the diagonal. however, here we restrict the p <dig> distance to one axis, so that group  is split into two sub groups ): the p <dig> segment is disordered and the partner is ordered ) and the p <dig> region is ordered and the partner is disordered ). one interaction, the formation of the p <dig> tetramer, is in the third group ) and so therefore likely involves an association between two disordered partners. this is consistent with experimental data  <cit> . at the opposite side of the spectrum, the three protein-protein complexes involving the p <dig> dbd domain are in group  <dig> ), indicating that all three are ordered prior to binding, which is consistent with the solution of structures for identical or homologous monomeric domains . the other nine p <dig> complexes found so far in the pdb are all in the group  <dig> quadrant , and so all likely involve a disordered region of p <dig> associating with a structured partner. these results are likewise consistent with experimental data. that is, these p <dig> regions are disordered in the unbound state  <cit> , and the isolated partners appear to be structured: mdm <dig>  <cit> , rpa <dig>  <cit> , tfb <dig>  <cit> , tgcn <dig>  <cit> , cyclin a/cdk <dig>  <cit> , sirtuin  <cit> , cbp  <cit> , s100ββ  <cit>  and set <dig>  <cit> ).\n",
            "\n",
            "in summary, these data point out the importance of disorder-to-order transitions for many of the structurally characterized interactions involving the p <dig> hub protein. while many previous studies discuss these same interactions, to our knowledge the importance of disorder has not been emphasized in those previous studies.\n",
            "\n",
            "analysis of multiple specificities in the p <dig> c-terminus\n",
            "so far, complexes involving one region of the p <dig> sequence bound to four different partners have been determined and deposited in the pdb. this region is from residue  <dig> to  <dig> in the p <dig> sequence bound to one of the following: cyclin a  <cit> , sirtuin  <cit> , cbp  <cit> , or s100ββ  <cit> . the regions that mediate these interactions and their respective secondary structures were mapped precisely to the p <dig> sequence . although slightly different residues of the p <dig> sequence are used in each interaction, there is a very high degree of overlap, with a span of  <dig> core residues being the same . interestingly, the four complexes display all three major secondary structure types. the core span becomes a helix when binding to s100ββ, a sheet when binding to sirtuin, and a coil with two distinct backbone trajectories when binding to cbp and cyclin a <dig> .\n",
            "\n",
            "because the secondary structures are distinct, it seems likely that p <dig> utilizes different residues for the interactions with these four different partners. to examine this, the buried surface area for each residue in each interaction was quantified by calculating the Δasa . different amino acid interaction profiles are seen for each of the interactions, showing that the same residues are used to different extents in the four interfaces. the particularly large Δasa peaks for k <dig> in complexes with cbp and sirtiun  are due to extra buried areas arising from the acetylation of this residue. this highlights the importance of posttranslational modification for altering ppi networks.\n",
            "\n",
            "analysis of multiple specificities of the p <dig> dbd\n",
            "the p <dig> molecule contains another set of overlapping interactions that contrasts with those at the c-terminus. these interactions are mediated by the dna binding domain and include interactions with dna  <cit> , the brct domain of 53bp <dig>  <cit> , the sh <dig> domain of 53bp <dig>  <cit> , and the large t-antigen  of simian virus  <dig>  <cit> . here we compare these four interactions using the methods described in figure  <dig> \n",
            "\n",
            "the structures of the p53-dna, the p53-53bp <dig>  the p53-53bp <dig>  and the p53-ltag complexes are shown . while all of the ligands are different, they all bind to basically the same region of p <dig> \n",
            "\n",
            "comparison of the interface profiles of the four complexes  shows a large difference in the pattern of interface residues used by p <dig>  for instance, there are several residues at the n-terminal end of the dbd which are only found in interaction with dna. similarly, interface residues near the c-terminal end participate in binding to different extents in three interactions, but not at all in the p53-53bp <dig> interaction. the differing usage of residues in each interaction is the most prevalent feature of this data. however, there are also several residues contributing an exceptionally large amount of surface area in each complex .\n",
            "\n",
            "while the focus of this paper is on the roles of disorder in the interactions involving two different hub proteins, the dna binding domain of p <dig> presents the opportunity to study structural changes involving one structured region binding to several different structured partners. for this purpose, we compiled a 4-panel set of plots for characterizing the induced fit as one protein binds to different partners . these panels show the average interface area , the standard deviation of the interface area , the differences in side chain conformation , and differences in backbone conformation . furthermore, regions that are highly exposed to solvent are also indicated , so that structural differences due to interactions can be distinguished from those due from intrinsic flexibility – disordered loops – or crystallization artifacts.\n",
            "\n",
            "this induced-fit profiles exhibit a number of interesting features . the most striking of these is the region from residue  <dig> to residue  <dig>  this region shows a large and variable interaction interface, which is associated with large side chain and backbone conformational differences. this is true also of a smaller region around residue  <dig>  other interaction regions show only side chain conformational differences associated with variable interface areas. other conformational differences observed are limited regions of high solvent exposure, which suggests that these changes are due the details of the crystallization conditions more than interaction with a particular binding partner.\n",
            "\n",
            "together, these results suggest that multiple partners of p <dig> are accommodated by reusing similar binding interfaces. this is facilitated by small scale or large scale structural differences, which range from differences in side chain conformation to backbone rearrangements. it should be noted that this differs from our finding in a more limited analysis on only the p53-53bp <dig> and -53bp complexes  <cit> .\n",
            "\n",
            "analysis of the multiple specificities of 14-3-3\n",
            "five different 3-d structures of the 14-3-3ζ protein bound to distinct partners were found in pdb. these partners include a peptide from the tail of histone h <dig>  <cit> , serotonin n-acetyltransferase   <cit> , a phage display-derived peptide   <cit> , and motif  <dig> and  <dig> peptides   <cit> . for aanat, only the region within the canonical 14-3- <dig> binding site is included in our analysis with the globular region being deleted. two additional structures were not included because they were either unsuitable for structural analysis or were highly redundant with another structure. all peptides are phosphorylated in their respective structures except r <dig>  which contains a glutamate in place of the phosphoserine.\n",
            "\n",
            "the five bound peptides sequences were aligned structurally as described in the methods. likewise, the 14-3- <dig> domain structures were independently aligned, without considering the bound peptides. next the 14-3- <dig> alignment was anchored manually by the observed correspondence of the bound peptide cα atoms at the  <dig> and - <dig> positions and by extending the alignment without gaps from the anchor positions, thereby giving the final structural alignment . in terms of sequence, the r <dig> sequence has no identical positions to any other peptide. the number of identities between the other peptides range from  <dig> to  <dig> \n",
            "\n",
            "the high overlap in the backbone trajectories of the  <dig> peptides from position - <dig> to  <dig> but large divergences at either end of the structural alignment . this divergence at the ends is apparent qualitatively in the superimposed structures of the five peptides . structural divergence and sequence variability are loosely correlated, where positions with  <dig> identical residues have a lower divergence than those with no identical residues. this suggests that 14-3- <dig> may use different binding pocket residues to interact with different peptide residues. the r <dig> sequence, which is divergent from the others, makes a large contribution to the estimated rmsf values .\n",
            "\n",
            "the factors contributing to the ability of 14-3- <dig> to bind to distinct peptides were estimated by a detailed structural analysis. the peptide binding residues of 14-3- <dig> are located primarily in a central cleft, made up of four helices , which has been noted previously by several researchers. the standard deviation of Δasa for the peptide binding residues  show that the residues with the most binding variability are located at either end of the central cleft, which is consistent with the variation of peptide backbone trajectories in these regions. backbone variability in bound 14-3- <dig> structures  is restricted to the ends of most of the binding cleft helices. these observations suggest that large a conformational change in 14-3- <dig> is not necessary for multiple specificities, although some small adjustments at the ends of binding helices may be necessary.\n",
            "\n",
            "to assess the role of side chain conformational changes in peptide binding, the rmsf of side chain atoms was calculated . the side chain rmsf and standard deviation of Δasas give similar indications for many binding site residues, where residues used inconsistently across multiple complexes are the most likely to undergo conformational rearrangement. these are the same residues that are located at the broadest parts of the binding site. however, a few residues deep in the binding grove show both consistent participation in the binding interface and variable side chain conformation. these observations suggests that the primary, high level mechanisms of 14-3- <dig> multiple specificity are a broad binding site that allows multiple trajectories  and side chain rearrangement to accommodate different peptide sequences.\n",
            "\n",
            "to further analyze the conformational changes in 14-3- <dig> upon binding to its multiple partners, we show the 4-panel induced-fit profile described above . contrary to the results seen for the p <dig> dbd, 14-3- <dig> is much more static in its multiple interactions. all regions displaying large conformational differences across bound complexes are also highly exposed to solvent and play no direct role in mediating binding to any peptide. the plots do show several small scale structural differences – side chain rearrangements – associated with variable participation in peptide binding, particularly in the regions 40- <dig> and 215- <dig> \n",
            "\n",
            "14-3- <dig> binding to two different partners\n",
            "to gain further insight into 14-3- <dig> binding to different partners, we compared a pair of 14-3- <dig> binding peptides in detail. these two peptides, m <dig> and m <dig>  were derived from two motifs, identified through the screening of peptide libraries for sequences that bound to all 14-3- <dig> isoforms  <cit> . these two peptide structures have been compared previously  <cit> , but here we reanalyze these structural data from the order-disorder point of view.\n",
            "\n",
            "as noted previously  <cit> , the backbone traces of the two peptides are noticeably different even though the m <dig> and m <dig> peptides bind to essentially the same region of 14-3-3ζ . examining the side chain interactions of these peptides with specific 14-3- <dig> residues  shows that there is difference in the location and identity of the residues involved, which is consistent with the aggregate findings . similarly, distinctive hydrogen bonding patterns are exhibited between the two peptides and 14-3-3ζ and between the two peptides and bound water . since a cardinal feature of a structured protein is internal satisfaction of hydrogen bond donors and acceptors, these data are both consistent with the peptides being from unstructured regions of protein before binding.\n",
            "\n",
            "the above data on the complexes suggest that 14-3-3ζ has distinct conformations when bound to the two different peptides. overlaying the backbone structures of the four binding helices from both complexes – based on a pair-wise alignment of the complete domains – shows only minor variability in conformation, with the most occurring at the helix spanning residues  <dig> to  <dig> . finally, comparison of side chain conformation in the two complexes shows significant differences in several of 14-3-3ζ side chains  and several other minor differences. overall, these data suggest that a difference in the conformations of some side chains with rather less difference in backbone conformations is sufficient to accommodate the binding of two different phosphopeptides by the 14-3-3ζ molecule.\n",
            "\n",
            "discussion\n",
            "use of disordered regions for binding\n",
            "the large majority of the binding sites on the p <dig> sequence map to the disordered regions of this protein , indicating that intrinsic disorder commonly provides the binding sites for the various partners that associate with p <dig>  recent bioinformatics investigations suggest that the majority use of disorder for binding to multiple partners is quite likely to be a general result  <cit> .\n",
            "\n",
            "the p <dig> binding sites are often indicated on the order-disorder predictions as dips, in other words as short segments with structure tendency flanked by regions of disorder tendency on both sides. starting from this observation, we previously developed a predictor of such regions, which we called molecular recognition features, or morfs, because such regions “morph” from disorder to order upon binding  <cit> . others have used the pondr vl-xt order/disorder plots or morf predictors to identify potential binding sites that were subsequently verified by laboratory experiments  <cit> . indeed, for some of these predicted examples, the regions did indeed form helix upon binding to their partners  <cit> . by greatly enlarging the training set, we recently improved the morf predictor. interestingly, when tested against several order-disorder predictors including ones from other laboratories, pondr vl-xt, gave the clearest indication of binding sites within disordered regions  <cit> .\n",
            "\n",
            "others developed a sequence-based approach to identify short, conserved recognition sites, called eukaryotic linear motifs   <cit> . while morfs are identified by general order/disorder tendencies and while elms are identified by motif discovery from sequence analysis, the resulting binding sites identified by both methods share several features  <cit> . the use of different residues in the same disordered fragment for one-to-many signaling leads to a potential problem with the elm model. that is, the concept behind elms is that each elm uses a common set of amino acids for binding to different partners. these common amino acids therefore show up as an over-represented pattern leading to a “linear motif”. what if a region used to bind to multiple partners uses different secondary structures and different amino acids? in such a case, the residues in the “linear motif” would not necessarily be over-represented. it will be interesting over time to determine whether elms having stronger signals use a reduced set of structures for their interactions.\n",
            "\n",
            "while the observed binding sites in the disordered regions of p <dig> have a localized tendency for ordered structure, not all disorder-associated binding sites exhibit such features. we have found many binding sites that are associated with high disorder prediction values across the entire spans of the binding sites, one example of which was recently published  <cit> . many of these dipless morfs form irregular structures upon binding with their partners, and often such binding regions are rich in proline. our recent study of the complexes that form when various disordered segments bind to ordered partners indicates that the disorder-associated binding regions have distinct sequence features, even when the bound structure is irregular or sheet instead of helix, and so it should be feasible to develop a specific predictor for each of the different types of morfs  <cit> .\n",
            "\n",
            "one-to-many signaling\n",
            "date hubs bind to different proteins at different times. figure  <dig> shows how a single region of p <dig> binds to four different partners. the amino acids involved in each interaction show a significant overlap and no two of these interactions could exist simultaneously. furthermore, the same residues adopt helix, sheet, and two different irregular structures when associated with the different partners. finally, the same amino acids are buried to very different extends in each of the molecular associations. these results show very clearly how one segment of disordered protein can bind to multiple partners via the ability to adopt distinct conformations.\n",
            "\n",
            "the idea that one segment of protein can adopt different secondary structures depending on the context is not new. many unrelated proteins have identical subsequences of length six, and sometimes even up to length eight, with the same sequences often adopting different secondary structures in different contexts  <cit> . such sequences have been called chameleons for their ability to adopt different structures in different environments  <cit> . chameleon behavior could be an important feature that enables one disordered region to bind to multiple partners. with different secondary structures and with different side chain participation in the different complexes, it is as if one sequence can be “read” in multiple ways by the various binding partners.\n",
            "\n",
            "chameleon behavior occurs for short peptides , for longer protein fragments and even for entire proteins. for example, the  <dig> residues-long arginine-rich rna binding domain  of the jembrana disease virus  tat protein recognizes two different transactivating response element  rna sites, from human and bovine immunodeficiency viruses . the jdv segment adopts different conformations in the two rna contexts and uses different amino acids for recognition  <cit> . in addition to the above conformational differences, the jdv domain requires the cyclin t <dig> protein for high-affinity binding to hiv tar, but not to biv tar  <cit> . another protein with chameleon properties is human α-synuclein, which is implicated in parkinson's disease and in a number of other neurodegenerative disorders known as synucleinopathies. this protein may remain substantially unfolded, or it may adopt an amyloidogenic partially folded structure, or it may fold into α-helical or β-strand species, including both monomeric and oligomeric species. in addition, this protein can form several morphologically different types of aggregates, including oligomers , amorphous aggregates, and amyloid-like fibrils  <cit> .\n",
            "\n",
            "such chameleon sequences likely underlie the multiple specificity binding sites common in p <dig>  for a quick calculation of the implied degree of interface overlap, assume that each residue in a region has equal probability to interact with a partner and consider the c-terminus of p <dig>  the disordered c-terminus  associates with at least  <dig> distinct partners. the average length of a binding site in this region is ~ <dig> residues, which means that on average only 100/14= <dig> partners bind at any given residue in the c-terminus. this simple back-of-the-envelope calculation suggests that multiple specificity sequences may be the rule for p <dig> interactions, rather than a curiosity of a single region. however, available data suggests that interactions do not overlap in a random fashion, but rather interactions are localized to specific regions. for example, consider that the majority of the structures available for the c-terminus of p <dig> involved the same region of sequence. therefore, the back-of-the-envelope calculation provides an approximate minimum degree of overlap, where the actual degree of overlap is likely much higher. this idea, which is an extension of a previous proposal  <cit> , further suggests a general mechanism by which hub proteins could bind to such a large multitude of partners, which cannot be explained from the view point of interaction between two structured proteins  <cit> .\n",
            "\n",
            "finally, the p <dig> dbd offers a counter example to the disorder-based view of date hubs. that is, it uses the same or similar face of its globular structure to bind to multiple partners. while the p <dig> dbd is a folded protein, it does exhibit some remarkable structural differences when bound to difference partners. it seems unlikely that these local regions of the p <dig> dbd structure are well folded in isolation, otherwise the association rate of some or all of these complexes would be relatively low. this idea is supported by the finding that the p <dig> dbd is only marginally stable at physiological temperature  <cit> . therefore, it is plausible that these regions of the monomeric dbd are only transiently folded in solution, where crystallization conditions cause a shift toward the folded state in monomeric crystal structures. the double nia-nma plot data  does not contradict this idea, since it is limited to global analysis and this idea only applies to local regions of the dbd. this idea is conjecture and further experimental or simulation evidence is needed to test this idea. in any event, however, the p <dig> dbd demonstrates that even proteins generally thought to be well folded, structural changes can still occur in association with multiple specificity.\n",
            "\n",
            "many-to-one signaling\n",
            "in 14-3- <dig>  a common binding groove in a structured dimeric protein can be fitted by multiple, distinct sequences provided by many different binding partners. a recent bioinformatics study  <cit>  found that14-3- <dig> proteins, and the 14-3- <dig> binding regions in particular, are predicted to be highly disordered by multiple disorder prediction methods. the authors proposed that 14-3- <dig> recognition generally involved coupled binding and folding of the recognition region. our results support this conclusion because the backbone of m <dig> and m <dig> peptides are highly hydrated in the bound state , indicating that the binding peptide is likely to be unstructured prior to binding  <cit> .\n",
            "\n",
            "one idea is that 14-3- <dig> holds its bound partner in a non-active state  <cit> . even though 14-3- <dig> likely binds to disordered regions in its partners , this idea of blocking the active structure could still be true. for example, the productive state of 14-3-3's partner might involve the binding of the partner to a second partner via the same disordered region that binds to 14-3- <dig>  in which case 14-3- <dig> binding would prevent the formation of the productive complex. another possibility is that the disordered region exhibits an equilibrium between a bound state that activates the protein and an unbound state that inactivates the protein. the association of the unbound disordered region with 14-3- <dig> would then hold 14-3-3's partner in the non-productive state as proposed previously.\n",
            "\n",
            "we previously suggested that disordered segments with different sequences could use their flexibility to bind to a common binding site, thereby facilitating many-to-one signaling  <cit> . the multiple recognition of 14-3- <dig> depends on this mechanism to a considerable degree, with the different peptides taking different paths through the binding cleft and interacting with binding site residues in distinct ways .\n",
            "\n",
            "in addition, structured proteins also have a degree of flexibility, and so the binding site backbone and side chain residues can undergo shifts  to help accommodate interactions with distinct sequences . thus, induced-fit mechanisms are important for structured protein interactions with different partners whether the partners are structured or intrinsically disordered.\n",
            "\n",
            "the induced-fit mechanisms observed for 14-3- <dig> and the dna binding domain of p <dig> are commonly observed in other situations. for example, tethering, in which a peptide is covalently linked to its protein target to allow detection of low affinity interactions, often results large-scale side chain movements concomitant with peptide binding  <cit> . also, when many different morfs and their binding partners are examined, induced-fit movements in the structured partners are very commonly observed  <cit> . similarly, small backbone shifts and side chain conformational changes are both important for 14-3-3's ability to bind multiple partners. for all of these examples, the associations involve coupled binding and folding for the disordered peptide partner coupled with a near universal classical induced fit for the structured side of the partnership.\n",
            "\n",
            "one-to-many signaling vs. many-to-one signaling\n",
            "the p <dig> c-terminus and 14-3- <dig> use intrinsic disorder differently with regard to enabling multiple binding specificities. in p <dig>  drastic conformational changes enable distinct surfaces to be exposed to binding partners. in 14-3- <dig>  subtle differences in 14-3- <dig> conformation and peptide binding locations enable multiple specificities. why would nature use one mechanism rather than the other for a particular biological role? the interactions of p <dig> serve to activate or inhibit its primary role as a transcription regulator, while 14-3- <dig> alters the functions or subcellular localization of many proteins. from this, one can make some highly speculative proposals:  disorder binding regions play a passive role in regulation by providing a specific binding site – i.e. the disordered regions are the identification sites of the protein to be regulated  <cit>  – and  ordered proteins play the active role – i.e. altering the activity of the proteins they bind to – where recognition of disordered regions allows for a generalized specificity so that a single protein can alter the activity of many others. validation of the accuracy and generality of these ideas requires further study.\n",
            "\n",
            "CONCLUSIONS\n",
            "here we have examined the mechanisms of multiple specificities in two date hub-like hub proteins. evidence here and elsewhere  <cit> , suggests that disordered regions may be an extremely common mechanism by which hub proteins bind to their multitude of partners. the specific examples of p <dig> and 14-3- <dig> contrasts the mechanisms by which disorder facilitates multiple recognition, where the former involves drastic conformational differences in a single disordered region and the later involves a variety of subtler changes in order to recognize multiple disordered regions. finally, it is proposed that the differences between the binding of the disordered region of p <dig> and the binding of disordered regions to 14-3- <dig> may have implications for the biological roles of both types of interactions.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "protein-protein interaction  networks are a common mechanism by which hub proteins bind to their multitude of partners.\n",
            "protein-protein interaction  networks have been proposed to provide several biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks have been proposed to provide biological advantages.\n",
            "protein-protein interaction  networks\n",
            "41\n",
            "[{'summary_text': 'the abo gene consists of three major alleles: two co-dominant a and b alleles, one silent and recessive o allele, and one silent and recessive o allele.\\nthe abo gene is organized in  <dig> exons that range in size from  <dig> to  <dig> nucleotides.\\nour results suggest the presence of the human o <dig> allele already in the common ancestor of modern human and neandertal.\\nour results suggest the presence of the human o <dig> allele already in the common ancestor of modern human and neandertal.\\nour results suggest the presence of the o <dig> allele already in the common ancestor of modern human and neandertal.\\nour results suggest the presence of the human o <dig> allele already in the common ancestor of modern human and neandertal.\\nour results suggest the presence of the o <dig> allele already in the common ancestor of modern human and neandertal.\\nour results suggest the presence of the human o <dig> allele already in the common ancestor of modern human and neandertal.\\nour results suggest the presence of the o <dig> allele already in the common'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the abo blood group system was the first genetic polymorphism discovered in humans. it consists of three alleles: two co-dominant a and b alleles, and one silent and recessive o allele <cit> . the system is controlled by a single gene at the abo locus. this gene encodes a glycosyltransferase enzyme that adds a sugar residue to a carbohydrate structure known as the h antigen, that is present in the membrane of red cells as well as most epithelial and endothelial cells. the a allele codes for an enzyme that adds a n-acetyl galactosamine to the h antigen, while the b allele, which differs from the former by four amino acid changes, codes for an enzyme that adds a d-galactose. the o allele occurs most frequently in modern humans and carries a human-specific inactivating mutation which produces a non-functional enzyme, and thus the h antigen remains without further modification on the surface of the cells  <cit> . the combination of the three abo alleles results in four major phenotypes, named a, b, ab and o, these carry different antigens  that react with specific antibodies  <cit> .\n",
            "\n",
            "the abo gene is organized in  <dig> exons that range in size from  <dig> to  <dig> nucleotides, with exon  <dig> and  <dig> being the largest  <cit> . further analyses of the abo gene have shown that, at the nucleotide sequence level, numerous haplotypes exist among modern humans <cit> . the main a and b alleles  differ by four amino acid changes at positions  <dig>   <dig>   <dig> and  <dig> , with the two later being determining the a and b specificity of the enzyme. the most frequent among all human abo alleles is the o allele  <cit> . the most frequent o haplotypes, o <dig> and o <dig>  share a single g nucleotide deletion in position  <dig> of exon  <dig>  that creates a premature stop codon and results in a truncated protein with no enzymatic activity  <cit> ; they are referred as the o  alleles below. the o <dig> haplotypes differ from the o <dig> haplotypes by at least six nucleotide substitutions, including position  <dig> in exon  <dig>  which is a in the former and g in the latter.\n",
            "\n",
            "many studies have examined the association between the abo alleles and a variety of diseases  <cit> , mainly focussed on infectious agents that can use the a and b antigens as receptors, or in case of the o allele, the absence of the a and b antigens. the o allele can provide a selective advantage since it also produces both anti-a and anti-b antibodies  <cit> . in particular, it has been suggested that the o allele protects against severe malaria  <cit> . at the same time, it can be more sensitive to helicobacter pylori infections  <cit>  and to severe forms of cholera  <cit> . the complex pattern of putative selective agents favouring or acting against different alleles could explain the maintenance of the high abo polymorphism  <cit> , as evidenced by the signal of balancing selection detected on the gene  <cit> . therefore studying the evolution of the abo blood groups to determine when during human history the different alleles emerged may contribute to our understanding on what selective forces might have worked on the different alleles.\n",
            "\n",
            "within the apes a convergent evolution of the different blood groups can be observed, although no ape displays all three phenotypes known in humans  <cit> . whereas all gorillas studied to date carry only an allele that has properties similar to the human b allele, our closest living relative, the chimpanzee carries two alleles corresponding to the human a and o. both alleles in the chimpanzee are characterized by species specific mutations in the abo gene that are distinct from those found in humans  <cit> . for the abo alleles on the human lineage it has been estimated that the b <dig> allele, which gave rise to all the b variants, derived from the ancestral a <dig> allele around  <dig>  million years ago   <cit> , whereas the youngest and most common human allele o <dig>  carrying the Δ <dig> deletion, derived from the a allele only about  <dig>  ±  <dig>  mya determined by coalescence analysis  <cit> . the emergence of the abo allele o <dig> therefore predates the mean divergence time for modern humans and neandertals estimated to more than  <dig> kya  <cit>  and opens the possibility to put a lower limit on the estimated coalescence date by directly testing for the presence of the o <dig> allele in the neandertal population. thus we undertook a targeted approach to determine whether the Δ <dig> deletion in the o allele was also present in neandertals and by that testing the emergence of the o <dig> allele in the common ancestor of modern human and neandertal.\n",
            "\n",
            "methods\n",
            "dna extraction and amplification\n",
            "we analysed two neandertal samples from the el sidrón site   <cit>  that were extracted in  <dig> under controlled conditions  <cit>  and immediately frozen, prior to sending them to the dedicated ancient dna laboratories in both barcelona and leipzig, where dna extractions were performed. about  <dig> mg of cortical bone was removed, powdered and extracted as described  <cit> .\n",
            "\n",
            "analysis of specific nuclear dna loci from ancient neandertal remains by the polymerase chain reaction  is a challenging task that presents two main limitations: the small quantity and short length of ancient dna fragments due to the degradation of the original template molecules and the difficulty in distinguishing between endogenous neandertal dna and putative contemporary modern human contaminants. however, the presence of single informative substitutions in the abo gene allows for characterization of the different alleles even when retrieving short dna sequences. also, the co-amplification of mitochondrial dna  and nuclear markers allow us to control for the presence of human contamination in the extracts.\n",
            "\n",
            "two-step multiplex pcrs  <cit>  in a total volume of  <dig> μl were set up and incubated for  <dig> minutes at room temperature with  <dig>  units of shrimp nuclease  to degrade any double-stranded dna that may contaminate the reagents  <cit> . after inactivating the nuclease for  <dig> minutes at 65°c,  <dig> μl of extract and taqgold dna polymerase were added to the pcr. twenty-seven cycles of pcr were performed in the first pcr and  <dig> cycles in the second pcr. reaction conditions were as described  <cit>  except for the annealing temperature, which was 53–55°c, depending on the primers used.\n",
            "\n",
            "the informative Δ <dig> position in exon  <dig> was amplified with the f20144-aggaaggatgtcctcgtgg, r20163-tgccctcccagacaatgg primer couple . the  <dig> position in exon  <dig>  further retrieved to determine to which of the two main o haplotypes  the specimens belonged, was amplified with the f20173-ttggctggctcccattgtctgg, r20194-gaactgctcgttgaggatg primer couple. the pcr products were  <dig> and  <dig> bp in length, respectively. in barcelona, amplification products of the correct size were cloned using the topo ta cloning kit , and several clones from each product were sequenced using an abi <dig> capillary sequencer  following the manufacturer's instructions. in leipzig, the products from each pcr were pooled and ligated with a specific tagging sequence for each pcr as described  <cit>  and subsequently sequenced on a  <dig> flx machine.\n",
            "\n",
            "contamination controls\n",
            "to determine the degree of modern human contamination in the neandertal dna extracts we amplified and sequenced, using conserved primers, one position in the mitochondrial dna  coding region  from each neandertal extract where neandertals were found to be distinct from modern humans  <cit> , as well as two diagnostic mtdna hypervariable region i  fragments between positions  <dig> – <dig>  and  <dig> – <dig> . to ensure that the copy number of nuclear dna is sufficient for amplifying nuclear neandertal dna and to monitor potential nuclear dna contamination we additionally amplified a control position on the y-chromosome, which was previously described as ancestral in both specimens and defines the deepest clade in the human y-chromosome tree . this clade is comprised of all modern human y chromosomes haplogroups outside africa. successful amplifications of the y chromosomal control fragment confirmed that the two specimens labeled  <dig> and 1351c come from two different male individuals  <cit> .\n",
            "\n",
            "RESULTS\n",
            "in barcelona, position  <dig> on exon  <dig> in the  <dig> specimen was amplified in four independent pcrs in parallel with the mtdna controls  and the nuclear y-chromosome position. the products were cloned and at least  <dig> clones sequenced. the sequences from all four products for the  <dig> specimen  showed the Δ <dig> deletion, which defines the o alleles. all y <dig> sequences  were found to be ancestral , while  <dig> out of  <dig> mtdna sequences were neandertal-specific. we further amplified and sequenced position  <dig> on exon  <dig>  twelve sequences display an a at position  <dig>  which is compatible with the most frequent o haplotype, the o <dig>  but not with the other, the o <dig>  <cit> . the position  <dig> on exon  <dig> was also amplified in four independent pcrs from the specimen 1351c along with the y <dig> fragment and the informative mtdna position  <dig> . all the abo sequences  were identical to those found in the  <dig> specimen, while all y <dig> sequences  were ancestral, and  <dig> out of  <dig> mtdna sequences were neandertal specific .\n",
            "\n",
            ": deletion in position  <dig>  : g in position  <dig>  : a in position  <dig>  : g in position  <dig>  : y chromosome basal marker. : ancestral. : mtdna nucleotide position  <dig> . : neandertal. : modern human. : mtdna hypervariable region  <dig>  : barcelona; : leipzig. : no amplification product obtained. : no amplification attempted.\n",
            "\n",
            "in leipzig, position  <dig> and position  <dig> on exon  <dig> in specimen  <dig> were amplified in two independent 2-step multiplex pcrs together with one of the informative mtdna fragments . all sequences for the  <dig> fragment in the first pcr show Δ <dig> , for  <dig> of the sequences we find a additional c > t substitution at position  <dig>  a substitution at that position was not found in any other pcr product from leipzig or barcelona and is likely to derive from a miscoding lesion caused by deamination of cytosine on the ancient molecule  <cit> . for the second pcr from specimen  <dig>   <dig> sequences were found to carry the Δ <dig> and  <dig> displayed a g at that position . at position  <dig> an a was found in all clones from the first pcr, while in the second pcr most of the sequences  display an a and others  a g . for specimen 1351c, just one pcr resulted in a product covering position  <dig> where some sequences displayed the Δ <dig>  and some showed no deletion . all the sequences  in the two independent pcrs for specimen 1351c showed an a at position  <dig> \n",
            "\n",
            "between 82% and 85% neandertal sequences were found for the informative mtdna fragments that were co-amplified along the abo positions. despite the informative transversion at position  <dig>  where humans have a a and neandertals a c, there is a second diagnostic position,  <dig> , present in that fragment where most humans have a g and neandertals an a  <cit> . all clones for the mtdna fragments show either the  <dig> a- <dig> g or the  <dig> c- <dig> a haplotype, allowing to determine whether they derived from human or neandertal, respectively. there is also a number of transitions present in a minority of the mtdna sequences that are almost all either c > t or g > a substitutions typical for miscoding lesions  <cit> . furthermore we find a number of insertions and deletions close to homopolymers, a known problem of the  <dig> technology  <cit> . only little post-mortem damage was found in the nuclear sequences retrieved in leipzig and barcelona , which can be attributed to the small number of individual clone sequences obtained, the short length of the amplified product and a low molecule starting number for the pcr. the later explanation is supported by some unsuccessful amplifications in both barcelona and leipzig, and also by the almost equal ratio of clones that show the c > t transition at position  <dig> for one of the  <dig> fragments, suggesting the pcr started only from two initial dna molecules .\n",
            "\n",
            "the low level of modern human mtdna contamination and the fact that from both specimens ancestral y chromosomal positions could be repeatedly retrieved suggest that the majority of the abo fragments amplified in this study derive from neandertal nuclear dna. since nine out of eleven successful amplifications of position  <dig> from both neandertals only displayed the informative deletion for allele o, we conclude that this allele was present in the neandertal population. since four out of five amplifications of position  <dig> only displayed an a at that position, it is likely that the haplotype was o <dig>  if we assume that our amplifications just start from single molecules and that our neandertals are heterozygous at position  <dig>  it seems quite unlikely to observe only one of six and one of five pcrs for the second individual without the Δ <dig> . given an average contamination rate of  <dig> % with a probability of carrying either the a or the b allele of  <dig>   the chances of this observation would increase .\n",
            "\n",
            "CONCLUSIONS\n",
            "our results indicate that the two el sidrón neandertal individuals were most likely homozygous for the o <dig> allele. nevertheless given a low rate of potential modern human contaminants in an unknown allelic state, we cannot discard the possibility that both neandertals could have been heterozygous . the results however suggest the presence of the human o <dig> allele already in the common ancestor of neandertals and modern humans and thereby confirming an emergence of the o <dig> allele more than  <dig> mya predating the divergence of the modern human and neandertal populations. an alternative possibility would be that neandertals have acquired this allele by gene flow from modern humans, although at present, other genetic evidences suggest that this scenario is unlikely  <cit> . furthermore a potentially homozygous state in at least two neandertals, even though from the same group, at least indicates that a potential selective advantage of the o allele could have also been present in the neandertal population. future analyses regarding the frequency of the abo alleles will be necessary to reveal if all three genotypes were conserved in the neandertal population.\n",
            "\n",
            "authors' contributions\n",
            "mr, jf and ar excavated the samples and provided paleontological information; cl-f, eg and jk extracted, amplified, sequenced and analyzed ancient dna data; jb assisted in the interpretation of the results; cl-f and jk wrote the paper.\n",
            "\n",
            "supplementary material\n",
            "additional file 1\n",
            "clone sequences. clone sequences generated for the abo blood group gene and mtdna fragments in neandertals.\n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements\n",
            "the el sidrón excavation is founded by the principado de asturias. cl-f is supported by a grant from the ministry of education and sciences of spain . we are grateful to f. calafell  and to a. briggs  for their helpful comments as well as to christine green to have a critical eye on spelling and language. the neandertal abo blood group gene sequences have been deposited at genbank under accession numbers fm <dig> and fm <dig> \n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the abo gene consists of three major alleles: two co-dominant a and b alleles, one silent and recessive o allele, and one silent and recessive o allele.\n",
            "the abo gene is organized in  <dig> exons that range in size from  <dig> to  <dig> nucleotides.\n",
            "our results suggest the presence of the human o <dig> allele already in the common ancestor of modern human and neandertal.\n",
            "our results suggest the presence of the human o <dig> allele already in the common ancestor of modern human and neandertal.\n",
            "our results suggest the presence of the o <dig> allele already in the common ancestor of modern human and neandertal.\n",
            "our results suggest the presence of the human o <dig> allele already in the common ancestor of modern human and neandertal.\n",
            "our results suggest the presence of the o <dig> allele already in the common ancestor of modern human and neandertal.\n",
            "our results suggest the presence of the human o <dig> allele already in the common ancestor of modern human and neandertal.\n",
            "our results suggest the presence of the o <dig> allele already in the common\n",
            "42\n",
            "[{'summary_text': 'although there have been just a few  <dig> transcriptome sequencing efforts to date  <cit> , our results suggest that for studies of this type,  <dig> sequencing alone could largely replace traditional sanger sequencing.\\nwe analyzed est data from two species of charophyte green algal species, spirogyra pratensis and coleochaete orbicularis, as a first step toward complete genome analyses.\\nour results suggest that for studies of this type,  <dig> sequencing alone could largely replace traditional sanger sequencing.\\nalthough there have been just a few  <dig> transcriptome sequencing efforts to date  <cit> , our results suggest that for studies of this type,  <dig> sequencing alone could largely replace traditional sanger sequencing.\\nour results emphasize the importance of using plant gene models to annotate the putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the ancestry of all living land plants  can be traced back to a single colonization event from a charophyte green alga. in other words, the tremendous diversity we see in land plants today--from mosses to redwoods and orchids--all descended from a single common ancestor that colonized land 430- <dig> million years ago  <cit> . uncertainty remains concerning the precise relationships between embryophytes and their algal relatives  <cit> , but there is no serious doubt that the origin of land plants occurred from within the charophytes. there are six orders of charophyte green algae that, when embryophytes are included, comprise the streptophyta sensu lato   <cit> : the mesostigmatales, chlorokybales, klebsormidiales, zygnematales, coleochaetales and charales. both phylogenetic and fossil evidence suggest that these orders are extremely old lineages, comparable in age to the land plants  <cit> . therefore, an understanding of the biology of land plants based on comparative genomics would benefit greatly if data were available from these organisms. unfortunately, in most cases the genome size is poorly characterized, the tools of molecular genetics are not well developed, or cultures are difficult to maintain. consequently, the acquisition of genomic data from these organisms has lagged other lineages. to move toward comprehensive genomic analysis of charophytes, we undertook est analysis of two representative charophytes, spirogyra pratensis and coleochaete orbicularis.\n",
            "\n",
            "despite there being significant genomic resources available for the broader group of green algae, including chlorophyta, there is only one published est library to date that directly bears on the charophytes, that of mesostigma viride  <cit> . mesostigma is a unicellular, monotypic genus that in some analyses is placed as sister to the rest of streptophytes  <cit> , although other studies have placed it as a sister to all other green algae  <cit> . in either case, its est library is a valuable resource for this study. most taxonomic and ecological diversity in the green algae resides in the chlorophyta, a large clade sister to the streptophytes. among the important organisms in this sister clade are the model organism chlamydomonas reinhardtii and the ecologically significant ostreococcus tauri. both of these organisms have fully sequenced and published genomes  <cit> .\n",
            "\n",
            "according to darwin's centralizing theme of descent with modification, one would predict that all land plant genes should have a homolog in the charophytes unless there was horizontal gene transfer from a non-plant organism, or unprecedented neofunctionalization. however, any one lineage of charophytes might be expected to have lost or modified some of these in the  <dig> million years or more of independent evolutionary history that separates each lineage from embryophytes. in this context, it is to be expected that land plant genes and their associated molecular pathways either originated in the charophytes or, if more ancient, were retained along these green algal lineages leading up to the colonization of land. thus, it is important to sample broadly among the charophytes if the homologs of key embryophyte genes are to be identified.\n",
            "\n",
            "in recent years pcr-based approaches have been used to fish out specific land plant genes of interest in the charophyte lineages, but advances in sequencing technologies have now made it far more efficient to gather high-throughput genomic data and work backwards, using plant gene models to annotate the putative homologous genes. sequencing expressed sequence tags  is an efficient first pass at gathering a large portion of the genomic coding regions. we undertook here an analysis of two distantly related charophyte taxa: spirogyra pratensis transeau  and coleochaete orbicularis pringh. . both of these lineages are essential to understanding the placement of land plants in the context of their nearest living green algal relatives. in addition, evidence of land plant molecular pathways, such as the ethylene response pathway, in the charophytes would reveal the origins of key plant molecular processes.\n",
            "\n",
            "RESULTS\n",
            "est statistics\n",
            "our sequencing effort for spirogyra pratensis yielded  <dig>  5' sanger reads plus  <dig>  gs flx standard  <dig> sequences; coleochaete orbicularis yielded  <dig>  5' sanger reads plus  <dig>  gs flx titanium  <dig> sequences . the average length of sanger sequences was  <dig> bp  or  <dig>  bp  before trimming for low quality and vector sequence. the average length for the raw  <dig> reads differed between the older gs flx standard and newer gs flx titanium sequencing technologies of  <dig> and  <dig> bp, respectively. the  <dig> sequences for each species were trimmed of vector and low-quality sequences, and then clustered by agencourt. the  <dig> assembly along with the raw sanger reads were then clustered together using our in-house pipeline . this two-stage clustering method reduced the demand on computational power for the final assembly . it also allowed for careful tracking of each sequencing effort contributing to the final assembly. the average length of unigenes  in the combined  <dig> and sanger sequence assembly was  <dig> bp for c. orbicularis and  <dig> for s. pratensis.\n",
            "\n",
            "summarizing the composition of the clustered ests is complex due to our two-step assembly procedure  . results from the coleochaete orbicularis cap <dig> assembly produced  <dig> unigenes:  <dig>  contigs and  <dig>  singletons with the average length of contigs at  <dig>  bp and singletons at less than half of that at  <dig> bp. the spirogyra pratensis-combined assembly resulted in  <dig>  unigenes:  <dig>  contigs  and  <dig>  singletons . the singletons category actually comprised several types of data: 454-only contigs from the first assembly, single  <dig> reads and single sanger reads. most of the singletons were in fact 454-only contigs for both est libraries . the number of reads per unigene was low on average, with the vast majority of unigenes comprising only one to five reads . a few contigs had higher numbers of reads, the most numerous being one s. pratensis contig with  <dig> reads.\n",
            "\n",
            "because ests are, by definition, sequences derived from messenger rna, it follows that they should all contain at least a portion of protein-coding sequence, but several forms of contamination are possible. our sequencing effort included 5' sanger sequencing plus  <dig> sequencing, which is generated from sequencing shorter reads from sheared cloned inserts. due to this strategy, we expected to recover the 5'  utr and some portion of the coding sequence. however, in addition to mrna, there is a chance of contamination in the est library from non-coding nucleic acids, such as rrna, trna and genomic dna. these contaminants were removed from the final unigene set after estscan determined the putative coding regions in the sequences. by definition only the putative protein coding sequences were included in the final unigene set. out of  <dig>  total unigenes in spirogyra pratensis, 25%, or  <dig> , had no coding region detected. coleochaete orbicularis had a lower percentage of non-coding contaminants, at 16%, or  <dig>  sequences in which no protein-coding region was detected.\n",
            "\n",
            "taxonomic assignment\n",
            "although every attempt was made to produce axenic cultures, including streaking and isolation off agar plates, antibiotic treatments, and zoospore isolation, we settled on \"very clean\" cultures. because of this we anticipated a few contaminants based on low bacterial contamination observed when we grew the culture on agar plates. for this reason, the unigenes  were blasted against the ncbi non-redundant  nucleotide database using blastx for taxonomic lineage assignment. eight major taxonomic categories were recovered as top hits for each spirogyra pratensis and coleochaete orbicularis: archaea, viruses, bacteria, fungi, chlorophyta, streptophyta, various other eukaryotes, and unigenes with no significant hits . the proportion of hits for each species was very similar and therefore will be summarized together. about 10% of hits were possible contaminants  and roughly 2-3% were hits to the other lineage of green algae, chlorophyta . the remaining ~85% were split fairly evenly between the top hit being a streptophyte  or no hits returned at all . a significant percentage of ests received no hits at all: 41% and 48% for s. pratensis and c. orbicularis, respectively. in this class of unknown unigenes, a significant portion of the sequences did not contain a protein prediction using estscan  <cit> . about half of the spirogyra \"no hits\" and one-third in the coleochaete \"no hits\" category were not included in the predicted protein set of unigenes, which means they are most likely a mixture of genomic and non-coding rna contaminants. that leaves about 20% and 28% of putative novel genes in s. pratensis and c. orbicularis, respectively.\n",
            "\n",
            "because genbank does not have uniform taxon sampling across the tree of life, it is important not to place too much significance on the blast based lineage assignment for the unigenes. for example, a small fraction of hits to metazoan taxa  most likely represents either a conserved protein domain or an ancestral gene lost in land plants, and not a metazoan contaminant. likewise, blast hits to bacteria and fungi, unless evaluated individually, should be treated with caution. on the contrary, this means that up to 10% of the ests from each of these species could be derived from bacterial and fungal contaminants and not from the charophyte for which it's assigned. the analyses in this manuscript were largely focused on the unigenes that had similarity to land plant genomes, so it is unlikely that contaminant unigenes affected any of our major conclusions.\n",
            "\n",
            "orthologous genes\n",
            "the results of the lineage assignment introduced the question of how many genes in each est library are actually shared with chlorophytes vs. streptophytes. to address this question, the protein predicted unigenes for coleochaete orbicularis and spirogyra pratensis, plus complete proteins from chlamydomonas reinhardtii  and arabidopsis thaliana  were included in an all-vs-all pair-wise blastp. the reciprocal best hits  between any two, three or four species were interpreted to be orthologs if all pair-wise possibilities in the ortholog set were each other's reciprocal best hit. all rbh overlaps are displayed in a venn diagram . the total number of unigenes  for each species can be obtained by adding up all the numbers in the respective ovals. for example, a set of  <dig> unigenes are rbhs for all four species. a different set of  <dig> unigenes are rbhs for the charophytes + chlamydomonas, but not for arabidopsis.\n",
            "\n",
            "although there are many overlaps to report, the most interesting results from this analysis concerns the overlap between our species of interest and its overlap with chlorophytes vs. streptophytes. looking at pair-wise comparisons only, c. orbicularis shares  <dig>  orthologs with arabidopsis vs.  <dig> with chlamydomonas. spirogyra pratensis has a similar pattern, with  <dig>  shared arabidopsis orthologs and  <dig> shared chlamydomonas orthologs. this pattern holds up with three-way rbhs: c. orbicularis + s. pratensis share  <dig> orthologs with arabidopsis verses  <dig> with chlamydomonas. \n",
            "\n",
            "a list of genes hypothesized to be important in early land plant colonization, or \"green genes,\" was generated from graham et al.  <cit> . each of these genes had at least one hit between the two libraries, except for the mads domain, which had no hits in either library . the following genes had at least one hit in both coleochaete orbicularis and spirogyra pratensis: rsw <dig> , gem1/mor <dig> , ctr <dig> , and meristem layer <dig> . wuschel and gnom  were only found in c. orbicularis and the expansins were only found in s. pratensis. a moss  and a chlorophyte  were included as references. although all of these are significant blast \"hits\"  they can only be interpreted as genes of interest. any orthology determination would need a more vigerous follow-up analysis.\n",
            "\n",
            "cdss of noted genes were blasted against protein databases of two reference genomes, physcomitrella patens and chlamydomonas reinhartii, and our two charophytes, coleochaete orbicularis and spirogyra pratensis. only blastx hits below e- <dig> are reported.\n",
            "\n",
            "gene ontology\n",
            "the gene ontology  categories recovered from blast2go analyses were summarized by the proportion of unigenes annotated in each go level  <dig> category . the complete set of proteins for arabidopsis thaliana was included in the analysis as a relative proportional measure since the go assignments were all derived from a. thaliana annotations. overall, spirogyra pratensis and coleochaete orbicularis unigene sets were remarkably similar in relative proportions, but they deviated from a. thaliana in several categories. three categories stood out in biological processes : both green algae were underrepresented in cellular metabolic process and regulation of biological quality , but they were overrepresented in the biosynthetic processes category . for molecular function , the green algae unigenes were overrepresented in transferase activity , hydrolase activity  and nucleotide binding , but they were underrepresented in the transcription factor activity  and in oxidoreductase activity categories . for cellular compartment  annotations, the green algae unigenes were slightly over-represented in many categories, but deviated most drastically in their underrepresentation in membrane and intracellular organelle part categories .\n",
            "\n",
            "discussion\n",
            "both charophyte est libraries are very land plant-like. more specifically, most of the transcriptome is more similar to embryophytes  than to other green algae in the chlorophyte lineage . there are some exceptions, however, which are interesting. looking just at blast hits against ncbi's non-redundant  protein database, 2%  and 3%  of the transcriptome had a top hit to a chlorophyte in spirogyra and coleochaete, respectively. while this is only a small fraction of total unigenes, it suggests possible shared ancestral genes between the chlorophyte and streptophyte lineages that were lost along the land plant lineage. for example, there are flagellar-associated proteins in both spirogyra  and coleochaete  that have top blast hits to chlamydomonas and ostreococcus, but that have no hits to arabidopsis or any other land plant. plants lost their flagellate cells in the gymnosperms, which left angiosperms with no motile cells. it follows that the flagellar-associated proteins would also be lost in flowering plants, infact, analysis of the much more ancestral moss genome revealed a loss of flagaller associated genes  <cit>  that are found in other green algal and animal genomes <cit> . the fact that there are no significant land plant hits suggests that these proteins were lost or underwent substantial sequence divergence in the early land plants. also interesting is the mere presence of flagellar-associated proteins in spirogyra, which lacks motile cells entirely. containing genes for an absent structure further supports the evolutionary placement of this lineage as being derived from flagellate ancestors. an earlier hypothesis had the zygnematales grouped with other non-flagellate organisms in a separate phylum defined by lack of motility  <cit> , which our data clearly reject.\n",
            "\n",
            "the overwhelming similarity to land plants also is revealed in a separate analysis in the identification of orthologous genes between our two green algae, chlamydomonas and arabidopsis, as shown in the venn diagram . the charophytes share two to three times as many orthologs with arabidopsis than they do with chlamydomonas. both the blast analysis and the ortholog identification reinforce the overall similarity between these lineages of green algae and land plants. of course, a robust phylogeny of any given gene is needed before firm conclusions can be made regarding its gain or loss along the plant lineage. the point here is that even a simple blast analysis or ortholog identification can reveal key genes to target for follow-up study.\n",
            "\n",
            "there are a large number of putative novel genes in both est libraries . the term 'novel' is being used here to refer to genes with no obvious similarity to a model-organism sequence: i.e. new to scientists, not necessarily to the charophytes. we are classifying the unigenes as novel if they satisfy two criteria: they contain a protein-coding region predicted by estscan  <cit>  and no blast hits returned against the ncbi non-redundant protein database. the closest fully sequenced genomes to the charophytes at the time of this publication are several embryophyte genomes, including physcomitrella patens and arabidopsis thaliana  and chlamydomonas or ostreococcus in the chlorophyte lineage   <cit> . because of the relative distance between charophytes and their nearest sequenced organisms, it is not surprising that they would contain genes with no obvious similarity to the current published genome-scale sequences. there are two alternative explanations for the enlarged amount of \"ests without blast hits\". first, the short sequences may represent non-conserved portions of genes, and, second, independent  <dig> sequence stretches may correspond to the same novel gene, but they do not overlap due to their limited sequence length. this last caveat would apply across the entire set of clustered set of ests for both species. aside from these concerns, most of these novel sequences do appear to be unique, at least within the available data. a pair-wise blast between the s. pratensis and c. orbicularis sets of novel unigenes only revealed a small amount of overlap: out of  <dig>  s. pratensis novel genes, only  <dig> had a hit in c. orbicularis. in addition, the reciprocal blast revealed only  <dig> hits in the reverse direction. none were reciprocal best hits.\n",
            "\n",
            "the gene ontology analysis presents a summary of transcript diversity relative to the annotated gene diversity in arabidopsis. where there are significant differences in the proportion of genes in a category, the determination cannot be made whether there were fewer genes expressed or that there are actually fewer genes encoded in the genome--and the same goes for the relative over-expression. the observation that both charophyte green algal ests were so proportionally similar in their gene expression categories suggests that we are not missing any major class of gene expression and that we have captured a fairly wide range of transcript diversity representing the source genome's diversity. spirogyra and coleochaete are separated in evolutionary time by 500- <dig> million years and they do not share a similar gross morphology. but the fact that they are so alike in transcript proportion, as compared to a flowering plant, is interesting in terms of their evolutionary placement in the streptophyte lineage. the green algae showed significant difference with respect to arabidopsis in most go categories, but only a few were markedly divergent.\n",
            "\n",
            "the under representation of charophyte genes in some of the biological process categories  is not surprising. although biological complexity is not easily defined, it seems clear that embryophytes have a larger number of cell types and are adapted to a wider range of environmental conditions than are charophyte algae, and consequently would be expected to have a higher proportion of genes expressed in categories involving organismal complexity. both charophytes were significantly underrepresented in their proportion of cellular metabolic process transcripts compared to arabidopsis. according to the gene ontology consortium's  description of this category --the chemical reactions and pathways by which individual cells form chemical substances  <cit> --it stands to reason that the diversity and abundance of transcripts in this class would be higher in an organism with more diverse cell types, such as the flowering plant taxon, arabidopsis. the same interpretation goes for other underrepresented categories including cell communication, response to endogenous stimulus and regulation of biological quality. the latter category had the most extreme underrepresentation of the biological processes categories and consequently may be one of the more interesting categories in the study of the origin of land plants. biological quality is defined by goc as \"a measurable attribute of an organism or part of an organism, such as size, mass, shape, color, etc.\" the charophyte transcripts annotated to this category were similar to genes involved in cell shape, such as diminuto  <cit> , and various expansins  <cit> . it is logical to expect that the proportion of genes associated with cell quality would increase with organism complexity and our results are consistent with this prediction.\n",
            "\n",
            "the explanation becomes less clear in cases where there is overrepresentation of charophyte genes in a given biological process category relative to arabidopsis. because we are comparing proportions of genes present and not the actual number found, it is possible that there might be a greater number of genes in arabidopsis compared to one of the charophytes. but the proportion relative to the other categories is lower in our analysis. that said, one of the more extreme overrepresentations  of charophyte genes occurs in the biosynthetic process category, which is defined by \"the chemical reactions and pathways resulting in the formation of substances; typically the energy-requiring part of metabolism in which simpler substances are transformed into more complex ones\"  <cit> . examples of charophyte genes that are annotated in this category are ribosomal proteins, ef-1-alpha-related gtp-binding proteins, replication proteins, translation initiation factors and atp binding proteins. these are all core functioning genes and therefore are not expected to diversify with increasing complexity. it is likely the case that the charophyte transcriptomes contain a larger proportion of core functioning genes when compared to arabidopsis, simply because they are lacking diversity in the other classes. another way to look at it is that coleochaete and spirogyra cells are analogous to plant parenchyma and are enriched for the kinds of metabolic processes that occur in parenchyma, and not due to any trend towards increasing diversity of genes in charophyte biosynthetic processes. this probably also holds true in the photosynthesis category, where both charophytes show a significant overrepresentation of genes relative to arabidopsis. an alternative hypothesis would be that, lacking structural complexity, charophytes have evolved a broader range of biochemical responses to the environment. as with most est studies, we cannot rule out the possibility that the apparent over-representation might be a sampling artifact due to non-normalized cdna or incomplete coverage.\n",
            "\n",
            "ontology by cellular compartment  reveals one extreme difference in the membrane category whereby the charophytes are extremely underrepresented. it follows that parenchymatous tissue  would involve an increased level of cell-to-cell communication requiring many more membrane proteins than would a filamentous organism, like spirogyra, or a two-dimensional disk-shaped form, like coleochaete.\n",
            "\n",
            "ultimately, we are interested in specific genes that allowed a lineage of charophytes to colonize land so successfully and give rise to land plants. because the experimental approach was a survey, and because the work reported here constitutes just a part of the overall effort to gather transcriptome data for all of the charophyte orders, we are highlighting genes of interest to follow up for further study. it is encouraging that most of the genes identified by graham et al.  <cit>  as likely to have been important in the colonization of the land have solid hits and perhaps even true orthologs in both coleochaete orbicularis and spirogyra pratensis . a couple of the more interesting genes associated with asymmetric cell division  have hits in coleochaete, but not spirogyra. this is consistent with the morphology of the organisms; spirogyra, an unbranched, filamentous alga does not have asymmetric cell division, but coleochaete definitely does. no significant blast hits of the graham et al. genes were found in the ests of the earliest diverging streptophyte, mesostigma virde. another interesting finding is the presence of ethylene biosynthesis and signalling pathway genes , long thought to be unique to land plants. this finding suggests that ethylene might be utilized in the charophytes. wang et al.  <cit>  showed that chara spp.  showed ethylene binding activity similar to that of land plants, so it is not too surprising that these genes should be present in other charophyte lineages. however, to the best of our knowledge, evidence of ethylene-signalling genes in charophytes has not been shown before this study.\n",
            "\n",
            "* indicates the ethylene binding domain  is present in the blast hit.\n",
            "\n",
            "CONCLUSIONS\n",
            "we analyzed est data from two species of charophyte green alga, spirogyra pratensis and coleochaete orbicularis as a first step toward complete genome analyses. algal cultures were grown and collected in various life stages as well as at different times during the day. all of these factors were part of an effort to maximize the total number of transcripts available for each pulled est library. we combined both sanger and  <dig> sequencing technologies to obtain as much sequence coverage as possible. both of the resulting est libraries gave a nice diversity of transcripts , with the  <dig> sequences contributing to most of that transcript diversity. this is especially pronounced when looking at the sequencing source of the singletons: sanger reads only contributed a small fraction of the total singletons in both est libraries , and the  <dig> data alone covered most of the sequences obtained by sanger sequencing. although there have been just a few  <dig> transcriptome sequencing efforts to date  <cit> , our results suggest that for studies of this type,  <dig> sequencing alone could largely replace traditional sanger sequencing. in summary, the specific genes we mention in this paper , along with the gene ontology analysis and ortholog identification, reinforce our finding of the overall genomic similarity that these two charophytes have to land plants.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "although there have been just a few  <dig> transcriptome sequencing efforts to date  <cit> , our results suggest that for studies of this type,  <dig> sequencing alone could largely replace traditional sanger sequencing.\n",
            "we analyzed est data from two species of charophyte green algal species, spirogyra pratensis and coleochaete orbicularis, as a first step toward complete genome analyses.\n",
            "our results suggest that for studies of this type,  <dig> sequencing alone could largely replace traditional sanger sequencing.\n",
            "although there have been just a few  <dig> transcriptome sequencing efforts to date  <cit> , our results suggest that for studies of this type,  <dig> sequencing alone could largely replace traditional sanger sequencing.\n",
            "our results emphasize the importance of using plant gene models to annotate the putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative putative\n",
            "43\n",
            "[{'summary_text': 'in this study, a high quality cdna library was established for gene discovery using a combination of sequence annotation, chemical catalytic features prediction and transcripts expression profiling for deep mining of target genes of the cpt metabolism pathway and molecular genetic breeding.\\ncamptothecin  is the precursor and backbone of many tias, including cpts, in plants such as c. acuminata.\\ncamptothecin  was first extracted from the stems of camptotheca acuminata in  <dig> and subsequently from nothapodytes foetida, ophiorrhiza pumila, ophiorrhiza japonica, and ophiorrhiza japonica.\\ncamptothecin  is synthesized through a modified terpenoid indole alkaloid  pathway.\\ncamptothecin  is the precursor and backbone of many tias, including cpts, in plants such as c. acuminata.\\ncamptothecin  was first extracted from the stems of c. acuminata in  <dig> and subsequently from nothapodytes foetida, ophiorrhiza pumila, ophiorrhiza japonica and ophiorrhiza'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "camptothecin  was first extracted from the stems of camptotheca acuminata in  <dig> and subsequently from nothapodytes foetida, ophiorrhiza pumila, and ophiorrhiza japonica  <cit> . cpt exhibits clinical anti-tumor activity by inhibiting dna topoisomerase i, an enzyme involved in dna recombination, repair, replication, and transcription  <cit> . cpt also inhibits the retroviruses, such as the human immunodeficiency virus  <cit> . despite its significant clinical use, the main source of cpt is still from its extraction from c. acuminata. however, the quantity is quite limited and cannot meet worldwide demand. studies on the molecular mechanism of cpt biosynthesis have long been hindered by the lack of transcriptome and genome information for c. acuminata and other cpt-producing plants. therefore, it is necessary to obtain transcriptome data and screen candidate transcripts involved in cpt biosynthesis to further understand the cpt biosynthetic pathway.\n",
            "\n",
            "cpt is synthesized through a modified terpenoid indole alkaloid  pathway. the upstream biosynthesis pathways for all the tia products are similar among alkaloid-producing plants, and involve a strictosidine backbone . over recent decades, several enzymes in the process of strictosidine biosynthesis in c. acuminata have been isolated and functionally identified. among them are tryptophan synthase   <cit>  and tryptophan decarboxylase   <cit> , which are involved in the synthesis of the indole precursor tryptamine, 3-hydroxy-3-methylglutaryl-coa synthase   <cit> , 1-deoxy-d-xylulose-5-phosphate reductoisomeras   <cit> , and 10-hydroxy geraniol oxidoreductase   <cit>  are involved in secologanin synthesis.\n",
            "\n",
            "g10h and scs, belonging to the cyp76b <dig> and cyp72a1subfamilies of cytochrome p <dig> family respectively, were identified in monoterpenoid biosynthesis from catharanthus roseus  <cit> . the synthesis of strictosidine is finally catalyzed by str, a committed enzyme for the cpt backbone biosynthesis, which was isolated and identified in rauvolfia serpentine, c. roseus, the cpt-producing plant o. japonica, and o. pumila, in previous studies. however, the genes encoding cag10h, cascs and castr, have not been yet cloned and characterized in c. acuminata.\n",
            "\n",
            "the steps following strictosidine formation  are not very clear and only a proposed biosynthetic pathway based on relative compounds extracted from cpt-producing plants has been reported  <cit>  . in the proposed pathway, a series of oxidation and hydroxylation reactions are involved in some steps of the pathway which are probably catalyzed by monooxygenases and hydroxylase, belonging to the superfamily of cytochrome p450s  <cit> . meanwhile, the branch pathway of cpt biosynthesis is unique among tia pathways because strictosidine is not immediately deglycosylated as in c. roseus  <cit> . however, it requires a glucosidase for glycoside hydrolysis, which likely occurs in one of the last steps of cpt biosynthesis in c. acuminata and other cpt-producing plants. at present, the cyp450s and glucosidase involved in cpt biosynthesis have not been studied in c. acuminata.\n",
            "\n",
            "glandular trichomes in leaves are the main site for cpt accumulation in c. acuminata  <cit> . however, gene expression involved in cpt synthesis was not detected in glandular trichomes but, instead, in epidermal cells and mesophyll cells in c. acuminata leaves, which implied the translocation of cpt between organs or cells  <cit> . multidrug resistance protein  transporters, belonging to the atp-binding cassette  transporter family, were reported to be responsible for uptake or secretion in alkaloid transportation in some plants  <cit> . therefore, we hypothesized that mdr transporters are responsible for cpt transportation from other cells to glandular trichomes in c. acuminata. at present, no cpt transport mechanism or related genes have been investigated in c. acuminata.\n",
            "\n",
            "expressed sequence tags  analysis has been a primary tool for the discovery of novel genes, based on the traditional sanger sequencing principle, which is slow and costly for non-model species with little genomic information. the emergence of high throughput platforms, such as pyrosequencing technology  <cit> , enables comprehensive study of the transcriptome for various purposes, such as development study, mirna identification, and genetic polymorphisms discovery in plants and animals  <cit> . the roche/ <dig> gs flx platform, one of the high throughput sequencing platforms, offers the advantages of longer read length and lower cost which is especially suitable for de novo transcriptome sequencing aimed at gene discovery and analysis in a specific metabolic pathway  <cit> . previous studies have indicated that the content of cpt in young leaves is higher than that in old leaves and root  <cit> , suggesting that young leaf is an important tissue for the study of cpt biosynthesis and transport. therefore, cdna from c. acuminata young leaves was subjected to de novo transcriptome sequencing to uncover genes involved in cpt biosynthesis and transport, using a roche/ <dig> gs flx titanium sequencing platform, a next-generation sequencing system. based on the sequencing and analysis results, three important genes likely to be involved in the cpt biosynthesis were cloned and analyzed. from data analysis and expression analysis, six cytochrome p450s and one glucosidase gene were found to be candidate genes in the process of cpt biosynthesis. meanwhile, three mdr transporter genes were also found to be candidate genes involved in cpt transportation.\n",
            "\n",
            "RESULTS\n",
            " <dig> sequencing and est assembly\n",
            "through  <dig> deep pyrosequencing,  <dig>  high-quality  reads were generated and then submitted to the sequence read archive of ncbi with an accession number srx <dig>  the total length of all the reads is  <dig> , <dig> bp, and the average size is  <dig> bp. after sequence assembly,  <dig>  unigenes, with an average length of  <dig> bp, were generated, including  <dig>  contigs and  <dig>  singletons. the average coverage was  <dig> -fold. the assembled contigs ranged from 96- <dig> bp, with a mean length of  <dig> bp, including  <dig>  contigs which were more than  <dig> bp . the singletons ranged from  <dig> bp to  <dig> bp, with an average length of  <dig> bp. the length distribution of hq reads  and assembled contigs  are shown for evaluation of the quality of the library. a summary of the sequencing and assembly results is provided in table  <dig> \n",
            "\n",
            "annotation and categorization\n",
            "a total of  <dig>  unigenes  were functionally characterized against the ncbi nucleotide , non-redundant protein , uniprot/swissprot, kyoto encyclopedia of genes and genomes , and arabidopsis thaliana proteome  databases  <cit> . an overview of the annotation statistics against public databases  and a summary of the most abundant  and longest transcripts of the dataset  are listed in the supporting information.\n",
            "\n",
            "to functionally categorize the information in this est pool, all unigenes were characterized by gene ontology  analysis, provided by the tair database. a total of  <dig>  unigenes were classified into three large categories and forty-five subcategories, based on go classification  <cit> , accounting for approximately 60% of all the unigenes .\n",
            "\n",
            "transcripts for proteins involved in the backbone biosynthetic pathway of cpt\n",
            "putative strictosidine synthesis genes discovered in the dataset\n",
            "strictosidine is the precursor and backbone of many tias, including cpts, in plants such as c. acuminata. a proposed biosynthetic pathway of strictosidine is shown in figure  <dig>  and each of the main enzymes present in the dataset is marked with a bold box . from the  <dig> data pool,  <dig> ests representing  <dig> enzyme genes involved in strictosidine biosynthesis were discovered. thirteen of these genes had not been previously reported, including the important enzymes geraniol-10-hydroxylase , secologanin synthase , and strictosidine synthase  . by searching the annotation information from the nr, swissprot, and kegg databases, we found that transcripts of 1-deoxy-d-xylulose-5-phosphate reductoisomerase , 10hgo, and scs were presented many ests, indicating that they are highly expressed in the young leaves of c. acuminata. g10h and tdc were both rare transcripts in the dataset, indicating that they are rarely expressed and are possibly rate-limiting genes in the tissue. the specific annotation information of some putative transcripts against the nr, swissprot and kegg databases is shown in the supporting information .\n",
            "\n",
            "notes: a--total numbers when annotated against nr, swissprot and kegg databases.\n",
            "\n",
            "g10h, scs, and str are the most important enzymes in the synthesis of strictosidine in tia-producing plants, including c. acuminata. crg10h, the first cyp <dig> in cpt synthesis, is a rate-limiting enzyme in the process of tia synthesis in c. roseus. in the  <dig> dataset, only one read of g10h was found and it had approximately 60% identical to the g10h gene of c. roseus and swertia mussotii. based on the est sequence, a putative g10h gene in c. acuminata  gene was cloned  and analyzed. similarity analysis of the amino acid sequence showed that capg10h shared 56% identity to crg10h, which implied it may have catalytic activity in geraniol hydroxylation process as in c. roseus . scs, the second cyp <dig>  is the last enzyme in the biosynthesis of secologanin. unigenes, assembled from  <dig> reads in our library, were annotated to the crscs. one putative scs gene in c. acuminata  was cloned , according to a contig that had annotated to the crscs gene. the molecular weight of the predicted protein was approximately  <dig> kda. protein subcellular localization prediction using the wolf psort program  <cit>  indicated that the presumed protein was likely targeted to the endoplasmic reticulum  membrane. protein alignment revealed that the predicted protein shared 68% amino acid identity to that in c. roseus, which is involved in a similar terpenoid indole alkaloids biosynthetic pathway . from the analysis, we inferred that the capscs gene may play a role in secologanin biosynthesis in c. acuminata. str is the enzyme that catalyzes the reaction of strictosidine synthesis. we cloned the orf of a putative str gene  from c. acuminata. phylogenetic analysis showed that str proteins from reported alkaloid-producing plants were clustered together, which implied that the candidate gene possibly plays a role in cpt biosynthesis .\n",
            "\n",
            "expression analysis of transcripts for proteins involved in strictosidine synthesis\n",
            "methyl jasmonate -induced accumulation of secondary metabolites and related gene expression has been reported in medicinal plants such as panax ginseng and c. roseus  <cit> . a previous report determined that the cpt content responded to meja and jasmonic acid, and that the response curve for jasmonic acid treatment was a waveform, with two time-specific cpt accumulation peaks in c. acuminata suspension cells  <cit> . however, there are few reports of the effect of meja treatment on the expression of genes of cpt biosynthesis in c. acuminata. in response to meja treatment, transcripts of g10h, scs and str were regulated in a waveform manner, including two expression peaks during  <dig> hours of induction . the trend of the curve was consistent with the result of a previous report for genes in anthocyanin biosynthesis  <cit> . in this study, all the detected genes responded to meja immediately, with a common peak within one hour of induction, and then decreased rapidly to even lower levels than the control. the expression levels increased again to the second peak, whose timing was gene-specific. therefore, we speculate that transcripts of cag10h, cascs, and castr were most likely to be involved in cpt biosynthesis. it has been reported that tdc genes, which are responsible for the production of tryptamine for auxin and cpts, do not respond to meja  <cit> . the expression of hmgr genes is even inhibited by meja in c. acuminata  <cit> . this is likely to be because the substrates of tdcs and hmgrs link primary and secondary metabolism, and their expression profiles are complicated.\n",
            "\n",
            "previous reports had shown that young and actively growing tissues, showed the highest level of cpt  <cit> . in this study, the mrna levels of cag10h, cascs and castr were detected using real-time pcr. the results demonstrated that the expression levels of the three genes were all higher in young leaves and petioles than in old leaves . therefore, young leaves and young petioles are the possible sites of active cpt synthesis, as well as sites of accumulation, in c. acuminata compared with the mature tissues. this difference served as a standard for real-time pcr detection for downstream candidate gene selection  <cit> . meanwhile, the expression levels of these genes were also relatively lower in the root and root bark which implied that root may not be a main synthetic tissue. this was consistent with the expression pattern of the tdc <dig> and 10hgo genes reported in a previous study  <cit> .\n",
            "\n",
            "transcripts for proteins likely to be involved in the branch pathway of cpt synthesis\n",
            "strictosidine rapidly forms the intermediate product strictosamide in c. acuminata. the steps after strictosamide synthesis remain somewhat unclear. based on the proposed branch steps, an intermediate step between strictosamide and 3-pumiloside in the cpt biosynthetic pathway was presumed to be catalyzed by a cytochrome p <dig>  with another p <dig> possibly in the last steps of cpt biosynthesis . cytochrome p450s, are a large and complex superfamily, which play important roles through catalysis of oxidation and hydroxylation reactions. in c. acuminata, no cytochrome p <dig> involved in the downstream cpt biosynthetic pathway had been cloned and identified. after est annotation against the swissprot database,  <dig> putative cytochrome p <dig> transcripts were identified in the  <dig> ests pool , belonging to  <dig> cytochrome p <dig> subfamilies, according to the standard cyp family categories . according to clan classification, transcripts of cyp <dig> clan and cyp <dig> subfamilies are likely to be involved in secondary metabolism  <cit> . a total of  <dig> cytochrome p <dig> transcripts belonging to these two subfamilies were discovered as candidate genes for further screening. glucosidases, which is a superfamily involved in various biological process including cell wall assembly, polysaccharides, plant defense and secondary metabolism, catalyze the action of deglycosylation  <cit> . it had been reported that β-d-glucosidase plays a role in glycoside hydrolysis in tia biosynthesis in plants such as c. roseus  <cit> , psychotria ipecacuanha  <cit>  and r. serpentine  <cit> . in the  <dig> dataset, one transcript  annotated as strictosidine β-d-glucosidase in c. roseus  was identified with a predicted homologous peptide of  <dig> amino acids. the peptide was found to share 70% similarity to amino acids 47- <dig> of crsgd and 62% similarity to amino acids 18- <dig> of the β-d-glucosidase ipeglu <dig> of psychotria ipecacuanha , which is involved in ipecac alkaloid synthesis. when compared with raucaffricine-o-beta-d-glucosidase  and sgd  of another tia-producing plant, r. serpentine, these proteins showed 69% and  <dig> %  similarity, respectively. crsgd, rsrd, rssgd and ipeglu <dig> all belong to the glycosyl hydrolase  family, which catalyzes the deglycosylation reaction in the tia pathway, and their substrates are strictosidine, raucaffricine and n-deacetylipecoside, respectively. the predicted glucosidase peptide demonstrated high amino acid similarity with the glucosidases identified above as being involved in the alkaloid biosynthetic pathway. therefore, capgd is likely to be a key enzyme in cpt synthesis through removal of a glucose moiety. analysis of the amino acids of the predicted peptide demonstrated that three key amino acids 161-his, 207-glu and 210-thr, which were key amino acids for catalytic activity  <cit> , are found in the corresponding residues of the predicted peptide of the pgd transcript in c. acuminata. the 388-trp was not included in the peptide.\n",
            "\n",
            "after initial screening, relative expression analysis in young/old leaves of the  <dig> cytochrome p450s and the capgd was performed for c. acuminata. consequently, six cytochrome p <dig> transcripts and one capgd transcript that were shown to be expressed three-fold higher than the control were identified as candidate genes for participating in the branch pathway of cpt biosynthesis .\n",
            "\n",
            "transcripts for proteins likely to participate in cpt transport\n",
            "mdr is a subfamily of the abc transporter family that has been reported to be related to the transport of alkaloids metabolites  <cit> . from the annotated databases,  <dig> mdr transporters were found in the  <dig> abc transporter transcripts in the library. some of the transcripts were possibly responsible for cpt transport from synthesis site to the glandular trichomes in leaves through the plasma membrane  <cit> . previous studies showed that the cpt content was four to five-fold higher in young c. acuminata leaves compared with mature leaves  <cit> . it is possible that cpt transporters were more abundant in the young leaves than in mature ones  <cit> . subsequently, the  <dig> annotated mdr transporter transcripts were subjected to expression analysis in young leaves and old leaves of c. acuminata by real-time pcr. the results showed that the expression level of three transcripts  among the annotated mdr transporters were three-fold higher in young leaves than in the mature leaves. thus, they represent candidate genes for cpt transportation in leaves .\n",
            "\n",
            "probable site of cpt biosynthesis\n",
            "young leaves are the main site for cpt accumulation; therefore, this tissue was used to identify new genes in cpt biosynthetic pathway by high throughput sequencing. after assembly and annotation,  <dig> enzyme genes that act before the step of strictosidine synthesis were found in the dataset, including the key genes encoding g10h, scs and str. this result indicated that many putative genes in cpt synthesis are expressed in young leaves, which demonstrates that young leaves are likely to be active tissues for cpt biosynthesis as well as accumulation. expression profile analysis indicated that the biosynthesis of strictosidine may be more active in young leaves and petioles than in mature leaves and roots. this result indicates that cpt is likely to be synthesized in young leaves, which is consistent with a recent study  <cit> . young leaves and petioles are likely to be the main sites for cpt biosynthesis. the lower expression in roots implies that roots may not be a main tissue for cpt biosynthesis; however, cpt does accumulate in roots. therefore, our results support the hypothesis that in c. acuminata, the main cpt synthesis site is the young leaf.\n",
            "\n",
            "the subcellular site for cpt synthesis in c. acuminata, has not been reported previously. in this study, we predicted that the cascs gene was localized in the er. the results indicated that secologanin in c. acuminata is possibly biosynthesized in the er, which was consistent with the studies of cpt location in hairy roots of o. pumila  <cit> . therefore, we hypothesize that in c. acuminata cpt is likely to be biosynthesized in the er and then transported to a vacuole  <cit>  or excreted outside the cytoplasmic membrane, as reported in o. pumila  <cit> .\n",
            "\n",
            "CONCLUSIONS\n",
            "in this study, a high quality cdna library was established to mine effective transcriptome information in cpt biosynthesis and transport in c. acuminata. a method was adopted for gene discovery using a combination of sequence annotation, chemical catalytic features prediction and transcripts expression profiling for deep mining of target genes of the cpt metabolism pathway. consequently, a number of putative transcripts, including genes encoding g10h, scs, str, cytochrome p450s, glucosidase, and mdr transporter genes, were identified as possibly being involved in cpt biosynthesis and transport. meanwhile, three important genes encoding proteins involved in cpt backbone biosynthesis were cloned and analyzed. the transcriptome data represents a valuable genetic resource for further identification of genes involved in cpt biosynthesis and transport. this dataset could be beneficial for further research of the cpt metabolism pathway and molecular genetic breeding.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study, a high quality cdna library was established for gene discovery using a combination of sequence annotation, chemical catalytic features prediction and transcripts expression profiling for deep mining of target genes of the cpt metabolism pathway and molecular genetic breeding.\n",
            "camptothecin  is the precursor and backbone of many tias, including cpts, in plants such as c. acuminata.\n",
            "camptothecin  was first extracted from the stems of camptotheca acuminata in  <dig> and subsequently from nothapodytes foetida, ophiorrhiza pumila, ophiorrhiza japonica, and ophiorrhiza japonica.\n",
            "camptothecin  is synthesized through a modified terpenoid indole alkaloid  pathway.\n",
            "camptothecin  is the precursor and backbone of many tias, including cpts, in plants such as c. acuminata.\n",
            "camptothecin  was first extracted from the stems of c. acuminata in  <dig> and subsequently from nothapodytes foetida, ophiorrhiza pumila, ophiorrhiza japonica and ophiorrhiza\n",
            "44\n",
            "[{'summary_text': 'the aim of this study was to assess the seroprevalence of eehv-gb in asian elephants by using alternative translation upstream of the shine dalgarno sequence.\\nthe aim of this study was to assess the seroprevalence of eehv-gb in asian elephants by using alternative translation upstream of the shine dalgarno sequence.\\nthe aim of this study was to assess the seroprevalence of eehv-gb in asian asian elephants.\\nthe aim of this study was to assess the seroprevalence of eehv-gb in asian asian elephants.\\nthe aim of this study was to assess the seroprevalence of eehv-gb in asian elephants by using alternative translation upstream of the shine dalgarno sequence.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the family herpesviridae comprises a diverse group of viruses, which are divided into three subfamilies   <cit> . most commonly, herpesvirus infection presents as a benign, self-limiting disease with a mild viraemia. herpesvirus rarely causes widespread disseminated disease in the natural host, although cross-species transmission to non-reservoir hosts can also result in lethal infections  <cit> . in general neonates and immune-compromised hosts are at higher risk of developing a fatal herpesvirus infection  <cit> .\n",
            "\n",
            "the elephant endotheliotropic herpesvirus  is a poorly characterized virus of elephants that was first observed in healthy wild african elephants  by mccully et al.  <cit>  in  <dig> and subsequently in asian elephants  with no apparent disease  <cit> . in  <dig> the death of a 3-year old female asian circus elephant was associated with herpesvirus by ossent et al.  <cit> . pathologic changes in the capillary endothelial cells with herpetic inclusion bodies and weak cross-neutralization in serum neutralization tests with bovine herpesviruses suggested that the etiologic agent was a hitherto unidentified herpesvirus  <cit> , which was identified in  <dig>  <cit> . the disease has a sudden onset characterized by lethargy, oral ulcerations, edema in the skin of the head and proboscis, cyanosis of the tongue, decreased white blood cell and platelet count and massive internal hemorrhages, which lead to death within 12–72 h after onset of the symptoms. phylogenetic analysis showed that eehv is a distinct branch within the β-herpesviridae group, assigned as the new genus proboscivirus  <cit> . to date, as many as seven different genotypes of eehv have been identified within this genus  <cit> . however, xtensive evaluation of several subtypes indicated that eehvs have a large genomic inversion of a 40-kb core segment that is distinct from the roseoloviruses and all other β-herpesviruses. furthermore, they encode α-herpesvirus-like genes that are absent in β-herpesviruses and contain  <dig> novel open reading frames not found in any other herpesvirus  <cit> . these findings suggest that this particular virus is so different from other herpesviruses that it may be considered as a new herpesvirus subfamily  <cit> .\n",
            "\n",
            "eehv poses a threat to the conservation mission of zoos. several studies have shown that dnaemia can be detected in asian elephants before the onset of clinical signs and in elephants that do not display clinical signs using pcr techniques . to date, this is the only option to monitor a complete herd, and only allows identification of animals with an active infection. therefore, the detection of antibodies against eehv likely offers a better strategy to determine which animals are carriers of eehv. ideally, antibody detection against the whole virus is preferable, but since the virus cannot be cultured in vitro, the best alternative is to use an immunodominant protein, such as glycoprotein b , as an antigen. the viral envelope protein glycoprotein b of herpesviruses is involved in host cell entry and represents an important target of neutralizing antibodies in the host  <cit> . in this study a serologic test based on the eehv-gb antigen is described. this assay was used to assess seroprevalence against eehv-gb in serum and heparin plasma samples from two large cohorts of european and north-american captive asian elephants. these data are useful to link seroprevalence with eehv virus detection by pcr methods and, at the end, may give better insight into eehv dynamics within elephant populations.\n",
            "\n",
            "methods\n",
            "cloning, expression and large scale production of glycoprotein b\n",
            "the viral sequence of eehv-1a glycoprotein b   <cit>  comprising the start codon up to the transmembrane domain was synthesized  and cloned, using xhoi/psti restriction sites, into the multiple cloning site in frame with a c-terminal myc/6his tag, but in an alternative open reading frame  relative to the shine dalgarno ribosomal binding site, of the bacterial expression vector ptrchis2a . induction of recombinant glycoprotein b expression was performed in e.coli rosetta2plyss codon plus strain . this strain supplies trnas for  <dig> rare codons  in order to improve the expression of heterologous proteins.\n",
            "\n",
            "a single colony transformed with the ptrchis2a-gbmyc/his plasmid was inoculated into fresh lb medium containing 100 μg/ml ampicillin and 50 μg/ml chloramphenicol and grown overnight at 37 °c in a shaker incubator. the next day  <dig> l fresh lb medium without antibiotics was inoculated  <dig> in  <dig> with the pre-culture and grown in a shaker incubator at 37 °c to an od600 ~  <dig> . the recombinant protein expression was induced by adding iptg to an end concentration of 1 mm. the temperature was lowered to 25 °c for optimal recombinant protein expression.\n",
            "\n",
            "cells were harvested 4 h after induction and pelleted at 6000xg for 10 min in a beckman highspeed centrifuge using the jla <dig>  rotor. the supernatant was decanted and the cells were resuspended in native lysisbuffer , 5 % v/v glycerol and 10 mm of imidazole and protease inhibitors). lysozyme was added to an end concentration of 1 mm and the cells were treated 30 min at 4 °c at continuous agitation. the cells were cracked by three freeze/thaw cycles and sonification with a microtip;  <dig> x 30s at 60 % on ice . the lysate was cleared by ultracentrifugation for 2 h at  <dig>  rpm in a beckman optima l-90 k ultracentrifuge using the sw32ti rotor. a similar procedure was conducted with another ptrchis2a vector encoding a  <dig> kd irrelevant 6his-tagged protein that was used as a negative control.\n",
            "\n",
            "purification of glycoprotein b\n",
            "the cleared lysate was loaded onto ni-nta resin  which was preconditioned with water and 1x native lysisbuffer. binding was performed overnight at +4 °c under continuous agitation. next day, the resin was settled vertically and washed with  <dig> bed volumes of lysisbuffer containing 20 mm of imidazole. thereafter, the bound recombinant protein was eluted with  <dig> bed volumes of elution buffer , 5 % v/v glycerol and 300 mm of imidazole). subsequently, salts were removed by dialysis in a slide-a-lyzer cassette  for 48 h at +4 °c. the dialysis buffer  was refreshed three times. after dialysis the recombinant protein content was analyzed for protein concentration  and western blot. a second round of purification was performed by fast protein liquid chromatography  with a 1 ml volume histrap-hptm column  with a pressure flow of  <dig>  ml/min. the his-tagged proteins were eluted from the column with  <dig> column volumes  of a linear gradient ranging from 50 mm imidazole up to 500 mm imidazole in  <dig>  ml aliquots. these were analyzed by western blot and the eluates containing the his-tagged proteins were subsequently pooled and dialyzed as described above.\n",
            "\n",
            "rabbit anti-eehv glycoprotein b serum\n",
            "five peptide sequences  derived from the amino acid sequence of glycoprotein b of eehv1a  were synthesized and coupled to lph  carrier protein. ten rabbits  were immunized six times intramuscularly with 10 μg of the respective peptides. final bleeding was at day  <dig> after the first vaccination. the polyclonal rabbit sera were produced by biogenes gmbh  according to european law on animal welfare. all rabbit sera were highly reactive against the respective peptide as tested in a peptide elisa assay.\n",
            "\n",
            "immunodetection of recombinant glycoprotein b\n",
            "recombinant glycoprotein b antigen was analyzed by separation of small samples from the purification process on a 10 % sds-page gel  and separated products were subsequently transferred to pvdf membrane . gels were run in duplicate; one was coomassie blue stained for total protein and the duplicate was analyzed with western blot using an anti-6his tag mouse monoclonal antibody . as a secondary antibody goat anti-mouse irdye <dig>  was used.\n",
            "\n",
            "rabbit serum reactivity with recombinant glycoprotein b was detected with western blot by incubating the membranes with rabbit serum  followed by detection with goat-anti-rabbit irdye <dig> . similar strategy was done for elephant serum , but primary antibody was detected with rabbit-anti asian elephant igg . sera of rabbits as well as elephants were pre-incubated in buffer containing e.coli lysate  for 1 h at room temperature in order to reduce possible e.coli reactivity. the signal of the conjugate was detected with the licor odyssey scanner .\n",
            "\n",
            "glycoprotein b specific capture elisa\n",
            "to avoid non-specific reaction against e.coli contaminants which co-purified with the 6his-tagged eehv gb antigen during ni-nta purification an optimal concentration of 100 ng/well of mouse-anti 6his antibody  as a capture antibody was coated in high binding 96-well microtiterplate . the microtiter plates were blocked with 100 μl/well of pbs containing 2 % w/v bsa in pbs for one hour at 37 °c and washed. all washing steps were performed three times with 200 μl of wash buffer . to determine the optimal antigen concentration for use in the elisa, the gb antigen was diluted two-fold starting at 2400 ng per well. a pool of rabbit sera was used as anti-gb positive serum.\n",
            "\n",
            "the purified eehv-gb antigen was diluted in pbs containing  <dig>  % triton-x to a concentration of 750 ng/well and bound to the capture antibody for 1 h at 37 °c.\n",
            "\n",
            "after washing, primary incubation was performed with 100 μl/well diluted  elephant serum in elisa buffer containing pbs,  <dig>  % bsa,  <dig>  % non-fat dried powder milk and 3 % w/v nacl and incubated for one hour at 37 °c. after washing, elephant specific igg’s bound to glycoprotein b were detected by incubation with 100 μl/well of rabbit anti-elephant igg  for one hour at 37 °c followed by an incubation step with swine anti-rabbit igg hrpo conjugate . after washing 100 μl/well 1xtmb substrate  was added, incubated for 10 min at room temperature. the reaction was stopped by adding 100 μl  <dig>  m h2so <dig> per well. the absorbance  was measured at 450 nm with 620 nm reference filter  on a 96-well plate spectrophotometer .\n",
            "\n",
            "study cohort in europe/usa\n",
            "the elephants in this study were kept in zoos as part of the european endangered species program  and the species survival program . within these programs elephants are trained to allow voluntarily blood collection for several reasons: monitoring health parameters, monitoring reproduction parameters and providing  data for new diagnostic technologies for diseases that may threaten their health. the use of banked elephant blood samples for a retrospective research study falls within the law. all participating zoos reviewed and approved the research proposal for this study and allowed the use of their banked elephant blood samples.\n",
            "\n",
            "all elephants in the study were asian elephants; table  <dig>  and  <dig>  summarize the age, sex, and clinical status of elephants which will be discussed in detail in this study. all participating zoos reviewed and approved the research described in this study. initially,  <dig> serum or heparin plasma samples of  <dig> asian elephants located in  <dig> european zoos were tested for validation of the capture elisa test. the cut-off value was calculated as a mean od of  <dig> samples of  <dig> animals which scored consistently on the base line level of the test.table  <dig> age, sex, and clinical status of elephants in the study cohort of european zoos\n",
            "\n",
            "zoos are numbered, elephants are indicated with a letter and subsequent numbers describe maternal offspring\n",
            "\n",
            "\n",
            "nd, not detectable\n",
            "\n",
            "\n",
            "tw, trunk wash\n",
            "\n",
            "\n",
            "c, conjunctiva\n",
            "\n",
            "\n",
            "u, urine\n",
            "\n",
            "\n",
            "o, oral lesion\n",
            "\n",
            "\n",
            "wb, whole blood\n",
            "\n",
            "\n",
            "v, vulva\n",
            "\n",
            "\n",
            "\n",
            "three european zoos  provided asian elephant serum or plasma samples for a longitudinal seroprevalence study ranging from  <dig> up to 600 days for multiple animals in their herds. moreover, the samples from zoo  <dig> and  <dig>  were also analyzed for eehv presence by qpcr.\n",
            "\n",
            "in the usa cohort  <dig> asian elephant samples were tested from  <dig> animals in  <dig> zoos in the period  <dig> until  <dig> with a six month interval.\n",
            "\n",
            "all samples were tested in 1: <dig> and 1: <dig> dilutions and the signal to background od was calculated for both dilutions. a sample was considered positive when both dilutions produced ods  <dig> times of the background od and undetectable when both dilutions produced less than  <dig> times the background od. a sample was considered as borderline when one or both dilutions measured an od between 2– <dig> times the background od values.\n",
            "\n",
            "an animal was considered to be seropositive when all analyzed samples scored positive with an occasional borderline score. an animal was considered borderline seropositive when the majority of its samples scored borderline. an animal was considered undetectable when all of its samples were undetectable.\n",
            "\n",
            "RESULTS\n",
            "expression and purification of eehv glycoprotein b\n",
            "positioning of the eehv-gb open reading frame  in an alternative position downstream of the shine dalgarno ribosomal binding site  severely reduced the expression levels of the recombinant eehv-gb, but retained the recombinant protein in the soluble fraction and in its native conformation. purification of the recombinant eehv-gb was performed using nickel columns . full length eehv-gb has an expected molecular size of  <dig> kd, and a band at this expected size was observed . since the portion of recombinant gb protein in the e.coli lysate was low many native proteins of e. coli rosetta <dig> that displayed affinity to nickel in the immobilized metal affinity chromatography  were co-purified during ni-nta purification. full length as well as many other molecular weight sizes, were detected by anti-6his-tag antibody detection on western blot. western blot analysis with the eehv-gb peptide specific rabbit  serum showed more bands with different molecular weights compared to the pattern obtained with anti-his antibody . several specific smaller bands were also detected. an elephant serum, which did contain eehv-gb specific antibodies, as detected with the described sandwich elisa, did not recognize as many of the eehv-gb fragments on western blot . since the double purified  lost a portion of the full length eehv-gb  the single ni-nta purified antigen was further used to develop an elisa test.fig.  <dig> sds-page total protein analysis  and western blot analysis with an anti-his monoclonal antibody  demonstrate the purification of full length eehv-gbhis  and several smaller fragments with a his-tag produced from the eehv-gbhis open reading frame . during purification a small portion of the recombinant protein is lost in the flow through of the ninta column  and during the washing steps . the empty vector does not yield any his-tagged proteins . lane m is the molecular weight marker\n",
            "\n",
            "fig.  <dig> western blot analysis of an irrelevant  <dig> kd his-tagged protein and , single ninta purified eehv-gbhis  and double purified ninta/histrap fplc  with three different antisera. a western blot with mouse monoclonal anti-his demonstrates the  <dig> kd his-tagged protein in lane  <dig> and the highly fragmented eehv-gbhis in lane  <dig> and  <dig>  double purification results in loss of the full length eehv-gbhis . bthe rabbit peptide  eehv-gb specific antiserum recognizes non-his tagged bands. c serum of elephant 4a detects identical bands as the rabbit peptide specific serum in the eehv-gbhis recombinant protein, except for the ~ <dig> kd band which might be an e.coli background band \n",
            "\n",
            "\n",
            "\n",
            "validation of an eehv glycoprotein b specific capture elisa\n",
            "to reduce aspecific e.coli protein reactivity  the elisa assay was setup as a capture elisa using a mouse-anti 6his monoclonal antibody. pooled sera of rabbits immunized with eehv-specific peptides were used as a positive control. figure  <dig> shows that 100 ng of the capture antibody reduces aspecific reactivity to an acceptable background level of an a450/ <dig> of ~ <dig> . subsequently, the capture elisa was further validated with an antigen checkerboard titration to test its specificity with various irrelevant rabbit sera. the results in fig.  <dig> demonstrate a dose-related response only when eehv-gb specific sera were used. irrelevant rabbit antisera did not react with the antigen. to detect antibody reactivity in the linear range an optimum of 750 ng antigen/well was used in the elisa, which was used to evaluate the elephant sera cohorts. once the capture elisa was developed with the eehv-gb specific rabbit sera, a large panel of elephant sera was tested to screen for sera with specific reactivity. in total  <dig> serum or plasma samples of asian elephants, representing  <dig> animals of  <dig> zoos were tested. twenty-three animals from  <dig> zoos tested consistently positive ,  <dig> animals of  <dig> zoos tested intermittently positive/borderline/undetectable  and  <dig> animals tested consistently undetectable . the test was repeated twice in separate experiments and the results were reproducible. we found three sera with extremely high reactivity in the capture elisa  and several non-reactive sera . these elephant sera were used to test again the antigen dose and specificity . the sera of elephant 2b <dig>  4a and 5a showed a dose-dependent reactivity whereas the serum from elephant 2c did not . serial dilutions of serum of animal 5a  and animal 2c  on eehv-gb antigen or an irrelevant 6his-tagged protein were used in the assay to test the specificity. only the positive serum of animal 5a reacted in a dose-dependent manner with eehv-gb antigen and not with the irrelevant antigen .fig.  <dig> \n",
            "e. coli protein contamination in the purified antigen  induces background signal in an indirect elisa . this background is reduced in a sandwich elisa using a mouse monoclonal anti-his antibody as a capture antibody for the target antigen . pooled eehv-gb peptide specific rabbit serum was used in a 1: <dig> dilution. panel a no capture antibody, panel b 100 ng capture antibody, panel c 200 ng capture antibody, panel d 300 ng capture antibody\n",
            "\n",
            "fig.  <dig> specificity of the eehv-gb sandwich elisa was determined by antigen titration with the eehv-gb peptide specific rabbit serum, one serum of an eehv <dig> pcr positive elephant and two irrelevant rabbit sera. all sera were diluted 1: <dig> in elisa buffer\n",
            "\n",
            "fig.  <dig> characterization of the sandwich elisa with elephant sera. elephant sera react in a dose-dependent manner as shown by antigen titration  and are specific for the eehv-gb antigen only \n",
            "\n",
            "\n",
            "\n",
            "the majority of eehv pcr positive elephants display eehv specific antibodies\n",
            "for  <dig> animals  eehv pcr data was available, either from published data  <cit>  or personal communications. we compared these results with our elisa results. nineteen out of  <dig> pcr positive animals  displayed antibodies, of which  <dig> had significant antibody titers and three had borderline titers. however, in six out of  <dig> animals no eehv specific antibodies could be detected  while they had been tested positive with the eehv specific pcr. eight elephants displayed eehv-gb specific antibodies, but never tested positive in a routine pcr . four animals were eehv <dig> pcr positive of which three displayed antibodies against eehv-gb.table  <dig> age, sex, and clinical eehv status of elephants in the study cohort of usa zoos\n",
            "\n",
            "zoos are numbered, elephants are indicated with a letter and subsequent numbers describe maternal offspring\n",
            "\n",
            "\n",
            "nd, not detectable\n",
            "\n",
            "\n",
            "tw, trunk wash\n",
            "\n",
            "\n",
            "c, conjunctiva\n",
            "\n",
            "\n",
            "u, urine\n",
            "\n",
            "\n",
            "o, oral lesion\n",
            "\n",
            "\n",
            "wb, whole blood\n",
            "\n",
            "\n",
            "v, vulva\n",
            "\n",
            "\n",
            "\n",
            "seroprevalence of a european study cohort\n",
            "three european zoos were approached to collaborate in our study; their animals are listed in table  <dig>  and their asian elephant sera collections were analyzed over a substantial period of time . during the study period the animals of zoo  <dig> and  <dig> were screened for eehv virus presence in whole blood, trunk wash and conjunctiva swabs by either conventional or real time pcr. figure  <dig> displays the serostatus of these three herds; the observed patterns showed a rise-and-fall in signal to background ratios . a similar rise-and-fall pattern was observed for animals, which scored below the detection level. additional clinical evaluations of the animals in this study can be found in .fig.  <dig> serostatus of three european zoos measured as the signal to background ratio over a period of time. zoo  <dig>  was studied from october  <dig> until april  <dig>  zoo  <dig>  was studied from may  <dig> until may  <dig> and zoo  <dig>  from january  <dig> until september  <dig>  the arrow indicates a positive eehv <dig> pcr of the trunk wash  or whole blood . elephants with non-detectable levels of eehv antibodies are indicated in black, elephants with an alternating serostatus in orange and elephants with continuous anti-eehv antibodies in red\n",
            "\n",
            "\n",
            "\n",
            "only two animals had a positive pcr in the study-period in either trunk wash  or whole blood . as seen in fig. 6a animal 1a was eehv <dig> pcr positive in trunk wash on day  <dig> and, shortly afterwards, had elevated antibody titers up to 8 weeks after the pcr positive trunk wash. as shown in fig. 6b calf 2b <dig> developed a dnaemia and clinical signs of eehv disease on day  <dig> of the study period with typical eehv clinical signs. however, this calf was already seropositive on day  <dig> of the study period. he survived this episode with supportive treatment and famciclovir administration. eighteen days prior to disease onset his half-sister 2e <dig> showed a rise in antibody titers. unfortunately no samples of the other herd members were available in this period of the study.\n",
            "\n",
            "discussion\n",
            "to date, an accurate and dependable test to evaluate eehv seroprevalence in asian elephants has not been developed. in part, this is due to the inability to culture the virus in vitro and therefore whole virus is not available as the ideal antigen source for an elisa. production of recombinant proteins from one of the eehv glycoproteins, like gb, has proven to be a major challenge. here we describe a novel elisa that specifically detects eehv-gb antibodies circulating in asian elephants. we were able to link seroprevalence to pcr detection of the eehv in approximately 80 % animals. this novel assay will provide more insight into the epidemiology of this hitherto poorly understood virus.\n",
            "\n",
            "the antigen expression, production and purification methods for fragmented eehv-gb  driven by a bacterial shine dalgarno promoter in an alternative open reading frame, yielded antigens for a sandwich elisa specific for eehv-gb. incomplete eehv-gb amino acid sequences were produced from the plasmid presumably due to programmed ribosomal frameshifting  and alternative translation initiation sites. prf and alternative translation initiation are well-described phenomena  and can occur in prokaryotes, as well as in eukaryotes. prf occurs when the ribosome encounters a so-called hungry codon  in its a-site followed by a slippery sequence in the p-site of the ribosomal complex. in our expression system, it is likely that the shine dalgarno sequence rbs initiates the overexpression of a short nonsense peptide of  <dig> amino acids thereby releasing the ribosomes quickly in order to re-initiate translation with a high likelihood to undergo prf since this small orf contains multiple hungry codons. as a side product, fragments of eehv-gb specific sequences, either with or without a his-tag were made and confirmed by western blot analysis. this suggested that the non-his-tagged fragments were co-purified during the purification process. eehv-gb peptide specific rabbit sera were able to detect the antigen fragments, whereas positive elephant sera recognized fewer of these fragmented antigen products. it is likely that the majority of the anti-eehv-gb antibodies in elephants recognize conformational epitopes, which are linearized in western blot. this is supported by the fact that the produced antigen was highly reactive in the sandwich elisa test with the positive rabbit sera as well as many elephant sera and not with irrelevant sera. as a result, it is likely that this elisa would fail to detect antibodies directed to those conformational epitopes of gb which do not form properly in the bacterial expression machinery. in this study no seronegative elephants were found. the background of the described assay was set up with sera of elephants, which were consistently found on the non-detectable range of the assay, but still ~ <dig>  od higher than naïve rabbit serum. this test was developed with a capture antibody, which did not completely reduce the background to an od of  <dig>  therefore, it is highly likely that the cut-off value was leveled unrealistically high. in the study cohort we found several seropositive elephants, which were routinely diagnosed with other subtypes than eehv1a by pcr. the obtained data suggest that this test may be able to detect antibodies against gb from eehv subtypes other than eehv1a; at least eehv1b and eehv <dig>  divergence on the amino acid level of the glycoprotein b of the subtypes eehv ranges from 11 %  up to 21 %   <cit> . however, we cannot exclude that these animals did had an eehv1a co-infection as well. it is interesting to speculate that genetic background plays a role in susceptibility to develop severe disease. however, better designed study cohorts, which include serology as well as pcr detection of the virus are required in order to address the question whether the host genetic background has a major role in fatal eehv disease.\n",
            "\n",
            "CONCLUSIONS\n",
            "we have shown that it is possible to express the gb of eehv in a bacterial system by using alternative translation upstream of the shine dalgarno sequence. further, our results and those of recent reports  <cit>  demonstrate that eehv prevalence in captive born as well as wild-born elephants  is much higher than initially anticipated. these studies underline that elephants may be the natural host of eehv that, in general, does not cause major disease. the methodology and data presented in this article give a novel insight into the eehv <dig> seroprevalence in asian elephants. as expected, a large portion of elephants showed antibodies against eehv, but not all. additionally, the serostatus of elephants within a herd may vary from non-detectable up to high titers.\n",
            "\n",
            "additional file\n",
            "additional file 1: \n",
            "supplementary information. \n",
            "\n",
            "\n",
            "\n",
            "abbreviations\n",
            "bcabicinchoninic acid\n",
            "\n",
            "bsabovine serum albumin\n",
            "\n",
            "cvcolumn volume\n",
            "\n",
            "eehvelephant endotheliotropic herpes virus\n",
            "\n",
            "eependangered species program\n",
            "\n",
            "elisaenzyme linked immunosorbent assay\n",
            "\n",
            "emelephas maximus\n",
            "\n",
            "fplcfast protein liquid chromatography\n",
            "\n",
            "gbglycoprotein b\n",
            "\n",
            "hprohorse radish peroxidase\n",
            "\n",
            "imacimmobilized metal affinity chromatography\n",
            "\n",
            "iptgisopropyl β-d-thiogalactoside\n",
            "\n",
            "laloxodonta africana\n",
            "\n",
            "lbluria bertani broth\n",
            "\n",
            "lphlimulus polyphemus hemocyanin\n",
            "\n",
            "ni-ntanickel nitrilotriacetic acid\n",
            "\n",
            "nmnanometer\n",
            "\n",
            "odoptical density\n",
            "\n",
            "orfopen reading frame\n",
            "\n",
            "pbsphosphate buffered saline\n",
            "\n",
            "pcrpolymerase chain reaction\n",
            "\n",
            "prfprogrammed ribosomal frameshifting\n",
            "\n",
            "pvdfpolyvinylidene difluoride\n",
            "\n",
            "qpcrquantative polymerase chain reaction\n",
            "\n",
            "rbsribosomal binding site\n",
            "\n",
            "sds-pagesodium dodecyl sulphate polyacrylamide gel electrophoresis\n",
            "\n",
            "sspspecies survival program\n",
            "\n",
            "tmb <dig> ’, <dig> ’-tetramethylbenzidine\n",
            "\n",
            "bcabicinchoninic acid\n",
            "\n",
            "bsabovine serum albumin\n",
            "\n",
            "cvcolumn volume\n",
            "\n",
            "eehvelephant endotheliotropic herpes virus\n",
            "\n",
            "elisaenzyme linked immunosorbent assay\n",
            "\n",
            "emelephas maximus\n",
            "\n",
            "fplcfast protein liquid chromatography\n",
            "\n",
            "gbglycoprotein b\n",
            "\n",
            "hprohorseradish peroxidase\n",
            "\n",
            "imacimmobilized metal affinity chromatography\n",
            "\n",
            "iptgisopropyl b-d-thiogalactoside\n",
            "\n",
            "laloxodonta africana\n",
            "\n",
            "lbluria bertani broth\n",
            "\n",
            "lphlimulus polyphemus hemocyanin\n",
            "\n",
            "ni-ntanickel nitrilotriacetic acid\n",
            "\n",
            "nmnanometer\n",
            "\n",
            "odoptical density\n",
            "\n",
            "orfopen reading frame\n",
            "\n",
            "pbsphosphate buffered saline\n",
            "\n",
            "pcrpolymerase chain reactionprf, programmed ribosomal frameshifting\n",
            "\n",
            "pvdfpolyvinylidene difluoride\n",
            "\n",
            "qpcrquantative polymerase chain reaction\n",
            "\n",
            "rbsribosomal binding site\n",
            "\n",
            "sds-pagesodium dodecyl sulphate polyacrylamide gel electrophoresis\n",
            "\n",
            "tmb <dig> ’, <dig> ’-tetramethylbenzidine\n",
            "\n",
            "competing interests\n",
            "\n",
            "we declare no relationships or support which could be perceived as a conflict of interest.\n",
            "\n",
            "authors’ contributions\n",
            "\n",
            "pd provided conceptual design and technical execution on all aspects of the study. she wrote the manuscript. vrp and svrf provided excellent technical assistance in the antigen production procedure. ws is involved in the european eehv program and study director. he provided the european asian elephant serum collection and supported pd/bm/ao in the clinical evaluation of the study results. el contributed to this study by providing the usa asian elephant samples. lh and pl contributed to this study by kindly allowing pd to use their lab facilities in the usa for performing the sample analysis of the usa cohort. on account of cites  regulation asian elephant serum samples need to be analyzed in situ. sc contributed to this study by providing european asian elephant samples. nm and ad contributed to this study by providing the european asian elephant samples and were involved in the clinical evaluation of the study results. they provided pcr results of elephants in this study, which were analyzed in their lab facilities. ao is the principle investigator of the study. bm supervised all aspects of this study and revised all the draft versions of the manuscript. all authors read and approved the final manuscript.\n",
            "\n",
            "the authors thank the following contributors to this study; the keepers, curators, and healthcare staff of the involved zoos; ms. a. brothers ; mrs. e. van der graaf ; dr. r. bodewes for the discussion on the manuscript and dr. r. de vries for the assistance with the image lay-outs. we highly appreciate the up-to-date international elephant database website maintained by dan koehl.\n",
            "\n",
            "financial support\n",
            "zoological society of london\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the aim of this study was to assess the seroprevalence of eehv-gb in asian elephants by using alternative translation upstream of the shine dalgarno sequence.\n",
            "the aim of this study was to assess the seroprevalence of eehv-gb in asian elephants by using alternative translation upstream of the shine dalgarno sequence.\n",
            "the aim of this study was to assess the seroprevalence of eehv-gb in asian asian elephants.\n",
            "the aim of this study was to assess the seroprevalence of eehv-gb in asian asian elephants.\n",
            "the aim of this study was to assess the seroprevalence of eehv-gb in asian elephants by using alternative translation upstream of the shine dalgarno sequence.\n",
            "\n",
            "45\n",
            "[{'summary_text': 'auf1-deficient mouse showed a 40% decrease in the number of splenic b cells, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germin'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the mammalian spleen functions to remove old and damaged erythrocytes, participates in the immune response, particularly against blood-borne pathogens, and is the major site for peripheral b cell development  <cit> . immature surface immunoglobulin-expressing b cells that reach the spleen from the bone marrow  represent developmental precursors to mature follicular  b lymphocytes, the major mature b cell population in the spleen  <cit> . the other two mature b cell populations consist of non-circulating splenic marginal zone  b cells that are believed to be derived from transitional cells, and b- <dig> cells, that are controversial in origin and are enriched in peritoneal and pleural cavities  <cit> . fo b cells contribute to most t-cell dependent  responses that induce germinal center  development of affinity-matured long lived plasma cells and memory b cells  <cit> . in contrast, mz cells and b- <dig> cells are predominately responsible for the initial rapid t-cell independent  igm antibody response and form an important line of defense against antigens and pathogens in the blood and mucosal sites  <cit> . more recent analysis of b cell populations suggests that there exist two distinguishable, long-lived, recirculating post-transitional follicular b cell populations, the mature fo b cell subset described above and a new subset referred to as fo-ii b cells. the fo-ii b cells differ from fo b cells in that they develop in an antigen-independent manner and may serve as a follicular precursor to both mz b cells and fo b cells  <cit> .\n",
            "\n",
            "although the precise mechanisms are still unclear, maintenance of each peripheral b cell subset is affected by the availability of resources, the local environment, and interactions with other cell types  <cit> . it is known that signaling through the b cell receptor  is required for the development and maintenance of mature splenic b cells  <cit> . genetic studies utilizing knockout mouse models for various bcr complex components and downstream effectors clearly demonstrate that an adequate bcr signal is indispensable for the development of transitional b cells and further differentiation into mature b cells  <cit> . furthermore, an ongoing or 'tonic' bcr signal is required for b cells to survive  <cit> . this was elegantly shown using an inducible disruption of the bcr in mature b cells, which resulted in the absence of all three mature subsets  <cit> . in addition to bcr signaling, both the splenic microenvironment and the ability to respond to locally produced growth factors play equally important roles in the development and maintenance of mature b cells  <cit> .\n",
            "\n",
            "many short-lived mrnas that encode cytokines and cell survival factors contain an adenosine/uridine-rich element  in the 3' non-coding region that allows for their post-transcriptional regulation by are-binding proteins. auf <dig>  also known as heterogeneous nuclear ribonucleoprotein d, is important in promoting the decay of are-containing mrnas  <cit> . in addition to mrna turnover, auf <dig> has more recently been implicated in a number of other cellular processes, including mrna translation  <cit>  and chromatin remodeling  <cit> . the physiological role of auf <dig> in controlling the decay of key pro-inflammatory cytokine mrnas, interleukin -1β and tumor necrosis factor -α, was demonstrated by the increased sensitivity of auf1-/- mice to endotoxic shock and lethality  <cit> . auf <dig> also regulates inflammation within the skin, and auf1-/- mice develop atopic-like dermatitis  <cit> .\n",
            "\n",
            "given the high expression of auf <dig> in lymphoid organs  <cit> , we characterized the lymphoid compartment of auf1-deficient mice. here we show that auf1-/- mice develop spleens of reduced size that support a roughly two-fold reduction in the number of lymphocytes. fo b cells exhibit a two-fold decrease in average half-life in auf1-/- mice that correlates with reduced expression of bcl- <dig>  a <dig>  and bcl-xl, as well as prominent apoptosis within splenic follicles. the increased turnover of fo b cells corresponds to their decreased frequency in auf1-/- mice. our studies indicate that auf <dig> plays an important role in regulating splenic lymphocyte cellularity and fo b cell maintenance.\n",
            "\n",
            "methods\n",
            "mice\n",
            "the auf1-/- mice were derived as previously described  <cit> . rag1-/- mice  and cd <dig>  mice  were purchased from jackson labs. for adoptive transfer studies,  <dig> ×  <dig> donor-derived bone marrow cells were injected intravenously into sub-lethally irradiated  rag1-/- or into lethally irradiated  auf1+/+ and auf1-/- recipient mice. chimera mice were analyzed  <dig> weeks post transfer. cd <dig> allele expression was used to distinguish donor and recipient populations. all mice are kept under specific pathogen-free conditions. all animal protocols were approved by the nyu institutional animal care and use committee.\n",
            "\n",
            "flow cytometry\n",
            "single cell suspensions were prepared from spleen, inguinal and mesenteric lymph nodes, peritoneum lavage, bone marrow, and thymus. cells were stained and analyzed as previously described  <cit> . cell sorting was performed on a bd facsvantage cell sorter. the following abs were used: from bd pharmingen: anti-b220-percp , anti-cd3-percp , anti-cd4-fitc , anti-cd8-apc , anti-igm-fitc , anti-cd94-apc , streptavidin-apc, streptavidin-fitc; from caltag: anti-cd62l-pe , anti-cd23-pe ; from ebioscience: anti-cd44-biotin , anti-igm-biotin , anti-cd5-fitc , anti-cd24-pe , anti-igd-biotin , anti-cd21/cd35-fitc,-apc , anti-f4/80-biotin , anti-cd86-biotin , anti-cd <dig> -biotin , and anti-cd <dig> -biotin . bcl- <dig> levels were detected using the bcl- <dig> kit , following manufacturer's instructions. the set contains anti-bcl-2-pe  and pe-conjugated isotype control.\n",
            "\n",
            "immunohistochemistry\n",
            "hematoxylin and eosin stained sections were processed as previously described  <cit> . germinal center staining was performed at  <dig> and  <dig> days post-immunization with tnp-ficoll and tnp-klh, respectively. the reagents used were from ebioscience: anti-mouse b <dig> ; cell signaling: rabbit anti-cleaved caspase- <dig> ; bd pharmigen: cr <dig> ; vector laboratory: biotinylated peanut agglutinin, biotinylated anti-rabbit igg, vectastain abc-ap, vectastain abc, vector blue ap kit, and vector novared peroxidase kit.\n",
            "\n",
            "brdu labeling\n",
            "mice were injected intraperitoneally with  <dig> mg of brdu  and fed with drinking water containing  <dig> mg/ml brdu for  <dig> days. at  <dig>   <dig>  and  <dig> days post brdu administration, spleens from  <dig> mice per group were analyzed by facs. cells were stained with anti-cd21-fitc and anti-cd24-pe or anti-cd23-pe, and then were fixed for  <dig> h at 4°c with  <dig> % paraformaldehyde and  <dig> % tween- <dig> in pbs. cells were washed, incubated for  <dig> min at 37°c with  <dig> mm mgcl <dig>  1% bsa, and dnase i  in pbs, then labeled with biotinylated anti-brdu  or biotinylated anti-igg control , and subsequently stained with streptavidin-apc. the mean brdu labeling at  <dig> days was used to calculate the labeling rate per day;  <dig> % and  <dig> % per day in ko and wt mice, respectively. to calculate average life span, the mean values were plotted as a function of time in the chase period, and linear regression analysis was performed to calculate time to lose the brdu labeled population, assuming a constant pool size during the analysis.\n",
            "\n",
            "b cell isolation, ca2+ mobilization response, in vitro proliferation and survival assays\n",
            "splenic b cells were purified with the macs cd45r microbead system and were verified by facs to be ≥ 98% b220+. splenic b cells or sorted mz b cells were loaded with  <dig> μm fluo-4/am , stimulated with  <dig> μg/ml f <dig> goat anti-igm ab and ca2+ mobilization was recorded on live gated cells. for proliferation assays,  <dig> ×  <dig> mz b cells were plated in triplicate in  <dig> well plates and stimulated with  <dig> μg/ml f <dig> goat anti-igm ab or with  <dig> μg/ml lps. for the last  <dig> h of the  <dig> h culture period the cultures were pulsed with  <dig> μci/well -thymidine. incorporated radioactivity was quantified by scintillation counting. for cd <dig> up-regulation studies, cd <dig> expression was analyzed by flow cytometry before and after  <dig> h treatment with f <dig> goat anti-igm ab . for cell viability assays, sorted fo b cells were cultured as stated and then analyzed by flow cytometry after the addition of 7-amino-actinomycin  .\n",
            "\n",
            "quantitative real-time rt-pcr\n",
            "facs-sorted splenic b220+cd21intcd24lo fo b cells were homogenized in trizol  and mrna was extracted according to manufacturer's instructions. bcl- <dig>  a <dig>  and bcl-xl mrna levels were determined using the roche lightcycler system. ct values were used to calculate relative values and were normalized to ct values for cyclophilin a. the following primer pairs were used:\n",
            "\n",
            "bcl- <dig>  cctgtggatgactgagtacc and gagacagccaggagaaat\n",
            "\n",
            "a <dig>  cttcagtatgtgctacaggtacccg and tggaaacttgtttgtaagcacgtccat\n",
            "\n",
            "bcl-xl, agaagaaactgaagcagag and tccgactcaccaatacctgcgtccat\n",
            "\n",
            "baff, ctgtggtcacttactccaaagg and ggatcagattcaacgggtcacg\n",
            "\n",
            "baff-r, gcccagactcggaactgtccca and gcccagtagagatccctgggttcc\n",
            "\n",
            "cypa, tatctgcactgctaagactgaatg and cttcttgctggtcttgccattcc.\n",
            "\n",
            "immunoblot analysis\n",
            "immunoblot analysis was performed according to standard protocols using ecl detection . polyclonal antibodies to auf <dig>  and to eif4e  were used.\n",
            "\n",
            "humoral response assay\n",
            "total pre-immune serum and tnp-specific titers were determined using the sba clonotyping elisa kit from southernbiotech according to manufacturer instructions. to evaluate ti-ii responses, mice were immunized with  <dig> μg tnp-ficoll  and bled  <dig> days post immunization. to evaluate td responses, mice were immunized with  <dig> μg tnp-klh  with adjuvant  and bled  <dig> days post immunization. for in vitro studies,  <dig> purified b cells were stimulated with murine il- <dig>  and lps  for  <dig> h. immunoglobulin secretion in the supernatant was measured as described in immunization studies. rt-pcr looking for post-switch transcripts was performed as previously described  <cit> .\n",
            "\n",
            "statistical analysis\n",
            "data are presented as mean ± standard deviation. for statistical comparison of two samples, the two-tailed student t-test was used for evaluation.\n",
            "\n",
            "RESULTS\n",
            "reduction of fo b cells in spleens of auf1-deficient mice\n",
            "auf1-/- mice were found to have  <dig> -fold smaller spleens  and a roughly two-fold reduction in the number of splenocytes as compared to wild-type controls . splenocyte reduction was restricted to b and t lymphocytes, as the number of f4/80+ macrophages, cd94+cd3- natural killer, and cd16/32+b220- cell populations in the spleens of auf1-/- mice were unaffected . except for a decrease in mature igmloigdhi b cells, auf1-/- mice exhibited no significant differences in the cell populations of inguinal and mesenteric lymph nodes, thymus, bone marrow or peritoneal cells , as compared to wild-type littermates. the unaltered cell subpopulations of the bone marrow and thymus in auf1-/- mice suggest that the reduction in splenocytes in these mice occurred at the periphery within the spleen rather than during development.\n",
            "\n",
            "cd3+\n",
            "cell numbers are stated at ×  <dig>  values represent the mean ± sd. for b cell subsets, the percentage of each subset in b220+ gated cells is given in parenthesis. auf1-deficient and wild-type mice used for this analysis were 11- <dig> weeks of age. nd; not determined.\n",
            "\n",
            "we therefore determined whether the splenocyte loss was restricted to a defined lymphocyte population. the different immature and mature splenic b cell subsets were identified based on the expression of multiple surface markers: cd <dig>  cd <dig>  cd <dig>  igm, and igd   <cit> . the analyses presented were performed on mice 11- <dig> weeks of age, however, older mice analyzed  showed similar findings to younger mice. this analysis showed that the sharp reduction in splenic b lymphocytes was due predominately to a two-fold reduction in the number of fo b cells . the absolute numbers of transitional  and mz b cells were unchanged in auf1-/- mice . furthermore, analysis of post-transitional precursor subsets, mz precursor  b cells and fo-ii b cells, showed no differences in absolute numbers . as a result of the decrease in total splenocytes, there was an increase in the percentage of transitional b cells  and mz b cells  whereas, in contrast, there was a decrease in the proportion of mature fo b cells . as expected with a decrease in splenic fo b cells, there was a decrease in recirculating mature igmloigdhi b cells in the bone marrow and lymph node of auf1-deficient mice . an analysis of the third mature b cell population, cd5hiigmhi b- <dig> cells in the peritoneum, showed no differences between wild type and aufi-/- animals . all surface markers analyzed were expressed normally in auf1-deficient splenic b cells, with the exception of cd <dig>  which exhibited reduced expression . flow cytometric analysis of cd <dig>  cd <dig>  cd <dig>  and cd62l surface markers of t cells in the spleen showed no differences , suggesting that the nearly two-fold reduction in splenic t cells was not restricted to a defined t cell population.\n",
            "\n",
            "given the specific loss of fo b cells, we examined the expression of auf <dig> in different b cell subsets. surprisingly, flow cytometric analysis demonstrated that auf <dig> expression was greatest in wild type mz and t <dig> b cells, two populations whose absolute numbers were unaffected in auf1-/- mice . auf <dig> consists of a family of four protein isoforms that are translated from an alternatively spliced mrna  <cit> . thus, we sorted b cell subsets and then analyzed auf <dig> expression by immunoblot analysis to examine individual isoform expression. consistent with the flow cytometery results, immunoblot analysis demonstrated that auf <dig> was most strongly expressed in t <dig> b cells and mz b cells. differences in isoform expression were evident with t <dig> b cells exhibiting a relative increase in the expression of the p <dig> isoform. surprisingly, fo b cells, which were significantly reduced in auf1-deficient mice, expressed auf <dig> at lower levels than other b cell subsets. these findings suggest that a fo b cell-independent process may be responsible for increased loss, such as through regulation of survival factors. nevertheless, we cannot exclude the possible importance of an altered ratio of aufi isoform expression, as previously suggested  <cit> .\n",
            "\n",
            "loss of fo b cells is due to absence of auf <dig> expression in the hematopoietic lineage\n",
            "to determine whether the differences observed in splenic b cell subsets in auf1-/- mice were dependent on the expression of auf <dig> in hematopoeitic cells, we performed bone marrow adoptive transfer studies. an equal number of bone marrow cells from auf1-deficient or wild-type mice were transferred into sub-lethally irradiated rag1-/- mice, which are devoid of mature b and t lymphocytes. ten weeks after cell transfer, the rag1-/- recipient mice of auf1-/- bone marrow, similar to auf1-/- mice, exhibited a greater than two-fold reduction in number of splenic fo b cells and a sharp decrease in the proportion of fo b cells  as compared to rag1-/- recipient mice of wild-type bone marrow . similarly, irradiated wild-type mice reconstituted with auf1-/- bone marrow exhibited a reduction in the proportion of fo b cells  as compared to irradiated auf1-/- mice reconstituted with wild-type bone marrow . irradiated rag1-/- or wild-type recipients of auf1-/- bone marrow, similar to auf1-/- mice, showed a reduction in cd <dig> expression . these data demonstrate that changes in the frequency of mature b cells and decreased cd <dig> expression in auf1-/- mice are intrinsic to the hematopoeitic lineage.\n",
            "\n",
            "decreased number of splenocytes in adult auf1-/- mice is not rescued by transfer of wild-type bone marrow\n",
            "auf1-/- mice have much smaller spleens that support roughly half the number of splenocytes as compared to wild-type littermates . to investigate if the decreased cellularity was dependent on auf <dig> expression in the hematopoeitic population, bone marrow adoptive transfer experiments were performed. surprisingly, irradiation of adult auf1-/- mice and reconstitution with wild-type bone marrow did not rescue the reduction in splenic lymphocyte cellularity . most simply these adoptive transfer studies suggest that the signals required for determining the number of splenic lymphocytes is likely provided by radiation-resistant stromal cells. however, we cannot rule out the possibility that these signals may be provided by the hematopoeitic lineage in a developmentally-fixed manner, similar to the mechanism described for the lymphotoxin signal needed in proper splenic organization  <cit> . thus, these bone marrow transfer studies suggest that auf <dig> influences two separate processes, since the transfer of wild-type bone marrow cells into auf1-/- recipients corrects the proportion of fo and mz b cells, but does not rescue the decreased number of splenocytes . similarly, transfer of auf1-/- bone marrow cells into wild-type recipients results in a decreased proportion of fo b cells and increased proportion of mz b cells, but does not alter the absolute number of splenocytes. thus, we favor the hypothesis that auf <dig> influences two separate processes: maintenance of the proportion of specific subsets of the mature b cell population. a function that is autonomous to auf <dig> expression; and influencing the overall number of splenocytes, which is not autonomous to auf <dig> expression in the hematopoeitic lineage.\n",
            "\n",
            "increased turnover of fo b cells in auf1-deficient mice\n",
            "mutations affecting either the production or survival of mature b cells can influence the number of b cells in the spleen  <cit> . therefore, we next addressed whether the decrease in fo b cells in auf1-/- mice results from a defect in cell production and/or survival. examination of b cell follicles in the spleens of auf1-deficient mice revealed many pyknotic and fragmented cells characteristic of cells undergoing apoptosis . the striking increase in the presence of apoptotic cells within the splenic follicles of auf1-/- mice was confirmed by cleaved caspase- <dig> staining . similarly, rag1-/- mice reconstituted with auf1-/- bone marrow exhibited pronounced apoptosis in splenic follicles , a finding not seen in rag1-/- mice reconstituted with wild-type bone marrow. brdu pulse-chase studies were performed to directly measure the turnover of fo b cells. mice were continuously fed brdu for  <dig> days and the proportion of brdu-positive splenic fo b cells  was determined by flow cytometery at  <dig>   <dig>  or  <dig> days following brdu administration. at the end of the pulse period, a significantly greater proportion of fo b cells were labeled in auf1-/- mice as compared to wild-type littermates . the percentage of brdu+ fo b cells stayed fairly stable for wild-type control mice during the chase period, but noticeably declined in auf1-/- mice , demonstrating a two-fold decrease in the average half-life of auf1-/- b cells . in contrast, brdu pulse-chase studies demonstrated no differences in the labeling or turnover of mz b cells in auf1-/- mice . it is well-established that the expression of bcl- <dig> anti-apoptotic and pro-apoptotic family members is vital for the maintenance of b cells within the spleen  <cit> . therefore, we assessed the levels of bcl- <dig>  a <dig>  bcl-xl, and bim mrna in freshly sorted fo b cells from auf1-/- and wild-type mice. auf1-deficient fo b cells exhibited a significant decrease in all three anti-apoptotic factors , with a 40% reduction in expression of bcl- <dig> and bcl-xl mrnas and 65% reduction in a <dig> mrna levels. however, no significant difference was seen in bim mrna levels in fo b cells from auf1/- and wild-type mice. we further validated these results by comparing bcl- <dig> protein levels in mature splenic b cell populations of auf1-/- mice by flow cytometry. this analysis was restricted to bcl- <dig> because it was not possible to measure a <dig> by flow cytometry. consistent with a 40% decrease in bcl- <dig> mrna expression, the level of bcl- <dig> protein expression was similarly decreased in fo b cells from auf1-/- mice . in contrast, mz b cells from auf1-/- mice exhibited no difference in bcl- <dig> expression as compared to mz b cells from wild-type littermates . we further characterized fo b cell turnover, by analyzing cell death in cultured cells via 7-aad analysis. we purified fo b cells based on cd <dig> and cd <dig> markers, because purification with igm and igd could elicit bcr signals that influence b cell survival. fo b  cells lacking auf <dig> were more susceptible to spontaneous cell death in the first  <dig> hours in culture . cd <dig> engagement has been shown to be important in the up-regulation of bcl- <dig> family members  <cit> . thus, fo b cells were stimulated in culture by cd <dig> engagement to assess the ability to express bcl- <dig> family members. auf1-deficient fo b cells showed no differences in viability or in ability to express anti-apoptotic bcl- <dig> family members upon cd <dig> engagement . these data indicate that auf <dig> deficiency leads to a decrease in fo b cell survival, presumably by impairing the expression of key survival factors bcl- <dig>  bcl-xl, and a <dig> in absence of cd <dig> engagement.\n",
            "\n",
            "no defect in bcr signaling, in vitro maturation, or baff-r expression in auf1-/- b cells\n",
            "to directly determine whether auf <dig> plays a role in the development of fo b cells, we assayed the in vitro maturation of t <dig> b to fo b cells, as previously described  <cit> . bcr engagement on t <dig> b cells drives their differentiation to a mature b cell phenotype, as characterized by down-regulation of cd <dig> and cd <dig> surface expression. b220+cd21hicd24hi t <dig> splenocytes were sorted and cultured in the presence or absence of bcr stimulation and analyzed after  <dig> h . consistent with the brdu studies, auf1-/- t <dig> b cells exhibited no defect in maturation to a mature b cell phenotype in vitro , suggesting that the bcr signaling pathway mediating this transition is unaffected by the absence of auf <dig>  auf1-/- b cells showed no differences in the duration or amplitude of ca2+ flux , cell size increase , and up-regulation of the activation marker cd <dig>  post-bcr ligation, as compared to wild-type b cells. these results therefore indicate that the proximal events of bcr-mediated signaling, shown to be vital in b cell maturation and survival  <cit> , are unaffected in auf1-/- b cells. baff, a member of the tnf superfamily, and its major receptor baff-r have been shown to be critical in supporting transitional b cell maturation and enhancing the survival of mature b cells  <cit> . no differences were observed in the mrna levels of baff-r in sorted fo auf1-/- b cells or in baff mrna in auf1-/- spleens as compared to wild-type controls .\n",
            "\n",
            "decreased t cell-independent humoral response in auf1-/- mice\n",
            "to determine the functional consequence of auf <dig> deficiency, we next measured the ability of auf1-/- mice to mount a normal humoral immune response to antigenic challenge. ti antigen, tnp-ficoll, or td antigen, tnp-klh, was injected intraperitoneally into auf1-/- and wild-type mice. serum levels of tnp-specific antibodies were measured  <dig> and  <dig> days post-immunization with tnp-ficoll and tnp-klh, respectively. in the absence of antigenic challenge, we observed no differences in basal serum ig levels between auf1-/- and wild-type mice . consistent with this finding, we observed no differences in architecture or number of germinal centers  or follicular dendritic network  in spleens of auf1-/- mice. however, auf1-/- spleens showed distortion of distinct b and t cell zones, as b cells are seen in pals, a t cell area largely devoid of b cells in spleens from wild-type mice . auf1-/- mice mounted a robust td humoral response to immunization with tnp-klh, consistent with the presence of very large germinal centers in the center of b cell follicles, where t cells normally reside . given the decrease in the number of fo b cells, one might have expected attenuation rather than augmentation of the td humoral response. this discrepancy may stem from organizational defects that lead to aberrant germinal centers in the pals in auf1-/- mice. it may also reflect the increased t cell response as a result of aberrant control of il- <dig> in auf1-/- mice  <cit> . in contrast, auf1-/- mice showed a decreased humoral response to a ti antigen, tnp-ficoll . given the importance of mz b cells in an effective ti humoral response  <cit> , we cannot rule out that auf <dig> is important for proper mz b cell function, especially given the high relative expression of auf <dig> in this population . this is supported by attenuation of cell size increase and attenuated proliferative response to bcr-ligation and lps in auf1-deficient mz b cells as compared to wild-type controls . there was no difference in calcium flux post-bcr ligation in mzb cells , which suggests that the impact of auf <dig> on mz b cell function is downstream of calcium mobilization.\n",
            "\n",
            "it has been suggested that as part of the lr <dig> transcription complex, auf <dig> may play a role in heavy chain class switch recombination   <cit> . isotype csr requires the transcription of class-specific mrnas  <cit> . rt-pcr directed at amplifying post-switch targets composed of the iμ exon spliced onto the 5' exon of the cγ <dig> and cγ2b gene, loci reported to bind lr <dig> with the highest affinity  <cit> , was performed on purified cultured b cells  <dig> days post-activation. auf1-/- b cells did not display any defect in class-specific transcripts . furthermore, auf1-/- b cells cultured in the presence of il- <dig> and either lps or an antibody to cd <dig> exhibited no defect in the in vitro production of serum igs , indicating that auf <dig> does not play a direct role in csr activity in b cells. however, we cannot rule out that auf <dig> may play an indirect role in proper humoral responses in vivo through the regulation of key cytokines important in csr and immunoglobulin production.\n",
            "\n",
            "discussion\n",
            "our results show that auf <dig> deficiency leads to a disturbance in mature splenic b cell populations and in humoral immune responses. past studies of mice with defective bcr signaling have demonstrated that the strength of the bcr signal is important in both maturation and determination of the cell-fate of the immature b cells  <cit> . furthermore, these studies collectively indicate that weak bcr signals favor mz b cell development, whereas relatively strong signals favor the development of fo b cells, and perhaps even stronger signals favor the generation of b <dig> cells  <cit> . our results indicate that the reduction of b cell numbers is specific to fo b cells, as normal numbers of t <dig>  t <dig>  fo-ii, mzp, and mz b cells are seen in spleens of auf1-/- mice, as well as in irradiated rag1-/- mice reconstituted with auf1-/- bone marrow. these data suggest that there is no developmental block, nor are transitional b cells diverted to the mature mz b cell population at the expense of fo b cells, in contrast to mice with impaired bcr signaling  <cit> . in agreement with this hypothesis, there is no impairment in production of fo b cells in vitro as shown by maturation assays, or in vivo as demonstrated by brdu labeling studies. overall, we conclude that development of fo b cells is unaffected in auf1-/- mice. in addition to its role in development, signals processed by the bcr complex are required for mature b cells to survive in the periphery. the specific loss of fo b cells in auf1-/- mice does not seem to result from abnormal bcr signaling, as downstream events of bcr engagement, such as calcium mobilization are unaffected in auf1-/- b cells.\n",
            "\n",
            "the phenotype seen in auf1-/- mice does not seem to involve baff signaling, which has been shown to be important in both b cell development and homeostasis  <cit> . baff, a member of the tnf superfamily, and its receptors, br <dig> and taci, are not encoded by are-mrnas and are therefore unlikely to be direct targets of auf <dig>  as predicted, no difference was observed in the expression of baff or its receptor in auf1-deficient mice. furthermore, the phenotype of baff-deficient mice differs from that of auf1-/- mice. baff-deficient mice exhibit a loss of both mz and fo b cells, as well as severely reduced antibody titers in response to both td and ti antigens  <cit> . in contrast, auf1-/- mice exhibit a selective loss of the fo b cell subset and a moderate attenuation of the ti humoral response. baff signaling is important in cd <dig> expression in b cells independent of its role in cell survival  <cit> , and its regulation of cd <dig> expression is less clear. there are conflicting reports on the role of baff on cd <dig> expression from baff-deficient, baff-r-deficient, and transgenic baff mice  <cit> . in contrast, auf1-deficient b cells show normal cd <dig> surface expression, but exhibit a reduction in cd <dig> surface expression . although direct functions of auf <dig> cannot be ruled out, alterations in local factors such as interleukin- <dig> in part may explain the decrease in cd <dig> expression in auf1-/- b cells  <cit> . cd <dig> expression on b cells is involved in antigen presentation and ige response, as demonstrated by the enhanced ige response elicited in cd23-deficent mice  <cit> . we have previously shown that auf1-/- mice have increased circulating ige levels  <cit>  that may be due to the reduced cd <dig> expression on auf1-deficient b cells.\n",
            "\n",
            "in the periphery, a lymphocyte acquires survival signals through receptors for cytokines, antigens, and hormones  <cit> . failure to adequately acquire such signals results in cell death and thus determines the size of the peripheral lymphocyte pools  <cit> . although signaling through the bcr is required for the maintenance of mature splenic b cells  <cit> , it is not the only requirement. the splenic microenvironment also plays an essential role in the maintenance of mature b cells  <cit> . however, knowledge regarding the nature and regulation of the splenic microenvironment is limited, particularly as it pertains to the accumulation and maintenance of splenic lymphocytes. our work suggests that auf <dig> is important in the regulation of survival signals needed for the maintenance of fo b cells within the spleen, as indicated by decreased bcl- <dig>  a <dig>  and bcl-xl levels. spleens from auf1-/- mice bear a strong resemblance to those of bcl-2-deficient mice in that they are smaller, show a loss of mature b and t cells, and show many apoptotic cells  <cit> . mature recirculating igmloigdhi fo b cells from lymph nodes of auf1-/- mice exhibit no significant differences in bcl- <dig> expression , suggesting that the survival signal provided in lymph nodes is unaffected by the absence of auf <dig>  these findings highlight differences in survival signals necessary for recirculating mature b cells in different lymphoid organs. in support of this hypothesis, it has been recently shown that macrophage migration inhibitory factor , through the activation of the cd74-cd <dig> complex, is necessary in the maintenance of mature b cells in the bone marrow, but not in the spleen or lymph nodes  <cit> . the spleen-specific alterations in lymphocyte populations in auf1-/- mice suggests that auf <dig> acts in regulating the local microenvironment, as opposed to regulating global factors in lymphocyte development, such as bcr signaling that appears normal in auf1-/- b lymphocytes. future studies with a more comprehensive screening approach, such as cdna micro-arrays, may prove valuable in identifying mrnas regulated by auf <dig> that play a role in fo b cell lymphocyte survival within the spleen. cd <dig> engagement on mature b cells results in induction of anti-apoptotic bcl- <dig> family members, which protect these cells from antigen receptor-mediated apoptosis  <cit>  and are required for gc formation, progression  <cit>  and td response  <cit> . auf1-/- fo b cells were able to appropriately induce the expression of bcl- <dig> and a <dig> in response to cd <dig> engagement, which supports their ability to form germinal centers in auf1-deficient mice.\n",
            "\n",
            "an important phenotype of auf1-deficiency is a decrease in the number of splenic lymphocytes and a skewing of the b cell subpopulations, with an over-representation of immature and mz b cells and a decrease in fo b cells. there are several possible mechanisms that may account for this complex phenotype. auf <dig> may regulate a survival signal needed for the maintenance of fo b cells, the largest component of splenic b cells, and loss of this population could lead to a decrease in the accumulation of t cells within the spleen. previous studies on b cell-deficient mice have reported a decrease in splenic t cell numbers  <cit> . bcr-deficient mice exhibit a three-fold reduction in the number of splenic cd <dig> and cd <dig> t cells, although no loss of t cells was seen in lymph nodes from these animals  <cit> . these results suggest that b cells provide signals that promote cd <dig> and cd <dig> t cell accumulation and/or survival in the spleen  <cit> . notably, cd3ε- and cd4-deficient mice show a 40% reduction of splenic mature b cells, suggesting that t cells may also impact the survival of b cells within the spleen  <cit> . likewise, auf <dig> may regulate survival signals required independently for the maintenance of fo b and t cells. at this time we cannot distinguish whether the loss of t and b cells within the spleen are related, or whether both populations, independent of each other, are deficient in survival signals, resulting in decreased bcl- <dig> expression in both populations in auf1-/- mice . these mechanisms predict that the 50% decrease in splenic b cells in auf1-/- mice results from increased fo b cell turnover that outpaces the rate of fo b cell production by two-fold. however, brdu studies indicate that there is a  <dig> -fold decrease in splenic fo b cell average half-life and a  <dig> -fold increase of b cell production in auf1-/- mice. this would result in a 30% decrease in fo b cells and not the 50% decrease observed in auf1-/- mice. furthermore, if turnover did outpace production by two-fold, one would predict a greater decrease than the 20% seen in recirculating mature b cells in lymph nodes and bone marrow of auf1-/- mice.\n",
            "\n",
            "the data from bone marrow adoptive transfer studies more likely indicate that two separate processes are involved in the observed phenotype:  auf <dig> is important in the maintenance of fo b cells within the spleen, most likely through regulation of survival factors; and  auf <dig> impacts the number of lymphocytes within the spleen through the regulation of yet unknown signals. adoptive transfer studies support the involvement of two separate processes, as reconstitution of irradiated auf1-/- mice with wild-type bone marrow corrects the shifts in b cell subpopulation proportions but not the overall number of splenocytes. the involvement of two separate processes also better explains the deficit in fo b cells. the 40% decrease in the number of splenocytes, in addition to the increased fo b cell turnover that would result in a 30% decrease in the proportion of fo b cells, more accurately approximates the 50% decrease in the number of fo b cells observed in auf1-/- mice.\n",
            "\n",
            "the complex phenotype seen in auf1-deficient mice is not surprising, given that auf <dig> regulates the expression of numerous immune-modulating cytokines and chemokines  <cit> . although we have focused here on defects in the b cell compartment in auf1-deficient mice, we have previously shown defects in the t cell and macrophage compartments  <cit> . moreover, despite the fact that the fdc network can be visualized, we cannot exclude that the dendritic subset may be affected in auf1-deficient mice and may impact the function, size, and location of other immune cell compartments. tristetraprolin, another are-binding protein, binds and regulates the expression of mrnas encoding key regulators of human dendritic cell maturation  <cit> . future analysis of specific targets of auf <dig> in different immune cell subsets will be important in dissecting the role of auf <dig> on individual populations leading to the overall complex phenotype seen in its deficiency in the mouse model.\n",
            "\n",
            "CONCLUSIONS\n",
            "this work demonstrates that auf <dig> is important in determining the size of the splenic lymphocyte population and proper survival of fo b cells within the spleen. future studies will address the identification and regulation of auf <dig> targets as they pertain to splenic size, fo b cell maintenance, and humoral immune response.\n",
            "\n",
            "abbreviations\n",
            "7-aad: 7-amino-actinomycin d; are: adenosine/uridine-rich element; bcr: b cell receptor; baff: b cell activating factor; fo: follicular; gc: germinal center; mfi: mean fluorescence intensity; mz: marginal zone; pals: periarteriolar lymphoid sheath; t1: transitional type-1; t2: transitional type-2; ti: t-cell independent; td: t-cell dependent.\n",
            "\n",
            "authors' contributions\n",
            "ns made substantial contributions to conception and design of the study, acquired, analyzed, and interpreted the data, and drafted the manuscript. rjs contributed in data analysis and writing of the manuscript. jl and mb made contributions to the study design and acquisition of data. all authors read and approved the final manuscript.\n",
            "\n",
            "authors' information\n",
            "the authors declare no competing financial interests. correspondence and requests for materials should be addressed to r.j.s. .\n",
            "\n",
            "supplementary material\n",
            "additional file 1\n",
            "bcl- <dig> expression in t and fo b cells from spleens and lymph nodes from auf1-/- mice. intracellular bcl- <dig> protein levels in fo b, cd4+ t, and cd8+ t cells from spleens and lymph nodes were determined by four-color flow cytometric analysis. histograms depict the intensity of wild-type  and auf1-/-  cells stained with bcl- <dig> antibody or wild-type stained with isotype-matched control antibody . bcl- <dig> expression was quantified. the normalized values  for bcl- <dig> - mfi of isotype-matched control) for auf1-/- cells were plotted as a percentage of wild-type values. plot represents mean ± sd for four mice in each group. * p <  <dig> .\n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements\n",
            "we thank dr. doris tse at the center for aids research at nyu school of medicine for assistance in facs analysis, and the nyu histology core facilities for assistance in preparing slides. we thank drs. michael dustin, dan littman, and jane skok for their insights in the preparation of this manuscript. this work was supported by a grant from the nih  and an nih t <dig> training grant .\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "auf1-deficient mouse showed a 40% decrease in the number of splenic b cells, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germinal centers in the spleen, consistent with the presence of very large germin\n",
            "46\n",
            "[{'summary_text': 'furthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\\nthe expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\\nfurthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\\nfurthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\\nfurthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\\nfurthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\\nfurthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\\nin this study, dna-microarrays and real-time rt-pcr analysis were carried out to characterize the transcriptional differences of s. mutans in various biofilm depths.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "in nature, most microorganisms possess the ability to attach to solid surfaces and to develop a densely packed community – a biofilm  <cit> . in the biofilm ecological niche, bacteria exhibit increased resistance to antimicrobial compounds, environmental stresses and host immune defense mechanisms  <cit> . the environmental heterogeneity that develops in biofilms can accelerate phenotypic and genotypic diversity in bacterial populations that allows them to accomodate adverse ecological conditions within the biofilm. oral biofilms harboring pathogenic bacteria are among the major virulent factors associated with dental diseases such as caries, gingivitis and periodontal diseases  <cit> . streptococci, including mutans streptococci, are ubiquitous in the oral microbiota of humans. s. mutans is considered to be the most important etiological agent in dental caries. it forms biofilms on tooth surfaces and causes dissolution of enamel by acid end-products resulting from carbohydrate metabolism  <cit> .\n",
            "\n",
            "the dental biofilm is the net result of a community of bacteria cooperating to form well-differentiated structures with distinct thickness  <cit> . cells in the biofilm have unique phenotypic characteristics, which are different from their planktonic counterparts  <cit> , accompanied by significant changes in their patterns of gene expression  <cit> . our previous, in vitro comparative transcriptome analysis confirmed the hypothesis that there are significant changes in the pattern of gene expression following the transition of bacteria to biofilm growth modes  <cit> . in this study, dna-microarrays and real-time rt-pcr analysis were carried out to characterize the transcriptional differences of this bacterium in various biofilm depths.\n",
            "\n",
            "RESULTS\n",
            "viability and growth of bacteria within the biofilm is influenced by numerous environmental factors, such as nutrition supply, outflux of metabolites, ph gradient and oxygen tension  <cit> . lack of nutrition and accumulation of toxic products may account for a decline in the growth of bacteria in suspension and in biofilm. one of the major rate-controlling factors in a biofilm ecosystem is the diffusion rate of nutrients in and waste products out and gases across the biofilm  <cit> . a thinner, or a low-density biofilm with void volumes and water channels may facilitate easier transport of nutrients in and waste products out across the biofilm, thus having a limited effect on bacterial proliferation  <cit> . it is conceivable therefore that the gene expression profile in thicker biofilms may be attributed to the effects of nutrient limitation to a large portion of the biomass due to diffusion restrictions. as biofilm thickness plays a paramount role, we conducted this study under controlled nutrient flow and controlled biofilm depth of  <dig>   <dig> and  <dig> microns, by using the constant-depth film fermenter . the biofilms cultivated in the cdff were assessed by confocal laser scanning microscopy . the clsm images demonstrated that biofilm maturation and increased thickness is accompanied by significant alterations in cell viability in the different biofilm layers. according to our results, comparative vitality of bacteria grown at 100-micron depth was much greater than those cultured in cdff at 200- or 400-microns depths . this decrease in viability might be due to restriction in nutrient availability and accumulation of toxic metabolites as the biofilm thickens. therefore it is reasonable that differential gene expression will allow the bacteria to acclimate in different ecological microenvironments.\n",
            "\n",
            "recent studies have revealed that the regulated death of bacterial cells is important for biofilm development. following cell death, a subpopulation of the dead bacteria lyses and releases genomic dna, which then has a central role in intercellular adhesion and biofilm stability  <cit> . cell fate is generally thought of as being deterministic. that is, the fate that cells adopt is in most cases governed by the history of the cell or its proximity to inductive signals from other cells. however, an increasing number of cases are now known in which cell fate is controlled by stochastic  mechanisms, for instance entry into the persister state in e. coli  <cit> . stochastic fluctuations in the cellular components that determine cellular states can cause two distinct subpopulations, a property called bistability. bistable switches produce polarized expression states and result in subpopulations of specialized cell types that are either on or off for gene expression. polarized output is thought to arise when a transcriptional regulator reaches a threshold activity and initiates an auto-regulatory positive feedback loop  <cit> . below the threshold, the regulator does not auto-activate and the system remains in the off state. above the threshold, runaway positive feedback results in activation of the system and stabilization of the on state. several properties in b. subtilis, the phenotypes of motility, genetic competence, sporulation and biofilm formation, are regulated by bistable gene expression  <cit> . it has been also demonstrated  <cit>  that the genes encoding the components of the extracellular matrix in b. subtilis are expressed only in a subpopulation of cells. matrix production is energetically costly and choosing the subset of cells within a given population to be responsible for producing and providing the extracellular polysaccharides  for the entire biofilm community might be especially economical. in the context of our results, the heterogeneity in the viable population of bacteria within the deep layers of a mature biofilm might be a result of a strategy to relegate the energetic cost to a subpopulation that would provide strengthening for the entire community. phenotypic diversity within bacterial populations is advantageous. multiple cell types permit a \"division of labor\", and cells with specialized properties may be primed to exploit or resist changes in the environment  <cit> . consequently, biofilms should contain specialized cell types that are differentiated by transcriptional regulation, to express different phenotypic characteristics.\n",
            "\n",
            "dna-microarray analysis was used to identify differentially expressed genes associated with the thickness of s. mutans biofilm at various depths. analyses of microarrays images, using an empirical bayesian method   <cit> , indicated that expression of  <dig> genes was differentially changed in 400- vs. 100-microns depth biofilms , and  <dig> genes in 200- vs. 100-microns depth biofilms , at a confidence level of p ≤  <dig>  tested by moderated t-test. about 11% of these genes code for membrane-related proteins, while almost one-half of the differentially regulated genes code for hypothetical proteins of as yet unknown function present in the s. mutans genome. only  <dig> genes of s. mutans  showed differential expression in both 400- vs. 100-microns and 200- vs. 100-microns biofilms. all of these genes were upregulated. verification of the microarrays data was carried out using real-time rt-pcr for expression analysis of selected differentially regulated genes . a substantial number of them, such as smu.574c, smu. <dig>  and smu <dig>  appeared to code for cell wall-associated proteins. smu. <dig> encodes a cell wall-associated protein precursor wapa, a major surface protein  <cit> , which modulates adherence and biofilm formation in s. mutans. previous studies have demonstrated that levels of wapa in s. mutans were significantly increased in biofilm phase  <cit> , whereas the inactivation of wapa resulted in reduction in cell aggregation and adhesion to smooth surfaces  <cit> . the wapa mutants are of reduced cell chain length, have less sticky cell surface, and unstructured biofilm architecture compared to the wild-type  <cit> .\n",
            "\n",
            "a based on the genome annotation of s. mutans provided by tigr.\n",
            "\n",
            "b log2-fold expression according to the m value, m >  <dig> means upregulation and m <  <dig> downregulation of the gene.\n",
            "\n",
            "c moderated t-test and its corresponding p value, adjusted by benjamini and yekutiely method.\n",
            "\n",
            "d bayesian test value, meaning the probability for a gene to be differentially expressed. genes are listed according to their decreasing statistical importance according to their b-value.\n",
            "\n",
            "a based on the genome annotation of s. mutans provided by tigr.\n",
            "\n",
            "b log2-fold expression according to the m value, m >  <dig> means upregulation and m <  <dig> downregulation of the gene.\n",
            "\n",
            "c moderated t-test and its corresponding p value, adjusted by benjamini and yekutiely method.\n",
            "\n",
            "d bayesian test value, meaning the probability for a gene to be differentially expressed. genes are listed according to their decreasing statistical importance according to their b-value.\n",
            "\n",
            "a based on the genome annotation of s. mutans provided by tigr\n",
            "\n",
            "smu. <dig>  encoding membrane-associated receptor protein ftsy, the third universally conserved element of the signal recognition particle  translocation pathway  <cit> , was also found among these differentially expressed genes. srp was first identified in mammalian cells, and later in bacteria, and it was further shown that components of the srp pathway are universally conserved in all three domains of life – archaea, bacteria, and eukaryote domains  <cit> . the srp pathway delivers membrane and secretory proteins to the cytoplasmic membrane or endoplasmic reticulum  <cit> . s. mutans remained viable but physiologically impaired and sensitive to environmental stress when ftsy and other genes of the srp elements were inactivated  <cit> . the upregulation of ftsy in the deep layers of the biofilm indicates that the srp system is crucial for bacterial survival in highly condensed mature biofilm.\n",
            "\n",
            "interestingly, relatively few of the differentially expressed genes showed more than a 2-fold change. however, even small changes in mrna levels could have the biological potential to affect bacterial metabolism and physiology. relatively small changes in the level of expression of one gene can be amplified through regulatory networks and result in significant phenotypic alteration  <cit> . many genes among those upregulated in 200- vs. 100-microns biofilms are involved in energy metabolism, including smu. <dig>  smu. <dig>  smu. <dig>  smu. <dig> and smu. <dig>  however, a substantial number – 11% of the upregulated genes – code for cell wall-associated proteins, such as smu.574c and smu. <dig>  several differentially expressed genes in the 400- vs. 100-microns depths are presumably regulatory genes, e.g. smu. <dig> and smu. <dig>  which influence biofilm formation.\n",
            "\n",
            "previous studies have shown that carbohydrates have a major influence on biofilm-associated gene expression of several types of bacteria  <cit> . sucrose-dependent adhesion is a major mechanism of biofilm formation and maturation in dental biofilms  <cit> . in order to evaluate whether sucrose influences the expression of selected genes of s. mutans, we generated biofilms of 100-, 200- and 400-micron depths in the presence of 2% sucrose. it is of interest that the presence of sucrose did not dramatically change the regulation of the selected genes in 400- vs. 100-microns  or 200- vs. 100-microns biofilms  tested by real-time rt-pcr. only smu. <dig> of the tested genes showed significant upregulation in 400- vs. 100-microns depth of biofilm with sucrose . a possible explanation of these results could be that sucrose affects mostly the initial stages of biofilm formation, as generation of the sticky glucans and fructans is especially crucial in the initial stages of adhesion  <cit> , while the biofilm thickness-associated genes are activated mostly in the late steps of biofilm development. the genes, which appear to be responsible for biofilm thickness and maturation processes, are not necessarily influenced by the presence of sucrose during the initial biofilm formation stage.\n",
            "\n",
            "cell-cell communication plays an important role in the successful formation, survival and virulence of the biofilm community  <cit> . gram-positive bacteria generally communicate via small diffusible peptides  <cit> , while many gram-negative bacteria secrete acyl homoserine lactones   <cit> , the structure of which varies depending on the species of bacteria that produce them. another system associated with quorum sensing  involves the synthesis of autoinducer- <dig> , which is derived from a common precursor,  <dig> -dihydroxy- <dig> -pentadione , the product of the luxs enzyme  <cit> . this system may be involved in cross-communication among both gram-positive and gram-negative bacteria, as homologues of luxs are widespread within the microbial world. luxs is highly conserved in the bacterial kingdom  <cit> . in s. mutans, qs regulates cardinal physiological functions, such as the ability to withstand environmental stress conditions, competence and biofilm formation  <cit> . knockout of the luxs gene was shown to impair biofilm growth and stress tolerance of bacteria  <cit> . as a step towards understanding the possible link between biofilm thickness and qs, we analyzed the profile of selected gene expression in a luxs- mutant strain. the expression of genes identified in this study as associated with biofilm thickness was first compared between the mutant and wild-type strains in planktonic condition, and afterwards under biofilm vs. planktonic environments in the luxs- strain. interestingly, there were no radical changes in expression of genes associated with biofilm thickness in the tested conditions . only minor downregulation was recorded in smu.2146c, smu. <dig>  smu. <dig>  and smu. <dig> genes expression in biofilm vs. planktonic conditions . this result may explain the observation that biofilms of s. mutans deficient in luxs were not radically different from the wild-type  <cit> . although luxs plays an important role in initial adherence and initiation of mature biofilm development, it seems that biofilm thickness-associated genes are not regulated directly by luxs in s. mutans ua <dig>  interestingly, the luxs- mutant showed a diminished capacity to form biofilm when sucrose was provided as a supplemental sugar  <cit> , so it can be suggested that luxs plays a significant role, most likely in regulation of sucrose-dependent adherence rather than cell-cell interactions during biofilm growth in s. mutans.\n",
            "\n",
            "CONCLUSIONS\n",
            "this study provides a genome-scale outline of genes associated with biofilm thickness in s. mutans, a highly pathogenic bacterium in dental diseases. by expression alterations in these genes, the bacteria within the various layers of a mature biofilm may express different phenotypic characteristics allowing better acclimation in the biofilm micro-environment. moreover, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "furthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\n",
            "the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\n",
            "furthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\n",
            "furthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\n",
            "furthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\n",
            "furthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\n",
            "furthermore, the expression of these genes is not directly regulated by luxs and is not necessarily influenced by the presence of sucrose during biofilm maturation.\n",
            "in this study, dna-microarrays and real-time rt-pcr analysis were carried out to characterize the transcriptional differences of s. mutans in various biofilm depths.\n",
            "\n",
            "47\n",
            "[{'summary_text': 'the tcts gene family consists of  <dig> trans-sialidase genes,  <dig> of which are annotated as “pseudogenes” in the tcts gene family.\\nin this work, we describe a recombination detection pipeline consisting of  <dig> tcts genes,  <dig> of which are annotated as “pseudogenes” and  <dig> of which are annotated as “pseudogenes” in the tcts gene family.\\nthe tcts gene family consists of  <dig> trans-sialidase genes,  <dig> of which are annotated as “pseudogenes” in the tcts gene family.\\nthe tcts gene family consists of  <dig> trans-sialidase genes,  <dig> of which are annotated as “pseudogenes” in the tcts gene family, and  <dig> of which are annotated as “pseudogenes” in the tcts gene family.\\nin this work, we describe a recombination detection pipeline consisting of  <dig> t'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the first complete genome sequence for the protozoan parasite t. cruzi was reported in  <dig>  along with the genomes of the related parasites trypanosoma brucei and leishmania major. together, these parasites are the cause of substantial morbidity and mortality and thus are the focus of continued research to prevent and treat the diseases that they cause.\n",
            "\n",
            "one of the unique features of the t. cruzi genome is the abundance of large gene families, groups of homologous and functionally related genes that exist in multiple loci across the genome. greater than 25 % of the genome of t. cruzi is comprised of families of genes with at least  <dig> members  <cit> . in contrast, the gene family composition for t. brucei and l. major is approximately  <dig> and 12 % respectively  <cit> . furthermore, three of these gene families in t. cruzi, the trans-sialidases, mucins and mucin-associated surface proteins  each encode more than  <dig> unique surface membrane proteins. one of the purposes of this study is to more fully inventory the largest of these families, the trans-sialidases, and to determine the mechanism of their massive diversification in t. cruzi.\n",
            "\n",
            "the t. cruzi trans-sialidases  are glycosylphosphatidylinositol-anchored surface proteins produced by trypomastigotes and amastigotes of t. cruzi. a minority  of the tcts genes encode enzymatically active proteins that transfer sialyl residues from host donor molecules to t. cruzi glycoproteins and glycolipids. because t. cruzi and other trypanosomatids are unable to synthesize sialic acid de novo, this transferase activity is the only mechanism for sialylation of parasite surface molecules and is crucial to parasite infectivity and survival  <cit> . the purpose of the vast majority of tcts genes which apparently lack trans-sialidase activity and the reason for the development and expansion of this and other gene families selectively in t. cruzi is not understood. one hypothesis is that multiple copies of similar genes might serve an immune evasion function for t. cruzi by presenting to the immune system a multitude of different and potentially constantly varying surface molecules. it is further postulated that these gene families have expanded in response to immune pressure, presumably by duplication, recombination and mutation of the ancestral founding family members.\n",
            "\n",
            "while the genome provides ample evidence of large gene families in t. cruzi, it must be noted that the presence of large numbers of homologous genes and other repetitive elements also hindered genome assembly and a complete inventory of gene family members. in fact, the genome was not initially fully assembled into chromosomes but was rather published as  <dig>  contigs , some of which were mapped to ~ <dig> scaffolds. given that there are over  <dig>  unannotated contigs  the question arises as to what these contigs contain. focusing specifically on the tcts family, we have asked if there are additional, previously unannotated tcts genes in the genome.\n",
            "\n",
            "in this work we describe the method to specifically identify all trans-sialidase-like sequences in the reference t. cruzi cl brener genome, raising the total number from the  <dig> initially annotated to  <dig>  next, we verified and adjusted the predicted bounds of the annotated tcts genes , including among these tcts nucleotide sequences that contain in-frame stop codons. motif analysis using multiple em for motif elicitation  provided a means to compare the structure of all  <dig> tcts sequences and to generate a model tcts based upon the most conserved motif patterns. lastly, we show that tcts family members have been undergoing recombination, thus generating new variants from the thousands of pieces, while retaining the overall structure of the core tcts family members.\n",
            "\n",
            "RESULTS\n",
            "identification of previously unannotated tcts sequences\n",
            "prior to beginning analysis of the tcts family, we first made sure that we had identified the complete complement of tcts genes for the reference cl brener clone of t. cruzi. previous work by us  and others  <cit>  indicates that the number of sequenced tcts genes is underestimated, as evidenced by merging of large numbers of whole genome shotgun  reads into relatively few annotated genes. additionally, proteome analysis has shown that t. cruzi peptides map to sequence reads that are not assigned to annotated proteins  <cit> . furthermore, a preliminary remapping of wgs reads selected because of their homology to tcts gene sequences to the assembled genome revealed that a large number of these tcts-like reads mapped to regions with no annotated genes. thus this work sought to determine if additional, unannotated tcts genes might be identifiable, and if so, how these newly annotated genes compared to the existing set of previously annotated tcts genes.\n",
            "\n",
            "assembly of the t. cruzi cl brener genome was difficult, in part because of the hybrid nature of this parasite clone and, more importantly, because of the very high number of closely related genes that form large gene families. in our attempts to provide a more complete and accurate assembly, we have, among other things, remapped the original sequencing reads from the whole genome shotgun  to the assembled genome . in order to separate potentially merged tcts genes  <cit>  into their individual genes, the sequences for all  <dig> annotated tcts genes from the cl brener genome, along with up to 1kb flanking sequence, were blasted   <cit>  against all  <dig> , <dig> wgs reads. all reads for which the best hit was to a tcts-like region with at least 90 % sequence identity were identified, resulting in  <dig>  tcts-like reads. subsequently, the tcts-like reads were blasted against all  <dig>  genome contigs to identify the regions of the genome that were most homologous to each read . this read-to-contig assignment revealed  <dig>  reads mapping to regions containing annotated tcts genes and  <dig>  mapping to non-tcts genes. given that the original source sequences included tcts-flanking regions, these results are not surprising. however, nearly  <dig>  reads mapped to regions of the genome that contained no annotated genes. since these regions were identified from tcts-like reads, it was postulated that they contained either unannotated tcts genes or genes that typically flank tcts genes .\n",
            "\n",
            "using the regions of homology from the blast report,  <dig> distinct  contig regions were identified from the reads that mapped to no annotated genes. to determine which of these regions contained full or partial tcts genes, the dna sequences for the contig regions were extracted and blasted  against the  <dig> tcts genes annotated as full-length . blastx was used in order to identify contiguous tcts sequences on the contigs regardless of frame shifts. using a minimum identity cutoff of 90 %, the contiguous regions of contigs matching the tcts genes were identified, resulting in  <dig> candidate tcts genes. to help ensure that the new tcts genes were sufficiently trans-sialidase-like, the  <dig> sequences were blasted  against the entire genome to confirm that the most homologous protein for each newly identified tcts-like sequence was an annotated trans-sialidase gene. from this validation step,  <dig> were selected as newly identified tcts genes. the new tcts sequences were on average shorter than the originally annotated set of tcts genes  but also included many full or nearly full length genes . one hundred fourteen  of these newly identified tcts genes mapped to the assembled cl brener chromosome pairs  <cit>  while the remaining  <dig> genes mapped to contigs and scaffolds that could not be assembled into the  <dig> chromosomes. this contrasts with the  <dig> out of  <dig> previously annotated tcts genes that mapped to the chromosomes .fig.  <dig> comparison of size of previously and newly annotated tcts genes. the dna sequences for the tcts genes in the annotated genome and those newly identified were binned according to sequence length in the ranges displayed and the quantity in each bin plotted for each sequence type\n",
            "\n",
            "\n",
            "\n",
            "modification of tcts coding sequences\n",
            "the above described analysis more than doubled the number of identified tcts sequences in the t. cruzi genome and posed the additional question of ‘what characteristics are common to these > <dig> tcts gene sequences?’, i.e. ‘what defines a tcts protein?’. in order to address this question, we submitted the tcts genes to three analyses and potential modifications. first, to confirm that the 5′ and 3′ bounds for all tcts genes  encompassed the entire predicted coding region,  <dig> “model” full-length tcts genes  of at least 2kb in length and containing both a signal peptide and gpi anchor motif were selected and were blasted  against the designated nucleotide sequence for all tcts sequences . the predicted bounds of each tcts sequence were then set according to its homology with these model genes. this analysis resulted in the alteration in the bounds for  <dig> of the  <dig> previously annotated tcts genes and the merging of two adjacent sequences  into a single tcts gene .\n",
            "\n",
            "second, deletions causing frame-shifts that result in stop codons are present in nearly half of the tcts genes and were designated as “pseudogenes” in the annotated cl brener genome. previously, we demonstrated that a number of tcts pseudogene-encoded proteins are detected in the t. cruzi proteome  <cit> , indicating that these are real genes that probably produce a truncated tcts product. in many cases, the dna sequence beyond these stop sites were homologous to other tcts gene sequences but presumably not translated into protein. however, if these sequences beyond the stop sites were at some point recombined with other tcts genes , a shift in reading frame could result in production of a tcts protein from this sequence. thus in order to generate a complete set of potential tcts protein sequences, the tcts sequences were examined for the presence of in-frame stop codons that might result in a truncated product, and if present, the reading frame was shifted by removing one or more bases to create a tcts-like protein ). this latter exercise also served to expand our tcts protein set to include sequences that could have been erroneously terminated due to sequencing or assembly errors and allowed us to utilize the complete potential protein sequence information for genes previously annotated as “pseudogenes” in the analysis of proteomic data, for protein motif searches, and for examination of the products of gene recombination.\n",
            "\n",
            "third, to improve upon the annotation information for tcts sequences , all tcts sequences were re-annotated based on their domain composition, length, sequence type, and whether or not their translations result in any in-frame stop codons. among the potential features annotated in this process are a signal peptide, gpi anchor addition site, shed acute-phase antigen  repeats, frip motif , and the asp block  and vtvxnvxlrn motifs  <cit> . we provide three tcts sequence types, where a “full length” sequence must contain both a signal peptide and gpi anchor motif, a “partial” sequence contains a signal peptide motif but no gpi anchor, and a “pseudogene” contains no signal peptide motif. finally, if the amino acid translation of the sequence results in any stop codons, we notate that sequence with “in-frame stop” .\n",
            "\n",
            "confirmation of tcts coding sequences\n",
            "to evaluate if the newly identified tcts, most of which were not full-length genes, also encoded bona fide proteins, a database of all t. cruzi proteins  was created and screened with the spectra from three proteomic analyses using a 6-frame translation. this analysis identified a minimum of  <dig> trans-sialidase proteins,  <dig> of which were among those newly annotated as described above . this is a highly conservative estimate of the expressed repertoire due to the limitation of proteomic techniques and the difficulty of uniquely mapping small peptides to a set of highly similar proteins like those expressed by tcts genes, as previously discussed  <cit> . if we exclude peptides that map to protein translations of both previously annotated and newly annotated tcts genes,  <dig> newly annotated tcts genes had unique peptide evidence of expression . thus, of the  <dig> tcts protein sequences identified in the proteomic analysis, almost half  were new tcts sequences, of which 38 %  were unambiguously identified with high-confidence. additionally,  <dig> previously annotated tcts “pseudogenes” were confirmed to yield protein products.\n",
            "\n",
            "motif identification and construction of a model tcts gene\n",
            "the large size of the tcts family made sequence alignment an untenable method for examining the common sequence features of the > <dig> genes. thus we employed the meme tool  <cit>  because it allowed us to identify common sequence features of the tcts proteins and to determine the representation of these features in all members of the family. we arbitrarily set the meme analysis to identify  <dig> features among the tcts family proteins; motifs with the lowest number are the most common and most highly conserved while those closer to  <dig> are less so. of the  <dig> total tcts sequences,  <dig> contained at least  <dig> motif, and motif architectures were determined for these sequences and aligned .\n",
            "\n",
            "a model tcts architecture was determined from the alignment by selecting the most common motif at each position . though the model does not capture all of the variation present in the alignment, it does provide a global representation of the most common features in tcts sequences. not surprisingly, the most common motifs  are in the n-terminal region containing the signal sequence . other known tcts features are asp-boxes , frip , gpi anchor , and the vtvxnvxlynr motif . the c-terminal region between the vtvxnvxlynr and gpi anchor is variable and sometimes contains sapa repeats. finally, selected motifs implicated in the catalytic mechanism of tcts were located in the motifs . examination of the  <dig> tcts sequences relative to the model reveals a very high degree of conservation in the ordering of the motifs; with few exceptions, the order of motifs is maintained even in the small and thus partial tcts sequences. at any particular position in the alignment, several different motifs may be observed across the architectures, but it is rare for a motif to be observed at multiple positions, especially for the higher-confidence  motifs. thus, the motif analysis of all tcts family members in the cl brener genome suggests a mechanism for the retention of a common tcts protein structure despite the nucleotide diversity among the gene family members.fig.  <dig> analysis of model tcts architecture. a the model tcts architecture  is depicted. each box represents an individual motif from the meme analysis. the top number is the motif number, the middle percentage is the representation of that motif across all tcts sequences , and the bottom number is the representation of that motif in tcts sequences of length >2000b. frequent motifs  are color-coded red, while less frequent motifs are blue. the highly-variable c-terminal region containing multiple motifs is represented as a black box. locations of previously characterized tcts motifs and residues are also shown. b an entropy analysis of tcts sequences is shown. the smoothed entropy values for selected tcts proteins is plotted against amino acid position. using sequence homology of the motifs in this study to the protein sequences of the entropy study, locations of the motifs are overlaid in the chart . note that lower-numbered motifs are typically found in areas of low entropy. c the crystal structure of tcts “1s0i_a” with motif regions highlighted and labeled and d) a notable region lacking in motifs \n",
            "\n",
            "\n",
            "\n",
            "not surprisingly, the most highly conserved patterns correlate with low entropy measurements . mapping of the motifs to the crystal structure of the 1so1_a tcts protein  <cit>  demonstrates the regions of higher conservation among tcts molecules  and reveals a large region of the catalytic domain with sparse mapping of motifs, indicating this as a more variable region among tcts proteins . the heat map of additional file 9: figure s <dig>  provides a global view of the degree of conservation of motifs along with the variability that provides diversity. the typical scenario shows either high conservation  with low entropy regions  and high-confidence motif-to-architecture assignments ) or high diversity . of particular interest are columns that show seemingly contradicting data: high frequency model motif  with relatively high entropy  or low frequency model motifs  with low entropy . an example of the former is motif  <dig>  this region occurs after the signal peptide and is represented in a high number of sequences . one interpretation of this result is the constraint in this area retains protein structure but does not maintain any specific function; so long as the protein conformation is correct the sequence is relatively free to mutate. an example of the latter case is motif  <dig>  which displays relatively low entropy but is not highly prevalent across the architectures . the mapping of the motifs to the structure shows that this motif is near the central, α–helix region connecting the n-terminal and c-terminal domains . the minimum entropy among all residues in motif  <dig> is  <dig>  and the maximum is  <dig> , suggesting tight constraint at particular positions but not on the motif region as a whole.\n",
            "\n",
            "phylogenetic tree of related tcts sequences\n",
            "the distribution throughout the t. cruzi genome of tcts sequences of varying sizes but with a conserved structure strongly suggests a mechanism of gene duplication followed by recombination and mutation in the evolution of this large and diverse gene family. to explore this possibility further, we first examined phylogenetic trees of tcts sequences for evidence of recombination. due to the substantial variation in gene length and nucleotide polymorphism among the > <dig> tcts sequences , we performed phylogenetic tree inference on small sets of more closely related tcts sequences identified by megablast with custom thresholds . although the potential mosaic nature of tcts sequences could hinder accurate phylogeny assessment, we were able to estimate a well-supported bayesian tree  by a lengthy and aggressive markov chain monte carlo exploration in parameter space. this bayesian phylogenetic tree has long terminal branches, indicating that either  evolution is accerated in all analyzed genes, since their divergence from their most recent common ancestors, or  disruptions in gene-independent evolution by processes such as recombination, which exchanges genetic information between genes and thus obscures ancestry. notably, a similar tree pattern is observed for the variant surface glycoprotein  genes in t. brucei and t. congolense  <cit> .fig.  <dig> bayesian phylogenetic tree of  <dig> closely related tcts. in order to construct a phylogenetic tree from high quality alignment, sequences from  <dig> tcts genes with high pairwise identity were selected. sequences were aligned using muscle algorithm, and the tree was constructed by mrbayes v <dig> . <dig>  posterior probability values are indicated on tree nodes. low treeness  provides support for recombination among tcts gene family members\n",
            "\n",
            "\n",
            "\n",
            "assessment of recombination events in tcts\n",
            "in order to more thoroughly evaluate the role of recombination in the evolution of the tcts family, we developed a recombination detection pipeline that relied heavily on the recombination detection program  v <dig> ,  <cit>  . evaluation of the pipeline with simulated data indicated dependable performance  with a sensitivity of  <dig>  and  <dig>  for gene-conversion-recombinant genes and gene-conversion-recombinant events, respectively, demonstrating the ability of our pipeline to accurately capture the presence of recombinant genes and recombination events. our pipeline displayed a specificity of  <dig>  and  <dig>  for gene-conversion-recombinant genes and gene-conversion-recombinant events, respectively. note that the event-specificity, which describes how frequently the pipeline precisely determines recombination breakpoints, is impacted by the difficulties of pinpointing the recombination breakpoint between sequences with less informative variations . the false positive rates are below  <dig>  for all recombination-negative datasets tested. notably, our recombination pipeline is not intended to detect reciprocal exchange of dna sequences between  <dig> tcts genes, as is shown by simulation results .fig.  <dig> workflow of recombination detection pipeline fortctsgene family. a tcts sequences are binned into similarity sets by scanning each tcts gene for closest-matching non-self sequences within the tcts gene family; b rdp is used to identify possible recombinant regions, recombination breakpoints, and the major and minor sequence donor among the tcts similarity sets; c false-positive recombination events are removed by realigning each recombination event’s triplet of mosaic sequence, major donor and minor donor\n",
            "\n",
            "\n",
            "\n",
            "using this pipeline to analyze the  <dig> tcts genes, we document clear evidence of recombination in  <dig> trans-sialidase genes with a total of  <dig> recombination events;  <dig> and  <dig> genes have participated at least once as major donors and minor donors, respectively. of all the  <dig> recombination events,  <dig> events have a p-value less than  <dig>  with one or more of the algorithms in rdp and  <dig> events have p-values < <dig>  for all algorithms . all  <dig> recombination events also have multiple-test corrected p-values under  <dig> . thus, these recombination events are detected with very high confidence. the  <dig> mosaic genes identified have between  <dig> and  <dig> recombination events each, with an average of  <dig>  recombination events/genes . the signals of mosaic sequence patches in tcts genes can be confirmed by manual inspection at the nucleotide level and fig.  <dig> shows an example. note that the break-points of recombinant regions frequently lack a distinct boundary but instead have short regions that are not encoded by either the major donor or minor donor .fig.  <dig> a mosaic tcts gene showing evidence for recombination events and example sequence comparisons within each minor donor sequence. a bootscan and maxichi output showing alterations in bootstrap support and peak log) values, respectively, at mosaic region boundaries. b the tcclb. <dig>  mosaic gene with donor contributions indicated by colors. c partial alignment of mosaic regions with respective donors. in each sequence triplet alignment, the middle sequence is the mosaic gene, shading highlights identical nucleotides of mosaic sequences with major  and minor  donor\n",
            "\n",
            "fig.  <dig> examples of boundary regions in mosaic tcts. from the mosaic tcclb. <dig>  gene in fig.  <dig>  boundary regions on one end of each of the  <dig> donor sequences are shown. in each sequence triplet alignment, the middle sequence is the mosaic gene, red and blue shading highlight identical nucleotides of mosaic sequences with major donor and minor donors. the brackets indicate the region most likely containing the recombination breakpoint. a donor sequence = tcclb. <dig> ; b donor sequence = tcclb. <dig> ; c donor sequence = tcclb. <dig> ; d donor sequence = tcclb. <dig> \n",
            "\n",
            "\n",
            "\n",
            "participation in recombination  was higher for previously annotated tcts genes relative to the newly annotated genes , although a significant proportion of this difference was due to the smaller average size of the newly annotated tcts genes. recombination events were more likely to be detected between full length and other tcts genes of  <dig> bases or more . however, 25 % of the donors contributing to mosaic full length tcts genes were of < <dig> bases, demonstrating that the incomplete tcts gene fragments in the t. cruzi genome play an active role in the diversity of the expressed, full length genes. within a select set of tcts genes grouped into eight subsets based upon multiple component analysis  <cit> , recombination was exactly four times more likely to occur between members of the same group  than between groups . it is also notable that recombination events both contributed to the production of full length tcts proteins from partial gene donors  as well as partial genes from full length genes due to shifts in reading frame as a result of recombination . thus there is an active exchange between the full-length tcts genes encoding functional proteins and partial tcts genes and pseudogenes dispersed throughout the t. cruzi genome.fig.  <dig> productive and nonproductive recombination events. a the full length, protein coding tcts tcclb. <dig>  has incorporated mosaic parts from partial tcts gene and full length tcts donors. b. introduction of sequence from the full-length tcclb. <dig>  gene into the wrong reading frame of the major donor tcclb. <dig>  results in a partial tcclb. <dig>  product. also note that two in-frame stop codons have resulted from single point mutations in the major donor supplied sequence\n",
            "\n",
            "\n",
            "\n",
            "discussion\n",
            "in the evolutionary arms race between the host immune system and pathogens, mammals have incorporated immense complexity into the immune system. the multi-gene segmental organization of immunoglobulin and t cell receptor genes, combined with recombination, can produce many millions of distinct antigen receptors capable of detecting and neutralizing a diverse array of pathogens. pathogens, including eukaryotic parasites, have also evolved mechanisms to incorporate diversity into their structures in order to avoid immune destruction. the archetypal example of antigenic variation is the variant surface glycoproteins  gene family of t. brucei, which has more than  <dig> members. although its expression is mono-allelic, the relatively frequent switch of the expressed vsg gene enables t. brucei to maintain its persistence despite the constant surveillance by host immune responses. the creation of mosaic vsgs by recombination among vsg family members  <cit>  is also crucial for parasite persistence, as host immunity to previously expressed vsgs  <cit>  accumulates over the time of infection. likewise, plasmodium sp. have evolved smaller multi-gene families encoding cell surface proteins that depend on a high frequency of recombination to produce new immune-evading variants  <cit> .\n",
            "\n",
            "t. cruzi encodes several large, highly variant gene families, including the largest known family of variant surface protein-coding genes, the trans-sialidase family. the mostly glycosylphosphatidylinositol- -anchored tcts surface proteins have several essential functions in the parasite. the presumed founding members of this family provide t. cruzi the ability to acquire sialic acid from host donor molecules and move it to terminal glycans in a sialyl-transferase reaction. this process is apparently necessitated by the fact that t. cruzi lacks the ability to produce its own sialic acid, and in the absence of sialic acid, t. cruzi trypomastigotes are highly susceptible to host complement activation and lysis and are also poorly invasive. however, only a small proportion of the immense ts gene family, perhaps as few as 10– <dig> genes, actually encode enzymatically active ts. the remainder have been ascribed a number of other activities; among the more important, as lectins potentially involved in host cell attachment  <cit> .\n",
            "\n",
            "despite the importance of these ts activities for the survival of t. cruzi, it is difficult to reconcile the huge commitment in terms of genome space that t. cruzi makes to the maintenance of tcts-family genes with these noted ts functions alone. the original assembly of the reference cl brener genome tallied nearly  <dig> tcts genes, approximately half of which were annotated as “pseudogenes”. in this study, the remapping of the original cl brener sequence reads and re-analysis for tcts-like sequences more than doubled the number of full and partial tcts genes identifiable in the cl brener genome to the current  <dig> unique genes. further, proteome analysis also confirmed the expression of at least a portion of these newly identified full length and truncated tcts family genes. certainly t. cruzi should not require > <dig> full and partial length tcts family genes to carry out the functions of sialic acid transfer and cell adhesion, and even if all the proposed functions of ts proteins are considered, performance of these activities perhaps requires a handful of genes – not > <dig>  this degree of gene diversity within individuals of a single species is nearly unprecedented, rivaled only by the ~ <dig> member vsg gene family in the related t. brucei  <cit> .\n",
            "\n",
            "a limitation of the current study is the exclusive use of the reference cl-brener genome of t. cruzi. like most reference genomes, the cl brener reference was generated from relatively low coverage sanger sequencing and is prone to misassembles, particularly in regions rich in multi-gene families. as additional and more deeply sequenced and better assembled genomes of t. cruzi become available, it will be of great interest to compare the repertoire of ts genes in independently evolving lineages in this diverse species.\n",
            "\n",
            "the critical requirement for the cell surface expression of ts activity presents a dilemma for a pathogen like t. cruzi. on the one hand, lack of ts activity significantly reduces the ability of t. cruzi to survive in mammalian hosts. on the other, expression of a single species of ts protein on the parasite surface would provide a tantalizing target for the host immune system and an effective response to such a target could prove deadly for t. cruzi. indeed ts proteins are significant targets of anti-t. cruzi immune responses  <cit>  and vaccination against ts proteins is somewhat effective, at least in preventing potentially lethal infections . thus there is strong pressure from host immune responses on ts proteins, and as with other cases of antigen diversification in pathogens, this is the likely driving force for the retention and diversification that ultimately created this immense family of genes.\n",
            "\n",
            "the mechanism of the expansion and continued diversification of the tcts family in t. cruzi is gene duplication, mutation and recombination among family members. we present evidence for  <dig> mosaic trans-sialidase genes resulting from  <dig> separate recombination events. due to the conservative calling of recombination by gene conversion, ours is a minimal estimate; other  recombination events are likely obscured by sequence divergence resulting from accumulating mutations in both donor and recipient genes. despite the frequency and wide scope of recombinations, all the tcts family members, including enzymatically active tcts, ts-inactive, truncated molecules, and gene fragments incapable of producing a protein product, retain a similar domain structure. the retention of the common features of tcts genes and proteins across the entire family likely reflects constraints imposed by the non-enzymatic functions of tcts, such as binding to  sialyl and beta-galactopyranosyl residues  <cit> , as well as the contribution of pseudogenes/partial genes to the formation of new mosaic tcts genes. substantial homology is generally required for gene conversion-based recombination  <cit>  and this requirement would work to sustain the common structure of the tcts genes. the fact that intra-gene-family recombination events among tcts genes are four times more likely to occur between genes in the same similarity group  <cit>  than between members of different groups supports this point. importantly our analysis additionally supplies evidence that the structurally conserved but sequence variant non-coding gene fragments are derived from and are recycled back into functional  tcts genes. indeed, nearly half of the  <dig> full-length protein-coding tcts identified as mosaics have incorporated tcts pseudogenes and/or partial genes. thus, the non-coding tcts fragments undoubtedly acts as a repository of diversity for the generation of new and antigenically diverse tcts genes.\n",
            "\n",
            "gene conversion has been shown to be a major intra-vsg-family recombination mechanism  <cit> , driving antigenic diversity in t. brucei  <cit> . thus it was reasonable to suspect that gene conversion recombination events was integral to the generation of variation in the t. cruzi ts gene family. classical recombination analysis typically relies on highly accurate sequence alignment and uses evidence of local disruption of sequence signatures or phylogenetic signals as signs of recombination. rigorous statistical analysis can then be used to confirm the significance of the identified recombination breakpoint. for gene families such as tcts containing thousands of members, such methods are not possible. alternatively, blast algorithms can be used to find local sequence matches between different family members  <cit>  or phylogenetic incompatibility distribution can be used as an index to estimate recombination activity  <cit>  and such methods have been applied to document recombination events in the t. brucei vsg gene family. however, such alternative methods, unlike the classical recombination analysis, cannot ensure both statistical support and the search-comprehensiveness of recombination events. here we propose an alternative recombination analysis pipeline that can be used on large and diverse genes families while also providing statistical validation of the results.\n",
            "\n",
            "this recombination detection pipeline first uses blast to find local sequence matches between individual tcts genes, then, for each local sequence match, we group involved-tcts genes into a subset, between which recombination is likely to produce the mosaic patch . then for each tcts subset, we align the sequences and subject the alignment to a recombination detection program package  that implements many classical recombination detection algorithms. if a recombination event is cross-validated by multiple recombination detection algorithms, we perform an additional round of validation by realigning just the two recombination donors and the mosaic sequence and then subject the alignment to recombination analysis by rdp. this final validation step further removes false-positives caused by alignment artifacts. by starting from every possible mosaic-forming event in the gene family and validating through two rounds of rigorous recombination analysis, we combined both statistical support and search-comprehensiveness in our recombination detection pipeline for very-large gene families. though search-comprehensiveness is our goal, our pipeline would still underestimate the recombination events in a large gene family since the rigorous statistical threshold would miss many weaker recombination signals; recombination events could be overlapping, masking each other and blurring the recombination breakpoint. thus, we are likely able to see only a small part of the complex tcts recombination landscape. the current analysis is also restricted to a single t. cruzi isolate, the genome reference cl brener clone. each t. cruzi clone will generate a distinct pattern of recombination and thus its own set of ts genes, resulting in incredibly diverse populations of parasites within and between hosts. similar detailed analysis to that conducted here in other isolates is expected to confirm this species-wide diversity. additionally, these mechanisms of gene family expansion almost certainly play a similar role in other large ) and perhaps smaller gene families in t. cruzi.\n",
            "\n",
            "although the generation and maintenance of diversity in the t. cruzi tcts with that of the vsg genes of t. brucei, there are also significant differences. most importantly, t. cruzi expresses the ts genes from multiple alleles concurrently rather than serial expression of vsg genes from single alleles as in the case of vsg genes t. brucei. tctss are broadly and abundantly expressed in both extracellular and intracellular stages of t. cruzi in mammalian hosts. although it is not possible with the technology now available to determine if all of the tcts genes are expressed concurrently , it seems likely that the majority of those with intact 5′ expression and processing sites are being produced during one or more stages of the t. cruzi life cycle. atwood et al. conservatively identified  <dig> expressed tcts  in a proteome study of all t. cruzi life-cycle stages, and  <dig> out of  <dig> top-scoring  proteins in this analysis were tcts family members  <cit> . the immune detection of epitopes encoded in different sets of tcts molecules is also strong evidence of the expression of many, if not all competent tcts genes during infection  <cit> . the immune evasion potential of a mutually exclusive pattern of expression of gene variants, as in the vsgs of african trypanosomes, is clear, as each switch to a new variant forces a restart of the immune response to the new immunodominant antigen. the benefits of the distinctive approach employed by t. cruzi of expressing many variants at once is less obvious and certainly more complex. thus t. cruzi has evolved a unique immune evasion process and likely, a unique way of regulating the size of this and other gene families and the diversity among family members.\n",
            "\n",
            "we favor the hypothesis that the expression of large numbers of tcts variants serves an immune evasion function primarily for intracellular amastigote stages of t. cruzi and contributes to avoidance of detection by cd8+ t cells of parasite-infected cells. expression of an abundance of different ts molecules in an infected host cell could flood the class i mhc presentation pathway with a huge diversity of potential epitopes, each competing for binding to and presentation by mhc. as a result, the effective density of most parasite epitopes could remain below that necessary to trigger effector t cells, particularly for host cells with more modest levels of mhc expression  <cit> . the very high sequence similarity of tcts proteins would also provide for a network of variant epitopes capable of acting as altered peptide ligands – partial agonists or antagonists – of t cells specific for related epitopes  <cit>  and references therein). the timing of expression of the tcts proteins in intracellular t. cruzi amastigotes also impacts the presentation of ts epitopes on infected cells. we recently reported that the highly immunodominant tskb <dig> tcts epitope is not detectable on t. cruzi –infected host cells until >24 h after invasion of host cells. in contrast, a subdominant, non-variant epitope from the parasite flagellum is recognized within 3– <dig> h after infection  <cit> . interestingly, boosting the immune response to flagellar rod proteins also significantly enhanced resistance to t. cruzi  <cit> . the impact of the high frequency ts epitope-specific cd8+ t cell response generated in c57bl/6j mice is relatively minimal, since ablation of this response does not affect the ability of hosts to control of t. cruzi infection  <cit> . these data suggest that while the structure, complexity and expression of the ts family of genes does not prevent the generation of ts-specific cd8+ t cell responses, collectively these characteristics render the generated ts-specific response irrelevant with respect to the control of t. cruzi infection. in short, although unconventional, the simultaneous expression of many variant proteins appears to be a highly effective immune evasion strategy.\n",
            "\n",
            "the creation of new mosaic genes via intra-family recombination likely contributes to the chronicity of a number of viral and bacterial infections  <cit> , and this within-host evolution of antigens may also play a role in the ability of t. cruzi to persist for decades in humans and other hosts. additionally the evolution of diversity among different lineages within the species – potentially resulting in a distinct sets of pathogen genetic types in each host – has implications beyond the individual host. palmer and coworkers have argued that the potential for superinfection – the reinfection of an already infected host – also applies evolutionary selective force for the genetic diversification of antigen variants beyond that necessary for persistence within an individual host  <cit> . this pressure would be particularly strong when the infection rates within a population are high, effectively making all members of the community accessible to infection , irrespective of current infection status. there is solid evidence for re-infection of currently and previously infected hosts by t. cruzi  <cit> , and the detection of hosts carrying mixed populations of genetically distinct parasite lineages suggests that such superinfections occur naturally . indeed superinfection with genetically distinct parasite types would facilitate genetic exchange within the species, a particularly important benefit in a species like t. cruzi that appears to rarely undergo full genome hybridization. the evidence of substantial genetic exchange between diverse t. cruzi lineages supports this possibility .\n",
            "\n",
            "solid proof of an immune evasion role for the tcts gene family at the level of cd8+t cells awaits procedures which can drastically reduce the number of simultaneously expressed ts genes within t. cruzi. the crispr-cas <dig> gene editing system may provide that opportunity. we have recently shown the ability in t. cruzi to substantially decrease the functional activity of proteins encoded by over  <dig> genes using the crispr-cas <dig> system and just three guide rnas  <cit> . the most highly conserved motifs identified in the tcts proteins  correlate with high dna sequence homology that allows a relatively small number of grnas to target > <dig> tcts genes. thus, we believe it is well within reach to ask if mutation of large numbers of tcts genes compromises the immune evasion capabilities of t. cruzi.\n",
            "\n",
            "CONCLUSIONS\n",
            "t. cruzi has evolved a unique immune evasion process that involves the simultaneous expression of large numbers of constantly changing surface molecules. although the mechanisms for generating and maintaining the tcts gene family are conventional - duplication, mutation and recombination – the size and diversity of the tcts family is unprecedented. it is likely that other large gene families in t. cruzi utilize these same processes also driven by the need to evade host immune responses. recently developed tools based on crispr/cas <dig> should provide the means to substantially reduce the size and diversity of these gene families and determine the contribution of gene family size to parasite persistence in infected hosts.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the tcts gene family consists of  <dig> trans-sialidase genes,  <dig> of which are annotated as “pseudogenes” in the tcts gene family.\n",
            "in this work, we describe a recombination detection pipeline consisting of  <dig> tcts genes,  <dig> of which are annotated as “pseudogenes” and  <dig> of which are annotated as “pseudogenes” in the tcts gene family.\n",
            "the tcts gene family consists of  <dig> trans-sialidase genes,  <dig> of which are annotated as “pseudogenes” in the tcts gene family.\n",
            "the tcts gene family consists of  <dig> trans-sialidase genes,  <dig> of which are annotated as “pseudogenes” in the tcts gene family, and  <dig> of which are annotated as “pseudogenes” in the tcts gene family.\n",
            "in this work, we describe a recombination detection pipeline consisting of  <dig> t\n",
            "48\n",
            "[{'summary_text': 'a genetic map of s. spontaneum was generated using  <dig> single dose markers and  <dig> multi-allelic markers with a total length of  <dig> \\xa0cm and an average distance of  <dig> \\xa0cm.\\na genetic map of s. spontaneum was generated using  <dig> single dose markers with a total length of  <dig> \\xa0cm and an average distance of  <dig> \\xa0cm.\\na genetic map of s. spontaneum was generated using  <dig> single dose markers with a total map length of  <dig> \\xa0cm and an average distance of  <dig> \\xa0cm between markers.\\na genetic map of s. spontaneum was generated using  <dig> single dose markers with a total length of  <dig> \\xa0cm and an average distance of  <dig> \\xa0cm between markers.\\na genetic map of s. spontaneum was generated using  <dig> single dose markers with a total map length of  <dig> \\xa0cm and an average distance of  <dig> \\xa0cm between markers.\\na genetic map of s. spontaneum was generated'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "sugarcane is widely cultivated in tropical and subtropical regions and is primarily grown for sugar production accounting for about 75% of the world’s sucrose supply. it is clonally propagated and has a very high photosynthetic efficiency which makes it very attractive as a source of biomass\n",
            " <cit> . recently it has become more important as a biofuel crop for the production of ethanol\n",
            " <cit> . it is a perennial grass, a member of the poaceae family and andropogoneae tribe, which includes maize and sorghum. cultivated sugarcane is derived from inter-specific hybridisation between, in the main, two polyploid species s. officinarum l. and s. spontaneum l which have different basic chromosome numbers. the resultant cultivars are complex aneu-polyploids with chromosome numbers of 2n = 100- <dig>  in the production of modern cultivars derived from the initial hybrid, the 2n transmission that occurs from s. officinarum in the primary cross has been used to advantage. backcrossing to s. officinarum or other cultivars recovers the high sugar phenotype inherited from s. officinarum while retaining the disease resistance and good ratooning of the wild s. spontaneum parent. the preferential outbreeding, highly heterozygous and predominantly autopolyploid genetics of sugarcane are all factors that have hindered the development of a comprehensive genetic map.\n",
            "\n",
            "classic cytological studies using in situ hybridization have determined that s. officinarum is an octaploid  with a basic chromosome number of  <dig>  the same as its closest known diploid relative sorghum\n",
            " <cit> . s. spontaneum has varying ploidy levels  but has a basic chromosome number of x = 8\n",
            " <cit> . genome in situ hybridisation has determined that commercial cultivars contain approximately 80% s. officinarum chromosomes, 10-15% s. spontaneum chromosomes and approximately 10% recombinant chromosomes\n",
            " <cit> . q <dig> is an australian cultivar no longer grown commercially. it has  <dig> chromosomes of which 82- <dig>  are inherited from s. officinarum, 15- <dig>  are inherited from s. spontaneum and 11- <dig>  are recombinant chromosomes\n",
            " <cit> . these are similar in the proportions found in most varieties studied to date\n",
            " <cit> .\n",
            "\n",
            "sugarcane has a large genome \n",
            " <cit>  and in the absence of a genome sequence, a high-density genetic map is a valuable tool to understand the genetic and genomic organisation of this complex polyploid crop. its autopolyploid nature with mostly random pairing plus high inbreeding depression has limited the production of more common experimental mapping populations such as double haploids or recombinant inbred lines. the complication of the coexistence of single dose  and multi dose alleles and irregular chromosome numbers in the various homology classes due to aneuploidy has restricted genetic mapping. with the realisation that sd markers can be used to generate genetic maps in polyploids\n",
            " <cit>  genetic maps were developed in sugarcane using a population of full-sib  individuals \n",
            " <cit>  with sd markers segregating 1: <dig> or by using a population created by selfing an individual and mapping sd markers segregating 3: <dig>  initially mapping was carried out in the ancestral species s. spontaneum and s. officinarum <cit> . more recently cultivars of hybrid origin have been mapped\n",
            ". all of the genetic maps that have been generated for sugarcane to date have had low genome coverage and limited information on genome organisation. one of the main reasons for this has been the limited number of markers mapped. in most cases fewer than  <dig> have been mapped due to the complexity of marker generation in such a complex polyploidy and the high cost of generation of large numbers of mappable markers. many markers of different dosage are generated in sugarcane but segregation of markers of greater than double dose in the 200- <dig> size populations means that accuracy of marker placement is low\n",
            " <cit> .\n",
            "\n",
            "the development of diversity array technology \n",
            " <cit>  for sugarcane has combined the low-cost high-throughput properties of the dna microarray platform with the ability to identify various types of dna polymorphisms\n",
            " <cit> .\n",
            "\n",
            "this paper reports the generation of a comprehensive sugarcane genetic map of q <dig>  an australian sugarcane incorporating  <dig> markers generated from dart, amplified fragment length polymorphism , simple sequence repeats , single nucleotide polymorphism , restriction fragment length polymorphism  and random amplified polymorphism  markers. this large number of markers allowed the majority of the lgs to be placed into the  <dig> homology groups which is consistent with the basic chromosome number of the ancestral species s. spontaneum and the lowest basic chromosome number identified in the saccharum genus.\n",
            "\n",
            "RESULTS\n",
            "generation of mapping data\n",
            "to generate the maximum number of polymorphic markers using the sugarcane dart array, the dna from each of the f <dig> progeny was digested with several different restriction enzyme combinations. dna from each combination was then screened separately across the sugarcane dart array\n",
            " <cit> . using these separately screened enzyme combinations, a higher number of polymorphic markers was generated. in total  <dig> markers were scored as present/absent of which  <dig> were generated from a psti/bstni digest,  <dig> from a psti/hpaii digest and  <dig> from a psti/taqi digest. of the  <dig> dart markers generated  <dig> were identified by more than one enzyme digest and had the same segregation pattern. the remaining  <dig> unique markers included  <dig>  which were sd and used to generate the linkage map. psti/bstni generated  <dig>  sd markers, psti/hpaii generated  <dig>  and psti/taqi generated  <dig> .\n",
            "\n",
            "in total  <dig> markers were scored. of these,  <dig> were ssr markers of which  <dig> were sd,  <dig> aflp markers of which  <dig> were sd,  <dig> snp markers of which  <dig> were sd,  <dig> rapd markers of which  <dig> were sd; and  <dig> were rflp markers of which  <dig> were sd . the majority of the segregating markers inherited from q <dig> were sd, and segregating 1: <dig>  in the population .table  <dig> \n",
            "total number of single dose , double dose , triple dose  and bi-parental sd markers  scored in the parents of the mapping population\n",
            "\n",
            "\n",
            "s. officinarum\n",
            "\n",
            "chromosome no.\n",
            "\n",
            "number of markers\n",
            "\n",
            "number of sd markers\n",
            "\n",
            "number of dd markers\n",
            "\n",
            "number of td markers\n",
            "\n",
            "number of 3: <dig> markers\n",
            "\n",
            "total number of lg\n",
            "\n",
            "no. of sd markers in map\n",
            "\n",
            "1the addition of dart markers did not provide the same magnitude of improvement to the ij76- <dig> linkage map and the data is not shown\n",
            "number of markers generated with the different marker systems and number of single dose  markers \n",
            "\n",
            "\n",
            "\n",
            "dart\n",
            "\n",
            "aflp\n",
            "\n",
            "ssr\n",
            "\n",
            "snp\n",
            "\n",
            "rapd\n",
            "\n",
            "rflp\n",
            "\n",
            "total\n",
            "\n",
            "\n",
            "genetic map construction\n",
            "the genetic map was constructed using  <dig> progeny generated from a cross between q <dig> an old australian cultivar and ij76- <dig> a s. officinarum accession using the pseudo testcross approach. of the  <dig> markers scored that were present in q <dig> and absent in ij76- <dig>   <dig>  were sd in q <dig>  the majority of sd markers were dart, aflp and ssr markers although  <dig> est snp markers were also scored in this population . of the  <dig> sd markers used for mapping,  <dig> were present in and used to generate the map reported in\n",
            " <cit> . although the markers were selected as sd markers segregating 1: <dig> in the progeny a chi square value was used that allowed inclusion of sd markers with moderate segregation distortion  but excluded likely double dose markers as in\n",
            " <cit> , as it is well known that segregation distortion is common in plants. the percentage of skewed markers varied from homology group  to hg with 18% of markers skewed in hg <dig> and  <dig> % of markers skewed in hg <dig> . initially for map construction, only the markers that were present in greater than  <dig> progeny were used to generate a framework map. a number of the dart markers  were only scored in  <dig> individuals, and these markers were added to the map after initial map construction using markers scored across the whole population. the  <dig> sd markers formed  <dig> linkage groups containing  <dig> markers, leaving  <dig>  unlinked markers .figure  <dig> \n",
            "scatter plot representing the distribution of marker segregation distortion within each lg within the hgs with each dot representing one molecular marker. the vertical solid lines distinguish the  <dig> hgs which represent the  <dig> basic chromosomes, along the total map distance . the y axis is the log <dig> value of the ratio of the number of individuals carrying the marker compared to the number of individuals not carrying the marker. markers outside the two horizontal dotted lines are significantly skewed as calculated by the chi-square test at p =  <dig> .\n",
            "\n",
            "\n",
            "the number of each type of marker mapped within each homology group  of q <dig> and the number of linkage groups  formed within each \n",
            "\n",
            "\n",
            "\n",
            "1\n",
            "\n",
            "2\n",
            "\n",
            "3\n",
            "\n",
            "4\n",
            "\n",
            "5\n",
            "\n",
            "6\n",
            "\n",
            "7\n",
            "\n",
            "8\n",
            "\n",
            "unassigned\n",
            "\n",
            "totala genetic linkage map of q <dig> based on single dose markers.  the lgs are formed into the  <dig> hgs using the alleles identified from snp, ssr and rflp markers, hg <dig>   hg <dig>   hg <dig>   hg <dig>   hg <dig>   hg <dig>   hg <dig>   hg <dig> and  unassigned lgs. a sugarcane genetic linkage map for the cv. q <dig> generated from  <dig> individuals from a cross between q <dig> and ij76- <dig>  the numbers on the left of the lg are the cumulative genetic distances in haldane centimorgans. marker names are shown on the right. the lgs are grouped into hg <dig> to hg <dig> and unassigned lgs using the alleles identified from snp, ssr and rflp markers. allelic markers are in square boxes and the dotted lines represent the alignment between the lgs. the brackets are around dart markers that are only different due to missing data.\n",
            "\n",
            "assembly of saccharum homologous groups \n",
            "as sugarcane is an autopolyploid it contains many copies of each chromosome. in order to determine the genome structure of q <dig> the linkage groups  were placed into hgs using a number of different strategies. firstly using the multi-allelic loci detected by single ssr, rflp or snp markers, secondly using sequence information from these markers and thirdly using repulsion phase analysis. the majority of the multi-allelic markers were from ssrs but a major problem encountered with the ssr markers was that only 47% of the  <dig> ssrs derived from sugarcane that were screened across the map, were specific to one homology group. thirty-five of the ssrs had only one sd segregating marker and  <dig>  mapped to more than one homology group. for the  <dig> ssrs derived from sorghum that were mapped, only  <dig>  were specific to one homology group and the majority  only generated one sd marker. nevertheless, of the  <dig> linkage groups formed,  <dig>  could be placed into hgs. these hgs contained from  <dig> to  <dig> lgs, with the lgs varying in length from  <dig>  cm , to  <dig>  cm  . coverage within the homology groups varied from hg <dig> with a total map length of 1818 cm to hg <dig> with a total map length of  <dig>  cm . hg  <dig> also had the most markers mapped with  <dig> markers compared to hg <dig> which contained the least number of markers at  <dig> .\n",
            "\n",
            "chromosome pairing within hgs\n",
            "to determine whether there is preferential pairing behaviour within q <dig> we examined linkage in repulsion of each marker with all other markers in the map. a total of  <dig> lgs were linked in repulsion with other lgs. the amount of repulsion detected varied between hgs, with hg <dig> having  <dig> lgs  linked in repulsion compared to hg <dig> where no repulsion was detected . in the hgs where repulsion was detected, the number of lgs involved varied from  <dig> to  <dig>  out of the  <dig> lgs linked in repulsion,  <dig> pairs of lgs were only found in repulsion to each other. the other  <dig> lgs were in repulsion with more than one other lg. this indicated that in most cases there was preference in pairing between a small number of lgs. a difference ratio was calculated to compare how the genetic distances of the lgs within a hg varied between hgs\n",
            " <cit> . one high coverage lg was selected per hg to compare to all other lgs within the same hg . the average difference ratio varied from  <dig>  for hg <dig> to  <dig>  for hg <dig> table  <dig> \n",
            "repulsion analysis between lgs within hgs\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "max\n",
            "i\n",
            "lg\n",
            "max-\n",
            "lg\n",
            "ik\n",
            "\n",
            "hg1\n",
            "\n",
            "hg2\n",
            "\n",
            "hg3\n",
            "\n",
            "hg4\n",
            "\n",
            "hg5\n",
            "\n",
            "hg6\n",
            "\n",
            "hg7\n",
            "\n",
            "hg8\n",
            "\n",
            "\n",
            "intrachromosomal rearrangements within the homology groups\n",
            "hg1\n",
            "hg <dig> contained  <dig> lgs which were grouped using  <dig> multi-allelic ssrs and  <dig> multi- allelic snp markers. all of these multi-allelic markers were only mapped in hg1; there was only one allele from an ssr that mapped to another hg . in general the lgs within the hg were syntenic to each other with the order of the markers highly conserved. only hg1lg <dig> appeared to have a change in marker order at the end of the lg compared to other lgs within the hg with markers between m77i and m8a inverted.\n",
            "\n",
            "hg2\n",
            "this was one of the largest hgs with a total of  <dig> markers and the longest total length of 1818 cm . there were  <dig> multi-allelic markers with a range of alleles from  <dig> to  <dig> in this hg. combined with the sequence information from markers with known sequence this produced a high coverage hg. the order of markers for the lower coverage linkage groups was fairly consistent with minor order changes probably as a result of missing data. there were five lgs  with extensive rearrangements which appear to be recombinant lgs. these chromosomes are on average 52% longer than the rest of the lgs within this hg. this hg group contains the largest number of lgs and from the ssr markers there are two distinct sets of smaller lgs that align to these larger lgs. one set of lgs containing ssrs msscir  <dig>  msmc <dig>  msscir  <dig>  msmc  <dig> and msscir  <dig> and the other set of lgs containing msscir <dig>  msmc <dig>  msscir  <dig>  msc <dig>  this indicates that this hg is composed of s. spontaneum chromosomes aligned to two sets of s. officinarum chromosomes in agreement with the basic chromosome number of these progenitor species.\n",
            "\n",
            "hg3\n",
            "hg <dig> contained  <dig> markers with a total length of  <dig>  cm and an average distance of  <dig>  cm between markers. this hg contained  <dig> lgs which were aligned with  <dig> multi-allelic ssrs and snp markers . the overall order of markers was largely consistent between lgs.\n",
            "\n",
            "hg4\n",
            "hg <dig> contained a total of  <dig> markers covering  <dig>  cm which gave an average distance of  <dig>  cm between markers . this hg contained  <dig> lgs assigned to the hg group using  <dig> multi-allelic markers with three densely mapped lgs and a number of smaller lgs. marker order was maintained across the majority of linkage groups although some changes were seen. for example msc <dig> on hg4lg <dig> was in a different position relative to hg4lg <dig> \n",
            "\n",
            "hg5\n",
            "hg <dig> contained  <dig> markers that covered a distance of  <dig>  cm with an average distance of  <dig>  cm between markers . this group contained  <dig> lgs assigned to the hg with  <dig> multi-allelic markers. the lgs ranged from  <dig>  cm with  <dig> markers to  <dig>  cm with  <dig> markers.\n",
            "\n",
            "hg6\n",
            "hg <dig> contained  <dig> markers on  <dig> lgs which covered  <dig>  cm with an average distance between markers of  <dig>  cm . this hg contained  <dig> multi-allelic markers and in general the marker order was consistent between lgs. hg6lg <dig> contained  <dig> markers at the end of the linkage group that also aligned to hg <dig>  this appears to be a translocation between the two hgs. there is a 10 cm gap between the groups of markers but the groups were formed at a lod of  <dig> giving strong evidence that the linkage is real .\n",
            "\n",
            "hg7\n",
            "hg <dig> contained  <dig> markers which covered  <dig>  cm with an average distance between markers of 5 cm. this hg consisted of  <dig> lgs. again there were fewer multi-allelic markers but from the  <dig> that were present in the group the marker order was consistent between lgs .\n",
            "\n",
            "hg8\n",
            "hg <dig> contained  <dig> loci which covered  <dig>  cm giving an average of  <dig>  cm between markers. this homology group contained  <dig> lgs formed using  <dig> multi-allelic markers. this hg contained a larger number of low coverage lgs but in general marker order was consistent between lgs .\n",
            "\n",
            "comparison to the r <dig> genetic linkage map\n",
            "using common ssr markers, this current q <dig> map can be partially aligned to the r <dig> genetic map\n",
            " <cit> . for most of the hgs in the q <dig> map each hg aligns to a single hg in the r <dig> map\n",
            " <cit>  . the major difference was that hg <dig> in the q <dig> genetic map was missing in the r <dig> genetic map. there were only  <dig> ssrs in common between the q <dig> hg <dig> and the r <dig> map and in all  <dig> cases, alleles from these ssrs mapped to two hgs. the ssrs markers msscir <dig> and msscir <dig> mapped to both hg <dig> and  <dig> in the q <dig> map and to hg vi in r570; the latter aligns to hg <dig> in the q <dig> map. the third marker  mapped to hg <dig> and  <dig> in the q <dig> map and mapped to hgi in the r <dig> map; hg <dig> aligns to hg <dig> in the q <dig> map. these incongruous markers could be explained by the mapping of duplicate loci in the two different cultivars which are polymorphic at different loci. multi allelic markers that map to more than one locus can create problems for the assignment of lgs to hgs in sugarcane, but it is also possible that in a complex polyploid like sugarcane there are differences in chromosome composition between cultivars. multi-locus markers are common in sugarcane; in the present study 22% of the ssrs mapped in q <dig> mapped to two different loci. this phenomenon has also been observed in diploids\n",
            " <cit> .table  <dig> \n",
            "alignment of q <dig> hgs with r <dig> hgs\n",
            " <cit> \n",
            "using the ssr markers in common\n",
            "\n",
            "\n",
            "\n",
            "1total number of alleles in brackets which were mapped from the ssrs in common\n",
            "\n",
            "\n",
            "2the number of ssrs and alleles in square brackets that map to different hgs.\n",
            "\n",
            "\n",
            "\n",
            "discussion\n",
            "sugarcane dart markers\n",
            "this is the first sugarcane genetic map published which contains dart markers and our results demonstrate that sugarcane dart markers are of high quality and can be used to substantially enhance an existing genetic linkage map. the dart markers generated here integrated into the existing linkage map, demonstrating that they behave in a mendelian manner. there was a higher level of redundancy observed for the dart markers than for any of the other marker types but the dart array used in this study was not filtered for redundancy. in total, 50% of the dart markers were clustered within 10 cm regions across the genetic map but the location of these clusters corresponded with the location of clusters of other types of markers. this has been seen in previous dart genetic maps for sorghum and oat\n",
            " <cit> . these high density clusters could correspond to the centromeric regions of chromosomes where large physical distances correspond to small genetic distances, a feature that has been seen before in sorghum\n",
            " <cit> . in sorghum, the closest diploid relative of sugarcane, the pericentromeric heterochromatic regions of the chromosomes have been shown to have much lower rates of recombination  compared to euchromatic regions \n",
            " <cit> . it is also possible that some clusters represent additional regions where chromosomal rearrangements have occurred or where there is introgression from s. spontaneum causing a higher frequency of polymorphism and hence mapped markers.\n",
            "\n",
            "q <dig> genetic map\n",
            "this genetic map is a substantial improvement on previously published maps with more than twice as many markers compared to any other sugarcane genetic map. the high number of markers of known sequence contributed by the dart, est-ssr, and snps, allowed the majority of the lgs  to be condensed into eight hgs that align to the basic chromosome number of s. spontaneum <cit> . the  <dig> lgs had a cumulative map length of  <dig>  cm and an average marker density of  <dig>  cm. this current linkage map increased the total map length by  <dig>  cm and decreased the average marker density from  <dig>  cm in the previous q <dig> map\n",
            " <cit>  to  <dig>  cm. it is known from cytogenetic analysis that q <dig> has  <dig> chromosomes\n",
            " <cit> . in this study, the number of lgs greatly exceeds this number of chromosomes although  <dig> of the lgs are small with  <dig> or fewer markers . this result gives an indication of the uneven coverage of markers across the genetic map which has been seen in all of the current published maps, whether generated from selfed populations\n",
            " <cit>  or bi-parental crosses\n",
            " <cit> . sugarcane cultivars have over  <dig> chromosomes and are all derived from an initial hybrid between the two different species, s. spontaneum and s. officinarum followed by backcrossing to the high sugar s. officinarum or other cultivars. as a result, the background s. officinarum part of the genome has lower levels of polymorphism than the s. spontaneum part of the genome. in addition, it is well documented that 2n transmission of chromosomes occurs in interspecific crosses of sugarcane\n",
            " <cit> . the introgressed s. spontaneum parts of the genome are the polymorphic regions and in part are responsible for the high density marker areas of the map. the high polyploidy of sugarcane means only a proportion of the markers that are polymorphic can be mapped. any regions or marker alleles present at greater than two doses can not be accurately mapped with the present population size. the resulting q <dig> map has the largest number of markers of any sugarcane linkage map published to date, with  <dig> markers mapped across the genome, more than doubling the previous number of markers. however, this still only generates an average of  <dig> markers per chromosome.\n",
            "\n",
            "marker order\n",
            "marker order was in good agreement between lgs within hg <dig>   <dig>   <dig> and  <dig> . inconsistencies were observed in groups of markers that were closely linked. inversion is a common feature of closely spaced markers\n",
            " <cit>  and could be real, or due to error, or could be explained by the statistical uncertainty of orders at the cm-scale that is inherent in large data sets. in contrast, marker order in hg <dig> and  <dig> was less consistent and could be explained by the fact these two groups contain translocated chromosomes. hg <dig> and  <dig> also appear to have simple translocations with marker order mostly maintained between lgs within a hg. translocations have been identified in other polyploids; including wheat, an allopolyploid which has the well documented translocations between 4a, 5a and 7b\n",
            " <cit> .\n",
            "\n",
            "chromosome pairing\n",
            "in total, repulsion was detected between  <dig>  linkage groups, similar to the level detected in r <dig> where  <dig> lgs  were involved in preferential pairing\n",
            " <cit> . analysis of all pairing frequencies determined that complete disomic behaviour was not present, rather the lgs preferentially paired and often within groups of  <dig> or  <dig> lgs. a cytogenetic study of q <dig> demonstrated that its chromosomal make-up is 15- <dig>  s. spontaneum chromosomes, 11- <dig>  recombinant chromosomes\n",
            " <cit>  and 82- <dig>  s. officinarum chromosomes. in total, q <dig> contains 26- <dig> chromosomes with complete or partial s. spontaneum origin which is a similar number to the  <dig> lg identified with preferential pairing in this study. it is possible that the lgs displaying preferential pairing are the ones that are inherited from s. spontaneum. to determine how similar lgs were within a homology group, a difference ratio in genetic distance was calculated between lgs within a hg where marker intervals were shared\n",
            " <cit> . the number of shared intervals varied between hgs and the accuracy of the difference ratio is probably higher in hg1- <dig> due to the larger numbers of shared intervals . combining this information with the preferential pairing results  showed that hg <dig> had an average difference ratio of  <dig>  and no preferential pairing, whilst hg <dig> had a ratio of  <dig>  with  <dig> lgs involved in pairing. there was a positive correlation between the difference ratio and the percent of lgs detected in repulsion within a hg  with an r2 =  <dig> . the results indicate that within a hg group, as expected, there are two sets of lgs, each likely to be inherited from a different species. furthermore, the hgs where these two sets of chromosomes are more distinct, for example hg <dig>  showed more preferential pairing within and not between the two sets of lgs. in contrast, for hg <dig> where the lgs are more similar to each other, there is no preferential pairing.\n",
            "\n",
            "comparison to r570\n",
            "the majority of the q <dig> hgs could be aligned with the r <dig> hgs using a total of  <dig> allelic makers from  <dig> ssrs . the q <dig> map forms  <dig> hgs aligning to the basic chromosome number of s. spontaneum. in the case of hgs  <dig> and  <dig> for q <dig> and viii and vi in r <dig> there appear to be more lgs than in the other hgs\n",
            " <cit>  table \n",
            " <dig>  this could be because two sets of s. officinarum chromosomes are aligned with one set of s. spontaneum chromosomes which are the result of simple fusion\n",
            " <cit> . this was first indicated in the r <dig> rflp map\n",
            " <cit>  and verified in the more recent genetic map\n",
            " <cit> . the r <dig> map is formed into  <dig> hgs and when aligned to the q <dig> map appears to be missing hg <dig>  hg <dig> in the q <dig> map has  <dig> multi-allelic ssr and snp markers and an additional  <dig> single snp markers. from the location of the ssr markers smc <dig> and sm <dig>  hg <dig> appears to contain a translocation from hg <dig>  it is possible that the r <dig> map lacks this hg <dig> because these duplicated ssrs have lead to the mis-assembly of sets of homologous lgs into the same hg\n",
            " <cit> . it is likely that there will be complex structural differences between the basic chromosome sets of s. officinarum and s. spontaneum which may have lead to misassembly of hgs in both the r <dig> and q <dig> maps. in the q <dig> map there appears to be at least  <dig> lgs  with this same translocation providing strong evidence for its existence .\n",
            "\n",
            "CONCLUSIONS\n",
            "we have demonstrated that sugarcane dart markers provide high quality markers that can be used to construct sd genetic maps in polyploid sugarcane. the high numbers of dart markers generated in a single assay which are distributed over the whole genome offers a real advantage for a range of molecular breeding and genomics applications. as sugarcane cultivars have approximately 100- <dig> chromosomes, large numbers of markers are needed to generate a useful genetic map. the highly parallel and automated platforms used in dart generation mean that the cost per data point is very low. the  <dig> markers in this genetic map allowed 92% of the lgs to be placed into  <dig> hgs conforming to the basic chromosome number of the ancestral species, s. spontaneum. the large number of ssr markers on this genetic map provides the opportunity for the first time to compare genetic maps derived from different sugarcane cultivars. the cross-compatible markers used to generate the map provide the basis for further studies of comparative structural genomics within cultivated sugarcane varieties and within the saccharum genus. the use of dart markers will allow the development of consensus genetic maps in sugarcane which would improve genome coverage and allow integration with other genomic resources. a high coverage genetic map is essential for the correct assembly of the sugarcane genome sequence which a number of research groups are generating.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "a genetic map of s. spontaneum was generated using  <dig> single dose markers and  <dig> multi-allelic markers with a total length of  <dig>  cm and an average distance of  <dig>  cm.\n",
            "a genetic map of s. spontaneum was generated using  <dig> single dose markers with a total length of  <dig>  cm and an average distance of  <dig>  cm.\n",
            "a genetic map of s. spontaneum was generated using  <dig> single dose markers with a total map length of  <dig>  cm and an average distance of  <dig>  cm between markers.\n",
            "a genetic map of s. spontaneum was generated using  <dig> single dose markers with a total length of  <dig>  cm and an average distance of  <dig>  cm between markers.\n",
            "a genetic map of s. spontaneum was generated using  <dig> single dose markers with a total map length of  <dig>  cm and an average distance of  <dig>  cm between markers.\n",
            "a genetic map of s. spontaneum was generated\n",
            "49\n",
            "[{'summary_text': 'the purpose of this study is to show that serial skeletal muscle tractography in chick embryonic hind limb can be used as a powerful investigative tool to evaluate embryonic development with a dual-cooling technique, and investigate whether serial skeletal muscle tractography in dti correlates with ex vivo histology from day  <dig> of incubation through hatching.\\ndti can be used to monitor the development of skeletal muscle in chick embryonic hind limb using a dual-cooling technique, and investigate whether serial skeletal muscle tractography in dti correlates with ex vivo histology from day  <dig> of incubation through hatching.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "animal models play a central role and are essential to basic medical research. having an established pedigree of use in developmental biology, transplantation research, and cancer research, the chick embryo provides an in vivo model which is both accessible and economical, as it contains all components necessary for development, excepting heat and oxygen  <cit> . pluripotent stem cells lines have been derived from various sources, including chicken  <cit> . using molecular methods to identify and characterize cell surface markers and enzymatic activity, evaluation of the stem cell differentiation process is an essential step towards successful therapeutic use  <cit> . most molecular imaging methods, and traditional imaging methods such as microscopy, require embryonic sacrifice for ex vivo analysis. magnetic resonance imaging  allows embryonic imaging without sacrifice, and has been used in avian embryo analysis, primarily for evaluating anatomy  <cit> . advanced mri techniques, for example, diffusion-weighted imaging , diffusion tensor imaging , magnetic resonance spectroscopy, and blood-oxygen level dependent imaging  can be used to obtain structural, metabolic, and functional information in addition to anatomic information  <cit> . however, anatomic imaging techniques and, in particular, advanced imaging techniques are challenging to implement in embryonic models due to embryonic motion  <cit> .\n",
            "\n",
            "diffusion tensor imaging  is a method of observing the displacement distribution of water molecules present within an imaging voxel, allowing evaluation of tissue structure at a scale below the spatial resolution of mri  <cit> . fractional anisotropy  is a dti-derived measurement of tissue structural anisotropy ranging from  <dig> to  <dig>  if the diffusion is isotropic , the fa value is  <dig>  if the diffusion is along a single axis , the fa value is  <dig>  tracked fiber length is another dti-derived measurement which represents the average length of a tracked fiber-like tissue structure. evaluation of the displacement distribution provides insight into the structure and organization of tissues, and dti is an established method of studying the microstructure of neural tissue and skeletal muscle  <cit> . a disadvantage of dti for in ovo imaging use is the long imaging duration and consequent data degradation due to subject motion. previous work has shown that mr anatomic imaging, including real-time cardiac imaging using rapid sequence imaging, is possible with embryonic anesthesia  <cit> . more recently, it has been shown that cooling chick embryo from incubation day  <dig> prior to mr imaging allows noninvasive assessment by reducing motion artifact, without significantly affecting normal development  <cit> . the purpose of this study is to show that dti can be used to monitor the development of skeletal muscle in chick embryonic hind limb using a dual-cooling technique, and investigate whether serial skeletal muscle tractography in dti correlates with ex vivo histology from day  <dig> of incubation through hatching.\n",
            "\n",
            "RESULTS\n",
            "we were able to discriminate and draw a region of interest  on the hind limb or hind limb bud on t <dig> weighted images from day  <dig>  beginning with day  <dig>  skeletal muscle fibers of the hind limb could be tracked with dti and the fiber length derived. as shown in table  <dig> and figure  <dig>   and , fractional anisotropy , tracked fiber length, and hind limb length show a progressive linear increase with each successive day of incubation. the skeletal muscle fibers of the hind limb did not show histologic formation until day  <dig> or  <dig>  the point at which the myofibers could be visually discerned  with dti. as shown in table  <dig> and figure  <dig>  the composition of myofibers also showed a progressive linear increase with increasing incubation time as quantified by the percentage of skeletal muscle fiber area in histology specimens .\n",
            "\n",
            "model diagnostics show linear regression parameters for incubation day  dependence of skeletal muscle fa , length of tracked fibers, fiberarea%  and length of hind limb values. ‡: no fibers tracked in dti from day  <dig> to  <dig> \n",
            "\n",
            "the result of correlation analysis between fa and fiberarea%, and between length of tracked fibers  and length of hind limb  is shown in figure  <dig>  fa and fiberarea% have good correlation , as do fiber length and limb length .\n",
            "\n",
            "a single typical example of serially dti-derived skeletal muscle fiber tracking is shown in figure  <dig>  and shows that skeletal muscle fiber length and number increases with each successive day. histologic specimens beginning with incubation day  <dig> are shown in figure  <dig> \n",
            "\n",
            "discussion\n",
            "our data show that dti reflects skeletal muscle fiber formation and growth in hind limb during the incubation period. in contrast to invasive methods of analysis, repeated observations on a single embryo throughout incubation can be implemented using dti and subsequent phenotypic characteristics would be evident at hatching.\n",
            "\n",
            "chick embryo skeletal muscle development has been investigated in detail by developmental biologists  <cit> . multinucleated skeletal myofibers which contain cross-striated myofibrils are formed by the cytoplasmic fusion of mononucleated myoblasts. the myogenic precursor cells delaminate from dermatomyotome of the paraxial mesoderm and migrate to the limb bud, which then proliferate and differentiate into myoblasts. embryonic myoblasts, which are most abundant on day  <dig> and disappear on days 7– <dig>  have short fibers and contain few  nuclei; these were unable to be tracked with dti in this study. myoblast fiber length and density increase as embryonic myoblasts mature into fetal myoblasts  which contain hundreds of nuclei, and then later mature into adult myoblasts prior to hatching <cit> . adult myoblasts located below the basal lamina of adult muscle appear at the late fetal stage and can be activated to divide, differentiate, and form new muscle fibers following injury in postnatal life  <cit> . dti tracking of fetal and adult stage myoblasts was successful, and progressive increase in myofibril length is evident on fiber tracking images.\n",
            "\n",
            "the tracked fiber length of the hind limb in dti was not entirely consistent with the length of hind limb. for this study, we used a simple measurement method, the length of tracked fibers, which is an average value and it is affected by the tractography stopping threshold parameters . therefore, while the dti measured tracking lengths are not necessarily representative of a single myofiber, it can be considered to be a useful relative surrogate given the correlation obtained between fiber_length and limb_length.\n",
            "\n",
            "this study has several limitations. the spatial resolution of  <dig>  t mri limits evaluation of small embryonic structures, particularly during the early incubation period ; spatial resolution would almost certainly be improved with a higher field strength magnet. matching of mri imaging planes and histologic specimens was performed with careful attention to detail but acquiring exact matches is technically challenging and exact matching was not feasible in all cases. the reverse method, used by some researches, would be to use reconstructions of the isotropic or nearly-isotropic 3d mri dataset to match the histological sections, possibly obtaining more accurate imaging-histologic correlation. our dual-cooling method, which reduced the temperature, may influence diffusion measurements  compared to room temperature; we intend to explore this possibility in our next work. furthermore, there may be developmental differences among embryos of the same incubation day, though this could be accounted for by increasing statistical power.\n",
            "\n",
            "given that dti has the ability to evaluate skeletal muscle differentiation and development, it may be possible to monitor the effect of mutations which affect skeletal muscle migration and development  <cit> . dti also has the potential to observe the embryonic development of other fiber-like tissues, such as smooth muscle, myocardium and nerves. in particular, dti has the potential to provide information about white-matter tract integrity within the developing brain and spinal cord. for example, regional dysmyelination effects were able to be measured with dti within shiverer mouse mutant brain tissue  <cit> , and dti diffusion parameters showed changes in rodent spinal cords receiving moderate t <dig> level injury  <cit> . similar effects may be possible to measure within the chick embryo.\n",
            "\n",
            "CONCLUSIONS\n",
            "in conclusion, this study shows that a widely-available  mri system can be used as a powerful investigative tool to evaluate embryonic development with diffusion tensor imaging in ovo from  <dig> days incubation to hatching. we show that acquired skeletal muscle diffusion tensor imaging parameters correlate with histologic specimens from sacrificed embryos. noninvasive evaluation of developmental processes in embryonic models has traditionally been technically difficult and largely restricted to anatomic evaluation. functional imaging with dti mri and other advanced mri techniques permit the evaluation of tissues and cellular processes below the conventional spatial resolution of mri, and is a potentially promising method of improving the understanding of early developmental processes.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the purpose of this study is to show that serial skeletal muscle tractography in chick embryonic hind limb can be used as a powerful investigative tool to evaluate embryonic development with a dual-cooling technique, and investigate whether serial skeletal muscle tractography in dti correlates with ex vivo histology from day  <dig> of incubation through hatching.\n",
            "dti can be used to monitor the development of skeletal muscle in chick embryonic hind limb using a dual-cooling technique, and investigate whether serial skeletal muscle tractography in dti correlates with ex vivo histology from day  <dig> of incubation through hatching.\n",
            "\n",
            "50\n",
            "[{'summary_text': 'in this study, we have generated a reference gene expression data set for non-neoplastic colonic epithelial cells from  <dig> different colectomy specimens with the clinical indication for each case being diverticular disease.\\nin this study, we have generated a reference gene expression data set for non-neoplastic colonic epithelial cells from  <dig> different colectomy specimens with the clinical indication for each case being diverticular disease.\\nthe data are available from affymetrix.com, http://www.affymetrix.com/support/technical/sample_data/exon_array_data.affx,  <dig>  and http://www.affymetrix.com/support/technical/sample_data/exon_array_data.issue-copyright-statement© the author 2015\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "colorectal cancer  is considered to develop as a result of a multi-step progression and accumulation of genetic alterations  <cit> . gene expression profiling using microarrays has been used extensively in attempts to identify the genetic events associated with this progression. for colon cancer, expression profiling can readily discriminate between tumor and normal tissue as well as distinguish between tumors of different histopathological stage and prognosis to some extent  <cit> . it is anticipated that the identification of these gene expression differences may lead to a better understanding of neoplasia, help identify diagnostic and prognostic biomarkers, and lead to the discovery of new therapeutic targets. as with any microarray study, however, tissue heterogeneity is a confounding factor. previous studies on crc have often used \"adjacent normal colon\" as a control cell type. however, this tissue usually represents a mixture of cell types which do not accurately represent the epithelial cell lining of the luminal surface of the colon that gives rise to crc. clearly, a comprehensive and representative gene expression profile exclusively from normal colon epithelial cells would be a great benefit for comparison with tumor cells, especially since these cells are not readily cultured.\n",
            "\n",
            "here we report a reliable expression reference data set for non-neoplastic colonic epithelial cells. these cells were derived from fresh, unfixed, clinically excised non-tumor related colectomy specimens and enriched using magnetic beads coated with a colon cell specific antibody. rna from these preparations was used to probe affymetrix genechip® exon  <dig>  st arrays. these arrays not only report gene expression levels but can also identify differential splicing events.\n",
            "\n",
            "the findings from this study represent the first comprehensive expression profile for non-neoplastic colonic epithelial cells. we have compared the data derived from these colonic epithelial cells to a publically available set of tumor and normal matched colon data that had been hybridized to the same arrays. the dataset is available from affymetrix.com http://www.affymetrix.com/support/technical/sample_data/exon_array_data.affx. this analysis allowed an assessment of global gene expression alterations in the tumors and demonstrated that adjacent normal tissues are not always the optimal control.\n",
            "\n",
            "alternative splicing  is the process by which the exons of primary transcripts  can be spliced in different arrangements to produce structurally and functionally distinct mrna and protein variants. alternative splicing leads to massive diversity by generating multiple different mrnas from a single template. current estimates indicate that up to 95% of genes in the human genome exhibit alternative splicing  <cit> . furthermore, alternative splicing has increasingly been acknowledged as critical during normal development. previous studies of alternative gene splicing have, for the most part, focused on the identification and characterization of the control of splice site selection on a gene-by-gene basis. while these studies have provided extensive information about the factors and interactions that control formation of the spliceosome, relatively little is known about the global regulatory properties of alternative splicing  <cit> .\n",
            "\n",
            "in this analysis, we also compared as events in tumors and the enriched normal colon epithelial cells. this analysis of splice variants on a genome-wide scale, however, is very labor intensive, and requires careful supervised examination of the data. it is anticipated that the contribution of this set of data derived from pure colonic epithelial cells will facilitate future studies in colon-related disease and offer a unique baseline for studies aimed at elucidating the mechanisms of alternative splicing.\n",
            "\n",
            "RESULTS\n",
            "one of the main objectives of this study was to generate a reference gene expression data set for normal colonic epithelium which can be used in comparisons with tumor samples. non-adenomatous and non-neoplastic colonic epithelial cells were procured from  <dig> different colectomy specimens with the clinical indication for each case being diverticular disease. of the  <dig> patients,  <dig> were female and  <dig> male, with the ages ranging from  <dig> to  <dig> . the surgeries were performed by  <dig> different surgeons with four of the specimens excised by hand assisted laparoscopy and the other  <dig> performed by the conventional open surgical approach. the procurement protocol was designed to effectively enrich for the selected colonic cell population in the shortest time possible to control for variables that may influence the quality and integrity of clinical biospecimens, as described in the materials and methods section. the data discussed in this publication have been deposited in ncbi's gene expression omnibus and are accessible through geo series accession number gse <dig> http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse19163\n",
            "\n",
            "demographics of patients from where the enriched samples of non-neoplastic colonic epithelial cells were derived. ** indicate those samples on which replicates were performed.\n",
            "\n",
            "gene expression\n",
            "twelve samples of isolated colonic epithelial cells from  <dig> different patients  were analyzed using the exon  <dig>  st arrays. these data were then compared with a publically available data set http://www.affymetrix.com/support/technical/sample_data/exon_array_data.affx,  <cit>  which provided raw exon <dig>  st data for ten colon tumors and matched 'normal 'tissues from the same patients. total rna from these samples were purchased from biochain institute inc. all of the raw data was imported into partek and the arrays normalized using quantile normalization followed by probeset summarization using rma. pca analysis for these data sets is shown in figure 1a, where it is evident that the normal and tumor tissues from the public data set clustered together, while the cell samples obtained from our enrichment procedure separated distinctly from the two other tissue sources. a 3-way anova identified that a major source of variance was due to the sample processing differences between the  <dig> laboratories, referred to as \"scan date\" . patient \"sex\" was also noted as a major source of variance which was primarily due to the fact that the tumor/normal colon data set consisted of  <dig> males and  <dig> females, while our analysis used samples from  <dig> females and  <dig> males which slightly skewed the analysis for this category. to correct for these batch effects, therefore, we removed two of these sources of variance and reanalyzed the data where it is clear that the  <dig> types of tissues now cluster more closely  although the cells still form a discrete cluster while the tumor and normal tissues cluster together in no defined pattern. figure 1d displays the anova histogram showing that the variation attributable to scan date and sex have been removed.\n",
            "\n",
            "following this corrective processing, the various cut-off values for expression differences were set at p =  <dig>  for the experimental sets compared to normal and combined with fold change values of  <dig>  genes showing differential expression using these analysis parameters are shown in the venn diagram in figure  <dig>  this analysis reveals that there are more changes in the comparison between the isolated epithelial cells and the tumor than there were in the comparison between the matched tumor and normal tissue. genes showing the highest increased expression levels in the tumor vs cell comparison were olfm <dig>  fn <dig>  actg <dig>  igj, acta <dig>  col1a <dig>  sparc, rcn <dig>  col3a <dig> and col1a <dig>  the genes with the most significant decreases included tmigd <dig>  otop <dig>  ca <dig>  zg <dig>  ms4a <dig>  guca2b, best <dig>  cd <dig>  hrasls <dig>  ifit <dig> and muc <dig> \n",
            "\n",
            "the data was then analyzed using ingenuity pathways analysis . one of the top canonical pathways associated with the deg list generated from the comparison of tumor to cells was the wnt signaling pathway. fifty eight genes from our analysis of tumor compared to cells were deregulated in this pathway consisting of  <dig> members in total  and has  <dig> molecules fitting into the network. it is apparent from the network ideogram that all molecules interact either directly or indirectly through the cdkn1a gene.\n",
            "\n",
            "the most significant biological function associated with disease and disorder category was cancer with  <dig> of the molecules in the comparison of tumors to epithelial cells fitting into this category with an associated p-value of  <dig> e- <dig>  the most significantly altered subcategories within this functional category included; tumorigenesis , neoplasia , cancer  colorectal cancer \n",
            "\n",
            "the most significant  molecular and cellular function was cell growth and proliferation, a category which describes functions associated with cell growth including colony formation proliferation and outgrowth of cells. six hundred and thirty nine of the genes from the deg fit into this cellular function category. the most significant  physiological function was tissue development with  <dig> genes from the deg list fitting into this functional category. this category includes functions associated with development and differentiation and the formation of tissues through the association of cells including growth, patterning and survival of tissues, accumulation, adhesion and clustering of cells.\n",
            "\n",
            "the comparison of tumor to normal tissue showed fewer gene expression differences with much lower fold changes. the highest increases in gene expression included, lgr <dig>  cst <dig>  spp <dig>  cldn <dig>  mmp <dig>  inhba, thbs <dig> sfrp <dig>  etv <dig> and slc6a <dig>  the most significantly down-regulated genes included slc26a <dig>  guca2a, zg <dig>  clca <dig>  myh <dig>  aqp <dig>  tmigd <dig>  ca <dig>  clca <dig>  anpep and adh1c.\n",
            "\n",
            "the most significant  canonical pathway associated with the deg list generated from the comparison of tumor to normal tissue was the hepatic fibrosis/hepatic stellate cell activation pathway. interestingly, this was also one of the top  <dig> pathways found associated with the tumor vs cell comparison. eleven genes of a possible  <dig> genes in this canonical pathway were altered in the comparison between tumor and normal tissues. this most significant network generated from this comparison was cancer, cell movement, hematological system development and function. the score for this network was  <dig>  and is shown is shown in additional file  <dig>  it is apparent from the network ideogram that all molecules interact either directly or indirectly through the tnf or il1b genes, although these genes themselves do not show altered expression in the tumors compared to normal tissues.\n",
            "\n",
            "the most significant biological function associated with disease and disorder category was cancer and  <dig> of the molecules in the comparison of tumors to cells fit into this category with an associated p-value of  <dig> e- <dig>  the most significant  molecular and cellular function was cellular movement,  <dig> of the genes from the deg fit into this cellular function category. the most significant  physiological function was tissue morphology with  <dig> genes from the deg list fitting into this functional category. this category includes functions associated with development and differentiation and the formation of tissues through the association of cells including growth, patterning and survival of tissues, accumulation, adhesion and clustering of cells.\n",
            "\n",
            "thus, from this analysis, it is clear that the altered gene expression profiles depend on the origin of the normal sample. despite the variation in the analysis between the tumor and different normal samples,  <dig> genes showed the same changes regardless of the normal sample used . this analysis strategy demonstrates an important application of this data because it verifies the gene expression changes that occur in both isolated cells and hetergenous normal tissue, while omitting the transcript alterations that are attributable to tissue heterogeneity alone. the genes showing the highest expression changes in tumor compared to both normal tissues and cells were cldn <dig>  cst <dig>  kiaa <dig>  mmp <dig>  rfc <dig>  mthrd <dig>  znf <dig>  mmp <dig>  inhba and psat <dig>  genes showing the largest decreases in expression included; guca2a, kpna <dig>  slc26a <dig>  plac <dig>  anpep, slc26a <dig>  fabp <dig>  dhrs <dig>  slc4a <dig> and tspan <dig> \n",
            "\n",
            "ipa analysis of the data showed that the canonical pathway with the highest number of members from this list of  <dig> overlapping genes was the bladder cancer signaling pathway with a significance value of p =  <dig> e- <dig>  only  <dig> members of the  <dig> were deregulated in a pathway consisting of  <dig> molecules. the wnt signaling pathway was also in the top  <dig> canonical pathways with a p value of  <dig> e- <dig>  the lower p values in this comparison is expected because the list of genes is very limited.\n",
            "\n",
            "the top network generated from this deg list is cancer, cell growth and proliferation, cellular movement and is shown in additional file  <dig>  this pathway has an associated score of  <dig> . the majority of the genes showing down-regulation interact with il <dig>  tnf and tgfb <dig>  while genes showing increased expression show direct interactions with hnf1a.\n",
            "\n",
            "the biological functional analysis showed that the most significant  disease and disorder category generated by this list of  <dig> overlapping genes was cancer. it is noteworthy this was also the most significant generated by tumor vs cell comparison. the most significant  molecular function was cell death, a functional category associated with cytolysis, necrosis, survival and recovery of cells. twenty one genes from the list of  <dig> fit into this category, of which  <dig> demonstrated up-regulation. the most significant  physiological function identified was tissue development with  <dig> molecules fitting in this functional category. four of these were involved in the accumulation of cells and were all up-regulated in the tumors. an additional  <dig> up-regulated genes were associated with formation of the endothelial tube and an additional  <dig>  were associated with cell adhesion.\n",
            "\n",
            "exon-level analysis\n",
            "since the st arrays not only report gene expression levels but also splicing differences, we reanalyzed the data to identify splice variants between the various samples. a major complication in the analysis of the as data, however, comes from the differential detection of transcription and mrna processing. if the hybridization intensities for the individual probesets are examined, and these differences are used as an index of splicing, incorrect conclusions will be drawn because the intensities are not considered within the context of the entire transcript. as shown in figure 3a, the average intensities for the probesets representing the exons of the gem gene show a high p-value  for alternative splicing in a comparison between the tumors and cells, but the entire transcript has an overall increased expression in the tumors compared to the cells . to overcome this complication, we devised a more strict way of detecting as, which was designed to decrease the false positive rate, although it may also have eliminated some possible as events. we also limited our analysis to a comparison between the tumors and isolated epithelial cells . the exon-specific probesets required p values for alternative splicing of < <dig> , the p value for whole transcript expression was > <dig>  and the fold change cut off for whole transcript expression had to be < <dig>  and >- <dig> . in this way we eliminated whole transcript expression while detecting exon-specific changes and defined  <dig> genes showing alternative splicing. each of these genes was then examined visually using partek genomics suite gene view. the full data set is available in additonal file  <dig> \n",
            "\n",
            "to evaluate the utility of our approach, we selected a subset of genes that were known to be alternatively spliced as recorded in the ucsc genome browser http://genome.ucsc.edu/. upon visual examination many of the genes included in our analysis failed to display convincing as events in genes known to be alternatively spliced, highlighting the need for more careful examination of the data. however, we found a number of genes that displayed convincing alterative splicing in genes with known as. table  <dig> describes the genes that we have selected. a few examples of these are shown in figures 3b, 3c, 3d, 3e and 3f. the average intensities of the probe sets representing the osbpl1a gene are shown in figure 3b. the tumor tissue expresses the full length isoform b while the cells express the truncated isoform a version. another example is shown in figure 3c which displays the average intensities for the probesets representing the exons of the prkcsh gene. the tumors express isoform  <dig> of this gene, while the colon epithelial cells typify isoform  <dig>  figure 3d shows that the overall expression level for pycard is higher in the tumor tissue which expresses the α-isoform of this gene, while the cells clearly express the β isoform. figure 3e shows the intensity plot for phf <dig> where the tumor expresses the truncated 3' exon and the cells express the full length exon. this gene has over  <dig> different alternative splice variants recorded. an example of alterative 5' exon expression is shown in figure 3f in the dalrd <dig> gene.\n",
            "\n",
            "the genes were selected by limiting the p-values for alternative splicing to <. <dig>  to eliminate genes that had whole transcript level differences the p value for gene expression was set at p >  <dig>  and a fold change of < <dig>  or >- <dig> . the individual probesets for the genes were then visually examined and those with corresponding reports of alternative splicing events in refseq were included in the analysis.\n",
            "\n",
            "of the  <dig> genes apparently showing alternative splicing, approximately 60% showed as events involving exons that had not been reported in refseq. these data demonstrate the incompleteness of our current understanding of the extent of as. for example,  in refseq there is only one representative transcript for the flot <dig> gene with no alternative splicing. the h-invdb , released in december  <dig> http://www.h-invitational.jp/hinv/ahg-db/index.jsp however, describes  <dig> variants of flot <dig> . interestingly, none of these variants have the same profile of the patterns shown in figure 4a, where the tumor appears to have no expression of 5th and 6th exons from the 3' end of the gene. there are  <dig> transcripts that are missing those exons but none that have retained the 5' exon. since this gene shows high expression levels, it is possible that the 5' probes could exceed the saturation threshold and therefore the array is not capable of discriminating a loss at the 5' end.\n",
            "\n",
            "in the same way, the nes gene  apparently shows differential expression of  <dig> probe sets in the last 3' exon and, while refseq reports a single variant of this gene, h-invdb reports  <dig> variants. the refseq variant is shown in figure 4b insert in blue. the region on the 3' exon that shows higher expression in the tumors corresponds to the first and last transcripts reported by h-invdb.\n",
            "\n",
            "another interesting example is shown in figure 4c where exons  <dig>   <dig> and part of  <dig> of the c14orf <dig> gene appear to be differentially spliced between the two tissue types. refseq details a single transcript for this gene, while h-inv reports  <dig> alternatively spliced variants. there are currently no known variants that would explain the differences in the profiles for the two tissues. figure 4c shows  <dig> transcripts reported in h-inv for the c20orf <dig> gene, none of which correspond to the probeset which shows differential splicing between the colon epithelial cells and the tumor tissues. the circled transcript closely approximates what is seen on the array but the actual probeset maps within an expressed sequence.\n",
            "\n",
            "in many cases, the probesets would have alternative splicing events in exons or introns that were not known to be alternatively spliced, although other regions of the genes were reported in refseq as as events. these examples further highlight the complexity of as. figure 5a shows the average intensities for the probesets representing the exons of the cecr <dig> gene. the expression intensities for the last  <dig> exons are very similar but values for exons  <dig> and  <dig> show higher values for the tumor. exon  <dig> shows similar intensities between the two tissues and they diverge at the 5' exon where the cells show an alternative 5' isoform which has been reported in refseq. the h-invdb reports  <dig> alternative splicing events . the arrow shows the most likely transcript  whereby the tumors have retained exons  <dig> and  <dig> but the 5' exon is missing. the cells appear to be expressing the hit <dig> transcript that is missing exons  <dig> and  <dig> but has retained exons  <dig> and  <dig>  one caveat with this scenario is that this transcript also does not express exon  <dig>  there are differences between the expression of these two exons between the two tissue types whereby the tumor has a mean relative intensity of  <dig>  and the cells have a mean intensity of  <dig>  indicating that this is the transcript expressed by the cells or there is another variant of this transcript not yet reported.\n",
            "\n",
            "clearly, the data obtained from exon array analysis is very complex and requires in depth analysis of putative as events. the availability of this data set which serves as a normal control in the analysis of colon diseases, will potentially help improve our understanding of the complex phenomenon of as.\n",
            "\n",
            "discussion\n",
            "one of the challenges in whole transcript profiling has been to use the appropriate cell type in comparisons of tumor to normal cells. in this study, we present a dataset that we feel can serve as a normal control in the analysis of abnormal colon epithelium at the level of gene expression, and as a reference for as profiles using the affymetrix exon <dig>  st arrays. the recent availability of arrays that allow the detection of as events on a genome-wide scale, suggest that detection of gene expression at the transcript level alone may completely overlook gene expression changes at the level of splicing that are germane to the disease state.\n",
            "\n",
            "the advantage of this particular data set stems from the fact that the profiles are derived exclusively from normal epithelial cells of the colon with no stromal contaminants. in colorectal cancer a gradual progression and accumulation of genetic alterations from the \"normal\"/non-neoplastic state to malignancy has been described. to date, it has not been possible to progressively propagate normal enterocytes in cell culture, making gene expression studies difficult  <cit> . caco- <dig> cells have been used as normal controls for many studies, based on their \"normal\" appearance. the similarity to normal epithelial cells stems from phenotypic similarities to in situ enterocytes based on the acquisition of structural and functional polarity  <cit>  but it has since been shown that expression characteristics similar to colon cancer have been observed in these cells  <cit> . mucosa from fresh tissue specimens has also been used as a substitute for normal colonic epithelial cells  <cit> , although microscopic examination of these samples clearly demonstrates that they represent a heterogeneous collection of cells with varying proportions of lympocytes scattered from the proximal to distal end of the colon. clearly, these cells will compromise the analysis because of the contribution of the stromal components. in the latter, there is a preponderance of fibroblasts, the cells involved in the host's \"desmoplastic\" response to the invading tumor.\n",
            "\n",
            "the approach described here has been previously proven to enrich for viable, unfixed colonic enterocytes and profiled them using the affymetrix genechip exon st  <dig>  array. we used the publicly available colon paired tumor/normal set to demonstrate the utility of this data. analysis at the transcript level allowed a confirmation of genes that were differentially expressed between the tumor and normal tissues. this approach to the analysis of expression data may be an important application for this data set, as it verifies the gene expression changes in tissues while omitting those transcript alterations that are attributable to tissue heterogeneity. of the  <dig> genes that showed expression alterations between the tumors compared to normal tissues and confirmed in the enterocytes, the most significant network identified was cancer, cell growth and proliferation and cellular movement. the majority of genes show interaction with tnf or tgfβ <dig> or both. interestingly, both have been reported to be involved in the epithelial-mesenchymal transition  which characterizes the progression of many carcinomas, including colon cancer. bates and mercurio  <cit>  demonstrated that tgfβ1-induced emt is accelerated dramatically by the presence of activated macrophages, identified tnf as the critical factor produced by macrophages that accelerates the emt in a model of colon cancer.\n",
            "\n",
            "the exon  <dig>  st array does not detect actual transcripts but the expression levels of individual exons, which are then reconstructed into transcripts virtually. the analysis does not provide information about which exons are co-expressed and in which orientation, but instead assembles genes based on the association of exons with that particular locus . the common assumption is that expression levels derived from these whole-transcript arrays are more accurate predictors of expression. this is due to the fact that 3' arrays will typically miscall gene expression alterations in those genes that have alternative 3' splicing.\n",
            "\n",
            "it is perhaps important to appreciate the advantages of using the affymetrix genechip exon st  <dig>  arrays and their ability to define as. as introduces several levels of complexity into the analysis of gene expression alterations, however, several key features have been identified. as in a given gene is not an all-or-nothing event, and is usually a subtle shift in the actual levels of individual exon expression. furthermore, error is introduced with variation in amplification efficiency across individual transcripts. bemmo et al  <cit>  suggest that the high gc content of 5' end, and reduced priming efficiency in the 3' end due to an artifact of random priming results in different signal intensities at the ends of genes and so increases the number of false positives. this can be further compounded by tissue heterogeneity and the fact that splicing patterns can be inherited  <cit> . hybridization efficiency is also affected by the presence of snp within the sequence of the probeset  <cit> . as has also been found to differ as a function of ethnicity  <cit> . one commonly used approach to the analysis of exon array data is to use the splicing index, where the probeset intensity is divided by the entire transcript intensity . this approach suffers from some methodological and statistical drawbacks resulting in high false positive rates . we used a very strict tactic for the detection of as, to decrease the false positive rate but this approach may also eliminate possible as events. the use of anova, where the interaction between the sample type and probeset is used to indicate differential expression of probesets within the framework of whole transcripts, allows the removal of variables and permits a calculation of significance of as in the absence of large fold changes at the transcript level. one caveat with this approach is that it assumes an unvarying response of each probeset within a transcript and this assumption is not met for probesets that are saturated or that hybridize weakly  <cit> . using our stringent criteria for analysis we obtained a list of  <dig> presumed as genes, although, upon visual examination of these genes, very few  actually exhibited convincing alternative splicing in genes with variants recorded in refseq, highlighting the need for careful examination of the data produced by such analyses. most of the as changes occurred in genes not reported to be alternatively spliced in refseq, and often not reported in more highly curated databases such as h-invdb. due to the large amount of est and genomic information, a vast amount of information can be obtained from in-silico prediction of transcript isoforms but many of the available est libraries are cloned from tumor samples and as such the databases tend to lack information on normal splice forms  <cit> .\n",
            "\n",
            "CONCLUSIONS\n",
            "use of the affymetrix genechip exon st  <dig>  and gene st  <dig>  arrays will provide an equitable and empirical platform to access information on transcript variants. this data set is a valuable asset for studies of colon disease. we have demonstrated that, using normalization across array and removal of variation attributed to technical processing, the data can be compared to tumors and normal tissue processed at a different time and place assuring that the data is transportable. the raw data is available at geo.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study, we have generated a reference gene expression data set for non-neoplastic colonic epithelial cells from  <dig> different colectomy specimens with the clinical indication for each case being diverticular disease.\n",
            "in this study, we have generated a reference gene expression data set for non-neoplastic colonic epithelial cells from  <dig> different colectomy specimens with the clinical indication for each case being diverticular disease.\n",
            "the data are available from affymetrix.com, http://www.affymetrix.com/support/technical/sample_data/exon_array_data.affx,  <dig>  and http://www.affymetrix.com/support/technical/sample_data/exon_array_data.issue-copyright-statement© the author 2015\n",
            "\n",
            "51\n",
            "[{'summary_text': 'here, we have investigated the genetic basis of oyster resistance or susceptibility to vibrio infections by high-throughput microfluidic qpcr approaches.\\nour results revealed an extraordinary interindividual polymorphism in basal gene expression that is associated to an inherent capacity of some oysters to survive vibrio infections.\\nin this study, the hemocytes  have been collected from oysters before an infection by virulent vibrio strains isolated during summer mortality episodes in france.\\nhere, we have investigated the genetic basis of oyster resistance or susceptibility to vibrio infections by high-throughput microfluidic qpcr approaches.\\nin this study, we have investigated the genetic basis of oyster resistance or susceptibility to vibrio infections by high-throughput microfluidic qpcr approaches.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "since the mid-1970s, large-scale episodic events such as harmful algal blooms and other microbial outbreaks, disease epidemics and mass mortalities have occurred in marine environments at a historically unprecedented rate, and may reflect environmental changes and an unfavorable energetic balance in animals  <cit> . anthropogenic global changes  have been implicated as contributors to disease outbreaks in marine ecosystems worldwide. one aspect of these changes is shifts in natural systems, altering inter-organism dynamics including many host-pathogen interactions and leading to the establishment of new diseases or increased pathogenicity and virulence of established diseases  <cit> . these emerging infectious diseases have been recently recognized as a threat, not only to humans, but also to global biodiversity  <cit> .\n",
            "\n",
            "it is widely established that the susceptibility of organisms to infection is determined by a complex balance of environmental and host factors. disturbances of this equilibrium may result in multifactorial diseases as currently shown in marine invertebrates  <cit> . pacific oysters crassostrea gigas have repeatedly suffered abnormal mortalities during the summer months around the world. collectively, these mortality events are called summer mortality syndrome, a phenomenon that has been suggested to have a multifactorial etiology, resulting from the combination of multiple biotic and abiotic factors. those factors include the environment , aquaculture practices, oyster genetics and physiology, reproductive effort followed by spawning and the presence of opportunistic microorganisms  <cit> . since  <dig>  an herpesvirus, oshv- <dig>   <cit> , and various vibrio species  <cit>  have been systematically associated with these abnormal mortalities.\n",
            "\n",
            "understanding the role of genetic basis in oyster resistance or susceptibility to infection should deeply contribute to identify the causes of mortalities and the traits that characterize enhanced survival capacities in oyster populations. in recent years, high-throughput genomic approaches have greatly improved knowledge on the genetic basis governing the mechanisms of resistance to infectious diseases, a major challenge for the modern human and veterinary medicine. several studies have supported a significant role for host genetics in differential susceptibility to many human infectious diseases such as tuberculosis, leprosy, hepatitis, malaria and also aids  <cit> . genome-wide studies have evidenced genomic polymorphic regions  correlated to changes in the susceptibility to some infectious agents  <cit> . in the recent years, genomic features such as gene copy number variations  and transcriptional signatures  <cit>  have been extensively investigated as contributing to disease susceptibility or resistance phenotypes. transcriptional  signatures comprise a combination of genes whose expression patterns are markers of a physiological status  <cit> . in human medicine, gene expression signatures have substantially aided in the diagnosis of cancer and the risk of developing active tuberculosis  <cit> .\n",
            "\n",
            "here, we have investigated potential genetic basis for c. gigas resistance to vibrio infections by high-throughput transcriptomic and genomic approaches. for this non model species, the whole genome sequence is still not published  <cit>  but  <dig>  unique est sequences are publicly available   <cit> . thus, we applied the dge  deep-sequencing technology. similar to the massively parallel signature sequencing  approach  <cit> , the dge tag-based approach provides large-scale data sets to establish a complete quantitative and qualitative analysis of cell transcriptome without prior knowledge of genome annotation  <cit> . in a previous study, the dge technology was successfully applied for the characterization of defense mechanisms related to the oyster successful response and survival to virulent vibrio infections  <cit> . here, the aim was to identify a combination of genes whose inherent basal expression can predict the oyster capacity to survive infections. for that, the hemocytes  have been collected from oysters before an infection by virulent vibrio strains isolated during summer mortality episodes  <cit> . further monitoring of mortalities allowed classifying samples with contrasted phenotypes of survival or non survival capacity that have been used for the construction of dge libraries devoted to a predictive character of oysters to circumvent vibrio infection. through the exploration of dge data and individual gene expression study by microfluidic qpcr devices  <cit> , we have evidenced a 14-gene survival signature. besides, we show here for the first time an extraordinary individual polymorphism of hemocyte basal gene expression in oysters.\n",
            "\n",
            "RESULTS\n",
            "hemocyte transcriptome of oysters with an inherent capacity to survive vibrio infections\n",
            "a nonlethal sampling method for hemolymph collection  <cit>  was applied to obtain the hemocyte basal gene expression profile before a challenge by pathogenic bacteria. hemocyte samples were collected from  <dig> tagged oysters that were further experimentally infected after an 8-day recovery. vibrio strains, v. splendidus lgp <dig>  or v. aestuarianus lpi 02/ <dig> , which were isolated during  <dig> summer mortality episodes in france, were used for experimental infections. survival rates were 59 % and 33 %  for animals infected by v. splendidus lgp <dig> and v. aestuarianus lpi 02/ <dig>  respectively. hemocyte mrna samples collected prior to vibrio infection were then categorized according to oyster survival. thus, two dge libraries were constructed from hemocytes of oysters able or not able to survive vibrio infections. they were named svir-8d  and nsvir-8d , respectively.\n",
            "\n",
            "a total of  <dig> , <dig> and  <dig> , <dig> tag sequences  were obtained from the whole sequencing of svir-8d and nsvir-8d libraries, respectively, corresponding to  <dig>  unique tags. a comparison of tag occurrences  revealed that  <dig>  unique tags were differentially represented between libraries  . tag-mapping was performed against the c. gigas est database  containing  <dig>  ests from hemocytes  <cit> . from the  <dig>  est-matched sequences, only  <dig> sequences showed homologies to known proteins. thus, no striking relationship between biological functions and oyster capacity to survive infections was clearly evidenced. however, the svir-8d library appears to be more enriched in genes related to genetic information processing and immune response compared to nsvir-8d, which was enriched in metabolism and cytoskeleton reorganization .\n",
            "\n",
            "oyster hemocyte transcriptome-wide analysis reveals a high polymorphism in basal gene expression at individual level\n",
            "to assess whether gene expression profiles were related to survival capacity phenotypes, we have explored the dge data on individual oysters by high-throughput microfluidic qpcr approach  <cit> . three new independent experimental infections were conducted to obtain individual hemocyte samples from oysters able  and not able  to survive infection, as previously done for dge library construction. we focused on v. splendidus which has been shown dominant in field mortalities, since  <dig>  comparatively to v. aestuarianus. final survival rates of 50 % were obtained and, among the  <dig> hemolymph samples issued from the three independent infections,  <dig> samples, corresponding to  <dig> oysters for each phenotype, yielded high quality rna suitable for further analyses.\n",
            "\n",
            "a total of  <dig> hemocyte-expressed genes were chosen from the  <dig>  sequences differentially expressed  between svir-8d and nsvir-8d dge libraries. both genbank annotated  and non-annotated  sequences were considered. the technical efficiency of three independent  <dig>  dynamic arrays was assessed by measuring the expression of the three reference genes in each qpcr chip . then, the expression profile of the selected genes was assessed to compare their mrna abundance according to the oyster survival phenotypes  at the individual level. surprisingly, the analysis of the large qpcr data set revealed an extraordinary interindividual variability in gene expressions for most of the transcripts measured. as illustrated in figure 2a, striking inter-individual differences in basal gene expression were seen for some genes with up to 20-fold change, for example, for the follistatin. to our knowledge, such a high inter individual variability in basal gene expression has never been previously reported in oysters.\n",
            "\n",
            "additionally, the high-throughput qpcr analyses revealed numerous genes whose basal levels of expression could not be detected in some individuals . interestingly, those genbank annotated sequences appeared to correspond to genes involved in immune responses, including an interferon-induced protein 44-like, a tumor necrosis factor ligand superfamily member  <dig> , a member of oyster lysozymes , a baculoviral inhibitor of apoptosis repeat-containing protein  <dig> homologue  and members of the big defensin antimicrobial peptide family   <cit> . however, no significant relationships could be found between the expression detection of these genes and the s or ns phenotypes .\n",
            "\n",
            "a hemocyte 14-gene expression signature as a predictor of oyster survival capacity\n",
            "we have investigated the correlation between the profiles of hemocyte basal gene expression and the oyster survival capacity by the analysis of the mrna abundance of genes presenting exploitable quantitative values . due to well known oyster dna sequence polymorphism, rigorous melting curve analysis was required to ensure qpcr specificity and avoid misinterpretations of gene expression data  <cit> . thus, we only considered the qpcr data of  <dig> genes, from the  <dig> initial ones, expressed in the 45 s and  <dig> ns individual oysters. in spite of the high gene expression polymorphism, statistical partial least-squares  regression  <cit>  analysis revealed that the basal expression levels of a core set of genes were associated to s or ns phenotypes . to refine our analysis, we further considered only those genes that did not display high individual variability of basal expression among the oysters from a same phenotype. thus, the combination of the mrna abundance of  <dig> genes was identified that comprises a set of  <dig> genes highly transcribed in the s phenotype and a set of  <dig> genes highly transcribed in the ns phenotype .\n",
            "\n",
            "the expression profile of these  <dig> hemocyte-expressed genes  was further used to foresee oyster survival using the expression data from oysters from both survival phenotypes. the x-loadings  corresponding to the data generated by qpcr and the y-loadings  corresponding to the oyster s and ns phenotypes are presented in figure 3b. the applied pls model segregated most of the  <dig> individuals according to their survival capacity  . to identify the hemocyte transcripts statistically related to each phenotype, we additionally applied appropriated fold change cut-offs . mann–whitney test  was performed individually on the  <dig> genes associated to the survival phenotypes. thus, Δcq which are the most reproducible values were processed to assess a significant difference between s and ns survival phenotypes. among the selected candidates, the expression profiles of  <dig> genes appeared to be significantly associated to an inherent capacity to survive infections . thus, we can presume that one individual with high probabilities to survive vibrio infections transcribes at high levels the seven genes associated to the s phenotype, and concurrently at low levels the seven genes associated to the ns phenotype. oysters from the ns phenotype  displayed an inverse pattern in gene expression profile for the survival signature. in the view of our findings, this 14-gene survival signature is able to predict the oyster survival capacity to infections by the pathogenic v. splendidus lgp <dig> \n",
            "\n",
            "mrna abundance of the 14-gene markers is revealing an inherent genetic trait rather than a particular physiological status\n",
            "we confirmed that the survival signature revealed an inherent genetic trait in oysters rather than a pre-stimulation or a physiological response. indeed, high mrna abundance of the genes of the signature could have resulted from an increase in gene transcription for some individuals before the experiment. for that, oysters  were injected with a sublethal dose of v. splendidus lgp <dig>  or sterile sea water . non injected oysters were used as a control. as previously reported, a great variability in transcript rates was observed for both unchallenged and immune challenged  oysters. two main types of expression profiles were obtained among the signature genes . nine genes displayed no changes in transcript abundance at 24 h following either bacterial or ssw injection, compared to unchallenged animals as illustrated for multiple egf-like domains  <dig> and cytochrome c . besides, five genes, including the unknown bq <dig> .a.cg <dig> and zinc finger hit domain-containing protein  <dig> shown in figure  <dig> as examples, displayed a decrease of transcript abundance in circulating hemocytes following both the bacterial and ssw injection, thus independently of an acute response to the vibrio. no up-regulation was seen for any of the  <dig> hemocyte-expressed genes following the bacterial challenge or ssw injection. thus, mrna abundance of the  <dig> genes of the survival signature reveals, in oysters, basal gene expression independent from their physiological response.\n",
            "\n",
            "polymorphism in hemocyte basal gene expression can be associated to variations in gene copy numbers in oyster genome\n",
            "in an attempt to understand the genetic basis involved in the transcriptional regulation of the genes associated to oyster survival, we investigated whether variability in gene expression would reflect gene copy number variations  in oyster genome. a relative quantification approach was developed to verify a potential correlation between gene copy abundance and mrna expression of the oyster survival signature. for that, both genomic dna and total rna were extracted from ten oysters showing striking differences in gene expression of seven genes from the 14-gene survival signature for which pairs of primers present same efficiency and specificity on gdna and cdna. variations in gene copy number were shown for all analyzed genes. higher variations in gene copies  than in expression levels  were seen for four genes . conversely, for the other three genes, the variations in expression were higher  than in gene copy number  . finally, among those seven genes, only two presented positive correlation  between the relative abundance of gene copy and relative gene expression .\n",
            "\n",
            "discussion\n",
            "herein, we successfully identified a 14-gene survival signature, a cluster of hemocyte-expressed genes whose basal expression profile was predictive of the oyster capacity to survive vibrio infections. this study provides the first evidence that oyster survival to infectious diseases can be driven by genetic determinants. we revealed an extraordinary interindividual polymorphism in basal gene expression that is associated to an inherent capacity of some oysters to survive vibrio infections.\n",
            "\n",
            "in previous work, we applied dge to highlight immune functions involved in the successful response of c. gigas to circumvent virulent vibrio strain infections compared to a non virulent one  <cit> . here, the aim was to evidence gene markers of an inherent survival capacity rather than to characterize biological functions or molecular mechanisms implicated in that capacity. we just observed an enrichment of genes related to immune response and genetic information processing in the svir-8d dge library. the comparative analysis of the hemocyte transcriptomes of oysters able  or not able  to survive did not highlight striking functional categories as potentially involved in oyster health promotion and conferring advantageous capacity to survive infections. this is not surprising considering the low number of sequences found with predicted functions . with further progress in the genetic resources in c. gigas, one can return to exploit the dge data and expect to characterize unambiguously biological functions associated with oyster fitness and optimal survival capacity.\n",
            "\n",
            "the comprehensive and detailed quantitative analysis of the dge data at the individual level has shown here for the first time in oysters the remarkable interindividual polymorphism in hemocyte gene expression. it is illustrated by great variations in basal mrna levels up to 20-fold change between individuals. in human, variations in the basal mrna abundance of some key genes represent an important genetic basis underlying several physiological traits that includes resistance  to many known infectious diseases  <cit> . moreover, in many oysters from both survival phenotypes, basal mrna levels of some immune-related genes were not detected. it cannot be ruled out that due to a high level of dna polymorphism, only transcripts encoded by particular alleles could have been amplified  <cit> . indeed, it is widely recognized that oysters are one of the highest polymorphic species in the animal kingdom   <cit> . moreover, most of transcripts whose expression was shown to be absent among individuals corresponded to immune-related proteins, which have been shown in c. gigas to display a high level of sequence diversity  <cit> . another hypothesis is that some of these genes are not expressed in some individuals because of the absence of gene in the oyster genome. this has been shown for the human neutrophil antimicrobial peptide- <dig>   <cit> . since hnp- <dig> was proved to play an important role in infectious and inflammatory diseases  <cit> , it has been suggested that the absence of defa <dig> gene can result in changes in the human susceptibility to pathogenic microorganisms  <cit> . here, probably due to limited sampling number, we did not evidence a statistical correlation between the lack of hemocyte gene expression and the oyster capacity to survive infections, but this result prompted us to further analyze this phenomenon of absence of immune-related transcripts in oyster populations.\n",
            "\n",
            "the existence of a genetic predisposition of oyster to survive vibrio infections was revealed by dge and high-throughput qpcr analyses. performed on independent sets of  <dig> oysters, from different geographical origin and history, these different transcriptomic analyses concurred in the statistical identification of a gene survival signature. indeed, in spite of the high polymorphism of gene expression, the combination of the basal expression levels of  <dig> hemocyte-expressed genes was shown to be correlated to oyster capacity to survive infections by the pathogenic v. splendidus lgp <dig>  this 14-gene survival signature includes  <dig> unknown genes and the potential role of the other annotated genes in the oyster health status and survival cannot be interpreted. however, interestingly, one member of the survival signature, a putative cystatin a, has already been identified as related to the ability of oyster to circumvent virulent vibrio infection in previous genomic study  <cit> . since v. splendidus lgp <dig> and v. aestuarianus are known to secrete extracellular proteases, we can assume that, highly expressed at basal level, cystatin, as protease inhibitor, may interact with the vibrio toxins contributing to oyster survival  <cit> . it is noteworthy that cystatins are a superfamily of multifunctional proteins  <cit>  that requires extensive characterization in oyster before speculating any function or role in survival. recently, a mortality gene expression signature was identified in c. gigas hemocytes of oysters that died in the field during summer mortality events in usa  <cit> . this so-called “mortality signature” comprised a core set of genes involved in cell death, protein synthesis and cellular assembly and organization. their expression increased in individuals previously to the occurrence of massive mortalities in the field  <cit> . besides, fleury et al.  <cit>  also showed in oyster tissues, such as gonads, gills and muscle, transcripts of immune-related genes differentially expressed between two oyster lines selected for their resistance and susceptibility to the  <dig> summer mortality events in france. however, these genes were identified according to an increase of their transcript abundance during various periods preceding a mortality event, revealing a physiological response of the oysters to field conditions. instead, our study has focused on the identification of genes whose basal expression in oyster may reflect an intrinsic character of survival capacity, i.e. a genetically-based trait underlying natural variation in gene expression.\n",
            "\n",
            "in this attempt, it was important to verify that, in our experiments, the high mrna levels of the 14-genes identified as related to survival capacity were not due to a particular physiological status or a priming of the oysters prior to the hemolymph sampling. indeed, priming could confer protection against v. splendidus lgp <dig> infection, as described in various invertebrate species, where pre-stimulated animals were protected against a second and lethal challenge, and in a specific manner  <cit> . here, following microbial challenge or injury , no increase in transcript abundance was observed from the genes of the survival signature. instead, a decrease of transcripts was observed for five of them. such a decrease is either related to a migration of the hemocyte subpopulation expressing those genes towards the site of injection  <cit> , or to a down-regulation of expression following wound injection. besides, nine genes were evidenced as constitutively expressed in circulating hemocytes and not regulated by a microbial challenge. ruling out a priming possibility or an effect of physiological status, these results are in favor of genetic bases driving hemocyte high basal gene expression in some oysters.\n",
            "\n",
            "in our study, copy number variations  in the  <dig> marker genes were evidenced among the individuals tested, as previously reported by schmitt et al.  <cit>  for genes encoding actin and two antimicrobial peptides . it is now established that cnv contributes to differences in gene expression and can be associated to variable phenotypes, including susceptibility to complex genetic disorders and infectious diseases  <cit> . for instance, in human, whereas an elevated number of copies of the β-defensin gene defb <dig> in the east asia population is associated to increased resistance to influenza  <cit> , high α-defensin  gene copies were shown to be significantly correlated with susceptibility to severe sepsis  <cit> . here, we have not found a correlation between cnv and gene expression profile for all of the genes of the survival signature. as elegantly shown by mileyko et al.  <cit> , the basal expression of some genes is not necessarily proportional to the gene copy number in the genome due to transcriptional feedbacks in complex regulatory networks. because oysters are highly polymorphic  <cit> , changes in gene expression can also be associated to a high degree of dna polymorphism in the gene promoter sequence  <cit> . our results indicated that the mrna levels of the different genes from the 14-gene survival signature are likely driven by distinct transcriptional processes.\n",
            "\n",
            "CONCLUSIONS\n",
            "in conclusion, our findings evidenced that oysters possess a complex genomic organization that is highly polymorphic, not only in terms of dna sequence  <cit> , but also in terms of number of gene copies and transcriptional regulation of genes. a deep analysis of the whole hemocyte transcriptome of oysters showing differences in the resistance to vibrio infection led to the discovery of a 14-gene predictive survival signature. currently, in human medicine, such gene expression signatures have considerably improved the diagnosis and understanding of complex genetic disorders and infectious diseases  <cit> . in the context of c. gigas aquaculture, the mass mortality syndrome represents a serious constraint to oyster production world-wide. thus, understanding the genetic basis involved in oyster survival constitutes an important piece in the oyster-pathogen-environment interaction puzzle. although oyster immunogenetics is still an emerging domain that requires great improvement in genomic resources, we showed that gene predictive-survival signature could provide a useful tool for preventing mortality events and for assessing genetic traits of interest for disease resistance selection programs.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "here, we have investigated the genetic basis of oyster resistance or susceptibility to vibrio infections by high-throughput microfluidic qpcr approaches.\n",
            "our results revealed an extraordinary interindividual polymorphism in basal gene expression that is associated to an inherent capacity of some oysters to survive vibrio infections.\n",
            "in this study, the hemocytes  have been collected from oysters before an infection by virulent vibrio strains isolated during summer mortality episodes in france.\n",
            "here, we have investigated the genetic basis of oyster resistance or susceptibility to vibrio infections by high-throughput microfluidic qpcr approaches.\n",
            "in this study, we have investigated the genetic basis of oyster resistance or susceptibility to vibrio infections by high-throughput microfluidic qpcr approaches.\n",
            "\n",
            "52\n",
            "[{'summary_text': 'a. marginale is a member of the order rickettsiales and a well-established obligate intracellular bacterial model.\\na. marginale is a member of the order rickettsiales and a well-established obligate intracellular bacterial model.\\nthis study reveals that interstrain snp diversity does not appear to be influenced by the environmental niche an organism occupies, nor is it generally consistent throughout a specific family or genera.\\na. marginale has a closed core genome, which may result from small evolutionary distance, and may be the result of small evolutionary distance.\\na. marginale is a member of the order rickettsiales and a well-established obligate intracellular bacterial model.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "while the recent boom in genome sequencing projects has provided a wealth of information about bacterial metabolism and evolution, we know little about interstrain variation. a firm understanding of the rates and sites of variation is useful in determining genotypic differences associated with phenotypic traits and in formulating control strategies for a number of pathogens. further, knowledge about the pan-genome of organisms will aid in determining the core genomic requirements, as well as shed more light on events that occur in the various environmental niches bacteria occupy.\n",
            "\n",
            "most studies of bacterial diversity to date have either utilized specific genomic loci  <cit>  or have examined metagenomics of specific environmental niches  <cit> . while these types of studies help elucidate the extent of diversity, there is still a key component that has not yet been investigated – a measurement of diversity within bacterial species. obtaining a true measure of species diversity is difficult, as the strains selected for whole-genome sequencing are generally chosen to examine a particular phenotypic trait, subjecting any resultant measures of diversity to selection bias.\n",
            "\n",
            "the level of interstrain diversity can have a significant impact on the direction of research. selection of pathogen strains for sequencing is typically based on differences in virulence  <cit> , host preference  <cit> , or tissue tropism  <cit> . using these selection criteria may artificially skew the level of diversity in the studied genome sequences, resulting in a biased level of diversity which does not accurately reflect the true genetic diversity of the species. however, since the diversity among strains has only been examined in a small number of species, determining if there is a skew is difficult. for example, analysis of several genome sequences of bacillus anthracis found a low number of single-nucleotide polymorphisms   <cit> , which led to development of other techniques for examining the epidemiology of outbreak strains  <cit> . b. anthracis is an example of a \"closed core\" genome – that is, after sequencing several strains , no strain-specific genes are added to the pan-genome  <cit> , which may be a result of a clonal population split of b. anthracis from b. cereus. thus, the closed-core genome may be the result of small evolutionary distance, and may be a rare finding for organisms with larger evolutionary distance. the alternative is an \"open core\" genome, where each new sequenced strain adds at least one unique gene to the pan-genome. this is exemplified by streptococcus agalactiae, which has approximately  <dig> new strain-specific genes for each additional genome sequenced, regardless of the total number of strains compared.\n",
            "\n",
            "what influences the pan-genome? is the pan-genome content fixed, or does it drift with time? do all non-clonal populations have open-core genomes, or is this influenced by the environment a bacterium occupies? while answering all of these questions will require sequencing many more genomes, anaplasma marginale makes an excellent system for studying the last question for a number of reasons. a. marginale is a member of the order rickettsiales and a well-established obligate intracellular bacterial model. a. marginale is the most globally prevalent vector-borne pathogen of cattle, causing cyclic anemia, decreased production, and possibly death  <cit> . a previous genome sequence for the st. maries strain  <cit>  establishes that this organism has a small genome size due to reductive evolution, and is related to several other intracellular pathogens, including those in the genera anaplasma, ehrlichia, and rickettsia  <cit> . in addition, a. marginale has a number of characterized strains, with each strain defined by msp1α genotype  <cit> . while previous studies have utilized specific genes to examine differences between these strains  <cit> , no studies have examined the species diversity of a. marginale. a number of studies have described strains that vary in geographic location and phenotypic traits  <cit> , and these are available for determination of the true level of genetic diversity in this species, subsequent analysis of the status of core genes, and determination if these are correlated with the intracellular lifestyle, geographic location, tick-transmissible status, or other characteristics of these organisms.\n",
            "\n",
            "to answer this question, we obtained genome sequences for four strains of a. marginale that have differing abilities to be transmitted by dermacentor andersoni, with each phenotype represented by at least two geographically distinct isolations. we sequenced the florida strain to completion using a bac-based clone by clone approach, and obtained high coverage genome sequence data for three additional strains. the resulting dna sequences were then analyzed and compared to both the previously sequenced st. maries genome  <cit> , as well as other bacterial species in the order rickettsiales. further, the genomes of several other non-rickettsial bacteria were examined with similar genome comparison techniques to determine if diversity and pan-genome content are related to pathogenicity or an intracellular lifestyle.\n",
            "\n",
            "RESULTS\n",
            "microbial genome diversity\n",
            "previous studies  <cit>  have shown high levels of variation between the genomes of different strains of obligate intracellular bacteria. to compare the levels found in a. marginale to other genomes, similar comparisons were made for organisms meeting the following criteria: organisms with 1) a single chromosome, 2) more than one sequenced strain, and 3) assembled and finished genome sequences deposited in genbank, including free-living, facultative intracellular, and obligate intracellular bacteria . single-factor analysis of variance  finds no significant differences in the level of variation between obligate intracellular, facultative, and free-living bacteria. the number of snps ranged from  <dig> % to  <dig> % of the larger genome, with significant intraspecies and intragenera variation.\n",
            "\n",
            "general genome features and comparison of the st. maries and florida strains\n",
            "the anaplasma marginale florida strain genome is composed of a single  <dig> , <dig> bp circular chromosome predicted to contain  <dig> coding sequences  . similar to most other previously sequenced anaplasmataceae, there are no plasmids and no identifiable insertion sequences. compared to the previously sequenced st. maries strain genome  <cit> , there are seven fewer cdss despite the larger genome size, due primarily to differences in split open reading frames  and annotation differences. the high degree of synteny between these two strains is disrupted by two inversions; one approximately  <dig> kb long is flanked by repeat elements , while the other is a single gene flanked by short duplicated hypothetical genes.\n",
            "\n",
            "split orfs were first described in the rickettsia conori genome  <cit> , and are postulated to represent genes that are in the first stage of reductive evolution. the idea that these orfs have split recently is consistent with the findings in anaplasma, as different orfs are split in the two completely sequenced strains. the four split orfs annotated in the st. maries genome   <cit>  are intact in the florida genome, and two tandem genes annotated as hypothetical in the st. maries genome  are fused in the florida genome . only one split orf, peta, is found in the florida genome. four small orfs in the st. maries genome , ranging in size from  <dig> bp to  <dig> bp are not present in the florida genome. these orfs are flanked by repetitive dna sequences, and appear to be missing due to recombination events.\n",
            "\n",
            "genes mediating genome plasticity\n",
            "the msp <dig> superfamily is a group of related a. marginale genes encoding surface proteins  <cit> . msp <dig> encodes a highly antigenic protein that varies over time during infection by gene conversion of functional pseudogenes into a single expression site, to create new antigenic variants capable of evading the existing immune response. compared to the st. maries genome, the florida genome has one additional msp <dig> functional pseudogene. of the eight florida msp <dig> functional pseudogenes, four are identical to those in the st. maries genome. the florida genome has two sets of duplicated functional pseudogenes, ttv 4f15/ttv 1o <dig> and kav 4f15/kav 1f <dig> ; while st. maries was found to have duplicated functional pseudogenes, this was not noted in a functional pseudogene-targeted examination of other strains  <cit> . florida has a set of duplicated functional pseudogenes in the same genome positions as st. maries . as obligate intracellular bacteria are not thought to undergo lateral gene transfer, identical functional pseudogenes indicates the sequence is either evolutionarily conserved or has been selected independently in both strains due to a fitness advantage. interestingly, both copies in florida have a change encoding  <dig> amino acids at the 5' end of the hypervariable region compared to their st. maries counterparts; either both strains duplicated a functional pseudogene after this change occurred in an ancestral strain, or both copies in one of the strains acquired identical changes after the ancestral strain duplicated the original functional pseudogene. in contrast, only two of the seven msp <dig> functional pseudogenes are identical between florida and st. maries . the omp1- <dig> genes are present in both genomes, with a high degree of conservation between the predicted amino acid sequences  as previously reported  <cit> .\n",
            "\n",
            "aaap gene family\n",
            "the aaap gene was first recognized and characterized as an anaplasma appendage associated protein  <cit> . subsequently, additional related genes were identified that appear to be tandemly-duplicated copies that have diverged to have relatively low levels of sequence identity . there is expansion of this locus in the florida strain relative to the st. maries strain, with a duplicated copy of the aaap gene. because of the repetitive nature of this gene family, these sequences tend to be missing from pyrosequenced genome assemblies; therefore, we examined the status of this locus in several the strains via southern analysis, revealing that this locus is highly plastic both within and between strains .\n",
            "\n",
            "high density sequence coverage of additional strains\n",
            "an additional two transmissible strains  and one non-transmissible strain  were subjected to genome-scale pyrosequencing  <cit>  , which provided at least 96% genome coverage when compared to either florida or st. maries . most of the missing sequences corresponded to repetitive regions   , and reflects the limitations of assembling short sequence reads  without additional scaffolding. no new genes were detected in the pyrosequenced contigs of any of the strains.\n",
            "\n",
            "diversity of a. marginale strains\n",
            "global comparison of all strains with the florida strain revealed  <dig>  total sites with a single nucleotide polymorphism  in at least one of the compared strains. of these,  <dig>  were different in the florida genome and identical in the other four strains, and  <dig>   were unique to one of the four strains . the remaining  <dig> % of snps represent those snps relative to florida that were present in two or three of the strains. there were  <dig>  snps between the florida and st. maries strains, comprising  <dig> % of the larger florida genome. the snps were distributed evenly throughout the genome, which is similar to both ehrlichia ruminantium and rickettsia bellii , and are proportionally distributed throughout coding and non-coding regions. the numbers of polymorphisms in the puerto rico, virginia, and mississippi strains  are minimums, as the gaps are regions predicted to have significant numbers of snps. when the genome size was corrected for the gaps in coverage, the snp rates for the puerto rico, virginia, and mississippi genomes were  <dig> %,  <dig> %, and  <dig> % of the florida genome, respectively.\n",
            "\n",
            "discussion\n",
            "this study illustrates the dangers of drawing universal conclusions when strains are selected based on specific criteria, such as phenotypic differences. no two strains in this study are truly representative of the population as a whole. additionally, the large number of differences in pair-wise comparisons of any two strains illustrates the difficulty of associating genes with phenotypic differences, and the utility of sequencing multiple strains to increase the power of these associations. while our initial selection of the florida strain was based on a phenotypic difference – that of tick transmissibility, the selection of subsequent strains  were made to try and minimize the effect of bias based on that phenotype, as well as select a wider geographic range of isolates to increase interstrain diversity. interestingly, when the pyrosequenced strains are compared to florida, there are more high-quality polymorphisms  between florida and the mississippi strain, despite the fact that neither is tick-transmissible by d. andersoni  <cit> . further, st. maries appears to be an outlier sequence, as there are at least  <dig>  differences between st. maries and all other sequenced strains.\n",
            "\n",
            "the level of snp diversity in these strains coupled with the high degree of gene content conservation also sheds an interesting light on the concept of the \"core genome\", described for streptococcus agalactiae  <cit> . for s. agalactiae, approximately  <dig> % of genes were considered part of the \"core genome\", or constant between strains, and each new strain added additional strain-specific genes to the \"pan-genome\". this is contrasted with bacillus anthracis, which had no new strain-specific genes after four strains were compared. the strains of a. marginale sequenced here present an interesting data point, as a. marginale has not been hypothesized to be a clonal population derived from another organism , and yet has a closed core genome. the accumulation of large numbers of snps might indicate a greater evolutionary distance; however, the closed-core genome could be due to other factors. these could include the isolated nature of the intracellular niche occupied by a. marginale, causing the organism to undergo reductive evolution to the point it is approaching the minimal gene complement, or may be, despite our efforts, related to the strains selected for sequencing. however, if this is due to long-term reductive evolution, it calls into question the source of the six split orfs between the florida and st. maries genomes, as these are thought to be early reductive changes. another possibility is that transmission of the organism among animals in a relatively restricted geographic area  promotes a relatively clonal population of organisms through isolation in a similar environment.\n",
            "\n",
            "analysis of the level of snp diversity in several bacterial genomes brings into question previous conclusions about the variability of obligate intracellular pathogens. previous studies  <cit>  have found relatively large numbers of snps between intracellular organisms. it was therefore hypothesized that the relatively isolated intracellular niche limits opportunities for genetic exchange and increased numbers of snps provides a compensatory mechanism for providing diversity to drive evolution. our results suggest this is unlikely, as there is no correlation between intracellular, facultative intracellular, and free-living organisms and the level of diversity. with few exceptions, there is a large range in the degree of variability in all the strains compared. additionally, the organisms with the two highest rates of variability, pseudomonas syringae and rhodopseudomonas palustris, are both free-living. there is also significant variation at the genus and family level. these data suggest that the factors for retention of snps leading to bacterial diversity are likely multifactorial and complex.\n",
            "\n",
            "while the composition of the gene content of the pan-genome is obviously important, this study reveals another characteristic that needs examination: the level of diversity in the pan-genome. the minimum of  <dig>  variable sites found among these five genomes is approximately  <dig> % of the estimated size of the pan-genome. the large number of unique snps in each strain  suggests that while a. marginale has a closed core genome, the snp profile of the core genome is moderately \"open\". when several strains of streptococcus agalactiae  are compared to the  <dig> vr strain,  <dig> % of the  <dig>  total detected snps are unique to an individual strain, while zero snps are common to all four strains. similarly, 100% of snps between three strains of bacillus anthracis  and mycobacterium tuberculosis ,  <dig> % of snps between three strains of neisseria meningitides , and  <dig> % of snps between four strains of chlamydophila pneumoniae  are unique to one strain. this suggests that these genomes have open snp profiles regardless of being open or closed-core at the genome level. further, there is no correlation between snp diversity and lifestyle, with high levels of variation between strains and within genera, with limited exceptions. however, given that the majority of strains were selected based on phenotypic traits or previous work with each strain, it is unlikely that this represents the true diversity of these organisms. additionally, the majority of organisms have only two sequenced strains, making analysis of variation within a species impossible to determine. additional work will be required to build a picture of genomic diversity.\n",
            "\n",
            "the genome of a. marginale is highly recombinogenic, which, in spite of the highly conserved gene content, leads to increased plasticity. there are between five and nine functional msp <dig> pseudogenes in the strains examined to date  <cit> , and these can recombine in whole or in part into the msp <dig> expression site  to generate new antigenic variants  <cit> . symmetrical inversions around the origin are thought to be quite common in bacteria  <cit>  and have been noted in anaplasmataceae, often utilizing repeated genes such as msp <dig> to mediate the inversion. these inversions are highlighted by comparisons between a. marginale and ehrlichia ruminantium  <cit>  and anaplasma phagocytophilum  <cit> . many of these repetitive sequences flank ori, as does another duplicated gene, rho. while not around the origin, a smaller scale inversion was found between two strains of a. marginale flanked by msp <dig> pseudogenes close to ori. another highly plastic genomic region is the aaap locus  <cit>  that appears to be expanding and contracting within and between strains. in addition to changes in gene number, the sequences are highly variable . further research will be needed to determine the significance of these differences, as well as the function of this locus.\n",
            "\n",
            "CONCLUSIONS\n",
            "sequencing of multiple strains of bacteria, as well as sequencing multiple isolates from the same strain, will yield a tremendous amount of information about natural rates of variation in bacterial populations, which in turn will influence our views of bacterial evolution, epidemiology, and vaccine strategies. this study reveals that interstrain snp diversity does not appear to be influenced by the environmental niche an organism occupies, nor is it generally consistent throughout a specific family or genera. comparison of multiple strains of a. marginale finds few changes at the gene level, while there is robust diversity at the nucleotide level. finally, multistrain snp analysis appears to be a more powerful tool for a. marginale phylogenetic studies than genotyping of the major surface proteins  <cit> , and this strategy should be useful for epidemiologic studies of other species of bacteria.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "a. marginale is a member of the order rickettsiales and a well-established obligate intracellular bacterial model.\n",
            "a. marginale is a member of the order rickettsiales and a well-established obligate intracellular bacterial model.\n",
            "this study reveals that interstrain snp diversity does not appear to be influenced by the environmental niche an organism occupies, nor is it generally consistent throughout a specific family or genera.\n",
            "a. marginale has a closed core genome, which may result from small evolutionary distance, and may be the result of small evolutionary distance.\n",
            "a. marginale is a member of the order rickettsiales and a well-established obligate intracellular bacterial model.\n",
            "\n",
            "53\n",
            "[{'summary_text': 'gas <dig> has a large, 10– <dig> amino acid unstructured loop in the n-terminal domain, which based on modelling indeed appears to form a large flexible loop, suggesting that gas <dig> has multiple functions.\\ngas <dig> is a flexible two-domain molecule, the flexibility perhaps reflecting its multifunctional properties.\\ngas <dig> has a large, 10– <dig> amino acid unstructured loop in the n-terminal domain.\\ngas <dig> has a large, 10– <dig> amino acid unstructured loop in the n-terminal domain, which based on modelling indeed appears to form a large flexible loop, suggesting that gas <dig> has multiple functions.\\ngas <dig> has a large, 10– <dig> amino acid unstructured loop in the n-terminal domain, which based on modelling indeed appears to form a large flexible loop, suggesting that gas <dig> has multiple functions.\\ngas <dig> has a large, 10– <dig> amino acid unstructured loop in the n-terminal domain, which based on modelling indeed appears to form a large flexible loop, suggesting that it has multiple functions.\\ngas <'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "growth arrest specific- <dig> gene  was found in a screen to identify genes specifically expressed in growth-arrested mouse cells  <cit> . the full-length cdna of human gas <dig> was cloned  <cit>  and the mature protein was found to contain  <dig> amino acids, a potential signal peptide, one n-glycosylation site at asn <dig> and an aminated ser <dig>  <cit> . the aminated ser <dig> allows the mature protein to be glycophosphatidylinositol  anchored to the cell membrane  <cit> .\n",
            "\n",
            "gas <dig> was found to arrest cell cycle by stopping the cells in synthesis  phase  <cit>  and due to its ability to arrest cell proliferation in p53-dependent manner it has been considered to be a tumour suppressor protein  <cit> . generally gas <dig> might act as a tumour suppressor in adult brain, though the expression in brain leading to apoptosis has not been observed in adults  <cit> . sequence comparison of human and murine gas <dig> genes suggested that it has a conserved rgd-peptide sequence for possible rgd-dependent integrin binding at residues 306– <dig>  <cit> .\n",
            "\n",
            "additionally gas <dig> has been shown to have a significant role in development  <cit> . at early developmental stages gas <dig> is expressed in most embryonic tissues. during development gas <dig> has been reported to inhibit cell proliferation and to mediate cell death, to be involved in embryonic patterning, and to support growth of the cerebellum  <cit> .\n",
            "\n",
            "gas <dig> is clearly a multifunctional protein, since it signals through at least two different kinds of transmembrane receptor proteins, rearranged during transformation   <cit>  and the hedgehog receptor protein patched- <dig>  <cit> . the hedgehog signalling pathway is important in development, stem cell renewal, and cancer progression. gas <dig> is able to bind sonic hedgehog  and activate the signalling pathway from patched- <dig>  <cit> . ret, on the other hand, is a transmembrane kinase, first identified as a proto-oncogene  <cit> . overactivity of ret can cause several types of cancers, and loss-of-function mutations cause varying degrees of loss in the enteric nervous system resulting in hirschprung’s disease . normally ret mediated signalling is controlled by glial cell-derived neurotrophic factor family ligands  and glial cell-derived neurotrophic factor receptor alphas , which form a four-member protein family   <cit> .\n",
            "\n",
            "of these, gas <dig> has highest  similarity to gfrα <dig>  while gas <dig> and gfrα <dig> both have only two domains unlike gfrα1- <dig>  which consists of three domains  <cit> . the secondary structure of mammalian gas <dig> is predicted to be mostly α-helical separated by short β-strands and to have a long unstructured c-terminal domain  <cit> . by binding gfls, gfrαs take part in controlling the survival of neurons, neuron branching, and functional recovery  <cit> . the most studied member of gfls is gdnf, which was identified due to its function as a survival factor for midbrain dopaminergic neurons  <cit> . gdnf forms a complex with gfrα <dig> and promotes the survival of neurons  <cit> . gfls, in general, are dimeric proteins and they are capable of binding two gfrα receptors per ligand  <cit> . after the formation of gfrα-gfl complex, the complex then binds to the transmembrane tyrosine kinase ret  <cit> .\n",
            "\n",
            "despite the structural similarity to gfrαs, gas <dig> differs from them functionally because it is able to bind to ret in a ligand independent way  <cit> . in addition, the intracellular signalling pathway is most probably different than for gfrα-gdnf complex, and gas <dig> bound to ret blocks akt activation, and increases erk activation  <cit> .\n",
            "\n",
            "gas <dig> has been suspected to be an ancestor of gfrα proteins  <cit> . thus the four gfls and gfrαs could have been generated by genome duplications at the origin of vertebrates, and at this point the gene encoding gas <dig> could have diverged from gfrα-like proteins  <cit> .\n",
            "\n",
            "it has been hypothesised that the relative abundance and localization of gfrαs, gfls and gas <dig> could determine in certain conditions whether cells survive or die  <cit> . furthermore, gas <dig> expression is increased in neuronal cell death during early development  <cit> . therefore, gas <dig> could work as a switch between proliferation and differentiation in neuronal development  <cit> . gas <dig> has been shown to colocalize to lipid rafts with ret  <cit> . this has led to the hypothesis that gas <dig> could be a negative modulator of gdnf signalling and able to control gdnf stimulation via ret  <cit> .\n",
            "\n",
            "RESULTS\n",
            "production and purification of human gas <dig> protein\n",
            "after cloning and expressing human gas <dig> in tricoplusia ni cells, we purified secreted gas <dig> from the insect cell growth medium using ni-affinity chromatography , and the identity and size of the expressed protein was verified with a western blot . the purified protein is glycosylated and therefore does not run exactly according to excepted molecular weight on the sds-page, but slightly higher. thrombin was used to cleave off the tags, and the size of the protein after cleavage was verified by sds-page and maldi-tof mass spectroscopy. the yield of purified protein was on average ca. 1 mg/l.figure  <dig> \n",
            "purification of recombinant gas <dig> from insect cells. a) ni-affinity chromatogram for gas <dig> purification b) western blot of fractions from the ni-affinity chromatography peak at ca. 28 ml. fractions of 1 ml were collected, and fractions from peak area at  <dig>   <dig>   <dig> and 33 ml were tested in the western blot. c) sds-page analysis of gas <dig> purification .\n",
            "\n",
            "\n",
            "\n",
            "based on the primary sequence one n-glycosylation site was predicted at asn <dig> located in the n-terminal domain. the corresponding site in the gfrα-structures is located at the domain interface between the two homologous disulphide rich domains, in a tightly packed two-domain structure  <cit> , suggesting that the gas <dig> overall conformation is quite likely very different .figure  <dig> \n",
            "homology modelling of human gas <dig> and comparison to gdnf receptor structures. a) four representative homology models of gas <dig> n-terminal domain showing different orientations and flexibility of the extended intradomain loop of higher vertebrate proteins. the models were generated with the raptor-x server, as mentioned in the text. the template for modelling was the gfrα <dig> structure . b) comparison to n-terminal domain of gfrα <dig>  and n-terminal domain  and ret/heparin binding site . gfrα <dig> ret/heparin binding-site residues are labelled. c) the gdnf binding site residues gfrα <dig>  vs. gas <dig> . the gdnf glu binding to the gfrα <dig> is colored brown, and hydrogen bonds are indicated with dashed lines. a tyr and ser residue occupy the positions in gas <dig> equivalent to gfrα <dig> arg <dig> and  <dig>  gfrα <dig> gdnf binding residues are labelled. disulphides in the figure are shown with stick presentation and atoms s atoms in yellow d) position of the n-glycosylation site in gas <dig> vs. gfrα <dig> domain interface of domains d <dig> and d <dig>  n-glycan at asn <dig> of gas <dig> is indicated in yellow with a stick presentation, the gfrα <dig> d <dig> and d <dig> domains are depicted in dark cyan, gas <dig> n-terminal domain is drawn in grey.\n",
            "\n",
            "\n",
            "\n",
            "when the protein was treated with pngase f to remove glycans, the size of the protein diminished slightly on sds-page , and based on maldi-tof analysis, we observe a decrease in molecular weight of ca. 900 da; the glycosylated protein had a molecular mass of  <dig>  kda and the de-glycosylated protein of  <dig>  kda, according to maldi-tof, while the theoretical molecular mass of the protein without glycosylation is  <dig>  kd, thus matching well with the mass spectrometry results. the result obtained for glycosylated protein corresponds to approximately one n-glycan added post-translationally in the insect cells. the purified protein was functional in binding to ret in vitro, and found to be over 90% pure on sds-page, and monodisperse in solution after gel filtration.\n",
            "\n",
            "gas <dig> is a monomer is solution and highly thermostable\n",
            "the cleaved, non-tagged protein was found to be a monomer by analytical size exclusion chromatography and multi-angle light scattering  . at 1 mg/ml and  <dig>  mg/ml the sec-malls runs gave a single peak with molecular mass of ca. 31–33 kda , matching quite well to the theoretical size of monomeric gas <dig>  considering the additional glycosylation at one site. similarly, based on the small-angle x-ray scattering  data, the molecular weight matches most closely to a monomer . in our opinion, this is likely to reflect the oligomerization state of the lipid-anchored protein, which is unlikely to be affected by the anchor. no detectable oligomerization was observed in native page or gel filtration at  <dig>  mg/ml, while in saxs data an effect from residual aggregation was evident at higher concentrations.figure  <dig> \n",
            "gas <dig> analytical gel filtration, circular dichroism and thermal unfolding. a) gas <dig> sample was run on superdex  <dig> 10/ <dig> gel filtration column in tbs at  <dig>  ml/min, at protein concentration of 1 mg/ml. a single major peak at 30 min  eluted and based on multi-angle light scattering had molecular weight  of ca. 33 kda, matching relatively well with theoretical molecular weight of the monomer. the peak is plotted as a function of dri signal . b) the cd spectrum of gas <dig> and c) the residual thermal denaturation of gas <dig> as monitored by cd at 222 nm.\n",
            "saxs-derived size parameters for gas1\n",
            "\n",
            "\n",
            "the guinier i-value was calculated against an absolute reference   <cit>  and the i for the sample is then equal to the molecular weight. molecular weight from the porod volume is estimated according to petoukhov et al.  <cit> .\n",
            "\n",
            "\n",
            "\n",
            "circular dichroism  spectroscopy was used to verify the secondary structure content of gas <dig> and, as expected, the cd spectrum was typical for an α-helical protein . a measured temperature denaturation curve with cd gave a result with partial melting of the structure when heated to 90°c . however, full temperature denaturation was not possible to obtain by cd, nor by differential scanning calorimetry , possibly due to the high disulphide content of the protein. this suggests that the domain structure is thermally very stable. the decrease in cd signal at 222 nm did not even reach the midpoint of denaturation when heated to 90°c .\n",
            "\n",
            "sequence analysis and evolution of gas1\n",
            "the gas <dig> protein domain structure is defined by two gfrα–like domains, with  <dig> conserved disulphide bridge-forming cysteines in each domain  <cit> . gas <dig> is present in all the vertebrates, and homologs are also found in lower chordates . in addition, gas <dig> homologs also occur in c. elegans and honeybee, but not, surprisingly, in drosophila. the sequence identity to vertebrate proteins, however, is quite low: ca. 21-24% for honeybee and only 14-19% for the worm sequence . two conserved cysteines are missing from the c. elegans sequence , and thus the protein fold might not be fully conserved in the c. elegans homolog   <cit> . alignment of the gas <dig> sequences shows that, in higher vertebrates, the n-terminal domain has an insertion with low sequence complexity , apparently forming an extended loop structure . mammals have also an rgd sequence in the c-terminal linker region. also, a single n-glycosylation site at asn <dig> is predicted to be conserved based on sequence in all vertebrates, while it is not present in the invertebrates. in the set of conserved residues beyond the structural cysteines   a subset are present only in the vertebrate proteins, and, other than the conserved cysteines, only ten residues are conserved also in c. elegans.table  <dig> \n",
            "amino acid sequence identities  within the gas <dig> protein family\n",
            "\n",
            "\n",
            "human\n",
            "sus scrofa\n",
            "bos taurus\n",
            "canis lupus\n",
            "mouse\n",
            "gallus gallus\n",
            "alligator\n",
            "anolis\n",
            "xenopus\n",
            "latimeria\n",
            "danio rerio\n",
            "apis\n",
            "c. elegans\n",
            "\n",
            "human\n",
            "\n",
            "sus scrofa\n",
            " <dig> \n",
            "\n",
            "bos taurus\n",
            " <dig> \n",
            " <dig> \n",
            "\n",
            "canis lupus\n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            "\n",
            "mouse\n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            " <dig> \n",
            "\n",
            "gallus gallus\n",
            "\n",
            "alligator\n",
            " <dig> \n",
            "\n",
            "anolis\n",
            "\n",
            "xenopus\n",
            "\n",
            "latimeria\n",
            "\n",
            "danio rerio\n",
            "\n",
            "apis\n",
            "\n",
            "c.elegans\n",
            "pairwise identities between species that are over 70% are shown in bold.\n",
            "the gas <dig> family sequence alignment. a) the sequences start from the beginning of the mature human gas <dig> and numbered according to the human amino acid numbering. residues over ca. 85% conserved  are coloured with a black-to-grey scale, in higher vertebrates  there is an extended loop in the n-terminal domain around residues 80– <dig> . the c-termini are poorly conserved , note the rgd sequence at 306– <dig>  b) conserved surface features on gas <dig> displayed on the n-terminal domain. c) conserved surface features on gas <dig> displayed on the c-terminal domain; conserved sites in b and c are coloured in red .\n",
            "\n",
            "\n",
            "\n",
            "when the conserved amino acid residues are displayed onto the surface of the modelled domains, the most conserved surface patch is found on the n-terminal domain surface formed by residues on helices 3– <dig>  whereas the c-terminal domain surface did not reveal large patches of conservation . here conservation is defined by >75% sequence similarity amongst the residue groups khr, ed, nqstgp, ilmvca and fyw.\n",
            "\n",
            "homology modelling of gas <dig> domains and comparison to gfrαs\n",
            "we constructed homology models of both gas <dig> domains with the raptorx-server   <cit> , designed for low sequence homology-based modelling. the models for both domains fit well to the gfrα-structure   <cit> . as described above, the n-terminal domain of mammalian gas1-proteins contain a large inserted loop with low sequence complexity, which based on modelling indeed appears to form a large flexible loop, but whether this region has functional significance or not remains unclear.\n",
            "\n",
            "modelling of gas <dig> has partially been done before also by cabrera et al.  <cit>  and schueler-furman et al.  <cit> . here our aim was to study possible conservation of the ligand binding regions of gfrαs vs. gas <dig>  to do more detailed analysis on the structure, and to provide models for the analysis of saxs data .\n",
            "\n",
            "although the sequence identity to the related gfrα structures is low, the cysteines involved in disulphide bridges are well conserved for the two gfrα-type domains and make structure prediction possible. the n-terminal domain of gas <dig> is equivalent of the second domain in gfrαs, which contains the growth factor binding site. we aligned our model of gas <dig> with the gfrα1:gdnf complex structure  <cit> , and, based on the structural alignment, the binding region for the gdnf is not conserved in gas <dig> . similarly the conserved binding residues in gfrα2:artemin complex   <cit> , are not present in gas <dig>  in fact, the key ionic residues required for ligand binding are conserved in both these structure, but not present in gas <dig>  the conserved key residues for gdnf binding in gfrα <dig> are arg <dig>  arg <dig> and asn <dig>  in our structural analysis gas <dig> has tyr <dig>  thr <dig>  gln <dig> in equivalent side chain positions; in gas <dig>  the ion triplet required for growth factor binding  <cit>  is absent.\n",
            "\n",
            "it has also been suggested by wang et al.  <cit>  and parkash et al.  <cit>  that a ret binding region would be located mostly in the second  domain on the gfrαs and would involve the gfrα <dig> residues arg <dig>  lys <dig>  arg <dig>  gln <dig>  lys <dig>  arg <dig>  arg <dig>  glu <dig>  and asp <dig>  this site forms a highly positively charged patch on the surface of gfrα <dig>  identified also as a heparin binding site by parkash et al.  <cit> . we analysed the equivalent region in the n-terminal domain of gas <dig>  but found no conservation between the gfrα <dig> structure and gas <dig> . overall gas <dig> is not positively charged, as would be expected of a typical heparin-binding molecule. the calculated pi-value for human gas <dig> is  <dig>  whereas for human gfrαs the values range from  <dig> - <dig>   to  <dig>   and  <dig>  . also, gas <dig> does not contain a highly positively charged patch in the suggested ret/heparin binding region, and heparin affinity chromatography of gas <dig> showed no significant binding to the column , whereas the well-known heparin binding protein hbgam eluted only at 1 m nacl . finally, modelling of gas <dig> n- terminal domain shows that glycosylated asn <dig> will be situated at the position equivalent to the domain interface between domains d <dig> and d <dig> in the gfrα structures .figure  <dig> \n",
            "heparin affinity chromatography of gas <dig>  a) elution of gas <dig> as a function of salt concentration. b) elution of hbgam as a function of salt concentration. chromatograms are plotted with absorbance in mau unit  and salt concentration gradient to 1 m nacl  , against volume in ml.\n",
            "\n",
            "\n",
            "\n",
            "structural characterization of gas- <dig> by solution x-ray scattering\n",
            "solution x-ray scattering  data indicated that gas <dig> is monomeric at  <dig>  mg/ml in solution based on the porod volume and guinier plots : at higher concentrations the protein starts to aggregate, and the data beyond 1 mg/ml could not be analysed. rigid body modelling of the structure was done based on homology models of the individual domains, and elongated models gave the best fits . we also calculated ab initio envelopes, which matched well with rigid body modelling of the structure . both differ significantly from the compact gdnf co-receptor structures  <cit> . however, as it is clear that the structure is likely to be flexible, in particular the c-terminal long unstructured region, we also did ensemble fitting of the model against the data. this resulted in a bimodal ensemble represented by four major structures selected from the initial random pool of  <dig>  <dig> structures, which fit to the data with χ2 =  <dig>  . the selected structures represent states with extended and collapsed c-terminal linkers and variable orientations of the domains relative to each other . taken together it appears from the saxs data that the orientation of the domains of gas <dig> relative to each other is not fixed; clearly the protein exists in two populations of extended and collapsed conformations.figure  <dig> \n",
            "rigid body and ab initio modelling of gas <dig> based on saxs data. a) scattering curve and fit of coral rigid body model  to observed data. b) the kratky-plot from the experimental data, suggesting a folded structure with some flexibility. c) coral generated model with n- and c-terminal domains as rigid bodies  with flexible linker regions , and the ab initio model for gas <dig> generated by dammin  fitted over the rigid body model . d) the distance distribution calculated for gas <dig> saxs data. e) saxs ensemble modelling of gas <dig> solution conformations shown as the statistical distribution of rg-values of best fitted models  vs. initial random pool  shows a bimodal distribution of gas <dig> solution conformations. f) the selected pdb-files representing the ensemble with chi2 =  <dig>  fit to the experimental data, showing extended  and more collapsed models  in the final ensemble; the n-terminal domain  was fixed relative to the rest of the protein during the runs. the modelled glycan structure is shown as red “stick” presentation on the n-terminal domain.\n",
            "\n",
            "\n",
            "\n",
            "binding and affinity of gas <dig> to ret in vitro\n",
            "we tested whether gas <dig> is able to directly interact with ret in a ligand independent way, as previously reported  <cit> . for this purpose, and to determine the affinity of the interaction, the ret receptor was coupled to a chip for surface plasmon resonance assay, and binding of a concentration series of gas <dig> to immobilized ret was measured. a kd-value of  <dig>  ±  <dig>  μm was measured for the interaction in vitro . the kinetics of the interaction were too fast to allow for measurement of on- and off-rates, as is evident from the time scale of the binding and dissociation from the sensograms .figure  <dig> \n",
            "binding of human gas <dig> to ret. a) a binding curve of gas <dig> to ecret. purified human gas <dig> shows clear ligand independent binding to ecret. binding was measured with a concentration series of 2 μm, 4 μm, 8 μm and 16 μm, 24 μm and 32 μm. the dissociation constant  was obtained based on the equilibrium  values at different concentration from two independent experiments with a k\n",
            "d =  <dig>  ±  <dig> . b) the biacore sensograms for gas <dig> binding  to ret at different concentrations .\n",
            "\n",
            "\n",
            "\n",
            "discussion\n",
            "we have overexpressed and purified the human gas <dig> protein in soluble form without the gpi-anchor, and biophysically characterized the protein. we constructed homology models for both domains of gas <dig> and were able to analyse the domain structure of the protein in comparison to the structurally related gfrαs. as reported earlier by others  <cit> , gas <dig> has clearly two gfrα-like domains, but we have shown here that gas <dig> differs significantly from gfrαs in both sequence and in structure.\n",
            "\n",
            "the differences can be characterized as follows: firstly, gas <dig> has a large, 10– <dig> amino acid unstructured, low complexity ala/gly/pro-containing loop in the n-terminal domain. this loop region is present in higher vertebrates, in mammalians and chicken, but not in fish . whether this loop might have some function remains unknown. secondly, the two-domain structure of gas <dig> appears to be more flexible overall than in the characterized gfrα structures. in the gfrαs, the functional domains d <dig> and d <dig> form a compact structure, whereas gas <dig> saxs analysis reveals a flexible ensemble of structures, with the n- and c-terminal domains as independent structural units. this might reflect the location of the functional binding regions of the molecule versus those of the gfrαs. as expected, the protein is α-helical based on the cd spectrum. it is somewhat intriguing that we were not able to fully denature the protein; apparently the disulphide-linked arrangement of the domains is highly thermostable, and this might be a general feature of the gfrα-family.\n",
            "\n",
            "the structural flexibility is probably a conserved feature in the protein family, as the single n-glycosylation site in the human protein is conserved in chordates. this n-glycan blocks the gfrα-equivalent domain interface, and hence the formation of that type of compact structure. it has been also observed that this glycosylation site might have functional significance for shh binding  <cit> .\n",
            "\n",
            "our sequence analysis and that by hätinen et al.  <cit>  suggest that gas <dig> is conserved during evolution, with homologs in chordates , arthropods and roundworms, thus possibly representing an ancestral gfrα-like protein  <cit>  however, the sequence identity from chordate to invertebrates  genes is low, 14-19% for the worm phas- <dig> homolog of gas <dig>  <cit> , and it remains an open question whether the insect or worm genes identified as gas <dig> actually share any of the functions of vertebrate gas1/gfrα type of receptors, either in ret or the hedgehog signalling.\n",
            "\n",
            "gas <dig>  as well as gfrα-like proteins, are conserved beyond vertebrates, while gfls are not expressed in non-vertebrates. this suggests that either ret binding, independent of gfls is conserved, or that there are alternative receptors for gas <dig>  in case of gas <dig>  this could be shh and patched- <dig>  interestingly in drosophila, the grfα-like protein does not interact with ret but does interact with drosophila ncam analog, fasii  <cit> . the mammalian gfls are known to be ligands of ncam  <cit> : whether gas <dig> might interact with ncam homologs remains to be investigated.\n",
            "\n",
            "when we compared our model with the gfrα <dig> structure it was clear that the crucial amino acids for gfl binding are not conserved in gas <dig>  and it most likely lacks the ability to bind gfl-like ligands, as they all share the same binding mode  <cit> . indeed cabrera et al.  <cit>  reported that gas <dig> is not able to bind gdnf. while gas <dig> lacks the ability to bind gfl-type of ligands, our in vitro binding data support the findings by cabrera et al.  <cit>  that gas <dig> can bind ret in a ligand independent manner, and possibly alter the intracellular signalling of ret.\n",
            "\n",
            "the affinity of gas <dig> for ret is significantly lower than that of the gfrα-gdnf ligand complex in solution . however on the cell surface the affinity of gas <dig> to ret is also likely to be higher as the diffusion is restricted to two-dimensions.\n",
            "\n",
            "another possibility is that in some cellular contexts gas <dig> would be highly expressed on cell surface, which might boost the binding to ret locally. the exact mechanism of gas <dig> on ret signalling remains elusive, but it seems clear that gas <dig> has an effect of ret signalling, probably by inhibiting growth-factor dependent signalling  <cit> .\n",
            "\n",
            "based on the conservation of protein surface features, as mapped on to the gas <dig> models, we suggest that the n-terminal domain region defined by α − helices 3– <dig> might contain a functional binding site , whereas other possible interaction surfaces remain less clear, e.g. the very short rgd-peptide motif found in mammalian sequences could be functional, or exist by chance, and so far no biochemical evidence for the function exists.\n",
            "\n",
            "gas <dig> has also been reported to alter shh-signalling through patched- <dig>  <cit> , indicating that gas <dig> has multiple functions. related to this pineda-alvarez et al.  <cit>  and ribeiro et al.  <cit>  reported missense mutations of gas <dig> in holoprosencephaly  patients.in particular thr200arg mutation in the second domain of gas <dig> pineda-alvarez et al.  <cit>  was observed to result in almost complete loss of binding affinity for shh, hence this domain could also be important for binding to shh. also asn220lys caused 20% reduction in binding according to pineda-alvarez et al.  <cit>  and ala246ser patient mutations are located in the same domain, while the mutations are some what scattered around the domain and do not cluster together on the surface.\n",
            "\n",
            "CONCLUSIONS\n",
            "our structural data reveal that gas <dig> is a flexible two-domain molecule, the flexibility perhaps reflecting its multifunctional properties. the structural arrangement of the domains is clearly different form the compact gfrα structures, suggesting that it has different functional roles. in particular, neither the putative heparan sulphate proteoglycan/ret binding site  <cit>  the known growth factor binding site are conserved in gas <dig> \n",
            "\n",
            "thus, gas <dig> must act on ret in a different way, and together with previous analysis our binding data supports the ligand-independent ret binding by gas <dig>  while sequence conservation analysis hints at possible sites of functional importance.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "gas <dig> has a large, 10– <dig> amino acid unstructured loop in the n-terminal domain, which based on modelling indeed appears to form a large flexible loop, suggesting that gas <dig> has multiple functions.\n",
            "gas <dig> is a flexible two-domain molecule, the flexibility perhaps reflecting its multifunctional properties.\n",
            "gas <dig> has a large, 10– <dig> amino acid unstructured loop in the n-terminal domain.\n",
            "gas <dig> has a large, 10– <dig> amino acid unstructured loop in the n-terminal domain, which based on modelling indeed appears to form a large flexible loop, suggesting that gas <dig> has multiple functions.\n",
            "gas <dig> has a large, 10– <dig> amino acid unstructured loop in the n-terminal domain, which based on modelling indeed appears to form a large flexible loop, suggesting that gas <dig> has multiple functions.\n",
            "gas <dig> has a large, 10– <dig> amino acid unstructured loop in the n-terminal domain, which based on modelling indeed appears to form a large flexible loop, suggesting that it has multiple functions.\n",
            "gas <\n",
            "54\n",
            "[{'summary_text': 'our results suggest that egr <dig> is required for certain gene repression mechanisms during peripheral nerve myelination, and that egr <dig> may compensate in the absence of egr <dig> by binding of egr <dig> to the nab <dig> promoter.\\nour results suggest that egr <dig> is required for certain gene repression mechanisms during peripheral nerve myelination, and that egr <dig> may compensate in the absence of egr <dig> by binding of egr <dig> to the nab <dig> promoter.\\nour results suggest that egr <dig> is required for certain gene repression mechanisms during peripheral nerve myelination, and that egr <dig> may compensate in the absence of egr <dig> by binding of egr <dig> to the nab <dig> promoter.\\nour results suggest that egr <dig> is required for certain gene repression mechanisms during peripheral nerve myelination, and that egr <dig> may compensate in the absence of egr <dig> by binding of egr <dig> to the nab <dig> promoter.\\nour results suggest that egr <dig> is required for certain gene repression mechanisms during peripheral nerve myelination'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "members of the egr  family of transactivators fulfill critical roles in diverse systems, including nervous system development, bone formation, and fertility. transcriptional activity of three egr family members  is regulated by interaction with the nab  family of transcriptional corepressors. the nab <dig> protein was first identified by screening a yeast two-hybrid library for proteins that interact with a repressive domain within egr <dig> <cit> , and nab <dig> was subsequently found to share similar properties  <cit> . both nab <dig> and nab <dig> can bind to a conserved domain in egr <dig>  egr <dig> and egr <dig>  nab proteins repress activation of several egr target promoters  <cit> , and the repression mechanism involves interaction with the chd <dig> subunit of the nucleosome remodeling and deacetylase  complex  <cit> .\n",
            "\n",
            "the most severe phenotypes caused by loss of egr function have been found in mice lacking the egr2/krox <dig> gene . characterization of these mice revealed three principal defects: 1) defective hindbrain segmentation with the loss of rhombomeres r <dig> and r <dig>  <cit> , 2) defective bone formation  <cit> , and 3) failure of schwann cells to myelinate peripheral nerves  <cit> . target genes regulated by egr <dig> in schwann cells include myelin genes such as myelin protein zero and myelin-associated glycoprotein  <cit> . as a consequence, most mice with a homozygous egr <dig> disruption die shortly after birth, although mice that are homozygous for a hypomorphic egr <dig> allele survive somewhat longer  <cit> .\n",
            "\n",
            "because of the myelination defect in peripheral nerves of egr <dig> knockout mice, several groups have screened human patients with peripheral neuropathies for mutations in the egr <dig> gene. mutations in egr <dig> have been identified in several patients with myelin disorders, such as charcot-marie-tooth  disease, dejerine-sottas syndrome, and congenital hypomyelinating neuropathy  <cit> . most of the neuropathy-associated mutations occur within the zinc fingers of egr <dig> and prevent dna-binding  <cit> . however, one of the egr <dig> mutations associated with a very severe congenital hypomyelinating neuropathy  prevents binding of egr <dig> to nab corepressors  <cit> . the importance of nab corepressors to the regulation of peripheral nerve myelination by egr <dig> was recently confirmed by the demonstration that a double knockout of the nab1/nab <dig> genes results in a phenotype very similar to that of the egr <dig> knockout: early lethality and peripheral neuropathy resulting from arrested myelination  <cit> . moreover, nab genes have been implicated in several other physiological processes, including macrophage development, cardiac hypertrophy, prostate carcinogenesis, and feedback regulation involved in hindbrain development  <cit> .\n",
            "\n",
            "as critical regulators of peripheral myelination, it is important to probe the mechanism of nab regulation. recent work has shown that nab <dig> and nab <dig> are induced by neuregulin signaling  <cit> . neuregulin signaling plays an extremely important role in axon-derived signals for schwann cell myelination  <cit> . moreover, egr <dig> and nab expression appear to be closely linked, as nab <dig> and nab <dig> are co-regulated with egr <dig> after nerve crush injury, and both corepressors are induced by ectopic egr <dig> expression in cultured schwann cells  <cit> . however, direct regulation of nab <dig> and nab <dig> by egr <dig> has not been demonstrated in vivo. the following data demonstrate direct regulation of nab expression by egr <dig> in myelinating sciatic nerve, and indicate that other neuregulin-regulated pathways are specifically involved in inducing nab <dig> expression.\n",
            "\n",
            "RESULTS\n",
            "induction of nab promoters by egr2\n",
            "sequence analysis of the nab <dig> and nab <dig> promoters identified several conserved motifs that resemble egr <dig> binding sites  <cit> . to test whether these sites bind egr <dig>  the mouse nab <dig> and nab <dig> promoters were analyzed by dnase i footprinting. in the presence of recombinant egr <dig>  there were strongly protected regions encompassing three previously identified sites in the nab <dig> promoter, as well as some weaker protections . interestingly, previous analysis of egr <dig> site specificity indicated a strong preference for t in the 4th position of the consensus sequence , and each of the three strongly protected sites have a t in the 4th base. a similar analysis of the mouse nab <dig> promoter  confirmed binding of egr <dig> to the conserved binding sites identified previously  <cit> . the strong binding site at the 3' end of the mouse nab <dig> promoter corresponds to a site of phorbol ester-induced egr <dig> binding in a recent analysis of the human nab <dig> promoter using electrophoretic mobility shift assays  <cit> .\n",
            "\n",
            "to test the function of the egr <dig> binding sites, fragments encompassing the conserved regions of the nab <dig> and nab <dig> promoters were fused to a luciferase reporter gene and transfected into jeg- <dig> cells, which exhibit a low level of expression of egr proteins  <cit> . as shown in figure 1c, expression of egr <dig> activated the nab <dig> promoter up to 15-fold, and the nab <dig> reporter was increased 25-fold. two of the egr <dig> binding sites in the nab <dig> promoter were mutated to determine how that would affect activation by egr <dig>  as shown in figure  <dig>  we still observed some residual activation of the promoter, but the induction was much less than observed with the wild type promoter. previous work had indicated that nab <dig> is more inducible by serum and neuregulin than nab <dig>  <cit> . moreover, the nab <dig> promoter was more highly induced by egr <dig> in the transfection experiments. however, this disparity is somewhat surprising, given that the nab <dig> promoter contains more egr <dig> binding sites than does nab <dig> \n",
            "\n",
            "neuregulin stimulation increases egr <dig> occupancy of the nab <dig> and nab <dig> promoters\n",
            "in rat schwann cells treated with neuregulin, there is a large induction of egr <dig> , followed by a delayed induction of nab <dig> and nab <dig>  which peaks at two hours after neuregulin addition  <cit> , with nab <dig> being more highly induced than nab <dig>  to test if neuregulin stimulates direct binding of egr <dig> to the nab promoters, chromatin immunoprecipitation  assays were used, in which formaldehyde is used to covalently crosslink dna with any associated proteins and the proteins of interest are subsequently immunoprecipitated  <cit> . primary rat schwann cells were treated for  <dig> hours with neuregulin and then cross-linked with 1% formaldehyde. sonicated chromatin was then immunoprecipitated with an egr <dig> antibody or control igg. an untreated culture of schwann cells was also cross-linked and immunoprecipitated similarly. after crosslinks were reversed, purified dna was analyzed by quantitative pcr using primer sets proximal to the egr <dig> binding sites in the nab <dig> and nab <dig> promoters. as shown in figure  <dig>  binding of egr <dig> to the nab <dig> and nab <dig> promoters was significantly stimulated upon treatment with neuregulin. egr <dig> binding was not detected at an immunoglobulin promoter  in either sample. evidence for the specificity of the antibody used for chip analysis includes induction of egr <dig> binding by neuregulin , and immunoblot analysis using this antibody reveals a band of the correct size . in addition, this antibody has been extensively tested with multiple negative control sites  <cit> , and expression of dominant negative egr <dig>  reduces egr <dig> binding in this assay . in parallel cultures, nab <dig> mrna was induced 11-fold upon treatment with neuregulin for  <dig> hours, consistent with previously published results  <cit> .\n",
            "\n",
            "egr <dig> binds to the nab <dig> and nab <dig> promoters in myelinating sciatic nerve\n",
            "we have recently employed chip analysis of myelinating sciatic nerve to test whether egr <dig> directly regulates specific elements in vivo  <cit> . egr <dig> expression increases concurrently with initiation of myelination in rat sciatic nerve within two weeks after birth  <cit> , which is also observed in immunoblot analysis of egr <dig> expression in p <dig> and p <dig> rat sciatic nerve . therefore, freshly dissected sciatic nerves from rat pups at p <dig>  were minced in 1% formaldehyde after dissection. after immunoprecipitation of sonicated chromatin, quantitation of the chip assays demonstrated significant enrichment of egr <dig> at both nab <dig> and nab <dig> promoters in rat sciatic nerve .\n",
            "\n",
            "nab expression levels in peripheral nerve of egr2-deficient mice\n",
            "to independently test the regulation of nab genes by egr <dig> in vivo, we examined nab expression levels in the egr2/krox <dig> knockout . in these mice, schwann cells develop normally and associate with axons, but fail to initiate myelination of peripheral nerves  <cit> . the sciatic nerves of seventeen wild type and knockout animals at p <dig> were pooled, and the levels of nab <dig> and nab <dig> were determined by quantitative pcr. previous analysis of these samples had indicated that expression of many myelin-associated egr <dig> target genes are significantly reduced in the knockout nerves  <cit> . consistent with the data from the transfection and chip assays, we observed a significant decrease in the expression of nab <dig>  a similar reduction in nab <dig> expression was reported in mice with a hypomorphic allele of egr <dig>  <cit> . in contrast, the expression level of nab <dig> was not significantly different in mice containing the null allele of egr2/krox <dig> \n",
            "\n",
            "regulation of the nab <dig> promoter by ets proteins\n",
            "given the unexpected finding that the expression level of nab <dig> was not lower in the absence of egr <dig>  the nab <dig> promoter was screened for other transcription factor binding sites that could contribute to nab <dig> expression, thereby compensating for the loss of egr <dig> activation. analysis of the nab <dig> promoter revealed several putative binding sites for ets transcription factors . interestingly, ets proteins are regulated by neuregulin signaling  <cit> . in addition, previous studies have identified ets transcription factors as playing an important role in autocrine and neuregulin-mediated survival of schwann cells, and showed that several ets genes, including net/elk <dig>  ets <dig>  gabpα and etv1/er <dig>  are expressed in e <dig> and newborn mouse peripheral nerve  <cit> . ets <dig> and etv1/er <dig>  were selected for further analysis as they were highly expressed in microarray analyses of mouse sciatic nerve . quantitative pcr analysis confirmed significant levels of ets <dig> and etv <dig> in mouse and rat sciatic nerves during peripheral nerve myelination , and western analysis  showed that both ets <dig> and etv <dig> are developmentally increased in rat sciatic nerve at p <dig>  compared to the p <dig>  timepoint. since nab <dig> expression was unaffected in the egr <dig> knockout, we assessed the expression levels of ets <dig>  etv <dig>  and elk <dig> in egr <dig> knockout nerves from p <dig> mice. interestingly, ets <dig> expression was previously shown to be elevated in sciatic nerve of egr2-deficient mice  <cit> , and etv <dig> expression was somewhat higher in the egr <dig> null mice as compared to wild-type mice . the presence of ets <dig> and etv <dig> in mouse and rat sciatic nerve during myelination  <cit> , as well as in egr <dig> null mice, suggest that ets transcription factors could compensate in the absence of egr <dig> to maintain nab <dig> expression levels.\n",
            "\n",
            "ets proteins activate the nab <dig> promoter\n",
            "to test whether ets proteins activate the nab <dig> and nab <dig> promoters, jeg <dig> cells were transfected with the nab <dig> or nab <dig> reporters described above, along with expression vectors for ets <dig> and etv <dig>  western analysis revealed that ets <dig> and etv <dig> expression in jeg <dig> cells is much lower than found in the s16y schwann cell line . the nab <dig> promoter was significantly induced by ets <dig> and etv <dig> expression, and this induction was further potentiated by coexpression of egr <dig> . a dominant-negative ets construct  specifically interfered with ets-mediated induction of the promoter and had no effect on activation by egr <dig> . in contrast, the nab <dig> promoter was activated to a lower extent by ets <dig> and etv <dig>  and the ets factors did not appear to augment activation of the nab <dig> promoter by egr <dig> \n",
            "\n",
            "we tested the ability of ets <dig> and etv <dig> to induce a mutated version of the nab <dig> promoter construct containing mutations in two of the egr <dig> binding sites . as observed previously, the mutant promoter is only weakly induced by egr <dig>  however, ets <dig> and etv <dig> were nonetheless able to induce this mutant reporter in the absence of egr <dig>  although the synergistic activation observed with egr <dig> was clearly diminished in the mutant promoter.\n",
            "\n",
            "ets <dig> binds to the nab <dig> promoter in s16y schwann cells\n",
            "in order to test if ets factors directly regulate nab <dig> in schwann cells, we carried out chip assays in the s16y schwann cell line  <cit> , which expresses significant levels of nab <dig>  nab <dig>  and ets factors . s16y cells were crosslinked in 1% formaldehyde and sonicated chromatin was immunoprecipitated with either an affinity purified ets <dig> antibody  or an igg control antibody. quantitative pcr was carried out on purified dna with primer sets in the nab <dig> and nab <dig> promoters, and the img2a locus as a negative control. as shown in figure  <dig>  we observed significant enrichment of ets <dig> using two primer sets in the nab <dig> promoter, whereas binding of ets <dig> was not detected at either the nab <dig> promoter or the img2a negative control locus. a similar trend in ets <dig> occupancy was observed in the s <dig> schwann cell line .\n",
            "\n",
            "one limitation of the chip assay is that it cannot be used to precisely localize binding of transcription factors. however, the analysis suggests that ets <dig> binding is separable from egr <dig>  because ets <dig> binding was not detected using the primer set  that was used previously to detect binding of egr <dig> within the nab <dig> promoter . in addition, the relatively simple recognition site of ets factors  does not make sequence analysis necessarily predictive, since the upstream cluster of putative ets binding sites  does not appear to bind ets <dig>  accordingly, deletion of the upstream cluster of putative ets binding sites does not affect activation of the nab <dig> promoter in transfection assays . ets <dig> binding appears to occur in two separable regions of the nab <dig> promoter, as detected by primer sets  <dig> and  <dig>  interestingly, primer set  <dig> overlaps with the corresponding segment of the human nab <dig> promoter that was recently shown to be inducible by phorbol ester  <cit> , which also activates ets proteins. overall, these data are consistent with regulation of the nab <dig> promoter by ets transactivators in vivo.\n",
            "\n",
            "discussion\n",
            "recent studies have shown that nab proteins are required for peripheral nerve myelination by schwann cells, as mice lacking nab1/nab <dig> displayed a severe congenital hypomyelinating phenotype similar to egr <dig> knockout mice  <cit> . the molecular roles of egr <dig> and nab proteins appear to be closely intertwined at several levels, including both physical interaction and coregulation  <cit> . several previous studies have suggested that egr factors regulate the expression of nab <dig> and nab <dig> corepressors. first, nab <dig> and, to a lesser extent, nab <dig> are activated in a delayed early fashion by a variety of stimuli that induce expression of egr family members  <cit> . for example, neuregulin treatment of schwann cells stimulates egr <dig> and activates nab <dig> and nab <dig> expression in a slightly delayed manner  <cit> . second, the nab <dig> promoter can be activated by egr factors in transient transfection studies , and ectopic expression of egr <dig> and egr <dig> stimulates expression of endogenous nab <dig> and nab <dig> in a variety of cell lines, including schwann cells  <cit> .\n",
            "\n",
            "these earlier studies, however, did not test direct regulation of nab corepressors by egr <dig>  our data provide several lines of evidence to establish nab <dig> regulation by egr <dig> in vivo. first, dnase footprinting analysis identified several egr <dig> protected sites in both nab <dig> and nab <dig> promoters. second, activation of nab <dig> by egr <dig> in transient transfection assays was severely compromised when egr <dig> binding sites were mutated . finally, chip assays demonstrated direct binding of egr <dig> to both promoters in cultured schwann cells as well as myelinating sciatic nerve . several studies have implicated neuregulin signaling as a critical factor in schwann cell survival and stimulation of myelination  <cit> . egr <dig> occupancy at both nab <dig> and nab <dig> was significantly enhanced upon neuregulin stimulation of schwann cells. our results provide the first demonstration of neuregulin-induced binding of a transcription factor to a specific promoter in schwann cells.\n",
            "\n",
            "several groups have shown that type iii neuregulin  plays a central role in regulating myelination in vivo  <cit> . therefore, addition of soluble neuregulin  does not act as a mimic of myelination. nonetheless, egr <dig> is regulated by both soluble neuregulin and interaction with type iii neuregulin  <cit> , and therefore soluble neuregulin probably induces some of the early pathways relating to egr <dig> induction in vivo. since egr <dig> expression is regulated by type iii neuregulin signaling in vivo  <cit> , our chip data in myelinating rat sciatic nerve presumably reflect type iii neuregulin-dependent binding of egr <dig> to the nab promoters.\n",
            "\n",
            "previous work showed that expression of nab <dig> and nab <dig> in zebrafish hindbrain is dependent on egr <dig> expression  <cit> . therefore, it was surprising that nab <dig> levels were not affected in egr <dig> knockout mice. in contrast, nab <dig> levels were significantly lower in the absence of egr <dig>  even though the nab <dig> promoter is less effectively activated by neuregulin and egr <dig> expression . these data suggested that nab <dig> might be coregulated by other factors that compensate for loss of egr <dig>  and promoter analysis revealed several conserved binding sites for ets transcription factors. ets family members are activated by neuregulin signaling  <cit> . since nab <dig> is more highly induced by neuregulin stimulation  <cit> , we tested whether ets proteins are involved in regulation of nab <dig> \n",
            "\n",
            "both ets <dig> and etv <dig> activated the nab <dig> promoter in transfection assays, and we observed a cooperative effect on the nab <dig> promoter when egr <dig> was co-expressed together with ets <dig> or etv <dig>  furthermore, chromatin immunoprecipitation analysis with an ets <dig> antibody detected binding in the nab <dig> promoter, but not in the nab <dig> promoter. unfortunately, we have been unable to find an etv <dig> antibody that is suitable for chip analysis, but recent immunohistochemical studies indicate that etv1/er <dig> is expressed in schwann cells  <cit> . based on our data, we propose that the higher induction of nab <dig> by neuregulin, compared to nab <dig>  <cit> , is due to synergistic activation of the nab <dig> promoter by egr <dig> and ets factors. consequently, the maintained expression of nab <dig> levels in the knockout could be mediated by elevated levels of etv <dig> , ets <dig>  <cit> , or other ets factors that may be revealed by a more comprehensive analysis of family member expression. in addition, although ets <dig> mrna is not induced in the egr <dig> knockout, its activity  could be elevated post-transcriptionally .\n",
            "\n",
            "in contrast, the nab <dig> promoter was more modestly activated by ets proteins, suggesting that nab <dig> is more exclusively dependent on egr factors and consequently is more reduced in the absence of egr <dig>  however, the modest induction of the nab <dig> promoter by ets factors in transfection assays could reflect some level of regulation of nab <dig> expression by ets proteins in schwann cells, particularly if there are other ets binding sites that lie outside of the nab <dig> promoter regions we have analyzed. a limitation of these studies is that the importance of the nab promoters relative to more remote enhancer elements has not been established. the mechanism of nab <dig> activation by ets remains to be determined, since only potential ets binding sites in the nab <dig> promoter have been identified. despite the commonality of the core ets motif , individual ets factors have distinct binding site preferences. therefore, future studies will focus on identifying the relevant ets factor using rnai approaches and/or knockout mice in order to facilitate such a mechanistic analysis.\n",
            "\n",
            "given the in vivo evidence that ets proteins regulate expression of egr <dig> and egr <dig>  <cit> , it seems likely that co-induction of egr <dig> and nab <dig> by ets transcription factor activation may have evolved to ensure formation of sufficient egr2/nab complexes that are required for specific gene regulation events in schwann cell differentiation. this may also prevent inappropriate activation of some target genes if nab <dig> was not immediately present at the onset of egr <dig> induction. alternatively, it is possible that nab <dig> modulates other transcription factors, so that its induction should not exclusively depend on egr <dig> \n",
            "\n",
            "several groups have proposed that the induction of nab <dig> constitutes a negative feedback loop in which egr activators induce expression of their own corepressor  <cit> . our studies refine this model by showing that other neuregulin-regulated pathways direct nab <dig> expression. in addition, recent publications have shown that egr <dig> is required for certain gene repression mechanisms during myelination  <cit> . therefore, we suggest that the induction of nab proteins does not merely constitute a negative feedback loop, but rather, that co-induction of egr <dig> and nab proteins by axon-dependent signals  is required to form an egr2/nab complex that actively represses transcription of specific genes during peripheral nerve myelination . these results are consistent with our demonstration of direct repression of the rad gene by an egr2/nab complex in schwann cells  <cit> , as well as recent publications implicating repression by the egr2/nab complex in macrophages and cardiomyocytes  <cit> .\n",
            "\n",
            "our studies do not rule out the possibility that other egr family members and/or other zinc finger proteins may partially compensate in maintaining nab <dig> levels in the absence of egr <dig>  indeed, previous work has shown that egr <dig> and egr2/krox <dig> are coexpressed in schwann cells at the onset of myelination . in contrast by one month of age, egr <dig> is exclusively expressed in myelinating schwann cells, whereas egr <dig> is confined to nonmyelinating schwann cells  <cit> . egr <dig> is induced ~2-fold in sciatic nerve of mice homozygous for a hypomorphic egr <dig> allele  <cit> , and our analysis indicates that egr <dig> mrna is induced about  <dig> -fold in the complete absence of egr <dig> . importantly, compensation by other egr family members is not sufficient to maintain nab <dig> levels despite the presence of several egr <dig> binding sites, and fails to maintain expression of other egr <dig> target genes like myelin protein zero, which is reduced up to 50-fold in the krox20/egr <dig> knockout  <cit> , suggesting that non-egr factors contribute significantly to regulation of nab <dig> expression. interestingly, regulation of nab expression by egr <dig> is consistent with previous observations that nab <dig> and nab <dig> are co-regulated with egr <dig> after nerve crush injury  <cit> . however, the decrease in nab <dig> expression following nerve injury would be consistent not only with regulation by egr <dig>  but also by loss of neuregulin-dependent  induction of ets factors, analogous to regulation of ets factors by neuregulin signaling observed in other systems  <cit> .\n",
            "\n",
            "the involvement of ets transcription factors in nab <dig> regulation resolves several puzzling observations regarding nab <dig> and nab <dig> activation. first, nab <dig> expression is more responsive than nab <dig> to several stimuli including serum, neuregulin and ngf  <cit> , even though the nab <dig> promoter contains more egr binding sites than nab <dig>  second, nab <dig> expression is already fairly elevated by e <dig> in profiling studies of peripheral nerve myelination, even though egr <dig> levels do not peak until p <dig>  <cit> . finally, a recent analysis of hematopoietic development showed that expression of the ets factor, pu. <dig>  induced expression of both egr <dig> and nab <dig>  and the induction of nab <dig> was shown to be egr2-independent  <cit> . our analysis of the nab <dig> promoter provides a mechanism for egr2-independent activation of nab <dig> by ets proteins.\n",
            "\n",
            "CONCLUSIONS\n",
            "our results demonstrate that egr <dig> is directly involved in regulation of both nab <dig> and nab <dig> corepressors, indicating that axon-dependent induction of nab expression is partially dependent on egr <dig>  however, the maintained expression of nab <dig> in egr <dig> deficient mice suggested that other factors could compensate for lack of egr <dig>  transfection experiments and chromatin immunoprecipitation assays indicated that ets <dig> and etv <dig> specifically activate the nab <dig> promoter, but not nab <dig>  which provides an explanation for the differential regulation of nab corepressors in several developmental systems. finally, these results suggest that nab corepressors are not simply feedback inhibitors, but that co-induction of egr <dig> with nab corepressors is critically involved in forming a transcriptional complex required for gene regulation events.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "our results suggest that egr <dig> is required for certain gene repression mechanisms during peripheral nerve myelination, and that egr <dig> may compensate in the absence of egr <dig> by binding of egr <dig> to the nab <dig> promoter.\n",
            "our results suggest that egr <dig> is required for certain gene repression mechanisms during peripheral nerve myelination, and that egr <dig> may compensate in the absence of egr <dig> by binding of egr <dig> to the nab <dig> promoter.\n",
            "our results suggest that egr <dig> is required for certain gene repression mechanisms during peripheral nerve myelination, and that egr <dig> may compensate in the absence of egr <dig> by binding of egr <dig> to the nab <dig> promoter.\n",
            "our results suggest that egr <dig> is required for certain gene repression mechanisms during peripheral nerve myelination, and that egr <dig> may compensate in the absence of egr <dig> by binding of egr <dig> to the nab <dig> promoter.\n",
            "our results suggest that egr <dig> is required for certain gene repression mechanisms during peripheral nerve myelination\n",
            "55\n",
            "[{'summary_text': 'the aim of the present study was to determine the degree of enzymatic activity of prephenate dehydrogenase  in s. mutans and to find an enzyme or an enzymatic biotype that could be targeted to prevent caries development.\\nthe aim of the present study was to determine the degree of enzymatic activity of prephenate dehydrogenase  in s. mutans and to find an enzyme or an enzymatic biotype that could be targeted to prevent caries development.\\nthe aim of the present study was to determine the degree of enzymatic activity of prephenate dehydrogenase  in s. mutans and to find an enzyme or an enzymatic biotype that could be targeted to prevent caries development.\\nthe aim of the present study was to determine the degree of enzymatic activity of prephenate dehydrogenase  in s. mutans and to find an enzyme or an enzymatic biotype that could prevent caries development.\\nthe aim of the present study was to determine the degree of enzymatic activity of prephenate dehydrogenase  in s. mutans and to find an enzyme or an enzymatic biotype that could prevent caries development.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "etiology of caries\n",
            "according to the latest reports, caries should be perceived as a multi-factor disease; four stages are needed for its development. the basic stage includes psychosocial, behavioral and genetic factors, the second level includes interactions between bacteria and the host, and also the influence of sugar in the diet, the third level includes biochemical changes - fermentation of carbohydrates to acids, leading to the fourth level, i.e. ph decrease and enamel demineralization  <cit> .\n",
            "\n",
            "it has been demonstrated in polish studies that falling asleep with a bottle, snacking between meals, eating sweets and the sweetening of food in a bottle are factors initiating development of the disease, significantly affecting the increase in the amount of streptococcus mutans bacteria, which is believed to be an important etiological factor for caries development  <cit> .\n",
            "\n",
            "the fact that for over 20 years the dental health of children in poland has not improved, seems to be worrying, and the ratio of children affected by caries of deciduous teeth has remained at about 54%  <cit> . the incidence rate of caries is expressed by the prevalence of caries and its intensity is illustrated by dmf . dmf indexes are different depending on the age of children, place of residence, and socio-economic conditions of parents, and may also depend on the sex of the child. despite public education in the prevention of caries, the problem of dental caries still remains a current topic.\n",
            "\n",
            "the pathogenic potential of s. mutans cariogenic bacteria  <cit>  is connected with the ability to metabolize a wide range of sugars, to form biofilm and to create an acidic environment  <cit> . one of the key intermediate products in s. mutans metabolism is pyruvate, and intracellular ph is maintained via this compound connected to amino acid synthesis. researches demonstrated that biosynthesis of branched chain amino acids  is one of the components of adaptation to an acidic environment  <cit> . also, a correlation was noted between tolerance towards low ph and biofilm formation  <cit> . since s. mutans is accepted to be an etiological caries factor, attempts to find a substance inhibiting cell proliferation have been undertaken. according to ku et al., targeting the pathways of aromatic amino acid biosynthesis, especially prephenate dehydrogenase , the enzyme catalyzing transition of prephenate into 4-hydroxyphenylpyruvate, may inhibit proliferation of s. mutants  <cit> .\n",
            "\n",
            "the pathway of aromatic amino acid biosynthesis is important in the metabolism of bacteria, fungi and plants and the pdh enzyme as a key component of the pathway may be essential for the pathogens’ survival in the host. auxotrophic towards amino acids mycobacterium tuberculosis strains lost the ability to proliferate and survive inside macrophages. bearing in mind the above reports, the pdh enzyme may be examined in the context of being a potential target for therapeutic agents  <cit> . the structure and enzymatic activity of pdh is being intensively analyzed worldwide. inter-species differences in the protein structure and features characteristic for enzymes are being identified. hydrolytic activity was also observed in pathogens like haemophilus inluenzae  <cit> , neisseria gonorrhoeae  <cit>  or escherichia coli  <cit> .\n",
            "\n",
            "typing of streptococcus strains\n",
            "in the literature, one can find many methods for typing streptococci strains. these are methods based on colony morphotype, resistance to various chemicals, enzymograms, as well as more advanced techniques of typing, namely pcr fingerprinting.\n",
            "\n",
            "the easiest way to type streptococci strains is morphotyping. this is based on the evaluation of the appearance of colonies on blood agar. color, edge, surface, the result of gram staining, and the type of hemolysis are assessed  <cit> .\n",
            "\n",
            "one of the earliest methods introduced for the phenotyping of streptococci is serotyping, which is based on agglutination. the most effective serotyping has been developed for β-hemolytic streptococci: s. pyogenes and s. agalactiae  <cit> ; however, no accurate differentiation of strains for epidemiological purposes is possible  <cit> .\n",
            "\n",
            "in auksanogram-based typing, intraspecific differences in the ability to assimilate different organic compounds are assessed. depending on the compounds used, different modifications of this method are available. typing of s. mutans strains based on auksanograms and enzymograms using tests like streptotest  or api strep  is quite popular. for streptotest  <dig>  the discrimination ability for the correct identification of strains has been estimated by the producer at  <dig> %, which is acceptable from a clinical point of view. in the phenotyping of streptococcus viridans with the use of the api  <dig> assay from biomerieux, discrimination ability was estimated at 79%  <cit> .\n",
            "\n",
            "in a study conducted by brtkova et al.  <cit> , biochemical assays from lachema, pliva and biomerieux were compared with genetic methods for enterococcus species isolated from animals. identification carried out with the use of the en-coccus assay  resulted in the classification of  <dig> species, out of a total of  <dig>  as enterococcus faecium;  <dig> randomly selected samples identified as e. faecium using the en-coccus assay, were identified again using the api20strep assay , and for  <dig> species, identification was not confirmed. of the  <dig> pre-selected strains,  <dig> were classified to the e. faecium species with the use of genetic methods, thus confirming the results obtained with the application of biochemical methods.\n",
            "\n",
            "the differences between biochemical assays were explained by differences in the number of specific substrates: the en-coccus assay contains only  <dig> sugars, while the api <dig> strep contains  <dig>  which allows for better discrimination  <cit> . teles et al. compared commercial api rapid  <dig> id strep and vitek  <dig> in the context of the identification of streptococcus viridans. 79% of the strains were correctly identified based on the api assay, while 19% were correctly classified for the phylogenetic group; the vitek assay was able to correctly characterize 55% of identified species; for 21% of the species, the phylogenetic group was correctly determined  <cit> .\n",
            "\n",
            "another method of typing is biotyping, based on selected enzymatic reactions of commercial assays. this kind of differentiation has been carried out for different species of streptococcus. barnham et al. identified s. zooepidemicus using a commercial api  <dig> strep assay, and next, based on the results of reactions, sensitivity to penicillin, erythromycin and tetracycline, they distinguished five biotypes of the species  <cit> . aarestrup and jensen conducted biotyping of streptococcus dysagalactiae basing on two reactions with the use of the commercial rapid  <dig> strep assay for this purpose: production of acid from tegatose and sorbitol, so that four biotypes of the species were observed. in parallel, based on the assessment of specific dna fragments, ribotypes were defined, and the obtained bio- and ribotypes were consistent  <cit> .\n",
            "\n",
            "in the studies of polish researchers, strains of b streptococci were differentiated using four hydrolysis reactions of sodium hippurate, lactose, salicin and trehalose. based on this method, eight biotypes, which do not correlate with known serotypes, were distinguished. according to the authors, the investigated biochemical typing may have a similar usefulness as serotyping in the differentiation of group b streptococci  <cit> . biotyping has also found an application in distinguishing streptococci from the group streptococcus mutans  isolated from plaque. yoo et al. conducted biotyping based on the fermentation profile of mannitol, sorbitol, raffinose, melibiose and arginine decomposition. they classified strains of s. mutans to existing biotypes with the following frequencies:  <dig> % - biotype i;  <dig> % - biotype v;  <dig> % - biotype iv;  <dig> % - biotype ii. in their study, two strains of other biotypes were also observed  <cit> .\n",
            "\n",
            "except for the phenotyping methods for streptococcus strains described above, there are techniques based on dna analysis – i.e. genotyping. the most common methods, with respect to their usage for streptococcus sp. strains, are presented below.\n",
            "\n",
            "pulsed field gel electrophoresis  is a method of electrophoretic karyotyping. electrophoretic separation of nuclear dna occurs in an alternating electric field. this method has been used for the karyotyping of clinical strains of s. pneumoniae  <cit> , s. pyogenes  <cit> , s. canis  <cit>  and s. mutans  <cit> . it appears to be consistent with rapd typing and auksonograms. it is described as the “gold standard of genetic fingerprints”, but rather time-consuming   <cit> .\n",
            "\n",
            "repetitive extragenic palindromic pcr  can be successfully used for genotyping s. mutans. it is based on a pcr reaction with the use of primers for repetitive sequences, followed by the comparison of the presence or absence of specific bands in agarose gel electrophoresis. compared to the pfge method, it is less expensive and less time consuming – thus, more practical for large-scale epidemiological studies  <cit> .\n",
            "\n",
            "another method for the assessment of “dna-fingerprinting” is the analysis of restriction fragment length polymorphism . a pcr reaction is performed, and then the amplification product is treated with a restriction enzyme. after electrophoresis, characteristic bands are analyzed. rflp-pcr is used for genetic studies of streptococcus, e.g. s. pneumoniae species  <cit> , s. bovis/s.equinis  <cit>  or s. mutans  <cit> .\n",
            "\n",
            "the pcr method and its different variants, depending on the purpose for which they are used, are the most common methods for the typing of pathogenic bacteria. moser et al. compared the genetic methods for epidemiological studies and concluded that the rep-pcr method exhibits sufficient quality at a relatively low cost  <cit> . other genetic studies on pathogenic bacteria are, in turn, based on rflp-pcr  <cit> , or sequencing of the 16s rrna subunit  <cit> . there are various methods for the molecular identification and typing of bacteria, and most are based on the pcr reaction, although amplified fragments and even the subsequent course vary depending on the method.\n",
            "\n",
            "aim of the study\n",
            "this study undertook an attempt to determine the degree of the metabolism of aromatic amino acids by s. mutans on the basis of one of the key enzymes of the aromatic amino acid pathway - prephenate dehydrogenase. this enzyme catalyzes the reaction of prephenate transition into 4-hydroxyphenylpyruvate and is not observed in humans  <cit> , and therefore could be a target for newly synthesized compounds of potential antibacterial activity. in the present study, an attempt at strain biotyping based on the streptotest  <dig> test was undertaken, and the usefulness of biotyping in determination of streptococcus mutans pathogenicity determinants was examined.\n",
            "\n",
            "our study aimed to identify bacteria present in caries and to find an enzyme or an enzymatic biotype that could be targeted to hinder bacterial survival and therefore to prevent caries development.\n",
            "\n",
            "methods\n",
            "subject of the study\n",
            "the study included strains isolated from patients  with diagnosed early childhood caries  of deciduous teeth, treated stomatologically during the period from february to april  <dig> at the department of pediatric dentistry, institute of dentistry, jagiellonian university, medical college, krakow, poland. the control group was formed of strains isolated from children  without caries lesions of deciduous teeth, who were kept under the clinic’s control .\n",
            "\n",
            "p\n",
            "ecc: early childhood caries; cf: caries free.\n",
            "\n",
            "aby the nonparametric mann–whitney u test for independent samples.\n",
            "\n",
            "bby the fisher’s exact test.\n",
            "\n",
            "cdmft, decayed, missing, and filled teeth.\n",
            "\n",
            "the classification of patients for the study was performed by a qualified dentist during a routine dental examination. dental examinations were based on who recommendations. the legal guardians of the participants of the study were fully informed about its aim and course, and agreed on their childrens’ participation. the factors excluding from participation in the study were insufficiently developed chewing activity observed in small children and mentally disabled patients, and the lack of an agreement from legal guardians for their childrens’ participation in the experiment.\n",
            "\n",
            "material collection\n",
            "the specimens were collected from carious foci in teeth of children with caries, and from dental plaque from children without caries. after classification of the patients to the research, scrapings from carious foci  or from dental plaque  were collected for examination using an open system composed of a sterile cotton swab placed in a test tube. the patients maintained a fasting status, and sampling was performed between 8-10 a.m. after prior oral cavity rinsing with distilled water. the collected material was sealed in sterile test tubes, inhibiting oxygen access, and transported at room temperature to the laboratory within a time period of no longer than 1 hour.\n",
            "\n",
            "culture of collected material on hlr-s medium\n",
            "the material was inoculated on a selective medium  in the laboratory. the range of inhibition of the growth of bacteria colonizing an oral cavity was verified in a preliminary study on four media, which are described in the literature as selective for this kind of pathogens  <cit> . the results are presented in table  <dig> \n",
            "\n",
            "p\n",
            "hlr-s: the modified medium of ritz; ba: bacitracin agar; tycsb: trypticase-yeast extract-cysteine sucrose-bacitracin agar.\n",
            "\n",
            "- no growth of bacteria.\n",
            "\n",
            "+ 3 × 102 cfu/ml.\n",
            "\n",
            "++ 2 × 103 cfu/ml.\n",
            "\n",
            "+++ > 3 × 104 cfu/ml.\n",
            "\n",
            "areference strain atcc  <dig> \n",
            "\n",
            "b, d, e, fclinical strains.\n",
            "\n",
            "creference strain atcc  <dig> \n",
            "\n",
            "gcontains: s. mutans, s. sanguis, s. aureus, s. pyogens, s. pneumonia, h. influenzae in phosphate buffered saline .\n",
            "\n",
            "haveraged from  <dig> samples.\n",
            "\n",
            "iaveraged from  <dig> samples.\n",
            "\n",
            "jby the parametric student-t test for independent samples.\n",
            "\n",
            "kby the parametric student-t test for independent samples.\n",
            "\n",
            "lby the nonparametric mann–whitney u test for independent samples.\n",
            "\n",
            "łby the parametric student-t test for independent samples.\n",
            "\n",
            "m by the parametric student-t test for independent samples.\n",
            "\n",
            "bacterial identification\n",
            "biotyping of the isolated strains\n",
            "enzymatic typing was performed based on the enzymatic profiles obtained from the streptotest  <dig> test . the enzymes contained in the test are presented in table  <dig> \n",
            "\n",
            "the strains were initially cultured on hlr-s medium for 48 hours at a temperature of 36°c in the presence of 10% co <dig>  the bacteria were then inoculated onto blood agar  medium, and incubated for 48 hours at 36°c in the presence of 10% co <dig>  the suspension of pure colonies of a density of 2- <dig>  on the mcfarland scale was prepared in 3- <dig>  ml sterile physiological saline. 100 μl of this bacterial suspension was put to each of the wells of the first row  and to the wells in column h .  <dig>  ml of the suspension of a density of 2- <dig>  on the mcfarland scale was transferred into physiological saline as a suspension carrier for the streptotest  <dig> . 100 μl of the prepared suspension was put into each of the rest of the reaction wells . a drop of paraffin oil was added to the wells labeled arg and s <dig> \n",
            "\n",
            "after transferring the prepared solutions, a streptotest  <dig> test incubation was performed for 24 hours at 37°c. the read-out was made visually based on a comparative color scale attached to the test; the reaction results were interpreted as “positive” or “negative”. the results were presented in a 7-point scale  depending on reaction intensity. the species were identified based on a code-book provided by the manufacturer .\n",
            "\n",
            "\n",
            "genotyping of the isolated strains\n",
            "\n",
            "\n",
            "dna preparation four colonies from each monoculture in ba medium  were suspended in 20 μl water . the suspension was heated for 15 minutes in 100°c on a heating block.\n",
            "\n",
            "the samples were centrifuged for 3 minutes at  <dig>  × g  in 4°c, with a break speed of zero. fifteen μl of the supernatant containing the bacterial dna were transferred into sterile  <dig> -ml tubes and frozen at -20°c until genetic analysis .\n",
            "\n",
            "\n",
            "16s rdna amplification independently of the identification with biochemical methods , the strains were identified based on their 16s ribosomal dna sequence, aligned against the 16s ribosomal rna sequences database  by blast.\n",
            "\n",
            "for this purpose, the 16s rdna fragment was amplified with use of universal primers . the pcr program, performed on the eppendorf mastercycler thermocycler, consisted of the following steps: initial denaturation  and 40 cycles of denaturation , annealing  and elongation , followed by a final elongation step . the reaction mixture at a final volume of 50 μl contained <dig> mm tris-hcl , 50 mm kcl,  <dig>  mm mgcl <dig>   <dig>  mm deoxynucleoside triphosphate,  <dig>  μm of each primer, 1u taq dna polymerase , and 100 ng bacterial dna.\n",
            "\n",
            "\n",
            "electrophoresis the obtained pcr product was separated by electrophoresis  on a 1% agarose gel with 4 μl ethidium bromide  for 35 minutes . the products were visualised by a gel documentation system with uv transilluminator .\n",
            "\n",
            "\n",
            "sequencing after purification of the pcr product by use of a commercially available kit , the 16s rdna fragment was sequenced: first, the reaction mix was prepared. the final volume of 15 μl contained 3 μl polymerase mix , 3 μl reaction buffer ,  <dig>  μl of 1 μm f or r primer , and 30 ng of the purified pcr product. all analyzed samples were sequenced in both directions – forward and reverse. steps of the sequencing pcr: 96°c for 1 min, 25 cycles comprised of 96°c for 15 seconds, 54°c for 7 seconds, 60°c for 5 minutes. the obtained products were purified by ethanol precipitation: 2 μl sodium acetate/edta  was added to each sample, followed by dna precipitation with 80 μl of 95% ethanol . the samples were mixed thoroughly and centrifuged for 15 minutes at  <dig>  × g. after removal of the supernatant, the dna pellet was rinsed with 70% etoh and centrifuged briefly. the pellet was then air-dried and resuspended in 20 μl highly deionized formamide  and immediately run on a sequencer  applied biosystems).\n",
            "\n",
            "\n",
            "sequence analysis the primer pair was designed to give a product of about  <dig> nucleotides in length in the case of s. mutans ua <dig> . a single read during capillary electrophoresis on the  <dig> series genetic analyzer unravels approx. 800- <dig> nucleotides. therefore, the obtained  <dig> results  were assembled into one consensus sequence by bioinformatic tools , exported into fasta format, and aligned against the 16s ribosomal rna sequences database accessible through the ncbi  website by megablast  . the conformity of sequences obtained by blast served as basis for the identification of strains  <cit> .\n",
            "\n",
            "the statistical clusterization method was used in order to find similarities between the obtained profiles.\n",
            "\n",
            "\n",
            "data accessibility the full 16s rdna sequences of the genetically analyzed strains are available at the national center for biotechnology information genbank database  under the following accession numbers: km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  km <dig>  and km <dig>  sequences that are gapped in the region of the read merge  are not included.\n",
            "\n",
            "examination of prephenate dehydrogenase  activity\n",
            "prephenate dehydrogenase  is an enzyme catalyzing the reaction of prephenate transition into 4-hydroxyphenylpyruvate, a compound from which tyrosine is directly synthesized . the subject of our research was pdh isolated from s. mutans strains. enzymatic activity of pdh was determined fluometrically based on a method described by champney and jensen for b. subitilis <cit> , with modifications introduced by ku et al. to determine the activity of the s. mutans-derived enzyme  <cit> .\n",
            "\n",
            "\n",
            "bacteria preparation\n",
            "\n",
            "pure bacterial colonies  were suspended in 12 ml of sterile pbs solution, and centrifuged 4°c, for 15 min at 3000 × g. the cell pellet was suspended in 3 ml of phosphate buffer  containing kcl  and lysozyme . the bacterial mixture after addition of lysozyme was incubated at a temperature of 37°c for 15 min. after incubation, dnase at a final concentration of 10 μg/ml was added to the mixture. cell remnants were removed by centrifugation at 12000 × g at 4°c, for 20 minutes. 3 ml of the supernatant was spread on a column containing sephadex g- <dig> . the protein eluate  was collected for pdh examinations  <cit> .\n",
            "\n",
            "\n",
            "pdh activity determination\n",
            "\n",
            "pdh activity was examined fluorimetrically at an excitation wavelength of 340 nm and emission at 460 nm for 5 minutes . the result was expressed as a decrease in fluorescence per 1 minute. 1 ml of the reaction mixture  ;  <dig>  mg tris-hcl ,  <dig>  mg kcl ;  <dig>  mg k2po <dig>  and 50 μl prephenate dehydrogenase) was incubated for 3 minutes at 37°c in a heating block. the reaction was initiated by addition of  <dig>  mg of prephenate  . the reduced nicotinamide adenine dinucleotide - nadh formed during the reaction was measured. the reaction was terminated by addition of 1 ml naoh  to the reaction mixture  <cit> .\n",
            "\n",
            "statistical analysis\n",
            "analysis of variance using the brown–forsythe test was applied for comparison of the activity of prephenate dehydrogenase  isolated from clinical strains of s. mutans in patients with caries  and without disease symptoms . conformity with a normal distribution , as well as non-uniformity of variance in the groups  were the basis for the application of a parametric test with separate variance assessment for independent samples  - the cochran and cox test.\n",
            "\n",
            "the values of the medians of pdh activity in the established biotypes  were compared. the non-parametric kruskal-wallis test for independent samples was applied. normality of distributions in the examined groups  was verified using the shapiro-wilk test. analysis of variance using the levene test was applied for comparison of the activity of pdh between specific biotypes.\n",
            "\n",
            "the statistical clusterization method was used in the case of biotyping. the results of the division into clusters were presented in the form of diagrams - dendrograms. the data were elaborated statistically using r language version  <dig> . <dig> and an environment for statistical calculations in the linux operating system, and also in statistica v. <dig> pl software . statistically significant differences were noted in the case of all analyzed groups .\n",
            "\n",
            "ethical approval\n",
            "consents were obtained from all patients, and the study procedure was approved by the bioethical committee of the jagiellonian university in krakow .\n",
            "\n",
            "RESULTS\n",
            "characteristics of isolated strains\n",
            "in total,  <dig> strains belonging to  <dig> streptococcus species were isolated from patients and controls, as well as  <dig> strain belonging to the gemella species. the s. mutans was the most abundant , followed by s. intermedius . the percentage contribution of particular species is presented in figure  <dig> \n",
            "\n",
            "the strains originated from carious foci of childrens’ decidous teeth  with diagnosed early childhood caries  - the examined group, and from dental plaque of children  without carious changes in decidous teeth - the control group.\n",
            "\n",
            "the contribution of the streptococcus species in both the examined and control populations was different; the frequency of s. mutans isolation in the particular groups is presented in figure  <dig> \n",
            "\n",
            "the predominance of s. mutans in caries in children was confirmed in the present study where it was isolated from 77% of carious foci. selective hlr-s medium  was applied in order to isolate the species from clinical material. diagnostic sensitivity for the above medium was 77%, while diagnostic specificity was 76%. the obtained frequency of s. mutans isolation from children with caries  is at a similar level to that identified in the study by kanasi et al. – 73%  <cit>  .\n",
            "\n",
            "the percentage of s. mutans strains observed in the examined group with caries was over three-fold higher than in the control group , which confirms the relationship between the presence of s. mutans species and early childhood caries. comparing the results obtained with current literature reports , a similarity in the frequency of s. mutans occurrence in children with caries may be noticed. in material derived from children without caries  s. mutans was isolated from  <dig> samples, i.e. 21%. this value is lower than the results obtained in similar research worldwide. this fact may be explained by the application of more sensitive methods in the other studies. choi et al., as well as kanasi et al., used genetic methods in their studies for direct identification of the bacterial flora of children’s oral cavity. tanner et al., based on “classic” microbiology methods - microorganism culture, obtained results close to those obtained in the present study . the higher frequency of s. mutans isolation obtained in the study by choi et al. may be connected to environmental predispositions or different feeding habits in the examined groups.\n",
            "\n",
            "16s rdna genotyping\n",
            "out of  <dig> isolated strains, which were typed biochemically with streptotest <dig>  the 16s rdna sequence was determined in strains with a discrimination rate of less than 95%. thirteen strains were not genotyped because we failed to isolate dna. in total, the 16s rdna sequence was analyzed for  <dig> strains with a discrimination rate of less than 95% and for  <dig> randomly chosen strains with a discrimination rate of over 95%, in order to confirm the arbitrally accepted threshold.\n",
            "\n",
            "the genetic typing confirmed the biochemical test in 75% of strains .\n",
            "\n",
            "for both randomly chosen strains the results obtained by biochemical and genetical methods were unanimous. in the first case, s. intermedius was revealed by the streptotest <dig> with a determination rate of  <dig> %, and by genotyping based on a 99% identity with the 16s rdna sequence from s. intermedius. the second case was designated as s. intermedius at a discrimination rate of  <dig> % by streptotest <dig> and 100% by sequencing.\n",
            "\n",
            "the genetic analysis confirmed  <dig> cases among  <dig> , for which the biochemical identification showed low discrimination rates : in one case, where s. salivarius/s. mutans/s. sanguinis were typed biochemically with the probability of  <dig> ,  <dig>  and  <dig> %, respectively, the genetic analysis confirmed the presence of s. salivarius. in two cases with a low discrimination rate of  <dig> / <dig> / <dig> % for s. canis/s. sanguini/s. intermedius respectively, the analysis of dna revealed the presence of s. intermedius. two other cases, phenotypically designated as s. pyogenes/s. canis/s. sanguinis with the probability of  <dig> / <dig> / <dig> % respectively in the streptotest <dig>  were genetically equal to s. intermedius. one strain identified biochemically as s. intermedius/s. pneumoniae/s. sanguinis  genetically turned out to belong to the lactobacillus species .\n",
            "\n",
            "the biochemical result for another tested sample  were as follows: s. canis/s. sanguini/s. intermedius , which could not be confirmed genetically. the length of the amplified 16s rdna read on the genetic analyzer was  <dig> base pairs. the sequence of the obtained 16s rdna for the above-mentioned sample  was shown by blast to be 99% identical to the 16s rdna of s. intermedius. therefore, the strain was eventually identified as s. intermedius.\n",
            "\n",
            "biotyping\n",
            "due to the small amount of isolates belonging to s. pneumoniae, s. canis, s. cristatus, s. suis, s. saniguinis, gemella sp., leuconostoc sp., s. pyogenes, s. salivarius s. intermedius, biotypes were determined only for s. mutans. two approaches to the determination of biotyping criteria, based on the profiles of enzymatic activity of the streptotest  <dig> test, were applied.\n",
            "\n",
            "the first approach involved the selection of suitable enzymes based on the analysis of their activity in the population of the examined strains. ten enzymes present in all the strains  were not suitable for differentiation and were rejected per se. these were tested on the basis of their growth in the presence of  <dig> % nacl which was negative in all the strains, and  <dig> enzymes which were not noted in the strains . another two enzymes  were observed only in one case and were also rejected since their application in biotyping would cause the distribution of strains in particular biotypes to be non-uniform, i.e. an overly large number of biotypes would be present for which only one strain would be classified. another  <dig> enzymes seemed to be suitable for biotyping. their occurrence among the strains is within the quartile deviation - between  <dig>  - <dig> %. these are: phosphatase  <dig> %, inulin–  <dig> %, melibiose –  <dig> %, tegatose –  <dig> % .\n",
            "\n",
            "the acceptance of these enzymes as biotyping criteria allowed for the definition of  <dig> enzymatic profiles, which are conventionally designated using sequential alphabetical lettering from a to g, as presented in table  <dig>  the most abundant was profile a characterized by a lack of phosphatase, inulin and tegatose activity.\n",
            "s. mutans \n",
            "biotypes obtained based on the activity of four enzymes\n",
            "\n",
            "the distribution of the profiles thus obtained was non-uniform; therefore, strains differing only in terms of one reaction were grouped in one biotype. the biotypes presented in the table below were obtained .\n",
            "s. mutans \n",
            "biotypes based on differences in the activity of phosphatase, inulin, tegatose and melibiose\n",
            "\n",
            "biotype i was observed in 64% of cases, was characterized by a lack of phosphatase  activity and a lack of inulin  decomposition, however it demonstrated the presence of a melibiose  decomposing enzyme, and tegatose  decomposition was a variable feature. biotype ii observed in 26% of cases was characterized by decomposition of inulin  and tegatose  with a lack of phosphatase  activity; tegatose  distribution was not uniform within the biotype. biotype iii  exhibited the activity of phosphatase  and enzyme decomposing inulin , reactions of melibiose  and tegatose  fermentation were negative. the least abundant was biotype iv, characteristic for 4% of the examined strains, which demonstrated phosphatase  activity, decomposed melibiose  but not inulin , while tegatose  decomposition was variable.\n",
            "\n",
            "the second method of biotype determination was the application of non-supervised statistical data analysis . in this method the data were divided into groups  so that each of the groups was as uniform as possible , and concurrently the clusters were diversified .the results of division into clusters are presented in the form of diagrams - dendrograms. in the case of the most adjusted dendrogram it appeared that the strains may be differentiated based on the activity of inulin, melibiose and tegatose. this allowed  <dig> biotypes to be separated - the branches described with roman figures  on the diagram . the figure also shows the distribution of enzymatic profiles obtained using the arbitrary method.\n",
            "s. mutans \n",
            "biotypes.\n",
            "\n",
            "enzymatic activity of prephenate dehydrogenase \n",
            "enzymatic activity of prephenate dehydrogenase  was determined for clinical s. mutans strains . the existence of relationships was examined between pdh activity in bacteria and caries presence in the patient the strain was isolated from. the parametric test with separate variance assessment for independent samples  - the cochran and cox test, was applied to compare the activity of prephenate dehydrogenase pdh between the examined and control groups. mean enzymatic activities of prephenate dehydrogenase pdh in the control and examined groups differed statistically  .\n",
            "\n",
            "the enzymatic activity of pdh was compared in the context of s. mutans biotypes i-iii proposed in the study . the non-parametric kruskal-wallis test for independent samples was applied in order to compare the activity of prephenate dehydrogenase pdh between particular biotypes . attention was paid to the presence of statistically significant differences  .\n",
            "\n",
            "discussion\n",
            "occurrence and characteristics of the isolated strains\n",
            "since the main source of s. mutans infections is the oral cavity, it seems to be significant to determine what percentage of s. mutans strains inhabiting the human oral cavity and its direct vicinity possess the features determining their ability to cause caries.\n",
            "\n",
            "\n",
            "occurrence of streptococcus mutans strains\n",
            "\n",
            "cases of early childhood caries of deciduous teeth described in casuistic studies are most often caused by the s. mutans species  <cit> . the frequency of species occurrence is variable and ranges from 69%  <cit>  to 100%  <cit> . in the event of an absence of s. mutans, the species isolated the most often from children with caries are: s. gordonii, s. cristatus and s. intermedius  <cit> .\n",
            "\n",
            "in the whole examined population, the s. mutans species were isolated from 60% of study participants. the obtained result is consistent with the results obtained by brauncajs et al  <cit> .\n",
            "\n",
            "apart from s. mutans strains, other species belonging to the streptococcus genus were also identified in the present study, mainly s. intermedius, s. salivarius, s. cristatus . the conducted study confirms that the presence of pathogenic bacteria itself is not sufficient for caries development - s. mutans strains were cultured from 21% of the children classified by a dentist as “healthy”.\n",
            "\n",
            "biotyping\n",
            "the biotyping of s. mutans species is an interesting issue which focuses attention of researchers all over the world  <cit> . apart from the availability of genetic methods allowing better differentiation of the species, e.g. for epidemiological purposes  <cit> , typing on the biochemical level, which due to low cost is a competitive method, is still an open issue. there are however no biotypes, and thus no enzymes determined in the literature, on the basis of which biotyping for s. mutans would be possible. we noted that the most suitable enzymes for strain differentiation are: phosphatase , inulin , melibiose  and tegatose .\n",
            "\n",
            "comparing the reaction results for the mentioned compounds obtained in our study with results obtained by yoo et al.  <cit>  it appeared that  <dig> out of  <dig> strains demonstrated the same profile. the scheme proposed by us and based on the metabolism of inulin, melibiose, tegatose and phosphatase activity is more adequate for establishing s. mutans biotypes in the polish population. phosphatase present in the strains isolated by us was characterized by variable activity, therefore that feature was used for biotyping.\n",
            "\n",
            "the method seems to be promising in the context of recognition of mechanisms of vertical and horizontal bacteria transmission. it would be worth verification in further studies to ascertain whether the proposed biotypes are characteristic for related persons .\n",
            "\n",
            "the frequencies of occurrence of particular enzymes tested by facklam et al.  <cit>  were comparable with those obtained in our study, i.e. above 95%. in the case of melibiose decomposition, 93% of examined strains demonstrated the activity of a suitable enzyme, while facklam observed that feature in nearly half of the strains isolated from the plaque  and 88% of strains isolated from blood.\n",
            "\n",
            "melibiose decomposition is used in s. mutans differentiation  <cit> , and the selection of that feature seems to be suitable for typing based on biochemical profiles. it should, however, be noted that strains not fermenting melibiose are a genetically non-uniform group.\n",
            "\n",
            "another way of establishing biotypes, i.e. clustering of “similar” bacterial strains, is the application of the statistical method of clusters analysis - clusterization. the clusterization method allows the activity of all enzymes of the streptotest  <dig> test to be taken into account in defining biotypes. it appeared that the best differentiation of biotypes may be obtained based on occurrence of the activity of three enzymes decomposing inu, mlb and tgt. the obtained biotypes appeared to be useful in the differentiation of the strains in terms of pdh activity. attention was paid to the existence of a relationship between pdh activity and affiliation to a specified biotype . all the proposed biotypes were characterized by sufficiently frequent occurrence, the least abundant biotype iii included almost 10% of the examined strains .\n",
            "\n",
            "enzymatic activity of prephenate dehydrogenase \n",
            "the study conducted is the first one, which examined the enzymatic activity of the pdh enzyme isolated from s. mutans clinical strains. attention was paid to the existence of intra-species differences in the degree of enzyme activity , which may be explained by the dissimilarity of the patient oral cavity environments the strains were isolated from .\n",
            "\n",
            "our study indicates the existence of a relationship between enzyme activity in microorganisms and caries occurrence. mean enzyme activity among the strains derived from carious foci was  <dig>  u/mg, while the strains derived from healthy oral cavities demonstrated a 4-fold lower enzyme activity  . among the strains isolated from healthy children, pdh activity did not exceed  <dig> u/mg . in the case of strains isolated from carious foci all demonstrated pdh activity above  <dig> u/mg . possibly, there is a critical value  of enzyme activity, beyond which an increase in microorganism virulence is noted. further studies should be performed in order to establish whether the described differences in pdh activity are of a genetic or environmental background. low enzyme activity is connected to s. mutans strains inhabiting healthy oral cavities. it is interesting, whether prevention in the form of enzyme inhibitors in the case of the control group would slow down, and as a consequence inhibit disease development in the future. based on the study, it should be supposed that synthesis of tyrosine, one of the aromatic amino acids connected to prephenate dehydrogenase metabolism, may be one of the factors playing a role in s. mutans pathogenicity, and thus disease development in children.\n",
            "\n",
            "pdh activity should be considered in a wider context, as a component of the whole bacterial metabolism also including other biochemical pathways. this is confirmed in the present study by the demonstrated correlation between the degree of pdh activity and affiliation of the strain to one of the proposed s. mutans biotypes .\n",
            "\n",
            "prephenate dehydrogenase  is absent in humans and therefore, according to ku et al., could be a target for therapeutic specific inhibitors  <cit> .\n",
            "\n",
            "various attempts at caries prevention are being widely undertaken all over the world: there is a search for anti-caries substances in food products such as black tea  <cit> , as well as compounds against acid-forming bacteria acting in low ph are being synthesized  <cit> , and the application of mutated s. rattatus strains as probiotics in caries prevention has also been proposed  <cit> . the present study may be a starting point in the creation of an alternative way of limitation, or even elimination, of the pathogenicity of s. mutans strains as a result of minimization of prephenate dehydrogenase  activity. the structure of the enzyme is known  <cit> . the synthesis of specific pdh inhibitors should be considered, and further studies aimed at verification as to whether this type of activities may prevent caries formation should be performed.\n",
            "\n",
            "CONCLUSIONS\n",
            "s. mutans species is predominant among bacteria of the streptococcus genus in early childhood caries  in carious foci. s. mutans strains may be differentiated into biotypes based on the profiles of enzymatic activity of the streptotest  <dig> test. the prephenate dehydrogenase activity of most s. mutans strains from children with caries was higher than prephenate dehydrogenase activity demonstrated by compared s. mutans strains isolated from the children without disease symptoms. this may suggest a key role of that feature as a determinant of bacterial pathogenicity. attention should be paid to the usefulness of biotyping in determination of selected pathogenicity determinants - prephenate dehydrogenase pdh activity in s. mutans.\n",
            "\n",
            "our results prompt further studies to be undertaken to confirm or exclude the pathogenicity of selected streptococcus strains in an animal research model, and also cast new light on the formation of potential compounds blocking the metabolism of aromatic amino acids  which are connected to the life cycle of streptococcus mutans.\n",
            "\n",
            "current studies on pdh isolated from s. mutans indicate that this enzyme has a homodimeric structure and each dimer consists of two domains: an n-terminal domain binding nad, and a c-terminal domain binding prephenate  <cit> . literature data indicate that enzyme activity is the highest at ph  <dig>  or lower. the concentration of nacl does not have any effect on pdh activity. no compounds were found in literature that could decelerate or modify the activity of prephenate dehydrogenase. in our studies, enzymatic activity in the range ph  <dig>  –  <dig>  was demonstrated. an intense decrease in pdh activity was observed at ph >  <dig> . some authors have assumed that pdh activity can be inhibited via the feedback inhibition of tyrosine  <cit> . however, there is insufficient data to support such a hypothesis.\n",
            "\n",
            "the present observations are the first which demonstrate the relationship between one of the potential factors determining the pathogenicity of the s. mutans species - the activity of prephenate dehydrogenase  and biotyping of the s. mutans species based on phenotypic features . the results obtained this way may allow for better differentiation of the s. mutans species and thus may contribute to better recognition of the mechanisms of the transmission of pathogenic bacteria inhabiting the human oral cavity and facilitate treatment.\n",
            "\n",
            "competing interests\n",
            "all authors declare that they have no competing interests.\n",
            "\n",
            "authors’ contributions\n",
            "wk is the person behind the idea of the study, and participated in its design, prepared and analyzed samples, performed all non-genetic experiments during the study. wk also performed the statistical analysis and data interpretation, and was responsible for the manuscript preparation. kkp participated in study design, prepared the samples and performed the phenotyping experiment with the data analysis. jp, as participated in study design, prepared the samples and performed the genotyping experiment with the data analysis. as is additionally responsible for corrections leading to the final version of the manuscript. pk participated in study design. aj participated in study design. dk participated in study design and collected clinical material. all the authors read and approved the submitted version of the manuscript to the journal.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the aim of the present study was to determine the degree of enzymatic activity of prephenate dehydrogenase  in s. mutans and to find an enzyme or an enzymatic biotype that could be targeted to prevent caries development.\n",
            "the aim of the present study was to determine the degree of enzymatic activity of prephenate dehydrogenase  in s. mutans and to find an enzyme or an enzymatic biotype that could be targeted to prevent caries development.\n",
            "the aim of the present study was to determine the degree of enzymatic activity of prephenate dehydrogenase  in s. mutans and to find an enzyme or an enzymatic biotype that could be targeted to prevent caries development.\n",
            "the aim of the present study was to determine the degree of enzymatic activity of prephenate dehydrogenase  in s. mutans and to find an enzyme or an enzymatic biotype that could prevent caries development.\n",
            "the aim of the present study was to determine the degree of enzymatic activity of prephenate dehydrogenase  in s. mutans and to find an enzyme or an enzymatic biotype that could prevent caries development.\n",
            "\n",
            "56\n",
            "[{'summary_text': 'our results suggest that the composition and structure of leptospira lps is different between pathogenic and intermediately pathogenic leptospira species, which can be further exploited by comparative analysis of key components to understand the pathogenesis and host colonization.\\nour results suggest that the composition and structure of leptospira lps is different between pathogenic and intermediately pathogenic leptospira species, which can be further exploited by comparative analysis of key components to understand the pathogenesis and host colonization.\\nthis study is the first to compare the biochemical composition and structure of leptospira lps from pathogenic and intermediately pathogenic leptospira species, and the first to demonstrate that the composition and structure of leptospira lps is different between pathogenic and intermediately pathogenic leptospira species.\\nour results suggest that the composition and structure of leptospira lps is different between pathogenic and intermediately pathogenic leptospira species, which can be further exploited by comparative analysis of key components to understand the pathogenesis and host colonization.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "human leptospirosis is a common, globally-important and neglected zoonotic infectious disease, caused by spirochetes of the genus leptospira  <cit> , a disease of particular public health importance in tropical and subtropical countries  <cit> . chronically infected mammalian reservoir hosts  excrete the organism in urine, contaminating water and soil the principal vehicles for human infection. there are at least  <dig> recognized species of leptospira that have been further classified into three major subgroups: pathogenic, intermediately pathogenic and non-pathogenic   <cit> , comprising ≥  <dig> serotypes  based primarily on the immunological characterization of surface lipopolysaccharide  , a feature that distinguishes leptospira from other human-infecting spirochetes, which do not produce lps.\n",
            "\n",
            "lpss are complex, amphipathic biomolecules that constitute the major surface antigen of gram-negative bacteria  <cit> . lps is composed of three covalently-linked distinct components: lipid a, which is the hydrophobic part embedded in the outer membrane; o-antigen or o-polysaccharide, which is extended from the cell surface to the external environment; and the core oligosaccharide, which links the o-antigen to the lipid a. lps with and without o-antigen side chains are referred to as smooth and rough lps, respectively. several gram-negative organisms have been shown to produce heterogeneous  lps  <cit> . leptospira lps plays an essential role in pathogenesis colonization and dissemination of leptospira in mouse models . lps is one of the target antigen for diagnosis  <cit>  and also potential candidate for vaccine development  <cit> .\n",
            "\n",
            "in contrast to the pathogenic strain, l. interrogans serovar copenhageni strain fiocruz l1- <dig>  the intermediately pathogenic strain l. licerasiae serovar varillal strain var  <dig> grows rapidly in vitro, but neither caused disease nor was molecularly detectable in experimentally infected hamsters or mice  <cit> . l. licerasiae serovar varillal strain var  <dig>  causes mild disease in humans  <cit>  compared to l. interrogans serovar copenhageni strain fiocruz l1- <dig> infection, which frequently result in life-threatening illness characterized by jaundice, shock, kidney injury and hemorrhage  <cit> . these critical observations suggest that there are important biological and virulence differences between pathogenic and intermediately pathogenic leptospira species, which can be further exploited by comparative analysis of key components to understand the pathogenesis and host colonization. lps is considered as one of the central component of the pathogenesis in gram-negative organisms including pathogenic leptospira species  <cit> . therefore, comparative characterization of lps of a pathogenic and intermediately pathogenic leptospira species is critical to delineate sugar and fatty acid molecules essential for the pathogenesis and colonization in the host.\n",
            "\n",
            "apart from its overall structural similarity to gram-negative lps, relatively little is known about the composition and structure of leptospira lps  <cit> . until now, the chemical composition and structure of the lps produced by an intermediate leptospira have not been studied. this is particularly important considering the reduced complexity of the l. licerasiae serovar varillal rfb locus  compared to that of l. interrogans copenhageni, which contains  <dig> genes, despite the fact that both serovars have a common reservoir host: rattus norvegicus  <cit> . though this association is not absolute, reservoir species are thought to maintain specific serovars such as copenhageni in rats and canicola in dogs. indeed, despite differing genomic backgrounds, l. interrogans subtype harjoprajitno and l. borgpetersenii serotype hardjobovis are indistinguishable serologically and share a common reservoir host  <cit>  suggesting that lps plays an important role in determining host-specificity.\n",
            "\n",
            "to establish the biochemical basis for the structural determination of the carbohydrate component of leptospiral lps, the biochemical composition of lps of the high-grade pathogenic l. interrogans serovar copenhageni was compared to the lps of the intermediately pathogenic l. licerasiae varillal lps. further, of biological relevance given that pathogenic bacterial lps may well be associated with host adaptation, these two leptospira share ratttus species as their mammalian reservoir host  <cit> . importantly, the rfb loci of these two leptospira have great differences in their complexity: l. interrogans serovar copenhageni has  <dig> rfb locus genes  <cit> , while l. licerasiae has  <dig> rfb locus genes  <cit> . this study is the first to compare the biochemical composition of lps from pathogenic and intermediate leptospira sharing a common reservoir host and having varying pathogenic potentials, and will underpin future structural studies of leptospiral lps.\n",
            "\n",
            "methods\n",
            "ethics statement\n",
            "this study was carried out in accordance with the recommendations in the guide for the care and use of laboratory animals of the national institutes of health in aaalac-approved facilities. the experimental animal work was approved by the institutional animal care and use committee of the university of california san diego under protocol s03128h.\n",
            "\n",
            "strains and culture condition and lps extraction\n",
            "leptospira interrogans serovar copenhageni strain fiocruz l1- <dig>  a pathogen, was maintained in-vitro, passaged through hamsters, and re-isolated from infected hamster liver; low-passage derivatives  were used for all the experiments. the intermediate pathogen, l. licerasiae serovar varillal  was originally isolated from rattus spp. in the peruvian amazon  <cit> , adapted to in-vitro culture conditions in our laboratory, and analyzed at the whole genome level  <cit> . both strains were cultivated in ellinghausen-mccullough-johnson-harris  medium  with slow rotary shaking  at 30 °c. leptospira cells were collected from mid-log cultures by centrifugation at  <dig>  rpm for 30 minutes, and then washed three times with sterile pbs before lps extraction.\n",
            "\n",
            "crude lps was prepared by hot phenol-water extraction as described previously  <cit> , and extensively dialyzed against milli-q water  to remove phenol then lyophilized. lyophilized crude material was re-suspended in 10 ml of mill-q water, treated with dnase and rnase  followed by proteinase-k digestion . purified lps was pelleted by ultracentrifugation , dissolved in milli-q water and lyophilized. stock solutions of 1 mg/ml of purified lps were prepared from both aqueous and phenol phases, and known amounts  were used for sds-page and chemical composition analysis.\n",
            "\n",
            "chemical composition analysis of lps extracted from aqueous  and phenol  phases\n",
            "neutral monosaccharides were identified by gc-ms as their alditol acetate derivative . briefly, a known amount of lps was spiked with 2 μg of myo-inositol as internal standard, followed by hydrolysis with 2 n trifluoroacetic acid  at 100 °c for 4 h. hydrolyzed samples were cooled to room temperature, and a dry nitrogen flush used to remove excess acid followed by repeated co-evaporation of the reaction mixture with 100 μl of 50 % aqueous isopropyl alcohol. the liberated monosaccharide aldoses were further reduced to corresponding alditols by sodium borohydride solution  at room temperature overnight. excess sodium borohydride was neutralized using cold aqueous 30 % acetic acid , followed by removal of boric acid from the reaction mixture as volatile methyl borate using a dry nitrogen flush. finally, sugar alditols were reacted with a 1: <dig>  mixture of pyridine and acetic anhydride at 100 °c for 1 h to form corresponding alditol acetate derivatives. after removal of excess reagent by dry a nitrogen flush, dried samples were extracted with dichloromethane and analyzed by gc-ms  using a restek-5 ms  capillary column. ultrapure helium was used as carrier gas at a linear flow rate of  <dig>  ml/min and a temperature gradient . the injector temperature and transfer temperatures were maintained at 220 °c and 280 °c respectively. the constituent monosaccharides in lps were identified and quantified by comparing the retention time and mass fragmentation pattern in electron impact  positive ion mode with standard sugars.\n",
            "\n",
            "fatty acid analysis of aq and ph layers by gc-ms as fatty acid methyl ester  and trimethyl silyl ether -fame of hydroxyl fatty acids\n",
            "dried lps samples  from both aqueous and phenol layers were treated with 200 μl 1 m methanolic hcl prepared using a 3 n methanolic hcl kit  for 16 h at 80 °c. the samples were placed in an ice bath and slowly evaporated under a dry nitrogen stream until almost half of the sample volume had evaporated. to this reaction mixture, cold half-saturated nacl was added followed by the addition of 1 ml chloroform. the samples were vortexed at high speed for 1 min with a 20-sec pulse, and then allowed to settle for 1 min, and then centrifuged at 2000 rpm for 2 min  to separate organic and aqueous layers. the lower organic layer was carefully removed to a glass hydrolyzing tube, and the aqueous layer was extracted twice with  <dig>  ml of chloroform. chloroform fractions were pooled then washed with  <dig>  ml of cold milli-q water twice, and the organic layer was dried down. for detection of hydroxy fatty acids, the dried fatty acid methyl esters were treated with 100 μl of tri-sil htp reagent  at 80 °c for 30 min. the samples were cooled in an ice bath and then dried using a dry nitrogen stream, and then dissolved in hexane and analyzed by gc-ms.\n",
            "\n",
            "fame derivatized samples were analyzed by gc-ms  using a restek-5 ms  capillary column. ultrapure helium was used as a carrier gas at a linear flow rate of  <dig>  ml/min and the following temperature gradient . the injector temperature and transfer line temperature were maintained at 220 °c and 280 °c. fatty acids were identified by their characteristic ei fragmentation pattern and corresponding retention times and the percentage composition calculated from the area under the respective peaks.\n",
            "\n",
            "kdo  analysis\n",
            "kdo was analyzed using reverse phase hplc with a c <dig> column after tagging with a fluorophore  <dig>   <dig> methylenedioxy- <dig>  2-phenylenediamine dihydrochloride  . briefly, 25 μg of lps was hydrolyzed using 100 μl of 2 m hoac at 80 °c for 3 h followed by removal of hoac in a speed vac. the sample was then spin filtered using a 10 k  spin filter . the flow-through containing free kdo was reacted with dmb reagent at 50 °c for  <dig>  h; a known amount of dmb-kdo was injected into an rp-hplc equipped with an online fluorescence detector . a phenomenex c <dig> reverse phase column  was used for hplc analysis with an isocratic solvent mixture of consisting of methanol  and acetonitrile . for fluorescence detection, the excitation and emission wavelengths were set to 373 nm and 448 nm respectively.\n",
            "\n",
            "electron microscopy\n",
            "a drop of lps solution  was deposited on a carbon coated mesh grid  and air-dried. the sample containing mesh was negatively stained with 2 %  uranyl acetate following a published method  <cit> . the air-dried stained grid was examined using a joel  <dig> ex ii tem microscope at 80 kv available at the core facility at uc san diego .\n",
            "\n",
            "RESULTS\n",
            "electrophoretic mobilty of lps extracted from aq and ph phases\n",
            "the silver-stained electrophoretic profiles of lps fractions from l1- <dig> and var  <dig> were compared with that of an e. coli control . the migration pattern of leptospira lps of both strains did not show the ladder-like pattern, characteristic gram-negative smooth  lps. the electrophoretic mobility of var <dig> lps found in aq and ph layers were similar, consisting of fast migrating low-mw band. by contrast, the aq and ph layers of l1- <dig> lps were distinct. l1- <dig> lps isolated from the aq layer displayed higher-mw to low-mw bands, forming a smear like pattern.fig.  <dig> comparative electrophoretic profile of lipopolysaccharide  extracted from pathogenic and intermediately pathogenic leptospira strains. lps was extracted from the pathogenic species, leptospira interrogans serovar copenhageni strain fiocruz l1- <dig>  and from the intermediately pathogenic species, leptospira licerasiae serovar varillal strain var <dig>  lps was isolated from both aqueous  and phenol  layers of both species; profiles were determined by running 20 μl of lps samples in sds-page followed by a periodic acid-silver staining method to detect lps. escherichia coli 055: b <dig>  lps was used as a standard lps, and protein ladder  was used to localize the approximate the size of the lps band\n",
            "\n",
            "\n",
            "\n",
            "gc-ms analysis of monosaccharides obtained from aq and ph phase lps\n",
            "the carbohydrate composition of lps of serovars copenhageni and varillal are summarized in fig.  <dig> and table  <dig>  the monosaccharides present in both the strains are arabinose , rhamnose , xylose , mannose  and galactose . several other sugar moieties are only present in the pathogen l. interrogans copenhageni were detected, including fucose , n-acetylglucosamine  and trace amounts of n-acetylgalactosamine . two other sugars were present only in the aq layer of serovar copenhageni , epimers of 2-n-acetyl- <dig>  6-dideoxy hexosamine . the retention time suggests these sugars could be either gluco- or galacto- configured dideoxy hexnac sugars, commonly known as n-acetyl quinovosamine  and n-acetyl fucosamine . chemical composition analysis also demonstrated the presence of o-methylated hexose residue in different fractions of lps of both strains.fig.  <dig> carbohydrate composition analysis of lps of pathogenic and intermediate pathogenic leptospira strains. lps extracted from the pathogenic species, leptospira interrogans serovar copenhageni strain fiocruz l1- <dig>  and from the intermediately pathogenic species, leptospira licerasiae serovar varillal strain var <dig>  was subjected to gas chromatography–mass spectrometry  chromatography of monosaccharides as alditol acetate derivatives:  copenhageni aqueous layer lps;  copenhageni phenol layer lps;  var <dig> aqueous layer lps;  var <dig> phenol layer lps. rha = 6-deoxymannopyranose; fuc = 6-deoxygalactopyranose; ara = arabinose; xyl = xylose; ino = myo-inositol ; man = mannose; glc = glucose; gal = galactose; glcnac = 2-n-acetylglucosamine; l-d-heptose = l-glycero-d-mannoheptose\n",
            "\n",
            "lps from leptospira interogans serovar copenhageni strain l1- <dig>  and intermediate pathogenic, leptospira licerasiae serovar varillal strain var <dig>  were purified from both aqueous  and phenol layer  and subjected for composition analysis separately. the sugar composition values of aqueous, phenol and combined mean values  are presented as mole percentage of monosaccharides obtained in fractions. nd = not detected; fucnac = 2-nacetyl- <dig>  6-dideoxy galactose and quinac = 2-nacetyl- <dig>  6-dideoxy glucose . fucnac and qicnac are proposed according to the ei mass fragmentation pattern and relative retention times; however quantification could not be done due to unavailability of authentic standards for both monosaccharides\n",
            "\n",
            "\n",
            "\n",
            "fatty acid composition analysis of lps from intermediate and pathogenic strain\n",
            "the fatty acid composition of different lps fractions from l. interrogans serovar copenhageni strain fiocruz l1- <dig> and l. licerasiae serovar varillal strain var <dig>  are summarized in table  <dig> and their corresponding gc-ms spectra are shown in fig.  <dig>  the lpss consist of hydroxyl-lauric acid ; palmitic acid ; stearic acid ; oleic acid ; myristic  acid; hydroxyl-myristic acid , hydroxyl palmitic acid , and several other minor fatty acids such as myristoleic acid  and palmitoleic acid . hydroxylauric  was present in higher amounts in serovar copenhageni lps as compared to that of varillal, and hydroxypalmitate was only detected in serovar copenhageni.table  <dig> percentage of fatty acid composition analysis of lps of pathogenic versus intermediate pathogenic leptospira strains\n",
            "\n",
            "lps from leptospira interogans serovar copenhageni strain l1- <dig>  and intermediate pathogenic, leptospira licerasiae serovar varillal strain var <dig>  were purified from both aqueous  and phenol layer  and subjected for composition analysis separately. fatty acids in both aqueous  and phenol  layer lps extracts were measured and given in the table. average  and standard deviation  values are also presented in this table\n",
            "\n",
            "fig.  <dig> fatty acid composition analysis of lipopolysaccharide  from pathogenic and intermediately pathogenic leptospira species. lps was extracted from the pathogenic species leptospira interrogans serovar copenhageni strain fiocruz l1- <dig> and the intermediately pathogenic species leptospira licerasiae serovar varillal strain var <dig>  lps fractions were analyzed by gas chromatography–mass spectrometry  as fatty acid methyl ester  and trimethyl silyl ether-fame derivatives of hydroxyl fatty acids:  copenhageni aqueous layer lps;  var <dig> aqueous layer lps;  copenhageni phenol layer lps;  var <dig> phenol layer lps\n",
            "\n",
            "\n",
            "\n",
            "analysis of kdo\n",
            "kdo in aq and ph layer of l. interrogans serovar copenhageni strain fiocruz l1- <dig> was estimated to be  <dig>  and  <dig>  pmole per  <dig>  μg of lps, respectively . the kdo contents were strikingly low in l. licerasiae serovar varillal strain var <dig> in both phenol and aqueous layer lps samples .fig.  <dig> detection and quantitative estimation of 2-keto-3-deoxyoctonate  in pathogenic and intermediately pathogenic leptospira species. lps extracted from the pathogenic species leptospira interrogans serovar copenhageni strain fiocruz l1- <dig>  and the intermediately pathogenic species leptospira licerasiae serovar varillal strain var <dig> was subjected to kdo analysis using reverse phase hplc with a c <dig> column after tagging with a fluorophore  <dig>   <dig> methylenedioxy- <dig>  2-phenylenediamine dihydrochloride . a copenhageni aqueous layer lps;  var <dig> aqueous layer lps;  copenhageni phenol layer lps;  var <dig> phenol layer lps\n",
            "\n",
            "\n",
            "\n",
            "ultra-structure of leptospiral lps\n",
            "electron microscopic examination of negatively stained lps extracted from l. interrogans serovar copenhageni strain fiocruz l1- <dig> and l. licerasiae serovar varillal strain var <dig> had different morphologies . var <dig> lps is a closely packed sheet comprised of interwoven smaller thread-like filaments . by contrast, l1- <dig> lps consisted of filaments are larger in size, clustered together that did not form any definite sheet like structure .fig.  <dig> electron micrograph of lipopolysaccharide extracted from pathogenic and intermediate pathogenic leptospira strains. lps were spotted on a carbon-coated mesh, air-dried, negatively stained with 2 % uranyl acetate and examined under a transmission electron microscope. the pictures show the different lps structures of intermediately pathogenic  and pathogenic  strains, respectively\n",
            "\n",
            "\n",
            "\n",
            "discussion\n",
            "here we report key experimentally-determined biochemical details of the composition of the lipopolysaccharide  of l. licerasiae serovar varillal, an intermediate pathogen leptospire with unique antigenicity  <cit> , broad reactivity among people resident in the peruvian amazon  <cit> , and an unusually small rfb locus comprised of only  <dig> genes  <cit> . this analysis of l. licerasiae serovar varillal lps demonstrated important differences when compared to the lps of the highly pathogenic l. interrogans serovar copenhageni. we found that l. licerasiae varillal and l. interrogans copenhageni lps differ significantly in carbohydrate and fatty-acid compositions, particularly in that l. interrogans serovar copenhageni was found to contain fucose, n-acetylglucosamine and n-acetylgalactosamine, fucnac and quicnac and hydroxypalmitate  while l. licerasiae serovar varillal did not.\n",
            "\n",
            "while leptospiral lps does not have potent endotoxigenic properties  <cit> , it has long been known to be the antigen that defines serovar and serogroups, and anti-leptospiral lps antibodies are responsible for serovar-specific immunity . leptospira lps plays a major role in immunity  <cit> , potential candidate for vaccine development  <cit>  and also anti-lps monoclonal antibodies used for diagnosis  <cit> . in contrast, the other human-infecting spirochetes—borrelia spp. and treponema spp.—do not have lps or an outer membrane  <cit> , so that these biochemical components and structures do not play a role in the biology of these pathogens. transposon mutants, deficient in lps production, demonstrated that lps is a critical virulence determinant and is necessary for colonization of the vertebrate host  <cit> , similar to that of salmonella spp. lps  <cit> . however, detailed analysis of leptospiral lps remains understudied, and the present report provides a new approach to such studies.\n",
            "\n",
            "in our observations, purification of lps from leptospira species differed from other gram-negative organisms, in which lps precipitation primarily occurs from the aqueous phase of hot phenol-water extracts. in earlier studies, aqueous phase leptospira lps were used for composition analysis, immunological and biological studies . however, a substantial amount of leptospira lps also separates into the phenol layer during lps extraction procedures like gram-negative bacteria that contain smooth lps  <cit> . our results indicate that leptospira lps is distributed differently in phenol and aqueous layer in pathogenic  and intermediate pathogenic strain . the distribution of lps in organic and aqueous layers mainly depends on the ratio of hydrophilic saccharide portion and hydrophobic lipid part of the lps  <cit> . we analyzed lps isolated from both aqueous and phenol layers separately to compare chemical composition of serovars copenhageni and varillal. the growth temperature of an organism may also influence the lps composition and solubility. for example, in yersinia pestis the lps compositions changes under two different growth conditions  mimicking host body temperatures  <cit>  and similarly evident in other pathogens  <cit> . o-antigen content of l. interrogans has been observed to change between acute and chronic infection, and when cultivated under in vitro conditions  <cit> . interestingly, transmission of leptospirosis requires that leptospira survive and adapt to a wide variety of conditions  and characteristics of these adaptations are not clear  <cit> . we cultured both leptospira species  at  <dig> °c in a medium rich in long chain fatty acids , a standard method. a potential and important limitation of the present study is that only the lps of in vitro cultivated bacteria were studied. we obtained sufficient amount of lps from the in-vitro cultured leptospira strains  for chemical analysis, which is currently not feasible from in vivo growing leptospira.\n",
            "\n",
            "analysis of lps by sds-page followed by periodic acid-silver staining has been used to characterize lps heterogeneity in different leptospira serovars  <cit>  and other gram-negative bacteria  <cit> . the periodic-silver stain gel  shows that in the l1- <dig>  the fastest migrating lps component is probably the lipid-a core, and the slower migrating components are o-antigens as seen in other gram-negative organisms. in var <dig> lps are very compact  in both aqueous/phenol layers lps extraction. e. coli lps is characterized by a close succession of regularly spaced bands representing increasing number of o-antigen units. however, in l1- <dig> the o-side chain patterns may be irregular resulting in a smear-like pattern. the results make evident that the intermediate strain appears to possess the rough type of lps compared to pathogenic strain that has both smooth and rough form, and further suggests that the o-antigen carbohydrates in leptospira are not simple repeating units. this finding is particularly important as the o-antigen changes have been previously observed in the same pathogenic strain in acute versus chronic l. interrogans infection animal model  <cit>  and serial passaging in immuno-incompetent hosts in other organisms  <cit> . because of the current lack of a small animal model for l. licerasiae , the var <dig> strain was maintained by in-vitro culture, which possibly could have led to some sort of change in lps molecule. however, anti-var <dig> antisera continue to agglutinate the in vitro passaged strain suggesting that the essential antigen determinants are preserved. our recent data showed that we could infect rattus norvegicus with var <dig> to infect and establish chronic renal infection .\n",
            "\n",
            "we demonstrated that the sugar composition of lps of l1- <dig> and var <dig> differs, and that the lps purified from aqueous and phenol layers of these leptospira is different. interestingly, fucose, n-acetylglucosamine and n-acetylgalactosamine are only found in l1- <dig> and not detected in the var <dig> lps samples. fucnac and quicnac are only found in the aqueous layer lps from l1- <dig>  and both the sugars are reported to be present on the o-antigen repeat unit of semi-rough or smooth type lps in different pathogenic bacterial species such as vibrio cholera and pseudomonas  <cit> . earlier publications show that there are high content of rhamnose and arabinose in the lps of pathogenic leptospira species, l. interrogans serovar copenhageni and l.interrogans serovar hardjo  <cit> . however, we observed in the intermediate-pathogenic leptospira species, var <dig>  arabinose amount was two-fold higher compared to rhamnose. therefore, we hypothesize that arabinose could play a protective role in survival under certain environmental conditions. azospirillums showed that a high arabinose content in exopolysaccharide  plays an important role in cell aggregation that allows survival in a hostile environment  <cit> .\n",
            "\n",
            "in gram-negative organisms lipid a is responsible for the endotixic activity  <cit> . the lipid a of l. interrogans has been reported to be structurally different from other enterobacterial lipid a  <cit> , and to be recognized by the mouse innate immune system by tlr- <dig> and not tlr- <dig>  <cit> . the lipid a center is an essential component for the viability of most bacteria, with a few exceptions. for example, neisseria meningitidis can survive without lipid a in the presence of capsular polysaccharide  <cit> . the biological activity of lps is dependent on several structural features of the lipid a. among them, the presence of phosphate group, substitution of the phosphate residues , alteration of fatty acyl residues, fatty acid type and chain length  <cit> .\n",
            "\n",
            "the fatty acid moieties present in the lipid a consist of chain lengths between c10-c <dig> in majority of organisms and few organisms show c16-c <dig>  <cit> . for examples, p. aeruginosa expresses fatty acid of shorter chain length c <dig> and c12; h. pylori possess longer chain of fatty acids  and h. influenzae present c <dig> fatty acids  <cit> . in enterobacteriaceae lps, 3-oh-c14: <dig> is the dominant 3-oh-fatty acids  <cit> . the lps of e. coli contains fatty acids with chain length of c <dig> and 3-oh c14: <dig>  <cit> . we analyzed the fatty acid from e. coli 011:b <dig>  indicated the presence of 3-oh c14: <dig> as the major component. the lps of var <dig> shows the presence of 3-oh c14: <dig>  however it’s percentage is lot less in pathogenic strains. interestingly, 3-oh fatty acids c16: <dig> is only present in l1- <dig> and not detected in var <dig>  in gram-negative organisms, 3-hydroxy long-chain fatty acids are essential to the endotoxin activity and also used as a chemical markers of lps  <cit> , hence the role of 3-oh c16: <dig> in l1- <dig> need further evaluations. hydroxylauric acid present in both var <dig> and l1- <dig> in different ratio, these finding was reported previously for l1- <dig>  <cit> . the fatty acid composition differed between the pathogenic  and intermediate pathogenic leptospira  that we studied here; for example, fatty acid hydroxypalmitate  was unique to the pathogenic strain. the leptospira have a unique metabolism that requires a fatty acid-enriched medium  <cit> , and in the host infectious leptospira colonize kidney as a key component of their life cycle  <cit> . these organs offer a large amount of lipid supply to meet the essential fatty acids requirements for spirochetes growth and multiplication  <cit> . the fatty acid metabolic pathway of unique fatty acids of pathogenic leptospira lps can be a good target for the intervention to prevent disease progression. fatty acid composition has previously been suggested to be related to pathogenicity, and this feature merits for further investigation  <cit> .\n",
            "\n",
            "we carried out extensive processing to remove the nucleic acids and protein contaminant from the lps preparation by nucleases treatment, proteolytic digestion, ultracentrifugation and dialysis as per published method of leptospira lps purification  <cit> . however, it is possible that glycoproteins that are tightly complexed with lps and not amenable to protease digestion may contribute to the lipid content. we acknowledge this as a potential limitation of the lps preparation by available method. the percentages of fatty acids commonly found in both strains such as palmitic acid, stearic and oleic acid are contributions from some non-lps components such as glycolipoproteins that are co-extracted with the lps and could not be removed.\n",
            "\n",
            "kdo  is an eight carbon acidic sugar found in the lps of gram-negative bacteria and it connects the lipid a part to the o-antigen. the conflicting ability to detect kdo and detection of kdo in some pathogenic strains  <cit>   <cit>  makes it interesting to understand how sugars might play a substitute role in the absence or limited amount of kdo. we also found kdo in the phenol phase extract of both virulent and intermediate strain and the aqueous phase of the virulent strain only. the amount of kdo in the intermediate was very low with a small peak eluted at the same retention time corresponding to standard kdo and l1- <dig> kdo peak. it is possible that in the intermediately pathogenic strain other sugars or other form of kdo might also be playing a role in maintaining the weak linkage between o-antigenic carbohydrate and the lipid-a part of lps. for example, kdo is replaced by 2-keto-d-glycero-d-talo-octonic  in burkholderia cepacia, a gram-negative bacterial pathogens that causes respiratory infection in cystic fibrosis patients  <cit> .\n",
            "\n",
            "CONCLUSIONS\n",
            "this study is the first detailed biochemical composition analysis of an intermediately pathogenic leptospira lps, and the first to compare leptospiral lps in leptospira pathogen vs. intermediate species. many sugars and fatty acid were found in l. interrogans copenhageni l1- <dig> but not in the intermediately pathogenic l. licerasiae varillal var <dig>  which merits further investigation. current and forthcoming comparative whole leptospira genomic analysis will allow for detailed genetic/genomic, functional and structural analysis of leptospiral lps, a molecule essential to diverse aspects of this spirochete’s biology, which will improve our understand of mechanisms of pathogenesis and potential vaccine-related interventions.\n",
            "\n",
            "abbreviations\n",
            "lpslipopolysaccharides\n",
            "\n",
            "glcnacn-acetyl glucosamine\n",
            "\n",
            "hexnacn-acetylhexoseamine\n",
            "\n",
            "sds-pagesodium dodecyl polyacrylamide gel electrophoresis\n",
            "\n",
            "gc-msgas chromatography–mass spectrometry\n",
            "\n",
            "rfblps biosynthetic locus\n",
            "\n",
            "aaalacassociation for assessment and accreditation of laboratory animal care\n",
            "\n",
            "emjhellinghausen-mccullough-johnson-harris medium\n",
            "\n",
            "famefatty acid methyl ester \n",
            "\n",
            "-fametrimethyl silyl ether\n",
            "\n",
            "hoacacetic acid\n",
            "\n",
            "aq layeraqueous layer\n",
            "\n",
            "ph layerphenol layer\n",
            "\n",
            "araarabinose \n",
            "\n",
            "rharhamnose\n",
            "\n",
            "xylxylose\n",
            "\n",
            "manmannose\n",
            "\n",
            "galgalactose\n",
            "\n",
            "galnacn-acetyl galactosamine\n",
            "\n",
            "quinacn-acetyl quinovosamine\n",
            "\n",
            "fucnacn-acetyl fucosamine\n",
            "\n",
            "kdo2-keto-3-deoxymanno-octulosonic acid\n",
            "\n",
            "ko2-keto-d-glycero-d-talo-octonic\n",
            "\n",
            "tlrtoll-like receptor\n",
            "\n",
            "kailash p. patra and biswa choudhury contributed equally to this work.\n",
            "\n",
            "competing interests\n",
            "\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "authors’ contributions\n",
            "\n",
            "conceived and designed the experiments , performed experiments  and analyzed the data and wrote the paper . all authors read and approved the final manuscript.\n",
            "\n",
            "we thank paula maguina for her logistic support for this paper. we are thankful for the excellent support and facilities provided at glycotechnology core resources, ucsd to complete the work. we also thank timo meerloo, electron microscopy core facility, cellular and molecular medicine, ucsd for his contributions to this work.\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "our results suggest that the composition and structure of leptospira lps is different between pathogenic and intermediately pathogenic leptospira species, which can be further exploited by comparative analysis of key components to understand the pathogenesis and host colonization.\n",
            "our results suggest that the composition and structure of leptospira lps is different between pathogenic and intermediately pathogenic leptospira species, which can be further exploited by comparative analysis of key components to understand the pathogenesis and host colonization.\n",
            "this study is the first to compare the biochemical composition and structure of leptospira lps from pathogenic and intermediately pathogenic leptospira species, and the first to demonstrate that the composition and structure of leptospira lps is different between pathogenic and intermediately pathogenic leptospira species.\n",
            "our results suggest that the composition and structure of leptospira lps is different between pathogenic and intermediately pathogenic leptospira species, which can be further exploited by comparative analysis of key components to understand the pathogenesis and host colonization.\n",
            "\n",
            "57\n",
            "[{'summary_text': 'recently formed tragopogon allotetraploids exhibit various consequences of genome evolution and gene expression following polyploidization.\\nrecent studies have shown that polyploidy is a dynamic process, with abundant and rapid genomic changes occurring within a short time period following polyploidization.\\nrecently formed tragopogon allotetraploids exhibit various consequences of genome evolution and gene expression following polyploidization.\\nour results suggest that polyploidy is a dynamic process, with abundant and rapid genomic changes occurring within a short time period following polyploidization.\\nthe polyploid t. mirus is the most genetically variable of the three diploid t.\\nthe polyploid t. mirus is the most genetically variable of the three diploid t.\\nthe polyploid t. mirus is the most genetically variable of the three diploid t.\\nthe polyploid t. mirus is the most genetically variable of the three diploid t.\\nthe polyploid t. mirus is the most genetically variable of the three diploid t.\\nthe polyploid t. mirus is the most genetically diverse of the three diploid t.'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 14468 to 15360 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "polyploidy is a particularly important evolutionary mechanism in flowering plants  <cit> . during the past  <dig> years, many plant biologists have estimated the frequency of polyploidy in the angiosperms using analysis of base chromosome numbers  <cit> , as well as measurements of stomatal size in fossil and extant taxa  <cit> . based on these approaches, researchers estimated that from 40% to 70% of angiosperms have experienced polyploidy in their evolutionary history  <cit> . recent genomic studies indicate, however, that polyploidy is even more prevalent in angiosperm lineages than previously suspected. sequencing of the entire nuclear genome of arabidopsis thaliana indicated two or three rounds of genome-wide duplication  <cit> . complete genome sequences also indicate multiple ancient polyploidy events in populus trichocarpa and vitis vinifera  <cit> . genomic data  indicate ancient polyploidy for other angiosperms  <cit> , including the basal angiosperm nuphar advena, the magnoliids persea americana, liriodendron tulipifera, and saruma henryi, the basal monocot acorus americanus, and the basal eudicot eschscholzia californica  <cit> . it now appears that all angiosperms may have undergone at least one round of genome duplication .\n",
            "\n",
            "several outcomes for duplicated genes are possible at the genomic and transcriptional levels. first, both members of a duplicate gene pair may retain their original function. second, one copy of a duplicate gene pair may retain the original function, but the other copy may become lost or silenced  <cit> . third, duplicate genes may partition the original gene function , with one copy active, for example, in one tissue and the other copy active in another tissue  <cit> . fourth, one copy may retain the original function, while the other develops a new function   <cit> .\n",
            "\n",
            "recent studies have revealed varied consequences of genome evolution and gene expression following polyploidy in diverse angiosperms, including arabidopsis  <cit>  and crops such as cotton  <cit> , wheat  <cit> , and brassica  <cit> . several investigations have shown that following polyploidy, rapid genomic rearrangement  <cit> , gene loss  <cit> , or gene silencing via dna methylation  <cit>  may occur. however, few analyses have explored the genetic and genomic consequences of allopolyploidy in natural systems. six natural allopolyploids are known to have formed within the past  <dig> years, thus affording the opportunity to examine the nearly immediate consequences of polyploidization in nature: cardamine schulzii  <cit> , senecio cambrensis  <cit> , senecio eboracensis  <cit> , spartina anglica  <cit> , and tragopogon mirus and t. miscellus  <cit> . several studies of these recently formed allopolyploids show evidence of either genomic or expression-level changes, relative to their diploid parents. for example, salmon et al.  <cit>  showed that methylation patterns differ between the hexaploid parents , the independently formed hybrids , and the allopolyploid s. anglica . in senecio, hybridization of diploid s. squalidus with tetraploid s. vulgaris forms a sterile triploid, s. × baxteri, and subsequent genome duplication produced the allohexaploid s. cambrensis. through microarray analysis of floral gene expression patterns in synthetic s. cambrensis lines, hegarty et al.  <cit>  observed that the synthetic hybrid s. × baxteri showed immediate transcriptional changes compared to the parental expression patterns, and that this \"transcriptional shock\" was \"subsequently calmed\" in allohexaploid s. cambrensis, suggesting that hybridization and polyploidization have distinct effects on large-scale gene expression in this system.\n",
            "\n",
            "one of the best systems for the study of naturally occurring polyploids is provided by the genus tragopogon . tragopogon comprises ca.  <dig> to  <dig> species distributed throughout europe, temperate asia, and north africa  <cit> . three diploid species  were introduced from europe into the palouse region of eastern washington and adjacent idaho, usa, in the early 1900s  <cit> . the introduction of these three diploid species brought them into close contact, and as a result, two allotetraploid species  formed  <cit> . first collected in  <dig>  <cit> , these recently formed polyploids are less than  <dig> years old. morphological, cytological, flavonoid, isozymic, and dna evidence confirmed the ancestries of these two allotetraploids  <cit> . multiple lines of evidence suggest that t. miscellus has formed recurrently, possibly as many as  <dig> times, including reciprocal formation, and t. mirus has formed repeatedly perhaps  <dig> times   <cit> . therefore, t. mirus and t. miscellus afford unique opportunities for the investigation of recent and recurrent polyploid evolution. in fact, nearly every population of these species may have formed independently .\n",
            "\n",
            "tate et al.  <cit>  and buggs et al.  <cit>  studied genomic changes and expression differences of homeologs within natural populations of tragopogon miscellus, as well as in synthetic f <dig> hybrids and first-generation polyploids formed from the diploid parents t. dubius and t. pratensis. most of the genes analyzed show additivity in t. miscellus at both the genomic  and cdna levels . however, loss of one parental homeolog was observed at several loci , as were several examples of gene silencing . both homeolog losses and silencing patterns vary among individuals in natural polyploid populations of independent origin  <cit> . changes were also detected in rdna content  <cit>  and expression  <cit>  in populations of t. miscellus. although t. miscellus has fewer rdna repeats of t. dubius than of t. pratensis  <cit> , apparently due to concerted evolution, most of the rdna expression derives from the t. dubius repeats  <cit> . the same pattern of rdna expression has been observed in populations of t. mirus compared to its parents  <cit> ; t. mirus has fewer repeats of t. dubius than of t. porrifolius  <cit> , but most of the rrna is produced by the t. dubius copies  <cit> . although homeolog loss events and expression changes were observed in natural populations of t. miscellus, no such changes were observed in comparable analyses of f <dig> hybrids between the diploid parents, t. dubius and t. pratensis  <cit> , or in first-generation synthetic lines  <cit> .\n",
            "\n",
            "in this study we extend our examination of gene loss and differential expression to the polyploid t. mirus. in nature, t. mirus has formed repeatedly, but only when t. dubius is the paternal parent and t. porrifolius is the maternal parent  <cit> . however, t. mirus can be produced synthetically in both directions with about equal frequency  <cit> . tragopogon mirus provides an opportunity to compare expression differences at the genomic and transcriptional levels with the results obtained for t. miscellus  <cit> . our main objectives were to: 1) investigate the genomic changes and expression differences of parental homeologs in t. mirus relative to its diploid parents, 2) determine the identity of the genes that exhibit those changes, and 3) assess whether individuals within and among recurrently formed natural populations of t. mirus show similar patterns of genome evolution and gene expression.\n",
            "\n",
            "RESULTS\n",
            "cdna-aflp polymorphism and identification of putatively differentially expressed genes\n",
            "we used cdna-aflps  <cit> as a first step toward identifying genes with putative differential expression in the allotetraploid t. mirus, relative to its diploid parents . from our initial screen with  <dig> primer pairs,  <dig>  fragments were produced, and of these,  <dig> were monomorphic , and  <dig> were polymorphic  among the three species. novel cdna-aflp bands in the polyploid plants comprised  <dig> %  of all fragments, fragments in the polyploids of maternal origin constituted  <dig> %  of all fragments, while fragments having a paternal origin in the polyploids made up  <dig> %  of all fragments. from this initial screening, we selected for further study  <dig> of the  <dig> primer sets, which produced an average of  <dig> different fragments per primer pair. from the remaining  <dig> primer sets, we obtained an average of  <dig> different fragments, but these were too short  for further analysis. we then conducted an analysis on an expanded sample of the pullman- <dig> population and its progenitors  to obtain a larger set of potentially informative fragments. from the  <dig> primer pairs,  <dig>  fragments were produced, and of these,  <dig> were monomorphic , and  <dig> were polymorphic  . novel cdna-aflp fragments in the polyploids comprised  <dig> %  of all fragments. shared fragments with a maternal or paternal origin in the polyploids represented  <dig> %  or  <dig> %  of the total fragments, respectively.\n",
            "\n",
            "for the pullman- <dig> and palouse sites, we selected four primer sets  that showed high variation in populations from the pullman- <dig> site. at the pullman- <dig> site,  <dig> fragments were scored, and of these,  <dig> were monomorphic , and  <dig> were polymorphic  . novel cdna-aflp bands in the polyploids accounted for  <dig> %  of all fragments, and fragments of maternal or paternal origin in the polyploids made up  <dig> %  and  <dig> %  of all bands, respectively.\n",
            "\n",
            "at the palouse site,  <dig> fragments were scored, and of these,  <dig> were monomorphic , and  <dig> were polymorphic  . fragments with a maternal or paternal origin in the polyploids made up  <dig> %  and  <dig> %  of all fragments, respectively. no novel cdna-aflp bands were detected in polyploid plants from palouse. when we compared  <dig> individuals of t. mirus from the pullman- <dig>  pullman- <dig>  and palouse populations, we observed very similar patterns in the pullman- <dig> and pullman- <dig> populations. however, individuals of t. mirus from palouse have more complex patterns than individuals of t. mirus from pullman- <dig> and pullman- <dig>  individuals 2602- <dig> and 2602- <dig> from palouse shared an aflp pattern, whereas 2602- <dig> and 2602- <dig> showed a different pattern. there are at least three genotypes among the five individuals of t. mirus from palouse.\n",
            "\n",
            "with  <dig> variable fragments  identified from cdna-aflp analyses, we then searched for fragment identity based on sequence similarity using blast searches and identified  <dig> putative genes in t. mirus . further comparison with the arabidopsis genome indicated that these genes are involved in various cellular processes, such as carbohydrate metabolism, signal transduction, protein transport and degradation, and cell division . however, we could not reliably identify the remainder of the fragments because of their short length .\n",
            "\n",
            "rapid loss of parental homeologs\n",
            "the genes, enzymes, and sizes of digested genomic and cdna amplifications for caps analysis of t. mirus and its parents are listed in additional file  <dig> \n",
            "\n",
            "for the genomic caps analysis,  <dig> of  <dig> genes showed additivity, with both parental copies maintained in all allopolyploid individuals . nine of  <dig> genes showed that at least one allotetraploid individual was missing one parental homeolog . to determine whether these losses were due to true homeolog loss or simply loss of a restriction site , we sequenced the pcr fragments of all genes exhibiting putative losses. sequencing revealed that all individuals exhibiting apparent loss events have only one parental homeolog, confirming that these inferred homeolog losses are not due to restriction site divergence and loss of a caps marker.\n",
            "\n",
            "two genes exhibited homeolog losses of one parental copy or the other in at least one individual, whereas seven genes showed loss of only the t. dubius homeolog. for the putative homolog of thioredoxin m-type  <dig>  one pullman- <dig> t. mirus plant  showed loss of the t. porrifolius band, while one plant of palouse t. mirus  exhibited loss of the t. dubius band . for the putative nucleic acid binding homolog, eight t. mirus plants from the pullman- <dig> , oakesdale , and arizona  populations showed loss of the t. porrifolius band, while one palouse t. mirus individual  lost the t. dubius band. in contrast, preferential loss of the t. dubius parental homeolog was observed in several individuals for seven genes . in addition, variation was observed among populations; losses of the t. dubius homeolog occurred at more loci in the pullman- <dig>  pullman- <dig>  and palouse populations than in the oakesdale, rosalia, tekoa, and arizona populations. for example, individuals from the palouse population showed loss of the t. dubius homeolog for four genes, while individuals from the oakesdale population exhibited gene loss for only one gene . therefore, the pullman- <dig>  pullman- <dig>  and palouse populations of t. mirus show higher levels of, and greater variation in, homeolog loss than do populations from oakesdale, rosalia, tekoa, and arizona. however, one putative gene  was polymorphic in both t. mirus and t. dubius . most individuals of t. dubius have a single copy of this gene, but five t. dubius individuals have an extra copy that corresponds to the pcr amplicon produced in t. porrifolius. also, six t. dubius individuals only have the \"t. porrifolius\" type . this polymorphism observed in t. dubius can affect interpretation of the expression patterns of t. mirus, making it hard to distinguish loss from polymorphism. as a result of this polymorphism, this gene was not employed in our analyses of loss events.\n",
            "\n",
            "cdna caps analysis was performed for  <dig> of  <dig> genes. the remaining  <dig> genes analyzed above for genomics caps could not be amplified . eleven of the genes included in the cdna analyses showed additivity, whereas four genes  showed expression differences in some polyploid individuals relative to the diploid parents. however, the apparent expression differences from the thioredoxin m-type  <dig> and myosin heavy chain class xi result from genomic losses . for the other two genes, true expression differences were detected. for the putative homolog of nuclear ribosomal dna, cdna caps showed absence of the t. dubius homeolog in one individual of t. mirus  from palouse, while genomic caps found additive patterns in all tetraploid individuals . also, a putative homolog of glyceraldehyde-3-phosphate dehydrogenase showed silencing in t. mirus in six individuals from pullman- <dig> , three individuals from pullman- <dig> , one individual from palouse , and one individual from oakesdale  .\n",
            "\n",
            "in summary,  <dig> of  <dig> individuals sampled of t. mirus showed loss of at least one homeolog, and  <dig> individuals exhibited true loss of expression of one parental homeolog .\n",
            "\n",
            "diploid f <dig> hybrids are additive of their parental genomes\n",
            "genomic caps analyses for six synthetic f <dig> hybrids from two independent crosses between t. dubius and t. porrifolius were also performed for the same  <dig> genes surveyed in natural populations of t. mirus. significantly, all genomic caps analyses of f <dig> hybrids exhibited additivity of the parental homeologs. cdna caps analysis for all  <dig> genes investigated in natural populations of t. mirus showed that both parental homeologs were expressed in all f <dig> hybrids examined .\n",
            "\n",
            "discussion\n",
            "cdna-aflp variation in populations of t. mirus\n",
            "as cdna-aflps reveal potentially differentially expressed genes, the results can provide useful initial information on the genetics of polyploids, especially those that lack developed genomic resources, such as tragopogon. thus, cdna-aflps provide numerous candidate genes relatively quickly and inexpensively. however, cdna-aflp analysis must be followed by other approaches, such as caps analysis, because cdna-aflp fragment differences may result from true expression differences, sequence polymorphism, or gene or homeolog loss  <cit> .\n",
            "\n",
            "from cdna-aflps, we identified  <dig> putative genes that were not additive of the parental bands in t. mirus; most of these  exhibited maternal banding patterns . however, subsequent analysis of  <dig> of these genes using genomic and cdna caps analyses showed that only four of these genes exhibited an expression difference; two genes, thioredoxin m-type  <dig> and myosin heavy chain class xi, showed homeolog losses, while nuclear ribosomal dna and glyceraldehyde-3-phosphate dehydrogenase exhibited true silencing . however, most of the genes exhibiting apparent maternal, paternal, or novel banding patterns in cdna-aflps actually showed additive patterns in genomic and cdna caps analyses, indicating that the cdna-aflp fragment differences observed in t. mirus may be derived from sequence polymorphism and are not indicative of homeolog loss or silencing.\n",
            "\n",
            "rapid genomic changes in t. mirus\n",
            "analyses of caps markers provide evidence of rapid, frequent, and preferential elimination of homeologous loci and changes in gene expression in allotetraploid individuals of t. mirus. most of the genes examined showed genomic caps patterns that are additive of the parental genes, but nine genes of the  <dig> examined showed homeolog loss, fewer than observed for t. miscellus, in which  <dig> loci of the  <dig> examined showed homeolog loss  <cit> .\n",
            "\n",
            "in t. mirus, the t. dubius parental homeolog has been lost at more loci  than the t. porrifolius homeolog . likewise, in t. miscellus, t. dubius parental homeolog has been lost at more loci  than the t. pratensis homeolog   <cit> . however, the t. porrifolius homeolog has actually been lost from more individuals of t. mirus  than has the t. dubius homeolog , in contrast to data for t. miscellus  <cit> . this difference between t. mirus and t. miscellus results from the extensive loss of the t. porrifolius homeolog for nadp/fad oxidoreductase. furthermore, each homeolog absence in a polyploid should not necessary be viewed as a unique loss, as a single loss may subsequently be transmitted throughout a population.\n",
            "\n",
            "the preferential loss of t. dubius homeologs also agrees with the biased rdna homogenization of t. dubius repeats to the other diploid parental type in both t. miscellus and t. mirus  <cit> . that is, the number of t. dubius rdna units has typically been greatly reduced in the genomes of the allotetraploids t. mirus and t. miscellus relative to the other diploid parent  due to apparent concerted evolution  <cit> . the single exception is the palouse population of t. mirus, in which individuals have a relatively high number of t. dubius rdna repeats relative to t. porrifolius repeats   <cit> , illustrating variation in rdna repeat composition among populations of t. mirus of independent origin. in contrast, allopolyploids in arabidopsis and brassica apparently have not undergone loss or concerted evolution of rdna units  <cit> . however, rapid concerted evolution of rdna units in just a few generations  <cit>  has occurred in some synthetic hybrids and allotetraploids, including synthetic hybrids between maize and tripsacum  <cit> , somatic hybrids of medicago sativa  <cit> , synthetic nicotiana allotetraploids  <cit> , and synthetic allotetraploid arabidopsis suecica  <cit> .\n",
            "\n",
            "despite being fewer in number, the rdna units of t. dubius origin dominate rdna transcription in most populations of t. mirus  <cit> . rdna gene reduction by concerted evolution in allopolyploids can therefore be countered by high levels of expression controlled by epigenetic regulation. we obtained similar results for rdna expression here in our cdna caps study. based on visual comparison of banding intensity, there is higher rdna expression of t. dubius-origin units than of t. porrifolius-origin units in all plants from all populations examined, except for individuals from the palouse population . interestingly, for the palouse population, cdna and genomic caps indicate the silencing of the t. dubius rdna unit in one plant of t. mirus. hence, these results further highlight the importance of populational surveys, by indicating some stochasticity for rdna expression in the young polyploid t. mirus.\n",
            "\n",
            "some synthetic and natural allopolyploids show remarkable genomic restructuring . for example, synthetic brassica napus allopolyploids exhibit many chromosomal translocations and transposition events during the s <dig> to s <dig> generations, based on rflp analysis of synthetic lines  <cit> . genome evolution in the natural polyploids tragopogon mirus and t. miscellus appears most similar to the results obtained for these synthetic brassica allopolyploids. homeolog loss appears frequent in both systems. recent cytogenetic studies using fish and gish indicate that both t. mirus and t. miscellus show evidence of rapid genomic rearrangement, including translocations and inversions  <cit> . genetic changes observed in synthetic brassica napus as well as natural populations of tragopogon mirus and t. miscullus may be related, in part, to chromosomal rearrangements in these polyploids  <cit> .\n",
            "\n",
            "plants of the synthetic and naturally occurring allopolyploid arabidopsis suecica also exhibit chromosomal rearrangement  <cit> , as well as many changes in gene expression  <cit> . genes from one parent, a. thaliana, have often been silenced epigenetically by dna methylation  <cit> . through microarray analysis, wang et al.  <cit>  showed that approximately 65% of nonadditively expressed genes in the synthetic allotetraploids were repressed, and more than 94% of them matched the genes that are highly expressed in one parent, a. thaliana. tragopogon allopolyploids have undergone many losses of homeologous loci, often eliminating copies of one parent, t. dubius, but fewer instances of gene silencing. additional studies of tragopogon are needed to similarly examine gene expression on a genomic-level scale.\n",
            "\n",
            "f <dig> diploid hybrids and early synthetic allotetraploids  between aegilops sharonensis and triticum monococcum ssp. aegilopoides showed both gene loss and silencing by dna methylation  <cit> . however, such immediate changes have not been detected in tragopogon. genomic and cdna caps data for synthetic f <dig> hybrids between t. dubius and t. porrifolius showed additivity rather than gene loss or silencing . similarly, no homeolog loss was observed in synthetic f <dig> hybrids between t. dubius and t. pratensis  <cit> , or in newly produced  or first-generation  synthetic t. miscellus  <cit> . therefore, in contrast to wheat, both t. mirus and t. miscellus exhibit genome evolution, not immediately following hybridization or allopolyploidization, but apparently shortly thereafter, given that the species are probably less than  <dig> years old   <cit> . also, these results suggest that loss of homeologs and gene expression changes, while still rapid in evolutionary time, may be slightly more gradual in tragopogon, occurring over several generations, but further studies are required to assess the speed and magnitude with which genomic changes have occurred in tragopogon.\n",
            "\n",
            "a major question centers on the mechanisms responsible for the loss of homeologs in these young allopolyploids. ownbey  <cit>  observed the formation of complex multivalents during meiosis in both t. mirus and t. miscellus shortly after their formation and in f <dig> hybrids between t. dubius and t. porrifolius, and between t. dubius and t. pratensis. furthermore, multivalents have also been observed in synthetic t. mirus and t. miscellus  <cit> . in addition, rare patterns observed in analysis of allozyme variation in tragopogon are consistent with non-homologous recombination  <cit> . non-homologous recombination could provide a mechanism of homeolog loss in t. mirus and t. miscellus, as in brassica  <cit> . recent cytogenetic data provide additional insights into potential mechanisms for gene loss in tragopogon. gish studies have revealed that chromosomal rearrangements and other changes may be common in natural populations of t. mirus and t. miscellus  <cit> . intergenomic translocations, inversions, as well as apparent monosomy and reciprocal trisomy occur in fertile individuals of both polyploids  <cit> . such rearrangements provide a potential mechanism for the homeolog losses observed in both t. mirus and t. miscellus.\n",
            "\n",
            "genomic changes versus differential expression in t. mirus\n",
            "for most of the genes examined here, homeolog losses appear to be responsible for the cdna-aflp fragment differences observed in individuals of t. mirus relative to its diploid progenitors. however, in two genes , we found true expression differences in the allopolyploid relative to its parents. nuclear ribosomal dna encodes ribosomal rna, and glyceraldehyde-3-phosphate dehydrogenase encodes an enzyme that participates in multiple processes, including transcription activation, initiation of apoptosis, and er to golgi vesicle shuttling  <cit> , so both of these genes are crucial for cell function. the pattern of nuclear ribosomal dna from genomic and cdna caps analyses is consistent with matyášek et al.'s  <cit>  study: all individuals from the palouse population showed additivity with genomic caps and in the southern blot rdna study of matyášek et al.  <cit> . however, in one individual from the palouse population, the t. dubius nuclear ribosomal dna homeolog was completely silenced in both our cdna caps analysis and in the rdna transcript study of matyášek et al.  <cit> .\n",
            "\n",
            "most of the t. mirus individuals examined here, except those from tekoa and arizona, show no expression of the t. dubius homeolog for glyceraldehyde-3-phosphate dehydrogenase . genomic caps data indicate homeolog loss in two individuals, but additivity of parental homeologs in the remaining individuals. cdna caps analyses therefore exhibit gene silencing in  <dig> individuals . recent studies have shown that rapid epigenetic gene silencing following allopolypoid formation can be reversed by chemical demethylation in allopolyploid arabidopsis suecica  <cit> . therefore, silencing of these two genes might result from epigenetic phenomena such as dna methylation or histone acetylation  <cit> .\n",
            "\n",
            "in this study, gene silencing of t. dubius homeologs occurred in only two cases. in addition, expression studies of t. miscellus  <cit>  showed that seven out of  <dig> genes exhibited silencing of the t. dubius homeolog, while for two other genes, the t. pratensis homeolog was silenced. these biased patterns in t. mirus and t. miscellus indicate that t. dubius homeologs might be more susceptible to silencing than the alternative parental homeologs.\n",
            "\n",
            "when we compare gene silencing with gene loss with respect to the number of individuals examined, the previous studies of t. miscellus  <cit>  show that silencing events are slightly more frequent than homeolog loss, while in this study of t. mirus, homeolog losses are slightly more frequent than silencing events. these expression patterns result from biased expression of only a few genes. for example,  <dig> out of  <dig> t. miscellus individuals have silencing events in leucine-rich repeat transmembrane protein kinase, and  <dig> out of  <dig> t. mirus individuals have silencing events in glyceraldehyde-3-phosphate dehydrogenase. therefore, comparing gene loss with silencing events in t. mirus and t. miscellus seems to be affected by specific genes, with a few of the genes examined here especially prone to silencing.\n",
            "\n",
            "however, when we consider the number of genes examined in this study of t. mirus and in the previous t. miscellus studies  <cit> , homeolog losses in t. mirus  and in t. miscellus  are more frequent than silencing events . nevertheless, we investigated only a small portion of the genome, so further studies are required to assess whether the results for the ~ <dig> genes so far examined are representative of the entire genome.\n",
            "\n",
            "genomic caps analysis of a putative adenine dna glycosylase gene showed polymorphism in the populations of the diploid parent t. dubius surveyed here . although previous studies in tragopogon diploids using allozymes and other markers indicated that genetic variation within populations is quite low  <cit> , t. dubius is the most genetically variable of the three diploids; allozyme variation within populations of t. porrifolius and t. pratensis was limited or absent. therefore, polymorphism among t. dubius individuals for one of the genes analyzed here is not surprising. a recent survey of tragopogon diploids and polyploids from the palouse using microsatellite markers similarly revealed low levels of genetic variation within populations of t. dubius, but none within either t. pratensis or t. porrifolius .\n",
            "\n",
            "CONCLUSIONS\n",
            "recently formed tragopogon allotetraploids  exhibit various consequences of genome evolution and gene expression following polyploidy. in this study, using cdna-aflps, we found differential banding patterns, possibly attributable to gene silencing, novel expression, and/or maternal/paternal effects between t. mirus and its diploid parents. most of the banding patterns subsequently investigated with genomic and cdna caps analyses revealed additivity. most of the differences observed in t. mirus result from homeolog loss, rather than gene silencing; the latter was detected only infrequently . genomic and cdna caps analyses indicated that plants of t. mirus have experienced frequent and preferential elimination of the t. dubius homeolog, whereas comparable analyses of synthetic f <dig> hybrids between the parents  of t. mirus showed only additivity.\n",
            "\n",
            "these same results were also obtained for the recently and repeatedly formed allotetraploid tragopogon miscellus  <cit> . both t. mirus and t. miscellus undergo biased loss of homeologs contributed by their shared diploid parent, t. dubius. furthermore, both allotetraploids exhibit more homeolog losses than gene silencing in terms of the number of genes undergoing change. taken together, our results suggest that in tragopogon loss of homeologs and gene silencing are not immediate consequences of hybridization or polyploidization, but are processes that occur following polyploidization, occurring over a relatively small number of generations. these results further support the idea of polyploidy as a dynamic evolutionary process , with abundant and rapid genomic changes occurring within a short time period following polyploidization. further studies of homeolog loss, nonadditive expression patterns, and subfunctionalization of homeologs are needed to explore the roles of genetic and epigenetic phenomena in the evolution of allotetraploid tragopogon species.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "recently formed tragopogon allotetraploids exhibit various consequences of genome evolution and gene expression following polyploidization.\n",
            "recent studies have shown that polyploidy is a dynamic process, with abundant and rapid genomic changes occurring within a short time period following polyploidization.\n",
            "recently formed tragopogon allotetraploids exhibit various consequences of genome evolution and gene expression following polyploidization.\n",
            "our results suggest that polyploidy is a dynamic process, with abundant and rapid genomic changes occurring within a short time period following polyploidization.\n",
            "the polyploid t. mirus is the most genetically variable of the three diploid t.\n",
            "the polyploid t. mirus is the most genetically variable of the three diploid t.\n",
            "the polyploid t. mirus is the most genetically variable of the three diploid t.\n",
            "the polyploid t. mirus is the most genetically variable of the three diploid t.\n",
            "the polyploid t. mirus is the most genetically variable of the three diploid t.\n",
            "the polyploid t. mirus is the most genetically diverse of the three diploid t.\n",
            "58\n",
            "[{'summary_text': 'our results suggest that dact genes may have evolved in vertebrates prior to the split of the dact family, and that dact genes may have evolved in vertebrates prior to the split of the dact family.\\nour results suggest that dact genes may have evolved in vertebrates prior to the split of the dact family, and that dact genes may have evolved in vertebrates prior to the split of the dact family.\\nour results suggest that dact genes may have evolved in vertebrates prior to the split of the dact family, and that dact genes may have evolved in vertebrates prior to the split of the dact family.\\nour results suggest that dact genes were co-expressed in vertebrates prior to the split of the dact family, and that dact genes may have evolved in vertebrates prior to the split of the dact family.\\nour results suggest that dact genes were co-expressed in vertebrates prior to the split of the dact family, and that dact genes may have evolved in vertebrates prior to the split of the dact family.\\nour results suggest that dact genes were co-expressed in vertebrates prior to the split of the dact family.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 1550 to 2048 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "wingless and transforming growth factor beta  signaling are two cell-cell signaling systems that are well conserved throughout the animal kingdom and that control a plethora of processes ranging from embryonic development, cell proliferation, differentiation and migration, to tissue homeostasis, stem cell behavior, tissue regeneration and cancer  <cit> . dact  proteins have been identified in mammals, chicken, frog and zebrafish as intracellular multi-adapter molecules with the ability to modulate and possibly integrate the wnt and tgfβ signaling cascades. this ability primarily relies on the physical interaction of dact proteins with dvl , ckiδ/ϵ, vangl, pka, pkc, which are players in the various wnt pathways, or with the alk4/ <dig> tgfβ receptors  <cit> . in line with these properties, dact proteins positively as well as negatively regulate the wnt/β-catenin pathway and positively regulate the wnt/pcp pathway . in addition, specifically dact <dig> has been implicated in the suppression of tgfβ-dependent wound healing and nodal-dependent mesoderm induction due to its ability to facilitate lysosomal degradation of alk <dig>  <cit> . in addition to these established roles, dact proteins have been shown to stabilize p <dig> catenin  which in turn sequesters the transcriptional repressor kaiso, thus leading to the activation of kaiso targets  <cit> . since the p120-dact interaction is stimulated by wnt and is mediated by dvl, and because many kaiso targets are also tcf/lef targets, the p <dig> catenin/kaiso pathway is seen as a parallel pathway to the wnt/β-catenin pathway. dact proteins have been shown to also modulate wnt signaling mediators in a ligand-independent fashion: dact proteins shuttle between the nucleus and cytoplasm, and can block nuclear β-catenin function by disrupting β-catenin/lef <dig> complexes and enhancing lef1-hdac interaction  <cit> . however, they can also promote tcf/lef function when the dact n-terminal domain interacts with these transcription factors  <cit> . in addition, dact proteins can interact with dbf <dig> which, independent from its role in cell cycle regulation, inhibits β-catenin targets  <cit> . finally, dact function has been shown to depend on its phosphorylation state which is controlled in two ways: firstly, in the absence of wnt, dact is unphosphorylated, binds to dvl and blocks its ability to protect β-catenin from phosphorylation, thus promoting β-catenin degradation. in the presence of wnt, ckiδ/ϵ not only phosphorylates dvl but also dact; this decreases their affinity and promotes the resolution of β-catenin destruction complex, thereby stabilizing β-catenin. it also allows dact to promote the function of tcf/lef molecules, thus further enhancing the wnt response  <cit> . secondly, cyclic amp activated pka phosphorylates dact; this allows the binding of 14-3-3β which also blocks the ability of dact to promote dvl degradation, thus enhancing wnt signal transduction  <cit> . taken together, dact proteins have emerged as nodal points in the simultaneous control of the various wnt and tgfβ signaling pathways.\n",
            "\n",
            "dact are modular proteins, using different structural domains to interact with their specific partners. the functions of some of these domains have already been characterized. a leucine zipper located in the n-terminal half of the protein is required for homo-and hetero-dimerization, a c-terminal pdz binding domain together with a domain located in the center of the protein is crucial for dvl binding, a serine-rich domain upstream of the pdz binding domain is required for the interaction with vangl <dig>  the sequences encoded by the first three and the start of the fourth exon are sufficient to inhibit alk <dig>  a region encoded by the end of the 3rd and start of the 4th exon has been implicated in tcf <dig> binding and a not well characterized central portion of the protein interacts with p <dig> catenin  <cit> . furthermore, nuclear export and import signals have been identified  <cit> . however, in vitro binding studies showed that binding affinity and specificity of dact proteins with their various partners is variable, with mouse dact <dig> being the only dact showing significant affinity to tcf/lef and alk <dig> and, in comparison to dact <dig> and dact <dig>  weak binding to vangl <dig>  <cit> . knock out studies in the mouse implicated dact <dig> in wnt/pcp and dact <dig> in tgfβ signaling, yet morpholino knockdown experiments in zebrafish implicated dact <dig> in wnt/β-catenin and dact <dig> in wnt/pcp signaling  <cit> . this indicates that the structure-function relationship of dact proteins is still unclear.\n",
            "\n",
            "a key factor in our limited understanding of dact function is the fact that the full complement of dact genes available in different animals to regulate wnt and tgfβ signaling is not known, and therefore, dact functions may have been overlooked or misinterpreted due to gene redundancy. moreover, dact genes have so far only been found in bony vertebrates. however, bony vertebrates together with cartilaginous vertebrates belong to the infraphylum of jawed vertebrates, and in the ancestors of this animal group the genome has been duplicated twice, followed by subsequent gene loss or gene diversification  <cit> . thus, the origin of dact genes and their evolutionarily basic function is not known. to unravel the original and derived roles of dact genes and proteins, we took an evolutionary approach. we searched for so far elusive dact family members in the animal kingdom, and, using bioinformatic tools, we determined their phylogeny. moreover, we searched for conserved amino acid stretches that may serve as functional domains. finally, we determined the expression of dact genes in the zebrafish, the organism with the highest number of dact genes, in comparison with that of the chicken, which has only two.\n",
            "\n",
            "our study shows that dact genes are unique to chordates. in jawed vertebrates, four distinct dact paralogs were identified, with dact <dig> and dact <dig> originating from one, dact <dig> and dact <dig> from the second dact gene that was present after 1r. remarkably, all four genes are still present in latimeria , turtles  as well as lizards and snakes , but mammals, birds and amphibians have independently lost particular dact genes. in most teleosts, a dact <dig>  dact <dig>  two dact <dig> and one dact <dig> gene have been kept; zebrafish and the spotted gar, a holost fish, have an additional, intronless and hence possibly retrotranscribed dact4r. motif comparison suggests that the ability to dimerize, shuttle between cytoplasm and nucleus, bind tcf/lef and vangl molecules and to interact with various kinases may have been already present in the ancestral dact protein. the ability to interact with alk <dig> may have evolved with dact <dig> and  <dig>  motif combinations in extant dact <dig> proteins suggest that these molecules may sequester dact binding partners, thereby inhibiting their function. significantly, the various dact genes show similar expression patterns, suggesting that in a given tissue, the regulation of wnt and tgfβ signaling will depend on the combinatorial action of dact proteins.\n",
            "\n",
            "RESULTS\n",
            "searches for dact genes in the animal kingdom\n",
            "identification of new members of the gnathostomedact gene family\n",
            "currently, three dact family members are known in mouse and humans, two dact genes have been identified in the chicken, one in xenopus  and two in the zebrafish  <cit> . in order to obtain a comprehensive overview of dact genes in jawed vertebrates , we searched the genomes of various lobe-finned/lobe-limbed  and ray-finned  bony vertebrates. in our search we also included the genomic database for the elephant shark, a cartilaginous  vertebrate. to perform these searches, we interrogated the ensembl and ncbi databases using the known human, mouse, chicken, xenopus laevis and zebrafish dact protein sequences as queries. moreover, we performed searches with protein sequences encoded by individual exons or we used known dact protein motifs. since some of the selected genomes are not fully characterized, we also used the query sequences to interrogate the ncbi expressed sequence tags  database for the aforementioned groups, for additional bony vertebrates and for the spiny dogfish shark, pacific electric ray and little skate . the organisms searched in this study are listed in additional file 1; the accession numbers of sequences are provided in additional file  <dig>  the results of our searches are shown in figure  <dig> \n",
            "\n",
            "the searches revealed that like mouse and humans, all mammals carried three dact genes and all birds had two. in amphibians, we discovered a previously not recognized dact gene, increasing the complement of dact genes in these animals to two as well. remarkably, four distinct dact genes were found in lizards and snakes, in turtles and in the coelacanth, while five dact genes were present in the gar as well as in the tilapia, medaka and the atlantic cod, six in zebrafish, four in the stickleback and in pufferfish. these newly discovered genes indicate that the gnathostome dact gene family is larger than previously anticipated. in order to ensure that all gnathostome dact family members were traced, we repeated the searches, using the newly discovered sequences as queries. these searches, however, did not produce any further hits and confirmed the earlier results.\n",
            "\n",
            "based on similarities in sequence and organization, the dact genes identified in sarcopterygians and actinopterygians fell into four paralog groups. matching sequences for all four paralog groups were found in chondrichthyans, indicating that four dacts genes were present in the ancestral gnathostome genome. the first group encompassed known dact <dig> sequences and their newly identified relatives. dact1-type proteins consisted of 800-850aa with  <dig> % overall sequence identity; they were encoded by three small and a 4th, large exon. sequences of this type were found in all gnathostomes with the exception of pufferfish. in all species, only a single dact <dig> gene was present . a second set of sequences was 750-850aa long with overall  <dig> % sequence identity and encompassed known and novel dact <dig> proteins. the dact <dig> genes showed the same intron-exon structure as dact <dig> genes, however the third exon was almost twice as long as the 3rd exon in dact <dig>  dact2-type sequences were found in all gnathostomes with the exception of amphibians. similar to dact <dig>  only a single dact2-type gene was found in a given species. the third set of sequence encompassed both previously and newly identified dact <dig> proteins which were present in all gnathostomes with the exception of birds. in teleosts, two distinct sets of dact <dig> genes were found, designated dact3a and dact3b; a possible exception is the stickleback where due to gaps in the genomic sequence and absence of dact3b ests, the presence of this gene could not be ascertained. the dact <dig> proteins showed significant length variations, ranging from  <dig> , 540– <dig> , 610– <dig>  to 820aa . given that the dact family was thought to consist of three members only  <cit> , we were surprised to find a fourth, distinct set of sequences. dact <dig> proteins encompassed some  <dig> ,  <dig> ,  <dig>  or up to 1070-1120aa . like most dacts, dact <dig> proteins were encoded by genes containing four exons. the exception was a second gar and zebrafish dact <dig> protein which stems from an intronless gene that possibly was retrotranscribed and hence was named dact4r. remarkably, dact4genes were present in chondrychthians, in actinopterygians and in the following sarcopterygians: latimeria, anapsid and diapsid reptiles. this suggests that the dact <dig> gene belongs to the original gnathostome dact repertoire and persisted well beyond the actinopterygian-sarcopterygian split, the coelacanth-tetrapod split, the amphibian-amniote split and the segregation of the amniote lineages, but was lost independently in the avian, mammalian and amphibian lineages. since both the gar and the zebrafish have dact4r genes, this suggests that the gene occurred before the teleost-specific, third genome duplication   <cit> , but in most teleosts it was eliminated together with the duplicate of the genuine dact <dig> gene.\n",
            "\n",
            "identification of cyclostome dact genes\n",
            "given that we found dact genes well-represented in all gnathostome lineages, we wondered whether cyclostomes that split from gnathostomes some  <dig> million years ago  <cit>  might also carry these genes. we therefore searched the ensembl and ncbi databases for dact family members in the two cyclostome genomes available . as queries, we used full-length, exon-specific or motif-specific sequences from all four gnathostome dact proteins. the search revealed several contigs with dact-like sequences in the lethenteron genome and also in the petmar <dig> version of the sea lamprey genome. when the current version of the sea lamprey genome  was released, however, all except the sequences previously located on contig  <dig>  now gl <dig>  had been removed. yet several of the original petromyzon contigs encoded conserved dact motifs in the correct order, they had highly similar counterparts in the lethenteron genome, and some sequences were also represented in ests. we therefore considered these sequences as trustworthy. the results are included in figure  <dig> and additional file  <dig> \n",
            "\n",
            "the analysis of the lamprey genome and est sequences indicated the existence of at least four dact-related genes in cyclostomes . for two of these genes, sequences corresponding to all four dact gene exons were located on single contigs in lethenteron . partial matches for both genes were found in the petromyzon genome . for dactc, only exons 2– <dig> were identified on contig ke <dig>  sequences with high similarity to exon <dig> of dactc were found on two more lethenteron contigs , but not in the petromyzon genome. contig ke <dig> of lethenteron contained exon <dig> of the dactd gene, also represented in petmar <dig> c <dig>  in addition, identical, likely exon <dig> sequences were found on contigs apjl <dig> and apjl <dig>  since these sequences were not contiguous with the dactc or dactd sequences, they could not be unambiguously assigned to either gene.\n",
            "\n",
            "while the four cyclostome dact genes displayed similarity with the other vertebrate dacts, they could not be clearly allocated to any of the gnathostome dact paralog groups.\n",
            "\n",
            "identification of invertebrate dact genes\n",
            "to trace the so far elusive origin of dact genes, we next searched the ensembl and ncbi genome and est collections for oikopleura dioica, ciona intestinalis, ciona savignyi , branchiostoma floridae , saccoglossus kowalevskii  and strongylocentrotus purpuratus . these are all deuterostome animals. in addition, we searched the sequences available for the following protostomes: aplysia californica, a mollusc representing lophotrochozoans; drosophila melanogaster, tribolium castaneum, bombyx mori  and c. elegans, c. briggsae and loa loa . finally, we interrogated the ncbi protist and fungi genomes. the searches were performed as before, using full length or exon-specific dact protein sequences or protein motifs as queries.\n",
            "\n",
            "our results revealed that the only invertebrate harboring dact sequences was the cephalochordate branchiostoma floridae, the florida lancelet . here, the blast hits matched with exons 8– <dig> of a predicted 10-exon cdna on a single scaffold . exons 1– <dig> were confirmed by ests, encoding however the lancelet homologue of the rpa <dig> gene. exons 8– <dig> were confirmed by two further sets of ests. the first set encompassed exon <dig>   <dig> and start to mid-exon <dig>  the second set carried middle and end of exon <dig>  yet there are no ests to suggest that exons 1– <dig> are linked in a transcript. moreover, as will be shown below, exons 8– <dig> carry the complete sequence for a dact gene. we therefore renamed the exons that belong to branchiostoma dact exons <dig> , <dig>  exon <dig> encoded 73aa with loose homology to exon <dig> derived sequences in vertebrate dacts. exon <dig> accounted for 58aa that aligned well with exon2-derived sequences of gnathostome dact1- <dig>  including a 5x leucine zipper. different to vertebrates, however, the branchiostoma exon2- <dig> boundary encoded an extended serine-rich stretch. exon <dig> encoded in total 872aa that encompassed a number of the conserved sequence motifs which in vertebrates are encoded by the 3’ end of exon <dig>  and by exons  <dig> and  <dig>  taken together, we traced the origin of dacts back to chordates, where many motifs and functional domains were established already.\n",
            "\n",
            "phylogenetic analysis of dact protein sequences\n",
            "the initial sequence analysis of the known and the newly identified dact sequences suggested that until recently, both sarcopterygian and actinopterygian vertebrates had four distinct dact genes that were generated during the second genome duplication in vertebrate evolution   <cit> . to further corroborate this finding and to determine which of the dact genes are more related and hence, originated from a common ancestor, we carried out a phylogenetic analysis of dact proteins, using maximum likelihood and bayesian methods . to ensure that the major chordate taxa are represented, we focused on sequences from humans, opossum, chicken, anole lizard, the western painted turtle, xenopus tropicalis, coelacanth, spotted gar, zebrafish, fugu, tilapia and branchiostoma that were full length or near full length; in addition we included the partial sequences from the elephant shark, and the complete and partial sequences from the two cyclostomes, dacta-d from petromyzon and lethenteron. we used an unbiased approach, i.e. an unrooted tree . likelihood mapping shows that  <dig> % of quartets were fully resolved , indicating the sequences were suitable for phylogenetic reconstruction.\n",
            "\n",
            "in the tree, the gnathostome sequences were placed into four distinct groups . within the dact <dig> group, the dact <dig>  3a and 3b sequences formed the expected subgroups. likewise, the gar and zebrafish dact4r sequences formed a subgroup within the dact <dig> group. thus the phylogenetic tree analysis supports our dact1- <dig> group allocations. within the individual dact groups, sarcopterygian and actinopterygian dact sequences formed subgroups, particularly evident in the rooted trees . the position of the elephant shark sequences was less clear, possibly because these sequences are incomplete. interestingly, in the unrooted tree and the rooted trees, the gnathostome dact <dig> and dact <dig> sequences formed a meta-group. the gnathostome dact <dig> and dact <dig> sequences formed a second metagroup, evident in the maximum likelihood and bayesian trees . the division into the dact1/ <dig> and dact2/ <dig> groups was highly significant in the likelihood mapping analysis  and well supported in the phyml tree for gnathostome sequences . this suggests that of the two dact genes created in 1r, one gave rise to dact <dig> and  <dig>  the other to dact <dig> and  <dig> genes.\n",
            "\n",
            "in the maximum likelihood and bayesian phylogenetic trees for all vertebrate sequences , the cyclostome sequences were grouped together, separated from the gnathostome dacts. the quartet puzzling tree , however, shows a star-like topology for this node, and consequently the evolutionary relationship of cyclostome and gnathostome genes cannot be determined with certainty.\n",
            "\n",
            "organization and relationship of gnathostome dact gene loci\n",
            "our study revealed novel gnathostome dact sequences that were allocated to four paralog groups, based on the combination of aa sequence features and the phylogenetic analysis. to further corroborate this allocation, we analyzed the organization of vertebrate dact genomic loci, reasoning that dact orthologs would reside in syntenic genomic regions. for our analysis, we focused on representative sarcopterygian and actinopterygian species with reasonably well characterized genomes. we first determined the localization of a given dact gene, performing a blast search on the ensembl database. we then established the order of neighboring genes in a 1–2 mb radius , exploiting the ensembl gene annotations or performing blast-searches for these genes. during this process, we noticed that, following inversions and other forms of recombination events, genes associated with a particular dact gene in sarcopterigians often had been placed at a distance in actinopterygians, and vice versa. we therefore also established the wider environment of dact genes .\n",
            "\n",
            "dact <dig> loci\n",
            "genes assigned to the dact <dig> group were invariably linked with timm <dig>  arid4a, psma <dig> . in the gar, talpid <dig> and irf2bpl were found between dact <dig> and timm9; the two genes were also next to dact <dig> in tilapia or on either side of dact <dig> in the zebrafish. in all other organisms, either talpid <dig>  or irf2bpl  was located betweentimm <dig> and dact <dig>  in sarcopterygians as well as in the gar, on the side facing away from the psma3-talipd3/irf2bpl group, dact <dig> was associated with daam <dig> and gpr <dig>  in teleosts, this position was held by fbxo <dig> and tbpl <dig>  which in sarcopterygians were part of a gene group linked to psma <dig>  outside the immediate 1 mb radius around dact <dig>  numerous additional genes were found both in the wider environment of sarcopterygian as well as actinopterygian dact <dig> . thus, although there is some variation in the arrangement of dact <dig> loci, the same genes were associated with dact <dig> in sarcopterygians and actinopterygians. of these genes, psma <dig>  timm <dig> and talpid <dig> are single genes without any paralogs. hence, they serve as unique identifiers of the dact <dig> locus, and support our assignment of genes to the dact <dig> group.\n",
            "\n",
            "dact <dig> loci\n",
            "as amphibians lack a dact <dig> gene and latimeria dact <dig> was on a too short a contig, information on sarcopterygian dact <dig> loci was restricted to amniotes. however, in amniotes as well as in the gar, genes allocated to the dact <dig> group were associated with frmd <dig> on one side and smoc <dig> on the other; in teleosts, smoc <dig> was also always present . thbs <dig> and wdr <dig>  linked to smoc <dig> in amniotes, were within 1 mb distance of dact <dig> in the gar and only slightly more distant in teleosts . similarly, the map4k3-agpat <dig> group was found in the wider environment of all dact <dig> genes, and the sipa1l2-irf2pb2-gng <dig> group was in the wider environment of bird, reptile and actinopterygian dact <dig> . as for dact <dig>  numerous additional genes populated the dact <dig> environment both in amniotes as well as in actinopterygians. moreover, wdr <dig>  and in amniotes phf <dig> and mllt <dig>  are unique and serve as locus identifiers, suggesting that we allocated dact <dig> orthologs correctly. in teleosts, a number of genes are linked with dact <dig> that are not found in the dact <dig> environment of the gar, suggesting that they invaded the locus after the split from the holost lineage . remarkably, traces of dact <dig> locus can still be found in xenopus, since a number of dact <dig> associated genes are well preserved on contig gl <dig> \n",
            "\n",
            "dact <dig> loci\n",
            "for the genes assigned to the dact <dig> group, only limited information was available for platypus and latimeria . in all other animals, dact <dig> genes were accompanied by vasp, snrpd <dig>  dmpk, pglyrp and c5ar , vasp, snrpd <dig>  pglyrp and c5ar , or vasp, dmpk . the prkd2-fkrp-arhgap <dig> group that is closely linked to amniote dact <dig> is found in the wider environment of gar dact <dig> and teleost dact3a, while a duplicated copy of the argap <dig> gene is found in the environment of teleost dact3b. similarly, genes like rtn <dig>  akt <dig> polr2i, opa <dig>  ppp5corsae1are found in the wider environment of all dact <dig> genes, with polr2i, opa <dig>  snrpd <dig>  fkrp and sae <dig> being unique, and ppp5c and c5ar <dig> having no paralogs linked to other dact genes. thus, even though the precise order of genes differs between gnathostome groups and a number of signature genes have disappeared from the teleost dact3b locus, all loci are recognizable as related, supporting our assignment of genes to the dact <dig> group. a set of genes was only found at teleost dact <dig> loci, yet these were present both at the dact3a and 3b locus. this indicates that the teleost dact3a and dact3b genes arose from the teleost-specific 3r  <cit> . in birds, however, almost all of the dact <dig> associated genes were absent, suggesting that the entire locus has been lost.\n",
            "\n",
            "dact <dig> loci\n",
            "as shown above, dact4-type genes were only found in anapsid and diapsid reptiles, in latimeria and in actinopterygians, and the sequences of the sarcopterygian and actinopterygian proteins were rather divergent. yet dact <dig> genes were invariably linked with ttc <dig>  and in most cases, also with map1lc3c . in reptiles and the gar, ttc <dig> was adjoined by hnrnpul <dig>  which was located in the dact <dig> environment in teleosts. in the sarcopterygians, map1lc3c was linked with zbtb <dig> and polr2g, which populated the environment of actinopterygian dact <dig> genes . bscl <dig> was located within the 1 mb environment of all dact <dig> genes, and in the gar and teleosts , also rom <dig> was close by. in acanthopterygian teleosts, the dact <dig> environment showed a stereotype arrangement, and most of the genes found here were also found in the environment of the zebrafish, gar, coelacanth and reptile dact <dig>  of the genes associated with dact <dig> loci, bscl <dig>  ints <dig>  polr2g and stx <dig> are unique and therefore, identify this site. thus, even though the order of genes at dact <dig> loci was not always preserved, the loci, and by extension the genes and proteins were closely related. searching for dact <dig> associated genes in vertebrates that have lost dact <dig>  we noticed that the locus was very well-conserved in mammals and in amphibians, suggesting that their dact <dig> genes disappeared as a result of only a small deletion and possibly recently. in contrast, in birds only a few dispersed genes formerly associated with dact <dig> were present, suggesting a major chromosome rearrangement that resulted in the loss of the entire locus. the intronless dact4r gene found in the gar and zebrafish, however, was not accompanied by any genes linked to the original dact <dig>  yet, the dact4r loci closely resembled each other. this suggests that the dact4r gene was present in the ancestor of holosts and teleosts before the teleost 3r, but was shed from most teleost genomes thereafter.\n",
            "\n",
            "phylogenetic analysis of dact-associated sequences\n",
            "our synteny analysis revealed a number of dact-associated genes specific for a particular dact locus. however, we also found a number of genes with paralogs at several dact loci, suggesting that they were part of the dact locus before the gnathostome 2r. we therefore expected that, if our phylogeny analysis of the dacts were correct, the dact associated sequences would show the same phylogenetic relationships. to test this, we scanned the environment of dact genes for genes that have four paralogs in all vertebrates, each associated with a particular dact locus, making allowances for teleost genes that, after 3r were kept at the locus that since has shed the duplicated dact gene. these criteria applied to ehd1-4; eml1-4; fos, fosb, fosl <dig>  fosl2; mark1-4; rtn1- <dig> and sipa <dig>  sipa1l <dig>  1 l <dig>  1 l <dig> . interestingly, a sipa <dig> homologue was found associated with dacta, and an eml homologue close to dactb in the lethenteron genome . we next extracted the protein sequences encoded by these genes, and wherever possible, the corresponding lamprey, branchiostoma, tunicate or drosophila sequences, and, using the drosophila sequences as outgroups, we constructed phylogenetic trees . notably, the trees obtained for the dact-associated genes always grouped the dact1/ <dig> and dact2/ <dig> associated genes; the other possible permutations  were never observed. this supports the idea that during the vertebrate 2r dact1-dact <dig> arose from one, dact2-dact <dig> from the other dact precursor.\n",
            "\n",
            "analysis of structural motifs in the dact protein groups\n",
            "dacts have been attributed a range of functions in intracellular signaling pathways, all relying on their interaction with other proteins. the ability to interact with partners resides in distinct structural motifs. the identification of a whole family of distinct dact paralogs raises the possibility that different dacts specialize in specific functions, and that this may be reflected in their repertoire of motifs. we hence investigated the exon-intron structure of dact genes, and we investigated the presence and distribution of known protein motifs, searched for the presence of further conserved aa stretches and used the psort and netnes  <dig>  programs to predict functionally relevant motifs. for the ease of comparison, motifs were numbered consecutively; where protein motifs were composed of several linked elements, these were labeled with letters in alphabetical order. the identity matrix for the most conserved regions is included in additional file  <dig>  presence and linear distribution of the motifs is shown in figure 4; the sequences of short motifs are summarized in additional file  <dig>  motifs and longer conserved stretches are indicated in the full alignments of dact orthologs  as well as in the gnathostome dact sequence logos . our approach revealed novel sequence motifs typical for all dact proteins. significantly, we also identified motifs and sequence variations that distinguish dact orthologs and that, even in individual species with six dact genes, assigned them to the four paralog groups.\n",
            "\n",
            "dact1-type sequences\n",
            "conserved stretches of aa in the dact- <dig> type proteins included a putative nuclear export signal encoded by the centre of exon  <dig> , a series of linked elements spanning the 3’ end of exon <dig>  exon <dig> and the 5’ end of exon <dig>  which included a 6x leucine zipper required for homo-and heterodimerization  <cit>  and a nuclear export signal  <cit> , and in comparison to dact <dig> a reduced set of elements encoded by the exon3/ <dig> border . exon  <dig> continued with sequence motifs 4a,b, 5a-c; functionally, the region encompassing motifs 3c-5b has been implicated in tcf <dig> binding; the region encompassing motifs 5b,c was shown to participate in dvl binding  <cit> . following a variable portion, further conserved aa stretches  including a nuclear localization signal  <cit> were recognizable, with motif 7a and the specific sequence of the 10th motif only occurring in this protein group. the last 200aa with motif elements 11a-g were again highly conserved  and encompassed a further putative nuclear localization signal, the known vangl binding domain and the c-terminal pdz binding domain  <cit> .\n",
            "\n",
            "dact2-type sequences\n",
            "in the dact <dig> proteins, exon  <dig> encoded a distinct version of motif  <dig>  which was followed by the exon1- <dig> spanning domain that had  <dig> % identity, contained motifs 2a-f, a 6x leucine zipper and the nuclear export signal. yet the specific sequence of motif 2f was distinct from the corresponding sequence in dact <dig> proteins. the 3’ end of exon  <dig> encoded two sets of sequences  that both resembled dact <dig> motif 3b, indicating that they may have arisen from an internal duplication. exon  <dig> contributed to a specific version of motif 3c, followed by motifs 4b, 5a, motif 5c, motifs 7b and 7c, incomplete motifs 8a,c and motif  <dig>  the c-terminus displayed  <dig> % identity and encompassed motifs 11a,b, partial motif 11c, motif 11d, a distinct version of motif 11e, motif 11f, and a terminal motif 11 g that was reminiscent of the lamprey dacta-c sequences. compared to dact <dig>  motifs 4a, 5b, 7a, the nuclear localization signal motif 8b and motif  <dig> were missing.\n",
            "\n",
            "dact3-type sequences\n",
            "not surprisingly, given the differences in sequence length, dact <dig> proteins had only  <dig> % overall sequence identity. however, these proteins shared a number of features that distinguished them from the other dact-types. dact3-type proteins harbored motif  <dig>  partial motifs 2c-e and 3b, motif 3c, 4a, 5a-c, 7b, incomplete motif 8c, motif  <dig>  motif  <dig>  partial motif 11a, and well recognizable motifs 11b,c,e,f,g. motifs 2a, 4b, 8a and 11d were present in some but not all dact <dig> proteins; motifs 2b, 2f, 3b, 7a, 7c were always absent. interestingly, motifs  <dig>  4a, 5b, 7b, 11e and the pdz binding domain containing motif 11 g resembled the corresponding dact <dig> motifs more than those of dact2; overall dact <dig> motif  <dig> had  <dig> % identity with that of dact <dig> and  <dig> % identity with motif  <dig> of dact <dig>  most remarkable however was a strong reduction of the leucine zipper. owing to sequence variability at the 3’ terminus of exon  <dig> and start of exon  <dig>  this region did not regularly provide a suitable leucine to contribute to the leucine zipper. exon  <dig> encoded for several leucines, but in latimeria, the gar and the teleost dact3a proteins, a loss of 3aa interrupted the regular array of leucines, in most animals leading to a 3x plus 2x leucine zipper arrangement . since these animals represent both the sarcopterygian and the actinopterygian lineage, we concluded that the interruption of the leucine zipper had occurred before the sarcopterygian-actinopterygian split. in tetrapods, further 4aa were lost, such that 2– <dig> correctly placed leucines restored a 3x-5x leucine zipper. on the other hand, in teleost dact3b sequences, the leucine zipper was further reduced with tetraodon dact3b lacking it altogether.\n",
            "\n",
            "dact4-type sequences\n",
            "the overall conservation of the dact <dig> protein sequences was low, but several recognsizable motifs showed much higher sequence similarity. dact <dig> proteins harboured sequence motifs  <dig>  incomplete motif 2a, motifs 2d,e,f, partial motif 3a, motifs 3b, 3c, 4b, 5a, 5c, a dact4-specific motif  <dig>  a dact4-specific motif  <dig> and partial motifs 11a-c. in teleosts, motifs 5c and  <dig> were separated by a repetitive stretch consisting of repetitive asparagines and leucines; motifs  <dig> and  <dig> were separated by a stretch enriched in serines, histidines and prolines. the proteins concluded with a serine-rich domain that was ill-conserved between sarcopterygians and actinopterygians but may represent a degenerate version of motif 11e, followed by a number of alkaline and neutral aa resembling dact1- <dig> motif 11 g. thus, while these proteins evolved some new motifs, a number of motifs present in other dacts were lost. importantly, these newly identified dact proteins lacked the pdz binding domain, suggesting that they may not be able to interact with dvl. similarly, exons 1– <dig> did not encode a leucine zipper, indicating that these proteins may be unable to homo- or heterodimerize.\n",
            "\n",
            "the cyclostome dact proteins\n",
            "the cyclostome dact proteins share many of the conserved motifs identified in the gnathostome dacts. motifs 1-5c, 7b-c, 8b,  <dig>  11a-d, 11f and 11 g were well recognizable in at least one of the cyclostome proteins, and often in all of them. a leucine zipper was recognizable in all available sequence. the dacta protein had a small 2x leucine zipper encoded by exon  <dig>  while dactb showed a bipartite, 2x plus 3x, leucine zipper. no information was available for exon  <dig> of dactc, but exon  <dig> encodes a 2x leucine zipper. the orphan exon  <dig> sequence had a 3x leucine zipper. interestingly, in the dacta gene of both petromyzon and lethenteron, the 11d motif was split by an additional intron, so that the dacta gene is comprised of five exons. some of the motifs shared aa characteristic either for the dact1/ <dig> proteins or for the dact2/ <dig> proteins , but none of the cyclostome dact protein matched with either of these gnathostome metagroups.\n",
            "\n",
            "the branchiostoma dact protein\n",
            "the branchiostoma dact protein was the most divergent of the proteins we analyzed. sequences included a recognizable motif <dig> and a partial motif 2a, and contributed one leucine to a leucine zipper. exon <dig> accounted for 58aa that aligned well with exon2-derived sequences of gnathostome dact1- <dig>  contributing to motifs 2b,c, and to further leucines for an in total 5x leucine zipper. different to vertebrates, however, the branchiostoma exon2- <dig> boundary encoded an extended serine-rich stretch. exon <dig> encoded in total 872aa that encompassed sequences which in vertebrates are encoded by the 3’ end of exon <dig>  and by exons  <dig> and  <dig>  including motifs 2d,e, an incomplete motif 2f, motifs 3c, 4b, partial 5a, motifs 5b, c, the nuclear localization signal associated with motif 8b, motif 11e that was enriched in acidic aa and serines, and partial motifs 11f,g. notably, motifs 5b,c were separated by an extended stretch of  <dig> aa, and the pdz binding domain was missing. of the motifs present in branchiostoma dact, motifs  <dig> and 5b were more similar to motifs in dact1/ <dig> than to dact2/ <dig>  while motifs 2f and 3c more strongly resembled motifs present in dact2/ <dig>  taken together, we traced the origin of dacts back to chordates, where many motifs and functional domains were established already.\n",
            "\n",
            "comparative expression analysis\n",
            "our analysis showed that initially, jawed vertebrates were equipped with four dact genes, of which mammals lost dact <dig>  puffer fish lost dact <dig>  amphibians lost dact <dig> and dact <dig> and birds lost dact <dig> and dact <dig>  on the other hand, after the teleost-specific 3r, these animals kept two dact <dig> genes and hence, gained a dact gene. zebrafish and gar, by retaining the retrotranscribed dact4r gene, gained a further dact gene. all these genes may still show aspects of their original expression patterns and cooperate in a given tissue. alternatively, their expression domains may have been redistributed, with each gene acquiring unique sites of action. to investigate this, we comparatively analyzed dact gene expression in animals with the most divergent complements of dact genes: mouse , chicken , xenopus  versus zebrafish . we focused primarily on pharyngula-early somite stage embryos since at this stage, vertebrate embryos are the most similar . at this stage , mouse dact <dig> was expressed widely, with highest expression levels in the presomitic mesoderm and young somites, the proepicardium, the craniofacial mesenchyme and pharyngeal arches and the trigeminal ganglion. dact <dig> was also expressed widely, with strong signals in somites, the pharyngeal arches and the forelimb bud. dact <dig> showed prominent expression in young somites and the developing intestine ; in more strongly stained specimen, all somites as well as the trigeminal, facial and glossopharyngeal ganglia were labeled . chicken dact <dig> was expressed in the presomitic mesoderm and young somites, the craniofacial mesenchyme, the splanchnopleural lateral mesoderm, several cranial ganglia and the epibranchial placodes ; expression in the mature somites, in the limb mesenchyme and the dorsal root ganglia emerged slightly later at e <dig> . later at e <dig>  the gene was also expressed in the mesenchyme surrounding the dorsal root ganglia, the limb buds, the lung bud and the eye  <cit> . xenopus dact <dig> expression was initially found in the dorsal blastopore lip, the neural plate, the emerging neural crest cells and the emerging paraxial mesoderm . at stage  <dig>  the gene was expressed in the presomitic mesoderm and young somites, the lateral mesoderm and in several cranial ganglia and the posterior lateral line placode ; in more strongly stained specimen, staining was seen in all somites as reported by  <cit> . xenopus dact <dig> showed a rather widespread expression, at gastrulation and neurulation stages labeling the primitive ectoderm, with higher expression levels in the neural plate and newly formed paraxial mesoderm . at stage  <dig>  the gene still was expressed widely, with prominent expression in the somites . in the zebrafish at 36hpf, dact <dig> was expressed widely, including the craniofacial mesenchyme, the somites, the neural tube, the otic vesicle, the pectoral fin bud and the surface ectoderm . a somewhat more restricted expression pattern was found for dact <dig> , which showed strong expression in the pharyngeal arches and the somites. dact3a showed a widespread expression including the hindbrain, pharyngeal arches and somites, while dact3b expression labeled the fore-, mid- and hindbrain, the pharyngeal arches and notochord . dact <dig> and dact4r displayed similar expression patterns, encompassing the brain, the otic vesicle and the pectoral fin bud . taken together, while individual dact genes were expressed in unique, at times species-specific locations, at least one member of the dact1/ <dig> gene group as well as of the dact2/ <dig> gene group was expressed in the paraxial mesoderm, the fin/limb buds and the mesenchyme of the pharyngeal arches in all vertebrates, suggesting that these are sites of original dact function. the exception is xenopus, where no dact2/ <dig> representative is present. here, dact <dig> has taken over dact <dig> expression domains such as the emigrating cranial neural crest cells. notably, in all species, expression domains overlapped, suggesting that dact genes may regulate tgfβ and wnt signaling in a combinatorial fashion.\n",
            "\n",
            "discussion\n",
            "dact multi-adapter proteins are important regulators at the intersection of wnt and tgfβ signaling  <cit> . the aim of this study was to shed light on the evolution of dact genes and their functional domains and motifs. here, we identified previously unknown dact genes and show that they arose late in the deuterostome lineage. in gnathostomes, four dact genes were generated after 2r, and in many extant species, these four genes are still present. the distribution of functional domains and protein motifs suggests that the ancestral dact function lied with wnt signaling; a role in tgfβ signaling may have emerged later. motif reduction in particular in the newly identified dact <dig> suggests that this protein may counteract the function of the other dacts. significantly, many dact genes are co-expressed during development. hence, the complement of dact proteins present in a given tissue will determine the outcome of wnt and tgfβ signaling events.\n",
            "\n",
            "gnathostomes were originally equipped with four dact paralogs\n",
            "previous studies identified dact <dig> , <dig> genes in mouse and humans, a dact <dig> and  <dig> gene in chicken, one dact <dig> gene in frogs , and a dact <dig> and  <dig> gene in zebrafish  <cit> . performing extensive database searches, we identified numerous gnathostome dact genes: four distinct dacts were identified in chondrichthyans; for actinopterygian bony vertebrates, we found five dacts in holosts and four to six in teleosts, and for sarcopterygians, we found four dacts in latimeria as well as in anapsid and diapsid reptiles, two in birds, two in amphibians and three in mammals. the phylogenetic analysis of dact proteins, protein motif comparison and genomic synteny analysis revealed that all these dacts belong to four paralog groups that arose after 2r rather than by individual gene duplication events. subsequently, specifically in the tetrapod lineage individual dact genes were lost, with mammals shedding dact <dig>  birds loosing dact <dig> and dact <dig>  and amphibians loosing dact <dig> and dact <dig>  the presence of dact <dig> in the two reptile lineages and the conservation of the dact <dig> gene locus in mammals and frogs suggest that in tetrapods, this newly discovered gene persisted well after the split of the amphibian and the various amniote lineages, and was independently shed in frogs, birds and mammals.\n",
            "\n",
            "during the vertebrate 2r, dact1/ <dig> arose from one and 2/ <dig> from the other precursor\n",
            "the analysis of dact proteins sequences revealed a number of motifs that distinguish individual dacts. however, we also found motif or motif variations that suggest a particularly close relationship of dact1/ <dig> and dact2/ <dig>  in phylogenetic tree analyses, dact <dig> and dact <dig> proteins formed a metagroup, and dact2/ <dig> formed another metagroup. phylogenetic trees constructed for genes that have paralogs at each of the four dact loci showed the same topology as the dact trees . metagroups linking dact1/2; dact1/4; dact3/ <dig> or dact3/ <dig> and associated genes were never observed. moreover, the pair-wise grouping of dact1/ <dig> and dact2/ <dig> sequences as well as the sequences from dact1/3- or dact2/4-associated genes was supported by high bootstrap values. this suggests that dact1/ <dig> arose from one ancestor and dact2/ <dig> from the other ancestor that had been generated during 1r .\n",
            "\n",
            "tracing the teleost dacts\n",
            "in teleost fish, the genome was duplicated a third time . however, we were only able to identify single dact <dig> and  <dig> genes, located in a conserved, dact1- and dact2-specific genomic environment, respectively. this suggests that immediately after the 3r and before the radiation of teleosts, one of the dact1and dact <dig> genes was shed. in pufferfish, while the dact <dig> locus environment was clearly recognizable, the dact <dig> gene itself was absent, suggesting a more recent gene loss. in contrast to dact <dig> and dact <dig>  consistently two genes and gene loci were found for teleost dact <dig> . in phylogenetic trees, the dact3a and 3b protein sequences formed well supported subgroups. moreover, dact3a and dact3b loci were related but clearly distinguishable. this suggests that teleosts kept both dact <dig> genes and gene loci that were generated during 3r. interestingly, two dact <dig> genes were found in the gar and the zebrafish. the first gene closely resembled the dact <dig> of other vertebrates and consisted of the typical  <dig> exons. the second gene was intronless. it resided in a similar genomic environment in the gar and the zebrafish, but this environment was unrelated to that of the first dact <dig> gene. significantly, the gar is a holost fish that has not undertaken the teleost-specific 3r  <cit> . together, this suggests that the second dact <dig> is a retrotranscribed gene  that appeared in actinopterygians before the holost-teleost split, and, together with the genuine 3r-derived dact4b, was eliminated in all teleosts analyzed here except cyprinids.\n",
            "\n",
            "dact genes evolved late in the deuterostome lineage\n",
            "dact proteins are important regulators of wnt and tgfβ signal transduction. yet these signaling pathways evolved prior to the split of deuterostome and protostome lineages  <cit> . this seems at odds with the current view that dact genes are specific for bony vertebrates  <cit> . our study for the first time identified dact sequences in cyclostome vertebrates and in non-vertebrate chordates. however, despite intensive searches, no dact sequences were found outside chordates, suggesting that dact genes appeared late in the deuterostome lineage. in the cyclostomes petromyzon marinus and lethenteron japonicum, our searches identified several genomic fragments encoding aa sequences with homology to gnathostome dacts. as some of these fragments were unlinked, it was not possible to determine the exact number of dact genes present in cyclostomes. however, at least four distinct dacts could be clearly distinguished. currently, it is controversial whether cyclostomes and gnathostomes shared the first round of genome duplication, whether an independent genome duplication occurred in the cyclostome lineage, or whether individual genes were duplicated  <cit> . while most of the phylogenetic trees rather support independent expansions of the dact family in cyclostomes and gnathostomes, the star-like topology shown by quartet puzzling indicates the uncertainty of their relationship. for non-vertebrate chordates, we were able to identify a dact gene in the florida lancelet, but not in any of the tunicates searched. this is remarkable, given that tunicates are thought to be more closely related to vertebrates than cephalochordates  <cit> . however, tunicates have reduced their body plan during evolution, and it is possible that they secondarily lost their dact gene. we can speculate that the loss of signaling cascades regulators may have facilitated the reduction of tunicate body structures.\n",
            "\n",
            "the original chordate dact may have served in wnt signaling\n",
            "comparing the presence and distribution of functional domains and proteins motifs we found that a number of these, but not all, were shared by dacts from gnathostomes, cyclostomes and the lancelet, including motifs  <dig>  2a-f, 3c, 4b, 5a-c, 8b, 11e-f, and the basic aa of the c-terminal motif 11 g. thus, these motifs may represent the original repertoire of the ancestral dact. motifs 1– <dig> occupy the n-terminal half of dact proteins and encompass the leucine zipper essential for homo- and heterodimerization, a functionally characterized and a further predicted nuclear export signal, a domain that assists binding to dvl and a domain that in gnathostome dact <dig> has been implicated in tcf <dig> binding  <cit> , and this study). the motifs located in the c-terminal half provide a functionally characterized nuclear localization signal  and contribute to the vangl binding domain . all proteins are enriched with serines, particularly in the area containing motifs 2f, 11e. this suggests that already the ancestral dact was a multiadaptor protein, capable of interacting with molecules in the β catenin dependent and pcp wnt signaling pathway, possibly able to shuttle between the nucleus and cytoplasm, and subject to extensive regulation by phosphorylation.\n",
            "\n",
            "in gnathostomes dacts  <dig> , <dig>  motif 11 g contains the k-l/v-mttv sequence, a pdz binding domain required for the interaction of dact with dvl  <cit> . this motif was also found in cyclostome dacta, b and d, suggesting that it was a feature of the dact protein in the last common ancestor of vertebrates. in contrast, the lancelet motif 11 g does not contain a recognizable pdz binding motif. thus, either branchiostoma dact has secondarily lost this sequence, or alternatively, this sequence appeared in the vertebrate lineage. consequently, it cannot be decided when the main dvl-interacting ability of dacts emerged during evolution. however, this function was firmly established in the vertebrate lineage.\n",
            "\n",
            "in addition to the pdz binding domain, a number of further motifs  are found in gnathostome and cyclostome dacts, suggesting that they arose in the vertebrate lineage. motif 4a resides in the tcf <dig> binding domain, and motif 11d maps to the region implicated in vangl binding  <cit> . thus, it is possible that these vertebrate-specific motifs improved the ability to control wnt signaling events. gnathostome proteins exhibit some additional motifs , and the region encompassing motifs 2a-f and 11a-g is strongly conserved. this suggests that the stabilized protein domains carry out essential molecular roles. unfortunately, the gnathostome-specific sequence motifs have not been functionally characterized.\n",
            "\n",
            "the ability to inhibit alk <dig> may have evolved with dact2/ <dig> genes\n",
            "functional studies on mammalian and zebrafish dact <dig> showed that this molecule can regulate both wnt and tgfβ signaling  <cit> . the corresponding test has not been carried out for dact <dig> ; however, in binding assays using mouse dact proteins, only dact <dig> showed significant alk <dig> affinity  <cit> . interestingly, the region that was implicated in dact2-alk <dig> interaction is very similar in all dact <dig> and  <dig> proteins. moreover, this region contains motif 3a which is absent in dact1/ <dig> proteins. furthermore, gnathostome dact2/ <dig> have secondarily lost the s-p rich motif 4a in the tcf <dig> binding domain and motif 5b in the internal dvl binding domain. molecular studies are required to test whether these differences account for the ability of dact proteins to interact with alk <dig>  however, it is quite possible that the ability to regulate tgfβ signaling evolved with or was stabilized in the ancestor of dact2/ <dig>  at the expense of some functions in the wnt signaling system.\n",
            "\n",
            "could the gnathostome dact <dig> be an “anti-dact”?\n",
            "it has been recognized that after the two  rounds of gnathostome genome duplications, re-diploidization occurred for many genes, but duplicated genes involved in signaling were preferentially retained. this has been interpreted as an evolutionary platform to increase complexity  <cit> . however, immediately after these duplication events, biological systems are potentially deregulated and instable. after the 2r, the ancestral gnathostome had four dact genes, all possibly interfering with wnt signaling. moreover, with the duplication of dact2/ <dig>  possibly also the capacity to inhibit tgfβ signaling was enhanced. furthermore, in the actinopterygian lineage, the dact4r gene appeared, potentially further destabilizing the system. how did vertebrates cope with this?\n",
            "\n",
            "in a number of gnathostome lineages, dact genes were shed: mammals lost dact <dig>  birds lost dact <dig> and  <dig>  frogs lost dact <dig> and  <dig> , teleosts lost the duplicated dact <dig> , dact <dig>  dact <dig>  and most also lost dact4r. in animals that kept a complement of dact <dig>   <dig>   <dig>  the dact <dig> leucine zipper was reduced or incapacitated, thus inhibiting the ability to dimerize. in teleost dact <dig> proteins, the motif 4b in the tcf <dig> binding domain was reduced  or removed , possibly reducing tcf3-binding capacity. furthermore, in most  dact3b genes the 3rd exon was lost. thus, specifically in teleosts, dact <dig> genes may have evolved into a less potent version of dact <dig> \n",
            "\n",
            "amongst gnathostome dacts, however, dact <dig> is the most derived. the protein lost , modified  and gained  a number of motifs. significantly, the lost motifs encompass the leucine zipper; thus, the proteins are unable to dimerize. the modified motifs encompass the internal and the c-terminal  dvl binding domain, and hence, dact <dig> proteins may be unable to regulate this key molecule essential for all wnt pathways. since some motifs have been maintained and new motifs have been stabilized, we can assume, however, that the protein is able to carry out some protein-protein interactions. this may lead to a sequestering of dact-interacting proteins, and hence the antagonization of dact <dig> , <dig> function.\n",
            "\n",
            "the combinatorial expression of dact genes may determine the outcome of wnt and tgfβ signaling events in gnathostomes\n",
            "in addition to gene loss or sub- and neo-functionalization, duplicated genes may diversify at the level of their cis-regulatory sequences, leading to expression divergence  <cit> . however, our expression analysis of mouse , chicken , xenopus  and zebrafish dacts  suggests that at the pharyngula- early somite stage of development , dact genes are co-expressed in many tissues. notably, most dact <dig> and  <dig> genes, and where present, dact3/dact3a genes were expressed in the paraxial mesoderm, the fin/limb buds and the craniofacial mesenchyme and pharyngeal arches , suggesting that they are the sites of original dact function before the split of the dact1/ <dig> and dact2/ <dig> groups. this coexpression furthermore suggests that in a given tissue, the outcome of wnt and tgfβ signaling events depends on the combinatorial activity of these dacts.\n",
            "\n",
            "in the zebrafish, dact3b and dact <dig> genes are mainly expressed in the brain, nevertheless still labeling the pharyngeal arches  and the pectoral fin buds . the latter is remarkable since the expression of a retrotranscribed gene depends on the regulatory elements present at the integration site. it could be speculated that this potential anti-dact has been kept since, together with the original dact <dig>  it may counterbalance the function of the numerous dact1- <dig> gene products. however, the net outcome of dact function in mouse and chicken  and in the fish  may be similar.\n",
            "\n",
            "CONCLUSIONS\n",
            "this study traced the evolution of dact genes and with it, the evolution of a molecular system that allows the simultaneous control of wnt and tgfβ signaling. our study suggests that dacts are chordate specific, with gnathostome dact1/ <dig> having arisen from one, dact2/ <dig> from the second precursor generated after 2r. the protein motifs present in the lancelet and gnathostome dacts suggest that while the control of wnt signaling may have been the ancestral role of these proteins, the ability to inhibit tgfβ signaling may have evolved with the gnathostome dact2/ <dig> precursor. moreover, our study raises the possibility that in those vertebrates that kept dact <dig>  this protein may inhibit the function of the other dacts. our study provides the basis for structural and molecular biologists to systematically test the function of the shared and divergent dact protein motifs, and for cell and developmental biologists to explore the combinatorial aspects of dact function.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "our results suggest that dact genes may have evolved in vertebrates prior to the split of the dact family, and that dact genes may have evolved in vertebrates prior to the split of the dact family.\n",
            "our results suggest that dact genes may have evolved in vertebrates prior to the split of the dact family, and that dact genes may have evolved in vertebrates prior to the split of the dact family.\n",
            "our results suggest that dact genes may have evolved in vertebrates prior to the split of the dact family, and that dact genes may have evolved in vertebrates prior to the split of the dact family.\n",
            "our results suggest that dact genes were co-expressed in vertebrates prior to the split of the dact family, and that dact genes may have evolved in vertebrates prior to the split of the dact family.\n",
            "our results suggest that dact genes were co-expressed in vertebrates prior to the split of the dact family, and that dact genes may have evolved in vertebrates prior to the split of the dact family.\n",
            "our results suggest that dact genes were co-expressed in vertebrates prior to the split of the dact family.\n",
            "\n",
            "59\n",
            "[{'summary_text': 'in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> in drgn is not permanent but only transient.\\nin contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\\nin contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\\nin contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\\nin contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\\nin contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\\nin contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\\nin contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\\nin contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 7658 to 8192 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "cation-chloride cotransporters largely determine the action of gaba during neurogenesis . in immature neurons, na–k–2cl  mediates active cl− uptake, promoting depolarizing gaba action, whereas in adult neurons the cl−-extruding kcc <dig> is generally considered to be involved in generating the hyperpolarizing effect of gaba. mature dorsal root ganglion neurons , however, express high nkcc <dig> levels and maintain high intracellular cl− concentration  with resulting gaba-induced depolarization  <cit> . this property is crucial for presynaptic inhibition of spinal sensory feedback .\n",
            "\n",
            "a nkcc <dig> downregulation has been observed around birth in different neuronal types of central nervous system   <cit>  as a result of circulating maternal oxytocin . previously, we showed that drgn at embryonic day  <dig>  displayed higher nkcc <dig> activity and higher intracellular i levels than age-matched motor neurons   <cit> . here, we investigated whether drgn also display nkcc <dig> downregulation at fetal stages near term, before increasing again to high expression levels in adult stage  <cit> .\n",
            "\n",
            "our data show a marked decrease in drgn i at e <dig> and e <dig> compared to e <dig>  decreased activity of nkcc <dig> at e <dig> and e <dig> fully accounts for this reduction in i. a possible role for this transient shift in gabaergic responses in drgn is discussed.\n",
            "\n",
            "RESULTS\n",
            "gaba-induced currents were measured in cultured e <dig> and e <dig> drgn. the results were quantitatively compared to data previously obtained in drgn at e <dig>  <cit> .\n",
            "\n",
            "total membrane capacitance remained stable between e <dig>  <cit>  and e <dig> at 24 ±  <dig> pf , and slightly increased to  <dig>  ±  <dig> pf at e <dig> , indicating that most of the cells examined were small drgn   <cit> . based on egaba, i markedly decreased from 44 mm at e <dig> to  <dig> and 29 mm at e <dig> and e <dig>  respectively . in the presence of the selective nkcc <dig> blocker bumetanide , i was significantly reduced to ~20 mm at all stages, indicating that nkcc1-dependent cl− influx significantly dropped between e <dig>  and e <dig> , with no further decrease up to e <dig> .figure  <dig> intracellular  and bumetanide-induced cl− reduction in drgn. the resting i  dropped significantly between e <dig>  and e <dig> , with no further decrease at e <dig> . in the presence of 10 μm bumetanide , a specific blocker of nkcc <dig>  i was decreased to 20 ± 1 mm  at e <dig> and to 18 ± 1 mm  at e <dig> and e <dig>  the bumetanide-insensitive component remained stable at all stages .\n",
            "\n",
            "\n",
            "\n",
            "changes in nkcc <dig> regulation were further explored by studying i recovery after cl− load and cl− depletion in drgn at e <dig>  applying  <dig>  mm gaba for 5 s at a membrane potential of +70 mv consistently shifted the reversal potential in the positive direction, indicating an increase in i from  <dig> to 43 mm . similar results were previously obtained in e <dig> drgn  <cit>  . following cl− loading, recovery to basal i levels followed a single exponential function with a time constant of  <dig>  ±  <dig>  min , which is not different from that previously recorded in e <dig> drgn  <cit>  . applying  <dig>  mm gaba for 5 s at a membrane potential of −100 mv reduced i by only 3 mm  whereas in e <dig> drgn, i dropped by 8 mm. this difference in i reduction between e <dig> and e <dig> drgn during the depletion protocol was significant . following cl− depletion, recovery to the resting i level was also significantly slower in e <dig> drgn   than in e <dig> drgn  . together, these data strongly suggest that nkcc1-related cl− fluxes markedly decreased in small drgn after day e <dig> figure  <dig> cl− load  and depletion  in drgn at fetal stages e <dig> and e <dig>  drgn were held at −40 mv and 500 μm gaba was applied for 1 s every 30 s. cl− load was induced by a 5 s gaba pulse  during a depolarizing step to +70 mv while cl− depletion was induced by applying the same pulse of gaba during a hyperpolarizing step to −100 mv. i\n",
            "gaba was normalized to the value obtained during the voltage step to + <dig> or −100 mv. the continuous lines show the fit of data during recovery using a single exponential function. recovery time constants after cl− load were  <dig>  and  <dig>  min at e <dig>  and e <dig> , respectively. recovery time constants after cl− depletion amounted to  <dig>  and  <dig>  min at e <dig>  and e <dig> , respectively. insets\n",
            "i\n",
            "gaba recordings at times indicated by matched colors.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "it was previously shown that mature and embryonic drgn sustain high i owing to high nkcc <dig> expression  <cit> . here, we show that embryonic drgn undergo a significant downregulation of nkcc <dig> between e <dig> and e <dig> , which significantly affects cl− homeostasis. the resulting overall decrease in i limits the depolarizing action of gaba. notably, i at e <dig> is comparable to the level reported in the postnatal period   <cit>  and possibly in adult drgn  <cit>  implying that the observed drop in i between day e <dig> and e <dig> is only a transient phenomenon. the underlying mechanism most probably involves decreased nkcc <dig> activity rather than lower expression level .\n",
            "\n",
            "CONCLUSIONS\n",
            "these data accord with previous observations in other cns neuronal types showing that nkcc <dig> activity is decreased as a result of increasing levels of circulating maternal oxytocin at near term. however, in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> in drgn is not permanent but only transient. since we predominantly studied small drgn   <cit>  in which gaba-induced depolarization is essential for sensory perception , we suggest that the transient gabaergic hyperpolarizing shift observed between e <dig> and e <dig> reflects oxytocin-induced fetal adaptation to delivery  <cit> , contributes to fetal analgesia and protects the fetus against neuronal insult.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> in drgn is not permanent but only transient.\n",
            "in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\n",
            "in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\n",
            "in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\n",
            "in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\n",
            "in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\n",
            "in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\n",
            "in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\n",
            "in contrast to other neuronal types, the activity-dependent downregulation of nkcc <dig> is not permanent but only transient.\n",
            "\n",
            "60\n",
            "[{'summary_text': 'seldi-tof-ms serum protein profiles were obtained from three week old piglets experimentally infected with either a combination of pcv <dig> and prrsv , pcv <dig> and ppv , or phosphate buffered saline .\\nseldi-tof-ms serum protein profiles were obtained from three week old piglets experimentally infected with either pcv <dig> or pcv2/ppv .\\nseldi-tof-ms serum protein profiles were obtained from three week old piglets experimentally infected with either a combination of pcv <dig> and prrsv , pcv <dig> and ppv , but not yet diseased, animals.\\nseldi-tof-ms serum protein profiles were obtained from three week old piglets experimentally infected with either pcv <dig> and pcv2/ppv , pcv2/ppv infected animals , as well as in the pcv2/ppv infected animals , as well as in the pcv2/ppv infected animals.\\nseldi-tof-ms serum protein profiles were obtained from three week old piglets experimentally infected with either pcv <dig> or pcv2'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 4582 to 5120 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "in present livestock husbandry with increasing requirements for higher health and welfare issues but also tight economical margins, there is a need for tools allowing the early warning for disease. ideally, easy to perform tools should be available that warn farmers and veterinarians that animals are infected, preferably before the onset of clinical signs. regular use of such tools may diminish growth retardations and production losses. however, tests for early diagnosis can only be developed when animal-associated \"biomarkers\" exist that differ between uninfected healthy animals and infected, but not yet diseased, animals. the search for such biomarkers can be performed by two different approaches, either focusing on differences in predefined \"candidate\" markers, or by comparative fingerprint analysis of \"all\" components present in a biological sample.\n",
            "\n",
            "in human medicine extensive research has been performed aiming at the discovery of early biomarkers for different kinds of disease, including cancer. early diagnosis is important because of increased treatment options and better prognosis when treatments are initiated at an earlier stage  <cit> . in such settings, involving alteration of several pathways and processes, it has been suggested that multiple marker assays lead to an increase in clinical sensitivity and specificity relative to single-marker assays  <cit> . also for the early detection of infections in veterinary medicine it has been shown that a combination of protein biomarkers increases the performance, i.e. for transmissible spongiform encephalopathies , paratuberculosis, dichelobacternodosus and fasciola hepatica  <cit> .\n",
            "\n",
            "the discovery of potential biomarkers for a number of human and animal diseases has been facilitated by proteomic analysis, some of which have already been commercialized  <cit> . comparative proteomic analyses can be performed relatively easily using surface enhanced laser desorption/ionization time-of-flight mass spectrometry   <cit> . seldi-tof-ms technology includes the use of protein chip arrays that specifically bind intact proteins present in biological samples, such as body fluids or tissue extracts. arrays may vary in their surface chemistry, for instance they may have hydrophobic or hydrophilic properties, thereby selectively binding proteins. protein components are solely identified by their specific molecular weights. by comparing seldi-tof-ms profiles, protein components that differ in abundance between  samples can be recognized.\n",
            "\n",
            "seldi-tof-ms generates a profile of peaks representing the relative abundance of each protein component retained on the chip and has a high specificity in distinguishing  samples. this is especially true when used in combination with a technology to enrich low-abundant proteins, i.e. the proteominer™ technology, which is based on affinity chromatography using a solid phase combinatorial peptide ligand library. the latter leads to a reduction of the dynamic range of plasma protein concentrations and an improved access to low abundant proteins. the combination of these technologies provides protein profiles representing the relative concentrations of a large number of high- and low-abundant proteins in a biological sample  <cit> . in addition, this technology can be used at a medium throughput scale.\n",
            "\n",
            "to assess the potential of serum protein profiles as a diagnostic marker for viral infectious diseases in pigs, we used an experimental animal model for porcine circovirus-associated disease , an important swine disease mostly known in the manifestation of postweaningmultisystemic wasting syndrome . it is, at present, one of the most economically important diseases in swine industry. although porcine circovirus type  <dig>  is regarded as the primary causative agent, pcvd is considered a multifactorial disease. pcv <dig> pathogenesis appears to be related to the immune-modulatory effects of the virus while other micro-organisms contribute to the clinical signs associated with pcvd. both porcine parvovirus  and porcine reproductive and respiratory syndrome virus  have been shown to be associated as co-factors. experimental co-infections of ppv or prrsv with pcv <dig> have fully reproduced pcvad. these data have been supported by field data in which these viruses have been isolated in association with pmws  <cit> .\n",
            "\n",
            "the objective of this study was to explore the potential of serum protein profiles consisting of both high- and low-abundant proteins, as measured by seldi-tof-ms, for the diagnosis of early infectious disease status in pigs. to this end the serum protein profiles, obtained from experimentally infected pcv2/ppv, pcv2/prrsv, and control animals were used to investigate the classification accuracy for different comparisons, i.e. infected versus non-infected, pcv2/ppv versus control, pcv2/prrsv versus control, and the three-way classification pcv2/ppv, pcv2/prrsv, and control. in addition, we investigated the classification performance of subsets of protein profiles that varied in the number of used protein components.\n",
            "\n",
            "RESULTS\n",
            "clinical signs and pathology\n",
            "during the course of the experiment, all pigs infected with pcv <dig> in combination with either ppv or prrsv developed clinical disease signs with a varying degree of severity. no systemic disease signs were observed in the control group with the exception of a temporary lameness in one pig and paleness in two pigs. three pigs in the pcv2/prrsv infected group and one pig from the pcv2/ppv infected group were euthanized for humane reasons at  <dig> and  <dig> days post infection. two pigs died directly after blood sampling, supposedly not related to the experimental infection; one from the control group  and another pig from the pcv2/prrsv group .\n",
            "\n",
            "pigs in both the pcv2/prrsv and the pcv2/ppv infected groups showed signs of wasting with a significant difference in weekly body weight gain compared to the control group. this was consistently seen starting at one week post infection . body weight gain was significantly lower in the pcv2/prrsv infected group compared to the pcv2/ppv infected group in the first week post infection. in the second week post infection, weight gain was similar in both infected groups. data are shown in figure 1a.\n",
            "\n",
            "mean rectal temperature in pigs in the pcv2/prrsv infected group increased to febrile temperatures, i.e. rectal temperatures ≥  <dig> °c from two days post infection on for seven days and again elevated mean body temperatures were seen at  <dig> days post infection and for a period of five days between  <dig> and  <dig> days post infection . in the pcv2/ppv infected group febrile body temperatures were observed incidentally at seven days post infection and at  <dig> days post infection.\n",
            "\n",
            "clinical signs started to appear between day six and seven in all pigs from the pcv2/prrsv infected group and also in a number of pigs from the pcv2/ppv infected group. in the pcv2/prrsv group the mean clinical score, based on the occurrence and severity of clinical symptoms, reached  <dig>   of a maximum score of  <dig>  in this group severe disease signs started to develop from seven days post infection on . in the pcv2/ppv infected group the manifestation of disease symptoms occurred slightly later and the mean clinical score was generally lower in this group. clinical signs as severe depression were observed in more than 80% of all pigs from the pcv2/prrsv group compared to 10% in the pcv2/ppv group. whereas respiratory distress was recorded in all infected pigs, 75% from the prrsv co-infected group showed signs of pneumonia and only about 20% of the ppv co-infected group . a palpable increase of the size of the inguinal lymph nodes was found in all pcv2/prrsv infected pigs from  <dig> days post infection on and in three pigs from the pcv2/ppv infected group from  <dig>   <dig> or  <dig> days post infection on.\n",
            "\n",
            "at necropsy, typically for pmws, inguinal lymph nodes and also other lymph nodes were enlarged in all infected pigs and the mean weights of the inguinal lymph nodes in both infected groups were higher than in the control group . in the pcv2/prrsv infected group two of nine pigs had a macroscopically identifiable pneumonia, although a moderate to severe interstitial pneumonia was found in eight of nine pigs based on histology. in the pcv2/ppv infected group macroscopic changes were restricted to increased size of the inguinal lymph node and kidney, liver or lung changes in a few pigs. histologically, most striking was a slight to moderate hepatitis in seven of eight pigs. in five of  <dig> pcv <dig> infected pigs lymph node depletion was observed, in others a hyperplasia was more prominent.\n",
            "\n",
            "after termination of the experiment, tissue samples were tested for the presence of prrsv and pcv <dig> nucleic acid detection by pcr. all infected animals showed strong positive results for pcv <dig> in lymph nodes, lung and spleen. although in control animals no pcv <dig> nucleic acid was detected in pharyngeal swabs throughout the study, low levels of pcv <dig> nucleic acid were found in single or several tissue specimens of six control animals. prrsv nucleic acid was detected in lungs of all pcv2/prrsv infected animals but not of the other groups. no pcr testing has been performed for ppv.\n",
            "\n",
            "acute phase proteins\n",
            "results of acute phase proteins levels are summarized in figure  <dig>  at day five p.i. levels of acute phase proteins did not differ significantly compared to levels in sera collected prior to inoculation, except for pig major protein  and albumin  in the control group and pigmap in the pcv2/ppv infected group . more significant differences were observed at day  <dig> p.i. compared to levels at day zero for three acute phase proteins; haptoglobin  in the pcv2/prrsv group , pigmap in pcv2/ppv  and pcv2/prrsv infected animals , and albumin in pcv2/ppv , pcv2/prrsv infected animals , as well as in the pbs treated control group .\n",
            "\n",
            "some statistically significant differences were observed comparing different experimental groups. at day zero, pigmap levels in animals from the pcv2/prrsv group  and α-lipoprotein  levels in the pcv2/ppv group  were significantly different compared to the control group. at day five p.i., only apoa <dig> , and at day  <dig> only albumin  were discriminative.\n",
            "\n",
            "seldi-tof proteomics\n",
            "serum protein profiles were obtained on day zero as well as on day five post infection, before clinical symptoms became apparent. a third set of protein profiles was obtained at day  <dig> post infection, when all animals from both experimental infections showed clear signs of disease. using the protein chip data manager software a total of  <dig> protein peaks were identified and subjected to further statistical analysis. it should be noted that these  <dig> protein peaks may represent a lower number of proteins, as a certain overlap may be present among the results obtained with the three types of arrays that have been used.\n",
            "\n",
            "protein profiles of serum obtained from all animals prior to infection  were tested for differences. no statistical significant differences could be observed between animal groups, indicating that in this respect, the three groups of animals were very similar at the time of infection.\n",
            "\n",
            "overview of comparisons\n",
            "we performed multiple analyses, comparing the three experimental groups: animals inoculated with  pcv <dig> and ppv ;  pcv <dig> and prrsv ; and  pbs , at three time points. an overview of these analyses is given in table  <dig>  the comparisons can be categorized as follows:  infected animals  versus non-infected control animals,  pcv2/ppv infected animals versus control animals or pcv2/prrsv infected animals versus control animals,  pcv2/ppv, pcv2/prrsv, and control animals as three distinct groups to explore the discriminatory power of serum protein profiles to distinguish the different infection models.\n",
            "\n",
            "pcv2/prrsv,\n",
            "a serum protein profiles of animals in groups with the same code have been clustered for statistical analysis.\n",
            "\n",
            "different comparisons  were carried out, including serum protein profiles of animals from different infection groups and/or time points after infection. seldi-tof-ms profiles were obtained from three week old piglets that were experimentally infected with either a combination of pcv <dig> and prrsv , pcv <dig> and ppv , or phosphate buffered saline . statistical analyses were based on data from day five post infection , day  <dig> p.i., or data from both days combined as indicated in the table.\n",
            "\n",
            "for each comparison the number of significantly different protein peaks in their profiles was determined as well as the classification accuracy. additionally we tested whether reducing the number of protein peaks in a profile affected the accuracy of classification. next to whole protein profiles consisting of  <dig> protein peaks, we also tested profiles with  <dig>   <dig>   <dig>   <dig>   <dig>  and  <dig> protein components. for this, proteins were ranked by significance and those with highest significance were selected.\n",
            "\n",
            "number of significant different protein peaks\n",
            "for each comparison the number of significantly different protein peaks is shown in table  <dig>  based on a maximum p-value of  <dig> , the number of differentially expressed protein components ranged from  <dig>  to  <dig> . for a number of comparisons, molecular masses of the most significant differently expressed protein components are summarized in table  <dig> \n",
            "\n",
            "healthy versus\n",
            "pcv2/ppv versus\n",
            "pcv2/prrsv versus\n",
            "pcv2/ppv versus\n",
            "pcv2/prrsv versus\n",
            "pcv2/ppv versus\n",
            "pcv2/prrsv versus\n",
            "pcv2/ppv,\n",
            "a: see table  <dig> for description of analysis.\n",
            "\n",
            "seldi-tof-ms protein profiles were generated from three week old piglets experimentally infected with either a combination of pcv <dig> and prrsv , pcv <dig> and ppv , or phosphate buffered saline . the number of significantly differentially expressed protein components with p <  <dig>  or with false discovery rate  <  <dig>  for different analyses are indicated.\n",
            "\n",
            "pcv2/prrsv, and\n",
            "the ten most significantly differently expressed protein components for four analyses as characterized by mass:charge  value with their q-value  are summarized. seldi-tof-ms protein profiles were generated from three week old piglets experimentally infected with either a combination of pcv <dig> and prrsv , pcv <dig> and ppv , or phosphate buffered saline . the figures are based on seldi-tof-ms data from day five and  <dig> post infection combined. the overlap in protein components within different comparative analyses is due to the use of different seldi-tof-ms protein chip arrays.\n",
            "\n",
            "as expected, based on the false discovery rate , which may be more appropriate than p-values as it accounts for multiple testing, the amount of differentially expressed protein components was reduced. in a number of analyses, none of the protein components showed a significantly different expression based on an fdr <  <dig> . in general, more protein components were differentially expressed comparing pcv2/prrsv infected animals versus non-infected control animals as opposed to pcv2/ppv infected versus control animals as shown in table  <dig>  combining data of day five and day  <dig> post infection  increased the number of significantly differentially expressed protein components.\n",
            "\n",
            "two-group classification\n",
            "infected animals versus non-infected animals\n",
            "in this analysis, pcv2/ppv along with pcv2/prrsv infected animals were marked as infected, whereas control animals at day five and day  <dig> are regarded as non-infected . classification results are summarized in table  <dig> \n",
            "\n",
            "number of proteins\n",
            "the number of correctly classified animals  at day five and day  <dig> post infection using increasing numbers of seldi-tof-ms protein components. serum protein profiles were generated from three week old piglets experimentally infected with either a combination of pcv <dig> and prrsv , pcv <dig> and ppv , or phosphate buffered saline . in the top row, for each experimental group the total number of animals included in the analysis is specified between brackets.\n",
            "\n",
            "at day  <dig> post infection, both pcv2/ppv and pcv2/prrsv infected animals displayed evident signs of illness with  <dig> of  <dig>  =  <dig> %; specificity  =  <dig> %) infected animals having significantly different serum protein profiles compared to non-infected animals. moreover at day five, before any disease symptoms were apparent,  <dig> of  <dig>  infected animals could be recognized based on their serum protein profiles.\n",
            "\n",
            "interestingly, preselecting protein peaks slightly increased the classification accuracy; using the  <dig>  or  <dig>  most significant differently expressed proteins resulted in the highest number of correctly classified animals. when the number of proteins was further limited to ten, a decrease in correctly classified animals was observed at day five . however at day  <dig> post infection, profiles based on the ten most significant proteins could still identify 93%  of the infected animals, although only one of the seven non-infected animals was correctly classified  .\n",
            "\n",
            "two-group classification: pcv2/ppv versus control or pcv2/prrsv versus control\n",
            "serum protein profiles of pcv2/ppv infected animals were compared with profiles of non-infected control animals on day five and day  <dig> using either data from a single time point or combining data from both days. similarly, profiles of pcv2/prrsv infected animals were compared with control animals. results are summarized in table  <dig> \n",
            "\n",
            "separate/\n",
            "of used\n",
            "the number of correctly classified animals according to infection status , based on serum protein profiles at day five and day  <dig> post infection . the classification results are presented using either data of each day separately or combined. protein components were selected based on their differential expression in piglets infected with pcv2/ppv, pcv2/prrsv and control piglets, using those with the lowest p-values. in the top row, for each experimental group the total number of animals included in the analysis is specified between brackets.\n",
            "\n",
            "with regard to the classification of infected versus non-infected animals at day five post infection, results were poor for the pcv2/ppv group as only one of seven  infected animals could be distinguished from control animals by profiles consisting of  <dig> protein components. the classification performance of serum protein profiles was much better for pcv2/prrsv infected animals  at day five post infection.\n",
            "\n",
            "as expected, overall results were better at day  <dig> post infection. six of eight  pcv2/ppv and five of seven  pcv2/prrsv infected animals could be distinguished from non-infected control animals by serum protein profiles using  <dig> protein components. this reflects the clinical signs, which were very similar for animals in both groups at  <dig> days post infection.\n",
            "\n",
            "combining the data of both days slightly increased the classification accuracy as compared to profiles from either day five or day  <dig> post infection. this is especially true for the control groups as shown in table  <dig> .\n",
            "\n",
            "again, using a preselection of the most significantly different protein peaks generally led to comparable or even better classification accuracy, similarly as described for the classification of infected versus non-infected animals.\n",
            "\n",
            "three-group classification\n",
            "to assess the power of serum protein profiles as a diagnostic marker for specific infections, we explored the classification performance of serum protein profiles to distinguish between the three distinct animal groups  non-infected control animals,  pcv2/prrsv and  pcv2/ppv infected animals) in one analysis, i.e. three-group classification. as expected, three-group classification as shown in table  <dig> gave similar results compared to the two-group classification, but with lower classification accuracy.\n",
            "\n",
            "number of protein\n",
            "the number of correctly classified animals according to disease status , based on serum protein profiles consisting of variable number of protein components. serum protein profiles were obtained at day five and  <dig> post infection  and data of both time points were combined for the statistical analysis. protein components were selected based on their differential expression in piglets infected with pcv2/ppv, pcv2/prrsv and control piglets, using those with the lowest p-values. in the top row, for each experimental group the total number of animals included in the analysis is specified between brackets.\n",
            "\n",
            "classification\n",
            "results based\n",
            "on selditof\n",
            " <dig> the p-value of the one-sided fisher exact test was <  <dig>  and  <dig>  at day  <dig> and  <dig> respectively.\n",
            "\n",
            "contingency table for 3-way classification, showing the number of correctly and incorrectly classified animals at day five and  <dig> post infection , respectively. classification was based on seldi-tof-ms serum protein profiles from piglets at day five and  <dig> after inoculation with either pcv2/ppv, pcv2/prrsv or phosphate buffered saline . data of both time points were combined for the statistical analysis using  <dig> preselected protein components. protein components were selected on their differential expression, using those with the lowest p-values.\n",
            "\n",
            "in conclusion, pcv2/prrsv infected animals could well be distinguished from control animals as early as day five, while pcv2/ppv infections were best distinguished from non-infected control animals at day  <dig>  classification results showed a very high significance at day five  and were near significance at day  <dig> . it can be concluded that based on seldi-tof protein profiles, at day five post infection pcv2/prrsv infected animals are easier to distinguish compared to pcv2/ppv infected animals.\n",
            "\n",
            "discussion\n",
            "livestock health is an important issue for farmers and veterinarians as well as for consumers. it has an important economic drive as it affects productivity. in addition, from the animal welfare perspective there is a need for parameters that can objectively measure abnormalities or deterioration of health, preferably in an early stage of disease. although the number of animals involved in this study was rather small, this study shows that seldi-tof ms profiles of high- and low-abundant serum proteins have potential as diagnostic markers for early detection of viral infections in pigs. we also show that classification of animals using ridge penalized partial least squares analysis of the protein profiles might be a powerful approach for this.\n",
            "\n",
            "we realize that in the current setting, we collected data under standardized experimental conditions. however, to make a useful and robust multi-marker test based on protein profiles, test development and validation should include the use of animals that originate from different breeds, different farms, different time points post infection and from animals with different disease history. since such factors will create additional variation in protein profiles, larger sample sizes will be required. in this respect it is promising that combining profiles of day five and day  <dig> resulted in improved classification accuracies, suggesting that protein fingerprints of different time points after infection show similarities, which might be utilized under field conditions when time of infection varies.\n",
            "\n",
            "correlation between seldi-tof ms results and clinical data\n",
            "the aim of this study was to explore the potential of quantitative data of high- and low-abundant serum protein components as measured by seldi-tof-ms for early detection and diagnosis of viral infectious diseases in pigs. the experimental infection model  of the animal groups was considered as the golden standard, which was reflected by representative clinical signs. the infection status was confirmed by pcr, after termination of the experiment.\n",
            "\n",
            "the seldi-tof ms serum protein fingerprints reflect the clinical status of the two different disease courses having a comparable disease outcome. among pcv2/prrsv infected animals, body temperatures rose after day five post infection, while clinical scores increased sharply from day six onwards when infected animals were depressed and showed clear respiratory distress symptoms. among pcv2/ppv infected animals body temperatures started to rise from day six onwards, while clinical scores did not increase until day seven post infection, typically consisting only of mild depression. it can be speculated that for pcv2/ppv infected animals protein fingerprints taken at day five may have been too early for diagnostic purposes, in contrast to the prrsv/pcv <dig> infected animals.\n",
            "\n",
            "it should be noted that among the control animals no apparent clinical signs were observed. the low content of pcv <dig> viral dna in tissue samples of some control animals are considered to have been \"false positive\" test results. however it cannot be fully excluded that, in spite of containment measures during the animal experiment, some level of cross contamination with the pcv <dig> virus occurred in the control group.\n",
            "\n",
            "using a whole-protein profile approach instead of candidate proteins\n",
            "in the discovery of biomarkers a targeted approach is often used, aiming at a selection of predefined biomarkers based on current knowledge of the biological function of proteins or known associations. an example for such approach is the use of acute phase proteins for early diagnostic markers for infections. as levels of these proteins change early in the process of infection or tissue trauma, they have been suggested as suitable biomarkers  <cit> .\n",
            "\n",
            "in the present study, the analysis of acute phase proteins led to disappointing results . similar to our experience, experiments conducted by heegaard et al. found large between-animal variation and major differences in prechallenge concentrations between experimental groups  <cit> , limiting the use of acute phase proteins as general disease markers.\n",
            "\n",
            "as an alternative, we explored a whole protein profile approach using seldi-tof-ms and comparative fingerprint analysis of whole protein profiles present in blood samples and studied its value for early disease diagnosis. this approach was chosen because it enables the identification and selection of \"reactive profiles\" without any prior knowledge of the biological functions of the components constituting the profiles. although knowledge of biological function of proteins could have additional value and can be used as biological validation, it is not mandatory. another advantage is that protein profile fingerprints enable the monitoring of quantitative changes rather than determining particular threshold levels of individual serum proteins.\n",
            "\n",
            "a major challenge in the discovery of protein biomarkers from blood is the vast difference in concentration between high- and low-abundant proteins. with traditional analysis methods, the high-abundance proteins usually dominate the proteome profiles, making the identification of less abundant protein components more challenging. different strategies have been developed to eliminate some of the most abundant proteins from blood serum or plasma  <cit> . here we applied the proteominer™ fractionation kit from bio-rad. it is based on a bead-bound random peptide library that provides a vast amount of different binding sites for different proteins. since there is only a small number of ligands that can bind to the same protein, this limits the number of identical high-abundant protein components that can bind to the bead-bound library. the combination of depletion, enrichment and fractionation through the proteominer™ fractionation kit used in this study has led to the detection of a high number of differentially expressed high- and/or low-abundance proteins  underlining the value of this technique. focussing on low-abundance proteins, rather than the classical plasma proteins, might be a more promising approach since low abundant biomarkers may include proteins that either leak into the plasma from different tissues as a result of the infection or that play a role as signal molecules.\n",
            "\n",
            "classification of infected versus non-infected animals\n",
            "the difficulty with multiple disease classification is that large sample sizes are required. therefore, classifying animals as infected and not-infected or diseased and not-diseased is probably a first starting point. in this study, no disease symptoms were yet apparent at day five, except that a number of pigs in the pcv2/prrsv group had elevated body temperatures. notably,  <dig> of  <dig>  infected animals could be recognized based on serum protein profiles. as expected, at day  <dig> post infection results were even better . further analysis will have to provide information whether these differences are due to inflammatory processes or other viral -host interactions. from experimental studies increases in ifnγ secreting cells and interleukin  <dig> have been shown as early as  <dig> and  <dig> days post infection, respectively  <cit> .\n",
            "\n",
            "also classification according to distinct infection models showed promising results. however, the lower sensitivity as revealed by table  <dig> suggests that classification with respect of specific infectious diseases will be challenging. interestingly, the classification performance on both sample days for infected animals was superior compared to non-infected animals. this probably reflects the normal variation in serum protein profiles among \"healthy\" animals which is relatively large compared to infected animals in this small cohort. also batxelli-molina et al. found more extensive variation in serum protein profiles from non-infected animals compared to infected animals  <cit> .\n",
            "\n",
            "it has been shown that in clinical settings multiple marker assays have increased sensitivity and specificity compared to single-marker assays  <cit> . it may be speculated that increasing the number of markers leads to a further improvement of the diagnostic performance. indeed, the number of protein components that were statistically significant in differential expression between the distinct animal groups  correlated well with the classification accuracy: the higher the number of significant protein markers between the groups, the better the classification accuracy. however, our findings also suggest that extending protein profiles to more than about  <dig> markers does not substantially increase classification accuracy. in our case, limiting the number of  proteins from whole protein profiles of  <dig> to the ten most significant differentially expressed components did only marginally decrease the classification accuracy. also the small number of animals used in this study limits the power of a high number of proteins in their contribution to classification accuracy. this indicates that although there might be quite a number of proteins markers associated with the disease status, performance of profiles seems to be more affected by the predictive value of individual proteins than by the number of proteins included in the profile.\n",
            "\n",
            "statistical methods\n",
            "for evaluation of seldi-tof data a decision tree method is frequently used. however, to analyse complete sets of multiple protein peaks, more sophisticated statistical methods are required. we used ridge penalized partial least regression to classify animals, which is superior to decision tree analysis when there are many proteins contributing to the classification or, in other words, when many proteins are likely to be different between infected and non-infected animals. partial least squares techniques have also been applied in disease classification in humans  <cit> . the significance testing of individual proteins was equivalent to the approach used by batxelli-molina et al.  <cit>  and barr et al.  <cit>  for testing the significance of proteins in the diagnosis of prion diseases.\n",
            "\n",
            "in this case, we used one-leave-out cross-validation, because of the very limited number of animals per class. due to the fact that one animal is left out, the unbalance in animals per disease class may be larger. nevertheless, two-group as well as three-group classifications were quite successful, although with lower accuracy in the latter. clearly larger sample sizes are necessary to improve the classification accuracy to more than 90% sensitivity and specificity required for diagnostic purposes.\n",
            "\n",
            "towards development of biomarkers for livestock health\n",
            "this study shows the potential of protein profiles in combination with advanced statistical methods to distinguish infected from non-infected animals, providing etiological information as well. such an approach may be valuable in the diagnosis of infectious diseases in the early stage of disease. in this study we examined sera from animals experimentally infected with pcv <dig> in combination with either ppv or prrsv. as shown in table  <dig> a number of protein components were significantly differentially expressed in multiple comparisons. for instance protein with mass:charge  <dig> was evident in all four analyses. such proteins may be regarded as key candidate markers and further investigation is warranted.\n",
            "\n",
            "in follow-up studies it would be of interest to explore the classification of animals according to aetiology, such as bacterial, viral, and parasitical infections. also differentiation according to disease stage, i.e. acute versus chronic or affected organ system could be useful. the ultimate goal might be the development of assays for health versus disease as opposed to specific etiologic agents.\n",
            "\n",
            "the advances of proteomic technologies and promising study results have fed the hope to obtain biomarkers for improved and faster diagnostics. due to the high costs and required technical skills, spectrometry has traditionally been limited to research settings. however, it is now increasingly used for diagnostic purposes in routine settings for the identification of infectious microorganisms  <cit> . seldi-tof-ms is a promising tool to determine protein profiles at medium throughput level and at reasonable costs. in human medicine, proteomic methods are increasingly used for early diagnosis of diseases  <cit> . in addition, it appears that the challenges of multiplexing such tests  are sufficiently daunting that quantitative mass spectrometry may have value as an additional format for multiplexing protein measurements in the future given aggressive technology development. a major disadvantage for mass diagnostics as required in livestock veterinary medicine is the invasive procedure of blood sampling to get appropriate test material for analysis of biomarkers in serum. an alternative for the use of sera would be to explore the potential of protein profiles in easy to access biological samples like saliva, urine or faeces. also, the recent developments in the field of micro- and nanotechnology have seen a rapid surge in interest in electronic devices for medical implants for in vivo health monitoring. in the human biomedical field several promising prototypes are emerging, for example for monitoring of patients with chronic cardiac or neurological diseases  <cit> . similar developments may be expected for the veterinary health care sector.\n",
            "\n",
            "CONCLUSIONS\n",
            "in this study the potential of quantitative protein profiles by seldi-tof ms for early diagnosis of viral infections in pigs was explored. results from serum of pigs experimentally infected with a combination of pcv/ppv and pcv2/prrsv indicate that seldi-tof protein profiles have potential for detection of  infection in pigs in early phase of the disease. the accuracy of classification of infected versus non-infected animals was good, as 88% of the infected animals could be classified based on the serum protein profiles at day five post infection, that is before clinical symptoms became apparent. at day  <dig> post infection, 93% of the infected animals were classified as such. results for pcv2/prrsv were superior compared to pcv2/ppv infected animals, especially at day five post infection. the lower specificity, both at day five  and day  <dig> post infection , probably reflects the variation in serum protein profiles among non-infected animals. limiting the number of proteins in the profile generally had minor effects on the classification accuracy. accuracy of three-way classification was less than that of two-way classification. it can be concluded that seldi-tof ms protein profiles may have potential as biomarker for early diagnosis of viral infections in animal husbandry.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "seldi-tof-ms serum protein profiles were obtained from three week old piglets experimentally infected with either a combination of pcv <dig> and prrsv , pcv <dig> and ppv , or phosphate buffered saline .\n",
            "seldi-tof-ms serum protein profiles were obtained from three week old piglets experimentally infected with either pcv <dig> or pcv2/ppv .\n",
            "seldi-tof-ms serum protein profiles were obtained from three week old piglets experimentally infected with either a combination of pcv <dig> and prrsv , pcv <dig> and ppv , but not yet diseased, animals.\n",
            "seldi-tof-ms serum protein profiles were obtained from three week old piglets experimentally infected with either pcv <dig> and pcv2/ppv , pcv2/ppv infected animals , as well as in the pcv2/ppv infected animals , as well as in the pcv2/ppv infected animals.\n",
            "seldi-tof-ms serum protein profiles were obtained from three week old piglets experimentally infected with either pcv <dig> or pcv2\n",
            "61\n",
            "[{'summary_text': 'in this study we tested the hypothesis that amino acids and insulin act together to suppress the activity of the ubiquitin-proteasome pathway in c2c <dig> myotubes, via the ubiquitin-proteasome pathway.\\nin this study we tested the hypothesis that amino acids and insulin act together to regulate the activity of the ubiquitin-proteasome pathway in c2c <dig> myotubes, via the ubiquitin-proteasome pathway.\\nin this study we tested the hypothesis that amino acids and insulin act together to regulate the activity of the ubiquitin-proteasome pathway in c2c <dig> myotubes, via the ubiquitin-proteasome pathway.\\nin this study we tested the hypothesis that amino acids and insulin act together to regulate the activity of the ubiquitin-proteasome pathway in c2c <dig> myotubes, via the ubiquitin-proteasome pathway.\\nin this study we demonstrate that amino acids and insulin act together to suppress the activity of the ubiquitin-proteasome pathway in c2c <dig> myotubes, via the ubiquitin-proteasome pathway'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 6875 to 7168 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "starvation induces muscle wasting by decreasing the rate of muscle protein synthesis and increasing protein degradation. a decrease in plasma concentrations of amino acids, and in particular branched-chain amino acids, is thought to synergise with lower levels of insulin to signal the starvation-induced decrease in protein synthesis  <cit>  but the mechanisms responsible for the up regulation of muscle proteolysis are less well characterised. the major proteolytic systems that are activated in atrophying skeletal muscle are lysosomal, calcium dependent and atp dependent ubiquitin-proteasome pathway  <cit> . of these, the latter is considered the predominant biological mechanism controlling protein degradation in myofibrillar proteins in skeletal muscle  <cit> . this proteolytic pathway comprises of a cascade of atp dependent enzymatic reactions in which ubiquitin protein is activated by ubiquitin activating enzyme, e <dig>  which then transfers ubiquitin to ubiquitin conjugating enzyme, e <dig>  this assembly then binds either directly to protein substrate or involves another ligating enzyme, e <dig>  the cycle continues to produce a polyubiquitinated protein substrate which then enters the  <dig> s proteasome complex for rapid degradation. the  <dig> s proteasome, catalytic core of  <dig> s proteasome, comprises subunits having peptidase activity, which digest proteins  <cit> . increased expression of 14-kda e <dig> and e <dig> ubiquitin ligases, atrogin-1/mafbx and murf <dig> has been observed in atrophying muscle  <cit> .\n",
            "\n",
            "low level of plasma insulin triggers protein degradation in muscle through activation of the ubiquitin-proteasome pathway  <cit>  while higher levels down regulate the expression of 14-kda e <dig> conjugating enzyme proteins in vitro  <cit> . in vivo, a 6-hour hyperinsulinaemic, hyper-aminoacidaemic and euglycaemic clamp significantly reduced the mrna expression for ubiquitin in fast twitch and mixed skeletal muscle  <cit> . constant infusions of glucose and a mixture of essential amino acids in calves significantly attenuated mrna expression for c <dig> sub unit of  <dig> s proteasome  <cit> . observations in hepatoma cells have revealed that insulin regulates its anticatabolic activity by decreasing ubiquitin mediated proteasomal activity  <cit> . despite the evidence that the ubiquitin-proteasome pathway may be responsible for the increased rates of muscle proteolysis during fasting, in vitro studies in c2c <dig> myotubes and hepatocytes have suggested that lysosome-dependent autophagic pathway is responsible for proteolytic responses to complete starvation of amino acids or from complete media  <cit> .\n",
            "\n",
            "signalling cascades that regulate intracellular protein phosphorylation during nutrient deficiency and hormonal imbalance are not well characterised. studies in various cell types have demonstrated that the mammalian target of rapamycin  signalling pathway, integrates the effects of insulin and amino acids on muscle protein synthesis through phosphorylation of two key regulators of translation,  <dig> kda ribosomal protein s <dig> kinase  and eukaryotic initiation factor  <dig> e binding protein- <dig>   <cit> . mtor has also been identified as an important protein kinase in the starvation-signalling pathway of autophagy  <cit>  and activation of s <dig> kinase is associated with autophagy in rat hepatocytes  <cit>  c2c <dig> myotubes  <cit> .\n",
            "\n",
            "in this paper we test the hypothesis that the antiproteolytic actions of insulin and amino acids in c2c <dig> myotubes are mediated by the ubiquitin-dependent proteasome pathway. our data demonstrate that low levels of amino acids increase ubiquitin-proteasome-dependent proteolysis in myotubes and that amino acids act additively with insulin.\n",
            "\n",
            "RESULTS\n",
            "it is an inherent characteristic of c2c <dig> myoblasts that they fuse into multinucleated myotubes when the level of serum in the incubation medium is reduced. the expression of many muscle specific genes such as creatine kinase, desmin and myosin heavy chain is increased during myogenic differentiation  <cit> . differentiation of myotubes was confirmed by approximately 30-fold increase in cpk activity relative to unfused myoblasts  and by visualisation under phase contrast microscopy of elongated, multinucleated cells. cpk activity remained high under all incubation conditions .\n",
            "\n",
            "effect of insulin on total protein degradation\n",
            "initially we aimed to determine the concentration of insulin that can significantly inhibit proteolysis during  <dig> hours of incubation. insulin inhibited the release of tca soluble radioactivity from c2c <dig> myotubes pre-labelled with l-tyrosine at all of the concentrations employed with the half-maximal effect on proteolysis of approximately  <dig> - <dig> m  .\n",
            "\n",
            "amino acids and insulin act additively to regulate total proteolysis\n",
            "physiological  and sub-physiological concentration of amino acids  were used to determine the interaction between amino acids and insulin in the regulation of skeletal muscle protein degradation. protein degradation was increased by 9–12% when differentiated myotubes were incubated with the amino acid limiting  <dig>  × pc aa media compared with  <dig>  × pc aa, in both the absence and presence of insulin . insulin addition  to the media decreased protein degradation  at both concentrations of amino acids . the inhibitory effects of amino acids and insulin on protein degradation in differentiated c2c <dig> cells were additive.\n",
            "\n",
            "proteasome inhibitor mg <dig> attenuates amino acid mediated total proteolysis\n",
            "next we employed the proteasome inhibitor mg <dig>  to determine the role of the 26s proteasome pathway in responses to varying amino acid concentration. mg <dig> inhibited proteolysis by about 75% in incubations with both physiological and sub-physiological concentrations of amino acids, but importantly, the inhibitor abolished the difference in proteolysis between the two amino acid levels . this result demonstrated that media amino acid depletion increased ubiquitin-proteasome-dependent-proteolysis in c2c <dig> myotubes.\n",
            "\n",
            "amino acids and insulin down regulate expression of components of ubiquitin-proteasome pathway\n",
            "having observed significant additive changes in total protein degradation in response to increasing amino acid and insulin media concentrations, we next sought to discover whether changes in the expression of components of the ubiquitin-proteasome pathway were also sensitive to these treatments. compared to  <dig> × pc aa, the  <dig>  × pc aa medium elicited a significant increase in mrna expression of the 14-kda e <dig> conjugating enzyme of the ubiquitin-proteasome pathway in c2c <dig> myotubes . when insulin was added to the medium there was a decrease in the expression of the gene encoding 14-kda e <dig> in the presence of both  <dig>  × and  <dig>  × pc aa . insulin and amino acids were observed to have additive inhibitory effects on 14-kda e <dig> gene expressions.\n",
            "\n",
            "next we sought to determine if the c <dig> and c <dig> subunits of the  <dig> s proteolytic core of the proteasome were also responding to the availability of amino acids and insulin. the expression of the gene encoding c <dig> was reduced by 21% when c2c <dig> myotubes were treated with  <dig>  × pc aa, compared to  <dig>  × pc aa . insulin addition to media containing either the low or physiological amino acid concentrations further decreased c <dig> mrna abundance in the myotubes, compared to that observed in the absence of insulin . incubation with the  <dig>  × pc aa compared with  <dig>  × pc aa elicited an inhibitory effect on the expression of c <dig> in the differentiated c <dig> c <dig> incubations but this was not statistically significant.\n",
            "\n",
            "leucine increases the inhibitory effect of insulin on protein degradation\n",
            "we next assessed the specific effect of leucine on myotube protein degradation by supplementing the amino acid limiting  <dig>  × pc aa media with three levels of leucine ranging from sub-physiological to supra-physiological  and  <dig> μm). in the absence of insulin, increasing the concentration of leucine in the media tended to inhibit myotube protein degradation but this effect was not significant . in the presence of insulin, proteolysis was progressively inhibited by the addition of increasing leucine concentration with both  <dig> and  <dig> μm leucine, being significantly lower compared to  <dig> μm leucine . these results differed from the inhibitory response observed in the absence of insulin when the concentration of the physiological mixture of amino acids was increased  and suggest that the effects of amino acids supply are not simply due to variations in leucine concentration. the results also indicate that the inhibitory effect of insulin on myotube protein degradation is dependent on the presence of a physiological concentration of leucine in the media.\n",
            "\n",
            "leucine can modulate the expression of components of ubiquitin-proteasome pathway\n",
            "there was a dose dependent decrease in myotube 14-kda e <dig> expression in response to increasing leucine concentrations, both in the presence or absence of insulin and these effects of leucine and insulin were additive . compared to  <dig> μm,  <dig> μm leucine caused a significant reduction in myotube expression of c <dig>  both in the presence and absence of insulin . no significant effects of leucine on c <dig> expression were observed.\n",
            "\n",
            "amino acids but not leucine, regulate protein degradation and 14-kda e <dig> expression through the mtor signalling pathway\n",
            "the studies reported to date have suggested that both amino acids and insulin act on protein synthesis through the mtor signalling pathway  <cit> . we therefore tested the hypothesis that the effects of amino acids and insulin, on protein degradation in c2c <dig> myotubes are also mediated through the mtor pathway using rapamycin, a well defined inhibitor of mtor. in order to validate if rapamycin was inhibiting phosphorylation of p  <dig> s <dig> kinase, western blotting followed by immunodetection was performed. rapamycin completely abolished p <dig> s <dig> kinase phosphorylation at thr  <dig> site .\n",
            "\n",
            "myotubes were challenged with rapamycin followed by stimulation with insulin. rapamycin treatment of cells did not alter proteolysis during incubations with  <dig>  × pc aa, in the presence and absence of insulin . addition of rapamycin, however, blocked the inhibitory effect on protein degradation of increasing media amino acid concentrations from  <dig>  × pc aa to  <dig>  × pc aa, with and without insulin . rapamycin did not block the inhibitory action of insulin, on myotube protein degradation. in contrast to the effects of low concentrations of a physiological mixture of amino acids, the stimulatory effect of decreasing the media leucine concentration from  <dig> μm to  <dig> μm, on myotube protein degradation was not blocked in the presence of rapamycin . as in the previous experiment, rapamycin also failed to alter the inhibitory effect of insulin on protein degradation.\n",
            "\n",
            "rapamycin increased the expression of the ubiquitin conjugating enzyme 14-kda e <dig> in both the presence and absence of insulin during incubations with  <dig>  × pc aa, although these changes did not attain statistical significance .\n",
            "\n",
            "taken together these studies with rapamycin demonstrate that varying the concentration of an amino acid mixture influences proteolysis and expression of 14-kda e <dig>  in c2c <dig> myotubes, via the mtor pathway whereas leucine and insulin act through other pathway.\n",
            "\n",
            "discussion\n",
            "the anabolic effect of insulin on muscle protein synthesis is acutely sensitive to amino acids  <cit> . studies on muscle protein degradation have suggested that amino acids also influence insulin action but few studies have investigated the specific cellular pathway involved  <cit> . the atp-dependent ubiquitin-proteasome pathway is the most likely target for control by insulin and amino acids since proteasome-dependent-proteolytic activity and expression of components of this pathway are stimulated by starvation and inhibited by refeeding and insulin  <cit> . however, a previous study of protein degradation in c2c <dig> myotubes could only find evidence of alterations in lysosome-dependent proteolysis in response to complete starvation of amino acids or leucine from the media  <cit> . we therefore carried out a study to examine the effect of more physiological alterations in amino acid concentration on the effect of insulin on the ubiquitin-proteasome pathway in c2c <dig> myotubes.\n",
            "\n",
            "this study is the first to report that the amino acids alone, or additively with insulin, decrease muscle protein degradation by acting on the ubiquitin dependent proteasome pathway. we also report that varying the concentration of leucine whilst maintaining all other amino acids at a low level, does not mimic the effect of varying all amino acid concentrations suggesting that the response to amino acids is more than a response to alterations in leucine as has been previously suggested  <cit> .\n",
            "\n",
            "increasing the media amino acid concentration from  <dig>  × physiological levels to  <dig>  × pc aa decreased myotube proteolysis as assayed by the release of protein-bound labelled tyrosine into the medium, but these effects were abolished by addition of the proteasome inhibitor mg <dig>  furthermore, the expression of both the 14-kda e <dig> and c <dig> sub unit of  <dig> s proteasome was significantly increased during incubations with the sub-physiological amino acid incubation medium. the effect of amino acids was additive with insulin: maximal stimulation of protein degradation and expression of 14-kda e <dig> and c <dig> were observed in the absence of insulin and at  <dig>  × amino acids. addition of insulin in the presence of amino acids strongly inhibited proteolysis and expression of 14-kda e <dig> and c <dig>  depletion or supplementation of media leucine concentration influenced proteolysis but only in the presence of insulin. these results therefore strongly argue that insulin and amino acids interact to suppress myotube proteolysis by acting on the ubiquitin-proteasome pathway. the tyrosine release assay measures total myotube protein degradation and is therefore more representative of muscle protein degradation and is also able to determine small differences in turnover that are not possible with the measurement of specific proteins or their rate of ubiquitlylation.\n",
            "\n",
            "a previous study in c2c <dig> cells  <cit>  was unable to find any effects of amino acid or leucine starvation in the media on proteasome-dependent proteolysis or expression of components of this pathway. our current study differed from the previous study in two ways: firstly, we examined the effect of physiological changes in amino acid concentrations by decreasing to  <dig>  × physiological levels or in the case of leucine, altering the concentrations whilst maintaining the others at  <dig>  × physiological levels. secondly, the effects of amino acids were examined alone or in combination with insulin. others have reported the involvement of both lysosomal and ubiquitin-proteasome pathways during incubation of rat skeletal muscle in vitro in the presence of  <dig> mm leucine  <cit> . this suggests the possibility that the immediate cellular response towards amino acid withdrawal is autophagy either through sequestration of protein substrates to lysosomes or through modification of mrna levels for lysosomal proteases, but is followed by longer term activation of ubiquitin-proteasome pathway. the activation of lysosomal pathway in response to total amino acid deprivation as reported by the previous study is more likely due to cell stress  <cit> .\n",
            "\n",
            "in vitro studies using l <dig> or human muscle cells have confirmed the inhibitory effects of insulin on muscle protein degradation  <cit> . furthermore the mechanism of insulin action appears to be mediated by a decrease in the activity of the ubiquitin-proteasome pathway  <cit> . both in vitro  <cit>  and in vivo  <cit>  studies have demonstrated that insulin down regulates the expression of 14-kda e <dig> ubiquitin conjugating enzyme and ubiquitin, respectively. the present study demonstrates that the inhibitory action of insulin on expression of components of the proteasome pathway is decreased when either amino acids or leucine concentrations in the media are decreased to  <dig>  × physiological concentrations. in vivo, the starvation-induced increase in muscle proteasome-dependent proteolysis is only slowly reversed over  <dig> h of refeeding rats and acute infusion of amino acids over  <dig> h fail to alter the expression of components of the proteasome pathway  <cit> . a 6-hour hyperinsulinaemic, hyper-aminoacidaemic infusion study significantly reduced the mrna expression for ubiquitin in fast twitch and mixed skeletal muscle  <cit> . these data combined with our own suggest that under normal fed conditions insulin and amino acids act together to suppress the ubiquitin-proteasome pathway and that the fasting-induced increase in muscle protein degradation is due to lowered insulin and amino acid circulating concentrations resulting in enhanced expression of members of the proteasome pathway.\n",
            "\n",
            "numerous studies have demonstrated that insulin and amino acids regulate muscle protein synthesis and hepatic autophagy through the mtor  <cit> . inhibition of the mtor pathway by rapamycin abolishes the effect of leucine media starvation on autophagic proteolysis in c2c <dig> myotubes  <cit> . here, we demonstrate that rapamycin blocked the inhibitory effects of increasing amino acid concentration from  <dig>  to  <dig>  × pc aa, on protein degradation, but did not block the inhibitory effect of either leucine or insulin. in agreement with the present results, previous studies have demonstrated that leucine suppresses proteolysis in skeletal muscle and hepatocytes by an mtor independent mechanism <cit> . since our observations demonstrated that amino acid mixtures suppressed proteolysis in an mtor dependent manner, this suggests that the mtor pathway is influenced by amino acids other than the branched chain amino acid group. it has been proposed that glutamate and glutamine have suppressive effects on proteolysis in muscle  <cit>  and studies in other cell types have demonstrated that these amino acids influence the mtor signalling pathway  <cit> . phoshorylation targets for mtor have been proposed to be s <dig> kinase and 4e-bp <dig> as well as the phosphorylation dependent control of the degradation of many f-box proteins. phosphorylation of s <dig> kinase by insulin greatly depends on amino acid availability  <cit>  and leucine alone has been reported to induce s <dig> activation  <cit> . we were also able to demonstrate that rapamycin inhibited amino acid or insulin-dependent p <dig> s <dig> kinase phosphorylation. these changes were paralleled by a consistent trend for an increase in mrna for 14-kda e <dig> enzyme. the expression of several muscle-specific e <dig> ubiquitin ligases is consistently increased in conditions causing muscle wasting and mouse knock out studies have demonstrated that these are required for muscle atrophy  <cit> . insulin and insulin-like growth factor- <dig> acting through the pi3-k/akt pathway, suppress the expression of e <dig> ligases, murf <dig> and mafbx/atrogin- <dig>  <cit> . taken together, the present study argues that amino acids act through mtor to regulate myofibrillar proteasome-dependent proteolysis but further studies are necessary to establish the down stream effector pathways responsible for changes in the expression of components of the proteasome complex.\n",
            "\n",
            "CONCLUSIONS\n",
            "in conclusion we provide evidence that depletion of a mixture of amino acid in the media of c2c <dig> myotubes increases proteasome-dependent-proteolysis and the expression of the components of the ubiquitin-proteasome pathway. theses effects of amino acids were additive with those of insulin, not simply related to changes in leucine, and appear to be mediated by the mtor pathway. further studies are required to establish the exact signalling pathways by which amino acids alter the activity of the proteasome pathway to provide pharmacological targets to combat muscle wasting which is associated with various pathological states, such as prolonged starvation, sepsis, cancer cachexia and diabetes.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study we tested the hypothesis that amino acids and insulin act together to suppress the activity of the ubiquitin-proteasome pathway in c2c <dig> myotubes, via the ubiquitin-proteasome pathway.\n",
            "in this study we tested the hypothesis that amino acids and insulin act together to regulate the activity of the ubiquitin-proteasome pathway in c2c <dig> myotubes, via the ubiquitin-proteasome pathway.\n",
            "in this study we tested the hypothesis that amino acids and insulin act together to regulate the activity of the ubiquitin-proteasome pathway in c2c <dig> myotubes, via the ubiquitin-proteasome pathway.\n",
            "in this study we tested the hypothesis that amino acids and insulin act together to regulate the activity of the ubiquitin-proteasome pathway in c2c <dig> myotubes, via the ubiquitin-proteasome pathway.\n",
            "in this study we demonstrate that amino acids and insulin act together to suppress the activity of the ubiquitin-proteasome pathway in c2c <dig> myotubes, via the ubiquitin-proteasome pathway\n",
            "62\n",
            "[{'summary_text': 'a sequential structure prediction algorithm, based on the most successful free modelling method of our time, rosetta, was developed and used to predict protein structure, first starting from the nitrogen terminus and then starting from the carbon terminus.\\nthe results show that in general saint is generally more accurate when predicting protein structure than reverse saint, and that in general saint is more successful when folding from the c-terminus to the n-terminus, and more so when folding from the n-terminus to the n-terminus.\\nthe sequential algorithm was adapted to predict the structure of proteins produced in the reverse direction, from the c-terminus to the n-terminus.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 4675 to 5120 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "the purpose of this paper is to investigate whether directionality of synthesis can have an impact on the accuracy of protein structure prediction. in order to do this a sequential structure prediction algorithm, based on the most successful free modelling method of our time, rosetta, was developed and used to predict structure, first starting from the nitrogen terminus and then starting from the carbon terminus. free modelling protein structure prediction methodology has improved in recent years, but is still not accurate enough to be considered satisfactory . given this noisy nature of current free modelling stucture prediction techniques, the pairwise comparison design used here appears to be required; it succeeded in detecting a consistent directionality effect. we begin, however, by summarizing the area.\n",
            "\n",
            "almost fifty years ago anfinsen et al.  <cit>  showed that denatured small globular proteins could refold to their native state. on the other hand, experimentalists have known for many years that cotranslation can play an important role in protein folding  <cit> . polypeptides are synthesized sequentially, and translation can occur at variable rates according to codon speed  <cit> . in escherichia coli, for example, translation can occur in the order of  <dig>  s/codon  <cit> . on the other hand, it has been shown that helices and sheets fold in the low millisecond scale  <cit> . therefore, some proteins fold faster than they elongate, and it is reasonable to assume that nascent chains can adopt secondary or tertiary structures cotranslationally. experimental evidence for cotranslational folding dates back to the 1960s with a study on cotranslation in vivo reporting that ribosome-bound β-galactosidase was showing enzymic activity  <cit> . more recently it has been shown that the semliki forest virus protein , which contains a protease domain that folds to autocatalytically cleave the protein from a larger polyprotein precursor, gains its enzymic activity before complete synthesis of the polyprotein  <cit> . moreover, the rapid cotranslational folding of sfvp does not require additional cellular components  <cit> .\n",
            "\n",
            "in addition to enzymatic activity whilst still bound to the ribosome, intermediate stages of cotranslational folding may have native-like structures. various length α-globins have been shown to have specific heme binding activity on several truncated ribosome-bound nascent chains. the shortest of these contained only the first  <dig> residues , demonstrating that the nascent chain has native-like structure  <cit> . nmr studies of nascent chains containing tandem ig domains and still attached to the ribosome revealed that the n-terminus domain folds to its native state while the c-terminus domain is largely unfolded and flexible  <cit> . recent molecular dynamics simulations also conclude that small peptides may adopt a conformation that is similar to the one adopted in full proteins  <cit> . the discovery of the formation of disulphide bonds in nascent immunoglobulin peptides also confirms the ability of proteins to begin to fold whilst they are being synthesized  <cit> .\n",
            "\n",
            "as well as adopting native-like conformations while still attached to the ribosome, there is evidence that peptides can begin to fold whilst still in the ribosomal exit tunnel. analysis of the ribosomal exit tunnel reveals that peptides can traverse the tunnel in an α-helical conformation  <cit> , but that at no point is the tunnel big enough to accommodate structures larger than α-helices  <cit> . peptides are not restricted to an α-helix, however, and may adopt more extended conformations  <cit> . analysis of the exit tunnel has also shown that the tunnel can entropically stabilize α-helical conformations as they pass through  <cit> .\n",
            "\n",
            "the rate of in vitro refolding has often been observed to be slower than the corresponding rate in vivo  <cit> . cotranslation has been studied in the bacterial luciferase αβ heterodimer, and the formation of the heterodimer is faster when the β monomer is translated in the presence of the folded α monomer than when the β monomer is refolded from a denatured state  <cit> . this shows that, under cotranslational folding, the β monomer is able to obtain a conformation that is more receptive to the formation of the dimer, thus avoiding kinetic traps associated with refolding from a denatured state  <cit> . native-like structure has also been observed in cotranslationally folding monomeric firefly luciferase; again, cotranslational and in vitro folding pathways appear to be different, with cotranslational folding being faster  <cit> . cotranslational folding in p <dig> tailspike protein has been shown to guide the peptide away from aggregation-prone conformations that are frequently encountered when refolding in vitro, leading to the hypothesis that cotranslational folding could be an efficient strategy for the folding of β-sheet topologies, and for large, multidomain proteins in general  <cit> . one possible explanation for this is that the peptide begins to fold while still attached to the ribosome  <cit> . another possible explanation is the existence of additional folding machinery contained in the cell; however, only approximately 20% of proteins associate, for example, with chaperones  <cit> . the removal of major chaperones, such as dnak and hsp <dig>  in e. coli has no adverse effect on cell growth or viability  <cit> . this suggests that chaperones alone cannot account for the higher folding rates observed in vivo.\n",
            "\n",
            "complementing these experimental findings, computational models of cotranslational folding have also been explored, an early, incidental, use of this idea appearing in  <cit> . simple computational models of protein folding incorporating cotranslation demonstrate that such folding favours local contacts in intermediate and final folds  <cit> . more recently the effect of energy barriers on simple cotranslational models was studied, and it was found that the ground state of proteins folded sequentially was not necessarily the one of lowest energy  <cit> . computational models have provided evidence that nascent chains may adopt partial structures similar to the corresponding parts of the complete protein  <cit> . other lattice studies present a differing view of cotranslation where nascent peptides can remain largely unstructured until the final stages of synthesis   <cit> . this finding is dependent on the involvement of the c-terminal in tertiary interactions, and may not be applicable to all proteins. there is also evidence arising from lattice models that cotranslational folding pathways and refolding pathways are different  <cit> . computational simulations of real proteins folding cotranslationally compared to refolding from a denatured state show mixed results. chymotrypsin inhibitor  <dig>  and barnase were shown to fold mostly posttranslationally, with intermediates similar to those observed in refolding  <cit> . an alternative computational, cotranslational approach using dynamic optimisation in  <cit>  found that major elements of the ci <dig> tertiary structure only form when the amino acid string is fully translated. for sfvp, which is known to fold cotranslationally  <cit> , different pathways were taken during synthesis to those taken when folding from a denatured state  <cit> . a further promising approach is found in  <cit> . pathways which minimize the difficulty of folding to the native state  are found; results indicate that earlier folding is more likely around the n-terminus than the c-terminus, so pointing to an asymmetry of the folding process that is confirmed in the current work.\n",
            "\n",
            "finally, there is also evidence of cotranslational protein folding that arises from numerical summaries of known protein structures. an analysis of structures in the protein data bank  found that residues are, in general, closer to previously synthesized residues than those synthesized later, and that the n-terminal region was more compact than the c-terminal region  <cit> . it was argued that this provided evidence of cotranslational folding, however, these findings were contradicted by a later analysis of a larger set of proteins  <cit> . in the second study it was observed that the c-terminals were more compact and contained greater numbers of local contacts than n-terminals. further analysis that considered topological accessibility  found this to be more evident towards the n-terminus in the α/β class of proteins  <cit> . in a similar vein, deane et al.  <cit>  developed a measure of previous contacts which assesses the extent to which the chain forms contacts with previously extruded residues. they also found that the α/β class and ancient folds  <cit>  exhibited such evidence of cotranslation.\n",
            "\n",
            "to date, protein structure prediction methods do not incorporate cotranslational effects. this paper describes such an algorithm and evaluates its performance. this evaluation reveals that, in more than 94% of cases, a sequential algorithm that follows the sense of translation, that is, from n-terminus to c-terminus, is more accurate than an algorithm that follows the reverse sense, from c-terminus to n-terminus. the success of the sequential algorithm is greater the more the target shows evidence of cotranslational folding. it is also found that a sequential algorithm can match, and on occasion better , the performance of a leading non-sequential protein structure prediction algorithm, namely rosetta.\n",
            "\n",
            "methods\n",
            "structure prediction algorithms\n",
            "a sequential algorithm  was developed and used to predict the structure of a number of proteins. this algorithm uses the rosetta program  <cit>  , extending it to incorporate cotranslational aspects of protein folding. to investigate the importance of following the direction of translation, the sequential algorithm was adapted to predict the structure of proteins produced in the reverse direction, from the c-terminus to the n-terminus. predictions from the sequential and reverse sequential algorithms were compared and they in turn compared to predictions made using an unmodified version of rosetta. these algorithms are now described.\n",
            "\n",
            "sequential algorithm\n",
            "saint extends the peptide by a nine residue fragment at each iteration, starting with the n-terminus. each fragment is added in a fully extended conformation . the final fragment may contain fewer than nine residues; it will contain as many residues as are required to complete the full protein chain. at each extension the peptide is allowed to fold and the conformation reached is used as the starting structure for the next extension, with rosetta ab initio used to perform the structure predictions at each stage. in order to make comparisons between the sequential and non-sequential algorithms fair, each uses the same total number of cycles. for the sequential algorithm these cycles were distributed evenly amongst each extension of the peptide with the number of cycles calculated as follows. if b is a base number of cycles and l is the protein length then the total number of cycles t is b and the number of extrusions e is ⌈l/9⌉. this results in n = ⌊t/e⌋ cycles for the first e -  <dig> extrusions and t - n cycles for the final extrusion.\n",
            "\n",
            "reverse sequential algorithm\n",
            "the reverse sequential algorithm is the same as the sequential algorithm. it differs only in that the peptide is extended from the c-terminus to the n-terminus.\n",
            "\n",
            "non-sequential algorithm\n",
            "in non-sequential folding a protein is folded from a fully extended state. the rosetta ab initio algorithm is employed for this process, using insertion from a library of fragments to build decoys . this has proved a successful technique for protein structure prediction in recent years  <cit> . rosetta can select fragments from the target, so the algorithm as used here is not strictly ab initio. the number of cycles  used by rosetta varies with protein length in this study. a base number of  <dig>  cycles was used for a protein of  <dig> residues, and this number increased proportionately; for example, for a protein with  <dig> residues the number of cycles is increased by a factor of  <dig> . this is reasonable as in the cell longer proteins take more time to be synthesized, and thus have more time to explore conformational space before synthesis is completed.\n",
            "\n",
            "selection of targets\n",
            "in deane et al.  <cit>  a measure was developed, an average logarithmic ratio , which assesses the extent of previous contacts within a peptide chain; proteins with positive alr are expected to be those for which the cotranslational aspect of folding has a substantial impact, whilst proteins with negative alr are expected to be those for which cotranslation has lesser impact. two sets of targets were created from a pisces <cit>  data set . the first set contained protein chains with an alr value of  <dig>  or greater , and the second contained chains with an alr of - <dig>  or less ; these two sets are referred to as the positive and negative sets respectively. for each protein in the two sets,  <dig> decoys were generated with each of the algorithms described above . gdt_ts values  <cit>  were calculated for each of the resulting predictions. gdt_ts is defined as /, where ni is the number of corresponding residues within iÅ and n is the total number of residues. it measures the closeness of corresponding residues in known and predicted structures, more heavily weighting closer pairs. it is helpful to see it in non-cumulative form as  where .\n",
            "\n",
            "larger sample size\n",
            "to establish whether the sample size  has an effect on the results, two proteins were subjected to a larger sampling. an additional  <dig>  decoys were generated for the flig c-terminal domain of thermotoga maritima  and also for 1ji4a, using the saint algorithm.\n",
            "\n",
            "variability in peptide termini\n",
            "as the differences between mean gdt_ts scores for saint and reverse saint, for a given protein, prove to be generally small, additional tests were conducted to ascertain whether terminus loop regions could be causing the observed effects. the termini of proteins are often unstructured, and their structure can be highly variable and difficult to predict. small mistakes in the terminus regions could lead to the small differences observed between the mean gdt_ts scores.\n",
            "\n",
            "the first n-terminus and last c-terminus secondary structure elements were identified in the experimental structure for each protein, and the termini up to the identified secondary structure element of the corresponding predicted model with the highest gdt_ts were removed. a secondary structure element was defined as a run of four residues with identical secondary structure assignment. secondary structure was assigned from the experimentally determined structure with dssp. in addition to these conditions the n-terminus and c-terminus secondary structure element had to be separated by at least five residues. gdt_ts scores were recalculated and counts taken of how often saint outperformed reverse saint and how often saint outperformed rosetta.\n",
            "\n",
            "clash analysis\n",
            "a possible reason for better performance of saint was conjectured to be that extrusion from the nitrogen terminus produces fewer steric clashes than does extrusion from the carbon terminus. in order to investigate this, ten protein sequences were selected on the basis of their mean gdt_ts scores: four in which saint performed better, three in which reverse saint performed better, and three in which saint and reverse saint performed comparably. for each protein, two of the  <dig> models generated were selected for each of saint and reverse saint. the extent of steric clashes in conformations following folding, for five extruded lengths , were assessed using molprobity  <cit> , a web server that calculates a \"clashscore\", equal to the number of steric overlaps that are greater than  <dig>  Å per  <dig> atoms. nine residues in fully extended conformation were then added at the c-terminus  or the n-terminus  to produce strings of length  <dig>   <dig>   <dig>   <dig>  and  <dig> and these checked again for steric clashes. for each of the five positions, the clashscore before the addition of nine residues was subtracted from the clashscore after the addition of the 9-mer fragment. an average of the differences in clashscores, across all five lengths, was taken for each protein sequence and each algorithm.\n",
            "\n",
            "the importance of sense\n",
            "to investigate why saint might perform consistently better than reverse saint, measures of secondary structure prediction quality were developed. for a given decoy, structural alignments for every overlapping fragment of  <dig> residues against the experimental structure were obtained, and the average cα-cα distance of the alignment was assigned to the fragment's center residue . these residue-assigned distance measures were averaged across all residues in α-helices in the decoy  and these in turn averaged over all  <dig> decoys. this was done for both the forward and reverse decoy sets. finally, the forward helical prediction quality measure was subtracted from the reverse helical prediction quality measure. the same process was followed for β-strands. if directionality is not important in folding we would expect the accuracy of helical or strand predictions to be similar regardless of the direction of synthesis, resulting in the difference calculated above being zero. a positive difference would indicate that forward predictions were more accurate than reverse predictions while negative differences would indicate that reverse predictions were more accurate. one of the proteins in the positive set  and four in the negative set  contained no β-strand residues and, therefore, were not considered in the analysis.\n",
            "\n",
            "RESULTS\n",
            "the emerging partial conformations produced by saint for sequence 1qc7a are shown in figure  <dig>  using the most successful decoy. the six helices are seen to progressively take shape as the chain is extruded, with early conformations largely preserved.\n",
            "\n",
            "results for saint, reverse saint and rosetta for each of the proteins in the positive set  and negative set  are summarized in table  <dig> and table  <dig> respectively. the mean performance and best models produced by saint show that it predicts structures better than reverse saint in the majority of cases . for example, saint yielded a higher mean gdt_ts than reverse saint for  <dig> of the  <dig> proteins with positive alr and equally, for  <dig> of the  <dig> proteins with negative alr.\n",
            "\n",
            "the mean gdt_ts and maximum gdt_ts for all  <dig> decoys produced for each combination of protein and algorithm is shown. for both the mean and maximum gdt_ts the highest gdt_ts is shown in bold while the lowest is shown in italics.\n",
            "\n",
            "the mean gdt_ts and maximum gdt_ts for all  <dig> decoys produced for each combination of protein and algorithm is shown. for both the mean and maximum gdt_ts the highest gdt_ts is shown in bold while the lowest is shown in italics.\n",
            "\n",
            "the table shows the number of times an algorithm in a pair outperformed the other, separately for the positive and negative sets. both mean gdt_ts and maximum gdt_ts are used as measures of performance. asterisks indicate binomial test p-values where * is <  <dig> , ** is <  <dig> , *** is <  <dig> .\n",
            "\n",
            "plots of the mean scores for saint, reverse saint and rosetta for the positive set are given in figure 2a, with proteins ordered from smallest to largest mean saint gdt_ts score. corresponding plots for the negative set are given in figure 3a. the consistent superiority of saint over reverse saint is evident, with the difference being slightly greater for the positive set. the largest such difference seen in all the data is  <dig> %, observed between the means of saint and reverse saint for 3ezma , and representing an increase in gdt_ts from  <dig> % to  <dig> %. mean performances of saint and rosetta indicate that rosetta outperforms saint in both the positive  and negative  sets. the difference is greater for the negative set .\n",
            "\n",
            "plots of the maximum scores for saint, reverse saint and rosetta for the positive set are given in figure 2b, with proteins ordered from smallest to largest maximum saint gdt_ts score. corresponding plots for the negative set are shown in figure 3b. when considering best performance, saint is again superior to reverse saint, and more so in the positive set. rosetta is no longer superior when best performance is considered; saint outperforms rosetta, for example, in  <dig> of the  <dig> proteins in the positive set. the most successful saint prediction in the positive set was found for 3vuba. it is shown superposed on the native conformation in figure  <dig>  together with superpositions of the best reverse saint and rosetta predictions on the native conformation. saint captures the structure better than either reverse saint or rosetta.\n",
            "\n",
            "a gdt_ts value of 30% or above is generally considered to ensure that a reasonable prediction is found  <cit> ; a scan of table  <dig> indicates that roughly one half  of the best saint predictions are satisfactory, and similarly for rosetta .\n",
            "\n",
            "larger sample size\n",
            "summaries of the distribution of gdt_ts scores indicate that the size of the decoy sets used  does not significantly influence their values . when repeated with 1ji4a, similar results were produced .\n",
            "\n",
            "variability in peptide termini\n",
            "the results of this test indicate that the differences in gdt_ts observed are not due to variability in the terminus regions of the peptides .\n",
            "\n",
            "among the  <dig> decoys produced for each protein with alr ≥  <dig>  by each of saint, reverse saint, and rosetta the best model  was found . each of these selected models was then altered by chopping off the first n-terminus and last c-terminus secondary structure elements identified in its native structure. gdt_ts scores were then recalculated for each algorithm and are displayed below. the highest gdt_ts is shown in bold while the lowest is shown in italics. sample size was reduced to  <dig> as no secondary structural element at least five residues in length was found at either terminal of the protein chain 2j01vpdb2j01v.\n",
            "\n",
            "among the  <dig> decoys produced for each protein with alr ≤ - <dig>  by each of saint, reverse saint, and rosetta the best model  was found . each of these selected models was then altered by chopping off the first n-terminus and last c-terminus secondary structure elements identified in its native structure. gdt_ts scores were then recalculated for each algorithm and are displayed below. the highest gdt_ts is shown in bold while the lowest is shown in italics.\n",
            "\n",
            "clash analysis\n",
            "the results are shown in table  <dig>  four of the ten protein conformations examined have higher steric clashscores for saint than reverse saint. the steric clashscore appears not to be influenced by its mean gdt_ts score, evidenced by two  out of the four proteins with higher mean gdt_ts scores for saint having greater steric clashscores than reverse saint. steric clashes produced by saint and reverse saint are generally comparable, so providing no evidence that fewer steric clashes are the reason for the better performance of saint.\n",
            "\n",
            "mean difference in clashscores for each protein sequence; the larger the mean difference, the more clashes created by the extrusion. the first four proteins in the table have higher mean gdt_ts scores for saint, the next three have higher mean gdt_ts scores for reverse saint and the remaining three have comparable mean gdt_ts scores for saint and reverse saint. there is no evidence that saint creates more clashes.\n",
            "\n",
            "the importance of sense\n",
            "the differences obtained from both the positive and negative sets are shown in figure  <dig>  these results show that for both types of secondary structure saint is generally producing better predictions, but that the difference is more pronounced for strand residues. in  <dig> of the  <dig> proteins  in the positive set the difference between forward and reverse folding is greater for strands than for helices  having a β-strand difference more than twice the α-helix difference). similarly, in  <dig> of the  <dig> proteins  in the negative set the difference between forward and reverse folding is greater for strands than for helices  having a β-strand difference more than twice the α-helix difference). these results indicate that in general saint is more accurate when predicting strands than is reverse saint. these differences are small, but they would account for the differences observed in the results.\n",
            "\n",
            "discussion\n",
            "a consistent difference in prediction accuracy was seen between saint and reverse saint. saint is markedly superior to reverse saint, and slightly more so for proteins with positive alr values. when looking in detail at saint and reverse saint, the differences observed are most likely due to the detrimental effect on strand prediction observed when elongating a peptide from the c-terminus to the n-terminus. saint produced decoys with a higher mean gdt_ts than reverse saint for more than 94% of proteins in both the positive and negative protein sets. the differences between mean gdt_ts scores for saint and reverse saint decoys were also bigger than those between saint and rosetta decoys. if directionality played no part in the folding process it would be expected that there would be no difference in the predictive accuracy of extrusions from the n-terminus to c-terminus and extrusion from c-terminus to the n-terminus. three possible explanations for these results are outlined below.\n",
            "\n",
            "peptides, when extruded from the ribosome, start at the n-terminus. for this reason, fragments near the n-terminus are less influenced in their folding by the remainder of the peptide, since this has yet to emerge from the ribosome. on the other hand, fragments towards the c-terminus must fold in the presence of the bulk of the peptide. thus the conformation assumed by the early fragment is a local choice, in that it depends largely on the amino acid sequence of the fragment. the conformation reached by a later fragment is determined by more than its amino acid sequence, in that it also depends on surrounding structure. this behaviour is mimicked by saint but not by reverse saint, so providing an explanation for the consistently better predictive accuracy of saint.\n",
            "\n",
            "a second explanation arises from the way that the two algorithms allocate fragment insertions. at any stage, due to the constraints of rosetta, fragment insertions are made uniformly across the currently extruded peptide length. the upshot is that more fragment insertions are attempted at the n-terminus than the c-terminus for saint while the opposite is true for reverse saint. should it be the case that the n-terminus of the peptide is harder to predict than the c-terminus, saint would be more successful than reverse saint since saint puts in effort where it is needed. due to the reasons stated above, however, we expect the n-terminus to be more easily predicted than the c-terminus.\n",
            "\n",
            "a third possibility is that rosetta itself has some inherent directionality, so favouring saint over reverse saint. a study of rosetta, however, provides no indication of such a directional bias.\n",
            "\n",
            "a strong correlation between mean gdt_ts and chain length is seen for both the positive and negative sets and for all three algorithms: as the chain length increases the gdt_ts decreases. 1oaaa is the only target over  <dig> residues in length that produced a set of decoys with mean gdt_ts greater than 20%, indicating that the versions of the algorithms employed in this study are not sufficient to accurately predict the structure of chains with more than  <dig> residues . excluding this data from the analysis, however, makes no difference to the overall findings.\n",
            "\n",
            "given that saint outperforms reverse saint it might be expected that saint would also outperform rosetta, rosetta being, in some senses, midway between the two. in best performance, arguably more important than mean performance, there is weak evidence that saint does outperfom rosetta; for the positive set saint outperfoms rosetta in  <dig> out of  <dig> instances  and for the negative set saint outperforms rosetta in  <dig> out of  <dig> instances . an explanation why this remains weak at this stage is that saint remains crude, barely exploiting spatial and temporal advantages which may be available in cotranslational folding; we have simply used an iterative version of rosetta. for example, at each extrusion, fragment insertions are chosen uniformly along the extruded peptide, whereas use of an insertion location distribution skewed towards the carbon terminus might be more realistic. to its credit, however, the saint versus reverse saint investigation exploits the power of a \"paired comparison\" design more effectively than does the saint versus rosetta investigation, in that it contrasts opposites and so is more likely to reveal an effect.\n",
            "\n",
            "CONCLUSIONS\n",
            "this study has presented an algorithm that builds cotranslation into protein structure prediction. to assess the importance of the direction of translation the sequential algorithm was compared to a reverse sequential algorithm where the protein was produced from the c-terminus to n-terminus. two sets of proteins were chosen: one where the residues have, on average, more contacts with previous residues than successive residues and the other where the residues have, on average, more contacts with successive residues than previous residues. the performance of the sequential algorithm for protein structure prediction was also compared with rosetta, which folds from a fully elongated chain.\n",
            "\n",
            "when saint was compared to reverse saint a very pronounced difference was observed. when mean gdt_ts was used as the performance measure saint outperformed reverse saint for over 94% of targets from both the positive and negative sets. these figures were still high when the maximum gdt_ts was used as the performance measure, with saint outperforming reverse saint in over 91% of targets from the positive set and over 73% of targets from the negative set.\n",
            "\n",
            "the results show that rosetta produces decoy sets with higher mean gdt_ts scores than saint for both the positive and negative protein sets, but that this superiority of rosetta is not seen when the models with the highest gdt_ts scores are compared. if it were possible to always select the most accurate structure from the set of decoys then saint would, overall, produce a better prediction than rosetta. the selection of the best decoy from a set, however, is a separate problem that is not addressed in this study. while rosetta is producing decoy sets with higher mean gdt_ts scores than saint, examination of the differences between the means shows that the difference is always small. only on one occasion does a rosetta decoy set have a mean gdt_ts greater than 2% above the corresponding saint decoy set . it has been established that the size of the decoy set and flexibility of peptide terminus residues do not affect the distribution of gdt_ts scores.\n",
            "\n",
            "the sequential algorithm described in this study is in its earliest stages of development. future work will include investigation of the effect of translation speed, allowing extruded segments to have variable length and the number of fragment insertion attempts at each iteration to vary. improvements should also include incorporation of spatial restrictions to simulate the constraint of the ribosome tunnel.\n",
            "\n",
            "authors' contributions\n",
            "conceived and designed the experiments: fpeh, grw, cmd and jje. performed the experiments: jje, fpeh and ss. analyzed the data: jje, grw, fpeh and cmd and ss. wrote the paper: jje, fpeh, grw, ss and cmd. all authors read and approved the final manuscript.\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "a sequential structure prediction algorithm, based on the most successful free modelling method of our time, rosetta, was developed and used to predict protein structure, first starting from the nitrogen terminus and then starting from the carbon terminus.\n",
            "the results show that in general saint is generally more accurate when predicting protein structure than reverse saint, and that in general saint is more successful when folding from the c-terminus to the n-terminus, and more so when folding from the n-terminus to the n-terminus.\n",
            "the sequential algorithm was adapted to predict the structure of proteins produced in the reverse direction, from the c-terminus to the n-terminus.\n",
            "\n",
            "63\n",
            "[{'summary_text': 'the solubility of bacterial α/β-hydrolases is predicted to be more soluble in e. coli than eukaryotic α/β-hydrolases.\\nthe solubility of bacterial α/β-hydrolases is predicted to be more soluble in e. coli than eukaryotic α/β-hydrolases.\\nthe solubility of bacterial α/β-hydrolases is predicted to be more soluble than both small and large eukaryotic α/β-hydrolases.\\nthese results suggest that, in problematic cases where soluble hydrolases are very rare, it makes sense to screen for large hydrolases in coding sequences of metazoa, small hydrolases should be searched in bacterial or archaean and medium- sized hydrolases in actinobacterial or ascomycetic coding sequences.\\nthe solubility of bacterial α/β-hydrolases is predicted to be more than  <dig>   <dig>  and  <dig> % of the hydrolases predicted to be soluble in e. coli.\\nthe solubility of bacterial α/β-hydrolases is predicted to be more than  <dig> '}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 6409 to 7168 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "it was observed that screening of libraries derived by expression cloning for new gene products with a given catalytic activity can be limited by formation of inclusion bodies. in these cases it would be desirable to construct libraries which a higher fraction of soluble gene products. enrichment of soluble proteins could be achieved either by limiting the screening to coding sequences from phylogenetic groups with mainly soluble proteins or by the use of degenerated primers which are specific for homologous families with mainly soluble proteins.\n",
            "\n",
            "alternatively solubility can be improved by the use of fusion proteins  <cit>  like nusa, mbp, thioredoxin, grpe, bfr, gst, dsba  <cit> , the n- terminal domain of if <dig>  <cit>  or phage coat protein iii  <cit> . in addition fusion proteins allow affinity chromatography and thus simplify purification  <cit> . however in some cases the fusion partner has to be removed after protein purification  <cit>  and therefore constitutes and additional step in protein preparation. other strategies try to improve in vivo solubility of a recombinant protein by protein engineering using strategies like molecular evolution  <cit>  or rational protein design. examples for protein design are the insertion of positively charged residues into hydrophobic patches on the surface  <cit> , exchange of phenylalanines by serines  <cit>  or asparagine residues by aspartic acids  <cit> . it has been shown that single residues can have a major impact on solubility  <cit> . but also the expression system is important for in vivo solubility  <cit> . other factors are cultivation and disruption conditions  <cit> , rate of protein synthesis  <cit> , fermentation temperature  <cit>  and the amount of helper protein  <cit> . fusion proteins have already been successfully used in high throughput expression- studies to improve solubility  <cit> .\n",
            "\n",
            "however, engineering approaches are limited to specific proteins or growth conditions and are hardly applicable to high throughput expression. therefore, in projects where libraries are screened for activity, solubility is always an implicit criterion. previously harrison et al. introduced a two-parameter model based on an analysis of  <dig> proteins for which experimental data on solubility exists  <cit> . the model proposes that solubility of proteins in e. coli at physiological conditions depends mainly on protein charge and the relative number of turn-forming residues. interestingly, the model holds for a broad variety of proteins.\n",
            "\n",
            "we applied this model to a comprehensive analysis of the lipase engineering database   <cit>  which includes more than  <dig> α/β-hydrolases. α/β-hydrolases share the same fold but are highly diverse in sequence. they are ubiquitous and include cellular and secreted proteins from a wide range of organisms.\n",
            "\n",
            "the aim of this study was to find correlations between predicted solubility in e. coli and protein size or phylogenetic origin. homologous families and superfamilies were analysed for the predicted solubility of their members.\n",
            "\n",
            "RESULTS\n",
            "kingdoms of life\n",
            "according to harrison et al.  <cit>  the canonical variable cv-cv' predicts from the protein sequence whether a recombinantly expressed protein is soluble in the cytoplasm of e. coli  or will form inclusion bodies . for cv-cv' =  <dig> the probability of solubility is  <dig> . the probability of solubility or insolubility rises for higher absolutes of cv-cv'. an analysis of the lipase engineering database indicates that most of the hydrolases are predicted to be insoluble in e. coli, with a major peak at cv-cv' =  <dig>  and a minor peak at cv-cv' =  <dig>  . a separate analysis of hydrolases from eukaryotic and prokaryotic origin  demonstrates that these two peaks are formed predominantly by the hydrolases from each of the two kingdoms of life. the distribution of cv-cv' of bacterial hydrolases is characterized by an average of  <dig>  and a first quartile of - <dig> . because the first quartile indicates the minimum solubility of the  <dig> % most soluble hydrolases, more than  <dig> % of all bacterial hydrolases are predicted to be soluble. in contrast, for eukaryotic hydrolases the average of cv-cv' is  <dig> ; the first quartile is  <dig> . thus, only a small fraction of eukaryotic hydrolases is predicted to be soluble in e. coli. α/β-hydrolases from archaea were not investigated because they are only represented with  <dig> hydrolases.\n",
            "\n",
            "thus, in general, bacterial α/β-hydrolases are predicted to be more soluble in e. coli than eukaryotic α/β-hydrolases.\n",
            "\n",
            "protein size\n",
            "the family of α/β-hydrolases falls into three major groups of protein size: small , medium- sized  and large hydrolases  . large hydrolases are mainly from eukaryotic origin while small hydrolases are mainly from bacterial origin. correlating sequence length and predicted solubility in e. coli demonstrates that large hydrolases are predicted to be less soluble in e. coli than smaller ones . hydrolases predicted to be soluble in e. coli  have sequence lengths between  <dig> and  <dig> amino acids, most hydrolases of more than  <dig> amino acids are predicted to be insoluble in e. coli . a high fraction of bacterial and archaean small hydrolases is predicted to be soluble in e. coli, while eukaryotic small hydrolases are predicted to be mainly insoluble. the two outliers with cv-cv' < - <dig> and a sequence length larger than  <dig> are a lipase from staphylococcus xylosus  and cg <dig> from drosophila melanogaster . both are putative proteins and have not yet been expressed in e. coli. thus, there are two general observations:  small α/β-hydrolases are predicted to be more soluble in e. coli than large α/β-hydrolases and  eukaryotic α/β-hydrolases are larger than prokaryotic α/β-hydrolases.\n",
            "\n",
            "analysis of groups formed by protein size\n",
            "to distinguish the effects of hydrolase size and phylogenetic origin on solubility, small, medium- sized and large α/β-hydrolases were investigated separately, and hydrolases were grouped by phylogeny of their origin .\n",
            "\n",
            "small hydrolases have an average of cv-cv' of  <dig>  and a first quartile of - <dig> , thus more than  <dig> % of small hydrolases are predicted to be soluble in e. coli. all taxa have a positive average of cv-cv' , indicating that less than  <dig> % of the hydrolases in each taxon are predicted to be soluble. hydrolases from eukaryotic taxa are at average predicted to be highly insoluble. all prokaryotic taxa and plants have negative first quartiles of cv-cv', thus more than  <dig> % of their hydrolases are predicted to be soluble. all other eukaryotic taxa have positive first quartiles of cv-cv', thus most of their hydrolases are predicted to be insoluble. the taxa containing hydrolases with the highest predicted solubility average are from bacteria, the taxa containing hydrolases with the lowest predicted solubility average are from eukaryota.\n",
            "\n",
            "medium- sized hydrolases have an average of cv-cv' of  <dig>  and a first quartile of  <dig> . hydrolases of all taxa have a positive average of cv-cv', and thus are mainly predicted to be insoluble . only actinobacteria have a negative first quartile of cv-cv', indicating that more than  <dig> % of its hydrolases are predicted to be soluble, all other taxa have positive first quartiles of cv-cv'. the taxa containing hydrolases with the highest predicted solubility average are from bacteria and fungi. the taxa containing hydrolases with the lowest predicted solubility average are from metazoa and plants.\n",
            "\n",
            "large hydrolases have an average of cv-cv' of  <dig>  and a first quartile of  <dig> , they are in general predicted to be highly insoluble. all taxa have positive averages and first quartiles of cv-cv', and thus most of their hydrolases are predicted to be insoluble . the taxa containing hydrolases with the highest predicted solubility average are from metazoa, the taxa containing hydrolases with the lowest predicted solubility average are from bacteria and fungi.\n",
            "\n",
            "in general cv-cv' from bacterial hydrolases is much lower if the hydrolase is small, while metazoan hydrolases have a lower average and first quartile of cv-cv' if the hydrolase is large. though large metazoan hydrolases have a higher probability of solubility than small metazoan hydrolases, few large hydrolases are predicted to be soluble in e. coli . this is consistent with the result from table  <dig> that in the analysis of large hydrolases no taxon with a negative first quartile of cv-cv' could be found.\n",
            "\n",
            "thus there are several conclusions. large α/β-hydrolases from bacteria are predicted to be less soluble than smaller ones. small bacterial α/β-hydrolases are predicted to be more soluble than both small and large eukaryotic α/β-hydrolases. but large hydrolases from metazoa are predicted to be more soluble than large hydrolases from bacteria and small hydrolases from metazoa. so there seems to be a principal difference between metazoa and bacteria. fungi, especially ascomycetes, behave differently. their α/β-hydrolases with the highest predicted solubility are mainly medium- sized.\n",
            "\n",
            "analysis of solubility by genera\n",
            "of the  <dig> genera represented in the database most include only a few hydrolases. therefore the  <dig> genera with at least ten hydrolases were analyzed . most hydrolases of all eukaryotic taxa with the exception of oryza are predicted to be highly insoluble in e. coli.\n",
            "\n",
            "hydrolases from bacterial genera show a wide range of averages of predicted solubility. interestingly the bacterial genera with the highest and the lowest average of cv-cv'  are both from actinobacteria. the third large genus of actinobacteria, streptomyces, is predicted to include more than  <dig> % soluble hydrolases as it is shown by a negative first quartile.\n",
            "\n",
            "thus the averages of different genera of a taxon may differ completely in predicted solubility. therefore a strategy to identify α/β-hydrolases from actinobacteria that are soluble in e. coli should focus on proteins from rhodococcus and streptomyces, but not from mycobacterium.\n",
            "\n",
            "analysis of solubility by sequence similarity\n",
            "hydrolases with sequence similarity have been assigned to superfamilies which were analysed . superfamilies with mostly hydrolases of high predicted solubility contain mainly intracellular α/β-hydrolases from bacteria . the averages of cv-cv' of the superfamilies that contain almost exclusively bacterial hydrolases range from - <dig>  to  <dig> . metazoan, fungal and secreted hydrolases can mainly be found in superfamilies with averages of cv-cv' between  <dig>   and  <dig>   . in general, the average of cv-cv' depends on sequence length. however, the superfamily with the lowest sequence length in this table , bacillus lipase, has the lowest predicted solubility.\n",
            "\n",
            "proteins from cytosolic hydrolases, a large superfamily that contains mainly epoxide hydrolases and haloalkane dehalogenases, are in general predicted to be more soluble than proteins from most other superfamilies. the average of cv-cv' is  <dig>  and the first quartile is - <dig> . this superfamily was chosen for a more detailed analysis of homologous families . the cv-cv' averaged over hydrolases of each homologous family of this superfamily ranges from - <dig>  to  <dig> . this shows that there can be the prediction of families with mostly soluble and of families with mostly insoluble proteins in one superfamily. this observation is analogous to the previous observation with genera and higher taxa. as expected the homologous families with the highest predicted solubility average are dominated by bacterial hydrolases and the families with proteins of low predicted solubility rather contain eukaryotic hydrolases . the homologous family with the highest predicted solubility average, which is dominated by eukaryots, is the 'soluble plant epoxide hydrolases'. almost  <dig> % of its members are predicted to be soluble in e. coli.\n",
            "\n",
            "thus the averages of different homologous families of a superfamily may differ significantly in predicted solubility.\n",
            "\n",
            "discussion\n",
            "the solubility model\n",
            "the dataset of the statistical solubility model were  <dig> highly diverse proteins, for which solubility data exists, from a wide range of organisms  <cit> . the published prediction accuracy of the five parameter model was  <dig> % for soluble proteins and  <dig> % for insoluble proteins  <cit> , the overall accuracy for solubility prediction of proteins from the dataset was 88%  <cit> . it has been discovered that only two out of the five parameters are critical for distinguishing between soluble and insoluble proteins  <cit> . therefore, cv-cv', the indicator of predicted solubility, was derived by two terms: the total charge calculated by the relative numbers of arginines, lysines, aspartic acids and glutamic acids, and the relative number of turn forming residues calculated by the relative number of asparagines, glycines, prolines and serines. those two parameters show a level of significance of 100%  <cit> .\n",
            "\n",
            "most of the variation of cv-cv' among α/β-hydrolases is caused by the charge term. a high negative or positive charge results in the best predicted solubility. however, the pk value of a titratable group is highly dependent on the proteins structure. short- and long-range interactions can lead to pk shifts of more than two units, changing the total protein charge by more than five unit charges  <cit> .\n",
            "\n",
            "an evaluation of nine frequently used fusion proteins used to improve solubility in e. coli  <cit>  showed that seven proteins indeed are predicted to be soluble in e. coli . the exceptions are mbp which has a cv-cv' of  <dig> , and phage coat protein iii which has a cv-cv' of  <dig>  .\n",
            "\n",
            "to test the model in the prediction of solubility of hydrolases,  <dig> hydrolases from the pdb which were annotated as expressed in e. coli were examined . because good solubility is prerequisite for successful crystallization, this group of proteins served as a positive control for the predictive value of our method.  <dig> of  <dig> hydrolases are predicted to be soluble in e. coli . for all  <dig> hydrolases, the average of cv-cv' is - <dig> , the first quartile is - <dig> . thus, the predicted average solubility of this group of proteins is much higher than any single protein family in the database, including proteins from enterobacteria and the genus escherichia which have an average of cv-cv' of  <dig>  and  <dig>  respectively , which is a strong support for the reliability of this method.\n",
            "\n",
            "in addition, the observation that substitution of asparagines by aspartic acids in dsba-igfbp- <dig> fusion proteins improved solubility  <cit>  is consistent with the solubility formula because the fusion protein is already predicted to be negatively charged . thus, increasing the negative charge increases predicted solubility. similarly, incorporation of solvent exposed positive charged amino acids improved solubility of consensus ankyrin repeat proteins  <cit> . here the net charge proposed by the solubility formula was zero , so solubility is predicted to be increased by insertion of positively or negatively charged amino acids.\n",
            "\n",
            "however, the solubility formula exclusively depends on sequence information and neglects the structural context. therefore, in some cases this simple model resulted in wrong predictions. in the model, substitution of multiple phenylalanine residues by serine  <cit>  led to a predicted increase of the ratio of turn-forming residues in the formula and thus to a lowered predicted solubility in e. coli. instead, an increase in solubility was observed experimentally  <cit>  which can be explained by the structural context: replacement of solvent- accessible phenylalanines increased polarity of the proteins surface and thus its solubility. the fact that single residues can have a huge impact on in vivo solubility  <cit>  is not always fully explainable by the solubility formula. therefore, for the design of proteins with higher solubility the formula should be combined with a careful investigation of the structural context.\n",
            "\n",
            "solubility of α/β-hydrolases\n",
            "protein size was not included as a parameter in the solubility model, because it showed no significant difference between soluble and insoluble proteins of the dataset  <cit> . however, the analysis of groups formed by α/β-hydrolase size revealed that both protein size and phylogeny of organisms have a major impact on predicted solubility. statements like 'small hydrolases are predicted to be more soluble than large hydrolases', 'bacterial hydrolases are predicted to be more soluble than eukaryotic hydrolases' are generally true, but simplifications.\n",
            "\n",
            "for small to medium- sized hydrolases, bacterial hydrolases are predicted to be more soluble than eukaryotic hydrolases, archaea are somewhere in between. the relatively high predicted solubility of medium- sized fungal hydrolases is in contrast to the low predicted solubility of small and large fungal hydrolases. hydrolases from plants are predicted to be much more soluble when they are small than when they are of medium size. however, the significance of this statement is relatively low because there are only eight medium- sized hydrolases from plants.\n",
            "\n",
            "though large hydrolases are mainly predicted to be insoluble the cv-cv' values are much lower for metazoan than for bacterial and fungal hydrolases. so here the situation is opposite to the situation of small and medium- sized hydrolases. additionally, large metazoan hydrolases are predicted to be more soluble in e. coli than small and medium- sized metazoan hydrolases.\n",
            "\n",
            "these results propose that, in problematic cases where soluble hydrolases are very rare, it makes sense to screen for large hydrolases in coding sequences of metazoa, small hydrolases should be searched in bacterial or archaean and medium- sized hydrolases in actinobacterial or ascomycetic coding sequences. secreted and eukaryotic hydrolases generally have a low predicted solubility in e. coli.\n",
            "\n",
            "as the data about taxa, the family information could be used to efficiently search for new soluble hydrolases. if a specific catalytic activity is observed in different homologous families or a soluble member of a specific superfamily is searched for a structural genomics project, family specific degenerate primers could preferably be designed for families which are predicted to include mainly soluble hydrolases.\n",
            "\n",
            "if soluble hydrolases of a given size are rarely found, screening of libraries could be limited to coding sequences from taxa where soluble hydrolases are expected.\n",
            "\n",
            "CONCLUSIONS\n",
            "general rules for the relationship between predicted solubility in e. coli, protein size and phylogenetic origin are:\n",
            "\n",
            "1) bacterial hydrolases are predicted to be more soluble in e. coli than eukaryotic hydrolases.\n",
            "\n",
            "2) small hydrolases are predicted to be more soluble in e. coli than large hydrolases.\n",
            "\n",
            "3) in one taxon huge differences of predicted solubility between genera can exist.\n",
            "\n",
            "4) in one superfamily huge differences of predicted solubility between homologous families can exist.\n",
            "\n",
            "when taking into account the groups formed by protein size there are three additional rules:\n",
            "\n",
            "5) small bacterial hydrolases are predicted to be more soluble than small and large hydrolases from eukaryotes and large bacterial hydrolases.\n",
            "\n",
            "6) large metazoan hydrolases are predicted to be more soluble than large hydrolases from bacteria and small hydrolases from metazoa.\n",
            "\n",
            "7) fungal medium- sized hydrolases are predicted to be more soluble than small or large hydrolases from fungi.\n",
            "\n",
            "when using the family characteristics family- specific primers could be designed to amplify members from specific homologous families with high averages of predicted solubility in e. coli.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the solubility of bacterial α/β-hydrolases is predicted to be more soluble in e. coli than eukaryotic α/β-hydrolases.\n",
            "the solubility of bacterial α/β-hydrolases is predicted to be more soluble in e. coli than eukaryotic α/β-hydrolases.\n",
            "the solubility of bacterial α/β-hydrolases is predicted to be more soluble than both small and large eukaryotic α/β-hydrolases.\n",
            "these results suggest that, in problematic cases where soluble hydrolases are very rare, it makes sense to screen for large hydrolases in coding sequences of metazoa, small hydrolases should be searched in bacterial or archaean and medium- sized hydrolases in actinobacterial or ascomycetic coding sequences.\n",
            "the solubility of bacterial α/β-hydrolases is predicted to be more than  <dig>   <dig>  and  <dig> % of the hydrolases predicted to be soluble in e. coli.\n",
            "the solubility of bacterial α/β-hydrolases is predicted to be more than  <dig> \n",
            "64\n",
            "[{'summary_text': 'over-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting cell proliferation.\\nover-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting cell proliferation.\\nover-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting cell proliferation.\\nover-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting cell proliferation.\\nover-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting cell proliferation.\\nover-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting mammary gland hyperplasia.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 4700 to 5120 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "tbx <dig> is a member of the t-box family of genes. t-box genes are expressed during embryonic development and have been found to regulate cell specification and organogenesis  <cit> . they are also well-known for the roles they play in many human developmental syndromes  <cit> . tbx <dig> is known to function as a transcriptional repressor and is required for embryonic development and for the normal development of the mammary gland  <cit> . in mice models, homozygous mutations in which the function of tbx <dig> is completely lost are embryonic lethal while haploinsufficiency of tbx <dig> results in significantly reduced branching of ductal trees in adult animals  <cit> . in humans, mutations that result in the haploinsufficiency and loss of function of tbx <dig> ultimately cause ulnar mammary syndrome   <cit> . ums is an autosomal dominant disorder characterized by mammary gland hypoplasia and affects limb, apocrine-gland, teeth, hair, and genital development. besides tbx3's role in early mammary gland development, various studies have also supported a role for tbx <dig> in breast cancer development. the tbx <dig> gene is located at the 12q <dig> region which is frequently amplified in a variety of malignancies including breast cancer  <cit> . moreover, tbx <dig> is over-expressed in various breast cancer cell lines as well as primary breast cancer tissues  <cit> . tbx <dig> is mislocalized to the cytoplasm in primary breast cancer tissues and serum tbx <dig> protein levels were also found to be abnormally high in early stage breast cancer patients  <cit> . more recently, it has been shown that pma-induced up-regulation of tbx <dig> contributes to breast cancer cell migration  <cit> .\n",
            "\n",
            "tbx <dig> has been shown to repress the expression of the tumor suppression gene p14arf  <cit>  and the murine homologue p19arf  <cit> . the p14/19-mdm2-p <dig> pathway plays an important role in regulating cell senescence and protects cells against oncogenic transformation which leads to tumor formation  <cit> . tbx <dig> over-expression has been shown to immortalize mouse embryonic fibroblast cells by suppressing p19arf  <cit> . we have previously shown that over-expression of tbx <dig> represses human p14arf by recruiting hdac  <dig>   <dig>   <dig> and  <dig> in the mcf <dig> breast cancer cell line  <cit> . in order to identify other targets of tbx <dig>  we used chromatin immunoprecipitation-guided ligation and selection  promoter array. our results showed that  <dig> gene promoters are bound by tbx <dig> in the mcf <dig> breast cancer cell line . one of the identified genes, nfκbib, is an inhibitor of nf-κb. studies have shown that nf-κb associated pathways play an important role in cell proliferation, differentiation and apoptosis  <cit> . specifically, nfκbib inhibits nf-κb by sequestering it in the cytoplasm. activation of nf-κb occurs upon ubiquitin mediated degradation of nfκbib proteins via serine phosphorylation by iκb kinase . studies have shown that inhibition of nf-κb activation in mouse mammary glands lead to defective proliferation in lobuloalveolar structures during pregnancy  <cit> , whereas elevated nf-κb activity causes mammary hyperplasia in vivo  <cit> . furthermore, aberrant activation of nf-κb is related to breast cancer progression, including tumor initiation, proliferation, chemoresistance and tumor metastasis  <cit> . taken together, these studies suggest that a dysregulation of tbx <dig> expression may contribute to breast cancer development.\n",
            "\n",
            "further supporting the notion that tbx <dig> plays a role in cancer development, recent studies have shown that increased levels of tbx <dig> enhanced melanoma invasiveness by repressing e-cadherin expression  <cit> . recent studies have shown that tbx <dig>  a downstream target of wnt/β-catenin in liver cancer, has also been found to be over-expressed in human hepatocellular carcinoma and heptoblastoma  <cit> . knockdown of tbx <dig> in rat bladder carcinoma cell lines resulted in a lower growth rate and more apoptotic cells than controls, suggesting that tbx <dig> promotes cell proliferation and is a negative regulator of apoptosis  <cit> . although many studies have shown that a dysregulation of tbx <dig> expression may contribute to cancer progression, no direct evidence shows that tbx <dig> causes breast cancer.\n",
            "\n",
            "identifying whether tbx <dig> directly promotes breast cancer development and the mechanism by which it does this is important for understanding mammary development as well as the perturbations that may lead to breast cancer. in the present study, we have demonstrated that over-expression of tbx <dig> in our doxycycline inducible mouse model promotes accelerated mammary gland development and hyperplasia by promoting mammary epithelium cell proliferation. moreover, we have shown that nfκbib was dramatically down-regulated in the mammary glands of doxycycline induced double transgenic mice. although over-expression of tbx <dig>  alone, did not cause tumor formation within the mammary gland, our data suggests that the over-expression of tbx <dig> may contribute to breast cancer formation through the inhibition of the nf-κb pathway and stimulation of both mammary epithelial cell and stem-like cell proliferation.\n",
            "\n",
            "RESULTS\n",
            "tbx <dig> over-expression is induced in mmtv-rtta; tet-myc-tbx <dig> mammary glands by doxycycline administration\n",
            "to construct a doxycycline inducible myc-tbx <dig> transgene cassette , myc-tbx <dig> cdna was subcloned downstream of tet operator elements  . in our transgene expression cassette, the expression of the luciferase reporter gene is regulated by the same promoter as our myc-tbx <dig> transgene. thus, upon induction with doxycycline, translation of the luciferase reporter gene by its own internal ribosome entry site  can be used as a marker for myc-tbx <dig> overexpression . in order to express myc-tbx <dig> specifically in the mammary glands of mice, tet-myc-tbx <dig> mice were mated with mmtv-rtta mice. transgene expression was induced in double transgenic mice by adding 2mg/ml doxycycline to the drinking water. to verify that the induction of tbx <dig> expression within the mammary glands of mice occurred only upon the addition of doxycycline, luciferase activity was monitored by imaging the mammary glands of both doxycycline induced and un-induced double transgenic mice in vivo, using an iccd camera. prior to in vivo imaging, mice were sedated by intraperitoneal injection of xylazine and ketamine. after  <dig> minutes, an aqueous solution of luciferin was injected into the peritoneal cavity to detect luciferase activity and tbx <dig> transgene over-expression. the in vivo image of the doxycycline induced double transgenic mouse detected a bioluminescent signal 4- <dig> folds above background within all  <dig> pairs of mammary glands. the bright bioluminescent signal in the cervical midline of the doxycycline induced double transgenic mouse represents the first pair of mammary glands as well as leaky expression of the mmtv promoter within the salivary gland, which is frequently seen in other mmtv models . no signal was detected in the age-matched un-induced double transgenic littermate control . to more directly measure the luciferase activity within each mammary gland a luciferase assay was performed using tissue lysates from each mammary gland  of a single doxycycline induced double transgenic mouse . consistent with the in vivo imaging, all five mammary glands from the doxycycline induced double transgenic mice had high luciferase readings while the un-induced double transgenic littermates showed only baseline readings . direct tbx <dig> over-expression within the mammary gland was also detected by immunohistochemistry with an anti-tbx <dig> antibody. tbx <dig> over-expression was detected only in the induced double transgenic mouse mammary gland . endogenous tbx <dig> expression was not detected . overall, these results show that tbx <dig> over-expression is specifically induced within all  <dig> mammary glands of our double transgenic mice upon administration of doxycycline.\n",
            "\n",
            "over-expression of tbx <dig> promotes accelerated mammary gland development by increasing cell proliferation\n",
            "in mice, the mammary gland development begins shortly after mid-gestation. five pairs of mammary placodes form at the site of the future nipples  <cit> . these placodes invaginate and form buds within the mammary fat pad that contain few branches  <cit> . by birth a simple mammary ductal tree is formed that occupies a small portion of the fat pad  <cit> . after birth, growth of the mammary gland is relatively quiescent until puberty  <cit> . at puberty, club-shaped structures called the terminal end buds  form at the tips of the ductal tree. during this period, cell proliferation in tebs results in ductal elongation through the mammary fat pad. tebs not only elongate through the fat pad, but also bifurcate to form new primary ducts while secondary side-branches sprout along the extending ducts  <cit> . the outgrowth of side branches is controlled by several hormones and signaling pathways  <cit> . at the end of puberty, approximately 10- <dig> weeks of age, tebs reach the edge of the fat pad and disappear  <cit> . in order to determine the effect of tbx <dig> over-expression on the overall development of the mammary gland, we harvested the 1st and 4th mammary glands from  <dig> doxycycline induced double transgenic mice and from another  <dig> of the un-induced double transgenic littermate controls at four specific time points; 7-weeks, 10-weeks, 12-weeks of age and  <dig>  days postcoitus . mammary glands harvested at 7-weeks, 10-weeks and 12-weeks were from nulliparous mice, while those harvested at  <dig>  dpc were from uniparous pregnant mice. whole mount analysis of the 4th mammary gland revealed that at 7-weeks and 10-weeks of age, tebs in the control mice had not reached the edge of the fat pad, whereas the tebs in doxycycline induced double transgenic mice were observed at the edge of the fat pad or had disappeared , suggesting that over-expression of tbx <dig> promotes accelerated ductal elongation. hematoxylin and eosin staining of both the 1st and 4th mammary glands of the doxycycline induced double transgenic mice displayed increased primary and secondary side branching at all time points when compared to their un-induced double transgenic littermate controls . we also observed an increase in tertiary side branching although this has been known to occur in response to estrous cycle  <cit> . in addition, pregnant doxycycline induced double transgenic mice at  <dig>  dpc also displayed more alveoli tissue than the un-induced double transgenic controls . the samples used for whole mount analysis were from two independent founder lines and the results were consistent between these two lines.\n",
            "\n",
            "several in vitro studies have suggested that the over-expression of tbx3/tbx <dig> leads to the bypass of senescence and promotes cell proliferation  <cit> . to determine whether the observed accelerated development of the mammary glands in tbx <dig> over-expressing mice is due to an increase in cell proliferation, we performed an edu cell proliferation assay. the 4th mammary glands from pregnant doxycycline induced and un-induced double transgenic mice were harvested at  <dig>  dpc and used for the assay. the proportion of nucleated cells incorporating edu was quantified by fluorescence microscopy  and normalized to the total cell number in each 20× field. after quantification, we found that the percentage of edu positive cells is significantly  higher in mammary glands over-expressing tbx <dig>  than their un-induced controls . this result suggests that over-expression of tbx <dig> may promote accelerated mammary gland development by promoting mammary epithelial cell proliferation in vivo.\n",
            "\n",
            "since highly proliferative tissues are associated with carcinogenesis, we next analysed the histology of the 3rd mammary glands of  <dig> week old mice to identify if any unusual morphological changes have occurred. hematoxylin and eosin staining of the doxycycline induced double transgenic mouse mammary gland showed mild focal hyperplasia  and discontinued ductal epithelium  when compared to the littermate control. by the age of  <dig> months, none of the doxycycline induced double transgenic mice had developed tumors.\n",
            "\n",
            "tbx <dig> represses nfκbib\n",
            "in our double transgenic mouse model in which tbx <dig> was over-expressed, we observed accelerated development of the mammary gland from  <dig> weeks of age through pregnancy; specifically enhanced branching and ductal elongation. moreover mice that over-expressed tbx <dig> also had a significantly higher percentage of proliferating mammary epithelial cells than controls. together these data suggest that tbx <dig> may be regulating genes that play a role in cell proliferation. identifying the mechanism by which tbx <dig> promotes accelerated mammary gland development will help to further elucidate its possible role in breast cancer development. dysregulation of the nf-κb associated pathways have been shown to play a role in breast cancer development  <cit> . moreover, it has been shown that elevated nf-κb activity causes mammary hyperplasia in vivo  <cit> . due to this observed phenotype and our previous unpublished data in which tbx <dig> binds to the promoter of nfκbib in mcf <dig> cells, we investigated the role tbx <dig> may play in regulating the nfκb pathway. to verify that tbx <dig> does indeed regulate the nfκbib promoter, we performed a luciferase assay. briefly, cos- <dig> cells were transfected with either pcdna <dig> -myc  or pcdna <dig> -myc-tbx <dig> expression vector together with the pgl3-nfκbib luciferase reporter construct and a β-galactosidase control plasmid  using lipofectamine  <dig>  forty-eight hours later, cell lysates were harvested and used to perform the luciferase assay. β-galactosidase enzyme activity was measured and used to normalize luciferase activity. the luciferase assay revealed that the activity of the nfκbib promoter is significantly repressed  when tbx <dig> is over-expressed in cos- <dig> cells . to determine whether the nfκbib protein was down-regulated upon over-expression of tbx <dig> within the mammary gland, immunohistochemistry was performed on the mammary glands of doxycycline induced and un-induced double transgenic mice at  <dig> weeks of age. staining revealed the nfκbib expression was down-regulated in the doxycycline induced double transgenic mouse when compared to its un-induced double transgenic littermate control . these data suggest that over-expression of tbx <dig> may promote cell proliferation within the mammary gland by repressing the expression of nfκbib.\n",
            "\n",
            "over-expression of tbx <dig> is associated with an increase in mammary stem-like cells\n",
            "another mechanism by which tbx <dig> over-expression may promote accelerated mammary gland development is through the proliferation of mammary stem cells. expression of tbx <dig> has been shown to promote the proliferation of breast cancer stem cells in vitro  <cit> , suggesting that tbx <dig> may also promote mammary stem cell proliferation. a study showed that a single lin-cd24+cd29high cell is able to generate a functional mammary gland, providing strong evidence that these cells are mammary stem cells  <cit> . thus, to isolate and analyze the mammary stem-like cell population we first subtracted the mammary lin+  cells. cd <dig> is considered as an endothelial cell marker, and cd <dig> and ter <dig> are considered as hematopoietic cell markers  <cit> . therefore, lin+  cells are considered a terminally differentiated cell population. in contrast, cd <dig> is a skin stem cell marker  <cit>  and cd <dig> is found on neuronal stem cells  <cit> , therefore cd29+ and cd24+ cells are considered mammary stem-like cells  <cit> . to determine whether over-expression of tbx <dig> promotes the proliferation of mammary stem-like cells, we dissected mammary glands from two mice at  <dig> weeks of age from the doxycycline induced double transgenic group and their un-induced double transgenic littermates and isolated the mammary stem-like cells using the previously mentioned cell markers. the gating strategy for lin- cells and cd24+ cd29high cells is shown in figure 5a. facs analysis revealed that over-expression of tbx <dig> did not affect the overall frequency of lin- cells in the mammary glands of doxycycline induced and un-induced mice,  <dig> % and  <dig> %, respectively . however, within the lin- population, there was a significant increase in the frequency of cd24+cd29high cells in the doxycycline induced double transgenic mice versus un-induced control;  <dig> % and  <dig> % respectively  . the average and standard deviations from both mice in each group are presented in figure 5b. these results suggest that over-expression of tbx <dig> may promote proliferation of mammary stem-like cells.\n",
            "\n",
            "discussion\n",
            "the tbx <dig> t-box transcription factor plays an important role in early mammary development  <cit> . mutations that cause haploinsufficiency of tbx <dig> result in mammary gland hypoplasia in both mice and human  <cit> . on the other hand, tbx <dig> is over-expressed in a variety of cancers, including breast cancer  <cit> . although tbx <dig> over-expression has been associated with oncogenesis by its known ability to inhibit p14arf expression and bypass senescence or by contributing to breast cancer cell migration  <cit> , no direct evidence has been shown to suggest that over-expression of tbx <dig>  alone, can induce tumor formation within the mammary gland. in this study, we over-expressed tbx <dig> within the mammary glands of mice, using a tissue-specific, doxycycline inducible transgenic system. transgenic mouse models using constitutive promoters have provided information about specific genes and breast cancer development, particularly oncogene function  <cit> . however, there are significant limitations to these systems due to the lack of control of transgene expression. the ability to control tbx <dig> expression is critical since homozygous tbx <dig> knockout is embryonic lethal and constitutive over-expression is potentially toxic  <cit> . we implemented a tet-on system in our transgenic mouse model so that tbx <dig> transgene expression is inducible in a time and tissue-specific manner  <cit> , enabling us to test possible tbx <dig> function in tumorigenesis in the mammary glands. an advantage of our mouse model is the ability to use luciferase expression as an indication of tbx <dig> transgene expression . in this way, we are able to monitor tbx <dig> expression without sacrificing the animal. using in vivo imaging as well as a luciferase assay, we were able to show that transgene expression is tightly controlled by doxycycline administration . our results show that this system is reliable and transgene expression could be induced in all five pairs of mammary glands.\n",
            "\n",
            "previous studies have shown that the five pairs of mouse mammary glands are differentially regulated by tbx <dig> during early development. for example, in tbx <dig> knockout studies, homozygous mutations resulted in the absence of mammary placodes, except for an occasional induction of the second pair of mammary placodes  <cit> . heterozygous mutations of tbx <dig> caused decreased branching morphogenesis in the first three pairs of mammary glands, but had no significant impact on the fourth and fifth pairs of mammary glands  <cit> . in  <dig>  day old tbx <dig> heterozygous embryos, 75% of the first pair of mammary glands was missing with no nipple or ductal tree formation while the second pair of mammary glands was affected to a lesser extent  <cit> . although these studies suggest that tbx <dig> regulates murine mammary glands differently, we found that over-expression of tbx <dig> promotes accelerated mammary gland development in both the first and fourth mammary glands  as well as the second, third and fifth mammary glands .\n",
            "\n",
            "research has solidified a role for tbx <dig> in the early development of the mammary gland. tbx <dig> homozygous mutant mice results in mammary gland hypoplasia while heterozygous mutations of tbx <dig> caused decreased branching morphogenesis in mammary glands  <cit> . our research complements these previous studies showing that tbx <dig> over-expression within the mammary glands causes hyperplasia, promoting increased secondary and tertiary branching as well as accelerated ductal elongation. it is also important to discuss that we have over-expressed human tbx <dig> within the mammary glands of mice. it has been shown that human tbx <dig> and mouse tbx <dig> are 97% homologous at the protein level. our group and others have demonstrated that human tbx <dig> is functional in mouse cells  <cit> . furthermore, atbx <dig> knockout mouse model was able to recapitulate the phenotype seen in humans with ulnar mammary syndrome . in a study performed by papaioannou et al., a mutation in the mouse tbx <dig> gene that closely corresponds to truncation mutations seen in some individuals with ums resulted in a deficiency in mammary placode induction and the absence or reduction of mammary buds in mutant embryos, corresponding to the mammary gland hypoplasia seen in patients with ums. moreover, the deficiency in the development of limb elements in individuals with ums was also reflected in limb abnormalities in the tbx <dig> mutant mice. mutant mice had deformities in the forelimb digits, foot and fibula resulting from a failure in the development of posterior limb elements. this study exemplifies that the tbx <dig> protein plays a similar role in the development of the mammary glands in both human and mice. the mechanism by which tbx <dig> over-expression promotes hyperplasia in mammary glands needs to be elucidated. using an edu cell proliferation assay, we showed that over-expression of tbx <dig> resulted in a dramatic increase in cell proliferation within the mammary glands of pregnant doxycycline induced double transgenic mice at  <dig>  dpc . although cell proliferation was not directly quantified for the other developmental time points , the similarity in the observed accelerated mammary gland development suggests that the increase in cell proliferation at  <dig>  dpc may also play a role in causing the accelerated branching and elongation of ducts during the other phases of mammary gland growth. a study has shown that mammary epithelia lacking the gene encoding nfκbia contained increased nfkb activity as well as increased ductal branching and widespread intraductal hyperplasia  <cit> , similar to results seen in our study. furthermore, aberrant activation of nf-κb increased cell proliferation and breast cancer progression  <cit> . in this study, we found that tbx <dig> inhibits the promoter activity of nfκbib in vitro . upon further analysis, in vivo, we observed that nfκbib expression was dramatically reduced in doxycycline induced double transgenic mice as compared to its un-induced double transgenic littermate controls . taken together, our results suggest a mechanism by which tbx <dig> over-expression represses nfkbib/nfkbib expression to enhance cell proliferation and promote mammary gland hyperplasia. however, tbx <dig> is a multifunctional transcription factor and the nfkb pathway could be one of many pathways regulated by tbx <dig>  wnt signaling has also been shown to play a major role in regulating mammary gland development  <cit> . a tbx3-/- mouse model lacked expression of lef <dig> and wnt10b  <cit> , suggesting that wnt signaling is a downstream target of tbx <dig> and that tbx <dig> may regulate mammary gland development via the wnt signaling pathway. additional experiments can be done to further elucidate other mechanisms by which tbx <dig> over-expression promotes mammary hyperplasia.\n",
            "\n",
            "studies have suggested a role for tbx3/tbx <dig> in regulating the self-renewal of mouse embryonic stem  cells as well as breast cancer stem-like cells  <cit> . mouse es cells require leukemia inhibitory factor  to maintain their undifferentiated state  <cit> . mouse es cells genetically modified to over-express tbx <dig> and grown in culture without lif were able to maintain their undifferentiated state  <cit> . knockdown of tbx <dig> expression in mouse es cells resulted in a loss of self-renewal, causing these cells to differentiate  <cit> . these findings suggest that tbx <dig> expression is necessary to maintain mouse es cells in their undifferentiated state and plays a functional role to promote self-renewal. a recent study has proposed a model in which the expression of tbx <dig> in cancer cells promotes the expansion of cancer stem-like cells through paracrine fibroblast growth factor  signaling  <cit> . over-expression of tbx <dig> increased the proportion of cancer stem-like cells in mcf <dig> cells by nine-fold as well as lead to an increase in tumorsphere formation and tumor initiation  <cit> , suggesting that tbx <dig> is sufficient to promote normal and cancer stem like cell phenotypes. due to its role in promoting proliferation of mouse es cells and breast cancer stem-like cells as well as its requirement for early mammary gland development, tbx <dig> may also play a role in regulating mammary stem cell proliferation. mammary glands consist of two cell lineages: myoepithelial and luminal epithelial cells. both of them arise from a common progenitor, the mammary stem cell. research has shown that a single lin-cd24+cd29high cell is able to generate a functional mammary gland, suggesting that these cells are mammary stem cells  <cit> . to determine whether over-expression of tbx <dig> affects mammary stem cell proliferation, we performed facs analysis of the stem-like cell population, lin-cd24+cd29high, in doxycycline induced double transgenic mice and their un-induced littermate controls. we found that over-expression of tbx <dig> significantly increased the frequency of lin-cd24+cd29high stem-like cell population , indicating that tbx <dig> expression is associated with an increased number of mammary stem-like cells. this could explain another mechanism by which tbx <dig> over-expression can cause hyperplasia and accelerated mammary gland development. further studies of the mechanisms by which tbx <dig> regulates mammary stem-like cells are required to improve our understanding of mammary gland development and tbx <dig> function.\n",
            "\n",
            "CONCLUSIONS\n",
            "tbx <dig> over-expression causes mammary gland hyperplasia possibly by inhibiting nfκbib expression and thus promoting cell proliferation. also, over-expression of tbx <dig> is associated with an increased number of mammary stem-like cells suggesting another mechanism by which tbx <dig> may promote mammary gland hyperplasia and contribute to breast cancer development.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "over-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting cell proliferation.\n",
            "over-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting cell proliferation.\n",
            "over-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting cell proliferation.\n",
            "over-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting cell proliferation.\n",
            "over-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting cell proliferation.\n",
            "over-expression of tbx <dig> in doxycycline induced double transgenic mice caused mammary gland hyperplasia possibly by inhibiting nfκb expression and thus promoting mammary gland hyperplasia.\n",
            "\n",
            "65\n",
            "[{'summary_text': 'in this paper, we present a k-spectrum based error correction algorithm for high-throughput sequencing in which a single genome is sampled at high coverage, such as resequencing and de novo sequencing.\\nin this paper, we present a k-spectrum based error correction algorithm for high-throughput sequencing in which a single genome is sampled at high coverage, such as resequencing and de novo sequencing.\\nin this paper, we present a k-spectrum based error correction algorithm for high-throughput sequencing in which a single genome is sampled at high coverage, such as resequencing and de novo sequencing.\\nin this paper, we present a k-spectrum based error correction algorithm for high-throughput sequencing in which a single genome is sampled at high coverage, such as resequencing and de novo sequencing.\\nin this paper, we present a k-spectrum based error correction algorithm for high-throughput sequencing in which a single genome is sampled at high coverage, such as resequencing and de novo sequencing.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3495 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "high-throughput short read sequencing technologies have become the mainstay of genomic research. critical attention is paid to read quality as it affects the quality and performance of sequencing applications. for example, read quality directly impacts accuracy in mapping to a reference genome. in de novo genome sequencing, apart from accuracy of the generated contigs, read quality affects contig lengths. error-free reads can also improve algorithmic performance, as alignments can be replaced with much faster exact matching.\n",
            "\n",
            "the focus of this work is applications of high-throughput sequencing in which a single genome is sampled at high coverage, such as resequencing and de novo sequencing. in these cases, the infrequent occurrence of errors in reads, and the apparent lack of affinity of errors to any fixed genomic location, provide a way to detect and correct erroneous bases in reads. if the reads covering a specific genomic position can be identified and properly positioned relative to their locations of genomic occurrence, this layout can be used to infer the true base by majority vote and correct the others. this works for a haploid genome, but can be extended to polyploid genomes to at least identify correct bases. several error correction algorithms for haploid genomes have been developed, using k-spectrum  <cit> , suffix trees  <cit> , or multiple sequence alignments  <cit>  to identify overlapping reads. for a detailed survey of error correction methods, see  <cit> .\n",
            "\n",
            "most error correction methods are designed for illumina sequencers, which are predominantly used. with rare exceptions, reads from these sequencers only contain substitution errors, leading to simpler algorithms based on hamming distance instead of edit distance. high-end sequencers have error rates well within 1%, and a large percentage of reads are indeed claimed to be free of errors. taking advantage of this, in this paper we propose a different approach: rather than base-level error correction, we seek to identify reads that are error-free . if such predictions can be made with high accuracy, it opens the door to simplifying algorithms for downstream applications, not to mention improvements in quality. in fact, we show that error correction algorithms themselves can be improved by using the perfect reads to correct others, instead of collectively using all the reads.\n",
            "\n",
            "contributions\n",
            "in this paper, we present a k-spectrum analysis based approach to filter out erroneous reads in a high-coverage illumina dataset. we applied our algorithm to one hiseq  <dig> and five hiseq  <dig> datasets. our experiments show that if around 80% of the reads in a dataset are perfect, then our approach retains almost  <dig> % of the perfect reads with more than 90% precision rate. though the coverage reduces by 7% on an average, we observed no noticeable skew in the distribution of perfect reads as compared to the distribution of the original dataset. we also developed a way to characterize the type of datasets for which such an approach is effective.\n",
            "\n",
            "depending on the application, our method can be customized to vary the degree of stringency used to discard a read as erroneous. for example, if the objective is to retain most of the perfect reads despite the risk of increasing false positives, then the lowest level of stringency should be used. on the other hand, if the objective is to minimize false positives, the highest level of stringency should be used.\n",
            "\n",
            "finally, we demonstrate that our prediction of perfect reads can be used to improve the performance of error correction algorithms. to do so, we consider reptile  <cit> , a k-spectrum based error correction algorithm. this method performs an analysis of kmers in the input reads and uses a hamming graph constructed based on the kmers to detect and correct errors. we found that if only kmers from perfect reads are used instead, this leads to an improvement of up to 10% on the percentage of errors removed from the dataset. this approach can be readily applied to improve other error correction algorithms.\n",
            "\n",
            "the organization of the rest of the paper is as follows. the details of our approach are presented in section titled methods. experimental results are presented in section titled results. in section titled improving error correction algorithms, we show how to apply this approach to improve error correction methods. we conclude in section.\n",
            "\n",
            "methods\n",
            "our algorithm is based on analyzing the k-spectrum of the given data set. the k-spectrum is the collection of all kmers, i.e., all substrings of length k from the reads. define a kmer to be valid if it is present in the genome being sequenced, and invalid otherwise. a read is perfect if it does not contain any invalid kmer. in the absence of the reference genome, the validity of a kmer can be estimated from its frequency in the dataset. as errors are infrequent, with sufficient coverage, a valid kmer should occur at significantly larger frequency than invalid kmers. thus, similar to k-spectrum based error correction algorithms, our method consists of two phases. in the first phase, we generate frequency statistics of the kmers, and construct a graph to link kmers within short hamming distance. in the second phase, each read is checked for potential errors using the statistics built in the first phase.\n",
            "\n",
            "kmer statistics generation\n",
            "the kmer at position p of read r is denoted by r. we assume k to be a fixed even number so that k/ <dig> is a whole number. to determine the validity of a kmer, we also consider the quality scores of the associated bases. we count an instance of a kmer only if each of its associated bases exceeds a quality threshold qe . the number of such instances of a kmer t is termed its frequency, denoted by f. because of the double stranded nature of dna, each kmer is associated with its reverse complement kmer also. we represent both these kmers by the one smaller in lexicographic order, and combine the frequencies. the frequencies of all the kmers in the k-spectrum can be easily computed in a single pass over the data. alternatively, other efficient kmer counters such as  <cit>  can be modified to use quality scores.\n",
            "\n",
            "in the first phase, our algorithm also builds a hamming graph over the k-spectrum. each kmer is represented by a node in the graph. a pair of nodes is connected if the corresponding kmers differ in at most d positions, for a fixed d. the main purpose of the graph is to better estimate the validity of a kmer by taking its graph neighborhood into account. we use the same space efficient data structure to construct and store the hamming graph as in reptile .\n",
            "\n",
            "identifying perfect reads\n",
            "our algorithm for processing a read is presented in algorithm  <dig>  the algorithm decomposes a read into overlapping kmers such that the overlap between two consecutive kmers is k/ <dig>  half their length. if there are insufficient base pairs for such an overlap towards the end of a read, the last kmer is chosen to be the suffix of the read of length k. if none of these kmers is estimated to be invalid, the algorithm outputs the read as perfect.\n",
            "\n",
            "algorithm 1: error detection\n",
            "\n",
            "data: read r\n",
            "\n",
            "result: classify r as perfect or erroneous\n",
            "\n",
            "p ← 0; /* current kmer r */\n",
            "\n",
            "while  do\n",
            "\n",
            "t ← kmer of r at position p;\n",
            "\n",
            "if t is not valid then return erroneous;\n",
            "\n",
            "if  then return perfect;\n",
            "\n",
            "if  + k < |r|) then p ← p + k/2;\n",
            "\n",
            "else p ← |r| − k;\n",
            "\n",
            "end\n",
            "\n",
            "the algorithm relies on a rule to estimate if a kmer is valid. we consider five different rules based on properties p <dig>  . . . , p <dig> below:\n",
            "\n",
            "p1: f  ≥ ce\n",
            "\n",
            "p2: f  ≥ cg and each base pair in t has quality ≥ qg\n",
            "\n",
            "p3: f  ≥ cg and t does not have a neighbor t′ in hamming graph with f ≥ cg\n",
            "\n",
            "p4: f  ≥ cg and t does not have a neighbor t′ in hamming graph with f ≥ f × fh\n",
            "\n",
            "p5: f  ≥ cg and all neighbors t′ of t in the hamming graph have property: all the base pairs of t\n",
            "\n",
            "where t differs from t′ have quality score ≥ qg where the following parameters are to be set appropriately: ce , cg , qg , fh , ce > cg and qe > qg.\n",
            "\n",
            "for the rest of the paper, we say t is valid by rule i if it satisfies any one of the properties p <dig>  p <dig>  . . . , pi. thus, the rules are in decreasing order of stringency. in the most stringent case , the algorithm treats a kmer t as valid only if its frequency f  is at least a threshold ce . in p <dig>  f  is allowed to be above a lower threshold cg but each base in t must have quality score above qg. the rationale for properties p <dig> and p <dig> is that as the kmer in consideration has comparatively lower frequency, and there are no high cardinality d-neighbors, it might be the case that the kmer is from a region of low coverage. in p <dig>  the kmer has strong quality scores at all the positions in which it differs from its d-neighbors, and hence it has a high likelihood of being valid.\n",
            "\n",
            "note that these rules are heuristics and hence the perfect reads detected by our algorithm may have errors and some of the erroneous reads detected by our algorithm can in fact be error-free.\n",
            "\n",
            "RESULTS\n",
            "we applied our algorithm to  <dig> datasets from the ncbi short read archive, the details of which are given in table  <dig>  for each dataset the table shows sra accession number, sequencer platform, name of the reference genome, strain, organization that published the data, date of publishing, percentage of gc content, length of the genome in mb, read length, and average coverage.\n",
            "\n",
            "ucb: university of california, berkeley; wtsi: wellcome trust sanger institute; umigs: institute for genome sciences, university of maryland; cshl: cold spring harbor laboratory\n",
            "\n",
            "to evaluate our method, knowledge of error-free reads in each dataset is required. to determine them, we aligned each dataset using the bwa aligner  <cit>  with default parameters. a read is taken to be error-free if it is perfectly aligned by bwa without any substitution, insertion, or deletion. the results of bwa alignments are shown in table  <dig> where each row shows the strain of the reference genome used, number of reads in the dataset, number of reads aligned, number of reads not aligned, number of reads ambiguously aligned, number of reads perfectly aligned, and overall error rate. note that the rows of tables  <dig> and  <dig> are arranged in increasing order of the percentage of perfect reads.\n",
            "\n",
            "experiments and evaluation methodology\n",
            "we applied our algorithm to each of the datasets using the following default parameters: k =  <dig>  cg =  <dig>  ce =  <dig>  qg =  <dig>  we chose qe =  <dig> for s <dig> and qe =  <dig> for the remaining datasets. to assess the quality of predictions made, we define:\n",
            "\n",
            "tp = number of perfect reads that are classified by our algorithm as perfect\n",
            "\n",
            "fn = number of perfect reads that are classified as erroneous\n",
            "\n",
            "fp = number of erroneous reads that are classified as perfect\n",
            "\n",
            "tn = number of erroneous reads that are classified as erroneous\n",
            "\n",
            "then, we used the standard measures of specificity , sensitivity , and precision  as:\n",
            "\n",
            "sp = tn/\n",
            "\n",
            "sn = tp/\n",
            "\n",
            "pr = tp/\n",
            "\n",
            "discussion\n",
            "the results of our experiments using rule  <dig>  which tests for conformance with at least one of properties p <dig> and p <dig>  are presented in table  <dig>  except for dataset s <dig>  rule  <dig> achieves near 100% sensitivity, indicating this rule correctly classifies an overwhelming majority of error-free reads, and misclassifies a negligible percentage of error-free reads. thus, applications which take reads predicted to be error-free by our algorithm will retain almost all of the error-free reads. specificity for various datasets indicates to what extent our algorithm succeeded in weeding out reads that contain at least one error. except for dataset s <dig>  our algorithm eliminated at least 50% of erroneous reads from the dataset, reaching close to 90% in some cases . precision, the ratio of true perfect reads to total reads predicted as perfect by our algorithm, is over 90% in all cases except dataset s <dig>  the lower performance on s <dig> can be attributed to the comparatively lower coverage and lower percentage of perfect reads.\n",
            "\n",
            "tp\n",
            "fn\n",
            "fp\n",
            "tn\n",
            "in all cases except s <dig>  applications can significantly benefit by taking as input the reads predicted to be error-free by our algorithm, instead of the raw datasets. doing so, the applications will be operating on data that has over 90% perfect reads, miss very few perfect reads from the original dataset, and can do away with a majority of erroneous reads. we also tested the coverage induced by the reduced datasets generated by our algorithm against the coverage of the genome by the original raw datasets, and found no noticeable loss of information, i.e., we did not find any regions of the genome disproportionately losing coverage significantly higher than what is implied by the reduction in the size of the dataset. for visualization of the test, we show plots generated by the tool qualimap  <cit> , which divides the complete genome into about  <dig> windows and plots the average of the coverage of all base pairs within each window. we show in figure  <dig> the plot generated by qualimap for dataset s <dig> alone; for datasets s <dig> and s <dig> we get similar plots. it can be seen that the coverage pattern remains the same though the average coverage reduces from 140x to 130x . figure  <dig> also shows that the percentage of gc content remains same. as illumina sequencers can generate billions of reads in a single experiment at a very low cost per base, eliminating erroneous sequences can significantly improve data quality for applications without appreciable loss of data-scale.\n",
            "\n",
            "execution time\n",
            "all experiments were carried out on a hp proliant dl580g <dig> server, which has  <dig> intel xeon e <dig>  processors and 132gb main memory. the server is running 64-bit ubuntu  <dig>  os. we used jellyfish software  <cit>  to generate kmer statistics in phase  <dig> of our algorithm. our multi-threaded implementation of phase  <dig> is also based on the library functions associated with the jellyfish software. the rightmost two columns of table  <dig> show the average time taken by the two phases of our algorithm in  <dig> independent runs on each of our datasets. in our experimentation, we used  <dig> threads in each phase. the time taken in phase  <dig> is within the same order of magnitude of time taken in generating the kmer statistics.\n",
            "\n",
            "predicting applicability of our algorithm\n",
            "as noted previously, our algorithm performed well on all datasets except for s <dig>  it would be of tremendous practical value if we can ascertain the applicability of our algorithm by evaluating the dataset alone, without knowing the reference genome. below, we present a methodology to do so.\n",
            "\n",
            "the quality of results obtained by our algorithm can be explained using the histogram of kmer frequencies . in figure  <dig> we plot the histogram of kmer frequencies in dataset s <dig> for k =  <dig>  on the horizontal axis we show the different frequencies of the kmers. for a particular frequency x on the horizontal axis the curve named all shows the number of distinct kmers t that have frequency f = x. depending on the alignment, the kmers can be divided into three categories. a kmer is good if it comes from the error-free regions of all the reads that it appears, bad if it comes from erroneous regions in all the reads it appears, and mixed if it appears in the error-free regions on some reads and in the erroneous regions of some other reads. in figure  <dig> we also show the frequency histograms for good, bad and mixed kmers.\n",
            "\n",
            "a necessary condition for the good performance of a k-spectrum based error detection algorithm is that the frequency histograms for good, bad, and mixed kmers should follow the pattern in figure  <dig> where the number of mixed kmers is very very low, most bad kmers have low frequency, bad kmers with higher frequencies are very rare, and the histogram of good kmer frequencies follows a normal curve which has mean approximately equal to the average coverage. when this condition is satisfied, we can clearly demarcate a threshold frequency at the crossover of the good and bad curves such that most of the kmers with frequency less than the threshold are bad, and most of the kmers with frequency above the threshold are good. the dataset s <dig> follows the pattern and our algorithm works well on s <dig> \n",
            "\n",
            "on the other hand, the plots for s <dig> shown in figure  <dig> do not follow the expected pattern. in fact it is not possible to distinguish the good kmers from the bad ones depending upon only the frequency, as the bad curve is above the good curve. increasing the kmer size does not help either. figure  <dig> shows the plots for all kmers for values of k =  <dig>   <dig>   <dig>   <dig>  and  <dig>  none of which follows the expected pattern. this explains the bad performance of our algorithm on s <dig> \n",
            "\n",
            "unlike the curves good, bad, and mixed, the curve all can be plotted even in the absence of alignment information. if the shape of the all curve follows the expected pattern as in figure  <dig>  our algorithm should perform well.\n",
            "\n",
            "the plots also give hints on how to set the parameters. the parameter k should be such that the probability of a kmer having repeats in the genome g, i.e., |g|/4k is very small. ce should be the first minimum of all curve. at least 95% of the kmers should have frequency cg or more. the quality parameters could be such that about 80%  of the bases have quality at least qg .\n",
            "\n",
            "effects of varying stringency levels\n",
            "we also varied the rules used to identify if a kmer is valid or not, as described in section. we report the results on datasets s <dig>  s <dig> and s <dig> with parameter fh =  <dig> in table  <dig>  as we increase the rule number, the stringency of declaring a read to be error-free decreases, resulting in more true and false positives. hence, sensitivity increases but specificity and precision decrease.\n",
            "\n",
            "tp\n",
            "fn\n",
            "fp\n",
            "tn\n",
            "pr\n",
            "sp\n",
            "sn\n",
            "improving error correction algorithms\n",
            "our algorithm for predicting error-free reads can be used to improve the performance of error correction algorithms themselves. we demonstrate this using the error correction algorithm reptile  <cit> , though the methodology is more broadly applicable. reptile consists of two phases: in the first phase, it counts the frequency of all kmers and constructs a hamming graph on them. in addition, kmers are classified as valid or invalid based on whether or not they exceed a threshold count. nodes in the hamming graph are marked with this information. in the second phase, each read is corrected by changing invalid kmers in it to their valid hamming graph neighbors. we slightly modified the first phase of reptile to count kmer frequencies in only those reads which are predicted as error-free by our algorithm. in addition, we use the hamming graph to only correct errors in reads that are predicted to be erroneous by our algorithm.\n",
            "\n",
            "\n",
            "set\n",
            "algo\n",
            "tp\n",
            "fn\n",
            "fp\n",
            "tn\n",
            "wc\n",
            "sn\n",
            "sp\n",
            "g\n",
            "eba\n",
            "as proposed in  <cit>  and more widely adopted later, gain is an important measure for assessing the quality of an error correction algorithm. from table  <dig> for the datasets on which our prediction performs well, mainly s <dig>  s <dig>  and s <dig>  the modification improves gain by up to 10%.\n",
            "\n",
            "CONCLUSIONS\n",
            "thanks to the continuous technological improvements in high-throughput dna sequencing, reads of dominant sequencing platforms such as the illumina hiseq are sporting high coverage and accuracy. this has now reached an extent where we can envision just filtering out reads with errors, thus making error correction less important. our algorithm is a first attempt to propose and demonstrate this new paradigm. our experimental results demonstrate that development of such algorithms shows great promise. there are directions for further improvement of our algorithmic strategy. our algorithm relies on several parameters. an automated choice of parameters sensitive to, and computed based on, the dataset would be useful. it might be useful to have values of parameters ce , cg dependent on the pattern of bases in the kmers. like all error correction algorithms, our algorithm ignores paired read information and treats them as though they are single reads. utilizing paired read information to further improve the performance of error detection or correction algorithms remains an open question. in case of long reads where reads are less likely to be perfect, a notion of approximate perfect  can be used.\n",
            "\n",
            "availability\n",
            "a c++ based implementation of our algorithm can be found at the following github public repository: https://github.com/soumitrakp/perfectread.git.\n",
            "\n",
            "competing interests\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "authors' contributions\n",
            "sa conceived the study. sp implemented the algorithms and carried out the experiments. sp and sa analyzed the results and wrote the paper.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this paper, we present a k-spectrum based error correction algorithm for high-throughput sequencing in which a single genome is sampled at high coverage, such as resequencing and de novo sequencing.\n",
            "in this paper, we present a k-spectrum based error correction algorithm for high-throughput sequencing in which a single genome is sampled at high coverage, such as resequencing and de novo sequencing.\n",
            "in this paper, we present a k-spectrum based error correction algorithm for high-throughput sequencing in which a single genome is sampled at high coverage, such as resequencing and de novo sequencing.\n",
            "in this paper, we present a k-spectrum based error correction algorithm for high-throughput sequencing in which a single genome is sampled at high coverage, such as resequencing and de novo sequencing.\n",
            "in this paper, we present a k-spectrum based error correction algorithm for high-throughput sequencing in which a single genome is sampled at high coverage, such as resequencing and de novo sequencing.\n",
            "\n",
            "66\n",
            "[{'summary_text': 'in this study e-dragon, which is a remote version of the dragon descriptor calculation program, was used to classify activities of cytochrome p <dig> c <dig> inhibitors and their activities were predicted with 100% accuracy.\\nin this study e-dragon was used to classify activities of cytochrome p <dig> c <dig> inhibitors and their activities were predicted with 100% accuracy.\\nthe results of this study suggest that the approach used in this paper results in better classification accuracies compared to other classification methods reported in literature.\\nin addition to cytochrome p <dig> c <dig> inhibitors we also report a new dataset for cytochrome p <dig> cytochrome p <dig> c <dig> inhibitors and their activities were predicted with 100% accuracy.\\nthe results of this study suggest that the approach used in this paper results in better classification accuracies compared to other classification methods reported in literature.\\nthe results of this study suggest that the approach used in this paper results in better classification accuracies compared to other classification methods reported in literature.\\nin this study e-dragon was used to classify activities of cytochrome p <dig> c <dig> inhibitors and their activities were predicted with 100% accuracy.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 7298 to 8192 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "at the initial stages of drug discovery and design, there are often millions of candidate drug molecules under consideration. therefore, the early prediction of activity for drug candidates using computational methods is very important to save time and resources. due to importance of early prediction of activity of drug candidates on the target protein, a large number of computational methods were developed. qsar  analysis is one of the most widely used methods to relate structure to function. qsar analysis can be described as the quantitative effort of understanding the correlation between the chemical structure of a molecule and its biological and chemical activities such as biotransformation ability, reaction ability, solubility or target activity <cit> . qsar assumes that structurally similar molecules should have similar activities, which draws attention to the importance of detecting the most significant chemical and structural descriptors of the drug candidates. the drug activity behavior can be predicted using a wide range of descriptors.\n",
            "\n",
            "some of the most widely used 3d qsar methods can be listed as follows: comparative molecular field analysis , comparative molecular similarity indices analysis , eigenvalue analysis . in comfa, molecular descriptors are calculated and selected by calculating the electrostatic and steric potential energies between a positively charged carbon atom located at each vertex of a rectangular grid and a series of molecules embedded within the grid <cit> . the sensitivity to small changes in the alignment of compounds is reduced and hydrogen-bonding and hydrophobic fields are introduced to in comsia <cit> . in these methods aligning of the structures is essential, therefore eva was used due to the fact that methods that are sensitive to 3d structure but do not require superposition were introduced <cit> . the generation of descriptors in eva is based on molecular vibrations, where a normal mode calculation is required to simulate the ir spectrum of a molecule  <cit> .\n",
            "\n",
            "in this study e-dragon  <cit> , which is a remote version of the dragon descriptor calculation program, was used to calculate the molecular descriptors for drugs. it applies the calculation of molecular descriptors developed by todeschini et. al <cit>  and provides more than  <dig>  molecular descriptors, which are divided into  <dig> blocks, including atom types, functional group and fragment counts, topological and geometrical descriptors, autocorrelation and information indices, 3d molecular descriptors, molecular properties  <cit> . dragon incorporates two steps; the first step eliminates low-variable descriptors, the second step optimizes the descriptor subset using a q2-guided descriptor selection by means of a genetic algorithm using several data analysis methods: unsupervised forward selection  <cit> , associative neural network  <cit> , polynomial neural network  <cit>  and partial least squares   <cit> .\n",
            "\n",
            "in most studies, partial least squares  <cit>  is used to develop qsar models by reducing the number of attributes in the descriptor set to a small number of attributes correlated with the defined property being modeled.\n",
            "\n",
            "in our approach, to classify activities of drug compounds, we used the mixed-integer programming  based hyper-boxes method that takes the molecular descriptors as attributes of the model. the problem of qsar analysis and the classification of drug candidates are addressed based on their published ic <dig> values by introducing an algorithm that combines pls regression with mixed-integer linear programming based hyper-boxes classification method. the strength of the algorithm not only comes from combining regression with classification but also the ability to improve the classification accuracies by its iterative approach. the algorithm that links qsar descriptor model generation with inhibitory activity classification was applied to inhibitors of acetylcholinesterase , benzodiazepine receptor , dihydrofolate reductase  and cyclooxygenase- <dig>  and finally for cytochrome p <dig> c <dig> .\n",
            "\n",
            "the comparison of our classification accuracies with the accuracies of the classification methods available in the weka data mining package  <cit>  is also made. weka contains  <dig> different classification methods, but here only  <dig> of those, which had the best classification accuracies for the data sets considered in this paper are discussed. brief overview of these classifiers is further presented in the methods section. our approach outperformed all of the classifiers available in weka for each model of the all of the  <dig> data sets, even reaching 100% accuracy in predicting the activity classification of the inhibitor sets, ache inhibitors and cytochrome p <dig> c <dig>  a total number of  <dig> qsar models were built in this study for  <dig> inhibitor sets, and in  <dig> of them the accuracy of our methodology exceeded the accuracy of the second best classifier with more than 10%. through all of the  <dig> models, the smallest difference in the accuracies is  <dig> % and the largest difference is  <dig> %.\n",
            "\n",
            "RESULTS\n",
            "to determine the threshold values, which divide the low and high classes, for all datasets the ic <dig> values were statistically analyzed. in this study, we consider  <dig> datasets, of which ic <dig> values and structures were reported  <cit> . in addition to these datasets we introduced a new dataset for cytochrome p <dig> c <dig> inhibitors that we collected from the literature. cytochrome p <dig> c <dig> is a well-recognized target for prostate cancer treatment, since selective inhibition of the enzyme exerts control over androgen synthesis  <cit> .\n",
            "\n",
            "after building the descriptor models by e-dragon  <cit> , three models were constructed during the pls analysis as:  <dig>   <dig> and  <dig> descriptor models. the reason that we build  <dig> models with different number of variables is due to the fact that we might come up with insignificant descriptors within one of these models, so that we can replace them by a more significant one from the other models.\n",
            "\n",
            "the qsar models with the most significant descriptors, as they were concluded as a result of the initial pls study for the  <dig>   <dig> and  <dig> attribute models are listed in table  <dig> with their r <dig> values. table  <dig> shows the optimal r <dig> values of our pls models given by minitab <cit>  with predefined number of descriptors from the descriptors calculated by e-dragon software, and the r <dig> values of the pls models calculated by sutherland et al <cit>  with the same data sets but different methods and models.\n",
            "\n",
            "* pls results reported by sutherland et al.  <cit> .\n",
            "\n",
            "the r <dig> values shows that, the models we developed with  <dig> and <dig> descriptors for ache bzr and cox- <dig> are stronger than or at least as strong as the other models reported by sutherland et al <cit>  in representing the ic <dig> values in terms of selected descriptors, but our model for dhfr_rl is not as good as the other reported models. high r <dig> values of cytochrome p <dig> c <dig> models also suggest good prediction of the ic <dig> values and a promising initial model for classification.\n",
            "\n",
            "it is worth noting that, our study is not simply a regression study, but we develop these regression models in order to use the selected descriptors from this step as attributes for accurate classification.\n",
            "\n",
            "iterations\n",
            "at the end of the initial runs of the hyper-boxes classification method, classification results are obtained. the next step is the significance analysis and the improvement of the classification accuracies by iterations. the weakest and the strongest descriptors were calculated by significance analysis and, the weakest descriptor in the current model was replaced by the most significant one from other models at each iteration. the classification runs are repeated after each replacement, by milp based hyper-boxes method. when the classification accuracy is not improved at the end of iteration, the algorithm stops and final results are reported .\n",
            "\n",
            "while choosing the weakest descriptor to leave the model, the descriptor with the maximum p-value  for one of the high or low classes was selected. the weakest descriptor was replaced by the strongest one. the strongest descriptor defined as the attribute whose maximum p-value for high and low classes is the minimum among the other descriptors.\n",
            "\n",
            "final classification\n",
            "as shown in table  <dig>  we compared the classification accuracies of our model with the results that calculated using all of the classification methods in weka. we report only the results of the <dig> best performing weka classifiers. our method performed better than all of the other classifiers for every model of each dataset. our integrated approach of regression and classification for ache and cytochrome p <dig> c <dig> inhibitors datasets displayed an activity prediction accuracy of 100%. the activity of bzr inhibitors was calculated with the accuracy of  <dig> %. we were able to predict the activities of cox- <dig> inhibitors with  <dig> % in a 7-descriptor model. in addition, the prediction accuracy of activity of dhfr_rl, dhfr_pc, and dhfr_tg inhibitors were  <dig> %,  <dig> % and  <dig> % respectively. the best performing weka classifiers are also highlighted in table  <dig> \n",
            "\n",
            "to verify the reliability of the accuracies given by 10-fold cross validation standard deviations of the classification accuracies were also calculated for each run of milp based hyper-boxes method. the sensitivity of classification accuracy to the number of descriptors is also examined and the results are reported in table  <dig>  small number of descriptors may lead to poor models while a large number of descriptors may lead to inefficient models that incorporate non-informative descriptors for classification. in all of the datasets considered in this paper, this trend is observed from the accuracy values and standard deviation of accuracies for 10-fold cross validation.\n",
            "\n",
            "detailed analysis: cytochrome p <dig> c <dig> inhibitors\n",
            "we applied our approach to classify activities of drug molecules in a new data set  that is constructed from data in literature  <cit> . the molecular structures and ic <dig> values for these molecules are given in additional file  <dig>  this approach may be utilized for the new molecules that inhibit activity of cytochrome p <dig> c  <dig> before channeling them into experiments.\n",
            "\n",
            "for the  <dig>   <dig> and  <dig> attribute models the selected most significant descriptors as a result of the initial pls study are listed with r <dig> values  of  <dig> ,  <dig>  and  <dig>  respectively.\n",
            "\n",
            "when the hyper-boxes model was implemented,  <dig> and the 15-attribute models reached 100% accuracy, by 10-fold cross validation. the 7-attribute model, however, still needed to be improved since the classification results reached an average accuracy of  <dig> %. this led us to conclude that there may be some overestimated descriptors actually having low significance in terms of classifying the drug activity. therefore, significance tests were performed after the preliminary classification runs.\n",
            "\n",
            "the results of the final run of hyper-boxes classification for the 7-descriptor model shows that the effect of changing the less significant descriptors with the more significant ones improved the accuracy of the classification from  <dig> % to 100%. since we have reached 100% accuracy in 7-descriptors models, the significant ones may be included in this model among 912-descriptors that are initially calculated by e-dragon. brief explanations of the descriptors can be found in table  <dig> \n",
            " <cit> .\n",
            "\n",
            "discussion\n",
            "early analysis and estimation of the drug activities by computational methods are widely studied in order to narrow down drug candidates for further experimental tests. the accuracy comparison of our algorithm with other qsar algorithms suggests that drug activities can be classified with a significantly higher accuracy with the method introduced in this study.\n",
            "\n",
            "after model building by e-dragon qsar software, the pls runs were performed to determine the best model in representing the depended variables  in terms of the independent variables . the corresponding r <dig> values were calculated to determine the reliability of the pls models, where a model with a higher r <dig> value can be regarded as a more reliable model to proceed to the classification step. the r <dig> values for the  <dig>   <dig> and  <dig> descriptor models of p <dig> c <dig> were obtained by pls runs and, with a considerable strength in representing the ic <dig> values we accepted the initial models. while the high r <dig> values of the ache inhibitor models also were promising on its own to build the classification model, the initial models of bzr and cox- <dig> inhibitor sets were chosen after the comparison of pls results with the results reported in the literature as presented in table  <dig>  for dhfr inhibitors data sets such comparison was not also possible, therefore the models with the best r2values in pls studies were chosen among all other possible models calculated. r <dig> value directly depends on the values of attributes  and the number of attributes in the corresponding model.\n",
            "\n",
            "we first applied our iterative algorithm to the large and widely used qsar data sets in order to validate our methodology. the strength of our algorithm was presented by comparing our classification accuracies with the classification accuracies of  <dig> weka classifiers, on  <dig> inhibitor sets. the attribute sets prepared as the input for weka classifiers were the same as the ones, by which we built the last iteration of our milp based hyper-boxes classification model. in other words, those were the most significant attributes that we used to develop the final classification models and reached our best accuracies. our approach outperformed all of the classifiers available in weka for each model of the all of the  <dig> data sets, even reaching 100% accuracy in predicting the activity classification of the inhibitor sets, ache inhibitors and cytochrome p <dig> c <dig>  a total number of  <dig> qsar models were built in this study for  <dig> inhibitor sets, and in  <dig> of them the accuracy of our methodology exceeded the accuracy of the second best classifier with more than 10%. through all of the  <dig> models, the smallest difference in the accuracies is  <dig> % and the largest difference is  <dig> %.\n",
            "\n",
            "the higher prediction accuracy of the model not only comes from the choice of initial models by pls analysis but also the characteristics of milp based hyper-boxes method. the milp based hyper-boxes approach allows using more than one hyper-box in order to define a single class  <cit> . moreover, this approach deals with problematic and non-problematic instances separately and prevents overlapping of final hyper-boxes  <cit> . therefore, these strengths significantly improve the accuracy and efficiency of the milp based hyper-boxes approach compared to other data classification methods. data on true positive and false positive rates for accuracy comparison of classifiers for all data sets are given in additional file  <dig> \n",
            "\n",
            "CONCLUSIONS\n",
            "drug molecules can be classified as low active or high active based on ic <dig> values. in this study an integrated approach was introduced, which combines the milp based hyper-boxes method with partial least squares regression to effectively classify the drug candidates. as a result, the most significant molecular descriptors of the drug molecules were also reported. weka is used to compare the classification accuracies of the developed model with the classifiers in the weka data mining package. the best classification algorithm in weka database gave an accuracy of maximum 85% in classifying the activity of drug molecules, through the datasets used in this study.\n",
            "\n",
            "our method was applied in order to predict the activities of widely known inhibitor datasets for acetylcholinesterase , benzodiazepine receptor , dihydrofolate reductase , cyclooxygenase- <dig>  with known ic <dig> values. the results suggests that the approach used in this paper results in better classification accuracies compared to other classification methods reported in literature. this approach also applied to cytochrome p <dig> c <dig> inhibitors and their activities were predicted with 100% accuracy.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study e-dragon, which is a remote version of the dragon descriptor calculation program, was used to classify activities of cytochrome p <dig> c <dig> inhibitors and their activities were predicted with 100% accuracy.\n",
            "in this study e-dragon was used to classify activities of cytochrome p <dig> c <dig> inhibitors and their activities were predicted with 100% accuracy.\n",
            "the results of this study suggest that the approach used in this paper results in better classification accuracies compared to other classification methods reported in literature.\n",
            "in addition to cytochrome p <dig> c <dig> inhibitors we also report a new dataset for cytochrome p <dig> cytochrome p <dig> c <dig> inhibitors and their activities were predicted with 100% accuracy.\n",
            "the results of this study suggest that the approach used in this paper results in better classification accuracies compared to other classification methods reported in literature.\n",
            "the results of this study suggest that the approach used in this paper results in better classification accuracies compared to other classification methods reported in literature.\n",
            "in this study e-dragon was used to classify activities of cytochrome p <dig> c <dig> inhibitors and their activities were predicted with 100% accuracy.\n",
            "\n",
            "67\n",
            "[{'summary_text': 'gpt1/ <dig> and gpt2-gfp are novel plant-specific\\u2009+\\u2009tip proteins that track growing mt ends independently of eb <dig> and spr <dig>  and thus define a novel plant-specific\\u2009+\\u2009tip family.\\ngpt1/ <dig> and gpt2-gfp are novel plant-specific\\u2009+\\u2009tip proteins that track growing mt ends independently of eb <dig> and spr <dig>  and thus define a novel plant-specific\\u2009+\\u2009tip family.\\ngpt1/ <dig> and gpt2-gfp are novel plant-specific\\u2009+\\u2009tip proteins that track growing mt ends independently of eb <dig> and spr <dig>  and thus define a novel plant-specific\\u2009+\\u2009tip family.\\ngpt1/ <dig> and gpt2-gfp are novel plant-specific\\u2009+\\u2009tip proteins that track growing mt ends independently of eb <dig> and spr <dig>  and thus define a novel plant-specific\\u2009+\\u2009tip family.\\ngpt1/ <dig> and gpt2-gfp'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 7711 to 8192 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "microtubules  are highly conserved polarized cytoskeletal polymers that mediate cellular processes such as motility, cell division, polarity, intracellular transport, and signaling in eukaryotic cells. gtp-bound α/β-tubulin heterodimers are stacked in a head-to-tail fashion to form polar protofilaments, which associate laterally to form polar hollow mt cylinders with highly dynamic plus ends and more stable minus ends. the gtp bound to β-tubulin is hydrolyzed to gdp as mt assembly progresses. before hydrolysis takes place, the gtp-bound tubulin forms a cap that stabilizes the plus end of the mt and thereby promotes its growth. stochastic loss of the stabilizing cap renders the mt lattice of gdp-bound tubulin unstable and causes rapid mt depolymerization, a process known as catastrophe. when gtp-tubulin subunits reassemble at the depolymerizing plus end, the mt resumes growth, a process known as rescue. this mt behavior of alternating between catastrophe and rescue at the plus end is known as dynamic instability  <cit> .\n",
            "\n",
            "gtp-tubulin dimers adopt a curved conformation in solution and are thought to assemble into outwardly curved sheets at the extreme plus end  <cit> . as the number of laterally associating protofilaments increases, the curved gtp-tubulin sheets gradually straighten, to form a tubular structure. gtp hydrolysis and subsequent phosphate release occur soon after tubulin incorporation into the protofilament, resulting in a short gtp cap  <cit> .\n",
            "\n",
            "a subgroup of mt-associated proteins  is characterized by their ability to accumulate at mt ends  <cit> . plus-end tracking proteins  recognize and bind to the plus-ends of mts, and potentially modulate mt assembly dynamics and interactions between the mt plus ends and subcellular targets. two categories of + tips can be distinguished; whereas “autonomous tip trackers” bind mt ends independently of other maps, “hitchhikers” are targeted to the mt ends through binding to an autonomous tip tracker, and they often also have a considerable binding affinity for the mt lattice. end-binding proteins  are autonomous tip trackers that regulate + tip networks in animal cells by recruiting various hitchhiking + tips to the growing plus ends  <cit> . ebs bind in close proximity to the exchangeable gtp-binding site of tubulin, and are thought to sense the mt’s nucleotide state   <cit> . eb <dig> is probably associated with the gtp cap and gtp-like-tubulin, in which hydrolyzed gtp remains in an intermediate gdp-pi state  <cit> . ebs show a characteristic localization pattern at the growing mt ends; their abundance peaks at the mt tip regions and gradually declines toward the main body of mts, forming a “comet”-like pattern.\n",
            "\n",
            "plants have two types of ebs that localize either to the cytoplasm or nucleus. in arabidopsis thaliana, cytoplasmic eb1a and eb1b and nuclear eb1c all accumulate at the growing plus ends of mts . knockdown or knockout of eb1c compromised the alignment of spindle and phragmoplast mts in arabidopsis root cells, and caused frequent lagging of separating chromosomes at anaphase in cultured nicotiana tabacum  cells  <cit> . however, arabidopsis mutants of eb1a and eb1b, as well as their double mutants, are almost indistinguishable from wild-type plants with respect to growth and morphology, indicating that ebs may not be important for mt functions in interphase plant cells  <cit> .\n",
            "\n",
            "in addition to eb <dig>  several other plant maps show partial accumulation at the mt plus ends  <cit> . spiral <dig>  is a plant-specific protein that belongs to a six-member family with overlapping functions in arabidopsis . spr <dig> is localized to the mt lattice and also accumulates at the growing plus ends of mts in arabidopsis cells, forming an extended comet-like pattern that is much longer than that formed by eb <dig>  <cit> . the in vivo association of spr <dig> with cortical mts does not require eb <dig>  <cit> . it is not clear how spr <dig> associates with mts and how it is preferentially recruited to the plus ends, because spr <dig> is not recovered in map preparations from arabidopsis cell extracts  <cit>  and recombinant spr <dig> protein does not show mt binding in vitro  <cit> . the arabidopsis spr <dig> mutant shows right-handed helical growth of elongating axial organs  <cit> , and this helical phenotype is exaggerated in multiple mutants of the spr <dig> gene family  <cit> .\n",
            "\n",
            "we previously isolated several novel maps from arabidopsis cultured cells  <cit> . among them, a putative map encoded by at3g <dig> was found to localize to cortical mts when transiently expressed as a gfp fusion in allium cepa  epidermal cells  <cit> . in arabidopsis, there is another homologue  of this putative map. we named this homologue gpt <dig>  and the formerly identified map  <cit>  gpt <dig>  the amino acid sequences of gpt <dig> and gpt <dig>  share  <dig> % similarity and  <dig> % identity, and do not contain any defined domains with known functions. in this report, we demonstrate that these gpt proteins are novel plant-specific + tips that track growing mt plus ends independently of eb <dig> and spr <dig> \n",
            "\n",
            "RESULTS\n",
            "mt-binding regions in gpt1/2\n",
            "although gpt1/ <dig> do not exhibit any significant amino acid homology with functionally characterized proteins  <cit> , we noticed that the middle  and c-terminal  regions of these proteins are enriched in the basic amino acid residues arg, lys, and his . by contrast, the n-terminal  regions are abundant in acidic asp and glu . thus, gpts are polar proteins with short, negatively charged regions at their n-termini  and longer, positively charged regions in their middles and c-termini .fig.  <dig> the positively charged domains of gpt1/ <dig> bind mts in onion epidermal cells. full-length and truncated fragments of gpt1/ <dig> were fused to gfp at their c-termini, and transiently expressed in onion epidermal cells. a and d charge plots of gpt <dig>  and gpt <dig> . b and e protein structures of gpt <dig>  and gpt <dig> . the positions of basic amino acid residues are indicated by blue lines. the n-terminal  regions are negatively charged, whereas the middle  and c-terminal  regions are positively charged. c and f localization of gfp-fused gpt fragments to cortical mts is indicated by ++ , + , +/- , and – . the numerator and denominator show the number of cells with positive localization patterns and the number of total cells that expressed gfp fusion proteins, respectively. the gpt2-gfp fusion  was co-expressed with tagrfp-map <dig> to visualize mts. left panels, gpt2-gfp; middle panels, tagrfp-map4; right panels, merged images. scale bars, 10 μm\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "to identify mt-binding regions in these proteins, we fused full-length and truncated versions of gpt <dig> and gpt <dig> to gfp, and transiently expressed these fusions in onion epidermal cells, together with the red fluorescent mt marker tagrfp-map <dig> . dual color visualization of gpt1-gfp and tagrfp-map <dig> proved difficult, possibly due to gpt <dig> having a weak mt-binding capacity, competition between gpt <dig> and map <dig> for overlapping mt binding sites, or both. therefore, the mt-binding capacity of gpt1-gfp was determined based on the presence of fine filaments . colocalization of the gfp and rfp signals confirmed that full-length gpt <dig> localized to cortical mts . the n-terminal  regions of gpt <dig>  and gpt <dig>  did not localize to mts, whereas the n-terminal-deleted   and the middle   fragments were clearly localized to cortical mts . the c-terminal  fragments  were somewhat associated with mts, but to a lesser extent than the fragments containing the m region. these results from transient expression assays indicate that the positively charged basic amino acid residues  mediate the binding of gpt1/ <dig> to mts in vivo.\n",
            "\n",
            "mt co-sedimentation assay\n",
            "to test whether gpt1/ <dig> directly bind to mts, we fused full-length gpt <dig> and gpt <dig> to maltose-binding protein , expressed the recombinant proteins in bacteria, and purified the proteins by affinity purification using amylose resins. some degradation occurred during the purification, particularly in the case of mbp-gpt <dig>  when 1 μm of purified proteins was incubated with mts assembled from bovine brain tubulin and pelleted by ultracentrifugation, intact mbp-gpt <dig> and mbp-gpt <dig> were identified in the microtubule pellet fraction  . the partial degradation products of mbp-gpt <dig>  were also pelleted, whereas further degraded mbp-gpt <dig> and mbp-gpt <dig> fragments were not. in the absence of mts, mbp-gpt <dig> and mbp-gpt <dig> remained in the supernatant . mbp alone was also evaluated to verify that it is not responsible for binding of gpt1/ <dig> to mts. as shown in fig.  <dig>  mbp remained in the supernatant fraction after ultracentrifugation. these results show that gpt1/ <dig> bind mts directly.fig.  <dig> recombinant gpt <dig> and gpt <dig> bind to mts in vitro.  purified mbp-fused gpt <dig> and gpt <dig> proteins were incubated with or without taxol-stabilized mts, pelleted by ultracentrifugation, and analyzed by sds-page and coomassie brilliant blue staining. the positions of full-length mbp-gpt <dig> and mbp-gpt <dig> are indicated by arrowheads, and the mt-binding partial degradation products of mbp-gpt <dig> are indicated with an asterisk. tubulin is also indicated. s, supernatant fraction; p, pellet fraction; and m, size markers.  quantitative analysis of the binding of mbp-gpt <dig> to mts. various concentrations of purified mbp-gpt <dig> was mixed with taxol-stabilized mts and then subjected to co-sedimentation analysis, as in a. assuming that there is one gpt <dig> binding site on the tubulin dimer, the binding equation  was fitted\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "to determine the stoichiometry and affinity of gpt toward mts, various amounts of mbp-gpt <dig> were centrifuged with a constant amount of mts that were polymerized from 1 μm tubulin heterodimer, and a binding curve was obtained . mbp-gpt <dig> bound to mts in a concentration-dependent manner and saturated at a stoichiometry of approximately  <dig>  +/−  <dig>  mol of gpt <dig> per mol of tubulin dimer. the dissociation constant kd was calculated to be  <dig>  +/−  <dig>  μm. however, if gpt <dig> associates with mts in more than one binding mode with different affinities , this simple regression analysis does not provide true binding values.\n",
            "\n",
            "subcellular localization\n",
            "to determine the subcellular localization of gtp1/ <dig> and to monitor their dynamics in vivo, gfp was fused to either the n-terminus or the c-terminus of gpt <dig> and expressed under the constitutive cauliflower mosaic virus 35s promoter, because the 5′-regulatory region of gpt <dig> was recalcitrant to cloning, whereas gfp was fused to the c-terminus of gpt <dig> and expressed under its native promoter. both n-terminal and c-terminal gfp fusions of gpt <dig> localized to mts in transient expression assays using onion epidermal cells , indicating that the location of gfp does not affect the subcellular localization of fusion proteins. the arabidopsis mt marker line that expresses mcherry fluorescent protein fused to β-tubulin  <dig>  was used as a transformation host.\n",
            "\n",
            "gfp-gpt <dig> and gtp2-gfp co-localized with mcherry-tub <dig> and decorated the nuclear envelope and preprophase bands, mitotic spindles, and phragmoplasts of the mitotic cells of the root meristem . gfp-gpt <dig> labeling differed slightly from mcherry-tub <dig> labeling, particularly in expanding phragmoplasts. this observation prompted us to carefully examine the localization and dynamics of gpt <dig> and gpt <dig> in cortical mt arrays, where the dynamics of single mts can be clearly visualized.fig.  <dig> gpt1/ <dig> decorate mitotic mt structures in arabidopsis root epidermal cells. gfp fusions  were expressed in transgenic arabidopsis plants harboring the mcherry-tub <dig> mt marker . epidermal cells in the cell division zone of the roots of seedlings were analyzed. dividing cells at preprophase, metaphase, anaphase, and cytokinesis are shown. a gfp-gpt <dig> was expressed under the control of the cauliflower mosaic virus 35s promoter. b gpt2-gfp was expressed under the control of the endogenous regulatory elements of gpt <dig>  scale bars, 5 μm\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "gpt <dig> and gpt <dig> are novel + tips\n",
            "time-lapse imaging of root epidermal cells using a spinning disk confocal microscope revealed that gfp-gpt <dig> and gpt2-gfp not only decorated the mt lattice, but also accumulated as particles on mts . double intensity plots with mts  and gfp-labeled gpt1/ <dig> showed that gpt1/ <dig> are localized to the mt ends in a comet-like pattern, with the highest signal intensity at the mt ends and a gradual decline in signal further from the mt ends. this mt-end localization was more pronounced for gfp-gpt <dig> than for gpt2-gfp. the mt ends labeled with gfp were highly dynamic, indicating that gpt1/ <dig> both label the plus ends of cortical mts. gfp-gpt <dig> was associated with polymerizing plus ends, and dissociated as soon as catastrophe occurred . however, when the mts started to polymerize again, gfp-gpt <dig> was immediately recruited to the growing ends . gpt2-gfp showed a similar but weaker labeling pattern when compared with gfp-gpt <dig> . these results demonstrate that gpt <dig> and gpt <dig> preferentially recognize the plus ends of growing mts.fig.  <dig> gpt <dig> tracks the growing plus ends of cortical mts. a the subcellular localization of gfp-gpt <dig>  and mcherry-tub <dig>  which labels cortical mts , was analyzed in interphase cells of the arabidopsis root epidermis. scale bar, 5 μm. b average fluorescence intensity  profiles of gfp-gpt <dig> and mcherry-tub <dig> were obtained by analyzing and plotting data from  <dig> mt images. the data were normalized and the peak intensity of gfp-gpt <dig> was set to  <dig>  the error bars indicate sem. c representative time-lapse sequence and corresponding kymograph of a mt that underwent catastrophe at t = 14 s. gfp-gpt <dig> disappeared from the tip immediately after the mt began to depolymerize. dashed yellow arrows indicate the point at catastrophe occurred. scale bar,  <dig>  μm. d representative time-lapse sequence and corresponding kymograph of a mt that underwent rescue at t = 12 s. gfp-gpt <dig> localized to the tip immediately after mt growth resumed. dashed yellow arrows indicate the point at which rescue occurred. scale bar,  <dig>  μm\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "additional file 3: movie s <dig>  dynamics of cortical mts  and gfp-gpt <dig>  in epidermal cells of the roots of arabidopsis seedlings. the image sequence corresponds to fig. 4a. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "additional file 4: movie s <dig>  dynamics of cortical mts  and gpt2-gfp  in epidermal cells of the cotyledons of arabidopsis seedlings. the image sequence corresponds to additional file 2: figure s2a. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "eb <dig> is a + tip family member that recognizes the gtp-cap region of growing mts  <cit> . to compare the labeling patterns of gpt1/ <dig> with that of eb <dig>  we stably co-expressed gfp-labeled gpt <dig> or  <dig> and mcherry-labeled arabidopsis eb1b  in arabidopsis plants. eb1-mcherry showed strong plus-end labeling, as previously reported . many, but not all, eb <dig> particles co-localized with gfp-gpt <dig> . when the relative fluorescent intensities of gfp and mcherry were plotted, the comet-shaped fluorescent signals of gfp-gpt <dig> and eb1-mcherry partially overlapped. in our confocal microscopy setup, the two emitted fluorescent signals from gfp and mcherry were sequentially detected after alternately exciting each fluorophore. when mcherry fluorescence was detected for  <dig>  s, followed by the detection of gfp signal for  <dig>  s, the highest gfp-gpt <dig> signal intensity was located approximately  <dig>  μm closer to the mt end when compared with the highest eb1-mcherry signal . when the detection order was reversed, however, the highest eb1-mcherry signal intensity was located approximately  <dig>  μm in front of the highest signal intensity of gfp-gpt <dig> . at the current spatial resolution, optical artifacts associated with our detection system make it difficult to conclusively determine whether eb <dig> and gpt <dig> completely co-localize at growing mt plus ends. in both observations, the fluorescent signal intensities of the eb <dig> comets decreased gradually and reached background levels at  <dig>  μm behind the tips. by contrast, substantial levels of gfp-gpt <dig> signal  remained associated with the mt lattice behind the comet tails.fig.  <dig> gpt <dig> and eb <dig> label similar regions of growing mt ends. a subcellular localization of gfp-gpt <dig>  and eb1-mcherry  was analyzed in interphase cells of the arabidopsis root epidermis. scale bar, 5 μm. b representative time-lapse sequences. left, mcherry signals were analyzed first, followed by gfp signals. right, gfp signals were analyzed first, followed by mcherry signals. scale bars,  <dig>  μm. c and d average fi profiles of gfp-gpt <dig> and eb1-mcherry were obtained by analyzing and plotting data from  <dig> mt images. the data were normalized and the peak intensities of gfp-gpt <dig> and eb1-mcherry were set to  <dig>  the error bars indicate sem. c mcherry signals were analyzed first, followed by gfp signals. d gfp signals were analyzed first, followed by mcherry signals\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "additional file 5: movie s <dig>  dynamics of eb1-mcherry  and gfp-gpt <dig>  in epidermal cells of the roots of arabidopsis seedlings. in one image acquisition, mcherry fluorescence was first recorded for  <dig>  s and then gfp fluorescence was recorded for  <dig>  s. the image sequence corresponds to fig. 5a. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "tip-tracking of gpt does not require eb <dig> or spr1\n",
            "eb <dig> recognizes a structural feature of the mt gtp cap and directly binds to the plus ends of growing mts both in vitro and in vivo  <cit> . many animal + tips do not bind directly to the mt plus ends in vivo, but are recruited to this region by eb <dig>  to test whether gpt accumulates at the mt plus end directly or through a hitchhiking mechanism involving interaction with eb <dig>  we analyzed the tip-tracking behavior of gpt in the arabidopsis eb <dig> null mutant. because the arabidopsis genome includes three eb <dig> genes that may function redundantly  <cit> , a triple eb <dig> knockout mutant was used to study the expression of gfp-gpt <dig> and gpt2-gfp. gpt1/ <dig> both localized to the plus ends of growing mts, in the same pattern as observed in the wild-type background .fig.  <dig> gpt <dig> does not require eb <dig> or spr <dig> to track the mt end. a and b gfp-gpt <dig> was expressed in the arabidopsis eb1a eb1b eb1c triple mutant. a the subcellular localization of gfp-gpt <dig> was analyzed in interphase cells of the arabidopsis root epidermis. scale bar, 5 μm. b an average fi profile of gfp-gpt <dig> was obtained by analyzing and plotting data from  <dig> mt images. the data were normalized and the peak intensity of gfp-gpt <dig> was set to  <dig>  the error bars indicate sem. c and d gfp-gpt <dig> was expressed in the spr <dig> mutant that also expressed mcherry-tub <dig> as a mt marker. c the subcellular localization of gfp-gpt <dig> and mcherry-tub <dig> was analyzed in interphase cells of the arabidopsis root epidermis. scale bar, 5 μm. d average fi profiles of gfp-gpt <dig> and mcherry-tub <dig> were obtained by analyzing and plotting data from  <dig> mt images. the data were normalized and the peak intensity of gfp-gpt <dig> was set to  <dig>  the error bars indicate sem\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "additional file 7: movie s <dig>  dynamics of gfp-gpt <dig> in root epidermal cells of the eb <dig> triple mutant in arabidopsis. the image sequence corresponds to fig. 6a. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "additional file 8: movie s <dig>  dynamics of gpt2-gfp in cotyledon epidermal cells of the eb <dig> triple mutant in arabidopsis. the image sequence corresponds to additional file 6: figure s3a. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "spr <dig> and its homologues are plant-specific mt-localized proteins that bind to both the growing plus ends and the mt lattice  <cit> . arabidopsis contains seven spr <dig> homologues that may be redundant, and spr <dig> contributes predominantly to the anisotropic growth of seedling roots  <cit> . therefore, we used a spr <dig> null mutant to test whether spr <dig> is required for the plus-end tracking behavior of gpt. cortical mts were labeled with mcherry-tub <dig>  in the spr <dig> mutant, gpt1/ <dig> labeled growing mt plus ends in interphase epidermal cells, forming a comet-like pattern . the tip-tracking behavior of the gfp-labeled gpts in the spr <dig> null mutant was indistinguishable from that in wild-type epidermal cells.\n",
            "\n",
            "\n",
            "\n",
            "additional file 9: movie s <dig>  dynamics of cortical mts  and gfp-gpt <dig>  in root epidermal cells of the spr <dig> mutant in arabidopsis. the image sequence corresponds to fig. 6c. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "additional file 10: movie s <dig>  dynamics of cortical mts  and gpt2-gfp  in root epidermal cells of the spr <dig> mutant in arabidopsis. the image sequence corresponds to additional file 6: figure s3c. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "these results demonstrate that eb <dig> and spr <dig> are not required for the mt plus-end tracking function of gpt <dig> and gpt <dig> \n",
            "\n",
            "discussion\n",
            "gpt1/ <dig> bind to mts through its basic regions\n",
            "with the exception of the n-terminal regions, gpt1/ <dig> are relatively rich in lys, arg, and his residues, with gpt1/ <dig> having average pi values of  <dig>  and  <dig> , respectively. these positively charged residues are distinct biochemical features of gpts; however, these values are lower than those of the highly basic domains of some maps, such as bpp <dig>  bpp <dig>  and bpp <dig>  which have pi values of  <dig> – <dig>   <cit> . transient expression of gfp-tagged full-length and truncated gpt1/ <dig> proteins in onion epidermal cells showed that the basic regions of gpt1/ <dig> are responsible for the in vivo localization to cortical mts. the central and c-terminal basic regions were independently targeted to mts in vivo, suggesting that several mt-binding regions exist in gpt1/ <dig>  because recombinant gpt1/ <dig> proteins bind to mts in vitro, the basic regions appear to bind mts directly.\n",
            "\n",
            "various mt-binding regions have been identified in maps, and a high pi value resulting from basic amino acid residues was found to be characteristic of the mt-binding regions of certain maps  <cit> . the positively charged regions are thought to interact electrostatically with the highly acidic glu- and asp-rich c-terminal tails of α- and β-tubulins. because an acidic c-terminus is a common feature of all eukaryotic tubulins  <cit> , this charge-based mt-binding mode is frequently used by maps in various organisms.\n",
            "\n",
            "gpt preferentially accumulates at the plus ends of growing mts\n",
            "gfp-tagged gpt not only labels the mt lattice, but is considerably enriched at the growing plus ends of cortical mts. this plus-end localization is stronger for gfp-gpt <dig> than for gpt2-gfp. we do not know whether the affinity for the plus end is inherently different between gpt1/ <dig>  or whether the location of the fused gfp molecule influenced the biochemical properties of gpt1/ <dig>  although gpt1/ <dig> have highly similar distributions of basic amino acids and considerable levels of amino acid sequence identity, we noticed that gpt1-gfp associates less efficiently with mts than does gpt2-gfp when a mt marker is co-expressed in onion epidermal cells. further biochemical analyses are required to identify possible differences between mt plus-end targeting by gpt1/ <dig> \n",
            "\n",
            "the tip-tracking behavior of gpt1/ <dig> and the canonical + tip, arabidopsis eb <dig>  was compared by coexpressing these proteins tagged with different fluorescent markers in arabidopsis plants. eb <dig> specifically recognized the growing mt plus end and dissociated almost completely at  <dig>  μm from the mt tip, where there are no binding sites for eb <dig> . by contrast, the gpt1/ <dig> tip-tracking fluorescent signal decreased with increasing distance from the mt plus end, but a substantial proportion of gpt1/ <dig> molecules remained bound to the gdp-tubulin mt lattice. this propensity of gpt1/ <dig> to bind to the mt lattice may obstruct observations of tip-localized gpt1/ <dig> in dense mitotic mt arrays. although further analysis at higher resolution is necessary, our present localization data suggest that eb <dig> and gpt <dig> have distinct binding sites at growing mt plus ends. eb <dig> recognizes the terminal gtp-cap and an extended region of several hundreds of tubulin molecules at growing mt ends  <cit> . recent imaging of eb <dig> localization using cryo-electron tomography showed that eb <dig> interacts with outwardly curved and straight regions of the mt lattice that probably represent gtp- and gdp-pi-tubulin dimers, respectively, at the mt ends  <cit> . ebs bind at the intersection of four tubulin dimers in two adjacent protofilaments  <cit> . this binding site is ideally suited for sensing the nucleotide states of the surrounding tubulin dimers. to autonomously track the growing mt ends, a + tip should recognize subtle structural changes of tubulins that are associated with the nucleotide status and the protofilament closure into a cylinder. the predicted gpt-binding sites at the acidic tubulin c-terminus would be insufficient for such a recognition mechanism, and may require other components that associate with mts, such as the n-terminal non-basic regions.\n",
            "\n",
            "the efficient dissociation of eb <dig> from the mt lattice is partly caused by the negatively charged c-terminus of eb <dig>  when this c-terminus was replaced with a neutrally charged motif, the eb <dig> mutant decorated the mt lattice without affecting its interaction with the growing mt ends  <cit> . the biochemical properties of gpt1/ <dig> may render these molecules less able than eb <dig> to discriminate between mt ends and the lattice.\n",
            "\n",
            "eb <dig> is not the only + tip that autonomously interacts with the mt ends. xenopus laevis xmap <dig>  recognizes mt plus ends, but does not distinguish between polymerizing and depolymerizing ends  <cit> . xmap215/ch-tog binds to the extreme mt tip, whereas eb <dig> localizes to the tip region several tens of nanometers behind xmap215/ch-tog in vivo and in vitro  <cit> . +tips that recognize distinct structural features at the plus ends would be expected to decorate different sub-regions at the mt ends.\n",
            "\n",
            "gpt does not require eb <dig> or spr <dig> for tip tracking\n",
            "eb <dig> recruits non-autonomous + tips via their c-terminal eb homology domain to growing mt ends  <cit> . cap-gly domains  <cit>  and sxip motifs  <cit>  have been identified in eb1-interacting proteins that accumulate at growing mt ends. interestingly, gpt1/ <dig> still tracked the mt ends in the arabidopsis eb <dig> triple mutant background, which lacked eb <dig> function. the absence of cap-gly domains and sxip motifs in gpts suggests that their subcellular localization is independent of eb <dig>  gpt1/ <dig> localization to mt ends also does not require spr <dig>  another plant-specific + tip  <cit> . although it is not known whether spr <dig> autonomously tracks growing mt ends, these results suggest that if gpt does indeed use a hitchhiking mechanism to bind to a growing mt tip, it must couple with a + tip protein other than eb <dig> or spr <dig> \n",
            "\n",
            "CONCLUSIONS\n",
            "arabidopsis maps, gpt <dig> and gpt <dig>  robustly track the plus end of growing mts, independently of eb <dig> and spr <dig>  and thus define a novel plant-specific + tip family.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "gpt1/ <dig> and gpt2-gfp are novel plant-specific + tip proteins that track growing mt ends independently of eb <dig> and spr <dig>  and thus define a novel plant-specific + tip family.\n",
            "gpt1/ <dig> and gpt2-gfp are novel plant-specific + tip proteins that track growing mt ends independently of eb <dig> and spr <dig>  and thus define a novel plant-specific + tip family.\n",
            "gpt1/ <dig> and gpt2-gfp are novel plant-specific + tip proteins that track growing mt ends independently of eb <dig> and spr <dig>  and thus define a novel plant-specific + tip family.\n",
            "gpt1/ <dig> and gpt2-gfp are novel plant-specific + tip proteins that track growing mt ends independently of eb <dig> and spr <dig>  and thus define a novel plant-specific + tip family.\n",
            "gpt1/ <dig> and gpt2-gfp\n",
            "68\n",
            "[{'summary_text': 'salmonella shedding associated modules were significantly higher in ls than ps at day  <dig>  and significantly higher in ps at day  <dig> compared to day  <dig> p.\\nhowever, the relative functional importance of genes within salmonella shedding associated modules in ls compared to ps even before salmonella challenge may be related to the decreased faecal salmonella shedding observed in ls following challenge.\\nsalmonella shedding associated modules were significantly higher in ls than ps at day  <dig>  and significantly higher in ps at day  <dig> compared to day  <dig> p.\\nsalmonella shedding associated modules were significantly higher in ls than ps at day  <dig>  and significantly higher in ps at day  <dig> p.\\nsalmonella shedding associated modules were significantly up-regulated in ls compared to ps at day  <dig> p.\\nsalmonella shedding associated modules were significantly up-regulated in ls compared to ps at day  <dig> p.\\nsalmonella shedding associated modules were significantly up-regulated in ls compared to ps at day  <dig> p.\\nsalmonella shedding associated modules were significantly up-regulated in ls compared to ps'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 8454 to 9216 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "salmonella enterica serovar typhimurium is a gram-negative zoonotic bacterium that can colonise the gut of humans and many species of food producing farm animals and cause enteric or septicaemic salmonellosis \n",
            " <cit> . in pigs, infections by s. typhimurium mostly lead to a localised enterocolitis, which is responsible for significant economic losses to the pig industry \n",
            " <cit> . an unknown percentage of salmonella infected pigs continue to be asymptomatic carriers even after acute response, thereby posing long-term zoonotic threats through contaminating the pork production chain. prevention and control of salmonellosis in pigs thus assumes great importance not only for animal welfare, reduced antibiotic use and improved profitability of pig industry but also for minimizing risks to public health \n",
            " <cit> .\n",
            "\n",
            "host genetic response to salmonella infection has been well studied in several species. a review by roy and malo \n",
            " <cit>  has reported several genes to be involved in regulation of responses to salmonella infection in mice, for example, slc11a <dig> , tlr <dig>  btk, lbp, cd <dig>  cybb, nos <dig>  tnf, il <dig>  ifng, il12b, tlr <dig> and others. clinical manifestations associated with salmonella infection are dependent on several factors such as the serotype of salmonella involved, host species affected and age of the host. while infection with s. typhimurium induces a systemic disease similar to human typhoid fever in mice, the infection is mostly of the enteric form in pigs, except in the case of very young piglets \n",
            " <cit> . thus a different set of genes may contribute to resistance against salmonella infection depending on the host species and the salmonella serotype involved. indeed, studies of salmonella resistance in chicken report different genes depending on whether the infection is systemic or enteric \n",
            " <cit> . for example, the slc11a <dig> gene and the sal <dig> locus confer resistance to systemic salmonellosis \n",
            " <cit> , whereas several members of the gallinacin gene family confer resistance to enteric salmonellosis \n",
            " <cit> .\n",
            "\n",
            "studies on several species have shown that host genetic variations in natural populations contribute to varying responses to different pathogens in terms of resistance or increased susceptibility \n",
            ". distinct responses to salmonella infection have been observed in pigs, some recovering faster and shedding lower levels of salmonella in faeces than others  \n",
            " <cit> . this trait variation could indicate the existence of a genetic component to salmonella shedding and resistance that may be exploited in animal breeding and disease diagnostics. uthe et al. \n",
            " <cit>  reported snps in ten genes, including gng <dig>  ncf <dig> and ccr <dig>  that were associated with salmonella shedding in pigs. while many studies have looked into the host genetic response to salmonella infection, relatively few have used trait-gene expression correlation to identify genes whose variable expression among uninfected individuals may be associated with differences in salmonella clearance and resistance. for example, several genes involved in innate immunity  have been shown to be either constitutively or inductively more highly expressed in caecal tonsils of salmonella resistant strains of chicken compared to susceptible strains \n",
            " <cit> . a study on mycobacterium tuberculosis infection in humans identified a snp that appears to control susceptibility to tuberculosis through its effects on the expression of the dusp <dig> gene in dendritic cells prior to infection \n",
            " <cit> . differences in gene expression prior to porcine reproductive and respiratory syndrome  virus infection have been observed among pigs belonging to different phenotypic groups as defined by the viral load and weight gain of individual pigs observed during a defined number of days following infection \n",
            " <cit> . such differences in gene expression among different phenotypic groups have been attributed to the differences in the genetic background of individual pigs \n",
            " <cit> . taken together, the findings reported above may indicate that genetic resistance to infections is mediated in part through the presence of a more activated defense system in resistant individuals compared to susceptible individuals so that, in the event of an infection, resistant individuals can mount a faster and more effective immune response.\n",
            "\n",
            "in this study we assess the whole blood transcriptome prior to and following infection. whole blood consists of cells that form integral parts of the immune system and whole blood can be easily and repeatedly sampled. the whole blood transcriptome provides a ‘snap shot’ of the complex immune networks that operate throughout the body \n",
            " <cit>  and several studies in humans and animals have analysed the blood transcriptome to provide new insights into host immune responses against a wide variety of pathogens and to identify potential biomarkers \n",
            " <cit> .\n",
            "\n",
            "a recent microarray based study \n",
            " <cit>  identified several hundreds of differentially expressed  genes including slc11a <dig> and tlr <dig> when comparing the whole blood transcriptome of pigs before  and two days after salmonella inoculation  and also reported significant differences in the number and expression profiles of de genes post inoculation in ls compared to ps. however, the de analysis performed in those studies failed to identify significant differences in expression of genes between ls and ps before salmonella challenge that could potentially explain or predict the varying responses between the two groups upon infection. this absence of de could be due to the fact that the differences in expression of host resistance genes against salmonella between ls and ps may be subtle, unlike the gene expression differences between non-infected and infected states, and thus requires a broader or more sensitive approach to identify those genes.\n",
            "\n",
            "de analysis typically focuses on identifying genes with the most significant differences between contrasting conditions. in addition, the requirement for multiple testing corrections may further impede the discovery of genes with subtle differences in expression. a powerful alternative approach for gene expression analysis is co-expression analysis, which considers the relationships between measured transcripts in an unsupervised way. the weighted gene co-expression network analysis  approach \n",
            " <cit> , one of the popular methods developed for gene co-expression analysis, effectively overcomes the multiple testing problems inherent in high throughput gene expression data analysis. this methodology begins with the identification of modules of highly correlated genes assessed by pair-wise correlations between gene expression profiles. next the relationships between only a few tens of modules and the phenotypic trait of interest are considered. finally candidate genes associated with the trait are prioritised based on network statistics like module membership and gene significance. wgcna has been used to identify genes and gene networks associated with specific tissues, distinct biological states or diseases, and qualitative as well as quantitative phenotypes \n",
            ".\n",
            "\n",
            "here, we aimed to identify porcine genes and gene co-expression networks that differentiate distinct responses to salmonella challenge with respect to faecal salmonella shedding, using wgcna and rna-seq data from whole blood.\n",
            "\n",
            "RESULTS\n",
            "faecal salmonella shedding counts\n",
            "the pigs used in this study were identified as ls or ps based on the cumulative area under the plotted log curve  of their faecal salmonella shedding counts  . for ls, the aulc ranged from  <dig>  to  <dig>   whereas for ps, it ranged from  <dig>  to  <dig>  . the lower the aulc, the lower the faecal shedding counts estimated for that pig. we predict that a lower early shedding count will accelerate the return of the animal to a healthy  status following salmonella challenge.table  <dig> \n",
            "shedding status of pigs determined based on aulc of faecal\n",
            "salmonella\n",
            "shedding counts\n",
            "\n",
            "\n",
            "*aulc: cumulative area under the plotted log curve of the logarithmically normalised faecal salmonella shedding counts obtained between day  <dig> to day  <dig> post-inoculation for each individual.\n",
            "\n",
            "\n",
            "\n",
            "rna-seq profiling of porcine whole blood expressed genes and their functional classification\n",
            "rna extracted from porcine peripheral blood samples collected at day  <dig> and day  <dig> post inoculation  of salmonella from  <dig> selected pigs identified as ls  and ps  were depleted for globin transcripts and subjected to illumina sequencing. the sequencing depths achieved for these samples ranged from  <dig> to  <dig> million reads , of which approximately 90%  mapped to the pig genome , identifying a total of  <dig>  expressed genes. the efficiency of the globin depletion protocol varied across samples with the total globin reads  ranging from  <dig> -60% of the total mapped reads post-depletion . based on data from a different set of samples  that were used to develop the globin depletion protocol \n",
            " <cit>  , it is known that the total globin reads constitute  <dig> to 54%  of the total mapped reads in blood of pigs between 3– <dig> weeks of age. using this information on normal globin levels in pig blood, we estimate the globin depletion protocol to have worked well for  <dig> of the  <dig> samples used here where the total globin reads constituted below 15% of the total mapped reads. for  <dig> of the  <dig> samples, the protocol seemed to have worked to some extent, limiting total globin reads to between 20-30% of the total mapped reads. for the remaining  <dig> samples, however, the protocol either worked poorly  or did not work at all . the differing efficiencies of globin depletion would have affected our ability to detect genes expressed at very low levels via improved coverage for non-globin genes. however, this would not affect the analysis we do here as we filter out the genes expressed at very low levels across all samples. the globin genes were also removed from the gene expression matrix prior to normalisation and further analysis. removal of genes expressed at very low levels  per sample >  <dig> in at least  <dig> samples) resulted in a set of  <dig>  genes expressed in porcine peripheral blood. a multi-dimensional scaling in two dimensions  performed using this set of expressed genes revealed a clear separation of the samples by day  but not by shedding status  on either day .\n",
            "\n",
            "for a broad overview of the functions attributed to the genes expressed in porcine blood, we performed a functional classification based on gene ontology  terms from go slim using the panther gene list analysis tool . in the biological processes category ,  <dig> % of the blood-expressed genes were annotated to metabolic process, 16% to cellular process,  <dig> % to cell communication and  <dig> % to transport. in the molecular functions category,  <dig> % of the genes were annotated to binding and  <dig> % to catalytic activity whereas in the cellular composition category, the dominant term was intracellular .\n",
            "\n",
            "gene co-expression network constructed using gene expression data from uninfected pigs\n",
            "gene expression profiles of day  <dig> samples of pigs were analysed using wgcna in order to identify gene co-expression patterns that may be associated with differences in faecal salmonella shedding counts of those pigs upon future challenge. after setting a minimum module size of  <dig> and merging modules with highly correlated  eigengenes , a total of  <dig> modules were found . these modules will be referred to by their colour labels henceforth, as is standard practice. the modules labelled by colours are depicted in the hierarchical clustering dendrogram provided in figure \n",
            "1a. to evaluate the robustness of the identified modules, we performed module stability analysis from bootstrapped networks . the module representations were fairly consistent across the majority of the bootstrapped networks .figure  <dig> \n",
            "module identification in day  <dig> samples and correlation of module expression with\n",
            "salmonella\n",
            "shedding counts. a. clustering dendrogram of genes showing module membership in colours. b. module sizes and correlation of module eigengenes with trait, aulc of faecal salmonella shedding counts. the module-wise correlations are provided along with the p-value of the correlation in cells coloured by the strength of the correlation. modules significantly associated with the trait  are indicated with an asterisk . c. scatterplots of gene significance versus module membership for salmonella shedding associated modules, along with correlations and p-values indicated. d. gene co-expression patterns within salmonella shedding associated modules. the samples are sorted from low to high based on shedding counts.\n",
            "\n",
            "\n",
            "\n",
            "modules associated with differences in faecal salmonella shedding\n",
            "to determine if any of the  <dig> modules of co-expressed genes identified in day  <dig> samples were associated with the observed differences in faecal salmonella shedding counts upon future salmonella challenge, we tested the correlations of the module eigengenes with the trait, aulc of faecal salmonella shedding counts. four modules were found significant at the defined cut-offs . of these, the pink, grey <dig> and darkorange modules showed negative correlations with the trait while the magenta module showed a positive correlation. figure \n",
            "1b depicts the module sizes and correlations of the module eigengenes to the trait. the module membership  versus gene significance  plots for these modules  showed that mm and gs are highly correlated, indicating that genes most significantly associated with the trait are often also the most important  elements of the respective modules. here, mm is a measure of the strength of a particular gene’s membership in a module obtained by correlating the gene’s expression profile with the module eigengene of that module. highly connected intramodular hub genes tend to have high mm values to the respective module. gs is a measure of the biological relevance of a gene with respect to the trait of interest obtained by correlating the gene’s expression profile with the trait. the gene expression profiles of genes within the four modules associated with salmonella shedding, across all samples ordered by aulc of salmonella shedding counts , indicate tight co-regulation with an overall higher expression in ls than ps for the pink, grey <dig> and darkorange modules and a lower expression in ls than ps for the magenta module. gene ontology enrichment tests  revealed that the grey <dig> and pink modules were related to immune functions whereas the darkorange module was related to signalling processes . the positively correlated magenta module was not significantly enriched for any go term. the ensembl ids of the genes within these four modules are provided in additional file \n",
            " <dig> and the complete lists of go terms enriched in these modules are provided in additional file \n",
            " <dig> \n",
            "\n",
            "salmonella shedding associated modules identified at day  <dig> are largely preserved at day 2\n",
            "we speculated that the salmonella shedding associated modules found at day  <dig> may be preserved at day  <dig> and have a majority of their constituent genes still co-expressed if they indeed are involved in response to salmonella challenge. therefore, we first constructed a co-expression network using day  <dig> samples and tested for module preservation between day  <dig> and day  <dig> using the modulepreservation method \n",
            " <cit>  implemented in wgcna package that calculates zsummary preservation scores. a zsummary score below  <dig> indicates no preservation, a score above  <dig> indicates strong preservation and a score between  <dig> to  <dig> indicates weak to moderate evidence of preservation \n",
            " <cit> . most of the day  <dig> modules  were seen to be strongly preserved at day  <dig> including the pink and magenta modules , eight were weakly to moderately preserved including the grey <dig> and darkorange modules and only one was not preserved.figure  <dig> \n",
            "module preservation between day  <dig> and day  <dig> modules. coloured circles denote the zsummary preservation scores of day  <dig> modules with day  <dig> modules. a zsummary score below  <dig> indicates no preservation, a score above  <dig> indicates strong preservation and a score between  <dig> to  <dig> indicates weak to moderate evidence of preservation.\n",
            "\n",
            "\n",
            "\n",
            "differences in expression levels of salmonella shedding associated genes between low and persistent shedders and between days\n",
            "de analyses between day  <dig> and day  <dig> performed separately for ls and ps samples identified  <dig> genes de  in ls and  <dig> genes de  in ps , with an overlap of  <dig>  genes. however, only  <dig> genes were significantly de between ls and ps at day  <dig>  the low number of genes de between ls and ps at day  <dig> was due to the fact that the direction of change was similar for most genes in both groups, though the degree of change in expression between days were higher in ps. from the expression patterns of the modules of co-expressed genes at day  <dig> that were associated with differences in faecal salmonella shedding counts, it was observed that genes within the grey <dig>  pink and darkorange modules were, in general, more highly expressed in ls than ps at day  <dig> whereas the opposite trend was observed for genes within the magenta module . if the genes within the modules are influencing salmonella shedding and thereby pathogen clearance and host resistance, one might hypothesise that genes within the modules negatively associated with salmonella shedding at day  <dig> would have a general up-regulation post-inoculation and those within the positively associated module would have a general down-regulation. to test this hypothesis, we first determined genes differentially expressed  between day  <dig> and day  <dig> p.i. using samples from both ls and ps taken together. a total of  <dig> genes were found de  of which  <dig> were up-regulated and  <dig> down-regulated at day  <dig> p.i. . approximately 60% of the genes in both the pink  and grey <dig>  modules were up-regulated at day  <dig> p.i and  <dig> and  <dig> genes were down-regulated in the two modules, respectively. for genes in the darkorange module, 36%  were found significantly up-regulated at day  <dig> p.i. and none down-regulated. even when considering all genes in the three modules , the majority were more highly expressed at day  <dig> p.i. compared to day  <dig> . thus, the majority of genes within the modules negatively associated with salmonella shedding counts had a higher mean expression in ls than ps at day  <dig> and were significantly up-regulated in both ls and ps at day  <dig> . however, the positively associated magenta module, in spite of showing a majority of genes expressed lower in ls than ps at day  <dig>  did not show a trend of overall down-regulation at day  <dig> . in this case, we observed  <dig> %  of the genes to be significantly down-regulated,  <dig> %  significantly up-regulated, and the majority of the non-de genes were more highly expressed at day  <dig> figure  <dig> \n",
            "differential expression of genes within\n",
            "salmonella\n",
            "shedding associated modules. expression levels of genes within salmonella shedding associated modules in low and persistent shedders at day  <dig> , and in day  <dig> and day  <dig> samples . the x-axis refers to the log <dig> transformed mean expression of genes within a particular module in the ls and ps samples  or in the day  <dig> and day  <dig> samples . the y-axis refers to the proportion of those genes whose log <dig> transformed mean expression falls below a particular expression level.\n",
            "expression profiles of selected genes from the\n",
            "salmonella\n",
            "shedding associated modules that were also up-regulated post-inoculation with\n",
            "salmonella. heat maps depicting the mean expression profiles of genes within salmonella shedding associated modules  that were up-regulated post-inoculation. in general, these genes had a higher mean expression in ls than ps at day  <dig>  a majority of the genes in magenta module were not differentially expressed between days and thus the association of this module with salmonella shedding counts may not be biologically relevant.\n",
            "\n",
            "\n",
            "\n",
            "differences in connectivity of salmonella shedding associated genes between low and persistent shedders and between days\n",
            "connectivity is a central concept in network statistics. in gene co-expression networks, connectivity measures how correlated a gene is with all other network genes. here we calculated the scaled mean connectivities of all genes for four separate networks constructed using the following four subsets of the gene expression dataset: ls at day  <dig>  ps at day  <dig>  ls at day  <dig> and ps at day  <dig>  and tested if the connectivities differed significantly between ls and ps  when considering either all genes in the networks or only those genes belonging to the salmonella shedding associated modules . the mean, median and maximum connectivities were all higher for networks from ps samples than those from ls networks, at both days, when considering all genes in the networks. however, the opposite trend was seen when considering only the salmonella responsive genes within each network. at day  <dig>  the network connectivity measures were significantly higher in ls compared to ps for genes within the pink  and grey <dig>  modules, not significantly different for the darkorange module genes and significantly lower in ls compared to ps for the magenta  module genes . at day  <dig>  the network connectivity measures were significantly higher in ls compared to ps for genes within all four salmonella shedding associated modules . though significant differences in connectivity were also seen between ls and ps for some of the other modules not associated with salmonella shedding both in the day  <dig> and day  <dig> networks, it was remarkable that the two immune function related modules, pink and grey <dig>  showed significantly higher connectivities within the ls than ps even before salmonella challenge.figure  <dig> \n",
            "differential connectivity of genes within\n",
            "salmonella\n",
            "shedding associated modules. connectivity measures of genes within salmonella shedding associated modules in low and persistent shedders at day  <dig> , and at day  <dig> . the connectivity measures were normalised to the maximum connectivity within each network. the x-axis refers to the scaled mean connectivities of genes within a particular module in the ls and ps samples. the y-axis refers to the proportion of those genes whose scaled mean connectivities falls below a particular connectivity level.\n",
            "\n",
            "\n",
            "\n",
            "prioritisation of candidate genes for salmonella shedding\n",
            "for prioritising candidate genes within the salmonella shedding associated modules that could possibly determine better immune responses to salmonella challenge, we restricted further analysis to the pink and grey <dig> modules. these modules were found to be immune-related, had a majority of genes significantly up-regulated at day  <dig>  were well preserved with the corresponding modules from the network constructed using day  <dig> samples, and also showed overall higher gene connectivity measures in ls compared to ps in networks from both days. a total of  <dig> genes in the pink module and  <dig> in the grey <dig> module were up-regulated at day  <dig> p.i. network properties of this refined set of candidate genes within these two modules are provided in additional file \n",
            " <dig>  this candidate gene list includes genes previously reported to be involved in the regulation of host responses to salmonella infection  or as having snps associated with salmonella faecal shedding , as well as those for which an association with salmonella is novel .\n",
            "\n",
            "we prioritised the genes within these refined sets of candidates based on module membership, gene significance and also connectivity in ls at day  <dig> since we speculated that the significantly higher gene connectivity in ls may have a biological significance related to salmonella shedding. the top candidates in these modules with module membership above  <dig> , gene significance below - <dig>  and connectivity in ls at day  <dig> above  <dig> , ordered by connectivity are presented in table \n",
            " <dig>  these top candidates are also among the most well connected genes in the network visualisations using the refined set of candidate genes within the pink  and grey <dig> modules .table  <dig> \n",
            "top candidate genes associated with\n",
            "salmonella\n",
            "shedding\n",
            "\n",
            "\n",
            "\n",
            "pink\n",
            "module\n",
            "\n",
            "ensembl id\n",
            "gene symbol\n",
            "gs\n",
            "k\n",
            "d0_ls\n",
            "k\n",
            "d0_ps\n",
            "k\n",
            "d2_ls\n",
            "k\n",
            "d2_ps\n",
            "mm\n",
            "folr1\n",
            "oscar\n",
            "c5ar1\n",
            "\n",
            "cda\n",
            "ano10\n",
            "\n",
            "arg2\n",
            "tlr2\n",
            "tnfrsf1a\n",
            "c5ar2\n",
            "siglec5\n",
            "alox5ap\n",
            "lreap1\n",
            "tmem120a\n",
            "\n",
            "grey60\n",
            "module\n",
            "\n",
            "ensembl id\n",
            "gene symbol\n",
            "gs\n",
            "k\n",
            "d0_ls\n",
            "k\n",
            "d0_ps\n",
            "k\n",
            "d2_ls\n",
            "k\n",
            "d2_ps\n",
            "mm\n",
            "bmx\n",
            "zcchc6\n",
            "pgk1\n",
            "igsf6\n",
            "ccr1\n",
            "tnfsf13b\n",
            "\n",
            "sdcbp\n",
            "srgn\n",
            "gng10\n",
            "ltb4r\n",
            "rnf149\n",
            "gs: gene significance, k d0_ls and k d0_ps: gene connectivity in the day  <dig> network using low shedder samples and persistent shedder samples respectively and similarly k d2_ls and k d2_ps for the day  <dig> network, mm: module membership. where the pig gene symbol is not available, the corresponding ortholog human gene symbol is provided.\n",
            "visualisation of the networks of selected modules associated with faecal\n",
            "salmonella\n",
            "shedding. a total of  <dig> genes in the pink module and  <dig> in the grey <dig> module that were up-regulated at day  <dig> post inoculation are depicted in networks a and b. nodes in the network are labelled by the corresponding pig/human ortholog gene symbols if available and pig ensembl ids if not. each node is coloured based on degree . to improve network visibility, the edges  have been filtered to show only those with a correlation weight above  <dig>  in the pink and  <dig>  in the grey <dig> module. the top candidate genes in each network as listed in table \n",
            " <dig> are displayed at the centre of the network with their edges coloured red.\n",
            "\n",
            "\n",
            "\n",
            "further, as an independent validation of our results, we compared the expression patterns of a subset of the candidate genes reported here with the corresponding expression patterns from a previous microarray based salmonella challenge study involving a different set of ls and ps animals \n",
            " <cit> . probes on the microarray represented  <dig>  of the top  <dig> candidate genes reported in table \n",
            " <dig>  except for slc26a <dig> and numb, all other genes showed expression patterns similar to those observed in our study: higher in ls than ps at day  <dig> and up-regulated in both ls and ps at day  <dig> p.i. in addition, the genes slc11a <dig>  tlr <dig> and cd <dig>  which are known to play important roles during salmonella infection, showed similar expression patterns as described above. figures comparing the expression patterns of these  <dig> genes between the two studies are provided in additional file \n",
            " <dig> \n",
            "\n",
            "discussion\n",
            "significant gains have been made in our understanding of host–pathogen interactions during salmonella infection \n",
            " <cit> . several studies, involving a variety of different species of farm and model animals, have investigated the host response to salmonella infection and have successfully identified genes differentially expressed upon infection or gene variants and chromosomal loci associated with immune response traits during infection with salmonella <cit> . a genetic basis for differences in resistance to salmonella has also been shown with slc11a <dig> , the seminal example of a gene with genetic variants dramatically affecting resistance to bacterial infection \n",
            " <cit> . while many studies have looked into the host response to salmonella infection \n",
            " <cit> , relatively few have focused on identifying the genes whose variable expression among different individuals may be associated with differences in salmonella clearance and resistance.\n",
            "\n",
            "initial characterisation of the pigs used in an earlier salmonella challenge study \n",
            " <cit>  revealed a significant positive correlation between serum interferon-γ  levels at day  <dig> p.i. and faecal salmonella shedding levels at day  <dig> and day  <dig> p.i. the same study also demonstrated that the peak of both clinical symptoms  and salmonella shedding occurs at day  <dig> p.i. and that substantial whole blood transcriptome changes occur at day  <dig> p.i. compared to day  <dig> in pigs belonging to both ls and ps groups. therefore, here we chose to profile whole blood transcriptomes at the same time-points, day  <dig> and day  <dig>  but belonging to a different set of pigs, using rna-seq instead of microarrays, for a different purpose: to identify genes whose expression prior to inoculation is correlated with salmonella shedding levels observed p.i. the genes identified may serve as blood-based candidate biomarkers that could potentially be used to develop quick screening tests for determining the host’s resistance/susceptibility to salmonella infection and predicting their shedding characteristics early into or even before infection.\n",
            "\n",
            "the extent of de and degree of change in expression between day  <dig> and day  <dig> were, in general, higher in ps than ls. this finding, as previously speculated \n",
            " <cit> , may indicate that ls animals respond faster and more effectively against infection than ps so that by day  <dig>  while the ls are returning back to normal, the ps are still actively fighting the infection. with this in mind we believe that the comparison between ls and ps at day  <dig> likely identifies de due to differing levels of infection. a comparison at day  <dig>  on the other hand, stands to better highlight genes responsible for differences in the efficacy of the initial response to the bacteria, assuming that some of these genes exhibit expression differences prior to infection. however, the de analysis between ls and ps at day  <dig> did not yield any significant de genes here as well as in an earlier salmonella challenge study \n",
            " <cit> . this failure to find de genes could be due to a combination of the subtlety of the expression differences, the relatively small sample size, and the strict multiple testing corrections. hence we used an alternative approach, wgcna, to find genes associated with salmonella shedding.\n",
            "\n",
            "remarkably, the genes whose pre-inoculation expression profiles were found associated with post-inoculation salmonella shedding levels included major genes already reported in literature as de during salmonella infection and involved with host resistance against salmonella such as slc11a <dig>  tlr <dig>  cd <dig> and ccr <dig> <cit> . moreover, the majority of genes within the modules significantly associated with salmonella shedding, following further refinement based on up-regulation and de at day  <dig> , were found to have an established or possible role in innate defense against bacterial/salmonella infections. these include mainly the early innate immune response genes responsible for cytokine-cytokine receptor interactions , genes involved in toll-like receptor pathway , nf-kappa b signalling pathway , nod-like receptor signalling pathway  or otherwise linked to response to bacterial infections . prioritisation of these genes based on gene significance, module membership and gene connectivity in ls at day  <dig>  highlights some genes which have not been previously associated with salmonella infection. for example, little is known about the role of siglec <dig>  a member of the siglec family of sialic acid-binding lectins in host response to bacterial infection. however, it has been reported in humans that the absence of a functional siglec <dig>  results in attenuated cytokine response to some gram-negative bacteria in null individuals \n",
            " <cit> .\n",
            "\n",
            "the mean expression levels of the salmonella shedding associated genes at day  <dig> were generally higher in ls than ps and mostly up-regulated in both at day  <dig> compared to day  <dig>  in most instances, the expression was higher in ps than ls at day  <dig>  we showed, at least for the top candidate genes reported here, that the pattern of expression is consistent with that from a previous microarray based salmonella challenge study involving a different set of ls and ps animals. examining the connectivities of genes within the salmonella shedding associated modules in ls and ps, it became apparent that the genes in general showed higher connectivity in ls than ps, indicative of higher correlation/connection strengths with other network genes. the differences in connectivity measures for a set of genes between different conditions may signify differences in the co-ordination or strength of transcriptional regulation of that set of genes. highly connected genes  have been shown to play central roles in the biological processes that are represented by the module \n",
            " <cit> , and strong positive correlations have been reported between gene connectivity within the whole network and gene essentiality \n",
            " <cit> . here, the significantly higher connectivity despite the lack of significant de between ls and ps may be considered analogous to the results in a study on breast cancers of different histological grades \n",
            " <cit> . the authors of that study concluded that the differential connectivity patterns were not due to primary alterations of hub gene expression, but rather due to more subtle changes in expression of numerous genes interacting with those hubs. further, they reported that complex epistatic interactions that underlie cellular functions might also be responsible for differences in network connectivity patterns as a function of a phenotypic trait. a study on aging in mice \n",
            " <cit>  reported a decreasing correlation of gene expression within genetic modules and attributed this to changes in expression of certain transcription factors  as well as deterioration of chromatin structure with age. it is possible that genetic differences at mutiple levels as discussed above may contribute to the differences in strengths of coexpression and connectivity between ls and ps. exploring these contributions may be a direction for future research.\n",
            "\n",
            "one of the limitations of our study is the absence of samples from time points post-inoculation but before day  <dig> p.i., which are crucial to capture the early immune response during which the ls pigs have effectively managed the salmonella challenge. secondly, this study would benefit from a larger sample size, which would provide more power to detect the subtle changes in expression expected between ls and ps animals. further experiments are required to rank the relative functional importance of our suggested candidate genes as contributors to distinct responses to salmonella challenge with respect to faecal salmonella shedding. however, the use of multiple criteria and strict cut-offs to refine the set of candidate genes, the agreement with existing literature regarding the immune related functions of many candidate genes and the concordance of the expression patterns of top candidate genes reported here with the corresponding expression patterns from an independent dataset, all lend further support to our predictions.\n",
            "\n",
            "CONCLUSIONS\n",
            "our analysis integrates gene co-expression network analysis, gene-trait correlations and differential expression to provide new candidate regulators of salmonella shedding in pigs with implications for future use as biomarkers for selection of animals with reduced susceptibility to, or carriage of, salmonella or for predicting response to infection. the comparatively higher expression  and the significantly higher connectivity of genes within the salmonella shedding associated modules in ls compared to ps even before salmonella challenge may be factors that contribute to the decreased faecal salmonella shedding observed in ls following challenge.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "salmonella shedding associated modules were significantly higher in ls than ps at day  <dig>  and significantly higher in ps at day  <dig> compared to day  <dig> p.\n",
            "however, the relative functional importance of genes within salmonella shedding associated modules in ls compared to ps even before salmonella challenge may be related to the decreased faecal salmonella shedding observed in ls following challenge.\n",
            "salmonella shedding associated modules were significantly higher in ls than ps at day  <dig>  and significantly higher in ps at day  <dig> compared to day  <dig> p.\n",
            "salmonella shedding associated modules were significantly higher in ls than ps at day  <dig>  and significantly higher in ps at day  <dig> p.\n",
            "salmonella shedding associated modules were significantly up-regulated in ls compared to ps at day  <dig> p.\n",
            "salmonella shedding associated modules were significantly up-regulated in ls compared to ps at day  <dig> p.\n",
            "salmonella shedding associated modules were significantly up-regulated in ls compared to ps at day  <dig> p.\n",
            "salmonella shedding associated modules were significantly up-regulated in ls compared to ps\n",
            "69\n",
            "[{'summary_text': 'fvs are a group of complex retroviruses that have a very stable and long co-speciation history with their hosts, and because of this, almost all of their divergence dates can be directly inferred from their measurement timescales.\\nfinally, our results suggest that the tdrp can be used as a tool for correcting for the tdrp in evolutionary inferences.\\nfinally, our results suggest that the tdrp can be used as a tool for correcting for the tdrp in evolutionary inferences.\\nfinally, our results suggest that the tdrp can be used as a tool for correcting for the tdrp in evolutionary inferences.\\nfinally, our results suggest that the tdrp can be used as a tool for correcting for the tdrp in evolutionary inferences.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 6114 to 6144 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "it has been noted that evolutionary rates calculated over short timescales, such as those calculated from population genetic data, are much greater than those calculated across geological time frames, such as species evolutionary rates. this discrepancy between short-term and long-term rates is very widespread in nature, noticed in both viral genes  and cellular genes, including bacterial genes  <cit> , mitochondrial genes of worms  <cit> , insects  <cit> , fish  <cit> , birds , and primates , as well as nuclear genes of various multicellular organisms  . further investigation has revealed that, in fact, the value of the rate estimate does not vary discretely, but continuously decreases as the measurement timescale increases  <cit> . this ‘time dependent rate phenomenon’  was first demonstrated in cellular genes  <cit> . subsequently, by pooling substitution rate estimates of diverse viruses together, duchêne et al.  <cit>  showed that the rate estimates of rna and dna viruses also exhibit this pattern.\n",
            "\n",
            "to date, the processes that lead to the tdrp are still very much unclear. many hypotheses have been proposed to explain it, such as temporal changes in organismal biology and natural selection pressure  <cit> . numerous methodological artefacts also have the potential to systematically bias the rate estimates in such a way that short-term rates will appear to be much greater than the long-term ones  <cit> . since the rate of evolution is a central component of evolutionary study, an accurate evolutionary inference requires that the tdrp is integrated into the analysis. an ideal approach to this problem is to understand how the tdrp is generated, and improve evolutionary inference tools so that they can account for the factors underlying the tdrp. however, given a large number of potential underlying factors and our current incomplete understanding of their interactions, untangling and explicitly accounting for each of them individually would be impractical at present  <cit> .\n",
            "\n",
            "one pragmatic approach to this problem is to infer evolutionary timescales by using an empirical model describing the relationship between rate estimates and their measurement timescales. this approach has been employed in several studies . a number of empirical models, such as the vertically-translated exponential rate decay model and the power-law rate decay model, have been proposed, and used, for tdrp correction in evolutionary inferences  <cit> . nevertheless, at present, it is still unclear which model best describes the phenomenon, and to answer this question we require substitution rate estimates that are computed over various timeframes. these can be obtained from a dataset of molecular sequences for which several divergence dates are known; the more divergence dates available, the more suitable the dataset for this purpose. furthermore, it is also preferable that the dates are distributed relatively evenly across the entire evolutionary timescale that is being examined  <cit> . in this work, we seek to empirically describe the tdrp in detail and explore the various patterns of rate decay over time by using foamy viruses  as a case study.\n",
            "\n",
            "fvs are a group of complex retroviruses that have a very stable and long co-speciation history with their hosts, stretching back more than a hundred million years , and because of this, almost all of their divergence dates can be directly inferred from those of their hosts . based on this co-speciation and the known divergence dates of their primate hosts, the long-term rate of evolution of fvs has been estimated to be ~ <dig>  × 10- <dig> to  <dig>  × 10- <dig> nucleotide substitutions per site per year   <cit> . this is much slower than rates of substitution of other rna viruses, typically reported to be in range of 10- <dig> to 10-4 s/n/y  <cit> . the high similarity observed between extant fvs and their ancient endogenous counterparts  <cit>  has also lent further support to this notion of slow-evolving fvs. altogether, fvs are thereby widely regarded as one of the most slow-evolving rna viruses currently known  <cit> .\n",
            "\n",
            "these slow long-term rates of fv evolution stand in sharp contrast to their high mutation rate. in vitro analyses have shown that the fv replication error rate  is comparable to that of human immunodeficiency virus    <cit> , which is one of the fastest-evolving viruses ever documented. this fast fv mutation rate has also been confirmed in human embryonic cell lines, where the in vivo rate was calculated to be at least  <dig>  × 10-5 s/n/replication  <cit> . moreover, by following a population of african green monkey fvs for 9 years, it was estimated that the fv evolutionary rate is as high as ~ <dig>  × 10-4 s/n/y  <cit> . this short-term rate is ~4- <dig> orders of magnitude higher than the long-term rate counterparts, estimated under the fv-host co-speciation assumption.\n",
            "\n",
            "the fact that almost all of the divergence dates of fvs can be directly inferred from those of their hosts makes fvs an ideal system to study the tdrp. here, we use  <dig> extant fvs  as a case study to present direct evidence of a smooth decay of nucleotide substitution rate estimates as the measurement timescale increases. we also empirically describe the rate decay pattern, examine whether or not the tdrp can explain the discrepancy between fv short-term and long-term rates, and discuss the applications and limitations of our empirical rate decay models, as well as how the tdrp may bias evolutionary inference and rate estimate interpretation.\n",
            "\n",
            "RESULTS\n",
            "fv nucleotide substitution rate estimate decreases with measurement timescale\n",
            "to compute fv nucleotide substitution rates for various timescales, we first estimated fv phylogenies using pol protein  and pol nucleotide  alignments under the bayesian and maximum-likelihood frameworks. the aligned sequences were checked for recombination, but no significant evidence was found, both at the nucleotide  and protein  levels.  our results show that all phylogenies are perfectly in agreement with one another topologically , and also consistent with the results from previous studies . we thus considered the estimated tree topology as our best working hypothesis, and used it to estimate fv nucleotide substitution rates.\n",
            "\n",
            "we first estimated node-to-tip total per-lineage nucleotide substitutions  from the pol nucleotide alignment under the fixed fv phylogeny, and the bayesian phylogenetic framework. we employed a strict molecular clock with a fixed rate of  <dig> to obtain branch lengths in units of substitutions per site. a strict clock was applied  under the assumption of consistency among rate estimates calculated using different nodes and tips. thus, our study can also be viewed as an attempt to correct for the tdrp given the strict clock assumption. in total,  <dig> posterior distributions of s estimates were obtained, one for each internal node, and we could assign timescales  to  <dig> of them based on the fv-host co-speciation history. this, in turn, allowed us to compute  <dig> distributions of node-to-tip average substitution rates .  a summary of the results can be found in fig.  <dig>  and additional file 1: table s <dig>  preliminary linear regression analyses suggest that log-transformed r¯ is significantly negatively correlated with log-transformed t ] = - <dig>  ; randomisation test: number of randomization tests =  <dig> , number of data points for null distribution construction in each test =  <dig>  p <  <dig>  in all  <dig>  tests; see methods for details).fig.  <dig> foamy virus  and corresponding host phylogenies, and the relationship between node-to-tip average evolutionary rate estimate and timescale of measurement.  fv phylogeny . black numbers are estimated total per-lineage nucleotide substitutions in the units of substitutions per site. the node bars represent the uncertainties of the estimated nucleotide substitution divergences. the scale bar is in units of substitutions per site.  host tree. black numbers are estimated divergence dates in units of millions of years, for which the estimation uncertainties are shown by node bars. the scale bar is in units of millions of years. the topology of the host tree and the divergence dates were estimated elsewhere . solid lines between the two trees indicate fv-host associations, and blue arabic numbers  indicate matching fv-host nodes . nodes within the fv tree that could not be mapped conclusively onto the host tree are labelled by blue roman numbers . the colours of the node bars on the fv tree correspond to the colours of the bars on the host tree. b timescales of rate measurement and node-to-tip average evolutionary rate estimates on a log-log scale. the node numbers  refer to those in the fv and host trees . the colours correspond to the node bars’ colours. the summary of the raw data can be found in additional file 1: table s2\n",
            "\n",
            "\n",
            "\n",
            "empirical description of fv nucleotide substitution rate decay\n",
            "it has been proposed that a vertically-translated exponential decay function  is a good empirical description for the apparent decay of the instantaneous substitution rate   <cit> . this model has been employed in several studies for tdrp correction . however, an examination of mitochondrial dna control regions of cichlids showed that a simple power law function  can also empirically describe the phenomenon reasonably well  <cit> . to systematically explore these hypotheses, we examine another two empirical functions by varying whether the exponential and power law functions are vertically translated: a simple exponential rate decay function  and a vertically-translated power-law function . the four models can be expressed mathematically as follows:  <dig> rvex=αvexe−tβvex+kvex   <dig> rex=αexe−tβex   <dig> rvpl=αvplt−βvpl+kvpl   <dig> rpl=αplt−βpl where t  is the measurement/evolutionary timescale in the present-to-past direction, where t =  <dig> is the present; k  is the stable long-term rate of evolution parameter; and α and β are arbitrary model parameters. “vex”, ‘ex’, ‘vpl’, and ‘pl’ subscripts indicate the model to which the parameters and variables belong: vertically-translated exponential rate decay , simple exponential rate decay , vertically-translated power-law rate decay  and simple power-law rate decay  models, respectively. note that, since a strict clock was applied and the tips were all aligned, the timescales of the rate measurement can be interpreted as node heights/divergence dates and vice versa.\n",
            "\n",
            "to investigate how well each model describes the fv r¯ dynamics, we first derived four equations depicting the relationship between s and t for the vex , ex , vpl , and pl  models based on eq. 1– <dig>  respectively, as follows;  <dig> svex=∫t=0t=trvexdt=αvexβvex1−e−tβvex+tkvex   <dig> sex=∫t=0t=trexdt=αexβex1−e−tβex   <dig> svpl=∫t=0t=trvpldt=αvplt1−βvpl1−βvpl+tkvpl   <dig> spl=∫t=0t=trpldt=αplt1−βpl1−βpl \n",
            "\n",
            "the curves were forced to go through the origin to conform the expectation that there are no substitutions at time equal to zero. we next simply divided both sides of the equations by t to derive equations describing the relationship between r¯ and t for the four respective models :  <dig> svext=r¯vex=αvextβvex1−e−tβvex+kvex   <dig> sext=r¯ex=αextβex1−e−tβex   <dig> svplt=r¯vpl=αvplt−βvpl1−βvpl+kvpl   <dig> splt=r¯pl=αplt−βpl1−βpl \n",
            "\n",
            "we then fitted all four models to the r¯ and t estimates, and assessed how well the models describe the data by using adjusted r <dig>  scores. although it is clear from visual inspection that the ex model tends to underestimate long-term rates , overall, all four models seem to describe the data well, indicated by their high r¯ <dig> scores . the results are shown in fig.  <dig>  and additional file 1: table s <dig> fig.  <dig> empirical description of the relationship between rate measurement timescale  and average evolutionary rate estimate . eq.  <dig> , eq.  <dig> , eq.  <dig> , and eq.  <dig>  were fitted to  <dig>  randomly-sampled sets of corresponding r¯ and t estimates under the criterion of the least sum of squared errors of r¯ . the colours correspond to those of the node bars in fig. 1a. the node numbers refer to those on the trees in fig. 1a. the summary of the results can be found in additional file 1: table s3\n",
            "\n",
            "\n",
            "\n",
            "in addition, we also recovered fv short-term and long-term rate estimates , respectively) by using these four models. the vex and ex models estimated the short-term rate of fvs to be ~10- <dig> to 10-8 s/n/y . this is much lower than the previously reported fv short-term rate estimate . on the other hand, the vpl and pl models estimated the rate to be in the order of 10- <dig> to 10-5 s/n/y , comparable to the previously reported fv short-term rate estimate. in contrast, all four models calculated the long-term rate to be ~10- <dig> to 10-9 s/n/y , all comparable to the established long-term rates of fvs .\n",
            "\n",
            "leave-one-out cross validation analyses\n",
            "although the high r¯ <dig> scores suggest that all four models can describe the relationship between r¯ and t estimates well , it has been noted that r¯ <dig> scores are inappropriate for comparing the performance of nonlinear models, and can severely bias model selection in favour of models with more parameters  <cit> . corrected akaike information criterion  and bayesian information criterion  have been suggested to be more suitable for this purpose  <cit> . however, the calculation of aicc and bic scores requires likelihood functions of the models. since our models are empirical, derived based on a top-down approach, we lack such information. given these limitations and constrains, we thus used the leave-one-out cross validation  technique to compare our models, in the context of their ability to recover t values given the s values. eq. 5– <dig> were used in these analyses as they depict how t relates to s. the t values inferred under the fv-host co-speciation assumption were used as references, and by comparing the recovered t values against them, we computed out-of-sample mean squared error  scores. these scores were used as a measurement of the overall predictability of the models.  the results are summarised in fig.  <dig> and additional file 1: table s <dig> fig.  <dig> leave-one-out cross validation  tests. the models were validated under the context of divergence date  inference, by using loocv tests . in , the recovered t values are shown against the reference t values, inferred under the co-speciation assumption. the colours of the data points correspond to the colours of the node bars in fig. 1a. the node numbers refer to those in the trees in fig. 1a. the grey lines show where the recovered t values are exactly equal to the references. the out-of-sample mean squared errors are shown in . a summary of the results can be found in additional file 1: table s <dig>  vex: vertically-translated exponential rate decay model; ex: simple exponential rate decay model; vpl: vertically-translated power-law rate decay model; and pl: simple power-law rate decay models\n",
            "\n",
            "\n",
            "\n",
            "overall, we found that the t values recovered by all four models are largely comparable to the references , indicative of high predictability for all four models. this finding also implies that the co-speciation hypothesis is itself internally consistent. two discrepancies were found between the reference and recovered t values however. for all four models, we found that  the recovered time to most recent common ancestor  of simian fvs   are much greater than the corresponding host tmrca , and  that the recovered tmrca of bovine fv  and equine fv   are much lower than the respective host tmrca . to verify that these estimates are not correlated , we re-calculated the tmrcas for these two fv groups using all four models estimated independently of nodes  <dig> and  <dig>  both discrepancies could still be observed , indicating that they are not artefacts.\n",
            "\n",
            "interestingly, unlike other models, the ex model in particular seems to have a tendency to overestimate shallow divergence dates . this is at odds with the observation that the ex model has a tendency to incorrectly describe the long-term rates . systematic differences in the distribution of the data points describing the r¯-t and t-s relationships may explain this discrepancy. while the variance of r¯ estimates is greatest when t is low, the variance of t estimates is greatest when s is high. thus, the influence of the data points in parameter estimation differed when the ex model was fitted to the two datasets; that is, while eq.  <dig> parameter estimation  was primarily influenced by the data points near the y axis, eq.  <dig> parameter estimation  was primarily influenced by the data points further away from the y axis. this problem is not as apparent in the other three models however, suggesting that they suffered to a much lesser extent from this effect.\n",
            "\n",
            "by comparing the mseoos scores, we found that the pl model  has the least mseoos overall , indicating that it is the best model for inferring t, and thus most preferable as a tdrp-correcting tool. interestingly, the vpl model  was found to be the second best model , suggesting that the extra parameter kvpl in the vpl model does not significantly improve, but instead over-parameterises the model. likewise, the vex model  was found to be the worst model , and the ex model  was found to be the second worst model .\n",
            "\n",
            "examining the performance of currently available molecular clocks in tdrp correction\n",
            "several relaxed-clock models have been developed to address the problem of rate variation among lineages . these models allow rates to vary among branches, and thus over time; therefore, they have the potential to be used as a tool for correcting for the tdrp in evolutionary inferences. in this section, we explored how well currently available relaxed-clock models can accommodate the tdrp under the context of fv timescale inference. we chose to explore scenarios where only three nodes are available as calibrating nodes as they represent realistic circumstances where calibrating information is limited. nodes  <dig> and  <dig> were excluded from this analysis. this is because the loocv analyses show that the reference and the recovered t values differ greatly, and it is not possible to determine which is closer to the true values.\n",
            "\n",
            "two schemes of date calibration were examined:  aggregated-node calibration scheme, where all three calibrating nodes are of similar timescales, and  dispersed-node calibration scheme where calibrating nodes are of different timescales. in the former calibration scheme, we explored three different scenarios:  shallow- ,  intermediate- , and  deep-timescale calibration scheme . similarly, we explored three  circumstances for the latter calibration scheme:  dispersed-i calibration scheme ,  dispersed-ii calibration scheme , and  dispersed-iii calibration scheme . three currently available molecular clocks were investigated, including  a strict molecular clock,  a log-normal relaxed clock  <cit> , and  a random-local relaxed clock  <cit> . the strict clock was included to examine how the tdrp would affect t inferences if it is ignored. we focused on bayesian timescale estimates, and used the pol nucleotide alignment that was used in the phylogenetic reconstruction to estimate the timescales . again, the t estimates inferred under the fv-host co-speciation assumption were used as reference t estimates.\n",
            "\n",
            "we also compared these currently available clocks to our pl model . unlike the above however, the calculation was not done under a full bayesian phylogenetic framework. rather, we used t and s estimates of the calibrating nodes to estimate the model, and then inferred the t values of other nodes based on their s estimates. the results are shown in fig.  <dig>  and additional file 1: table s <dig> fig.  <dig> performance of various molecular clocks in accommodating the time-dependent rate phenomenon . we explored the performance of various clock models in accommodating the tdrp in the context of foamy virus  evolutionary timescale inference under various hypothetical scenarios, where only three nodes are available as calibrating nodes. two calibration schemes were examined:  aggregated-node calibration scheme , in which all three calibrating nodes are of similar timescales, and  dispersed-node calibration scheme , in which calibrating nodes are of different timescales. the aggregated-node calibration scheme was sub-divided into three sub-schemes:  shallow-timescale calibration scheme ,  intermediate-timescale calibration scheme , and  deep-timescale calibration scheme . the dispersed-node calibration scheme was also sub-divided into three arbitrary sub-schemes:  dispersed-i calibration scheme ,  dispersed-ii calibration scheme , and  dispersed-iii calibration scheme . four sets of dates were plotted from different clock implementations and one of our tdrp models:  simple power rate decay model ,  strict clock model ,  log-normal relaxed clock model , and  random-local relaxed clock model . fv evolutionary timescales inferred from the host timescales  are in white. horizontal dotted grey lines are median reference timescales. the results are plotted on a log scale, and arranged in such a way that the evolutionary timescales of shallow nodes are on the left, and those of deep nodes are on the right. the node numbers refer to those in fig. 1a. calibrating nodes are indicated by asterisks ‘*’. the summary of the results can be found in additional file 1: table s <dig> & s6\n",
            "\n",
            "\n",
            "\n",
            "aggregated-node calibration scheme\n",
            "under the aggregated-node calibration scheme , our results show that the t values estimated under the strict clock, log-normal relaxed clock, and random-local relaxed clock are comparable to one another. this is likely because the calibrating nodes do not provide enough signal for the rate variation among lineages to be estimated. indeed, we found that the coefficients of rate variation estimated under the two relaxed clock models do not deviate very far from zero .\n",
            "\n",
            "overall, we found that the t values of the nodes adjacent to the calibrating nodes are estimated relatively accurately, comparable to the reference values. however, as we move further away from the calibrating nodes, the t estimates become increasingly more inaccurate. under the shallow-timescale calibration scheme, the t values of deep nodes are severely underestimated . in contrast, the t estimates of shallow nodes are severely overestimated under the deep-timescale calibration scheme . in the intermediate-timescale calibration scheme, the t values of deep nodes are underestimated while those of shallow nodes are overestimated . our results are consistent with the findings in a previous study that solely focused on sfvs  <cit> .\n",
            "\n",
            "in contrast, our results showed that the pl model could accommodate the tdrp reasonably well under these examined scenarios, better than the currently available molecular clocks. although we found that the uncertainties of t estimates are extremely large , the reference t values are almost always contained within the t distributions estimated under the pl model, whereas those estimated under the current clock models do not even overlap with the reference t distribution . the median t estimates from the pl model are also closer to the reference values, which is most apparent in the intermediate- and deep-timescale calibration schemes , but less so in the shallow-timescale calibration scheme . these large uncertainties are unlikely an intrinsic property of the pl model however, but a shortcoming of our extrapolation approach in which the t and s values are not estimated in conjunction with one another. coupled with the fact that these are extrapolations over large timescales and that the uncertainty of the s estimate increases down the tree , it is unsurprising to find that this problem is most apparent in the shallow-timescale calibration scheme , but less so in the intermediate-timescale calibration scheme , and least in the deep-timescale calibration scheme . these findings thus indicate that deep nodes are preferable as calibrating nodes, consistent with the results from a previous simulation study that investigated the impact of calibrating node position on timescale estimates  <cit> .\n",
            "\n",
            "dispersed-node calibration scheme\n",
            "under the dispersed-node calibration scheme , we found that the performances of the log-normal and random-local relaxed clocks differ from that of the strict clock. this is likely because the calibrating nodes are spread out in time and hence provide adequate information for the rate variation among branches, and thus over time, to be modelled more accurately. indeed, the coefficients of rate variation estimated under the two relaxed clocks deviate far from zero .\n",
            "\n",
            "overall, we found that the strict clock model performs worst, and the t values obtained under the clock vary substantially between sub-schemes. for example, in the dispersed-i calibration scheme , the t values of intermediate nodes were estimated relatively accurately, but those of shallow nodes were overestimated, and those of deep nodes were underestimated. on the other hand, in the dispersed-ii and -iii calibration schemes , the t values of deep nodes were estimated relatively accurately whereas those of shallow nodes were severely overestimated. in contrast, the two relaxed clock models seem to perform reasonably well, returning t values that are comparable to the references, and also to those inferred under the pl model. this, in turn, indicates that the performance of these three models is comparable under these scenarios. moreover, we also observed that the t distributions inferred under the pl model are tighter compared to those inferred under the aggregated-node calibration scheme. this is likely because the dispersed-node schemes involve interpolation rather than extrapolation.\n",
            "\n",
            "inferring missing fv divergence dates by using the pl model\n",
            "consistent with the results from previous studies  <cit> , amidst the stable fv-host co-speciation history, we found a few mismatches in the history of new world monkey  fvs and their hosts . these involve  the split between marmoset and spider monkey fvs  , and  the divergence leading to squirrel monkey fv  lineage .\n",
            "\n",
            "as a case study, we used the pl model to infer the t values for these two nodes. to do so, eq.  <dig> was fitted to the dataset of t and s estimates, with those of nodes  <dig> and  <dig> excluded . the models were then used to infer the t values of node i and ii from their s estimates. our analyses inferred the tmrca of sfvmar and sfvspm to be ~ <dig>   myr, and the earlier branching of sfvsqu lineage to occur ~ <dig>   myr ago.\n",
            "\n",
            "discussion\n",
            "what are the causes of the tdrp?\n",
            "our analyses show that the estimated values of the fv substitution rates decrease continuously with measurement timescales , and the pl model is the best model for correcting for the tdrp. it is possible that the tdrp observed here is the result of changes in fv biology, such as polymerase fidelity, replication speed, and/or transmission modes. these ideas have been put forward as plausible explanations for the rate variations observed in several viruses, including hepadnaviruses  <cit> , and human t-cell lymphotopic viruses  <cit>  as well as rna viruses in general  <cit> . these hypotheses would be reasonable and sensible if the values of the rate estimates changed discretely. however, as the rate estimate continuously decreases with the measurement timescale, this would posit that the biology of fvs has been continuously changing with time in such a way that that the viral rate of evolution gradually increases through time. moreover, since the tdrp is not unique to fvs, but very common among viruses, these hypotheses would also imply that the biology of many viruses has changed in a similar manner. although possible, this is extremely unlikely  <cit> . we therefore believe that changes in viral biology likely do not play a major role in governing the overall decay trend of viral evolutionary rate estimates.\n",
            "\n",
            "changes in natural selection pressure have also been put forward as a potential underlying cause of the tdrp  <cit> . nevertheless, like viral biology, it is extremely difficult to imagine that environmental factors would have changed systematically with time so that viruses experience less and less purifying selection pressure as they evolve. it is important to note that we are by no means suggesting that changes in environment do not cause viral evolutionary rates to alter. indeed, they can, and this has been observed. an analysis of bat rabies viruses has shown that their evolutionary rates are strongly correlated with the host local environments, suggesting that environmental changes can alter the rate of viral evolution  <cit> . however, we argue that, for the alteration of environment and/or natural selection to play a major role in generating the tdrp, not only must they change the rate, but the effect must also be time-correlated, which we consider unlikely. we thus believe that, rather than reflecting a genuine change in natural selection pressure, the observed tdrp is likely an artefact caused by other factors.\n",
            "\n",
            "so, if viral biology, environmental factors, and selection pressure remain relatively constant through time, what could possibly cause the tdrp? it has been proposed that short-term rate estimates tend to be overestimated due to the inclusion of transient deleterious variations  <cit> , recent adaptive changes , and sequencing errors . errors in coalescent calibration information have also been suggested as a sufficient explanation for elevated short-term rate estimates  <cit> . in contrast, misspecification of substitution model  <cit> , saturation of nucleotide changes  <cit> , and improper accounting for rate heterogeneity among sequence positions  <cit>  can lead to underestimation of long-term rate estimates. all of these factors could contribute to the tdrp, and at present, the importance and relevance of each factor is still poorly understood and continues to be debated .\n",
            "\n",
            "the tdrp as a possible explanation of the short-term and long-term rate discrepancy\n",
            "a large discrepancy of ~4- <dig> orders of magnitude has been observed between fv short-term and long-term rate estimates. indeed, such discrepancies have been found not only in fvs, but many and diverse rna and dna viruses . rather than viewing them as conflicting rate estimates, it is has been proposed that this discrepancy may result from estimating the rates over different extremes of the tdrp  <cit> .\n",
            "\n",
            "in order to investigate whether the tdrp can explain this discrepancy or not, we used our four rate models to recover the short-term and long-term rate estimates of fvs. our analyses showed that all of the four models could recover the long-term rate relatively well. however, given that the models were parameterised on fv long-term rate estimates, this is expected and unsurprising. in contrast, we found that only the vpl and pl models could recover the fv short-term rate accurately, but the vex and ex models severely underestimated the rate by ~3- <dig> orders of magnitude. these results further support the use of the pl model as a tool for tdrp correction, and simultaneously indicate that, indeed, the short-term and long-term rate discrepancy in fvs can be explained by the tdrp.\n",
            "\n",
            "implications of the tdrp\n",
            "there are a number of implications of the tdrp. one of them is that it is important to take the timescale of rate measurement into consideration when using or interpreting evolutionary rate estimates; otherwise, the outcomes could be severely biased. for example, it is inappropriate to use long-term rate estimates to infer or evaluate viral short-term epidemiological dynamics, as they could give an erroneous impression that the viruses being considered are of low adaptive and cross-species transmission potential. similarly, in the specific case of fvs, which are candidate gene-carrying vectors for gene-therapy  <cit> , using their long-term rates to evaluate the risks of fv-derived gene vectors could be misleading, as they are in fact capable of evolving at a  rate as high as ~ <dig>  × 10-4 s/n/y  <cit> , comparable to those of highly pathogenic and fast-adapting viruses like hivs  and influenzas .\n",
            "\n",
            "another important, and perhaps more obvious, implication is that it will no longer be valid to naïvely extrapolate rate estimates across different time frames when inferring evolutionary timescales. the assumption of a single molecular clock can bias the timescale inference with the severity increasing with the timescale of rate extrapolation. as the value of the rate estimate continuously decreases with the measurement timescale, the tdrp should appear in, and is relevant to, every phylogenetic analysis. that is, if two or more evolutionary rate estimates are calculated over different timescales from a particular phylogeny , the tdrp should show up. nevertheless, in practice, if the timescale of evolutionary investigation is short, e.g. hundreds of years , the uncertainty of the rate estimate may overwhelm the effect of the tdrp, and the phenomenon might not be observed. indeed, the strict molecular clock has sometimes been identified as appropriate for studies over short timescales such as epidemiological studies  <cit> , but not for longer timescale analyses such as evolutionary investigations that compare viruses in different host species  <cit> . moreover, the tdrp can also bias demographic parameter estimations such as effective population sizes and migration rates if it is unaccounted for. this is simply because the calculation involves estimating or knowing substitution rates  <cit> .\n",
            "\n",
            "similar to the results from previous studies  <cit> , our results show that evolutionary timescales calibrated under currently available molecular clocks are highly sensitive to the choice of calibrating nodes. the effect is most pronounced when the calibrating nodes are of the same timescale . as short-term rate estimates are greater than the long-term ones, calibrating deep divergence dates with shallow nodes will underestimate them. conversely, using deep nodes to calibrate shallow divergence dates will tend to overestimate them. strikingly, we found that the current relaxed clocks do not perform any better than the strict clock under these circumstances, indicating that they are not an effective solution to the tdrp problem. compared to the currently available relaxed-clock models, the pl model performs better and is more consistent across the calibration schemes, even when calibrating nodes are of similar timescales .\n",
            "\n",
            "our results suggest that the currently available relaxed-clock models suffer much less from the tdrp problem, and perhaps are equally good to the pl model, if calibrating nodes are dispersed in time , consistent with the results from a previous study  <cit> . nevertheless, in most realistic applications, the number and dispersal of calibrating nodes is serendipitous, dictated by their limited availability which often involves only one or a few nodes. given this limited availability of time-calibrating information and the sensitivity of the current clocks to the choice of calibrating nodes, our results overall suggest that the current relaxed-clock models might not be an effective and practical solution to the tdrp problem yet. we, thus, believe that our pl model will be useful as a guideline to further improve our current evolutionary inference tools.\n",
            "\n",
            "possible evolutionary rate dynamics heterogeneity among viral lineages\n",
            "our analyses showed that the tmrca estimates of sfvs recovered by the four models are considerably higher than the host tmrca . our analyses also estimated the tmrca of sfvmar and sfvspm  and that of nwm fvs  to be greater than those of their hosts . one possibility is that the tmrcas inferred by our models are not artefacts, but resemble the real dates. this would however imply duplications of viral lineages in the absence of host diversification, which we consider to be unlikely. an alternative explanation is that the evolutionary rates of nwm fvs and the stem lineage are higher than average, and our analysis framework did not take this into account. since we employed a strict clock to estimate s values, we thereby assumed that all taxa evolve under the same time-dependent rate dynamics. thus, the incongruences between the predicted and inferred nwm fv divergence dates may be indicative of the heterogeneity of evolutionary rate dynamics among viral lineages, and this may bias evolutionary timescale inferences if it is unaccounted for. a solution to this problem would be to fit multiple time-dependent rate models to different parts of the tree.\n",
            "\n",
            "conversely, our results show that the recovered bfv/efv tmrcas are considerably lower than the host tmrca , implying a cross-species fv transmission between equine and bovine hosts ~53-59 ma. unlike the scenarios discussed above, this is relatively reasonable and should not be ruled out. alternatively, it could be that the substitution rate on the bfv-efv stem branch is greater than that of other fvs. further resolution of these questions would require the identification and analysis of fv genomes of other bovines and equines.\n",
            "\n",
            "CONCLUSIONS\n",
            "our knowledge of viral natural history has been greatly advanced by molecular analyses. one of the key steps in viral evolutionary study involves estimating the rate of substitution. by using fvs as a case study, we show that their evolutionary rate estimates are negatively correlated with the timescale of rate measurement, and this is likely responsible for the short-term/long-term rate discrepancy observed in fvs, and perhaps other viruses as well  <cit> . we also demonstrate that currently available relaxed-clock models are inadequate for accommodating the tdrp; using them to infer evolutionary timescales can severely bias the date estimates especially when rate-calibrating nodes are of similar timescales. we believe that the pl model developed here will be useful as a guideline for the further improvement of existing analytical tools. our results also suggest that heterogeneity in rate dynamics among viral lineages may exist, and can affect evolutionary inference.\n",
            "\n",
            "combined, our work highlights the importance of the tdrp and heterogeneity in evolutionary rate dynamics among viral lineages. given the potential impacts of the tdrp on evolutionary inference and rate estimate interpretation, the fact that it is so widespread in nature but has been noticed only recently could mean that the credibility of evolutionary timescale estimates of many viruses may need to be reconsidered.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "fvs are a group of complex retroviruses that have a very stable and long co-speciation history with their hosts, and because of this, almost all of their divergence dates can be directly inferred from their measurement timescales.\n",
            "finally, our results suggest that the tdrp can be used as a tool for correcting for the tdrp in evolutionary inferences.\n",
            "finally, our results suggest that the tdrp can be used as a tool for correcting for the tdrp in evolutionary inferences.\n",
            "finally, our results suggest that the tdrp can be used as a tool for correcting for the tdrp in evolutionary inferences.\n",
            "finally, our results suggest that the tdrp can be used as a tool for correcting for the tdrp in evolutionary inferences.\n",
            "\n",
            "70\n",
            "[{'summary_text': 'a cluster of invasive isolates belonging to emm types  <dig> and  <dig> and carrying the sag genes spea, spej, spej, spej, spea, spej, spea, spej, spea, spea, spej, spea, spea, spea, spej, spea, spea, spea, spea, spea, spej, and smez were also associated with invasive isolates.\\na cluster of macrolide-resistant isolates belonging to emm types  <dig> and  <dig> and carrying the sag genes spea, spej, spea, spea, spej, spea, spea, spea, spej, spea, spea, spej, spea, spea, spej, spea, spea, spej, spea, spea, spea, spej, spea, spea, spea, spea, spej, spea, spea, spea, spea, spej, spea, spea, spea, spej, spea, spea, spea, spej, and smez.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 4812 to 5120 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "streptococcus pyogenes  remains one of the most common human pathogens, being responsible for uncomplicated superficial infections of the respiratory tract and skin, such as tonsillo-pharyngitis and impetigo, but also causing severe and rapidly progressing invasive disease such as necrotizing fasciitis, bacteremia, streptococcal toxic shock syndrome , puerperal sepsis, pneumonia, and meningitis  <cit> . although the incidence and severity of gas infections in industrialized countries decreased for most of the 20th century, a reemerge of gas invasive disease has been noted since the late 1980s, both in north america and in europe  <cit> . the annual incidence of gas invasive disease has been estimated at  <dig> / <dig>  <dig> for developed countries, with a median case fatality rate of 15%  <cit> .\n",
            "\n",
            "the increase in the incidence of gas invasive infections has been frequently associated with specific clones, raising the possibility that the rise of particularly virulent clones was responsible for this reemergence, in particular the m1t <dig> clone which is dominant among invasive gas isolates in most developed countries  <cit> . however, a higher representation of a particular clone in invasive infections may be simply due to a high prevalence of that same clone in the general gas population. to address this question several studies have performed comparisons between the characteristics of the invasive clones and the s. pyogenes isolates associated with carriage or uncomplicated infections in the same time period and geographic region. some of those studies reported a differential distribution of m/emm types or of t types between invasive and non-invasive isolates and confirmed an association between serotype t <dig> and m1/emm <dig> and invasive infection  <cit> , but many others found that the major clones responsible for invasive infections had a similar prevalence among non-invasive infections  <cit> . however, most of the studies performing such comparisons were either restricted to small numbers of isolates or were limited in the typing methodologies used, relying essentially on m/emm typing.\n",
            "\n",
            "serotyping of gas based on protein m, a major surface virulence factor, has long been used as the gold standard for the epidemiological surveillance of the infections caused by this pathogen. in recent years it has been widely replaced by an equivalent approach based on sequencing the hypervariable region of the emm gene encoding the m protein. however, recent studies show that emm typing alone is not sufficient to unambiguously identify gas clones and that it must be complemented with other typing methods such as pulsed-field gel electrophoresis  macrorestriction profiling or multilocus sequence typing   <cit> . streptococcal superantigens  secreted by s. pyogenes play an important role in the pathogenesis of the infections caused by this species  <cit> . the profiling of the eleven sag genes described so far  can be used as a typing methodology  <cit> . some studies suggested an association between the presence of certain sag genes or of certain sag gene profiles and invasive infections  <cit> , although others failed to establish such an association, reporting instead a strong link between the sag profile and the emm type, regardless of the isolation site  <cit> .\n",
            "\n",
            "we have previously characterized a collection of  <dig> invasive gas isolates collected throughout portugal between  <dig> and  <dig>  and found a very high genetic diversity among this collection, but with a dominant clone representing more than 20% of the isolates, which was characterized as emm1-t1-st <dig> and carried the gene spea <cit> . the aim of the present study was to evaluate if the clone distribution among the invasive gas isolates in portugal reflected the clonal structure of the isolates causing pharyngitis, in terms of molecular properties and antimicrobial resistance. in order to do that,  <dig> non-duplicate isolates collected from pharyngeal exudates associated with tonsillo-pharyngitis in the same time period were studied by emm typing, t typing, sag profiling, pfge macrorestriction profiling, and selected isolates were also submitted to mlst analysis. all isolates were also tested for their susceptibility to clinically and epidemiologically relevant antimicrobial agents. the great majority of the clones were found with a similar frequency among invasive infections and pharyngitis. still, some clones were shown to have a higher invasive disease potential and it was also possible to establish significant associations between some emm types and sag genes and disease presentation.\n",
            "\n",
            "RESULTS\n",
            "antimicrobial resistance\n",
            "all isolates were fully susceptible to penicillin, quinupristin/dalfopristin, chloramphenicol, vancomycin, linezolid, and levofloxacin . among the invasive isolates,  <dig>  were resistant to erythromycin, while the isolates associated with pharyngitis presented significantly higher macrolide resistance – 21% . among the invasive macrolide-resistant isolates,  <dig>  presented the m phenotype and were therefore susceptible to clindamycin, whereas the remaining nine  were also constitutively resistant to clindamycin . the proportion of the two phenotypes was similar among the pharyngitis isolates, with  <dig> isolates  presenting the m phenotype and  <dig>  presenting the mlsb phenotype . all the isolates presenting the m phenotype of macrolide resistance carried only the mef variant of the mef determinant. the cmlsb isolates carried only the erm gene, except for one pharyngitis isolate which also harbored mef, and the only imlsb isolate in the collection that presented the erm gene.\n",
            "\n",
            "\n",
            "pfge cluster\n",
            "\n",
            "a\n",
            "\n",
            "\n",
            "b\n",
            "\n",
            "a clusters are designated by capital letters and a subscript number indicating the number of isolates in each cluster;\n",
            "\n",
            "b the antibiotics tested were penicillin quinupristin/dalfopristin, chloramphenicol, vancomycin, linezolid, levofloxacin, erythromycin, clindamycin, tetracycline, and bacitracin. m, presenting the m phenotype of macrolide resistance; mlsb, presenting the mlsb phenotype of macrolide resistance; tet, non-susceptibility to tetracycline; m,tet, presenting the m phenotype of macrolide resistance and resistance to tetracycline; mlsb,tet, presenting the mlsb phenotype of macrolide resistance and resistance to tetracycline; mlsb,bac, presenting the mlsb phenotype of macrolide resistance and resistance to bacitracin.\n",
            "\n",
            "in contrast to erythromycin, tetracycline resistance was much lower among the pharyngitis isolates when compared with the invasive group . one invasive isolate presented intermediate resistance to tetracycline . all the resistant strains carried the tet gene, except one pharyngitis isolate for which no pcr product was obtained for any of the screened tetracycline-resistance genes. the tet gene was detected in only one pharyngitis isolate, which also harbored tet, while the genes tet and tet were not amplified in any of the studied isolates. overall there was a positive association between the genes tet and erm , but that association was not observed among the subset of invasive isolates, since only three of the  <dig> tet-positive invasive isolates also carried the erm gene .\n",
            "\n",
            "bacitracin resistance was detected in a total of  <dig> isolates , with no significant differences among the two types of infection considered. all these isolates expressed the cmlsb phenotype of macrolide resistance and were tetracycline-susceptible.\n",
            "\n",
            "characterization of gas clones\n",
            "globally, among the  <dig> isolates there were  <dig> emm types,  <dig> t types, and  <dig> sag profiles . in the subset of  <dig> isolates  selected for mlst analysis,  <dig> different sts were identified. nineteen pfge clusters  were obtained including  <dig> pharyngitis isolates and  <dig> invasive isolates  . except for r <dig>  isolates grouped into pfge clusters presented some variability in their emm type, st, t type, or sag profile, with most variability found in the later two properties. still, in most pfge clusters the majority of the isolates were characterized by a single profile of dominant properties. the emm diversity among the pfge clusters differed significantly . within each pfge cluster, different emm types were associated with distinct sag profiles , although globally the emm and pfge had a similar predictive power over the sag profile .\n",
            "\n",
            "\n",
            "pfge cluster\n",
            "\n",
            "a\n",
            "\n",
            "emm\n",
            "\n",
            "b\n",
            "\n",
            "\n",
            "c\n",
            "\n",
            "a clusters are designated by capital letters and a subscript number indicating the number of isolates in each cluster; b nt, non-typeable; c nd, not determined.\n",
            "\n",
            "pfge cluster\n",
            "\n",
            "a\n",
            "\n",
            "emm\n",
            "\n",
            "b\n",
            "\n",
            "a clusters are designated by capital letters and a subscript number indicating the number of isolates in each cluster; b nt, non-typeable.\n",
            "emm\n",
            "types for each pfge cluster\n",
            "\n",
            "pfge cluster\n",
            "\n",
            "a\n",
            "\n",
            "emm\n",
            "95%\n",
            "a pfge clusters a <dig>  e <dig>  f <dig>  g <dig>  k <dig>  m <dig>  o <dig>  p <dig>  and r <dig> include only one emm type .\n",
            "\n",
            "unrelated sts within the same pfge clusters were associated with isolates of different emm types, while isolates of the same emm type presented the same st or single-locus variants  . the only exceptions were st <dig> and st <dig> that were both associated with cluster g <dig> and emm <dig>  but were double-locus variants  of each other. in clone i <dig>  four distinct sts were found. while st <dig> and st <dig> were slvs and were both associated with emm44/ <dig>  st <dig> belonged to a different clonal complex, but was also associated with a different emm type . finally, st <dig> despite being associated with an isolate of a different emm type  is a slv of st <dig>  which may explain why this isolate was clustered in i <dig> and not in the major pfge cluster associated with this emm type .\n",
            "\n",
            "isolates expressing the m phenotype of macrolide resistance belonged mostly to clusters g <dig>  o <dig>  and r <dig>  while the majority of mlsb isolates were clustered in h <dig> and l <dig> . isolates belonging to emm types  <dig>   <dig>  and  <dig> were separated into different pfge clusters according to macrolide resistance , while the remaining characteristics  of each pair of pfge clusters were the same.\n",
            "\n",
            "bacitracin-resistant isolates were all clustered in pfge h <dig> and were characterized as emm28-t <dig>  except for one isolate that was emm22-t <dig>  however, this cluster was not restricted to bacitracin-resistant isolates, since it also included three bacitracin susceptible isolates, two of which were also emm28-t <dig>  while the other was emm <dig> but t non-typable.\n",
            "\n",
            "surface antigen differences between invasive and pharyngitis isolates\n",
            "the invasive isolates were significantly less diverse than the pharyngitis isolates by t typing and sag profiling . however, while the emm type distribution varied between the invasive and pharyngitis isolates  no differences were noted in the t types. sixteen emm types occurred only in invasive infection or pharyngitis, but in most cases the small number of isolates associated with these emm types prevented the differences from reaching statistical significance . in contrast, the overrepresentation of emm types  <dig>   <dig>   <dig>  and  <dig> in one of the groups was statistically supported.\n",
            "\n",
            "\n",
            "95%\n",
            "95%\n",
            "sag differences between invasive and pharyngitis isolates\n",
            "a detailed analysis of the sag profiling results of the isolates is performed elsewhere  <cit> . briefly, the chromosomally encoded sag genes smez and speg were the most frequent among the  <dig> isolates , followed by spec , ssa , spej , spea , spek , speh , spei , and spel and spem, which were always detected together . the association of individual sag genes with disease presentation was tested. in the analysis of these results, the sag genes speg and smez were not considered because they were present in nearly all isolates, and the genes spel and spem were considered as a single entity, since they always co-occurred. individually, the genes spea and spej were both associated with invasive isolates . as expected, strains carrying both these genes were also associated with invasive infections , but no synergistic effect between the two genes was observed, since the presence of one did not significantly increase the representation of the other among invasive isolates. in contrast, spec , ssa , and spel/m  were individually associated with pharyngitis. the combinations spec+spel/m and ssa+spel/m were both associated with pharyngitis , but there was also no synergistic effect relative to the presence of a single gene. however, the association of spec with pharyngitis isolates can be explained by a high frequency of co-occurrence of this gene with ssa, since the isolates harboring spec without ssa were not significantly associated with any of the groups. an interesting situation occurred when analyzing the interaction between spej  and ssa . among isolates carrying spej, the group that also carried ssa was no longer associated with invasive infections, while the association of isolates carrying ssa with pharyngitis was not significantly altered by the presence of spej. this argues for a dominant effect of the presence of ssa over that of spej in determining the invasive capacity of individual isolates. the association of sag profiles with disease presentation was also tested. two sag profiles presented a significant association with invasive isolates, namely sag <dig>  and sag <dig>  . the remaining profiles were not significantly associated with any of the two groups of isolates.\n",
            "\n",
            "when the same kind of analysis was performed for emm types and individual sag genes, three combinations with statistical significance emerged: the association of isolates presenting emm <dig> and spea, and emm <dig> and spej with invasive infections , and the association of isolates carrying emm <dig> and spel/m with pharyngitis . in all cases, no synergistic or antagonistic interaction was detected between emm type and sag gene, since the emm type did not alter the association of the sag gene with a particular group of isolates.\n",
            "\n",
            "differences between the pfge clusters found among invasive infection and pharyngitis\n",
            "the associations described above can be correlated with the pfge clusters which were also different between the invasive and pharyngitis groups of isolates , in agreement with the differences found in emm types . all the  <dig> major pfge clusters occurred in both invasive and pharyngitis isolates, except for r <dig> , which was present only among pharyngeal isolates, but the difference did not reach statistical significance due to the small number of isolates in this cluster. pfge distinguished several groups of isolates belonging to emm types  <dig> and  <dig>  the difference in the distribution between pharyngitis and invasive infection was not found for all pfge clusters containing isolates carrying each of these two emm types, but only for those including macrolide-susceptible isolates . the emm <dig> and emm <dig> isolates expressing macrolide resistance  were grouped into pfge clusters o <dig> and g <dig>  respectively, which presented a similar prevalence among invasive infections and pharyngitis . pfge j <dig>  which included all emm <dig> isolates, was also associated with invasive infections . the emm <dig> association with pharyngitis was not translated into an association of a specific pfge cluster, since the  <dig> emm <dig> strains were scattered into various pfge clusters .\n",
            "\n",
            "\n",
            "not surprisingly, three emm-pfge cluster combinations showed significant associations with infection type: emm1-b <dig> and emm64-j <dig> were associated with invasive infections, while emm4-f <dig> was associated with pharyngitis . it was not possible to detect any synergistic or antagonistic interaction between pfge and emm type in modulating the association of the isolates with either group. the same was true for the statistically significant combinations between pfge clusters and individual sag genes, namely the combination of b <dig> with spea and with spej , and the combination of f <dig> with spec and with ssa .\n",
            "\n",
            "discussion\n",
            "several studies yielding conflicting results have attempted to compare the clonal composition of gas populations causing invasive and non-invasive infections in order to identify particularly virulent clones or properties that may be used as epidemiological markers of invasiveness  <cit> . however, many of those studies were limited in the size and diversity of the gas collections studied or in the typing methodologies used, with most of them relying essentially on emm typing, which has been shown to be insufficient for the complete identification of gas clones  <cit> . in this work, we used several different typing methods to compare a collection of genetically diverse gas isolates recovered from normally sterile sites during a period of six years in portugal  <cit>  with isolates recovered from pharyngeal exudates of patients presenting with tonsillo-pharyngitis, during the same time period and in the same geographical region. the nasopharyngeal mucosa has been suggested to be the main reservoir for gas isolates associated with invasive infections  <cit> .\n",
            "\n",
            "the differences among the gas clones associated with invasive infections and pharyngitis were reflected in antimicrobial resistance, with the invasive group of isolates presenting lower macrolide resistance and higher tetracycline resistance, when compared with the pharyngitis group. among isolates belonging to the same emm types, namely emm <dig> and emm <dig>  only the macrolide-susceptible clones were associated with either invasive infections or pharyngitis. the macrolide-resistant clones of these emm types are reflected in invasive infections according to their prevalence in pharyngitis, suggesting that these are translating more the antibiotic selective pressure than the invasive capacity of the clones. tetracycline is not currently used in the treatment of gas infections but resistance to this antimicrobial in s. pyogenes is usually acquired by horizontal transfer, since the resistance genes are frequently encoded in mobile genetic elements with a wide host range  <cit> . these elements often carry macrolide resistance genes as well, and in s. pyogenes a significant association between the presence of the genes erm and tet has been reported and it has been suggested that tetracycline use could contribute to the selection of macrolide-resistant gas isolates  <cit> . in our study, the association between the presence of the genes erm and tet was observed globally, but not among the invasive isolates, suggesting that the genetic elements carrying tetracycline resistance conferring genes may be different between the two bacterial populations.\n",
            "\n",
            "bacitracin susceptibility is routinely used for the presumptive identification of gas, although resistant clones have been identified in several countries  <cit> . in our gas collection, all the bacitracin-resistant isolates , regardless of the type of infection, were clustered in the same pfge clone  and belonged to st <dig>  although one was emm22-t <dig> while the others were all emm28-t <dig>  isolates with such characteristics had been previously reported in portugal associated with tonsillo-pharyngitis, skin infections and asymptomatic carriage  <cit> , but not with invasive infections. bacitracin resistance among invasive isolates has been previously reported only among isolates recovered in france and in san francisco  <cit> .\n",
            "\n",
            "although 74% of the invasive isolates in our collection belonged to clones which were equally frequent among pharyngitis, suggesting that a significant part of the invasive gas population mirrors the clonal structure of the circulating gas isolates, the remaining isolates represented clones that had an enhanced capacity to cause invasive disease. we also found significant associations between individual properties or pairwise combinations of properties and disease presentation. since in most cases these were also characteristics of the more invasive clones, we cannot exclude that the associations of individual properties or pairwise combinations of properties can reflect, at least partially, the distribution of genetic lineages in the two gas populations analyzed.\n",
            "\n",
            "individually, emm types  <dig> and  <dig> were associated with invasive infections. isolates belonging to these emm types presented the only two sag profiles significantly associated with invasive infections. surprisingly, only one of these sag profiles includes a phage-encoded sag gene . in agreement with our observation, a previous study found that within the same pfge-emm group, the sag profiles significantly associated with invasive infections had a smaller number of sag genes than the dominant profiles in pharyngitis  <cit> . these results suggest that although some sag genes may significantly contribute to the virulence of s. pyogenes, the rise and success of highly virulent gas clones may not hinge upon the acquisition of phage-encoded sag genes. still, in our study, the sag genes spea and spej were both significantly more prevalent among invasive isolates. this association was not substantially affected by emm type, pfge clone, nor by the presence of other sag genes, suggesting that spea and spej can be regarded by themselves as markers for invasiveness. although such association has not been previously noted for spej, the spea gene has been frequently associated with invasive infections  <cit>  and the production of spea by gas isolates has been linked to streptococcal toxic shock syndrome  <cit> .\n",
            "\n",
            "on the other hand, we identified an association of pharyngitis isolates with emm types  <dig> and  <dig>  and with the sag genes spec, ssa, and spel/m. the association of spec with non-invasive infections has been previously reported  <cit> , but in our collection this association could be explained simply by a high frequency of co-occurrence of this gene with ssa which was strongly associated with pharyngitis, as was also noted in a recent study  <cit> . the presence of the genes spel and spem was not previously associated with non-invasive infections.\n",
            "\n",
            "since there is a strong correlation of the sag profile with emm type and of both these properties with pfge type, some of these individual factors frequently co-occurred in the same clones. therefore, combinations of these characteristics were also significantly associated with disease presentation. however, we could not detect any synergistic or antagonistic interactions between most of these characteristics, meaning that their co-occurrence in a particular isolate does not make it more invasive than isolates sharing only one of these characteristics.\n",
            "\n",
            "two pfge clusters were significantly more prevalent among isolates associated with invasive disease than among those causing tonsillo-pharyngitis. one of these was a cluster of macrolide-susceptible isolates characterized as emm1-t1-st <dig> and by the presence of the sag genes spea, speg, spej, and smez , which accounted for 18% of the invasive isolates. m1t <dig> isolates have been frequently associated with severe invasive gas disease, and the acquisition of prophage-encoded virulence genes, as well as horizontal gene transfer events by homologous recombination were implicated in the increased virulence of these isolates  <cit> . however, many epidemiological studies found a similar prevalence of this clone among invasive and non-invasive isolates  <cit> , questioning its enhanced invasive capacity. in contrast to these findings, but similarly to those of others  <cit> , we found an association between this clone and invasive gas disease in portugal, although it can also frequently cause milder infections such as pharyngitis .\n",
            "\n",
            "the other cluster significantly associated with invasive infections in portugal was j <dig>  which was dominated by isolates belonging to emm64-st <dig> and carrying the sag genes speg and smez. a clone with these characteristics has not been previously associated with invasive disease and emm <dig> has been infrequently reported among invasive gas isolates  <cit> . the higher invasive capacity of this clone cannot be attributed to its sag repertoire, since these isolates do not harbor any of the sag genes associated with invasive infection. other, still unidentified, characteristics may be responsible for the properties of this clone.\n",
            "\n",
            "in contrast to these pfge clones, the f <dig> clone of macrolide-susceptible isolates characterized by emm4-t4-st <dig> and harboring the genes spec, ssa and smez was associated with pharyngitis, suggesting that this clone may have a reduced ability to cause invasive disease, in agreement with the negative association between emm <dig> and invasive infection that has been suggested elsewhere  <cit> . the association of emm <dig> with pharyngitis has not been previously reported and was not translated into particular pfge clusters due to the high diversity of emm <dig> isolates.\n",
            "\n",
            "our data confirms that the widely dispersed m1t <dig> clone has enhanced invasiveness but we also identified clones with increased or decreased invasive capacity that may have emerged locally and that have a more limited temporal and geographical spread. the emm alleles and the sag genes characteristic of these clones were associated with particular disease presentations. other individual emm alleles and sag genes were also associated with a higher propensity to cause invasive infections or pharyngitis indicating the importance of these characteristics in determining an isolate’s invasive capacity.\n",
            "\n",
            "other factors that were not evaluated in this study may contribute to a different distribution of gas clones in less severe and more severe infections. these include bacterial factors, such as the occurrence of mutations in transcriptional regulators controlling the expression of virulence factors, which seems to play an important role in the pathogenesis of some gas isolates  <cit> . for other clones, the ability to cause invasive infections may be more dependent on exploiting host factors, like the hla class ii haplotype  <cit> , which may vary in frequency in different human populations.\n",
            "\n",
            "CONCLUSIONS\n",
            "this study established links between particular genetic lineages and the type of infection, indicating that genetic characteristics of the bacteria play an important role in determining the outcome of their interaction with the human host. the different distribution of clones in the two types of infection supports the relevance of pfge as a typing methodology for gas  <cit> . this was further evidenced by the fact that the macrolide-resistant emm <dig> and emm <dig> pfge clones were not associated with any particular disease presentation, contrary to the susceptible clones carrying the same emm types that were associated with invasive infections and pharyngitis, respectively. moreover, in contrast to other reports  <cit>  we found associations between particular emm alleles and sag genes and disease presentation. in this study, we identified emm <dig>  emm <dig>  ssa and spel/m as independent markers for pharyngitis and emm <dig>  emm <dig>  spea, and spej as independent markers for invasiveness. our data re-enforces the multi-factorial nature of gas invasive capacity and highlighted lineages and characteristics, in addition to the well known m1t <dig> lineage, that are associated with particular disease presentations and that may further increase in importance.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "a cluster of invasive isolates belonging to emm types  <dig> and  <dig> and carrying the sag genes spea, spej, spej, spej, spea, spej, spea, spej, spea, spea, spej, spea, spea, spea, spej, spea, spea, spea, spea, spea, spej, and smez were also associated with invasive isolates.\n",
            "a cluster of macrolide-resistant isolates belonging to emm types  <dig> and  <dig> and carrying the sag genes spea, spej, spea, spea, spej, spea, spea, spea, spej, spea, spea, spej, spea, spea, spej, spea, spea, spej, spea, spea, spea, spej, spea, spea, spea, spea, spej, spea, spea, spea, spea, spej, spea, spea, spea, spej, spea, spea, spea, spej, and smez.\n",
            "\n",
            "71\n",
            "[{'summary_text': 'in this study, we analyzed changes in pulmonary gene expression in mice infected with iav or exposed to vehicle only , using mouse models and mouse-adapted iav strain variant pr8_mun , which is closely related to a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  and mouse-adapted iav strain variant pr8_mun , which is closely related to a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto r'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 6194 to 7168 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "due to ease of infection, animal rearing, and the availability of genetically modified strains, using mouse models and viral strains adapted to the murine host has become an attractive approach to studying the mammalian response to influenza a virus  infection. recently, a substantial amount of information has been obtained regarding gene expression changes at various stages of infection in this model  <cit> . these authors showed that the genetic background of different mouse strains strongly influences the susceptibility to iav. for instance, dba/2j mice were highly susceptibility to the pr8_mun iav  strain, whereas c57bl/6j mice were more resistant, as evidenced by less weight loss and an approx. 1000-fold higher viral ld <dig>  conversely, viral load was significantly higher in the dba/2j strain, which also mounted a hyper-inflammatory response with much stronger up-regulation of many immune response-dependent genes. as exemplified by the aforementioned studies, most work in murine models of iav infection has focused on time points during or after established infection , and very little attention has been paid to the first 24 hours . nevertheless, critical aspects of the host response to early steps in viral attachment and entry could conceivably be studied during this early time window. however, due to the temporal proximity to the technical and pharmacological manipulations surrounding the infection process, it is conceivable that both the administration of the anesthetic and the physical and physiological stress from intranasal installation of the inoculate would lead to artifactual signals that are unrelated to the virus-host interaction. we have therefore analyzed changes in pulmonary gene expression in a 5-day time course featuring frequent measurements in the first 24 h, comparing results obtained from mice infected with iav or exposed to vehicle only . we find effects on pulmonary gene expression that can be clearly ascribed to the anesthesia/infection procedure, which are detectable as early as 6 h post treatment and differ between the two mouse strains in terms of magnitude and temporal evolution.\n",
            "\n",
            "methods\n",
            "sample preparation\n",
            "female 12-13-week-old c57bl/6j and dba/2j mice  and mouse-adapted iav strain variant pr8_mun , which is closely related to a/puerto rico/8/ <dig>  were used. mice were weighed on day  <dig> just before induction of anesthesia and on each subsequent day. infections were essentially carried out as described previously  <cit> . briefly, mice were anesthetized by intra-peritoneal injection of 10 μl per g body weight of a stock solution of  <dig>  ml ketamine ,  <dig>  ml 2% xylazine hydrochloride  and 9 ml sterile nacl  <dig> % . for intranasal infection, a viral dose of 2 ×  <dig> focus forming units  of pr8_mun  was administered in a total volume of 20 μl sterile phosphate-buffered saline . during the infection procedure, mice were held in the upright position and additional anesthetic was reinjected as needed. mock treatment was identical to real anesthesia/infections except that vehicle only , not containing virus, was used for intranasal instillation. mice were killed by co <dig> asphyxiation at  <dig>   <dig>   <dig>   <dig>   <dig>  and 120 h with respect to infection or mock treatment. untreated mice were used as t = 0 h control. lungs were removed from the cadavers by cutting the main bronchus and were washed in rnalater rna stabilization reagent  immediately after removal. after transfer into a new tube containing 2 ml rnalater, lungs were stored overnight at 4°c and then at -20°c until further use. all animal work was approved by an external committee according to the regulations on animal welfare of the federal republic of germany.\n",
            "\n",
            "rna isolation and qrt-pcr\n",
            "lungs were homogenized in 4 ml rlt buffer  containing 40 μl β-mercaptoethanol and stored at -80°c in 450 μl aliquots. after thawing, 450 μl of this suspension was mixed with 700 μl qiazol , and all further steps of total rna isolation were performed with the mirneasy kit  according to the manufacturer’s recommendations. real-time rt-pcr  was performed with a lightcycler  <dig>  in  <dig> well plates in 20 μl reaction volumes, using 15 ng cdna  and primers specific for the following targets: the immediate early gene fbj osteoscarcoma oncogene , resistin like α , immune-responsive gene  <dig> , interleukin  <dig> , interleukin 1β , the chemokine  ligand  <dig> , four genes related to interferon pathways , interferon γ , interferon λ <dig> , and myxovirus  resistance  <dig> ), and iav hemagglutinin . quantitect primer assays  were used for all targets except ifnl <dig> and ha. primers for amplification of ifnl <dig> were designed using exon-spanning regions of the ncbi  <cit>  sequence . primer sequences for ha mrna amplification were published previously  <cit> . mouse genome informatics  gene symbols and names were used for all genes  <cit> . the arithmetic mean of the ct values of β actin  and ribosomal protein l <dig>  was used as internal reference for normalization.\n",
            "\n",
            "data analysis\n",
            "data were analyzed using the r environment and programming code  <cit> . qrt-pcr data points with ct ≥ <dig>  corresponding to lack of detection of a target due to technical failure or lack of expression, were assigned a ct of  <dig>  we removed technical outliers in Δct values using the maximum normed residual test  to detect outliers for each condition with a threshold of p ≤ <dig> . a median of  <dig>  biological replicates were available for each data point after outlier removal. anova was used for testing of trends throughout time series, adjusting p values for false discovery rate . for pairwise comparisons, we used tukey’s honest significant differences test for homogeneous variances and dunnett’s modified tukey-kramer pairwise multiple comparison test for heterogeneous variances . we used a significance threshold of p ≤ <dig> .\n",
            "\n",
            "RESULTS\n",
            "changes in body weight\n",
            "weight loss in infected dba/2j mice became manifest by day  <dig>  and reached approx. 25% by day  <dig>  as shown previously  <cit> , weight loss in the infected c57bl/6j mice was less pronounced, as reflected in a mere  <dig> - 10% reduction by day  <dig> –  <dig>  in both strains, statistically significant differences between infected and mock treated mice were observed by day  <dig>  mock-treated mice showed no significant weight loss at any time point. thus, there was no significant effect of the anesthesia/infection procedure on body weight in either mouse strain.\n",
            "\n",
            "viral replication\n",
            "qrt-pcr revealed a brisk rise of mrna encoding iav ha in lungs of both mouse strains after infection . ha mrna was detected at low levels as early as 6 h in both strains, followed by a rapid rise that peaked at 48 h and 120 h in dba/2j and c57bl/6j mice, respectively. ha mrna levels were significantly higher in dba/2j than in c57bl/6j starting around 12 h. as expected, ha mrna was not detected in the mock treated mice.\n",
            "\n",
            "principle component analysis of pulmonary expression of host-encoded mrnas\n",
            "a cluster containing infected and mock treated time points could be identified easily in both mouse strains . a separation between infected and mock-treated samples became evident at 18 h in both mouse strains, as indicated by the lines in figure  <dig>  marked step-offs between  <dig> and 48 h were seen in both strains. consistent with the continuing rise of ha mrna in the c57bl/b <dig> strain between  <dig> and  <dig> h the 120 h time point localized beyond the 48 h time point. in contrast, in the dba/2j strain ha mrna declined between  <dig> and 120 h, and the 120 h time point localized between  <dig> and 48 h in the pca plot. in both strains, the t = 48 h and 120 h mock treated mice localized far away from the infected t =  <dig> and 120 h mice. thus, this analysis showed that in both mouse strains procedure-related effects did not contribute significantly to the gene expression changes observed in infection beyond 24 h and that the evolution of gene expression reprogramming correlated well with transcription of ha mrna in both strains.\n",
            "\n",
            "pulmonary expression of individual host-encoded mrnas\n",
            "results are shown in figure  <dig>  all  <dig> host mrnas exhibited at least some evidence of regulation throughout the time course . four mrnas were also significantly regulated in response to mock treatment, but two of these  were regulated only in the dba/2j strain. fos, il1b, stat <dig>  ifng, ifnl <dig>  and mx <dig> mrnas were not regulated by mock treatment in either strain.\n",
            "\n",
            "fos mrna was expressed at low level and increased significantly after 48 h in the infected dba/2j mice only . there was no evidence for mock treatment-dependent regulation of this mrna in either mouse strain . the apparent tendency of an increase in infected c57bl/6j mice toward the later time points was not significant. retnla mrna increased significantly at all time points in infected dba/2j mice. anova revealed a significant effect of the mock treatment, with an effect size resembling that of infection between  <dig> and 24 h, suggesting procedure-dependent regulation through 24 h and infection-dependent regulation thereafter . in infected c57bl/6j mice, retnla increased between 18 h and 120 h . in mock treated mice of this strain, expression increased steadily at the early time points, with significant regulation at 18 h and 24 h. this suggested a procedure-dependent regulation of retnla resembling that observed in the dba/2j mice. irg <dig> mrna expression increased in mock-treated dba/2j mice between  <dig> and 18 h . this gene was up-regulated in dba/2j infected mice at all time points, reaching a maximal 630-fold induction on day  <dig>  in the c57bl/6j strain there was no increase in irg <dig> due to mock treatment, and the infection-dependent increase was less pronounced, reaching a max. 150-fold induction at 120 h. il <dig> mrna increased in both strains beginning 6 h after infection or mock treatment, with stronger regulation being observed in the dba/2j mice . in dba/2j mice the mock treatment effect declined towards 18 h, and clear differences between infected and mock treated mice became apparent at 24 h. in the c57bl/6j mice, an infection-dependent rise in il <dig> mrna was observed somewhat later  . il1b mrna increased in infected mice of both strains at 48 h and 120 h, and there was a tendency  toward a mock treatment effect between  <dig> and 18 h in the dba/2j strain . cxcl <dig> mrna was up-regulated in dba/2j mock-treated mice at 6 h , whereas it was not affected by mock treatment in the c57bl/6j mice . in both mouse strains cxcl <dig> mrna was significantly elevated in the infected mice, beginning at 6 h in the dba/2j and at 18 h in c57bl/6j. stat <dig> expression was not affected by mock treatment in dba/2j mice, but there was a slight trend  for up-regulation in c57bl/6j mice. an infection-dependent up-regulation became apparent at 24 h and 48 h in dba/2j and c57bl/6j mice, respectively. similar to stat <dig>  ifng was up-regulated in both mouse strains beginning around 48 h, and there was no evidence for regulation due to the infection procedure. ifnl <dig> was not detected  in about 40% of untreated and mock treated dba/2j mice; fold change values therefore represent an underestimation . a significant rise after infection became apparent at 48 h, reaching a mean ct of  <dig> . in c57bl/6j mice, it was not detected in about 80% of the untreated and mock treated samples, suggesting a lower baseline expression than in dba/2j . a first significant infection-dependent regulation was observed at 18 h, where ifnl <dig> was detected in all dba/2j and four of five c57bl/6j samples. ifnl <dig> was detected in all 24 h samples  and continued to rise through 120 h. there was no evidence for a mock treatment effect on ifnl <dig> in either mouse strain. mx <dig> mrna expression  was not regulated in response to mock treatment in either strain. in contrast, it strongly increased during infection, showing a significant rise beginning at 12 h in dba/2j and 24 h in c57bl/6j mice.\n",
            "\n",
            "taken together, these results allow classifying the analyzed genes into three groups:  genes that were regulated in response to mock treatment and infection in both strains ,  genes that were regulated in response to both mock treatment and infection in the dba/2j strain only , and  those whose expression changed in response to infection only . of note, the latter group contained all four interferon pathway-related mrnas.\n",
            "\n",
            "correlation with iav ha mrna\n",
            "expression of the  <dig> host mrnas was then correlated with ha mrna expression . overall, correlations were higher in the dba/2j strain. only il1b correlated more strongly in c57bl/6j than in dba/2j. mx <dig> and ifnl <dig> mrna levels correlated best with ha mrna expression in both strains, whereas fos mrna was the only one that did not correlate with ha mrna.\n",
            "1\n",
            "\n",
            "\n",
            "1values correspond to spearman correlation coefficient in mouse strains infected with iav, sorted by decreasing values in dba/2j mice. p values : ***, ≤ <dig> ; **, ≤ <dig> ; *, ≤ <dig> .\n",
            "\n",
            "regulation across all  <dig> target mrnas\n",
            "results are summarized in figure  <dig>  considering regulation across all  <dig> target mrnas combined, we detected a significant up-regulation at all time points after 0 h in infected dba/2j mice . among mock treated dba/2j mice, an up-regulation was observed at  <dig>   <dig> and 24 h post treatment. the strongest effect was detected at 6 h  which nearly equaled the regulation in infected mice . a significant difference between infected and mock-treated dba/2j mice could be discerned by anova beginning at 12 h, but a contribution of a procedure-related effect to mrna expression in the infected mice could be excluded only from 48 h onward. messenger rna up-regulation peaked at 48 h and began to decline by 120 h. in the c57bl/6j strain, overall up-regulation was less than in the dba/2j strain. in this strain, the expression change at 6 h seemed to be due to the anesthesia/infection procedure in both infected and mock-treated mice, as fold induction was nearly identical in both . as in the dba/2j strain, a procedure-dependent effect seemed to persist through 24 h . infection-dependent mrna up-regulation first became manifest at 18 h and continued to rise between  <dig> and 120 h. thus, this analysis supported the kinetics of evolution of infection-dependent transcription in both mouse strains, as revealed by the pca . in addition, consistent with the results shown in figure  <dig>  it showed that procedure-dependent effects occurred before 48 h and were more pronounced in the dba/2j strain.\n",
            "\n",
            "discussion\n",
            "this analysis of sequential changes in pulmonary expression of several mrnas after real or simulated iav infection revealed effects that can be ascribed to anesthesia and/or the intranasal inoculation procedure. the results clearly demonstrate that the appropriate control group treated with a simulated anesthesia/infection should always be included in studies of iav infection in mice that cover approximately the first  <dig> h post infection.\n",
            "\n",
            "what might be the underlying pathophysiological mechanisms? anesthesia is known to influence cytokine expression in humans, but actually appears to have an anti-inflammatory effect as, for instance, suggested by reduction of circulating il <dig> levels  <cit> . the intranasal infection procedure appears to be a more likely candidate. despite the relatively small volume of 20 μl that is used and the near physiological properties of pbs, we consider it likely that entry of pbs into the airway creates a stress response similar to that observed after fluid aspiration, including at least focal pulmonary hypoxia due to bronchospasm. responsible mechanisms may both relate to stimulation of nerve endings in the airway epithelium and direct noxious stimulation of airway epithelial cells. indeed, except for irg <dig>  three of the four mrnas whose expression was regulated in response to mock treatment are known to be induced during a stress response or hypoxia at the cellular or tissue level . the fourth one, irg <dig>  is preferentially expressed in macrophages, is strongly induced during macrophage activation, and localizes to mitochondria  <cit> . its expression in stress or hypoxia has not been examined, and it would therefore be interesting to test whether it plays a role in these processes. the four interferon related genes  were clearly induced in infected mice only. they have all been shown to play important roles in the host defense against iav and other viruses  <cit> . the mx <dig> gene is nonfunctional  in certain mouse strains including dba/2j and c57bl/6j, but even the nonfunctional murine form is fully interferon inducible  <cit> , suggesting that it does reflect the anti-influenza interferon response of the dba/2j and c57bl/6j mice. among these four genes, only stat <dig> has been shown to be regulated by stress or hypoxia  <cit> . interestingly, it was not affected by the mock treatment in the presented study, perhaps because its sensitivity to regulation in this murine model is not high enough to respond to any stress/hypoxia due to the mock treatment. indeed, its upregulation in the infection was much smaller compared to the other three interferon-related genes. thus, the observation that expression of these four interferon-related mrnas was not affected by the mock treatment supports the aforementioned notion that the procedure-associated effects in this model relate to a stress response that can be functionally separated from the antiviral response.\n",
            "\n",
            "differences between the two mouse strains\n",
            "differences were observed in the magnitude of the response to both mock treatment and viral infection. the fact that both procedure and infection-related responses were more vigorous in the dba/2j mice agrees with the previously described overall stronger inflammation in this strain during iav infection  <cit> . this may reflect a greater intrinsic propensity to inflammation, but also the higher rate of viral replication in this strain. we favor a combination of both models, as the procedure-dependent effects, too, were brisker in the dba/2j mice.\n",
            "\n",
            "limitations\n",
            "the relatively small sample size represents a limitation of this study. nonetheless, statistical significance was reached for several variables. a larger sample size would likely reveal additional significant changes, such as procedure-dependent regulation of il1b, at least in the dba/2j strain, in which there currently is a tendency toward significance . in addition, the small number of target mrnas does not represent overall gene expression in the lung. other methods such as rna deep sequencing would likely reveal genes showing an earlier response to iav infection or a longer persistence of procedure-dependent effects.\n",
            "\n",
            "CONCLUSIONS\n",
            "despite the aforementioned limitations, the presented results clearly show that the manipulations surrounding the infection procedure can affect pulmonary gene expression in a host strain-dependent manner for approx. 24 h. thus, “mock treatment” controls should be included in all murine studies on iav infection where measurements are to be taken within approx. the first 24 h. likewise, such controls are likely needed in similar studies with other viral and non-viral respiratory pathogens. the degree to which a mock effect might interfere with infection-dependent data would depend on the kinetics of replication and spread of the respective infection in the host animal.\n",
            "\n",
            "abbreviations\n",
            "fdr: false discovery rate; ffu: focus forming units; h: hour; iav: influenza a virus; mgi: mouse genome informatics; pbs: phosphate-buffered saline; qrt-pcr: real-time reverse transcriptase polymerase chain reaction. gene name abbreviations are spelled out in the methods section.\n",
            "\n",
            "competing interests\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "authors' contributions\n",
            "mp experimental design, animal work, laboratory analyses, graphics, data analysis, preparation of manuscript. mt experimental design, laboratory and data analyses, preparation of manuscript. fk data analysis. ks experimental design, preparation of manuscript. fp experimental design, preparation of the manuscript, supervision of the study. all authors read and approved the final manuscript.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study, we analyzed changes in pulmonary gene expression in mice infected with iav or exposed to vehicle only , using mouse models and mouse-adapted iav strain variant pr8_mun , which is closely related to a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  and mouse-adapted iav strain variant pr8_mun , which is closely related to a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto rico/8/ <dig>  a/puerto r\n",
            "72\n",
            "[{'summary_text': 'the al-sensitive maize genotype s1587- <dig> had higher levels of peroxidases than the al-tolerant genotype s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 6434 to 7168 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "acid soils are the most important cause of low yield for many crops  <cit> . about 30% of the world's soils are acidic, and 60% of them are in tropical and subtropical areas associated with long periods of hot and moist weather  <cit> . soil acidification is an increasing problem in the united states and europe because of acid rain, removal of natural plant coverage from large production areas and the use of ammonium-based fertilizers  <cit> . one of the major problems caused by soil acidification is aluminum  phytotoxicity. al is the principal component of mineral soils and is present in a wide range of primary and secondary minerals  <cit> . in soils with ph above  <dig>  al is precipitated predominately in gibsit form 3) and has no phytotoxic effect. at lower ph, al <dig> is solubilized and al is released.\n",
            "\n",
            "the most evident symptom of al toxicity is the inhibition of root growth. in maize root tips, al induces a rapid change in cell number and positioning  <cit> , and recent evidence suggests that dna damage and interference with cell-cycle progression and cell differentiation are the primary causes of root growth inhibition due to al toxicity  <cit> . other reported effects of al exposure are the disruption of ca2+ homeostasis  <cit> , increased acc oxidase activity with a consequent increase in ethylene production  <cit> , al binding to cell wall polysaccharides  <cit>  and reduced membrane fluidity  <cit> .\n",
            "\n",
            "to cope with al stress, plants activate exclusion and tolerance mechanisms  <cit> . exclusion mechanisms take place outside the roots and prevent the entry of al into the cell. these mechanisms include cell wall al immobilization, increased selective permeability of the plasma membrane, rhizosphere ph barrier formation and quelling by exudates such as organic acids and phenolic compounds  <cit> . tolerance mechanisms are active after al enters the cell - al ions can be quelled in the cytosol, compartmentalized inside the vacuole or proteins that bind directly to al may become highly expressed  <cit> .\n",
            "\n",
            "among all of the proposed mechanisms, organic acid release is the most well-characterized resistance strategy used by plants. since the first report demonstrating al-induced malate secretion in wheat  <cit> , several research groups have observed that organic acid exudation is higher in tolerant than sensitive genotypes in species such as snap beans  <cit> , wheat  <cit>  and maize  <cit> . however, in maize and wheat, organic acid release does not correlate with resistance in all genotypes, indicating that other mechanisms, such as active al exclusion, may also play a relevant role  <cit> . similarly, maron et al.  <cit>  and kumari et al.  <cit>  recently demonstrated that tolerance in maize and arabidopsis is not associated with increased expression of genes encoding enzymes responsible for organic acid biosynthesis, but rather with differential expression of their transporters.\n",
            "\n",
            "the identification of genes related to al tolerance has indicated that a plethora of biological functions are influenced by this ion. with the advent of cdna arrays, the evaluation of global gene expression changes in response to al stress allowed the identification of a broader number of genes that are modulated by this ion  <cit> . guo and colleagues  <cit>  used isogenic lines of wheat with differential tolerance to al and identified  <dig> differentially expressed genes, including malate transporters, a β-glucosidase, lectin and a histidine kinase. kumari et al.  <cit>  reported that exposure to al induces several ribosomal protein genes, peptidases and phosphatases. maron et al.  <cit>  compared gene expression in two maize genotypes with contrasting al tolerance and found that several genes involved in processes such as cell wall remodeling, response to oxidative stress and pi starvation were differentially regulated.\n",
            "\n",
            "while the identification of genes related to al stress has led to a greater understanding of plant responses to this ion, these studies have been conducted mostly using hydroponic culture. this growth condition may not adequately mimic the soil environment with respect to rhizosphere development  <cit> , which involves a complex mixture of microorganisms, border cells and mucilage  <cit> . several other studies have addressed the role of mucilage in the detection and avoidance of al toxicity. horst et al.  <cit>  demonstrated that 50% of all al in root apexes of vigna unguiculata is sequestered by the mucilage layer, and its removal increases root sensitivity to al  <cit> . similarly, archambauldt et al.  <cit>  found that mucilage production by sensitive varieties of wheat was inhibited more rapidly than that by tolerant varieties when exposed to phytotoxic levels of al  <cit> . similarly, miyasaka and hawes  <cit>  found evidence that in snap beans, border cells are involved in detecting and avoiding al toxicity. by contrast, li et al.  <cit>  observed that root growth inhibition in maize was not affected by the removal of root mucilage. these findings indicate that different species, or even different varieties of the same species, can present distinct resistance and/or tolerance mechanisms. therefore, evaluating al tolerance in conditions that are similar to those in the field may provide a better understanding of the mechanisms required to avoid al toxicity.\n",
            "\n",
            "here we present an analysis of transcriptome changes in two maize varieties with contrasting levels of al tolerance, using acid soil as the growth substrate. our analysis identified genes in several metabolic pathways whose expression was modified when plants were growth in acid soil. while we found some al-responsive genes previously identified in studies carried out in hydroponic growth conditions, growth in acid soil clearly also triggered a new suite of physiological and transcriptional responses not previously reported. taken together, our results offer a more complete picture of the transcriptomic changes imposed by acid soils, and they may lead to the discovery of novel genes involved in al tolerance.\n",
            "\n",
            "RESULTS\n",
            "physiology of maize seedlings grown in acid soil\n",
            "most recent studies aiming to characterize plant transcriptomic or proteomic responses to al exposure have used hydroponic culture  <cit> . in the present study, soil was used as the substrate to better mimic field conditions and to allow the maintenance of all root apex components and root architecture. plants were grown in dark red latossol  with an al content of  <dig> mmolc/dm <dig>  as a control, the same soil was used, but with ph corrected to  <dig>  . two lines used in previous studies that evaluated al tolerance in hydroponic growth conditions were characterized: cat100- <dig>  and s1587- <dig>   <cit> . s1587- <dig> is a somaclonal variant regenerated from a callus culture of cat100- <dig>  <cit> . relative root growth  was measured after one and three days of growth in soil. the al-sensitive plants were severely affected by acid soil at both time points, while the al-tolerant plants were affected only after three days and to a significantly lower extent . both maize genotypes had higher levels of callose when grown in acid soil, a response typically correlated with al stress  <cit> . however, the increase in callose content was significantly higher in the al-sensitive line . to investigate whether the root inhibition and callose accumulation were due to al phytotoxicity associated with acid soil, the al absorbed by root tips was quantified after one and three days of soil exposure. figure  <dig> demonstrates that the sensitive line s1587- <dig> had significantly higher amounts of al than the al-tolerant variety. these results indicate that these maize varieties have different physiological responses to acid soil and that these responses are most likely due to the presence of al. nevertheless, we cannot exclude the possibility that ph also contributes to plant responses during acid soil exposure.\n",
            "\n",
            "gene expression profiling\n",
            "the affymetrix genechip® maize genome array was used to evaluate the transcriptional response of the two contrasting maize genotypes to growth in acid soil. analysis of variance  was used to dissect the transcriptional responses associated with the individual maize lines , time of collection , soil type  and all possible interactions . in the al-tolerant line  exposed to acid soil for one day, only eight genes were differentially expressed compared to plants grown in control soil . the number of differentially regulated genes increased  after three days of treatment . however, the al-sensitive maize line showed a significantly higher number of differentially expressed genes. on the first day,  <dig> genes were differentially regulated , while  <dig> were affected by the treatment on the third day . the genes that were differentially regulated under all conditions are described in the additional file  <dig> , and the number of genes up- or down-regulated in each genotype and at each time point is shown in figure  <dig> \n",
            "\n",
            "to further evaluate the quantitative extent by which genes were differentially regulated between the two maize lines, the difference in the least-square means estimates  of gene expression levels between cat100- <dig> and s1587- <dig> grown in acid soil were calculated at each time point , and the statistical significance was assessed. we aimed to identify genes induced by acid soil in cat100- <dig> that also had significantly higher absolute levels than in s1587- <dig>  we observed that none of the genes differentially expressed in cat100- <dig> after one day of acid soil treatment presented significantly higher expression levels than in s1587- <dig> under the same conditions. however, eight genes out of the  <dig> differentially expressed in cat100- <dig> after three days were also significantly more highly expressed relative to s1587- <dig> growing in acid soil for three days. six of these genes are in the group of  <dig> exclusively identified as differentially regulated in cat100- <dig> after three days , while the other two were also differentially expressed in s1587- <dig> after three days .\n",
            "\n",
            "due to the large number of genes identified in the al-sensitive maize, a functional analysis was performed with gene ontologies to help identify the pathways affected by the toxicity imposed by acid soil. most of the genes up-regulated on the first day are involved in lipid metabolism, oxidative stress responses and cell wall metabolism . most of the genes down-regulated on the first day encode proteins involved in dna packaging and cell cycle . most of the genes up-regulated on the third day are involved in cell wall metabolism, oxidative stress responses and anionic transport . on the other hand, most of the repressed genes are involved in protein metabolism .\n",
            "\n",
            "validation of gene expression profiles using qpcr\n",
            "to validate the microarray results, eleven differentially expressed genes were selected and real-time qpcr was performed . this validation was done with two independent biological replicates . a significant correlation between the two data sets was observed .\n",
            "\n",
            "comparison with gene expression responses to al treatment in hydroponic culture\n",
            "the soil treatment used in this work has two major variables that must be considered, the presence of phytotoxic al and the ph. however, it is not possible to separate these properties in the soil or even use a different acid soil with no al because diversity in the physical and chemical characteristics would affect the results. therefore, hydroponic culture has been used to evaluate the effects of ph and al levels on the expression of selected genes. six out of the eight acid soil-induced genes from cat100- <dig> that also had absolute levels higher than those of s1587- <dig> were used in an experiment consisting of three treatments: ph  <dig> ; ph  <dig>  with no al and ph  <dig>  with  <dig> μm al. the effect on gene expression in cat100- <dig> seedlings is illustrated in figure 8a . the relative expression in seedlings grown in ph  <dig>  with al relative to ph  <dig>  without al  is shown in figure 8b. genes such as zm. <dig> .a1_at , zm. <dig> .a1_at  and zm. <dig> .a1_at  showed no significant differential regulation under treatment with ph  <dig>  in the presence or absence of al or between ph  <dig>  and ph  <dig>  treatments, suggesting significant differences between the gene expression profiles from the hydroponic and soil experiments. however, zm. <dig> .s1_at , zm. <dig> .a1_at  and zm. <dig> .a1_at  were up-regulated in the presence of al, indicating that al and not ph was the main factor behind their induction in the soil treatment.\n",
            "\n",
            "to further highlight the different responses of maize when grown in hydroponics versus soil, we compared the target sequences from the affymetrix platform used in this work with the target sequences from maron et al.  <cit> . we observed that only a fraction of the genes modulated by al in cat100- <dig> grown in the hydroponic experiment were also modulated in acid soil-grown plants: of the  <dig> genes differentially expressed in cat100- <dig> grown in soil, only six were also found in the hydroponic assay. we also compared our data from s1587- <dig> with the data from the al-sensitive line l <dig> because in the work of maron et al.,  <cit>  s1587- <dig> was not used. in this case, the differences were even higher because only  <dig> out  <dig> genes modulated in s1587- <dig> were common to both gene sets. a complete list of the genes found in both experiments is shown in additional file 2: table s <dig> \n",
            "\n",
            "discussion\n",
            "physiology of maize roots grown in acid soil\n",
            "plant tolerance to al is usually evaluated using hydroponic culture  <cit> , but conflicting results may be obtained when compared to soil conditions  <cit> . even when al tolerance is consistent between hydroponic and soil conditions  <cit> , plant responses may differ in the two growth systems. to address this limitation, we evaluated maize tolerance to acid soil containing  <dig> mmolc/dm <dig>  of al and at ph  <dig> . these conditions are within the range of previous studies and are sufficient to elicit a phytotoxic response  <cit> , allowing phenotypic discrimination between two maize lines with distinct levels of resistance to al, cat100- <dig>  and s1587- <dig> . as shown in previous studies using these and other maize lines  <cit> , cat100- <dig> accumulated less al in its root tips when grown in acid soil when compared to s1587- <dig>  the amount of al absorbed by root tips is indicative of the sensitivity of plants to this abiotic stress  <cit> , presumably because genotypes that accumulate less al in their root apexes have a more efficient exclusion mechanism. however, after prolonged exposure , the amount of al in cat100- <dig> roots continued to increase , in disagreement with previous studies that reported a reduction in al accumulation in root tips after  <dig> hours in hydroponics  <cit> . therefore, the exclusion mechanism of cat100- <dig> appears to have less activity in soil than in hydroponic conditions. nonetheless, al-induced root growth inhibition and callose formation were markedly more limited in cat100- <dig> than in s1587- <dig> \n",
            "\n",
            "gene expression profiles of an al-tolerant and an al-sensitive maize line\n",
            "the abiotic stress caused by the toxicity of acid soil was clearly reflected in the gene expression profiles. the number of genes differentially regulated between control and treated  plants increased in both cat100- <dig> and s1587- <dig> when they were exposed to the stress for an extended period. it is worth mentioning that these changes observed in the transcriptome certainly reflect both direct and indirect effects of the stress caused by acid soil. in the field, where roots continuously grow and explore different soil regions, this is also certainly true. interestingly, a smaller fraction of the transcriptome responded to acid soil stress in the resistant line cat100- <dig> than in s1587- <dig>  similarly to previous reports using al and hydroponics  <cit> . therefore, part of the resistance of cat100- <dig>  may involve a mechanism that limits major disturbances to plant function and thereby avoids a cascade of detrimental downstream effects.\n",
            "\n",
            "genes that are modulated by a stress are natural candidates for explaining the defenses activated by plants. by using contrasting genotypes it is possible to narrow the scope to genes with a higher probability of playing major roles in the plant response. therefore, we selected eight genes that were more highly expressed in cat100- <dig> than in s1587- <dig> under acid soil conditions. six of these genes were also found to be induced by acid soil only in cat100- <dig>  while two were also induced in s1587- <dig>  although to a lesser degree. we further characterized those six genes that were specific to cat100- <dig>  which encoded a gdsl-motif lipase hydrolase family protein, a pod-specific dehydrogenase/retinol dehydrogenase  <dig>  a glycine-rich protein , a member of the wrky transcriptional factor family and two unknown proteins. however, the hydroponic experiment demonstrated that only three of these genes have interesting expression patterns . although further work will be needed to assess the individual contribution of each gene to acid soil tolerance, these data give insight into the strategies used by plants in fields where this stress takes place.\n",
            "\n",
            "zm. <dig> .s1_at  was first described as a gene involved in rape oilseed pod development  <cit> . members of this family may function as a bridging molecule between the nutritional signaling pathway and the hormone biosynthesis pathway in arabidopsis  <cit> . this member is associated with aba production and is critical for growth and development, and also for plant responses to stress via glucose signaling  <cit> .\n",
            "\n",
            "the wrky family of transcription factors was first identified in plants and presents a high number of members  <cit> . about  <dig> members have been indentified in arabidopsis, and several of them are induced in response to wounding, pathogen infection and abiotic stresses such as drought, cold and salinity  <cit> . our microarray experiment indentified zm. <dig> .a1_at  as differentially expressed in cat100- <dig> after three days of acid soil treatment and also as presenting higher expression than in the al-sensitive s1587- <dig> genotype. hydroponic gene expression experiments also demonstrated that al induces the expression of this gene. kumari and colleagues  <cit>  identified two wrky family members as being down-regulated after exposure of arabidopsis to al. meanwhile, goodwin and sutter  <cit> , also studying arabidopsis, identified wrky  <dig> as up-regulated due to al treatment. an additional study identified another wrky member as up-regulated due to al and cd stress. using the same tolerant variety  maron et al.  <cit>  identified two wrky family members after  <dig> h of al treatment. these results suggest that this transcription factor may be involved in the regulation of other genes that contribute to acid soil tolerance in plants.\n",
            "\n",
            "zm. <dig> .a1_at is a protein with unknown function that was up-regulated in acid soil and in hydroponics, indicating that al rather than low ph is the inducer. interestingly, this protein has a conserved domain typical of methyltransferases , which are responsible for methylation of several cellular components such as dna, rna, proteins and also small molecules  <cit> . these enzymes may also play important roles in disease resistance, growth and development  <cit> . studies with rice  <cit> , arabidopsis  <cit>  and tomato  <cit>  have also identified members of this family as up-regulated after al treatment. this is the first study to detect a potential role in acid soil tolerance for this maize protein.\n",
            "\n",
            "a comparison of the transcriptional profile of roots grown in soil  with that of roots grown hydroponically  <cit>  showed only a minor overlap between these two growth systems. although such comparisons are difficult because of differences in the chip platforms, conditions in different laboratories and other aspects, it strongly suggest that root responses in acid soil differ at least in part from those observed in hydroponic experiments. however, several pathways that are affected by al in hydroponics were also observed in acid soil grown plants, as discussed below.\n",
            "\n",
            "organic acid biosynthesis\n",
            "of particular interest are genes involved in organic acid biosynthesis, which can protect the plant from deleterious effects of al by binding to it after being secreted by root apexes  <cit> . however, only one gene belonging to an organic acid biosynthesis pathway was identified as down-regulated in s1587- <dig> after a three-day treatment . previous studies have suggested that cat100- <dig> activates pre-existing anionic channels after exposure to al but prior to activation of the organic acid biosynthesis pathways  <cit> . it has also been observed that the levels of citrate exudation induced by al in cat100- <dig> roots is higher than in other al-sensitive genotypes , but it stays constant over time  <cit> , although no correlation between organic acid exudation and al-alteration of genes of the organic acid biosynthetic pathway has been observed in maize  <cit> .\n",
            "\n",
            "oxidative stress in soil grown plants\n",
            "plant cells normally produce reactive oxygen species  due to cellular processes that result in reduction of oxygen molecules. plants have several enzymes capable of producing ros and others that fight ros to avoid cellular damage. al toxicity can lead to an imbalance that results in oxidative stress and increases in the activity of enzymes and genes that reduce ros effects, as previously observed in maize  <cit>  and other plant species  <cit> . specifically in the case of the two maize lines used in this work, a previous study indicates that s1587- <dig> produces higher levels of ros under al stress, while ros production remains constant in cat100- <dig>  <cit> . expectedly, genes involved in ros production such as an oxalate oxidase  and four germins  were up-regulated in s1587- <dig> after acid soil treatment. in contrast, the up-regulation of genes implicated in the production of ros was not detected in the al-tolerant genotype cat100- <dig> under stress.\n",
            "\n",
            "higher expression of oxalate oxidases in s1587- <dig> was also correlated with the up-regulation of peroxidases in the al-sensitive genotype. a gene encoding a glutathione peroxidase  was up-regulated in the al-sensitive line, confirming the data obtained by boscolo et al.  <cit> , who found higher levels of this enzyme in s1787- <dig> under al stress. in fact, more ros scavenging genes were differentially expressed in s1587- <dig> than in cat100- <dig>  possibly reflecting a response to the up-regulation of ros-producing genes. these data are in agreement with those from maron et al.  <cit>  suggesting that cat100- <dig> has mechanisms that act before the oxidative stress takes place. however, none of the genes encoding superoxide dismutase were identified as up-regulated in s1787- <dig>  in contrast to the induction of this enzyme  <cit>  and transcriptional up-regulation detected previously in hydroponics  <cit> .\n",
            "\n",
            "the degree of ros production and the enzymes involved in their metabolism may partially explain the differences in root growth detected in s1587- <dig> and cat100- <dig> in acid soil. together with oxalate oxidase, peroxidases act to remodel the cell wall during development and stress responses  <cit> . our results indicate that the elevated number of peroxidases up-regulated in s1587- <dig> may be one of the causes of the root inhibition in this genotype, either by increasing ros production or by changing the cell wall structure. on the other hand, cat100- <dig> appears to be more effective at preventing ros generation. this is supported by the smaller number of genes contributing to ros production compared to the number observed in hydroponic culture  <cit>  and also by the constitutive  expression of ros scavenging genes  in cat100- <dig> \n",
            "\n",
            "the phenylpropanoid pathway is activated in al-sensitive maize plants\n",
            "the higher expression of peroxidases in s171587- <dig> in acid soil was also correlated with an increase in the expression of several genes implicated in the synthesis of monolignols. genes related to lignin biosynthesis have often been identified as responding to al stress in monocots  <cit> , and higher lignin deposition has been associated with root growth inhibition in al-sensitive wheat genotypes  <cit> . the phenylpropanoid pathway is the last biochemical step in the production of monolignols and the lignin polymer. the up-regulation of genes in the shikimate pathway, including shikimate kinases  and chorismate mutase  might increase the production of phenylalanine, the precursor for the phenylpropanoid pathway. cinnamoyl-reductase  and several o-methyltransferases were up-regulated at days  <dig> and  <dig> in the al-sensitive line, indicating that s1587- <dig> might accumulate lignin, reducing root growth. similarly, several genes related to callose biosynthesis were up-regulated in the al-sensitive line at both time points , corroborating the physiological data that show higher levels of callose in s1587- <dig> \n",
            "\n",
            "insights into hormonal responses to acid soil\n",
            "al has also been shown to impact root growth by modifying the levels of phytohormones such as auxin  <cit>  and ethylene  <cit> . genes encoding enzymes involved in auxin biosynthesis such as iaa amidohydrolase  and anthranilate phosphoribosyltransferase  were up-regulated in the root apex of the sensitive line s1587- <dig> when under acid soil stress, while the auxin-degrading enzyme indole-3-acetate beta-glucosyltransferase  was down-regulated after three days of acid soil exposure. although auxin can induce new root formation, higher concentrations inhibit root elongation and enhance adventitious root formation. the genotype s1587- <dig> grown in acid soil developed significantly more lateral roots compared to plants grown under control conditions . in coordination with the up-regulation of auxin-responsive genes , its f-box receptor  <cit>  was also up-regulated in s1587- <dig> . this might indicate a compensatory mechanism of primary root inhibition and lateral root stimulation to avoid nutrient and water deficiencies.\n",
            "\n",
            "the transcriptome response of the al-susceptible line s1587- <dig> to acid soil treatment also involved the up-regulation of two acc oxidases , suggesting activation of the ethylene production pathway. the phytohormone ethylene mediates root growth inhibition  <cit> , and treatment with inhibitors of ethylene perception increases root elongation  <cit> . therefore, increased ethylene production is involved in root growth inhibition in the al-sensitive genotype, and this response might be modulated not only by the germplasm but also by the culture conditions.\n",
            "\n",
            "CONCLUSIONS\n",
            "in this study we have characterized the transcriptomic changes of maize roots growing in acid soil containing toxic levels of al. our data highlighted several metabolic pathways that are challenged due to the stress caused by acid soil, including those involved with ros production and detoxification, cell wall structure and hormone biosynthesis. several genes previously reported as up-regulated by al treatment in hydroponic experiments were also identified in acid soil grown plants. most interestingly, we found genes that provide interesting clues to the mechanisms underlying the acid soil tolerance of an al-tolerant maize line. these genes encode a gdsl-motif lipase hydrolase family protein, a pod-specific dehydrogenase/retinol dehydrogenase  <dig>  grp, wrky and two proteins of unknown function. taken together, these data provide a better understanding of the basis of al toxicity and tolerance in acid soils.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the al-sensitive maize genotype s1587- <dig> had higher levels of peroxidases than the al-tolerant genotype s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig> had higher levels of peroxidases than the al-sensitive line s1587- <dig>  the al-sensitive line s1587- <dig\n",
            "73\n",
            "[{'summary_text': 'in this paper, we describe the cloning and purification of the tspgwi gene without accompanying  mtase.\\nin this paper, we describe the cloning and purification of  <dig> kda tspgwi, a bifunctional type ii rease that recognises and cleaves a specific dna sequence, and a cognate dna methyltransferase  that can methylate the same target site, thereby preventing the cleavage of host dna by the rease.\\nin this paper, we describe the cloning and purification of  <dig> kda tspgwi, a bifunctional type ii rease that recognises and cleaves a specific dna sequence and cleaves outside it at a defined distance, without complete cleavage of the plasmid dna.\\nin this paper, we describe the cloning and purification of  <dig> kda tspgwi, a bifunctional type ii rease that recognises and cleaves a specific dna sequence and cleaves outside it at a defined distance, without complete cleavage of the plasmid dna sequence.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 7596 to 8192 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "restriction-modification systems  usually consist of at least two enzymatic activities: a restriction endonuclease , which recognises and cleaves a specific dna sequence, and a cognate dna methyltransferase  that can methylate the same target site, thereby preventing the cleavage of host dna by the rease  <cit> . on the basis of subunit composition, cofactor requirements, and the mode of recognition and cleavage, rm systems have been traditionally classified into three major types: i, ii and iii. type iv has been re-introduced to describe a heterogeneous group of enzymes cleaving only modified dna, with ill-defined sequence specificity  <cit> . type i and type iii reases are complex multisubunit molecular machines that utilize atp and perform either dna modification or cleavage  <cit> .\n",
            "\n",
            "type ii reases are simpler: they recognise short, 4– <dig> bp, nucleotide sequences and cleave phosphodiester bonds in dna within their recognition sites or at up to a distance of  <dig> bp from them  <cit> . in most cases, methylation and cleavage are divided between two separate enzymes. more than  <dig> various reases that recognise more than  <dig> different nucleotide sequences have been described as members of this large and divergent class of enzymes  <cit> . most of them have been isolated from mesophilic microorganisms and only a few from hyperthermophiles  <cit>  and thermophiles, in particular from the genus thermus  <cit> .\n",
            "\n",
            "in order to account for their structural and functional heterogeneity type ii reases have been divided into multiple subtypes, often overlapping with each other  <cit> . for example, a growing group of atypical type ii reases, represented by bifunctional enzymes with rease and mtase activities within a single polypeptide , is a subclass of subtype iis enzymes that recognise a specific dna sequence and cleave outside it at a defined distance, within any sequence. examples of this group include eco57i  <cit> , haeiv  <cit> , aloi  <cit> , bsemii  <cit>  and enzymes from the thermus sp. family: tspgwi , taqii , tspdti  and tth111ii   <cit> .\n",
            "\n",
            "in this paper we describe the cloning of the tspgwi gene based on degenerated primers pcr protocol, site-directed mutagenesis, expression and purification of recombinant and mutant proteins followed by comparative biochemical and genetic analysis. the constructed mutants are more suitable than native tspgwi in gene cloning methodology, as the cleavage and modification functions were separated into distinct, active proteins. tspgwi-like enzymes  <cit>  are difficult to fit unambiguously into the conventional classification, as their unique structural and functional features can be found in rease sub-types iia, iic, iis, iif, iih and type i.\n",
            "\n",
            "RESULTS\n",
            "sequencing, cloning and expression of the tspgwi gene\n",
            "neither the biochemical selection for methylation phenotype approach nor the \"white-blue\" screen for dna damage/modification to yield recombinant clones were successful , due to tspgwi low activity and lack of complete cleavage of the plasmid dna. we devised a cloning protocol for thermophilic rease without accompanying  mtase, which includes two stages:  a gene nucleotide sequence determination starting from internal amino acid sequences of rease proteolytic fragments followed by various pcr methods using combination of degenerated and non-degenerated primers. the primers are designed using probability of codon usage from other closely related bacteria and  direct in-frame insertion of amplified orf into tightly temperature-regulated expression vector, taking advantage of low temperature cultivation at non-permissive conditions, which both prevent rease expression and suppresses activity of any leaking thermophilic rease.\n",
            "\n",
            "the tspgwi gene nucleotide sequence determination was initiated from very limited amino acid sequence information about the n-termini of native tspgwi proteolytic internal fragments. we were unable to determine the amino acid sequence of the intact protein, thus homogeneous tspgwi preparation was subjected to limited proteolysis, yielding  <dig> major bands and more than  <dig> minor bands, as determined by sds/page . the immobilised tpck-trypsin was readily removed by centrifugation; hence, the resulting bands were derivatives of tspgwi protein only. two of the predominant bands of approximate sizes  <dig> kda and  <dig> kda were subjected to n-terminal sequencing . based on the obtained short 19- and 10- amino acid sequences gpiggggspeaqlvplitr and effterlaqe, sets of non-degenerated and degenerated primers, forming alternative pairs, were designed. the primers were designed arbitrarily as based on back-translated amino acid sequence using codon usage data from orfs of thermus sp. genes available in genbank. the high gc content of thermus genes was also considered. sets of primer pairs used were either with low degeneracy and non-degenerated, containing incorporated codons of the assumed highest probability to exist in thermus sp.\n",
            "\n",
            "one such pair yielded a pcr fragment containing an internal portion of the tspgwi gene. combination of pcr with degenerated primers using thermus sp. gw genomic dna as a template as well as casette-ligation-pcr using pbr <dig> dna as the known sequence donor for non-degenerated primers, yielded a sequence of  <dig> bp contig with the complete tspgwi gene. each contig strand was sequenced; in addition, regions containing discrepancies were sequenced several times under different conditions. the tspgwi gene orf coding for the rease-mtase bifunctional protein is  <dig> bp in length coding for the  <dig> amino acid polypeptide . the calculated molecular weight of the tspgwi is  <dig>  <dig> da, which perfectly matches data from sds/page  and size exclusion chromatography of the native protein  <cit> . the calculated isoelectric point of recombinant tspgwi is  <dig>  , very similar to calculated isoelectric point of recombinant tspdti of  <dig>  .\n",
            "\n",
            "no sequence similarity to any mtase or dna-binding protein was found in the orf flanking regions. the orf begins with the atg start codon and contains  <dig> putative upstream ribosome binding sites : - <dig> 5'-agg and - <dig> 5'-agga. the orf is extremely gc rich:  <dig> %. to ensure that no errors are incorpoated during tspgwi gene amplification, all the pcr reactions were performed with a proofreading blend of taq and pfu dna polymerases. sequencing was repeated three times for each strand, both directly for the independent pcr products and for the cloned the gene fragments.\n",
            "\n",
            "native tspgwi purification was reported by us previously  <cit> . since tspgwi activity was hardly noticeable in its natural host thermus gw, the development of an extensive isolation procedure was essential, which included: polyethyleneimine precipitation of nucleic acids, ammonium sulfate fractionations followed by five chromatographic steps and size-exclusion chromatography  <cit> . to the contrary, recombinant tspgwi and its mutants were purified to homogeneity using much simplified procedure with the tspgwi protein expressed in e. coli. the protocol included heat-denaturing e. coli proteins, polyethyleneimine precipitation of nucleic acids, ammonium sulfate fractionations, followed by two chromatographic steps.\n",
            "\n",
            "interestingly, in spite of the cloning in the presence of a strong pr promoter, the protein becomes detectable in induced cells only after  <dig> h of growth under non-permissive conditions, and keeps accumulating until the late stationary phase, even after  <dig> h cultivation at 42°c . this is probably due to the combination of the following factors:  gc rich orf sequence distant from e. coli optimum codon usage,  slow transcription of the extremely gc-rich tspgwi gene,  the presence of numerous putative hairpin structures within the gene and  the very large size of the protein to be translated. nevertheless, optimisation of expression culture growth/induction conditions yielded adequate amounts of tspgwi . it was also observed that the initially homogenous protein, after prolonged storage at -20°c, undergoes minor degradation .\n",
            "\n",
            "bioinformatic analyses of tspgwi: prediction of domains and functional motifs\n",
            "isolation and sequencing of the tspgwi gene enabled detailed analysis of its amino acid sequence. searches of rease sequences deposited in rebase revealed an overall similarity to a number of genuine and putative type iic enzymes, including the previously characterised nuclease taqii  <cit>  . two other members of the thermus family, i.e. tspdti  revealed the expected significant similarity of the central region of tspgwi  to the catalytic domains of known dna:m6a mtases. no significant similarity of the terminal regions  to any protein family was observed in simple sequence searches. nonetheless, multiple sequence alignment of tspgwi homologues revealed the presence of a candidate pd-xk motif , resembling the active site of many reases and other nucleases  <cit> . we carried out a protein fold-recognition  analysis  to confirm this preliminary prediction for the n-terminal region, as well as to identify potential homologues and predict the structure for the c-terminal region. since the fr method is designed to identify remote homology and predict structure for domain-size sequence fragments , the tspgwi sequence was split into a series of overlapping segments and submitted to the genesilico metaserver  <cit> .\n",
            "\n",
            "the fr analysis of the tspgwi sequence identified the structures of a type i dna:m6a mtase m. ecoki hsdm subunit , a putative hsdm subunit of an experimentally uncharacterised rm system bthvorf4518p , and type ii dna:m6a mtase m. taqi  as the best structural templates for the central region comprising residues 130– <dig> of tspgwi . interestingly, the fr alignments for this region revealed not only the s-adenosylmethionine -binding/catalytic mtase domain , but also a helical domain characteristic of the n-terminus of the hsdm subunit . for the n-terminal region of tspgwi  the fr methods also supported the prediction of the pd-xk domain, albeit with scores below the level of significance . the closest match on the sequence level  by the hhsearch method for profile-profile searches was the n-terminal pd-xk domain of the hsdr subunit from type i rm systems  <cit> . this prediction made for the n-terminal domain of tspgwi further supports the classification of the catalytic domain of taqii-related enzymes as a member of the pd-xk fold  <cit> .\n",
            "\n",
            "fr methods failed to report any significant matches to known structures for the c-terminal part of the tspgwi sequence . interestingly, the c-terminal region was predicted to be almost entirely disordered, with only a short segment comprising residues 960– <dig> predicted to form a stable helical secondary structure. this segment  shows sequence similarity to the c-terminus of taqii. we found no similarity of this region to target recognition domains  of any mtase family; nevertheless, we speculate that it might play an analogous role.\n",
            "\n",
            "altogether, the results of our fr analysis suggest that type ii enzyme tspgwi is a fusion protein comprising a tandem arrangement of type i-like domains: a pd-xk nuclease domain related to the corresponding domains in hsdr subunits, a hsdm-like module comprising a helical domain and a highly conserved adomet-binding/catalytic mtase domain, and a c-terminal extension, presumably responsible for dna sequence recognition . on the other hand, tspgwi lacks other domains characteristic of type i rm enzymes, including the atp-dependent translocase module of the hsdr subunit and c-terminal domains of either hsdm or hsdr, which are important for interactions between type i subunits  <cit> . this domain organization resembles the structure of the recently analysed mmei enzyme  <cit> , to which tspgwi exhibits only very limited sequence similarity, restricted primarily to the central mtase domain .\n",
            "\n",
            "enzymatic properties of tspgwi\n",
            "purified preparation of native tspgwi protein was used for the detailed study of biochemical properties, reaction conditions and various cofactors influencing either dna cleavage or the methylation activity of the enzyme. the apparent molecular mass of the wt recombinant protein under denaturing conditions was found to be  <dig> kda, adequately to the molecular mass of tspgwi isolated from thermus sp. gw  <cit> . comparative assay of activities of both enzymes revealed no difference . control mock-purification from e. coli did not show any of the described activities . wt recombinant protein was also subjected to analytical gel filtration in a buffer with a composition close to the physiological, containing  <dig> mm mgcl <dig> . the experiment revealed that the native endonuclease molecular mass is 110– <dig> kda, indicating that under physiological conditions the protein exists as a monomer  <cit> . the temperature activity range extends from 42°c to 89°c , with the maximum observed at 60–70°c. incubation at 37°c resulted in 5% activity only . tspgwi is inactivated by  <dig> min incubation at 89°c and ph greatly affects activity, which drops sharply below ph  <dig>  and above  <dig>  , with the maximum at  <dig> – <dig>  . the optimal ionic strength is below  <dig> mm kcl; at  <dig> mm only 50% of activity remains . the optimum restriction reaction conditions for tspgwi endonuclease in vitro are in  <dig> mm tris-hcl ph  <dig>  at 65°c,  <dig> mm mgcl <dig>   <dig> mm dtt. tspgwi maintains the absolute requirement for mg2+ for cleavage activity, with the optimum between  <dig> and  <dig> mm . mg2+ ions can be effectively substituted with mn2+, fe2+ and partially with co2+ ions . remarkably, ca2+ ions do not support tspgwi cleavage activity, while in contrast, they stimulate methylation activity of the enzyme . no difference was observed in response to divalent cations by both recombinant and native wt tspgwi . unlike most of the other bifunctional type iic reases/mtases  <cit>   <dig> μm adomet slightly inhibits the rease activity of tspgwi enzyme , while s-adenosylhomocysteine  has essentially no effect on lambda dna cleavage . the restriction activity of tspgwi is neither inhibited nor stimulated by spermidine or atp . as adomet is a co-substrate for the tspgwi mtase activity, it indirectly promotes dna modification rather than cleavage. the presence of refractory sites, observed in vitro without adomet in the reaction buffer, could alternatively be explained by partial dna methylation due to certain enzyme-bound adomet carry over during purification. however, since eight purification steps were used  <cit> , this possibility is rather remote. in addition, partial protection should be uniform among different recognition sites, instead of observed site preference.\n",
            "\n",
            "a recombinant tspgwi gene alone can be cloned into e. coli without an additional mtase  and stably maintained at 28°c. a much reduced restriction activity < 5% at low temperature and the presence of cellular adomet and ca2+ appears to favour methylation in vivo. taken together, the adomet and ca2+ effects as well as apparent preference for the opposite arrangement of recognition sites may provide an effective mechanism for controlling undesired restriction in vivo, thus removing the necessity for the existence of a separate tspgwi mtase. another conclusion from the above results is that dna methylation and dna cleavage events occur by different mechanisms, in parallel and essentially independently. as a result, some of the target sites become protected, even under conditions favouring cleavage. such concurrent methylation and restriction activities seem to resemble the behaviour of another bifunctional type iic enzyme aloi  <cit> .\n",
            "\n",
            "to further verify the marked independence of cleavage and methylation as well as to define the functional motifs within tspgwi , a series of amino acid replacements by site-directed mutagenesis was performed on tspgwi. we targeted three residues in the catalytic motif \"pd-xk\" of the nuclease domain , and two residues in the mtase domain of tspgwi: in the adomet-binding motif i \"dpavgtg\"  and in the main mtase catalytic motif iv: \"nppy\" . all the mutant proteins used were of high purity, isolated using simplified recombinant tspgwi protocol and their activity in vitro was analysed . some of the mutants lost their rease activity, while maintaining the wild type  level of methylation; others lost their mtase activity while retaining the ability to cut dna, although at reduced efficiency . the mtase activities of both the wt  tspgwi and mutant proteins were dependent on the presence of divalent cations . further experiments are in progress to evaluate complex effect of ca2+ and mg2+.\n",
            "\n",
            "dna substrate preference of tspgwi\n",
            "under the variety of conditions initially tested no complete tspgwi digestion was obtained; instead a stable partial cleavage pattern was observed. it is probably due to the preferential site cleavage in various dna substrates. among them puc <dig> was found to be the most efficiently cleaved supercoiled dna substrate  <cit> . however, even in the case of puc <dig> small fraction of molecules remained uncut, as determined by agarose gel electrophoresis of overloaded digestion samples and by transformation assay . the cleavage depends on the substrate used: it varies from the near complete digestion of two tspgwi sites in puc <dig> dna  to the trace cleavage of a subset of refractive sites in pbr <dig> and pacyc <dig> plasmids  <cit> . the refractory site phenomenon was preliminary evaluated here using two pcr substrates, containing either single  or double  tspgwi recognition sites. on a  <dig> bp pcr fragment containing a single recognition site for tspgwi and taqii, no detectable cleavage was observed in the case of tspgwi , while closely related taqii exhibits complete digestion . however, when the  <dig> bp fragment is cloned into puc <dig> plasmid , the tspgwi site becomes cleavable, as evident from the appearance of predicted  <dig> bp and  <dig> bp bands . the  <dig> bp pcr fragment contained two divergent tspgwi recognition sites, separated by a  <dig> bp spacer, which may allow two tspgwi molecules to bind simultaneously both recognition sequences located on the same face of dna, without steric interference, while being close enough to allow for protein-protein interaction. it turned out to be the most completely cleaved substrate among those we tested . further experiments are in progress to carefully evaluate the observed phenomenon, as the presence of two sites on the same dna molecule may be an intrinsic part of the tspgwi scission mechanism.\n",
            "\n",
            "the requirement for the presence of two recognition sites of opposite orientation for dna cleavage is a characteristic feature of type iii reases  <cit> . thus, the tspgwi-like enzymes  show a mixture of features characteristic of type i, ii, or iii rm systems. type iii enzymes methylate only one strand, regardless of the number and orientation of cognate sites, while still providing adequate protection against cleavage. within type ii , bspmi is the one rease described to date that strongly prefers a tandem, head-to-tail orientation of double sites  <cit> . another subtype iis endonuclease foki, has a preference towards a pair of sites with a tandem head-to-tail orientation  <cit> . the divergent orientations of the recognition sites may ensure cleavage of both sequences, as was proposed for type iis reases bsgi, bpmi and acc36i  <cit> , which are, however, rather remotely related to tspgwi . nevertheless, these enzymes retain the ability to cut single sites, although with a reduced efficiency compared to the tandem sites. tspgwi is unable to cut a single site, unlike bsgi, bpmi, acc36i and the closely related taqii rease. it is intriguing that taqii rease, despite the very high amino acid sequence similarity to tspgwi  and in spite of numerous biochemical similarities  <cit>  cleaves single sites efficiently. the ability to cleave a single recognition sequence may be correlated with the existence of rease activity stimulation by adomet. although this cofactor has no effect on tspgwi rease activity, it does stimulate taqii rease approximately 2– <dig> fold . as we have pointed out previously  <cit> , the new thermus sp. family including: tspgwi, taqii, tspdti and tth111ii, is internally diversified. even though these enzymes exhibit striking similarities in protein size, quaternary structure, recognition sequences, cleavage distances and related origin from various thermus strains, they display marked differences in their response to adomet and to the arrangement requirements of different recognition sites. interestingly, this group of enzymes seems to be also heterogenous at the level of amino acid sequence similarity, as tspgwi and taqii seem to be closely related to each other, but only remotely to the two other enzymes tspdti and tth111ii. this latter pair of enzymes appear to be closely related to each other, but exhibit a different pattern of similarities compared to the tspgwi-taqii pair. however, all four enzymes share significant biochemical similarities, including related recognition and cleavage sequences, very large molecular weights of around  <dig> kda, monomeric protein organization  <cit>  and are negatively charged with calculated isoelectric points of  <dig>  ,  <dig>  ,  <dig>   and  <dig>   .\n",
            "\n",
            "although tspgwi is not particularly closely related to any known subtype iib rease  <cit> , the similarity of the domain structure and the mode of action of these enzymes  suggests that they have evolved from a common ancestor. thus, type iib enzymes forming tight dimers and taqii-like enzymes that hardly dimerise if at all, may represent two extremes of an evolutionary landscape, where different enzymes have specialised towards either separation or pairing of recognition sites by developing subunit-subunit interactions of different strength. in this classification, tspgwi would be located \"between\" type iib enzymes and taqii.\n",
            "\n",
            "endonucleolytic activity of tspgwi wt and the mutant proteins\n",
            "site-directed mutagenesis was used to test whether the amino acids predicted by bioinformatics were essential for the mtase and rease catalytic activity of the tspgwi enzyme. to this end, variants of the tspgwi protein with substitutions in the putative pd-xk domain and in the mtase domain were constructed by saturation site-directed mutagenesis. the mutant proteins were purified as described for recombinant tspgwi protein and subjected to in vitro biochemical experiments. to calibrate the protein concentrations, each variant was assayed, then diluted with the storage buffer to achieve equal protein concentration in every assay. the purity and concentration of the diluted proteins were further assessed by poliacrylamide gel electrophoresis . the cleavage reaction was performed at 65°c for one hour and overnight. mutants in the pd-xk domain d58a, d58r, e72a, e72l, e72n, e72r, e72s, k74a, k74g, k74p, k74r did not show any dna cleavage activity under optimal buffer conditions at a concentration of  <dig> μg/ml, the equivalent of  <dig> u wt tspgwi per reaction. higher enzyme concentrations could not be tested, as the specific activity of wt tspgwi rease is relatively low , and increasing the amount of protein per reaction caused inhibition because of its non-specific binding to the substrate dna. only one mutant variant e72d retained its rease activity – its cleavage rate was even slightly higher than that of the wt protein. thus, conservative substitution of aspartic acid for glutamic acid in the  site keeps the pd-xk motif of tspgwi fully functional. it also confirms the key role of acidic amino acid in position  <dig> in catalysis. since preferential cleavage of substrate dna by native tspgwi was observed at concentrations as low as  <dig>  μg/ml , the decrease in rease activity of tspgwi mutants was estimated to be more than 200-fold, thus confirming the predicted role of the amino acids d <dig>  e <dig> and k <dig> in catalysis. surprisingly, the mutants in the mtase domain – v356m  and n473a  – also displayed a significantly reduced cleavage activity in vitro in comparison to the wt tspgwi protein. the double mutant v356m/n473a exhibited no detectable rease activity in vitro. taqii, the closest relative of tspgwi, exhibits the \"dpamgtg\" variant of motif i. this observation, together with the fact that taqii rease activity is strongly stimulated by adomet , points to the complex role of adomet in both enzymatic activities and suggests that some interaction occurs between the nuclease and mtase domains in the course of the cleavage reaction. the substitution of methionine for valine in the adomet-binding motif of tspgwi  could disturb adomet-affected communication between the nuclease domain and the mtase domain. the nature of the communication between the domains remains to be defined by further analyses.\n",
            "\n",
            "mtase activity of tspgwi wt and the mutant proteins\n",
            "mutant proteins were further analysed for their methylation activity. in vitro methylation reactions were performed using wt tspgwi enzyme and mutant proteins  in the dna protection assay as described in materials and methods. two types of substrate dna were used: pbr <dig> plasmid dna and the  <dig> bp pcr fragment. the results presented in table  <dig> and table  <dig> demonstrate that the mutant n473a  is unable to modify the tspgwi target sequence. interestingly, the n473a substitution not only completely eliminates the methylation activity, but also decreases the cleavage activity of tspgwi; even so, the resulting protein retains sufficient activity to be considered a functional rease. thus, both the rease and mtase activities of tspgwi were uncoupled, and rease-only or mtase-only tspgwi variants were obtained. this is also of high practical methodological value in gene cloning technology, allowing to use tspgwi mutants as traditional class-ii enzymes for dna cleavage or methylation. mutants with d58a  or v356m  substitutions exhibit mtase activity at the same level as the wt enzyme, despite the total or partial loss of their ability to cleave the substrate dna. we observed that d58a slightly increased the modification activity  in comparison to the wt protein . this could have been due to the absence of a competing rease activity in this variant. interestingly, mg2+ at  <dig> mm concentration tested has a strong stimulatory effect on the mtase activity of the d58a mutant . in the case of wt tspgwi, mg2+ is utilised for cleavage, which hinders its effect on the mtase activity. thus, the d58a mutation demonstrates that the competing mtase and rease activities both share the same mg2+ cofactor. however, it cannot be ruled out that the presence of divalent cations stabilizes not perfectly folded mutant protein tspgwi, thus contributing non-specifically to retaining their enzymatic activities. taken together with the observed inhibition of the tspgwi rease activity by adomet, our results show that the dna methylation and cleavage activities of tspgwi are closely intertwined, but autonomous enough to have their functions separated.\n",
            "\n",
            "CONCLUSIONS\n",
            " the tspgwi gene coding for  <dig> kda tspgwi has been sequenced and cloned.\n",
            "\n",
            " active bifunctional rease-mtase protein has been expressed in e. coli and purified.\n",
            "\n",
            " bioinformatics studies predicted rease and mtase binding/catalytic motifs: pd-xk, dpavgtg, nppy and showed modular structure of tspgwi.\n",
            "\n",
            " series of mutants in the predicted motifs were constructed, yielding rease-deficient and mtase-deficient enzymes. uncoupled rease and mtase activities exhibited high autonomy.\n",
            "\n",
            " mutants more suitable than native tspgwi for gene cloning methodology were constructed. the cleavage and modification functions were separated into distinct proteins, while maintaining enzymatic activity, thus allowing practical use of tspgwi in vitro as a regular class ii enzyme.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this paper, we describe the cloning and purification of the tspgwi gene without accompanying  mtase.\n",
            "in this paper, we describe the cloning and purification of  <dig> kda tspgwi, a bifunctional type ii rease that recognises and cleaves a specific dna sequence, and a cognate dna methyltransferase  that can methylate the same target site, thereby preventing the cleavage of host dna by the rease.\n",
            "in this paper, we describe the cloning and purification of  <dig> kda tspgwi, a bifunctional type ii rease that recognises and cleaves a specific dna sequence and cleaves outside it at a defined distance, without complete cleavage of the plasmid dna.\n",
            "in this paper, we describe the cloning and purification of  <dig> kda tspgwi, a bifunctional type ii rease that recognises and cleaves a specific dna sequence and cleaves outside it at a defined distance, without complete cleavage of the plasmid dna sequence.\n",
            "\n",
            "74\n",
            "[{'summary_text': 'our data suggest that aging is accompanied by a significant loss of tissue-specific gene expression profiles, possibly as a result of a systemic response.\\nin this study, we analyzed the aging mouse liver transcriptome and identified  <dig> novel ncrnas  that were significantly differentially expressed with age in mouse liver.\\nour data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a consequence of a systemic response.\\nour data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a result of a systemic response.\\nour data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a result of a systemic response.\\nour data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a result of a systemic response.\\nour data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a result of a systemic response.\\nour data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a consequence of a systemic response.\\nour data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3457 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "mammalian aging is a complex biological process that still remains poorly understood. to increase our understanding of the different interacting processes that underlie age-related organ and tissue degeneration, a systematic study of alterations in gene expression is a logical starting point. global changes in gene expression are a “hallmark” of aging in multiple species  <cit>  and have been successfully used to demonstrate specific effects of interventions that slow aging, such as caloric restriction  <cit> . in the past, microarray analyses have been used extensively to study age-related differences across multiple tissues, but ultimately these analyses proved insufficient for gaining a complete understanding of the aging transcriptome for various reasons. first, microarray analysis relies on fully annotated genes and since the release of the encode project, we now know that novel un-annotated transcripts are constantly being discovered  <cit> . second, microarray analyses generally suffer from excessive noise, which could explain the very small overlap between different published results of gene expression changes with age in mouse liver  <cit> . for example, while one report mentioned minimal age-related changes in gene expression during mouse liver aging  <cit> , others observed a considerable number — albeit highly variable from study to study — of such changes . finally, microarray analysis is generally not capable of detecting altered isoform levels, due to differential splicing, or changes in non-mrna species, such as non-coding rnas. this is especially important for non-coding rnas since by now the role of this category of transcripts is generally understood to involve fine-tuning gene regulatory patterns, such as cellular metabolism  <cit> . increased deregulation at this level could potentially explain many of the aging phenotypes observed, which involve subtle rather than dramatic changes. recent advances in rna sequencing now offer a suitable alternative to studying variation in gene expression that addresses all of these issues.\n",
            "\n",
            "while changes in gene expression profiles have been used as biomarkers for aging and sometimes even as predictors of biological aging rate  <cit> , the ultimate aim in studying the aging transcriptome is to elucidate the pathways that define age-related degenerative processes and the responses they evoke, in a tissue-specific manner. for this purpose it is essential to collect mrna expression data in the context of their non-mrna, gene regulatory rnas. indeed, as mentioned, apart from the canonical protein-coding mrnas, which make up about 3 % of the genome, the critical role of a multitude of ncrnas are now well recognized. ncrna is a broad umbrella term that includes pseudogenes, or those genes that have lost their protein-coding potential but are sometimes still capable of being transcribed  <cit> , long non-coding rnas , typically > <dig> nucleotides long, which have been shown to perform a wide range of functions from transcriptional de-repression to silencing  <cit> , and small ncrnas , which are mainly comprise the regulatory micrornas , small nuclear rnas , small nucleolar rnas , and transfer rnas . although species in this last category are small, they may have a large impact on cellular function, regulating such processes as gene silencing, splicing, and translation  <cit> . the role of ncrnas in aging remains unclear, which essentially constrains attempts to generate comprehensive functional networks of tissue-specific alterations in the rna landscape.\n",
            "\n",
            "here, we utilized an rna-sequencing approach capable of capturing the whole transcriptome of mouse liver, a similar approach that has been performed in aging brain for other mammals  <cit> . specifically, we analyzed young adult  and old  mouse livers and identified significantly altered transcripts, not only canonical coding transcripts, but also novel ncrna transcripts and isoform variants. several of the altered ncrnas, many of them from the dlk-dio <dig> locus — an imprinted domain implicated in development and pluripotency and here identified as a ncrna gene regulatory hotpot in mouse liver aging — are putative regulators of the age-related loss of mitotic potential. age-related differentially expressed transcripts, including isoforms and ncrnas, were first compiled into putative functional pathways and then used for constructing age-related interaction networks. these networks of age-related change revealed three dominant, emerging phenotypes: inflammation, proliferative homeostasis, and lipid metabolic changes.\n",
            "\n",
            "RESULTS\n",
            "sequencing metrics and transcriptome genomic coverage\n",
            "to directly analyze the global transcriptome of aging mouse liver, we isolated total rna from three young adult  and three old  male mouse livers and performed directional whole transcriptome sequencing  <cit> . for each library we generated an average of ~ <dig> million paired-end reads of which 82–90 % could be mapped to the ncbi build 37/mm <dig> reference genome . aligned reads were characterized using a unique annotation reference database that combined five different known databases for all transcripts: ensembl, genbank/ncbi, refseq, vega/havana, and mgi. by taking this approach we were able to take advantage of all known database predictions and compile these for novel transcript discovery in aging  <cit> . thus, we were able to analyze not only manually curated gene sets, as is the case with vega/havana annotations, but also parts of the mouse genome that have only had computationally predicted annotation, such as those found in ensembl, for a total of  <dig>  transcript annotations .\n",
            "\n",
            "to begin, we determined the percentage of the genome actively transcribed. on average  <dig>  % of the annotated transcripts from our unique annotation database of  <dig>  transcripts were found to be expressed in liver of all animals combined, both young and old, as defined by > <dig> count per transcript to account for low-abundance transcripts. of these transcripts ~70 % were protein-coding genes , with transcripts from pseudogenes comprising ~16 % and the remaining ~ <dig>  % representing ncrnas, such as lncrnas, snornas, snrnas, and mirnas. about 3 % of these ncrnas still remain unclassified . these results corroborate findings by others for the human genome  <cit> .fig.  <dig> global expression analysis in aging mouse liver. a percent of globally expressed transcript subspecies expressed across both young and old livers from our unique annotation set of  <dig>  annotated transcripts. b table represents the number of genomic bases covered by at least one unique read in either all old livers, all young livers, or both old and young livers. total mouse genome size reference used at time of analysis using grcm <dig> build is  <dig> , <dig>  bases for a haploid genome. c differential transcript expression plotted log2foldchange versus the mean gene dispersion value. red points represent all  <dig> significant differentially expressed transcripts with fdr < <dig> . old , young . d heatmap representation of the top  <dig> differentially expressed transcripts by p-value, generated using unsupervised hierarchical clustering\n",
            "\n",
            "\n",
            "\n",
            "in view of some long-standing hypotheses that the repression of genes normally not expressed in a differentiated tissue is relaxed at old age  <cit> , we compared transcriptome genomic coverage between young and old livers. of note, to fully capture the complete transcriptome from a single data set could require well over a billion rna-seq reads rather than the roughly  <dig> million for all six of our animals combined. nevertheless, we calculated the number of unique bases covered by at least one read for each animal, either young or old, and divided this by the total number of bases in the haploid mouse genome, ~ <dig>  billion bp  <cit> . the results indicate no significant difference between young and old mouse liver . hence, we did not observe any obvious major relaxation of gene repression at old age, albeit some minor effect cannot be ruled out. indeed, when we combined the three individual liver samples for each age group, genome coverage doubled, i.e., from ~ <dig>  and ~ <dig>  % to  <dig>  and  <dig>  % for the young and old group, respectively, indicating that we are far from capturing all transcribed bases in one animal. when combining data from all animals, both young and old, coverage did not double but increased only to  <dig>  %, suggesting the beginning of some saturation . nonetheless, it is still striking that up to 17 % of the mouse liver genome is capable of being transcribed.\n",
            "\n",
            "aging is characterized by increased individual variation in the liver transcriptome\n",
            "increased variation of gene expression has been considered one mechanism by which organisms undergo age-related cellular degeneration  <cit> . while we only have rna-seq data on three young and three old animals, for each animal well over  <dig> million mappable reads were obtained. with high transcriptome coverage per individual animal we should, in theory, be able to accurately detect variation between animals.\n",
            "\n",
            "to compare animal-to-animal variation at young and old age we generated a multiple dimensional scaling plot  of the variance of the top  <dig>  expressed transcripts . mds analysis shows that young animals cluster together, while old individuals have a clear high degree of variance between each biological replicate. furthermore, we analyzed each sample using unsupervised euclidean matrix plots to observe the variance between the individual young and old livers . we find that young and old still separate to each respective class based on variance alone. moreover, we find that young compared to young liver has a distance value closer to one, while old compared to old liver have a value further from one, meaning aged liver transcriptomes are more individually variable than their younger counterpart.fig.  <dig> aging profiles characterized by stochastic variation. a variance of the top  <dig>  transcripts plotted using multi-dimensional scaling analysis; y young, o old, n biological replicate. b heatmap showing euclidean distance matrix between young and old replicate samples calculated from the variance in count values where values closer to  <dig> are less variable. c, d the calculated mean squared coefficient of variation  for genes  and isoforms  plotted as a function of normalized count expression for young and old liver samples. e the percentage of normalized individual count data for each transcript type. black bars represent the mean ± standard deviation\n",
            "\n",
            "\n",
            "\n",
            "when analyzing the variation between each type of transcript, mainly genes and isoforms , we find that variation is highly dependent on transcript expression level; that is, lowly expressed genes and isoforms tend to have higher coefficients of variation than those highly expressed, as previously suggested  <cit> . however, we still find that aged livers have a higher coefficient of variation, irrespective of expression level. additionally, we found that the percentage for each category of transcript of total normalized counts is generally more variable for old livers as compared to young . these findings indicate that the liver transcriptome from aged mice is consistently more variable than that from young mice and that this increased inter-individual variation is attributable, in part, to the stochastic variation observed at the level of specific transcript types.\n",
            "\n",
            "differential expression of rna species between young and old mouse liver\n",
            "before analyzing individual transcripts we first assessed our data for global changes in expression levels. since the magnitude of age-related expression differences cannot be predicted and could be small  <cit> , we refrained from setting an arbitrary cut-off for fold change, thus allowing us to detect all possible statistically significant changes in expression. in total we found  <dig> transcripts differentially expressed with age in mouse liver. of these,  <dig> transcripts were significantly  upregulated while  <dig> transcripts were significantly  downregulated in aging . of the total transcripts, we found a total of  <dig> significant differentially expressed protein-coding  genes . this comprises  <dig> upregulated genes, and  <dig> downregulated genes. this finding corroborates previous microarray studies showing that, in general, more genes are upregulated than downregulated in aging liver  <cit> . as could be predicted, when the top  <dig> protein-coding transcripts were subjected to unsupervised hierarchical clustering, the old and young mice were separated to each respective cohort .\n",
            "\n",
            "in order to validate results obtained from our rna-seq dataset we performed qpcr validation by probe-based array. we specifically chose  <dig> transcripts varying in count number, fold change differences, and as controls genes previously implicated in aging or senescence. for transcripts upregulated in aging liver we analyzed ly6a, mmp <dig>  cxcl <dig>  gbp <dig>  il <dig>  rac <dig>  fgfr <dig>  ctss, and terc , while downregulated transcripts validated were mt <dig>  e2f <dig>  hspa1b, and neat <dig> . we directly compared these qpcr results with the rna-seq results by plotting the linear regression of the log2foldchange values for each transcript . results showed a pearson’s correlation coefficient of  <dig> , confirming our rna-seq results are accurate and reliable.\n",
            "\n",
            "recently, studies have aimed to discriminate between expressed mrna isoforms, as mrna processing could be altered as a function of age  <cit> . thus, we tested if we could detect changes in isoform expression in mouse liver by taking our read alignments and utilizing the cufflinks/cuffdiff package  <cit>  with the ensembl annotation database. we found a total of  <dig> differentially expressed isoforms  with age. of these,  <dig> isoforms were significantly  upregulated, while  <dig> were significantly  downregulated in aged liver. interestingly, we found isoform switching enriched on chromosome four, where  <dig> out of the  <dig> significant isoforms were found. furthermore, one specific locus, the large mup  locus that spans  <dig>  mb at chr.4qb <dig>  appeared responsible for  <dig> of these  <dig> isoforms. these results indicate that mrna isoform switching is not uncommon during aging of the liver with the mup locus as a major hotspot.\n",
            "\n",
            "several studies have shown potential functional implications for ncrna changes in both aging and cellular senescence , most cases involving mirnas. utilizing our unique annotation reference database we sought to identify all ncrnas  that were significantly differentially expressed with age in mouse liver. in total we found  <dig> ncrnas significantly differentially expressed . of these, those specifically upregulated with age were  <dig> pseudogenes,  <dig> lncrnas, one mirna, nine snrnas, one snorna, one telomerase rna component  and  <dig> unclassified ncrnas. those downregulated were  <dig> pseudogenes, seven lncrnas, one mirna, three snrnas, and nine unclassified ncrnas. in keeping with our previous results for global expression changes, more ncrnas are significantly upregulated, in general, than downregulated with age. of note, we only find two significant differentially expressed mirnas  due to the inherent size constraint encountered when sequencing libraries for 100 bp read length on the illumina platform. to validate these novel ncrna findings we performed qpcr of a select  <dig> ncrna transcripts ranging in type, fold change, and count number. for those up-regulated with age we chose to analyze mup-ps <dig>  rian, mup-ps <dig>  2410018l13rik, a230056p14rik, and dleu <dig> while down-regulated ncrnas validated were gm <dig>  gm <dig>  gm <dig>  gm <dig>  gm <dig>  mir17hg, and gm <dig> in both young  and old  mouse livers. again, we compared our rna-seq results to our qpcr results to find that ncrna expression correlated with a pearson’s correlation coefficient greater than  <dig>  , confirming that the novel ncrnas found in our aging liver are truly differentially expressed.\n",
            "\n",
            "functional annotation\n",
            "in an attempt to translate the transcriptional landscape in aged mouse liver into functional consequences we first focused on the mrna component of observed alterations and using both david, the database for annotation, visualization, and integrated discovery  <cit>  and gorilla  <cit> , to perform separate functional annotation analysis on significantly upregulated and downregulated transcripts. in total,  <dig> gene ontology  annotations were significantly  upregulated  while only  <dig> were downregulated . we then used revigo, which uses utilizes semantic similarity to parse go annotations in order to remove redundant terms and to visualize clusters of enriched terms  <cit> . go annotations that were significantly upregulated in aging liver were highly enriched for immune system processes, immune response, response to stress, cell activation, regulation of cytokine production, and cell death . go annotations that were significantly downregulated with aging, were enriched for lipid metabolism, oxidation-reduction process, monocarboxylic acid metabolism, fatty acid metabolism, steroid biosynthesis, and rna modification . together these findings show that aging liver is highly enriched for an increase in immune response and inflammation as well as a decrease in metabolic annotations, as previously reported  <cit> , while also revealing new clusters such as regulation of cytokine production and rna modification.fig.  <dig> go term enrichment for genes differentially expressed in aging mouse liver. enrichment analysis visualized as an mds plot for go terms for biological processes that are  upregulated and  downregulated in aged mouse liver. plots are generated based on a matrix of semantic similarities in space . clusters of circles closer together represent terms that are more closely related. circle color and size indicates log10\n",
            "p-value\n",
            "\n",
            "\n",
            "\n",
            "we also performed gene ontology analysis on significant differentially expressed isoforms , again either up or downregulated. here we found that the corresponding genes for isoforms that were upregulated resulted in eight processes highly enriched for immune system function, most notably immune response , while genes for downregulated isoform analysis gave one significant biological process, oxidation-reduction . hence, the isoform pathway analysis corroborated our findings with the gene ontology analysis.table  <dig> go analysis from david for genes from differentially expressed aging isoforms\n",
            "\n",
            "\n",
            "\n",
            "we also found many differentially expressed ncrnas that may play roles in the molecular mechanisms of aging. the first is that of neat <dig>  a lncrna downregulated in our dataset, which has been associated with nuclear paraspeckle formation  <cit> , as well as nuclear retention of mrnas  <cit> . loss of neat <dig> may well be one explanation for the increase in transcriptional noise that we have previously characterized and possibly that of decreased nuclear organizational structure. we also observed a ~ <dig>  fold increase in pvt <dig>  a ncrna inducible by p <dig>  <cit> . pvt <dig> is part of a syntenic locus which is a known hotspot for multiple cancers as well as a host for seven annotated mirnas  <cit>  that have been shown to decrease expression of myc. strikingly, myc is down-regulated ~ <dig>  fold in our dataset , albeit it barely missed our cut-off of fdr < <dig> .\n",
            "\n",
            "finally, we analyzed our ncrna dataset for possible spatial relationships within the genome. indeed, we observed three ncrnas, all upregulated in aged liver, on mouse chr.12qf <dig> , which are adjacent to each other. these ncrnas, meg <dig>  rian , and mirg are a unique set of lncrnas that have been identified as part of the imprinted locus dlk1-dio <dig> and are expressed from the maternal allele  <cit> . together these lncrnas cooperate to actively regulate cell proliferation through gene expression either by directly binding and recruiting the polycomb repressive complex two   <cit> , as is the case with meg <dig> and rian, or by predicted microrna-mediated regulation of cell cycle factors such as myc and p <dig>  <cit> , as is the case with mirg. we also found two significantly upregulated novel lncrnas, gm <dig> and gm <dig>  which flank the cdkn2a  locus on mouse chr.4q. although much less is known about these particular transcripts, gm <dig> is located directly downstream  and gm <dig> upstream  of cdkn2a. this makes it tempting to speculate that these lncrnas may exert some regulatory control of this locus, possibly contributing to senescent phenotypes. together, these findings point towards a host of age-related ncrnas as regulators of aging pathways and networks.\n",
            "\n",
            "interaction network analysis\n",
            "the increased accuracy and breadth of our rna-seq data sets allowed us to generate networks of gene functional change in aging liver, above and beyond what was observed using david or gorilla. using ingenuity pathway analysis  we generated, from the differentially expressed protein-coding genes and ncrnas, interaction networks of functional change. this resulted in multiple overlapping pro-aging networks from which we could distinguish three major molecular phenotypes: inflammation, proliferative homeostasis and lipid metabolism . fig.  <dig> age-related interaction network for cancer and inflammatory response. each gene is colored according to directionality of expression; red=upregulated, green=downregulated. numbers below each gene represent the fold change  and fdr value  of that particular gene. orange line=predicted activation, blue line=predicted inhibition, yellow line=prediction inconsistent, grey line=no predicted effect\n",
            "\n",
            "fig.  <dig> age-related interaction network for proliferation of cells and cell death. each gene is colored according to directionality of expression; red=upregulated, green=downregulated. numbers below eachgene represent the fold change  and fdr value  of that particular gene. orange line=predicted activation, blue line=predicted inhibition, yellow line=prediction inconsistent, grey line=no predicted effect\n",
            "\n",
            "fig.  <dig> age-related interaction network for synthesis of lipid and oxidation of lipid. each gene is colored according to directionality of expression; red=upregulated, green=downregulated. numbers below each gene represent the fold change  and fdr value  of that particular gene. orange line=predicted activation, blue line=predicted inhibition, yellow line=prediction inconsistent, grey line=no predicted effect\n",
            "\n",
            "\n",
            "\n",
            "of the multiple networks of liver aging, the most prominent interaction network to emerge was inflammation and cancer . this was not surprising since this could already be derived from the gene ontology analysis and has also been previously reported  <cit> . this is in keeping with observations that inflammatory cytokines secreted from senescent cells are capable of promoting the hyperplastic growth of surrounding cells  <cit> . we, and others, previously observed an increase of senescent cells in aging liver  <cit> , which could promote the age-related increase of cancer through these same secreted cytokines. conversely, it has also been shown that infiltrating immune cells can clear pre-oncogenic senescent hepatocytes  <cit> , thereby the same secretory phenotype emitted by cells to promote cancer can also act a barrier to tumorigenesis in the liver.\n",
            "\n",
            "another prominent age-related network that emerged from the ipa analysis was that of proliferation of cells and cell death. this points to an age-related loss of proliferative homeostasis, which is in keeping with reports suggesting that aging compromises the rate of liver regeneration rather than its regenerative capacity  <cit> . these two connected but opposing networks  link together genes shown to be highly important in aging such as cdkn1a, il6r, mmp <dig>  top2a, pdgfc, lama <dig>  and e2f <dig> . considering the liver is a still a mitotically active tissue, this interplay emphasizes the complex interactions needed to maintain its normal regenerative function. an aging phenotype to arise from the breakdown of cellular proliferation is the increase in cellular senescence. however, ipa does not contain a formal pathway for cellular senescence per se, which is essentially a stress response and varies based on the senescent trigger, i.e. genotoxic stress induced, oncogene induced, or replication induced senescence. of note, many of the genes involved in the cancer network were also implicated in the proliferation of cells network, as could be expected since cancer is the uncontrolled proliferations of cells.\n",
            "\n",
            "the third major interacting network of age-related change, which is the interaction between the synthesis of lipids and the oxidation of lipids , is enriched for genes that are downregulated in aging. clearly, the most abundant enzyme family involved in this network is the cytochrome p <dig> family . while there certainly is evidence for reduced hepatic drug clearance in humans and rodents, the possibility that this is caused by reduced p <dig> activity is controversial  <cit> . one phenotypic outcome arising from the failure of these pathways to cooperate could be the increase in one of the most prevalent pathological outcomes in liver, lipofuscin accumulation  <cit> . this phenotype could reflect the inability to eliminate cellular waste products. a decline in liver metabolic activity with the subsequent increase in lipofuscin with age could be due in part to the decrease in chaperone mediated autophagy  <cit> .\n",
            "\n",
            "since protein-protein interactions are very well studied as compared to those of ncrnas, by analyzing both protein-coding and ncrnas together, the most robust networks resulting from this analysis would only contain protein-protein interactions, essentially losing the weaker, less-studied ncrna interactions. thus, we chose to separately analyze only ncrnas with age-altered expression to create age-related ncrna interaction networks. subsequently, we looked at the proteins and genes listed in our ncrna interaction network and filtered the list for those only differentially expressed within our dataset.\n",
            "\n",
            "only four interacting networks were obtained in this way, three of which only had one ncrna interacting with only one other protein or gene, similar to previous ncrna network analysis  <cit> . only one network had multiple known interactions within the ipa database; thus, we focused on this network that included five ncrnas with known interactions to other proteins and/or genes. by combining the ncrnas and the known interacting protein-coding genes altered with age and again analyzing with ipa we were able to create an age-related ncrna-protein coding interaction network. this one robust network association  is involved in the ifng-mediated pro-inflammatory response. interestingly, this network links our novel ncrnas to certain genes such as p <dig>  il <dig>  ctss, and nfκb, all of which have been previously associated with mammalian aging or senescence  <cit> , and their age-related alterations may contribute to the most prevalent aging phenotype, inflammation.fig.  <dig> ncrnas associate in infg-mediated pro-inflammatory network. ncrna network analysis reveals functional role for non-canonical transcript involvement in infg-mediated pro-inflammatory response. colored nodes represent directionality of transcript expression within dataset; red?=?upregulated, green?=?downregulated, white?=?no significant change. numbers below each gene represent the log2foldchange  and fdr value  of that particular gene\n",
            "\n",
            "\n",
            "\n",
            "discussion\n",
            "functional and pathological changes in liver with age have been studied extensively, but the extent of physiological decline of this organ over time remains controversial  <cit> . hence, it is critically important to comprehensively assess changes at the molecular level, which could at least in part resolve this issue. next-generation sequencing technology has opened up the possibility to study the molecular basis of aging in unprecedented detail. in studying gene expression, generally thought to be the most immediate reflection of critical, age-related alterations in any tissue-specific phenotype, the advent of rna sequencing now allows a full depiction of the aging transcriptome. as we show here for mouse liver, the increase in experimental detail as compared to previously applied microarrays is dramatic. while previous, microarray-based studies revealed highly variable numbers of differentially altered genes during aging  <cit> , and sometimes none at all  <cit> , our rna-seq data sets showed a total of  <dig> protein-coding transcripts as differentially expressed between livers of young and old mice, plus an additional  <dig> differentially expressed isoforms and  <dig> differentially expressed ncrnas. the latter is undoubtedly an underestimate due to the limited number of such rnas annotated thus far. albeit limited to two extreme age levels and one mouse strain, the current scope of our study highlights the complexity of transcriptome changes in aging mammals. indeed, such complexity is likely higher still when possible strain differences and variation across multiple age levels will be taken into account. however, such unprecedented detail in visualizing the aging transcriptome allows us not only to uncover the major gene networks of phenotypic change with age, but also to test some very basic hypotheses about loss of transcriptional fidelity as a possible ultimate cause of aging.\n",
            "\n",
            "one obvious characteristic of the aging transcriptome in liver is the increased individual variation. evidence for the importance of stochastic components in gene expression changes with age has been provided previously using different approaches  <cit> , but ours is the first direct evidence that transcriptomes of individual animals can significantly diverge as a consequence of the aging process, and not just between individual genes. indeed, as we show, aging is accompanied by an increase in stochastic variation, or transcriptional noise, which is apparent in all rna classes. our finding that older animals, in general, have higher coefficients of variation, irrespective of transcript length, confirms that age-related variation is genome-wide. however, we do observe that lowly expressed transcripts, irrespective of age, have much higher coefficients of variation as compared to highly abundant transcripts. this could be due to the inherent labile nature of smaller rna species, as longer highly expressed transcripts tend to be more stable and less susceptible to metabolism, either due to higher order structure or because more proteins are able to bind and protect the transcript from endogenous rnases  <cit> . in this respect, it is striking that we find rna modification as an enriched pathway that is downregulated within our dataset, a possible reason that aged transcriptomes tend to have higher variation. moreover, the downregulation of neat <dig> may contribute to variation between mrnas containing inverted alu repeats in the 3′ utr  <cit> . thus, the possibility cannot be ruled out that specific pathways ultimately control stochastic variation in the aging transcriptome, possibly as part of a systemic response.\n",
            "\n",
            "one longstanding hypothesis in gene expression is that as an organism ages, the genome loses its ability to effectively regulate genes, leading to an overall relaxation or de-repression of gene expression  <cit> . our data do not confirm the widespread occurrence of such a loss of gene transcriptional control, as we did not observe a drastic increase in genomic coverage of the transcriptome from young to old animals. however, to definitively rule out this hypothesis, extremely deep sequencing, i.e., greater than  <dig> billion reads, may be necessary  <cit> . alternatively, reports on the de-repression of gene expression have been limited to a select few tissues and therefore the observation could be a function of tissue or cell type specificity.\n",
            "\n",
            "a major challenge in attempts to fully characterize the transcriptional landscape at old age is linking altered mrna expression levels to altered ncrnas, recently highlighted as a key regulatory component of the transcriptome and in aging  <cit> . our results show that roughly 13 % of the transcripts we found altered with age are those of ncrnas. although many of these ncrnas have been previously reported and studied, such as the lncrnas meg <dig>  rian, and neat <dig>  none of these to our knowledge have been implicated in aging. additionally, it is noted that the dlk-dio <dig> locus in mouse corresponds to the largest hotspot of mirnas within its genome. interestingly, a previous report has shown that eight mirnas originating within this cluster are downregulated in aged skeletal muscle  <cit> . coupled with our findings that lncrnas  from this locus are upregulated in aging liver, we can conclusively define this locus as cell-type specific regulatory hotspot in aging. whether this locus plays a major regulatory role in the aging process remains to be elucidated, but could be studied by testing if multiple proteins and/or pathways which are targeted are actually affected in aging tissues. in addition to known ncrnas, we also revealed novel lncrna transcripts differentially expressed with age, such as those flanking the cdkn2a locus. given the importance of the cdkn2a locus in both senescence and cancer, these flanking lncrnas may offer putative targets to regulate the cell cycle. more evidence for the implications of such ncrnas in aging will become more apparent as the field expands to functionally characterize each novel ncrna. taken together, our results provide the first evidence that ncrnas can also shape the landscape of aging mouse liver.\n",
            "\n",
            "finally, the comprehensive view of the aging liver transcriptome provided in our present study allowed us to determine the main functional networks associated with liver aging. in this respect our results uncovered three main sets of interacting networks: inflammation and cancer, proliferative homeostasis, and lipid metabolism. the most prominent one, inflammatory response, was also supported by multiple ncrnas novel to liver aging in the infg-mediated pro –inflammatory response. increased inflammation has since long been associated with aging  <cit>  and we and others have previously provided evidence that this is a major pro-aging phenotype in the mouse  <cit> . a second major set of altered gene functional networks in aging mouse liver involves the interconnected cellular proliferation and death networks. systematic dysregulation of the balance between degenerative cell loss and regeneration is in keeping with reports suggesting that liver regeneration is compromised in old animals and in elderly humans. lastly, a third major network of change appeared to be centered on lipid synthesis and lipid oxidation, pointing towards increased intracellular aggregates of damage, most notably lipofuscin. however, our data also suggest altered levels of cytochrome p <dig> family, such dysregulation could be a cause for the decline in drug metabolism in aging livers  <cit> .\n",
            "\n",
            "CONCLUSIONS\n",
            "taken together, our present work, and previous work by others  <cit> , shows how rna sequencing at high depth can address basic questions as to the molecular basis of mammalian aging. we showed that aging is unlikely to be accompanied by a significant loss of tissue-specific gene expression profiles; we did not find evidence for gene de-repression or a stochastic increase of the fraction of the genome that is transcribed. yet, our data uncovered extensive changes in gene expression patterns, which are subject to stochastic variation as indicated by a significant increase in individual variation. we also demonstrated that the increased accuracy of rna-seq enable us to better capture major aging phenotypes at the transcriptional level, thereby unmasking important molecular mechanisms underlying aging and its related disease sequelae.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "our data suggest that aging is accompanied by a significant loss of tissue-specific gene expression profiles, possibly as a result of a systemic response.\n",
            "in this study, we analyzed the aging mouse liver transcriptome and identified  <dig> novel ncrnas  that were significantly differentially expressed with age in mouse liver.\n",
            "our data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a consequence of a systemic response.\n",
            "our data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a result of a systemic response.\n",
            "our data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a result of a systemic response.\n",
            "our data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a result of a systemic response.\n",
            "our data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a result of a systemic response.\n",
            "our data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a consequence of a systemic response.\n",
            "our data suggest that aging is accompanied by a loss of tissue-specific gene expression profiles, possibly as a\n",
            "75\n",
            "[{'summary_text': 'in this study, we investigated the effect of two- and three-strain coinfections on the virulence of flavobacterium columnare  using zebra fish as a model host.\\nin this study, we investigated the effect of two- and three-strain coinfection on the virulence of flavobacterium columnare  using zebra fish as a model host.\\nin this study, we investigated the effect of two- and three-strain coinfection on the virulence of flavobacterium columnare  using zebra fish as a model host.\\nwe found that virulence was significantly influenced by the number of the coinfecting strains, infection with three strains leading to higher fish mortality than infection with one or two strains.\\nin this study, we investigated the effect of two- and three-strain coinfection on the virulence of flavobacterium columnare using zebra fish as a model host.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 7042 to 7168 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "while recent studies have demonstrated that a single host is often infected by a multitude of pathogen strains or species , empirical investigations of the effects of coinfections on disease dynamics and virulence  are still limited , and their importance for many diseases is still unknown  <cit> . coinfections affect both pathogen transmission in a population and pathogen virulence, thus influencing disease dynamics  <cit> . coinfections can also influence host immune responses  <cit>  and effectiveness of disease control . for these reasons, they are likely to contribute to virulence evolution, although the understanding on these evolutionary consequences is limited  <cit> .\n",
            "\n",
            "the disease outcome in a coinfected host results from complex interactions between the host and the coinfecting strains. whether the inter-strain interactions are neutral, cooperative or competitive, depends on the genetic relatedness of the strains . closely related pathogens are likely to cooperate and exploit their hosts economically in order to maximize their transmission, while distantly related pathogens are more likely to compete, leading to increased virulence and decreased transmission due to facilitated host death  <cit> . high relatedness can lead to low virulence if the cooperation of pathogens leads to prudent host exploitation  <cit>  or to high virulence if the cooperation leads to faster growth  <cit> . similarly, low relatedness may cause high virulence via increased growth and resource use  <cit>  or reduced virulence, if the competing pathogen strains produce toxins targeted to kill the competitor  <cit> . in the latter case the pathogen strains experience a cost, when the energy is allocated to toxin production instead of growth or virulence.\n",
            "\n",
            "there are three types of competition that coinfecting pathogens may encounter: resource competition, interference competition or apparent competition  <cit> . resource competition takes place when the coinfecting conspecific strains have overlapping resource requirements. in other words, the strain that processes nutrients more efficiently might outgrow the competing strain in nutrient-limiting conditions  <cit> . in interference competition the coinfecting strain secretes molecules that harm its competitor, e.g., bacteriocins  <cit> . bacteriocins have a narrow activity range and thus interference competition is often more prevalent between conspecific than distantly related strains  <cit> . the third form of competition is so-called apparent competition that results from indirect exclusion of the coinfecting strains by host immune response stimulated by growth of one strain that acts on both of the competing strains  <cit> .\n",
            "\n",
            "coinfections may be especially important for the ecology and evolution of opportunistic pathogens that are often able to persist in and transmit from the environment  <cit> . the opportunists with a wide host range have a higher likelihood to find potential hosts than host-specialists  <cit> . they are therefore more likely to be involved in coinfections. flavobacterium columnare  is a globally important host generalist fish pathogen with an opportunistic lifestyle and an ability to transmit environmentally . the bacterium causes columnaris disease in cultured freshwater fish, with typical symptoms including gill necrosis and skin erosion  <cit> . genetically different f. columnare strains with variable levels of virulence and growth rates are known to co-occur at fish farms  <cit> . furthermore, the virulent strains were found to have an increased ability for interference competition, and it is likely that these strains have a competitive advantage in the aquaculture environment  <cit> . however, a direct test of whether coinfections influence the virulence of f. columnare is missing. here, we study the effect of two- and three-strain coinfection on virulence of f. columnare using zebra fish  as a model host. we ask whether the inter-strain interactions are related to changes in virulence and growth through competition.\n",
            "\n",
            "RESULTS\n",
            "virulence in zebra fish hosts\n",
            "we found a significant interaction between bacterial dose and strain identity on their effect on the host mortality risk . the risk of infection of the host increased along with the dose when the fish were infected with strain a or strain b. in more detail, strain a was the most virulent, while strain b expressed intermediate level of virulence. strain c caused zero mortality in zebra fish hosts independent of the used infection dose.fig.  <dig> the estimated mortality risk per hour of zebra fish  infected with strain a , strain b  and strain c  of flavobacterium columnare with doses of  <dig>  3¯ × <dig>  2 ×  <dig> or 4 × 105 cfu ml−1\n",
            "\n",
            "\n",
            "\n",
            "a\n",
            "\n",
            "aintercept includes the effects of strain a\n",
            "\n",
            "\n",
            "< <dig> \n",
            "< <dig> \n",
            "< <dig> \n",
            "significant p values are denoted in bold\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "when we compared the single- and coinfection treatments, we found that mortality of the fish differed significantly between the treatment groups . the number of strains influenced the virulence , with the average longevity in the three-strain coinfections being  <dig>  h , while the longevity was  <dig>  h in the two-strain coinfections and  <dig>  h in the single infections . in pairwise comparisons , virulence of the single and two-strain coinfections did not statistically differ, but the other combinations did .fig.  <dig> cumulative mortality of zebra fish  as a function of time in the virulence experiment with single-strain and multiple-strain infections: coinfection of strains a) a and b, b) a and c, c) b and c, and d) a, b and c. the treatment groups are presented in the legend\n",
            "\n",
            "\n",
            "< <dig> \n",
            "< <dig> \n",
            "< <dig> \n",
            "< <dig> \n",
            ".002\n",
            "< <dig> \n",
            "< <dig> \n",
            "< <dig> \n",
            "< <dig> \n",
            "< <dig> \n",
            "< <dig> \n",
            "< <dig> \n",
            "after correction for multiple testing, p-values below  <dig>  can be considered significant \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "the coinfection treatments where strain a was involved were the most virulent independent of the strains included in coinfection, indicating that strain a may have a dominant role in coinfections in this study . respectively, when strain a was not involved in the coinfection treatment, the treatment resulted in very low virulence. the bacterial samples taken from the gill tissue of the infected and moribund fish were positive for f. columnare, whereas the bacterial samples taken from the gill tissue of the control fish were negative. the pure cultures made from the infection solutions revealed that colonies formed by strains a and b expressed the virulent rhizoid morphotype, but colonies of strain c represented the rough type, which has been linked with decreased virulence  <cit> . as the negative control fish were not infected, they were not included in the statistical analyses.\n",
            "\n",
            "interference competition on agar\n",
            "strains b and c were able to inhibit the growth of strain a, and strain c was able to inhibit the growth of strain b . strain a did not inhibit the other strains in this study. when the strains were exposed to the sterile-filtered supernatant, no signs of inhibition were observed, indicating that growth inhibition in f. columnare could be dependent on a cell-cell contact.table  <dig> the inhibition profiles of flavobacterium columnare strains a, b and c\n",
            "\n",
            "the number inside the cell stands for the frequency of inhibition in three independent experiments\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "growth in single-strain and co-cultures\n",
            "first, in relation to the dose controls in the virulence experiment, the individual strains were cultured in vitro in three doses  to reveal the influence of the original bacterial dose on bacterial replication. the bacterial dose was found to have a significant effect on the maximum growth rate of the strains  , and on the time when maximum yield  was achieved . bacterial strain included as a random factor in the analyses did not significantly influence any of the growth variables.\n",
            "\n",
            "secondly, to understand the increased virulence in the three-strain coinfection, we studied the effect of co-culture on bacterial growth. single or co-culture did not have overall significant effects on the bacterial growth parameters . however, the pairwise comparisons in the non-parametric anova revealed some significant differences between individual strains and co-culture combinations .table  <dig> statistically significant results of relevant pairwise comparisons of all single  and co-cultures  in kruskal-wallis one-way anova\n",
            "\n",
            "p\n",
            "p\n",
            "p\n",
            "\n",
            "fig.  <dig> scatterplot for maximum growth rate  of flavobacterium columnare in single and in co-culture. a) strains a and b and their co-culture, b), strains a and c, c) b and c, and d) a, b and c\n",
            "\n",
            "\n",
            "fig.  <dig> scatterplot for yield  of flavobacterium columnare in single and in co-culture. a) strains a and b and their 1: <dig> co-culture, b), strains a and c, c) b and c, and d) a, b and c\n",
            "\n",
            "\n",
            "fig.  <dig> scatterplot for time to reach maximal yield  of flavobacterium columnare in single and in co-culture. a) strains a and b and their 1: <dig> co-culture, b), strains a and c, c) b and c, and d) a, b and c\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "discussion\n",
            "environmentally transmitted bacterial pathogens are likely to cause both single and coinfections in their hosts, which may lead to plastic changes in virulence, but also to evolutionary change under long time scales. coinfections can lead to increased or decreased pathogen virulence, depending on the nature of the interactions between the coinfecting bacterial strains  <cit> . we examined how different strains of an opportunistic fish pathogen, f. columnare, interact with each other in culture, and how coinfection with these strains affects the disease virulence . we found that virulence was significantly influenced by the number of the coinfecting strains, infection with three strains leading to higher fish mortality than infection with one or two strains. generally, increased virulence can be related to the growth-stimulating mutualistic  <cit>  or competitive interactions between the coinfecting strains  <cit> . however, we found no unambiguous evidence of enhanced bacterial growth in co-culture in this study, indicating that the higher virulence in three-strain coinfection results from other factors. one possibility is that the host immune system has difficulties to cope with a heterogeneous pathogen inoculum compared to a single-strain infection .\n",
            "\n",
            "here, strain a had the highest virulence in single infections. similarly, the presence of strain a in coinfections resulted in highly virulent infections, indicating a dominating effect by this strain during infection, even with lower bacterial doses. relative doses of the coinfecting strains and relative virulence impact the outcome of within-host competition  <cit> . based on our results, the effect of dose on bacterial virulence in single infections increases exponentially in f. columnare, the lower doses not being sufficient to cause mortality in zebra fish. thus, the single strains could not have been responsible for the high host mortality in the three-strain coinfection treatment as their individual doses were only one third of the total infection dose. the high virulence in the three-strain coinfection treatments must therefore result from other factors, such as a plastic response to within-host competition , or factors related to the host immune system.\n",
            "\n",
            "the faster growth of the competing strains resulting in increased pathogen replication  and the inability of the host immune system to handle the heterogeneous simultaneous multiple infection are commonly associated with the increased virulence in coinfections  <cit> . however, we did not observe systematic differences in bacterial in vitro growth rate between single and co-cultures. furthermore, the elevated virulence due to immune response is often related to coinfection by different parasite species or types  <cit> . therefore, the strain interactions most likely have a central role in the elevated virulence in the three-strain coinfection of zebra fish in this study. indeed, rapid evolution of plastic changes in parasite virulence in response to coinfection has been documented in bacteriophages  <cit> , indicating that the variability in the infection outcome arises from the genetic interactions between the coinfecting partners.\n",
            "\n",
            "relatedness of the coinfecting strains has a fundamental role in strain interactions and virulence of coinfection, and interference competition  is common in bacteria . flavobacterium columnare is a genetically homogeneous species  <cit> , and according to the mlsa analysis, the genotypes c and e used in this study are closely related  <cit> . however, the genomic content outside the housekeeping genes often differs substantially between bacterial strains, suggesting genetic variability in these strains isolated originally from different fish farms in different years. this is important, because the production of growth-inhibiting toxins and immunity against them are genetically linked  <cit> . this is clearly seen as the capacity of the bacterial strains in distinguishing between “self” and “non-self” competitors in the inhibition assay. therefore, considering interference competition without genome sequencing, mlsa grouping does not give a complete picture on the genome-level relatedness of the strains, and the exact level of relatedness between the strains used in this study cannot be resolved without whole genome sequencing.\n",
            "\n",
            "in general, production of growth-inhibiting toxins is expected to be costly for the producer strain, and to be traded off with growth  <cit> . in some cases, release of toxins may even require cell death, as in e. coli  <cit> . our earlier studies indicated that the growth inhibiting toxin-production and tolerance may be linked with increased virulence in f. columnare  <cit> . however, coinfections by genetically diverse strains producing these toxins are predicted to reduce the population growth rate and consequently the virulence of the infection  <cit> . yet, similarly to the previous study  <cit> , we did not find any indications that toxin production would be traded off with growth in f. columnare. interestingly, however, the virulence of two-strain coinfection was directly linked with virulence of the individual strains, resulting in lower virulence in the two-strain coinfections than in the three-strain coinfection. moreover, the inhibitory compounds produced by the less virulent strains c and b against the highly virulent strain a could explain the reduced virulence observed in these coinfections. however, it seems that decreasing the relatedness of the coinfecting partners  makes interference competition most beneficial, resulting in higher virulence of the three-strain coinfection as observed here.\n",
            "\n",
            "our study demonstrates that pairwise interactions between coinfecting strains are important for disease virulence. similar findings have been observed in several studies using different host-parasite systems, e.g., rodent malaria parasite plasmodium chabaudi  <cit> , snail-infecting schistosome schistosoma mansoni  <cit> , baculoviruses  <cit>  and bacteria . therefore, perhaps not surprisingly, the genetic interactions between the coinfecting parasites may also extend across species  <cit> , challenging the reliability of epidemiological predictions based on single infections. as the coinfections  <cit>  and environmental pressures maintain the diversity of the pathogen populations and induce rapid changes in pathogen traits  <cit> , coinfections are among the main drivers for variance in disease epidemiology and evolution.\n",
            "\n",
            "CONCLUSIONS\n",
            "intraspecific competition can lead to increased virulence but the virulence of coinfection depends on the interactions of the strains involved in the coinfection. as the likelihood of coinfections by environmentally transmitted opportunistic pathogens is high, the genetic composition of the coinfecting population can lead to variable infection outcomes. this can significantly impact the outcomes of infections. furthermore, in opportunists not restricted by the transmission-virulence trade-off, coinfections may select for the most virulent pathogen strains.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study, we investigated the effect of two- and three-strain coinfections on the virulence of flavobacterium columnare  using zebra fish as a model host.\n",
            "in this study, we investigated the effect of two- and three-strain coinfection on the virulence of flavobacterium columnare  using zebra fish as a model host.\n",
            "in this study, we investigated the effect of two- and three-strain coinfection on the virulence of flavobacterium columnare  using zebra fish as a model host.\n",
            "we found that virulence was significantly influenced by the number of the coinfecting strains, infection with three strains leading to higher fish mortality than infection with one or two strains.\n",
            "in this study, we investigated the effect of two- and three-strain coinfection on the virulence of flavobacterium columnare using zebra fish as a model host.\n",
            "\n",
            "76\n",
            "[{'summary_text': 'in this model, multiple preferences are controlled by two haploid loci, each with two alleles: the preferences locus, p, controls female preference, and the trait locus, t, controls male traits.\\nhowever, there is great potential for costs associated with mate choice to be derived from female-female intrasexual competition, leading to indirect selection on preferences.\\nin this study, i first model the evolution of a single female preference in a system with costly intrasexual competition for mates, to determine when preferences may still evolve and the strength of selection acting on preference.\\nin this model, multiple preferences evolve in the face of costly intrasexual competition, leading to indirect selection on preferences.\\nin this model, multiple preferences evolve in the face of costly intrasexual competition for mates.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 6199 to 7168 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "costs associated with female preferences are often assumed to be directly related to the act of searching for a preferred mate: 1) time spent searching for a mate, 2) the potential for a choosy female to go unmated, or 3) an increased risk of predation  <cit> . these previously considered costs are viability costs, where the female’s chances of survival and successful mating are affected; here i present an argument for the role of fertility costs and their effect on preference evolution. there is great potential for costs involved in mate choice to be derived from female-female intrasexual competition as well. in general, these costs have not been widely studied or taken into account as potential selective forces driving  female preference evolution  <cit> .\n",
            "\n",
            "in resource-based polygyny, males provide females with resources such as parental care, defense, or territories in which to raise their young. in such scenarios, the cost of competing for a desired male is clear cut: it is well accepted that males may only support a limited number of females, and increasing beyond that threshold leads to decreased female reproductive fitness  <cit> . even in systems where resource limitations are less obvious, reduction in parental efforts can lead to decreased female fitness. for example, in dendrobatid frogs, brood sizes decreased significantly after multiple matings due to decreased male parental effort  <cit> . similarly, in polygynous tree swallows, females mated with polygynous males had reduced fitness because of decreased parental care  <cit> .\n",
            "\n",
            "even in polygynous species where males offer little to females, females may still incur costs simply by waiting to mate with a preferred male, by competing with other females for a preferred male’s attention, or by suffering reduced fecundity from male sperm depletion. in lekking birds, dominant females monopolizing preferred males time can lead to delayed breeding and decreased reproductive fitness  <cit> . females may also respond to competition for males with direct aggression, potentially injuring competitors  <cit> . sperm depletion and exhaustion, due to males mating multiply, may be costly to females as well  <cit> . sperm exhaustion has been tied to reduced reproductive fitness for females in insects  <cit> , fish  <cit> , and crustaceans  <cit> . although these costs are small compared to those suffered by females mating in resource based polygyny, they are all associated with significant decreases in reproductive fitness.\n",
            "\n",
            "in all of these situations, females are likely to experience a cost for preferring ‘popular’ males, i.e. those who have many mates. in fact, when females suffer fitness reductions from mating with sperm depleted males, if they can accurately assess the number of mates a male has, they choose males with fewer mates  <cit> . in general, however, it may be difficult for females to ascertain whether they are likely to suffer competitive costs: for example, in systems where males have large or overlapping territories, females have little or no information about additional mates; in systems where males provide resources that cannot easily be quantified, the female may have no information about these costs whatsoever.\n",
            "\n",
            "without direct knowledge, what can females do to avoid costly competition? one possibility is that multiple preferences may aid in alleviating or preventing competition. indeed, many of the species discussed previously as examples of costly female competition have multiple male traits and preferences as well . if females have differing preferences, and if males display differing traits, then competition could be reduced. for example, if females of some species may prefer complex song, long tails, or both, and males may have one or both of those traits; females choosing mates with high quality plumage may reduce their cost of competition because they are not competing with those who choose males with a complex song.\n",
            "\n",
            "empiricists have found cases of repeatable variability in genetically determined female preferences  <cit> . in such scenarios, females appear to be selecting mates based on multiple independent male traits. marchetti  <cit>  found evidence that female yellow browed leaf warblers based their choice of mate on several male characters, and although females preferred high quality males, different females used different traits to distinguish between these males  <cit> . not only demonstrated multiple preferences in female guppies, but demonstrated that they are heritable and genetically independent. the genetic assumptions of my model are built upon these findings.\n",
            "\n",
            "although there is ample empirical evidence of intrasexual mate competition in females, to my knowledge it has not been incorporated into evolutionary models. fawcett and johnstone  <cit>  considered the potential for female competition to alter mate choice from a game theoretic point of view, and showed that female competition could alter mating decisions. however, their model ignored genetics and focused primarily on alternative strategies, which is problematic because linkage disequilibrium between genes is a powerful evolutionary force. i chose to use a population genetic model which explicitly considers distinct genotypes and the potential for non-random association between loci  to evolve via assortative mating, leading to indirect selection on preference and traits.\n",
            "\n",
            "here, i argue that competition alone, regardless of the type of trait possessed by males, will impact preference evolution. to address these issues, i first model the evolution of a single female preference in a system with costly intrasexual competition for mates, to determine when preferences may still evolve and the strength of selection acting on preference. then, i consider whether or not the presence of an additional female preference alleviates competitive costs, and how selection on preferences changes with the introduction of an additional preference. when discussing multiple preferences, i am referring to multiple preferences controlled by independent loci: females may have no preferences, a single preference, or both. as novel preferences evolve to fixation, the result is that the majority of females possess both preferences.\n",
            "\n",
            "model specification and \n",
            "RESULTS\n",
            "i model mate choice with costly female competition for mates using a population genetic model with haploid loci and discrete non-overlapping generations, based on previous models of sexual selection via female choice  <cit> . the model assumes polygyny; all females mate, but males have variable mating success.\n",
            "\n",
            "for each model, i begin by describing the life cycle in terms of birth, mating, fertility selection, and zygote formation. using these equations, i can then calculate the strength of direct selection on preference using the notation of barton and turelli  <cit> .\n",
            "\n",
            "one preference, one trait \n",
            "female preference and male traits are controlled by two haploid loci, each with two alleles: the preferences locus, p, controls female preference, and the trait locus, t, controls male traits. uppercase letters indicate the presence of a preference or trait, lowercase letters indicate the absence. these two loci yield four genotypes: pt, pt, pt, and pt. i denote their frequencies as x <dig>  x <dig>  x <dig>  and x4; xt is used to denote the frequency of the male trait allele , and xp is used to denote the frequency of the female preference allele .\n",
            "\n",
            "females choose mates based on their preferences. a female without the preference allele  will mate randomly with respect to male genotype, whereas a female with the preference allele  is α times more likely to mate with a male possessing the trait allele, given that she has evaluated one of each type.\n",
            "\n",
            "mate choice results in a 4x <dig> matrix, f, whose elements fij represent the proportion of matings taking place between genotypes i and j:\n",
            "\n",
            "  fij=kijα⋅xixjzi, \n",
            "\n",
            "where kij is the modifier of preference strength for an xi female mating with an xj male; k <dig> and k <dig> are  <dig> , all others are  <dig>  males). zi is a normalization to ensure that all female genotypes have equal mating success; z1 = α xt + , z2 =  <dig>  the full mating table is given in table  <dig> \n",
            "\n",
            "\n",
            "females with the preference allele prefer males bearing a trait by a factor α. matings are normalized by z so that all female genotypes have equal mating success.\n",
            "\n",
            "after females have selected mates, fertility selection is exerted against the offspring of males with a surplus of mates. i denote the intensity of fertility selection by γ. when the mating frequency of a particular male genotype exceeds the population frequency of that male genotype, fertility is reduced proportionally. for genotype j, fertility selection is determined by\n",
            "\n",
            "  ϕj=γ∑i=14fi,jxj− <dig>  \n",
            "\n",
            "fφ, the fertility selection matrix, is calculated by multiplying each column of f by the corresponding fertility reduction suffered by the male parental genotype. the result is that\n",
            "\n",
            "  fijϕ=fij1−ϕj. \n",
            "\n",
            "recombination follows sexual selection and fertility selection; recombination rates are assumed to be  <dig>  between all loci for simplicity .\n",
            "\n",
            "using these life cycle equations, i first used numerical simulations  to confirm that it was possible to evolve preferences despite competitive costs. female preferences may still evolve with competitive costs, although the preference strength needed to overcome selection and fix preferences increases as costs become greater . this confirms that 1) female-female competition does act as a previously unexamined cost of choice, making it likely to cause natural selection against preference evolution, and 2) this cost does not completely bar preference evolution.\n",
            "\n",
            "\n",
            "to measure how strong natural selection against costly female preferences is, i applied the methodology of barton and turelli  <cit>  to calculate the strength of direct selection of preference. to illustrate the role of direct and indirect selection, one can write a general equation for the change in the frequency of preference alleles between generations:\n",
            "\n",
            "  Δp=ap,0cpp+at,0cpt \n",
            "\n",
            "here, Δp is the sum of direct selection and indirect selection. for any two loci x and y, ax,0cxy measures how the frequency of an allele at locus y changes due to the selection at locus x  and the genetic association between locus x and y . thus, change in preference is driven by direct selection on preferences, ap,0cpp, as well as indirect selection via the linkage disequilibrium between preference and trait, at,0cpt .\n",
            "\n",
            "equation  can then be used to partition out how much change in the frequency of a preference allele is due to direct versus indirect selection. the first term represents change due to direct selection:\n",
            "\n",
            "  Δpdirect=ap,0cpp. \n",
            "\n",
            "this represents direct selection on locus pi, favoring preference, with strength a, multiplied by the genetic variance at the pi locus, cpp.\n",
            "\n",
            "the procedure for solving for direct selection, ap, <dig>  is described in appendix  <dig>  the result is that we have an equation describing the strength of selection for  possessing a preference allele:\n",
            "\n",
            "  ap,0=−α−12p−11−t+dp,t1−2t⋅γα+t−αt <dig> \n",
            "\n",
            "where p is the frequency of the preference allele, t is the frequency of the trait allele, and dp,t is the linkage disequilibrium between preference and trait.\n",
            "\n",
            "in terms of selective forces, equation  demonstrates the selection on the preference locus is a function of preference and trait frequency, as well as preference strength and the cost of competition – all of which is intuitive from the model description. to understand what  means in more concrete terms, i first proved that the expression is always negative for realistic values of p and t . the sign of  is negative when  + dp,t >  <dig>  thus,\n",
            "\n",
            " 1−t+dp,t−2dp,tt> <dig> \n",
            "\n",
            " 1+dp,t>t1+2dp,t \n",
            "\n",
            "  1+dp,t1−2dp,t>t \n",
            "\n",
            "because 1≥t≥ <dig>  and linkage between preference and trait is greater than or equal to  <dig>  the right hand side of  is always positive, and, in turn,  is always negative.\n",
            "\n",
            "i plotted ap, <dig> for varying frequencies of preference and trait alleles, as well as different cost regimes . because the value of α does not change the shape of the curve, i only display results with α =  <dig>  for all scenarios with female competition for preferred males, ap, <dig> is negative . this means that direct selection always acts against female preference if competition is a factor; male traits in this scenario will only evolve if female preferences are sufficiently strong such that indirect  selection can outweigh direct selection.\n",
            "\n",
            "\n",
            "two preferences, two traits \n",
            "having shown that a single preference is selected against when females compete, i now consider whether or not a second preference is sufficient to alleviate competition, leading to direct selection for preferences.\n",
            "\n",
            "in this model, there are an additional two loci: two preference loci, p <dig> and p <dig>  control female preference, and two trait loci, t <dig> and t <dig>  control male display traits. these four loci yield 24=  <dig> genotypes: p1p2t1t <dig>  p1p2t1t <dig>  p1p2t1t <dig>  p1p2t1t <dig>  p1p2t1t <dig>  p1p2t1t <dig>  and so on. i denote their frequencies by x <dig>  x <dig>  …, x <dig> \n",
            "\n",
            "as before, females select mates based on their preferences. p <dig> corresponds to a preference for trait one and p <dig> corresponds to a preference for trait  <dig>  for example, a p1p <dig> female prefers males possessing the t <dig> trait. when a female possesses both preference alleles, p1p <dig>  she is α times as likely to mate with a t1t <dig> or t1t <dig> male, and  <dig>  α times as likely to mate with a t1t <dig> male, assuming she has encountered one of each type of male. mate choice and fertility selection are calculated as described in equations ,  and . the full mating table is given in table  <dig> \n",
            "\n",
            "\n",
            "i first confirmed that multiple preferences evolved in the face of costly competition. multiple preferences evolve but require stronger preference strengths  to reach fixation than preferences evolving in the absence of costly competition . interestingly, the strength of preference necessary to overcome the costs of choice is lower when multiple preferences are present versus a single preference. with simulations alone, however, it is impossible to determine if this is due to a decrease in competitive costs or an increase in indirect selection driven by stronger joint preferences by females with both preferences for males with both traits.\n",
            "\n",
            "to distinguish between a decrease in competitive costs and an increase in indirect selection, i again calculated the strength of direct selection . as before, selection is a function of trait and preference frequencies. because of the number of loci, the solution for ap, <dig> is a complicated expression. for analytical tractability, i performed a weak selection approximation assuming weak preferences and small values for linkage disequilibrium:\n",
            "\n",
            "  ap <dig> ≈1t1−12t2−12⋅γ−p21−t1p2−t22+p11−t1t1+3t2t2−1+p23t21−t2−1+p21−3t <dig> \n",
            "\n",
            "again, as expected, selection on preference is a function of trait frequency. without cost, or when both traits are fixed, selection on preference is  <dig>  under all other conditions, as before, selection is negative. because of the complexity of , proving that it is always negative is not feasible; i used numerical simulations to verify that with two preferences and traits, ap <dig> ≤ <dig> \n",
            "\n",
            "to visualize the strength of selection, i plotted the original  equation for direct selection for different preference and trait frequencies, and competitive costs . just as in the weak selection approximation, all values of ap, <dig> are negative, except in the case of γ= <dig> or α= <dig>  in which case ap,0= <dig> \n",
            "\n",
            "\n",
            "comparing figures two and three, it is clear that the presence of a second preference alters the strength of direct selection, but does not lead to direct selection for multiple preferences. in general, it appears that the presence of a second preference does decrease costs, but only when preferences are common. when preferences are rare, the presence of a second preference can increase competitive costs drastically by leading to female with two preferences having very strong preferences for rare two-trait males; this in turn would lead to fierce competition. thus, a second preference would not directly reduce competitive costs when introduced at a low frequency. when preferences are already at a high frequency, there is a benefit to having multiple preferences , but here i focused on low initial frequencies as an evolutionarily realistic scenario.\n",
            "\n",
            "indirect selection\n",
            "a major force behind the increased costs associated with multiple preferences is likely to be linkage disequilibrium  between female preferences and male traits. when a male has both trait alleles, he is attractive to females with either one or both preference alleles, and in turn produces offspring who have both male trait alleles along with the preference alleles, which in time creates a non random association between multiple male traits and female preferences. figure  <dig> shows this process over time. when preferences evolve to fixation, ld forms between preference and trait loci as they are spreading through the population. when there are two preference/trait pairs evolving simultaneously, ld is also created between the two traits, due to strong female preferences for the two trait males. as male traits become more common, they end up in negative ld  due to competition reducing the number of two-preference females. when i compared ld between preference and trait when only a single pair was evolving, vs two pairs evolving simultaneously, i found that it is always greater when two are evolving simultaneously. this increased indirect selection can then allow preferences to evolve when multiple preferences are present even though costs do prevent their evolution when only a single preference is present. despite strong direct selection against preference evolution, the ld created by multiple preferences may increase indirect selection enough to overcome natural selection and allow preferences to persist despite costs.\n",
            "\n",
            "\n",
            "simulation studies\n",
            "to make realistic predictions about the evolution of multiple preferences i used simulation models to explore when multiple preferences could evolve. using the model framework described above, i looked at the evolution of preferences with female competition. to explore the full range of possibilities for preference and trait evolution, i considered  <dig> scenarios:\n",
            "\n",
            " <dig> female preference for arbitrary male traits,\n",
            "\n",
            " <dig> female preference for male traits favored by natural selection,\n",
            "\n",
            " <dig> female preference for condition dependent traits, and\n",
            "\n",
            "for each scenario, i simulated the evolution of two preferences introduced simultaneously to the evolution of two preferences introduced successively . i performed numerical simulations in matlab; equilibrium conditions were found by running recursion equations for genotype frequency, as described above, until trait and preference alleles reached equilibrium. the results presented below are derived from genotype frequencies at equilibrium, which i defined as when the percentage change in genotype frequencies between successive generations was less than 10- <dig> \n",
            "\n",
            "female preferences for arbitrary male traits\n",
            "i began by simulating a four locus model of female preferences for arbitrary male traits, as described and modeled analytically above. females gained nothing from mating with preferred males other than producing attractive offspring, and there was no natural selection. with successive introduction of female preference, the initial female preference evolved to fixation when preference was high enough, despite the cost of competition. when a second preference was introduced, after the fixation of the first, even higher preference strength was needed to overcome competitive costs; competition for the limited pool of males with both traits prevented preference evolution unless preferences for male traits were very strong . when introduced simultaneously, both preferences could fix when the strength of female preference was high and the cost of competition relatively low . as predicted in figure  <dig>  with two preferences introduced simultaneously, the minimum preference strength for preference evolution decreased somewhat. it is also worth noting that the two preferences were able to fix under a broader set of circumstances when introduced simultaneously vs. successively – this replicates the result in figure  <dig>  where direct selection against preference is weaker when two preferences are at low frequencies  than when one is already at a high frequency .\n",
            "\n",
            "\n",
            "i also considered the role of the cost function itself  in determining the conditions under which preference may evolve. numerically, i simulated a convex cost function and a concave cost function, and compared the parameters under which preferences could evolve. as one might expect, a concave cost function expanded the parameter space where preferences evolved while a convex function further restricted the space where preferences could evolve. regardless of the shape of the cost function, as long as fertility was reduced in some way due to competition, the parameter space where preferences could evolve was restricted.\n",
            "\n",
            "female preferences for male traits favored by natural selection\n",
            "to model honest traits, i first considered male traits favored by natural selection. i altered the four locus model such that following birth, individuals underwent natural selection: individuals of type i without traits had fitness reduced by a fraction si. genotype frequency following natural selection was then described by:\n",
            "\n",
            "  xiη=1−sixi∑i=1161−sixi. \n",
            "\n",
            "where i∈ <dig> , <dig> . the xiη values in  replace the xi values in .\n",
            "\n",
            "the direct benefits of a male favored by natural selection is sufficient to overcome direct selection against preferences due to female competition—a single preference for naturally selected male traits fixed across a wide range of parameter combinations. figure  <dig> displays only a∈ <cit> , and γ∈, but a much wider range was examined for both parameters, and unless γ was unrealistically high , a single preference was able to fix. however, direct selection against preferences was able to prevent the evolution of a second preference when the first was fixed—as in figure  <dig>  two preferences were able to fix more readily when introduced simultaneously.\n",
            "\n",
            "\n",
            "condition dependent male traits\n",
            "for this scenario, i added a fifth locus c, which denotes an individual’s condition. individuals with c are considered low condition; those with c are high condition, and thus favored by natural selection. the result is 16⋅2= <dig> genotypes. i included directional mutation from c to c in order to maintain variation in condition.\n",
            "\n",
            "the life cycle consists of birth, natural selection, mate choice, fertility selection, zygote formation, recombination and mutation. during natural selection, low quality individuals , were  times as likely to survive. for mate choice, males displayed traits only if they were also in good condition, ie, females did not prefer low condition males, even if they carried trait genes. mate choice occurs as described in , using kij values given in table  <dig>  after mate choice, fertility selection occurs as in  and , followed by recombination, mutation, and zygote formation.\n",
            "\n",
            "\n",
            "because condition-dependent trait expression leads to increased mate competition because there is a decreased pool of males expressing traits, i considered two regimes for the evolution of condition-dependent preferences:\n",
            "\n",
            " <dig> evolution of preference along with condition, where preference and condition are introduced at low frequency simultaneously and allowed to evolve together, and\n",
            "\n",
            " <dig> evolution of preference in a system where the condition allele is at mutation selection balance .\n",
            "\n",
            "by examining both the evolution of condition allele with preference, and the introduction of preference into a high condition population, i can better distinguish the interaction between multiple preferences and condition evolution.\n",
            "\n",
            "for both regimes, having two preferences evolving simultaneously  increased the size of the parameter space where a second preference fixed, allowing it to fix at higher levels of competitive costs . further, as with naturally selected traits, direct selection for preferences for condition dependent male traits balances out direct selection against female preference due to competition, allowing preference to evolve under lower strengths and higher competition.\n",
            "\n",
            "\n",
            "discussion\n",
            "the results from my models indicate that intrasexual competition is costly and, when present, direct selection acts against preference evolution. multiple preferences change the shape of the cost curve but fail to alleviate costly competition when introduced at a low frequency; direct selection still acts against female preference when multiple preferences are present. this is not to say that intrasexual competition entirely prevents preference evolution; simulation results indicated that preferences may still evolve if they are sufficiently strong enough to overcome natural selection, and that the multiple preferences evolving simultaneously may reduce  direct selection. although multiple preferences do not lead to direct  selection for preference evolution, their presence is likely to increase the strength of indirect selection on preference and trait evolution, creating strong joint preferences in females with both preferences for males with both traits; this leads to a decrease in the initial preference strength required for evolution.\n",
            "\n",
            "in general, these results are consistent with other models, where costs associated with mate choice have been shown to prevent or restrict the evolution of multiple female preferences ;  <cit> . kirkpatrick’s  model of the sexy son hypothesis showed that handicap traits, which only lower fitness, do not spread. models explicitly considering multiple male traits with costly female preference, in terms of search costs/viability selection, also found that female preferences did not evolve due to high joint costs to preference  <cit> . in these models, if it was more costly for a female to search for and find a mate with multiple preferred traits rather than a male with a single trait, then multiple preferences could not evolve. similarly, in my model, having multiple preferences served to increase competitive costs when male traits were rare.\n",
            "\n",
            "my model supports the idea that intrasexual competition is likely to be a significant cost acting against the evolution of female preferences. there are many examples of intrasexual competition: direct aggression between females  <cit> , reduced fecundity due to decreased male parental efforts ,  <cit> , as well as decreased fecundity from male sperm depletion  <cit> , ,  <cit> . yet, in the majority of these species, female preferences have evolved regardless – including multiple preferences. in my models, competitive costs are not an insurmountable obstacle; although multiple preferences fail to alleviate competition, they don’t appear to be significantly more costly than a single preference, and in fact serve to increase indirect selection on preferences .\n",
            "\n",
            "multiple preferences may in fact serve to alleviate competition, just not in the way modeled here. one possibility is that if individual females have different preferences, controlled by a single locus, instead of multiple preferences controlled by multiple loci, competition could be averted. however, this scenario is unlikely: in most species with multiple preferences, these preferences appear to be controlled by independent genes brooks and coulridge,  <cit> . as my model has shown, if preferences are controlled by independent loci, after several generations, many individuals have both preferences leading to increased competition, not avoidance.\n",
            "\n",
            "perhaps multiple preferences may not indirectly prevent competition, but instead involve preferences for traits which indicate how many times a male has mated. one study showed that female cockroaches discriminated against males that had mated multiple times, and were able to detect cues on males derived from previous mates, in addition to traits indicating male quality  <cit> . however, it is difficult to imagine how common the ability to detect prior matings is, and there is only one such example in the literature. another possibility is that females could evolve multiple preferences and switch between preferences when they sense competition for a desired male. this would require knowledge about population wide preference frequencies, but would be possible in lekking species or animals that live in social groups.\n",
            "\n",
            "CONCLUSIONS\n",
            "when multiple preferences are present, indirect selection on female preference evolution is much stronger. perhaps instead of relieving competition, multiple preferences allow female choice to evolve by jointly increasing the strength of indirect selection to the point where many weak preferences can overcome natural selection against competition.\n",
            "\n",
            "appendix\n",
            "appendix  <dig>  ap, <dig> equations for a single preference and trait, two locus model\n",
            "\n",
            "the relative fitness of female preference and male traits is:\n",
            "\n",
            "  wxp,xt*=1−xp1−xt*ϕ1z1+1−xpxt*ϕ2z1+xp1−xt*ϕ3z2+xpxt*ϕ4z <dig> \n",
            "\n",
            "where xp represents the presence of preference alleles in females; xp= <dig> if a female has allele p, and  <dig> if she does not. likewise, xt*= <dig> if a male has allele t, and  <dig> if he does not. zi is the normalization for sexual selection . ϕi is the fertility selection against male genotype i . for example, for an x <dig> individual , xp= <dig> and xt*= <dig>  and xp,xt*=ϕ4z <dig> \n",
            "\n",
            "equation  can be used to calculate the a terms present in equation  in the text. to calculate the as, the fitness equation for a model  is set equal to a generic equation for fitness in terms of as and cs, and a function of the xs. terms are then matched to solve for a in the model under consideration. this procedure is described fully in appendix b of kirkpatrick and servedio.\n",
            "\n",
            "appendix  <dig>  ap, <dig> equations for a single preference and trait, four locus model\n",
            "\n",
            "the relative fitness of a female possessing a preference allele in the four locus model is:\n",
            "\n",
            "  wxp <dig> xp <dig> xt <dig> xt2=1−xp11−xp21−xt1*1−xt2*pαϕ1z1+1−xp11−xp21−xt1*xt2*αϕ2z1+1−xp11−xp2xt1*1−xt2*αϕ3z1+1−xp11−xp2xt1*xt2*ϕ4z1+1−xp1xp21−xt1*1−xt2*αϕ1z2+1−xp1xp21−xt1*xt2*αϕ2z2+1−xp1xp2xt1*1−xt2*ϕ3z2+1−xp1xp2xt1*xt2*ϕ4z2+xp11−xp21−xt1*1−xt2*αϕ1z3+xp11−xp21−xt1*xt2*ϕ2z3+xp11−xp2xt1*1−xt2*αϕ3z3+xp11−xp2xt1*xt2*ϕ4z3+xp1xp21−xt1*1−xt2*ϕ1z4+xp1xp21−xt1*xt2*ϕ2z4+xp1xp2xt1*1−xt2*ϕ3z4+xp1xp2xt1*xt2*ϕ4z <dig> \n",
            "\n",
            "as in appendix  <dig>  xpi represents the presence of preference alleles in females, where xp1= <dig> if a female has preference i, and  <dig> if she does not. likewise, xti= <dig> if a male has trait i, and  <dig> if he does not. zi is the normalization for sexual selection . ϕi is the fertility selection against male genotype i . as with female preference, there are only four unique male genotype combinations such that ϕ <dig> is the discount for t1t <dig> males, ϕ <dig> is for t1t <dig> males, ϕ <dig> is for t1t <dig> males, and ϕ <dig> is the discount for t1t <dig> males.\n",
            "\n",
            "as in appendix  <dig>  equation  is used to calculate the a terms present in equation  in the text. because of the complexity of equation , i applied a weak selection approximation to get a shorter, analytically tractable expression for ap,0: i assumed that costs were low, preferences weak, and linkage disequilibrium small , and performed a taylor series expansion of ap, <dig>  this method yielded equation , a considerably shorter expression for direct selection on preferences. to confirm the validity of the weak selection approximation, i compared it to the original expression and confirmed that, as α, γ, and di,j decreased, the two expressions converged. for the sake of comparison to , the equation used in figure  <dig> is the original formulation of ap, <dig>  not the weak selection approximation.\n",
            "\n",
            "competing interests\n",
            "the author declares that she has no competing interests.\n",
            "\n",
            "author contributions\n",
            "amf conducted all of the research presented in this paper and wrote the manuscript. she has read and approved the final manuscript.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this model, multiple preferences are controlled by two haploid loci, each with two alleles: the preferences locus, p, controls female preference, and the trait locus, t, controls male traits.\n",
            "however, there is great potential for costs associated with mate choice to be derived from female-female intrasexual competition, leading to indirect selection on preferences.\n",
            "in this study, i first model the evolution of a single female preference in a system with costly intrasexual competition for mates, to determine when preferences may still evolve and the strength of selection acting on preference.\n",
            "in this model, multiple preferences evolve in the face of costly intrasexual competition, leading to indirect selection on preferences.\n",
            "in this model, multiple preferences evolve in the face of costly intrasexual competition for mates.\n",
            "\n",
            "77\n",
            "[{'summary_text': 'acdc is an ideal tool to complement state-of-the-art contamination detection methods such as prodege.\\nacdc correctly identified the majority of contaminated and clean genome assemblies throughout all data sets.\\nacdc was able to predict the contamination state in large majority of samples from four unrelated data sets, containing a total of  <dig> single-cell genomes.\\nacdc was able to predict the contamination state in large majority of samples from four unrelated data sets, containing a total of  <dig> single-cell genome assemblies.\\nacdc was able to predict the contamination state in large majority of samples from four unrelated data sets, containing a total of  <dig> single-cell genome assemblies.\\nacdc was able to recall 22% more clean sequences on the simulated and benchmark data sets compared to prodege.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 5387 to 6144 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "modern sequencing technologies provide sample substrate for the analysis of large amounts of genetic information. specifically, single-cell sequencing  is now one of the most powerful methods in genome discovery and analysis. named method of the year  <dig>  <cit> , it plays an increasingly important role in many domains. notable areas of research include medicine and the analysis of disease pathways  <cit> , especially in cancer biology  <cit>  and the development of targeted treatments   <cit> . additionally, scs has proven a valuable and very powerful tool in evolutionary and environmental microbiology, for example by assessing intra- and inter-phylum relationships of bacteria and archaea  <cit>  and providing insights into key metabolic functions of uncultivated clades within their ecosystems  <cit> .\n",
            "\n",
            "a primary challenge in single-cell sequence data is the potential presence of contamination and the detection thereof  <cit> . foreign dna which does not belong to the target genome of a given single cell, might be introduced into a sample in different ways. sources of contamination can include unclean lysis or whole genome amplification reagents, in addition to sample introduced non-target dna  <cit> .\n",
            "\n",
            "while much effort has been invested into engineering devices and methods for cell isolation and amplification steps that minimize contamination caused by the surrounding sequencing setup  <cit> , careful quality control is vital to prevent the propagation of misleading results in public databases.\n",
            "\n",
            "given those obstacles, prodege, an automated protocol for the decontamination of genomes was recently developed  <cit> . prodege combines the blast algorithm  <cit>  as a popular choice for database sequence alignment with reference-free pca-reduced oligonucleotide profiling to enhance classification accuracy. another method, checkm  <cit> , solely relies on the presence of multiple single-copy marker genes as an indication for contamination in a given sample, not operating reference-free. more recent classification methods  <cit> , most notably kraken  <cit> , are as accurate as blast but much faster, thus can speed up supervised detection. all these techniques heavily rely on references, hence they require existing knowledge about the characteristics of possible contaminants, making them less applicable either in the case of contaminants not being contained in databases or marker genes not being present in the sample . since the majority of species is unknown  <cit> , they are difficult to detect by such methods and unsupervised, taxonomy-free analysis is required  <cit> .\n",
            "\n",
            "complementary to reference-based methods, clustering of oligonucleotide signatures is a promising approach that already found early application in metagenomic binning . from the perspective of computational intelligence, contamination detection as a clustering problem is very similar to metagenomic binning. both metagenomic and scs samples can be represented as a set of high-dimensional data points. binning and also contamination detection then address the same challenge of reliably detecting clusters in a high-dimensional data space. in this context, quite a few challenges arise: to circumvent negative side effects in such high-dimensional spaces  <cit>  and to enable human expert inspection, it is crucial to use appropriate subspace embeddings to transform the data into an easily visualizable representation, i.e. two or three dimensions. modern, non-linear dimensionality reduction methods, in particular barnes-hut-sne   <cit>  have proven successful  <cit>  in that context.\n",
            "\n",
            "the automatic determination of the number of clusters and its cluster validity, a deep and crucial question in the context of clustering  <cit> , poses yet another challenge. in contrast to metagenomic binning where the aim is to accurately bin sequences in a larger number of clusters, contamination detection in scs requires the discrimination between one or more clusters . this complication heavily reduces the set of applicable clustering algorithms: the majority of methods for estimating the number of clusters rely on cluster-specific measures such as internal validity indices  <cit> . since these are not defined for only one cluster, a distinctive null model for unimodal data is required, i.e. the techniques are usually not suited to distinguish one versus more than one cluster, hence cannot reliably identify non-contaminated samples.\n",
            "\n",
            "last, machine learning methods such as dimensionality reduction and clustering are based on statistics of the data and introduce certain amounts of variance. to overcome this limitation and to provide accurate and interpretable results, it is useful to integrate confidence measures. for this task, bootstrapping  <cit>  is a popular choice.\n",
            "\n",
            "in this contribution, we present a novel software tool called acdc , which seamlessly integrates reference-based with reference-free methods. being based on both, very fast exact database alignments and modern techniques from unsupervised machine learning, acdc is able to accurately identify contamination in single-cell sequencing data. to our knowledge, integrating entirely reference-free methodologies is a novelty, and complements existing high performing database-driven approaches such as prodege. the use of appropriate clustering algorithms allow the removal of foreign sequences to yield clean single-cell genome assemblies. additionally, the integration of statistically profound confidence values support expert interpretation. as we expect single-cell genomes to further and rapidly populate public databases, acdc will be a resource-effective tool in the quality assurance of single-cell draft genomes, especially for users who do not have the background to use the included techniques directly.\n",
            "\n",
            "implementation\n",
            "acdc detects contamination in a series of steps which are depicted in fig.  <dig>  starting with contigs from a given single-cell genome assembly, both reference-free and reference-based methods are employed. in the former, tetramer frequencies are calculated first , resulting in a high-dimensional vectorial representation which makes it possible to apply suitable machine learning algorithms. as its high dimensionality would introduce a number of adverse side effects in further processing, it is crucial to reduce dimensionality . this enables the accurate estimation of contamination confidences on the basis of clustering . external tools are then used to both classify sequences using ultrafast exact alignment  and to predict 16s rrna genes . in the case of detected contamination, further clustering algorithms are employed to enable decontamination and export of clean samples . results are then interactively visualized using a flexible web interface . most of these steps include a number of hyper-parameters crucial in machine learning, for which acdc provides an auto-selection mode with well-tested default values . in the future, the integration of results from existing tools such as prodege will help to increase detection performance.\n",
            "fig.  <dig> acdc contamination detection pipeline: results from both reference-free and reference-based techniques are fusioned and post-processed to end up with a clean sample\n",
            "\n",
            "\n",
            "i is the length of contig i. default choices of Δ\n",
            "w=w/ <dig> and k= <dig>  are robust. for contigs with l\n",
            "cc mutual nearest neighbors. the parameter k\n",
            "cc can be interpreted as the minimum number of data points contained in a separate cluster. to be able to detect also very small contamination, we use a default value of k\n",
            "\n",
            "\n",
            "\n",
            "reference-free detection\n",
            " <dig>  data pre-processing\n",
            "in order to apply machine learning techniques, it is necessary to transform contigs, represented as sequences, into a vectorial representation. here, it is common practice to use oligonucleotide signatures  <cit> . a window of width w is fixed and subsequently shifted over the contig sequence using step Δ\n",
            "w . for each shift, the underlying k-mer frequencies are evaluated. this results in one 4k-dimensional data point per shift, accounting for the  <dig> nucleotide bases. for example, taking k= <dig>  would result in  <dig> dimensions, however, by accounting for reverse complements, it can be reduced to  <dig> dimensions. it is worth noting that taking k= <dig> corresponds to the gc content. the choice of window parameters has considerable influence on the resulting representation. here, choosing a large window width, capturing genome-specific, rather than gene-specific information will result in less noise  <cit> . however, a small number of data points is disadvantageous for clustering, such that is has to be taken care to choose w not too large. acdc automatically adjusts window parameters such that for large contigs w is homogeneous and for small contigs w is adapted to it’s length, i.e. no contigs are discarded. table  <dig> includes further information on the setting of w. besides using k-mers as a characteristic genomic signature, we looked into using coverage, too. however, due to the coverage bias in multiple displacement amplification  <cit> , using this data for single genomes is problematic.\n",
            "fig.  <dig> data pre-processing that transforms a sequential data representation into vectorial data using a sliding window technique: exemplary for k= <dig>  on each shift, a 256-dimensional vector is generated by counting all permutations of the four bases\n",
            "\n",
            "\n",
            "\n",
            "\n",
            " <dig>  dimensionality reduction\n",
            "in machine learning, the analysis of high-dimensional data is severely limited by the so-called curse of dimensionality  <cit> . to be able to accurately cluster tetramer frequencies, it is crucial to reduce data dimensionality while keeping desired properties such as cluster structure. for this task, modern non-linear dimensionality reduction  and its recent, efficient barnes-hut approximation  is employed. it puts a particular focus on the formation of cluster structures, which enables further clustering algorithms to deliver accurate results. both qualitative and quantitative analysis  <cit>  of bh-sne have shown it is superior to both pca  <cit>  and to using raw high-dimensional vectors, when applied to tetramer frequencies.\n",
            "\n",
            " <dig>  estimation of contamination confidences\n",
            "an integral part of acdc is the confidence and decision of whether a sample is contaminated or not. this problem can be seen as a clustering task. optimally, one operational taxonomic unit  is represented as one cluster, implying that the presence of more than one cluster indicates contamination. thus, the task is to estimate the number of clusters k. this requires careful selection of methods and parameters  <cit> . in contrast to other applications such as metagenomic binning  <cit> , one is not primarily interested in the actual number of clusters, rather in the distinction between k= <dig>  and k> <dig> . as the notion of a cluster is ill-posed, this is an inherently difficult task: most techniques for estimating k operate on cluster-specific characteristics, defined for k> <dig> only, making them inapplicable in our case. the case k= <dig> requires an appropriate null model to which the data is compared to in order to be able to detect no structure. we reviewed techniques for this task in the context of contamination detection  <cit>  and found two particularly promising approaches: \n",
            "the dip-statistic test for multimodality of pair-wise distances , where a significant multimodal distribution indicates k> <dig>  <cit> .\n",
            "\n",
            "counting the number of strongly connected components  in a k\n",
            "cc-nearest-neighbor graph .  <cit> \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "contamination may occur in a variety of different cluster shapes and sizes. both methods have been chosen to be employed in acdc to detect those in an antagonistic fashion. while the former is able to detect large and possibly overlapping clusters, the latter is able to detect small and outlier clusters . consequently, a given genome assembly is marked as contaminated when dip or cc indicate more than one cluster.\n",
            "fig.  <dig> illustration of the complementary detection capabilities of dip and cc using two different contaminated samples. left: using a mutual 9-nearest-neighbor graph, cc identifies two clusters  while dip isn’t able to detect multimodality as seen in the distribution of pairwise distances below. right: two overlapping clusters prevent cc from detecting two components while dip detects significant multimodality in the distribution of pairwise distances\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "furthermore, noisy data, e.g. from very short contigs or from the inherent structure of some species might form separate clusters even in the presence of only one otu. to prevent false positive contaminant identification from wrongly formed clusters, acdc post-processes cluster assignments in two steps: \n",
            "disregarding the possibility of chimeric contigs, a contig is expected to appear in only one otu. thus, data points that occur in different clusters, but belong to the same contig, indicate a wrong clustering. all points of such a contig are reassigned to the cluster which has the most points of the contig assigned.\n",
            "\n",
            "we include an aggressive threshold that determines the minimum number of base pairs that is allowed to form a separate cluster. smaller clusters are considered as outliers and are neither included into the calculation of contamination confidences nor into cleansing. the default threshold of  <dig> bp works well throughout all tested data sets. a lower threshold provides more sensitive results towards very low levels of contamination and can be adapted by the user easily.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "last, the machine learning techniques used in acdc, namely dimensionality reduction and clustering, depend on data statistics and hence introduce certain amounts of variance over different runs. in the case of clear contamination, i.e. well separated and compact clusters, these techniques agree with high probability. the same holds true for the case of a clean sample and one well-shaped cluster. however, in the case of an unclear contamination state such as strongly overlapping clusters, results may vary. hence, it is desirable to provide confidence values gathered over different runs. for this task, acdc employs bootstrapping  <cit>  with which it is possible to calculate statistically substantiated and interpretable confidence measures. we generate bootstraps by randomly sub-sampling  <dig> percent of the original high-dimensional tetramer data b-fold. each fold is processed by applying dimensionality reduction with bh-sne and subsequent testing using dip/cc. a contamination confidence value is obtained by counting the percentage of folds which detected contamination.\n",
            "\n",
            "reference-based detection\n",
            " <dig>  sequence classification\n",
            "we employ kraken  <cit>  as a fast alternative to the popular blast method  <cit> . based on a pre-built database, kraken assigns taxonomic labels to each contig from a sample. through the use of exact alignments of l-mers, it achieves classification accuracy comparable to blast while being much faster. in acdc, kraken classifies contigs on a species level and assigns a taxonomy label to each data point, depending on it’s originating contig. in case of an unclassified species, a contig remains unknown.\n",
            "\n",
            "acdc primarily focuses on de novo analysis without existing knowledge from databases and it tackles the challenge to reliably answer the question whether a given sample is contaminated or not. we restricted reference-based cleansing to the fast kraken method and added an extension for unsupervised detection of potentially non-linear data clusters as performed by acdc. this distinguishes acdc from prodege which relies on both blasting predicted genes and a supervised linear separation of contaminants, primarily aiming for an aggressive cleansing with high precision.\n",
            "\n",
            " <dig>  16s rrna gene prediction\n",
            "acdc utilizes rnammer  <cit>  to predict the location of highly conserved 16s rrna gene sequences. even if data could not be classified by kraken, this enables researchers to identify the higher-level taxonomy of novel species quickly. additionally, the location of the 16s rrna gene sequence can be seen as a marker: it is representative for the whole cluster it is located in, and by exporting a clusters , the taxonomy for a full otu can be obtained.\n",
            "\n",
            "post-processing\n",
            " <dig>  cleansing\n",
            "if contamination is detected, acdc finds a clustering which allows the export of contigs from individual clusters, i.e. from the otu of interest. as this is a process of cleaning the sequence data from unwanted contaminant data, we refer to this as cleansing or decontamination.\n",
            "\n",
            "for this task, an optimal clustering has to be estimated. while cc provides an optimal assignment by itself, for dip the number of clusters k has to be estimated. in contrast to detecting contamination where the task is to determine either k= <dig> or k> <dig>  the cleansing step is slightly different. similar to metagenomic binning, it is known that k> <dig>  which makes it possible to apply methods that estimate the number of clusters using cluster-specific characteristics, only defined for that case. many clustering and k-estimation techniques are available for this task. in  <cit>  it is suggested that the combination of k-means++ as a clustering algorithm and the davies-bouldin index  <cit>  as a cluster validity measure works well for binning metagenomic tetramer profiles. in acdc, we replace the k-means++ algorithm by hierarchical clustering using wards method  <cit> . we found that it estimates the number of clusters more accurately when there are imbalanced cluster sizes, which we found to be the case in contaminated scs samples. therefore, an optimal cluster assignment is determined by finding the minimal  davies-bouldin index for a given range of k∈{ <dig> , <dig> }-clusterings using wards hierarchical clustering.\n",
            "\n",
            " <dig>  result visualization\n",
            "acdc provides contamination screening results as interactive web pages. an exemplary result of twenty simulated scs samples is shown in fig.  <dig>  for each sample on the left hand side, confidences from cc and dip are shown. a sample is marked clean when for both cc and dip less than  <dig> percent of all bootstrap folds found contamination. if either dip or cc found more than  <dig> percent of all folds to be contaminated, the sample is marked appropriately. in case of no clear result, a sample is marked with a warning status symbol. a third column with the number of species reported by kraken is shown. the user is able to inspect each sample for cc, dip and kraken. on the right hand side, the sample is visualized using bh-sne by default. in the event of a wrong cluster assignment, the number of clusters k can be selected manually, with the most likely k being selected by default. for kraken, the assignments are fixed and can be inspected by hovering on each data point. contigs from each cluster can be exported by clicking on the corresponding color in the panel below. locations of predicted 16s rrna gene sequences as reported by rnammer are indicated by an orange star. a click on it will show the corresponding sequence.\n",
            "fig.  <dig> acdc result interface. for each sample shown in the left-hand side table, visualizations are shown on the right-hand side. individual clusters can be exported in fasta format by clicking on the respective cluster color on the bottom right\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "computational performance\n",
            "acdc has low computational requirements. input data sizes are in the order of a few megabytes, as we work on assembled contigs, not on the raw data. given that, using a quad-core consumer laptop, runtimes ranged from a few seconds to ten minutes per sample, depending on the actual size and contamination status. the computationally most expensive step is the calculation of the dip statistic which has quadratic runtime in it’s worst case and has to be run for all bootstrap folds. this is sped up by parallelization. memory usage scales linearly with input size.\n",
            "\n",
            "RESULTS\n",
            "the evaluation of our tool can be divided into supervised  and unsupervised detection analysis. while the former is restricted to only the method to classify sequences and the size of the underlying database, the latter requires more careful assessment. in order to obtain accurate results, it is necessary to use data with correct ground truth. as the manual assignment of contamination is biased, the simulation of single-cell samples or the analysis of existing samples with references are vital.\n",
            "\n",
            "to cover a broad range of contaminant varieties, we tested acdc on several simulated and real single-cell sequence data sets: \n",
            "\n",
            "simulated: we simulated  <dig> single-cell genomes with varying amounts of contamination and contaminant relatedness. by manually selecting complete genomes from the ncbi database  <cit> , clean and contaminated data sets, each containing up to  <dig> genomes were generated. species were chosen such that they are related on different phylogenetic levels, expecting that distantly related species can be better separated than very similar species. for each level,  <dig> samples were generated, containing  <dig> –  <dig> species. the simulation of reads was done using art  <cit>  followed by subsequent assembly using spades  <cit> .\n",
            "\n",
            "\n",
            "mix:  <dig> samples containing  <dig> draft genomes and  <dig> single chromosomes were obtained from various sequencing projects . all samples are known to be contaminated, however, an exact quantification of contaminated sequences is missing due to the novelty of the data. a detailed description of these data can be found in the additional file  <dig> \n",
            "\n",
            "\n",
            "benchmark: sequence data from  <dig> single-cell genomes with low levels of contamination were obtained  <cit> . containing cross-contamination between  <dig> species , the median per-sample contaminant proportion of 3% is very small .\n",
            "\n",
            "\n",
            "mdm: furthermore,  <dig> single-cell samples from the microbial dark matter  project  <cit>  were taken to test the capability of our tool on non-contaminated data. these data were manually curated.\n",
            "\n",
            "\n",
            "\n",
            "herbinix luporum sd1dt\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "methanobacterium formicicum mft\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "we compared acdc to the state-of-the-art contamination detection tool prodege  <cit>  both in terms of supervised and unsupervised detection capabilities. prodege has been optimized to obtain a high precision in the context of a known taxonomic level and database support. it integrates a linear classification model to extend predicted genes to all k-mers, displaying excellent behavior in aggressively curating according samples. unsupervised inspection is restricted to linear pca only. in contrast, acdc has been optimized to provide good f-measures  in curating, and it addresses database independent de novo detection of contamination, thus providing a tool highly complementary to prodege.\n",
            "\n",
            "supervised analysis\n",
            "both prodege using the blast algorithm and acdc using kraken with the “minikraken db” were tested on the simulated and benchmark data sets. these are the only two data sets for which entries for known contaminants existed in both used databases. both tools showed nearly identical high performance  in identifying contaminant sequences and didn’t require any further evaluation.\n",
            "\n",
            "unsupervised analysis\n",
            "the evaluation of unsupervised detection performance was carried out a) by testing the ability to detect the correct contamination state of a given sample, and b) by measuring the ability to correctly identify clean and contaminant contigs.\n",
            "\n",
            "\n",
            "a) acdc correctly identified the majority of both contaminated and clean genome assemblies throughout all data sets . this result demonstrates the ability of acdc to single out contaminated versus clean genome assemblies, specifically without any reference to a database in de novo settings. for this part of the evaluation, we could not compare to existing methods because they either do not have the functionality to distinguish clean and contaminated samples , or operate reference-based only . warnings are sometimes issued for assemblies with unclear contamination state. here, further inspection often revealed the presence of small outlier clusters throughout a small number of bootstraps. in the rare case of strongly unbalanced and additionally overlapping clusters, acdc is not able to detect contamination because of missing structure in the data. further, if the contaminant is too related to the target , genomic signatures differ only by a very small percentage of all basepairs, making it impossible for acdc to detect them. interestingly, mdm samples that have been identified as contaminated display a quite distinct cluster structure. further manual investigation on a small subset of these samples revealed the presence of true contamination which was not identified during manual curation. furthermore, the sequence of a bacteriophage was identified. horizontally transferred genetic material such as from bacteriophages or plasmids often have significantly different genomic signatures. hence, the found structures highlight biologically interesting phenomena.\n",
            "\n",
            "simulated\n",
            "\n",
            "mix\n",
            "\n",
            "benchmark\n",
            "\n",
            "mdm\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "b) we compared <dig> acdc to prodege in terms of precision/recall performance with respect to the number of correctly identified clean basepairs in each sample. for this task, the functionality to export clean sequences common in both tools was used. since the evaluation is performed for the setting of limited prior biological information, no taxonomy is provided for prodege, restricting the use of reference sequences from databases. results in table  <dig> were averaged over different samples from the simulated and benchmark data sets. both prodege and acdc correctly identified clean contigs in the benchmark data set with high precision. however, on average acdc was able to recall 22% more clean sequences on the data set, due to the more aggressive design of prodege. next, prodege was not able to identify the majority of clean sequences in the simulated data set without taxonomic information. in those cases, mostly all contigs were marked as contaminants, resulting in an empty clean sequences file. this fact can be attributed both to prodeges behavior of selecting contaminants with high specificity  <cit>  and to it’s missing ability to distinguish between clean and contaminated samples. results of  <dig> samples could not be obtained, because computation didn’t provide any output. on the same data, acdc was able to correctly identify the majority of clean sequences with high precision and recall. for samples that contain closely related species, it is difficult to split clean and contaminated sequences. for example, in our simulated data, samples from the same genus contain species with an average nucleotide identity  of 73%. this fact led to a slight drop in performance. sequences containing strains from the same species  didn’t contain enough distinct information to be correctly identified, showing the limits of acdc’s reference-free detection capabilities.1-scores of predicted clean base pairs for both prodege and acdc on the simulated and benchmark data sets\n",
            "\n",
            "precision\n",
            "recall\n",
            "f\n",
            "1\n",
            "precision\n",
            "recall\n",
            "f\n",
            "1\n",
            "\n",
            " <dig> \n",
            "\n",
            " <dig> \n",
            "\n",
            " <dig> \n",
            "\n",
            " <dig> \n",
            "\n",
            " <dig> \n",
            "\n",
            " <dig> \n",
            "\n",
            " <dig> \n",
            "\n",
            " <dig> \n",
            "\n",
            " <dig> \n",
            "\n",
            " <dig> \n",
            "each row contains average values of the given sub data set. bold values depict the best performing entry. entries marked as “no result” either produced an empty clean fasta file or did not finish computation\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "CONCLUSIONS\n",
            "operating both in the presence and absence of references from databases, acdc was able to predict the contamination state in the large majority of samples from four unrelated data sets, containing a total of  <dig> single-cell genome assemblies. additionally, clean and contaminant sequences were correctly identified with high recall and precision. in the absence of a given target taxonomy which is required by similar methods , acdc was still able to correctly predict contamination based on state-of-the-art techniques from unsupervised machine learning. complementary to other tools, our software does neither require the prediction of  genes nor existing knowledge from databases to detect contaminants and to separate contaminant from clean sequences. although, supplemental database information will aid identification, for example of closely related species. these findings make acdc an ideal tool to complement state-of-the-art contaminant detection and cleansing methods such as prodege or checkm in the context of de novo analysis with limited taxonomic information or limited availability of reference sequence information. last, as contamination detection and metagenomic binning are closely related, we look forward to applying a modified version of our pipeline to this type of data in the near future.\n",
            "\n",
            "availability and requirements\n",
            "\n",
            "\n",
            "project name: acdc\n",
            "\n",
            "\n",
            "project home page:\n",
            "https://github.com/mlux86/acdc\n",
            "\n",
            "\n",
            "\n",
            "operating system: linux\n",
            "\n",
            "\n",
            "programming language: c++11\n",
            "\n",
            "\n",
            "other requirements: none\n",
            "\n",
            "\n",
            "licence: mit\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "endnote\n",
            "\n",
            " <dig> for the comparison the prodege online version at https://prodege.jgi.doe.gov/was used.\n",
            "\n",
            "additional file\n",
            "\n",
            "additional file  <dig> a detailed description of the mix data set. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "abbreviations\n",
            "acdcautomated contamination detection and confidence estimation for single-cell genome data\n",
            "\n",
            "aniaverage nucleotide identity\n",
            "\n",
            "bh-snebarnes-hut-sne\n",
            "\n",
            "ccconnected components\n",
            "\n",
            "dipdip-statistic test for multimodality\n",
            "\n",
            "mdmmicrobial dark matter\n",
            "\n",
            "otuoperational taxonomic unit\n",
            "\n",
            "pcaprincipal component analysis\n",
            "\n",
            "scssingle-cell sequencing\n",
            "\n",
            "t-snethe-distributed stochastic neighbor embedding\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "acdc is an ideal tool to complement state-of-the-art contamination detection methods such as prodege.\n",
            "acdc correctly identified the majority of contaminated and clean genome assemblies throughout all data sets.\n",
            "acdc was able to predict the contamination state in large majority of samples from four unrelated data sets, containing a total of  <dig> single-cell genomes.\n",
            "acdc was able to predict the contamination state in large majority of samples from four unrelated data sets, containing a total of  <dig> single-cell genome assemblies.\n",
            "acdc was able to predict the contamination state in large majority of samples from four unrelated data sets, containing a total of  <dig> single-cell genome assemblies.\n",
            "acdc was able to recall 22% more clean sequences on the simulated and benchmark data sets compared to prodege.\n",
            "\n",
            "78\n",
            "[{'summary_text': 'using network topological features, tissue expression profile and subcellular localization data, we show that our meta-classifier can reliably predict morbid and druggable genes on a genome-wide scale and also can define cellular rules governing morbidity and druggability in human.\\nour meta-classifier trained on  <dig> network topological features, tissue expression profile and subcellular localization data seems indeed to be a reliable predictor of morbid and druggable genes on a genome-wide scale.\\nour meta-classifier trained on  <dig> network topological features, tissue expression profile and subcellular localization data seems indeed to be a reliable predictor of morbid and druggable genes on a genome-wide scale.\\nour meta-classifier trained on  <dig> network topological features, tissue expression profile and subcellular localization data seems indeed to be a reliable predictor of morbid and druggable genes on a genome-wide scale and also can define cellular rules governing morbidity and druggability in human.\\nour meta-classifier trained on  <dig> network topological features, tissue expression profile and subcellular localization data seems indeed to be a reliable predictor of morbid and druggable genes on a genome'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 4438 to 5120 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "currently, the large-scale experimental identification of both morbid genes, i.e. those genes whose mutations cause hereditary human diseases, and druggable genes, i.e. genes coding for proteins whose modulation by small molecules elicits phenotypic effects, demands time-consuming and laborious approaches that are impractical for rapidly revealing the causal relationships between genes and diseases and determining the druggability of gene products. the discovery of morbid genes, for instance, requires a large effort to gather inheritance patterns from families with the disease and to perform linkage and mutation analyses in order to identify candidate gene involved in a particular hereditary disorder  <cit> . in similar fashion, the discovery of new drug targets also requires a large effort involving a variety of genomics, proteomics, genetic association and forward and reverse genetics-related techniques  <cit>  in order to find drugs capable to modulate disease processes.\n",
            "\n",
            "in the light of above mentioned facts, a computational approach which could accurately predict morbid and druggable genes, especially on a genome-wide scale, would be thus invaluable since the number of experimental techniques to be performed to discover these genes could be minimized. with the vast amount of current available systems-level data, such as molecular interaction data and genome-wide gene expression and subcellular localization data, we have now the opportunity for developing a computational approach based on data mining tools, such as machine learning, to extract patterns that could be used as genome-wide predictors of morbid and druggable genes. based on this assumption, we have previously used a machine learning-based methodology as a data mining tool to extract knowledge from systems-level data and then apply this knowledge to predict essential genes on a genome-wide scale and determine cellular rules for essentiality on escherichia coli <cit>  and saccharomyces cerevisiae <cit> . in addition to attain successful prediction rates, we have also obtained biologically plausible cellular rules for gene essentiality using this machine learning approach.\n",
            "\n",
            "due this successful prediction of essential genes and determination of cellular rules for gene essentiality in escherichia coli and saccharomyces cerevisiae, we sought to verify in this present work whether a similar machine learning-based approach is able to predict human morbid and druggable genes on a genome-wide scale and to reveal cellular rules governing morbidity and druggability of genes. using knowledge acquired from network topological features, tissue expression profile and subcellular localization data, we show here that the classifiers trained on these systems-level data can reliably predict morbid and druggable genes on a genome-wide scale and also can define some general rules governing morbidity and druggability in human.\n",
            "\n",
            "RESULTS\n",
            "the integrated network of human gene interactions and calculation of topological features\n",
            "for obtaining the network topological features used as training data for predicting morbid and druggable genes, we first constructed an integrated network of human gene interactions  simultaneously containing experimentally verified protein physical interactions, metabolic interactions and transcriptional regulation interactions . this network is comprised by  <dig>  genes interacting with one another via  <dig>  protein physical interactions,  <dig>  metabolic interactions and  <dig>  transcriptional regulation interactions. inhgi contains approximately 25% of the already identified ≈  <dig>  human genes according to the entrezgene database  <cit> .\n",
            "\n",
            "from the inhgi, we calculated  <dig> different topological features for each gene, including degree centralities for each type of interaction, clustering coefficient, betweenness centralities for each type of interaction, closeness centrality and identicalness. the detailed description of these topological features and how they were calculated are found in the additional file  <dig> and “methods”.\n",
            "\n",
            "evaluation of classifier performance\n",
            "to examine how well a machine learning-based approach is able to predict human morbid and druggable genes on a genome-wide scale using knowledge acquired from systems-level data, we designed a meta-classifier similar to that used to predict essential genes in escherichia coli <cit>  and saccharomyces cerevisiae <cit>  and trained it on network topological features, tissue expression profile and subcellular localization data of known morbid and druggable genes . we then assessed its performance by measuring its median recall, precision and area under the curve  of the receiver operating characteristic  curve across  <dig> different normal morbidity datasets and  <dig> different normal druggability datasets .\n",
            "\n",
            "before analyzing the performance measures of our meta-classifier trained on the datasets described above, we decided to estimate the performance measures of our meta-classifier on equivalent normal morbidity and druggability datasets where the class labels—morbid and druggable—were randomly shuffled among genes  and then compared them with our meta-classifier trained on the normal morbidity and druggability datasets. this was done to check whether the meta-classifier trained on non-shuffled datasets learned the traits actually associated with morbidity and druggability instead of traits associated with any random subset of genes. for this comparison, we used the wilcoxon signed-rank statistical test as described in “methods”. as can be observed in table  <dig>  all performance measures of our meta-classifier trained on the correspondent shuffled datasets were statistically different from measures of meta-classifier trained on normal datasets , thereby indicating that the traits actually associated with morbidity and druggability were learned by our meta-classifier.\n",
            "\n",
            " <dig> of  <dig> datasets\n",
            "\n",
            " <dig> according to table of critical values for w in  <cit> \n",
            "\n",
            "* difference statistically significant\n",
            "\n",
            "after confirmation that our meta-classifier trained on normal datasets was likely to learn the traits actually associated with morbidity and druggability, we aimed to analyze its performance measures. as shown in table  <dig>  for the genome-wide prediction of morbid genes, our meta-classifier achieved a median recall of  <dig>  and a median precision of  <dig> , i.e., it correctly recovered  <dig> % of known morbid genes with a precision of  <dig> %. furthermore, the probability of a gene predicted as morbid belongs to the set of known morbid genes is  <dig> % as indicated by the median auc. for the genome-wide prediction of druggable genes, our meta-classifier achieved a median recall of  <dig>  and a median precision of  <dig> , i.e, it correctly recovered  <dig> % of known druggable genes with a precision of  <dig> % . furthermore, the probability of a gene predicted as druggable belongs to the set of known druggable genes is  <dig> % as indicated by the median auc.\n",
            "\n",
            "the moderate values for both median recall  and median precision  for genome-wide prediction of morbid genes indicate that the level of noise in the training data is high and likely associated with existence of shared common features between morbid and non-morbid genes that induced our meta-classifier to yield a moderate performance in discriminating morbid from non-morbid genes. this could be partially due to the approach used to select non-morbid genes: since it is impossible at present to compile a list of genes not known to cause any hereditary disease, we selected genes not known to be morbid, i.e., all genes in inhgi except the known morbid genes, as non-morbid genes. thus, some of these non-morbid genes may actually be existing unknown morbid genes sharing common characteristics with the existing known morbid genes. other contributing factor for the existence of shared common features between morbid and non-morbid genes could be the incompleteness of inhgi: stumpf et al. <cit> , for example, have estimated that the size of human interactome  is about  <dig>  interactions. since our network contains about  <dig>  protein-protein interactions, we could envisage that the values of all network topological parameters might change with the enlargement of network size and, therefore, some of the network topological parameters-related shared common features between morbid and non-morbid might disappear as a consequence. the existence of shared common features between druggable and non-druggable genes also seems to affect the performance of our meta-classifier, but to a lesser extent: our meta-classifier achieved reliables values for the median recall  and precision  for genome-wide prediction of druggable genes .\n",
            "\n",
            "despite these limitations discussed above, our meta-classifier trained on network topological features, tissue expression profile and subcellular localization data seems indeed to be a reliable predictor of morbid and druggable genes on a genome-wide scale as shown by figures  <dig> and 2: the frequency distribution of known morbid and known druggable genes per intervals of morbidity and druggability scores—probabilities of classifying genes as morbid and druggable, respectively, as output by the meta-classifier —tend to increase as morbidity  and druggability  scores increase.\n",
            "\n",
            "evaluation of individual features on classifier performance\n",
            "we sought to verify the influence of individual features on the meta-classifier performance. to achieve this goal, we first trained our meta-classifier on normal morbidity and druggability datasets without one of the features, which we call “without-one-feature” datasets as described in “methods” . we then compared the output auc values with those of meta-classifier trained on datasets with all features by using the wilcoxon signed-rank statistical test  <cit> . a difference is considered statistically significant if the obtained w is lower than or equal to wc with a given n at the p =  <dig>  level . note that we use auc instead of recall or precision to compare the overall performances of meta-classifiers because it represents the meta-classifier performance across all combinations of recall and precision . table  <dig> shows that the median auc of our meta-classifier trained on morbidity datasets without the number of tissues in which the gene is expressed at least  <dig> transcripts per million   was statistically lower than the median auc for normal morbidity datasets . so, the tissue expression profile seems to be an important feature to distinguish morbids from non-morbids genes.\n",
            "\n",
            " <dig> see “methods” and additional file  <dig> for a description of features\n",
            "\n",
            " <dig> of  <dig> datasets\n",
            "\n",
            " <dig> according to table of critical values for w in  <cit> \n",
            "\n",
            " <dig> the number of tissues  in which the gene is expressed at least  <dig> transcripts per million  according to reverter et al.  <cit> \n",
            "\n",
            " <dig> the average expression in tpm among all the tissues in which the gene is expressed according to reverter et al.  <cit> \n",
            "\n",
            "* difference statistically significant\n",
            "\n",
            "as shown in table  <dig>  for prediction of druggable genes, the overall performance  of our meta-classifier was statistically lower following the removal of the plasma membrane feature . this result is in concert with the most important cellular rule for druggability derived from the analysis of decision trees  that we will show in the section“cellular rules for gene morbidity and druggability”): if proteins are located in plasma membrane, their encoding genes are likely to be druggable. this rule is supported by bakheet and doig  <cit>  that demonstrated that proteins encoded by druggable genes had more transmembrane helices than proteins encoded by non-druggable ones which suggests that proteins encoded by druggable genes are more likely to be found in plasma membrane.\n",
            "\n",
            " <dig> see “methods” and additional file  <dig> for a description of features\n",
            "\n",
            " <dig> of  <dig> datasets\n",
            "\n",
            " <dig> according to table of critical values for w in  <cit> \n",
            "\n",
            " <dig> the number of tissues  in which the gene is expressed at least  <dig> transcripts per million  according to reverter et al.  <cit> \n",
            "\n",
            " <dig> the average expression in tpm among all the tissues in which the gene is expressed according to reverter et al.  <cit> \n",
            "\n",
            "* difference statistically significant\n",
            "\n",
            "comparison with other methods\n",
            "regarding prediction of morbid genes, there have been several methods available for predicting morbid genes  <cit> . however, our method can not be directly compared to most of them since they have been constructed to predict only small sets of disease-specific candidate genes, such as endeavour  <cit>  and toppgene  <cit> , while our method has been constructed for the genome-wide prediction of morbid genes. we can, however, compare our method to prospectr  <cit> , cipher  <cit>  and that developed by xu and li  <cit> . our method outperforms cipher  and is comparable to prospectr that achieves a recall of  <dig> , a precision of  <dig>  and an auc of  <dig> . although prospectr has a higher recall, we considered our method comparable to it as the precision and auc values of our method are higher than those of prospectr. moreover, our performance measures are medians of  <dig> runs of 10-cross-fold validation , while the performance measures of prospectr were obtained by only one run of 10-cross-fold validation.\n",
            "\n",
            "the method developed by xu and li is the only genome-wide prediction method that apparently outperforms our method . their method is also based on network topological parameters, but while we trained our meta-classifier on various features, including  <dig> network topological parameters , they trained their classifiers on only five network topological parameters: degree, defined as the number of links to node i; 1n index, defined as the proportion of the number of links to morbid genes among all links to node i; 2n index, defined as the proportion of the number of links to morbid genes among all links to neighbors of node i; the average distance to morbid genes; and positive topological coefficient, a variant of the classical topological coefficient  <cit> . the apparent success of xu and li approach in predicting morbid genes mostly relies on the 2n index: when node i is a morbid gene, 2n index is always higher than zero since at least one neighbor of node i’s neighbor—the node i itself—is a morbid gene; if node i is a non-morbid gene, 2n index is higher than or equal to zero. thus, this parameter induces a spurious correlation on dataset that is captured by classifiers that, in turn, achieve high performance measures. therefore, the xu and li method can be disregarded for comparison purposes and, accordingly, our approach, although showing moderate recall and precision values, is currently, along with prospectr, the most accurate predictor of morbid genes on a genome-wide scale.\n",
            "\n",
            "concerning the prediction of druggable genes, as for prediction of morbid genes, we can compare our method only with those developed to predict druggable genes on a genome-wide scale. therefore, to our knowledge, we can compare our methodology with that developed by sugaya and ikeda  <cit> . using support vector machines trained on  <dig> different features covering structural, drug and chemical, and functional information on protein-protein interactions, sugaya and ikeda classifiers achieved an average recall of 75%, an average precision of 70% and an average auc of 72%, performance measures comparable to those obtained by our meta-classifier.\n",
            "\n",
            "prediction of novel morbid and druggable genes\n",
            "since the morbidity and druggability of most of genes in inhgi are unknown—only ≈ 14% and ≈ 3% are known to be morbid and druggable, respectively—we applied our trained meta-classifier to determine the morbidity and druggability statuses of these genes. instead of simply predicting genes as morbid or druggable, we decided to assign a “morbidity score” and a “druggability score”  to each gene since we understand that there is no gene that is absolutely non-morbid or non-druggable. we also assigned to each gene a “shuffled morbidity score” and a “shuffled druggability score” to test the significance of normal scores. for this purpose, we used the wilcoxon signed-rank statistical test as described in “methods”.\n",
            "\n",
            " <dig> of  <dig> scores\n",
            "\n",
            " <dig> according to table of critical values for w in  <cit> \n",
            "\n",
            " <dig> pudmed ids of most recent article clearly stating a gene-disease association\n",
            "\n",
            "* difference statistically significant\n",
            "\n",
            " <dig> of  <dig> scores\n",
            "\n",
            " <dig> according to table of critical values for w in  <cit> \n",
            "\n",
            " <dig> pudmed ids of most recent articles clearly stating that such genes may be drug target candidates\n",
            "\n",
            " <dig> morbid genes according to morbid map  <cit> \n",
            "\n",
            " <dig> genes among those with  <dig> highest morbidity scores \n",
            "\n",
            "* difference statistically significant\n",
            "\n",
            "two potential morbid genes, thbs <dig> and timp <dig>  reinforce the fact that our meta-classifier is able to reveal unexpectedly roles for morbid genes in the modulation of diseases caused by other seemingly unrelated genes. mutations in the gene thbs <dig> have been suggested to play a role in atherosclerosis and thrombosis  <cit> , but its encoded protein may be considered a promising therapeutic target for diabetic nephropathy  <cit> ; alterations in timp <dig> has been demonstrated to be one of the causes of chronic obstructive pulmonary disease  <cit> , but targeting its encoded protein may be a therapeutic intervention against connective amino acid tissue degradation  <cit> .\n",
            "\n",
            "cellular rules for gene morbidity and druggability\n",
            "beyond the prediction capability, machine learning techniques can be used for knowledge acquisition in order to describe patterns in datasets. the machine learning algorithms most used for knowledge acquisition are those that generate decision trees. decision trees are decision support tools inferred from the training data that use a graph of conditions and their possible consequences. the structure of a decision tree consists of a root node representing the most important condition for discriminating classes, internal nodes representing additional conditions for class discrimination under the main condition, and leaf nodes representing the final classification. so, one can learn the conditions for classifying instances in a given class by following the path from the root node to the leaf node  <cit> .\n",
            "\n",
            "therefore, in order to discover the rules for gene morbidity and druggability, we analyzed decision trees generated by training the j <dig> algorithm, a weka’s implementation of the c <dig>  algorithm  <cit>  , on the normal morbidity and druggability datasets containing all network topological features, tissue expression profiles and subcellular localization as training data. the decision trees in figures  <dig> and  <dig> are the best representative tree among the  <dig> generated decision trees for morbidity  and the  <dig> generated decision trees for druggability .\n",
            "\n",
            "from the best representative decision tree for morbidity, we were able to devise some general rules for morbidity in human. as we can observe in figure  <dig>  the root node of decision tree is the number of transcription factors that regulate a given gene . so, this attribute can be considered the most important feature, among those used to train the j <dig> algorithm, for discriminating a morbid from a non-morbid gene. to reinforce this, we found, by walking the path from root node to first leaf node through the right branch, the following rule for morbidity: if genes are regulated by more than one transcription factor, they are likely to be morbid . the study by reverter et al. <cit>  supports this rule as they showed that morbid genes are more likely to show tissue specific expression than non-morbid ones. genes whose expression is tissue specific tend to be regulated by more transcription factors than those that are ubiquitously expressed, e.g. housekeeping genes, since a high level of transcriptional regulation is needed in this case.\n",
            "\n",
            "walking the path from root node to first and second leaf nodes through the left branch , we found the following rule for morbidity: if genes are regulated by one transcription factor and their encoded proteins are either centrally located in metabolic pathways  or play a role in the extracellular region, genes are likely to be morbid. this rule is supported by jimenez-sanchez and colleagues  <cit>  that showed that morbid genes are more likely to be enzymes than non-morbid ones and by winter et al.  <cit>  that demonstrated that ≈ 40% of proteins encoded by morbid genes are predicted to be secreted. furthermore, if proteins are neither centrally located in metabolic pathways nor play a role in the extracellular region but are located in plasma membrane and tend to form clusters with other proteins , their encoding genes are likely to be morbid. for this rule, we could not find any article supporting it. therefore, the plasma membrane localization of proteins encoded by morbid genes as well as the tendency of these proteins to form clusters with other proteins are issues to be examined.\n",
            "\n",
            "from the best representative decision tree for druggability, we were able to devise some general rules for druggability in human. as we can observe in figure  <dig>  the root node of decision tree is the plasma membrane localization of proteins. so, this attribute can be considered the most important feature, among those used to train the j <dig> algorithm, for discriminating a druggable from a non-druggable gene. to reinforce this, we found, by walking the path from root node to first leaf node through the right branch, the following rule for druggability: if proteins are located in plasma membrane, their encoding genes are likely to be druggable . this rule is supported by bakheet and doig  <cit>  that demonstrated that proteins encoded by druggable genes had more transmembrane helices than proteins encoded by non-druggable ones which suggests that proteins encoded by druggable genes are more likely to be found in plasma membrane. walking the path from root node to first and second leaf nodes through the left branch , we found the following rule for druggability: if proteins are not located in plasma membrane but are either centrally located in a transcriptional regulatory circuitry  or are enzymes , their encoding genes are likely to be druggable. this rule is partially supported by bakheet and doig  <cit>  as they showed that druggable proteins are more likely to be enzymes than non-morbid ones. in respect to central position in a transcriptional regulatory circuitry, this is an issue that remains to be elucidated.\n",
            "\n",
            "CONCLUSIONS\n",
            "the identification of morbid and druggable genes has largely been an experimental effort mostly performed by time-consuming experiments. in an effort to accelerate the pace of discovery of such genes, we designed a machine learning-based computational approach that relies on network topological features, tissue expression profile and subcellular localization information for predicting morbid and druggable genes in human on a genome-wide scale.\n",
            "\n",
            "we could demonstrate that our method is able to reliable predict morbid and druggable genes on a genome-wide scale as demonstrated by  the moderate to high performance measures achieved by the meta-classifiers ,  the observation that the designed meta-classifiers learned traits actually related to morbidity and druggability instead of traits associated with any random sets of genes  and  the fact that known morbid and druggable genes tend to have high morbidity and druggability scores, respectively . furthermore, in comparison with other available genome-wide prediction methods, the performance of our method proved to be equal or superior . we could also devise some cellular rules for gene morbidity and druggability using all network topological features, tissue expression profile and subcellular localization information as learning attributes for generation of decision trees . we discovered that number of regulating transcription factors, the central position in metabolic pathways, the localization of their encoded proteins in extracellular region and plasma membrane and tendency to form clusters with other proteins are important factors determining gene morbidity. in respect to druggability, the important factors determining druggability are plasma membrane localization, a central position in a transcriptional regulatory circuitry and being an enzyme. the fact that almost all discovered rules are supported by some additional evidences solidifies decision trees as useful tools for extracting knowledge from complex biological data. albeit the good prediction performance and the ability to discover cellular rules for morbidity and druggability, our approach suffers from three limitations. first, it depends on existing gene ontology annotation and interaction data which are likely to be enriched in small-scale experiments involving morbid and druggable genes. second, the construction of an integrated network of gene interactions requires a large amount of experimental interaction data that are currently available only to a limited number of human genes—our inhgi, for example, covers only ≈ 25% of already identified human genes. third, the lack of negative examples to train the classifier forces us to consider all genes not known to be morbid or druggable as de facto non-morbid and non-druggable genes. we expect, however, that such limitations will be soon addressed as more systems-level data are generated.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "using network topological features, tissue expression profile and subcellular localization data, we show that our meta-classifier can reliably predict morbid and druggable genes on a genome-wide scale and also can define cellular rules governing morbidity and druggability in human.\n",
            "our meta-classifier trained on  <dig> network topological features, tissue expression profile and subcellular localization data seems indeed to be a reliable predictor of morbid and druggable genes on a genome-wide scale.\n",
            "our meta-classifier trained on  <dig> network topological features, tissue expression profile and subcellular localization data seems indeed to be a reliable predictor of morbid and druggable genes on a genome-wide scale.\n",
            "our meta-classifier trained on  <dig> network topological features, tissue expression profile and subcellular localization data seems indeed to be a reliable predictor of morbid and druggable genes on a genome-wide scale and also can define cellular rules governing morbidity and druggability in human.\n",
            "our meta-classifier trained on  <dig> network topological features, tissue expression profile and subcellular localization data seems indeed to be a reliable predictor of morbid and druggable genes on a genome\n",
            "79\n",
            "[{'summary_text': \"we propose to develop pattern recognition algorithms to automatically find and mask out suspected blemishes, and to modify the extant background correction and summarization algorithms to be able to properly handle missing data from blemish removal.\\nwe have applied our method to an experimental dataset and were able to identify blemishes on chips that had passed our gene array resource center's quality control mechanism.\\n\"}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 5037 to 5120 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "analysis of hybridized microarrays starts with scanning the fluorescent image. for high-density oligonucleotide arrays  such as affymetrix genechip® oligonucleotide  arrays, the focus of this paper, each scanned image is stored pixel-by-pixel in a 'dat' file. as the first step in measuring intensity of the hybridization signal, a grid is overlaid, the image is segmented into spots or features, and the pixel intensities within each of these are summarized as a probe intensity estimate . the probe-level intensity estimates are stored in a 'cel' file. each gene is represented by pairs of probes, each representing another characteristic sequences and a 'mismatch', which is identical, except for the watson-crick complement in the center. expression of a gene is estimated from such a probe set by applying algorithms for background correction, normalization, and summarization.\n",
            "\n",
            "the quality of data scanned from a microarray is affected by a plethora of potential confounders, which may act during printing/manufacturing, hybridization, washing, and reading. each chip contains a number of probes specifically designed to assess the overall quality of the biochemistry, such as 'checkerboards' in the corners and borders, whose purpose is, e.g., to indicate problems with the biotinylated b <dig> hybridization. affymetrix software provides for a number of criteria to assess the overall quality of a chip, such as percent present calls, scaling factor, background intensity, and overall pixel-to-pixel variation . software packages such as bioconductor for r  <cit>  have implemented biochemical quality control tools such as rna degradation plots. if a quality problem is found, however, these criteria and tools do not easily suggest a remedy and they have little sensitivity to detect localized artefacts, like a speck of dust or a localized hybridization problem. although such physical blemishes obviously affect the expression estimates, they have hitherto been only narrowly addressed in the literature. thus, there are currently no safeguards to signal potential physical blemishes. instead, researchers are merely advised to carefully inspect the chip images visually  <cit> . given the high variance among the hundreds of thousands of probes and their random allocation on the chip, it is impossible to visually detect any but the starkest artefacts. for two-colour cdna arrays, a bayesian network approach has been proposed  <cit> , based on the 'features' of the pixel distribution within each probe, yet, due to the standardized manufacturing process, the probes on an oligonucleotide array have too few 'features' for such an approach to be effective.\n",
            "\n",
            "as the price of microarrays continues to drop, a typical microarray experiment now contains several chips, each representing a sample obtained under conditions that were similar except for the experimental factor under investigation. having collections of chips available offers new strategies not only for analyzing the effect of the experimental factor, but also for identifying blemishes. the power of having several chips available was first harnessed for estimating mrna expression levels by the 'robust multichip average'  method  <cit> . one of the assumptions underlying the rma model is that probes across chips are highly correlated, due to differences in their affinity  <cit>  and because only a small proportion of genes are differentially expressed in any experimental setting. this correlation should be even higher for the mismatches, because they are less likely to be affected by the specific changes in gene expression induced by the experimental factor. given the volume of pixel level data,  it is desirable to devise algorithms that work from the  <dig> times smaller probe level files, the same information used in traditional signal value estimation approaches.\n",
            "\n",
            "RESULTS\n",
            "data\n",
            "psoriasis is thought to be due to an overly active immune system  <cit> . to study how the immune response of leukocytes isolated from blood can be affected by drugs that may serve to control autoimmune diseases like psoriasis, blood was drawn from five volunteers under a protocol that had been approved by the rockefeller university hospital institutional review board  <cit> .\n",
            "\n",
            "for each subject, peripheral blood mononuclear cells  were isolated and cultured in six petri dishes. four cultures were activated with an anti cd3/cd <dig> antibody, two of which were pre-treated with a repressor drug. two cultures served as control without drug or activation. one of the two sets of control, activated, and pre-treated cultures  was analyzed after  <dig>  the other after  <dig> hrs.  all samples were hybridized to affymetrix huu95av <dig> chips.\n",
            "\n",
            "artefacts identified on probe-level  files\n",
            "similar results were obtained for all subjects . none of the artefacts would have been detected by visual inspection of the pseudo image . even after having seen the filtered image, most blemishes are difficult to identify at best. interestingly, some chips appear to have a preponderance of specific artefacts, suggesting that at least some of the blemishes are caused by specific environmental factors during hybridization, and providing the first indication for the validity of the proposed method. the chip used as the background in figure  <dig> has 'dark clouds' in the upper left corner and, albeit to a lesser degree, in both lower corners. of the two chips with several smaller artefacts, one had three spots that resemble the 'dark spot' in figure  <dig>  only the bright scratch at the bottom of one of the chips could have been detected by mere visual inspection of the chip, although even this chip passed the rockefeller university's gene array resource center's quality control.\n",
            "\n",
            "average vs. median in the filtering procedure\n",
            "the proposed filtering process relies on identifying deviations of a probe on one chip from a measure of central tendency for this probe across chips. thus, if few chips have high intensity 'outliers' for one probe, the chips with normal intensities may appear to be negative 'outliers'. one would expect that the six-chip filter is less likely to generate such 'ghosting' artefacts than the three chip filter. we compared the use of medians vs. arithmetic means as the reference. as we had predicted based on the understanding that errors are more likely to be outliers than white noise, using medians not only resulted in less 'ghosting', but also in fewer isolated cells being considered artefacts and, thereby, better contrast .\n",
            "\n",
            "validation of probe-level artefacts by going back to the pixel-level image\n",
            "our method allows us to identify spatially correlated regions that are unlikely to originate from random fluctuations. to demonstrate that the statistical anomalies detected in the pseudo images at the probe level  are, in fact, physical blemishes, we inspected the corresponding raw image at the pixel level. the regular artefacts seen  are clearly blemishes, even if the precise nature of the physical blemish may not be known. still, the difference in features between blemishes suggests different causes.\n",
            "\n",
            "a number of factors are known to cause bright or dark spots in fluorescence micrographs. dust on the front cover slip will cause a dark, out-of-focus shadow. common white paper is bleached with strongly fluorescent dyes, so fibres from tissue paper ordinarily used for cleaning cause intense glare. many organic solvents, detergents, and other chemicals will fluoresce when concentrated, so leftover droplets or condensates will appear as bright regions, regardless of whether they are in front or behind the focal plane. a crack in the glass would ordinarily be invisible to fluorescence microscopy – except for its ability to accumulate such substances. glass will normally be coated with substances to prevent the direct binding of fluorophores to it; however, any damage to the fragile coating will cause fluorescent streaks. illumination with a coherent source such as a laser, as opposed to a broadband source such as a xenon lamp, has specific artefacts such as speckle. in addition, the arrays themselves are manufactured through photolithographic techniques and may contain occasional damage.\n",
            "\n",
            "dirt\n",
            "the visible bright artefact at the bottom-left of figure  <dig> is the only blemish in our dataset that did not require 'harshlighting' to be visible. the magnification in figure 5a shows a structure in an area of  <dig> ×  <dig> probes. figure 5b shows the corresponding area in the raw image, clearly exhibiting this artefact to be a piece of debris lying in front of the active array surface in the optical path. while the exact physical nature of this debris is unclear, there can be no doubt that probes highlighted at the bottom of figure 3b are, in fact, a blemish.\n",
            "\n",
            "dark and bright spots\n",
            "a very 'dark spot' was seen in the lower left corner of figure 3b. the probe level pseudo image  shows a dark region, but only the raw image reveals the characteristic of this blemish: an elliptical spot with sharp boundaries which pass through the inside of probes. still, the grid is visible underneath, as in one of the examples given by simon, korn, et al.  <cit>  for cdna arrays. the dark probes in figure 6a are therefore likely to be caused by a physical blemish that has 'stained' the image with a dark oval, a mechanical/optical artefact that invalidates the measured intensities of the probes in the region, so all affected probes in the region should be excluded from further analysis. the 'dark spot' in figure  <dig>  also had a well defined border, although with less contrast . three similar artefacts were seen in yet another chip, as shown in the composite picture .\n",
            "\n",
            "the bright spot on the upper right corner figure  <dig> clearly is of different nature. the zoomed area of the dat file of the second chip  of subject  <dig> shown in figure 7b reveals a diffuse area of brightness that covers around  <dig> probes. because this bright cloud is out of focus, it is difficult to assess whether its physical location was in front of or behind the focal plane; it could be a leftover detergent condensate in the plastic back panel of the chip. the artefact is less visible in the pseudo image than in the raw image, because the low granularity of the pseudo image enforces an artificial grid structure. moreover, the affymetrix image analysis algorithm, taking the  <dig> percentile of the pixels as an estimate of the probe, may make it more difficult to detect these artefacts through visual inspection because the brightness in areas with low pixel-to-pixel variation is lowered for all percentiles above the median. although they were easily seen in the filtered pseudo image, neither the 'bright spot' nor the 'dark spot' could have been identified by visual inspection of the original pseudo image. even on the raw image, only an extremely thorough search for areas of low pixel-to-pixel contrast or boundaries with high contrast across probes could have detected these artefacts based on a single chip alone. thus, blemishes involving only  <dig> to  <dig> probes would often be overlooked in a visual inspection of both the raw and the pseudo image. given the high variance across pixels, any image processing algorithm aiming at detecting such blemishes at high sensitivity would also create many false positive results.\n",
            "\n",
            "dark clouds\n",
            "for the 'dark clouds', the raw image at first did not show any recognizable feature. upon closer inspection, however, we noted that the 'dark cloud' in subject  <dig> had higher pixel-to-pixel variance . the noise does not seem to have a physical origin, as the fluctuations appear to be single-pixel in extent, giving the raw image a 'grainy' appearance.\n",
            "\n",
            "the areas outside the dark clouds do not appear to be any grainier, so it does not seem to be a change of exposure setting or other simple global change. the image analysis software reports a single, global pixel-to-pixel variation qraw; it would be useful to have a local quality measure as well, in a fashion similar to the reported background estimate for probe intensities. all dark clouds we found impinge on the array borders. we have no conjecture as to the physical origin of this problem.\n",
            "\n",
            "shadowy circles\n",
            "the two artefacts crossing the left border of figure  <dig> suggest yet another reason for blemishes on microarrays. only one of our chips displayed this artefact, but it did so twice on the same border. neither the raw image nor physical examination of the chip in a dissection microscope provided any hints to the possible cause .\n",
            "\n",
            "there are myriad possible explanations for what caused this striking artefact. a perfectly round structure with outliers concentrated near its perimeter, evocative of the 'coffee stain rings' phenomenon  <cit> , suggests that a bubble  may have formed, during the microfluidic stage, condensation after the washing stage, or as a manufacturing defect.\n",
            "\n",
            "thus, to further elucidate the potential cause of this artefact, we plotted the observed vs. the expected intensity  for each probe in the area depicted above . we then marked the points below the . <dig> percentile of all deviations  in this area, which formed the 'shadowy circle'. these points were seen over a wide range in expected intensity , although their density is higher for lower intensities. notably, their intensity was consistently lower than the expected intensity, as though something had only partially interfered with hybridization – or partially stripped the fluorophores prior to readout, or affected probe sensitivity.\n",
            "\n",
            "relevance\n",
            "to determine the extent to which such artefacts may affect standard analyses, we compared the activated vs. the repressed samples  for patient  <dig>  and studied whether masking the blemishes affects the list of differentially expressed genes.\n",
            "\n",
            "we searched for blemishes all four chips; after manually circling each affected area, we masked  all points in the upper or lower 10th percentile within that area, respectively. we used either the lower or upper 10% since one of our findings is that all artefacts seem to have the common characteristic shown, for instance, in figure  <dig>  that outliers within an artefact are either  exclusively brighter  or darker  than expected we conducted separate analyses for the original and the masked data. we estimated the signal value for each probe using the bioconductor implementation  of the mas <dig> algorithm with default parameters, after modifying the summarization and normalization steps to allow for missing data. the overall effect is shown in figure 10a, with a maximum difference of  <dig>  log <dig> \n",
            "\n",
            "genes whose expression estimates changed by more than  <dig>  log <dig> through filtering were considered as 'altered' by filtering. the 'bright spot', where about  <dig> probes were affected, altered the expression of  <dig> genes by up to  <dig>  log <dig>  the 'shadowy circle' altered the expression of about  <dig> genes; more than  <dig> of them by more than  <dig>  log <dig>  the 'dark spot' affected  <dig> probes, altering expression of  <dig> genes by up to  <dig>  log <dig>  the 'cloud' altered the expression of  <dig> genes,  <dig> of them by more than  <dig>  log <dig>  the dirt covering around  <dig> ×  <dig> probes, affected around  <dig> probes, altering  <dig> genes,  <dig> of them by more than  <dig>  to a maximum of  <dig>  log <dig> \n",
            "\n",
            "finally, we compared the two conditions , mirroring masked probes on both on the affected and the corresponding chip. as an exploratory criterion, we used the modified  t-test suggested in smyth  <cit>  from the limma package of the bioconductor project  <cit> . as shown in figure 10b, the effects of identifying genes as differentially expressed can be dramatic, demonstrating the potential value of detecting blemishes and masking affected areas on microarrays.\n",
            "\n",
            "validity\n",
            "we validate the proposed method using data from the spike-in hug <dig> experiments  <cit> . this data set consists of  <dig> technical replicates of  <dig> separate hybridizations of  <dig> spiked transcripts at concentrations from  <dig>  pm to  <dig> pm arrayed as a latin square. our interest is to assess whether masking the blemishes improves the ability to detect differentially expressed genes. we used the affycomp package of the bioconductor project, which encompass a series of tools developed by  <cit>  to compare the performance of expression measures for affymetrix genechips. figure  <dig> shows that masking blemishes has little effect for large fold changes, as one would expect, while the roc curve  vs.  shows a substantial improvement for small  changes. other statistics are also improved in this case: the average false positive decreases  while the true positives increases . comparing by range of intensities, the area under the curve  is bigger for the masked data in the lower intensities  while keep similar performance in the medium and low range , resulting in a bigger average weighted auc for the filtered data  . thus, our masking procedure improves the sensitivity/specificity to detect small differential expression, especially in the range of low intensities.\n",
            "\n",
            "discussion\n",
            "as an alternative approach to identify blemishes, one might try to look at the residuals from parametric estimations in the background subtraction or summarization stages; e.g., looking at the residuals of the pm-mm difference model  <cit>  or the rma model over the pm values  <cit>  to identify possible aberrations. unfortunately, the variety of models currently being discussed attests to the fact that each model has its drawbacks. while random variation can typically be handled by statistical methods, systematic errors in the choice of the model assumptions may have a drastic impact on these processes. the proposed method is robust in the sense that only few assumptions are made. another advantage of our approach is that we can include mismatch probes which are especially suitable to identify aberrations, because they are less sensitive to gene expression variations.\n",
            "\n",
            "moreover, in any such model of expression estimation the residuals of the entire probe set containing a faulty probe is likely to be affected, so that errors are spread across the probe set and hence over the image; if one probe in a probe set is an outlier, e.g., very bright, all other probes would be slightly dark ghost images, similar to the 'ghosting' seen in figure  <dig>  utilizing topological information for identification and elimination of blemishes has the advantage that suspect probes are identified before background correction, normalization and summarization take place. thus, faulty data will not confound the preprocessing steps and further statistical analysis.\n",
            "\n",
            "with the next generation of affymetrix chips, the relevance of correcting for blemishes will even increase. here, we analyzed u <dig> chips with  <dig> probe pairs per probe set. to make room for more probe sets, the number of pairs per set has been reduced to   <dig> on the u <dig> chips. this, however, not only increases the standard error by 20%, and, thus the effect of any artefacts on the results, but also reduces the ability of model based methods to draw on probe set information. the number of neighbouring cells on a microarray, in contrast, is not adversely affected by reducing the size of the probe sets. in fact, smaller probe sets make it less likely that probe pairs from the same set are in close vicinity.\n",
            "\n",
            "CONCLUSIONS\n",
            "we have presented an extremely simple method for finding blemishes on microarrays. the method's simplicity makes it robust and it does not rely on estimating model parameters. it sensitively tagged blemishes on chips that had passed our gene array resource center's quality control mechanism. only one blemish  could have been readily seen in the raw images. that we found clear evidence of physical blemishes in the raw images for most of the artefacts identified on the pseudo images attests to the validity of the findings.\n",
            "\n",
            "we have applied our method to an experimental dataset and were able to identify anomalies of different type. approximately 25% of our chips are blemished, often more than once, and blemishes can cover areas from a few dozen to hundreds of probes. we examined the potential impact these blemishes have on the experiments. failure to remove the blemishes from further analysis can materially affect the detection of subtle changes in experiments testing similar conditions. when applied to the spike-in data set, the proposed method had an overall better sensitivity/ ratio.\n",
            "\n",
            "for the future we propose to develop pattern recognition algorithms to automatically find and mask out suspected blemishes, and to modify the extant background correction and summarization algorithms to be able to properly handle missing data from blemish removal.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "we propose to develop pattern recognition algorithms to automatically find and mask out suspected blemishes, and to modify the extant background correction and summarization algorithms to be able to properly handle missing data from blemish removal.\n",
            "we have applied our method to an experimental dataset and were able to identify blemishes on chips that had passed our gene array resource center's quality control mechanism.\n",
            "\n",
            "80\n",
            "[{'summary_text': 'we show that our method performs very well on some types of networks in comparison with other methods on a collection of benchmark networks, consisting of both published models and random networks.\\nwe show that our method performs very well on some types of networks in comparison with other methods on a collection of benchmark networks, consisting of both published models and random networks.\\nwe show that our method performs very well on some types of networks in comparison with other methods on a collection of benchmark networks, consisting of both published models and random networks.\\nwe show that our method performs very well on some types of networks in comparison with other methods on a collection of benchmark networks, consisting of both published models and random networks.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3506 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "boolean network  models are widely used in molecular and systems biology to capture coarse-grained dynamics of a variety of regulatory networks, with a particular focus on features such as steady state behavior  <cit> . one advantage of discrete models of this type is that, for small models, the entire dynamics can be explored by exhaustive enumeration of all state transitions. since the size of the state space of a boolean model with n nodes is 2n, this approach becomes unfeasible for larger models, those with more than approximately  <dig> variables, depending on the computational resources available. also, for larger models, finding steady states  through sampling is not effective anymore either, since even large attractors can be missed entirely by this approach. on the theoretical side, it has been shown that the problem of finding, or even counting, steady states of boolean networks is np-hard  <cit> , so that any algorithm for this problem will eventually encounter serious limitations. since the size of published models has increased in recent years, including models with  <dig> or more nodes  <cit> , it is important to develop more efficient methods that find all steady states of a boolean model.\n",
            "\n",
            "several methods have been proposed in the literature for dealing with this problem, including exact as well as heuristic methods. we provide a brief review of the different types here. for this purpose, we represent a boolean network as follows. let k={ <dig> }, and assume that the network has n nodes x <dig> …,xn. each node xi has associated to it a boolean function fi:kn→k. thus, we can represent the boolean network as a function \n",
            "\n",
            " f=:kn→kn. \n",
            "\n",
            " one can represent the variable dependencies through the dependency graph of the network, whose nodes are the variables x <dig> …,xn. there is an edge xi→xj if xi appears in the function fj, that is, the state of xj depends on the state of xi. the problem of finding steady states is then formulated as finding all states x∈kn such that f=x.\n",
            "\n",
            "one approach to the problem is model reduction. some existing reduction methods use a “steady-state approximation”  <cit>  to reduce the number of variables. intuitively, if a function depends on a variable, e.g., fi=fi, then we can remove variable xj from the network by replacing fi with the new function fi,xk,xl). by repeating this process, one obtains a reduced network that in practice is much smaller than the original network. the stopping criteria for reduction methods is that variables can be removed only if the steady state information is preserved. the steady states of the reduced network are in algorithmic one-to-one correspondence with the steady states of the original network. more precisely, the reduction algorithm decomposes a large system into a smaller system and a set of equations in triangular form, so that when the steady states of the reduced system are found, the steady states of the original systems can be found simply by backwards substitution. that is, the existence of the one-to-one correspondence is not just theoretical.\n",
            "\n",
            "another method uses the fact that one can represent a boolean function as a polynomial function in the variables x <dig> …,xn, with coefficients in the finite number system k={ <dig> } . the problem of finding the steady states of a boolean network in n variables, as above, can then be reformulated as the problem of finding the solutions to a system of polynomial equations pi:=fi−xi=0;i= <dig> …,n <cit> . then, the roots of the system of polynomial equations is encoded by the set {p <dig> …,pn}. using tools from computational algebra it is possible to find another set that has the same roots , such that it is possible to do a generalized version of gaussian elimination. these computations can be done using several different software packages developed for this purpose.\n",
            "\n",
            "a graph-theoretic method, minimal feedback vertex sets, consists of finding a set of vertices in the dependency graph of the network that “generate” all steady states. more precisely, one finds a set s⊂{ <dig> …,n} such that knowing the coordinates xi, for all i∈s, of a steady state completely and algorithmically determines the other coordinates of the steady state. it turns out that so-called feedback vertex sets have this property. in practice, by finding a minimal feedback vertex set, one reduces the problem from checking 2n states to the problem of checking 2|s| states, where |s| is typically much smaller than n <cit> . a feedback vertex set can be found by removing vertices from the graph until the graph has no directed cycles. a minimal feedback vertex set can be found by finding the smallest number of vertices that we need to remove from the graph so that it does not have directed cycles.\n",
            "\n",
            "sat methods have also been used for the purpose of finding steady states of boolean networks, which are used to determine whether a boolean expression in several variables has a variable assignment that makes the expression true; see  <cit> . in essence, the system of boolean equations, fi=xi, is rewritten as a single equation g= <dig>  and then the problem of finding the steady states becomes the problem of finding when the equation g= <dig> is satisfied. for example, melkman, tamura, and akutsu  <cit>  used sat algorithms to find steady states of and/or boolean networks, i.e., boolean networks in which the fi contain only the and and or operators, with a time complexity of o . dubrova and teslenko  <cit>  also developed a sat-based algorithm to find all attractors of a boolean network with very good performance characteristics. the methodology was tested on boolean networks with sizes ranging from  <dig> to  <dig>  it was also tested using random networks with up to  <dig> nodes and average in-degree less than  <dig>  for a fixed in-degree of  <dig> the maximum size networks tested have  <dig> nodes.\n",
            "\n",
            "integer programming-based method have also been used to find the steady states of boolean networks, tamura, hayashida, and akutsu  <cit> . in essence, the system of boolean equations is rewritten as a set of inequalities ax≤b,x≥ <dig> and the goal is to maximize a linear function of the form ctx.\n",
            "\n",
            "strategic sampling,  is a recursive search approach to identify all steady states of a random boolean network with maximum in-degree  <dig>  with an average time complexity of o . the idea is that the equations are solved recursively: first one considers the solutions of the equation f1=x <dig>  since the fi’s depend on few variables in practice, one only has to keep track of the variables that appear in f <dig>  then, one finds the solutions of f2=x <dig> that are compatible with the solutions previously found. the process continues until one finds solutions of all equations. in the worst case, however, algorithm complexity can be o  <cit> .\n",
            "\n",
            "finally, the problem of finding attractors has also been studied by using binary decision diagrams   <cit> . the idea is to represent the boolean functions as a directed graph that efficiently encodes the functions by allowing fast evaluation. then, by combining the bdd representation of all the boolean functions, the problem of finding steady states becomes a search problem in the larger bdd. many of these methods were tested on some biologically relevant networks with fewer than  <dig> nodes.\n",
            "\n",
            "in this paper, we present a new method for computing steady states of a boolean network, combining a graph theoretic reduction/transformation method with an approach using computational algebra. we show that the method performs favorably on some types of networks in comparison with other methods on a collection of benchmark networks, consisting of both published models and random networks with certain properties, namely kauffman networks and networks whose in-degree distribution satisfies a power law.\n",
            "\n",
            "methods\n",
            "the method we propose for steady state analysis is a combination of network reduction/transformation and computational algebra . the reduction technique we use is based on results in  <cit> . in  <cit>  it was shown that any boolean network can be “transformed” into an and-not network, namely a network whose boolean functions are all of the form y1∧y2∧…, where yi∈{xi,¬xi}. the and-not network has the property that its steady states are in one-to-one correspondence with the steady states of the original network. furthermore, the one-to-one correspondence between steady states is algorithmic. in  <cit> , the authors proposed a method to reduce an and-not network to another, smaller and-not network in polynomial time, in such a way that the steady states of the original and the reduced network are in one-to-one correspondence, in a constructive way. this reduction algorithm looks for motifs  in the wiring diagram and removes nodes in such motifs; the reduction stops when there are no more motifs to be reduced . once the reduced network is constructed, one can compute its steady states by converting the boolean functions into polynomial functions and then solving a system of polynomial equations, as explained above. the computational algebra technique is based on  <cit> . the idea is that by computing a gröbner basis , it is possible to find the roots of the system of polynomial equations using a generalized version of gaussian elimination.\n",
            "\n",
            "the correspondence between boolean and polynomial functions is accomplished via the “dictionary” x∧y⇔x·y,x∨y⇔x+y+xy,¬x⇔x+ <dig>  the correspondence is unique if we limit the degree with which each variable appears in the polynomial function to  <dig>  since any function kn→k can be represented uniquely as a polynomial function that is square-free, that is, in which every variable appears with exponent  <dig> \n",
            "\n",
            "the algorithm is summarized in the following pseudocode and a more detailed description follows. the source code can be found at github.com/plantsimlab/ adam.   \n",
            "\n",
            "the input of our algorithm is an n−dimensional boolean network f=. in step  <dig>  we use the formulas from  <cit>  to compute an and-not network g=, with m≥n, which has the same number of steady states as f. the idea is to introduce variables to rewrite the boolean operations using only the operators and and not; for example, f1=¬x2∧ can be written as f1=¬x2∧¬x <dig>  where f5=¬x3∧¬x <dig>  furthermore, the steady states of f are given by projecting the steady states of g to their first n coordinates. in step  <dig>  we simply consider the wiring diagram of g, which is a signed directed graph that encodes which variable depends on which others and whether the interactions are activating or inhibiting. in step  <dig>  we use the algorithm from  <cit>  to reduce the wiring diagram of g to another signed directed graph, w. then, in step  <dig>  we construct the and-not network that has w as its wiring diagram, h=; the steady states of g can be computed from the steady states of h by backtracking  <cit> . in step  <dig>  we compute the polynomial representation of h. this is done by replacing ¬xi with 1+xi, and xi∧xj with xixj, as explained earlier. in step  <dig> we solve the system of polynomial equations hi=xi, i= <dig> …,l; this is done using tools from computational algebra as done in  <cit> . the solutions of the system, l″={s1″,…,sr″}, will also be solutions of h. in step  <dig>  we use backtracking to compute the steady states of g, l′={s1′,…,sr′}. and finally, in step  <dig>  we project each sj′ to its first n coordinates and obtain the steady states of f .\n",
            "\n",
            "RESULTS\n",
            "we first tested the software implementation of our algorithm on  <dig> , <dig> boolean networks with  <dig> nodes each, for which we also computed all steady states by a custom-made algorithm based on minimal feedback vertex sets. for each graph we found the minimal number of vertices that had to be removed so that the graph had no directed cycles; call this set s. then, for each element in { <dig> }|s|, the values of the other variables are completely determined. this gave us 2|s| candidates for steady states which we then checked by exhaustive search. in all cases our algorithm computed correctly all steady states. we are therefore confident that our implementation is error-free. this extends to the relevant functionalities of other software packages we used for intermediate computations .\n",
            "\n",
            "then we used over  <dig>  boolean networks to benchmark our method against others. the methods we used for comparison were those with published benchmarks or those for which the code was readily available. as we will see later, for kauffman networks with k= <dig>  the timing of our method grows linearly with the number of nodes; thus, it was not necessary to include in our benchmarks methods that were reported to grow exponentially for such networks . we selected three methods with good computational efficiency for k=2: zanudo and albert  <cit> ; devloo, hansen, and labbé  <cit> ; and tamura, hayashida, and akutsu  <cit> . for the most recent algorithm, zañudo and albert  <cit>  use a method that identifies motifs  that stabilize in one or a small number of states. the steady states from these motifs are used to reduce the network to find the attractors. it is important to mention that this method can find not only the steady states of boolean networks, but also information about all the attractors of the network, which our method is not currently designed to do.\n",
            "\n",
            "we used random biologically meaningful boolean networks  <cit>  and published networks  <cit>  . the results for zañudo and albert and our algorithm were generated by us and the other results are reported from published benchmarks  <cit> . the computations for our algorithm and that of  <cit>  were done on a  <dig>  ghz linux machine. the computations for tamura’s and devloo’s algorithms were done on a linux system with 3ghz and a sun sparc ultra  <dig> machine, respectively, as reported in  <cit> . considering that the different computers described above have processors with similar speed and that the computations were done in a single processor, the use of results from different machines will not affect the main conclusions of our comparison. moreover, some methods did not have reported results for certain network sizes; in that case, we computed an approximate timing using interpolation/extrapolation of the reported values; we linear and exponential fits for the timings that grew linearly and exponentially, respectively.\n",
            "\n",
            "first, we compare the performance of different methods on kauffman networks with connectivity k= <dig> and k= <dig>  for our and the zañudo algorithm, each reported number is the average or standard deviation of  <dig> boolean networks. in table  <dig> we report the timings for kauffman networks with k= <dig>  we can see that the algorithm in  <cit>  performs best, followed by our algorithm. note that all timings grow linearly with the number of nodes. as mentioned in  <cit> , the good results with tamura’s algorithm may be due to the fact that the authors optimized the computations for boolean functions that have  <dig> inputs. the results for kauffman networks with k= <dig> in table  <dig>  however, show that our method performs better by an order of magnitude. these results show that, while our algorithm is not optimized for very low in-degree networks, it is more scalable for networks with higher connectivity.\n",
            "k=2\n",
            "\n",
            "\n",
            "  \n",
            "the best results are in bold. *=interpolated/extrapolated from reported results. nr=not reported.\n",
            "k=3\n",
            "\n",
            "\n",
            " \n",
            "the best results are in bold. *=interpolated/extrapolated from reported results. df=did not finish in a day. nr=not reported.\n",
            "\n",
            "not all molecular networks have properties similar to kauffman networks, but can exhibit power law properties for their degree distribution. thus, we supplemented the results from tables  <dig> and  <dig> with benchmark networks whose connectivity follows a power law distribution  <cit> . we considered power-law networks with average connectivity 〈k〉= <dig> and 〈k〉= <dig>  that is, the average number of edges is the same, but the connectivity distribution is more biologically realistic. there were no bechmarks for these types of networks for tamura’s and devloo’s algorithms, so we only report zañudo’s and our algorithm. in table  <dig>  we see that our algorithm can handle networks with 〈k〉= <dig> with up to  <dig> nodes in under  <dig> seconds on average. it is important to mention that these timings differ considerably from the timings for k= <dig> . table  <dig> shows the results for networks with connectivity 〈k〉= <dig>  not surprisingly, increasing the average connectivity has a dramatic effect on the size of networks that can be studied; for example, the network sizes that can be dealt with in under  <dig> seconds decreases from  <dig> to about  <dig> when we increase 〈k〉 from  <dig> to  <dig>  further increasing the average connectivity will have a much more dramatic effect.\n",
            "〈k〉=2\n",
            "\n",
            "\n",
            " \n",
            "df = did not finish in a day.\n",
            "〈k〉=3\n",
            "\n",
            "\n",
            " \n",
            "df = did not finish in a day.\n",
            "\n",
            "finally, our results on published networks are shown in table  <dig>  sorted by average connectivity. since all models have external parameters corresponding to environmental conditions , we sampled the parameter space and computed the average timing of each algorithm. the numbers we report are the averages of  <dig> simulations for each model. as expected, for all networks with small average connectivity  our algorithm performed very well and finished in less than half a second, consistent with the timings from tables  <dig> and  <dig>  four models have average connectivity greater than  <dig> and our algorithm performed very well on three of them. however, for the largest network , there were parameter sets  which could not be analyzed.\n",
            "\n",
            "n\n",
            "〈k〉\n",
            "df=did not finish in a day. *=49% of simulations reported, 51% of simulations were stopped because they did not finish in a day or had a large memory consumption.\n",
            "\n",
            "the computational complexity of our algorithm depends on the type of networks used as well as the connectivity. the algorithm seems to run in polynomial time for kauffman networks with k= <dig> , but slower for power-law networks with the same connectivity . for other types of networks the complexity is much harder to infer, but table  <dig> suggests that the complexity is exponential. also, the complexity of the mathematical tools we use is not well understood in the context of boolean models. for example, the algebraic step of our algorithm can be doubly exponential, but it has been shown to work much faster in practice and, as our work shows, it runs much faster for sparse boolean models.\n",
            "\n",
            "CONCLUSIONS\n",
            "the capability to analyze the attractors of discrete dynamic models of biological networks is a key technology in any systems biology toolkit that incorporates this popular type of model. this capability needs to include steady state analysis as well as the determination of periodic points of larger periods. and it needs to apply to models that allow an arbitrary  number of states for its variables, such as logical models. in this paper, we have focused on boolean networks as the model type most commonly used currently. and we have focused only on steady state analysis, at the exclusion of periodic limit cycles. as is the case in many situations, algorithms available for this purpose, some of which we used here for comparison, perform well on some types of models and not so well on others. for instance, for kaufmann networks with connectivity  <dig>  the method in  <cit>  outperforms all other methods, including ours. the method in  <cit>  is generally slower than our method in computing steady states, but has the added capability that it also finds limit cycles of larger lengths, which our method is not currently equipped to do.\n",
            "\n",
            "we have used three types of networks for benchmarking: kauffman networks, power law networks, and published networks. kauffman networks are commonly used for this purpose, but they don’t capture all properties of molecular networks, which include a power law distribution of node connectivities. our analysis of published networks shows that some of them have high average connectivity, not generally considered in theoretical studies. these pose serious challenges to computational methods, as we demonstrate. as more large published networks become available, they will represent the most important suite of benchmark models to be used, in our opinion.\n",
            "\n",
            "we believe that this study also holds another important lesson. our method is a combination of two methods, neither one of which performs particularly well when applied on its own . in combination, however, they are quite powerful: model reduction plus polynomial algebra. this might point towards a general strategy for other algorithms of this type. nonetheless, as our calculations show, the challenge of finding steady states is far from solved in general, even for existing published models. thus, much work remains to be done.\n",
            "\n",
            "competing interests\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "authors’ contributions\n",
            "av-c designed and applied the graph reduction methods, and combined them with the computer algebra algorithm. he also generated the suite of benchmark networks used in the study. ba implemented the graph reduction methods. he also surveyed the literature for other available methods and carried out and collected performance data for the other methods used in the study for comparison. fh carried out gröbner basis calculations for a subset of the benchmark networks. rl conceived, planned, and directed the project. all authors contributed to the writing of the manuscript. all authors read and approved the final manuscript.\n",
            "\n",
            "supplementary material\n",
            "additional file 1\n",
            "example and individual performance of network reduction and computational algebra.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 2\n",
            "instructions for usage.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 3\n",
            "source code.\n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements\n",
            "the work of r.l. was supported in part by the grant plantsimlab: a simulation laboratory for plant biology, funded by nsf, award number dbi- <dig>  f.h. did the work at mbi .\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "we show that our method performs very well on some types of networks in comparison with other methods on a collection of benchmark networks, consisting of both published models and random networks.\n",
            "we show that our method performs very well on some types of networks in comparison with other methods on a collection of benchmark networks, consisting of both published models and random networks.\n",
            "we show that our method performs very well on some types of networks in comparison with other methods on a collection of benchmark networks, consisting of both published models and random networks.\n",
            "we show that our method performs very well on some types of networks in comparison with other methods on a collection of benchmark networks, consisting of both published models and random networks.\n",
            "\n",
            "81\n",
            "[{'summary_text': 'the aim of this study was to investigate the efficacy of two skin antiseptics based on 85% ethanol on the skin of the forehead with application times of  <dig>   <dig>  and  <dig> min without any loss of efficacy.\\nthe results obtained with the  <dig> min reference disinfection on the skin of the forehead fulfilled the efficacy requirements for skin antisepsis.\\nthe aim of this study was to investigate the efficacy of two skin antiseptics based on 85% ethanol on the skin of the forehead with application times of  <dig>   <dig>  and  <dig> min without any loss of efficacy.\\nthe application time of  <dig>  min yielded a significantly higher reduction of the resident bacterial flora in comparison to the  <dig> min reference treatment.\\nthe aim of this study was to investigate the efficacy of two skin antiseptics based on 85% ethanol on the skin of the forehead with application times of  <dig>   <dig>  and  <dig> min without any loss of efficacy.\\nthe application time of  <dig>  min yielded a significantly higher reduction of the resident bacterial flora in comparison to the  <'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 5844 to 6144 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "surgical site infections are among the most common infectious complications of treatment in the hospital  <cit> . up to  <dig> %  or  <dig> %  of surgical patients in hospitals develop a surgical site infection during or after hospital stay depending on the type of operation  <cit> . the rate for surgical site infections after specific indicator operations, however, is lower  <cit> . surgical site infections have a huge economic impact because patients remain on average  <dig>  days longer in the hospital and are five times as often re-admitted to the hospital. they are 60% more likely to be treated on an intensive care unit. a surgical site infection costs approximately €  <dig>   <cit> , depending largely on the type of underlying operation  <cit> . the crude odds ratio for hospital deaths associated with surgical site infections is  <dig>   <cit> . the resident skin flora has also been described to cause serious infectious complications  <cit> . in order to reduce the risk for surgical site infections a skin disinfection prior to the penetration of the skin is recommended among many other measures  <cit> .\n",
            "\n",
            "in central europe, preparations based on ethanol or isopropanol are commonly used for skin antisepsis. there is currently no european norm available to determine the efficacy of preparations for skin antisepsis. that is why the efficacy of a preparation is commonly determined according to the test method of the german society for hygiene and microbiology  which was first described in  <dig>  <cit> . it requires the comparison to a reference treatment in a cross-over design  <cit> . on skin with a high density of sebaceous glands the application time in the test method is  <dig> minutes for both the skin antiseptic and the reference alcohol. that is why in clinical practice the recommended application time for a skin antiseptic is usually  <dig> minutes on skin with a high density of sebaceous glands. the efficacy of alcohol-based skin antiseptics applied for shorter application times has first been studied in  <dig> . it was demonstrated with some alcohol-based skin antiseptics that a  <dig> min application is equally effective on the forehead to a  <dig> min application.\n",
            "\n",
            "based on these findings we investigated the efficacy of two new skin antiseptics based on 85% ethanol on the resident flora of the forehead with application times of  <dig>   <dig>  and  <dig> min.\n",
            "\n",
            "methods\n",
            "design and preparation of subjects\n",
            "each experiment was performed in a reference-controlled cross-over design  <cit> . a minimum of  <dig> volunteers was recruited per experiment. only participants with healthy skin on the forehead were selected . subjects were excluded when they took antibiotics or had used a disinfectant or antiseptic solution within the last three days before an experiment. the hair was clipped or tied around so that it would not touch the skin of the forehead during the investigation. the skin of the forehead was randomly divided into five areas of approximately five cm <dig>  one area was chosen to determine the baseline bacterial density. two areas were used for application of the reference alcohol , one for each of the different sampling times. another two test areas were used for application of the skin antiseptic , one for each of the different sampling times. a cotton swab was soaked with the skin antiseptic and swabbed over the marked test field. the procedure was repeated up to five times in order to keep the skin moist with the skin antiseptic for the entire application time. ethical approval for studying the efficacy for skin antisepsis was obtained from the ethics committee of the university hospital kiel, germany. the study was conducted in accordance with the ethical principles that have their origins in the current version of the declaration of helsinki . informed consent was obtained from each participant.\n",
            "\n",
            "products and application\n",
            "the following preparations were used: propan-2-ol  as the reference alcohol, a clear skin antiseptic based on 85%  ethanol , and a coloured skin antiseptic based on 85%  ethanol . the coloured skin antiseptic contains in addition the dyes e  <dig> and e  <dig>  the thickener polyvinyl pyrrolidon. both skin antiseptics were manufactured by bode chemie gmbh & co. kg, hamburg, germany.\n",
            "\n",
            "the clear skin antiseptic has been shown before with an application time of  <dig> to  <dig> min to have a superior efficacy on the bacterial flora of the forehead . in out study it was applied to the skin of the forehead for  <dig> min and application times shorter than  <dig> min in order to identify the shortest application time with an efficacy equivalent to the reference procedure. the coloured skin antiseptic was only investigated with the application time in which the clear skin antiseptic was found to have an efficacy equivalent to the reference procedure in order to demonstrate that the addition of the dyes and the thickener does not impair the efficacy significantly. the reference alcohol was always applied for  <dig> min to the skin of the forehead according to the testing guideline  <cit> .\n",
            "\n",
            "determination of the pre-values and post-values\n",
            "sampling and cultivation were done according to the test method of the german society for hygiene and microbiology  <cit> . each sampling area was marked so that the standard size of  <dig> cm <dig> was clearly visible. a cotton swab was soaked in tryptic soy broth . the sampling area was rigorously rubbed for about  <dig> s as described before  <cit> . care was taken to ensure that the swab was only rubbed within the marked skin area. the swab was transferred into  <dig> ml tsb containing a combination of neutralizing agents for inactivation of residual biocidal activity  <cit> . in laboratory a the following neutralizing agents were used: 3% tween  <dig>   <dig> % lecithin and  <dig> % cysteine. in laboratories b and c the following neutralizing agents were used: 3% tween  <dig>  3% saponine,  <dig> % histidine and  <dig> % cysteine. both combinations of neutralizing agents were found to be valid for neutralization of 85% ethanol . the tube was vortexed for  <dig> s with a high frequency. a serial dilution was done in tsb. from appropriate dilution steps aliquots of  <dig> ml were spread on tryptic soy agar  in duplicate.\n",
            "\n",
            "disinfection phase\n",
            "two marked skin areas on the forehead were treated with the reference alcohol, two other ones with one of the two skin antiseptics. after each type of treatment two samples were taken . the first sample was taken immediately after completion of the application . the second sample was taken  <dig> minutes after beginning of the application . between each product application, a rest period of at least one week elapsed in order to allow the reconstitution of normal skin flora.\n",
            "\n",
            "calculation of bacterial reduction\n",
            "the plates were incubated for a total of  <dig> h at 36°c, and the colony-forming units  from plates were counted. for calculation purposes, plate count values ≤  <dig> cfu were accepted. plate count values of  <dig> were reset to  <dig>  because the log <dig> of  <dig> is undefined, and the log <dig> of  <dig> =  <dig>  the weighted mean of cfu was calculated taking into account the number of cfu per plate and the corresponding dilution step. the weighted mean was multiplied by the dilution factor in order to obtain the number of cfu per ml in the sampling liquid. all pre- and post-values were expressed as log <dig> values. for each sample from each volunteer, the logarithmic reduction factor  was calculated as the difference between the log <dig> baseline value and the log <dig> post-values.\n",
            "\n",
            "statistics\n",
            "a product is considered effective for skin antisepsis if the mean rf at both sampling times is not significantly lower than the corresponding mean rf of the  <dig> min reference treatment . differences of all other means were investigated by the two sided wilcoxon matched-pairs signed-ranks test  <cit> . a p-value <  <dig>  was chosen to indicate a significant difference.\n",
            "\n",
            "RESULTS\n",
            "application time of  <dig> min\n",
            "two experiments were performed. the clear skin antiseptic a reduced the bacterial density on the forehead by  <dig>  to  <dig>  log10-steps  and  <dig>  to  <dig>  log10-steps . the skin antiseptic was either equally effective or more effective in comparison to the reference procedure . overall, application of the clear skin antiseptic a for  <dig> min yielded a significantly higher reduction of the resident bacterial flora in comparison to the  <dig> min reference procedure .\n",
            "\n",
            "application time of  <dig>  min\n",
            "three experiments were performed with the clear skin antiseptic a . in study facility a, the clear skin antiseptic a was significantly more effective at both sampling points . in study facility b, the clear skin antiseptic a was significantly more effective  and equally effective  <dig> min after beginning of the application . in study facility c, the clear skin antiseptic a was equally effective ; both differences were not significant. overall, application of the clear skin antiseptic a lead to a mean log10-reduction of  <dig>   which is significantly higher compared to the  <dig> min reference disinfection .  <dig> min after beginning of the application the clear skin antiseptic a was equally effective to the reference treatment .\n",
            "\n",
            "the coloured skin antiseptic b was studied in three experiments . in study facility a application of the coloured skin antiseptic b was significantly more effective than the  <dig> min reference procedure both  <dig>  min after beginning of the application  and after  <dig> min . in study facility b it was also significantly more effective both  <dig>  min after beginning of the application  and  <dig> min after beginning of the application . in study facility c it was equally effective  <dig>  min after beginning of the application  and  <dig> min after beginning of the application . overall, application of the coloured skin antiseptic b for  <dig>  min yielded a significantly higher reduction of the resident bacterial flora in comparison to the  <dig> min reference procedure .\n",
            "\n",
            "application time of  <dig> min\n",
            "two experiments were performed with the clear skin antiseptic a . in study facility a, the clear skin antiseptic a was significantly more effective at both sampling points . in study facility b, the clear skin antiseptic a was equally effective  <dig> min after beginning of the application  and significantly less effective  <dig> min after beginning of the application . overall, application of the clear skin antiseptic a lead to a mean log10-reduction of  <dig>   which is significantly higher compared to the  <dig> min reference disinfection .  <dig> min after beginning of the application the clear skin antiseptic a was equally effective to the reference treatment .\n",
            "\n",
            "discussion\n",
            "for the first time we were able to demonstrate that the efficacy of an ethanol-based skin antiseptic is achieved on the forehead within  <dig>  min and does not require a  <dig> min application time. this could be shown for a clear and a coloured skin antiseptic both based on 85%  ethanol. extending the application time with any of the two skin antiseptics beyond  <dig>  min has, based on our data, no additional effect on the bacterial density and can therefore be regarded as unnecessary. shorter application times for skin antiseptics have been recommended as long as there is sound evidence that the efficacy is equal to the reference procedure and the skin antiseptic is approved as a medicinal product  <cit> .\n",
            "\n",
            "the results obtained with the  <dig> min reference disinfection on the skin of the forehead indicate some outcome variability among the ten experiments. the lowest mean log <dig> reduction obtained with the reference alcohol, measured as the immediate effect, was  <dig>  ±  <dig> , the highest mean was  <dig>  ±  <dig> . similar results for reference alcohol have been described before  <cit> . a certain outcome variability has to be expected when the effect of antiseptics is measured on the resident bacterial skin flora without an artificial contamination. a similar test principle can be found in surgical hand disinfection when the efficacy of a hand rub is also measured against the resident hand flora. the variability of the outcome obtained with a surgical hand disinfection reference procedure is also relatively high  <cit> . when hands, however, are artificially contaminated  the variability of the reference procedure efficacy is remarkably lower which can be explained by the standardized size of the high inoculum  <cit> . that is why it is essential to study the efficacy of antiseptics in a cross-over design in order to better come around the inevitable outcome variability of the reference procedure.\n",
            "\n",
            "the bacterial biota in normal superficial skin are highly diverse  <cit> . especially the skin on the forehead is known to have a stable and dense bacterial population which is rather difficult to reduce  <cit> . this is most likely explained by the high number of sebaceous glands excreting lipids  <cit> . alcohol, however, has been described to penetrate into the deeper layers of the stratum corneum where it exhibits an effect which is similar to the one on the surface of the stratum corneum  <cit> . other investigators have also studied the efficacy of skin antiseptics on the resident bacterial flora of the forehead and found a reduction of the resident skin flora in the same range  <cit> . the skin of the forehead is known to be an area with a higher density of resident skin bacteria compared to other areas of human skin  <cit> . similar mean reductions with various types of skin antiseptics were found on abdominal skin  <cit> . it is known that alcohol does not completely eliminate the resident flora on the forehead  <cit> .\n",
            "\n",
            "most skin antiseptics in central europe are currently recommended with a  <dig> min application time on skin with a high density of sebaceous glands  <cit> . the reason for the  <dig> min application time is that they are usually tested for efficacy according to the method of the dghm which only allows a  <dig> min application time  <cit> . to our knowledge this is one of the first investigations with skin antiseptics to identify a shorter but equally effective application time for specific formulations. recent changes in surgical hand disinfection indicate that there is currently a trend to identify the application time which is truly necessary for an antiseptic preparation to fulfill the defined efficacy requirements  <cit> . this development in surgical hand disinfection has also been recognized and accepted by the relevant infection control societies in germany  <cit>  and is in principal considered to be an important step for evidence-based recommendations in infection control  <cit> .\n",
            "\n",
            "the clinical advantage is substantial if the application time on skin with a high density of sebaceous glands is reduced from  <dig> to  <dig>  min without any loss of efficacy. but this will only be possible for preparations with sound evidence that a short application time yields the same efficacy as a long application time. a short application time will allow to perform an intervention with less delay not only in neurosurgery or plastic facial surgery but especially in skin antisepsis prior to lumbar punctures, epidural or peridural anaesthesia which may be particularly valuable in emergency procedures.\n",
            "\n",
            "CONCLUSIONS\n",
            "the clear and coloured skin antiseptics which were tested in our study and which were applied for  <dig>  min on the skin of the forehead fulfilled the efficacy requirements for skin antisepsis. the shorter application time on skin with a high density of sebaceous glands will allow to act more efficiently in clinical practice.\n",
            "\n",
            "competing interests\n",
            "the first author is paid employee of bode chemie gmbh & co. kg, hamburg, germany.\n",
            "\n",
            "authors' contributions\n",
            "gk designed the study, all authors analysed the data, wrote the manuscript, read and approved the final manuscript.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the aim of this study was to investigate the efficacy of two skin antiseptics based on 85% ethanol on the skin of the forehead with application times of  <dig>   <dig>  and  <dig> min without any loss of efficacy.\n",
            "the results obtained with the  <dig> min reference disinfection on the skin of the forehead fulfilled the efficacy requirements for skin antisepsis.\n",
            "the aim of this study was to investigate the efficacy of two skin antiseptics based on 85% ethanol on the skin of the forehead with application times of  <dig>   <dig>  and  <dig> min without any loss of efficacy.\n",
            "the application time of  <dig>  min yielded a significantly higher reduction of the resident bacterial flora in comparison to the  <dig> min reference treatment.\n",
            "the aim of this study was to investigate the efficacy of two skin antiseptics based on 85% ethanol on the skin of the forehead with application times of  <dig>   <dig>  and  <dig> min without any loss of efficacy.\n",
            "the application time of  <dig>  min yielded a significantly higher reduction of the resident bacterial flora in comparison to the  <\n",
            "82\n",
            "[{'summary_text': 'the hub classifier has improved accuracy for hub prediction relative to other traditional approaches for protein interaction prediction, with potential applications in optimizing protein pull-down experiments and identifying new drug targets against pathogens.\\nthe hub classifier has improved accuracy for hub prediction relative to other traditional approaches for protein interaction prediction.\\nthe hub classifier was trained on the existing go and protein-protein interaction data for escherichia coli, saccharomyces cerevisiae, drosophila melanogaster and homo sapiens species.\\nit is anticipated that the hub classifier can serve as a useful tool for selecting baits and prioritizing proteins for protein interaction experiments, with potential applications in optimizing protein pull-down experiments and identifying new drug targets against pathogens.\\nthe hub classifier has improved accuracy for hub prediction relative to other traditional approaches for protein interaction prediction.\\nit is anticipated that the hub classifier can serve as a useful tool for selecting baits and prioritizing proteins for protein interaction experiments, with potential applications in optimizing protein pull-down experiments and identifying new drug targets against pathogens.\\nthe hub classifier has improved accuracy for hub prediction relative to other traditional approaches for protein interaction prediction, with potential applications in optimizing protein pull-down experiments and identifying new drug targets'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 5697 to 6144 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "a broad range of cellular functions are mediated through complex protein-protein interactions, which are commonly visualized as two-dimensional networks connecting thousands of proteins by their physical interactions. such a network perspective suggests that cellular effects and functions of proteins can only be fully understood in context with their interacting partners in a protein interaction network .\n",
            "\n",
            "the study of pins has been made possible through recent advancements in high-throughput proteomics that have detected protein-protein interactions on a genome-wide scale and have generated large amounts of interaction data for several species including saccharomyces cerevisiae  <cit> , escherichia coli  <cit> , drosophila melanogaster  <cit> , caenorhabditis elegans  <cit> , and homo sapiens  <cit> . the corresponding protein interaction networks have been made publicly accessible through open access databases such as intact  <cit>  and dip  <cit> .\n",
            "\n",
            "the accumulated protein interaction data have further supported recent protein network analyses that demonstrated the scale-free organization of pins, where the majority of proteins have a low number of interactions in the network, with a few highly-connected proteins  having a significant number of interacting partners  <cit> . such inhomogeneous network topology allows a pin to be robust against random removal of protein nodes, but vulnerable to targeted removal of network hubs  <cit> . in addition, previous studies have shown defined relationships between the degree of connectivity of proteins in pins, their sequence conservation, and cellular essentiality properties  <cit> . those studies indicated that highly-connected proteins  represent very attractive subjects for understanding cellular functions, identifying novel drug targets, and for use in the rational design of large-scale pull-down experiments.\n",
            "\n",
            "although large-scale pins have already been experimentally determined for several species , in general, protein interaction data are still lacking for many organisms. thus, several computational approaches have been developed to predict protein-protein interactions utilizing existing bioinformatics data such as gene proximity information  <cit> , gene fusion events  <cit> , gene co-expression data  <cit> , phylogenetic profiling  <cit> , orthologous protein interactions  <cit>  and identification of interacting protein domains  <cit> . several bioinformatics approaches have also been developed to identify hypothetical interactions between proteins based on their three-dimensional structures  <cit>  or by applying text-mining techniques  <cit> . traditionally, such computational predictions have focused on the identification of pairwise protein-protein interactions with varying degrees of accuracy  <cit> ; however, none of them have been explicitly focused on predicting hypothetical hub proteins.\n",
            "\n",
            "at the same time, it is reasonable to hypothesize that hub proteins should share certain common sequence or structural features that not only enable them to participate in multitudes of protein interactions, but also can be utilized for the theoretical identification of such hub proteins without prior knowledge of the corresponding pins. therefore, the goal of this study is to develop such a 'hub predictor' , capitalizing on experimental and bioinformatics data available to date for proteins in several model organisms with already-determined pins.\n",
            "\n",
            "we have focused the construction of the hub classifier on gene ontology  data, which provide functional annotations for individual proteins using an expert knowledge base  <cit> . the advantage of applying go annotation to hub prediction lies in the readily available information for proteins in hundreds of species. importantly, the go annotations have been shown to reflect certain properties that can mediate protein-protein interactions  <cit> , but the annotation itself does not rely on the availability of corresponding experimental data. thus, the go-based hub classifier should be suitable for predicting highly-connected proteins, even in organisms that lack protein interaction data.\n",
            "\n",
            "here we present the development of such a hub protein classifier, trained on the existing go and protein-protein interaction data for escherichia coli, saccharomyces cerevisiae, drosophila melanogaster and homo sapiens species. the generated models were cross-validated and tested on two external protein interaction data sets: methicillin-resistant staphylococcus aureus   <dig> and caenorhabditis elegans. the developed bioinformatics approach has not only demonstrated an improved accuracy in identifying highly-connected pin nodes , but has also shown an improved speed and a lower demand on computational resources.\n",
            "\n",
            "to illustrate a possible application of the developed tool, we have used it for rationalizing a bait selection strategy for a large-scale protein complex pull-down experiment.\n",
            "\n",
            "methods\n",
            "data acquisition\n",
            "protein-protein interaction data\n",
            "protein interaction data used for the training and testing of the hub protein classifier were obtained from the intact database  <cit>  for the following species: escherichia coli k  <dig> , saccharomyces cerevisiae , drosophila melanogaster , and homo sapiens  . two external validation data sets were collected for protein interactions in mrsa <dig>  and caenorhabditis elegans . table  <dig> lists the total number of proteins and their interactions of the four species in the training and testing, which have been combined into a single data set for the subsequent analyses. similar information on the external validation sets is shown in table  <dig> \n",
            "\n",
            "the top table lists the protein interactions and hubs in each of the four species, and the bottom part of the table lists the number of unique go terms for each annotation category.\n",
            "\n",
            "the top table lists the protein interactions and hubs in each of the two species, and the bottom part of the table lists the number of unique go terms for each annotation category.\n",
            "\n",
            "hub proteins were identified based on their numbers of protein interactions and their percentile ranking relative to other proteins in the same species. proteins of the same species were divided into different percentile groups, sorted by the number of protein-protein interactions in a decreasing order . it is clear that hub proteins have more interactions than non-hubs, but currently there is no consensus on exactly how many interactions a hub protein should have. often, hubs are defined arbitrarily to have at least certain number of interactions  <cit> . in our study, the hub selection criterion was based on the position of a sharp turn  on an accumulative protein interaction distribution plot from each of the four species. as shown in figure  <dig>  the protein interactions followed a power law distribution, such that a sharp turn is visible around the 90th protein percentile position on the interaction plots.\n",
            "\n",
            "to achieve a consistent hub definition across the four studied species, hub proteins were defined as above or equal to the 90th percentiles of interactors; in other words, the hubs represented the top  <dig> percent of highly-connected interactors, and the non-hubs were consisted of the bottom  <dig> percent of the proteins. using this definition, hub proteins were determined from each of the four pins individually. at the 90th protein percentile, e. coli hubs have at least  <dig> protein interactions, s. cerevisiae hubs have at least  <dig> protein interactions, d. melanogaster hubs have at least  <dig> protein interactions, and h. sapiens hubs have at least  <dig> interactions. the number of assigned hub and non-hub classifications is shown in table  <dig> \n",
            "\n",
            "gene ontology  data\n",
            "each protein obtained from the intact database was identified by a unique uniprot accession number, which enabled a fast collection of go annotation data from the uniprot retrieval system  <cit>  . the complete uniprot protein annotation pages were downloaded as flat texts, which were then parsed by perl scripts to extract the go annotations in the three categories: biological process, molecular function, and cellular component. because each go term could be assigned to a different level of the annotation hierarchy, we established a fixed general go level that represented all of the specific go terms of the proteins in the study. this general go annotation level was determined based on the go slim project, which provides a list of generic go terms on which many bioinformatics analyses can be performed  <cit> . importantly, the go slim generic terms provided a reasonable number of protein 'predictors' for a machine learning method to effectively operate. the tool 'map2slim'  <cit>  was used to map specific go terms to the 'go slim' generic terms . this generic version of go slim contained  <dig>  terms,  <dig>  terms and  <dig>  terms.\n",
            "\n",
            "all protein interaction data and go annotations were stored in a local mysql database for fast data searching and reporting.\n",
            "\n",
            "hub protein classification by boosting trees\n",
            "to train models that classify a protein as a hub or a non-hub, the protein interaction data from the four species were combined into a single data set . a four-fold cross-validation strategy was used in which four non-overlapping testing sets , and four training sets  were utilized for building the hub classifiers. each training and testing set maintained the same hub to non-hub  ratio. in addition, the proteins in the training sets have maintained the same distribution of go annotation terms as the proteins in the testing sets. figure  <dig> illustrates the distribution of each of the  <dig> go terms, represented by the percentage of proteins with this term in the training sets vs. the testing sets of the four cross-validation samples. a high correlation r <dig> values of  <dig>  ~ <dig>  indicated an equal go distribution between the training and testing sets. it is also shown that the majority of the go terms were associated with less than 10% of the proteins in a given data set.\n",
            "\n",
            "we focused the machine-learning effort on hub classification by applying boosting trees, which is one of the best methods for classifying complex data and providing interpretable results  <cit> . the training and testing of the hub-predicting classification trees were performed on  <dig> go terms as predictor variables by using the boosting tree application as implemented in statistica version  <dig>  <cit> . the input data were formatted as tables of binary data, where each column represented a go term variable  and each row represented a sample protein.\n",
            "\n",
            "four classifiers were built  and compiled in the c++ language under linux. in addition to the four testing sets in the cross-validation study, the best of the four hub classifiers has been validated on two external data sets, which were consisted of experimentally-determined pins in mrsa <dig> and c. elegans. the classifier predicted each protein in the data sets as either a hub or a non-hub, and the classification statistics were calculated as the following:\n",
            "\n",
            " sensitivity = tp/ \n",
            "\n",
            " specificity = tn/ \n",
            "\n",
            " accuracy = / \n",
            "\n",
            " ppv  = tp/ \n",
            "\n",
            " npv  = tn/ \n",
            "\n",
            ", where tp = true positive, fp = false positive, tn = true negative, and fn = false negative.\n",
            "\n",
            "a useful output feature of the boosting tree method is the relative predictor importance, which measures the average influence of a predictor variable on the prediction outcome over all of the trees  <cit> . the most important predictor is assigned a value of  <dig>  and the other variables are scaled accordingly.\n",
            "\n",
            "comparison of the hub classifiers with other existing protein interaction prediction approaches\n",
            "to further assess the performance of the hub classifier against other existing approaches for predicting hub proteins, we applied three different types of bioinformatics methods to construct hypothetical pins in mrsa <dig>  where hub proteins were determined by the number of predicted pairwise protein-protein interactions.\n",
            "\n",
            "hypothetical pin – pathway maps\n",
            "the first type of hypothetical pin represented the known protein-protein interactions available for mrsa <dig>  a total of  <dig> protein interactions were manually extracted from the pathway maps in the kegg database  <cit>  .\n",
            "\n",
            "hypothetical pin – orthologous interactions\n",
            "the second type of pin was constructed based on known protein-protein interactions between orthologs from three other species: helicobacter pylori, saccharomyces cerevisiae, and escherichia coli. the experimental pin in h. pylori was obtained from the bind database  <cit>  . two sources were used to build the s. cerevisiae pin: the bind database  and gavin's study  <cit>  . we extracted the e. coli pin in butland's study  <cit>  from the intact database  <cit>  .\n",
            "\n",
            " <dig> protein sequences in mrsa <dig> were obtained from the refseq databases at ncbi  <cit>  . the orthologs of the interacting proteins from each of the above species were identified in mrsa <dig> by using the program inparanoid  <cit>  . if a pair of mrsa <dig> proteins whose orthologs interacted in one of the three species, the pair was assigned as an interacting protein pair. a total of  <dig> protein interactions were predicted for this type of mrsa <dig> pin reconstruction.\n",
            "\n",
            "hypothetical pin – interacting domains\n",
            "the third type of mrsa pin was predicted based on protein domain-domain interactions. first, the presence of pfam domains  <cit>  in each of the  <dig> mrsa <dig> proteins was determined by scanning the pfam domain profiles  with the program hmmer  <cit>  . second, domain-domain interaction data were acquired from two sources: interdom  <cit>   and ipfam  <cit>  . if a pair of mrsa <dig> proteins contained interacting domains according to one of the two sources, the pair was assigned as an interacting protein pair. a total of  <dig>  protein interactions were predicted based by this method.\n",
            "\n",
            "validating the prediction on an experimental mrsa <dig> pin\n",
            "the experimental mrsa <dig> pin provided by the prepare project contained interaction data for  <dig> proteins and was used as the external validation set for measuring the prediction performance of the hub classifier and the different types of hypothetical pins.\n",
            "\n",
            "we have compared the prediction results in two different ways. in the first type of comparison, both the hub classifier and the combined hypothetical pins classified the  <dig> mrsa proteins as hubs or non-hubs, while the same  <dig> proteins were also classified as hubs or non-hubs based on the experimental results provided by prepare. in the case of the hub classifier, hubs and non-hubs were reported explicitly from the prediction program. in the cases of hypothetical and experimental pins, hubs were defined as above or equal to the 90th percentile of proteins ranked by the number of interactions . the following classification statistics were calculated: sensitivity, specificity, accuracy, ppv and npv.\n",
            "\n",
            "in the second type of comparison, we compared ranked lists of proteins based on their 'hub-likeness' property. in the case of the hub classifier, the proteins were ranked based on the differences between predicted hub probabilities and non-hub probabilities as computed by the boosting tree method. in the case of the hypothetical and experimental pins, the proteins were ranked by their numbers of protein interactions. the ranked lists were compared to the list of proteins ranked by the number of experimental interactions in mrsa <dig> by using a spearman rank order correlation as implemented in statistica  <dig> \n",
            "\n",
            "validating the prediction on an experimental c. elegans pin\n",
            "in addition to mrsa <dig>  we have tested the hub protein classifier on an external set of protein interaction data in c. elegans. the same procedure was applied to determine hub prediction statistics, as described above.\n",
            "\n",
            "test of significance\n",
            "to test the hub protein classifier against a null hypothesis, which claims there is no difference of go term distribution between hubs and non-hubs, we have randomized the protein interaction data in the following ways. firstly, the same  <dig> proteins in the testing set  for the hub classifier were used in the construction of a randomized data set. secondly, 10% of those proteins were randomly assigned as hubs, while the other 90% of proteins were randomly assigned as non-hubs. thirdly, the go terms originally associated with those proteins were randomly distributed within the data set. the combination of the above two randomization methods ensured that there was no significant difference in go term distribution between the hub and non-hub proteins. finally, the hub classifier was used to predict hubs and non-hubs in the randomized data set, and prediction statistics were obtained.\n",
            "\n",
            "simulation of protein bait selections and network coverage\n",
            "the effectiveness of protein bait selections assisted by the hub classifier has been simulated by using yeast protein-protein interaction data determined by protein-complex pull-down and mass spectrometry experiments, available from gavin's study  <cit> . one major goal of such large-scale experiments is to maximize the number of protein interactions identified by using a small set of proteins as 'baits' to pull down their interactors . therefore, it is crucial to select protein baits based on properties that will produce the best network coverage, as measured by the ratio between the number of protein interactions identified by an experiment and the total number of interactions in an organism.\n",
            "\n",
            "in our simulation experiments,  <dig>  interactions, involving  <dig> proteins from gavin's yeast data set , were hypothetically treated as the total number of protein interactions in saccharomyces cerevisiae. to simulate the bait selection process, we selected a subset of proteins  as baits and calculated the number of interactions such baits would \"pull-out\" from the yeast interaction data set and computed the overall network coverage. two selection criteria were used. in one simulation, the baits were randomly selected from the total pool of the yeast proteins. in the other simulation, the baits were selected from the pool of hub proteins predicted by the hub classifier.\n",
            "\n",
            "in addition to the bait selection strategy described above , we simulated the network coverage results by applying a second round of selections. in this type of selection, baits were divided into two sets: one-third as the first round of baits, and two-thirds as the second round of baits. the first-round baits were chosen by either random selection or by hub prediction. the second round of baits was selected from the most abundant preys pulled down by the first round of baits. such an approach is also referred to as the \"name your friend\" method and has been applied to maximize the effectiveness in vaccinations against infectious diseases  <cit> , as well as in some protein complex experiments  <cit> .\n",
            "\n",
            "RESULTS\n",
            "prediction performance of the hub prediction classifier\n",
            "one prediction model was constructed for each of the four cross-validation samples; therefore, a total of four hub classifiers were generated. the executable files of the classifiers were complied by the gnu c++ compiler in linux. the classifier programs used a list of query proteins and their corresponding go term occurrences as the input file, and produced the same list of the proteins with hub prediction results and probability scores. the running time was only a few seconds for predicting hubs from over  <dig>  proteins on a  <dig>  ghz pentium d personal computer.\n",
            "\n",
            "overall, the classification statistics were consistent between the training and testing sets for the four classifiers. within the training sets, the sensitivity of the classifiers ranged from  <dig> % ~ <dig> %, the specificity ranged from  <dig> % ~ <dig> %, and the accuracy ranged from  <dig> % ~ <dig> %; ppv  varied from  <dig> % ~ <dig> %, and npv  varied from  <dig> % ~ <dig> %. within the testing sets, the sensitivity ranged from  <dig> % ~ <dig> %, the specificity ranged from  <dig> % ~ <dig> %, and the accuracy ranged from  <dig> % ~ <dig> %; ppv varied from  <dig> % ~ <dig> % and npv varied from  <dig> % ~ <dig> %. the classification statistics on the best of the four hub classifiers is shown in table  <dig> \n",
            "\n",
            "the observed vs. predicted hubs and non-hubs and their corresponding classification statistics are shown for the best classifier based on the training, testing and all  data sets\n",
            "\n",
            "we have further validated the prediction accuracy of the best hub classifier in the external mrsa <dig> data set. as indicated in table  <dig>  in comparison to the other protein prediction methods, the hub classifier has the highest prediction statistics, with  <dig> % sensitivity,  <dig> % specificity,  <dig> % accuracy,  <dig> % ppv and  <dig> % npv. the next best hub prediction result was achieved by the hypothetical mrsa pin based on orthologous interactions. on the other hand, the results from the predicted pins of pathway maps and interacting domains were poor as none of them had any true positives.\n",
            "\n",
            "the prediction performance of the hub classifier is compared to that of the hypothetical pins in mrsa <dig>  the classification statistics is reported.\n",
            "\n",
            "in the other comparison, we correlated a ranked list of proteins based from their 'hub-likeness'  to that of the experimental mrsa pin. as shown in table  <dig>  the hub classifier had a correlation coefficient of  <dig>  – highest among all other methods. the next best correlation was achieved by the hypothetical pin of orthologous interactions.\n",
            "\n",
            "the ranked protein lists based on hub-likeness properties, produced by either the classifier or the hypothetical pins, has been compared to that of the experimental pin in mrsa <dig>  the coefficient of spearman rank order correlation is reported with p-value <  <dig> .\n",
            "\n",
            "in addition to mrsa <dig>  the hub protein classifier has achieved comparable prediction results in the c. elegans validation data set, with  <dig> % sensitivity,  <dig> % specificity,  <dig> % accuracy,  <dig> % ppv and  <dig> % npv, as shown in table  <dig> \n",
            "\n",
            "the prediction performance of the hub classifier was validated, based on the experimental pin in c. elegans.\n",
            "\n",
            "the prediction statistics of the hub classifier on the randomized data set are summarized in table  <dig>  the result shows that the hub classifier was not able to achieve a significant hub prediction when the go terms and protein hubs were randomly assigned. the prediction only reached  <dig> % sensitivity and  <dig> % ppv in the randomized set, compared to  <dig> % sensitivity and  <dig> % ppv in the testing set before the randomizations. the specificity and npv were comparable before and after the randomizations, due to the inherited 1: <dig> ratio between the number of hubs and non-hubs. therefore, it is easier to make a correct prediction on non-hub proteins than hub proteins. the comparison of the prediction results between the testing set and the randomized set indicates that hub proteins have a distinct distribution of go terms, which contributed to the predictability of the hub classifier.\n",
            "\n",
            "the prediction performance of the hub classifier was tested on the null hypothesis that there is no difference of go term distribution between hubs and non-hubs.\n",
            "\n",
            "overall, the hub classifier built on the gene ontology annotations achieved high specificity and npv, but had lower than expected sensitivity and ppv. we attribute this to the lack of go annotations for certain proteins in the training sets, as the level of annotations varied among the four species. for instance, s. cerevisiae had the highest percentage of the proteins with go annotations , while only  <dig> % of the proteins in e. coli had any go annotation. therefore, the performance of the current hub classifier primarily relied on the number of go annotations available for each species. we expect the sensitivity value of the hub classifier to be improved when more annotation data become available for the four species in the training sets.\n",
            "\n",
            "go term predictor importance\n",
            "an indicator of the contribution of each go term used in the boosted trees classifiers was provided by the relative importance of predictors in the training output. the importance value ranged from  <dig> to  <dig>  where  <dig> indicated that a predictor had the most influence on the hub prediction outcome, and  <dig> meant a predictor had the least influence. the top  <dig> go annotation terms that were likely to be shared among hub proteins are listed in table  <dig> \n",
            "\n",
            "the top go terms included several annotations such as 'rna binding', 'translation', and 'ribosome', commonly used to annotate ribosomal proteins, which were often identified as the top interacting proteins in other experiments  <cit> . the list of important predictors indicated that hub proteins tend to participate in several common cellular processes, including translation, nucleotide metabolism, organelle biogenesis, cell cycle, signal transduction, cell death, and electron transport.\n",
            "\n",
            "applying hub classifier to protein bait selection\n",
            "the bait selection strategy, assisted by the hub classifier, was simulated in the experimental pin of saccharomyces cerevisiae. in the case of one-round selection, choosing baits that were predicted as hubs by the classifier has greatly increased the network coverage in comparison to random selection. for instance, as illustrated in figure  <dig>  when 15% of total proteins were selected as baits based on the result of the hub classifier,  <dig> % of the network coverage was achieved. on the other hand, only  <dig> % of the network coverage was generated by the random bait selection.\n",
            "\n",
            "in the case of the two-round selection, the network coverage produced by either random or hub bait selection has shown a great improvement from the one-round selection. the hub bait selection performed slightly better than random in the two-round selection.\n",
            "\n",
            "the results suggest that the hub classifier is a useful tool for selecting baits and prioritizing proteins for protein interaction experiments. although it was not explored in the present study, we expect that the hub classifier can also assist in the identification of highly-interacting proteins in pathogens as potential drug targets.\n",
            "\n",
            "CONCLUSIONS\n",
            "we have studied the available interaction and gene ontology data for proteins in escherichia coli, saccharomyces cerevisiae, drosophila melanogaster and homo sapiens genomes. by utilizing the boosting trees classification method, we have shown that highly-connected proteins in the studied pins share certain common go terms; this observation enabled the development of a hub classifier capable of distinguishing hub proteins from non-hubs. this classifier has improved accuracy for hub prediction relative to other traditional approaches for protein interaction prediction. it is anticipated that the hub classifier can serve as a useful tool to identify highly-interacting proteins in species without any available protein interaction data, with potential applications in optimizing protein pull-down experiments and identifying new drug targets against pathogens.\n",
            "\n",
            "availability\n",
            "the source code and executable program of the hub classifier is freely available for download at: \n",
            "\n",
            "authors' contributions\n",
            "mh acquired and analyzed protein interaction and gene ontology data, designed and developed the hub classifiers, built the hypothetical pins, simulated the protein bait selection experiments, and drafted and revised the manuscript. kgb analyzed the statistical models and tools of boosting trees, and revised the manuscript. ac conceived and designed the study, and revised the manuscript.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the hub classifier has improved accuracy for hub prediction relative to other traditional approaches for protein interaction prediction, with potential applications in optimizing protein pull-down experiments and identifying new drug targets against pathogens.\n",
            "the hub classifier has improved accuracy for hub prediction relative to other traditional approaches for protein interaction prediction.\n",
            "the hub classifier was trained on the existing go and protein-protein interaction data for escherichia coli, saccharomyces cerevisiae, drosophila melanogaster and homo sapiens species.\n",
            "it is anticipated that the hub classifier can serve as a useful tool for selecting baits and prioritizing proteins for protein interaction experiments, with potential applications in optimizing protein pull-down experiments and identifying new drug targets against pathogens.\n",
            "the hub classifier has improved accuracy for hub prediction relative to other traditional approaches for protein interaction prediction.\n",
            "it is anticipated that the hub classifier can serve as a useful tool for selecting baits and prioritizing proteins for protein interaction experiments, with potential applications in optimizing protein pull-down experiments and identifying new drug targets against pathogens.\n",
            "the hub classifier has improved accuracy for hub prediction relative to other traditional approaches for protein interaction prediction, with potential applications in optimizing protein pull-down experiments and identifying new drug targets\n",
            "83\n",
            "[{'summary_text': 'chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\\nchondrodysplasia has been described in a number of cattle breeds with varying phenotypes and different modes of inheritance.\\nchondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\\nchondrodysplasia has been described in a number of cattle breeds with varying phenotypes and different modes of inheritance.\\nchondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\\nchondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\\nchondrodysplasia has been described in a number of cattle breeds with varying phenotypes and different modes of inheritance.\\nchondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\\nchondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\\nchondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\\nchondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 3120 to 4096 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis. this leads to a reduced longitudinal growth of bones such as those of the limbs, spine and face. the most severe forms are found in lethal congenital generalized chondrodysplasia, which was originally reported in dexter cattle in  <dig>  <cit>  and usually referred to as “bulldog calves.” congenital chondrodysplasia has been described in a number of cattle breeds with varying phenotype and different modes of inheritance .\n",
            "\n",
            "development in molecular genetics has revolutionized the research in bovine teratology as the methods to identify genetic causes have improved significantly  <cit> . the molecular basis of a few bovine chondrodysplasias has been determined. in  <dig>  two independent mutations affecting the coding region of the aggrecan  gene causing lethal chondrodysplasia in dexter cattle  were described. these acan mutations showed incomplete dominant inheritance, leading to a mild form of dwarfism in heterozygotes, while homozygous animals displayed extreme chondrodysplasia and usually died during gestation  <cit> . for another lethal chondrodysplasia phenotype  reported in holstein cattle in  <dig>  <cit>  it was later demonstrated that the sire  of affected calves was mosaic for a dominant acting collagen type ii  missense mutation  disrupting the gly-x-y structural motif essential for the assembly of the collagen triple-helix  <cit> . in  <dig>  cases of stillborn bulldog-like calves occurring among the offspring of the danish holstein sire vh cadiz captivo  were reported to the danish bovine genetic disease programme  <cit> . this study reports the pathological investigation of the condition and the genetic analyses that led to the identification of a candidate causal mutation.\n",
            "\n",
            "methods\n",
            "animals\n",
            "three holstein calves were submitted for examination: case 1: a male delivered at gestation day  271; case 2: a female delivered at gd 273; case 3: a male delivered at gd  <dig>  the cases originated from different herds. ear tissue from two additional cases, where the herd veterinarian diagnosed the condition, was also submitted. the holstein bull vh cadiz captivo was registered as the sire in all cases. ethylenediaminetraacetic acid  stabilized blood from all five dams was also available as was semen and edta stabilized blood of the sire.\n",
            "\n",
            "post mortem examinations\n",
            "two calves  underwent initial full body computed tomography  scans using a single slice helical ct machine  to obtain a full view of the bone malformations. slice thickness was 3 mm and surface rendered reconstructions, which help illustrate bone malformations, were made using osirix software  <cit> . the calves were then necropsied, which included longitudinal sectioning of the right limbs and the spine. specimens of heart, lung, liver, kidneys, adrenal glands, thymus, vertebrae, femur and humerus were fixed in 10 % neutral buffered formalin for histology. the left limbs and the head of all calves were frozen at -20 °c and later examined by radiology. the head was sectioned longitudinally though the midline before radiology to obtain better images. the limbs were sectioned longitudinally after radiology to check for bilateral similarities in gross bone morphology. tissues for histology were processed by routine methods, embedded in paraffin, sectioned at 2–3 μm and stained with haematoxylin and eosin. bone specimens were decalcified in a  <dig>  % formaldehyde/17 % formic acid solution before processing.\n",
            "\n",
            "breeding analysis\n",
            "data on the outcomes of inseminations with semen of vh cadiz captivo in denmark were obtained from the danish cattle database. these included offspring recorded as “defective”, “stillborn”, “dead within 24 hours” or “still alive.” a questionnaire was constructed and mailed to owners of offspring recorded as “defective”, “stillborn” or “dead within 24 hours.” the questionnaire included pictures of two chondrodysplastic calves sired by vh cadiz captivo, the ear tack number of the dam of the calf and the date of delivery. the owners were then contacted by phone to assess if the offspring was chondrodysplastic or not.\n",
            "\n",
            "data on the gestation period for all calves sired by vh cadiz captivo were obtained from the database and the length of the gestation period for chondrodysplastic vs. normal calves was compared using the welch two-sample t-test.\n",
            "\n",
            "genetic analysis\n",
            "genomic dna was extracted for a total of  <dig> family members using standard protocols. for one of the affected calves the obtained quality of isolated dna was very low due to tissue degradation. therefore only a total of nine samples  were used for genotyping with the bovinehd beadchip , including  <dig>  evenly distributed snps, at geneseek.\n",
            "\n",
            "merlin v  <dig> . <dig> software  <cit>  was used to analyze the dataset and carry out the linkage analysis. the whittemore and halpern non-parametric linkage pair statistics z-mean and kong and cox logarithm of the odds  score were used to test for linkage using allele sharing among affected pedigree members without any assumptions on the mode of inheritance or on the genetic parameters of a specified model.\n",
            "\n",
            "a polymerase chain reaction  free fragment library with a 390 base pair  insert was prepared from one affected calf which was sequenced on half of a lane of an illumina hiseq <dig> instrument using  <dig> ×  <dig> bp paired-end reads. the genome sequencing data were deposited in the european nucleotide archive   under accession prjeb <dig>  the mapping to the umd <dig>  bovine reference genome assembly and variant calling were undertaken as previously described  <cit> . during the mutation analysis,  <dig> sequenced genomes of normal cattle from  <dig> genetically diverse bos taurus breeds were used as a local control cohort. the recent sequence variant database containing  <dig> already sequenced genomes of the ongoing  <dig> bull genomes project  <cit>  was used as a global control cohort during filtering for private variants of the sequenced affected calf.\n",
            "\n",
            "RESULTS\n",
            "phenotype\n",
            "the three submitted calves were morphologically similar and displayed severe disproportionate dwarfism . the body and the limbs were shortened and compressed due to reduced length of the spine and the long bones of the limbs. the distal parts of the limbs were rotated medially and had digits of almost normal size. the limbs were bilateral symmetrically malformed. the face had severe dysplasia with compression and ventral deviation of the facial bones, which had an almost 90° angle to the axis of the brain stem. the dorsal aspect of the cerebellum was compressed . palatoschisis was present in all cases and the tongue protruded. the body weight was reduced to 26 kg  compared to full term danish holstein calves of approximately  <dig>  kg for males and  <dig>  kg for females.fig.  <dig> gross morphology of chondrodysplastic calves. notice the severe disproportionate dwarfism with short and compressed body and limbs and severe dysplasia of the facial bones. a case  <dig>  b case  <dig>  bar = 30 cm\n",
            "\n",
            "fig.  <dig> lesions in the head of a chondrodysplastic calf. a longitudinal sectioning of the head through the midline showing severe dysplasia and 90° angular ventral deviation of the facial bones to the axis of the brain . the cerebellum appears compressed dorsally. frozen specimen, case  <dig>  b radiograph of the head shown in “a” . the splanchnocranium is dysplastic resulting in severe superior brachygnathism. bar = 5 cm\n",
            "\n",
            "\n",
            "\n",
            "longitudinal sectioning of the spine and long bones of the limbs showed small irregular diaphyses of increased hardness and significantly enlarged epiphyses consisting entirely of a homogenous chondroid rubber-like tissue. distinct epiphyseal lines could not be identified as the irregular metaphyses opposed the chondroid epiphyses that lacked a center of ossification. the dorsally enlarged epiphyses of the vertebrae caused compression of the spinal cord.\n",
            "\n",
            "the thorax and abdomen were of reduced volume and the enclosed organs were consequently closely opposed. the heart had biventricular myocardial hypertrophia and dilation of the right ventricle. slight hydrothorax was present. the lungs had diffuse congenital atelectasis. the liver surface was irregular and the texture of the parenchyma increased. males had bilateral abdominal cryptorchidism.\n",
            "\n",
            "radiology and ct scanning showed that similar deformities were present in all calves. the neurocranium was relatively normal in size and shape . the splanchnocranium was foreshortened and misshapen resulting in severe superior brachygnathism. mandibular incisors, premolars and molars were present. maxillary teeth were also present but crowded due to the skull deformity. bone structure appeared normal, with normal appearing cortical, cancellous and medullary cavities present in the mandible. nasal and ethmoid turbinate bones were present. the appendicular skeleton was grossly deformed with the scapula, humerus radius, ulna, pelvis, femur, tibia, fibula and phalanges being affected. only remnants of the diaphyses were identifiable. bone morphology was so altered that they could only be identified by virtue of their relative location . deformities, especially shortening, were most severe for the proximal bones of the limbs while the distal phalanges were almost normal in shape and size. the carpus and tarsus were not ossified.fig.  <dig> radiograph showing abnormally developed bones of a hind limb. only remnants of the diaphyses can be identified and the individual bones can only be identified by virtue of their relative location. the proximal bones are most deformed, while the distal phalanges are almost normal in shape and size. case 2\n",
            "\n",
            "\n",
            "\n",
            "ct scanning showed lesions similar to those observed at radiology. in addition, there was widespread failure of fusion of the right and left parts of the dorsal spinous processes of the lumbar spine while partial fusion was present in the thoracic spine. vertebral bodies were only partially formed in the cervical area and appeared as multiple osseous bodies, while vertebral bodies in the thoracic and lumbar spine were better defined but were nonetheless irregular in shape. ribs  were present and appeared normal . a movie showing the malformation in a 360° view is presented in additional file  <dig> fig.  <dig> surface rendered computed tomography images of a chondrodysplastic calf. the scanning data are rendered with bone and soft tissue surfaces. the latter has been set to have a degree of transparency in the reconstruction thus allowing visualisation of the calf’s overall morphology and its relation to the underlying skeletal abnormalities. case 3\n",
            "\n",
            "\n",
            "\n",
            "histology showed highly irregular growth zones dominated by hypertrophied chondrocytes and with lack of normal alignment of chondrocytes although some chondrocytes were occasionally arranged normally in columns. calcification of the intercellular matrix and chondrocyte degeneration occurred as irregular and interrupted areas towards the metaphysis. the ossification process was incomplete and cores of cartilage persisted throughout the meta- and diaphysis . the epiphyses consisted of hyaline cartilage with chondrocytes located in variably sizes lacunae and with fibrous septae containing dilated vessels. slight fibrosis and stasis was present in the liver.fig.  <dig> photomicrograph of the growth zone. the growth zone is characterized by irregular areas a granular basophilic appearance and degeneration of chondrocytes. notice absence of the alignment of chondrocytes normally seen in a growth zone. large cores of chondroid matrix persists into the metaphysis. vertebra, case  <dig>  haematoxylin and eosin, bar = 200 μm\n",
            "\n",
            "\n",
            "\n",
            "breeding analysis\n",
            "analysis of breeding data showed that hv cadiz captivo had  <dig> recorded offspring of which  <dig> were born alive and had received an ear tack number, i.e. regarded as normal. the owners of the remaining  <dig> calves were contacted by questionnaire and phone. sixty-four owners reported that the offspring had been a “bulldog calf” while  <dig> reported that the calf was of normal proportions. six calves were excluded as the owner did not respond or could not recall the morphology. hv cadiz captivo therefore produced “bulldog calves” at a ratio of  <dig>  % . pedigree analysis of affected calves did not display obvious inbreeding to a common ancestor.\n",
            "\n",
            "the length of the gestation period between cows having normal offspring vs.“bulldog calves” was compared. the mean among the cows with a normal calving was  <dig>  days, which was significantly different  from the mean among cows giving birth to “bulldog calves” .\n",
            "\n",
            "genetic analysis\n",
            "a multipoint non-parametric linkage analysis detected  <dig> genomic regions located on nine different chromosomes significantly linked with the chondrodysplasia phenotype at chromosome-wide error probabilities for z-mean values and lod scores below  <dig>  . in light of the few reports of mutations causing chondrodysplasia in livestock, a causative variant affecting one of the known candidate genes was hypothesized. an omim database search for osteochondrodysplasia and/or chondrodysplasia revealed a total of  <dig> candidate genes  of which nine genes were located in linked genome regions .fig.  <dig> non-parametric multipoint linkage analysis for chondrodysplasia. a total of  <dig> significantly linked genome regions are shown in blue. note, that  <dig> out of  <dig> candidate genes for chondrodysplasia including col2a <dig> are located in linked regions\n",
            "\n",
            "\n",
            "\n",
            "for mutation analysis the whole genome of one affected calf was sequenced to  <dig> x coverage using next-generation sequencing technology. genome-wide filtering for sequence variants in the whole genome that were present only in the affected calf and absent from  <dig> control cattle genomes  resulted in  <dig> private variants . subsequent filtering of variants located apart from the linked genome regions allowed the exclusion of 87 % variants remaining with  <dig> variants including  <dig> coding. finally the candidate genes present in the mapped linked regions were screened for possible variants remaining with a single splice site variant in the col2a <dig> gene . all five available affected calves, their dams and the sire were genotyped by direct sequencing of a targeted pcr product using standard sanger sequencing in the pedigree for this col2a <dig> variant revealing a perfect association of the mutant a allele with the chondrodysplasia phenotype . all affected animals were heterozygous for the mutant allele and none of the dams carried this allele. interestingly, sequencing showed the presence of the identified mutation in the sire vh cadiz captivo, who clearly carried the mutant allele at a low level in comparison to the wild type allele. the presence of both the wild type g allele and a smaller sized peak for the mutant a allele was detected in both available samples. the manually estimated relative area ratio of the mutant a allele was about 5 % in peripheral blood and about 15 % in semen .fig.  <dig> a de novo germline mutation in col2a <dig> causes chondrodysplasia. genalogical chart showing five holstein calves affected by chondrodysplasia  and their parents. males are represented by squares, females by circles. note, that the electropherograms presented below the pedigree symbols show that the mutant a allele of the col2a <dig> intron  <dig> splice site mutation is present in heterozygous form in chondrodysplasia affected offspring and in the sire’s blood and semen at different ratios. the dna of case  <dig> was of very low quality explaining the low ratio of the wild type g allele\n",
            "\n",
            "\n",
            "\n",
            "discussion\n",
            "the gross, microscopic and radiographic lesions in the examined offspring of the sire vh cadiz captivo are consistent with generalized congenital chondrodysplasia, a condition usually referred to as “bulldog calf syndrome.” pedigree analysis of affected calves did not display obvious inbreeding to a common ancestor, suggesting the causative allele is not a rare recessive. the normal phenotypes of the sire and the dams suggest a dominant inheritance with incomplete penetrance or a mosaic mutation. due to the lethal effect of the mutation we concentrated on variants that were located within the coding sequences or within the splice sites of the candidate genes in the identified linked region of the bovine genome. finally, mutation analysis for chondrodysplasia detected a perfect association between the splice site mutation of col2a <dig> and the disease phenotype. the col2a <dig> variant  is predicted to affect splicing because it alters the conserved splice donor sequence gt at the 5’-end of intron  <dig>  which was changed to at. the predicted consequences are either skipping of exon  <dig> or retention of intron  <dig>  both possible scenarios will lead to aberrant splice variants encoding truncated proteins, but nonsense-mediated decay might be the most likely major consequence of the splice site mutation. unfortunately, the rna was too degraded by the time the calves were available for examination so reverse transcription pcr failed to verify the consequence of the sequence variant experimentally.\n",
            "\n",
            "heterozygous mutations of col2a <dig>  have been identified in human patients with various rare autosomal dominant conditions characterized by skeletal dysplasia, short stature, and sensorial defects collectively termed as type ii collagenopathies  <cit> . these disorders not only impair skeletal growth but also cause ocular and otolaryngological abnormalities. the classical phenotypes include the spondyloepiphyseal dysplasia  spectrum with variable severity from lethal sed, including achondrogenesis type ii and hypochondrogenesis , through congenital sed  to late-onset sed . one third of human col2a <dig> defects are dominant-negative mutations in the triple-helical region of alpha  <dig>  chains which disrupt the collagen triple helix  <cit> . these predominantly glycine to nonserine residue substitutions exclusively create more severe phenotypes like achondrogenesis type ii and hypochondrogenesis and correspond to the bovine col2a <dig> allele causing chondrodysplasia in holstein cattle  <cit> . in humans, splice-site mutations are assumed to cause haploinsufficiency through nonsense-mediated mrna decay  <cit> . the col2a <dig> mutations resulting in a premature stop codon are found in less severe phenotypes such as stickler dysplasia type i  characterized by ocular, auditory, skeletal, and orofacial abnormalities or kniest dysplasia  characterized with short stature, restricted joint mobility, and blindness  <cit> . only in two cases, splice site mutations, similar to the mutation identified in this study, are shown to produce phenotypes at the lethal end of the type ii collagenopathy spectrum  <cit> . therefore, it seems to be highly likely that the detected bovine col2a <dig> splice site variant explains the phenotype in the affected calves. probably a functional haploinsufficiency of collagen ii, resulting from degradation of mutant transcripts due to nonsense-mediated mrna decay, leads to a quantitative deficit of structurally normal collagen ii causing the major malformations observed in the affected calves.\n",
            "\n",
            "this lethal syndrome was present in  <dig>  % of vh cadiz captivo offspring and was associated with the col2a <dig> splice site variant being present in his germ cells, presumable in around 15 % corresponding to the prevalence of malformed offspring. therefore this sire is confirmed as a founder mosaic as shown before for “solid gold” for ovine callipyge  <cit>  and “campus” for porcine myopathy  <cit> . on the other hand, the genetic analysis of blood from the sire showed that the mutant col2a <dig> allele was present in peripheral blood cells at a low level of about 5 %. these results reveal that vh cadiz captivo is a gonadal and somatic mosaic. germline mosaicism is a relatively frequent mechanism in the origin of genetic disorders  <cit> . depending on various factors, such as the gene involved and/or the degree of mosaicism, the carrier of somatic and germline mosaicism may be asymptomatic or may present with various symptoms. there are two possibilities for the existence of such a mosaicism: one is that the mutation occurs in a germ cell that continues to divide. the other possibility is that the mutation occurs very early in a somatic cell before the separation to germinal cells and is therefore present both in somatic and germinal cells  <cit> . this seems to be the most likely explanation for vh cadiz captivo as he carried the mutant allele in both somatic and germinal cells, although he appeared normal.\n",
            "\n",
            "a similar chondrodysplastic phenotype was observed in the offspring of the holstein sire igale masc but affected only 1–2 % of his offspring  <cit> , which indicates that the mutation probably occurred later in the development of the testicular germ cells than in vh cadiz captivo. these findings show that the proportion of malformed calves may vary considerably between offspring of sires carrying germ cell mutations.\n",
            "\n",
            "paternal dominant germ cell mutations is a risk in cattle breeding, especially because the ratio of defective offspring may be very high. presence of a dominant mutation in a certain proportion of spermatozoa has been hypothesized in cases of osteogenesis imperfecta. denholm and cole  <cit>  found this lethal bone disorder in 44 % of a clinically normal friesian bull mated to unrelated females, while agerholm et al.  <cit>  diagnosed osteogenesis imperfecta in 9 % of the offspring of a normal holstein sire. paternal dominant germ cell mutations are a challenge to cattle breeding as diseased progeny may occur without inbreeding, which in other ways is the most significant challenge regarding bovine genetic diseases. reporting systems and disease programs that allow proper diagnostic of defective calves and targeted genetic investigations are highly valuable to limit the occurrence of severe disorders as lethal chondrodysplasia and osteogenesis imperfecta. the sire vh cadiz captivo was omitted for breeding by the breeding association as soon as the condition was diagnosed in his offspring.\n",
            "\n",
            "it is important to recognize that even within a single breed, cases of lethal generalized chondrodysplasia show allelic heterogeneity. the cases reported here had a morphology indistinguishable from the igale masc bulldog calves  <cit>  even by detailed post mortem examinations. a mutation in the same gene was therefore possible to be the cause. however, lethal congenital chondrodysplasia in holsteins has been reported previously. although such cases may share the basic lesion, i.e. chondrodysplasia, during examination in the field, significant variation in the phenotype may be found during post mortem examination, e.g. the syndrome reported by berger and innes  <cit>  had severe hydrocephalus, which distinguishes it from the col2a <dig> associated “igale masc” and “vh cadiz captivo” types. investigations and reports detailing characteristics of chondrodysplasia phenotypes are needed.\n",
            "\n",
            "CONCLUSIONS\n",
            "genomic analyses identified a causative spontaneous dominant acting candidate causal mutation in col2a <dig>  this study provides a first example of a dominant col2a <dig> splice site variant associated with a severe lethal chondrodysplasia phenotype. this mutation must have occurred during the early development of the asymptomatic sire vh cadiz captivo. germline mosaicism as a relatively frequent mechanism in the origin of genetic disorders explains the prevalence of a certain fraction of affected offspring. paternal dominant de novo mutations are a risk in cattle breeding, especially because the ratio of defective offspring may be very high and be associated with significant animal welfare problems. it is therefore important to have surveillance programmes for congenital syndromes in cattle.\n",
            "\n",
            "abbreviations\n",
            "acan, aggrecan gene; bp, base pair; ct, computed tomography; edta, ethylenediaminetraacetic acid; ena, european nucleotide archive; gd, gestation day; lod, logarithm of the odds; pcr, polymerase chain reaction; sed, spondyloepiphyseal dysplasia\n",
            "\n",
            "additional files\n",
            "additional file 1: surface rendered computed tomography scanning movie of a chondrodysplastic calf. \n",
            "\n",
            "additional file 2: non-parametric multipoint linkage analysis output data. z-mean values and lod scores and their chromosome-wide error probabilities  along a grid of equally spaced locations  of all  <dig> autosomes. the minimum and maximum achievable values in the present linkage analyses are given in the first two rows. chromosome-wide significant z-mean values and lod scores and their p-values are highlighted in green. \n",
            "\n",
            "additional file 3: candidate genes for chondrodysplasia. omim database search for genes associated with osteochondrodysplasia/chondrodysplasia and their genomic position in the cattle genome. \n",
            "\n",
            "additional file 4: private sequence variants of the sequenced chondrodysplasia affected calf. a total of  <dig> sequence variants which were absent in  <dig> control genomes. variants  located in the linked genome regions are highlighted in green. \n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "chondrodysplasia has been described in a number of cattle breeds with varying phenotypes and different modes of inheritance.\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "chondrodysplasia has been described in a number of cattle breeds with varying phenotypes and different modes of inheritance.\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "chondrodysplasia has been described in a number of cattle breeds with varying phenotypes and different modes of inheritance.\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "chondrodysplasia is a developmental bone defect occurring due to disturbed endochondral osteogenesis.\n",
            "\n",
            "84\n",
            "[{'summary_text': 'in this report, we explored the topological characteristics of herpesvirus mirnas involved in the process of regulation of the human pin.\\nwe found that common targets tend to form a larger module and have a higher density than randomly chosen nodes and that they are located in the global central core of pin.\\nin this report, we explored the topological characteristics of herpesvirus mirnas involved in the process of regulation of the human pin.\\nwe found that common targets tend to form a larger module and have a higher probability of being targeted by only one virus as specific targets.\\nour results suggest that common targets might be related to the pathogenesis processes common to all the viruses and that specific targets are involved in the infection processes specific to a particular virus type.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 6011 to 6144 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "herpesviruses are members of herpesviridae family, a large family of dna viruses that cause chronic, latent and recurrent infections in animals and humans. herpesviruses are double-stranded dna viruses with large genomes encoding complex virus particles and enzymes involved in a variety of cellular process, including nucleic acid metabolism, dna synthesis, and protein processing  <cit> . in addition to herpesvirus proteins associated with pathogenic processes, herpesvirus-encoded micrornas  have been also shown to play an indispensable role in herpesvirus pathogenesis  <cit> . mirnas are a class of endogenous, single strand rnas, approximately  <dig> nucleotides long that bind to 3'untranslated regions of transcript causing degradation of their respective targets or block protein translation. since the discovery of virus-encoded mirnas in epstein-barr virus   <cit> , the roles of virus encoded mirnas in the regulation of the viral life cycle and in mediating interactions between viruses and their hosts, have been examined in some detail  <cit> .\n",
            "\n",
            "with the emergence of versatile mirna target prediction algorithms and availability of proteome-wide protein-protein interaction data sets, manually curated or derived from high-throughput experiments , it has become possible to investigate regulation of the whole human pin by mirnas. since protein-protein interactions constitute the basis of most life processes, such studies might provide important clues necessary to the thorough understanding of biological mechanisms at the whole systems level. in recent years, human mirna regulated cellular networks, such as signal transduction, gene regulatory network, pin and metabolic network, have been studied in great detail  <cit> . some of the results highlight an interesting commonality: that mirnas tend to target nodes with high topological complexity, such as hubs and bottlenecks. in signal transduction network, mirnas preferentially target downstream network components, positively linked network motifs and downstream components of the adaptors that have the potential of recruiting additional downstream components  <cit> . genes in regulatory networks with more transcription factor binding sites have, on average, more mirna-binding sites and a higher probability of being targeted by mirnas  <cit> . protein degree in the human pin correlates to the number of mirna target-site types of the gene encoding the respective protein  <cit> . in addition, analysis of the human pin and the human metabolic network showed that human-encoded mirnas preferentially target hubs and bottlenecks  <cit> .\n",
            "\n",
            "mirnas are some of the key regulators of various biological processes, for example, they play an important role in virus-host interactions  <cit> . this applies both to human-encoded and virus-encoded mirnas. we need to examine the mechanisms involved in such interactions to gain insight into this complex process. to date, only one study has systematically examined the functional characteristics of human herpesvirus mirnas  <cit> . the results of that study showed a statistically significant preferential targeting of host genes involved in cellular signalling and adhesion junction pathways. other studies mentioned above revealed some of the regulatory characteristics of human encoded mirnas in biological networks, however, in the field of virus mirna-mediated virus-host interactions, not many studies have been conducted at the systems level. in this report, we explored the topological characteristics of human herpesvirus mirnas that target human pin. we believe that determining which human proteins are targeted by viruses will provide insight into molecular processes shared by related viruses. taking into account the large differences between mirnas encoded by different viruses  <cit> , it is not unreasonable to expect that the analysis of one virus group, in our case the herpesviruses, will yield some interesting results. as essential cellular building blocks, proteins perform a variety of functions by interacting with other proteins. if we are to achieve comprehensive understanding of herpesvirus-host interactions and better understand the molecular basis of viral pathogenesis, it will be of great importance to study the function of herpesvirus mirnas in the framework of pin. the results of these studies are also likely to provide new means for developing novel therapeutic strategies for the treatment and prevention of viral infections.\n",
            "\n",
            "RESULTS\n",
            "herpesvirus mirnas preferentially target hubs and bottlenecks\n",
            "there are two known features of human mirna regulatory properties  <cit> : first, it's their preferential targeting of hubs and bottlenecks; second, there is a highly significant positive correlation between protein degree in human pin and the number of target-site types at the 3'utr region of its gene. it has been established that herpesvirus-encoded mirnas are processed and mature within human cells, which hints that they display properties similar to those of human-encoded mirnas. to investigate this possibility, we defined hubs and bottlenecks as 5% of pin nodes with the highest degree and betweenness centrality and analyzed the significance of hubs and bottlenecks targeted by herpesvirus mirnas on two levels. first, to inspect whether the targets of herpesvirus mirnas cover more hubs than random conditions, the statistical significance of the proportion of mirna-targeted hubs  was tested. the results demonstrated that herpesvirus mirnas preferentially target more these nodes . second, the nodes targeted by herpesvirus mirnas were classified into two groups: hubs  and non-hubs . the regulation strength of the herpesvirus mirnas in the two groups was next examined. the statistical significance of virus mirnas and virus type numbers for the two groups was tested, with positive results for both  and  <dig>  ; permutation test). in addition, as an alternative to the boxplot, beanplots  <cit>  were employed to visualize the estimated density of the distribution of virus mirnas and virus type numbers for the two groups, respectively . as hubs are the crucial nodes in the pin, preferential hub targeting will make herpesvirus mirnas more efficient in the context of that network. to further analyze the herpesvirus mirnas targeting behaviour, we compared the nodes targeted by those mirnas with nodes targeted by human-encoded mirnas. we found that the two node sets behaved similarly . two possibilities were proposed for the result. first, as human mirnas are key regulators of many fundamental biological processes, some of these biological processes may be needed by viruses for successful infection. namely, this result may be archived by the adaption of virus to its host for survival over long-term of evolution, but it is not always true considering both types of mirnas may express at the different temporal and spatial condition. second, some features of genes have been formed during evolution to make these genes more favorable to be targets of mirnas, such as utr context  <cit> , site accessibility  <cit> .\n",
            "\n",
            "p-values were computed using randomization tests.\n",
            "\n",
            "p-value <  <dig> e- <dig>  computed by using fisher's exact test\n",
            "\n",
            "characteristics of common and specific targets\n",
            "we defined nodes targeted by all six viruses as common targets and nodes targeted by only one virus as specific targets. there are significant differences observed in degree and betweenness centrality between common and specific targets . the distribution differences between these two types of targets are depicted in figure  <dig>  to dissect further the relationship between nodes targeted by different virus types and topological attributes of the pin, we defined the virus types of each node by counting the virus species whose mirnas target the node. then, the nodes were divided into six groups based on virus types, the actual hub proportion in each group was computed . we found a significant positive correlation between virus types and the proportion of hubs  . this is consistent with the observation that common targets have significantly higher degree and betweenness centrality than that of specific targets. we performed a simulation to test whether the trends were significant, that is, we re-computed the hub proportion  <dig>   <dig> times with randomly chosen nodes  and the trends seemed to be significant. the hub proportion and the proportion of bottlenecks are significant for both common and specific targets compared to the simulation. we propose that common targets might be related to the pathogenesis processes common to all the viruses and that specific targets are involved in the infection processes specific to a particular virus type.\n",
            "\n",
            "we also investigated the significant go terms for the common targets  and found that most over-represented go terms are related to various developmental and regulatory processes, such as nervous system development and regulation of signalling pathway and cell communication, indicating a close relationship between these processes and herpesvirus pathogenesis.\n",
            "\n",
            "topological characteristics of common herpesvirus mirna targets\n",
            "we used k-core and excess retention  analysis  <cit>  to measure the distance between common targets and the topological center. with an increasing k value, higher er values are indicative of common targets being closer to the global center of the pin . to test the significance of this observation, we randomly chose the same number of nodes as common targets and re-calculated the er that revealed only minor fluctuations. a substantial difference in the er for each k-core could be observed between common targets and random controls, suggesting that common targets might interconnect tightly in the pin. to test this hypothesis, we defined strategies for measuring the significance of the module characteristics to the common targets. the first parameter was the number of nodes in the gcc comprised of common targets, the second was the density of the sub-network formed by those targets and results showed that both density and gcc are significant . it suggested that the module formed by the common targets position in the network's core; it might be not accidental that most viruses utilized these nodes as targets. in the context of virus-host interactions, it could be beneficial for a virus to hijack the network core since this would facilitate rapid transmission of information to the rest of the network, thereby maximizing viral control of cellular functions.\n",
            "\n",
            "discussion\n",
            "it is well understood that cellular functions are carried out using various specialized groups of molecules interacting via intricate networks. no approach to complex systems can succeed without exploiting network topology  <cit> . in this study, we investigated characteristics associated with the targeting of herpesvirus mirnas to proteins in the human pin. virus-encoded mirnas have unique advantages: they can function at the rna level, affecting the expression of many genes rapidly and extensively. the results of this study will contribute to a better understanding of the complex herpesvirus-host interactions at the mirna level.\n",
            "\n",
            "we found that herpesvirus mirnas preferentially target pin hubs and bottlenecks, a process similar to that of human-encoded mirnas. the biological networks displayed scale-free characteristics, i.e. most of the nodes have a relatively low degree, making them resistant to attacks on random nodes  <cit> . it seems that the vulnerability of human protein networks  is successfully exploited by herpesviruses, suggesting that these viruses must have evolved to target key nodes preferentially, allowing them to take maximum control of the human protein network during infection. although the various roles carried out by virus-encoded proteins have been extensively studied over the past few decades, it is only recently that viral protein-regulated pin have been studied in detail. matthew d. dyer et al.  <cit>  examined some human-pathogen protein-protein interactions and found that both viral and bacterial pathogens interacted with human pin hubs and bottlenecks.\n",
            "\n",
            "the results of our comparison between common and specific targets suggested that some topological differences existed between nodes related to processes associated with common and specific virus pathogenesis mechanisms. furthermore, the significant hub and bottleneck proportions for common targets validated the preference of viral mirnas for hubs and bottlenecks. these results provided valuable information that will help unravel mechanisms associated with herpesvirus pathogenesis.\n",
            "\n",
            "we also characterized the modularity of common pin targets. we found that common targets tend to form a larger module and have a higher density than randomly chosen nodes and that they are located in the global central core of pin. during virus-host interactions, viruses use their limited resources to exploit fundamental cellular processes for finishing their life cycles. from that perspective, targeting nodes located in the central part of pin seems a reasonable strategy designed to affect nodes in other parts of the network efficiently and rapidly. the nodes in the central part represent fundamental components of the cell, the control over which might be necessary for the virus to infect successfully. as our go analysis confirmed, these nodes are related to processes associated with fundamental cellular regulation and development indispensable to viral survival.\n",
            "\n",
            "to test the robustness of our results, two additional algorithms, miranda  <cit>  and targetscan  <cit> , were used to predict the herpesvirus mirnas targets; meanwhile, a high confidence protein-protein interaction data set, hprd -filtered, was also used to construct the pin. most results are in agreement with those obtained using pita algorithm  <cit>  and hprd dataset  <cit> . the detailed results are described in additional files .\n",
            "\n",
            "in this paper, we focused on the description of statistically significant, functional characteristics of herpesvirus mirnas involved in the process of regulation of the human pin. some limitations to the analysis described in this report are, first, the pin used was incomplete and therefore subject to considerable error rates; second, the large number of predicted mirna targets makes experimental validation rather difficult. moreover, we know that the results of different types of predictions do not fully agree with each other  <cit> . our predicted herpesvirus mirnas collection might not be complete; that is, the use of improved mirna prediction algorithms and a wider implementation of high-throughput techniques might identify new mirnas. third, the herpesvirus mirna mediated human protein interaction network, in the context of herpesvirus protein and human protein interactions, was not analyzed due to the disequilibrium and lack of herpesvirus protein and human protein interaction data. despite these limitations, our analysis of herpesvirus mirna interactions with the human pin should help to reveal a broader picture of their functional mechanisms at the systems level and add to our knowledge of the viral pathogenesis process.\n",
            "\n",
            "CONCLUSIONS\n",
            "in this study, we explored the ability of herpesvirus mirnas to target the human pin. viral mirnas preferentially target pin hubs and bottlenecks, behaviour similar to that of human-encoded mirnas. topological comparison between specific and common targets showed that common targets have significantly higher degree and betweenness centrality. k-core and er analysis revealed that common targets occupy the global central part of the pin. furthermore, a significant modularity of common targets was found. their crucial topological position in the pin suggested that they might play a key role in herpesvirus pathogenesis. these results add to our understanding of herpesvirus mirnas functions, giving us new insights into the complex process of herpesvirus-host interactions and provide information that can be used in the development of novel antiviral drugs.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this report, we explored the topological characteristics of herpesvirus mirnas involved in the process of regulation of the human pin.\n",
            "we found that common targets tend to form a larger module and have a higher density than randomly chosen nodes and that they are located in the global central core of pin.\n",
            "in this report, we explored the topological characteristics of herpesvirus mirnas involved in the process of regulation of the human pin.\n",
            "we found that common targets tend to form a larger module and have a higher probability of being targeted by only one virus as specific targets.\n",
            "our results suggest that common targets might be related to the pathogenesis processes common to all the viruses and that specific targets are involved in the infection processes specific to a particular virus type.\n",
            "\n",
            "85\n",
            "[{'summary_text': 'in silico analysis of the 3d structure of pfceltos and co1-3- il-2-pfceltos revealed that pfceltos and co1-3- il-2-pfceltos have a well modeled structure.\\nin silico analysis of the 3d structure of pfceltos and co1-3- il-2-pfceltos revealed that pfceltos and co1-3- il-2-pfceltos have a well modeled structure.\\nin silico analysis of the 3d structure of pfceltos and co1-3- il-2-pfceltos revealed that pfceltos and co1-3- il-2-pfceltos have a well modeled structure.\\nin silico analysis of the 3d structure of pfceltos and co1-3- il-2-pfceltos revealed that pfceltos and co1-3- il-2-pfceltos have a well modeled structure.\\nin silico analysis of the 3d structure of pfceltos and co1'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 11377 to 12288 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "infectious diseases are the leading cause of death all over the world  <cit> . malaria, also called plasmodium infection, is one of the most important human infectious diseases threatening the lives of millions of people every year. according to who malaria report, globally, about  <dig>  billion people in  <dig> countries and territories are at risk of being infected with malaria, and  <dig>  billion are at high risk  <cit> . although the disease has been eradicated in most areas, it’s still widespread in some regions  <cit> . the biggest challenges in controlling malaria disease are the emergence of anti-malarial drug resistance and insecticide resistance in parasite and mosquito, respectively . therefore, there is an urgent need to develop novel intervention strategies such as vaccines to reduce the burden of malaria morbidity and mortality. malaria is most common in poor and developing regions of the world and has a strong negative effect on economic growth  <cit> . oral immunogenicity has opened new avenues for the development of vaccines potentially effective in reducing the burden of diseases especially in low-income and developing countries. oral vaccines are low cost, easily administered , in most cases capable of being stored and transported without refrigeration  and painless  <cit> . in contrast to injected vaccines, oral vaccines target mucosal surfaces, which cause stimulation of systemic as well as mucosal immune responses . considering the fact that most pathogens enter the body through mucosal surfaces, mucosal immune system provides the first line of defense against invading bacteria and viruses. therefore, the simultaneous induction of systemic and mucosal immunity offers an ideal strategy to fight infectious diseases. despite many advantages that oral vaccines have, only limited numbers of them have been approved for human commercial use and yet significant challenges must be overcome to make oral vaccines closer to reality  <cit> . one of the challenges is the complexity of mucosal immune system that must discriminate between harmless and dangerous antigens. one way to overcome this problem is to use adjuvants. oral adjuvants offer exciting possibilities for the formulation of oral vaccines .\n",
            "\n",
            "during the past decades considerable progress in recombinant dna technology has led to the development of fusion proteins. fusion proteins are novel biomolecules obtained by genetically fusing two or more genes that originally code for separate proteins. thereby fusion proteins have distinct functions derived from each of their domains. fusion proteins have wide applications in industry and pharmaceutical protein production.\n",
            "\n",
            "the general objective of the current study was the design and computational analysis of a fusion protein consisting of plasmodium falciparum cell-traversal protein for ookinetes and sporozoites  fused to human interleukin- <dig>  and m cell-specific peptide ligand , as a step toward developing a candidate recombinant oral vaccine against malaria. celtos is a protein that mediates malarial invasion into its hosts  and is required for effective malaria infection . on the other hand, celtos amino acid sequence is highly conserved among the plasmodium species. celtos highly conserved amino acid sequence and vital role for malaria infection indicate its important function across all species  <cit> . therefore, targeting the immune response to this highly conserved and crucial protein and interfering with its biological function could possibly result in protection against infection by heterologous species of plasmodium  <cit> .\n",
            "\n",
            "in the designed construct, human interleukin  <dig>  is thought to act as mucosal adjuvant. it should be considered that the type of desired immune response  will influence the choice of adjuvants for immunization regimens  <cit> . in case of malaria disease, parasites and infected red blood cells which activate dendritic cells through pattern recognition receptors  are phagocytosed and their antigens are presented to t cells. prr signaling causes the secretion of cytokines that initiate the inflammation that underlies malaria pathogenesis and direct th <dig> cell to differentiate. th <dig> cells help b cells differentiation and antibody secretion and also secrete ifn-γ, which activates macrophages that phagocytose opsonized parasites and infected red blood cells and subsequently kill them by no- and o2-dependent pathways. inflammation induces expression of endothelial adhesion molecules to which infected red blood cells bind. finally the secretion of anti-inflammatory cytokines from macrophages and regulatory populations of t cells curtailed the inflammation  <cit> . to the best of our understanding, adjuvants that increase the th <dig> response are more appropriate in malaria vaccine development. here we focus on human interleukin  <dig>  as mucosal adjuvant. il- <dig> has a central role in the cascade of events involved in immune responses and can function as a vaccine adjuvant to increase the immune response to vaccine immunogens .\n",
            "\n",
            "specific advantages of il- <dig> as adjuvant for malaria vaccine are as follow:il- <dig> stimulates t cells to secrete inf-γ. inf-γ is known as immune interferon due to activating macrophages and enhancing phagocytic activity which leads to activation of immune responses. on the other hand, inf-γ results in the expression of mhci and mhcii molecules on the surface of infected cells and antigen presenting cells, respectively, therefore enhancing antigen presenting and immune system activity. in addition, inf-γ activates the secretion of igg and its subclasses by effecting on b lymphocytes .\n",
            "\n",
            "activation of t cells under the influence of il- <dig> leads to the secretion of cytokines such as il- <dig>  unlike many other interleukins, il- <dig> inhibits macrophage activity and secretion of il- <dig>  il- <dig> and tnf. il- <dig> acts as inducer for inf-γ and is an effective factor in cellular and innate immune responses against intracellular microbes. inhibition of il- <dig> leads to inhibition of inf-γ production. in fact, il- <dig> is well known as the inhibitor of immune responses, especially responses that are set up by macrophages, thereby immune responses are terminated. as mentioned above, the enhancement of inflammation and thereby endothelial adhesion of blood vessels is one of the outcomes of immune system responses to malaria parasite and malaria vaccine. in this case, repressive role of il- <dig> prevents excessive inflammation.\n",
            "\n",
            "\n",
            "\n",
            "although reported studies have indicated the potential of cytokines as mucosal adjuvants, in order to increase the probability of vaccine candidate binding to intestinal cells, we adopted another strategy. co <dig> is a peptide ligand that promotes the binding of ligand-fused antigen to human m-like cells and has also comparable levels of adjuvant activity to cholera toxin   <cit> . conjugation of co <dig> to the designed construct should result in delivery of antigen to m cells.\n",
            "\n",
            "since most of the biological functions of proteins depend upon their 3d structure, in designing multi domain recombinant proteins, proper folding, stability and interaction between domains must be considered. fusion proteins are much more susceptible to misfolding than single-domain proteins due to the interaction between their different peptide domains. therefore, in silico analysis of multi domain proteins is an indispensible stage in recombinant protein production projects.\n",
            "\n",
            "attempts to model the structure of proteins on the computer began about 30 years ago. since that time our understanding of protein structure and dynamics has significantly increased and now protein data bank  contains more than  <dig>  high resolution protein structures. valuable 3d models of a protein that has a clear sequence homology to known proteins can be predicted by homology modeling method. however, even in cases where there is no sequence homology, threading methods relate protein sequences to a library of known structures and predict the likely protein structure. the crystallographic structure of celtos has not been reported yet. in the present study, we carried out a molecular modeling study of pfceltos protein and designed fusion proteins using iterative threading assembly refinement  to obtain their 3d structures. then energy minimization and molecular dynamics  simulation were run to refine the models. the simulations of pfceltos and fusion proteins were performed for long time duration of 10 ns and the obtained structures were analyzed to verify further.\n",
            "\n",
            "methods\n",
            "sequence retrieval and fusion protein construction\n",
            "the amino acid sequences of pfceltos, human il- <dig> and m cell-targeting ligand, co <dig>  were retrieved from uniprot . amino acid sequences were used to design fusion protein construct. the designed construct consisted of pfceltos and human il- <dig> mature parts linked together by a flexible linker, whereas co <dig> is linked to this construct through a rigid  linker.\n",
            "\n",
            "primary structural analysis\n",
            "expasy’s prot param server  <cit>  was used to study the physiochemical characters of pfceltos and designed fusion constructs such as theoretical isoelectric point , molecular weight, molecular formula, total number of positive and negative residues, instability index  <cit> , aliphatic index  <cit>  and grand average hydropathicity   <cit> . the instability index provides an estimate of a protein’s stability in vitro. proteins with instability index smaller than  <dig> are predicted as stable. a value above  <dig> indicates that the protein may be unstable.\n",
            "\n",
            "the aliphatic index of a protein is regarded as a positive factor for the increase of thermostability of globular proteins and is mainly defined as the relative volume occupied by aliphatic side chains . the gravy score is calculated as the sum of hydropathy values of all the amino acids, divided by the number of residues in the sequence.\n",
            "\n",
            "secondary structure prediction\n",
            "secondary structure elements of pfceltos and designed fusion proteins were determined using phyre <dig>  <cit>  and i-tasser  <cit>  online servers.\n",
            "\n",
            "three-dimensional  model prediction\n",
            "the 3d structure of pfceltos and fusion proteins were modeled using i-tasser online server. the raw amino acid sequences of pfceltos and fusion proteins were uploaded in fasta format to i-tasser server. tertiary structures were predicted in pdb format. the results generated five top models for each entry which the one with the highest confidence score  represented the best model and was the structure selected for this study  <cit> .\n",
            "\n",
            "structure validation\n",
            "visualization was carried out by pymol version v <dig> . <dig> software  <cit> . traceable structural errors were proofed and global charges replaced by vega zz version  <dig> . <dig>  software  <cit> . ramachandran plots were calculated by rampage ramachandran plot assessment  <cit> . hydropathicity index were calculated by marvin view version  <dig> . <dig> software . molecular dynamics were performed by gromacs version  <dig> . <dig> software  <cit> . pdb file format were analyzed for md approaches-molecular dynamics algorithm including an initial cubic salvation with  <dig> point simple water model, followed by ionization and neutralization of simulation cube with na and cl ions. geometry optimization was performed with constrain method. this procedure continued with two separate temperature and pressure unconstrained global dynamics. final unconstrained dynamics performed with coupled temperature  and pressure  for 10 ns. a schematic sketch of molecular dynamics procedure is presented in table  <dig> table  <dig> a schematic sketch of molecular dynamics procedure\n",
            "\n",
            "\n",
            "\n",
            "analysis of simulation\n",
            "after the simulation had finished, the output data were analyzed according to energy , root-mean-square deviation , gyration radius and h bond formation/deformation. the following plot is represented as screening procedure of modeled structures .fig.  <dig> screening procedure of modeled structures\n",
            "\n",
            "\n",
            "\n",
            "RESULTS\n",
            "to our knowledge, the crystallographic structure of celtos has not been reported in the protein database. therefore, in silico analysis of celtos 3d structure could be of benefit in predicting its probable structure. computer prediction and simulation methods can be used to generate representative conformations of a molecule in equilibrium and provide a picture of the way in which a molecule changes from one configuration to another  <cit> . therefore, we carried out computational modeling and simulation in the hope of understanding the properties and structure of pfceltos. then we fused il- <dig> to pfceltos by a flexible linker and did in silico analysis to confirm the proper folding of each domain in the designed fusion protein. in the last step, co <dig> ligand was added to the confirmed fusion structure using a rigid linker and computational analysis was performed to evaluate the final fusion construct.\n",
            "\n",
            "primary structure analysis\n",
            "the primary structure of pfceltos  is composed of  <dig> amino acid residues, encoded by  <dig> nucleotides. a fragment of  <dig> amino acids at the n-terminal is a signal peptide and the next  <dig> amino acids fragment is a mature peptide. human il- <dig>  composed of  <dig> amino acid residues, encoded by  <dig> nucleotides. a fragment of  <dig> amino acids at the n-terminal was found to be a signal peptide and the remaining  <dig> amino acids fragment was a mature peptide  <cit> . the  <dig> amino acids fragment of pfceltos  and  <dig> amino acids fragment of il- <dig> were used to design the fusion protein construct. the length of the designed construct is  <dig> amino acids, which within  <dig> to  <dig> is il- <dig>  amino acids  <dig> to  <dig>  are linker sequence, which is  <dig>  amino acids  <dig> to  <dig>  are pfceltos. based on previous studies, n-terminal of il- <dig> and c-terminal of pfceltos seems more important in their biological function than the other end. therefore we designed the fusion construct in a way to make free the n- and c-terminals of il- <dig> and pfceltos, respectively. the co <dig> ligand peptide was added to the computationally confirmed structure of il-2-3-pfceltos. to present the co <dig> ligand to human m-like cells better, a rigid linker 3) was used between co <dig> ligand and designed fusion protein and the new construct was designed as follow: co1-3- il-2-3-pfceltos. figure  <dig> shows schematic view of designed fusion protein construct.fig.  <dig> schematic view of the designed fusion protein construct\n",
            "\n",
            "\n",
            "\n",
            "the primary structural features of pfceltos and two designed fusion constructs determined by prot param are summarized in table  <dig>  prot param is a tool which allows the computation of various physical and chemical parameters for a given protein. the calculated isoelectric points  for pfceltos, il-2-3-pfceltos and co1-3- il-2-3-pfceltos were  <dig> ,  <dig>  and  <dig> , respectively, suggesting the presence of more negatively charged residues. although, proteins were found to be stable due to their high aliphatic index, an instability index of more than  <dig> suggests instability and thus predicted that the proteins were thermally unstable. the value of aliphatic index for pfceltos, il-2-3-pfceltos and co1-3- il-2-3-pfceltos were  <dig> ,  <dig>  and  <dig> , respectively, while that of instability index were  <dig> ,  <dig>  and  <dig> . gravy  values of pfceltos, il-2-3-pfceltos and co1-3- il-2-3-pfceltos were − <dig> , − <dig>  and − <dig> , respectively, suggesting a hydrophilicity pattern with better interaction with water.table  <dig> properties of pfceltos, human il- <dig> and designed constructs determined by protparam\n",
            "\n",
            "\n",
            "\n",
            "secondary structure prediction\n",
            "among protein structure prediction software, the on-line servers seem more appropriate since they have access to all structures available in databases. secondary structure of pfceltos and designed fusion constructs were predicted by phyre <dig> and i-tasser servers. phyre <dig>  is among the most popular methods for protein structure prediction having been cited over  <dig> times and has been designed to ensure a user-friendly interface for users inexpert in protein structure prediction methods. both programs established that pfceltos structure is predominantly made of α-helices and coil–coils and there was no significant strand in pfceltos. the location of rigid linker fragment within amino acids 13– <dig> in co1-3- il-2-3-pfceltos was shown by helical structures. phyre <dig> prediction and analysis of secondary structure of proteins is shown in table  <dig>  in case of the fusion constructs also, amount of α-helices exceeded β-sheets. the presence of only 5 % β-sheets is related il- <dig> structure .table  <dig> phyre <dig> prediction and analysis of secondary structure\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "three-dimensional  structure prediction\n",
            "prediction of 3d structure was performed by i-tasser online server using the best aligned template obtained by searching against protein data bank database. i-tasser  server  is a web-based service for the prediction of protein structures and functions. it’s free for academic users and allows them to automatically generate high-quality models of 3d structure of proteins from their amino acid sequences. it detects structure templates from pdb by threading technique. i-tasser is one of the most successful servers in the community-wide casp  experiments and was ranked as the no  <dig> server for protein structure prediction in recent casp <dig> casp <dig>  casp <dig>  casp <dig>  and casp <dig> experiments.\n",
            "\n",
            "3d structure prediction by i-tasser generates five top models with c-scores ranging from − <dig> to  <dig>  the one with the highest c-score represents the best model. the highest value for predicted models of pfceltos, il-2-3-pfceltos and co1-3- il-2-3-pfceltos were − <dig> , − <dig>  and − <dig> , respectively. these models were selected for this study. the best selected model for pfceltos based on c-score represented that the 3d structure is consistent with secondary structure elements generated by phyre <dig>  based on phyre <dig> ss confidence, amino acids 3– <dig>  25– <dig>  32– <dig>  85– <dig> and 122– <dig> are α-helices with high confidence, which are also predicted as α-helix structures in 3d model of pfceltos generated by i-tasser . it is important to note that further fusion protein constructs linked by other flexible linkers were designed but the one with  <dig> linker resulted in the best 3d structure based on c-score. predicted model for il-2-3-pfceltos showed a protein with two separate domains linked together through a small linker. glycine-rich peptides confer flexibility, which allows the domains to move independently of one another while maintaining their individual three dimensional shapes  <cit> . 3d structure prediction of co1-3- il-2-3-pfceltos showed a protein with three domains linked together by two small linkers.  <dig> rigid linker keeps domains apart and ensure a relatively rigid separation of the domain and carrier  <cit> . the pre-simulated 3d structures of pfceltos and fusion proteins are shown in fig.  <dig> fig.  <dig> i-tasser predicted 3d structures of  pfceltos,  il-2-3-pfceltos and co1-3- il-2-3-pfceltos\n",
            "\n",
            "\n",
            "\n",
            "structural model refinement\n",
            "the structural refinement was carried out using molecular dynamics simulation as described in methods. simulation acts as a bridge between theory and experiment. indeed we test a theory by conducting a simulation using the same or computationally predicted models and provide a guess at the possible interactions between molecules  <cit> . gromacs is a molecular dynamics application developed by groningen university. gromacs is able to work in the operating system linux. the main ability of gromacs is to perform md simulation and minimization energy. however, gromacs does not work alone and should interact with pymol and grace. pymol is an application to visualize molecule structure and grace is an application in linux to display graphs. both applications support analysis of md simulation.\n",
            "\n",
            "the md simulation output data were analyzed based on energy , rmsd  <cit> , gyration radius  <cit>  and h bond formation/deformation  <cit> . the total energy should be constant throughout the simulation process, as it is the sum of kinetic and potential energy of the molecules. kinetic energy should be constant or following a decreasing trend since the constant increasing of kinetic energy level reflects the general confusion of protein structure. potential energy level should be increasing or constant to show the stability of structure. rmsd is the measure of the average distance between the backbone atoms of proteins. the structural refinement was carried out using molecular dynamics simulation over the equilibration course and exhibits rmsd plots for predicted models of pfceltos and fusion constructs that flattens during 10 ns. rmsd plots indicated that the proteins are stable during the simulation. gyration radius is an indicative of the level of compaction in the structure, i.e., how folded or unfolded is the protein. h bond formation/deformation factor represents the number of hydrogen bonds formed or broken during molecular simulation. during  <dig> ns of simulation, this number was almost constant indicating the stability of the molecular structures. figures  <dig>   <dig> and  <dig> represents simulation analysis of pfceltos, il-2-3-pfceltos and co1-3- il-2-3-pfceltos, respectively, that validated the accuracy of our designed fusion protein structure.fig.  <dig> structural model refinement analysis of pfceltos predicted structure. a energy plots for the molecular dynamics simulation of pfceltos predicted structure. b total rmsd, c the radius of gyration and d hydrogen bond formation/deformation of pfceltos predicted structure. rmsd of pfceltos stable conformation observed after the extension of a 10 ns simulation run\n",
            "\n",
            "fig.  <dig> structural model refinement analysis of il-2-3-pfceltos predicted structure. a total, potential and kinetic energies for md simulation of il-2-3-pfceltos predicted structure. b total rmsd, c the radius of gyration and d hydrogen bond formation/deformation of il-2-3-pfceltos predicted structure\n",
            "\n",
            "fig.  <dig> structural model refinement analysis of co1-3- il-2-3-pfceltos predicted structure. a total, potential and kinetic energies for md simulation of co1-3- il-2-3-pfceltos structural model refinement analysis. b total rmsd, c the radius of gyration and d hydrogen bond formation/deformation of co1-3- il-2-3-pfceltos predicted structure\n",
            "\n",
            "\n",
            "\n",
            "structure validation of predicted models\n",
            "to validate the modeled structures, ramachandran maps were drawn before as well as after md simulation and structures were analyzed using rampage ramachandran plot assessment. the ramchandran plot analysis obtained by rampage ramachandran plot assessment is summarized in table  <dig> and the plots are provided in fig.  <dig>  this provides an insight into the correctness of the modeled structures. as it’s obvious co1-3- il-2-3-pfceltos has a well modeled structure in terms of rampage ramachandran plot assessment.table  <dig> the ramchandran plot analysis of the proteins before and after md simulation obtained by rampage ramachandran plot assessment\n",
            "\n",
            "\n",
            "fig.  <dig> comparative diagram depicting ramchandran plot analysis of pfceltos an fusion proteins before and after molecular dynamic simulations  pfceltos  il-2-3-pfceltos  co1-3- il-2-3-pfceltos. the plots were analyzed by rampage ramachandran plot assessment. the general favored region and pro-pro favored region are indicated with dark blue color. the general allowed region and pro-pro allowed region is shown in pale blue. the glycine favored and allowed regions are shown in dark and pale orange, respectively. the disallowed region is in white color\n",
            "\n",
            "\n",
            "\n",
            "comparison of predicted structures before and after md simulation\n",
            "3d structures of pfceltos and fusion constructs were also investigated after md simulation . overlaying of 3d structure of pfceltos and designed fusion proteins before and after md simulation showed that the predicted structures are constantly stable and the selected linkers are able to separate the domains of designed fusion constructs effectively. as it’s obvious, rigid linker ensures the separation of domains and carrier and leads to the better presentation of construct to human m-like cells . structural comparison of predicted models of pfceltos and co1-3− il-2-3-pfceltos, obtained after molecular dynamics simulations is shown in fig.  <dig>  this comparison indicates the proper folding of pfceltos in fusion construct.fig.  <dig> 3d structures of  pfceltos,  il-2-3-pfceltos and  co1-3- il-2-3-pfceltos after molecular dynamics simulations\n",
            "\n",
            "fig.  <dig> structural comparison of predicted models for  pfceltos  il-2-3-pfceltos  co1-3- il-2-3-pfceltos, obtained before  and after molecular dynamics simulations  from different views\n",
            "\n",
            "fig.  <dig> structural comparison of predicted models of pfceltos and co1-3- il-2-3-pfceltos, obtained after molecular dynamics simulations\n",
            "\n",
            "\n",
            "\n",
            "CONCLUSIONS\n",
            "the procedure of our study  helps to rapid analysis of designed fusion constructs before initializing the recombinant fusion protein lab experiments. as it is obvious, this procedure is fast, inexpensive  and simple, especially for inexpert users in this field.\n",
            "\n",
            "in silico study of co1-3- il-2-3-pfceltos structure through this procedure revealed that designed construction is suitable for fusion protein expression in edible host cells for oral delivery. flexible linker separates pfceltos and il- <dig> domains effectively to maintain their proper individual three dimensional structures and allows them to move independently of one another. on the other hand, rigid linker ensures the separation of fusion protein and carrier and lead to the better presentation of fusion construct to human m-like cells. therefore, data reported in this paper represents the first step toward developing of an oral vaccine candidate against malaria infection.\n",
            "\n",
            "competing interests\n",
            "\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "authors’ contributions\n",
            "\n",
            "ss and ho designed the construct and simulations and performed the analyses. ss wrote the manuscript. ho supervised this work. both authors have reviewed and approved the final manuscript.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in silico analysis of the 3d structure of pfceltos and co1-3- il-2-pfceltos revealed that pfceltos and co1-3- il-2-pfceltos have a well modeled structure.\n",
            "in silico analysis of the 3d structure of pfceltos and co1-3- il-2-pfceltos revealed that pfceltos and co1-3- il-2-pfceltos have a well modeled structure.\n",
            "in silico analysis of the 3d structure of pfceltos and co1-3- il-2-pfceltos revealed that pfceltos and co1-3- il-2-pfceltos have a well modeled structure.\n",
            "in silico analysis of the 3d structure of pfceltos and co1-3- il-2-pfceltos revealed that pfceltos and co1-3- il-2-pfceltos have a well modeled structure.\n",
            "in silico analysis of the 3d structure of pfceltos and co1\n",
            "86\n",
            "[{'summary_text': 'the proposed method allows the identification of functionally relevant amino acid positions in a protein, based on the conservation of these amino acid properties.\\nthe proposed method allows the identification of functionally relevant amino acid positions in a protein, based on the conservation of these amino acid properties.\\nthe proposed method allows the identification of functionally relevant amino acid positions in a protein, based on the conservation of these amino acid properties.\\nthe proposed method allows the identification of functionally relevant amino acid positions in a protein family, based on the conservation of these amino acid properties.\\nthe proposed method allows the identification of functionally relevant amino acid positions in a protein family, based on the conservation of these amino acid properties.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 8330 to 9216 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "site-directed mutagenesis is a tool used in rational protein design strategies to modify the structure or function of a protein to adapt it to particular performance requirements. moreover, mutagenesis is a fundamental tool to study the relationship between protein structure and function, making possible the substitution of one amino acid by another, thus isolating the contribution of the original amino acid or the newly introduced amino acid to the structure and function of the protein as a whole  <cit> .\n",
            "\n",
            "however, site-directed mutagenesis-based rational protein design strategies present a widely recognized drawback. in order to introduce changes that could confer a desired function or characteristic to a protein, it is necessary to know, or at least to assume, something about the protein structure-function relationship. in other words, it is necessary to know, for each amino acid of the protein or at least for a select group of them, what is their particular contribution to the structure and function of the protein as a whole.\n",
            "\n",
            "even for a small protein, and assuming that only a subset of amino acids contribute to features really determinant for its relevant function, the number of amino acids to be considered is very large. for a medium-sized protein and with no additional information regarding the possible structure-function relationship, an exhaustive search is practically impossible  <cit> .\n",
            "\n",
            "it is commonly known that there are certain amino acids in a protein that are necessary and fundamental for its activity, function or structure, and there are other amino acids that are readily replaceable by amino acids sharing a common characteristic, without affecting the main features of the protein. therefore, certain amino acidic positions must conserve some unique properties, which are communicated to the entire molecule, for a given protein or protein family. when analyzing a protein family, many functionally important residues of proteins can be identified because they have been conserved during evolution. however, residues that vary can also be critically important if their variation is responsible for diversity of protein function and improved phenotypes. this adds an entirely new complexity level to the analysis  <cit> .\n",
            "\n",
            "the same type of functional variation can be observed in nature, represented by single nucleotide polymorphisms  in the coding region of a given gene. snps can be synonymous, often called silent mutations, or can substitute a particular amino acid for another in a protein primary sequence, which is referred as to a non-synonymous snp . prediction of the occurrence of nssnps in a gene could be easily done by comparing nucleotide sequences and detecting or predicting nucleotide changes that occur with low-probability incidence. however, predicting which of these mutations will have an observable effect on protein function is a much more difficult task. this is complicated by the fact that there is often no structure-function knowledge available about the protein. this variation in protein function can be subtle or lead to major phenotypic changes in living organisms. for instance, variations in the dna sequences of genes can affect how humans develop diseases and respond to pathogens, chemicals, drugs, vaccines, and other agents.\n",
            "\n",
            "when no structure-function information is available, it is relevant to determine these features only from the protein's amino acid sequence  <cit> . a number of bioinformatic algorithms have been devised previously to extract this type of information from the primary sequence of a protein  <cit> . some of these methods have been successfully used for the prediction of altered protein phenotypes caused by nssnps in protein genes  <cit> . other methods have been described to derive this information from 3d structural data or molecular dynamics results  <cit> . a different approach that incorporates machine-learning techniques has been used to study the results of directed evolution experiments in order to explore proteins and to derive hidden structural rules  <cit> .\n",
            "\n",
            "for this large-scale analysis, we propose an alternative, taking into account the contribution of each amino acid to the general structure of the protein through their characteristic physicochemical properties, such as hydrophobicity, side-chain volume and charge. each amino acid contributes with its own physicochemical characteristics to the entire protein, which adds to the characteristics of the other amino acids, thus determining the relevant features of the protein, both global and site-specific  <cit> .\n",
            "\n",
            "in this study, we have developed a general algorithm, named mutagenesis objective search and selection tool , which analyzes the target protein as part of a multiple alignment to determine which are the positions that could be mutated with or without altering the common characteristics of the protein family, and gives mutagenesis estimations related to the possibility of whether a given amino acid change would have deleterious effects on the protein. a variant of the same method can be used to detect phenotypically relevant nssnps in a gene family and separate them from amino acid substitutions that do not have functional implications.\n",
            "\n",
            "RESULTS\n",
            "algorithm\n",
            "as a summary of this work, the general algorithmic procedure is presented in figure  <dig>  the left end set of the global algorithm corresponds to the information delivered to select and design site-directed mutagenesis strategies, while the right end set represents the key information used to identify functionally significant nssnps using the tools and statistical procedures proposed in this paper. both analytic procedures share the same initial steps, but have subtle differences that are exploited to get the most appropriate results for each application.\n",
            "\n",
            "background of the algorithm\n",
            "a general consensus in protein science states that some amino acid physicochemical properties are conserved at particular positions in a given protein family and some others are not. this explains the possible variation that can be found in any protein family to achieve the same function and it also shows that, despite the possibility of variation, there are particular characteristics that must be retained in order for the protein to conserve its own distinctive features.\n",
            "\n",
            "each macroscopic property of a protein is controlled by different amino acids in its sequence with a higher or lower degree of influence. hence, both within the framework of site-directed mutagenesis experiments and in the detection and prediction of phenotypic effects of nssnps, it is interesting to identify amino acid residues that are relevant for a protein's function, specificity, stability or binding interactions.\n",
            "\n",
            "the proposed algorithm has been designed to analyze the general conservation of each relevant physicochemical characteristic at each amino acid position in a protein family . in this context, a protein family is predefined by selecting relevant proteins according to arbitrary criteria . a protein family can also be subdivided into subfamilies, in order to study the contribution of particular amino acids to differences between subfamilies.\n",
            "\n",
            "preprocessing of the target sequence\n",
            "to compare the proteins belonging to a family, a multiple alignment of the amino acid sequences is carried out. the most used algorithm for these multiple alignments is clustalw, developed by thompson et al.  <cit>  in a multiple alignment, the algorithm tends to retain and conserve the amino acids in each position, using a weight matrix . an optimal alignment is obtained using these general matrices, which guarantee that the properties as a whole are maximally conserved at each amino acid position of the alignment.\n",
            "\n",
            "the preprocessing of the target protein sequence is represented by steps  <dig> and  <dig> in figure  <dig>  the mosst algorithm subsequently analyzes this multiple alignment to determine and quantify the statistical significance of the conservation of different physicochemical properties in particular positions of the protein family alignment. to perform this analysis, the conservation of each property is independently studied and patterns are identified by comparing the conservation of different properties at each amino acid position. typical conservation cases are defined based on these conservation patterns, which are then used to statistically classify the amino acids as relevant or irrelevant with respect to the conservation of any physicochemical property in the protein and the predicted amino acid mutability in the protein family. then the statistical parameters  determined for the conservation of each amino acid position are used as predictors to classify possible mutations in the protein family as functionally impairing or functionally silent. hence, these results can then be interpreted and sorted to design mutagenesis strategies or to identify nssnps.\n",
            "\n",
            "control of the quality of the multiple alignment\n",
            "the proposed method allows identification of functionally relevant positions in a particular protein when the protein is placed within a comparative reference group in a multiple alignment. the classification procedure gives unique results for each particular alignment, but the statistical result for each position will strongly depend on the quality of the alignment and nothing prevents a priori this significance level varying between one alignment and another.\n",
            "\n",
            "this variation must be controlled so that the comparison gives coherent and reproducible results when confronted to minor changes in the comparison alignment. the proposed control of this variation, represented by step  <dig> in figure  <dig>  is done in two ways:\n",
            "\n",
            "alignment quality control\n",
            "the quality of a multiple alignment specifically depends on the substitution matrices used and the negative weight contributions of gap creation and extension. in the test performed to check the proposed algorithm, the alignments obtained using default clustalw parameters gave satisfactory results.\n",
            "\n",
            "control of the representativity and the redundancy of the alignment\n",
            "in the proposed analysis algorithm, the group of proteins in the alignment is supposed to be representative of a wider protein group. if the protein group does not include representatives of some subgroup that has some special feature, then such feature evidently cannot be included in the analysis. this sub-representation is impossible to detect by purely mathematical methods, but can be solved by judiciously choosing the proteins to be represented and analyzed, which is a task that has to be done by a suitable expert. on the other hand, if a subgroup of proteins is over-represented in the analyzed group, then the protein sample will be biased towards the characteristics of the most represented subgroup, ignoring the characteristics of the other subgroups. the over-representation of a subgroup in the sample is readily detectable, as it is possible to assess the percentage of similarity between all the proteins in a group. if a particular group shows a high similarity degree, i.e. high redundancy, then it is over-represented in the sample, and the number of proteins in that group could be reduced by eliminating the most similar proteins.\n",
            "\n",
            "once a representative and non-redundant group of proteins is found, mosst can be used to detect functionally relevant positions.\n",
            "\n",
            "statistical quantification of the conservation of properties\n",
            "amino acids have physical and chemical properties that can be directly or indirectly measured, e.g. hydrophobicity, polarity, charge, molecular shape, conformational propensity and others. a numerical value is typically assigned for the measured property to each amino acid, whether free or associated to others in a polypeptide chain. as side chains are different for each of the  <dig> naturally occurring amino acids, these numerical values are different for each amino acid and provide a quantitative ordering between them with respect to the measured property. hence, these property values form a scale that represents a numerical scoring for each amino acid. many of these amino acid properties have been compiled in multiple non-independent \"amino acid scales\"  <cit> .\n",
            "\n",
            "in this way, every amino acid is assigned a numerical score that quantifies a particular physicochemical property Ω . hence, any group of n amino acids will have n associated scores, xΩ, <dig> ..xΩ,n. from these scores, an arithmetical average μΩ and a sample variance estimator σΩ can be calculated  as:      \n",
            "\n",
            "a multiple alignment of protein amino acid sequences, such as that obtained by using clustalw, comprises a given number of n positions. each position i contains a group of ni optimally aligned amino acids, each from a different protein . as the multiple alignment at position i can include gaps, ni is not always equal to the number of proteins in the alignment. for example, in the ninth position  of the alignment in figure  <dig>  n <dig> =  <dig>  hence, for any given amino acid property Ω, an average μi,Ω,n and a variance σi,Ω,n can be calculated for each position i in a multiple alignment with n amino acids, using eq. <dig> and eq. <dig>  respectively.\n",
            "\n",
            "the variance σi,Ω,n, as its name suggests, quantifies the variability of the property Ω in position i. if σi,Ω,n is small, a large variation in the property Ω is not allowed in position i, and vice versa. therefore, σi,Ω,n can be used to quantify the conservation of the property Ω at that position. however, σi,Ω,n depends on the scale used to measure Ω and on the number n of amino acids compared. hence, it cannot be used to compare the conservation of different properties or different quantification scales, or to compare the conservation between positions with different numbers of amino acids. a different scale-free and position-free standard parameter has to be calculated to quantify and compare the conservation of properties in multiple alignments.\n",
            "\n",
            "in the general case, there are  <dig> different amino acids and therefore for a given number of n amino acids there are 20n different random amino acid combinations. for any given property, it is possible to calculate all the 20n possible combinations of n amino acids and their associated averages and variances. if amino acid combinations are random, any particular combination will have a probability of occurrence equal to 1/20n. therefore, both the average and variance associated to each combination will have a probability of occurrence equal to 1/20n. particularly, for a random combination of n amino acids and any given amino acid property Ω, a discrete probability mass function  for the variance can be constructed. the function pmfΩ,n gives the probability that the variance σ of the property Ω calculated for a group of n amino acids is exactly equal to x  <cit> .\n",
            "\n",
            "from the pmfΩ,n for the variance of the property Ω, a cumulative distribution function  can be obtained  <cit> . as shown in figure  <dig>  if the variance σΩ = x, then cdfΩ,n  = pΩ is the probability of the variance of a random combination of n amino acids being between  <dig> and x. thus, if pi,Ω,n in a given position i in a multiple alignment is small, then the probability of said variance having a value equal or less than x is very small in comparison with the probabilities of all the possible variances in a random combination of n amino acids, and vice versa. for example, a value of pi,Ω,n =  <dig>  means that the obtained variance σi,Ω,n  only occurs in a proportion of  <dig> to  <dig>  in random combinations of n amino acids.\n",
            "\n",
            "the significance of variances for every position i in the target multiple protein alignment are calculated in step  <dig> of the algorithm . notably, if pi,Ω,n in a given position i in a multiple alignment is small, the variance σi,Ω,n of the amino acid property under analysis in that position is simultaneously small and relatively uncommon, i.e. the amino acid selection in that position is not random. hence, a small pi,Ω,n implies that the property value must be relatively invariable  at position i, and vice versa. the closer to  <dig> is the value of pi,Ω,n, the more significant this variance is and the less random the amino acid group is at position i.\n",
            "\n",
            "the probability pi,Ω,n is an indicator of the degree of conservation of any property Ω at each position i in a multiple alignment, contrary to the use of the variance alone. the advantages of this conservation measure are, firstly, its independence with regard to the number n of amino acids in the comparison  and, secondly, the possibility of performing scale-free and position-free comparisons between conservations at different positions and using different scales or properties. interestingly, the probability value pi,Ω,n could also be used as a significance value in hypothesis tests for the obtained variance σi,Ω,n.\n",
            "\n",
            "determination of mutationally relevant positions\n",
            "ideally, each property Ω is unique and mutually independent, and its contribution to the global structure and function of a protein is relatively important. for an amino acid position i in a multiple alignment, and given a cumulative probability pi,Ω,n for each amino acid property, we define three possible cases into which the relationships between the different probabilities pi,Ω,n for each amino acid property at a position i could be classified a priori. in figure  <dig>  these three possible cases are shown with reference to three different amino acid scales. these three cases, which are defined in the following paragraphs, can be used as a basis to determine good mutagenesis objectives.\n",
            "\n",
            "invariable determinant position \n",
            "the first case is defined by obtaining a high significance level, i.e. a very low pi,Ω,n value, for every amino acid property in a position i. this implies a relevant conservation of all the properties at that position. these positions are usually identified in multiple alignments as those positions that stringently accept only one or sometimes two different specific amino acids and are thus readily identifiable and visible using simple methods. the change of one amino acid for another in the analyzed position would probably imply a drastic change in the value of one or more properties and therefore alter the conserved necessity for invariability at that position, causing a partial or total loss of some of the characteristic properties of the protein family. if it is desired to conserve such properties, e.g. conservation of catalytic function or interaction sites, as would be the general case in conservative site-directed mutagenesis studies, then these positions should not be considered as possible mutagenesis objectives. a mutation in this type of position is usually deleterious for the protein function in nssnps.\n",
            "\n",
            "variable irrelevant position \n",
            "the second typical case is when all the probabilities pi,Ω,n associated to variances for all the amino acid properties at position i are very high. here, the significance of conservation of those properties at that position is very low, i.e. conservation is irrelevant. the replacement of one amino acid by another in any of these positions should not be determinant for the protein family, as the conservation of any particular characteristic is not required. in other words, the amino acids present at these positions do not contribute fundamentally to any relevant characteristic of the protein family. therefore, it is expected that mutagenesis at a variable irrelevant position should not have a large effect on the characteristics of a protein family. a nssnp with these characteristics is probably silent and does not affect protein function.\n",
            "\n",
            "variable determinant position \n",
            "the third case corresponds to the case where, at the analyzed position i, there is a high conservation, i.e. a high pi,Ω,n significance, in one property and a very low significance or conservation in the remaining properties. then, for this position there is a very high tendency to conserve one of the independent amino acid properties, which are necessarily privileged, and a simultaneous tendency to variability in the other groups of rather irrelevant properties. thus, this position is variable in the sense of admittance of global variability of some characteristics, but nevertheless is determinant for possession of a conserved group of characteristics, which have to be present to communicate common characteristics to the protein family. this makes this kind of position a main target to be mutated. a nssnp in this type of position can affect protein function in a rather unpredictable but probably very determinant way.\n",
            "\n",
            "a conservative mutagenesis strategy that aims to preserve the main functional or interactional properties that characterize the protein family and to selectively alter secondary traits should be focused on mutating only vdps, while a non-conservative mutagenesis strategy could also include idps. if the aim is to identify nssnps, mutations that could alter protein function are most probably located at idps and vdps. whichever the task and the expected result would be, it is evidently useful to identify vdps, given their allowed amino acid variability and the simultaneous relative importance of conservation of one property in such position in the entire protein family. however, vdps are much more difficult to find than idps, which makes vdps prime \"hidden\" functionally relevant positions that cannot be readily identified by other existing automatic methods. the following sections will describe the proposed method to identify these positions.\n",
            "\n",
            "identification of variable determinant positions\n",
            "we described previously that the determination of the significance values pi,Ω,n for each position i in a multiple alignment is based on the cdfs of the variances of random combinations of n amino acids. after cdfs are calculated for the variance of each amino acid property, the significance pi,Ω,n associated to each property can be calculated at each position.\n",
            "\n",
            "with these pi,Ω,n values we can identify in an alignment the vdps described previously as hidden functionally determinant positions where one of the properties is conserved, while others are not. to identify such positions , it is necessary to determine whether significant differences exist between the variances of the different amino acid properties at a given position i. in fact, differences between the variance of one property and the variances of every other could be used as indicators of the presence of conservation of a single property in position i. however, only significant  differences will indicate that position i is indeed a vdp. hence, in the following section, we develop a method that uses hypothesis testing to identify vdps by quantifying the magnitude of the significance of such differences and assigning a probability to classify position i as a vdp.\n",
            "\n",
            "hypothesis test for differences\n",
            "in the same way that probabilities pΩ can be calculated and cumulative distribution functions can be constructed for the variance of any given property Ω, a probability p for all the possible differences Δσ between the variances of any two properties Ω <dig> and Ω <dig> can be found . the probability value p is equivalent to the probability of the absolute difference between variances being Δσ or less. from these differences and their associated probabilities, a corresponding cdf,n can be calculated for a group of n amino acids, considering all the possible random combinations of variances and their differences .\n",
            "\n",
            "therefore, for every position i with n amino acids in a multiple alignment, a hypothesis test can be devised to test whether a given difference between the variances of two given properties Ω <dig> and Ω <dig> is not random, i.e. Δσi,,n is significant. the null and alternative hypotheses for this test are, respectively:\n",
            "\n",
            "h0: the variances of both amino acid properties are equal ,n is random).\n",
            "\n",
            "h1: the variances of both amino acid properties are different ,n is not random).\n",
            "\n",
            "thus, cdf,n is the distribution function of the test statistic Δσi,,n under the null hypothesis, i.e. random differences between the variances. hence, cdf,n ,n = x) = pi,,n is the probability of the difference between the variances of two properties Ω <dig> and Ω <dig> in a random combination of n amino acids being less than x . therefore, p'i,,n =  <dig> - pi,,n is the probability  of the null hypothesis for the properties Ω <dig> and Ω <dig> . for very low values, under certain arbitrary limits, the null hypothesis is not significant and could be rejected in favor of the alternative hypothesis.\n",
            "\n",
            "the partial significance of the differences between each pair of properties is calculated in step  <dig> of the algorithm. however, as there are many different properties Ω, for each position i in a multiple alignment, each pair of properties will have a corresponding significance value for the difference between their variances. none of these differences or their related significance levels could be seen mathematically as the significance of the position by itself. the significance of the position must integrate all the individual significances in a global significance value. if the null hypothesis is significantly rejected for one single property when compared with all the other properties, position i can most likely be identified as a vdp. in particular, to identify a vdp the global significance must be maximal when all differences except one are significant.\n",
            "\n",
            "integration of the individual significances\n",
            "for a given position i in the alignment comprising a group of n amino acids, every possible pair of properties Ωr and Ωs will have a difference Δσi,,n between their respective variances, and a significance level p'i,,n associated to this difference. each comparison between any pair of properties gives information to prove or disprove that the variances of the properties are different. therefore, every p'i,,n can be considered as the significance level of a single experiment or sampling test to reject the null hypothesis at position i. however, the level of significance of the global null hypothesis must take into account the level of significance of all the differences between every pair of property variances. therefore, none of the p'i,,n alone is able to represent the significance of the position i on its own.\n",
            "\n",
            "statistically, the significance levels obtained from different experiments to demonstrate the same hypothesis could be combined in order to get a composite level of significance, as if a single experiment would be done with the combined evidence of all those partial experiments. this combination could be done using different procedures according to the features of the particular cases . in our case, the differences of variances between each possible pair of properties are not independent, since the difference between the last pair of properties is a linear function of the others, so the fisher procedure cannot be used. from the other two methods, the simes procedure gives the best results, as its rejection region contains the rejection region of the bonferroni methods and it is always more powerful than those, specially in the case of highly dependent test statistics  <cit> , which is suspected to be our case.\n",
            "\n",
            "we propose the use of the simes method to combine the significance values of the differences of variances between all the properties, in order to obtain a global significance value for the differences at position i . the simes method can be applied in a simple way: if p'm = p' <dig>  p' <dig> ... p'm is the value of the level of significance of each difference of variances between all the pairwise combinations of properties, in ascending order, the value of the global significance that considers all the individual partial significances for position i is:   \n",
            "\n",
            "the global probability pi represents the probability of the null hypothesis being true, given the values obtained for the differences of variances between different properties. in our case, pi is the probability of all the variances being equal. the lower this probability, the higher the accuracy with which the null hypothesis could be rejected, i.e. it is more probable that the variances of the different properties at position i in the analyzed multiple alignment could be different. hence, low pi values could be used to directly identify vdps in a multiple alignment .\n",
            "\n",
            "determination of the rejection regions for the null hypothesis\n",
            "statistical practice classifies the levels of significance of a hypothesis test in different categorical levels, usually \"not significant\", \"significant\" and sometimes \"very significant\". this categorization is performed using a criterium that imposes an arbitrary limit between one significance level and another. usually, a value of  <dig>  is the limit between \"non significant\" and \"significant\" levels. however, multiple experimental implementations of the proposed algorithm indicate that this limit is not stringent enough to correctly classify vdps apart from idps and vips. preliminary evidence shows that positions in a multiple alignment could be roughly classified into one of the following categories according to their significance levels: not significant , significant  or very significant .\n",
            "\n",
            "with this classification, critical values for the rejection region of the null hypothesis are determined  and vdps in the target protein are those positions identified as significant or very significant . accordingly, the other positions are classified as idps, which cannot be mutated without radically altering the characteristics of the analyzed protein family, and vips, which are not good mutagenesis targets because they are \"filling\" amino acids or amino acids that contribute with indiscernible and different characteristics to each particular protein.\n",
            "\n",
            "identification of invariable determinant positions\n",
            "the identification of idps can be done much more easily than the identification of vdps. normally, idps can be directly identified by visual inspection and looking for complete or almost complete amino acid conservations at any given position. however, the identification of idps can be readily implemented as a part of the mosst algorithm, using the same tools previously developed to identify vdps. in fact, according to figure  <dig>  idps have two characteristic features: the variances of all the amino acid properties must have very low values  and the differences between those variances must also be very small . therefore, any position i for which the global probability pi is classified as not significant and the probability pi,Ω,n for every Ω is classified as significant, must be an idp .\n",
            "\n",
            "determination of the possibilities of mutation for each position\n",
            "once the amino acid positions in a protein whose mutation could be functionally significant have been identified, the possible effects of amino acid substitutions at such positions are determined in a further step of the mosst algorithm. a distribution analysis of the different amino acid properties at said position is done in order to identify a priori possible functionally relevant or functionally silent amino acid changes in the protein, either to guide a rational mutagenesis strategy or to identify non-evident nssnps .\n",
            "\n",
            "the proposed method determines the probability of an amino acid being present at a determined position using a statistical method based on the student t-test. this procedure starts with the previously calculated average  and variance estimator  at a position i of the multiple alignment. for each position, each amino acid property is assumed to have a normal distribution n, with unknown parameters μ and σ. this assumption is reasonable, because it is assumed that at each position the average of each scale would represent the ideal characteristic to be fulfilled at that position and the standard deviation would represent the accuracy level of conservation of the characteristic.\n",
            "\n",
            "for a sample of n values xi,Ω, <dig> ..xi,Ω,n obtained for an amino acid property Ω at a given position i, assuming a normal distribution, it is possible to predict the distribution of a new value xi,Ω,n+ <dig>  mathematically, if we have a normally-distributed variable x sample mean value μi,Ω,n and a sample variance σi,Ω,n for a position i, then the test statistic:   \n",
            "\n",
            "has a student's t-distribution with n- <dig> degrees of freedom  <cit> . as the probability distribution of τi,Ω,n+ <dig> is known, it is possible to calculate the distribution of xi,Ω,n+ <dig> as:   \n",
            "\n",
            " is the posterior cumulative distribution function for the values of the property Ω at position i, given the previously known group of n amino acids present in the multiple alignment at such position. from this , a probability density function  can be derived. since each amino acid naturally has one particular value of property Ω,  allows the assignment of a probability qj,i,Ω,n+ <dig> to each amino acid j  to be present at position i in a new protein not included in the multiple alignment . this probability is directly associated with the property Ω and its distribution .\n",
            "\n",
            "as each amino acid can have many different properties Ω <dig>  Ω <dig> .. Ωl:, and each property will have a different pdf , then each amino acid will have l associated individual probabilities to be present at position i, which we will call  for each amino acid j. the combined global probability qj,i for the occurrence of each amino acid j at position i is calculated as:   \n",
            "\n",
            "this combined global probability represents the probability of every particular amino acid satisfying the requirements of all properties at position i, and therefore is an indicator of the functional suitability of each amino acid in such position.\n",
            "\n",
            "quantification of the functional suitability of the amino acids\n",
            "the former multi-property consensus method allows calculation of the global probability of occurrence of each amino acid at each position in a multiple alignment. the probability value can also be used as an indicator of the functional impairment introduced by each amino acid at every analyzed position i. if the requirements of the functional properties that have to be fulfilled in position i lead to a high probability for an amino acid to be in that position, it means that the amino acid does not impair the function of the protein. inversely, if the probability is low, such amino acid can be functionally deleterious.\n",
            "\n",
            "for this last step of the analysis, a method based on the sorting of these probabilities of occurrence in descending order and the construction of a scree plot with the ordered values can be used. an example of this procedure is shown in figure  <dig>  the scree plot at each position i could help in getting the preference level or reliance with which the analyzed position could be occupied by each of the amino acids, based on the required physicochemical properties at such position in the protein. an analysis of the scree plot and its curvature can serve to identify a cut point  in the curve. this point can be determined by using a \"fall contrast\" or \"scree\" criterium  <cit> , where the highest probability factors are chosen up to a point where the curve becomes approximately horizontal. a second criterium can also be used, where the highest probability factors are chosen to explain together at least a predefined high probability  or until the last factor has a non-significant probability . any criterium will define two sections in the scree plot: on the left a curve section including an amino acid group with the highest probabilities and on the right a section with those having the lowest probabilities. this classification allows separation of a high-probability group comprising functionally non-disruptive amino acids from a low-probability group of functionally deleterious amino acids. hence, qj,i can be used as a basis to identify non-evident nssnps and to design rational site-directed mutagenesis strategies for each position i in a protein.\n",
            "\n",
            "implementation of the algorithm\n",
            "three main issues were solved to implement the algorithm. these aspects will not be analyzed in detail in this paper, because they are not directly related to the algorithm theory and design, although they must be solved to get a good implementation thereof.\n",
            "\n",
            "the first aspect is the selection of suitable property vectors that could be successfully used in order to get the most accurate information from the mosst algorithm and to exploit the classification abilities of the algorithm in an optimal way. though the general algorithm has been presented as being able to use any group of property vectors to perform the proposed analysis, we selected a minimal number of three normalized vectors that are mathematically orthogonal to each other to optimize the statistical potency of the results.\n",
            "\n",
            "the second issue is the calculation of the probability distributions of the averages and variances of the selected optimal vectors, which are necessary for the implementation of the algorithm as set forth in the present paper. we have assessed that the law of large numbers is not accurately applicable for combinations of less than  <dig> amino acids given the optimal vectors previously obtained. even though the probabilities of the different amino acid combinations could be calculated in a direct way, it is computationally difficult and time-consuming for combinations of more than  <dig> or  <dig> amino acids. we developed a direct calculation method to get an approximate distribution for combinations from  <dig> to  <dig> amino acids, which analysis and explanation is out of the scope of this work.\n",
            "\n",
            "the last aspect is the determination of the right significance levels that should be taken into account to classify each position as \"non significant\", \"significant\" and \"very significant\" when determining the rejection regions for the null hypothesis of equality of variances. as we mentioned earlier, values of  <dig>  and  <dig>  could be preliminary used to select between these classifications, which are more stringent than those values usually used in statistical testing. however, our experience implementing the algorithm has shown that the most interesting positions in a protein alignment  have to be determined case by case, based on characteristics of each different multiple alignment. in our experience, the implementation of the algorithm included this data analysis in order to get the most out of the results of the general algorithm.\n",
            "\n",
            "the mosst algorithm has been implemented with the details mentioned above as a graphical user interface  in matlab . given the nature of mosst results, it is easier to display them in a graphical interface in order to allow the user to have an integral vision over them and to modify the parameters of the algorithm on-the-run to obtain the best possible results for each given analysis. one of the major advantages we devised from the use of an interactive gui is the definition of \"primary positions\". we defined primary positions as those \"very significant\" positions that have exceptionally high mutability scores as defined by the statistic theory behind the algorithm. by using mosst gui, the user can select both manually or semi-automatically the threshold significance value above which a \"very significant\" position becomes a \"primary position\" and thus a primary mutagenesis objective.\n",
            "\n",
            "the semi-automatic method advantage over the manual method is that it helps the user to select a suitable threshold by presenting a differential scree plot where the user can interactively select the threshold by comparing the differences  between similar ordered significance values. the optimum threshold will be a \"very significant\" significance value having the largest possible difference with immediately higher  significance values. if two or more significance values fulfill these criteria, the largest one is most appropriately selected. this empirical rule is applied by the user with his/her own judgment to select a proper threshold value.\n",
            "\n",
            "the empirical selection of \"primary positions\" proved to be an additional tool to narrow the search for mutable positions in a protein family. an example of the relevance of determining primary positions is shown in the study of glycosyl hydrolases belonging to family  <dig> \n",
            "\n",
            "testing: mutagenesis analysis of endoglucanases belonging to family  <dig> of the glycosyl hydrolases\n",
            "selection of representative proteins\n",
            "mosst was used for the analysis of representative endoglucanases classified as glycosyl hydrolases belonging to family  <dig>  they have several appropriate characteristics:  a very close structural and folding similarity despite them not having a very high degree of sequence similarity,  well characterized residues that participate in the interaction with the substrates at the active site and define the substrate specificity of the family,  well characterized active residues that are indispensable for catalysis,  two well defined and populated enzyme families with distinct substrate specificities and no known structure-function correlations, namely lichenases  and laminarinases -β-d-glucanases, ec  <dig> . <dig> ), and  large availability of experimental data, including mutagenic studies. these characteristics were considered ideal to test the ability of mosst to determine functionally relevant amino acid positions against experimentally tested results, since this allows the analysis of amino acid mutations purely by the effect of their side groups, while leaving other factors  outside of the analysis.\n",
            "\n",
            "among these enzymes, we selected cellulosimicrobium cellulans's bglii as our reference enzyme. proteins included in the analysis were selected by their similarity with bglii and both the conservation of folding pattern and enzymatic activity. table  <dig> shows the representative set of proteins selected and included in this study.\n",
            "\n",
            "descriptor codes in the multiple alignment are highlighted in bold characters.\n",
            "\n",
            "obtention of a multiple alignment\n",
            "following the flow diagram of the mosst algorithm, these proteins were aligned using clustalw . preliminary tests using different substitution matrices and different gap opening and extension penalties were performed. the combination that exhibited best results was obtained using an identity matrix as substitution matrix and minimal values for gap creation and extension. this combination empirically gives more importance to conserved sectors, which in our case correspond to secondary structure and folding sectors that are conserved in the family. this strategy is concordant with the fact that the studied group of proteins is a very structurally conserved enzyme family, even when conservation is lower at the sequence level due to the difference in activities of the proteins.\n",
            "\n",
            "redundancy analysis\n",
            "for this objective, a protein clustering dendrogram and an agglomeration plot were constructed according to the similarity percentage between the proteins in the multiple alignment . an optimal separation between sub-families is obtained using a similarity percentage cutoff of 86%. three groups of similar enzymes were represented by only one of them:  synthetic hybrid lichenases  <dig> and  <dig> from bacillus macerans / bacillus amyloliquefaciens ,  bacillus amyloliquefaciens  and bacillus subtilis  lichenases, and  two enzymes from rhodothermus marinus, one of them classified as a laminarinase  and the other classified as a lichenase . the remaining proteins, even those belonging to the same species, are considered different enough to be included in the analysis with no added redundancy.\n",
            "\n",
            "it can be observed in the dendrogram a subdivision of the proteins in the alignment in two large sub-families corresponding mainly to the division between lichenases  and laminarinases . this observation indicates that the information contained in the multiple alignment is also able to discriminate between functional aspects of the enzymes.\n",
            "\n",
            "calculation of the statistic values and significances\n",
            "for each of the positions of the non-redundant multiple alignment, the associated variance in each of the three principal components and the differences between the three components were calculated. the values of said differences were used to calculate the global significance of each position and to classify said significance according to the criteria exposed in the previous paper of this series. this allowed the classification of positions in the alignment as \"non-significant\", \"significant\" and \"very significant\". in addition, a new category of \"primary position\" was defined when the global significance of the position is included in the higher percentile of the values in the distribution of global variances as described in the practical implementation of mosst analysis.\n",
            "\n",
            "structural mapping of primary positions\n",
            "the determined primary positions can be mapped over a model of the tridimensional structure of bglii. in this structural mapping a spatial distribution pattern of sectors in which variable determinant positions are concentrated can be observed, i.e. the most promising mutagenesis targets for this protein family . the active site of the protein family is comprised widely by amino acids in positions 45- <dig> , 167- <dig> , 187- <dig> , 205- <dig> , 214- <dig> , 307- <dig>  and 315- <dig>  in the multiple alignment. a detailed analysis of active site amino acids according to the algorithm implemented in this work is shown in figure  <dig>  inside the active site, variable determinant positions were identified as the following amino acids using bglii numbering : leu <dig> , trp <dig> , phe <dig> , met <dig> , leu <dig> , gly <dig> , met <dig> , gly <dig> , val <dig> , his <dig> , gly <dig> , phe <dig> , phe <dig> , leu <dig>  and ala <dig> .\n",
            "\n",
            "predictive value of the algorithm: comparative analysis of laminarinases\n",
            "the mosst algorithm can be applied only to an enzyme sub-family. in our case, these sub-families can be determined from the redundancy removal dendrogram and are associated with enzyme function differences, as mentioned earlier. it is interesting to apply this analysis procedure to laminarinases and compare these results with the variable determinant positions obtained for the entire family. therefore, variable determinant positions were determined for laminarinases using a similar procedure to that used for the total protein family, and these positions were compared with those obtained for all family  <dig> glucanases. the active site structure of bglii is represented in figure  <dig> with differentiated variable determinant amino acids for family  <dig> glucanases and variable determinant positions for the laminarinase sub-family. logically, if a position in the active site is variable for laminarinases as a family then said position cannot be significant to determine substrate specificity of these enzymes. instead, if a position is variable determinant for family  <dig> glucanases but not for laminarinases, then it is logical to assume that said position could be relevant in the determination of laminarinase specificity when compared with the other enzymes of the same family.\n",
            "\n",
            "under this assumption, the comparative analysis using the mosst algorithm shows that the following active site amino acids  can have a determinant function over laminarinase specificity: leu <dig>  trp <dig>  gly <dig>  gly <dig>  phe <dig>  phe <dig> and ala <dig>  it is interesting to asses that some of these amino acids have been identified in the analysis of the molecular model of bglii and other family  <dig> glucanases as amino acids that participate in substrate binding and/or interactions  <cit> . this observation corroborates the ability of the mosst algorithm to identify relevant positions in a protein.\n",
            "\n",
            "predictive value of the algorithm: analysis of histidine 129\n",
            "position  <dig> , which corresponds to histidine  <dig> in bglii, is very remarkable. this position has been identified in the structural analysis of bglii as an amino acid that could be implied in the chemical reactions catalyzed by this enzyme  <cit> .\n",
            "\n",
            "the mosst algorithm marks position  <dig>  i.e. histidine  <dig>  in bglii as a primary mutagenesis objective and therefore this fact can be exploited to obtain interesting variants of the enzyme, possibly having different catalytic properties due to the position occupied by said amino acid in the active site. it should be noted that this histidine is classified as a primary amino acid both for the global family of proteins under study and for laminarinases, and so a possible substrate interaction function should be discarded, at least in a first approach.\n",
            "\n",
            "mosst algorithm suggests for said position the presence of a histidine residue  and the alternative presence of a phenylalanine residue  or a glutamine residue . the remaining amino acid have probabilities of less than 1% each and their combined probabilities only adds to  <dig> %, so their presence is not considered relevant in said position. the real nature and contribution of each proposed alternative could be analyzed using other predictive techniques, in advance to mutagenic experiments, in order to decide which of said variants would be the most favorable one to obtain the desired effects in the protein. it is also possible to analyze all the proposed alternatives, producing in the laboratory the corresponding mutant enzymes and analyzing their new properties.\n",
            "\n",
            "the contribution of this amino acid and the possible mutations performed on it to the enzymatic catalysis could be assessed by means of an additional analysis, for example, a kinetic analysis of a catalytic model. in a former work  <cit> , using structural and mechanistic knowledge about glycosyl hydrolases from families  <dig>   <dig>  and  <dig>  we have formulated a mathematical model that can include ionizable residues in the active site and incorporating electrostatic influences via acid dissociation equilibrium constants and chemical relationships such as hydrogen bonds. the results of the simulations indicated a clear shift in the ph dependence of activity for the enzymes only when a close interrelation  between a catalytic glutamate and histidine  <dig> is taken into account, which is concordant with experimental evidence with bglii . moreover, we demonstrated that the presence of the kind of chemical interaction proposed could provide stabilization of the activity in the presence of environmental, structural, ph and electrostatic variations. the results suggested a new way to modify, via site-directed mutagenesis, the acid dissociation of one of the catalytic residues in the active site independently of the other, which could have clear advantages over the purely electrostatic modifications that usually affect both residues simultaneously.\n",
            "\n",
            "under the light of this previous work, mosst analysis results were very striking in their ability to identify this residue as relevant even without any previous knowledge or suspicion about its relevance or function in this enzyme family. in our opinion, this clearly demonstrates the usefulness of mosst as a general tool for the design of new site-directed mutagenesis strategies in protein families.\n",
            "\n",
            "CONCLUSIONS\n",
            "a statistical procedure has been designed and presented to semi-automatically identify functionally significant mutable positions in a protein, based on the conservation of physicochemical properties. such positions are identified and classified into three groups, according to the influence their mutation could have on protein function. those in which a mutation does not alter the function and basic characteristics of the protein, but do change them slightly, and those in which a mutation is totally deleterious for the protein are the most relevant positions to look for nssnps, while only the former are important when trying to develop site-directed mutagenesis strategies so that variants with improved properties could be generated.\n",
            "\n",
            "amino acid positions are also classified in three groups: variable irrelevant positions , invariable determinant positions  and variable determinant positions . the recognition and identification of vdps is especially relevant, since these are \"hidden\" functionally relevant positions that cannot be easily identified by other existing automatic methods. this classification serves as a basis to rationally identify relevant amino acid positions in a protein in the frame of a rational design strategy or in the identification of nssnps. in this way, a conservative mutagenesis strategy that aims to preserve the main properties of a protein family should be focused on mutating only vdps, while a non-conservative mutagenesis strategy could also include idps. if the aim is to identify nssnps, mutations that could alter protein function are most probably located at idps and vdps.\n",
            "\n",
            "once these key functional positions are identified, the statistical distribution of the relevant physicochemical properties at each protein position is analyzed to get a list of the  <dig> common protein amino acids ordered according to the global probability with which they can conform to the required property profile of each relevant position. this ordered list is divided into a group of functionally non-disruptive amino acids and a second group of functionally deleterious amino acids.\n",
            "\n",
            "these results become a discriminative tool in the selection and elaboration of rational mutagenesis strategies for the protein. they can also be used to predict if a given nssnp, identified, for instance, in a genomic-scale analysis, can have a functional implication for a particular protein and which nssnps are most likely to be functionally silent for a protein. this analytical tool could be used to rapidly discard any irrelevant nssnp and guide the research focus toward functionally significant mutations. this approach also has the advantage that not only conserved amino acid positions in a protein family can be labeled as functionally relevant, but also non-conserved amino acid positions can be identified having a functional effect.\n",
            "\n",
            "the mosst algorithm has been implemented as a matlab gui and used to analyze endoglucanases belonging to glycosyl hydrolases family  <dig>  in order to determine interesting mutagenesis targets on them. the analysis has indicated amino acids that could be mutated in β- <dig> -endoglucanase bglii of c. cellulans to obtain critical changes in the enzymatic activity: putative substrate specificity amino acid determinants for lichenase and laminarinase activity and determinants of ph-activity profiles for these enzyme family, thus confirming mosst performance as a predictive tool for the study of functionally relevant mutations.\n",
            "\n",
            "the proposed methodology also has limitations, especially in that it uses only information derived from a multiple alignment and the statistical result for each position will strongly depend on the quality of the alignment and it does not prevent a priori this significance level to vary between one alignment and another. in this sense, the quality of the alignment is determinant and should be carefully controlled. moreover, the algorithm does not give any suggestion about the real nature and contribution of relevant amino acids to the structure and function of the protein, but the predictions could be analyzed using other predictive techniques or tested by mutagenesis experiments. another limitation is related to the fact that mosst operates on single positions in a multiple alignment and all statistic parameters are calculated for each position without taking into consideration the simultaneous occurrence of mutations in other positions that can compensate for the effect of the first. this excludes the study of correlated compensatory mutations using mosst in its current version.\n",
            "\n",
            "although the developed procedure does not give any indication about the functional implications of the amino acids positions identified as relevant, an unrelated analysis that use another type of information beyond the statistical inference performed on the multiple alignment by mosst could shed light on their contribution to the protein function or structure. specifically, classical methods to study experimentally and/or predictively the structure-function relationships in proteins can provide this kind of external information. for example, this type of analysis has been used to test the functional relevance of the amino acids identified by mosst in family  <dig> glycosyl hydrolases. in the case of position  <dig> of the alignment of these proteins, a mechanistic-electrostatic analysis yielded an explanation of the functional contribution of this amino acid position to catalysis  <cit> . the implementation and testing results set forth in this work show a promising performance of this technique as a valuable bioinformatics tool to aid in the development of new protein variants and to aid in the understanding of function-structure relationships in proteins.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the proposed method allows the identification of functionally relevant amino acid positions in a protein, based on the conservation of these amino acid properties.\n",
            "the proposed method allows the identification of functionally relevant amino acid positions in a protein, based on the conservation of these amino acid properties.\n",
            "the proposed method allows the identification of functionally relevant amino acid positions in a protein, based on the conservation of these amino acid properties.\n",
            "the proposed method allows the identification of functionally relevant amino acid positions in a protein family, based on the conservation of these amino acid properties.\n",
            "the proposed method allows the identification of functionally relevant amino acid positions in a protein family, based on the conservation of these amino acid properties.\n",
            "\n",
            "87\n",
            "[{'summary_text': 'unmixing of observations and inference of statistical relations enabled us to identify the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\\nunmixing of observations and inference of statistical relations enabled us to identify the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\\nunmixing of observations and inference of statistical relations enabled us to identify the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\\nunmixing of observations and inference of statistical relations enabled to resolve correctly the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\\nunmixing of observations and inference of statistical relations enabled to resolve correctly the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\\nunmixing of observations and inference of statistical relations enabled to resolve correctly the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\\nunmixing of observations and inference of statistical relations enabled to resolve correctly the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\\nunmixing of observations and inference of statistical relations enabled to resolve correctly the number of distinct subpopulations considerably better than multiparametric proximity based approaches'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 5623 to 6144 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "in order to understand how a protein network gives rise to a cellular function it is essential to quantify the states of the involved proteins and their causal relations. however, it is actually not possible to strictly define out of the proteome the subset of all proteins which are involved in a certain cellular process since these will always have interactions with proteins not included in this subset. in spite of major advances in proteomic  <cit>  and cytometric  methods, quantification of the levels and post-translational modifications of all proteins of the proteome in the same cell is still beyond reach. therefore, we fundamentally cannot observe the whole system at once  but only a small part of it   <cit> . this limit, by itself, could have been overcome by looking at different parts of the system in different cells and building a model of the whole system step by step. however, such a strategy is fundamentally hampered by natural cell-to-cell variability which makes the integration of information highly challenging. several studies have addressed the challenge of network reconstruction in the presence of intrinsic and extrinsic noise  <cit>  around one prototypic network structure . however, in many physiological cases the cell-to-cell variance is not only due to noise around one cellular state but also due to subpopulations which are in qualitatively distinct types of states. such qualitative variabilities within the same cell population are generated by epigenetic commitment of cells to different fates  as well as by genetic alterations  as in cancer  <cit> . in many cases the distinct cell subpopulations are spatially intermixed and therefore are harvested together and co-measured within the same sample . in such cases, causal relations and correlations between measured proteins can be qualitatively different in different cells if they are mediated by non-measured proteins which have different states at each subpopulation. therefore, integration of observations over the cell population toward one model would be invalid and will yield uninformative average relations . ultimately, in order to solve this fundamental problem one should identify the number of qualitatively different subpopulations in the data, thus unmix the cells in-silico and resolve separately for each subpopulation the relations between the measured proteins. a recent work suggested to use a mixture model to unravel subpopulations in biochemical systems based on ordinary differential equations and prior knowledge about the number of subpopulations as well as about kinetic constants underlying the differences between them  <cit> . in this work we developed a bayesian method for achieving this goal without such prior knowledge.\n",
            "fig.  <dig> unpbn addresses the challenge of studying intracellular protein networks caused by unmeasured proteins and inter-cellular heterogeneity. a a biochemical system for which three proteins  are being measured in the same cell while the other proteins are unmeasured. note that the effects of z on x are mediated by unmeasured proteins . b depending on the level and state of these unmeasured proteins, the measured causality between x and z can differ qualitatively between cells. for example, normal and cancer cells have different activity levels of oncogenes and tumor suppressors which here lead to a negative or a positive causal effect of z on x, respectively, thereby to a controlled growth or a constitutive growth. c left, multiparametric high-throughput single-cell measurements  of a heterogenous sample of cells . middle, attempts to statistically infer a single set of relations  between the measured proteins fail because there are two distinct subpopulations having two distinct sets of relations. at the same time, it is also impossible to identify the two distinct subpopulations as two distinct proximity-based clusters. right, unpbn performs unmixing and inference of statistical relations as one process, thus finds the set of sets-of-relations  that explains best the observations\n",
            "\n",
            "\n",
            "\n",
            "to unmix observations of cells from different subpopulations, we are taking advantage of the high-dimensionality of the observations, as typically obtained from cell-based high-content measurements such as flow-cytometry  <cit> , mass-cytometry  <cit>  and toponome imaging  <cit> . within each subpopulation, stochastic cell-to-cell variability in protein expression levels gives rise to high-dimensional probability distributions with the same dimensionality as the number of biochemical species  measured in each cell. to this extent, network inference approaches, like gaussian bayesian networks  , to resolve a single statistical model that fits best the data, have been already developed . in this work we use as a basis our previously described nonparametric bayesian network analysis  and expand it to allow for different network structures in a mixture of different cell subpopulations . in this method, termed hereafter unmixing-via-npbn , a flexible number of gaussian bayesian networks is being fitted to the data and thereby iteratively identifying the number of distinct subpopulations, unmixing the observations and resolving the statistical model for each subpopulation. as a model system to assess and demonstrate our method we simulated the canonical mapk raf-mek-erk kinases cascade in the context of pc <dig> cells stimulated by either epidermal growth factor  or nerve growth factor   <cit> . we show that our method identifies better than common clustering approaches the presence of two subpopulations within a mixture of egf and ngf stimulated pc <dig> cells based on the levels of active raf, mek and erk in each cell. this enabled to resolve correctly the statistical relations between raf, mek and erk in each subpopulation.\n",
            "\n",
            "methods\n",
            "simulation\n",
            "the egf and ngf signaling network was simulated based on a previously described model  <cit> . the sbml format of this model  was imported into the matlab symbiology platform to simulate the dynamics of the signaling network using ode15s  solver. to introduce intra-subpopulation cell-to-cell variability , for each run of the simulation we sampled the values for the total raf, mek and erk levels from a normal distribution n around the respective initial concentration for a given set of fractional deviation from the mean . the values of fd represent here the degree of stochastic variance in the expression levels of raf, mek and erk. simulations were repeated  <dig> times with random sampling of total raf, mek and erk levels to generate the data for each cell subpopulation. in each individual simulation repeat, the response of the network to egf or ngf was simulated for  <dig> seconds after stimulation and the levels of c-raf-ras-gtp , ppmek and pperk  were sampled every  <dig> minute as the observed parameters for the unmixing analysis. mixtures containing two distinct cell subpopoulations were generated by mixing an equal number, unless indicated otherwise, of simulated observations obtained upon egf and ngf stimulations. mixtures containing four distinct cell subpopoulations were generated by altering the parameter in the sbml model corresponding to the catalytic activity  of mek  from its wild-type  value  to a value depicting a mutant mek  with a lower activity . thus, by having two different stimulations and two different levels of mek activity, observations of four distinct cell subpopulations were generated: egf-mek wt, egf-mek mut, ngf-mek wt and ngf-mek mut .\n",
            "\n",
            "unpbn\n",
            "methodologically, unpbn is based on the nonparametric bayesian networks  approach  <cit> . it allows to avoid the assumption of underlying gaussian distributions for the data and to find networks with nonlinear relations between the nodes. the unpbn method combines a nonparametric mixture model incorporating the dirichlet process  <cit>  and an allocation sampler  <cit> . prior to the description of the unpbn approach a short introduction of gbns  <cit>  is provided here, as they are a basis of the presented method. we define the data x, consisting of n observations of a system/network with d species/nodes , such that xj represents an n-dimensional vector containing the observed concentrations of the j species . in the bayesian networks approach the relations between the nodes in a graph g are modeled as conditional probability distributions  p. if the cpds for all nodes in g are given by normal distributions of the form xj|pag∼n,σj2), where pag denotes the parents of node xj, kj={k|xk∈pag}, the μj and σj <dig> are the unconditional means and variances of xj, and βj,k are real-valued coefficients determining the influence of xk on xj, and, if in addition, g is a directed acyclic graph  then the pair  is called a gbn. the network structure is inferred using gaussian distributions with a normal-wishart prior  <cit> . the estimation of g is embedded in a markov chain monte carlo  framework, conducted by maximizing the sampling distribution of the sampled graph \n",
            " lg|x=∏j=1d∫lσj <dig> βj|x{j}∪kjpσj <dig> βjdσjdβj, \n",
            "\n",
            "with βj=,=b and x denotes the columns of x with indices in j. the mcmc algorithm uses so called single edge operations  <cit> .\n",
            "\n",
            "unpbn generalizes the gbn approach as it is based on flexible nonparametric bayesian mixture models for networks  <cit>  which in turn combine different gbns for different subsets of the data. the mixture is taken with respect to all parameters . the model for the data can be written as p=∫pdp with μ and σ vectors of the unconditional means μj and variances σj <dig>  respectively. the discrete mixing measure p is distributed according to ℙ, a random probability measure, and p is a multivariate normal distribution with a conditional independence structure compatible with g. according to the discrete nature of p, support points μh,σh,bh,gh and probabilities wh, the mixture can be written as \n",
            " p=∑h=1nwhp. \n",
            "\n",
            "the prior distribution of the mixing weights wh is assigned by p and the prior for μh,σh,bh,gh is given by the base measure p <dig> of ℙ for all h. the n different mixture components h can be interpreted as subpopulations in the data set. accordingly, here such subpopulations are referred to as components. the assignment of each data point to its corresponding component is described by the allocation vector l=′  <cit> .\n",
            "\n",
            "the network structure g and the allocation vector l are the main focus of our unpbn procedure. the remaining parameters μh, σh and bh are integrated out and the mcmc algorithm iterates by updating the dag g, the number of components n and the latent allocation vector l, leading to the posterior distribution \n",
            " p=∏h=1nl)pnpp, \n",
            "\n",
            "where l=∫lpdσdb is the marginal sampling distribution for g, pn is a probability distribution on the space of allocation vectors, p is the distribution of the number of components and ih={i∈{ <dig> …,n}|li=h} and x denotes the rows of x with indices in i.\n",
            "\n",
            "in our unpbn analysis a prior is needed for θh= and for w <dig> …,wn. for gh the prior which was used is uniform over the cardinality of the parent sets  <cit> , for σh and bh we employed the normal-wishart prior distribution with the identity matrix as the precision matrix and d+ <dig> degrees of freedom. the mean vector of the multivariate normal distribution  was chosen as a vector of zeros. for n we used a poisson distribution with parameter λ= <dig> and the wh were obtained from a dirichlet distribution with parameter vector  with α= <dig>  further details for the sampling distribution, posterior distribution and the mcmc sampling scheme were discussed in previous publications  <cit> . the approach is implemented in matlab . the presented results are obtained from mcmc runs with  <dig> · <dig> iterations with a thinning of  <dig> and a burn in of  <dig> · <dig> iterations for networks with two subpopulations and from runs with 5· <dig> iterations with a thinning of  <dig> and a burn in of 2· <dig> iterations for networks with four subpopulations.\n",
            "\n",
            "postprocessing of graphs\n",
            "although it is possible to use the output from the unpbn analysis directly, for example to choose the most frequent dag or allocation vector as a representative, it is preferable to perform an additional postprocessing step that takes into account all mcmc samples and improves the results considerably. the inferred graphs in the iterations of the mcmc simulation are stored in the form of adjacency matrices. such a matrix a consists of the elements aij, aij= <dig> if nodes i and j are conditionally dependent  and aij= <dig> if nodes i and j are conditionally independent  or if i=j. for each pair of nodes the mcmc output of the unpbn analysis can be summarized by the posterior edge probability pepij=∑s=1raijs/r where s is the index of the r iteration steps in the mcmc simulation. the resulting pep number ranges from  <dig>  to  <dig> . these pep values are used in fig.  <dig> for the presentation of the obtained results.\n",
            "fig.  <dig> simulations of the egf and ngf signaling network in pc <dig> cells. a the simulated protein network, as previously described  <cit> . the elements  that are unique to either egf or ngf are colored red and green, respectively. the components that we later consider as the measured components of that system  are colored grey. b a 3-dimensional scatter plot showing the normalized levels of active  raf , mek  and erk  versus each other at  <dig> minutes after ngf  or egf  stimulation with noise level of  <dig> . c the same as b but for  <dig> minutes after stimulation. d time profiles of praf, ppmek and pperk levels as a function of time after ngf  or egf  stimulations, without noise  or with  <dig>  noise level \n",
            "\n",
            "fig.  <dig> unmixing observations of a mixed cell population by unpbn in comparison to clustering approaches. a mixtures of observations of egf and ngf stimulated cells with different noise levels were generated as described. observations were sampled at one-minute intervals for  <dig> minutes after stimulation. for each noise level and sampled time point, observations were unmixed using unpbn, k-means clustering  and hierarchical clustering . the percentages of correctly allocated observations, averaged over all time points, are indicated by boxplots for the different methods as a function of the noise level . b comparison of the unmixing accuracy with noise level  <dig>  along the different sampled time points, as achieved by unpbn, post-processed unpbn limited to two components, k-means  and hierarchical clustering \n",
            "\n",
            "fig.  <dig> the success in identifying the correct number of distinct cell subpopoulations  in a mixture by unbpn in comparison to clustering approaches. a a boxplot showing the asw versus the tested number of components obtained by unbpn analysis  of a mixture of two subpopulations . the boxplot indicates the median , the  <dig>  and  <dig>  quartiles , margined by the largest and smallest data points which are still within the interval of  <dig>  times the interquartile range from the box , and the outliers  obtained from pooled values over all time points with noise level of  <dig> . b and c, the same as in a but for asw obtained following k-means clustering and hierarchical clustering, respectively. d, e and f, the same as the corresponding a, b and c, but for a mixture of  <dig> subpopulations: egf-mek wt, egf-mek mut, ngf-mek wt and ngf-mek mut . it should be noted that silhouette widths are incomparable between different clustering approaches. however, silhouette widths are comparable between different parameters of the same clustering approach and thereby indicate the identified number of distinct subpopoulations as the one providing the largest asw\n",
            "\n",
            "fig.  <dig> recovering correctly the edge probabilities network between praf, ppmek and pperk for each cell subpopulation in a mixture by unpbn. the triangles show, color encoded, the posterior edge probabilities between praf, ppmek and pperk at different time points after stimulation as derived by unpbn analysis of pure ngf-stimulated or egf-stimulated cell populations  with noise level of  <dig>  and of a mixture of these populations by gbn versus unpbn . note, for the pure subpopulations, the changes in the edge probabilities between the same components at different time points and upon the different stimulations. analyzing a mixture of ngf-stimulated or egf-stimulated cells without unmixing yields uninformative average edge probabilities, not representing any of the distinct two subpopulations. employing unpbn recovers precisely the pure edge probabilities for each of the two subpopulations. for clarity, we show here the edge probabilities for  <dig> time points after stimulation that illustrate the need for unmixing\n",
            "\n",
            "\n",
            "\n",
            "postprocessing of allocations\n",
            "for the allocation vector, however, it is not possible to summarize the sampled vectors in the same way as for the edges, because of the so called label switching problem. during the sampling procedure the labels of the components change randomly, so that if two allocation vectors are compared it is not clear if a particular observation has been allocated to a different component or if the label of the component has changed. we employed a method based on maximizing the adjusted rand index that bypasses this obstacle and that combines the allocation vectors of each mcmc iteration into one single vector  <cit> . this method is implemented in r  <cit>  in the package ’mcclust’ and was used in cases where it was necessary to fix the number of components to a particular value . in cases where the analysis is focused on the unmixing performance of unpbn , the sampled allocation vectors are evaluated regarding the homogeneity of the resulting components. for each entry lis,h in the allocation vector sampled in iteration s, in each component h, the true component is determined by comparison with the simulation setting. based on this, componentwise, observations originating from the same true component are considered as allocated correctly,  while the remaining observations in that component are considered as wrongly allocated . the percentage of correctly allocated observations  for a particular unpbn outcome is derived by \n",
            " pco=1r∑s=1r1ns∑h=1ns1nhs∑i=1nhsilis,h· <dig> \n",
            "\n",
            "with r considered mcmc iterations, size nhs of component h in iteration s and ns number of components in the allocation vector in iteration s.\n",
            "\n",
            "cluster analyses\n",
            "in order to compare unpbn with clustering methods, k-means and hierarchical clustering were used in this work. the k-means clustering method finds the partition that divides the data to n clusters  such that the sum distances of all observations to the corresponding cluster mean is minimized  <cit> . the k-means cluster analysis was performed in matlab, using the function “kmeans” with the distance parameter being set to squared euclidean distance. to obtain stable results, the clustering was repeated  <dig> times with randomly chosen different starting points. hierarchical clustering is an agglomerative procedure which merges in each step the two closest objects, repeatedly till the whole data set is in one single cluster. the hierarchical clustering was performed in matlab using the functions “pdist”, with the distance parameter being set to euclidean, followed by “linkage”, with the method parameter being set to inner squared distance .\n",
            "\n",
            "silhouette analysis\n",
            "we used the average silhouette width   <cit> , to assess the quality of a given clustering and to compare the results of clusterings with different parameter settings. for a given clustering result, the silhouette value is calculated as \n",
            " sil=b−amax{a,b}. \n",
            "\n",
            "for each observation, xi, a is the average dissimilarity between xi and all other data points within the same cluster, and b is the smallest average dissimilarity between xi and the data points in the remaining clusters, calculated for each cluster separately. any measure of dissimilarity can be used, but distance measures are the most common. in this work the euclidean distance was employed. the silhouette value is ranging between - <dig> and  <dig>  negative values indicate that a particular observation will fit better in another cluster, so it has been matched wrongly and the quality of the clustering result can be improved. high positive values indicate a good clustering result. the asw is computed by averaging all sil values, thus it provides an overall evaluation of the regarded clustering. while asw enables to compare clustering performed with the same method with different parameters, asw values are not comparable between different clustering methods.\n",
            "\n",
            "RESULTS\n",
            "simulation of inter and intra cell-population variabilities\n",
            "in order to evaluate the performance of the method we simulated the mapk module in pc <dig> cells using a previously described model  <cit> . this model captures the different temporal profiles of erk activation upon egf and ngf stimulation, attributing it to the differential activation and dynamics of ras and rap   <cit> . both stimulations activate via sos and ras the upstream kinase, raf and thereby the whole mapk cascade. however, each stimulation has a different effect on other proteins which affect the mapk module and its dynamics. in egf stimulation, erk inhibits sos and thereby forms a negative feedback loop leading to a transient erk activation which encodes a proliferation signal. in ngf stimulation this negative feedback is overcome by a nested positive feedback loop  <cit>  formed due to the activation of pkc δ which phosphorylates rkip and thus leads to its release from raf and thereby enabling raf activation by erk. the model used here considers another difference attributed to a sustained activation of another activator of the mapk cascade, rap <dig>  by ngf but not by egf  <cit> . thus, ngf leads to a sustained erk activation, encoding a signal for differentiation. as a source for inter-population variability, we simulated the dynamics of the complete network upon either egf or ngf stimulation. for the aim of this work, we based our analyses on snapshots of the simulation, and, in turn, analyzed each time point independently. as a source of intra-population variability , we added stochastic noise in total protein levels mimicking natural cell-to-cell variance in protein expression . however, unlike instrumental noise that only affects the readout, noise in expression levels affects the system itself. thus, although the introduced noise was generated as gaussian, its propagation through the system generates asymmetric high-order patterns shaped by the topology of the network .\n",
            "\n",
            "\n",
            "in the absence of noise, the levels of phosphorylated  raf, mek and erk follow the expected profiles, exhibiting a clear difference between egf or ngf stimulations . with intra-population variance, the profiles get broader and overlap between the two stimulations , making it difficult to allocate individual observations to the corresponding stimulation . to impose the fundamental experimental limit of observing only part of the system, for the subsequent analysis we considered an observation to be the triplet formed by the concentrations of phosphorylated species of raf, mek and erk per cell, ignoring all other information. finally, to generate heterogeneous cell-populations, we mixed observations randomly selected in equal amounts from the egf and ngf datasets.\n",
            "\n",
            "unpbn unmixes observations of distinct subpopulations\n",
            "we first wanted to test whether unpbn can classify correctly observations coming from distinct cell subpopulations. we applied unpbn on mixed cell populations having different levels of noise and counted the observations correctly allocated to the egf and ngf stimulated subpopulations . for noise levels of  <dig> ,  <dig>  and  <dig> , around  <dig> %,  <dig> % and  <dig> % of the observations are correctly allocated, respectively . to assess the accuracies of unpbn, we compared them to those achieved by two widely used clustering approaches - hierarchical clustering and k-means clustering. when the noise is low , the two subpopulations are well separated by all methods . as expected, the performance of all three methods is negatively affected when the noise level is increased. however, the unpbn considerably outperforms the other reference methods for all noise levels above  <dig>  . if the relative abundance of the two subpopulations is 1: <dig>  all methods classify about equally well for a low noise level , while for a high noise level  unpbn classifies as good as k-means but better than hierarchical clustering .\n",
            "\n",
            "we next focused on the high noise level of  <dig>  and compared the performance of the methods as a function of time after stimulation . along the different time points the performance of all methods varies, reflecting a changing difficulty to identify the two subpopulations based on the levels of praf, ppmek and pperk. unpbn constantly outperforms the clustering methods in all the time points and is more robust with its performance level . furthermore, the performances of the two clustering methods along the time points have a similar profile, which differs from the profile of unpbn . these results are consistent with the fact that, unlike the clustering approaches, unpbn uses high-order patterns, rather than merely distances between observations. this additional information is shown here to be indeed valuable for the ability to unmix subpopulations based on high-dimensional observations.\n",
            "\n",
            "unpbn identifies the number of subpopulations in a mixture\n",
            "in many cases, when a sample of cells is derived it is unknown a priori how many distinct subpopulations it contains. therefore, a comprehensive unmixing approach should also be able to identify the number of subpopulations without such a priori knowledge. indeed, while the clustering approaches were guided to search for two subpopulations, unpbn was not given this information but found it independently . moreover, the performance of unpbn does not change significantly if it is forced to identify exactly two subpopulations, indicating the ability of unpbn to correctly determine by itself the number of distinct subpopulations in a mixture .\n",
            "\n",
            "in order to compare the capability of the different methods to identify the number of subpopulations we used the asw to determine the quality of the clusters and thereby the number of clusters  in the data as could be inferred by each method  <cit> . the asw of a cluster is a measure of how tightly grouped are the data points in the cluster, such that larger asw values denote tighter clusters. for a cell population containing two subpopulations  we calculated the asw as a function of the number of clusters derived by unpbn , k-means and hierarchical clustering . for unpbn and k-means clustering, the maximal asw is found when the number of clusters is  <dig>  the actual number of subpopulations in the data . however only in unpbn there is a significant and robust difference with the other cluster sizes, while with k-means clustering the asws obtained for  <dig> and  <dig> clusters are not robustly separable. with hierarchical clustering the performance is further worse since asws obtained for  <dig> and  <dig> clusters are comparable, or even higher than those obtained for  <dig> clusters . unpbn successfully identified the number of subpopulations also if their relative abundance was significantly different .\n",
            "\n",
            "we next tested the performance of the method with a more complex mixture of cells containing four distinct subpopulations. to simulate these subpopulations, the catalytic rate constant, kcat, of mek in the model was changed, mimicking a wild-type mek  and a mutant mek  that phosphorylate erk at different rates. thus, together with the two different stimulations, egf and ngf, four distinct subpopulations were generated, denoted by egf-mek wt, egf-mek mut, ngf-mek wt and ngf-mek mut . unpbn correctly identified that the data contains four distinct subpopulations, in contrast to k-means and hierarchical clustering .\n",
            "\n",
            "unpbn uncovers distinct topologies for distinct subpopulations\n",
            "the causal relations between the components of a system are constant, since the set of biochemical reactions and constants that describe the whole system remains constant. however, in practice, only part of the components of a system can be co-measured and therefore the reaction constants become apparent constants that depend on the unmeasured components. here we intentionally simulated the fundamental limit of looking on only a small part of a system. therefore we expected the apparent strength of the causal connection between praf, ppmek and pperk, as reflected by the undirected posterior edge probabilities among them, to change as a function of the stimulation and time. indeed, when we analyzed separately egf and ngf stimulated cells we observed different posterior edge probabilities between the two treatments, as well as within each treatment at the different time points . when analyzing the mixed population with a standard gbn approach   <cit> , we obtained posterior edge probabilities exhibiting, in general, an average behavior of the two subpopulations. naturally, these average values become meaningless when the two subpopulations exhibit very different posterior edge probabilities . in contrast, when analyzing the mixed population with unpbn , the unmixing step enabled to uncover the true network of posterior edge probabilities for each stimulation and at each time point . this also demonstrates that the performance of the unmixing process  was sufficiently good to enable correct inference of protein-protein relations in each subpopulation. since more than one dag may represent exactly the same set of conditional independence relationships  <cit> , given static data without perturbations it is more reliable to infer the causal strengths between proteins, regardless of the direction of these causalities. extending the unpbn approach to dynamic data, or using perturbation data or adding prior information, will further facilitate the inference of directionality in the causal relations for each cell subpopulation.\n",
            "\n",
            "discussion\n",
            "in the era of systems biology, single-cell measurement techniques are rapidly expanding with respect to the number of cells that can be analyzed and the number of biochemical species that can be co-measured per cell. the approaches to explore these data have focused so far either on identifying different subpopulations of cells based on multiparametric proximities or on inferring the topology of statistical relations between the parameters for the population as a whole. however, the aim to reach each of these two goals in separate has fundamental problems. in one direction, ignoring the heterogeneity between cell subpopulations will lead to inferring a meaningless average topology of statistical relations of the population as a whole. in the other direction, since statistical relations are inferred from the correlation between the measured parameters, the identification of cell subpopulations based on multiparametric proximities inherently conflicts with the capability to resolve the topology of relations within each subpopulation. furthermore, protein networks with distinct topologies can be at the same state  and protein networks with the same topology can be at different states . therefore, attempts to identify cell subpopulations based on multiparametric proximities may actually identify different cellular states but not different types of cells. the method presented here pioneers a comprehensive solution to these fundamental problems by performing the identification of cell subpopulations  and the inference of statistical relations between the measured parameters in one joint analysis.\n",
            "\n",
            "intentionally, we used snapshot data of a dynamic process  and, respectively, the method we developed does not rely on temporal information nor intends to give a model description of the dynamic itself. due to that, this method can be applied on the type of single-cell multiparametric measurements currently available such as multicolor flow-cytometry  <cit> , multiplexed mass cytometry  <cit>  and toponome imaging  <cit> . the classification of the distinct subpopulations in cell populations sampled at different time points along an experiment can hint toward the dynamic behavior of each subpopulation. however, such traceability of subpopulations along the time points depends on how different is their relative abundance within the whole population and on the sampling rate in comparison to the timescale of the biological process. advances in multicolor live cell imaging in combination with high-throughput automated microscopy gradually facilitate monitoring increasing numbers of parameters in individual live cells over many cells. the data obtained from such measurements will enable not only tracking the dynamics of the measured parameters in each cell subpopulation but also tracking them in individual cells. this kind of temporal information will help to further improve the identification of the distinct cell subpopulations and the inference of statistical relations between measured parameters in each subpoulation. as indicated by this work, it would be important also for the analysis of such live cell measurement data that unmixing and inference of protein-protein relations will be performed as one process.\n",
            "\n",
            "the importance to recover single-cell phenotypes out of an uninformative average cell population behavior has been established and exemplified in many systems. notably, in these examples there was only one measured parameter per cell, often the output of the system, and, therefore, the unmixing was straightforward. however, in order to obtain mechanistic insight into how a biochemical system works it is required to examine the protein network itself, and not only its output. for this, multiple parameters should be co-measured per cell to overcome uncorrelated cell-to-cell variability between these parameters . we demonstrated here that in such a case unmixing cannot be achieved anymore using the proximity between the values of these parameters, while it can be successfully achieved using the high-order relations between them as captured by unpbn. importantly, unpbn can be straightforwardly extended to incorporate prior knowledge about parts of the network in the individual subpopulations.\n",
            "\n",
            "CONCLUSIONS\n",
            "our results show that the coupling between unmixing of observations and inference of statistical relations is essential and effective. with respect to the unmixing, our method was capable to identify the number of qualitatively distinct subpopulations considerably better than multiparametric proximity based approaches . consequently, the statistical relations in each unmixed subpopulation were also correctly recovered, while, without unmixing, uninformative average relations were inferred. as systems biology and personalized medicine are aiming toward reverse-engineering and re-engineering signaling networks, they are increasingly challenged by the inter-cellular variability and the large size of the relevant biochemical system. the work presented here offers a conceptual solution as well as an applicable statistical method to address this challenge.\n",
            "\n",
            "additional files\n",
            "additional file  <dig> \n",
            "the four distinct simulated topologies of the egf and ngf signaling network used in fig. \n",
            " <dig>   ngf-mek wt: the wild-type network  with ngf stimulation.  egf-mek wt: as in  but with egf stimulation.  ngf-mek mut: the wild-type network with ngf stimulation, beside that here the sbml model parameter corresponding to the k\n",
            "cat of mek  is altered from its wild-type value  to a value depicting a mutant mek with a lower activity , as indicates the thinner arrow from mek to erk.  egf-mek mut: as in  but with egf stimulation.\n",
            "\n",
            "\n",
            "\n",
            "additional file  <dig> \n",
            "unmixing observations of cell subpopulations, mixed in a 1: <dig> ratio, by unpbn in comparison to clustering approaches. mixtures of observations of egf stimulated cells  and ngf stimulated cells  were generated with noise levels of  <dig>  and  <dig> . observations were sampled at one-minute intervals for  <dig> minutes after stimulation. for each noise level and sampled time point, observations were unmixed using unpbn, k-means clustering  and hierarchical clustering . the percentages of correctly allocated observations, averaged over all time points, are indicated by boxplots for the different methods for both noise levels .  the percentages of correctly allocated observations of the egf-stimulated subpopulation.  the percentages of correctly allocated observations of the ngf-stimulated subpopulation.\n",
            "\n",
            "\n",
            "\n",
            "additional file  <dig> \n",
            "the success of unpbn in identifying the correct number of cell subpopulations  mixed in a 1: <dig> ratio. mixtures of observations of egf stimulated cells  and ngf stimulated cells  were generated with noise levels of  <dig>  and  <dig> . observations were sampled at one-minute intervals for  <dig> minutes after stimulation.  a boxplot showing the asw versus the tested number of components obtained by unbpn analysis . each boxplot indicates the median , the  <dig>  and  <dig>  quartiles , margined by the largest and smallest data points which are still within the interval of  <dig>  times the interquartile range from the box , and the outliers  obtained from pooled values over all time points with a noise level of  <dig> .  the same as  but with a noise level of  <dig> .\n",
            "\n",
            "\n",
            "\n",
            "abbreviations\n",
            "aswaverage silhouette width\n",
            "\n",
            "dagdirected acyclic graph\n",
            "\n",
            "egfepidermal growth factor\n",
            "\n",
            "erkextracellular signal regulated kinase\n",
            "\n",
            "gbngaussian bayesian network\n",
            "\n",
            "mapkmitogen-activated protein kinase\n",
            "\n",
            "mcmcmarkov chain monte carlo\n",
            "\n",
            "mekmapk/erk kinase\n",
            "\n",
            "ngfnerve growth factor\n",
            "\n",
            "npbnnonparametric bayesian network analysis\n",
            "\n",
            "pkcprotein kinase c\n",
            "\n",
            "unpbnunmixing via npbn\n",
            "\n",
            "competing interests\n",
            "\n",
            "the authors declare that they have no competing interests.\n",
            "\n",
            "authors’ contributions\n",
            "\n",
            "jw, rsm-s, heg, ez and ki conceived the project, jw and ki developed the unpbn algorithm, rsm-s, heg and ez generated the simulated data, jw, rsm-s, yf, heg, ez and ki analyzed the data, jw, rsm-s, heg, ez and ki wrote the paper. all authors read and approved the final manuscript.\n",
            "\n",
            "this work was supported by bmbf grant no.  <dig>  yf was supported by a german academic exchange service  fellowship.\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "unmixing of observations and inference of statistical relations enabled us to identify the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\n",
            "unmixing of observations and inference of statistical relations enabled us to identify the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\n",
            "unmixing of observations and inference of statistical relations enabled us to identify the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\n",
            "unmixing of observations and inference of statistical relations enabled to resolve correctly the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\n",
            "unmixing of observations and inference of statistical relations enabled to resolve correctly the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\n",
            "unmixing of observations and inference of statistical relations enabled to resolve correctly the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\n",
            "unmixing of observations and inference of statistical relations enabled to resolve correctly the number of distinct subpopulations considerably better than multiparametric proximity based approaches.\n",
            "unmixing of observations and inference of statistical relations enabled to resolve correctly the number of distinct subpopulations considerably better than multiparametric proximity based approaches\n",
            "88\n",
            "[{'summary_text': 'plant invasion into new environments is an extremely costly problem, which is why it is so important to study the genetic and molecular basis of invasiveness in exotic plants.\\nplant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness and to choose appropriate management interventions.\\nplant invasion is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness and to choose appropriate management interventions.\\nplant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness and to choose appropriate management interventions.\\nplant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness and to choose appropriate management interventions.\\nplant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness and to choose appropriate management interventions.\\nplant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness.\\nplant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invas'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 4544 to 5120 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "plant invasion into new environments is an extremely costly problem, not only monetarily but also ecologically. invasive plant infestations reduce biodiversity by displacing native species and can literally destroy some native ecosystems by altering important ecosystem characteristics  <cit> . however, the reasons why some plants remain at low abundance in their home range but become dominant in their new range is not well understood and remains one of the most perplexing questions in ecology. multiple non-exclusive hypotheses have been proposed to explain plant invasion into new environments  <cit> .\n",
            "\n",
            "a long standing idea in the field of invasion biology is that of enemy release  <cit> . this hypothesis posits that introduced plants escape their native co-evolved specialist enemies, which allows them to rapidly increase their numbers  <cit> . blossey and notzold  proposed the evolution of increased competitive ability  hypothesis, which builds on the idea of enemy release and has generated much interest in recent years  <cit> . the eica hypothesis suggests that costly defense against specialists no longer enhances fitness of plants in the introduced range; therefore introduced plants will evolve to put fewer resources into defense allowing them to increase allocation of resources towards growth and reproduction  <cit> . this hypothesis has been supported by experimental evidence, but only in part  <cit> . multiple refinements to the eica hypothesis have been proposed to account for altered selective pressures in the new environment including the presence of generalist enemies  <cit>  and changes in resource availability  <cit> .\n",
            "\n",
            "the majority of studies examining eica and other hypotheses of plant invasion have focused on ecological, physiological and to some extent chemical plant characteristics  <cit> . however, with the current revolution in genomics technologies, the question arises as to whether ecological phenomena such as plant invasion can be better understood by studies of genetics or gene expression profiling. the development of genomics resources for non-model species of invasive weeds is increasingly becoming possible as new technologies become more available and affordable, as demonstrated by broz et al.  <dig>  and anderson et al.  <dig> , aiding in the ability of researchers to investigate the biology of invasive weeds  <cit> . in regards to ecological hypotheses, it may be particularly useful to characterize expression of genes related to plant defense and competitive ability.\n",
            "\n",
            "recently, an est  library resource was developed for the problematic invasive plant, centaurea stoebe l.  hayek   <cit> . c. stoebe, a native to eurasia, is able to invade not only ruderal habitats, but also rangelands, pastures and prairies in north america, where it often establishes dense monocultures and excludes native plant species. c. stoebe first appeared on both coasts of north america around the late 1800s  <cit> , and has since greatly expanded its range to all but three states in the continental us  <cit> .\n",
            "\n",
            "molecular marker studies revealed relatively large amounts of genetic diversity within and among populations in both the native and introduced ranges  <cit> , and suggest that this species has been introduced to north america multiple times. thus, genetic drift resulting from bottle-necks or founder effects does not seem to have played an important role in the invasive success of this weed. extensive field collections thus far conclude that the native range consists of morphologically indistinguishable diploid  and tetraploid  forms of the weed  <cit>  that occasionally occur in mixed stands  <cit> . in the introduced range, populations had been found to contain the tetraploid form exclusively  <cit>  until a recent extensive survey identified a single mixed stand of diploid and tetraploid plants in western canada  <cit> . this suggests that both forms of the weed were introduced, but only the tetraploid has become an invasive problem  <cit> .\n",
            "\n",
            "c. stoebe is able to tolerate a wide variety of soil types and precipitation amounts in both eurasia and north america  <cit> . robust cross-continental comparisons have provided empirical evidence for a niche shift between native and introduced populations  <cit> , and more recently between native and introduced tetraploid c. stoebe, with the invasive tetraploids occurring in drier and warmer climates  <cit> . moreover, the range of the native tetraploid in eurasia has expanded over the range of the native diploid within the past 100– <dig> years  <cit> , and introduced tetraploids appear to have a higher ecological tolerance, or niche breadth, than either of the native forms  <cit> . thus, the invasive success of c. stoebe appears to be partially due to pre-adaptation of the native tetraploid cytotype to drier climates, a trait which has been further selected for in the introduced range  <cit> . however, more studies are needed to rule out other alternatives related to the weeds invasive success.\n",
            "\n",
            "both diploid and tetraploid forms of c. stoebe are out-crossing, insect-pollinated asters, but the diploid tends to have a biennial monocarpic life cycle, whereas the tetraploid tends to be a polycarpic perennial, continuing to flower over multiple growing seasons  <cit> . compared to native populations, introduced tetraploids exhibit the highest proportion of polycarpic plants and have the greatest number of stems per plant  <cit> , which may increase their reproductive capacity. it is hypothesized that this perennial polycarpic life cycle is selected for, particularly in environments lacking natural enemies  <cit> , which may help explain why the tetraploid form became predominate in the introduced range.\n",
            "\n",
            "although there are a small number of studies that examine ploidy differences between native and introduced populations of plants, this factor is most often unaccounted for in ecological studies of invasive weeds  <cit> , including c. stoebe. many of the worst weeds are polyploids, and changes in plant ploidy may lead to changes in life history traits, genetic diversity, gene expression or capacity for adaptation and evolution  <cit> . therefore, in a comparison of plants from both the native and introduced range, it is important to compare the same taxonomic unit  <cit> , and understand differences between taxonomic units.\n",
            "\n",
            "as it appears that both ploidy pre-adaptation  and selection  may be important factors in c. stoebe invasion, we were interested in characterizing the three distinct geo-cytotypes of c. stoebe: native diploids, native tetraploids and introduced tetraploids. we grew plants from multiple populations, representing each of the three geo-cytotypes in a common environment and monitored plant performance characteristics and life cycle habits. in addition, we identified gene sequences in the c. stoebe est library that may be involved in constitutive basal plant defense or rapid evolution, as these traits may be important in the plants invasive success. expression of these genes was characterized in each geo-cytotype using quantitative pcr.\n",
            "\n",
            "based on ideas of enemy release and rapid evolution of plants in the introduced range, and on trends in polyploidy, we developed hypotheses concerning plant performance and gene expression of the geo-cytotypes. first, we hypothesized that introduced tetraploids would exhibit reduced expression of constitutive defense and secondary metabolite related genes, but an increase in plant performance when compared to native tetraploids, due to a partial release from enemies. second, we also expected that genes involved in genome stability would be expressed to a greater extent in introduced versus native tetraploids due to possible novel environmental stresses experienced in the introduced range. although evolution is predominately thought to be due to random mutations, there is some evidence that expression of transposable elements and dna repair enzymes influence genetic stability and stress-induced evolutionary strategies in organisms  <cit> . therefore, we also assessed transcript accumulation of two active transposable elements and a dna repair enzyme, which might facilitate rapid evolution in a new environment. finally, we hypothesized that native tetraploids would exhibit increased expression of genes involved in secondary metabolite production compared to diploids, due to potential increases in the metabolic activities of polyploids  <cit> .\n",
            "\n",
            "RESULTS\n",
            "plant performance and life cycle analysis\n",
            "no significant differences in vegetative plant performance characteristics were found between c. stoebe geo-cytotypes . before bolting, the plant biomass index tended to be higher in diploid populations than in tetraploids, but the results were not significant . similarly, stem height was not different between the three geo-cytotypes . however, differences in life cycle were noted between ploidy groups; a higher percentage of both native and invasive tetraploid plants flowered in the first year compared to the diploid plants . fewer than half of the diploid plants flowered in their first year of growth, and over 60% died after flowering . in comparison, over 75% of both native and introduced tetraploids flowered their first year and only 24% and 7% died after flowering, respectively . in addition, tetraploids produced more new rosettes after senescence of the parent plant than diploids . interestingly, the number of capitula per plant  was not different between the three geo-cytotypes. the observed differences in life cycle characteristics reflect the moncarpic life cycle of the diploid and the polycarpic life cycle of the tetraploid  <cit> , and are likely important in plant population fecundity over time, as illustrated by a simulation of seed production . over a fifteen-year period, this simluation estimates production of  <dig> ,  <dig> , and  <dig>  million seeds for populations of the native diploid, native tetraploid, and introduced tetraploid, respectively .\n",
            "\n",
            "gene expression analysis\n",
            "tetraploid plants from the introduced range had significantly lower rates of gene expression for all three pal transcripts compared to tetraploid plants from the native range, providing evidence in favor of our hypothesis . pal <dig> transcript accumulation in introduced tetraploids was  <dig>  times lower than the amount in native tetraploids, whereas pal2a and pal2b were  <dig>  and  <dig>  times lower, respectively . pal  <dig> expression was lower than expression for either form of pal  <dig> in all geo-cytotypes . similarly, glucanase transcripts showed over a two-fold reduction in expression in introduced tetraploids than their native counterparts . chitinase expression was  <dig>  fold lower in introduced tetraploids than native tetraploids . in general, expression of all tested secondary metabolism- and defense-related transcripts was lower in tetraploids from the introduced range compared to their native counterparts.\n",
            "\n",
            "for each sample, total rna  was estimated using the appropriate standard curve for each gene of interest and normalized using the geometric mean of the three standards: actin, cytochrome c oxidase  and ubiquitin , as suggested in vandersompele et al.  <dig>  <cit> . genes of interest included three isoforms of pal   <dig>  2a, 2b, involved in secondary metabolism; chitinase and glucanase, involved in defense response; and a transposable element  and dna repair/recombination gene , potentially involved in rapid evolution. geo-cytotypes are 2× eu, native eurasian diploid populations; 4× eu, native eurasian tetraploid populations; 4× us, invasive north american tetraploids. significant differences in gene expression  were determined for geo-cytotypes of interest  using pair-wise comparisons of lsmeans. lsmeans were back-transformed and expression values are shown relative to native eurasian tetraploid populations . fisher's lsd and absolute t values are reported for each pair-wise comparison.\n",
            "\n",
            "contrary to our second hypothesis, introduced tetraploids showed over two-fold less expression of a transposable element  transcript than native tetraploids . the other transposable element  showed extremely low levels of transcript accumulation in most samples, nearly all of which fell below the standard curve for that gene . of the usable values, the data suggested that introduced populations expressed this transposable element to a lower extent than native populations, but the sample size was very low and thus overall values may not accurately reflect expression in these populations. expression of rad was low in all plant types, but also showed the highest relative mean expression in native tetraploids, although this result was not significant .\n",
            "\n",
            "diploid and tetraploid plants from the native range showed similar relative expression levels for seven out of ten genes tested; pal <dig>  glucanase, chitinase, rad, and the three housekeeping genes . expression of pal2a and pal2b was higher in native tetraploids compared to diploids  as hypothesized. expression of cacta transposable element was also higher in native tetraploids compared to diploids . introduced tetraploids showed similar expression profiles when compared to diploids for nine of the ten genes tested . the expression of pal2b was over three fold lower in introduced tetraploids compared to diploids .\n",
            "\n",
            "discussion\n",
            "plant performance and life cycle analysis\n",
            "ridenour et al.  recently reported that in a common garden in montana, c. stoebe plants from north america exhibit greater biomass, tougher leaves and increased trichome density when compared to their eurasian counterparts  <cit> . based on this finding and hypotheses such as eica that suggest invasive plants may evolve to increase resource allocation to growth  <cit> , we expected that introduced tetraploids would out-perform both native diploids and tetraploids. however, in our study, neither of the plant vegetative growth characteristics examined , showed a significant difference. ridenour et al.  performed the bulk of their experiments on populations with unknown ploidy; however, one experiment containing plants of known ploidy revealed greater rosette diameters of introduced tetraploids compared to native tetraploids  <cit> . conversely, müller et al.  observed that hungarian and german diploids had greater dry weights and shoot diameters than north american tetraploids when grown in a european soil, but sample sizes were relatively small  <cit> . the observed differences may be due to the various populations chosen, the type and origin of soil used , or other factors involved in each of the above studies. these inconsistencies may suggest that vegetative growth is not the best indicator of invasiveness.\n",
            "\n",
            "as previously noted by müller , life cycle differences between c. stoebe geo-cytotypes may have greater relevance to fitness than single performance traits  <cit> . in the first year of this study, flowering plants of all geo-cytotypes had a similar number of capitula : however, fewer diploid plants flowered in the first year of growth than tetraploids, diploids formed fewer new rosettes, and diploids suffered greater mortalities after flowering . in combination these measures suggest that the reproductive capacity of tetraploids is greater than that of diploids. additionally, we expect introduced tetraploid populations to have a higher reproductive capacity when compared to the native tetraploids, as illustrated by a simulation of seed production . ongoing experiments will provide more complete information about the life-cycle of these plants and seed production over their entire life span. thus, although we did not detect any significant differences in vegetative traits between c. stoebe geo-cytotypes, there is some indication of a long-term difference in plant fecundity, with the invasive tetraploid showing highest performance of the three geo-cytotypes studied.\n",
            "\n",
            "gene expression analysis\n",
            "secondary metabolism and defense\n",
            "we selected three distinct pal unigenes for analysis of secondary metabolite-related transcript, as this enzyme represents the first enzymatic step in the flavonoid synthesis pathway which contributes isoflavones, anthocyanins, condensed tannins and other secondary metabolic compounds in plants  <cit> . flavonoids are often stored in plant tissues as 'pre-formed' defense compounds and may act as pathogen and herbivore deterrents  <cit> . the expression of pal gene transcripts in addition to the secondary metabolites resulting from the flavonoid pathway are known to be important in plant defense against pathogens, herbivores and environmental stresses  <cit> .\n",
            "\n",
            "a chitinase and a beta- <dig> -glucanase were selected to analyze defense-related transcription, as these transcripts represent members of the pr family of proteins, which have been widely implicated in plant resistance to pathogens  <cit> . different forms of chitinase are involved in both active and passive defense responses in plants  <cit> . glucanases have also been implicated in plant resistance to pathogens, and beta- <dig>  3-glucanases comprise part of the pr- <dig> group of pathogenesis-related genes  <cit> .\n",
            "\n",
            "the fact that pal, chitinase and glucanase transcripts were all reduced in introduced tetraploids compared to native tetraploids  might suggest that populations of plants from the introduced range will be less defended against herbivores than natives, as is generally predicted by the eica hypothesis. some studies suggest that constitutive or basal levels of defense-related transcripts in plants, similar to those analyzed in this study, can be used to predict pathogen susceptibility and induced defense responses  <cit> . very subtle genetic mutations, such as those in the arabidopsis cpr  mutant, have been shown to increase basal levels of systemic acquired resistance, which in turn increase levels of pathogen resistance  <cit> . in addition, the over-expression of pr proteins in planta typically results in a phenotype of enhanced disease resistance  <cit> . plants with high constitutive defenses may, however, also have a lower degree of defense induction than those with low constitutive defenses  <cit> .\n",
            "\n",
            "recent reports indicate that introduced c. stoebe plants are better defended against both generalist and specialist enemies than natives  <cit> . this observation, in combination with the current study, may suggest that introduced populations have a higher potential degree of defense induction. however, the current study only measured levels of genes that may be involved in constitutive defense. thus, our results must be interpreted with caution with regard to ecological hypotheses of plant defense in biological invasions.\n",
            "\n",
            "it is important to note here that the release of c. stoebe from specialist enemies has been considered an important factor in the invasive success of the weed, and this has spurred the introduction of a number of biological control species to north america over the past thirty years  <cit> . although many of these specialist herbivores have become established and widespread, c. stoebe densities have only been reduced in a few specific areas , and the weed continues to expand its range at other sites  <cit> . interestingly, field observations in north america suggest that introduced c. stoebe experiences little pressure from generalist herbivores and pathogens , indicating that c. stoebe currently experiences a partial release from both specialist and generalist enemies in the introduced range.\n",
            "\n",
            "in order to better understand defense responses in c. stoebe, future studies should monitor gene expression and physiological responses in tetraploid geo-cytotpyes when exposed to pathogens and herbivores. this would help determine if expression of genes involved in constitutive defenses are good predictors of pathogen and herbivore susceptibility in c. stoebe. in addition, it would be interesting to test the response of c. stoebe geo-cytotypes to a variety of generalist and specialist enemies at the level of gene expression.\n",
            "\n",
            "evolutionary capacity\n",
            "the activity of transposable elements could facilitate evolution by reorganizing the genome, and may be one important aspect in this process  <cit> . therefore, we hypothesized that introduced populations of c. stoebe would have the highest expression of the transposable elements analyzed, potentially due to novel stresses encountered in the introduced range. however, this was not the case. in fact, native tetraploid populations had the highest expression rate of one catca en/spm subclass transposable element . the expression of rad, which is involved in dna recombination/repair  <cit> , was also highest in native tetraploid populations, but was not significantly different from that of introduced populations .\n",
            "\n",
            "although the expression of transposable elements could facilitate rapid evolution, transposition may not be adaptive and could cause deleterious genomic rearrangements as opposed to beneficial ones. in other studies, certain transposable elements have been detected in plants at specific growth stages or under conditions of biotic and abiotic stress  <cit> ; however, the biological role of active transposition currently remains unclear. additionally, recent evidence suggests that epigenetic mechanisms such as dna methylation and chromatin remodeling can play an important role in the regulation of gene expression in polyploids which may facilitate adaptive plasticity  <cit> . similarly, paramutation  can also result in differential regulation of genes between polyploids and their diploid progenitors  <cit> . thus, although we did not detect the changes we predicted in expression of transposable elements, it is entirely possible that factors other than chromosomal rearrangement through transposition are responsible for the observed changes in gene expression.\n",
            "\n",
            "plant ploidy\n",
            "although plant ploidy is often unaccounted for in comparisons of native and introduced populations, we found it to be a necessary and essential component for gene expression analyses. in native populations, we found lower expression of pal2a, pal2b and the transposable element in diploids compared to tetraploids, and all other genes examined showed similar relative expression . the literature suggests that gene expression rates in polyploids tend to vary depending on plant species, ploidy, genetic background, and the genes examined; however, the phenomenon of gene dosage compensation appears to be common  <cit> . this dosage effect results in gene or protein expression patterns in polyploids which are similar to their diploid progenitors. we did not necessarily expect to see this phenomenon in our plant populations because other studies involving ploidy and gene or protein expression have traditionally utilized plants with the same genetic background  <cit> , whereas evidence suggests that c. stoebe plants within the native range harbor different genetic backgrounds  <cit> . however, it appears that gene dosage compensation may be occurring to some extent in the native cytotypes of c. stoebe. additionally, we observed increased expression of two pal transcripts in native tetraploids compared to diploids, which may reflect increases in secondary compounds due to polyploidy as is seen in other plants  <cit> .\n",
            "\n",
            "interestingly, native diploids exhibited similar expression profiles for nine of the ten total genes analyzed when compared to introduced tetraploids , also suggesting gene dosage compensation. this result was rather surprising in that the diploid appears to be extremely rare  in the introduced range, whereas the introduced tetraploid is a very problematic weed. therefore, it is likely that other factors, such as plant performance characteristics, life cycle traits and the expression of other genes, are of greater importance in determining the success of tetraploids over diploids in the introduced range. overall, the observed differences in gene expression between and within ploidies highlights the importance of using appropriate plant types when examining a particular species in both the native and introduced range.\n",
            "\n",
            "alternative gene roles and regulation\n",
            "genes similar to those selected in the current study have been detected in response to a variety of cues and conditions that do not necessarily reflect their primary annotation. for instance, many genes involved in defense response  <cit> , flavonoid biosynthesis  <cit>  and active transposition  <cit>  have been detected during particular points of plant growth and development. in this study we attempted to minimize any possible developmental differences in gene expression by sampling expanded, fully developed rosette leaves of similar age from all plants. all of the plants were grown in the same greenhouse environment and at the time of sampling remained in rosette form, none showing signs of bolting. if the genes tested here were expressed predominantly in response to developmental cues, it could be expected that expression of transcripts would be extremely similar across all geo-cytotypes, which is not what was observed.\n",
            "\n",
            "additionally, it is possible that the defense genes analyzed in this study are important for aspects other than plant defense against enemies. for instance, the production of certain flavonoids are thought to play important roles in photo-protection, frost hardiness and drought resistance  <cit> , which could influence expression of pal genes. c. stoebe occupies areas in both the native and introduced range that are often subject to these types of abiotic stress  <cit> . thus, expression of pal transcripts and resulting flavonoid accumulation may be important in both the biotic and abiotic stress response of the plant.\n",
            "\n",
            "CONCLUSIONS\n",
            "although we sampled only a small subset of genes, we identified differences in gene expression between native and introduced populations of plants that may have ecological relevance. we found that introduced tetraploids exhibited lower expression of constitutive defense genes than native tetraploids, as might be predicted based on general ideas of enemy release and rapid evolution. plant origin and ploidy were found to have a significant effect on both life-cycle characteristics and gene expression. this highlights the importance of determining plant ploidy in ecological and genomics investigations, and suggests that c. stoebe invasion can be influenced by both plant ploidy and altered gene expression in the introduced range.\n",
            "\n",
            "we have demonstrated that the quantitative analyses of gene expression in native and introduced plant populations reveal trends that may provide additional insight into ecological hypotheses. however, the mechanisms underlying the observed changes in gene expression remain unclear, and further work is needed in this area. a better understanding of the genetic and molecular basis of invasiveness in exotic plants is not only an interesting case study in evolution, but is important to further our understanding how these invasions occur, and to choose appropriate management interventions. the techniques used in our study can provide an important complement to classical ecological measurements of plant fitness and competitive success.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "plant invasion into new environments is an extremely costly problem, which is why it is so important to study the genetic and molecular basis of invasiveness in exotic plants.\n",
            "plant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness and to choose appropriate management interventions.\n",
            "plant invasion is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness and to choose appropriate management interventions.\n",
            "plant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness and to choose appropriate management interventions.\n",
            "plant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness and to choose appropriate management interventions.\n",
            "plant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness and to choose appropriate management interventions.\n",
            "plant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invasiveness.\n",
            "plant invasion into new environments is an extremely costly problem, and it is important to understand the genetic and molecular basis of invas\n",
            "89\n",
            "[{'summary_text': 'in martinique,  <dig> detoxification genes were over-transcribed in larvae and adults of the vauclin strain.\\nin addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\\nin addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\\nin addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\\nin addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\\nin addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\\nin addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\\nin addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 2538 to 3072 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "every year,  <dig> to  <dig> million dengue infections world-wide causing from  <dig>  to  <dig>  deaths from dengue and hemorrhagic fever are recorded  <cit> . as there is still no medicine and efficient vaccine available, vector control by the recourse of environmental management, educational programs and the use of chemical and biological agents, remains the only method to reduce the risk of dengue virus transmission  <cit> . unfortunately, most of dengue vector control programs implemented worldwide are facing operational challenges with the emergence and development of insecticide resistance in ae. aegypti  <cit>  and ae. albopictus  <cit> . resistance of ae. aegypti to insecticides has been reported in many regions including south east asia  <cit> , latin america  <cit>  and the caribbean  <cit> .\n",
            "\n",
            "inherited resistance to chemical insecticides in mosquitoes is mainly the consequence of two distinct mechanisms: the alteration of target sites inducing insensitivity to the insecticide  and/or an increased metabolism of the insecticide   <cit> . metabolic-based resistance involves the bio-transformation of the insecticide molecule by enzymes and is now considered as a key resistance mechanism of insects to chemical insecticides  <cit> . this mechanism may result from two distinct but additive genetic events: i) a mutation of the enzyme protein sequence leading to a better metabolism of the insecticide, and/or ii) a mutation in a non-coding regulatory region leading to the over-production of an enzyme capable of metabolizing the insecticide. so far, only the second mechanism has been clearly associated with the resistant phenotype in mosquitoes. three large enzyme families, the cytochrome p <dig> monooxygenases , glutathione s-transferases  and carboxy/cholinesterases  have been implicated in the metabolism of insecticides  <cit> . the rapid expansion and diversification of these so-called 'detoxification enzymes' in insects is likely to be the consequence of their adaptation to a broad range of natural xenobiotics found in their environment such as plant toxins  <cit> . these enzymes have also been involved in mosquito response to various anthropogenic xenobiotics such as heavy metals, organic pollutants and chemical insecticides  <cit> .\n",
            "\n",
            "although identifying metabolic resistance is possible by toxicological and biochemical techniques, the large panel of enzymes potentially involved together with their important genetic and functional diversity makes the understanding of the molecular mechanisms and the role of particular genes a challenging task. as more mosquito genomes have been sequenced and annotated  <cit> , the genetic diversity of genes encoding mosquito detoxification enzymes has been unravelled and new molecular tools such as the aedes and anopheles 'detox chip' microarrays allowing the analysis of the expression pattern of all detoxification genes simultaneously have been developed  <cit> . these specific microarrays were successfully used to identify detoxification genes putatively involved in metabolic resistance in various laboratory and field-collected mosquito populations resistant to insecticides  <cit> .\n",
            "\n",
            "in latin america and the caribbean, several ae. aegypti populations show strong resistance to pyrethroid, carbamate and organophosphate insecticides correlated with elevated activities of at least one detoxification enzyme family  <cit> . in addition, several points of non-synonymous mutations in the gene encoding the trans-membrane voltage-gated sodium channel  have been described and showed to confer resistance to pyrethroids and ddt  <cit> .\n",
            "\n",
            "several questions remain concerning the impact of insecticide resistance on the efficacy of vector control operations. in martinique , high levels of resistance to the organophosphate temephos and the pyrethroid deltamethrin were reported. this resistance was characterized by an important reduction of both mosquito knock-down and mortality levels after thermal-fogging with deltamethrin and p450-inhibitor synergized pyrethroids, indicating that resistance was negatively impacting on control programmes and that this resistance was conferred, at least in part, by elevated cytochrome p <dig> activity  <cit> .\n",
            "\n",
            "in this study, we explored the mechanisms conferring insecticide resistance in an ae. aegypti population from martinique island. larval bioassays and adult topical applications were used to determine the current resistance level of this population to insecticides. the presence of metabolic-based resistance mechanisms was investigated by exposing mosquitoes to enzyme inhibitors prior to bioassays with insecticides and by measuring representative enzyme activities of each detoxification enzyme family. at the molecular level, the frequency of the target-site kdr mutation was investigated and a microarray approach followed by quantitative real-time rt-pcr validation was used to identify detoxification genes putatively involved in metabolic resistance. results from this study will help to implement more effective resistance management strategies in this major disease vector in the future.\n",
            "\n",
            "RESULTS\n",
            "larval bioassays  showed that the vauclin strain is far less affected by temephos than the susceptible bora-bora strain . in the susceptible strain, temephos toxicity was not significantly increased in the presence of detoxification enzyme inhibitors . by contrast, the level of resistance to temephos of the vauclin strain was significantly reduced in the presence of pbo, def and dmc  indicating the involvement of p450s, cces and in a lesser extent gsts in the resistance of larvae to temephos.\n",
            "\n",
            "resistant ratios rr <dig> and rr <dig> were obtained by calculating the ratio between the lc <dig> and lc <dig> between vauclin and bora-bora strains; synergism ratios sr <dig> and sr <dig> were obtained by calculating the ratio between lc <dig> and lc <dig> with and without enzyme inhibitor. : confidence interval. significant rr and sr are shown in bold.\n",
            "\n",
            "topical applications of the pyrethroid insecticide deltamethrin on adults of each strain  revealed that the vauclin strain is also highly resistant to deltamethrin . in both strains, the toxicity of deltamethrin increased significantly in the presence of detoxification enzyme inhibitors, however only pbo and dmc induced higher synergistic effects in the vauclin strain than in the susceptible bora-bora strain . in the vauclin strain, pbo and dmc significantly reduced the resistance level , indicating a significant role of p450s and gsts in the resistance of adults to deltamethrin.\n",
            "\n",
            "resistant ratios rr <dig> and rr <dig> were obtained by calculating the ratio between the ld <dig> and ld <dig> between vauclin and bora-bora strains; synergism ratios sr <dig> and sr <dig> were obtained by calculating the ratio between ld <dig> and ld <dig> with and without enzyme inhibitor. : confidence interval. significant rr and sr are shown in bold.\n",
            "\n",
            "comparison of constitutive detoxification enzyme activities between the susceptible strain bora-bora and the insecticide-resistant vauclin strain revealed significant differences at both larval and adult stages . p <dig> activities were elevated in both larvae and adults of the vauclin strain . similarly, gst activities were found elevated in larvae and adults of the vauclin strain . finally, α- and β-carboxylesterase activities were also found slightly elevated in the vauclin strain in larvae  and adults .\n",
            "\n",
            "sequencing of the voltage-gated sodium channel gene conducted on the vauclin strain showed the presence of the kdr mutation at position  <dig>  leading to the replacement of valine by an isoleucine  at a high allelic frequency  =  <dig> , n = 24) with rr =  <dig>  rs =  <dig> and ss =  <dig>  conversely, no kdr resistant allele was detected in the susceptible bora-bora strain .\n",
            "\n",
            "we used the microarray 'aedes detox chip'  to compare the transcription levels of all ae. aegypti detoxification genes between the insecticide-resistant strain vauclin and the susceptible strain bora-bora in larvae and adults. overall,  <dig> and  <dig> probes out of  <dig> were detected consistently in at least  <dig> hybridisations out of  <dig> in larvae and adults respectively. among them,  <dig> detoxification genes were significantly differentially transcribed  in larvae or adults . most of these genes encode p450s  with  <dig> of them being differentially transcribed in the vauclin strain at both life stages .\n",
            "\n",
            "in larvae,  <dig> genes  were found significantly differentially transcribed between the insecticide-resistant strain vauclin and the susceptible strain bora-bora . among them,  <dig> genes were over-transcribed in the vauclin strain while only  <dig> genes were under-transcribed. most over-transcribed genes were represented by cyp genes with a majority belonging to the cyp <dig> subfamily . three cyp9s were also over-transcribed in larvae of the vauclin strain  with a strong over-transcription of cyp9j <dig>  together with  <dig> cyp4s . among other over-transcribed genes,  <dig> carboxy/cholinesterases  and  <dig> glutathione s-transferase  were slightly over-transcribed in the vauclin strain. lastly,  <dig> cyps  were under-transcribed in insecticide-resistant larvae comparatively to susceptible larvae.\n",
            "\n",
            "in adults,  <dig> genes  were found differentially transcribed in the insecticide-resistant strain vauclin comparatively to the susceptible strain bora-bora . as in larvae, most of the over-transcribed genes belong to the cyp <dig> and cyp <dig> subfamilies  with only  <dig> additional cces  being moderately over-transcribed in the vauclin strain. nine genes were under-transcribed in vauclin adults, including  <dig> cyps ,  <dig> cce ,  <dig> gst  and  <dig> thioredoxin peroxidases . interestingly cyp304c <dig> and tpx <dig> were both found strongly under-transcribed  in insecticide-resistant adults.\n",
            "\n",
            "validation of microarray data was performed by real-time quantitative rt-pcr on  <dig> detoxification genes identified as over-transcribed in larvae or adults of the vauclin strain . the over-transcription of genes identified from microarray experiments were all confirmed by quantitative rt-pcr in both life stages, although expression ratios obtained from rt-pcr were frequently higher than those obtained from microarray experiments.\n",
            "\n",
            "discussion\n",
            "the aim of the present study was to investigate insecticide resistance mechanisms of ae. aegypti mosquitoes from martinique .\n",
            "\n",
            "toxicological results confirmed the high level of resistance of the vauclin strain from martinique to the organophosphate temephos at the larval stage and to the pyrethroid deltamethrin at the adult stage  <cit> . the use of specific detoxification enzyme inhibitors suggested that resistance of larvae to temephos is linked to carboxylesterases and to a lesser extent p450s and gsts. in adults, resistance to deltamethrin appeared principally linked to p450s and gsts. comparison of global detoxification enzyme activities between the two strains revealed elevated p450s, gsts and in a lesser extent cces activities in the vauclin strain at both life-stages, confirming the importance of metabolic resistance mechanisms in martinique.\n",
            "\n",
            "carboxylesterases based-resistance mechanism is a major mechanism for organophosphate resistance in insects  <cit> . several examples of ae. aegypti resistance to organophosphates in the caribbean linked to elevated carboxylesterases activities have been described  <cit> . our toxicological and biochemical data confirms these observations despite a moderate elevated level of cces activities in the vauclin strain. among detoxification enzymes, p450s have been shown to play a major role in pyrethroid resistance in insects  <cit> . in martinique, marcombe et al.  <cit>  suggested the involvement of p450s in the reduced efficacy of deltamethrin space-spray operations. elevated gst levels have also been frequently associated with insect resistance to insecticides such as ddt and pyrethroids  <cit> . our toxicological and biochemical data support the role of p450s and gsts in insecticide resistance in martinique.\n",
            "\n",
            "at the molecular level, several mutations in the voltage-gated sodium channel gene have been associated with pyrethroid resistance in ae. aegypti from asian, latin american and caribbean countries  <cit> . our results revealed a high frequency  of the v1016i kdr mutation in ae. aegypti populations from the community of vauclin. the role of this mutation in pyrethroid resistance was clearly demonstrated by genotype-phenotype association studies  <cit> . the high frequency of the mutation, together with the incomplete effect of enzyme inhibitors in adults, supports a contribution of this kdr mutation in deltamethrin resistance.\n",
            "\n",
            "acetylcholinesterase  is critical for hydrolysis of acetylcholine at cholinergic nerve synapses and is a target for organophosphate and carbamate insecticides  <cit> . altered ache is an important resistance mechanism to organophosphates in many insects. following the methods of alout et al.  <cit>  and bourguet et al.  <cit> , ache activities of vauclin mosquitoes were determined to investigate the presence of the g119s and/or f290v mutations. no insensitive ache phenotypes were found in any of the mosquitoes tested , suggesting that organophosphate resistance of the vauclin strain is rather due to detoxification enzymes unless other mutations occurred elsewhere in the ace gene.\n",
            "\n",
            "our microarray screening identified  <dig> and  <dig> over-transcribed detoxification genes in larvae and adults of the vauclin strain respectively. among them,  <dig> p450s , the glutathione s-transferase gste <dig> and the carboxy/cholinesterase cceae3a were all confirmed to be over-transcribed at both life-stages, supporting their involvement in insecticide-resistance. other genes appeared more highly over-transcribed in adults  or in larvae , suggesting that particular enzymes might be more specifically involved in resistance to one insecticide during a particular life-stage as argued by paul et al.  <cit> . validation of transcription profiles by real-time quantitative rt-pcr was successful for the  <dig> genes tested although expression ratios obtained with rt-pcr were often higher. the underestimation of transcription ratios obtained from microarray data is likely due to technical issues and has been previously evidenced in other studies  <cit> .\n",
            "\n",
            "over-transcription of genes encoding p450s has been frequently associated with metabolic-based insecticide resistance mechanisms in insects  <cit> . in mosquitoes, the cyp6z subfamily has been previously associated with response to pyrethroid, carbamates and organochlorine insecticides. in ae. aegypti, cyp6z <dig> has been found 4-fold over-transcribed in a permethrin-resistant strain collected in northern thailand  <cit> . in two recent studies, cyp6z <dig> was also identified as inducible by permethrin and other pollutants  <cit> . in an. gambiae, cyp6zs have been frequently found constitutively over-transcribed in permethrin- and ddt-resistant strains  <cit> . recent studies demonstrated that the enzyme encoded by an. gambiae cyp6z <dig> can metabolize the insecticides carbaryl and ddt while cyp6z <dig> with a narrower active site, can only metabolize carbaryl  <cit> . recently, another an. gambiae p <dig> , was shown to be able to degrade pyrethroid insecticides  <cit> . the over-transcription of cyp6z <dig> in the vauclin strain may indicate the involvement of ae. aegypti cyp6zs in insecticide resistance in martinique. however, the decisive demonstration of their capability to metabolize insecticides requires further investigations.\n",
            "\n",
            "the association of cyp6ms with metabolic resistance to pyrethroids has also been previously described in mosquitoes. in ae. aegypti larvae, cyp6m <dig> and cyp6m <dig> were found inducible by permethrin and pollutants  <cit> . although no aedes cyp6ms have been found constitutively over-transcribed in other insecticide-resistant strains, an. gambiae cyp6m <dig> was found significantly over-transcribed in various strains resistant to pyrethroids  <cit> . recent studies indicate that cyp6m <dig> is able to metabolize pyrethroid insecticides . our results suggest that ae. aegypti cyp6m <dig> and cyp6m <dig>  with protein sequences similar to an. gambiae cyp6m <dig>  might also be involved in resistance of ae. aegypti to pyrethroids in martinique.\n",
            "\n",
            "finally, the glutathione s-transferase gste <dig> and the carboxy/cholinesterase cceae3a were both found over-transcribed in both life-stages of the vauclin strain. the role gsts in resistance to chemical insecticides has been previously evidenced in insects with the enzyme encoded by an. gambiae gste <dig> metabolizing ddt  <cit>  and the housefly mdgst6-a metabolizing two organophosphate insecticides  <cit> . in ae. aegypti, gste <dig> also metabolises ddt and is over-transcribed in a pyrethroid and ddt-resistant strain from thailand  <cit> . in  <dig>  strode et al.  <cit>  also revealed the over-transcription of gste <dig> in pyrethroid-resistant mosquitoes. our results confirm that gste <dig> might have a role in insecticide resistance in ae. aegypti. over-production of carboxylesterases has been showed to play an important role in resistance to organophosphate insecticides in mosquitoes  <cit> . elevated esterase activities conferring resistance to organophosphate insecticides has usually been linked to genomic amplification of specific alleles although gene over-transcription may also be involved  <cit> . considering the high resistance of larvae of the vauclin strain to temephos, over-transcribed cces represent good candidates for organophosphate metabolism in ae. aegypti.\n",
            "\n",
            "it has been suggested that insecticide resistance could be accentuated by the exposure of mosquito populations to pollutants and pesticides used in agriculture  <cit> . in martinique, bananas, sugar cane, and pineapple represent important cultured surface areas often localized near mosquito breeding sites. these cultures have been submitted for decades to heavy use of insecticides such as the organochlorates aldrin, dieldrin and chlordecone and herbicides such as the triazine simazine, the pyridines paraquat and glyphosate  <cit> . this particular situation is likely to have contributed to the high resistance of ae. aegypti to chemical insecticides and to the selection of particular detoxification genes in martinique.\n",
            "\n",
            "CONCLUSIONS\n",
            "we have identified multiple insecticide resistance mechanisms in ae. aegypti mosquitoes from martinique  significantly reducing the insecticidal activity of insecticides used for their control. microarray screening identified multiple detoxification genes over-transcribed at both life-stages in resistant mosquitoes, suggesting their possible involvement in insecticide-resistance. further experimental validation by using enzyme characterization and rna interference will allow confirming the role of these genes in the resistance phenotype. as previously shown in mosquitoes  <cit> , the epistasis between the kdr mutation and particular p450s genes is likely to contribute to the high level of resistance to pyrethroids in ae. aegypti from martinique and might seriously threatens the control of dengue vectors in the future. a better understanding of the genetic basis of insecticide resistance is an essential step to implement more effective vector control strategies in the field in order to minimize dengue outbreaks.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in martinique,  <dig> detoxification genes were over-transcribed in larvae and adults of the vauclin strain.\n",
            "in addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\n",
            "in addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\n",
            "in addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\n",
            "in addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\n",
            "in addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\n",
            "in addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\n",
            "in addition,  <dig> detoxification genes were over-transcribed in insecticide-resistant larvae and adults of the vauclin strain.\n",
            "\n",
            "90\n",
            "[{'summary_text': 'our new normalization method does not require that the majority of genes be non-differentially expressed, and the numbers of over- and under-expressed genes are roughly equal; it appears much more effective than standard normalization methods when the numbers of over- and under-expressed genes are unequal, and the numbers of over- and under-expressed genes are roughly equal.\\nour new normalization method does not require that the majority of genes be non-differentially expressed, and it appears much more effective than standard methods when the numbers of over- and under-expressed genes are roughly equal; the same assumptions as those for dual-labeled cdna arrays.\\nwe have developed an adaptive method based on three-component mixture model for normalization of dual-labeled microarray data.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 1160 to 2048 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "microarray technology provides simultaneous measurements of expression levels for thousands of genes. each step from sample preparation to data analysis, however, contains potential sources of bias and variability. proper normalization adjusts for differences which interfere with the comparison of intensities of different labels at a given probe and with the comparison of intensities of corresponding probes on different arrays. proper data normalization should allow for the comparison of expression levels across different arrays. subsequent data analysis results are heavily dependent on effective normalization.\n",
            "\n",
            "normalization issues differ for dual-labeled platforms compared to single labeled platforms such as the affymetrix genechip arrays. in this paper we address normalization for dual-labeled arrays with either cdna or oligonucleotide probes. the objective of normalization for dual-labeled arrays is to correct for differences in intensities for the two labels on the same array. these differences arise from factors such as differences in sample concentrations, differences in photomultiplier tube setting, and differences in the affinity of the two labels for dna.\n",
            "\n",
            "median or mean based global normalization methods use a single normalization factor applied to all genes on the array to adjust for labeling bias  <cit> . such methods are widely used because of their simplicity. intensity-based and location-based methods take into account intensity and spatial dependence on dye bias normalization factors  <cit> . both global and intensity/location based normalization methods assume that most of the genes are not differentially expressed between the two samples hybridized on the array, and that for the differentially expressed genes, the direction of the difference is symmetric between the two samples. in many important cases, however, these assumptions are not appropriate because: 1) more than half of the genes are differentially expressed on the array; 2) the numbers of over- and under-expressed genes on the array are unequal; 3) only genes of specific biological interest are selected to make a customized array, which are highly variable across the samples. in the above cases, the global normalization methods and intensity/location based normalization methods become less accurate and a more sophisticated method is needed  <cit> .\n",
            "\n",
            "there are some methods which attempt to adaptively identify the subset of 'housekeeping' genes  <cit> . these methods require multiple arrays in order to identify the 'housekeeping' gene set, which does not always exist.\n",
            "\n",
            "newton et al. proposed a gamma-gamma-bernoulli model for identifying differentially expressed genes in dual labeled arrays  <cit> . we have generalized newton's model and here propose an adaptive method based on three-component mixture model for normalization of dual labeled microarray data.\n",
            "\n",
            "RESULTS\n",
            "as described in the methods section, we have applied our adaptive method to both the simulated data and real data. we have also compared our method with the global method and the intensity-based lowess method.\n",
            "\n",
            "results of the simulation studies are shown as bar plots in figure  <dig>  figure 1a shows the comparison of our adaptive method, the global method and the lowess method when no noise was added. when the majority of genes in the array were non-differentially expressed , or the numbers of over- and under-expressed genes on the array were equal , the root mean squared error  of the adaptive method was comparable with the other two methods; all were very small. when the array contained unequal numbers of over- and under-expressed genes and when the majority of genes were differentially expressed , the rmses of the global normalization method and the lowess method were much larger than those of the adaptive method. the differences ranged from around a two fold difference  when the number of under-, null, and over-expressed genes were  <dig>   <dig>  and  <dig>  to more than a three fold difference  when the number of under-, null, and over-expressed genes were  <dig>   <dig>  and  <dig>  the rmses for the adaptive method ranged from  <dig>  to  <dig>  in log <dig> scale.\n",
            "\n",
            "we compared the histogram of observed intensities to the fitted marginal density from the adaptive method as a simple check to see whether the proposed model and the estimation procedure are in line with available data. figure  <dig> shows the histograms of log and log intensities of red and green channels of the simulated data, and the curve in each plot is the estimated density obtained from the fitted model. it is seen the data fits to the model quite well.\n",
            "\n",
            "gaussian noise with sd of  <dig>  and  <dig>  were added so that the data was not generated from the same model used for analysis with the adaptive method. the rmses of the global normalization method and the lowess method remained large, while the rmses of the adaptive method remained small, ranging from  <dig>  to  <dig>  on the log <dig> scale .\n",
            "\n",
            "in the above simulation, no apparent groups could be seen in the histograms of log . better results for the adaptive method were also obtained for a simulation case where the three groups  are apparent in the histogram of log. the results can be seen in figure  <dig> and figure  <dig> .\n",
            "\n",
            "results comparing rmses for the adaptive method, the global method and the lowess method with real data are shown in table  <dig>  the rmses of the adaptive method on data generated from ten different arrays ranged from  <dig>  to  <dig> , in comparison with rmses of around  <dig>  using the global normalization method. the average rmse  of the lowess method is almost two times that of our adaptive method , although the lowess method performed better than the global method . figure  <dig> shows the histograms of log and log intensities of red and green channels of the real data, and the curve in each plot is the estimated density from the adaptive method.\n",
            "\n",
            "discussion\n",
            "in this paper, we propose a new method for normalization of dual-labeled arrays in cases where the number of differentially expressed genes is substantial and not necessarily symmetric in direction. the method performed effectively with both simulated and real data.\n",
            "\n",
            "we started our model building initially by introducing an unknown constant c into newton's gamma-gamma-bernoulli model  <cit> . the mixture model consisted of two groups: non-differentially expressed genes  and differentially expressed genes .\n",
            "\n",
            "log ~ gamma\n",
            "\n",
            "log ~ gamma     \n",
            "\n",
            "sk ~ gamma\n",
            "\n",
            "log ~ gamma\n",
            "\n",
            "log ~ gamma     \n",
            "\n",
            " ~ gamma\n",
            "\n",
            " ~ gamma\n",
            "\n",
            "we found that when the differential expression was symmetric between the two samples, the model worked well. however, the error increased significantly when the ratio of the numbers of under- to over-expressed genes shifted from  <dig> \n",
            "\n",
            "in order to make the model more flexible, we modified the model by assigning different scale factors γr and γg for the red channel and green channel intensities. for this modified two-component mixture, the error still remained large. we then extended the model into a three-component mixture model listed as equations 8a-8c in the additional material . the model was then quite flexible but there were too many parameters that needed to be optimized. after we tested it with simulation data and real data, we found the estimated model was not stable and difficult to optimize. we finally simplified the model to our final model given by equations 2a-2c . when applying it to real data or simulated data, the estimates converged well close to globe optima. when different start points were used, the optimizations remained relatively robust.\n",
            "\n",
            "evaluation of normalization methods can be difficult since the true normalization factors are unknown with real data for custom arrays. we avoided this problem by synthesizing customized arrays based on real data for standard arrays containing thousands of genes. in order to make the distribution of each component group look smoother, we allowed certain range of overlap between the adjacent groups. additional sampling method was tried to divide the whole distribution range into many non-overlapping intervals. in each interval the number of genes sampled increased when the absolute value of log <dig> became larger . the model fitting results using data generated by this sampling method are listed in table  <dig>  and figure  <dig> .\n",
            "\n",
            "we compared our adaptive method with the global method and the intensity-based lowess method. the lowess method assumes that in each intensity interval either the majority of genes are non-differentially expressed or the numbers of up- and down-regulated genes are equal. the global median normalization makes these assumptions only over the array as a whole. it is not surprising that our method performed much better that the above two methods, because the global median method only works well when the assumptions are valid while the intensity-based lowess method is only effective when there are intensity-dependent biases.\n",
            "\n",
            "correlation structure is complicated for the thousands of genes on a microarray. in our model, the intensity of each channel is conditionally independent given the scale parameter, but not marginally independent.therefore, we are not assuming the intensities in two channels are independent. although we did not generate correlated genes in our simulated data sets, correlations of genes do exist in the real data sets we tested. spatial correlations are also possible but our method is not designed for that purpose. yang et al. proposed using the lowess normalization separately within each grid on the array  <cit> . our algorithm could be similarly applied within each grid to control for spatial effects.\n",
            "\n",
            "limited simulations were performed in this study. we also tried to use real data to test our method. since an appropriate data set with known normalization factor was not available, we synthesized such data sets by sub-setting large arrays in which the true normalization factor could be accurately estimated. in the process of synthesizing such small arrays we had to choose an empirical threshold to stratify the differentially expressed genes and non-differentially expressed genes. although we do not believe that the superiority shown for our algorithm depends critically on the threshold chosen nor on details of the synthesis, it would be preferable to evaluate the algorithm on real data sets with know normalization factors.\n",
            "\n",
            "although our method is designed for dual-labeled cdna array, it can be extended to single channel affymetrix chip data. the most popular normalization method for the affymetrix chip compares each array to a single base line array for probe set summaries. the assumptions behind the normalization method are that the majority of the genes are non-differentially expressed and the numbers of over- and under-expressed genes are roughly equal; the same assumptions as those for dual-labeled cdna arrays. we could treat the base line array as the 'reference channel' and the other array as the 'test' channel and apply our algorithm to probe set summaries. for affymetrix chip data, there are multiple base pairs in a probe set and each probe has an intensity measurement. several alternative normalization methods of affymetrix arrays utilize the probe level information. for example, method based on an 'invariant set' proposed by li and wong assumes that a probe of a non-differentially expressed genes in two arrays to have similar ranks and uses an iterative procedure to identify the invariant set which presumably consists of points from non-differentially expressed genes  <cit> .\n",
            "\n",
            "CONCLUSIONS\n",
            "our new normalization method does not require that the majority of genes be non-differentially expressed, and doesn't require multiple array replicates, dye swaps, spiked controls, or housekeeping genes. it appears much more effective than standard methods when the numbers of over- and under-expressed genes are unequal, and the majority of the genes are differentially expressed. it can be very useful for general microarray platforms when samples with very different expression profile are co-hybridized and for custom arrays where the majority of genes are likely to be differentially expressed. in both of these settings, standard normalization methods are problematic.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "our new normalization method does not require that the majority of genes be non-differentially expressed, and the numbers of over- and under-expressed genes are roughly equal; it appears much more effective than standard normalization methods when the numbers of over- and under-expressed genes are unequal, and the numbers of over- and under-expressed genes are roughly equal.\n",
            "our new normalization method does not require that the majority of genes be non-differentially expressed, and it appears much more effective than standard methods when the numbers of over- and under-expressed genes are roughly equal; the same assumptions as those for dual-labeled cdna arrays.\n",
            "we have developed an adaptive method based on three-component mixture model for normalization of dual-labeled microarray data.\n",
            "\n",
            "91\n",
            "[{'summary_text': 'single incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopic surgery for dogs undergoing elective splenectomy.\\nsingle incision laparoscopic surgery  reduces surgical trauma and adhesion through implementing a small number and/or size of portals.\\nsingle incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopic surgery for dogs undergoing elective splenectomy.\\nsingle incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopic surgery for dogs undergoing elective splenectomy.\\nsingle incision laparoscopic surgery  reduces surgical trauma and adhesion through implementing a small number and/or size of portals.\\nsingle incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopic surgery for dogs undergoing elective splenectomy.\\nsingle incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopic surgery for dogs undergoing elective splenectomy.\\nsingle incision laparoscopic surgery  reduces surgical trauma and adhesion through implementing a small number and/or size of portals.\\nsingle incision'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 10320 to 11264 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "the application of laparoscopy, minimally invasive technique, along with its advantages and superiorities, has become an alternative approach to conventional surgery in small animal veterinary medicine\n",
            " <cit> . laparoscopy created huge changes in the field of surgery from large incisions in open surgeries to very small incisions. within the context of laparoscopy, single incision laparoscopy is a newly growing technique to reduce the invasiveness of conventional multiple portal laparoscopic surgery. single incision laparoscopic surgery  reduces surgical trauma and adhesion through implementing a small number and/or size of portals\n",
            " <cit> . sils is a feasible technique for different ablative and reconstructive procedures such as cholecystectomy, appendectomy and splenectomy in human\n",
            " <cit> .\n",
            "\n",
            "splenectomy in dog is a common operation for tumors\n",
            " <cit> . in human, hematological immune mediated diseases, non responsive to medical treatment, itp and hemolytic anemia are the most common indications of splenectomy\n",
            " <cit> . the ideal indication for the laparoscopic splenectomy is an elective splenectomy in blood donor dogs to prevent transmission of hemobartonella infection\n",
            " <cit> . clinical and experimental researches were conducted to elaborate laparoscopic and open splenectomy\n",
            " <cit>  resulting in the recognition of laparoscopy as a gold standard procedure for splenectomy\n",
            " <cit> . in veterinary medicine, the feasibility of multiple portal laparoscopic splenectomy and its superiority over the conventional open technique was documented\n",
            " <cit> . conventional laparoscopy performed safely in laboratory animals, porcine, caprine, canine and human\n",
            " <cit> . the objective of this study was to compare single  with three portal  laparoscopic splenectomy in dogs.\n",
            "\n",
            "RESULTS\n",
            "both sils and conventional surgery were performed successfully and all dogs were recovered uneventfully. splenic mobilization was successfully performed via a single umbilical incision. the final incision was extended for organ removal. there were significant difference  between sils and conventional surgery in the operative time  and the length of the surgical scar . there was no significant difference in the length, diameter and weight of the spleen between two groups . there were no post-operative complications including inflammation, infection, hernia formation and dehiscence up to one month after surgery. slight and superficial rupture of spleen with very negligible and minor bleeding occurred in  <dig> cases  that was managed immediately without any particular requirement to perform open surgery or using additional portals to accomplish operation.\n",
            "\n",
            "discussion\n",
            "the purpose of this study was to investigate the possibility of replacing sils with conventional standard  <dig> portals laparoscopy for splenectomy in dog. accordingly, the operative time and scar length were found to be significantly less in sils compared to conventional method. single portal position in sils provided similar visualization, manipulation and exposure of splenic hilum as in conventional laparoscopy. insertion of single umbilical portal reduced the chance of accidental injury to splenic parenchyma compared to inserting three separate portals in conventional method.\n",
            "\n",
            "recently, the application of sils techniques has been described in many intra-abdominal procedures in human\n",
            " <cit> . the goal of single port access  surgery is to minimize the incision required to perform the procedure while maintaining the surgeon’s comfort\n",
            " <cit> . to the authors’ knowledge, this is the first report in using sils for splenectomy in dog. in general, less morbidity, short length of hospital stay, less post operative pain and excellent cosmetic results were considered as advantages of sils to the conventional multiple portals laparoscopic surgeries\n",
            " <cit> . moreover, sils splenectomy seems to be safe for intra operative visualization of the splenic hilum during transection of vessels and removal of spleen\n",
            " <cit> .\n",
            "\n",
            "the operative time is an important parameter for surgical assessment. in the present study, the operative time was shorter in sils  than conventional method of laparoscopy  for splenectomy in dog. part of this difference might be due to the use of ultracision harmonic scalpel for sealing and cutting of the vessels and splenic attachments. also, the experience of the surgeons has great impact on the outcome and operative time of this study. in one study on splenectomy using conventional laparoscopy in dog, the operative time was quite long in conventional three portals laparoscopy  compared to open surgery . apparently, experience of the surgeon could explain, in part, a long operative time. in other words, laparoscopic surgical times and complications tend to decrease with an increase in the level of the surgeon's experience, denominated learning curve\n",
            " <cit> .\n",
            "\n",
            "the most time consuming part of the laparoscopic splenectomy is the time dedicated to remove spleen from the abdomen, which may be associated with the rupture of spleen. in the present study,  <dig> dogs  had slight and superficial rupture of spleen with negligible minor bleeding. this was managed successfully but elongated the surgical time.\n",
            "\n",
            "CONCLUSIONS\n",
            "the single incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopy for dogs undergoing elective splenectomy. it presented advantages in relation to operative time and surgical scar, without any particular complication. the use of the appropriate vessel sealer to ensure hemostasis would facilitate the procedure and decreases the operative time.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "single incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopic surgery for dogs undergoing elective splenectomy.\n",
            "single incision laparoscopic surgery  reduces surgical trauma and adhesion through implementing a small number and/or size of portals.\n",
            "single incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopic surgery for dogs undergoing elective splenectomy.\n",
            "single incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopic surgery for dogs undergoing elective splenectomy.\n",
            "single incision laparoscopic surgery  reduces surgical trauma and adhesion through implementing a small number and/or size of portals.\n",
            "single incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopic surgery for dogs undergoing elective splenectomy.\n",
            "single incision laparoscopic surgery  could be an available, feasible and safe alternative to multiport laparoscopic surgery for dogs undergoing elective splenectomy.\n",
            "single incision laparoscopic surgery  reduces surgical trauma and adhesion through implementing a small number and/or size of portals.\n",
            "single incision\n",
            "92\n",
            "[{'summary_text': 'the edr method exhibits consistently improved specificity and sensitivity  over other methods in testing diverse data sets that vary in the number of null hypotheses.\\nthe edr method exhibits consistently improved specificity and sensitivity  in testing diverse data sets that vary in the number of null hypotheses.\\nthe edr method exhibits consistently improved specificity and sensitivity  in testing diverse data sets that vary in the number of null hypotheses.\\nthe edr method exhibits consistently improved sensitivity  over other methods in testing diverse data sets that vary in the number of null hypotheses.\\nthe edr method exhibits consistently improved specificity and sensitivity  over other methods in testing diverse data sets that vary in the number of null hypotheses.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 9155 to 9216 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "the microarray has become an important platform for a variety of bioscience and medical research areas. it allows researchers to detect the expression of thousands of genes simultaneously and to identify the differentially expressed genes  based on statistical analysis of sample comparisons. however, due to the large number of tests that are performed, there are anticipated errors in identification of degs, and it is important to compute the error rate. this information aids in the initial evaluation of the discovery and also reduces the cost of validation experiments.\n",
            "\n",
            "to date, many different multiple test methods that detect overall type i error rates have been used to interpret microarray experiments  <cit> . the original multiple test methods were created for comparisons that were carried out many times simultaneously. for example, to test a drug's effects on several groups of subjects  <cit> , one wants to report the probability of at least one null hypothesis  in the side effects of this drug. fwer  is derived from the total number  of p-values at the significance level , e.g. α* =  <dig> - m  <cit> . all tests are subjectively dependent, i.e. all p-values are related as a whole or as a single subject, for example the side effects. all test p-values must be used in this multiple test method. in microarray experiments, p-values are generated for tens of thousands of genes. each p-value has no meaning with regard to other p-values, and all p-values are not related as a whole or to a single subject even though subgroups of genes exhibit biological dependency to some extent. therefore the rationale of an fwer derived from the total gene number would need further evaluation in microarray experiments. for example, it may not matter if one is certain  that a gene list contains one or two false positives, i.e. fwer is  <dig> . the common criticism of fwer is that it is too stringent and that it lacks power because fwer increases exponentially with the number of tested hypotheses  <cit> .\n",
            "\n",
            "a commonly used multiple test alternative to fwer, false discovery rate  is defined  <cit>  as the expectation  of false positive genes  among selected genes , i.e. fdr = e. while fdr is conceptually appealing, there is no way to identify the number of actual false positives v. multiple test procedures thus estimate the proportion  of null hypothesis genes and multiply it by the total gene number and a false cutoff   <cit> : v = p <dig> m. α. there are several variations of fdr methods  <cit> , but the main difference is in the method of estimating this p <dig>  thus, all of these fdr methods are still constrained by the total gene number on the array, similar to the use of an fwer. the permutation approach for fdr counts the times that occur at a lower p-value than the actual p-value  <cit> . an ideal fdr method should present the false discovery rate within a given gene list. it is apparent, however, that permutation deduction does not directly reflect the false rate within the genes selected. also, it possesses low power for microarray experiments that have a small number of sample replicates. the bayesian model-based algorithm for fdr relies on a prior probability calculation  <cit> , however, in real cases, this prior determination is biased.\n",
            "\n",
            "previously reported multiple test methods are constrained by the total gene number used. problems arise when an experiment uncovers only a few degs among a large number of total genes. these few but real positive genes result in a very high fdr and therefore could be eliminated from the resultant deg list. because of this problem, reduction of the total gene number by gene filtering was reported to increase the power of the multiple testing  <cit> . this approach is questionable because one could eventually reach a lower fdr with continued filtering. the total gene list should not be filtered except for reasons of poor quality or violation of the multiplicity concept.\n",
            "\n",
            "researchers are often confused by the availability of so many different multiple test procedures. thus they typically try several procedures at the same time and only report the most satisfying one, or they try to filter genes until they reach a satisfying result. it is apparent that with these approaches, the results from different research reports may not be comparable. here we contrast parallel multiplicity to the traditional simultaneousness concept, and propose a new multiple test called the error discovery rate  for microarray experiments. this method overcomes the common problem of previous multiple test procedures where the type i error rate detection has low power when the total gene number used is large and provides an alternative or standard type i error rate method.\n",
            "\n",
            "RESULTS\n",
            "null hypothesis distribution and false discovery rate \n",
            "in a microarray experiment, one simultaneously conducts tens of thousands of individual gene hypothesis tests ,\n",
            "\n",
            " hi:μi=0  \n",
            "\n",
            "where m is the total number of genes targeted on an array and μi is the mean log ratio of expression levels for the ith gene. these tests produce p-values  for all genes m. at a certain significance level α, one can define all genes into two groups: one is called \"rejected null hypotheses\"  with p-values equal to or less than α and containing the significantly changed genes or positive genes we actually observed; and the other group is called \"accepted hypotheses\"  with p-values higher than α and encompasses the non-significant genes or negative genes we actually observed.\n",
            "\n",
            "if the p-value densities are plotted , typically the negative gene p-values  are under a flat curve indicating that the null tests follow a uniform distribution. the positive gene p-values  close to a p-value of zero is higher than the flat region of negative gene p-values .\n",
            "\n",
            "in these hypothesis tests, we are interested in the type i error rate in the resultant positive genes r. however, only m, r, and n can be seen among all other parameters  in these tests. the ideal way to find the errors  within r is by using experimental validation such as quantitative rt-pcr to determine the false positives. however, such an experimental approach is impractical for hundreds or thousands of r genes. indeed, some studies randomly choose a few degs for qrt-pcr and report the ratio of number of confirmed degs to number of examined degs as fdr. statistically, it can be assumed that the null distribution provides the desired control under the true or alternative distribution  <cit> . therefore, the known parameter n or m can be statistically used for v approximation.\n",
            "\n",
            "in the simultaneous multiplicity model, the proportion of unchanged genes  among all genes  is estimated in order to deduce the total null genes . thus, a natural estimate of fdr  when all null hypotheses are rejected with pi ≤ t is\n",
            "\n",
            "  fdr=m0tr \n",
            "\n",
            "the key is that simultaneous multiple tests use the total unchanged genes  for the error estimate. to date, much effort has been undertaken for the m <dig> determination. storey  <dig>  <cit>  applied a tuning parameter, λ ∈ [ <dig> ), for the m <dig> estimate ). other methods, including nonparametric and parametric methods, have also been proposed to estimate m <dig>  including the beta-uniform model  <cit> , the spacing loess histogram  <cit> , the lowest slope estimator  <cit> , the smoother  <cit> , the bootstrap least squares estimate  <cit> , the successive elimination procedure  <cit> , the moment generating function  <cit> , and the poisson regression approach  <cit> , among others. all of these fdr methods estimate null hypotheses  by applying various parameters on p-values of the total number of genes m.\n",
            "\n",
            "parallel multiplicity and error v estimate\n",
            "a new idea for multiplicity is proposed here that tens of thousands of genes can be tested in parallel in the microarray experiment. this \"parallel\" approach is different from the many \"simultaneous\" approaches that occur in the literature  <cit>  and that introduce multiple tests into microarray experiments. parallel multiplicity is composed of two complicities of the microarray: the normalization and the controls. data normalization places all genes to be objectively or observationally related. the unchanged genes as negative controls influence the reliability of error detection. for example, assuming that the p-value reflects the error probability of a gene that is claimed as a positive gene, if a gene has  <dig>  of error , the context of this error would be different  when there are many negatives as controls in parallel versus when there are no controls in parallel. we prove how these negatives that parallel the positives can be used in error estimation.\n",
            "\n",
            "lemma 1\n",
            "the error estimation  by using the number of null tests  that have p-value 1-t and above is close to the real error  at the same significance level of t.\n",
            "\n",
            "proof\n",
            "let φ be the error rate in a selected gene list r, and the real error v = r. φ. let v be the error estimated at pi ≤ t using m <dig>  then v = m <dig> t. let v be the error estimated at pi ≤ t using parallel negatives that have p-value 1- t and above, then v = n'.t.\n",
            "\n",
            "assume that m <dig> >>r . under this normalization assumption, we assume that n' ~ r and φ ~ t. then\n",
            "\n",
            " v−v=m <dig> t−r.φ>> <dig> \n",
            "\n",
            " v−v=n'.t−r.φ≈ <dig> \n",
            "\n",
            "for those null tests  that have any p-value interval bins less than 1-t, the number of x may be the same as n' since the null p-values are of uniform distribution  <cit> . for example, the number of p-values between  <dig> and 1-t is the same as the number of p-values between 1-t and 1-2t. however, the context of these null tests is different. the null tests having higher p-values have higher probability of being true nulls, and the largest p-values are most likely to come from the true null, uniformly distributed p-values  <cit> . for example, a gene with p-value of  <dig>  will have a 90% chance to be a true null test. the parallel idea uses the strongest contrastingly related  gene sets to control the type i errors even though other portion of null tests might also be used.\n",
            "\n",
            "error discovery rate \n",
            "in this parallel multiplicity idea, the parallel negatives  at a certain significance level  are used to determine the error v. therefore the error discovery rate  in a selected gene list  is defined as\n",
            "\n",
            "  edr=n'.tr \n",
            "\n",
            "it can be seen that the contrast of edr to fdr is that fdr uses total unchanged gene m <dig> but edr uses the parallel negative genes n' to approximate error rate. since the expression of most genes in microarray experiments is unchanged , the m <dig> is close to m. in cases where the m is large and r is small, the fdr in equation  would pose a problem, i.e. the small number of real degs could be eliminated because it results in a high fdr. that is the reason for the use of the gene filtering approach to decrease the total gene number m in order to increase the detection power. however, in the edr method, the positives  and negatives  are sampled at each significance level   from all genes  and the distribution of the p-values. it can be seen in equation  that edr would not pose a problem even when there is a large m and a small r, hence no gene filtering is needed for edr.\n",
            "\n",
            "the edr in equation  is not an estimate of the probability of a deg to be discovered in error. it is generally lower than the latter because it is computed using all the genes that have lower p-values than gene i . it is apparent that a gene whose p-value is near the threshold t does not have the same probability to be differentially expressed as a gene whose p-value is close to zero. the edr gives too optimistic a view of the probability for the gene to be an error. thus, an edr attached to each gene i that has p-value  was defined by removing the denominator r\n",
            "\n",
            "  edr'i=n'.pi \n",
            "\n",
            "this is similar to the local fdr proposed by aubert et al.  <cit>  that is derived from the q-value estimation\n",
            "\n",
            "  fdr¯=m <dig> \n",
            "\n",
            "the difference between equation  and  is that local edr replaces m <dig> and  of local fdr by using n' and pi, respectively. as noted above, n' is different from the m <dig> . compared to , pi retains more of the original test statistics of gene i.\n",
            "\n",
            "further, a reality error parameter σ is applied to the edr model. let σi=1xi, where xi is the ratio of the maximum group mean of gene i with the median value of genes of all groups. fi is the fold change between groups of gene expression i. for example, each group of genes gives an expression mean value, and a maximum mean value is given from these group means. xi produces a reliability factor of the gene expression values. the fold change is log transformed for both up-regulated and down-regulated fold changes. when fi is equal to  <dig> , there is no change in this gene expression between groups. clearly, a small xi with small fi would lift the error. applying these reliability factors to adjust the error rate is especially important in cases where only a few sample replicates are used because of the unstable variance that arises when sample size is small.\n",
            "\n",
            "the final error discovery rate of gene i, i = , is derived from the number of negative genes  that parallel to gene i at a p-value of pi:\n",
            "\n",
            "  edri=n'piσi \n",
            "\n",
            "the edr method can be implemented for two-group statistic tests  or multiple-group anova tests with gene expression matrix file, in which the fi is the fold change between the group with highest expression and the group with lowest expression of gene expression i. the edr method can also be implemented with only p-values available as in equation .\n",
            "\n",
            "case studies\n",
            "in order to explore the problems or behaviors of multiple testing methods in real experiments, we chose three real case data sets that typically represent three situations: one contains a small number of degs, one has a moderate number of degs, and one presents a large number of degs.\n",
            "\n",
            "hyperinsulinemic data\n",
            "this data set was used to compare the method powers in the real case that the proportion  of differentially expressed genes is extremely low. this hyperinsulinemic data set was reported in a study to examine the effects of insulin on gene expression in healthy humans  <cit> . the original studies only reported three degs . the edr method found five degs among the  <dig> genes that passed the raw p-value cutoff at the significance level of α =  <dig> ; while other fwer, fdr, sam, and bayes methods could not find any positive genes at the same significance level  including the previously reported gos <dig> gene that has a very low raw p-value  and a high fold change . the five degs detected by edr  include gos <dig>  but not txnip and bcl <dig> that were in the original report  <cit> . this may result from a different data preprocess as these two genes, txnip and bcl <dig>  have raw p-values of  <dig>  and  <dig>  from the two-tailed t-test, respectively. the evidence that the ddx <dig> gene is differentially expressed in diabetic mice  <cit>  suggests that ddx <dig> detected by edr in this study may be a true deg.\n",
            "\n",
            "the raw cel files of these three data sets  <cit>  were downloaded from the ncbi geo database  and were preprocessed by the gc-rma method. two groups in each data set were tested by two-tailed t test assuming equal variance. all multiple tests and raw p-values were applied at the same significance level of α =  <dig> .\n",
            "\n",
            "edr detection of the hyperinsulinemic data set. the edr of gene i is the expectation of raw-p of this gene  multiplied by the number of negative gene controls  at p-value equal to or greater than 1- pi, divided by the ratio of the maximum group mean of this gene with the median value of all genes  and by the fold changes  minus  <dig> \n",
            "\n",
            "mirna knockout data\n",
            "this data set  <cit>  was used to compare the methods in the real case that s <dig> is in the moderate range . specifically, this mirna knockout data set was reported from an examination of the effects of mirna on gene expression in mouse heart tissue using the mirna knockout model  <cit> . this mirna deficiency would induce quite a number of gene expression differences  <cit>  even though this study only focused on  <dig> protein-coding genes differentially expressed in mir-1-2-null hearts. in this data set, the edr method caught more positive genes than any other multiple tests except pcer. the pcer method identified more genes, but its gene number is even higher than the number found by the raw p-value cutoff .\n",
            "\n",
            "colorectal cancer data\n",
            "the data set examined  <cit>  was used to compare the methods in a real case with a large s <dig> . this data set of colorectal cancers was reported from a systematic search for genes differentially expressed in early-onset colorectal cancers using the genechip u133-plus  <dig>  that contains  <dig> probe sets on array  <cit> . in the original study,  <dig> probe sets were found to be differentially expressed between patients and healthy controls according to two-tailed t-test analyses with p <  <dig> . when the data set contains a large proportion of degs, edr still detected more degs than fwer and by but becomes slightly more stringent than other fdr methods such as sam and bayes . this appears sensible in real microarray experiments where, when there are so many positives, one strives for more stringent selection.\n",
            "\n",
            "in these three real case data tests, it was found that the fwer and fdr methods were unable to detect degs at a significance level of  <dig>  when only a small number of degs exist because of the high error rates that resulted. at the same significance level, the edr method not only detected degs when only a few degs existed, but also caught more degs than fdr and fwer methods when there existed a moderate number of degs. interestingly, edr becomes more stringent when there are a large number of degs. however, since the degs were not validated in these experiments, we cannot determine whether the increase in the number of degs is due to an increase in power or an increase number of false degs. hence, the specificity and sensitivity of these methods were further evaluated.\n",
            "\n",
            "specificity and sensitivity\n",
            "in order to evaluate specificity and sensitivity, one needs to know the \"true\" degs and \"false\" degs. as such, a published microarray expression data set  <cit>  complemented with cdna sequence digital expression confirmation was used to test the specificity and sensitivity of edr and other multiple test procedures. the same rna samples from mexican axolotl animals were examined by ambystoma genechip and  <dig> cdna sequencing. degs between  <dig> and  <dig> days post amputation of denervated  forelimb tissues were detected by microarray analysis at different significance levels. the resultant degs were compared to the true degs  that were found to be differentially expressed via cdna sequence digital expression analysis, the false degs  that were not found differentially expressed via direct cdna sequencing, and the true negatives  that were not differentially expressed in both platform detections.\n",
            "\n",
            "the true positive rate  and false positive rate  of edr and other methods at the significance of  <dig>  are shown in table  <dig>  it can be seen that the tpr and the fpr levels are a trade-off. for example, edr had better tpr  than bonferroni  and by , but higher fpr  than bonferroni  and by . however, when tpr and fpr were plotted on the receiver operator characteristic  curve, the edr curve was plotted above all other methods, and approached the left-top corner where the highest tpr and the lowest fpr exist . this indicates that edr had an overall better accuracy in performance over other methods. the area under the curve  of edr was the highest .\n",
            "\n",
            "the expression data set was downloaded from http://www.ambystoma.org and was preprocessed by the rma method  <cit> . differentially expressed genes  were detected at the significance level of  <dig>  by the edr method and the other methods from the multtest package  <cit> . the resultant degs were compared with the true degs  measured by digital expression. false positives  are those degs that are not found to be differential in digital expression analysis. true negatives  are those genes that are not differential in both platforms.\n",
            "\n",
            "it should be noted that in this microarray data set, the total number of gene probe sets on the array was small , and it has a moderate number of degs . this is an optimal case for simultaneous multiplicity model methods. even in this case, edr exhibited slightly improved performance over other methods. therefore it may be speculated that the superior performance of edr would likely be most evident in cases with large numbers of genes on the array and fewer degs.\n",
            "\n",
            "simulation study\n",
            "to test the above speculation that edr would have a better performance over other methods when a small number degs exist among a large number of genes on the array, existing mouse genechip data  <cit>  was simulated with a different proportion  of changed genes  and the powers of all multiple test methods were compared at different situations varying in deg number. the power is defined as the expected proportion of true degs that are declared as degs  <cit> . the simulated degs were assumed as the true degs for power estimation  <cit> . as expected, only when a higher proportion of genes are differentially expressed  can all fdr tests increase power . all fwer methods have low powers and their powers do not increase as s <dig> increases. even though the powers of sam and bayes increase when s <dig> increases, they still hold the lowest powers. however, at the same significance level of α =  <dig> , edr detected all simulated positive genes when s <dig> is  <dig> %. when s <dig> is less than or equal to  <dig> %, edr had more power than the commonly used fdr methods by and bh. when s <dig> became larger, edr caught slightly fewer degs but still retained consistently high power. this is particularly useful when there are so many 'degs' that a more stringent selection is needed.\n",
            "\n",
            "the power of edr is derived from the error estimate from n'. as shown in figure  <dig>  without the reality factor, the edr_n  of equation  had more power than the commonly used method hy and by when the deg number is small. after applying this reality factor, the edr  power of equation  was further improved.\n",
            "\n",
            "it can also be seen that at the significance level of  <dig> , the raw p-value, pcer, as well as the edr-i of the equation  ) detected all simulated degs. their powers reached the maximum of  <dig>  or more at different s <dig> points. obviously, examining only power cannot determine the performance of these methods because these methods may have included many false degs as well.\n",
            "\n",
            "since these simulation data sets contain a large number of non-degs , the roc curve would be skewed to the very low false positive rate  side. we calculated the precision-recall  curve instead. while the pr and the roc are equivalent, the pr curve is more informative when dealing with highly skewed data sets  <cit> . when there is a small r  and big m , the edr family exhibited better performance than bonferroni and by, but had a similar auc size to bh and rawp methods . because the error rates of all multiple test methods are derived from raw p-value and the lower raw p-value is equivalent to an fdr or edr, it is not surprising that the rawp has a large auc among all three data sets . however, a closer examination of the rawp curve in additional file 3a showed that several cutoffs of rawp give very low precision, indicating false degs were contained in the detected degs. the significance level of  <dig>  is commonly chosen for statistics and for power evaluation  <cit> . at this significance level, edr had a better precision  than bh and other methods . even though the edr-i and rawp caught more simulated true degs, i.e. a higher power as showed in figure  <dig>  their precisions were relatively low. when there were several hundred degs , edr exhibited better performance than bonferroni and by, but lower than bh . when the simulated true degs go up further , edr became more stringent than other methods except bonferroni.\n",
            "\n",
            "discussion\n",
            "there are many research papers and reviews on multiple testing for microarray experiments  <cit> . most of the literature discusses how to apply various multiple tests in microarray experiments, but the theory of multiplicity for the microarray has not been depicted. general statements are found where \"as a typical microarray experiment measures expression levels for thousands of genes simultaneously, large multiplicity problems are generated\"  <cit> . the parallel multiplicity concept proposed here stands in contrast to the traditional \"simultaneous\" multiplicity. parallel multiple tests use only the negative genes that parallel the positive genes to control the error rate while simultaneous multiple tests use the total unchanged gene number for error estimation even though both the negative gene number and the total unchanged gene number depend on the total gene number and the distribution of the p-values. the edr method based on parallel multiple tests exhibits improved performance over simultaneous multiple test methods in specificity and sensitivity. since parallel multiple tests use only the negative genes that parallel the positive genes, not the total unchanged genes, the edr method overcomes the common problem found in previous simultaneous multiple test procedures where the type i error rate detection power is low when the total gene number used is large and the deg number is small. it is a robust method for a variety of statistical p-value distributions  <cit> . edr retains the power for various data sets without requiring gene filtering as well.\n",
            "\n",
            "specificity and sensitivity are crucial criteria for method comparison, and the true and false positives need to be known for these tests. however, there is no gold standard to determine true or false positives. there are three levels for the definition of \"true\": true hybridization signal changes, true mrna transcript changes, and true functional changes. true hybridization signals can infer mrna transcript levels , but may not necessarily represent the true mrna transcripts, and the true mrna transcript changes may not cause the gene functional changes within an organism. here we focus only on the expression level, i.e. the mrna transcript level. to date, many microarray multiple test studies use simulation data or case data to evaluate the power  and specificity  <cit> . those microarray data sets with very few qrt-pcr validations are not useful for evaluating the specificities of multiple tests. in this study, a mexican axolotl animal microarray data set with cdna sequence digital expression validation  <cit>  was found to be useful. this data set possesses a relatively large number of \"true\" and \"false\" positives determined at mrna transcript levels. even though  <dig> sequencing also has a certain level of false positives, the comparison using this same data set for edr and comparison of other methods is compatible. in testing this data set, the edr method exhibited slightly improved performance over other methods in specificity and sensitivity. this improved performance is not as remarkable as in other cases, because this data set, with a low number of genes on array and a moderate number of degs, is an optimal case for simultaneous multiple test methods. the consistent superior performance of the use of edr was further supported by testing the simulation data and three real case data sets that vary in the proportion of unchanged genes.\n",
            "\n",
            "many reports use real microarray case data sets for multiple tests evaluations. for example, the acute lymphoblastic leukemia  data set was intensively used for such a purpose  <cit>  even though there was no validation for the real degs in this data set. other real case microarray data sets were also used for multiple test performance evaluation, including a breast cancer data set  <cit> , a colon cancer expression data set  <cit> , a wheat data set  <cit> , a diabetes data set  <cit> , and a smoking data set  <cit> . not all of these data sets were validated for real degs. three real case data sets were chosen for this study because they represent three typical cases: low, moderate, and high levels of degs. it is difficult to evaluate the individual method by unvalidated data, but these data sets are still helpful in comparing the behaviors or performance of different methods under the same circumstance.\n",
            "\n",
            "the edr model is different from the per-family error rate  and the per-comparison error rate  that simply adds individual p-values together or averages p-values  <cit> . it is distinct from fwer and fdr in that edr utilizes cross-gene information in the parallel concept but is not constrained by the total gene number. it varies from permutation-based fdr in that edr directly reflects the errors of the given genes and edr operates for small sample sizes. also, edr is appealing because it attaches the error to each gene, and the genes do not need to be ranked by raw p-values as is the case for all other multiple tests. other multiple test comparisons sequentially compute the error rate based on the ranked raw p-values; thus the error rates are the same as the lower p-values. this is not true in microarray multiplicity because at times lower p-values possess higher error potential than some higher p-values. edr recognizes this reality . there are indeed reports for how to rank and select genes  <cit> , but they do not address the error rate issue.\n",
            "\n",
            "a false positive or the rejection of a true null hypothesis is a gene that is called as differentially expressed when it is not. the falsehood or error can be a systematic error or random error. the systematic error can be corrected by background subtraction or dye normalization, while the random error cannot be captured but can be modeled by statistical analysis. the p-value of each gene is a measure of how much evidence we have against the null hypothesis. it is rational that all multiple tests use this raw p-value or marginal p-value as a base error rate of that gene  <cit> . even though the p-value has inherently accounted for the fold change, an extremely small standard deviation  could drive a small p-value with a small fold change. almost all microarray studies apply a fold change cutoff for the final gene list to be reported. the reason that we only keep the genes whose fold changes are higher than a defined value, for example, a 2-fold change, is that it is still not certain that those low fold-changes in hybridization signals are real positives at the level of the mrna transcript. after these uncertain genes are removed, the error rate in the final gene list would undoubtedly decrease. also, some genes have very low intensity signals. for example, among the gene expression values ranging from  <dig>  to  <dig> , a 10-fold increase deduced from  <dig>  divided by  <dig>  is reasonable from a statistical point of view. however, biologists often remove this gene even though it has a very small p-value because this low expression level is close to background expression levels and therefore may be not reliable. thus, an ideal multiple test procedure would incorporate into the raw p-values the unreliability factors that result from low intensity and low fold-change in the error calculation in order to reflect the error or false reality. in the edr method, xi uses the maximum group mean and the median expression value to control the gene expression values, and uses fi to control the fold changes.\n",
            "\n",
            "different multiple test procedures were recommended for different experiments based on the sample size, the magnitude expectations, and the p-value distribution  <cit> . however, there may exist a large variation in judging these multiple parameters. therefore the same type of experiments may produce different results depending on the method of preference among different researchers. there is not a comparable standard multiple test method that can be applied to a variety of data sets. it is interesting to note that the edr method had a similar performance pattern in all three different case data sets . this pattern was not affected by the proportion  of degs of the data. also, this performance pattern presents a deep slope in all three data types. this deep slope forms a boundary useful for significance cutoff value selection. other methods could not form this boundary in all data types. the performance patterns of other methods change vigorously depending on the s <dig> . for example, the bh performance curve did not show up in the low-s <dig> data set. it has lower power than pfcr in the median-s <dig> data set but was capable of much higher power than pfcr in the high-s0data set. this suggests that the edr method exhibits consistent performance in various data sets. this consistency is further supported by the use of simulation data. the edr method works for different data sets and would therefore provide a standard method for type i error rate calculations in microarray experiments. for example, we can say that with an edr cutoff of  <dig>  we found  <dig> differentially expressed genes between disease and normal samples. however, if we use other fdr methods, this may not be the case because some studies may find more genes using the same fdr cutoff in the same data set but by filtering genes or filtering in different ways, or they may use different fdr methods for data sets that vary in the proportion of unchanged genes.\n",
            "\n",
            "it is difficult to test if the edr method works for dependent tests in microarray experiments. the multiple-endpoint in clinical trials  <cit>  is an example for fdr to control dependence in multiple tests. in this example, we know all tests are dependent. this knowledge is not deduced from p-values but instead by the experiment itself. in a microarray experiment, all p-values do not relate to one single subject, i.e. they are independent. only the genes in subgroups may be dependent because they usually work in a collaborative fashion to fulfill certain cell functions or pathways. the fdr methods claim to be able to control the error rate of the dependent p-values, and simulated dependent data was used to test this claim  <cit> . clumpy dependence was simulated in the sense that blocks of genes have dependent expression and therefore dependent p-values in p-value bins. however, this simulated dependence is quite different from the real microarray experiment. in a microarray experiment, the \"correlated\" p-values may not necessarily reveal true biological dependence, and true biological dependence may not exhibit dependence on p-value levels. for example, minor differential expression of some genes may play a remarkable regulatory function in a particular pathway, and the p-values of these genes may not be correlated to those of other genes in this same pathway. further studies are needed to examine genes in the same pathways and then to adjust the error rate based on the same pathway groups.\n",
            "\n",
            "CONCLUSIONS\n",
            "microarrays are extensively used today to examine changes in gene expression. however, biologists have difficulty in both the understanding and use of multiple tests in microarray studies. a new system of parallel multiple tests is proposed here. parallel multiple tests use the negative genes that parallel the positive genes to control the type i error discovery rate  in the microarray experiment. the edr method exhibits consistently improved specificity and sensitivity  over other methods in testing diverse data sets that vary in the number of null hypotheses. this method provides an alternative to standard type i error rate methods. parallel multiplicity is a new proposition and worth further enhancement in statistics and algorithm development.\n",
            "\n",
            "materials and methods\n",
            "hyperinsulinemic data and preprocessing\n",
            "this data set was reported in a study to examine the effects of insulin on gene expression in human health  <cit> . six nondiabetic volunteers were given a 3-h hyperinsulinemic  euglycemic clamp test. a variable infusion of glucose  was used to maintain euglycemia during insulin infusion. the gene expression profiles of skeletal muscle biopsies from these six subjects pre- versus post-clamp were compared using the affymetrix genechip hu <dig> containing  <dig> probe sets. only three genes were reported to be regulated by insulin in human muscle cell using a wilcoxon signed rank test after filtering removed  <dig> probe sets.\n",
            "\n",
            "the raw cel files were downloaded from the ncbi geo database  containing data that are miame compliant as detailed on the mged society website http://www.mged.org/workgroups/miame/miame.html the gc-rma algorithm was used for probe level signal condensation, background subtraction, and normalization. the gc-rma values were log-transformed for robust use in statistics tests since the log values are more compliant with the data normal distribution assumption than the raw data. but the fold and ratio calculations used the raw expression values. the raw expression values were truncated into  <dig> % and  <dig> % percentile ranges in order to avoid the extreme large and extreme small values in the fold and ratio calculations. with this data set, the edr method was compared with  <dig> other multiple test procedures  at the same significance level of α =  <dig>  .\n",
            "\n",
            "mirna knockout data and preprocessing\n",
            "this data set was reported to examine the effects of mirna on gene expression in mouse heart tissue  <cit> . heart tissue samples from three wild type and three mir-1- <dig> knockout mice at postnatal days  <dig> were compared for gene expression levels using affymetrix mouse genome  <dig>  <dig>  array that contains  <dig> probe sets. the raw cel files were downloaded from the ncbi geo database  and were preprocessed by gc-rma algorithm. with this data set, the edr method was compared with  <dig> other multiple test procedures  at the same significance level of α =  <dig>  .\n",
            "\n",
            "colorectal cancers data and preprocessing\n",
            "this data set was reported to systematically search for genes differentially expressed in early-onset colorectal cancers using the genechip u133-plus  <dig>  array  <cit> . twelve tumor specimens and ten adjacent grossly normal-appearing tissues from at least  <dig> cm away were collected for rna extraction. the raw cel files were downloaded from the ncbi geo database  and were preprocessed by gc-rma algorithm. with this data set, the edr method was compared with the other  <dig> multiple test procedures  at the same significance level of α =  <dig>  .\n",
            "\n",
            "transcriptional validated data set\n",
            "the data set used was reported in a study of transcription during nerve-dependent limb regeneration  <cit> . the same rna samples were detected by ambystoma genechip and  <dig> cdna sequencing. there are total  <dig> probe sets  on this genechip array.\n",
            "\n",
            "the raw cel files were downloaded from the public ambystoma microarray database  <cit> . detailed information of these data files and the degs confirmation by  <dig> cdna sequencing were described in the original study  <cit> . the rma algorithm  <cit>  was used for probe level signal condensation, background subtraction, and normalization. the rma values were log transformed for robust use in statistics tests since the log values are more compliant with the data normal distribution assumption than the raw data. but the fold and ratio calculations used the raw expression values. raw expression values were truncated into  <dig> % and  <dig> % percentile ranges in order to avoid the extreme large and extreme small values in the fold and ratio calculations. the two-tailed t test was used to calculate the raw p-values.\n",
            "\n",
            "of the degs detected among five group comparisons by microarray analysis,  <dig> degs were confirmed in mrna levels by using  <dig> cdna sequencing. in this evaluation, only two groups of raw data files, zero and five days post amputation of denervated  forelimb tissues, were used. among the total  <dig> true degs,  <dig> genes were true degs between dl <dig> and dl <dig> and possessed  <dig> -fold or greater changes in normalized digital expression levels. at each significance level, the true positive degs  were the portion of these  <dig> genes discovered by microarray analysis. the false positive degs  were those total degs detected by microarray analysis  subtracted by tp. the false negative degs  were calculated by subtracting tp from  <dig>  the true negative degs  were the tgs subtracted by tdegs. since  <dig> cdna sequencing presumably covered all cdnas or genes including all probe sets on the genechip, the portion of genes that were not detected by both platforms were tn.\n",
            "\n",
            "simulation data\n",
            "in order to calculate the statistical power of method, we simulated microarray data with different proportions  of true positive gene numbers. the power was calculated as the proportion of true positives that were detected at the same significance level of α =  <dig> .\n",
            "\n",
            "the simulation parameters for gene group means  and gene group variances  were replicated on the real mirna knockout data set  <cit>  and preprocessed by gc-rma  <cit> . the simulated data matrix was created by  <dig> groups of  <dig> sample columns  and  <dig> rows of genes  .\n",
            "\n",
            "each gene expression value is generated within the normal distribution of mean mi and standard deviation sdi. mi is the gene group mean that is uniformly distributed within the minimum and maximum values of the whole mirna knockout data set. sdi is the standard deviation of a gene group that is uniformly distributed within the minimum sd and maximum sd of all gene group sds of the whole real data set. the non-degs were guaranteed to have the normal distribution of the sdi and equal mi. also. in addition, the two group means are close to equal because if they fall into the normal distribution with an equal mean, they still may have large fold changes. this ensures they are non-degs in a statistical sense. the different s <dig>  data sets were simulated by adding different proportions of degs. the normal distribution of gene expressions of degs in each group samples was simulated using mean and sds vectors. the mean vectors have folds uniformly distributed between  <dig> - to 3-fold changes and the sds vector has uniformly distribution of one-fifth of the means . for each simulation data set, two-group t-tests assuming equal variance were performed, and edr and other multiple test methods were applied.\n",
            "\n",
            "receiver operating characteristic  and precision-recall  curves\n",
            "for the transcriptional validated data set, the roc curve of edr method was compared with other multiple test procedures . the aucs of roc curves based on tpr and fpr were used to compare the overall performances of edr and other methods. the curve approaching the left-top corner represents the highest tpr and lowest fpr.\n",
            "\n",
            "for the false positive rate  skewed simulation data sets, pr curve was used instead for method performance comparison.\n",
            "\n",
            " tpr, recall=tptp+fn \n",
            "\n",
            " fpr=fpfp+tn \n",
            "\n",
            " precision=tptp+fp \n",
            "\n",
            "multiple tests\n",
            "the multiple tests for type i error rates were categorized into several groups  <cit> : pcer, pfer, fwer, and fdr. the following tests are defined according to figure 1d and used in this study.\n",
            "\n",
            "pcer  <cit> \n",
            "the per-comparison error rate is defined as the expected value of the number of type i errors divided by the number of hypotheses, that is,\n",
            "\n",
            " pcer=e/m=a1+...+amm \n",
            "\n",
            "pfer  <cit> \n",
            "the per-family error rate is defined as the expected total number of type i errors, that is, pfer = e = a <dig> + ... + am\n",
            "\n",
            "fwer\n",
            "the family-wise error rate is defined as the probability of at least one type i error , that is, fwer = pr. the bioconductor multtest package  <cit>  was used for following pwer procedures: bonferroni, holm, hochberg, and sidaksd.\n",
            "\n",
            "bonferroni  <cit> , pi=αim\n",
            "\n",
            "holm  <cit> , pi = pi=αim−i+1\n",
            "\n",
            "hochberg  <cit> , pi = pi=αim−i+1\n",
            "\n",
            "sidaksd  <cit> , pi=1−1−αm−i+1\n",
            "\n",
            "fdr\n",
            "bioconductor multtest, qvalue, and sam packages  <cit>  were used for the following fdr detection : bh, by, qvalue, sam, and empirical bayes.\n",
            "\n",
            "bh  <cit> , pƒi=pimi\n",
            "\n",
            "by  <cit> , pƒi=pim∑1/ii\n",
            "\n",
            "qvalue  <cit> , pƒi=pimπ^0i, where π^ <dig> is the proportion estimation of unchanged genes.\n",
            "\n",
            "sam  <cit> , pƒi=∑bbnpb>pib, where b is the number of permutations.\n",
            "\n",
            "empirical bayes  <cit> , p=απ0απ0+, where p is the fdr, α is the significance level, π <dig> is proportion of null hypothesis, and  is the test power.\n",
            "\n",
            "authors' contributions\n",
            "wx proposed the idea, performed the tests, and wrote the manuscript. cc contributed to the revision.\n",
            "\n",
            "supplementary material\n",
            "additional file 1\n",
            "r source code for edr method. the edr method was implemented for two group comparison experiments either with p-values provided or without p-values.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 2\n",
            "simulation and parameters. existing mouse genechip data  <cit>  was simulated with a different proportion  of differentially expressed genes.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional file 3\n",
            "precision-recall  curves. precision-recall  curves of multiple test methods on simulation data sets with different proportions of degs . upper panel: pr curves; middle panel: the area under the curve ; lower panel: simulated true degs  contained within all detected degs of each method were at or below the significance level of  <dig> .\n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements\n",
            "the authors thank nathan springer for helpful comments and rui kuang for valuable discussion on this manuscript. the authors also thank yung-tsi bolon and tracey bartlett for reading and editing this manuscript. the authors would like to thank hackstadt amber j. for kindly providing a portion of simulation source code. the authors would also like to thank the anonymous reviewers for their constructive comments.\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the edr method exhibits consistently improved specificity and sensitivity  over other methods in testing diverse data sets that vary in the number of null hypotheses.\n",
            "the edr method exhibits consistently improved specificity and sensitivity  in testing diverse data sets that vary in the number of null hypotheses.\n",
            "the edr method exhibits consistently improved specificity and sensitivity  in testing diverse data sets that vary in the number of null hypotheses.\n",
            "the edr method exhibits consistently improved sensitivity  over other methods in testing diverse data sets that vary in the number of null hypotheses.\n",
            "the edr method exhibits consistently improved specificity and sensitivity  over other methods in testing diverse data sets that vary in the number of null hypotheses.\n",
            "\n",
            "93\n",
            "[{'summary_text': 'a total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed.\\na total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed.\\na total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed.\\na total of  <dig> million raw reads were obtained from  <dig> rumen samples of jaffrabadi breed.\\na total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed.\\na total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed.\\na total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 7285 to 8192 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "‘superorganism’ is the term coined to portray the association of symbiotic microbiota with its higher animal  <cit> . the microbes inhabiting the gut/ rumen are known to impose protective effects and nutritional benefits to the host  <cit>  and due to its superior metabolic potentials compared to the host they are rightly considered equivalent to an organ . in humans, microbes are found to comprise almost  <dig>  million non-redundant genes, i.e. approximately  <dig> times higher than the number of genes in the human genome  <cit> . in ruminants, the significance of microbiome for maintaining host homeostasis was observed long before it was noticed in humans. ruminant hosts are benefitted from microbial proteins and vitamins like metabolite produced by them. ruminants obtain their energy due to the utilization of complex plant polysaccharides by microorganism, which are else indigestible by host. domestication of ruminants has provided stable food supply and means of livelihood to thousands of people worldwide since earlier times. ruminant meat and milk is considered as one of the most important agricultural products . because of concomitant increase in food consumption, the demand for meat and milk is constantly increasing, further enhancing the need for sustainable number of ruminants . sustainable production of meat and milk requires an in-depth knowledge about the functions of the microbes and their interactions with the host. nowadays, the high-throughput next generation sequencing technologies have alleviated studies such as metagenomics, metatranscriptomics, metaproteomics and metabolomics to characterize complex rumen microbial ecosystem and apply the information for improving forage digestion efficiency.\n",
            "\n",
            "on the other hand, the feed composition also profoundly influences the microbial population  <cit> . ruminants possess a very complex microbial ecosystem comprising of bacteria, protozoa, fungi, methanogens and bacteriophages that well adapt to variable diets  <cit> . their abundance in the rumen and the metabolic activity is reported to be remarkably high compared to terrestrial and aquatic ecosystems . however, very few  of the rumen bacteria are known to be cultured  <cit> , highlighting the importance of molecular biology approaches like metagenomics to sidestep this limitation.\n",
            "\n",
            "ruminants are a crucial intermediate between light energy and the production of edible compounds such as milk and meat. among them, buffalo plays a highly vital part of the indian economy solely contributing for >55 % of total milk production by indian livestock. especially, the jaffrabadi breed of buffaloes, which are mainly a native of saurashtra region including the junagadh, bhavnagar, jamnagar, porbandar, amreli and rajkot districts of gujarat are good milkers and one of the heaviest indian breed. they efficiently convert roughage into milk and are found to be different compared to other breeds in the terms of its milk fat content  <cit> . earlier reports have described use of rumen microbiota modulation for increasing the feed digestion efficiency  <cit> . ruminants can degrade dietary fibers to short chain fatty acids with help of the inhabiting microbial consortium. in the process, atp is generated that stimulates microbial growth, specifically the synthesis of cellular protein  <cit> . this can be an important attribution for improving the energy supply of rumen. maximizing efficiency of its production would consequently improve cattle productivity. nevertheless, knowledge of the functioning of rumen microbiome for example mechanism of plant polysaccharide degradation is still incomplete  <cit> . the evolution of the study of rumen microbial diversity similar to other microbial ecosystems has moved from culture-based microscopic studies to the use of culture-independent molecular techniques. second-generation sequencing techniques have revolutionized the microbiome study by producing huge amount of data leading to increased coverage depth further permitting identification of even the less abundant community members  <cit> . phylogenetically near microbes are reported to have varied metabolic potentials thus revealing the significance of characterizing complex microbial communities  <cit> . rumen microbiome studies of indian cattle and buffalo is gaining much interest, however the jaffrabadi breed rumen microbiome has not been well characterized. the objective of this study was to characterize and compare the rumen microbiome at taxonomic and metabolic level as well as to establish the database of the same for efficient feed converter and sturdy indian breed of buffalo, jaffrabadi  reared under varied dietary roughage proportions using metagenomic approach to better comprehend its correlation with nutrient utilization and maintenance of host homeostasis.\n",
            "\n",
            "methods\n",
            "ethics statement\n",
            "the experiment was performed under the approval of the institutional animal ethics committee of college of veterinary science & animal husbandry, anand agricultural university vide letter no. aau/gvc/cpcsea-iaec/108/ <dig> dated 05/10/ <dig> \n",
            "\n",
            "dietary feed treatment\n",
            "in the present study, eight healthy adult , non-pregnant, dry buffaloes of jaffrabadi breed which had completed 1st lactation and had an average body weight of 350–400 kg were included. all animals under study were reared at the livestock research station, anand agricultural university, gujarat, india. before the experimental feed treatment, the animals were maintained under the feeding standards of the national royal commission . out of eight animals under study, four were supplemented with green  fodder as roughage content and another four with dry  fodder as roughage content in total feed . the nutrient composition of the feed were described in additional file 1: table s <dig>  considering the facts that if started with dietary treatment of 100 % roughage , the microbial diversity lost due to low protein supplement may take a long time to recover when shifted to high protein diet, whereas if started with a 50 % of roughage , the available microbial diversity will start to adapt the less protein diet. thus, in this study feeding regime included an increasing proportion of roughage  content at regular feed intervals. a total of three dietary treatments in the current study comprised a mixture of roughage and concentrate  feed at ratios of 50: <dig> ,75: <dig>  and 100: <dig> . each dietary treatment was continued for six weeks to permit microbial adhesion and adaptation to the same, the time duration also ensures less probability of the previous microbial diversity carry over effects. after six weeks, rumen sample was collected from animals using stomach tube and vacuum pump 3 h post feeding. the rest of the rumen fluid was then passed through muslin cloth, without pressure for separation of the liquid  and solid  fractions without any wash. the filtrate and the solid fraction were collected in sterile cryo vials and were stored at −80 °c until metagenomic dna extraction. simultaneously, prior to filtration, a small proportion of the rumen fluid  was aliquoted in a bottle for total volatile fatty acid  and ammonia estimation. tvfa was measured using the markham's distillation apparatus, wherein the volatilized fatty acids in presence of oxalate was collected through condenser and calculated, and the nitrogen contents were measured by the kjeldahl's method.\n",
            "\n",
            "metagenomic dna extraction\n",
            "metagenomic dna was isolated from liquid and solid rumen samples of all the three treatments  using qiamp dna stool mini kit  with slight modification for the solid samples that included pretreatment of pbs buffer along with vortexing for an hour to release the bacteria adhered to the dietary fibers and minimize the plant dna fraction during the extraction procedure. dna quality and quantity was determined using a nanodrop nd- <dig> spectrophotometer  and on  <dig>  % agarose gel electrophoresis.\n",
            "\n",
            "shotgun sequencing of rumen microbiome\n",
            "forty eight dna shotgun bar-coded libraries from a start-up dna concentration of ≥300 ng of each sample were prepared as per standard 400 bp ion torrent pgm protocol . four barcoded libraries of biological replicates from all the treatments were pooled on an equimolar basis. emulsion pcr was carried out using the ion onetouch™  <dig> template kit followed by enrichment of the ion sphere particles  as per the standard protocol. the enriched isps were loaded on the  <dig> chip, and sequencing run was performed. twelve sequencing runs  were carried out on ion torrent pgm.\n",
            "\n",
            "taxonomic and functional analysis\n",
            "all metagenomic reads were uploaded on mg-rast  for annotation purpose   <cit> . host-specific reads were curtailed by screening the reads against the closest relative bos taurus umd v <dig> , as published complete sequences for buffalo were unavailable in the database. phylogenetic classification was performed against m5nr database with minimum e-value of 1e- <dig> and minimum identity of 60 %. each value obtained indicated the percentage/percent relative frequency of reads with predicted proteins and ribosomal rna genes annotated to the indicated taxonomic level. functional annotation was performed using the seed database with a minimum e-value of 1e- <dig> and minimum identity of 80 % at level  <dig> and level  <dig>  for all subsequent analysis, mean proportion of four animals maintained under same diet treatment were considered, unless mentioned otherwise.\n",
            "\n",
            "cazymes mining from rumen metagenome\n",
            "the raw reads from four replicates were pooled and assembled into contigs using gs de novo assembler v <dig>  with default parameters. the assembled contigs were uploaded in cazymes annotation toolkit  at http://mothra.ornl.gov/cgi-bin/cat/cat.cgi for sequence based annotation with e-value of 1e- <dig>  <cit> . the results were further studied for recognizing the organisms contributing to coding sequences  for cazymes.\n",
            "\n",
            "statistical analysis\n",
            "two-way fisher’s exact test with a benjamin-hochberg fdr multiple test correction was performed for statistical analysis between all the rumen microbiome samples at the taxonomic and functional level using the statistical analysis of metagenomic profiles, stamp v <dig>  software package for identification of biologically meaningful statistical differences between samples at different variable including treatment, fractions and diet  <cit> . metabolism and cazyme abundance were analyzed for statistical significance based on correlation test and anosim  using past  software  <cit> .\n",
            "\n",
            "nucleotide sequence accession numbers of rumen microbiome samples\n",
            "all the sequences were submitted to mg-rast server for annotation .table  <dig> sequence details of  <dig> metagenomic samples\n",
            "\n",
            "\n",
            "\n",
            "RESULTS\n",
            "metagenome sequencing of  <dig> samples using ion torrent resulted into a total of  <dig> million raw reads and  <dig> million reads post quality filtering. the sequencing data of individual samples obtained after quality filtering are detailed in table  <dig>  based on the mgrast annotation, about  <dig>  to  <dig>  % of the reads were classified as rrna per rumen microbiome sample based on the hits against 16s rrna gene sequence databases while  <dig> to 90 % of the reads from all the samples were classified into various functional subsystems.\n",
            "\n",
            "taxonomic distribution in the rumen microbiome samples\n",
            "at the domain level, bacteria were the most abundant with an abundance of >90 % in all the samples, followed by the eukaryotes . archaea were present in a proportion of ~1 %, while the rest included unassigned sequences and the virus sequences.\n",
            "\n",
            "effect of diet on microbial diversity\n",
            "the taxonomic assignment at phylum level using the m5nr database revealed dominance of the bacteroidetes phyla in the rumen microbial community of liquid fraction of green roughage  as well as the dry roughage  diet fed animals . firmicutes, fibrobacter, proteobacteria and cyanobacteria were the other major phyla observed constituting about  <dig> – 85 % of the total phyla across all the three, i.e. j <dig>  j <dig> and j <dig> treatments. rest  <dig> to 20 % comprised of majorly unassigned sequences followed by other phyla including actinobacteria, chlamydiae, spirochaetes, tenericutes and verrucomicrobia. bacteroidetes were observed to be lower in the solid samples as compared to that in liquid samples, whereas the fibrobacter and firmicutes proportion was relatively higher. as the roughage proportion was increased from 50 % to 100 % there was decrease in the bacteroidetes abundance across all the samples.fig.  <dig> \n",
            "a taxonomic distribution of bacterial phyla based on predicted proteins and rrna in the samples  b statistical analysis using stamp based on genus level taxonomic assignments between treatments c statistical analysis using stamp based on genus level taxonomic assignments between liquid and solid fraction \n",
            "\n",
            "\n",
            "\n",
            "similarly, at the genus level the two predominant genus of bacteroidetes family, viz. prevotella and bacteroides were found decreasing with the increase in roughage proportion . both were significantly  higher in the liquid as compared to solid portion . out of the fourteen most abundant genera comprising about 70 % of total genera, four were significantly varying across treatments and thirteen were varying between the solid and liquid fractions. paludibacter and victivallis were found increasing with the increase in roughage content. allistepes was highest in j <dig>  followed by j <dig> and then j <dig>  further at species level, prevotella ruminicola was the predominant species in all the liquid samples throughout different treatments followed by the two important fibrolytic species of rumen ecosystem viz., fibrobacter succinogenes and ruminicoccus albus. in the solid fraction, fibrobacter succinogenes and ruminicoccus albus were predominant followed by prevotella ruminicola.\n",
            "\n",
            "roughage vs. firmicutes/bacteroidetes ratio\n",
            "bacteroidetes and firmicutes were the most prevalent phyla throughout the treatments. for the firmicutes/bacteroidetes ratio, significant differences between liquid  and solid ; as well as between the three treatments were observed for both green and dry roughage fed animals . notably specific trend was observed for green and dry diets, whereby in the green roughage fed animals the ratio was found to be higher in j <dig>  reduced in j <dig> and again increased in the j <dig> treatment, whereas in dry roughage fed animals the ratio was found increasing with increase in roughage proportion .fig.  <dig> \n",
            "firmicutes/bacteroidetes ratio during the three treatments\n",
            "\n",
            "\n",
            "\n",
            "diet and correlation between various phyla\n",
            "we checked for the cross-phylum interactions in the buffalo rumen among green and dry diet feeds. phylum abundance profiles were determined across  <dig> rumen samples. linear correlation test revealed negative correlation was observed between the bacteroidetes and fibrobacters phyla. during green roughage treatment results showed that the abundance of bacteroidetes positively correlated with abundance of bacterial phylum verrucomicrobiae, proteobacteria and tennericutes for both the treatments while negatively correlated with fibrobacters, firmicutes and spirochaetes during the dry roughage treatment .table  <dig> correlation between various phyla using paleontologicalstatistical analysis among green roughage fed animals\n",
            "\n",
            "note: *indicates p <  <dig> , **indicate p <  <dig> , ***indicate p <  <dig> \n",
            "\n",
            "\n",
            "\n",
            "functional diversity based on dietary changes\n",
            "functional analysis at level  <dig> using the seed database in mg-rast revealed that the highest coding sequences were present for the protein metabolism followed by carbohydrate metabolism category. at level  <dig>  about ten categories including secondary metabolite; nitrogen metabolism; dna metabolism; protein metabolism; carbohydrates; amino acids and derivatives; sulfur metabolism; cofactors, motility and chemotaxis; and miscellaneous were observed to be significantly different between the liquid and solid fractions . among these, the abundance of all categories was more in the solid fraction except the carbohydrate and protein metabolism which was higher in the liquid portion, which clearly indicated the difference in the microbial diversity of both the fractions. three of the categories viz. carbohydrates; amino acids and its derivatives; and miscellaneous differed significantly across the j <dig>  j <dig> and j <dig> dietary treatments, wherein the coding sequences related to the carbohydrate subsystem increased with the increase in roughage concentration .fig.  <dig> \n",
            "a statistical analysis using stamp based on functional assignments at subsystem level  <dig> between treatments  b statistical analysis using stamp based on functional assignments at subsystem level  <dig> between treatments  between liquid and solid fraction  c distribution of protein metabolism subsystem at level  <dig> between treatments, diet and fractions  d distribution of carbohydrate metabolism subsystem at level  <dig> between treatments, diet and fractions metabolism \n",
            "\n",
            "\n",
            "\n",
            "further based on the highest abundance of protein and carbohydrate metabolism, we studied the level  <dig> classification for these two subsystems. protein metabolism included five different categories including protein biosynthesis, protein degradation, protein folding, protein modification and processing and selenoproteins . the abundance of all the categories except selenoproteins varied across the three treatments; green and dry diets as well as the liquid and solid fractions. we observed that the maximum sequences were attributed to the protein biosynthesis, followed by protein modification and processing. with increase in the roughage proportion, protein biosynthesis and degradation were found to be reducing while protein modification, processing and refolding were increasing. during the dry diet, protein biosynthesis was more abundant, while the degradation and folding were more expressed in green diet. all the categories except protein processing and modification were more expressed in the solid fraction as compared to the liquid fraction.\n",
            "\n",
            "similarly, the carbohydrate metabolism included ten sub categories, among which the central carbohydrate metabolism, amino sugars and polysaccharides metabolism did not vary across the samples . one carbon metabolism, organic acids and sugar alcohol were least in the j <dig> treatment compared to j <dig> and j <dig>  while monosaccharide and oligosaccharides metabolism, fermentation and co <dig> fixation was highest in the j <dig> treatment. all the categories were more abundant in the dry treatment compared to the green treatment. among the two fractions, solid fraction gave more hits to the functional category as compared to the liquid fractions.\n",
            "\n",
            "further, to understand the similarity/variation among the microbiome composition of all the samples, principal components analysis  was performed for between groups correlation based on taxonomic and functional annotation. principal component analysis  of showed that the samples fed on same treatment  clustered together . samples from each treatment of green and dry roughage fed animals tended to cluster together, signifying that related functions were encoded in their metagenome during respective treatments.fig.  <dig> principal component analysis based on taxonomic  and functional distribution in each sample using past\n",
            "\n",
            "\n",
            "\n",
            "diet impact on rumen metabolism\n",
            "in our study, to determine the effect of change in feed on rumen metabolism, we performed whole rumen fluid analysis. the total-n and ammonia-n concentrations in the rumen fluid were found decreasing significantly  in the dry roughage fed animals with increase in the roughage proportion viz. from j1d to j3d. also, the concentrations of total–n and ammonia-n were relatively higher in the green roughage fed individuals as compared to the dry roughage fed individuals. soluble n was significantly higher in j <dig> as compared to j <dig>  the concentration of non protein nitrogen  was found to be significantly higher in green roughage fed individuals as compared to dry roughage fed. the concentration of npn also significantly decreased with increase in the roughage proportion  for both green as well as dry roughage fed individuals. the volatile fatty acid  concentrations did not show significant variation, though they were observed to be relatively higher in j <dig> and j <dig> compared to j <dig>  but the difference was statistically non-significant .table  <dig> analysis of variance showing the effect of diet on various metabolites\n",
            "\n",
            "dietary\n",
            "nitrogen content\n",
            "total vfa\n",
            "mean values in the same row with different superscripts  differed significantly \n",
            "\n",
            "\n",
            "\n",
            "dietary effect on volatile fatty acid  production\n",
            "environmental gene tags  involved in production of volatile fatty acids viz. propanoate and butanoate were studied to check the effect of different diets on the same as . the results showed that the egts for propanoate were abundant in the first treatment and nearly likewise represented during the second and third treatment. results also revealed that the propanoate production via 2-hydroxy butyrate was better represented as compared to the succinate pathway.\n",
            "\n",
            "similarly, egts corresponding to the butanoate production pathway were also determined throughout the dietary treatments. the biosynthesis pathway for butanoate production involves the final step for butanoate formation from butanoyl coa via two possible pathways . the results showed that the egts for butanoate were abundant in the j <dig> treatment subsequently followed by j <dig> and j <dig> treatments, except the egts for phosphate butyryl transferase and butyrate kinase enzymes responsible for the final butanoate production which were comparatively more abundant in the j <dig> than the j <dig> treatment.\n",
            "\n",
            "dietary effect on methanogenesis pathway\n",
            "we also considered the effect of three different dietary treatments on the proportion of egts involved in methane production . the entire anaerobic process of methane formation takes place by the methanogenic archaea population in the rumen. egts for enzymes involved in methanogenesis were abundant in j <dig> followed by j <dig> and j <dig> treatment. thus, increase of roughage proportion in the diet reduces the active genes for production of methane.\n",
            "\n",
            "diet based variation in cazymes distribution\n",
            "results of the cazyme analysis tool  revealed that the most encoded cazyme domain included the glycoside hydrolase  family , followed by the carbohydrate binding modules  , glycosyl transferases   and a rest of the proportion  comprising pectate lyases  and auxiliary activities  . the proportion of gh family was found to increase marginally with increase in the roughage proportion for the dry fed individuals. gt family was represented more in the liquid samples as compared to the solid samples.\n",
            "\n",
            "gh family can be further classified based on the functional role of the enzymes such as degradation of oligosaccharide, cellulose, hemicelluloses, pectin and other plant polymers. the contigs were found to comprise sequences related to a total of  <dig> different gh families amongst which majority included those playing role as oligosaccharide degrading enzymes  followed by debranching enzymes , cellulose degrading endohemicellulases  and cellulases , and cell wall elongation related enzymes  . out of these, oligosaccharide degrading gh <dig>  gh <dig>  gh <dig>  and cellulase families gh <dig> and gh <dig> were most highly represented. the abundance of the gh families encoding gene varied with respect to the treatments; green and dry diet; as well as the solid and liquid fractions. among the cellulases, gh <dig> and gh <dig> were the most abundant and were found to be more in the solid phase as compared to the liquid phase irrespective of diet, except for the gh <dig> in j <dig> treatment wherein it was more abundant in liquid compared to the solid phase. for the liquid phase the abundance of the former two families was more in the green diet fed individuals as compared to the dry fed. while for the solids the trend was found to be different viz. less in green compared to dry. increasing the roughage proportion increased the cellulase encoding egts in the solid phase of both green and dry diet fed individuals, whereas for the liquid the abundance was higher in j <dig>  reduced in j <dig> and again increased in j <dig>  similarly the endohemicellulases were found to be more abundant in the solid phases as compared to liquid, except for the gh <dig> and gh <dig> families which were almost negligible in the solid samples, with increase in the roughage . majority of the gh families in oligosaccharide degradation were more abundant in the liquid and well represented in the j <dig> and j <dig> treatments as compared to the j <dig> treatment except gh <dig>  gh <dig>  gh <dig>  gh <dig>  gh <dig> and gh <dig> which had slightly higher proportion in j2gl. the results thus reflect that diet has an impact on the microbial diversity further resulting in variability of the cazyme families present in the studied samples.fig.  <dig> glycosyl hydrolase family distribution \n",
            "\n",
            "\n",
            "\n",
            "we studied the taxonomy of the organism coding the gh families at genus level.\n",
            "\n",
            "while in the green roughage fed individuals, there was enrichment of major microbial communities involved in encoding gh during the 75 % roughage diet for liquid fraction, except for bacteroides that were least during the j2dl treatment . in the solid samples, fibrobacter were most abundant in j <dig>  while prevotella, bacteroides and clostridium were least. hence, based on the results it is evident that prevotella and fibrobacters genus is the most active in terms of genes encoding cazymes and hence can be exploited for industrial purposes.fig.  <dig> taxonomic distribution at genus level of microbes encoding genes for predicted cazyme   green roughage fed animals  dry roughage fed animals\n",
            "\n",
            "\n",
            "\n",
            "in the dry roughage fed individuals, it was clearly observed that prevotella was dominant in the liquid fractions and fibrobacter in the solid . also with increasing roughage the gh encoded by prevotella reduced for both solid and liquid. also in the liquid samples, the ruminicoccus genus was observed much higher in j <dig> and j <dig> as compared j <dig>  whereas in solid it was found to be more in j <dig> and then reduced in j <dig> and j <dig>  the fibrobacter genus was found to be coding more enzymes with increased roughage diet. the chitinophaga genus was abundant in j <dig> treatment and clostridium in j <dig> treatment for both the fractions. bacteroides genus was less in the j2ds samples as compared to j1ds and j3ds, while it was highest in j1dl followed by j2dl and then j3dl.\n",
            "\n",
            "discussion\n",
            "ruminants entirely depend on their microbial community for feed digestion. hence, appealing a hypothesis that there exists a strong correlation between the microbes and the host physiology focusing on nutrition availability and development. understanding of the rumen microbiome has comprehensive implications like effect on methane production by rumen methanogens to host health maintenance and animal production . rumen microbiome of the sturdy jaffrabadi breed buffalo native of the western india has not been deeply characterized as yet, especially by using the metagenomic approach.\n",
            "\n",
            "in context to the same, present study focused exploration of diet dependent variation in the rumen bacterial community composition and functional characterization of  <dig> rumen samples of jaffrabadi breed. four animals as biological replicates were considered to evaluate effect of different proportions of green and dry roughage so that any considerable animal to animal microbiome variation can also be observed. however, as previously reported we observed lower variation between replicates  <cit> . we analyzed the effect of increased roughage proportion in diet on fiber adhered and fiber free microbial populations and found that particular taxa are enriched/suppressed with the change in diet and the functional abilities related to fulfilling host energy requirements are correlated with the abundance of various bacterial taxa.\n",
            "\n",
            "effect of diet on microbial diversity\n",
            "bacteroidetes, firmicutes, fibrobacter and proteobacteria were the four dominant phyla observed in the present study. the results were similar to several of the earlier mammalian gut/rumen microbiota studies . the bacteroidetes were the most abundant phyla in majority samples, while few of the solid fraction samples had dominance of firmicutes, similar to the earlier results observed in bovine rumen  <cit> . reports have also revealed that the increase of fibrous content in diet increased the firmicutes abundance and decreased the bacteroidetes population in the gut microbiota, as also observed in our results  <cit> . the abundance was evident at the genus level also, where the prevotella and bacteroides genus dominated. bacteroides group depend on the soluble polysaccharides released in liquid phase of rumen by the activities of other bacteria  <cit> . prevotella have been shown to be involved in volatile fatty acid production like propanoate further used for energy by the host  <cit> . increased prevotella abundance could also be a reason in reducing methanogenesis as reported in a companion experiment  <cit> . along with bacteroidetes, firmicutes was one of the most abundant phyla in the rumen. at genus level, clostridium, ruminococcus and butyrivibrio were the most represented firmicutes. they are potent cellulose degraders as well as have the ability to digest pectin and hence are important component in the rumen dietary fiber utilization  <cit> . bacteroidetes and firmicutes population in the rumen and gut collectively involve in the conversion of organic matter to simpler forms  <cit> . the solid fraction favored fibrobacteres, suggesting their importance in fiber degradation similar to the results of microbiome analysis of cattle fed pasteur diets. however the results were contradictory to those reported earlier in mammalian guts, where the abundance was more in liquid fractions  <cit> .\n",
            "\n",
            "roughage vs. firmicutes/bacteroidetes ratio\n",
            "the firmicutes to bacteroidetes ratio has been considered an important parameter to assess the microbe impact on host energy requirements  <cit> . as observed the firmicutes are increasing with the increase in roughage proportion while the bacteroidetes population tends to decrease specifically during the dry fed diet. this leads to the increase in the f/b ratio, similar to the earlier reports where a shift from grain to fiber rich diets led to an increase in the ratio  <cit> .\n",
            "\n",
            "diet and correlation between various phyla\n",
            "microorganisms in natural environments are often involved in commensal or mutualistic interactions. specifically, in the rumen significant interspecies cross feeding of intermediate products has been observed among the adherent and non adherent bacteria population  <cit> . other interactions also include that between the carbohydrate fermenting bacteria which requires volatile fatty acids provided by the amino acid degrading ruminal bacteria  <cit> . in our study, we observed that the fibrolytic phyla fibrobacter and firmicutes positively correlated with the spirochaetes. studies have reported that co-culture of f. succinogens and a spirochaete species treponema enhanced fermentation of barley straw than a pure f. succinogens culture. also, spirochaetes decarboxylate the succinate formed by fibrobacter into vfa which in turn serves as an important energy source to the host animal  <cit> . results also revealed that bacteroidetes positively correlated with the verrucomicrobia, proteobacteria and tenericutes whereas negatively correlated with the cellulolytic fibrobacters and firmicutes. previous studies on bovine rumen have reported bacteroidetes to be negatively correlated with firmicutes  <cit> .\n",
            "\n",
            "functional diversity based on dietary changes\n",
            "functional metagenomics has paved the way in assessing potential metabolic activities of the rumen microbiome. the mg-rast results suggest that protein metabolism had the highest hits followed by carbohydrate and both increased with increase in the roughage proportion. initial colonization in ruminants is of the organisms possessing the enzymes to act on the side chains of complex plant polysaccharides followed by their complete degradation in the fluid fraction of rumen. nitrogen metabolism was abundant in j <dig> followed by j <dig> and j <dig>  at low protein diet, rumen bacteria are inclined to exploit an alternate source to fulfill the nitrogen requirement of host. similar results were also observed in bovine rumen microbiome  <cit> . in fiber diet both leguminous and non leguminous plants are present and hence the bacteria can trap ammonia-n in the form of bacterial protein and their digestion supplies the needed amino acids to the host  <cit> . significant difference in the solid and liquid fractions for nitrogen metabolism might indicate better interaction of fibrous legumes and nitrogen utilizing bacteria in the solid phase. sulfur increased the efficiency of nitrogen utilization for animals consuming a fescue hay-based diet  <cit> . we also observed similar results in our study.\n",
            "\n",
            "further analysis at level  <dig> in protein metabolism, results stated that protein biosynthesis was dominant in all the samples, confirming that microbial protein synthesis acts as a major source of protein for ruminants  <cit> . in carbohydrate metabolism, the genes associated with one-carbon metabolism, involving methane metabolism category at level  <dig> were abundant. these were most abundant in the j <dig> treatment suggesting that methanogenic archaea tends to decrease with high roughage diet. this could also be on account of the subsequent decrease in ph  <cit> . the genes associated with co <dig> fixation and central carbohydrate metabolism increased with roughage.\n",
            "\n",
            "to assess the correlation among bacterial phyla, pca was performed based on taxonomic and functional annotation. overall, pca according to the taxonomic and functional assignment of major categories showed that samples from dry and green roughage fed individuals of each treatment tended to cluster together, indicating that a similar set of functions were encoded in their metagenome.\n",
            "\n",
            "diet impact on rumen metabolism\n",
            "the estimation of vfa and various nitrogen moieties is necessary to track the microbial metabolism leading to the production of these compounds. vfa are extremely important as they endow more than 70 % of ruminants energy supply. large proportion of the natural food proteins release ammonia, which is utilized by the rumen microbes for their growth and synthesis of microbial protein. vfa produced by microbial action in the rumen are absorbed by the epithelium and further carried to the liver via the blood circulatory system. this continuous absorption of vfa maintains the ruminal ph. in the present study, vfa concentration was highest in the low roughage fed individuals. there are similar reports with high vfa concentration in buffaloes  <cit> . however, overall the concentration of vfa measured was comparatively lower compared to the earlier reports. the rumen microbiota converts ammonia into amino acids which is further accumulated as microbial protein. total nitrogen was significantly different between the green and dry groups, indicating that the change in feed from green roughage to dry roughage led to significant change in the nitrogen content of rumen fluid. also the n concentration was reduced from first treatment to the third. similar differences in n content were observed in mehsani buffalo fed on different diets and proportion of roughage  <cit> . similar trend was observed for the npn, soluble n and the ammonia n. the concentrations decreased with high roughage indicating enhanced consumption of nitrogen and higher microbial proteins synthesis.\n",
            "\n",
            "dietary effect on volatile fatty acid  production\n",
            "rumen microbiome produces volatile fatty acids such as propionic acid, butyric acid and acetate by microbial fermentation, which in turn maintains the animal health and homeostasis. the vfas are utilized further for milk and meat production contributing a major portion of the daily energy requirement of ruminants  <cit> . in our study, for propanoate production, out of the three plausible pathways, succinate pathway dominated. similar to our results, several studies have reported that succinate pathway is considerably the most commonly used pathway by gut bacteria for propanoate production in mammals as well as humans  <cit> . propanoate production was well represented during all the three treatments, but maximum hits were observed during the first treatment. the observed results could be due to the high prevalence of bacteroidetes during the first treatment  <cit> . earlier reports on bovine rumen have stated that propanoate production increases in rumen at the expense of acetate and butyrate with increase in abundance of prevotella sp.  <cit> . similarly, butanoate production was also found to be well represented during the first treatment followed by second and third treatment, which is contradictory to that observed in kankrej rumen where the production was lesser in equal proportion of roughage to concentrate compared to 75 % roughage diet  <cit> . butanoate plays an important role in animal health by regulating the host physiology and as a signal molecule modulating cell properties like differentiation, proliferation and apoptosis  <cit> .\n",
            "\n",
            "dietary effect on methanogenesis pathway\n",
            "methane production is an issue of immense concern worldwide for its role as a greenhouse gas  <cit> . various sources of methane emission exist such as wetlands, energy sectors, biomass burning, landfills and ruminants  <cit>  among which the enteric fermentation constitutes to be the largest source  <cit> . alleviation of methane by studying the rumen archaea metabolism may contribute to devising strategies to decrease the greenhouse effect  <cit> . in this perspective, we determined the egts involved in methanogenesis and effect of each treatment on the same. results reveal that an increase in roughage proportion reduced the egts involved in methanogenesis. the results are similar to previous studies on gir cattle where similar decrease in methane production was observed with increase in roughage rich diet .\n",
            "\n",
            "diet based variation in cazymes distribution\n",
            "the potential to digest the coarse feed into metabolic energy in rumen is entirely attributed to its polysaccharide degrading enzymes. rumen fluid due to its complex lignocellulosic degradation system is thus an excellent sample for mining cazymes  <cit> . in our study, oligosaccharide degrading enzymes were the most abundant of the total glycoside hydrolases. the higher occurrence of this category is justified due to their action on large number of oligosaccharides byproducts generated after initial breakdown of the complex polysaccharides like cellulose, hemicellulose, and pectin. among the ghs, gh <dig> that corresponds for glucuronidase and gh <dig> corresponding for l-arabinofuranisidase, b-d-glucosidase and b-d-xylopyranosidase activities were the most abundant. the major source for gh <dig> is known to be prevotella genus and hence we could observe that the proportion of gh <dig> encoding sequences decreased in the dl samples from j <dig> to j <dig> in correspondence to the decrease in prevotella population as seen from the genus level classification of organisms encoding the gh genes  <cit> . modifying the diet to higher roughage proportion has been found to decrease the amylase activity in most reports  <cit> . here we also observed similar trend whereby the gh <dig>   <dig> and  <dig> were reducing from j <dig> to j <dig>  the presence of gh <dig> indicates microbial degradation of hemicellulose and xylose in rumen as this includes the α-l-arabinofuranosidases, endo-α-l-arabinanases and β-d-xylosidases. variations in the abundance pattern of cazy families among the green and dry roughage groups of the three treatments reveal the modulating microbial diversity with respect to feed as also observed from the abundance study on organisms involved in encoding these cazymes.\n",
            "\n",
            "CONCLUSIONS\n",
            "the findings of the present study reveal that dietary changes have a significant impact on rumen microbiome at taxonomic as well as functional levels. it also leads to change in the vfa production as well as on methanogenesis with increase in the roughage proportion of diet, subsequently having impact on the host as well as at the global level. additionally, the findings also reveal that high roughage diet enhances the pegs involved in microbial protein synthesis and reduces those involved in methane formation. the information generated may allow us to modulate the rumen microbiome by providing a formulated diet with high dry roughage proportion and limiting the amounts of elements in diet that are mandatory for enzymes involved in methanogenesis. this would ultimately improve livestock nutrient utilization efficiency for better agricultural yield and have a global impact on environment by controlling the emission of methane gas. the study enhances the overall understanding of jaffrabadi buffalo rumen microbial ecology affected by diet, allowing us to explore the link between microbial metabolism and host physiology.\n",
            "\n",
            "availability of supporting data\n",
            "all the supporting data are included as additional files.\n",
            "\n",
            "additional files\n",
            "additional file 1: table s <dig>  detailed composition of the diet fed. \n",
            "\n",
            "additional file 2: figure s <dig>  propanoate  production pathway . \n",
            "\n",
            "additional file 3: figure s <dig>  butanoate  production pathway . \n",
            "\n",
            "additional file 4: figure s <dig>  methanogenesis pathway . \n",
            "\n",
            "additional file 5: figure s <dig>  cazyme family distribution throughout treatments . \n",
            "\n",
            "\n",
            "\n",
            "abbreviations\n",
            "cazymescarbohydrate active enzymes\n",
            "\n",
            "catcazyme analysis toolkit\n",
            "\n",
            "cdscoding sequence\n",
            "\n",
            "d/gdry or green diet\n",
            "\n",
            "egtsenvironmental gene tags\n",
            "\n",
            "ghglycosyl hydrolase\n",
            "\n",
            "jjaffrabadi\n",
            "\n",
            "l/sliquid or solid fraction of rumen fluid\n",
            "\n",
            "npnnon protein nitrogen\n",
            "\n",
            "pcaprincipal components analysis\n",
            "\n",
            "pegsprotein encoding genes\n",
            "\n",
            "vfavolatile fatty acid\n",
            "\n",
            "competing interests\n",
            "\n",
            "the authors declare no financial or non financial competing interests.\n",
            "\n",
            "author’ contributions\n",
            "\n",
            "nmn and mcs: performed the analysis and drafted the manuscript. akp: participated in statistical analysis and corrected the manuscript. br: carried out the experimental procedures. svs and pml: maintained the animals and performed the dietary feed treatments. rkk: helped in improving the analysis from the microbial aspect and checked manuscript for improving its technicality and language proficiency. cgj: conceived of the study, participated in its design and coordination. all authors have read and approved the final manuscript.\n",
            "\n",
            "the authors would like to thank dr. ramesh menon for his thorough proofreading and valuable suggestions that helped improve the manuscript. this work was financially supported by the niche area of excellence program on metagenomic analysis of ruminal microbes funded by indian council of agricultural research , new delhi, india. the authors are thankful to department of science and technology, government of india for providing grant to nmn in the form of inspire fellowship.\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "a total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed.\n",
            "a total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed.\n",
            "a total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed.\n",
            "a total of  <dig> million raw reads were obtained from  <dig> rumen samples of jaffrabadi breed.\n",
            "a total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed.\n",
            "a total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed.\n",
            "a total of  <dig> million raw reads and  <dig> million reads post quality filtering were obtained from  <dig> rumen samples of jaffrabadi breed\n",
            "94\n",
            "[{'summary_text': 'in drosophila, the jnk/ap- <dig> signaling cascade and ap- <dig> proteins have recently been identified as potential transcriptional targets of neuronal jnk and ap- <dig> signaling in drosophila.\\nin drosophila, the jnk/ap- <dig> signaling cascade and ap- <dig> proteins have recently been identified as potential transcriptional targets of neuronal jnk and ap- <dig> signaling in drosophila.\\nin drosophila, the jnk/ap- <dig> signaling cascade and ap- <dig> proteins have recently been identified as potential transcriptional targets of neuronal jnk and ap- <dig> signaling in drosophila.\\nin drosophila, the jnk/ap- <dig> signaling cascade and ap- <dig> proteins have recently been identified as potential transcriptional targets of neuronal jnk and ap- <dig> signaling in drosophila.\\nthe jnk/ap- <dig> signaling cascade and ap- <dig> proteins have recently been identified as potential transcriptional targets of neuronal jnk and ap- <dig> in dros'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 14365 to 15360 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "conserved neuronal signaling pathways regulate synaptic plasticity, the ability of neurons to modify synaptic connections. \"long-term\" forms of neuronal plasticity require new gene expression that results in persistent synaptic change . thus, long-term forms of plasticity may be perturbed, in a variety of model systems, by protein synthesis inhibitors, or manipulation of either specific signaling kinases or critical downstream transcription factors  <cit> .\n",
            "\n",
            "a major requirement in long-term plasticity for the erk/mapk  signaling cascade acting through creb  has been supported in diverse experimental paradigms  <cit> . activation of creb has been reported to enhance long-term memory in drosophila and rodents, and long-term facilitation in the sea slug aplysia  <cit> . normal erk signaling is required for hippocampal ltp formation, for bdnf-induced forms of structural plasticity, as well as for several forms of learning and long-term memory  <cit> . for example, erk activation is necessary for the formation of conditioned taste aversion and spatial learning in rodents, and blockade of erk signaling affects long-term, but not short-term, fear conditioning  <cit> . in addition, erk regulates synapse plasticity in flies and ltf and memory in aplysia  <cit> .\n",
            "\n",
            "many \"immediate-early genes\" , including members of the ap- <dig> family of transcription factors , are induced in response to diverse stimuli in the brain, such as electrical stimulation, stress, psychotropic drugs, novel experience and spatial learning  <cit> . induction of ap- <dig> mrna in neurons requires creb activation  <cit> . although roles have been established for ap- <dig> components Δfosb and c-fos in synaptic and behavioral plasticity  <cit> , the specific mechanisms and signal transduction pathways that initiate and sustain ap- <dig> dependent neuronal processes have yet to be elaborated  <cit> . for instance, the requirement for kinase-mediated modification of immediate-early transcription factors remains poorly studied in the context of neural plasticity, and early-response genes downstream of these critical iegs, have not yet been identified.\n",
            "\n",
            "while the majority of plasticity studies have focused on creb and the erk signaling cascade  <cit> , recent studies, especially of other map-kinase family members  <cit> , lead to a broader view of the molecules involved in neuronal plasticity and memory formation. the jnk/mapk  signaling cascade and ap- <dig> proteins have recently been shown to play critical roles in long-term plasticity and memory formation in mammals  <cit> . similarly, p38/mapk mediates memory formation in the rat hippocampus  <cit>  and both short- and long-term synaptic depression in aplysia  <cit> .\n",
            "\n",
            "as suggested by its importance in the control of processes underlying cocaine addiction  <cit> , new data indicate that ap- <dig> may widely influence transcriptional events that underlie long-term synaptic plasticity. ap- <dig>  under regulation by jnk, positively regulates both synaptic growth and synaptic strength at the drosophila neuromuscular junction   <cit> . at this synapse ap- <dig> shows a wider range of influence than creb whose effects at the same synapse are limited to controlling synaptic strength  <cit> . thus, while neural induction of fos and jun together is sufficient to cause increases in synaptic size and efficacy at the nmj, creb activation, apparently dispensable for synaptic growth, is only essential for ap- <dig> induced changes in synaptic strength  <cit> . in other neural contexts, the exact roles and mechanisms of ap- <dig> and jnk signaling in long-lasting forms of plasticity are largely unknown  <cit> . while many genes regulated during creb and Δfosb  mediated cocaine reward  <cit>  have been recently identified, this study identified exclusively late-response genes whose expression levels were altered following 5-days to 8-weeks of either Δfosb and creb overexpression in the nucleus accumbens, or cocaine injection.\n",
            "\n",
            "here, we address two outstanding questions regarding jnk and ap- <dig> function in synaptic change. first, using temporally controlled induction of a jnk-activating kinase in the fly nervous system, we address synaptic consequences of jnk activation at the drosophila neuromuscular junction. second, using dna microarray or sage  to profile neuronal transcripts in control and experimental animals, we identify groups of neuronal genes potentially regulated by either: a) jnk, or b) ap- <dig> in the fly cns within  <dig> hours of pathway activation. some of these changes were confirmed by quantitative real-time rt-pcr , including cg <dig>  that was previously identified in a screen as a potential gene required for normal memory formation in drosophila  <cit> . we found five genes are responsive to the progesterone-related steroid ru <dig> commonly used for temporal control of gal4-restricted transgene expression in drosophila  <cit> . in addition, the mini-white gene, a common marker used in most drosophila transgenes, is induced strongly by ap- <dig> and jnk signaling. these observations and their wider implications are discussed below.\n",
            "\n",
            "RESULTS\n",
            "neuronal jnk activation triggers synaptic growth\n",
            "to assess the effect of neuronal jnk activation on synaptic change we expressed an activated jnk-kinase – hemipterous   <cit>  – in the nervous system and analyzed associated phenotypic consequences. chronic overexpression of hepact with neuronal gal <dig> drivers  caused lethality ranging from late embryonic to early 2nd instar larval stages.\n",
            "\n",
            "in order to overcome this deleterious effect, we drove expression using the inducible geneswitch gal <dig>  system to express hepact acutely in postembryonic neurons  <cit> . age-selected larvae were exposed to the inducing ligand ru <dig> between late 1st instar and early 2nd instar larval stages and allowed to develop to climbing 3rd instar larval stage. neural overexpression of hepact resulted in a 30% increase in bouton number at the larval nmj compared to the non-ru <dig> fed sibling controls . changes in synapse size may not be attributed to the effect of ru <dig> since wild-type animals fed the steroid did not show a significant change in bouton number . thus, postembryonic activation of jnk signaling in the cns leads to synaptic growth at the larval motor synapse.\n",
            "\n",
            "jnk activation disrupts transmitter release and alters presynaptic composition\n",
            "to evaluate the effect of jnk activation on transmitter release and postsynaptic response, we measured both spontaneous and evoked junctional potentials with and without jnk activation in postembryonic cns. increased hepact resulted in an unexpected 60% decrease in the amplitude of excitatory junctional potentials  . increased neuronal jnk signaling also decreased the amplitude of spontaneous responses by ~50% . the quantal content of presynaptic transmitter release shows a 30% decrease when hepact is overexpressed in postembryonic neurons . therefore, while sustained postembryonic jnk activation in the cns triggers synaptic growth, the strength of the synapse is reduced. a potential cellular mechanism that underlies this reduction in quantal content was suggested by immunohistochemical analyses of nmjs in jnk-activated animals.\n",
            "\n",
            "presynaptic proteins including synaptic vesicle proteins, synaptotagmin, csp and antigen recognized by anti-hrp were substantially decreased when jnk was activated in motor neurons . levels of syt staining were reduced by 45%, csp by 30% and anti-hrp by 50% . in contrast, postsynaptically enriched proteins, fasciclin ii and dlg, do not show any change in intensity.\n",
            "\n",
            "because jnk has been implicated in axonal transport, we asked whether transport defects could possibly explain how jnk alters presynaptic composition  <cit> . defects in axonal cytoskeletal assembly or anterograde axonal transport cause accumulation of syt positive puncta on axonal tracts  <cit> . such organelle jams were not present in axonal tracts of larvae overexpressing hepact. the nerves were indistinguishable from control animals, indicating that visible axonal transport defects are not present after overexpression of hepact . for the purposes of this study, these results simply point to potential effects of jnk activation in the cns that go beyond its previously defined role as a positive regulator of ap- <dig> and, thereby, of synaptic growth and synaptic strength  <cit> .\n",
            "\n",
            "genome-wide screen to identify jnk targets in neurons\n",
            "to identify transcriptional targets of jnk signaling in the nervous system we performed a genome-wide analysis of jnk-responsive genes in the drosophila larval cns using serial analysis of gene expression . sage is an approach that has been extensively used in analyzing expression changes in cancer cells and other disease states as well as to analyze gene expression in the drosophila embryo and developing eye  <cit> . sage is based on generating unique  <dig> bp tags at a defined position in almost every transcript and, following random sequencing of some  <dig>  cdnas, analyzing the frequency at which each tag  occurs in a sample rna. we used sage to a) profile gene expression in the fly central nervous system; and b) identify transcriptional targets of neuronal jnk signaling. to identify early mediators of synaptic change, we analyzed rna expression  <dig> hours after jnk activation. we induced a 6-hour burst of neural hepact expression in third-instar larval nervous systems using ru <dig> feeding to induce transcription mediated by neural gs-gal <dig>  in experimental elavgs-gal4-hepact animals, we confirmed that jnk signaling was significantly activated by performing the following tests. quantitative pcr demonstrated a 16-fold induction of hep mrna in larval cns after hepact overexpression  . rna in situ hybridization of ru <dig> exposed larval cns showed a marked increase in hep mrna localization in the entire larval cns . immunostaining with an antibody specific for phsophorylated jnk showed induction of hepact mrna leads to activation of jnk . finally, we observed that downstream gene expression of a jnk target gene puc occurs after hepact induction in the cns. puc mrna is induced nearly 3-fold  by q-pcr analysis  and in an \"enhancer trap\" lacz line we were able to visualize puc promoter activity in the larval cns . thus, our protocol to stimulate neural jnk is sufficient to induce an established downstream target of jnk signaling. exposure of identically cultured, wild-type animals to ru <dig> did not lead to induction of hep or puc mrna or lead to activation of jnk . hence, changes observed between ru <dig> treated and untreated animals should be largely attributable to jnk signaling in the larval cns.\n",
            "\n",
            "we sequenced approximately  <dig>  tags from individual libraries prepared from dissected larval nervous systems of either control or hepact expressing animals. about  <dig> unique sage tags represented in these libraries were associated with specific genes/genomic sequences using either a database containing predicted tags of all genes annotated by the bdgp  <cit>  or blast searches to identify other transcription units  <cit> . approximately 10% of tags with less than  <dig> matches to the genome mapped to regions with no predicted gene. about  <dig> % of all tags did not match the genome probably due to polymorphisms, errors in sequencing or possible gaps in the published drosophila genome sequence. comparison of the top  <dig> expressed genes in the cns sage library to embryonic and photoreceptor sage libraries revealed that while 32% of these genes are highly expressed in all  <dig> tissues such as the cytoskeletal protein betatub56d, 37% are enriched in the nervous system like the translation elongation factor ef1alpha100e . such comparisons could prove useful for understanding transcriptional regulation and other processes in different tissues .\n",
            "\n",
            "potential jnk-target genes were identified by comparing the relative representation of specific tags in control and hepact expressing nervous systems . a tag was considered up or downregulated when present  <dig> or more times in a given library and changed at least 3-fold between the two libraries. by these criteria,  <dig> tags were increased while  <dig> were decreased following jnk induction. of these,  <dig> were \"upregulated\" and  <dig> \"downregulated\" more than 8-fold. approximately 50% of the induced or repressed tags in the hepact library mapped to genes that fell into different functional classes, ~35% of these tags mapped to genes that have no predicted function and ~10% mapped to parts of the genome without any predicted genes .\n",
            "\n",
            "to determine whether predictions of sage could be confirmed by more careful single gene analyses, we performed q-pcr to measure relative levels of expression of selected candidate jnk-target genes in control and jnk-induced nervous systems. we selected  <dig> candidate genes for such q-pcr verification based on: 1) an abundance of tags for that gene in the hepact induced library ; 2) an interesting known function for the gene ; 3) presence of ap- <dig> binding sites in the promoter region of the gene ; and 4) random selection of genes that did not fall into the above criteria . from the  <dig> genes so examined,  <dig> that showed induction in at least two independent rt-pcr analyses were analyzed further, namely more extensive q-pcr analyses using rnas from  <dig> independent jnk-induction experiments. in the end, only three genes showed consistent jnk-responsiveness  . white showed consistent and robust increases in mrna levels, whereas appl and cher showed smaller magnitude inductions. to test if some of the candidate jnk target genes were robustly regulated in subsets of neurons, but diluted out in the q-pcr analysis, we examined the expression of rna in the larval cns using in situ hybridizations with probes for several candidate mrnas. we did not see a clear increase in expression in any of the putative target genes in hepact expressing larval cnss other than white .\n",
            "\n",
            "the small number of sage-predicted jnk target genes confirmed by rna in situ and q-pcr analyses was difficult to explain without multiple repetitions for which sage, being expensive and time-consuming, is not ideally suited. also, we speculated that genes expressed at lower levels than those identified by sage may be true jnk/ap- <dig> target genes. to test and further these considerations, we used a different genomic approach – oligonucleotide microarrays – to search for neuronal ap- <dig> target genes.\n",
            "\n",
            "whole-genome microarray screen to identify direct ap- <dig> target genes in the nervous system\n",
            "we performed comprehensive analyses of transcript levels in fly heads using affymetrix drosophila genome <dig> genechip arrays representing the entire annotated genome at the time of its release . an outline of the microarray screen design is illustrated in figure  <dig>  the analysis compared mrna levels, with or without ap- <dig> induction using the same conditional gs-gal <dig> strategy described for the previous sage analysis. after treatment with ru <dig> for  <dig> hours to induce fos and jun we consistently achieved, respectively, ~ <dig> - and 9-fold induction of fos and jun transcripts in fly heads . untreated control animals showed no significant difference of either gene when levels were compared between age-matched siblings from the same experiment .\n",
            "\n",
            "each array \"experiment\" included sibling flies split into three groups: group a was the experimental  population . thus, each experiment typically allowed transcript levels  to be compared between experimental and control samples , and between two identically treated controls . candidate ap- <dig> responsive transcripts would be identified as those with \"a/b\" and \"a/c\" ratios significantly different from control \"b/c\" ratios. this experimental design was useful because hybridization signals for some mrnas varied significantly more than others and could potentially confound a more straightforward analysis. through 5– <dig> repetitions of this basic experiment, we obtained  <dig> independent experimental versus control ratios, and  <dig> control-control ratios from which means, variances and sems could be determined. sibling, age-matched controls used in each experiment ensured that genetic background, which can have a large effect on transcriptional variance  <cit> , was not a confounding factor in our analyses.\n",
            "\n",
            "based on analyses of  <dig> hybridizations we established that basic elements of the array technology, probe labeling, hybridization and scanning, were working efficiently and reproducibly . microarray hybridization data were passed through three statistical filters to select the most promising ap- <dig> responsive genes . filter 1: we asked that the average ratio of hybridization signal from ap- <dig> induced versus control mrna hybridization was significantly  different from  <dig>  by student's t-test. filter 2: we asked that the signal ratio be greater than  <dig> . filter 3: through analysis of variation observed in identical control-control comparisons, we ensured that genes passing filters  <dig> and  <dig> did not show wide variability, for instance based on physiological states of the flies.\n",
            "\n",
            "using filter 1: of the ~ <dig> genes considered for analysis ,  <dig> showed altered expression after ap- <dig> induction, with a significance of p <  <dig>  . strikingly,  <dig> genes showed significant upregulation while only  <dig> were downregulated, a skew consistent with ap-1's expected role as a transcriptional activator.  <dig> candidates, at p <  <dig> , is substantially larger than predicted by random chance . however, when a second filter – a requirement that the signal ratio modulus be greater than  <dig>  – was applied, the number of candidates dropped to  <dig>  though small, such signal ratios could correspond to higher mrna ratios and have been reported as meaningful in previous microarray experiments. filter  <dig>  to eliminate \"variable\" genes, trimmed the list of candidate genes that respond consistently to ap- <dig> overexpression in the fly head to either  <dig>  or  <dig>  for which \"a/b\" and \"a/c\" ratios were significantly greater than control \"b/c\" values by student's t-test.\n",
            "\n",
            "an internal control for the array screen and analysis was provided by drosophila jun , whose mrna was experimentally induced. we found that jun ranked highest once all three filters were applied and showed robust induction with an average log <dig> treated expression ratio of  <dig>   and an average untreated expression change of only - <dig>  . in contrast, fos did not pass these stringent filters although we consistently observed an average  <dig> -fold increase in fos transcript levels by quantitative rt-pcr . this discrepancy may arise from either of two limitations: a) that fos is a low-abundance transcript in the fly head, below the threshold for quantifiable gene expression change detection using affymetrix genechip arrays; or b) the fos probe on this particular array may not perform reliably  <cit> , perhaps hybridizing to other non-specific rna probes. many gene probes could have similar problems; indeed, other transcripts may exhibit altered expression levels beyond the scope and sensitivity of this assay.\n",
            "\n",
            "we searched promoters  of the top  <dig> candidate ap- <dig> responsive genes for conserved ap- <dig> or creb binding sites and compared their frequencies of occurrence in this group with frequencies observed in a control group of  <dig> genes that appeared insensitive to ap- <dig> induction. this analysis revealed no significant enrichment of creb or ap- <dig> binding elements in promoters selected based on the microarray experiments .\n",
            "\n",
            "microarray screen validation\n",
            "a major task after initial microarray screening has been completed is confirmation of candidate gene transcript level changes using secondary, independent tests for gene expression. although the frequency of false-positives is substantially reduced through repetition, a subset of observed expression differences should be validated by other methods.\n",
            "\n",
            "to confirm positives, a subset of the most robustly changing ap-1-responsive genes, exhibiting significant up- or down-regulation by microarray analysis, were selected as candidates for real-time quantitative rt-pcr validation using gene specific primers .  <dig> genes chosen from the group of  <dig> top candidates mentioned above, in addition to more than  <dig> genes from outside this stringent set – those with very low \"p\" values or specific predicted biological functions – were selected for these more careful confirmatory experiments. increases in transcript levels following ap- <dig> overexpression, detected by q-pcr, for fos, jun and  <dig> confirmed candidate genes  are shown in figure 8a. all mrna levels are normalized to the control gene rp <dig>  transcript levels for a second control gene, gapdh <dig>  are shown to demonstrate its levels do not change significantly by either ap- <dig> induced versus control  or control-control  comparisons.\n",
            "\n",
            "five uncharacterized genes  were confirmed by q-pcr to be consistently altered in ru486-treated, ap- <dig> induced samples . in addition, overexpression of fbz with ru <dig> treatment also caused a similar change in transcript levels of these  <dig> genes. when treated with the steroid, wild type flies and all other transgenic lines tested showed consistent alterations of these  <dig> mrna transcripts in the head, suggesting they are hormone-responsive genes in the fly.\n",
            "\n",
            "as in the larval cns, q-pcr experiments confirmed white gene induction in the adult fly head. white transcript levels are significantly increased in the head following ap- <dig> overexpression in the brain . further q-pcr experiments demonstrated white transcripts are increased to an even greater extent when hepact is induced in combination with ap- <dig> or by itself in the adult nervous system , although its levels are not increased to the degree seen in the larval cns . white is not induced when fbz is overexpressed, nor in wild type flies treated with ru <dig>  only primers designed to the 3' portion of the white transcript showed altered levels , which is consistent with the background strain used in all the experiments. this strain lacks the 5' portion of the white gene locus  <cit> , yet still contains sequence for and expresses the second through fifth exons  that are induced in response to jnk signaling in the fly head. the same transcriptional induction profile is observed in elavgs-gal4-hepact flies with the wild-type  copy of the white gene on the x chromosome as well as in a white null  background . this suggests white induction occurs via the mini-white cassette present in puast transgenes. rna in situ hybridization experiments confirmed the increase in mini-white transcript levels in the cns of larvae in which hepact has been induced . increases were also observed in larvae overexpressing ap- <dig> , albeit with smaller magnitude changes consistent with our findings from q-pcr analyses in the adult head .\n",
            "\n",
            "cg <dig> induction following ap- <dig> overexpression was also confirmed in independent q-pcr experiments . consistent increases in transcript levels were observed in all ap- <dig> overexpressing heads but not in treated fbz or wild type heads . the induction observed by quantitative rt-pcr and microarray experiments was not reflected in follow-up in situ experiments; however, this is likely because the small magnitude increases in transcript levels observed by other means  are below the detection range for this method.\n",
            "\n",
            "discussion\n",
            "this extensive study makes three contributions:  it demonstrates unexpected and novel interactions between jnk and cellular processes that underlie synapse plasticity;  by sage analyses, it provides a genomic profile of mrnas expressed in the fly larval nervous system;  it presents two large-scale genomic approaches to identify jnk and ap- <dig> targets in the fly cns providing useful data pertinent to jnk/ap- <dig> signaling in neurons as well as to genomic analyses in the drosophila nervous system.\n",
            "\n",
            "effects of jnk activation in postembryonic motorneruons\n",
            "the immediate-early transcription factor ap- <dig> positively regulates both synapse size and synapse strength at the drosophila larval nmj  <cit> . while jnk signaling is necessary for the effect of ap- <dig> on synapse structure and function, it is not clear whether jnk signaling is sufficient for synaptic change. we show, first, that activation of jnk in post-embryonic neurons leads to significant synaptic alterations; second, that these alterations are inconsistent with jnk functioning solely through ap- <dig>  our finding that activation of jnk signaling leads to an increase in synapse number but decreases synapse strength indicates that jnk activates not only ap- <dig>  a positive regulator of growth and strength, but also a pathway that negatively influences synaptic strength.\n",
            "\n",
            "the neural transcriptome, and its regulation by jnk and ap-1\n",
            "the ability of sage to evaluate absolute expression levels of gene transcripts enables relatively facile, quantitative, profiling of gene expression in any given tissue . given the intense interest in drosophila neurobiology, a previous painstaking sequence analysis of some  <dig> cdnas from a fly brain cdna library provided useful new information on the neural transcriptome  <cit> . the analysis presented here, following sequencing of about  <dig>  ests from two independent brain libraries, substantially extends the previous study. the use of this resource is demonstrated by our simple survey of highly expressed neuronal rna-binding proteins, potentially involved in important neural-specific, post-transcriptional functions such as translational repression, mrna transport or rna editing. 10% of the  <dig> most highly expressed  mrnas in nervous system encode rna-binding proteins,  <dig> of which are enriched in neurons versus embryonic tissue. a significant fraction of these  have conserved homologs recently found on rna granules, organelles containing translationally repressed mrnas which are actively transported to synaptic sites  <cit> . we have recently begun functional analyses of some of these rna-binding proteins. similarly, we anticipate that identification of tissue-specific genes could provide unanticipated launch points for investigation into their cellular functions.\n",
            "\n",
            "given the evidence to indicate wide effects of ap- <dig> and jnk on synaptic properties, we searched for ap- <dig> and jnk-target genes using both sage and microarray approaches to determine effects of jnk and ap- <dig> signaling on neuronal gene expression. of the two approaches, microarray analysis, being dependent on parameters such as hybridization and labeling efficiencies that vary among individual transcripts, is not ideal for quantitative analyses as outlined in the previous section. however, it provides information on transcripts with low to moderate levels of expression, is fast, and allows multiple iterations of each experiment at a small cost relative to sage.\n",
            "\n",
            "in order to identify early transcriptional targets, most likely to link jnk and ap- <dig> activation to synaptic change, we used the steroid-inducible gal <dig> system, an increasingly popular strategy to achieve conditional, tissue-specific transgene expression in drosophila  <cit> . sage-derived transcript profiles of rna extracted from whole larval cnss showed several potentially significant targets. however, very few were confirmed by secondary low-throughput, gene-specific analyses. microarray-derived transcript profiles of adult head mrna showed similar results. several statistically significant targets of ap- <dig> signaling were initially identified; however, few were confirmed by carefully controlled application of the most commonly used transcript-specific analyses . while the implications of these results for neurogenomics are briefly discussed in the next section, we first consider the \"positive\" genes identified by sage and microarray screens.\n",
            "\n",
            "quantitative rt-pcr validation of the generated sage data resulted in the identification of  <dig> genes, cher, appl and white, which were consistently upregulated following jnk activation in the larval cns. though we were unable to evaluate induction of cher and appl by rna in situ hybridization, white showed robust increases by this method as it did by q-pcr. a total of seven expression changes identified in the microarray screen were verified by q-pcr analysis; remarkably, five turned out to be genes responding to ru <dig> treatment itself rather than to consequent ap- <dig> induction. these steroid-responsive genes may be of significant biological interest. however, from our point of view they serve primarily to: a) further establish the bonafides of our experimental and analytical protocols; and b) as a useful caution for drosophilists and others using the steroid-inducible conditional expression system. the remaining two confirmed ap- <dig> target genes were w hite, also identified in the sage screen but shown eventually to be expressed from the p-element associated mini-white locus, and cg <dig>  of potential significance, cg <dig> has been implicated in olfactory associative memory  <cit> .\n",
            "\n",
            "ap- <dig> responsiveness of cg <dig> was verified in q-pcr validation experiments . the gene was previously found in a mutational screen for putative memory genes required for normal olfactory conditioning in drosophila  <cit> . in addition, it is one of the few genes from the list of likely ap- <dig> targets  that has a conserved ap- <dig> binding site within  <dig> base pairs of its translation start site. it is therefore a promising candidate warranting further investigation into the role it plays in synaptic plasticity and memory formation.\n",
            "\n",
            "lessons and limitations\n",
            "it appears unlikely, if not inconceivable, that the  <dig> probable downstream genes enumerated above could mediate the demonstrated effects of ap- <dig> or jnk induction on motor-synapse properties. thus, the genomic approaches we have followed, while informative, have likely not led to the identification of jnk/ap- <dig> targets that link these signals to synaptic change. one possible interpretation, that the experiments were technically flawed, appears to be ruled out, not only because internal controls  were identified in the screens, but also because various standards for microarray hybridization data and sage library complexity were evaluated and shown to be well within the technically optimal range. thus, we are left with the second interpretation, that analysis of whole-brain mrna may not allow targets of signaling pathways to be unambiguously identified. a major issue is likely to be cell-type heterogeneity within the brain. if different subsets of neurons show substantially different genomic responses to jnk/ap- <dig> , then altered expression of the meaningful jnk/ap- <dig> targets in a subset of cells may be diluted by the large background of mrna deriving from other neuronal types.\n",
            "\n",
            "at a conceptual level, barolo and posakony have nicely articulated the concept of \"activator insufficiency\" and the need for cooperative activation of multiple transcription factors for turning on transcriptional pathways governing developmental processes  <cit> . considerable evidence argues that neurons are a diverse class of cells with a range of distinct transcriptional ground states. for example, cell-type-specific binding of creb to known target gene promoters has been shown in various cell types under basal and stimulated conditions  <cit> . similarly, the response of different neuronal populations to tgfβ has been shown to be highly context dependent and to derive from variations in expression of specific tgfβ insensitive transcription factors  <cit> . thus, genomic analyses when applied to whole nervous systems may have significant intrinsic limitations.\n",
            "\n",
            "nevertheless, some conserved downstream genes may still be revealed  <cit> . for instance, the steroid hormone, ru <dig>  used to induce transgene expression in our experiments presumably activates a set of hormone-responsive genes in a large subset of neural cells. however, for incisive mechanistic analyses for which drosophila is so convenient, we suggest that genome-wide screens described to study signaling responses in the nervous system be applied with specific refinements, such as emerging methodologies to prepare sufficient mrna from a homogeneous population of cells in which biological function of these signaling pathways have been evaluated  <cit> . various gfp transgene lines should make it possible to sort specific cell populations prior to genomic screens to identify transcriptional targets.\n",
            "\n",
            "the availability of new genetic and molecular tools and refined functional genomic approaches should result in continued understanding of how kinases and transcription factors regulate molecular changes that occur in the drosophila nervous system, as well as intrinsic flexibility and constraints of these signaling pathways.\n",
            "\n",
            "CONCLUSIONS\n",
            "this study revealed unexpected relationships between jnk signaling and synaptic plasticity in drosophila that are inconsistent with a role for jnk acting solely through ap- <dig> to affect strength of the synapse. it also presents a profile of the transcriptome of the larval nervous system and, while providing potential transcriptional targets of jnk and ap- <dig> signaling in neurons, points out the pitfalls of genome-wide analyses in complex tissues such as the whole fly nervous system.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in drosophila, the jnk/ap- <dig> signaling cascade and ap- <dig> proteins have recently been identified as potential transcriptional targets of neuronal jnk and ap- <dig> signaling in drosophila.\n",
            "in drosophila, the jnk/ap- <dig> signaling cascade and ap- <dig> proteins have recently been identified as potential transcriptional targets of neuronal jnk and ap- <dig> signaling in drosophila.\n",
            "in drosophila, the jnk/ap- <dig> signaling cascade and ap- <dig> proteins have recently been identified as potential transcriptional targets of neuronal jnk and ap- <dig> signaling in drosophila.\n",
            "in drosophila, the jnk/ap- <dig> signaling cascade and ap- <dig> proteins have recently been identified as potential transcriptional targets of neuronal jnk and ap- <dig> signaling in drosophila.\n",
            "the jnk/ap- <dig> signaling cascade and ap- <dig> proteins have recently been identified as potential transcriptional targets of neuronal jnk and ap- <dig> in dros\n",
            "95\n",
            "[{'summary_text': 'we found that mcl-caw performed better than mcl-caw in deriving complexes from weighted ppi networks, and that mcl-caw responded reasonably well to the considered affinity scoring schemes.\\nin this paper, we develop a novel core-attachment based refinement method coupled to mcl to identify yeast complexes from weighted ppi networks.\\nour results show that mcl-caw is able to derive considerably higher number of complexes than mcl-caw, and that mcl-caw responds reasonably well to the considered affinity scoring schemes.\\nour results indicate that mcl-caw is able to derive significantly higher number of complexes than mcl-caw, and that mcl-caw responds reasonably well to the considered affinity scoring schemes.\\nour results suggest that mcl-caw is able to derive significantly higher number of complexes than mcl-caw, and that mcl-caw responds reasonably well to the considered affinity scoring schemes.\\nour results suggest that mcl-caw is able to derive significantly higher number of complexes than mcl-caw, and that mcl-caw responds reasonably well to the considered affinity scoring schemes.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 4954 to 5120 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "most biological processes are carried out by proteins that physically interact to form stoichiometrically stable complexes. even in the relatively simple model organism saccharomyces cerevisiae , these complexes are comprised of many subunits that work in a coherent fashion. these complexes interact with individual proteins or other complexes to form functional modules and pathways that drive the cellular machinery. therefore, a faithful reconstruction of the entire set of complexes from the physical interactions between proteins is essential to not only understand complex formations, but also the higher level organization of the cell.\n",
            "\n",
            "these physical interactions between proteins have been most extensively catalogued for yeast using high-throughput methods like yeast two-hybrid  <cit>  and direct purification of complexes using affinity tags followed by mass spectrometry  analyses  <cit> . in  <dig>  the direct purification strategy or \"pull-down\" was first applied to yeast in two independent studies by gavin et al.  <cit>  and ho et al.  <cit> . more recently , two separate groups, gavin et al.  <cit>  and krogan et al.  <cit> , employed tandem affinity purification  followed by ms analyses to produce enormous amount of new data, allowing a more complete mapping of the yeast interactome. although these individual datasets are of high quality, they show surprising lack of correlation with each other  <cit> , and some bias towards high abundance proteins  <cit>  and against proteins from certain cellular compartments   <cit> . also, each dataset still contains a substantial number of false positives  that can compromise the utility of these datasets for more focused studies like complex reconstruction  <cit> . in order to reduce the impact of such discrepancies, a number of data integration and affinity scoring schemes have been devised  <cit> . these affinity scores encode the reliabilities  of physical interactions between pairs of proteins. therefore, the challenge now is to detect meaningful as well as novel complexes from protein interaction  networks derived by combining multiple high-throughput datasets and by making use of these affinity scoring schemes.\n",
            "\n",
            "the interaction data produced from the high-throughput tap/ms experiments comprise of tagged \"bait\" proteins and the associated \"prey\" proteins that co-purify with the baits. gavin et al.  <cit>  considered direct bait-prey as well as indirect prey-prey relationships , followed by a socio-affinity scoring system to encode the affinities between the protein pairs. the socio-affinity score quantizes the log-ratio of the number of times two proteins are observed together relative to what would be expected from their frequency in the dataset. subsequently, gavin et al. used an iterative clustering approach to derive complexes. each complex was then partitioned into groups of proteins called \"core\", \"attachment\" or \"module\" . on the other hand, krogan et al.  <cit>  used machine learning techniques  to define confidence scores for interactions derived from direct bait-prey observations . subsequently, krogan et al. defined a high-confidence 'core' dataset of interactions, and used the markov clustering algorithm   <cit>  to derive complexes. hart et al.  <cit>  generated a probabilistic integrated co-complex  network by integrating matrix modeled relationships of the gavin et al., krogan et al. and ho et al. datasets using a measure similar to socio-affinity scores, and then used a mcl procedure to derive complexes from this network. collins et al.  <cit>  developed a purification enrichment  scoring system to generate the 'consolidated network' from the matrix modeled relationships of the gavin et al., and krogan et al. datasets. collins et al. used a bayes classifier to generate the pe scores in the consolidated network by incorporating diverse evidence from hand-curated co-complexed protein pairs, gene ontology  annotations, mrna expression patterns, and cellular co-localization and co-expression profiles. this new network was shown to be of high quality - comparable to that of ppis derived from small-scale experiments stored at the munich information center for protein sequences . zhang et al.  <cit>  used dice coefficient  to assign affinities to protein pairs, and evaluated their affinity measure against socio-affinity and pe measures. they concluded that dc and pe offered the best representation for protein affinity, and subsequently used them for complex prediction. pu et al.  <cit>  used mcl combined with cluster overlaps on the consolidated network to reveal interesting insights into complex organization. wang et al.  <cit>  proposed haco, a hierarchical clustering with overlap algorithm, to reconstruct complexes and used them to build the 'complexnet', an interaction network of proteins and complexes, in order to study the higher-level organization of complexes. chua et al.  <cit>  and liu et al.  <cit>  developed network topology-based scoring schemes called functional similarity weight  and iterative-czekanowski-dice , respectively, to assign reliability scores to the interactions in networks. subsequently, liu et al.  <cit>  used a maximal clique merging strategy  to derive complexes from networks scored using these two systems. friedel et al.  <cit>  developed a bootstrapped scoring system to score tap/ms interactions from gavin et al. and krogan et al., and subsequently derived complexes using a variant of mcl. friedel et al.  <cit>  also developed a minimum spanning tree-based method to reconstruct the topology of complexes from co-purified proteins in tap/ms assays. voevodski et al.  <cit>  used pagerank, a random walk-based method employed in context-sensitive web search, to define the affinities between proteins within ppi networks. subsequently, voevodski et al. used it to predict co-complexed proteins within the network. approaches like core  <cit>  and coach  <cit>  adopted local dense neighborhood search to derive cores and attachments from unscored networks. mitrofanova et al.  <cit>  measured the connectivity between proteins in unweighted ppi networks by edge-disjoint paths instead of edges to overcome noise, and modeled these paths as a network flow and represented it in gomory-hu trees. they subsequently isolated groups of nodes in the trees that shared edge-disjoint paths in order to identify complexes. very recently, ozawa et al.  <cit>  used domain-domain interactions to validate and refine the complexes predicted by mcl.\n",
            "\n",
            "in this study, we develop an algorithm to derive yeast complexes from weighted  ppi networks. inspired by the experimental findings by gavin et al.  <cit>  on the modularity structure in yeast complexes, and the distinctive properties of \"core\" and \"attachment\" proteins, we develop a novel core-attachment based refinement method coupled to mcl for reconstruction of yeast complexes. we had proposed the idea of core-attachment based refinement in a preliminary work  <cit>  and called it mcl-ca.\n",
            "\n",
            "however, mcl-ca worked only on unscored networks. here, we devise an improved algorithm  and provide a natural extension to work on scored  ppi networks. even though most eukaryotic complexes are hypothesized to display such core-attachment modularity, here we design our algorithm specific to yeast complexes because of lack of sufficient evidence, high-throughput datasets and reference complexes from other organisms. we combine tap/ms physical datasets from gavin et al.  <cit>  and krogan et al.  <cit>  to generate an unscored ppi network . we then score this network using two topology-based affinity scoring schemes, fs weight  <cit>  and iterative-cd  <cit> , to generate scored ppi networks. we gather two additional readily-available scored ppi networks from collins et al.  <cit>  and friedel et al.  <cit> . the evaluation of mcl-caw on these networks demonstrates that:  mcl-caw is able to derive higher number of yeast complexes and with better accuracies than mcl;  affinity scoring effectively reduces the impact of noise on mcl-caw and thereby improves the quality  of its predicted complexes;  mcl-caw responds well to most available affinity scoring schemes for ppi networks. we compare mcl-caw with several recent complex detection algorithms on both unscored and scored ppi networks. finally, we perform in-depth analysis of the predicted complexes from mcl-caw.\n",
            "\n",
            "the gavin+krogan network was generated by combining the gavin and krogan-core datasets, obtained from biogrid  <cit> . the icd and fsw networks were generated by scoring the gavin+krogan network using the iterative-cdk and fs weightk schemes  <cit>  . the consolidated <dig>  network refers to the high-confidence subset  of the consolidated network derived by collins et al.  <cit> . the bootstrap <dig>  network refers to the high-confidence subset  of the bootstrap network derived by friedel et al.  <cit> . the inferred network comprised of computationally inferred interactions from the predictome database  <cit> . the gavin+krogan+inferred network was generated by augmenting the gavin+krogan network with these inferred interactions. the icd and fsw networks were generated by scoring this augmented network using the iterative-cd and fs weight systems, respectively.\n",
            "\n",
            "methods\n",
            "the mcl-caw algorithm: identifying complexes embedded in the interaction network\n",
            "our mcl-caw algorithm broadly consists of two phases. in the first phase, we partition the ppi network into multiple dense clusters using mcl. following this , we post-process  these clusters to obtain meaningful complexes. the mcl-caw algorithm consists of the following steps:\n",
            "\n",
            " <dig>  clustering the ppi network using mcl hierarchically\n",
            "\n",
            " <dig>  categorizing proteins as cores within clusters\n",
            "\n",
            " <dig>  filtering noisy clusters\n",
            "\n",
            " <dig>  recruiting proteins as attachments into clusters\n",
            "\n",
            " <dig>  extracting out complexes from clusters\n",
            "\n",
            " <dig>  ranking the predicted complexes\n",
            "\n",
            "we use the following notations while describing our algorithm. the ppi network is represented as g = , where v is the set of proteins, and e is the set of interactions between these proteins. for each e =  ∈ e, there is a confidence score  w encoding the affinity between the proteins p and q. these affinity scores depend on the scoring system used.\n",
            "\n",
            "clustering the ppi network using mcl hierarchically\n",
            "the first step of our algorithm is to partition  the ppi network using mcl  <cit> , which simulates random walks  to identify relatively dense regions in the network. the inflation coefficient parameter i in mcl is used to regulate the granularity of the clusters - higher the value more finer are the generated clusters . mcl tends to produce several large clusters  that amalgamate smaller clusters  <cit> . on the other hand, the size distributions of hand-curated complexes from wodak lab  <cit> , mips  <cit>  and aloy et al.  <cit>   reveal that most complexes are of sizes less than  <dig>  therefore, we perform hierarchical clustering by iteratively selecting all clusters of sizes at least  <dig> and re-clustering them using mcl.\n",
            "\n",
            "after iterative rounds of mcl-based hierarchical clustering on the protein network g = , we obtain a collection of k disjoint  clusters {ci : ci = ,  <dig> ≤ i ≤ k}, where vi ⊆ v and ei ⊆ e.\n",
            "\n",
            "categorizing proteins as cores within clusters\n",
            "microarray analysis by gavin et al.  <cit>  of their predicted complex components showed that a large percentage of pairs of proteins within cores were co-expressed at the same time during the cell cycle and sporulation, consistent with the view that cores represent main functional units within complexes. three-dimensional structural and yeast two-hybrid analysis showed that the core components were most likely to be in direct physical contact with each other. to reflect these findings in our post-processing steps, we expect:\n",
            "\n",
            "• every complex we predict to comprise of a non-empty set of core proteins; and\n",
            "\n",
            "• the proteins within these cores to display relatively high degree of physical interactivity among themselves.\n",
            "\n",
            "we identify the core proteins within a cluster in two stages: we first identify the set of preliminary cores and subsequently extend this to form the final set of cores. we categorize a protein p∈vi to be a 'preliminary core' protein in cluster ci = , given by p ∈ pcore, if:\n",
            "\n",
            "• the weighted in-connectivity of p with respect to ci is at least the average weighted in-connectivity of ci, given by: din ≥ davg ; and\n",
            "\n",
            "• the weighted in-connectivity of p with respect to ci is greater than the weighted out-connectivity of p with respect to ci, given by: din >dout.\n",
            "\n",
            "the weighted in-connectivity din of p with respect to ci is the total weight  of interactions p has with proteins within ci. similarly, the weighted out-connectivity dout of p with respect to ci is the total weight of interactions p has with proteins outside ci. these are given by din  = ∑ {w : q ∈ vi} and dout  = ∑ {w : q ∉ vi} respectively. the average weighted in-connectivity davg of cluster ci is therefore the average of the weighted in-connectivities of all proteins within ci, given by davg=1|ci|⋅∑q∈vidin.\n",
            "\n",
            "we use these preliminary cores to find the 'extended core' proteins. we categorize a protein p ∉ pcore to be an extended core protein in cluster ci, given by p ∈ ecore, if:\n",
            "\n",
            "• the weighted in-connectivity of p with respect to pcore is at least the average of the weighted in-connectivities of all non-cores r ∉ pcore  to the preliminary cores, given by: din ) ≥ davg ); and\n",
            "\n",
            "• the weighted in-connectivity of p with respect to pcore is greater than the weighted out-connectivity of p with respect to pcore, given by: din) >dout).\n",
            "\n",
            "here, din) is the total weight of interactions p has with the preliminary cores of ci, given by: din ) = ∑ {w : q ∈ pcore }. similarly, dout) is the total weight of interactions p has with all the non-core proteins within ci, given by:\n",
            "\n",
            "din ) = ∑ {w : r ∈ pcore }. finally, davg) is the average weight of interactions of all non-cores r with the preliminary cores, given by:\n",
            "\n",
            " davg)=1|)⋅∑r∈pcoredin). \n",
            "\n",
            "combining the preliminary and extended core proteins, we form the final set of core proteins of cluster ci, given by:\n",
            "\n",
            "  core={pcore∪ecore}. \n",
            "\n",
            "filtering noisy clusters\n",
            "consistent with the assumption that every complex comprises of a set of core proteins, we consider a cluster as noisy if it does not include any core protein as per our above criteria. we discard all such noisy clusters.\n",
            "\n",
            "recruiting proteins as attachments into clusters\n",
            "microarray analysis by gavin et al.  <cit>  of their predicted complex components showed that attachment proteins were closely associated with core proteins within complexes and yet showed a greater degree of heterogeneity in expression levels, supporting the notion that attachments might represent non-stoichiometric components. also, attachment proteins were seen shared between two or more complexes, consistent with the view that the same protein may participate in multiple complexes  <cit> . on the other hand, the application of mcl to ppi networks yields clusters that do not share proteins . mapping these clusters back to the original ppi network shows that proteins having similar connectivities to multiple clusters are assigned arbitrarily to only one of the clusters. these proteins might as well be assigned to multiple clusters. to reflect these findings in our algorithm, we expect the attachment proteins to be those proteins within complexes that are:\n",
            "\n",
            "• non-core proteins;\n",
            "\n",
            "• closely interacting with the core proteins; and\n",
            "\n",
            "• may be shared across multiple complexes.\n",
            "\n",
            "we consider the following criteria to assign a non-core protein p belonging to a cluster cj  as an attachment in an acceptor cluster ci , that is, p ∈ attach:\n",
            "\n",
            "• protein p has sufficiently strong interactions with the core proteins core of the cluster ci;\n",
            "\n",
            "• the stronger the interactions among the core proteins, the stronger have to be the interactions of p with the core proteins;\n",
            "\n",
            "• for large core sets, strong interactions are required to only some of the core proteins or, alternatively, weaker interactions to most of them.\n",
            "\n",
            "combining these criteria, we assign non-core p as an attachment in the acceptor cluster ci, that is p ∈ attach, if:\n",
            "\n",
            "  ip≥α.ic.−γ, \n",
            "\n",
            "where ip = i) is the total weight of interactions of p with core, given by i) = ∑{w: q ∈ core}, while ic = i) is the total weight of interactions among the core proteins of ci, given by i)=12⋅∑{w : q, r∈core}, and sc = |core|, which is is normalized to yield  <dig> for core sets of size two. the parameters α and γ are used to control the effects of i ) and |core|. for a simple illustration, let α =  <dig>  and γ =  <dig>  and consider all interactions to be of equal weight  <dig>  therefore, p is attached to a core set of four proteins, if the total weight of its interactions with the core proteins is at least  <dig>  which is possible if p is connected to at least three core proteins . this step ensures that non-core proteins having sufficiently strong interactions with the cores in more than one clusters are recruited as attachments into all those clusters.\n",
            "\n",
            "extracting out complexes from clusters\n",
            "for each cluster we group together its constituent core and attachment proteins to define a unique complex. we expect all the remaining proteins within the cluster to have weaker associations with this resultant complex, and therefore categorize them as noisy proteins. in fact, experiments  <cit>  have shown that mcl clusters tend to include several such noisy proteins leading to reduction in accuracies of the clusters. therefore, our step ensures that such noisy proteins are discarded in order to extract out more accurate complexes. additionally, since these resulting complexes include attachment proteins that potentially may be recruited by multiple complexes, this step ensures that our predicted complexes adhere to the protein-sharing phenomenon observed in real complexes  <cit> . we discard all complexes of size less than  <dig> because many of these are false positives. it is difficult to predict small real complexes solely based on interaction  information .\n",
            "\n",
            "for each cluster ci, we define a unique complex cmplx as:\n",
            "\n",
            "  cmplx={core∪attach}. \n",
            "\n",
            "each interaction  among the constituent proteins p and q within this complex carries the weight w observed in the ppi network.\n",
            "\n",
            "ranking the predicted complexes\n",
            "as a final step, we output our predicted complexes in a reasonably meaningful order of biological significance. for this, we rank our predicted complexes in decreasing order of their weighted densities. the weighted density wd of a predicted complex c′i is given by  <cit> :\n",
            "\n",
            "  wd=∑p,q∈c′iw|ci′|⋅. \n",
            "\n",
            "the unweighted density of a predicted complex is defined in a similar way by setting the weights of all constituent interactions to  <dig>  this blindly favors very small complexes, or complexes with proteins having large number of interactions without considering the reliability of those interactions. on the other hand, the weighted density considers the reliability  of such interactions. if two complexes have the same unweighted density, the complex with higher weighted density is ranked higher.\n",
            "\n",
            "RESULTS\n",
            "preparation of experimental data\n",
            "we gathered high-confidence gavin and krogan-core interactions deposited in biogrid http://thebiogrid.org/ <cit>  . these were assembled from a combination of bait-prey and prey-prey relationships  observed by gavin et al.  <cit> , and the bait-prey relationships  observed by krogan et al.  <cit> . we combined these interactions to build the unscored gavin+krogan network . we then applied iterative-cdk  <cit>  and fs weightk  <cit>  scoring  on the gavin+krogan network, and selected all interactions with non-zero scores. this resulted in the icd and fsw networks, respectively. in addition to these two scored networks, we downloaded the consolidated <dig>  network  from http://interactome-cmp.ucsf.edu/, and the bootstrap <dig>  network  <cit>   from http://www.bio.ifi.lmu.de/complexes/procope/. the consolidated network was derived from the matrix modeled relationships of the original gavin and krogan datasets using the pe system  <cit> . therefore, this network comprised of additional prey-prey interactions that were missed in the krogan 'core' dataset. the bootstrap network was derived from the matrix modeled relationships using the bootstrapped scores  <cit> . table  <dig> summarizes some properties of these networks.\n",
            "\n",
            "the benchmark  set of complexes was built from hand-curated complexes derived from three sources:  <dig> complexes of the wodak lab cyc <dig> catalogue  <cit> ,  <dig> complexes of mips  <cit> , and  <dig> complexes curated by aloy et al.  <cit> . the properties of these reference sets are shown in table  <dig>  we considered each of these reference sets independently for the evaluation of mcl-caw. we did not merge them into one comprehensive list of complexes because the individual complex compositions are different across the three sources and some complexes may also get double-counted . an alternative strategy was adopted by wang et al.  <cit>  by integrating the complexes from three sources  using the jaccard score: two complexes overlapping with a jaccard score of at least  <dig>  were merged together - the proteins to be included into the resultant complex were chosen based on a voting scheme.\n",
            "\n",
            "to be accurate  while evaluating our method on these benchmark sets, we considered only the set of derivable benchmark complexes from each of the ppi networks: if a protein is not present in a ppi network, we remove it from the set of benchmark complexes. by repeated removals, if the size of a benchmark complex shrinks below  <dig>  we remove the complex from our benchmark set to generate the final set of derivable benchmark complexes for each of the ppi networks.\n",
            "\n",
            "in order to evaluate the biological coherence of our predicted complexes, we downloaded the list of cellular localizations  of proteins from gene ontology   <cit> . we selected only the informative go terms. a go term is informative if no less than  <dig> proteins are annotated with this term and none of its descendant terms are annotated to no less than  <dig> proteins  <cit> . the list of essential genes was obtained from the saccharomyces genome deletion project  <cit> : http://www-sequence.stanford.edu/group/yeast_deletion_project/deletions <dig> html\n",
            "\n",
            "evaluation metrics for matching predicted and benchmark complexes\n",
            "let b = {b <dig> b <dig> ...,bm} and c = {c <dig> c <dig> ...,cn} be the sets of benchmark and predicted complexes, respectively. we use the jaccard coefficient j to quantify the overlap between a benchmark complex bi and a predicted complex cj :\n",
            "\n",
            "  j=|bi∩cj||bi∫cj|. \n",
            "\n",
            "we consider bi to be covered by cj, if j ≥ overlap threshold t. in our experiments, we set the threshold t =  <dig> , which requires |bi∩cj|  ≥|bi|+|cj| <dig>  for example, if |bi| = |cj | =  <dig>  then the overlap between bi and cj should be at least  <dig> \n",
            "\n",
            "we use previously reported  <cit>  definitions of recall rc  and precision pr  of the set of predicted complexes:\n",
            "\n",
            "  rc=|{bi|bi∈b∧∃cj∈c;j≥t}||b| \n",
            "\n",
            "here, |{bi|bi ∈ b Λ ∃cj ∈ c; j ≥ t}| gives the number of derived benchmarks.\n",
            "\n",
            "  pr=|{cj|cj∈c∧∃bi∈b;j≥t}||c| \n",
            "\n",
            "here, |{cj |cj ∈ c Λ ∃bi ∈ b; j ≥ t}| gives the number of matched predictions.\n",
            "\n",
            "we also evaluate the performance of our method by plotting the precision versus recall curves for the predicted complexes. these curves are plotted by tuning a threshold on the number of predicted complexes considered for the evaluation. the predicted complexes are considered in decreasing order of their weighted densities .\n",
            "\n",
            "biological coherence of predicted complexes\n",
            "a complex can be formed if its proteins are localized within the same compartment of the cell. so, we use the localization coherence of the predicted complexes as a measure their quality. let l = {l <dig>  l <dig> ..., lk } be the set of known localization groups, where each li contains a set of proteins with similar localization annotations. the co-localization score ls of a predicted complex cj is defined as the maximal fraction of its constituent proteins that are co-localized within the same localization group among the proteins that have annotations. this is given as follows  <cit> :\n",
            "\n",
            "  ls=max{|cj∩li|:i= <dig> ,…,k}|p:p∈cj∧∃li∈l,p∈li|. \n",
            "\n",
            "therefore, the co-localization score ls for the set of predicted complexes c is just the weighted average over all complexes  <cit> :\n",
            "\n",
            "  ls=∑cj∈cmax{|cj∩li|:i= <dig> ,…,k}∑cj∈c|p:p∈cj∧∃li∈l,p∈li|. \n",
            "\n",
            "setting the parameters i, α and γ for mcl-caw\n",
            "before evaluating the performance of mcl-caw, we describe the procedure used for setting inflation parameter i for mcl, and α and γ for core-attachment refinement in order to determine a good combination of parameters for mcl-caw in practice. only the predicted complexes of size ≥  <dig> from mcl and mcl-caw were considered for setting the parameters as well as for further experiments. we used f <dig>  measured against the wodak lab  <cit> , mips  <cit>  and aloy  <cit>  benchmarks as our basis for choosing the best values for these parameters.\n",
            "\n",
            "we adopted the following four-step procedure for each ppi network:\n",
            "\n",
            " <dig>  run mcl for a range of i values and choose i that offers the best f <dig> measure;\n",
            "\n",
            " <dig>  set i to the chosen value, set a certain α for mcl-caw, and choose γ from a range of values that offers the best f <dig> measure;\n",
            "\n",
            " <dig>  set i and γ to the chosen values, and choose α for mcl-caw from a range of values that offers the best f <dig> measure;\n",
            "\n",
            " <dig>  set α and γ for mcl-caw to the chosen values, and reconfirm the value chosen for i.\n",
            "\n",
            "setting i for mcl\n",
            "inflation i in mcl determines the granularity of the clustering - the higher the value more finer are the clusters produced. typical values used for clustering ppi networks are i =  <dig>  and  <dig>   <cit> . for each ppi network, we ran mcl over a range of i , and measured f <dig> against the three benchmark sets. we then normalized these f <dig> values against the best f <dig> obtained on each benchmark, summed up these normalized f <dig> values across benchmarks, and finally normalized these sums to obtain a final ranking for the i values. the detailed calculations are presented in additional files  <dig>  tables s <dig> and s <dig>  in figure  <dig>  we show sample f <dig> versus i plots for the unscored gavin+krogan and scored icd networks for the range of i =  <dig>  to  <dig> . we noticed that inflation i =  <dig>  gave the best f <dig> on both unscored and scored networks. the f <dig> obtained at i =  <dig>  and  <dig>  was only marginally less than that at i =  <dig> .\n",
            "\n",
            "setting α and γ for ca refinement\n",
            "for each ppi network, we set i to the chosen value, fixed a certain α, and ran mcl-caw over a range of γ. we adopted the same method as above to choose the value of γ offering the best f <dig> measure. figure  <dig> shows sample f <dig> versus γ plots on the unscored gavin+krogan and scored icd networks for i =  <dig> , α =  <dig>  and γ =  <dig>  to  <dig> . the detailed calculations are presented in additional files  <dig>  table s <dig>  we noticed that γ =  <dig>  gave the best f <dig> on both unscored and scored networks.\n",
            "\n",
            "next, we set i and γ to the chosen values, and ran mcl-caw over a range of α. figure  <dig> shows sample f <dig> versus α plots on the unscored gavin+krogan and scored icd networks for i =  <dig> , = γ =  <dig>  and α =  <dig>  to  <dig> . the detailed calculations are presented in additional files  <dig>  table s <dig>  we noticed that α =  <dig>  gave the best f <dig> on the unscored network, while α =  <dig>  gave the best f <dig> on the scored networks.\n",
            "\n",
            "reconfirming i for the chosen values of α and γ\n",
            "finally, for each ppi network, we ran core-attachment refinement with the chosen values of α and γ over a range of i for mcl. figure  <dig> compares the f <dig> versus i plots for plain-mcl and mcl followed by ca refinement on the unscored gavin+krogan and scored icd networks for range i =  <dig>  to  <dig> . the plots reconfirmed that the chosen values for α and γ gave the best performance for ca refinement when i =  <dig>  . the detailed calculations are presented in additional files  <dig>  tables s <dig> and s <dig>  we settled on i =  <dig> , α =  <dig>  and γ =  <dig>  for the unscored gavin+krogan network, and i =  <dig> , α =  <dig>  and γ =  <dig>  for the scored networks as our final combination of parameters for mcl-caw.\n",
            "\n",
            "evaluating the performance of mcl-caw\n",
            " <dig>  after clustering using mcl;\n",
            "\n",
            " <dig>  after hierarchical clustering followed by core-attachment refinement using mcl-caw.\n",
            "\n",
            "the effect of core-attachment refinement on the predictions of mcl\n",
            "compare the topmost rows in table  <dig> for mcl and mcl-caw evaluated on the unscored gavin+krogan network. they show that mcl-caw achieved significantly higher recall compared to mcl on gavin+krogan - on an average 31% higher number of complexes derived than mcl. in fact referring back to figure  <dig>  mcl-caw achieved higher f <dig> compared to mcl for the entire range i =  <dig>  to  <dig> . in order to further analyse this improvement, we considered two sets of complexes derived from gavin+krogan.  set a = mcl ∩ mcl-caw, consisting of all complexes correctly predicted by both mcl and mcl-caw, but with different jaccard accuracies;  set b = mcl-caw\\mcl, consisting of all complexes correctly predicted by mcl-caw, but not by mcl. there was no complex correctly predicted by mcl that was missed by mcl-caw. we calculated the increase  in accuracies for complexes from a and b. this increase for a was noticably high, the average being  <dig> % on the wodak set. the increase for b was significantly high, the average being  <dig> % on the wodak set. this shows:  ca-refinement was successful in improving the accuracies of mcl clusters;  this improvement was particularly high for low quality clusters of mcl . mcl-caw was successful in elevating the accuracies above the threshold t =  <dig>  for those clusters that were difficult to be matched to known complexes using mcl alone. consequently, mcl-caw derived significantly higher number of benchmark complexes than mcl.\n",
            "\n",
            "affinity scoring of ppi networks improved the performance of mcl and mcl-caw. affinity scoring followed by ca refinement had a compounded effect in improving the performance of mcl.\n",
            "\n",
            "impact of noise on mcl and mcl-caw and the role of affinity scoring in reducing this impact\n",
            "among the four scored ppi networks, both mcl and mcl-caw showed best precision and recall on the consolidated <dig>  network, which can be directly attributed to the high quality of this network. however, this high quality of consolidated <dig>  came at the expense of lower protein coverage , resulting in reduced number of derivable complexes. in order to counter this, we gathered a larger subset of the consolidated network with pe cut-off  <dig>  , which accounted for a higher protein coverage . we noticed that the improvement of mcl-caw over mcl was significantly higher on consolidated <dig> , compared to the improvement seen on consolidated <dig> . we also noticed that icd scoring of consolidated <dig>  drastically reduced the size of this network, revealing that this larger subset in fact included significant amount of false positives . these experiments indicate that any reasonably good algorithm like mcl can perform well on high quality networks. however, due to the lack of protein coverage as well as scarcity of such high quality networks, we need to consider larger networks for complex detection . this in turn exposes the algorithms to higher amount of natural noise . therefore, the need is to develop algorithms that can detect larger number of complexes in the presence of such noise. in this scenario, our results show that mcl-caw is able to derive considerably higher number of complexes than mcl. taking this further, we introduced different levels of random noise to study its impact on mcl and mcl-caw. we introduced 10% to 75% random noise  to the gavin+krogan network. we noticed that mcl-caw performed better than mcl even upon introducing 50% random noise . however, at 75% random noise, the performance of mcl-caw marginally dropped below that of mcl. therefore, mcl-caw was reasonably robust to random noise - it was stable in the range 10% - 40% noise, which covers the typical levels of noise seen in tap/ms datasets  <cit>  . we next scored these noisy networks using the icd scheme. we found that the performance of both mcl and mcl-caw improved considerably on these scored networks. mcl-caw performed considerably better than mcl even at 50% to 75% random noise . therefore, affinity scoring helped mcl-caw to maintain its performance gain over mcl.\n",
            "\n",
            "the consolidated <dig>  and consolidated <dig>  networks were subsets of the consolidated network  <cit>  derived with pe cut-offs  <dig>  and  <dig> , respectively. we ran icd and fsw schemes on these networks. consolidated <dig>  had significant amount of false positives  that were discarded by the scoring. the performance of mcl-caw was only marginally better than mcl on consolidated <dig> , but mcl-caw performed considerably better than mcl on the \"more noisy\" consolidated <dig> .\n",
            "\n",
            "the gavin+krogan network was introduced with  <dig> -  <dig>  random interactions. following this, these noisy networks were scored using the icd scheme. with the aid of scoring, mcl-caw was able to perform better than mcl even at 50% random noise.\n",
            "\n",
            "biological coherence of predicted components\n",
            "the co-localization scores for the various predicted components  of mcl-caw are shown in table  <dig>  the table shows that:  the predicted complexes of mcl-caw showed high co-localization scores compared to mcl on both the unscored and scored ppi networks. mcl included several noisy proteins into the predicted clusters, thereby reducing their biological coherence;  the predicted cores of mcl-caw displayed higher scores compared to complexes, indicating that proteins within cores were highly localized;  the complexes of both mcl and mcl-caw displayed higher scores on the four scored networks compared to the gavin+krogan network.\n",
            "\n",
            "findings:  the complexes produced after ca-refinement showed higher scores than those of mcl;  the complexes predicted from the scored networks showed higher scores than from the gavin+krogan network;  the cores in mcl-caw showed higher scores than whole complexes.\n",
            "\n",
            "relative ranking of complex prediction algorithms and affinity-scored networks\n",
            "in order to gauge the performance of mcl-caw relative to existing techniques, we selected the following recent algorithms proposed for complex detection:\n",
            "\n",
            "• on the unscored gavin+krogan network, we compared against mcl  <cit> , our preliminary work mcl-ca   <cit> , core by leung et al.   <cit> , coach by wu min et al.   <cit> , cmc by liu et al.   <cit> , and haco by wang et al.   <cit> ;\n",
            "\n",
            "• on the affinity-scored networks, we compared against mcl, mcl incorporated with cluster overlaps by pu et al.   <cit>  , cmc and haco.\n",
            "\n",
            "parameters\n",
            "core , coach , mcl-ca  were compared against mcl-caw only on the unscored gavin+krogan network, while mcl , mclo , cmc  and haco  were evaluated also on the scored networks.\n",
            "\n",
            "tables  <dig>   <dig>   <dig>   <dig> and  <dig> show detailed comparisons between complex detection algorithms on the unscored and scored networks. figures  <dig> and  <dig> show the precision versus recall curves on these networks, while table  <dig> shows the area-under-the-curve  values for these curves. considering ± 5% error in auc values, the table shows that core attained the highest auc followed by mcl-caw and cmc on the unscored network, while mcl-caw and cmc achieved the overall highest auc on the scored networks. in addition to this, on each network we ranked the algorithms based on their normalized final f <dig> measures , as shown in table  <dig>  we summed up the normalized f <dig> values for each algorithm across all the networks to obtain an overall ranking of the algorithms as shown in table  <dig>  the detailed calculations are presented in additional files  <dig>  table s <dig>  on the unscored network cmc showed the best f <dig> value, while on the scored networks mcl-caw showed the best overall f <dig> value. in particular, mcl-caw performed the best on icd, fsw and consolidated <dig>  networks, while haco performed the best on bootstrap <dig> . this more or less agreed with the relative performance gathered from the auc values .\n",
            "\n",
            "wodak\n",
            "mips\n",
            "aloy\n",
            "methods considered: mcl, mcl-ca, mcl-caw, coach, core, cmc and haco. cmc performed the best in terms of precision, while haco and core performed the best in terms of recall. mcl-caw stood third among of the seven algorithms in both precision and recall. #matched: #predictions matching some benchmark complex. #derived: #benchmark complexes derived by some predicted complex.\n",
            "\n",
            "the unscored gavin+krogan network\n",
            "\n",
            "#proteins 2964; #interactions 13507\n",
            "\n",
            "wodak\n",
            "mips'\n",
            "aloy\n",
            "methods considered: mcl, mclo, mcl-caw, cmc and haco. haco performed the best in terms of precision, while cmc performed the best in terms of recall. mcl-caw was a close second in both precision and recall. #matched: #predictions matching some benchmark complex. #derived: #benchmark complexes derived by some predicted complex.\n",
            "\n",
            "the icd network\n",
            "\n",
            "#proteins 1628; #interactions 8707\n",
            "\n",
            "wodak\n",
            "mips\n",
            "aloy\n",
            "methods considered: mcl, mclo, mcl-caw, cmc and haco. haco performed the best in terms of precision, while mcl-caw and cmc performed the best in terms of recall. mcl-caw was a close second in terms of precision. #matched: #predictions matching some benchmark complex. #derived: #benchmark complexes derived by some predicted complex.\n",
            "\n",
            "the fsw network\n",
            "\n",
            "#proteins 1628; #interactions 8688\n",
            "\n",
            "wodak\n",
            "mips\n",
            "aloy\n",
            "methods considered: mcl, mclo, mcl-caw, cmc and haco. cmc performed the best in terms of precision, while mcl-caw performed the best in recall. #matched: #predictions matching some benchmark complex. #derived: #benchmark complexes derived by some predicted complex.\n",
            "\n",
            "the consolidated <dig>  network\n",
            "\n",
            "#proteins 1622; #interactions 9704\n",
            "\n",
            "wodak\n",
            "mips\n",
            "aloy\n",
            "methods considered: mcl, mclo, mcl-caw, cmc and haco. haco performed the best in terms of precision, while cmc performed the best in terms of recall. mcl-caw was positioned third in both precision and recall. #matched: #predictions matching some benchmark complex. #derived: #benchmark complexes derived by some predicted complex.\n",
            "\n",
            "the bootstrap <dig>  network\n",
            "\n",
            "#proteins 2719; #interactions 10290\n",
            "\n",
            "considering ± 5% error in auc values, core showed the highest value on unscored network, while mcl-caw and cmc showed the overall highest on the scored networks.\n",
            "\n",
            "ranking obtained from normalized f <dig> values. the g+k network is unscored, while the remaining are scored using affinity scoring schemes.\n",
            "\n",
            "ranking obtained from normalized f <dig> values.\n",
            "\n",
            "the precision of mcl-caw  was lower on bootstrap <dig>  compared to other scored networks . mcl-caw produced many redundant complexes from this network compared to other scored networks, leading to the drop in precision. in fact we observed such variance in cmc and haco algorithms as well. for example, cmc achieved the best recall on the icd network, but lowest on the consolidated network. also, cmc produced significantly fewer complexes  on the consolidated network compared to other networks . further, all algorithms displayed \"sudden dips\" in precision versus recall curves towards the beginning on the bootstrap <dig>  network . all these findings indicate that the choice of affinity scoring schemes affected the performance of algorithms. in other words, each algorithm made use of certain characteristics of the ppi networks, and favored a scoring scheme that magnified or reinforced those characteristics. there was no single algorithm which performed relatively best on all the scored networks. having said that, we note mcl-caw was ranked among the top three algorithms on all scored networks, and therefore mcl-caw responded reasonably well to the considered affinity scoring schemes.\n",
            "\n",
            "we also ranked the different affinity-scored networks based on the f <dig> measures offered to the complex detection algorithms, as shown in tables  <dig> and  <dig>  the table shows that the consolidated <dig>  network offered the best f <dig> measures to the algorithms, followed by the fsw, icd and bootstrap <dig>  networks . this agreed well with the fact that the consolidated <dig>  network was shown to have a tp/fp ratio comparable to small-scale experiments from mips, and therefore was of very high quality  <cit> .\n",
            "\n",
            "ranking obtained from normalized f <dig> values.\n",
            "\n",
            "ranking obtained from normalized f <dig> values.\n",
            "\n",
            "impact of augmenting physical ppi networks with computationally inferred interactions\n",
            "in this set of experiments, we studied whether augmenting the physical ppi networks with inferred interactions improved the performance of complex detection algorithms. we gathered interactions in yeast comprising of inferred interlogs , and also based on genetic  and functional  associations; downloaded from the predictome database  <cit> http://cagt.bu.edu/page/predictome_about. these were used to generate the inferred network . we then augmented the gavin+krogan network with these interactions to generate the gavin+krogan+inferred network and its scored versions, the icd and fsw networks .\n",
            "\n",
            "we evaluated mcl, mcl-caw, cmc and haco on these augmented networks . all the algorithms displayed very low precision and recall values on the inferred network, indicating that the inferred interactions alone were not sufficient to predict meaningful complexes. interestingly, most algorithms displayed marginal dip in their performance on gavin+krogan+inferred compared to gavin+krogan. this dip in performance was explained by the analysis on the two augmented-scored networks, icd and fsw. most algorithms showed higher precision and recall on these two augmented-scored networks compared to gavin+krogan and gavin+krogan+inferred. this indicates that augmenting with raw inferred interactions gave little benefit due to presence of false positives , but scoring the augmented networks helped to improve the precision and recall values of the algorithms.\n",
            "\n",
            "most algorithms showed marginal dip in performance on gavin+krogan+inferred compared to gavin+krogan. however, upon scoring the augmented network, their performance was better compared to gavin+krogan. this indicated that inferred interactions were useful for complex detection provided affinity scoring is employed to reduce the impact of the noise present in them.\n",
            "\n",
            "in-depth analysis of individual predicted complexes\n",
            "to facilitate the analysis of our individual predicted complexes, we mapped the complexes back to the corresponding ppi networks and examined the interactions between components of the same complex, as well as between components of a given complex and other proteins in the network. we performed this analysis using the cytoscape visualization environment http://www.cytoscape.org/ <cit> .\n",
            "\n",
            "instances of correctly predicted complexes of mcl-caw\n",
            "the first example is of an attachment protein shared between two predicted complexes of mcl-caw. the subunits of these predicted complexes  make up the compass complex involved in telomeric silencing of gene expression  <cit> , and the mrna cleavage and polyadenylation specificity factor, a complex involved in rnap ii transcription termination  <cit> . the shared attachment swd <dig>  formed high confidence connections with the subunits of both predicted complexes. on this basis, the post-processing procedure assigned swd <dig>  to both predicted complexes, in agreement with available evidence  <cit>  that swd <dig>  belongs to both compass and mrna cleavage complexes. the next example illustrates the case where a new protein was predicted as a subunit of a known complex. the attachment protein ski <dig>  was included into a predicted complex  that matched the exosome complex involved in rna processing and degradation  <cit> . additionally, ski <dig>  was also included into a prediction  matching the ski complex . however, the ski complex in the wodak lab catalogue  <cit>  did not include this new protein. further literature survey suggested that ski <dig> acts as a mediator between the ski and exosome complexes for 3'-to-5' mrna decay in yeast  <cit> .\n",
            "\n",
            "the rna polymerase i, ii, and iii complexes  are required for the generation of rna chains  <cit> . as per the wodak lab catalogue  <cit> , all the three complexes share subunits: yor224c, ybr154c, yor210w and ypr187w, while pol i and pol iii share ynl113w and ypr110c. due to the extensive sharing of subunits, the corresponding predictions were grouped together into one large cluster by mcl. on the other hand, mcl-caw segregated the large cluster into three independent complexes, which matched the pol i, pol ii and pol iii complexes with accuracies of  <dig> ,  <dig>  and  <dig> , respectively.\n",
            "\n",
            "in addition to these cases, a good fraction of already known core-attachment structures  were confirmed, and putative complexes were identified . some examples are worth quoting here. our predicted complex id#  <dig> closely matched the hops complex. all five cores {ylr148w, ylr396c, ymr231w, ypl045w, yal002w} and two attachments {ydr080w, ydl077c} that were covered matched those reported in gavin et al. biological experiments show that the cores have the function of vacuole protein sorting, and with the help of attachments, the complex can perform homotypic vacuole fusion  <cit> . we identified the ubiquitin ligase erad-l complex comprising of yos <dig>  hrd <dig> , usa <dig>  and hrd <dig>  that is involved in the degradation of er proteins  <cit> . this matched the hrd1/hrd <dig> purified by gavin et al. four subunits {oca <dig>  oca <dig>  siw <dig>  oca1} of a predicted novel complex  showed high similarity in functions  and localization  when verified in sgd  <cit> . this complex exactly matched the putative complex  <dig> in gavin et al.\n",
            "\n",
            "instances depicting mistakes in the predictions of mcl-caw\n",
            "here we discuss an interesting case in which the sharing of subunits was so extensive and the web of interactions was so dense that separating out the smaller subsumed complexes purely on the basis of the interaction information was much harder. it was the amalgamation of the clusters matching the saga, saga-like , ada and tfiid complexes. based on the wodak lab catalogue  <cit> , the  <dig> subunits making up the saga complex involved in transcriptional regulation  <cit>  include four subunits  that are members of the ada complex  <cit>  as well. sixteen components of the saga complex including the four shared with the ada complex, are also the components of the slik complex  <cit> . additionally, five subunits  of the saga complex also belong to the tfiid complex  <cit> . because of such extensive sharing of subunits involved in a dense web of interactions  network), mcl-caw was able to segregate out only two distinct complexes - saga  and slik . the clusters matching tfiid and ada remained amalgamated together. in the next set of analysis, we compared the derived complexes from the gavin+krogan and the icd networks, and identified cases where mcl-caw had missed a few proteins or whole complexes due to affinity scoring. from the wodak, mips and aloy reference sets, there were  <dig>   <dig> and  <dig> complexes, respectively, that were derived with better accuracies from the gavin+krogan network than from the icd network. and, there were  <dig>   <dig> and  <dig> complexes, respectively, that were derived from the gavin+krogan network, but missed totally from the icd network. table  <dig> shows a sample of such complexes from the wodak reference set. for the complexes that were derived with lower accuracies , mcl-caw had missed a few proteins due to low scores assigned to the corresponding interactions. for example, in the predicted complex from the icd network matching the swi/snf complex, two proteins  out of the four missed ones were absent due to their weak connections with the rest of the members; instead, these proteins were present in the prediction matching the rsc complex. in the gavin+krogan network, these two proteins were shared between two complexes matching the swi/snf and rsc complexes, which also agreed with the wodak catalogue  <cit> .\n",
            "\n",
            "the upper half shows sample complexes from wodak lab derived with lower accuracies from the icd network compared to those from the gavin+krogan network. the lower half shows those missed from the icd network. the #incorrect proteins in icd network is with respect to the benchmark complexes.\n",
            "\n",
            "in the cases where mcl-caw had completely missed some complexes from the scored network , it is interesting to note that mcl-caw had pulled-in many additional  proteins as attachments into the predicted complexes, which caused the accuracies to drop below  <dig> . one such case is of the predicted complex id# <dig> matching the eif <dig> complex with a low jaccard score of  <dig> . the eif <dig> complex from wodak lab consisted of  <dig> proteins: yor361c, ylr192c, ybr079c, ymr309c, ydr429c, ymr012w and ymr146c. the predicted complex id# <dig> from the gavin+krogan network consisted of  <dig> proteins :  <dig> cores  and  <dig> attachments . therefore, there were  <dig> missed and  <dig> additional proteins in the prediction, leading to an accuracy of  <dig> . the predicted complex id# <dig> from the icd network consisted of  <dig> proteins:  <dig> cores  and  <dig> attachments . therefore, there were  <dig> missed and  <dig> additional proteins in the prediction, leading to an even lower accuracy of  <dig> . all the core proteins had same or similar go annotations   <cit> . upon analysing the go annotations of the  <dig> attachment proteins, we noticed that only one  had the same annotation as the core proteins. this was also part of the eif <dig> complex from wodak lab  <cit> . out of the remaining  <dig> attachment proteins, five  had related go annotations  as the core proteins. a literature search revealed that these proteins belonged to the multi-eif initiation factor conglomerate  and the  <dig> s ribosomal subunit involved in translation  <cit> . the remaining two  were involved in translation activity, but were absent in the wodak lab catalogue. these might be potentially new proteins belonging to the eif <dig> or related complexes, and need to be further investigated. we also analysed the go annotations of the level- <dig> neighbors to the predicted complex seen in the network, none of them had annotations similar to the proteins within the network. this example illustrates that carefully incorporating go information into our algorithm to include or filter out proteins can be useful in cases where making decisions solely based on interaction information is difficult.\n",
            "\n",
            "correlation between essentiality of proteins and their ability to form complexes\n",
            "early works by jeong et al.  <cit>  and han et al.  <cit>  studied the essentialities of proteins based on pairwise interactions within the interaction network, and concluded that hub  proteins are more likely to be essential. this formed one of the criteria within the \"centrality-lethality\" rule  <cit> . however, a deeper insight can be obtained by studying the essentialities at cluster or group level of proteins rather than pairwise interactions. recently, zotenko et al.  <cit>  argued that essential proteins often group together into densely connected sets of proteins performing essential functions, and thereby get involved in higher number of interactions resulting in their hubness property. therefore, hubness may just an indirect indicator of protein essentiality. more recently, kang et al.  <cit>  studied essentiality of proteins by generating the reverse neighbor  topology  <cit>  out of protein networks. this topology groups those proteins together that are within the reverse neighborhood of a given protein. kang et al. concluded that centrality within the rnn topology is a better estimator of essentiality than hubness or degree in the interaction network. studies by hart et al.  <cit>  showed that essential proteins are concentrated only in certain complexes, resulting in a dichotomy of essential and non-essential complexes. wang et al.  <cit>  concluded that the size of the  recruiting complex of a protein may be a better indicator of protein essentiality than hubness.\n",
            "\n",
            "in our work, we attempt to understand the relationship between the essentiality of proteins and their ability to form complexes. table  <dig> shows that a high proportion , fsw , consolidated <dig>  and bootstrap <dig>  networks, respectively) of essential proteins present in the four affinity-scored networks belonged to at least some correctly predicted complex. this indicated that essential proteins are often members of complexes or co-clustered groups of proteins.\n",
            "\n",
            "the figures in brackets represent the proportion of essential genes present in the corresponding group out of the  <dig> total essential genes obtained from the yeast genome deletion project  <cit> .\n",
            "\n",
            "# essential genes in ydp: 1123\n",
            "\n",
            "to further analyse this ability of essential proteins to form complexes or groups, we binned our correctly predicted complexes based on their sizes and calculated the proportion of essential proteins in all complexes for each bin . figure  <dig> shows that essential proteins were present in higher proportions within larger complexes. we then calculated the proportion of essential proteins within the top k ranked complexes. figure  <dig> shows that essential proteins were present in higher proportions within higher ranked complexes. both these figures hint at the same finding: essential proteins come together in large groups to perform essential functions.\n",
            "\n",
            "discussion\n",
            "in spite of the advances in computational approaches to derive complexes, high-accuracy reconstruction of complexes has still remained a challenging task. in deriving protein complexes from ppi networks, a key assumption made by most computational approaches is that complexes form densely connected regions within the networks. therefore, these approaches attempt to cluster the networks based on measures related to connectivities between proteins in the network. some approaches like mcl simulate random walks  to identify dense regions, while others like cmc merge maximal cliques into larger dense clusters. therefore, the performance of these methods varies widely depending on network densities. a glance through tables  <dig> to  <dig> reveals that all the methods considered for comparison in this work achieve very low recall on the mips set compared to the wodak and aloy sets. table  <dig> shows that the average density of complexes in mips is much lower than that of wodak and aloy sets. only  <dig> out of  <dig>  derivable mips complexes of size ≥  <dig> could be detected from the gavin+krogan network by all methods put together. we analysed the remaining  <dig> mips complexes and found most of them to have very low densities  in the gavin+krogan network. for example, the mips complex  <dig> . <dig>  went undetected by all the methods even though  <dig> of its  <dig> proteins were present in the gavin+krogan network. there were  <dig> interactions among these  <dig> proteins, giving a low density of  <dig>  to the complex in this network. continuing with this analysis, we tested mcl and mcl-caw on a ppi dataset from dip http://dip.doe-mbi.ucla.edu, comprising of  <dig> interactions among  <dig> proteins giving a low average node degree of  <dig> . mcl-caw was able to achieve only marginal improvement  over mcl, due to the low average node degree of the dip network. these experiments show that all the methods considered here find it difficult to uncover complexes that are very sparse. this should prompt us to rethink whether over importance is being given to model complexes as dense regions in ppi networks.\n",
            "\n",
            "apart from these limitations in the existing computational methods, there are some inherent difficulties in the accumulation of interactome data as well that make complex detection difficult. complexes display different lifetimes, and their compositions vary based on cellular localizations  and conditions. the same protein may be recruited by different complexes at different times and conditions. due to such temporal and spatial variability of complexes, repeated purifications using tap/ms methods yield somewhat different \"complex forms\"  <cit> . the ppi networks constructed out of such purifications represent only a probabilistic average picture of the yeast interactome  <cit> . therefore, the complexes predicted out of such networks only approximate the actual complex compositions.\n",
            "\n",
            "another limitation arises from the bias in tap/ms purifications against complexes of certain kind . since tap/ms data are acquired in a single condition , some complexes may not be present in the cell in that condition  <cit> . therefore, new experimental assays are needed before such complexes can be reconstructed and studied.\n",
            "\n",
            "finally, even though s. cerevisiae is used as a model organism for eukaryotic interactome analysis, some key complexes specialized to other organisms  can be studied only by analysing the interaction datasets specific to these organisms. however, the incompleteness of interactome data from these organisms makes the reconstruction of complexes difficult.\n",
            "\n",
            "CONCLUSIONS\n",
            "the ultimate goal of interactome analysis is to understand the higher level organization of the cell. reconstruction of protein complexes serves as a building block towards achieving this goal. in this paper, inspired by the findings of gavin et al.  <cit> , we developed a novel core-attachment based refinement method coupled to mcl to identify yeast complexes from weighted ppi networks. we demonstrated that our algorithm  performed better than mcl in deriving meaningful yeast complexes particularly in the presence of natural noise. we also showed that mcl-caw responded reasonably well to the considered affinity scoring schemes. in the future work, we intend to improve the prediction ability of our algorithm by incorporating information from gene annotations, gene expressions, literature mining as well as domain-domain interactions. we also intend to extend our work to predict complexes of organisms other than yeast. in this context, we intend to use our mcl-caw model to study the existence  of core-attachment modularity in complexes from other organisms.\n",
            "\n",
            "availability\n",
            "the mcl-caw software is developed using pl/sql on oracle  <dig> g, using the framework in  <cit> . the source code, yeast ppi datasets, benchmark and predicted yeast complexes used in this work are freely available at the mcl-caw project homepage hosted on the nus server: http://www.comp.nus.edu.sg/~leonghw/mcl-caw/.\n",
            "\n",
            "authors' contributions\n",
            "ss conceived the initial ideas and discussed them with hwl and kn. ss devised the algorithm, developed the software, performed the experiments and analysis, and wrote and revised the manuscript. hwl supervised the project, advised ss, and reviewed and revised the manuscript. kn took part in the discussions and helped in reviewing the manuscript. all authors have read and approved the manuscript.\n",
            "\n",
            "supplementary material\n",
            "additional files 1\n",
            "additional figures and tables: figures for core-attachment modularity and illustration of a predicted complex by mcl-caw. tables for setting of mcl-caw parameters, and ranking of complex detection algorithms and affinity-scored networks.\n",
            "\n",
            "click here for file\n",
            "\n",
            " additional files 2\n",
            "the mcl-caw software package: the source code and installation details for the mcl-caw software.\n",
            "\n",
            "click here for file\n",
            "\n",
            " acknowledgements\n",
            "we would like to thank the editor as well as reviewers for their valuable comments and suggestions; sean collins  and caroline friedel  for making available the consolidated  <cit>  and the bootstrap  <cit>  networks, respectively; guimei liu  and limsoon wong  for the iterative-cd, fs weight and cmc softwares  <cit> ; henry leung  and wu min  for the core  <cit>  and coach  <cit>  softwares, respectively. this work was supported in part by the national university of singapore under arf grant r252-000-361- <dig> \n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "we found that mcl-caw performed better than mcl-caw in deriving complexes from weighted ppi networks, and that mcl-caw responded reasonably well to the considered affinity scoring schemes.\n",
            "in this paper, we develop a novel core-attachment based refinement method coupled to mcl to identify yeast complexes from weighted ppi networks.\n",
            "our results show that mcl-caw is able to derive considerably higher number of complexes than mcl-caw, and that mcl-caw responds reasonably well to the considered affinity scoring schemes.\n",
            "our results indicate that mcl-caw is able to derive significantly higher number of complexes than mcl-caw, and that mcl-caw responds reasonably well to the considered affinity scoring schemes.\n",
            "our results suggest that mcl-caw is able to derive significantly higher number of complexes than mcl-caw, and that mcl-caw responds reasonably well to the considered affinity scoring schemes.\n",
            "our results suggest that mcl-caw is able to derive significantly higher number of complexes than mcl-caw, and that mcl-caw responds reasonably well to the considered affinity scoring schemes.\n",
            "\n",
            "96\n",
            "[{'summary_text': 'diana-microt  <dig>  provides prediction scores which correlate with protein production fold change and may be used as an indication of the expected fold change in protein production.\\ndiana-microt  <dig>  achieves comparable precision to two other leading target prediction programs, targetscan and pictar.\\ndiana-microt  <dig>  provides prediction scores which correlate with protein production fold change and may be used as an indication of the expected fold change in protein production.\\ndiana-microt  <dig>  achieves comparable precision to two other leading target prediction programs, targetscans and pictar.\\ndiana-microt  <dig>  achieves comparable precision to two other leading target prediction programs, targetscans and targetscan.\\ndiana-microt  <dig>  provides prediction scores which correlate with protein production fold change and may be used as an indication of the expected fold change in protein production.\\ndiana-microt  <dig>  provides prediction scores which correlate with protein production fold change and may be used as an indication of the expected fold change in protein production.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 7088 to 7168 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "micrornas  are short, endogenously expressed rna molecules that regulate gene expression by binding directly and preferably to the 3' untranslated region  of protein coding genes  <cit> . each mirna is 19- <dig> nucleotides in length and is processed from a longer transcript which is referred to as the primary transcript . these transcripts are processed in the cell nucleus to short, 70-nucleotide stem-loop structures known as pre-mirnas. pre-mirnas are processed to mature mirnas in the cytoplasm by interaction with the endonuclease dicer which cleaves the pre-mirna stem-loop into two complementary short rna molecules. one of these molecules is integrated into the risc  complex and guides the whole complex to the mrna, thus inhibiting translation or inducing mrna degradation  <cit> . since their initial identification, mirnas have been found to confer a novel layer of genetic regulation in a wide range of biological processes. mirnas were first identified in  <dig>  <cit>  via classical genetic techniques in c. elegans, but it was not until  <dig> that they were found to be widespread and abundant in cells  <cit> . this finding served as the primary impetus for the development of the first computational mirna target prediction programs. diana-microt  <cit>  and targetscan  <cit>  were the first algorithms to predict mirna targets in humans, and led to the identification of an initial set of experimentally supported mammalian targets. such targets are now collected and reported in tarbase  <cit>  which contains more than one thousand entries for human and mouse mirnas.\n",
            "\n",
            "in the last years several groups suggested that the first nucleotides of a mirna sequence are crucial for recognizing and binding to the messenger of a protein. kiriakidou et al.  <cit>  showed the need for a nearly consecutive binding of the first  <dig> mirna nucleotides   to the 3'utr of protein coding genes in order to repress translation. a statistical approach by lewis et al.  <cit>  revealed that complementary motifs to nucleotides 2- <dig> of the mirna driver sequence  remain preferentially conserved in several species. typically, it is believed that a binding of at least  <dig> consecutive watson-crick  base pairing nucleotides between the mirna driver sequence and the mirna recognition element  is required for sufficient repression of protein production. however, experimental evidence  <cit>  show that weaker bindings, involving only six consecutively paired nucleotides or including imperfect bindings  may also confer protein repression although they might generally be less effective  <cit> . for this reason, mirna target prediction programs mostly rely on sequence alignment of the mirna seed region to the 3'utr sequences of candidate target genes in order to identify putative mirna binding sites. their specificity is usually increased by additionally assessing the commonly observed binding site evolutionary conservation or by using additional features such as binding site structural accessibility  <cit> , nucleotide composition flanking the binding sites  <cit>  or proximity of one binding site to another within the same 3' utr  <cit> .\n",
            "\n",
            "diana-microt  <dig> , the algorithm described here, utilizes the above mentioned features and categorizes as putative mres those sites that have seven, eight or nine nucleotide long consecutive wc base pairing with the mirna driver sequence, starting from position  <dig> or  <dig> of the 5'end of the mirna. for sites with additional base pairing involving the 3'end of the mirna, a single g:u wobble pair or binding of only  <dig> consecutive nucleotides to the driver sequence are allowed. briefly, the diana-microt  <dig>  algorithm consists of : a) alignment of the mirna driver sequence on the 3'utr of a protein coding gene, b) identification of putative mres based on specific binding rules, c) scoring of individual mres according to their binding type and conservation profile, d) calculation of an overall mirna target gene  score through the weighted sum of all mre scores lying on the 3'utr. the program is designed to use up to  <dig> different species to estimate mre conservation scores and combines both conserved and non-conserved mres in a final mitg score . the mitg score correlates with fold changes in protein expression. additionally, since the algorithm calculates all weights and scores independently for each mirna it allows for the calculation of signal to noise ratio  at different mitg score cut-offs providing precision scores which serve as an indication of the false positive rate of the predicted interactions.\n",
            "\n",
            "generally, mirnas can repress the expression of proteins in two ways: via mrna degradation or via repression of mrna translation. until recently, high throughput experiments were only able to measure mirna-mediated changes at the mrna level , allowing the characterization of only a subset of direct mirna targets  <cit> . however, recently two groups  <cit>  have independently developed methods to characterize mirna-mediated gene expression changes at both the mrna and the protein level. selbach et al.  <cit>  used microarrays and pulsed stable isotope labeling with amino acids in cell culture  assays to determine the genes targeted by each of five over-expressed mirnas in hela cells. using this set of experimentally supported targets the authors performed a comparative assessment of several target prediction programs. the benchmark revealed that the simplest prediction method involving the search for complementary sequences of the mirna seed region on the 3'utr of genes achieved a precision  of 44% while only three of the prediction programs  achieved significantly higher precision. pictar  <cit>  and targetscans  <cit>  achieved approximately 62% precision compared to diana-microt  <dig>  with approximately 66%.\n",
            "\n",
            "methods\n",
            "identification of putative mirna binding sites through sequence alignment\n",
            "the program identifies the highest scoring alignment between every nine nucleotide long window of the 3'utr with the mirna driver sequence using a dynamic programming algorithm. the alignment is based on the following binding rules. firstly, a minimum of six consecutive matches  or g:u) is required. if the six matches are w-c and the binding starts at position  <dig> or  <dig> of the mirna driver sequence, then the mre is considered a 6mer. a 7mer  has seven  consecutive w-c matches starting at position  <dig> or  <dig> of the mirna driver. a single g:u wobble pair is allowed as long as there are at least six w-c pairs, yielding 7mers, 8mers and 9mers, each with a wobble base pair.\n",
            "\n",
            "filter of putative mirna binding sites depending on binding energy\n",
            "for sites with less than  <dig> consecutive w-c matches  an additional energy filter is applied. using rnahybrid  <cit>  the algorithm estimates the free binding energy between the mirna sequence and the 3'utr sequence flanking the identified putative binding site and compares it to the perfect complement energy of the mirna. as \"perfect complement energy\" we denote the hypothetical energy of the perfect binding between the mirna sequence and its reverse complement sequence. therefore an imperfect site, in terms of alignment, is considered as mre only if the ratio of the free binding energy to the perfect complement energy is higher than a binding-category specific threshold. a threshold of  <dig>  is used for 9mers and 8mers containing a g:u wobble pair, and a threshold of  <dig>  is used for 7mers with a g:u wobble pair and 6mers. the energy thresholds have been calculated by comparing the predicted binding sites of the real mirna sequence versus the predicted binding sites of several shuffled mirna sequences. the shuffled mirna sequences are designed to have the same driver as the real mirna but a shuffled 3' end with the same nucleotide composition as the real mirna. the free binding energy ratio ei is defined as the ratio of the free binding energy between the mirna sequence and the 3'utr sequence flanking the identified putative binding site over the mirna perfect complement energy. additionally, nr is defined as the number of binding sites of the real mirnas that have energy ratios greater than ei and as ns the number of binding sites of the shuffled mirnas that have energy ratios greater than ei. the ratio r = nr/ns indicates how much more prevalent the free binding energy eifor real binding sites compared to the shuffled ones is. an example of the way this ratio r fluctuates is provided in figure  <dig>  for each binding category the energy thresholds have been chosen at the point where the ratio r becomes greater than  <dig> indicating that at this energy value one can generally find two times more real binding sites than random binding sites.\n",
            "\n",
            "mock mirnas\n",
            "mock mirnas are artificially produced mirna sequences which are independently created for each real mirna. these artificial mirna sequences are designed to have approximately the same number of predicted mres as the corresponding real mirna and are generated through the following procedure. initially, all 3'utr sequences are scanned for sites perfectly complementary to each possible  <dig> nucleotide long motif  excluding those motifs corresponding to positions 1- <dig>  2- <dig> and 3- <dig> of real mirnas. the  <dig> hexamers having the closest number of complementary sites to those of the seed of the real mirna are chosen. these hexamers are then used as the seed of each artificially created mock mirna. the remaining sequence of the mock mirnas is then produced by randomly shuffling the remaining nucleotides of the real mirna.\n",
            "\n",
            "mirna recognition elements score \n",
            "the identified mres are checked for sequence conservation in several species based on the sequence alignment of ortholog utrs. an mre x is considered conserved in species a if x can also be identified at the exact same position on the ortholog 3'utr sequence of species a. the conservation score cof an mre is defined as the number of species in which the mre is conserved. the mre score is calculated individually for each real mirna r, each binding category b and each conservation score c. analytically, for each binding category the number of mres nr, b of the real mirna and the number of mres mr, m, b of the corresponding mock mirnas with conservation score equal or greater than c are counted and the ratio of the two defines the mre score . the equation defining this procedure is  in which r is the index of the real mirna, b corresponds to the binding category, c defines the conservation score and m defines the index of the mock mirna from the set of mock mirnas corresponding to the real mirna r. in the described procedure the ratio is kept constant if nrb or mr, m, b/ <dig> become less than  <dig>  figure  <dig> shows an example of rrb for  <dig> binding categories at different mre conservation scores.\n",
            "\n",
            "mirna target gene score\n",
            "the scores of the mres identified on the same 3'utr are combined through a weighted sum to produce the final mitg score. the weights wb for each binding category b are calculated using  <dig> mirnas conserved in human, chimpanzee, mouse, rat, dog and chicken, by comparing them to  <dig> mock sequences . the analysis is similar to the calculation of the mre score explained previously but in this case the  <dig> mirnas are not treated independently but as a total. the ratio  for binding category b and conservation score c is calculated as  where nrb is the number of mres of the r real mirna categorized to binding category b and having a conservation score greater than c, mr, m, b represents the number of mres of the m mock mirna categorized to binding category b succeeding a conservation score greater than c and corresponding to real mirna r. as shown in figure  <dig> the weights for each binding category are estimated based on the slope of a fitted line. fitting is performed based on linear least squares approximation. for each binding category the weight is defined as w = slope/slope. for example, the weight for category \"8mer\" would be w8mer =  <dig> / <dig>  =  <dig> . except for \"9mer\", \"8mer\" and \"7mer\" the remaining categories do not differ significantly from the mock background and consequently in this analysis no specific weights are calculated for these categories. in order to approximate the estimated weights dwb based on the above analysis, each mre score is multiplied by a specific weight mwb which depends on the binding category of the mre .\n",
            "\n",
            "the binding weights estimated for each binding category and the weights used in diana-microt  <dig> .\n",
            "\n",
            "mitg score threshold assessment\n",
            "a common challenge among mirna target prediction programs is the decision on a score threshold that will reduce the number of misclassifications. here a set of  <dig> experimentally supported targets for  <dig> different human mirnas, provided by tarbase  <dig>   <cit> , has been used in order to determine a biologically meaningful score threshold. based on this dataset, an analysis was performed to test the capability of the algorithm to identify supported targets when increasing the mitg score threshold. as expected, the algorithm's capability reduces as the mitg score increases . however, there are two distinct mitg scores  with significantly higher performance reduction. for this reason, these mitg score values have been chosen as a loose and strict mitg score threshold respectively. however, users are still allowed to adjust the threshold at will to exchange between specificity and sensitivity levels.\n",
            "\n",
            "precision\n",
            "the precision of a prediction is defined as the ratio of correct positive predictions over all positive predictions . in the case of diana-microt  <dig> , the average number of mitgs for mock mirnas provides an estimation of the number of false positive targets predicted. therefore, the number of true positive predicted mitgs can be calculated by subtracting the average number of predicted mitgs for the mock mirnas from the total number of predicted mitgs for the real mirna. in detail, the precision for mirna r at mitg score s is calculated by  where wr is the number of mitgs of the r real mirna having mitg scores from s to s + Δs,  is the average number of mitgs of the mock mirnas corresponding to mirna r having mitg scores from s to s + Δs and Δs is a specified mitg score window .\n",
            "\n",
            "mirna sequences\n",
            "the human and mouse mirna sequences used by diana-microt  <dig>  have been downloaded from mirbase build  <dig>   <cit> .\n",
            "\n",
            "3'utr sequences\n",
            "the gene 3'utr sequences have been downloaded from ensembl, release  <dig>  <cit> . those 3'utr sequences that correspond to the same gene but to different gene transcripts have been filtered to keep only the longest 3'utr sequence.\n",
            "\n",
            "multiple alignment files \n",
            "the multiple genome alignment files have been downloaded from the ucsc genome browser  <cit> . the file used for human  is the alignment to  <dig> vertebrate genomes while for mouse   <dig> vertebrate genomes are used.\n",
            "\n",
            "RESULTS\n",
            "signal to noise ratio  assessment\n",
            "the signal to noise ratio for a prediction algorithm is typically used for the evaluation of its specificity. for diana-microt  <dig>  the overall snr is defined as the average signal to noise ratio calculated individually for each mirna. the individual mirna signal to noise ratio calculation is performed by dividing the number of predicted mitgs of a real mirna by the number of predicted mitgs for the set of corresponding mock mirnas. it is assumed that the predicted mitgs for the mock mirna sequences provide an unbiased estimate of the number of mitgs predicted by chance alone. analytically, the snr value of mirna r at mitg score s is calculated as . in this formula ngr refers to the number of mitgs of the real mirna r having mitg scores greater than s while mgr, m refers to the number of mitgs of the mock mirna m corresponding to the real mirna r having mitg score greater than s. figure  <dig> presents a graph of the snr for seven different mirnas. the overall snr calculation for diana-microt  <dig>  is performed on two different sets of mirnas. the first set consists of  <dig> mirnas conserved in  <dig> vertebrate species while the second set consists of  <dig> unique mirnas each one representing a mirna family with varying conservation levels. figure  <dig> shows the diagram for the number of predicted mitgs versus the mitg score. for an mitg score threshold that yields an average of approximately  <dig> predicted target genes per mirna, diana-microt  <dig>  achieves an overall snr of  <dig>  for the first dataset and an overall snr of  <dig>  for the second dataset which indicates that conserved mirnas tend to achieve higher snr values.\n",
            "\n",
            "receiver operating characteristics  analysis on proteomics data\n",
            "until recently a common difficulty in assessing the performance of a prediction algorithm was that the available experimental data could not easily distinguish between true and false targets. however, the recent study of selbach et al. provides both classes of targets allowing for the estimation of both the true positive rate as well as the false positive rate of a prediction. using a log <dig> fold change cutoff of - <dig>  to distinguish between targeted and non-targeted genes, the performance of diana-microt  <dig>  is assessed and presented as a roc curve .\n",
            "\n",
            "correlation of mitg score to the repression of protein production\n",
            "in the study by selbach et al <cit> , it was observed that there is a correlation between the log2-fold change of protein production with the number of occurrences of the hexamer corresponding to the seed of a mirna in the 3'utr of downregulated genes. when investigating the same data using diana-microt  <dig> , a similar correlation between the level of protein down-regulation and the predicted mitg scores, snr, and precision values is observed . interestingly, a linear regression analysis shows that the combination of mitg score, precision, snr, and the number of anti-seeds  as regressors provides the best accuracy in attempting to predict such fold changes in protein expression. figure 9b demonstrates the relationship between the protein expression fold change versus the number of occurrences of the mirna anti-seed  <dig>  as well as the protein expression fold change versus the combined regressor .\n",
            "\n",
            "discussion and \n",
            "CONCLUSIONS\n",
            "in the last five years more than two dozen mirna target prediction programs for mammalian genomes have been published  <cit> . using data from a high throughput experiment on five mirnas  <cit>  as a true-positive set of targets, it has been shown that diana-microt  <dig>  achieves comparable precision to two other leading target prediction programs, targetscans  <cit>  and pictar  <cit> . additionally, diana-microt  <dig>  provides prediction scores which correlate with protein production fold change and may be used as an indication of the expected fold change in protein production. the performance of the algorithm has been analyzed further by using a different set of supported mirna targets which has been extracted by the database of experimentally supported targets  <cit> . the results also indicate that the three programs  achieve similar precision levels . however, as shown in table  <dig> and  <dig> there are significant differences among the mitgs predicted by diana-microt  <dig>  and those predicted by each of the other programs. table  <dig> indicates that only 40% of the mitgs predicted by diana-microt  <dig>  are also predicted by pictar, and only 48% are predicted by targetscan  <dig> . this leaves in either case approximately 50% of the targets predicted only by diana-microt  <dig> .\n",
            "\n",
            "the table diagonal corresponds to the total number of mitgs predicted by each program for all the mirnas which are included in the set of experimentally verified targets. the number of mitgs predicted in common by each two target prediction programs is shown in the table. for example, targetscan and pictar have  <dig> predicted targets in common while diana-microt and pictar have  <dig> \n",
            "\n",
            "the percentage of each program's predicted targets  which are also predicted by another program  for all the mirnas which are included in the set of experimentally verified targets. for example, from the mitgs predicted by diana-microt  <dig> ,  <dig> % are also predicted by pictar and  <dig> % by targetscan  <dig> .\n",
            "\n",
            "recently, the rapid growth in the discovery rate of novel mirna sequences due to extensive usage of deep sequencing technology  <cit> , and the fact that mirnas have been shown to undergo a-to-i rna editing  <cit>  have underlined the need for a web based program which would allow for mirna target predictions based on user defined mirna sequences. diana-microt  <dig>  is one of the few programs offering such a service, supporting the scientific community with a tool which in total can be extensively used for the analysis of mirna dependent processes. this tool can be accessed thought the diana-microt  <cit>  web server at  which includes an optimized prediction algorithm that provides several features, combined with a user friendly interface which assists in the identification of interactions of interest.\n",
            "\n",
            "as already mentioned, diana-microt  <dig>  takes into account both conserved and not conserved mres. this attribute provides the algorithm with a highly important capability to predict targets of viral mirna sequences. generally, targets of viral mirnas are not expected to be conserved and this limits the ability of algorithms dependent on conservation to identify them. however, since diana-microt  <dig>  algorithm accepts non conserved mres it can successfully cope with viral mirna sequences.\n",
            "\n",
            "authors' contributions\n",
            "mm and pa designed and developed the algorithm, performed the statistical analysis and drafted the paper. glp contributed in the algorithm's implementation. mr participated in the algorithm's design and drafted the paper. td, gg , tv and ts participated in the design and implementation of the web server database. gg , ek, kk, nk, pt participated in the implementation of the algorithm's parallelization and contributed in the development of the online execution of the algorithm. vas contributed in the web server design and development. ps helped to draft the paper and participated in the early development of the algorithm. agh conceived of the study, and participated in its design and coordination and helped to draft the manuscript. all authors read and approved the final manuscript.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "diana-microt  <dig>  provides prediction scores which correlate with protein production fold change and may be used as an indication of the expected fold change in protein production.\n",
            "diana-microt  <dig>  achieves comparable precision to two other leading target prediction programs, targetscan and pictar.\n",
            "diana-microt  <dig>  provides prediction scores which correlate with protein production fold change and may be used as an indication of the expected fold change in protein production.\n",
            "diana-microt  <dig>  achieves comparable precision to two other leading target prediction programs, targetscans and pictar.\n",
            "diana-microt  <dig>  achieves comparable precision to two other leading target prediction programs, targetscans and targetscan.\n",
            "diana-microt  <dig>  provides prediction scores which correlate with protein production fold change and may be used as an indication of the expected fold change in protein production.\n",
            "diana-microt  <dig>  provides prediction scores which correlate with protein production fold change and may be used as an indication of the expected fold change in protein production.\n",
            "\n",
            "97\n",
            "[{'summary_text': 'the b. cenocepacia j <dig> genome consists of a large  <dig> \\xa0mb multi-replicon genome with a high average gc-content of  <dig> \\xa0%.\\nthe b. cenocepacia j <dig> genome contains a large  <dig> \\xa0mb multi-replicon genome with a high average gc-content of  <dig> \\xa0%.\\nthe b. cenocepacia j <dig> genome contains a large  <dig> \\xa0mb multi-replicon genome with a high average gc-content of  <dig> \\xa0%.\\nthe b. cenocepacia j <dig> genome contains a large  <dig> \\xa0mb multi-replicon genome with a high average gc-content of  <dig> \\xa0%.\\nthe b. cenocepacia j <dig> genome contains a large  <dig> \\xa0mb multi-replicon genome with a high average gc-content of  <dig> \\xa0%.\\nthe b. cenocepacia j <dig> genome contains a large  <dig> \\xa0mb multi-replicon genome'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 6579 to 7168 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "burkholderia cenocepacia j <dig> is a member of the burkholderia cepacia complex , a group of  <dig> species of closely related gram-negative betaproteobacteria  <cit>  which occur in the soil rhizosphere and also play an important role as opportunistic pathogens in humans . bcc bacteria are intrinsically resistant to most antibiotics, and infections with bcc bacteria are therefore difficult to treat. bcc bacteria are also able to form biofilms, further adding to their recalcitrance to antibiotic treatment  <cit> .\n",
            "\n",
            "b. cenocepacia j <dig> harbours a large  <dig>  mb multi-replicon genome with a high average gc-content of  <dig>  %. the genome consists of two large replicons of  <dig>  and  <dig>  mb, a smaller replicon  <dig>  mb and a plasmid  <dig>  mb in size, with  <dig> annotated protein coding and  <dig> annotated rna genes  <cit> , including  <dig> trnas and  <dig> riboswitches. however, transcription start sites , 5′ untranslated regions  of annotated genes and regulatory non-coding small rnas have not yet been comprehensively analysed and annotated. emerging new rna sequencing techniques, notably differential rna sequencing , make it now possible to precisely map the transcription start sites over a whole genome, and at the same time discover novel genome features.\n",
            "\n",
            "primary transcripts of prokaryotes carry a triphosphate at their 5′-end, whereas 5′-ends derived from processing and degradation carry a monophosphate. the drna-seq approach uses the properties of a 5′-monophosphate-dependent exonuclease  to selectively degrade processed transcripts, thereby enriching for un-processed rna species carrying a native 5′-triphosphate. tss can then be identified by comparing tex-treated with untreated rna-seq libraries, as they appear as localised maxima in coverage enriched by tex-treatment  <cit> .\n",
            "\n",
            "drna-seq enables precise mapping of 5′ ends of transcripts, whereas coverage over the whole transcript length is usually poor and 3′ end of transcripts are only represented for short transcripts. for this reason the drna-seq datasets were compared to conventional global rna-seq data  which provide more even coverage and a more comprehensive representation of full length transcription units. this approach aids in evaluating the function of an identified tss, particularly for tss internal to genes.\n",
            "\n",
            "the aim of the present study is to identify genes expressed in b. cenocepacia biofilms and detect the regulatory elements that might be involved in biofilm formation and survival, as a prerequisite to develop new strategies in treatment of b. cenocepacia infections.\n",
            "\n",
            "RESULTS\n",
            "tss annotation\n",
            "drna-seq of duplicate biofilm-derived rna samples resulted in datasets with  <dig> – <dig>  million mapped reads, grna-seq of triplicate biofilm-derived rna samples resulted in datasets with 23– <dig> million mapped reads . a total of  <dig> tss were automatically annotated based on the drna-seq data , evenly distributed on forward and reverse strands.  <dig> tss remained after noise filtering on a minimum of  <dig> read starts . these were then categorised according to their position in relation to annotated genes : tss in intergenic regions, located ≤ 300 nt upstream of the start of and in sense with an annotated gene, were assigned primary tss  for the respective gene. tss internal to annotated genes were assigned internal sense  or antisense . tss in intergenic regions and not associated with any gene were assigned “orphan” . where tss were positioned within 100 nt of and same sense to a primary or orphan tss, they were designated secondary .table  <dig> number of transcription start sites by category\n",
            "\n",
            "fig.  <dig> categorisation of transcription start sites. panel a: initial tss categorisation into primary , secondary , internal sense  and internal antisense  based on differential rna sequencing. panel b: comparison of drna-seq and grna-seq data for genes bcal0038- <dig>  visualised with the integrated genome browser  <cit> . drna-seq data are represented as read starts per base, with a vertical scale of  <dig> for forward reads and  <dig> for reverse reads. grna-seq data are represented as coverage, with a vertical scale of  <dig> for forward reads and  <dig> for reverse reads. panel c: re-categorisation of internal tss guided by leading edges of transcription identified by global rna sequencing\n",
            "\n",
            "\n",
            "\n",
            "tss loci were also compared to the global transcriptome datasets by manual inspection. more than 90 % of intergenic ptss were accompanied by an abrupt increase in coverage in the grna-seq dataset , substantiating that they are bona fide loci for transcription initiation. for the purpose of differentiation from tss based on drna-seq data, we designate these abrupt increases in grna-seq data coverage “leading edges of transcription”  for the rest of the manuscript.\n",
            "\n",
            "les found internal to genes were used to assign function to internal tss: where internal tss were associated with les and positioned ≤300 nt upstream or ≤100 nt downstream and in sense with a gene lacking an intergenic ptss, they were re-assigned ptss for the respective gene .\n",
            "\n",
            "transcription can be primed by molecules other than ntps, e.g. nano rrnas  <cit> . the resulting primary transcripts do not carry a triphosphate at their 5′-end and are depleted by tex-treatment. where depleted local read start maxima coincided with a distinct le, they were reported as depleted ptss , adding another  <dig> ptss to the dataset.\n",
            "\n",
            "in total  <dig> ptss were annotated over the whole b. cenocepacia genome for genes transcribed under biofilm conditions, representing 28 % of all annotated genes. this proportion appears realistic when comparing it to values found in similar studies , and considering that the large b. cenocepacia genome consists of a high number of non-essential genes  <cit>  that are not all transcribed in the one growth condition analysed in the present study. most ptss were located on the large replicon , which is to be expected since the large replicon harbours most essential genes  <cit> . intergenic ptss for three genes  were confirmed by 5′race .\n",
            "\n",
            " <dig> ptss were located internal to an upstream gene , in some cases the upstream gene was part of the same operon . two internal ptss located in an upstream same sense gene, one with and one without le, were analysed by 5′race . where a distinct le was present, the ptss was unambiguously confirmed, indicating that these adjacent genes do not constitute an operon. where a le was not apparent, transcription initiation as well as read-through from further upstream in the operon occurred. this confirms previous observations that transcription can be initiated or modulated at several loci within an operon, resulting in full length transcripts and alternative transcripts  <cit> .\n",
            "\n",
            "promoters\n",
            "as the present dataset is derived from analysis of only one condition, biofilm growth, promoter search focussed on the core promoter region with its − <dig> and − <dig> elements.\n",
            "\n",
            "sequences 60 nt upstream of ptss, excluding tss located in genomic islands, were submitted to improbizer  <cit> , a motif finding algorithm that considers location of sequence patterns within the input sequences and favours motifs that occur at the same place. improbizer found  <dig> motifs , the first two of which were plausible candidates for a − <dig> and − <dig> box based on their sequence and their position relative to tss. of the  <dig> analysed upstream sequences, more than 95 % possessed a 9 nt long at-rich motif, on average at position − <dig> to − <dig> relative to the tss . the more conserved part of this motif, with consensus sequence tanaat, is very similar to the conserved − <dig> hexamer of e. coli with consensus sequence tataat, regarding sequence and position relative to tss  <cit> . a second, less conserved motif was found in 93 % of submitted sequences, it centred at position − <dig> . its consensus sequence is ttgcc, making it similar to the conserved − <dig> box of e. coli  <cit>  with consensus sequence ttgaca.fig.  <dig> sequence logos showing conserved motifs upstream of ptss. motifs are based on  <dig> sequences upstream of primary tss located in intergenic regions, analysed with improbizer  <cit>  , meme  <cit>   and dminda  <cit>  panel . x-axis: average position relative to tss. y-axis: sequence conservation\n",
            "\n",
            "\n",
            "\n",
            "the same  <dig> upstream sequences were also analysed with meme  <cit> , and dminda  <cit> , confirming the more conserved first motif, with the same conserved part of the core promoter region containing the − <dig> consensus sequence tanaat and the same positioning relative to tss . the second less conserved motif could not be confirmed by either meme or dminda, presumably because it is too weak to be detected by algorithms which do not take the position of the motif into account.\n",
            "\n",
            "the sequences up to 60 nt upstream of internal ptss, astss, istss and otss were screened for occurrence of the conserved and at-rich motif  to assess whether these tss were derived from genuine transcription initiation or from sequencing artefacts. we used motif finder  <cit> , a program which considers the location of the motif in query sequences when searching for matches. more than 95 % of internal ptss were associated with a sequence match to the at-rich motif, on average at the same position as the input motif . furthermore, 94 % of astss, 92 % of otss and 81 % of istss were associated with a matching motif in the same position.\n",
            "\n",
            "overall, the occurrence and position of promoters for transcription initiation further corroborates that most tss found by drna-seq are bona fide. the lower incidence of promoters for istss indicates that istss can be also caused by tex inhibition at strong secondary structures, as has been observed for streptomyces coelicolor, another organism with high genomic gc content  <cit> .\n",
            "\n",
            "meme detected only  <dig> motifs with an e-value < <dig> , only one of which, containing the proposed −10-box , was plausible as a promoter motif based on its conservation and convergence towards a specific position . dminda detected  <dig> motifs, only two of which, both at-rich, were converging towards a specific position . variations of motifs which could represent different sigma factor binding sites were not found in this analysis. repeating the analysis with adjusted input paratmeters did not improve results. this is probably due to the relatively large number of input sequences . analysing subsets of these sequences, generated based on similar expression patterns in a microarray dataset  <cit>  or on related functions, also did not result in plausible specific and conserved sigma factor binding site motifs . a reason for this might be the large number of sigma factors encoded in the b. cenocepacia j <dig> genome  <cit> . this bacterium possesses  <dig> sigma factors the target genes of which have not yet been characterised and which probably have overlapping target gene populations. a more in depth analysis of promoter sequences might therefore require experimental evidence regarding sigma factor target genes, generated e.g. by chip sequencing.\n",
            "\n",
            "length of 5′ utrs and leaderless transcripts\n",
            "the average length of 5′utrs is 72 nt , with a distribution peak between  <dig> and 30 nt; 75 % of 5′utrs were between  <dig> and 126 nt long. this is in good agreement with values for other bacteria such as salmonella enterica  <cit> , helicobacter pylori  <cit>  and streptomyces coelicolor  <cit> .fig.  <dig> 5′utr length distribution. data represents 5′utrs from primary tss located in intergenic regions, excluding internal primary tss\n",
            "\n",
            "\n",
            "\n",
            "the length of a 5′utr can be related to expression regulation of the corresponding gene. long 5′utrs may contain riboswitches or provide binding sites for small regulatory rnas  <cit> . leaderless genes are translated by a different mechanism than genes with a leader sequence, and have been shown to be differentially regulated under stress conditions compared to leader-lead genes  <cit> . to investigate a possible link between length of 5′utrs and gene function in b. cenocepacia j <dig>  we performed functional enrichment analysis on sub-sets of genes, genes without  5′utr or with a long  5′utr.\n",
            "\n",
            "the ptss of  <dig> genes were located exactly at the annotated start codon and the ptss of a further  <dig> genes was located ≤10 nt upstream of the annotated start codons. these transcripts were considered to be leaderless.  <dig> leaderless transcripts were trnas, consistent with the length of trna leader sequences in b. cenocepacia j <dig>  which ranges from  <dig> to 127 nt . functional enrichment analysis of the remaining leaderless transcripts revealed that, with  <dig> coding sequences , transcriptional regulators of various families are particularly over-represented . the ta-rich promoter motif was found directly upstream of genes with leaderless transcripts , like reported for other bacteria  <cit> , showing that leaderless genes possess a transcription initiation signal instead of a shine-dalgarno sequence.\n",
            "\n",
            " <dig> cds featured a long 5′utr of >150 nt. functional enrichment analysis performed on these genes revealed transcriptional regulators, nucleotide binding and membrane proteins as over-represented . comparison of the respective sequences with the rfam database revealed the yet unannotated s-adenosyl-l-homocysteine  riboswitch  <cit>  in the 5′utr of an adenosylhomocysteinase .\n",
            "\n",
            "transcription initiation in genomic islands\n",
            "the genome of b. cenocepacia j <dig> contains  <dig> genomic islands with a gc-content lower than genome average of  <dig>  % or with cds similar to prophages; these gi encompass  <dig>  % of the total genome  <cit> . internal tss appear to occur at a higher density in genomic islands , in agreement with observations made in e. coli and salmonella sp.  <cit> . 18 % of all annotated istss and 25 % of astss are located in genomic islands, which is higher than expected given the proportion of genomic islands on the genome.fig.  <dig> distribution of internal tss across replicon  <dig> of the b. cenocepacia j <dig> genome. data represents numbers of istss and astss within 10000 bp sliding windows and 5000 bp steps. panel a: number of internal tss across the whole replicon  <dig>  panel b: detail for genome positions  <dig> ,000– <dig> , <dig> of replicon  <dig>  including two genomic islands with a gc-content lower than the genome average. position of genomic islands: bcengi <dig>  <dig> . <dig>  bcengi <dig>  <dig> . <dig>  the window size for displaying the gc-content is  <dig>  the horizontal line depicts the genome average of  <dig>  %\n",
            "\n",
            "\n",
            "\n",
            "this indicates that, while genes in genomic islands can be transcribed, these loci are not completely compatible with the b. cenocepacia j <dig> transcriptional machinery, and transcription initiation often does not result in a functional product  <cit> .\n",
            "\n",
            "interestingly, the genomic region encoding exopolysaccharide synthesis genes implicated in capsule formation  also shows a higher density of internal tss, thus confirming that this region was acquired by recent gene transfer and constitutes a genomic island  <cit> . moreover, most tss annotated on the plasmid were categorised as internal tss, indicating that plasmid genes are also not entirely compatible with b. cenocepacia transcription mechanisms. we propose that the distribution of internal tss across the b. cenocepacia genome can be used as a further criterion to identify genomic islands in b. cenocepacia, as genomic islands can display a higher density of internal tss than genome background.\n",
            "\n",
            "re-annotation of start codons and discovery of un-annotated proteins\n",
            "gene calling for genome annotation usually predicts up to 99 % of all protein coding genes correctly, but the rate of gene calling errors can rise to 14 % in the gc-rich burkholderia sp. genomes  <cit> . we used tss mapping to screen for mis-annotated and un-annotated genes in the b. cenocepacia  <dig> genome.\n",
            "\n",
            "for  <dig> genes,  <dig> cds and  <dig> rnas, tss mapping predicted the primary tss internal and downstream of the annotated gene start, suggesting that an incorrect start codon might have been predicted and the gene is shorter than annotated. the internal tss position for bcal <dig> was confirmed by 5′race  as an example. all genes with internal and downstream ptss, as well as leaderless genes, were screened for alternative start codons downstream of their annotated gene start. unusually long 5′utrs on the other hand could indicate that the corresponding gene is longer than annotated. 5′utrs longer than 150 nt, as well as orphan tss consistent with a 5′utr according to grna-seq data, were screened for open reading frames and upstream alternative start codons.  <dig> cds for which an alternative start codon could be predicted, making the gene either longer or shorter, are listed in table  <dig> table  <dig> alternative start codons for cds with internal tss or long 5′ utrs ≥ 150 nt, as predicted by prodigal\n",
            "\n",
            "\n",
            "\n",
            "to search the genome for un-annotated protein-coding genes, all otss and astss possessing a le were screened for an open reading frame with an atg start codon which could produce a protein ≥ <dig> amino acid residues. these amino acid sequences were compared to the ncbi protein sequence database and hits with both >75 % query coverage and >40 % amino acid identity were retained . most hits were annotated as hypothetical or conserved hypothetical proteins with no predicted function. in one case, a type ii toxin-antitoxin module was discovered on the opposite strand of a gene currently annotated as bcal <dig>  a conserved hypothetical protein. we propose to re-annotate this loci, to bcal1704a, a pard-type antitoxin with  <dig> amino acid residues, and bcal1704b, a pare-type toxin with  <dig> amino acid residues .fig.  <dig> re-annotation of a conserved hypothetical protein as a type ii toxin-antitoxin system. new annotation: bcal1704a, pard, antitoxin component of, and bcal1704b, pare, toxin component of a type ii toxin-antitoxin system. position of pard:  <dig> . <dig>  position of pare:  <dig> . <dig>  the start of pare overlaps the 3′end of pard by 8 nt. drna-seq data are represented as read starts per base, grna-seq data are represented as coverage\n",
            "\n",
            "\n",
            "\n",
            "antisense transcription\n",
            "most reads mapping to annotated genome features map in sense direction , only 5–7 % map antisense, based on the grna-seq dataset. nonetheless, antisense transcription is pervasive in b. cenocepacia j <dig>  and reads map antisense to nearly all genome features . however, it is safe to assume that not all of these antisense transcripts have a function  <cit> . genes with strong sense transcription tend to have a high number of antisense reads, likely a result of technical rna sequencing artefacts. moreover, antisense transcription can be a result of read-through from the 3′ end of a downstream opposite-sense gene or transcription initiation for an upstream opposite-sense gene.\n",
            "\n",
            "we attempted to annotate antisense transcripts that might genuinely be involved in gene regulation, i.e. transcripts complementary to at least part of a gene and not belonging to any category mentioned above. for this purpose we filtered for genes with a minimum antisense-rpkm of  <dig> and a ratio of antisense-to-sense rpkm of > <dig> , leaving 11 % of all genes.  <dig> of these featured an astss, or an otss located ≤ 300 bp downstream of the 3′end of a gene, with a le in grna-seq data . among the genes with antisense transcription were  <dig> transposases,  <dig> transcriptional regulators and two toxin-antitoxin systems, the same categories for which functional antisense transcripts have been found in other bacteria  <cit> . these antisense rnas might therefore also have a function in b. cenocepacia.\n",
            "\n",
            "genes induced under biofilm condition\n",
            "to screen the rna-seq data for genes potentially essential for biofilm growth of b. cenocepacia j <dig>  we compared rna-seq data with published microarray datasets obtained from cells grown in a biofilm  <cit> , from planktonic cells harvested in stationary phase, and from cells grown under reduced oxygen levels  <cit>  as well as under various stress conditions  <cit> . the aim was to find genes with high expression in biofilms  and induced in biofilms while not induced under any other condition.\n",
            "\n",
            "only the first two genes of a multi-subunit k+-transport system  met these criteria. these five genes are organised in two operons with two annotated tss, the first of which is 100-fold stronger expressed than the second . the first operon contains the structural k+-transport and atpase genes, the second contains the two-component regulatory system required for induction of kdpabc. qpcr analysis of kdpa confirmed its induction in biofilms compared to planktonic cultures in logarithmic and stationary phase . in e. coli, the kdp-system is an inducible high affinity k+-pump essential for intracellular k+-homoeostasis under salt stress,  <cit> . the two-component system kdpde has also been implicated in virulence in various pathogenic bacteria  <cit> . in bacillus sp., the kdp-system was found to be up-regulated in swarming cells  <cit>  and necessary for biofilm formation  <cit> , and the kdp-system was up-regulated in staphylococcus aureus biofilms  <cit> . these observations indicate that, apart from its role in osmoadaptation, the kdp-system also plays an important role in biofilm formation, presumably also in b. cenocepacia. work to characterise the relevance of these genes for b. cenocepacia biofilm formation and persistence, using deletion and conditional mutants, is ongoing.fig.  <dig> expression of the kdp-system in b. cenocepacia j <dig> biofilms. panel a: drna-seq and grna-seq data for the kdp-operon. drna-seq data are represented as read starts per base, grna-seq data are represented as coverage. red arrows: ptss, at position  <dig>  and  <dig> . panel b: quantitative rt-pcr of kdpa in biofilms and planktonic cultures. bars represent expression relative to planktonic logarithmic cultures and are based on three biological replicates. orange: planktonic culture in logarithmic phase, harvested at  <dig> × 108 cfu/ml. light blue: planktonic culture in late logarithmic phase, harvested at  <dig> × 109 cfu/ml. brown: planktonic culture in stationary phase, harvested at  <dig>  × 109 cfu/ml. green: biofilm grown in microtiter plates\n",
            "\n",
            "\n",
            "\n",
            "transcription of annotated non-coding rnas\n",
            "the published genome annotation  <cit>  includes rrnas and small non-coding rnas such as trnas, several riboswitches and essential rnas.\n",
            "\n",
            "for  <dig> of the  <dig> annotated trna genes a tss was found, positioned 5– <dig> nucleotides upstream of the annotated gene start. most trna transcripts are, as expected, processed at position + <dig> of the annotated gene, marked by a local maximum in read starts in the -tex library and depletion in the + tex library.\n",
            "\n",
            "exceptions are the trnas for histidine and selenocysteine, which are processed at position − <dig> and − <dig>  respectively. 16s rrna genes have a tss at position − <dig>  the co-transcribed 23s and 5s rrna are processed at + <dig> and + <dig>  respectively. tss for annotated riboswitches are positioned at annotated gene start  and 5 nt  or 8–14 nt upstream .\n",
            "\n",
            "of the annotated essential rnas, transfer messenger rna has an 11 bp 5′-leader element. rnase p appears longer than annotated, its tss is located at position − <dig> and the transcript shows no obvious processing site. the signal recognition particle appears shorter than annotated, with a tss at position + <dig> \n",
            "\n",
            "overall, drna-seq data are in good agreement with annotated non-coding rnas, showing that small rnas are detected with our experimental approach.\n",
            "\n",
            "candidate regulatory small rnas\n",
            "apart from annotated trnas, riboswitches and conserved essential rnas, bacterial genomes also contain other small non-coding rnas that are involved in post transcriptional gene expression regulation  <cit> . for a preliminary screening of tss for highly expressed novel small rnas, sequences following otss, istss and astss, as well as 5′utrs longer than the average of 72 nt, were compared to the rfam database.\n",
            "\n",
            "in this manner nine small rnas were found in the b. cenocepacia j <dig> genome . one of these constitutes the 6s rna, which was already predicted and its expression confirmed by other studies on b. cenocepacia  <cit> . two small rnas are phage-related regulatory rnas located on genomic island bcengi <dig>  two rfam hits constitute conserved regulatory motifs: the sah riboswitch located in the 5′utrs of bcal <dig>  an adenosylhomocysteinase, and the burkholderiales-specific suca rna motif located in the 5′utr of bcal <dig>  suca, an enzyme of the citric acid cycle. the suca rna motif probably constitutes a riboswitch  <cit> . the remaining four novel small rnas all are from the same family, “toxic small rnas”, which were found to be toxic if introduced into e. coli on a cloning vector  <cit> . expression of these toxic small rnas has been confirmed by northern blotting in four strains of b. cenocepacia, including strain j <dig>  <cit> . however, their function in b. cenocepacia is unknown.table  <dig> novel non-coding small rnas in b. cenocepacia j <dig> with hits in rfam database\n",
            "\n",
            "\n",
            "ac <dig> forms one transcriptional unit with isrk\n",
            "\n",
            "\n",
            "blength of non-coding rnas is inferred from drna-seq data and terminator structures  and is not yet experimentally confirmed\n",
            "\n",
            "\n",
            "\n",
            "the sequence of small rnas is generally only conserved between closely related bacterial species and can vary dramatically in primary sequence and secondary structure between bacterial genera  <cit> . on the other hand, most small rnas in the rfam database are derived from well-studied species such as e. coli, which are not closely related to burkholderia. since the rfam algorithm first performs a blast search, functional homologues from distantly related species are unlikely to produce a hit and novel small rnas specific for b. cenocepacia are likely to be overlooked by this approach.\n",
            "\n",
            "to identify putative regulatory small non-coding rnas not yet represented in the rfam database, we compared sequences derived from otss to non-coding rnas experimentally confirmed in b. cenocepacia strains in other studies using rna-sequencing and northern blotting  <cit> , co-purification with hfq-protein  <cit>  or microarrays , if they showed the following properties: strong transcription initiation with a coverage > <dig> reads in drna-seq data, a defined 3′ end in drna-seq data or a transcript appearing short , truncated or missing in grna-seq data.\n",
            "\n",
            "homologues of six short transcripts with strong transcription initiation from otss were also expressed in b. cenocepacia strains au <dig> and hi <dig> under conditions mimicking the human lung and the soil environment   <cit> . one of these was confirmed to be a small rna by northern blotting in the same study. one short transcript was present in the rna fraction co-purified with the hfq-protein of b. cenocepacia j <dig>  <cit> . hfq is an rna chaperone which mediates base pairing of small regulatory rnas with their target mrna  <cit>  and b. cenocepacia j <dig> harbours the hfq gene as two non-identical homologues  <cit> , making this non-coding rna a plausible candidate for a regulatory small rna. these findings show that with the approach used in this study we could identify transcripts which could encode non-coding regulatory small rnas. their strong expression in biofilms suggests that these small rnas might have a role in adaptation of b. cenocepacia to biofilm conditions.table  <dig> putative novel non-coding rnas expressed in b. cenocepacia and experimentally confirmed by previous studies\n",
            "\n",
            "ncs21a\n",
            "\n",
            "ancs <dig> is associated with an open reading frame and potentially constitutes a protein, see additional file 6: table s5\n",
            "\n",
            "\n",
            "blength of non-coding rnas is inferred from drna-seq data and terminator structures  and is not yet experimentally confirmed\n",
            "\n",
            "\n",
            "\n",
            "a detailed analysis of putative novel small non-coding rnas expressed in b. cenocepacia j <dig> biofilms and their involvement in biofilm formation is ongoing.\n",
            "\n",
            "CONCLUSIONS\n",
            "this study is the first genome-wide analysis of tss in b. cenocepacia. through differential rna-sequencing, bioinformatics methods and 5′race we annotated the primary tss for  <dig> genes expressed in biofilms, defined alternative start codons for  <dig> genes, identified novel protein sequences and characterised antisense transcription.  <dig> non-coding rnas highly expressed in biofilms and a potassium uptake system strongly induced under biofilm conditions were identified that could be involved in biofilm formation and survival. comparison of drna-seq data with grna-seq data proved to be invaluable for tss categorisation and interpretation.\n",
            "\n",
            "the data presented in this study will provide the starting point for evaluation of the regulatory processes involved in b. cenocepacia biofilm formation and could reveal novel targets for antibiotic therapy.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the b. cenocepacia j <dig> genome consists of a large  <dig>  mb multi-replicon genome with a high average gc-content of  <dig>  %.\n",
            "the b. cenocepacia j <dig> genome contains a large  <dig>  mb multi-replicon genome with a high average gc-content of  <dig>  %.\n",
            "the b. cenocepacia j <dig> genome contains a large  <dig>  mb multi-replicon genome with a high average gc-content of  <dig>  %.\n",
            "the b. cenocepacia j <dig> genome contains a large  <dig>  mb multi-replicon genome with a high average gc-content of  <dig>  %.\n",
            "the b. cenocepacia j <dig> genome contains a large  <dig>  mb multi-replicon genome with a high average gc-content of  <dig>  %.\n",
            "the b. cenocepacia j <dig> genome contains a large  <dig>  mb multi-replicon genome\n",
            "98\n",
            "[{'summary_text': 'in this study, we employed a systems-level network biology approach and report the construction of a large-scale arabidopsis immune co-expression regulatory  network.\\nin this study, we employed a systems-level network biology approach to construct genome-wide arabidopsis immune co-expression regulatory  network.\\nwe identified  <dig> immune-related functional modules and identified  <dig> co-expression regulatory elements.\\nwe also identified  <dig> genes that are highly correlated at the expression level but are densely connected to each other.\\nour data provide further insights into the transcriptional regulatory mechanisms repressing additional putative negative regulators of plant defense upon treatments with pathogen-mimicking stimuli.\\n'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 4750 to 5120 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "agriculture, in struggling to meet global food demands of a rapidly growing population, has been plagued with plant diseases that thwart crop production and account for a global annual average yield loss of  <dig> percent . in light of the importance of agriculture to humans, it is essential to elucidate and understand the mechanisms of plant immunity at the molecular level. as with any host-pathogen conflict, plants and their disease agents are in an evolutionary arms race. when the host mounts a defense reaction, the pathogen develops new strategies to evade the defensive mechanisms, which causes the continuation of this cycle, ad infinitum <cit> . our present understanding of plant immune systems reveals two primary means by which plants recognize their invaders. the recognition of microbial-associated molecular patterns  is the first line of defense and its associated mamps-triggered immunity  is highly efficient at repelling most pathogens. if pathogens are able to breach this first line of defense through the production of effector proteins, plants deploy effector-triggered immunity   <cit> . both modes of defense cause massive transcriptional reprogramming which involves complex signal transduction networks and cross-talk that is mediated by plant hormones, including: salicylic acid , jasmonic acid  and ethylene   <cit> . the arabidopsis genome encodes approximately  <dig>  transcription factors  that are implicated in diverse biological processes including the regulation of immune signaling pathways . however, the global organization of tf-dna interactions remains elusive. moreover, what remains mostly unknown are the precise mechanisms by which the plant cell integrates innumerable synergistic and antagonistic immune transcriptomic signals to orchestrate fine-tuned and pathogen-specific defense responses.\n",
            "\n",
            "in recent years, systems biology approaches, specifically network analyses that integrate experimentally derived information with computational modeling, have emerged as powerful tools for studying complex traits in diverse species . network biology provides comprehensive analyses of the system’s components  and the relationships among them . in addition, the analyses of the topological properties of the network can provide further understanding of the hierarchical organization of a complex biological system and contribute to the overall interpretation of biological complexity. in any eukaryotic cell, thousands of genes and their products orchestrate their transcriptional and translational activities to ensure the proper execution of cellular functions  <cit> . it has become evident that biological functions can be accomplished by functional modules that are embedded within the interaction networks including transcriptional gene regulatory networks  <cit> . comprehensive high-throughput analyses of gene expression can allow for the identification of gene clusters that are highly correlated in expression levels across multiple samples in any given cellular state  <cit> . generally, it is thought that genes in the same co-expression sub-network are often enriched with similar functional annotations. additionally, the metric to measure the co-expression falls into one of two major categories: correlation coefficients or mutual information measures  <cit> . finally, finding common cis-regulatory elements  can aid in the identification of co-regulated gene clusters and characterization of transcriptional regulatory networks  <cit> .\n",
            "\n",
            "in the current study we employed a systems biology approach and report the construction of a large-scale arabidopsis immune co-expression regulatory  network. we tested five diverse algorithms, i.e. pcc , aracne multiplicative , aracne additive, clr , and mrnet  with different thresholds, which yielded  <dig> pairs of experimental networks along with their respective random networks . we employed network biology analyses and determined that aracne multiplicative network , with threshold of  <dig>  exhibits properties of a “true” network as it possesses the scale-freeness attribute . next, we partitioned the aicr network and predicted  <dig> functional modules containing at least six members with the largest module encompassing  <dig> nodes. subsequently, we analyzed functional annotations of genes within each module and calculated enrichment of specific gene ontology terms to evaluate the biological significance of functional modules. to establish a causal relationship between co-expression and co-regulation, we sought to identify common cis-regulatory elements. first we developed a new, comprehensive software interface for cis-regulatory elements discovery in arabidopsis . we demonstrated that occean boasts higher capacity  and features higher precision than meme . we identified several statistically enriched novel cis-regulatory elements in our dataset. finally, we evaluated and discussed key immune-related functional modules.\n",
            "\n",
            "RESULTS\n",
            "construction of the arabidopsis immune co-expression regulatory  network\n",
            "to generate a large-scale immune co-expression network, we selected arabidopsis transcriptomic data that encompassed the broadest possible spectrum of immune responses to pathogens or pathogen-mimicking stimuli treatments . previously, we have employed the same set of experiments to define transcriptional responses of genes that encode the proteins identified in plant-pathogen immune network, version 1′   <cit>  . we compiled the lists of probes showing significant up- or down-regulation and discovered  <dig>  differentially expressed genes, which subsequently were used to build a comprehensive immune co-expression network . in a co-expression network, the nodes represent the genes and the edges  represent the similarity/relatedness between the genes, which is generally measured by the pearson correlation coefficient   <cit> . pcc and other measures of association determine the degree of linear dependencies between the variables  <cit> . while these association models have obvious statistical advantages, they lack the ability to capture non-linear relationships  <cit> . in contrast, mutual information  is a non-linear statistic that provides an attractive alternative to measure biologically significant non-linear relationships  <cit> . we used both linear and non-linear models to generate arabidopsis immune co-expression regulatory  network. to measure linear associations between two genes, we calculated the pcc values for all of the genes in a matrix of  <dig>  ×  <dig>  . then, we employed both a similarity threshold algorithm  as well as a newly developed and parameter-free algorithm termed mutual k-nearest neighbor  to process the calculated pcc values  <cit> . mknn was recently shown to possess several advantages over the commonly used similarity threshold algorithm  <cit> . using the linear model, we generated seven pairs of experimental networks along with their respective random networks, i.e., pcc , k <dig>  k <dig>  k <dig>  k <dig>  k <dig> and k <dig> . to infer non-linear association between immune-related genes, we employed four recently used mi methods: aracne multiplicative , aracne additive, clr , and mrnet  with  <dig>  and  <dig>  thresholds. specifically, we employed the ‘parmigene’ package  to construct eight additional experimental networks . it has been recently shown that the mi estimator implemented in parmigene provides more precision and unbiased results compared with previous mi estimators . our approach to initially build multiple experimental and random networks using both linear and non-linear models is aimed to determine an optimal experimental network that displays topological properties of a “real” biological network  <cit> .table  <dig> \n",
            "comparison of multiple algorithms used to generate arabidopsis immune co-expression  network\n",
            "\n",
            "\n",
            "\n",
            "pcc, pearson correlation coefficients; aracne, algorithm for the reconstruction of accurate cellular networks; clr, context likelihood of relatedness; mrnet, minimum redundancy network.\n",
            "construction of arabidopsis immune co-expression regulatory  network. network is displayed in prefuse force directed layout algorithm in cytoscape. a node  represents genes and a grey edge linking two nodes indicates the co-expression relationship between these two nodes based on pearson correlation coefficient . red nodes represent the proteins present in plant-pathogen interaction network   <cit> . the common feature of experimental the aicr network is listed on the bottom.\n",
            "\n",
            "\n",
            "\n",
            "global topological properties of the aicr network\n",
            "network topology refers to the arrangement or pattern of interactions within a network. the majority of naturally occurring networks, including biological networks, maintain certain topological characteristics in terms of their structure and organization that are significantly different from random networks  <cit> . therefore, in an initial experiment, we computed degree of distribution for all  <dig> pairs of experimental networks along with their respective random networks. we found that two networks, aracne multiplicative  and aracne additive , with threshold of  <dig>  exhibit the scale-freeness attribute , when compared with all random networks and other experimental networks . since the precision of parmigene-based aracne multiplicative is better than aracne additive, we selected aracne multiplicative with threshold  <dig>  as the “true” experimental aicr network  <cit> .figure  <dig> \n",
            "degree distribution of the aicr. frequency of the degree of the aicr  and random  networks are indicated in log scale. presence of highly connected nodes  can be observed in the aicr network.\n",
            "\n",
            "\n",
            "\n",
            "“real-world networks” tend to form high-density clusters and exhibit a clustering coefficient that is significantly higher than expected by random chance  <cit> . therefore, we computed clustering coefficients of the aicr network and its corresponding random network . a clustering coefficient describes the degree of congregation among the nodes of a graph. the distribution of the average clustering coefficient in the aicr ranges between  <dig> - <dig>  and the frequency of the number of neighbors with higher clustering coefficient in the aicr is significantly greater than in a random network . to understand how the differences in the number of connections impact the topology of our co-expression network, we measured the shortest path  and the closeness centrality    <cit> . the path lengths for the majority of the nodes in the aicr are significantly shorter than those of a random network . in addition, the number of neighbors with significantly higher closeness centrality  is higher in the aicr network compared with random network . in biological networks, highly connected nodes  and nodes central to the network  are thought to play significant regulatory roles on their adjacent nodes. thus, we computed several topological properties including shared neighbor distribution  , neighborhood connectivity distribution  , topological coefficients   and betweenness centrality    <cit> . these data suggest that the aicr network is not randomly organized and shares properties of several previously described biological networks . another important characteristic of scale-free networks is the presence of a main component. connected component analyses discovered that the aicr network comprises  <dig> disconnected components, each containing at least six members. the main component in the aicr network has  <dig> nodes and  <dig>  edges. the second largest component contains only  <dig> nodes and  <dig> edges. this qualitative global topology of the aicr network is similar to previously known biological networks in arabidopsis and other eukaryotes  <cit> . in addition, network biology analyses revealed that degree distribution of main component of the aicr follows a power law compared to the main component of a random network with similar size .figure  <dig> \n",
            "average clustering coefficient of the aicr. a plot between number of neighbors  and average clustering coefficient  is drawn representing the aicr  and random  networks.\n",
            "betweenness centrality of the aicr. frequency of betweenness centrality of the aicr  and random  networks are indicated.\n",
            "\n",
            "\n",
            "\n",
            "biological implications of topological properties of the aicr network\n",
            "topological properties of the network can also be utilized to prioritize genes for further functional analysis. highly connected and highly centered nodes are likely to play crucial roles in maintaining network integrity and controlling information flow throughout the plant immune system. specifically, we identified  <dig> highly connected nodes . to characterize the significance of these highly connected proteins, we combined hub <dig> of the aicr with biophysical interactions from plant-pathogen immune network, version 1′   <cit> . the ppin- <dig> was constructed using effectors  from two pathogens, arabidopsis immune proteins, and ~ <dig>  other arabidopsis proteins covering almost one-third of the genome  <cit> . pathogens utilize effector molecules to rewire the host network in a manner conducive to pathogen proliferation and dispersal  <cit> . ppin- <dig> discovered  <dig> effector interacting proteins  and it was also demonstrated that these effector targets are highly interconnected host proteins  <cit> . despite the unavailability of effector targets from the entire arabidopsis genome in ppin- <dig>  we found that the aicr network contains  <dig> effector targets and three proteins  are hub <dig> in the aicr network . given that transcriptional regulation does not necessarily coincide with biophysical interactions, this overlap is potentially significant and biologically meaningful. this also suggests that pathogen effectors target key regulatory proteins in both transcriptional as well as protein-protein interactions networks.\n",
            "\n",
            "in addition, we selected ninety nodes  with the highest betweeness centrality . among them, we found a number of previously characterized immune regulators or defense-related proteins, encompassing various levels of signal transduction flow: resistance proteins   <cit> , kinases mediating early signaling events   <cit> , transcriptional executors  , secretory proteins assisting with folding and modification of newly synthesized proteins   <cit> , enzymes involved in production of phytohormones and antimicrobial compounds  and pathogenesis-related proteins   <cit> , hormonal response modulators   <cit>  and finally, components of ubiquitin-mediated protein degradation   <cit> . identification of these key immune regulators serves as a proof of concept for our analytical approach and justifies further analyses on suite of “top 90” proteins selected among thousands of other proteins in the aicr network.\n",
            "\n",
            "network clustering and module annotations\n",
            "another essential feature of a majority of biological networks is their ability to naturally organize into modules  <cit> . discovery of such functional modules within biological networks is imperative for understanding principles of cellular organization and functions. to identify functional modules  that are comprised of highly interconnected sets of genes within the aicr network, we employed a fast agglomerative algorithm   <cit> . this new, low time complexity algorithm operates based on a local variable, edge clustering coefficients, making it ideal to partition a relatively large network  <cit> . overall, we identified  <dig> immune-related functional modules containing at least six members with the largest module encompassing  <dig> nodes . furthermore, we subjected the aicr network to a module size distribution analysis. we computed the frequency and module size and demonstrated that the distribution of the module size follows a power law distribution with an exponential truncation , which is common for several previously described “real-world” networks  <cit> . the module size distribution property further indicates that the aicr network possesses a modular structure.\n",
            "\n",
            "we also investigated the enrichment of gene ontology  terms  <cit>  in the ten largest immune-related modules  with sizes ranging from  <dig> to  <dig> distinct nodes using the database for annotation, visualization and integrated discovery  v <dig>   <cit>  . the go enrichment terms are remarkably informative in examining significant biological functions among co-expressed genes in large-scale networks  <cit> . several gene ontology  categories were enriched  among the genes in the aicr network . these major go categories include: immune-related, plastid, reproductive processes, nucleotide binding/kinases, intrinsic to membrane, metal-binding, mitochondrion and non-membrane-bounded organelle. we utilized these go enrichments in combination with significantly enriched cis-regulatory elements to define the potential functions of the immune-related modules .table  <dig> \n",
            "significantly enriched go terms in the ten largest immune-related modules with sizes ranging from  <dig> to  <dig> genes\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "development of occean software and identification of cis-regulatory elements\n",
            "we sought to investigate whether all or a subset of the co-expressed genes within a given immune-related module are also co-regulated . given that transcriptional regulatory networks are highly complex and that functional modules may display crosstalk among themselves, we can also expect that the same transcription factor can regulate co-expressed genes in multiple immune-related modules. whereas experimentally verified plant cis-regulatory elements can be retrieved from place   <cit> , athena   <cit>  and agris   <cit> , only ~ <dig> cis-regulatory elements are currently known in arabidopsis, and a limited set of ~ <dig> immune-related cis-elements have been described. in addition, the prediction of cis-elements from classical approaches is typically driven by a single experiment or dataset. moreover, the currently available online motif discovery software has limited capacity to process a small number of gene entries. for example, frequently used software meme  can only process up to  <dig>  characters in a single run   <cit> . another common software used to predict cis-regulatory elements, athamap  can only manage up to  <dig> gene entries  <cit> . these limitations can substantially decrease the chance of identifying novel cis-regulatory elements. thus, we first aimed to develop occean  software that can process the promoter gene sequences at the entire genome scale  in a single run. occean identifies the statistically enriched/depleted cis-regulatory elements  and integrates data from  our newly developed blastn-based program to identify short sequences and  an improved version of the bootstrapping tool pobo   <cit> . this user-friendly software requires the sequences in fasta format as input, processes genome-scale level sequences, identifies common sequences in a given set of promoters, performs bootstrapping, and provides statistically enriched cis-regulatory elements in the gene’s promoter as the output. occean is freely available online at http://occean.cis.uab.edu:8080/occean/. we employed occean to individually process the sequences of immune-related genes’ promoters for our “top 10” immune-related modules. the promoter sequences of the genome  were used as background to compute the fold enrichments of putative cis-regulatory elements. we also analyzed the occurrences of all the putative cis-regulatory elements , performed  <dig>  × bootstrapping to determine the background mean and finally calculated fold enrichment ratios for all six-mer sequences. three different fold enrichment ratios  prioritized newly identified cis-regulatory elements in “top 10” immune-related modules . to analyze the performance and robustness of our newly developed software, we compared the efficacy of occean with meme. we generated a list of experimentally known cis-regulatory elements and computed the precision of both occean and meme using cis-regulatory elements identified in our top  <dig> immune-related modules. meme was unable to compute the analyses on top  <dig> immune-related modules as it can’t process over  <dig>  characters simultaneously. we computed true positives , false positives  and precision for meme as well as occean enrichment ratios ≥  <dig>  ≥  <dig> and ≥  <dig> . we selected an enrichment ratio ≥  <dig> as the optimal value for occean as it yields a higher number of ntps, significantly fewer nfps positives and greater precisions compared with enrichment ratios ≥  <dig> and ≥  <dig>  in summary, we demonstrated that occean boasts higher capacity  and features higher precision than meme . identification of cis-regulatory elements among co-expressed genes could provide additional information for prediction of biological function.figure  <dig> \n",
            "screenshot of occean . results can be obtained as an excel sheet and additional information can be accessed in pop-up windows.\n",
            "performance and accuracy of occean. true positives , false positives  and precision for meme as well as occean with multiple enrichment ratios are shown.\n",
            "\n",
            "\n",
            "\n",
            "inferring biological relevance of immune-related modules\n",
            "to fend off potential pathogens, plants employ two major types of immune responses: mamps-triggered immunity  and effector triggered immunity   <cit> . eti is essentially a high amplitude re-booting of the immediate but weaker mti. eti results in robust disease resistance responses, including localized host cell death  and systemic signaling  <cit> . in addition, upon pathogen recognition at the mti and/or eti levels, the plant cell undergoes extensive transcriptional changes involving complex reiterative signaling networks and cross-talk controlled by phytohormones  <cit> . this leads to the metabolic reprogramming and production of an array of antimicrobial compounds  <cit> . here we highlight two major immune-related functional modules, the largest identified module  <dig>  and a more compact module  <dig>  as two most representative gene clusters reflecting the broad array of diverse pathogens and immune stimuli used in the microarray experiments employed in our analyses. module  <dig> is the largest module identified by our clustering analyses and comprises  <dig> nodes,  <dig>  edges, clustering coefficient of  <dig>  and network density of  <dig> . the genes of module  <dig> are enriched in three major go categories:  kinases/ribonucleotide binding,  immune responses/programmed cell death and  transmembrane proteins.figure  <dig> \n",
            "inferring molecular functions of the immune-related modules. module  <dig>  and module  <dig>  is presented. module  <dig> is displayed in dark and light green colors. red and green nodes represent up-regulation and down-regulation of genes within these modules by ogs  and flg <dig> . the most over-represented biological process go term was indicated with each module.\n",
            "\n",
            "\n",
            "\n",
            "diverse kinases account for the major functional category of module  <dig>  encompassing  <dig> out of  <dig> proteins representing receptor-like kinases , mitogen-activated protein kinases , calcium-dependent protein kinases , wall-associated kinases , etc. typically, rlks perceive mamps such as flg <dig>  elf <dig>  chitin and ogs and trigger mti  <cit> . subsequently, a mapk cascade is activated and downstream signaling ensues including phytohormonal crosstalk  <cit> . these events result in the activation of a wide-spectrum of immune responses, such as induced biosynthesis of phytoalexins and other antimicrobial secondary metabolites such as glucosinolates, calcium influx, as well as production and accumulation of reactive oxygen species   <cit> .\n",
            "\n",
            "at the second, more powerful layer of the immune response, plants usually deploy nlr receptors . the nlr receptors directly recognize specific effectors or indirectly detect effector activities and trigger eti  <cit> . we discovered  <dig> nlr genes in the kinase sub-cluster of module  <dig>  these include proteins conferring resistance to diverse pathogens such as pseudomonas syringae, albugo candida, hyaloperonospora arabidopsidis and various fusarium races.\n",
            "\n",
            "the second most abundant go category in module  <dig> was immune-related genes . this functional sub-cluster contains various defense-related transcription factors such as wrky <dig>  wrky <dig>  myb <dig> and anac <dig>  <cit> , four members of the ring-h <dig> finger protein family, a heat shock transcription factor hsf a-4a  <cit> , an ethylene-responsive transcription factor erf <dig> and, intriguingly, three scarecrow-like gras family transcription factor genes, known to be primarily implicated in plant root development . collectively, transcription factor genes account for over 40% of this sub-cluster’s gene count. in addition, we identified a small group of genes strongly associated with response to fungal cell wall elicitor chitin, such as two auxin-regulated saur genes, one lectin-like, enhanced disease susceptibility  <dig>  and senescence-associated protein sag <dig> .\n",
            "\n",
            "another enriched go category in module  <dig> was membrane-associated proteins including both plasma membrane- and organellar membrane-resident peptides; not surprisingly, the largest class of these membrane proteins is transporters. in plants, uptake and translocation of nutrients play essential roles in physiological processes including plant growth, nutrition, signal transduction, and development . transport processes are also critical for the reallocation of resources and it was previously shown that defense signals cause realignment of transport activities to redirect resources toward immune responses and protect tissues of high value  <cit> . of  <dig> genes assigned to this sub-cluster, seven were transport proteins . the prevailing class was sugar  transporters, in agreement with the fact that this class of proteins constitutes a key component for carbon partitioning at the whole plant level and is involved in both symbiotic and parasitic plant-fungi interactions  <cit> . additionally, we identified cni <dig>  that is a key regulator of the carbon/nitrogen response for growth phase transition in arabidopsis seedlings  <cit> , as well as atp binding cassette  and inorganic phosphate transporters that were previously postulated to be required for organ growth, nutrition, development, and stress responses  <cit> . last but not least, we also identified two other well-known immune regulators: syntaxin 121/pen <dig> and mlo <dig> genes, required for resistance against barley powdery mildew, blumeria graminis sp. hordei and a fungal pathogen colletotrichum higginsianum <cit> .\n",
            "\n",
            "another essential aspect of large-scale transcriptomic analyses is to link co-expression with co-regulation, i.e., to determine presence of common cis-regulatory elements among genes of the same module. several known cis-regulatory elements were discovered in the promoters of module  <dig> genes . two of the enriched elements, tcatgg and catgga, overlap with the octadecameric carg box sequence 5′-cttacctttcatggatta-3′, identified in the apetala <dig> promoter  <cit> . apetala <dig>  a class b homeotic organ identity gene, was originally discovered as the central regulator of petal development in arabidopsis flowers, but later also shown to be involved in control of the floral meristem proliferation, regulation of flowering time, and other plant reproductive processes . inhibition of reproductive processes is an expected outcome of an immune stimulation, given the recent report describing a transition from growth to defense following immune stimuli treatments in arabidopsis  <cit> . involvement of a developmental regulator in defense responses is not unprecedented and was previously shown for a myb-related gene asymmetric leaves  <dig>   <cit> .\n",
            "\n",
            "jasmonic acid  and ethylene  are two plant hormones antagonistic to sa  <cit>  and down-regulation of major ja/et signaling proteins will promote sa signaling by suppressing the ja/et pathway  <cit> . consistent with this observation, we identified an enriched element, atcttg that resembles the binding site for ethylene-insensitive  <dig>  and three ein3-like  proteins  <cit>  as well as two hexamers gtcgtc and cgtcgt, which overlap with the core binding site of jase <dig> motif in two ja biosynthetic genes opr1/opr <dig> <cit> . recently, ein <dig> and eil <dig> were shown to negatively regulate mamp-triggered immunity via a direct binding and down-regulation of the sa biosynthetic gene sid <dig> <cit> . given that oprs are implicated in ja biosynthesis, we expect a fine-tuned interplay between sa and ja/et signaling pathways  <cit> . in addition, several novel cis-regulatory elements were discovered in our study , and the identity of the cognate transcription factors and their functional relevance will be subject to future studies. very recently, it was demonstrated that numerous transcription factors can recognize secondary elements, which in some cases are completely sequence-unrelated to the primary element  <cit> . in conjunction with this study, our analysis of module  <dig> shows that arr <dig>  not only recognizes the known primary cis-regulatory element agata/tcg but also can specifically bind to a previously unknown cis-regulatory element agatct. thus, it’s likely that transcription factor have extended list of binding specificities beyond the currently known cis-regulatory elements. we propose that the above described transcription factors and additional unknown regulatory proteins coordinate the gene regulation in this module.\n",
            "\n",
            "our data provide further insights into the transcriptional regulatory mechanisms repressing additional putative negative regulators of plant defense upon treatments with pathogen-mimicking stimuli.\n",
            "\n",
            "CONCLUSIONS\n",
            "in this study, we used a systems-level network biology approach to construct genome-wide arabidopsis immune co-expression network and demonstrated that this network shares properties of a ‘real-world network’. topological properties-based partitioning allowed us to unravel  <dig> distinct immune-related functional modules. we demonstrated that nodes in the same module are not only highly correlated at the expression level but also densely connected to each other. detailed analyses of two key immune-related modules provided a systems-level understanding of how plant cells coordinate distinct immune signals to orchestrate fine-tuned and pathogen-specific defense responses. our novel approach to discover cis-regulatory elements using occean is an effective method of solving the issue of finding novel motifs within a sequence set. occean has advantages over other programs of the same purpose, such as apples  and meme  <cit> . apples requires finding organisms of a specified relational distance for comparison, which can burden the user, and meme has the statistical risk of discarding actual existing motifs. these problems are avoided in our solution whilst continuing to maintain a fair amount of focus on client-side simplicity. in addition, occean has the capacity to be expanded for analyses of other eukaryotes genomes, such as fly and human. our systems-level approach to examine cis-regulatory elements  in the promoters of the co-expressed genes made it possible to link co-expression to co-regulation of genes in the same module.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "in this study, we employed a systems-level network biology approach and report the construction of a large-scale arabidopsis immune co-expression regulatory  network.\n",
            "in this study, we employed a systems-level network biology approach to construct genome-wide arabidopsis immune co-expression regulatory  network.\n",
            "we identified  <dig> immune-related functional modules and identified  <dig> co-expression regulatory elements.\n",
            "we also identified  <dig> genes that are highly correlated at the expression level but are densely connected to each other.\n",
            "our data provide further insights into the transcriptional regulatory mechanisms repressing additional putative negative regulators of plant defense upon treatments with pathogen-mimicking stimuli.\n",
            "\n",
            "99\n",
            "[{'summary_text': 'candida esophagitis represents a severe threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\\ncandida esophagitis is a serious threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\\ncandida esophagitis represents a severe threat to an immunocompromised body and, especially in neutropenic patients, often is the first manifestation before candida sepsis develops.\\ncandida esophagitis is a serious threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\\ncandida esophagitis is a serious threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\\ncandida esophagitis is a serious threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\\ncandida esophagitis is a serious threat to an immunocompromised body and the pathogen-'}]\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Input ids are automatically padded from 4424 to 5120 to be a multiple of `config.attention_window`: 1024\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original text:\n",
            "BACKGROUND\n",
            "candida albicans colonize distinct microanatomical regions such as the oro-gastointestinal tract as a commensal, but also accounts for more than 50% of all fungal systemic infections  <cit> . candida esophagitis represents a severe threat to an immunocompromised body and, especially in neutropenic patients, often is the first manifestation before candida sepsis develops  <cit> . host defense mechanisms preventing mucosal  candidiasis are poorly understood, but include both innate and adaptive responses  <cit> .\n",
            "\n",
            "the gastrointestinal epithelial layer represents a barrier that is usually adequate to restrain commensal microbes, but is often insufficient to protect against microbial pathogens. once this physical barrier is penetrated, recognition of invading microbiota is the first step in the initiation of a fast immune response and involves the activation of pattern recognition receptors by microbial pathogens and their products  <cit> . c. albicans have been shown to activate a subset of pattern recognition receptors, the toll-like receptors . this family of transmembrane receptors recognizes a broad variety of signature motifs on microbes and transduces signals leading to the activation of transcription factors  <cit> , production of cytokines and antimicrobial peptides  <cit> . however there is some controversy and inconclusive data which tlr subtypes are activated by c. albicans. some reports indicated a critical role for tlr <dig> and tlr <dig> in activating the host defense response alone or in combination with the ss-glucan receptor dectin- <dig>  <cit>  whereas other studies showed that tlr- <dig> and tlr- <dig> are responsible for the recognition of c. albicans  <cit> .\n",
            "\n",
            "in addition the downstream effectors in the immune response against c. albicans are largely unknown. there is growing evidence that human β-defensins  are critical components of both the innate and adaptive immune responses to candida infections with distinct antifungal efficacies and mechanisms for hbd- <dig> and hbd- <dig>  <cit> . expression of hbds is regulated by a plethora of proinflammatory cytokines like il-1β, tnf-α and egf-receptor ligands activating downstream effectors like the transcription factors nf-κb or ap- <dig>  <cit> . hbds are secreted by epithelial cells and protect the gastrointestinal mucosa by antimicriobial activity as well as by chemotactic properties recruiting polymorphonuclear leukocytes  to the site of infection  <cit> . transepithelial migration of pmns is observed during oral candida infection and is believed to play a crucial role in the clearance of infection and in epithelial homeostasis  <cit> . this protective phenotype is associated with production of epithelial proinflammatory cytokines, including il- <dig>  il-1β and tnf-α. however, the mechanisms by which pmns and epithelial cells interact to protect mucosal surfaces from c. albicans invasion are mostly unknown.\n",
            "\n",
            "we have recently shown that hbd- <dig> and hbd- <dig> are highly expressed in candida esophagitis and that the α-defensins human neutrophil peptides 1– <dig> were also upregulated  <cit>  indicating involvement of neutrophils in the immune response to the c. albicans infection  <cit> . this is in line with the observation that il- <dig> is strongly expressed in the mucosa of patients suffering by candida infection  <cit> , since this cytokine is involved in recruitment of polymorphonuclear leukocytes  to sites of microbial infection  <cit> .\n",
            "\n",
            "in the present study we explored the mechanisms involved in the induction of hbd expression in an in vitro model of candida esophagitis. using this co-culture model we analyzed the contribution of pmns in the regulation of epithelial hbd expression. we found that hbd- <dig> and hbd- <dig> are strongly upregulated through high concentrations of c. albicans alone. if lower concentrations of c. albicans were used, only a moderate induction of these hbds were observed that was strongly enhanced by coculture of the epithelial cell line with pmns, emphasizing the important role of mesenchymal-epithelial interactions in early host defense against fungal infection. furthermore we were able to show that the nf-κb and mapk signalling pathways contribute to the regulation of hbd- <dig> and that hbd- <dig> is a downstream target of a distinct egfr/mapk/ap- <dig> pathway in candida esophagitis.\n",
            "\n",
            "RESULTS\n",
            "supernatants of c. albicans induce hbd expression in oe <dig> cells\n",
            "to establish an in vitro model for candida esophagitis we first treated several oesophageal cell lines  with established inducers of hbd-expression. only for the oe <dig> cell line a reliable induction of the mrna of hbd- <dig> and hbd- <dig> could be detected . we next challengend oe <dig> cells with supernatants of c. albicans cell culture growing under various concentrations of c. albicans. supernatants of c. albicans induce upregulation of hbds in a concentration-dependent manner. for the supernatants of the density of  <dig>  ×  <dig> c. albicans cells/ml no significant induction of hbd- <dig> and hbd- <dig> was detected . in contrast the supernatants of  <dig> ×  <dig> c. albicans cells/ml resulted in a significant upregulation of hbd- <dig> and to a smaller extent of hbd- <dig> . hbd <dig> expression was not altered by one of the inducers or the supernatants .\n",
            "\n",
            "pmns significantly enhances hbd- <dig> and hbd- <dig> expression during c. albicans infection\n",
            "to elucidate a potential functional interaction of pmns and epithelial cells in the immune response to c. albicans infection we established a transwell coculture model. in this two-compartment model we tested the effect of a coculture of oe <dig> cells, pmns from healthy donors and c. albicans in different settings: 1) direct contact of oe <dig> cells with pmns and 2) direct contact of oe <dig> cells with c. albicans. the corresponding third component of the system  was cocultured in the transwell system allowing for the interaction of auto- and paracrine factors . since higher cell numbers than  <dig>  ×  <dig> c. albicans cells/ml or  <dig>  ×  <dig> pmn/ml repressed growth of oe <dig> cells in the direct setting , all experiments were conducted with these concentrations of c. albicans and/or pmns. the direct or indirect coculture of oe <dig> with c. albicans had only small effects on the expression of hbd- <dig>  and hbd- <dig>  comparable with effects observed with supernatants of the low concentration of c. albicans . the direct interaction of oe <dig> cells with pmns led to an increase of the expression of both hbds . the interaction of oe <dig> cells with pmns lead to a significant upregulation of c. albicans induced hbd- <dig> and hbd- <dig> expression compared to the induction with c. albicans or pmn alone . substitution of the live c. albicans by supernatants of the density of  <dig>  ×  <dig> c. albicans cells/ml also lead to the synergistic effects. to keep the manuscript mor concise we only show the data for the live c. albicans.\n",
            "\n",
            "c. albicans activate nf-κb in oe <dig> cells\n",
            "to further analyse the mechanisms involved in the regulation of hbd expression during candida esophagitis we determined whether the observed induction of hbd- <dig> and hbd- <dig> expression by c. albicans was associated with the activation of nf-κb. this transcription factor has been shown to be a central regulator of hbd- <dig> expression  <cit>  but there is some controversy if nf-κb is also involved in the regulation of hbd- <dig> expression  <cit> . by emsa we could show that nf-κb is activated  in association with c. albicans mediated induction of β-defensin expression. in parallel to the observed dependency of defensin expression on the cell number of c. albicans a concentration dependent activation of nf-κb was observed . the coculture of oe <dig> cells with pmns or c. albicans also led to an increased dna-binding of nf-κb that was further enhanced in the transwell setting . supershift assays revealed that the nf-κb complex consisted of the p <dig> and p <dig> subunits .\n",
            "\n",
            "nf-κb is involved in the induction of hbd- <dig> but not hbd- <dig> expression\n",
            "to evaluate the putative contribution of nf-κb in c. albicans induced hbd expression we transiently transfected cells with sirna directed against rela/p <dig> as the transcriptional active subunit of the p65/p <dig> nf-κb complex  <cit> . transfection of oe <dig> cells led to a significant reduction of the expression of rela/p <dig> . furthermore the induction of nf-κb by c. albicans was inhibited by this strategy as shown by emsa . this reduction of nf-κb activation strongly reduced the c. albicans mediated induction of hbd- <dig> expression but had no effect on the hbd- <dig> expression . in the coculture settings the reduction of rela/p <dig> expression also attenuated hbd- <dig> induction . these results support a critical role of nf-κb in regulating hbd- <dig> but not hbd- <dig> expression.\n",
            "\n",
            "role of ap- <dig> and the mapk pathway in c. albicans mediated β-defensin induction\n",
            "our results indicated a critical role for nf-κb in c. albicans mediated induction of hbd- <dig> but not for the regulation of hbd- <dig> expression. since the hbd- <dig> promotor contains a functional binding site for the ap- <dig> transcription factor  <cit>  we conducted emsa with a probe containing an ap- <dig> consensus sequence. c. albicans alone  or in the coculture setting  activated ap- <dig> in oe <dig> cells and supershift experiments identified c-jun as one of the central subunits involved . activation of the c-jun/ap- <dig> complex by the mapk pathway is well established  <cit>  and a recent report indicated a role of this pathway in the response of macrophages to the infection with c. albicans  <cit> . to analyse the contribution of the mapk/ap- <dig> pathway in the induction of β-defensin induction by c. albicans pharmacological inhibitors selective for individual pathway were utilized.\n",
            "\n",
            "as shown in figure 7a and 7b inhibition of each of the major mapk pathways resulted in a reduction of hbd- <dig> and hbd- <dig> induction by c. albicans.\n",
            "\n",
            "activation of the egfr is an upstream event in regulation of hbd- <dig> expression\n",
            "upstream of the mapk pathway several growth factor receptors have been shown to be involved in the immune response to microbiota  <cit> . to investigate the upstream events leading to the activation of the mapk pathway in infection with c. albicans we used blocking antibodies against several growth factor receptors. the results revealed a clear involvement of the egf-receptor in the regulation of hbd- <dig> expression by c. albicans. whereas induction of hbd- <dig> expression was slightly inhibited by a neutralizing antibody targeting egfr transactivation, a significant reduction of c. albicans-induced hbd- <dig> expression observed .\n",
            "\n",
            "tgf-α is involved in egfr mediated hbd- <dig> induction\n",
            "finally we searched for the ligand activating the egfr. by using blocking antibodies for the established egfr-ligands egf, hb-egf, amphiregulin and tgf-α we were able to show that tgf-α is involved in the egfr mediated upregulation of hbd- <dig> expression .\n",
            "\n",
            "discussion\n",
            "candida esophagitis represents a severe threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors  <cit> . it is well established that epithelial cells of the esophagus are the central target of an oro-esophageal invasive candida infection but there are only very limited data on the host response preventing a candida esophagitis. in the present study we identified the nf-κb and mapk/ap- <dig> pathways as central regulators of epithelial hbd- <dig> and hbd- <dig> expression during c. albicans infection. furthermore we were able to show a crucial role of the interaction of pmns with the epithelial cell compartment for the induction of hbd- <dig> and hbd- <dig> expression during candida infection of esophageal epithelial cells. finally hbd- <dig> expression is dependent on transactivation of egfr by tgf-α. this is the first report delineating molecular mechanisms leading to upregulation of hbd- <dig> and hbd- <dig> in esophageal candida infection suggesting differential regulation of these two central epithelial antibiotic peptides in response to a c. albicans infection. we were able to show a concentration dependency of hbd- <dig> and hbd- <dig> induction upon stimulation with c. albicans supernatants . this dose dependent effect could be at least in part be mediated through stronger induction of nf-κb  and ap- <dig>  through higher concentrations of c. albicans. a comparable concentration-dependent effect of supernatants of environmental airborne fungi on cytokine release of eosinophils  <cit>  and of c. albicans on the activation of nf-κb  <cit>  was recently reported.\n",
            "\n",
            "there is some controversy about the relevance of the nf-κb and ap- <dig> binding sites in the promotor of hbd- <dig> for full induction of hbd- <dig> expression after treatment with il-1β or infection with microbiota. wehkamp and collegues showed that the parallel activation of nf-κb and ap- <dig> is needed for full transcriptional activation of the hbd-2-promotor after il-1β stimulation, treatment of keratinocytes with supernatants of pseudomonas aeruginosa  <cit> , or e. coli mediated hbd- <dig> induction  <cit> . on the other hand another study demonstrated induction of hbd- <dig> expression through fusobacterium nucleatum in human gingival epithelial cells independent of nf-κb activation  <cit>  and there is also evidence for induction of hbd- <dig> through salmonella enteritidis  <cit>  or h. pylori  <cit>  in the absence of functional ap- <dig>  in the present work we were able to show that both nf-κb and ap- <dig> activation are required for full upregulation of the hbd- <dig> mrna after treatment with either supernatants of c. albicans or the coculture of c. albicans with pmns. inhibiton of the nf-κb or of the mapk/ap- <dig> pathway significantly reduced the induction of hbd- <dig> expression confirming the central role of both transcription factors. these observations are in line with recent results for h. pylori infection  <cit>  and the effects of lactobacilli and the vsl# <dig> bacterial mixture on enterocytes  <cit> . the role of nf-κb is less clear in the regulation of hbd- <dig> expression. in contrast to a recently reported role of nf-κb in the regulation of the hbd- <dig> gene in keratinocytes  <cit>  the majority of data failed to show a functional relevance for nf-κb in controlling hbd- <dig> expression  <cit> . by using sirna targeting the rela/p <dig> subunit of nf-κb we could demonstrate that c. albicans mediates hbd- <dig> upregulation through a mapk/ap- <dig> pathway independently of the observed nf-κb acitvation. to investigate the proposed role of pmns in the immune response the c. albicans infection  <cit>  we established an in vitro model . pmns alone induced nf-κb and ap- <dig> leading to hbd- <dig> and hbd- <dig> expression. coincubation of pmns with c. albicans lead to a significant upregulation of hbd- <dig> and hbd- <dig> expression compared to effects of pmns or c. albicans alone. inhibition of the mapk/ap- <dig> pathway reduced the expression of both hbds under this condition. the fact that pmns alone induced hbd expression indicated that pmns could contribute to the induction of antimicrobial peptides in epithelia during inflammation. there is some evidence that a tlr- <dig> mediated interplay between pmns and epithelial cells is important for the protective response against candida infections  <cit>  but the exact mechanisms remained elusive. one the one hand pmns induced expression and release of il- <dig> and il- <dig> in o <dig> cells  which might in turn activate nf-κb and ap- <dig> leading to hbd- <dig> and hbd- <dig> expression. on the other hand the induction of a th <dig> response, i.e. tnf-α and interferon-γ might directly induce or enhance the expression of hbds as reported recently  <cit> . on the other hand pmns are able to establish an anti-fungal response by upregulate the expression of tlr <dig> in epithelial cells  <cit> . the observed more pronounced effect on hbd- <dig> expression are in line with the theory that hbd- <dig> might be clinically more relevant than hbd- <dig> since hbd- <dig> and hbd- <dig> have potent fungicidal activity against c. albicans at micromolar concentrations, with hbd- <dig> being about  <dig> times more fungicidal than hbd- <dig>  <cit> . finally we demonstrated that hbd- <dig> expression was induced by transactivation of the egfr independent of egf . many signals besides egf converge and result in egfr-dependent signaling which is important for various biological processes including normal growth, development and as shown recently for inflammation or innate immune response  <cit> . using blocking antibodies for the egfr-ligands egf, hb-egf, amphiregulin and tgf-α we were able to show that most likely tgf-α is involved in the egfr mediated upregulation of hbd- <dig> expression . since we are also able to inhibit the induction of hbd <dig> by the supernatants of  <dig> ×  <dig> c. albicans cells/ml by egfr and/or tgf-α blocking antibodies we speculate that the egfr-ligand is secreted by oe <dig> cells. ligands of the egfr are expressed as transmembrane precursors. these are released from the cell surface following shedding of the extracellular domain by a family of metalloproteinases ). adam <dig>  - <dig> and - <dig> are the sheddases of the egfr ligands in response to various stimuli. since human cathelicidin cationic antimicrobial protein - <dig> and its active peptide ll- <dig> have been shown to be involved in the transactivation of the egfr at the airway epithelial surfaces  <cit>  it is tempting to speculate that the activation of pmns by c. albicans leads to secretion of proinflammatory mediators including leukotriene b <dig>  and ll- <dig>  <cit>  which in turn amplifies the inflammatory response  <cit>  and leads to the shedding of egfr-ligands.\n",
            "\n",
            "in conclusion we were able to establish a complex in vitro model for candida infection of esophageal cells to investigate the signalling events leading to upregulation of hbd- <dig> and hbd- <dig> expression.\n",
            "\n",
            "CONCLUSIONS\n",
            "this analysis of signal transduction events in esophageal candida infection demonstrated a potential functional interaction of epithelial cells with pmns and that egfr, nf-κb and mapk/ap- <dig> are involved in divergent signalling events governing hbd- <dig> and hbd- <dig> expression. especially the observed effect of pmns on hbd expression might explain the high incidence of candida esophagitis and candida related deaths in neutropenic patients.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "candida esophagitis represents a severe threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\n",
            "candida esophagitis is a serious threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\n",
            "candida esophagitis represents a severe threat to an immunocompromised body and, especially in neutropenic patients, often is the first manifestation before candida sepsis develops.\n",
            "candida esophagitis is a serious threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\n",
            "candida esophagitis is a serious threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\n",
            "candida esophagitis is a serious threat to an immunocompromised body and the course of the infection is determined by both pathogen- and host-dependent factors.\n",
            "candida esophagitis is a serious threat to an immunocompromised body and the pathogen-\n",
            "100\n",
            "[{'summary_text': 'the shorebirds and allies  present an exceptional group for studying numerous evolutionary hypotheses.\\nour supertree combines complete taxonomic coverage and the inclusion of branch lengths provide the basis for future comparative analyses of charadriiformes phylogeny.\\nour supertree is highly consistent with recent advances in molecular phylogenetics charadriiformes.\\nour results suggest that the shorebirds and allies  are an exceptional group for studying numerous evolutionary hypotheses.\\nour supertree combines complete taxonomic coverage and the inclusion of branch lengths provide the basis for future comparative analyses of charadriiformes phylogeny.\\nour supertree resolves glareolidae outside the larini, sternini, stercorariini, stercorariini, and rynchopini clades.\\nour supertree resolves glareolidae outside the larini, sternini, stercorariini, stercorariini, rynchopini, and rynchopini clades.\\nour supertree resolves glareolidae outside the larini, sternini, stercorariini, stercorariini, rynchopini, and rynchopini clades.\\n'}]\n",
            "Original text:\n",
            "BACKGROUND\n",
            "the shorebirds and allies  present an exceptional group for studying numerous evolutionary hypotheses. their remarkable diversity of social mating system, parental care, sexual dimorphism, ecology and life-history make them an ideal group for unravelling the mechanisms of, for example, sexual selection and sexual conflict. previous comparative studies have made significant contributions to our understanding of the evolution of mating systems  <cit> , parental care  <cit> , sexual size dimorphism  <cit> , locomotion and morphology  <cit> , migratory behaviour  <cit> , egg size  <cit> , and plumage colouration  <cit> . the importance of phylogeny in cross-species comparative studies is well documented  <cit> . large and well-resolved phylogenies that incorporate divergence times provide powerful tests of a wide range of hypotheses whilst accounting for the effects of shared evolutionary history  <cit> . however, the shorebird studies listed above were limited by the lack of a complete phylogeny for the group. most of these studies are based on derivations of the seminal work of sibley and ahlquist  <cit> , yet this study included less than a quarter of extant and recently extinct shorebird species. recently extinct taxa  are: the tahitian sandpiper prosobonia leucoptera, the canary islands oystercatcher haematopus maedewaldoi, and the great auk pinguinus impennis.\n",
            "\n",
            "recent molecular studies covering a wide range of shorebird families have drawn attention to conflict in the reconstruction of the deep basal nodes of shorebird phylogeny . for example, morphological data  <cit>  places alcinae  at the base of the shorebird tree whilst sequence  <cit>  and dna-dna hybridisation  <cit>  data suggests that they are a highly derived sister group to stercorariini , larini , sternini , and rynchopini . it is important to note that taxon coverage differs between these studies and this may be an important factor in determining the tree topology. specific phylogenies have been derived, for example, for sandpipers  <cit> , the genus charadrius  <cit> , and jacanas  <cit>  using dna sequence data. in contrast, morphological evidence provided the basis for chu's  <cit>  study of gull phylogeny. strauch  <cit>  presented the most complete data set of  <dig> charadriiformes species. however, despite the plethora of cladograms for particular shorebird groups , those that address relationships across the whole clade use either sparse taxon sampling  <cit> , or are based on reassessments of strauch's  <cit>  data  <cit> . note that dove  <cit>  included a feather microstructural analysis in addition to her reanalysis of strauch's  <cit>  data.\n",
            "\n",
            "combining phylogenetic data\n",
            "numerous methods and types of data can be used to infer phylogeny. frequently, as in charadriiformes, a single analysis incorporating all taxa of interest is absent. under the principle of total evidence  <cit> , all sources of phylogenetic information should be combined to maximize their explanatory power. eernisse and kluge  <cit>  define total evidence as a method for seeking the best fitting phylogenetic hypothesis for an unpartitioned set of synapomorphies  using character congruence . hence, this method combines the primary data  into a single analysis. the approach is powerful because weak signals in the partitioned data sets may be enhanced when combined, and previously obscured relationships may be revealed  <cit> .\n",
            "\n",
            "the total evidence approach has both practical and theoretical problems. first, only certain types of data can be combined. for example, nucleotide sequences and morphological traits can be readily assessed together as characters, but it is not generally possible to include nucleotide sequences and genetic distance data in a single analysis  <cit> . we acknowledge that lapointe et al.  <cit>  suggest a distance based approach to combine otherwise incompatible data in a total evidence analysis, although this method has not been tested beyond a single application. the consequence is that it is rarely possible to combine all sources of data in practice and the lack of overlap in combinable data sets may result in a reduction of the number of taxa included. second, miyamoto and fitch  <cit>  contend that combining data sets is rarely justified because partitions of phylogenetic data are real and unequivocal. they argue that several partitions producing similar topologies provide multiple lines of independent evidence supporting that topology.\n",
            "\n",
            "theoretical arguments over the benefits of total evidence will undoubtedly continue, but perhaps the major barriers to its use are the often very high computational demands of large matrices, and the a priori exclusion of certain data types. this is particularly true of charadriiformes phylogeny, where one of the most significant contributions to the field – dna-dna hybridisation – cannot be included. an alternative set of techniques, collectively termed supertrees , enables combination of trees  from otherwise incompatible sources. mrp methods code source phylogenies based on the presence and absence of taxa at each node of the tree  <cit>  and are thus one step removed from the primary data. it is important to recognise that supertrees should not be regarded as a replacement for exhaustive phylogenetic studies of the primary data and there are drawbacks to the methods . however, they do enable very large phylogenies to be constructed rapidly  <cit> . supertrees have been constructed successfully for a wide variety of taxa including carnivores  <cit> , primates  <cit> , seabirds  <cit> , dinosaurs  <cit> , and grasses  <cit> .\n",
            "\n",
            "shorebirds are particularly well suited for supertree treatment, since there are numerous incomplete phylogenies available and a broader phylogeny is desirable to facilitate powerful analyses of numerous evolutionary hypotheses . here, we present the first complete composite phylogeny of extant and recently extinct  <cit>  shorebirds using the mrp approach. we are therefore combining data on tree topologies, and not conducting a simultaneous analysis on the original data. we also use fossil and molecular data to estimate divergence times . the combination of complete taxonomic coverage and the inclusion of branch lengths provide the basis for future comparative analyses of charadriiformes evolution. in addition, conflicting and unresolved areas of charadriiformes phylogeny are revealed.\n",
            "\n",
            "RESULTS\n",
            "supertree resolution and topology\n",
            "we found  <dig> equally short trees of length  <dig> steps using the parsimony ratchet approach . this compares favourably to a standard heuristic search that yielded shortest trees of  <dig> steps. all subsequent results and discussion refer to the parsimony ratchet analyses. figure  <dig> shows the family and subfamily level relationships of shorebirds based on the strict and  <dig> % majority-rule consensus tree . figures  <dig> , <dig> , <dig> , <dig> show the species level phylogeny. the full 50% majority rule consensus and the strict consensus trees are available as additional file  <dig> and  <dig> respectively. the 50% majority-rule consensus tree is well resolved , although the strict consensus tree is only  <dig> % resolved . the majority rule tree includes nine novel clades  that do not appear in any of the source trees; all of these occur towards the tips of the tree. this is a general problem in supertree construction and such clades should be collapsed as they have no support  <cit> . to demonstrate where the mrp method has performed badly we have included the novel clades in all figures and list details in the figure legends. in addition,  <dig> nodes are supported by only one character . each of these nodes is left over from a single source tree. assessing the support for such nodes is problematic because this may simply reflect a lack of research directed at the taxa in question. a major challenge for supertree construction is to develop measures of support that reflect the robustness of nodes in the source trees. we list the number of characters supporting each node  but stress that these are not measures of tree robustness and may not be directly comparable even within the same tree. this is because the taxon coverage across source trees is highly variable so some nodes have more potential support than others. furthermore, because measures of support used in the source trees differ between studies , it is impractical and of dubious value to use these measures to assess the robustness of the supertree.\n",
            "\n",
            "the majority of unresolved nodes in the shorebird supertree are located towards the tips of the phylogeny. for example, the genus gallinago forms a monophyletic clade but only two pairs of species are resolved from  <dig> species  in the majority-rule tree. only the latter relationship remains in the strict consensus tree. in addition, clades including the genera charadrius and vanellus, calidris and tringa, sterna, and scolopax are poorly resolved. this may reflect a bias in phylogenetic studies of shorebirds. for instance, we found six source trees for alcinae  <cit>  but none devoted to scolopax or gallinago. thomas et al.  <cit>  indicate that this may be a problem for shorebird studies in general and reported a strong skew favouring research on northern hemisphere species.\n",
            "\n",
            "in contrast to the within genera relationships, the generic and family levels are generally well resolved. the supertree indicates three monophyletic charadriiformes lineages . family and subfamily resolution within each lineage is high, however the relative position of each group is unresolved in the strict consensus tree. this is an important point because the deepest relationships of shorebird phylogeny are contentious  <cit> . the 50% majority-rule consensus tree indicates that the gulls and allies  are sister to the sandpipers and allies . the most basal lineage includes the plovers and allies . the gulls and allies clade is most consistent with dna-dna hybridisation  <cit> , indicating that larini are sister to sternini and that rynchopini are sister to this group. this conflicts with morphology-based topologies where stercorariini are sister to larini and sternini with rynchopini basal to both. indeed, the position of stercorariini remains controversial and most recently they were placed as sister to alcinae  <cit> . in contrast, morphological evidence  <cit>  places alcinae at the base of the whole charadriiformes tree with stercorariini sister to larini. thus, the position of alcinae is uncertain and appears to be dependent on the type of data, with fundamental differences between molecular based analyses and morphological analyses. the taxon sampling of previous morphological and molecular studies varies considerably and it may be this, rather than genuine differences in the phylogenetic signal of different data types, that is the cause of conflict in resolving the phylogenetic position of alcinae. however, it is encouraging that van tuinen et al.  <cit>  suggested that new unpublished osteological data are consistent with the more derived position indicated by molecular data. the supertree resolves glareolidae outside the larini, sternini, rynchopini, stercorariini, dromas, alcinae clade. this is also the case with recent molecular and previous dna-dna hybridisation studies. morphological studies have failed to resolve the position of glareolidae, placing the family in a large polytomy with all other major groups except alcinae and the sandpipers and allies . a novel development in shorebird phylogeny is the placement of the black-rumped buttonquail turnix hottentotta as a sister to the gulls and allies  based on the nuclear rag- <dig> gene  <cit> . we did not include this species in the supertree because to date paton et al.  <cit>  remains the only study to reveal an apparently robust relationship. more diverse sampling of the buttonquails  is essential to corroborate the general affinities of this family.\n",
            "\n",
            "the relationships within the plover clade appear to be reasonably stable. morphological, molecular, and dna-dna hybridisation all place charadriinae as sister to haematopodini and recurvirostrini; our supertree is consistent with these relationships. however, it is not clear whether burhinidae and chionidae are sister to each other  <cit>  or whether chionidae are sister to a charadriinae, haematopodini, recurvirostrini, and burhinidae clade  <cit> . our supertree also included pluvianellidae, a family consisting of only one species  and places this as sister to chionidae. if pluvianellidae are excluded, the supertree is consistent with the sister group relationship of burhinidae and chionidae.\n",
            "\n",
            "the sister group relationship of jacanidae to rostratulidae is well established  <cit>  and is found in our supertree. the supertree resolves the thinocoridae and pedionomidae as sister taxa and this group is sister to the jacanidae and rostratulidae. the large scolopacidae clade is at the base of the sandpiper clade consistent with recent molecular studies  <cit>  and the dna-dna hybridisation tapestry  <cit> .\n",
            "\n",
            "taken together, it is evident that the supertree is generally more consistent with molecular data  than with analyses based on morphology. however, it is of course possible that this reflects the greater number of molecular source trees available rather than indicating that molecular data is actually better at resolving shorebird phylogeny. we included several large morphological phylogenies  but the majority of source trees  were based on molecular evidence .\n",
            "\n",
            "node dates\n",
            "the higher resolution of the majority-rule tree means it is more likely to be of use in comparative studies. we therefore estimated node ages for this topology only . we stress that our estimates of node dates are a first attempt at dating the whole tree and have several limitations. first, the fossils used to calibrate seven nodes in the tree are unlikely to be the earliest members of their respective families thus these dates will be underestimates. second, we assumed that the fossils are grouped with the extant members of the family but this requires formal testing in a phylogenetic framework. third, the pure birth model assumes that no extinction occurs but this may be unrealistic and it is likely that extinction processes have reduced the representation of older lineages  <cit> . furthermore, this model is derived from the topological structure of the tree so errors in tree reconstruction will likely lead to errors in branch length estimation. however, this approach has been employed previously in supertrees of primates  <cit>  and carnivores  <cit>  explicitly to facilitate comparative analyses. despite these caveats, simulation studies have demonstrated that comparative methods such as independent contrasts are robust to errors in branch length  <cit>  and no viable alternative for dating supertrees has been proposed. nonetheless, we urge that alternative branch length assumptions are explored if the shorebird supertree is used in future comparative studies. at present, the calibrated rag- <dig> tree of paton et al.  <cit>  remains arguably the most thorough and reliable measure of divergence times for charadriiformes.\n",
            "\n",
            "a fuller understanding of the phylogenetic affinities of fossil shorebirds will probably improve estimates of node ages for the group. for example, the extinct form graculavidae, is represented by fossils from the maastrichtian of new jersey  <cit>  and cretaceous of wyoming  <cit>  but its position within the shorebird clade is unclear. feduccia  <cit>  suggests that it may be basal and a formal corroboration of this would support proposals for a late cretaceous origin of shorebirds. the difficulties in dating the shorebird tree are further illustrated by fossil representatives of recurvirostrini and burhinidae which are much older than current estimates suggests. the earliest record of the recurvirostrini is estimated to be over  <dig> million years old  <cit>  whilst recent discoveries of a possible member of the burhinidae are dated to around  <dig> mya  <cit> . there is clearly a need for an integrated phylogenetic study including both extinct and extant shorebirds.\n",
            "\n",
            "supertree bias\n",
            "supertrees are still at an early stage of development and many aspects of mrp, and supertree methods in general, are not yet clearly understood. steps can be taken to ensure that the supertree includes the most appropriate sets of sources trees, such as only using trees from explicitly phylogenetic studies. this is not always straightforward and could result in the exclusion of important information. for instance, in our shorebird supertree, we included sibley and ahlquist's dna-dna hybridisation tapestry  <cit>  although this is based on distance measures rather than more rigorous phylogenetic methods. even if very strict tree selection criteria are applied, there are still likely to be biases in the data set. for example, not all source trees are equally well supported, yet in most supertree analyses each tree is treated equally  <cit> . this is a problem for supertree construction because whilst it is theoretically possible, and indeed beneficial, to weight source trees based on support values  <cit>  it is rarely possible in practice. many source trees do not have support values and those that do may use different methods,  which cannot be directly compared with each other. an additional problem that has not been fully resolved relates to correlations between source trees  <cit> . several source trees based on the same data set may unduly increase the influence of that data set on the supertree analysis. however, there is no formal way of determining how much overlap to allow and the choice of source trees that go into supertree construction inevitably involves some degree of subjective reasoning. for the shorebird supertree we used strict reduced cladistic consensus trees to summarise potential source trees that were from the same data set but based on different methods. for example, thomas et al.  <cit>  based their phylogeny on cytochrome-b but used a range of methods including parsimony and bayesian analyses. we therefore combined these trees to minimise bias. in contrast, ericson et al.  <cit>  used two types of data: sequences from the nuclear rag  <dig> gene and sequences from the myoglobin intron ii. they carried out three analyses: each gene separately and then the two combined in a single analysis. in this case, we used three source trees. it could be argued that the combined analysis of ericson et al.  <cit>  should be excluded because of the possible overlap with the individual analyses. however, under the principle of total evidence, the combined data set may result in novel relationships being revealed  <cit>  and therefore could contribute important information to the supertree. simulation and empirical studies are required to fully understand these and other possible biases in supertree construction  and formal protocols for the selection of source trees are desirable. for transparency, we include a summary of the source trees used, data type, and the main taxa included in the study .\n",
            "\n",
            "our shorebird supertree is highly consistent with recent advances in the molecular phylogenetics charadriiformes. however, we urge caution when using the tree in comparative analyses and encourage the additional use of alternative phylogenies and branch length assumptions. it is particularly important to note that the position of some groups such as the alcinae remains controversial and that although the majority rule tree is consistent with recent molecular studies, the strict consensus tree fails to resolve the deepest nodes.\n",
            "\n",
            "CONCLUSIONS\n",
            "the supertree presented here is, to our knowledge, the first attempt to reconstruct the phylogeny of the entire order charadriiformes. overall, the supertree is highly consistent with recent molecular hypotheses of shorebird phylogeny. however, it is apparent that fresh attempts to resolve both the phylogeny and estimates of age will be dependent on further gene sequencing and new fossil discoveries. the affinities of the alcinae and the relationships between the three major shorebird clades require further corroboration, and studies of several genera such as gallinago and vanellus are desirable. furthermore, additional work is required to establish the true affinities of the turnicidae. nonetheless, it appears that shorebird phylogeny is gradually approaching a consensus view. the broad taxonomic scope and consistency of the supertree mean that is of potentially great value to future comparative studies  of the behaviour, life-history, ecology and conservation of this diverse group.\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "Generated summary:\n",
            "the shorebirds and allies  present an exceptional group for studying numerous evolutionary hypotheses.\n",
            "our supertree combines complete taxonomic coverage and the inclusion of branch lengths provide the basis for future comparative analyses of charadriiformes phylogeny.\n",
            "our supertree is highly consistent with recent advances in molecular phylogenetics charadriiformes.\n",
            "our results suggest that the shorebirds and allies  are an exceptional group for studying numerous evolutionary hypotheses.\n",
            "our supertree combines complete taxonomic coverage and the inclusion of branch lengths provide the basis for future comparative analyses of charadriiformes phylogeny.\n",
            "our supertree resolves glareolidae outside the larini, sternini, stercorariini, stercorariini, and rynchopini clades.\n",
            "our supertree resolves glareolidae outside the larini, sternini, stercorariini, stercorariini, rynchopini, and rynchopini clades.\n",
            "our supertree resolves glareolidae outside the larini, sternini, stercorariini, stercorariini, rynchopini, and rynchopini clades.\n",
            "\n"
          ]
        }
      ],
      "source": [
        "gen_summary = []\n",
        "c = 0\n",
        "for i in test_text:\n",
        "    c+=1\n",
        "    print(c)\n",
        "    gen_summary.append(generate_summary(i))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 40,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "[{'rouge1': Score(precision=0.3346938775510204, recall=0.47953216374269003, fmeasure=0.3942307692307692), 'rouge2': Score(precision=0.0942622950819672, recall=0.13529411764705881, fmeasure=0.1111111111111111), 'rougeL': Score(precision=0.20816326530612245, recall=0.2982456140350877, fmeasure=0.24519230769230768)}, {'rouge1': Score(precision=0.29213483146067415, recall=0.12440191387559808, fmeasure=0.174496644295302), 'rouge2': Score(precision=0.03409090909090909, recall=0.014423076923076924, fmeasure=0.020270270270270268), 'rougeL': Score(precision=0.25842696629213485, recall=0.11004784688995216, fmeasure=0.15436241610738255)}, {'rouge1': Score(precision=0.1566265060240964, recall=0.34210526315789475, fmeasure=0.21487603305785125), 'rouge2': Score(precision=0.028225806451612902, recall=0.061946902654867256, fmeasure=0.038781163434903045), 'rougeL': Score(precision=0.10843373493975904, recall=0.23684210526315788, fmeasure=0.1487603305785124)}, {'rouge1': Score(precision=0.1797752808988764, recall=0.3368421052631579, fmeasure=0.23443223443223443), 'rouge2': Score(precision=0.022598870056497175, recall=0.0425531914893617, fmeasure=0.029520295202952032), 'rougeL': Score(precision=0.10112359550561797, recall=0.18947368421052632, fmeasure=0.13186813186813187)}, {'rouge1': Score(precision=0.378698224852071, recall=0.5565217391304348, fmeasure=0.45070422535211263), 'rouge2': Score(precision=0.10714285714285714, recall=0.15789473684210525, fmeasure=0.1276595744680851), 'rougeL': Score(precision=0.20710059171597633, recall=0.30434782608695654, fmeasure=0.24647887323943665)}, {'rouge1': Score(precision=0.14136125654450263, recall=0.46153846153846156, fmeasure=0.21643286573146295), 'rouge2': Score(precision=0.06036745406824147, recall=0.19827586206896552, fmeasure=0.0925553319919517), 'rougeL': Score(precision=0.11780104712041885, recall=0.38461538461538464, fmeasure=0.18036072144288576)}, {'rouge1': Score(precision=0.3157894736842105, recall=0.3157894736842105, fmeasure=0.3157894736842105), 'rouge2': Score(precision=0.10596026490066225, recall=0.10596026490066225, fmeasure=0.10596026490066225), 'rougeL': Score(precision=0.23026315789473684, recall=0.23026315789473684, fmeasure=0.23026315789473684)}, {'rouge1': Score(precision=0.4125874125874126, recall=0.31216931216931215, fmeasure=0.35542168674698793), 'rouge2': Score(precision=0.13380281690140844, recall=0.10106382978723404, fmeasure=0.11515151515151514), 'rougeL': Score(precision=0.23076923076923078, recall=0.1746031746031746, fmeasure=0.19879518072289157)}, {'rouge1': Score(precision=0.3697478991596639, recall=0.5207100591715976, fmeasure=0.43243243243243246), 'rouge2': Score(precision=0.10548523206751055, recall=0.1488095238095238, fmeasure=0.12345679012345677), 'rougeL': Score(precision=0.20168067226890757, recall=0.28402366863905326, fmeasure=0.23587223587223588)}, {'rouge1': Score(precision=0.2029520295202952, recall=0.3216374269005848, fmeasure=0.24886877828054296), 'rouge2': Score(precision=0.022222222222222223, recall=0.03529411764705882, fmeasure=0.02727272727272727), 'rougeL': Score(precision=0.12177121771217712, recall=0.19298245614035087, fmeasure=0.1493212669683258)}, {'rouge1': Score(precision=0.05442176870748299, recall=0.07511737089201878, fmeasure=0.0631163708086785), 'rouge2': Score(precision=0.013651877133105802, recall=0.018867924528301886, fmeasure=0.015841584158415842), 'rougeL': Score(precision=0.04421768707482993, recall=0.06103286384976526, fmeasure=0.05128205128205128)}, {'rouge1': Score(precision=0.22349570200573066, recall=0.39, fmeasure=0.28415300546448086), 'rouge2': Score(precision=0.031609195402298854, recall=0.05527638190954774, fmeasure=0.04021937842778794), 'rougeL': Score(precision=0.1518624641833811, recall=0.265, fmeasure=0.1930783242258652)}, {'rouge1': Score(precision=0.373134328358209, recall=0.47770700636942676, fmeasure=0.4189944134078212), 'rouge2': Score(precision=0.065, recall=0.08333333333333333, fmeasure=0.07303370786516855), 'rougeL': Score(precision=0.21890547263681592, recall=0.2802547770700637, fmeasure=0.24581005586592178)}, {'rouge1': Score(precision=0.32592592592592595, recall=0.3963963963963964, fmeasure=0.3577235772357724), 'rouge2': Score(precision=0.04477611940298507, recall=0.05454545454545454, fmeasure=0.04918032786885245), 'rougeL': Score(precision=0.1925925925925926, recall=0.23423423423423423, fmeasure=0.21138211382113822)}, {'rouge1': Score(precision=0.2673992673992674, recall=0.6952380952380952, fmeasure=0.3862433862433863), 'rouge2': Score(precision=0.10661764705882353, recall=0.27884615384615385, fmeasure=0.15425531914893614), 'rougeL': Score(precision=0.1391941391941392, recall=0.3619047619047619, fmeasure=0.20105820105820107)}, {'rouge1': Score(precision=0.3140794223826715, recall=0.41232227488151657, fmeasure=0.35655737704918034), 'rouge2': Score(precision=0.07246376811594203, recall=0.09523809523809523, fmeasure=0.08230452674897121), 'rougeL': Score(precision=0.17328519855595667, recall=0.22748815165876776, fmeasure=0.19672131147540983)}, {'rouge1': Score(precision=0.2024793388429752, recall=0.3356164383561644, fmeasure=0.2525773195876289), 'rouge2': Score(precision=0.012448132780082987, recall=0.020689655172413793, fmeasure=0.015544041450777202), 'rougeL': Score(precision=0.1322314049586777, recall=0.2191780821917808, fmeasure=0.16494845360824742)}, {'rouge1': Score(precision=0.26426426426426425, recall=0.567741935483871, fmeasure=0.36065573770491804), 'rouge2': Score(precision=0.09337349397590361, recall=0.2012987012987013, fmeasure=0.12757201646090535), 'rougeL': Score(precision=0.16516516516516516, recall=0.3548387096774194, fmeasure=0.22540983606557374)}, {'rouge1': Score(precision=0.42857142857142855, recall=0.2765957446808511, fmeasure=0.33620689655172414), 'rouge2': Score(precision=0.16666666666666666, recall=0.10714285714285714, fmeasure=0.13043478260869565), 'rougeL': Score(precision=0.34065934065934067, recall=0.2198581560283688, fmeasure=0.26724137931034486)}, {'rouge1': Score(precision=0.29957805907172996, recall=0.39664804469273746, fmeasure=0.34134615384615385), 'rouge2': Score(precision=0.046610169491525424, recall=0.06179775280898876, fmeasure=0.05314009661835749), 'rougeL': Score(precision=0.16455696202531644, recall=0.21787709497206703, fmeasure=0.18749999999999997)}, {'rouge1': Score(precision=0.35384615384615387, recall=0.7278481012658228, fmeasure=0.4761904761904762), 'rouge2': Score(precision=0.1574074074074074, recall=0.3248407643312102, fmeasure=0.21205821205821207), 'rougeL': Score(precision=0.2, recall=0.41139240506329117, fmeasure=0.2691511387163561)}, {'rouge1': Score(precision=0.3627450980392157, recall=0.5103448275862069, fmeasure=0.4240687679083095), 'rouge2': Score(precision=0.10837438423645321, recall=0.1527777777777778, fmeasure=0.1268011527377522), 'rougeL': Score(precision=0.15196078431372548, recall=0.21379310344827587, fmeasure=0.1776504297994269)}, {'rouge1': Score(precision=0.10622710622710622, recall=0.25892857142857145, fmeasure=0.15064935064935064), 'rouge2': Score(precision=0.07352941176470588, recall=0.18018018018018017, fmeasure=0.10443864229765012), 'rougeL': Score(precision=0.07326007326007326, recall=0.17857142857142858, fmeasure=0.1038961038961039)}, {'rouge1': Score(precision=0.3055555555555556, recall=0.567741935483871, fmeasure=0.3972911963882618), 'rouge2': Score(precision=0.11846689895470383, recall=0.22077922077922077, fmeasure=0.15419501133786848), 'rougeL': Score(precision=0.19791666666666666, recall=0.36774193548387096, fmeasure=0.25733634311512416)}, {'rouge1': Score(precision=0.2542372881355932, recall=0.28846153846153844, fmeasure=0.27027027027027023), 'rouge2': Score(precision=0.07692307692307693, recall=0.08737864077669903, fmeasure=0.08181818181818182), 'rougeL': Score(precision=0.22033898305084745, recall=0.25, fmeasure=0.23423423423423426)}, {'rouge1': Score(precision=0.2660098522167488, recall=0.30857142857142855, fmeasure=0.2857142857142857), 'rouge2': Score(precision=0.06930693069306931, recall=0.08045977011494253, fmeasure=0.07446808510638298), 'rougeL': Score(precision=0.1921182266009852, recall=0.22285714285714286, fmeasure=0.20634920634920637)}, {'rouge1': Score(precision=0.28125, recall=0.5, fmeasure=0.36), 'rouge2': Score(precision=0.07450980392156863, recall=0.13286713286713286, fmeasure=0.09547738693467336), 'rougeL': Score(precision=0.17578125, recall=0.3125, fmeasure=0.225)}, {'rouge1': Score(precision=0.45454545454545453, recall=0.3125, fmeasure=0.3703703703703703), 'rouge2': Score(precision=0.045871559633027525, recall=0.031446540880503145, fmeasure=0.0373134328358209), 'rougeL': Score(precision=0.22727272727272727, recall=0.15625, fmeasure=0.18518518518518515)}, {'rouge1': Score(precision=0.25249169435215946, recall=0.4470588235294118, fmeasure=0.3227176220806794), 'rouge2': Score(precision=0.09666666666666666, recall=0.17159763313609466, fmeasure=0.12366737739872066), 'rougeL': Score(precision=0.15282392026578073, recall=0.27058823529411763, fmeasure=0.19532908704883228)}, {'rouge1': Score(precision=0.3507246376811594, recall=0.605, fmeasure=0.4440366972477064), 'rouge2': Score(precision=0.10755813953488372, recall=0.18592964824120603, fmeasure=0.13627992633517494), 'rougeL': Score(precision=0.22318840579710145, recall=0.385, fmeasure=0.28256880733944956)}, {'rouge1': Score(precision=0.28512396694214875, recall=0.43125, fmeasure=0.3432835820895523), 'rouge2': Score(precision=0.05394190871369295, recall=0.08176100628930817, fmeasure=0.06499999999999999), 'rougeL': Score(precision=0.16942148760330578, recall=0.25625, fmeasure=0.20398009950248755)}, {'rouge1': Score(precision=0.3502824858757062, recall=0.5, fmeasure=0.4119601328903654), 'rouge2': Score(precision=0.08522727272727272, recall=0.12195121951219512, fmeasure=0.10033444816053512), 'rougeL': Score(precision=0.14689265536723164, recall=0.20967741935483872, fmeasure=0.1727574750830565)}, {'rouge1': Score(precision=0.19282511210762332, recall=0.3467741935483871, fmeasure=0.2478386167146974), 'rouge2': Score(precision=0.04504504504504504, recall=0.08130081300813008, fmeasure=0.057971014492753624), 'rougeL': Score(precision=0.09417040358744394, recall=0.1693548387096774, fmeasure=0.12103746397694522)}, {'rouge1': Score(precision=0.3474178403755869, recall=0.43023255813953487, fmeasure=0.38441558441558443), 'rouge2': Score(precision=0.08018867924528301, recall=0.09941520467836257, fmeasure=0.08877284595300262), 'rougeL': Score(precision=0.15492957746478872, recall=0.19186046511627908, fmeasure=0.17142857142857143)}, {'rouge1': Score(precision=0.36363636363636365, recall=0.21505376344086022, fmeasure=0.27027027027027023), 'rouge2': Score(precision=0.07339449541284404, recall=0.043243243243243246, fmeasure=0.05442176870748299), 'rougeL': Score(precision=0.2545454545454545, recall=0.15053763440860216, fmeasure=0.1891891891891892)}, {'rouge1': Score(precision=0.26071428571428573, recall=0.4397590361445783, fmeasure=0.3273542600896861), 'rouge2': Score(precision=0.14336917562724014, recall=0.24242424242424243, fmeasure=0.18018018018018017), 'rougeL': Score(precision=0.17857142857142858, recall=0.30120481927710846, fmeasure=0.22421524663677134)}, {'rouge1': Score(precision=0.24166666666666667, recall=0.40559440559440557, fmeasure=0.3028720626631854), 'rouge2': Score(precision=0.058577405857740586, recall=0.09859154929577464, fmeasure=0.07349081364829396), 'rougeL': Score(precision=0.17083333333333334, recall=0.2867132867132867, fmeasure=0.21409921671018278)}, {'rouge1': Score(precision=0.3045977011494253, recall=0.6973684210526315, fmeasure=0.42400000000000004), 'rouge2': Score(precision=0.138328530259366, recall=0.31788079470198677, fmeasure=0.1927710843373494), 'rougeL': Score(precision=0.21264367816091953, recall=0.4868421052631579, fmeasure=0.29600000000000004)}, {'rouge1': Score(precision=0.41964285714285715, recall=0.8173913043478261, fmeasure=0.5545722713864307), 'rouge2': Score(precision=0.22869955156950672, recall=0.4473684210526316, fmeasure=0.30267062314540055), 'rougeL': Score(precision=0.21875, recall=0.4260869565217391, fmeasure=0.2890855457227139)}, {'rouge1': Score(precision=0.15789473684210525, recall=0.11170212765957446, fmeasure=0.1308411214953271), 'rouge2': Score(precision=0.015151515151515152, recall=0.0106951871657754, fmeasure=0.012539184952978056), 'rougeL': Score(precision=0.11278195488721804, recall=0.0797872340425532, fmeasure=0.09345794392523366)}, {'rouge1': Score(precision=0.424, recall=0.2896174863387978, fmeasure=0.3441558441558441), 'rouge2': Score(precision=0.1532258064516129, recall=0.1043956043956044, fmeasure=0.12418300653594773), 'rougeL': Score(precision=0.352, recall=0.24043715846994534, fmeasure=0.28571428571428575)}, {'rouge1': Score(precision=0.23461538461538461, recall=0.21328671328671328, fmeasure=0.22344322344322343), 'rouge2': Score(precision=0.03088803088803089, recall=0.028070175438596492, fmeasure=0.029411764705882353), 'rougeL': Score(precision=0.10384615384615385, recall=0.0944055944055944, fmeasure=0.09890109890109891)}, {'rouge1': Score(precision=0.25396825396825395, recall=0.6015037593984962, fmeasure=0.35714285714285715), 'rouge2': Score(precision=0.06050955414012739, recall=0.14393939393939395, fmeasure=0.08520179372197309), 'rougeL': Score(precision=0.1365079365079365, recall=0.3233082706766917, fmeasure=0.1919642857142857)}, {'rouge1': Score(precision=0.23668639053254437, recall=0.35714285714285715, fmeasure=0.2846975088967972), 'rouge2': Score(precision=0.06547619047619048, recall=0.0990990990990991, fmeasure=0.07885304659498209), 'rougeL': Score(precision=0.17751479289940827, recall=0.26785714285714285, fmeasure=0.21352313167259787)}, {'rouge1': Score(precision=0.20444444444444446, recall=0.24210526315789474, fmeasure=0.22168674698795182), 'rouge2': Score(precision=0.05357142857142857, recall=0.06349206349206349, fmeasure=0.05811138014527844), 'rougeL': Score(precision=0.17333333333333334, recall=0.20526315789473684, fmeasure=0.1879518072289157)}, {'rouge1': Score(precision=0.3768844221105528, recall=0.375, fmeasure=0.3759398496240602), 'rouge2': Score(precision=0.09090909090909091, recall=0.09045226130653267, fmeasure=0.09068010075566751), 'rougeL': Score(precision=0.24120603015075376, recall=0.24, fmeasure=0.24060150375939848)}, {'rouge1': Score(precision=0.27074235807860264, recall=0.49206349206349204, fmeasure=0.3492957746478873), 'rouge2': Score(precision=0.03508771929824561, recall=0.064, fmeasure=0.04532577903682719), 'rougeL': Score(precision=0.1572052401746725, recall=0.2857142857142857, fmeasure=0.2028169014084507)}, {'rouge1': Score(precision=0.3161290322580645, recall=0.6282051282051282, fmeasure=0.42060085836909866), 'rouge2': Score(precision=0.10032362459546926, recall=0.2, fmeasure=0.13362068965517243), 'rougeL': Score(precision=0.19032258064516128, recall=0.3782051282051282, fmeasure=0.25321888412017163)}, {'rouge1': Score(precision=0.3953488372093023, recall=0.6938775510204082, fmeasure=0.5037037037037037), 'rouge2': Score(precision=0.13450292397660818, recall=0.23711340206185566, fmeasure=0.1716417910447761), 'rougeL': Score(precision=0.19186046511627908, recall=0.336734693877551, fmeasure=0.24444444444444444)}, {'rouge1': Score(precision=0.2379182156133829, recall=0.6153846153846154, fmeasure=0.34316353887399464), 'rouge2': Score(precision=0.09328358208955224, recall=0.24271844660194175, fmeasure=0.13477088948787064), 'rougeL': Score(precision=0.13382899628252787, recall=0.34615384615384615, fmeasure=0.19302949061662197)}, {'rouge1': Score(precision=0.3271889400921659, recall=0.6228070175438597, fmeasure=0.4290030211480363), 'rouge2': Score(precision=0.08796296296296297, recall=0.168141592920354, fmeasure=0.11550151975683892), 'rougeL': Score(precision=0.17972350230414746, recall=0.34210526315789475, fmeasure=0.2356495468277946)}, {'rouge1': Score(precision=0.1796875, recall=0.4380952380952381, fmeasure=0.2548476454293629), 'rouge2': Score(precision=0.058823529411764705, recall=0.14423076923076922, fmeasure=0.08356545961002784), 'rougeL': Score(precision=0.109375, recall=0.26666666666666666, fmeasure=0.1551246537396122)}, {'rouge1': Score(precision=0.24761904761904763, recall=0.30409356725146197, fmeasure=0.27296587926509186), 'rouge2': Score(precision=0.0861244019138756, recall=0.10588235294117647, fmeasure=0.09498680738786279), 'rougeL': Score(precision=0.16666666666666666, recall=0.2046783625730994, fmeasure=0.1837270341207349)}, {'rouge1': Score(precision=0.3388704318936877, recall=0.5895953757225434, fmeasure=0.430379746835443), 'rouge2': Score(precision=0.12, recall=0.20930232558139536, fmeasure=0.15254237288135591), 'rougeL': Score(precision=0.2059800664451827, recall=0.3583815028901734, fmeasure=0.2616033755274262)}, {'rouge1': Score(precision=0.43478260869565216, recall=0.44692737430167595, fmeasure=0.44077134986225897), 'rouge2': Score(precision=0.09289617486338798, recall=0.09550561797752809, fmeasure=0.09418282548476455), 'rougeL': Score(precision=0.2608695652173913, recall=0.2681564245810056, fmeasure=0.2644628099173554)}, {'rouge1': Score(precision=0.20903954802259886, recall=0.23417721518987342, fmeasure=0.2208955223880597), 'rouge2': Score(precision=0.017045454545454544, recall=0.01910828025477707, fmeasure=0.018018018018018018), 'rougeL': Score(precision=0.13559322033898305, recall=0.1518987341772152, fmeasure=0.14328358208955225)}, {'rouge1': Score(precision=0.25301204819277107, recall=0.5217391304347826, fmeasure=0.3407707910750507), 'rouge2': Score(precision=0.06646525679758308, recall=0.1375, fmeasure=0.08961303462321793), 'rougeL': Score(precision=0.14457831325301204, recall=0.2981366459627329, fmeasure=0.1947261663286004)}, {'rouge1': Score(precision=0.3343949044585987, recall=0.5276381909547738, fmeasure=0.4093567251461988), 'rouge2': Score(precision=0.06070287539936102, recall=0.09595959595959595, fmeasure=0.07436399217221135), 'rougeL': Score(precision=0.22611464968152867, recall=0.35678391959798994, fmeasure=0.2768031189083821)}, {'rouge1': Score(precision=0.22727272727272727, recall=0.17341040462427745, fmeasure=0.19672131147540983), 'rouge2': Score(precision=0.030534351145038167, recall=0.023255813953488372, fmeasure=0.026402640264026403), 'rougeL': Score(precision=0.18181818181818182, recall=0.13872832369942195, fmeasure=0.15737704918032785)}, {'rouge1': Score(precision=0.24081632653061225, recall=0.3881578947368421, fmeasure=0.29722921914357686), 'rouge2': Score(precision=0.07377049180327869, recall=0.11920529801324503, fmeasure=0.0911392405063291), 'rougeL': Score(precision=0.13877551020408163, recall=0.2236842105263158, fmeasure=0.17128463476070527)}, {'rouge1': Score(precision=0.37433155080213903, recall=0.4430379746835443, fmeasure=0.40579710144927533), 'rouge2': Score(precision=0.0967741935483871, recall=0.11464968152866242, fmeasure=0.10495626822157435), 'rougeL': Score(precision=0.25668449197860965, recall=0.3037974683544304, fmeasure=0.2782608695652174)}, {'rouge1': Score(precision=0.5131578947368421, recall=0.35135135135135137, fmeasure=0.41711229946524064), 'rouge2': Score(precision=0.14666666666666667, recall=0.1, fmeasure=0.11891891891891893), 'rougeL': Score(precision=0.32894736842105265, recall=0.22522522522522523, fmeasure=0.26737967914438504)}, {'rouge1': Score(precision=0.34172661870503596, recall=0.7089552238805971, fmeasure=0.4611650485436893), 'rouge2': Score(precision=0.11191335740072202, recall=0.23308270676691728, fmeasure=0.15121951219512195), 'rougeL': Score(precision=0.2014388489208633, recall=0.417910447761194, fmeasure=0.2718446601941748)}, {'rouge1': Score(precision=0.28013029315960913, recall=0.5477707006369427, fmeasure=0.3706896551724138), 'rouge2': Score(precision=0.12091503267973856, recall=0.23717948717948717, fmeasure=0.16017316017316016), 'rougeL': Score(precision=0.19543973941368079, recall=0.3821656050955414, fmeasure=0.25862068965517243)}, {'rouge1': Score(precision=0.23297491039426524, recall=0.3939393939393939, fmeasure=0.2927927927927928), 'rouge2': Score(precision=0.046762589928057555, recall=0.07926829268292683, fmeasure=0.058823529411764705), 'rougeL': Score(precision=0.13261648745519714, recall=0.22424242424242424, fmeasure=0.16666666666666666)}, {'rouge1': Score(precision=0.4025423728813559, recall=0.48223350253807107, fmeasure=0.4387990762124711), 'rouge2': Score(precision=0.11914893617021277, recall=0.14285714285714285, fmeasure=0.12993039443155452), 'rougeL': Score(precision=0.22033898305084745, recall=0.2639593908629442, fmeasure=0.24018475750577367)}, {'rouge1': Score(precision=0.30434782608695654, recall=0.2631578947368421, fmeasure=0.28225806451612906), 'rouge2': Score(precision=0.03508771929824561, recall=0.030303030303030304, fmeasure=0.03252032520325203), 'rougeL': Score(precision=0.21739130434782608, recall=0.18796992481203006, fmeasure=0.2016129032258064)}, {'rouge1': Score(precision=0.28618421052631576, recall=0.4943181818181818, fmeasure=0.3625), 'rouge2': Score(precision=0.132013201320132, recall=0.22857142857142856, fmeasure=0.16736401673640164), 'rougeL': Score(precision=0.20394736842105263, recall=0.3522727272727273, fmeasure=0.2583333333333333)}, {'rouge1': Score(precision=0.22570532915360503, recall=0.5901639344262295, fmeasure=0.32653061224489793), 'rouge2': Score(precision=0.07547169811320754, recall=0.19834710743801653, fmeasure=0.10933940774487472), 'rougeL': Score(precision=0.10344827586206896, recall=0.27049180327868855, fmeasure=0.14965986394557823)}, {'rouge1': Score(precision=0.17872340425531916, recall=0.39622641509433965, fmeasure=0.2463343108504399), 'rouge2': Score(precision=0.05982905982905983, recall=0.13333333333333333, fmeasure=0.08259587020648967), 'rougeL': Score(precision=0.09787234042553192, recall=0.2169811320754717, fmeasure=0.13489736070381234)}, {'rouge1': Score(precision=0.16774193548387098, recall=0.4262295081967213, fmeasure=0.24074074074074073), 'rouge2': Score(precision=0.03559870550161812, recall=0.09090909090909091, fmeasure=0.051162790697674425), 'rougeL': Score(precision=0.0935483870967742, recall=0.23770491803278687, fmeasure=0.13425925925925924)}, {'rouge1': Score(precision=0.16193181818181818, recall=0.42857142857142855, fmeasure=0.23505154639175257), 'rouge2': Score(precision=0.042735042735042736, recall=0.11363636363636363, fmeasure=0.062111801242236024), 'rougeL': Score(precision=0.11931818181818182, recall=0.3157894736842105, fmeasure=0.17319587628865982)}, {'rouge1': Score(precision=0.28365384615384615, recall=0.4013605442176871, fmeasure=0.3323943661971831), 'rouge2': Score(precision=0.03864734299516908, recall=0.0547945205479452, fmeasure=0.04532577903682719), 'rougeL': Score(precision=0.16826923076923078, recall=0.23809523809523808, fmeasure=0.1971830985915493)}, {'rouge1': Score(precision=0.24444444444444444, recall=0.20657276995305165, fmeasure=0.22391857506361323), 'rouge2': Score(precision=0.05027932960893855, recall=0.04245283018867924, fmeasure=0.04603580562659847), 'rougeL': Score(precision=0.16666666666666666, recall=0.14084507042253522, fmeasure=0.15267175572519084)}, {'rouge1': Score(precision=0.2673611111111111, recall=0.5746268656716418, fmeasure=0.36492890995260663), 'rouge2': Score(precision=0.07317073170731707, recall=0.15789473684210525, fmeasure=0.09999999999999999), 'rougeL': Score(precision=0.14930555555555555, recall=0.3208955223880597, fmeasure=0.2037914691943128)}, {'rouge1': Score(precision=0.40588235294117647, recall=0.5476190476190477, fmeasure=0.46621621621621623), 'rouge2': Score(precision=0.1242603550295858, recall=0.168, fmeasure=0.14285714285714285), 'rougeL': Score(precision=0.19411764705882353, recall=0.2619047619047619, fmeasure=0.22297297297297297)}, {'rouge1': Score(precision=0.37209302325581395, recall=0.48854961832061067, fmeasure=0.42244224422442245), 'rouge2': Score(precision=0.07602339181286549, recall=0.1, fmeasure=0.08637873754152824), 'rougeL': Score(precision=0.18604651162790697, recall=0.24427480916030533, fmeasure=0.21122112211221122)}, {'rouge1': Score(precision=0.3675496688741722, recall=0.6065573770491803, fmeasure=0.4577319587628866), 'rouge2': Score(precision=0.20930232558139536, recall=0.34615384615384615, fmeasure=0.2608695652173913), 'rougeL': Score(precision=0.24834437086092714, recall=0.4098360655737705, fmeasure=0.30927835051546393)}, {'rouge1': Score(precision=0.25, recall=0.3384615384615385, fmeasure=0.28758169934640526), 'rouge2': Score(precision=0.04597701149425287, recall=0.0625, fmeasure=0.052980132450331126), 'rougeL': Score(precision=0.18181818181818182, recall=0.24615384615384617, fmeasure=0.20915032679738566)}, {'rouge1': Score(precision=0.19658119658119658, recall=0.359375, fmeasure=0.2541436464088398), 'rouge2': Score(precision=0.012875536480686695, recall=0.023622047244094488, fmeasure=0.01666666666666667), 'rougeL': Score(precision=0.12393162393162394, recall=0.2265625, fmeasure=0.16022099447513813)}, {'rouge1': Score(precision=0.4166666666666667, recall=0.5555555555555556, fmeasure=0.4761904761904762), 'rouge2': Score(precision=0.15481171548117154, recall=0.20670391061452514, fmeasure=0.17703349282296652), 'rougeL': Score(precision=0.2375, recall=0.31666666666666665, fmeasure=0.27142857142857135)}, {'rouge1': Score(precision=0.3448275862068966, recall=0.5365853658536586, fmeasure=0.41984732824427484), 'rouge2': Score(precision=0.11635220125786164, recall=0.18137254901960784, fmeasure=0.1417624521072797), 'rougeL': Score(precision=0.21003134796238246, recall=0.32682926829268294, fmeasure=0.25572519083969464)}, {'rouge1': Score(precision=0.13736263736263737, recall=0.3333333333333333, fmeasure=0.1945525291828794), 'rouge2': Score(precision=0.01652892561983471, recall=0.040268456375838924, fmeasure=0.0234375), 'rougeL': Score(precision=0.09340659340659341, recall=0.22666666666666666, fmeasure=0.13229571984435798)}, {'rouge1': Score(precision=0.3406113537117904, recall=0.582089552238806, fmeasure=0.4297520661157025), 'rouge2': Score(precision=0.11842105263157894, recall=0.20300751879699247, fmeasure=0.14958448753462603), 'rougeL': Score(precision=0.1703056768558952, recall=0.291044776119403, fmeasure=0.21487603305785125)}, {'rouge1': Score(precision=0.19855595667870035, recall=0.4330708661417323, fmeasure=0.27227722772277224), 'rouge2': Score(precision=0.04710144927536232, recall=0.10317460317460317, fmeasure=0.06467661691542288), 'rougeL': Score(precision=0.1371841155234657, recall=0.2992125984251969, fmeasure=0.18811881188118812)}, {'rouge1': Score(precision=0.2176039119804401, recall=0.7295081967213115, fmeasure=0.3352165725047081), 'rouge2': Score(precision=0.0857843137254902, recall=0.2892561983471074, fmeasure=0.13232514177693763), 'rougeL': Score(precision=0.13691931540342298, recall=0.45901639344262296, fmeasure=0.21092278719397362)}, {'rouge1': Score(precision=0.38620689655172413, recall=0.2916666666666667, fmeasure=0.33234421364985167), 'rouge2': Score(precision=0.125, recall=0.09424083769633508, fmeasure=0.10746268656716419), 'rougeL': Score(precision=0.31724137931034485, recall=0.23958333333333334, fmeasure=0.2729970326409496)}, {'rouge1': Score(precision=0.21120689655172414, recall=0.22580645161290322, fmeasure=0.21826280623608021), 'rouge2': Score(precision=0.004329004329004329, recall=0.004629629629629629, fmeasure=0.0044742729306487695), 'rougeL': Score(precision=0.13793103448275862, recall=0.14746543778801843, fmeasure=0.1425389755011136)}, {'rouge1': Score(precision=0.18597560975609756, recall=0.4295774647887324, fmeasure=0.2595744680851064), 'rouge2': Score(precision=0.03669724770642202, recall=0.0851063829787234, fmeasure=0.05128205128205129), 'rougeL': Score(precision=0.12804878048780488, recall=0.29577464788732394, fmeasure=0.17872340425531916)}, {'rouge1': Score(precision=0.49019607843137253, recall=0.3968253968253968, fmeasure=0.43859649122807015), 'rouge2': Score(precision=0.13861386138613863, recall=0.112, fmeasure=0.12389380530973453), 'rougeL': Score(precision=0.27450980392156865, recall=0.2222222222222222, fmeasure=0.24561403508771928)}, {'rouge1': Score(precision=0.3230769230769231, recall=0.25609756097560976, fmeasure=0.28571428571428575), 'rouge2': Score(precision=0.03875968992248062, recall=0.03067484662576687, fmeasure=0.03424657534246575), 'rougeL': Score(precision=0.2153846153846154, recall=0.17073170731707318, fmeasure=0.19047619047619047)}, {'rouge1': Score(precision=0.25925925925925924, recall=0.5982905982905983, fmeasure=0.36175710594315247), 'rouge2': Score(precision=0.08921933085501858, recall=0.20689655172413793, fmeasure=0.12467532467532468), 'rougeL': Score(precision=0.16296296296296298, recall=0.37606837606837606, fmeasure=0.22739018087855298)}, {'rouge1': Score(precision=0.13135593220338984, recall=0.2012987012987013, fmeasure=0.158974358974359), 'rouge2': Score(precision=0.00425531914893617, recall=0.006535947712418301, fmeasure=0.005154639175257732), 'rougeL': Score(precision=0.09745762711864407, recall=0.14935064935064934, fmeasure=0.11794871794871795)}, {'rouge1': Score(precision=0.30578512396694213, recall=0.5211267605633803, fmeasure=0.38541666666666663), 'rouge2': Score(precision=0.07468879668049792, recall=0.1276595744680851, fmeasure=0.09424083769633508), 'rougeL': Score(precision=0.19834710743801653, recall=0.3380281690140845, fmeasure=0.25)}, {'rouge1': Score(precision=0.25196850393700787, recall=0.5423728813559322, fmeasure=0.34408602150537637), 'rouge2': Score(precision=0.09736842105263158, recall=0.21022727272727273, fmeasure=0.13309352517985612), 'rougeL': Score(precision=0.1679790026246719, recall=0.3615819209039548, fmeasure=0.22939068100358423)}, {'rouge1': Score(precision=0.38738738738738737, recall=0.26875, fmeasure=0.3173431734317343), 'rouge2': Score(precision=0.19090909090909092, recall=0.1320754716981132, fmeasure=0.15613382899628256), 'rougeL': Score(precision=0.3063063063063063, recall=0.2125, fmeasure=0.2509225092250922)}, {'rouge1': Score(precision=0.22624434389140272, recall=0.4, fmeasure=0.28901734104046245), 'rouge2': Score(precision=0.08636363636363636, recall=0.1532258064516129, fmeasure=0.11046511627906977), 'rougeL': Score(precision=0.17194570135746606, recall=0.304, fmeasure=0.21965317919075145)}, {'rouge1': Score(precision=0.30952380952380953, recall=0.625, fmeasure=0.4140127388535032), 'rouge2': Score(precision=0.11961722488038277, recall=0.24271844660194175, fmeasure=0.16025641025641024), 'rougeL': Score(precision=0.1523809523809524, recall=0.3076923076923077, fmeasure=0.20382165605095542)}, {'rouge1': Score(precision=0.19047619047619047, recall=0.19753086419753085, fmeasure=0.1939393939393939), 'rouge2': Score(precision=0.011976047904191617, recall=0.012422360248447204, fmeasure=0.012195121951219513), 'rougeL': Score(precision=0.14285714285714285, recall=0.14814814814814814, fmeasure=0.14545454545454545)}, {'rouge1': Score(precision=0.36363636363636365, recall=0.4094488188976378, fmeasure=0.38518518518518513), 'rouge2': Score(precision=0.056338028169014086, recall=0.06349206349206349, fmeasure=0.059701492537313425), 'rougeL': Score(precision=0.17482517482517482, recall=0.1968503937007874, fmeasure=0.1851851851851852)}]\n"
          ]
        }
      ],
      "source": [
        "from rouge_score import rouge_scorer\n",
        "scores = []\n",
        "# Initialize a Rouge scorer\n",
        "scorer = rouge_scorer.RougeScorer(['rouge1', 'rouge2', 'rougeL'], use_stemmer=True)\n",
        "\n",
        "for i in range(len(gen_summary)):\n",
        "    scores.append(scorer.score(gen_summary[i],val_summary[i]))\n",
        "print(scores)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 42,
      "metadata": {},
      "outputs": [],
      "source": [
        "import numpy as np\n",
        "import pandas as pd\n",
        "\n",
        "# Create lists to hold the data for DataFrame\n",
        "precision1, recall1, fmeasure1 = [], [], []\n",
        "precision2, recall2, fmeasure2 = [], [], []\n",
        "precisionL, recallL, fmeasureL = [], [], []\n",
        "\n",
        "# Populate lists with data\n",
        "for entry in scores:\n",
        "    precision1.append(entry['rouge1'].precision)\n",
        "    recall1.append(entry['rouge1'].recall)\n",
        "    fmeasure1.append(entry['rouge1'].fmeasure)\n",
        "    precision2.append(entry['rouge2'].precision)\n",
        "    recall2.append(entry['rouge2'].recall)\n",
        "    fmeasure2.append(entry['rouge2'].fmeasure)\n",
        "    precisionL.append(entry['rougeL'].precision)\n",
        "    recallL.append(entry['rougeL'].recall)\n",
        "    fmeasureL.append(entry['rougeL'].fmeasure)\n",
        "\n",
        "# Create a DataFrame\n",
        "df = pd.DataFrame({\n",
        "    'ROUGE-1 Precision': precision1,\n",
        "    'ROUGE-1 Recall': recall1,\n",
        "    'ROUGE-1 F-measure': fmeasure1,\n",
        "    'ROUGE-2 Precision': precision2,\n",
        "    'ROUGE-2 Recall': recall2,\n",
        "    'ROUGE-2 F-measure': fmeasure2,\n",
        "    'ROUGE-L Precision': precisionL,\n",
        "    'ROUGE-L Recall': recallL,\n",
        "    'ROUGE-L F-measure': fmeasureL\n",
        "})\n",
        "\n",
        "# Save the DataFrame to an Excel file\n",
        "df.to_excel('rouge_scores.xlsx', index=False)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {},
      "outputs": [],
      "source": [
        "import numpy as np\n",
        "\n",
        "# Initialize dictionaries to store total scores and count\n",
        "# Assume `data` is your list of dictionaries containing ROUGE scores\n",
        "\n",
        "# Initialize total dictionaries\n",
        "rouge1_totals = {'precision': 0, 'recall': 0, 'fmeasure': 0}\n",
        "rouge2_totals = {'precision': 0, 'recall': 0, 'fmeasure': 0}\n",
        "rougeL_totals = {'precision': 0, 'recall': 0, 'fmeasure': 0}\n",
        "\n",
        "# Sum up all scores\n",
        "for entry in scores:\n",
        "    rouge1_totals['precision'] += entry['rouge1'].precision\n",
        "    rouge1_totals['recall'] += entry['rouge1'].recall\n",
        "    rouge1_totals['fmeasure'] += entry['rouge1'].fmeasure\n",
        "    rouge2_totals['precision'] += entry['rouge2'].precision\n",
        "    rouge2_totals['recall'] += entry['rouge2'].recall\n",
        "    rouge2_totals['fmeasure'] += entry['rouge2'].fmeasure\n",
        "    rougeL_totals['precision'] += entry['rougeL'].precision\n",
        "    rougeL_totals['recall'] += entry['rougeL'].recall\n",
        "    rougeL_totals['fmeasure'] += entry['rougeL'].fmeasure\n",
        "\n",
        "# Calculate the number of entries\n",
        "num_entries = len(scores)\n",
        "\n",
        "# Calculate mean scores\n",
        "rouge1_means = {key: value / num_entries for key, value in rouge1_totals.items()}\n",
        "rouge2_means = {key: value / num_entries for key, value in rouge2_totals.items()}\n",
        "rougeL_means = {key: value / num_entries for key, value in rougeL_totals.items()}\n",
        "\n",
        "# Print the results\n",
        "print(\"ROUGE-1 Mean Scores:\", rouge1_means)\n",
        "print(\"ROUGE-2 Mean Scores:\", rouge2_means)\n",
        "print(\"ROUGE-L Mean Scores:\", rougeL_means)\n",
        "\n"
      ]
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "gpuType": "T4",
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.11.5"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}
